<title>mesna-1983-aacvs.html</title>
Vol 20: 
<h2>ARABIDOPSIS THALIANA c-DNA BANK. RESEARCH FOR GENES CODING FOR ENZYMES INVOLVED IN PYRIMIDINE METABOLISM.</h2><p>

J.M. Mesnard and G. Lebeurier<p>

Laboratoire de Virologie, Institut de Biologie Moleculaire et
Cellulaire du C.N.R.S., 15, rue Descartes, F-67084 Strasbourg, France<p>

INTRODUCTION<p>

Werner et al. (1982) attempted to isolate various genes coding for the
biosynthesis of amino acids in plants by transforming different yeast
Saccharomyces cerevisiae mutants affected in arginine or leucine
metabolism with a cosmic gene bank of A. thaliana. We report in this
paper another attempt by cDNA cloning and transforming various E. coli
mutants affected in pyrimidine metabolism. We chose A. thaliana
because its genome is the smallest known among higher plants with 2 x
10 8 kilobase pairs per haploid genome (Bennet and Smith, 1976). We
chose the pathway for the biosynthesis of pyrimidines because, in this
case, the enzymes of an eukaryotic organism, namely the yeast
Saccharomyces cerevisiae, can be expressed in E. coli and complement
the pyrimidine mutants of E. coli by transformation with a DNA bank
(Bach et al., 1979; Loison et al., 1981). To apply this approach to A.
thaliana it is necessary to eliminate the introns which are present in
the genome of higher eukaryotes. Therefore, we synthesized
double-stranded cDNA from mRNAs and cloned the double stranded c DNA
in the *beta*-lactamase gene of pBR322 to obtain an expression of an
active eukrayotic protein due to prokaryotic promoter activity (Chang
et al., 1978).<p>

MATERIALS AND METHODS<p>

Extraction of polyadenylated RNAs<p>

The method employed was that of Dudley and Northcote (1978) modified
in order to obtain mRNA from A. thaliana. The leaves of three week old
plants were found until reduced to a fine powder in liquid nitrogen
with a pestle. Per gram of leaves, 10 ml of extaction buffer (400 mM
LiCl, 5 mM EGTA, 1% SDS, 200 mM Tris-HC1 pH 9.0) and 10 ml of phenol,
containing 5 g of 8-hydroxyquinoline/l, were added. The homogenate was
centrifuged 20 min. at 6 000 rpm at 4*degrees*C. The pellet was washed
again with extraction buffer. Both supernatants were filtered through
4 thicknesses of sterile gauze and immediately and repeatedly
extracted with an equal volume of phenol: chloroform (1:1 v/v) until
no material was visible at the interface. DNA and total cellular RNA
were precipitated from the aqueous phase with 2 volumes of ethanol and
centrifuged again. The pellet was washed with an equal volume of
phenol twice and chloroform once. Cellular RNA was precipitated from
the supernatant with 2 volumes of ethanol containing 100 mM NaCl.
Polyadenylated RNAs were separated from nonpolyadenylated molecules by
two cycles of chromatography on oligo (dT)-cellulose (P.L.
Biochemicals) as described by AVIV and Leder (1972). The
polyadenylated RNAs were precipated twice with ethanol.<p>

In vitro translation of polyadenylated RNAs<p>

The RNAs were translated in rabbit reticulocyte lysates. The
translation kit was purchased from New England Nuclear. We adopted the
conditions described by the manufacturer.<p>

Polyacrylamide gel electrophoresis<p>

The synthesized proteins were analysed in SDS acrylamide gels prepared
as described by Laemmli (1970) as modified by Burgess and Jendrisak
(1975)<p>

Double-stranded cDNA synthesis<p>

The first strand was synthesized at 42*degrees*C in a reaction
containing 10 mM Tris-HCl pH 8.2, 4 mM KCl, 2.4 mM DTT, 200 micro M
dATP, dGTP, dTTP and cCTP, 5 micro g/ml obligo (dT) and 15 units
reverse transcriptase per micro g polyadenylated RNAs. After
elimination of RNA by heating at 100*degrees*C for 3 min., the second
strand was synthesized at 22*degrees*C in a reaction containing 100 mM
Hepes pH 6.8, 100 mM KCl, 100 M dATP, dGTP, dTTP and dCTP, 100 micro
Ci *alpha* 32P dCTP (3000 ci/mole) and 20 units/ml Klenow fragment of
&#206;&#151;&#197; polymerase I. &#160;&#157;he reaction mixture was extreacted
with phenol: chloroform (1:1, v/v). The double standed cDNA was
ehtanol precipitated and treated with S1 nuclease in a reactio
ncontining 30 mM sodium acetate pH 4.8 300 mM NaCl, 3 mM ZnCl 2 and 50
U/ml enzyme. After 60 min. at 37*degrees*C the reaction mixture was
heated for 10 min. at 65*degrees*C in 0.1% SDS, 10 mM EDTA.<p>

Analysis of double stranded cDNA size<p>

The double stranded cDNA was fractionated on a 5-20% sucrose gradient
in TNE buffer (20 mM Tris-HC1 pH 7.5, 100 mM NaCl, 1 mM EDTA).
Centrifugation was for 15 hr at 30 000 rpm and 15*degrees*C in a
Beckmann SW60 rotor. Collected fractions (300 micro l) were ethanol
precipitated and dissolved in 20 micro l 0.1 SSC (0.075 mM Na-citrate,
0.75 mM NaCl). 1 micro 1 of each fraction was run on 1.5% agarose gel
and the migration was detected by autoradiography.<p>

Cloning of double stranded cDNA<p>

Terminal addition of dCTP to the double stranded cDNA by terminal
deoxynucleotidyl transferase was carried out as described by Chang et
al. (1978). The tailed material was annealed with pBR322 that had been
cleaved in the *beta*-lactamase gene by Pst I endonuclease and treated
with terminal transferase to add homopolymeric dG tails at the
cleavage site. Tailed pBR322 and tailed double stranded cDNA were
mixed in a molar ratio of 20:1 in 10 mM Tris HCl pH 7.5, 100 mM NaCl,
1 mM EDTA. The mixture was incubated first at 65*degrees*C for 2 min
and then at 42*degrees*C for 2 h. 10 micro l of annealing solution
were used for 100 micro l of competent cells. Transformation of E.
coli K12 type 5K was carried out as described by Mandel and Higa
(1970)<p>

<a href="/ais/images/vol20/vol20-pg82-fig1.gif">Click here for figure</a><p>

Figure 1: In vitro products of mRNA translated in rabbit reticulocyte
lysates. The translation products were analyzed by polyacrylamide gel
and autoradiography. Tracks 1 and 3: no mRNA added to the reticulocyte
lysate. Track 2: mRNA from New England Nuclear added. Track 4: A.
thaliana mRNA added. Track 5: markers with molecular weight as
indicated (in Kilodaltons).<p>

Plasmid content analysis<p>

The plasmid DNA was used to transform E. coli K12 mutants affected in
the metabolism of pyrmidines.<p>

RESULTS AND DISCUSSION<p>

The first step in the construction of a double stranded cDNA bank is
the isolation of mRNAs. To avoid RNA degradation by the very active
ribonucleases in plants we ground the leaves in the extaction buffer
with an equal volume of phenol containing 8-hydroxyquinoline. To avoid
the loss of RNA at the interface between the aqueous phase and the
phenol phase, the pellet was washed again with extraction buffer.
Starting with 10 micro g of leaves we obtained 1.65 mg of total
cellular RNAs and 20 micro g of polyadenylated RNAs after obligo(dT)
cellulose chromatography. The mRNAs activity was tested by
incorporation of 35 S methionine into proteins. The products of
translation were electrophoresed alongside with markers of known
cellular weight (Figure 1). The analysis showed that approximately 50%
of the synthesized polypeptides were of a molecular weight greater
than 40 000 daltons. This suggested that some isolated mRNAs were
probably not degraded and that they were being translated with a high
degree of fidelity.<p>

<a href="/ais/images/vol20/vol20-pg83-fig2.gif">Click here for figure</a><p>

Figure 2: Scheme used for cloning and expression of A. thaliana double
stranded cDNA in bacterial mutants.<p>

The rest of this study is summarized in the experimental scheme
(Figure 2). 250 ng of double stranded c DNA were synthesized from the
purified mRNAs. The double stranded cDNA, the size of which was
greater than 0.5 kilobase pairs (Figure 3), was inserted in the
*beta*-lactamase gene of pBR322 to allow efficient transcription and
translation (Chang et al., 1978). Constructed chimeric plasmids were
introduced into CaCl 2 treated E. coli K12 type K5 and tetracycline
resistant transformants were selected. 150 000 transformants per micro
g of insert were obtained and 90% of the tetracycline resistant
colonies contained inserts by the criterion of sensitivity to
ampicillin. 25% of the clones analysed by minipreparations (Holmes and
Quigley, 1981) were found to have inserts greater than 1 Kb as
expected from the profile of the 5-20% sucrose gradient.<p>

<a href="/ais/images/vol20/vol20-pg84-fig3.gif">Click here for figure</a><p>

Figure 3: Sucrose gradient fractions of synthesized double stranded
cDNA analysed on a horizontal 1.5% agarose gel and revealed by
autoradiography. The double stranded cDNA, the size of which was
greater than 0.5 kilobase pairs (dark arrows) was selected.<p>

Plasmid DNA extracted from the population was introduced by
transformation into mutants of E. coli K12 affected in pyrimidine
metabolism. The tetracycline resistant transformants were transferred
to minimal agar medium. For each of the genes, we tested about 5 x 10
5 transformants. None were able to complement the bacterial mutants.
Complementation by a double stranded cDNA bank of 63 000 clones
requires that at least one mRNA per 10 000 codes for an enzyme of the
pathway for the biosynthesis of pyrimidines. For Saccharomyces
cerevisiae, one mRNA per 6 000 codes for the same enzyme (Lacroute,
personal communication). So a frequency 1/10 000 seems right in the
case of A. thaliana. However, is the double stranded cDNA size
representative of the mRNA size? As seen above 50% of the polypeptides
synthesized in vitro (Figure 2) were greater than 40 000 daltons; this
allows us to estimate that 50% of the mRNAs are greater than 1
kilobase pairs. Now only 25% of the double stranded cDNA is greater
than 1 Kb. It is possible that the negative results are due to an
incomplete representation of the mRNAs greater than 1 Kb. Since in
higher eukaryotes, the first three reactions of the biosynthesis of
pyrmidines are controlled by a multienzymic protein, the mRNA size of
which is approximately 8 000 nucleotides (Padgett et al., 1979) in the
case of the hamster. By our approach we did not obtain a double
stranded cDNA corresponding to a mRNA of this size and which,
moreover, is little represented. However, for multienzymic proteins
perhaps it is not necessary to synthesize a full length double
stranded cDNA. Indeed a double stranded cDNA coding for just one
enzymatic activity could complement a bacterial mutant. Before giving
up this approach, it is necessary to obtain a new bank containing more
double stranded cDNA greater than 1 Kb.<p>

SUMMARY<p>

Attempts to clone A. thaliana genes by transformation of yeast
auxotrophic mutants were not successful. We attempted another approach
by producing a double stranded cDNA bank. We used this bank to
complement bacterial mutants affected in pyrimidine metabolism. The
first results are negative but it is necessary to synthesize a new
bank to be definitively sure that there is no compatibility between A.
thaliana and E. coli for pyrimidine metabolism. However, this bank can
be tested either with DNA probe from other organisms, or with
antibodies to screen colonies containing a fused protein bearing both
*beta*-lactamase and eukaryotic protein antigenic determinants
(Villa-KLomaroff et al., 1978).<p>

ACKNOWLEDGEMENTS<p>

We wish to thank Dr. P. Sondermeyer for help with the synthesis of the
double stranded cDNA and Professors Hirth and Lacroute for their
interest.<p>

REFERENCES<p>

Aviv, H. and P. Leder: Proc. Natl. Acad. Sci. USA 69, 1408-1412 (1972)<p>

Bach, M.L., F. LaCroute and D. Botstein: Proc. Natl. Acad. Sci. USA
76, 386-390 (1979)<p>

Bennet, M.D. and J.B. Smith: Proc. R. Soc. London Ser. B 274, 227-274
(1976)<p>

Burgess, R.R. and J.J. Jendrisak: Biochemistry 14, 4634-4638 (1975)<p>

Chang, A.C.Y., J.H. Nunberg, R.J. Kaufman, H.A. Erlich, R.T. Schimke
and S.N. Cohen: Nature 275, 617-624 (1978)<p>

Dudley, K. and D.H. Northcote: Planta 138, 41-48 (1978)<p>

Holmes, D.S. and M. Quigley: Anal. Biochem. 114, 193-197 (1981)<p>

Laemmli, U.K.: Nature 227, 680-685 (1970)<p>

Loison, G., R. Jund, M. Nguyen-Juilleret and F. LaCroute: Current
Genetics 3, 119-123 (1981)<p>

Mandel, M. and A. Higa: J. Mol. Biol. 53, 159-162 (1970)<p>

Padgett, R.A., G.M. Wahl, P.F. Coleman and G.R. Stark: J. Biol. Chem
254, 974-980 (1979)<p>

Villa-Komaroff, L., A. Efstratiadis, S. Broome, P. Lomedico, R.
Tizard, S.P. Naber, W.L. Chick and W. Gilbert: Proc. Natl. Acad. Sci.
USA 75, 3727-3731 (1978)<p>

Werner, M., M. Crabeel, N. Glansdorff and A. Pierard: Arabid. Inf.
Serv. 19, 1-10 (1982)<p>

