<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>

<HEAD>
	<META NAME="GENERATOR" Content="Symantec Visual Page 1.0">
	<META HTTP-EQUIV="Content-Type" CONTENT="text/html;CHARSET=iso-8859-1">
	<TITLE>Protocols</TITLE>
</HEAD>

<BODY BACKGROUND="Images/fond.GIF">

<P ALIGN="CENTER"><A NAME="Top"></A><IMG SRC="Images/protocols.gif" WIDTH="397" HEIGHT="54"
ALIGN="BOTTOM" BORDER="0"></P>
<P>
<HR>


<UL>
	<LI>1) <A HREF="#1">YAC library screening by PCR on three-dimensional DNA pools</A>
	<UL>
		<LI><A HREF="#Preparation">preparation of DNA Pools</A>
		<LI><A HREF="#PCR">PCR Screening</A>
	</UL>
	<LI>2) <A HREF="#2">YAC library screening by hybridization</A>
	<UL>
		<LI><A HREF="#Yeast">Yeast DNA preparation in MT plates</A>
		<LI><A HREF="#Preparation10">Preparation of hybridization filters</A>
		<LI><A HREF="#Hybridization">Hybridization</A>
	</UL>
	<LI>3) <A HREF="#3">Preparation of YAC end probes</A>
	<UL>
		<LI><A HREF="#Left">Left arm rescue</A>
		<LI><A HREF="#Right">Right arm rescue</A>
	</UL>
	<LI>4) <A HREF="#4">References</A>
</UL>

<P>
<HR>
<BR>
<BR>
Standard molecular techniques were essentially as decribed by <A HREF="#Sambrook">Sambrook<I>
et al.</I> (1989)</A>.<BR>
The CIC library (12 microtitre plates) and yUP library (24 plates) were propagated
in yeast strain AB1380 (<A HREF="#Burke">Burke<I> et al.</I>, 1987</A>) grown in
super-AHC selective medium (Yeast Nitrogen Base w/o amino acids 0.255%, (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB>
0.75%, glucose 3%, casamino acids 1.65%, drop-out powder w/ adenine, w/o tryptophan
and uracil, 0.195%).<BR>
<BR>
<BR>
<A NAME="1"></A><FONT SIZE="4" COLOR="#DD0000"><B>1) YAC library screening by PCR
on three-dimensional DNA pools</B></FONT><I><BR>
</I><BR>
<A NAME="Preparation"></A><FONT SIZE="4"><B>Preparation of DNA pools</B></FONT><BR>
Three replicas of the CIC YAC library were grown in 150 &#181;l super-AHC medium
in microtitre plates at 28&#176;C without agitation. After 24h of culture, clones
growing poorly were reinoculated from the glycerol master stock. After 1 additional
day of culture, clone cultures were mixed according to the following scheme : the
library was divided into two parts, plates 1-6 and plates 7-12, then each pile was
theoretically divided again in two (columns 1-6 and columns 7-12) to give four super-pools
(alpha, beta, gamma, delta) in matrix arrangement. Each super-pool contained six
half-microtiter plates (288 clones) and represented roughly one genome-equivalent.
Using the three replicas, clones were mixed along each dimension, resulting in twenty
3-D pools for each super-pool (6 pools of half-plates (48 clones each), 8 pools of
half-rows (48 clones each), 6 pools of columns (36 clones each)). The whole library
was therefore represented in a total of 80 pools. Each mixture of clone cultures
(about 5 ml) was extracted using a standard yeast DNA miniprep procedure (<A HREF="#Ausubel">Ausubel<I>
et al.</I>, 1995</A>). Final DNA concentrations were 100-200 ng/&#181;l. Preparation
of DNA pools for the yUP library was essentially as described for the CIC library.<BR>
<BR>
<A NAME="PCR"></A><FONT SIZE="4"><B>PCR Screening</B></FONT><BR>
For PCR screening, DNA stocks were diluted 50-fold. PCR reactions, which were performed
in a PTC100-96 thermal cycler (MJ Research), consisted of: 5 &#181;l (10-20 ng) of
diluted DNA stock, 10 mM Tris-HCl pH 9 at 25&#176;C, 50 mM KCl, 0.1% Triton X-100,
2.5 mM MgCl<SUB>2</SUB>, 200 &#181;M of each dNTP, 10 pmoles of each primer, 1 unit
Taq DNA polymerase, in 25 &#181;l total reaction volume. Cycle conditions were: 2
min at 94&#176;C; then 35 cycles of 20 sec at 94&#176;C, 30 sec at 50-55&#176;C,
1 min at 72&#176;C; then 2 min at 72&#176;C.<BR>
For each primer pair, a control experiment was first performed on AB1380 DNA, Columbia
genomic DNA, total library DNA, and DNAs from the 4 super-pools (alpha, beta, gamma,
delta). Three-dimensional pools corresponding to positive super-pools were then tested
using the same conditions as above. Positive clones, determined by the intersection
of three positive planes, were checked using a 1/200 dilution of the individual clone
DNA isolated as described below. When multiple positive clones were detected in the
same super-pool, all the possible intersections were tested individually. On average,
a total of 30 PCR reactions were sufficient for unambiguous characterization of any
positive clone.<BR>
<BR>
<BR>
<A NAME="2"></A><FONT SIZE="4" COLOR="#DD0000"><B>2) YAC library screening by hybridization
</B></FONT><I><BR>
</I><BR>
<A NAME="Yeast"></A><FONT SIZE="4"><B>Yeast DNA preparation in MT plates</B></FONT><BR>
The 1152 clones of the CIC YAC library were each grown individually in 1 ml of selective
medium (super-AHC) in 1.5 ml microtube racks (96-well format) at 28&#176;C without
agitation. After 24h of culture, clones growing poorly were inoculated again from
the glycerol master stock. After 1 additional day of culture, 800 &#181;l of medium
were aseptically removed without disturbing the yeast pellet using a home-made 96-channel
pipettor, and replaced with fresh medium. The cultures were further incubated overnight
at 28&#176;C. The following day, most of the culture supernatant was removed without
centrifugation, and the yeast cells were resuspended with a multi-channel pipetman
in the remaining medium (ca. 200 &#181;l) and transferred to regular MT plates. Following
centrifugation in a swing-out rotor (1200 rpm, 3 min), the medium was completely
removed using the 96-channel pipettor. The yeast pellets were then resuspended in
120 &#181;l TES (100 mM Tris-HCl pH 8.0, 100 mM EDTA, 0.9 M Sorbitol, 0.2% &#223;-mercaptoethanol,
2 mg/ml Zymolyase 20-T (Seikagaku)) using a multi-channel pipetman, and incubated
for 2 h at room temperature. Following spheroplasting, cells were pelleted by centrifugation
(1200 rpm, 5 min) and the supernatants removed using the 96-channel pipettor. The
spheroplasts were then lysed by resuspension in 70 &#181;l TNLE (10 mMTris-HCl pH
8.0, 200 mM NaCl, 1% Lithium Dodecyl Sulfate, 5 mM EDTA) and incubation 1h at room
temperature. DNA was then precipitated by addition of 70 &#181;l isopropanol. The
plates were centrifuged (4000 rpm, 10 min), and the pellets rinsed with 100 &#181;l
70% ethanol, and dried on the bench. The final pellets were resuspended in 50 &#181;l
TE + RNase A (0.01 mg/ml), incubated for 1 h at 50&#176;C, and then stored frozen
at -20&#176;C.<BR>
<BR>
<A NAME="Preparation10"></A><FONT SIZE="4"><B>Preparation of hybridization filters</B></FONT><BR>
For preparation of hybridization filters, 20 &#181;l of these DNA stocks were transferred
to 384-well plates containing 5 &#181;l of 5x denaturing solution (0.05 % xylene-cyanol,
2N NaOH, 0.05% SDS). Denatured DNAs (ca. 0.2 &#181;l) were simultaneously transferred
to 12 x 8 cm2 Hybond N+ membranes (Amersham) using a disposable 384-pin inoculator.
The amounts indicated here were enough for at least 80-90 membranes.<BR>
<BR>
<BR>
<A NAME="Hybridization"></A><FONT SIZE="4"><B>Hybridization</B></FONT><BR>
Membranes were prehybridized for a few hours in Church hybridization buffer (250
mM Na<SUB>2</SUB>HPO<SUB>4</SUB> pH 7.4, 7% SDS, 2 mM EDTA, 0.2 mg/ml heparin, 0.1
mg/ml denaturated salmon-sperm DNA) at 65&#176;C in an hybridization oven. The probe
was added and hybridization performed for 24 h at 65&#176;C. Following hybridization,
the membranes were washed 15 min in 2xSSC, 0.5% SDS, 0.2% Na pyrophosphate at 55&#176;C
and 5-10 min in 0.2xSSC, 0.5% SDS, 0.2% Na pyrophosphate at 55&#176;C. Membranes
were autoradiographed against regular X-ray films or the BAS-1500 system (Fuji).<BR>
<BR>
An example of a result is shown <A HREF="Images/screen.GIF">here</A>.<BR>
<BR>
<BR>
<A NAME="3"></A><FONT SIZE="4" COLOR="#DD0000"><B>3) Preparation of YAC end probes</B></FONT><I><BR>
</I><BR>
YAC end probes from yUP and CIC clones were either prepared using vectorette-PCR
as described by <A HREF="#Matallana">Matallana<I> et al.</I> (1992)</A>, or by right-
and left-arm rescue in <I>Escherichia coli</I>.<BR>
<BR>
<A NAME="Left"></A><FONT SIZE="4"><B>Left-arm rescue</B></FONT><BR>
For left-arm rescue, the YAC clone was grown in 5 ml super-AHC medium at 28&#176;C.
A standard yeast DNA minipreparation was performed (<A HREF="#Ausubel">Ausubel<I>
et al.</I>, 1995</A>), and 500 ng were digested to completion either by <I>Xho</I>I
or <I>Nde</I>I. The DNA was then precipitated, and self-ligated in 400 &#181;l total
volume with 10 units of T4 DNA ligase. After precipitation, the DNA was resuspended
in 10 &#181;l H2O, and 2 &#181;l were electroporated into competent <I>E. coli</I>
DH5a cells, which were plated on LB medium containing 200 mg/l ampicillin. The end
probe was isolated from the vector sequences using <I>Eco</I>RI and <I>Xho</I>I or
<I>Nde</I>I.<BR>
<BR>
<A NAME="Right"></A><FONT SIZE="4"><B>Right-arm rescue</B></FONT><BR>
For right-arm rescue, we used the ability of the URA3 gene to complement <I>pyr</I>F
mutations in <I>E. coli</I> (<A HREF="#Rose">Rose<I> et al.</I>, 1984</A>). We used
strain S&#216;5009, a <I>pyr</I>F derivative of MC1061 (a gift from K. Schnorr, Denmark)
(<I>ara</I>D139, D(<I>ara</I>-<I>leu</I>)7647, D<I>lac</I>74, <I>gal</I>U, <I>hsd</I>R,
<I>rps</I>L, <I>pyr</I>F30). DNA was extracted from YAC clones as above and 500 ng
were digested with <I>Xho</I>I, and co-precipitated with 500 ng pBluescript II-SK+
that had been cut by <I>Xho</I>I and dephosphorylated. Ligation was performed in
a total volume of 20 &#181;l with 1 unit T4 DNA ligase. The ligation mixture was
then digested by <I>Bam</I>HI or <I>Xba</I>I and precipitated again. A second ligation
was performed in diluted conditions (400 &#181;l total volume with 10 units T4 DNA
ligase) to favour circularisation. After precipitation, the DNA was resuspended in
10 &#181;l H<SUB>2</SUB>O, and 4 &#181;l were electroporated into competent <I>E.
coli</I> S&#216;5009 cells, which were plated on M9 selective medium (<A HREF="#Sambrook">Sambrook<I>
et al.</I>, 1989</A>), supplemented with 200 mg/l ampicillin. Clones appeared after
2 days incubation at 37&#176;C. This two-step cloning procedure dramatically improved
the number of complementing clones recovered. The clones obtained were then analysed
by restriction. The end probe was isolated from the vector sequences using <I>Eco</I>RI
and <I>Bam</I>HI or <I>Xba</I>I.<BR>
<BR>
<BR>
<A NAME="4"></A><FONT SIZE="4" COLOR="#DD0000"><B>4) References</B></FONT><BR>
<BR>
<A NAME="Ausubel"></A><B>Ausubel F.M., Brent R., Kingston R.E., Moore D.D., Seidman
J.G., Smith J.A. and Struhl K. </B>(1995). <I>Current Protocols in Molecular Biology</I>.
John Wiley &amp; Sons, Inc.<BR>
<B><BR>
<A NAME="Burke"></A>Burke D.T., Carle G.F. and Olson M.V.</B> (1987) Cloning of large
segments of exogenous DNA into yeast by means of artificial chromosome vectors. <I>Science</I>
<B>236</B>, 806-812.<BR>
<BR>
<A NAME="Matallana"></A><B>Matallana E., Bell C.J., Dunn P.J., Lu M. and Ecker J.R.</B>
(1992) Genetic and physical linkage of the <I>Arabidopsis</I> genome: methods for
anchoring yeast artificial chromosomes. In <I>Methods in Arabidopsis research</I>
(Koncz, C., Chua, N.H. and Schell, J., eds). Singapore: World scientific, pp. 144-169.<BR>
<BR>
<A NAME="Rose"></A><B>Rose M., Grisafi P. and Botstein D.</B> (1984) Structure and
function of the yeast URA3 gene : expression in <I>Escherichia coli</I>. <I>Gene</I>
<B>29</B>, 113-124.<BR>
<BR>
<A NAME="Sambrook"></A><B>Sambrook J., Fritsch E.F. and Maniatis T. </B>(1989). <I>Molecular
cloning : a laboratory manual</I>. Cold Spring Harbor, New-York.<BR>
<BR>

<HR>
<BR>
<BR>
(<A HREF="#Top">To top of page</A>) (<A HREF="chromosome3.html">Back to the Chromosome
3 page</A>)<BR>



</BODY>

