<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>
<head>
<title>ICAR 2007 - Metabolism</title>
<link rel="Shortcut Icon" href="/favicon.ico">
<link rel="stylesheet" type="text/css" href="/css/main.css" />
<link rel="stylesheet" type="text/css" href="/css/search.css" />
<link rel="stylesheet" type="text/css" href="/css/base.css" />

<style type="text/css">
@import "/css/page/pagelevels.css";
</style>

<style type="text/css">
.abs {
 margin-top: 30px;
}
.abs div {
 margin: 10px;
}
</style>
</head>
<body>

<div id="container">

<script src="https://ssl.google-analytics.com/urchin.js" type="text/javascript"></script>
<script type="text/javascript">
_uacct = "UA-1945230-1";
urchinTracker();
</script>

 <div id="nav">	
  <div id="logo"><a href="/index.jsp"><img src="/i/logo2.gif" width="87" height="61" alt="tair"/></a></div>
 </div>

<div id="content" class="clearfix">

<div class="abs">

<div style="font-weight:bold">The Overexpression of a Ser&#47;Thr Kinase Protein Altered Ethylene Responses In Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Liping Qiu</span>
	<sup>1</sup>, 

	<span  >
	Chi-Kuang Wen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of plant physiology and ecology,Shanghai,China

</div>

<div>To uncover new components involved in the ethylene signal transduction pathway, gene expression profiles were analyzed by Affymetrix&#210; arrays. Four genes were identified and their overexpressions alter ethylene responses. Here we report that the overexpression of ATR3 &#40;Altered Ethylene Responses 3&#41; conferred constitutive ethylene responses. ATR3 encodes a kinase protein and the kinase domain locates near the C terminus. In vitro assay suggest that ATR3 exhibits ser&#47;thr kinase activity. The overexpression of ATR3 kinase domain conferred ethylene insensitivity, implying that the ATR3 N terminus is a regulatory domain which negatively controls the C&#45;terminal function in the repression of ethylene responses. Possible roles of ATR3 in the ethylene signal transduction pathway were studied by expressing the full&#45;length ATR3 and ATR3 kinase domain in ethylene response mutants. Ethylene responses conferred by ATR3ox was completely masked by the etr1&#45;1 mutation and Ag&#40;I&#41; treatment. However, the ein3&#45;1 mutation and the overexpression of EBF1 only partially masked the overexpression of ATR3.The overexpression of ATR3 kinase domain further rescued the mutant phenotype of etr1 ers1 mutant. Possible roles of ATR3 in the regulation of ethylene responses are discussed.</div>

</div>







<div class="abs">

<div style="font-weight:bold">The ETR1 N&#45;Terminal Signaling is Primarily Dependent on Subfamily I Receptor and can be Covalent Independent</div>

<div>

	<span style="text-decoration: underline" >
	Fang Xie</span>
	<sup>1</sup>, 

	<span  >
	Qian Liu</span>
	<sup>1</sup>, 

	<span  >
	Chi-Kuang Wen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Key Laboratory of Plant Molecular Genetics, Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese

</div>

<div>Arabidopsis ETR1 is an ethylene receptor and plays unique roles in the regulation of ethylene responses. Although ETR1 has the conserved structures and domains of prokaryotic His&#45;kinases of the two&#45;component signaling modules, little is known about the mechanism by which the receptor signal is mediated to downstream components. To further study how the ETR1 signal is mediated to the repression of ethylene responses, we focused on the receptor signal output mediated through the N terminus. It has been shown that the wild&#45;type ETR1 N terminus, the etr1&#40;1&#45;349&#41; protein, cannot repress ethylene responses and that the covalent disulfide linkages are important to the N&#45;terminal signaling. In our study, using genetic and transformation approaches, we found that etr1&#40;1&#45;349&#41; was still capable of the repression of ethylene responses, implying receptor signal output through the ETR1 N terminus. However, the etr1&#45;7 ers1&#45;3, but not etr1&#45;7 ers1&#45;2, mutations completely blocks the etr1&#40;1&#45;349&#41; receptor signal output.  Moreover, the dominant N&#45;terminal signaling mediated by Ag&#40;I&#41; and etr1&#45;1&#40;1&#45;349&#41; was also masked in etr1&#45;7 ers1&#45;3, suggesting essential roles of subfamily I receptors in the ETR1 N&#45;terminal signaling. The loss of subfamily II receptors, however, did not perturb the etr1&#45;1&#40;1&#45;349&#41;&#45;mediated ethylene insensitivity. Roles of the disulfide linkages in the ETR1 N&#45;terminal signaling were examined. Mutations that disrupt the formation of covalent dimerization via Cys4 and Cys6 did not perturb the repression of ethylene responses mediated by ETR1, implying that the N&#45;terminal signaling can be non&#45;covalent. Non&#45;covalent protein&#45;protein interaction was examined in yeast two&#45;hybrid assay and the GAF domain may be involved in the ETR1 N&#45;terminal signal output.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis Information Resource &#40;TAIR&#41;</div>

<div>

	<span style="text-decoration: underline" >
	Donghui Li</span>
	<sup>1</sup>, 

	<span  >
	Eva Huala</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Carnegie Institution of Washington, Department of Plant Biology, Stanford, (CA), USA

</div>

<div>The Arabidopsis Information Resource &#40;TAIR; www.arabidopsis.org&#41; is a community database for Arabidopsis thaliana that provides access to curated data about the structure and function of this model plant. The workshop is designed for users who wish to more effectively utilise the curated data and software resources provided by TAIR. Specifically we will address curation of three major data types: gene structure, gene function and metabolic pathway annotation.<br/><br/>Arabidopsis serves as an important resource and potential benchmark for the annotation of other plant genomes. As such, a &#34;gold standard&#34; of gene structure annotation is demanded. We will describe our progress in meeting this aim and give details of the TAIR7 genome release. In addition we will outline our plans for the subsequent TAIR8 release and some of the annotation challenges which remain.<br/><br/>In addition to annotating gene structures, TAIR curators annotate gene function using terms developed by the Gene Ontology &#40;GO&#41; Consortium that describe the molecular function, biological process, and subcellular location of a gene product. We will use TAIR's GO annotations to demonstrate how controlled vocabularies allow for standardization of annotation and assist in comparative genomics. We will describe the process of GO annotation at TAIR and how the annotations can be used to classify large data sets or to facilitate annotation of other sequenced plant genomes. We will demonstrate how to access TAIR's GO annotations in bulk and to derive a global view of the status of functional annotation of the Arabidopsis genome.<br/><br/>Plant metabolic databases have been developed to visualize the universe of primary and specialized &#40;secondary&#41; metabolism, to present the flexibility and interconnectivity of biochemical networks and to serve as matrix to predict Pathway&#47;Genome Database's &#40;PGDB's&#41; based on sequenced genomes. We will introduce the contents and curation of the multi&#45;organism database MetaCyc and the Arabidopsis thaliana specific database AraCyc. The major functions and applications of MetaCyc and AraCyc will be presented including the visualization of large&#45;scale data on the Omics viewer. </div>

</div>





<div class="abs">

<div style="font-weight:bold">The dissection of the WUSCHEL C&#45;terminal domain reveals specificity for the interaction with TOPLESS</div>

<div>

	<span style="text-decoration: underline" >
	Elena Clerici</span>
	<sup>1</sup>, 

	<span  >
	Eric van der Graaff</span>
	<sup>2</sup>, 

	<span  >
	Thomas Laux</span>
	<sup>2</sup>, 

	<span  >
	Brendan Davies</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Leeds, UK, 

	<sup>2</sup>Institute of Biology III, Freiburg, Germany

</div>

<div>An important characteristic of plants is their ability to produce new organs during their entire life cycle. The homeobox gene WUSCHEL &#40;WUS&#41; encodes an important regulator of meristem maintenance in Arabidopsis, yet its mechanism of action in meristem regulation is presently unclear.  TOPLESS &#40;TPL&#41;, previously named WSIP1, a transcriptional co&#45;repressor like protein, interacts with the WUS protein and also plays a role in meristem maintenance. WUS and TPL belong to two small families, other members of which are also capable of interacting with each other in specific combinations.  An interaction matrix, testing all combinations of interactions between TPL and WUS proteins shows that specific protein domains are required for interaction. An ERF&#45;like domain, a domain previously associated with transcriptional repression, at the C&#45;terminus of WUS appears to be important for the interaction, suggesting a means by which WUS might repress genes promoting differentiation. A similar domain is also present in AUX&#47;IAA proteins, which are repressor proteins involved in auxin signaling. This suggests that members of the TPL family could mediate the repressive effects of AUX&#47;IAA proteins. Extending the interaction matrix to include AUX&#47;IAA proteins will reveal any specificity in these interactions. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Global Identification of Target Genes Regulated by Gibberellin Signaling Repressor RGA during Arabidopsis Flower Development</div>

<div>

	<span style="text-decoration: underline" >
	Wen-Wei Hu</span>
	<sup>1,2</sup>, 

	<span  >
	Xingliang Hou</span>
	<sup>2</sup>, 

	<span  >
	Hao Yu</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Temasek Life Sciences Laboratory, National University of Singapore, 

	<sup>2</sup>Department of Biological Sciences, National University of Singapore

</div>

<div>The phytohormone gibberellins &#40;GAs&#41; play crucial roles in regulating many aspects of plant development. Previous studies have shown that the default status for GA signaling is repression that is controlled by DELLA proteins, a family of nuclear growth repressors. The GA signal derepresses its signaling pathway by promoting DELLA protein degradation via 26S proteasome&#45;dependent ubiquitylation pathway. In GA&#45;deficient ga1&#45;3 mutants, the development of all floral organs was arrested, while various combinations of null mutants of DELLA proteins could gradually rescue floral defects in ga1&#45;3, suggesting that DELLA proteins, especially RGA, play critical roles in mediating the GA&#45;signaling pathway in the control of flower development. As a first step to study the target genes of GA signaling mediated by DELLA proteins in flower development, functional ga1&#45;3 rga rgl2 35S::RGA&#45;GR transgenic lines were selected for high&#45;throughput microarray analysis. A number of genes whose transcripts were modulated by induced RGA activity were identified. In particular, the RGA&#45;regulated genes included those responsible for synthesis, transport, and signaling of other phytohormones, indicating the presence of potential crosstalk between GA and other hormones. Semi&#45;quantitative RT&#45;PCR and in situ hybridization further confirmed that the expression of these identified genes was indeed regulated by RGA in the GA signaling pathway. Characterization of the over&#45;expression transgenic lines and their corresponding loss&#45;of&#45;function mutants is being carried out to further our mechanistic understanding of these target genes downstream of DELLA proteins during flower development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis LIM Protein WLIM1 Binds and Bundles Actin Filaments</div>

<div>

	<span style="text-decoration: underline" >
	Andre Steinmetz</span>
	<sup>1</sup>, 

	<span  >
	Clement Thomas</span>
	<sup>1</sup>, 

	<span  >
	Flora Moreau</span>
	<sup>1</sup>, 

	<span  >
	Stephane Tholl</span>
	<sup>1</sup>, 

	<span  >
	Jessica Papuga</span>
	<sup>1</sup>, 

	<span  >
	Monika Dieterle</span>
	<sup>1</sup>, 

	<span  >
	Celine Hoffmann</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Public Research Center-Health, Luxembourg, Luxembourg

</div>

<div>Flowering plants express several LIM proteins some of which are specific to mature pollen grains, while others are expressed only in vegetative structures, sometimes in a tissue&#45;dependent pattern. The aims of our studies are to understand the functional differences between the six LIM proteins identified in Arabidopsis by their coding sequences. One of these proteins we have been studying is WLIM1 &#40;At1g10200&#41;. To examine its subcellular distribution we expressed it as a GFP fusion under the control of an inducible promoter and analysed its localization in root tissues of Arabidopsis seedlings by laser scanning confocal microscopy. The fusion protein was found to associate with filamentous structures in several cell types. Following incubation of the seedlings with Latrunculin B &#40;LatB&#41; the cortical cytoskeletal structures disappeared while the more internal structures appeared to be more resistant. This sensibility to LatB of the WLIM1&#45;GFP&#45;labeled cables identified the structures as actin filaments. High&#45; and low&#45;speed centrifugation experiments showed that the bacterially&#45;expressed and affinity&#45;purified protein binds actin filaments in vitro without intermediary protein and that this association triggers F&#45;actin bundling. Interestingly also, while in root hairs the cytoskeletal organisation was identical to that observed for fimbrin&#45;GFP&#45;expressing cells, another root cell type showed marked differences in cytoskeletal structure: most notably, the WLIM1&#45;GFP&#45;expressing cells showed thick, more randomly organized actin cables, whereas in fimbrin&#45;GFP&#45;expressing cells the cables were much thinner and longitudinally oriented. These observations identify the Arabidopsis protein WLIM1 not only as an F&#45;actin bundling protein but also as an actin cytoskeleton organizer.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular genetics of temperature modulation of defense responses</div>

<div>

	<span  >
	Ying Zhu</span>
	<sup>1</sup>, 

	<span  >
	Yi Wang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Jian Hua</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Cornell University, Ithaca, NY 14853, USA

</div>

<div>Temperature regulates growth and development as well as biotic and biotic responses in plants. However, the molecular mechanisms underlying how temperature modulates these processes are largely unknown. We are particularly interested in how a relatively higher temperature suppresses defense responses and cell death in plant&#45;pathogen interactions. Both <span style="font-style: italic">bon1&#45;1</span> and <span style="font-style: italic">snc1&#45;1</span> mutants have constitutive defense responses and subsequently dwarf phenotypes at 22&#186;C and these phenotypes are absent at 28&#186;C due to an inhibition of defense response by higher temperatures.  Using the morphological phenotype as a readout of defense responses, we carried out genetic screens in <span style="font-style: italic">bon1</span> and <span style="font-style: italic">snc1</span> respectively for mutants defective in this temperature modulation. A number of isolated mutants exhibit dwarf phenotype at 28&#186;C in <span style="font-style: italic">bon1</span> or <span style="font-style: italic">snc1</span> but not in the wild type, and these mutants are named &#8216;<span style="font-style: italic">int</span>&#8217; for &#8216;insensitive to temperature&#8217;. Some of the <span style="font-style: italic">int</span> mutants appear to be enhancers of <span style="font-style: italic">bon1</span> or <span style="font-style: italic">snc1</span> while others appear to confer temperature insensitivity to <span style="font-style: italic">bon1</span> or <span style="font-style: italic">snc1</span> mutants. We are now in the process of isolating these <span style="font-style: italic">INT</span> genes through tagging or map&#45;based cloning. One of the <span style="font-style: italic">int</span> mutants is likely a new <span style="font-style: italic">snc1</span> allele that contains a missense mutation in the <span style="font-style: italic">R</span> gene <span style="font-style: italic">SNC1</span>, suggesting that temperature modulation of defense responses could occur at the <span style="font-style: italic">R</span> gene level. The revelation of the molecular identities of these <span style="font-style: italic">INT</span> genes will generate insights on the interplay between defense and temperature responses.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Differentiation of an unusual post&#45;transcriptional processing mediated by short, direct repeats in response to stress conditions between monocots and dicots</div>

<div>

	<span style="text-decoration: underline" >
	Yongsheng Liu</span>
	<sup>1</sup>, 

	<span  >
	Xiangli Niu</span>
	<sup>1</sup>, 

	<span  >
	Wenjing Zheng</span>
	<sup>1</sup>, 

	<span  >
	Bao-Rong Lu</span>
	<sup>2</sup>, 

	<span  >
	Di Luo</span>
	<sup>1</sup>, 

	<span  >
	Lijuan Chang</span>
	<sup>1</sup>, 

	<span  >
	Yuguo Wang</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>College of Life Science, Sichuan University, 

	<sup>2</sup>Institute of Biodiversity Science, Fudan University

</div>

<div>Various abilities to synthesize and accumulate glycine betaine &#40;GB&#41; are crucial for angiosperms to develop salt and drought tolerances. In higher plants, GB is synthesized by a two&#45;step oxidation of choline via an intermediate form of betaine aldehyde, and catalyzed by choline monooxygenase &#40;CMO&#41; and betaine aldehyde dehydrogenase &#40;BADH&#41;. In this study, numerous truncated and&#47;or recombinant transcripts of two BADH homologues resulted from an unusual post&#45;transcriptional processing were detected in rice &#40;Oryza sativa&#41; and other cereal crops, including maize &#40;Zea mays&#41;, wheat &#40;Triticum aestivum&#41; and barley &#40;Hordeum vulgare&#41;. The observed events took place at the 5&#8217; exonic region, and led to the insertion of exogenous gene sequences and a variety of deletions that resulted in the removal of translation initiation codon, loss of functional domain and frame&#45;shifts with premature termination by introducing stop codon. By contrast, the BADH transcripts from dicotyledonous species such as spinach &#40;Spinacia oleracea&#41;, Arabidopsis &#40;Arabidopsis thaliana&#41;, and tomato &#40;Solanum lycopersicum&#41;, had correctly processed mRNA. This suggests the differentiation of post&#45;transcriptional processing in BADH genes potentially contributes to the variation of GB synthesizing capacities among various plant species. In addition, comprehensive sequence analyses demonstrated that extensive sequence similarities &#40;named as short&#45;direct repeats, SDRs&#41; are of paired presence surrounding the junctions of both the deletion and&#47;or insertion sites in the unusual BADH transcripts. The sites selection for the deletion&#47;insertion was altered in response to the stress conditions. This indicates that the sequence elements of SDRs are probably required for the recognition of the deletion&#47;insertion sites. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Plasticity of root epidermal cell fate in response to nutrient starvation</div>

<div>

	<span style="text-decoration: underline" >
	Thomas Yang</span>
	<sup>1</sup>, 

	<span  >
	Natasha Savage</span>
	<sup>2</sup>, 

	<span  >
	Wolfgang Schmidt</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>IPMB, Academia Sinica, Taipei, Taiwan, 

	<sup>2</sup>Computational Systems Biology group, Sheffield University, Sheffield, UK

</div>

<div>Differentiation of root epidermal cells is controlled by a network of interacting factors, the core of which is a complex of MYB and bHLH transcription factors associated with a WD40 repeat protein. The fate of rhizodermic cells in the ageing root is not irreversibly fixed, but is reached by continuous integration of different signals. Both the stage at which the environmental signals are perceived and the mechanisms underlying the re&#45;programming of epidermal cells have yet to be understood. In the present study, we investigated the effect of P and Fe deficiency on the expression levels of a number of cell specification genes as well as genes known to be up&#45;regulated during RH formation. P&#45;deficient plants produce longer and additional root hairs that are located both in the normal position and in positions normally occupied by non&#45;hair cells. This was not observed in Fe&#45;deficient plants, in which the surface area was increased by the formation of branched hairs. Our results indicate that the respective deficiencies also produced a unique pattern at the gene expression level. Contrary to expectation, the expression of genes determining the non&#45;hair cell fate such as <span style="font-style: italic">WER</span> and <span style="font-style: italic">GL2</span> was not affected by both deficiencies. However, some cell specification genes, namely <span style="font-style: italic">SCM</span> and <span style="font-style: italic">CPC</span>, and <span style="font-style: italic">ETC1</span>, were differentially up regulated by either &#8211;Fe or &#45;P. In addition, some root hair associated genes were found to be induced by both deficiencies in correlation with increase in root hair number. To further investigate the plasticity in root hair development, we have developed a reaction&#45;diffusion model that accounts for previous observations on root epidermal patterning in mature roots in response to P deprivation. The model assumes an additional patterning mechanism that by&#45;passes the theorized cell fate regulatory circuit in the root epidermis. The presumed second mechanism is suggested to allow for plasticity in epidermal cell patterning by changing the cell fate of epidermal cells regardless of their position in a <span style="font-style: italic">de novo</span> patterning mode.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Alternative oxidase plays a central role in defining the levels of expression of stress induced genes in Arabidopsis and is regulated by stress signalling pathways</div>

<div>

	<span style="text-decoration: underline" >
	Lois Ho</span>
	<sup>1</sup>, 

	<span  >
	Estelle Giraud</span>
	<sup>1</sup>, 

	<span  >
	Ryan Lister</span>
	<sup>1</sup>, 

	<span  >
	Rachel Clifton</span>
	<sup>1</sup>, 

	<span  >
	Katharine A Howell</span>
	<sup>1</sup>, 

	<span  >
	Angela Glen</span>
	<sup>1</sup>, 

	<span  >
	David Thirkettle-Watts</span>
	<sup>1</sup>, 

	<span  >
	James Whelan</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ARC Centre of Excellence in Plant Energy Biology, University of Western Australia, Perth, Western Australia, Australia

</div>

<div>The alternative oxidase &#40;Aox&#41; is encoded by small multigene families in plants. In Arabidopsis &#40;Arabidopsis thaliana&#41;, the five members of this family have been shown to be differentially expressed during development and in response to stress. Functional analysis of the Arabidopsis Aox1a &#40;AtAox1a&#41; promoter indicated that regulation was responsive to stress treatments with hydrogen peroxide and, to a lesser extent, rotenone. Functional analysis of the cis&#45;sequence elements driving the expression of AtAox1a identified ten elements responsive to hydrogen peroxide treatment and&#47;or rotenone treatment. Analysis showed that six of these elements were also active in the alternative NAD&#40;P&#41;H dehydrogenase gene, NDB2, previously shown to be co&#45;expressed with AtAox1a &#40;Clifton et al 2005, Clifton et al 2006&#41;, indicating that co&#45;expression patterns observed were due to co&#45;regulation. Microarray analysis of two independent lines lacking in a functional AtAox1a gene revealed that all the transcripts induced and repressed after one hour of treatment with hydrogen peroxide were already similarly changed in the knock&#45;out lines. This reveals that AtAox1a plays a role in defining the basal level of expression of these stress induced genes under unstimulated conditions. The induction of AtAox1a and NDB2 was also compromised in the defence signalling mutants, npr1 and eds4, and, to a lesser extent, pad4. An analysis of the Arabidopsis genome for genes with six or more of the cis&#45;elements identified as functional in AtAox1a, revealed a set of genes which were involved in stress perception, signal transduction, stress activation of gene expression and other stress induced transcripts. Together, these results indicate that AtAox1a is co&#45;regulated with defence and&#47;or stress induced genes and that its activity feeds back to define the expression of stress induced genes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">EIN3&#47;EIL1 NEGATIVELY REGULATE BASAL&#13;<br/>DEFENSES THROUGH INHIBITING SA&#13;<br/>PATHWAY IN ARABIDOPSIS&#13;<br/></div>

<div>

	<span  >
	Huamin Chen</span>
	<sup>1</sup>, 

	<span  >
	Satya Chintamanani</span>
	<sup>2</sup>, 

	<span  >
	Huiqiong Lin</span>
	<sup>1</sup>, 

	<span  >
	Jie Zhang</span>
	<sup>1</sup>, 

	<span  >
	Xiaoyan Tang</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Jian-Min Zhou</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute of Biological Sciences, Beijing, 

	<sup>2</sup>Department of Plant Pathology, Kansas State University

</div>

<div>Innate immune responses induced by Pathogen&#45;Associated Molecular Patterns &#40;PAMPs&#41; form a critical line of defense against potential pathogens. However, the regulatory mechanisms involved are poorly understood. Here we show that Arabidopsis transcription factors EIN3 and EIL1, known components of the ethylene signaling pathway, negatively regulate PAMP&#45;induced defenses such as callose deposition induced by flg22. Consistent with this, ein3&#47;eil1 double mutant is more resistant to both virulent and nonpathogenic Pseudomonas syringae bacteria. Conversely, the ebf1&#47;ebf2 double mutant that over&#45;accumulates the EIN3 protein shows increased disease susceptibility to these bacteria. Surprisingly, a novel ein3 mutant, ein3&#45;5, showed a loss of ethylene&#45;induced triple response but enhanced inhibition of the PAMP&#45;induced cell wall defenses and increased disease susceptibility, uncoupling the ethylene&#45;regulated development and inhibition of the PAMP&#45;signaling pathway. Microarray and northern analyses showed that PR1 several genes involved in SA synthesis, such as isochorismate synthase 1 &#40;ICS1&#41;, chorismate synthase, phenylalanine ammonia&#45;lyase 1 &#40;PAL1&#41;, phenylalanine ammonia&#45;lyase 2 &#40;PAL2&#41;, are constitutively expressed in the ein3&#47;eil1 double mutant. Consistent with a role of EIN3 and EIL1 in SA regulation, the ein3&#47;eil1&#47;sid2 triple mutant shows susceptibility to the bacterium. Together these results indicate that EIN3 and EIL1 negatively regulate basal defenses by negatively regulating the SA pathway.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Isolation and characterization of oxalic acid insensitive mutant of Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Xiaoting Chen</span>
	<sup>1</sup>, 

	<span  >
	Fangfang Chen</span>
	<sup>2</sup>, 

	<span  >
	Lin Zheng</span>
	<sup>1</sup>, 

	<span  >
	Guodong Lu</span>
	<sup>2</sup>, 

	<span  >
	Zonghua Wang*</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>College of Life Science,Fujian Agriculture and Forestry University,Fujian ,China, 

	<sup>2</sup>College of Plant Protection,Fujian Agriculture and Forestry University,Fujian ,China, 

	<sup>3</sup>Overseas College,Fujian Agriculture and Forestry University,Fujian ,China

</div>

<div>A wide range of fungi can secrete oxalic acid&#40;OA&#41;, which facilitates the growth and reproduction of fungi during the interaction between pathogen and host , so OA is considered as one of the ubiquitous pathogenic factors. We screened OA insensitive mutants on modified MS culture medium from Arabidopsis mutant library based on the XVE chemical inducible gene expressing system constructed by Professor Jiruo Zuo&#40;about 6000lines&#41;. We obtained five mutants which can survive on MS culture medium containing 1.2 mmol&#47;L OA . Based on the sequences we got from TAIL&#45;PCR , we blasted the sequences in TAIR and  knew that the T&#45;DNA of mutant D33 was inserted between At2g39720 &#40;Zinc finger &#41; and At2g39730&#40;Rubisco activase&#41;,and the T&#45;DNA junctions of the other four mutants were the same ,all inserted in the same place of the first intron of At5g10450 &#40;14&#45;3&#45;3 protein GF14 lambda&#41;.We try to over expression At5g10440 ,At5g10460 ,At2g39720 and At2g39730 ,so far we obtained some trans&#45;genic plants and will identify their OA resistance and Sclerotinia sclerotiorum&#8217;s resistance. Resistance to Sclerotinia sclerotiorum was also tested on OA insensitive mutant by inoculation on detached leaves and found the mutants possess the enhanced resistance. These candidate OA insensitive mutants are under further molecular characterization.&#13;<br/>&#13;<br/>Key words : Arabidopsis,oxalic acid insensitive mutant , Sclerotinia sclerotiorum ,chemical inducible gene expressing system&#13;<br/>&#13;<br/>Acknowledgement:This work was supported by the National Natural Science Foundation of China &#40;30671347&#41;&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">An auxlin&#45;like protein of Arabidopsis controlled disease tolerance to Sclerotinia sclerotiorum</div>

<div>

	<span style="text-decoration: underline" >
	Airong Wang</span>
	<sup>1</sup>, 

	<span  >
	Deshu Lin</span>
	<sup>1</sup>, 

	<span  >
	Chunhua Zhang</span>
	<sup>1</sup>, 

	<span  >
	Wenwei Lin</span>
	<sup>1</sup>, 

	<span  >
	Guodong Lu</span>
	<sup>1</sup>, 

	<span  >
	Jie Zhou</span>
	<sup>1</sup>, 

	<span  >
	Zonghua Wang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Fujian Agriculture and Forestry University, Fuzhou, Fujian, 350002?China

</div>

<div>We obtained an activation&#45;tagged mutant showing enhanced disease tolerance to Sclerotinia sclerotiorum. Preliminary genetic analysis showed that this phenotype was caused by overexpression of an auxlin&#45;like protein. The bioinformatics analysis showed that this auxlin&#45;like protein had a J&#45;domain, resembling clathrin&#45;uncoating factor auxilin at its C terminus &#40;Schmid et al., Annu Rev Biochem, 1997, 66: 511&#8211;548&#41;. It also contained an Asp&#45;Pro&#45;Phe &#40;DPF&#41; adaptor protein&#45;binding motif and FxDxF motif at the N terminus, which is known to bind to the appendage domain of &#945; or &#946; subunits of an adaptor protein complex AP&#45;2 &#40;Brett et al., Structure, 2002, 10: 797&#8211;809&#41;. The bioinformatics data suggested that it may have ATP&#45;binding activity, serine&#47;thrine kinase or phosphatase activity and it might regulate cell cycle or phosphorylation or dephosphorylation of proteins. Subcellular localization by instant expression of GFP fusion protein on onion epidermis showed that it was localized in nuclei, which validated the bioinformatics deduction. Its&#8217; loss&#45;of&#45;function mutant showed more susceptible to S. sclerotiorum than wild type and grew slower on MS medium than wild type. We presumed that the Clathrin&#45;Coated Vesicles may participate in the defense pathway to S. sclerotiorum. Further genetic analysis is to confirm the linkage between the molecular or biological functions of the protein and the predicted phenotype.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Networks and Mechanisms of Brassinosteroid Regulated Gene Expression in Arabidopsis</div>

<div>

	<span  >
	Michelle Guo</span>
	<sup>1</sup>, 

	<span  >
	Lei Li</span>
	<sup>1</sup>, 

	<span  >
	Xiaofei Yu</span>
	<sup>1</sup>, 

	<span  >
	Addie Hall</span>
	<sup>1</sup>, 

	<span  >
	Tadao Asami</span>
	<sup>2</sup>, 

	<span  >
	Shigeo Yoshida</span>
	<sup>2</sup>, 

	<span  >
	Joanne Chory</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Yanhai Yin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Genetics, Development and Cell Biology, Plant Science Institute, Iowa State University, Ames, IA 50011, 

	<sup>2</sup>Plant Functions Lab, RIKEN, Wako-shi, Saitama 351-098, Japan, 

	<sup>3</sup>Howard Hughes Medical Institute and Plant Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, CA 92037

</div>

<div>Plant steroid hormone brassinosteroids &#40;BRs&#41; play important roles throughout growth and development, regulating diverse processes such as cell elongation, xylem differentiation, ethylene biosynthesis and stress responses. Recent molecular genetic studies established that BRs signal through membrane receptor BRI1 to regulate the protein levels and&#47;or activities of BES1&#47;BZR1 family transcription factors, which function to mediate many of the hormone responses.  Little is know about the genetic networks and molecular mechanisms by which BES1 and its homologs regulate target gene expression. To fill these gaps, multidisciplinary approaches have been used to identify BES1 partners, regulators and targets. First, Partner of BES1 &#40;POB1, a putative histone modification enzyme&#41; has been identified by a yeast genetic screen and has been found to interact with BES1 both in vitro and in vivo. Both loss&#45;of&#45;function and gain&#45;of&#45;function mutants of POB1 gene display BR response phenotypes including changed sensitivities to BR biosynthesis inhibitor brassinazole and altered expression of a subset of BR&#45;regulated genes. Second, several BES1 target genes encoding transcription factors and other signaling molecules have been identified by microarray and Chromatin Immunoprecipitation &#40;ChIP&#41; experiments. T&#45;DNA knock&#45;out mutants have been identified and the characterization of the mutant phenotypes will be presented. Finally, we have identified several mutants that have reduced BES1&#45;regulated gene expression and may identify proteins that function with BES1 to regulate BR target gene expression.  Our results suggest that BES1 interacts with other transcriptional partners to modify chromatin structure and therefore regulate the expression of different target genes for various BR responses. The research is supported by NSF &#40;IOS 0546503&#41; and a faculty start&#45;up fund from Iowa State University.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Transcription factors as potential tools to alter metabolic pathways in seeds</div>

<div>

	<span style="text-decoration: underline" >
	Armin Schlereth</span>
	<sup>1</sup>, 

	<span  >
	Martin Bringmann</span>
	<sup>1</sup>, 

	<span  >
	Alexander Erban</span>
	<sup>1</sup>, 

	<span  >
	Stefanie Wienkoop</span>
	<sup>1</sup>, 

	<span  >
	Klementina Kakar</span>
	<sup>1</sup>, 

	<span  >
	Mark Stitt</span>
	<sup>1</sup>, 

	<span  >
	Michael Udvardi</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Max-Planck-Institute of Molecular Plant Physiology, D-14476 Potsdam-Golm, Germany, 

	<sup>2</sup>The Noble Foundation, Ardmore 73401 (Oklahoma), USA

</div>

<div>Transcription factors &#40;TFs&#41; exhibit key regulatory functions and are crucial for the regulation of many developmental and physiological processes by influencing gene expression. Furthermore, during seed development and maturation TFs display an essential role in controlling the quality and quantity of the storage compound composition. Since seeds are an excellent and in many human populations the major source for nutrients as well as for animals food it could be beneficial to identify the determinants which control or affect the biosynthesis of the storage compounds.&#13;<br/>In Arabidopsis thaliana almost 1900 TFs are annotated. We designed primer pairs for all these TFs genes &#40;Czechowski et al., 2004&#41; and analysed their expression in comparison to the expression of marker genes either encoding for storage proteins or being relevant concering the synthesis of storage lipids during seed development using the quantitative Real&#45;time RT&#45;PCR. This allowed the identification of seed specific TFs displaying a developmental expression comparable to that of the marker genes. Therefore, these TFs could have the potential acting as key regulators controlling the storage compound synthesis. For the functional characterization 17 different T&#45;DNA insertion&#45; as well as RNAi&#45; lines of putative key regulators have been selected and will be analysed.&#13;<br/>To supplement the forward and reverse genetic approaches metabolite, lipid and protein profiling of Arabidopsis wild type seeds from various developmental stages from pollination till mature seeds was performed. These uncovered quantitative and qualitative changes in storage compound composition with a high temporal resolution and allowed a seasonal correlation of the expression of TFs&#45; and marker genes with the synthesis of the storage compounds. Moreover, these results can be used as a basis that allows the comparison from WT and transgenic seeds with respect to their differences in storage compound composition.&#13;<br/>Further studies will transfer the obtained results to grain legume seeds.&#13;<br/>&#13;<br/>Czechowski&#40;2004&#41;, Plant Journal: 38, 366&#45;79</div>

</div>





<div class="abs">

<div style="font-weight:bold">Methylation of Histone H3 Lysine 4 Residues in Response to Drought Stress in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Baoqiang Cao</span>
	<sup>1</sup>, 

	<span  >
	Karin van Dijk</span>
	<sup>1</sup>, 

	<span  >
	Yuannan Xia</span>
	<sup>1</sup>, 

	<span  >
	Jean-Jack M. Riethoven</span>
	<sup>1</sup>, 

	<span  >
	Eric P. Moss</span>
	<sup>1</sup>, 

	<span  >
	Jingyi Yang</span>
	<sup>1</sup>, 

	<span  >
	Michael E. Fromm</span>
	<sup>1</sup>, 

	<span  >
	Istvan Ladunga</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Center for Biotechnology, University of Nebraska-Lincoln, Lincoln, NE, USA, 

	<sup>2</sup>Center for Biotechnology and Department of Statistics, University of Nebraska-Lincoln, Lincoln, NE, USA

</div>

<div>Histone modifications are important epigenetic markers for the transcriptional status of genes. In this study, we mapped histone modifications at whole&#45;genome level in control and drought conditions and compared histone remodeling events with changes in gene expression. Histone H3 lysine 4 &#40;H3K4&#41; methylation changes were monitored by immunoprecipiting chromatin in vivo with modification&#45;insensitive H3 antibodies as well as with antibodies specific to mono&#45;, di&#45;, and tri&#45;methylated histone H3K4 residues. Then we used Affymetrix whole&#45;genome tiling arrays to map nucleosome occupancy and histone methylation using over 3 million genomic probes. We found that H3K4 di&#45; and tri&#45;methylation showed significant positive correlation with gene expression &#40;p&#60;&#60;0.05, Spearman test&#41;. We also investigated the changes in response to drought condition in various genetic regions of each gene, such as 5' untranslated region &#40;UTR&#41;, 3' UTR, coding region, up&#45; and downstream 500 base pair regions. We found that 5&#8217; UTR of upregulated genes exhibit a significantly higher level of H3K4 trimethylation than in the 3&#8217;UTRs of unchanged or downregulated genes. Similar results were obtained for dimethylation in coding regions. This study presents a whole genome view of H3K4 methylation and its changes in response to drought stress in Arabidopsis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Differences in chemical composition of &#34;Arabidopsis thaliana&#34; seeds and implications for plant&#45;herbivore interactions</div>

<div>

	<span style="text-decoration: underline" >
	Asghar Mosleh Arani</span>
	<sup>1</sup>, 

	<span  >
	Tom J de Jong</span>
	<sup>2</sup>, 

	<span  >
	H.K. Kim</span>
	<sup>2</sup>, 

	<span  >
	Nicole M. van dam</span>
	<sup>2</sup>, 

	<span  >
	Yong H. Choi</span>
	<sup>2</sup>, 

	<span  >
	Rob Verpoorte</span>
	<sup>2</sup>, 

	<span  >
	Eddy van der Meijden</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Yazd university, Yazd, Iran, 

	<sup>2</sup>Leiden university, Leiden, The Netherlands

</div>

<div>When plants of &#34;Arabidopsis thaliana&#34; that originated from dune or inland populations were reciprocally transplanted to dune or inland habitats, they were affected differently by the specialist weevils, &#34;Ceutorhynchus atomus&#34; and &#34;Ceutorhynchus contractus&#34; &#40;Curculinoidae&#41; which feed on flowers and fruits. &#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We grew plants in the growth room for one generation and performed a metabolomic analysis on the new seeds produced using NMR spectroscopy and multivariate data analysis. Major differences in chemical composition were found in the water&#45;methanol fractions: more thioglucosinolates and sucrose in dune and more sinapoylmalate in inland populations.&#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Quantitative analysis of glucosinolates was made with an HPLC analysis, using the same seed batches collected from three sites &#40;inland, dune and growth room&#41;. Glucosinolate composition and concentration differed between individual plants, populations and sites. &#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Fruit damage by adult weevils and their larvae was not significantly correlated with field concentrations of individual glucosinolates, glucosinolate groups and total concentration of glucosinolates in seeds.&#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We conclude that, given the range of glucosinolate concentrations in dune and inland plants of &#34;Arabidopsis thaliana&#34;, other factors must be involved in defense against herbivory by the specialist weevils.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A Study of Arabidopsis Ethylene Signal Transduction Identifies a Transcription Factor and an Ethylene Response Mutant</div>

<div>

	<span style="text-decoration: underline" >
	Qin Wang</span>
	<sup>1</sup>, 

	<span  >
	Xin Zhou</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences

</div>

<div>Genetic screen for mutants defective in the seedling triple response has identified genes involved in the ethylene signal transduction in Arabidopsis. Other approaches have been used to isolate more mutants that cannot be identified in the seedling triple&#45;response assay. In an attempt to find more components participating in the ethylene signal transduction pathway, both forward and reverse genetics approaches were used in our study. Here we report the isolation of a transcription factor, ATR2, whose overexpression confers ethylene insensitivity and completely restores the growth of the ctr1&#45;1 mutant. In a genetic screen, we identified a mutant, ain, which exhibits a mild constitutive ethylene response phenotype. Ethylene treatment caused a severer triple&#45;response phenotype in ain than in wild type and the silver treatment did not block the ethylene treatment in ain, suggesting AIN a component downstream of the receptors. In adult stage, the ain mutant was small and dwarf. More genetic analyses for the atr2 and ain mutants are in progress.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis Calcium Sensor CBL3 Inhibits the 5&#8242;&#45;Methylthio&#13;<br/>Adenosine Nucleosidase Activity of C38 in a Calcium&#45;Dependent Manner&#13;<br/></div>

<div>

	<span  >
	Seung-Ick Oh</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Kyung-Nam Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biology, Sejong University, Seoul 143-747, Korea

</div>

<div>Abiotic stresses such as cold, drought, and high salt induce changes in cytosolic calcium signatures of plant cells. Calcineurine B&#45;like &#40;CBL&#41; proteins are believed to sense and transmit the calcium signals to a group of serine&#47;threonine protein kinases called CIPKs, thereby activating the kinase activity. In this study, we have identified a new target of the CBL3 protein, designated C38, using a yeast&#45;two hybrid library screening. C38 interacted only with CBL3 but not with other CBL family members such as CBL1 and CBL4. Deletion analysis revealed that the CBL3 region spanning from 109 to 199&#45;amino acids is required and sufficient for the interaction with C38. In the case of C38, it appeared that almost entire sequence is necessary for the interaction with CBL3. Pull&#45;down assays and enzyme activity assays demonstrated that CBL3 associates with C38 only in the presence of calcium, and thereby significantly inhibits the 5&#8242;&#45;methylthioadenosine nucleosidase &#40;MTAN, EC 3.2.2.16&#41; activity of C38. Although no striking phenotypic differences were observed between a wild type and a 'c38' mutant in general, the 'c38' mutant was a little more sensitive to high osmotic or salt stresses in terms of germination rate.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification of a rice QTL regulating grain width and weight</div>

<div>

	<span style="text-decoration: underline" >
	Wei Huang</span>
	<sup>1</sup>, 

	<span  >
	Xian-Jun Song</span>
	<sup>1</sup>, 

	<span  >
	Jun-Xian Shan</span>
	<sup>1</sup>, 

	<span  >
	Min Shi</span>
	<sup>1</sup>, 

	<span  >
	Mei-Zhen Zhu</span>
	<sup>1</sup>, 

	<span  >
	Hong-Xuan  Lin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Key Laboratory of Plant Molecular Genetics, Shanghai Institute of Plant Physiology and Ecology, Shanghai Institute for Biological Sciences, The Chinese Academy of Sciences, 300 Fenglin Road, Shanghai 200032, China.

</div>

<div>We used map&#45;based cloning strategy to identify a QTL, GW2 as the gene encodes a RING&#45;type protein with E3 ubiquitin ligase activity, responsible for rice grain width and weight. GW2 promoter&#8211;GFP was expressed constitutively in various tissues and organs. Subcellular localization of GFP&#45;GW2 fusion driven by the CaMV 35S showed that GW2 localized to the cytoplasm. We observed a increase in grain width  and panicle number per plant in NIL&#40;GW2&#41; carrying GW2 allele of the larger grain size variety &#40;WY3&#41; compared with FAZ1 &#40;the recurrent parent, smaller grain size variety&#41;. Histological analyses of spikelet hulls of FAZ1 and NIL&#40;GW2&#41; demonstrated that the increased width of the NIL&#40;GW2&#41; spikelet hull resulted mainly from an increase in cell number, but not in cell size. On the contrary, the increase in NIL&#40;GW2&#41; endosperm size resulted mainly from cell expansion, not from an increase in cell number. We observed the grain milk filling rate in NIL&#40;GW2&#41; and FAZ1 and found that the rate of accumulation of dry matter was faster in NIL&#40;GW2&#41;. These data suggest that larger spikelet hull allows greater endosperm growth and provides a greater area of contact for endosperm with the seed coat leading to an accelerated milk filling rate .We examined grain quality traits in NIL&#40;GW2&#41; and FAZ1.,Results showed that the WY3 GW2 allele could increase grain size and yield with little influence on appearance and no reduction in cooking or eating quality. Our findings   will facilitate breeding efforts to improve grain yield in staple crops.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Role of Two Ubiquitin&#45;Like Proteins in <span style="font-style: italic">Arabidopsis</span> Peroxisome Biogenesis and Function</div>

<div>

	<span  >
	Navneet Kau</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Jilian Fan</span>
	<sup>1</sup>, 

	<span  >
	Sheng Quan</span>
	<sup>1</sup>, 

	<span  >
	Jianping Hu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>MSU DOE-Plant Research Laboratory, Michigan State University, East Lansing, MI 48824. USA

</div>

<div>Plant peroxisomes are dynamic organelles that play a vital role in plant growth and development. Peroxins are an eclectic set of proteins necessary for peroxisome biogenesis and import of matrix proteins from the cytosol. PEX2 is a RING family Peroxin which has been shown to be essential for viability and to be involved in the photomorphogenic response. Yeast two hybrid approach was employed to identify PEX2 interacting factors and resulted in the retrieval of an Ubiquitin&#45;Like protein, named AtPXI1 &#40;<span style="text-decoration: underline">P</span>E<span style="text-decoration: underline">X</span>2 <span style="text-decoration: underline">I</span>nteractor protein&#41;. <span style="font-style: italic">Arabidopsis</span> was found to have a highly conserved paralog of this gene, <span style="font-style: italic">AtPXI2</span>.<span style="font-style: italic">AtPXI</span> genes are present in tandem and oriented in head to tail fashion on the chromosome. The AtPXI proteins constitute an N&#45;terminal Ubiquitin&#45;Like &#40;UBL&#41; domain, a C&#45;terminal Ubiquitin Associated domain &#40;UBA&#41; and four chaperonin binding sites in the medial region. Both AtPXIs interact with the RING domain of PEX2. Deletion constructs are being made to delineate the AtPXI domain specific to this interaction. Using fluorescence microscopy we demonstrated that GFP&#45;fusions of AtPXI1 and AtPXI2 localize to the peroxisome. T&#45;DNA insertion lines of AtPXIs were identified and are being investigated to determine the functional role of the AtPXIs. RNAi lines are concurrently being analyzed to further elucidate the functions of the AtPXIs in <span style="font-style: italic">Arabidopsis</span>. UBL&#45;UBA proteins are believed to function as shuttle factors, relaying ubiquitinated proteins to the proteasome. Drawing analogy from the ERAD &#40;Endoplasmic Reticulum Associated Degradation&#41; model, the presence of RING proteins &#40;putative E3 ligases&#41;, E2, AAA&#45;ATPases and UBL&#45;UBA proteins in the peroxisomes prompts us to speculate that peroxisomes also have a ubiquitin&#45;proteasome type of proteolytic system associated with them, which regulates peroxisome biogenesis and function by targeting selective peroxisome proteins for degradation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">DELLA proteins regulate trichome initiation and branching through GIS and GL1 in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Yinbo Gan</span>
	<sup>1</sup>, 

	<span  >
	Hao Yu</span>
	<sup>2</sup>, 

	<span  >
	Jinrong Peng</span>
	<sup>3</sup>, 

	<span  >
	Pierre Broun</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CNAP, Department of Biology, University of York, York YO10 5YW, United Kingdom, 

	<sup>2</sup>Department of Biological Sciences and Temasek Life Sciences Laboratory, National University of Singapore, Singapore 117543, Singapore, 

	<sup>3</sup>Functional Genomics Lab, Institute of Molecular and Cell Biology, 61 Biopolis Drive, Protesos, 138673, Singapore

</div>

<div>Gibberellins &#40;GA&#41; are known to influence vegetative and reproductive phase change in Arabidopsis, as well as the development of trichomes, which are faithful epidermal markers of shoot maturation.  Downstream of GA production, hormone signaling integrators REPRESSOR OF &#47;ga1&#45;3&#47; &#40;RGA&#41; and GIBBERELLIC ACID INSENSITIVE &#40;GAI&#41; play a central role in the control of vegetative and reproductive phase change. <br/><br/>In this study, we have probed the relative roles of RGA, GAI, RGA&#45;LIKE 1 &#40;RGL1&#41; and RGL2 in the control of epidermal differentiation and shoot maturation and investigated molecular mechanisms through which they regulate trichome production.  We found that RGL1 and RGL2 act synergistically with RGA and GAI in the repression of vegetative and reproductive phase change in Arabidopsis.  RGL1 and RGL2 also play a significant role alongside RGA and GAI as negative regulators of trichome initiation.  In the absence of GA signaling, de&#45;repression of initiation on the inflorescence through loss of DELLA function requires GLABROUS INFLORESCENCE STEMS.  When DELLA function is restored, genes encoding activators of trichome initiation, including GIS and GLABROUS1, are rapidly repressed, but this appears to be through an indirect rather than a direct molecular mechanism.  In contrast to its relatively minor role in initiation, GAI is a major repressor of trichome branching, a role in which it is antagonized by RGL1 and RGL2. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional Analysis of A&#45;type ARRs in Shoot Apical Meristem Regulation</div>

<div>

	<span style="text-decoration: underline" >
	Zhong Zhao</span>
	<sup>1</sup>, 

	<span  >
	Stig Andersen</span>
	<sup>1</sup>, 

	<span  >
	Monika Demar</span>
	<sup>1</sup>, 

	<span  >
	Jan Lohmann</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max-Planck-Institute for Developmental Biology

</div>

<div>It has been known for decades that some phytohormones, such as cytokinin and auxin, play an essential role in meristem regulation, however, the underlying molecular mechanisms remained largely elusive. In <span style="font-style: italic">Arabidopsis thaliana</span>, stem cell fate in the shoot apical meristem is controlled genetically by a negative feed back loop of <span style="font-style: italic">CLAVATA3 &#40;CLV3&#41;</span> and the homeodomain transcription factor WUSCHEL &#40;WUS&#41;. We have shown previously that WUSCHEL, which positively regulates stem cell fate, represses the transcription of several A&#45;type <span style="font-style: italic">ARABIDOPSIS RESPONSE REGULATOR</span> genes &#40;<span style="font-style: italic">ARR5</span>, <span style="font-style: italic">ARR6</span>, <span style="font-style: italic">ARR7</span> and <span style="font-style: italic">ARR15</span>&#41;, in the meristem. Since A&#45;type <span style="font-style: italic">ARR</span> genes act in the negative feed back loop of cytokinin signaling, this finding provided the first mechanistic link between hormone signaling and plant stem cell regulation. <br/>However, we did not find any visible phenotype in <span style="font-style: italic">ARR5</span>, <span style="font-style: italic">ARR6</span> and <span style="font-style: italic">ARR7</span> single mutants, while strong alleles of <span style="font-style: italic">ARR15</span> caused female gametophytic lethality. To circumvent the lethality problem, we constructed conditional alleles by inducible artificial microRNAs against <span style="font-style: italic">ARR15</span> to further explore its functions in the shoot apical meristem. Our results show that down&#45;regulation of <span style="font-style: italic">ARR15</span> or expression of constitutively active forms of ARR15 cause meristematic defects, including disturbed phyllotaxis. Consistent with the observed phenotypes, RNA <span style="font-style: italic">in situ</span> hybridization showed that <span style="font-style: italic">ARR15</span> is expressed in the shoot apical meristem. Our data indicate that <span style="font-style: italic">ARR15</span> function might be one of the central elements in the regulation of meristem activity.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The SPT transcription factor acts as an activator in Arabidopsis ecotype <span style="font-style: italic">Landsberg erecta</span> &#40;<span style="font-style: italic">Ler</span>&#41; and a repressor in ecotype <span style="font-style: italic">Columbia</span> &#40;<span style="font-style: italic">Col&#45;0</span>&#41; to control seed germination</div>

<div>

	<span style="text-decoration: underline" >
	Yinbo Gan</span>
	<sup>1</sup>, 

	<span  >
	Eve-Marie Josse</span>
	<sup>2</sup>, 

	<span  >
	Steven Penfield</span>
	<sup>1</sup>, 

	<span  >
	Alison D. Gilday</span>
	<sup>1</sup>, 

	<span  >
	Karen J. Halliday</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>CNAP, Department of Biology, University of York, York YO10 5YW, United Kingdom, 

	<sup>2</sup>Institute of Molecular Plant Sciences, University of Edinburgh, Edinburgh, EH9 3JR, United Kingdom

</div>

<div>We previously described the role of the basic helix&#45;loop&#45;helix transcription factor SPATULA &#40;SPT&#41; in the control of seed germination by light and temperature &#40;Penfield et al., 2005, Current Biology, 15: 1998&#45;2006&#41;. In the current work we show that the role of SPT in seed germination is dependent on the ecotype. In <span style="font-style: italic">Col&#45;0</span>, SPT acts as a repressor of seed germination while in <span style="font-style: italic">Ler</span> it acts as an activator. The <span style="font-style: italic">spt&#45;11</span> mutation in <span style="font-style: italic">Col&#45;0</span> resulted in much higher germination rates of fresh seeds compared to the wild type. Light treatment alone results in 100&#37; germination in <span style="font-style: italic">spt&#45;11</span> whereas <span style="font-style: italic">Col&#45;0</span> requires both light and cold treatments to reach 100&#37; germination. Constitutive overexpression of SPT in <span style="font-style: italic">Col&#45;0</span> results in significantly lower germination rates following either light or cold treatments compared to the wild type. In contrast, the <span style="font-style: italic">spt&#45;1</span> and <span style="font-style: italic">spt&#45;2</span> mutants in the <span style="font-style: italic">Ler</span> background exhibit lower germination rates than the wild type following either cold or light treatments, while the overexpressing line results in higher germination rates under both treatments. In agreement with this overexpression phenotype we have found that the <span style="font-style: italic">spt&#45;10 Ler</span> allele previously reported to be a knockout mutant actually overexpresses <span style="font-style: italic">SPT</span> transcript despite the fact that the SPT ORF is disrupted by a transposon insertion &#40;Penfield et al., 2005&#41;. Consistent with these opposite roles for SPT in germination, we have also found that hypocotyl elongation also responds differently to SPT. Overexpression of SPT in <span style="font-style: italic">Col&#45;0</span> results in hypocotyls that are significantly longer than the wild type whereas overexpression in <span style="font-style: italic">Ler</span> does not result in a significant increase compared to wild type. These results lead us to conclude that the underlying mode of action of the SPT transcription factor in <span style="font-style: italic">Ler</span> and <span style="font-style: italic">Col&#45;0</span> is fundamentally different.</div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">Reduced leaflet</span> pathway controls leaf complexity and leaflet polarity in <span style="font-style: italic">Lotus japonicus</span></div>

<div>

	<span style="text-decoration: underline" >
	Jun Yan</span>
	<sup>1</sup>, 

	<span  >
	Xuefei Cai</span>
	<sup>1</sup>, 

	<span  >
	Shusei Sato</span>
	<sup>2</sup>, 

	<span  >
	Satoshi Tabata</span>
	<sup>2</sup>, 

	<span  >
	Jun Yang</span>
	<sup>1</sup>, 

	<span  >
	Xianglin Cao</span>
	<sup>1</sup>, 

	<span  >
	Wei Ma</span>
	<sup>3</sup>, 

	<span  >
	Da Luo</span>
	<sup>1,3</sup>

</div>

<div>

	<sup>1</sup>National Laboratory of Plant Molecular Genetics, Shanghai Institute of Plant Physiology and Ecology, Shanghai Institute for Biological Sciences, Shanghai 200032, China, 

	<sup>2</sup>Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan, 

	<sup>3</sup>School of Life Science and Biotechnology, Shanghai Jiao Tong University, Shanghai 200030, China

</div>

<div>   Leaf forms of seed plants can be simply classified as simple leaf and compound leaf. Compound leaves are normally composed of one petiole and several leaflets, whereas simple leaves have a single lamina. To address the question about the key components in the control of leaf complexity during compound leaf development, we have identified a series of mutants in <span style="font-style: italic">Lotus japonicus</span>: <span style="font-style: italic">reduced leaflet 1</span> &#40;<span style="font-style: italic">rel1</span>&#41;, <span style="font-style: italic">rel3</span>, <span style="font-style: italic">rel4</span> and <span style="font-style: italic">rel5</span>, which display the similar phenotype of reduced leaflet numbers in a compound leaf with defect of abaxial&#45;adaxial &#40;ab&#45;ad&#41; polarity in individual leaflet. Genetic analysis indicates that these four mutants are not allelic to each other, and these genes may act in the same pathway. It is found that the expression patterns of duplicated <span style="font-style: italic">LjPhantastica</span> genes in <span style="font-style: italic">Lotus japonicus</span> are altered in <span style="font-style: italic">rel1</span> and <span style="font-style: italic">rel3</span>, accounting for the abnormalities of ab&#45;ad development in the mutant compound leaves, and suggesting that <span style="font-style: italic">REL</span> may act upstream of <span style="font-style: italic">LjPhan</span> genes. Forward and reverse genetic approaches are conducted to analyze the function of <span style="font-style: italic">REL</span> genes and the regulatory network in regulating compound leaf development in <span style="font-style: italic">Lotus japonicus</span>.</div>

</div>





<div class="abs">

<div style="font-weight:bold">OmDSP1 having novel nuclease activity contributes to resistance against B. cinerea in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Min Kyoung You</span>
	<sup>1</sup>, 

	<span  >
	Hyung Young Shin</span>
	<sup>1</sup>, 

	<span  >
	Seung-Ick Oh</span>
	<sup>1</sup>, 

	<span  >
	Jeong-Kook Kim</span>
	<sup>1</sup>, 

	<span  >
	Jeong Sheop Shin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Life Sciences and Biotechnology, Korea University

</div>

<div>A blast &#40;Magnaporthe grisea&#41;&#45;, brown planthopper &#40;BPH; Nilaparvata lugens&#41;&#45; and wounding&#45;responsive OmDSP1 &#40;Oryza minuta Diverse Stress responsive Protein 1&#41; was isolated and identified as a novel defense related gene from O. minuta. OmDSP1 exists as single copy in its genome and encodes a predicted protein with molecular mass of 39 kDa and a pI of 6.2. OmDSP1 gene was expressed at a basal level under normal condition, and induced by diverse stresses; methyl jasmonic acid &#40;MeJA&#41;, ethephone, salicylic acid &#40;SA&#41;, abscisic acid &#40;ABA&#41;, methyl viologen &#40;MV&#41;, NaCl, wounding, blast and BPH. In vivo targeting of OmDSP1&#45;smGFP with onion cells revealed that OmDSP1 was targeted to nucleus. &#13;<br/>OmDSP1 was identified as a novel nuclease with DUF151 domain which is found just in plant and bacteria. OmDSP1 shared highly conserved N&#45;terminal region, DUF151 and UVR domains with other plant orthologs. The mRNA expression patterns of OmDSP1 plant orthologs, OsDSPs and AtDSPs genes, were induced also by MeJA, ethephone, SA and ABA, suggesting that orthologs also participate in similar cellular responses under diverse stress conditions. &#13;<br/>To investigate the function of OmDSP1, OmDSP1 gene was introduced into Arabidopsis. The 35S:OmDSP1 plants showed increased&#45;resistance against attacks of necrotrophic fungal pathogen, Botrytis cinerea and germination retardation on media containing ABA.  The results of RT&#45;PCR performed with defense&#45;related marker genes &#40;PDF1.2 and Thi2.1&#41; showed to be more sensitively increased by JA or ET in 35S:OmDSP1 plants. AtSAC1b and ABA1 are closely related to callose deposition against pathogen attacks. AtSAC1b and ABA1 are constitutively expressed in 35S:OmDSP1. These responses in 35S:OmDSP1 and atdsp1 plants were suggested to be caused by fine&#45;tuning regulation of crosstalk between JA&#47;ET and ABA signaling pathways. In yeast two hybrid screening, three putative counterparts of OmDSP1 were screened: putative E3 ligase, drought&#45;induced protein Di19 and putative PR10.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Developmental control of Arabidopsis seed oil biosynthesis</div>

<div>

	<span  >
	Hongyun Wang</span>
	<sup>1</sup>, 

	<span  >
	Jinhua Guo</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Yun Lin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Illinois, Urbana, IL, USA

</div>

<div>Arabidopsis transcriptional factors <span style="font-style: italic">LEAFY COTYLEDON1</span> &#40;<span style="font-style: italic">LEC1</span>&#41;, <span style="font-style: italic">LEAFY COTYLEDON2</span> &#40;<span style="font-style: italic">LEC2</span>&#41;, <span style="font-style: italic">FUSCA3</span> &#40;<span style="font-style: italic">FUS3</span>&#41;, <span style="font-style: italic">ABSCISIC ACID3</span> &#40;<span style="font-style: italic">ABI3</span>&#41;, and <span style="font-style: italic">ABSCISIC ACID5</span> &#40;<span style="font-style: italic">ABI5</span>&#41; are known to regulate multiple aspects of seed development. In an attempt to understand the developmental control of storage product accumulation, we observed the expression time course of the five transcripts. The sequential expression of these factors during seed fill suggests differentiation of their normal responsibilities. By extending the expression periods of the two early genes <span style="font-style: italic">LEC1</span> and <span style="font-style: italic">LEC2</span> in transgenic seeds, we demonstrated that the subsequent timing of <span style="font-style: italic">FUS3</span>, <span style="font-style: italic">ABI3</span>, and <span style="font-style: italic">ABI5</span> transcripts depends on <span style="font-style: italic">LEC1</span> and <span style="font-style: italic">LEC2</span>. Because a delayed onset or reduced level of <span style="font-style: italic">FUS3</span> mRNA coincided with reduction of seed oil content in the transgenic seeds, the role of <span style="font-style: italic">FUS3</span> in oil deposition was further examined. Analysis of published seed transcriptome data indicated that <span style="font-style: italic">FUS3</span> transcript increased together with nearly all the plastidial fatty acid biosynthetic transcripts during development. The ability of <span style="font-style: italic">FUS3</span> to rapidly induce fatty acid biosynthetic gene expression was confirmed using transgenic Arabidopsis seedlings expressing a dexamethasone &#40;DEX&#41;&#45;inducible <span style="font-style: italic">FUS3</span> and Arabidopsis mesophyll protoplasts transiently expressing the <span style="font-style: italic">FUS3</span> gene. By accommodating the current evidence, we propose a hierarchical architecture of the transcriptional network in Arabidopsis seeds in which the oil biosynthetic pathway is integrated through the master transcriptional factor <span style="font-style: italic">FUS3</span>.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Analysis of changes in protein expression during Arabidopsis pollen germination</div>

<div>

	<span  >
	Jun-Jie Zou</span>
	<sup>1</sup>, 

	<span  >
	Lian-Fen Song</span>
	<sup>1</sup>, 

	<span  >
	Wen-Zheng Zhang</span>
	<sup>1</sup>, 

	<span  >
	Yi Wang</span>
	<sup>1</sup>, 

	<span  >
	Song-Lin Ruan</span>
	<sup>1</sup>, 

	<span  >
	Ji-Dong Feng</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Wei-Hua Wu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>State Key Laboratory of Plant Physiology and Biochemistry, College of Biological Sciences, National Plant Gene Research Centre (Beijing), China Agricultural University, Beijing 100094, China

</div>

<div>To identify candidate proteins involving regulation of Arabidopsis pollen germination, proteomic approach was applied to analyze the changes in protein expression during the transition from the desiccated mature pollen to the germinated pollen.  By application of 2&#45;DE and silver staining, there were 499 and 494 protein spots resolved from the protein samples extracted from pollen grains and pollen tubes, respectively.  Among these proteins, 303 protein spots were analyzed by MALDI&#45;TOF and 213 protein spots representing 189 proteins were identified.  A comparative analysis revealed that 40 protein spots showed statistically significant and reproducible changes between mature pollen and pollen tubes. Among these 40 protein spots, 40&#37; of the protein spots were specific to developmental stages.  Mass spectrometry analysis allowed the identification of 17 downregulated and 6 upregulated differentially expressed proteins representing 21 diverse proteins.  Functional category analysis showed that these proteins were involved in signaling, cellular structure, transport, defense&#47;stress responses, transcription, metabolism and energy.  The changed pattern of these proteins indicated extensive changes in the energy metabolism of the pollen tube growth, accompanied by the activation of stress response pathway and modifications of the cell wall.  In addition, the detailed methods for collection of Arabidopsis pollen, extraction of proteins from pollen and pollen tubes were described and discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A conserved role of <span style="font-style: italic">SHORT VEGETATIVE PHASE &#40;SVP&#41;</span> in controlling flowering time of <span style="font-style: italic">Brassica</span> plants</div>

<div>

	<span style="text-decoration: underline" >
	Hanna Lee</span>
	<sup>1</sup>, 

	<span  >
	Jeong Hwan Lee</span>
	<sup>1</sup>, 

	<span  >
	Soo Hyun Park</span>
	<sup>1</sup>, 

	<span  >
	Jong Seob Lee</span>
	<sup>2</sup>, 

	<span  >
	Ji Hoon Ahn</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Signaling Network Research Center, School of Life Sciences and Biotechnology, Korea University, Seoul 136-701, Korea, 

	<sup>2</sup>School of Biological Sciences, Seoul National University, Seoul 152-742, Korea

</div>

<div>The control of flowering time in <span style="font-style: italic">Brassica</span> plants is an important approach for improving productivity, as vegetative tissues are not produced after the floral transition in <span style="font-style: italic">Brassica</span> plants. In order to determine the feasibility of modulating flowering time in Chinese cabbage plants, genes homologous to <span style="font-style: italic">Arabidopsis SHORT VEGETATIVE PHASE &#40;AtSVP&#41;</span> were isolated from spring&#45;type and fall&#45;type cultivars of Chinese cabbage plants, and their functions were determined. Their deduced amino acid sequences were 91&#8211;93&#37; identical with that of AtSVP. The expression of <span style="font-style: italic">BcSVP</span> was ubiquitously detected, and was unaffected by vernalization. Constitutive <span style="font-style: italic">BcSVP</span> expression induced late flowering with additional floral defects. This delayed flowering was attributed to the repression of <span style="font-style: italic">FLOWERING LOCUS T &#40;FT&#41;</span> and <span style="font-style: italic">SUPPRESSOR OF OVEREXPRESSION OF CONSTANS 1 &#40;SOC1&#41;</span>. <span style="font-style: italic">BcSVP</span> expression under the control of the <span style="font-style: italic">AtSVP</span> promoter also resulted in the complementation of the <span style="font-style: italic">svp</span> mutation in <span style="font-style: italic">Arabidopsis</span>. These results indicate that <span style="font-style: italic">BcSVP</span> is a functional equivalent of <span style="font-style: italic">AtSVP</span> and also suggest that <span style="font-style: italic">BcSVP</span> may prove useful for the genetic manipulation of flowering time in <span style="font-style: italic">Brassica</span> plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">An alternative splicing variants of rice pex5p, OsPex5pL, is required for PTS2 protein import</div>

<div>

	<span style="text-decoration: underline" >
	Hye Song Lim</span>
	<sup>1</sup>, 

	<span  >
	Sun Young Kim</span>
	<sup>2</sup>, 

	<span  >
	Young Jun Jung</span>
	<sup>2</sup>, 

	<span  >
	Min Gyung Chun</span>
	<sup>2</sup>, 

	<span  >
	Kyun Oh Lee</span>
	<sup>2</sup>, 

	<span  >
	Woe-Yeon Kim</span>
	<sup>2</sup>, 

	<span  >
	Sang Yeol Lee</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Envirovemental Biotechnology National Core Research center, PMBBRC &amp; Division of applied Life Science (BK21),Gyeongsang National University, Jinju 660-701 Korea, 

	<sup>2</sup>Environmental Biotechnology National Core Research Center, PMBBRC &amp;  Division of Applied Life Science (BK21), Gyeongsang National University, Jinju 660-701 Korea

</div>

<div>Using the rice PEX14 cDNA as a bait in a yeast two&#45;hybrid assay, two splice variants of the type I peroxisomal targeting signal &#40;PTS1&#41; receptor, OsPex5pL and OsPex5pS, were cloned from a pathogen&#45;treated rice leaf cDNA library. The proteins were produced from a single gene by alternative splicing, which generated a full&#45;length variant, OsPEX5L, and a variant that lacked exon 7, OsPEX5S. OsPex5pL contained 11 copies of the pentapeptide motif WXXXF&#47;Y in its N&#45;terminus, and seven tetratricopeptide repeats in its C&#45;terminus. Expression of OsPEX5L and OsPEX5S predominantly occurred in leaf tissues, and was induced by various stresses, such as exposure to the pathogen Magnaporthe grisea, and treatment with fungal elicitor, methyl viologen, NaCl or hydrogen peroxide. The Arabidopsis T&#45;DNA insertional pex5 mutant, Atpex5, which does not germinate in the absence of sucrose and was resistant to indole&#45;3&#45;butyric acid &#40;IBA&#41;, was perfectly rescued by over&#45;expression of OsPex5pL, but not by OsPex5pS. Using transient expression of OsPex5pL and OsPex5pS in the Atpex5 mutant, we show that OsPex5pL translocates both PTS1&#45; and PTS2&#45;containing proteins into the peroxisome by interacting with OsPex7p, whereas OsPex5pS is involved only in PTS1&#45;dependent import in Arabidopsis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A Novel Nuclear Protein Associated with CIPK11 Is Involved in Salt Response of Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Soojin Park</span>
	<sup>1</sup>, 

	<span  >
	Kyung-Nam Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biology, Sejong University, Seoul 143-747, Korea

</div>

<div>We have isolated an Arabidopis cDNA, named K50, which encodes a polypeptide interacting with CIPK11 using a yeast two&#45;hybrid system. Deduced amino acid sequence analysis revealed that K50 possesses a bipartite nuclear localization signal and a proline&#45;rich domain, suggesting that it may be a nuclear protein with transcriptional activation activity. In fact, GFP&#45;fusion protein analyses and yeast two&#45;hybrid assays supported that K50 may serve as a transcription activator. Pull&#45;down assays and split&#45;YFP analyses verified formation of the complex between CIPK11 and K50 in vitro and in vivo, respectively. Deletion analyses revealed that K50 interacts with the 363&#45;435 CIPK11 region lacking the CBL3&#45;interacting NAF domain. Interestingly, the CIPK&#45;K50 interaction affinity was enhanced in the presence of Ca2&#43;&#45;bound CBL3. Real&#45;time RT&#45;PCR analyses indicated that the K50 gene expression is down&#45;regulated by the salt stress. Interestingly, however, a T&#45;DNA knock&#45;out K50 mutant was more sensitive to the salt stress than the wild&#45;type plants. We have also observed that the stress genes such as RD29A&#47;B and Kin1&#47;Kin2 were highly expressed in the k50 mutant. Anyway, these findings suggest that K50 may play an important role in relaying the cytosolic Ca2&#43; signals induced by salt stress to the nucleus, thereby regulating gene expression.</div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">ECL1</span> promotes flowering by the repression of <span style="font-style: italic">FLC</span> in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Sung Myun Hong</span>
	<sup>1</sup>, 

	<span  >
	Seung Kwan Yoo</span>
	<sup>1</sup>, 

	<span  >
	Jeong Seob Lee</span>
	<sup>2</sup>, 

	<span  >
	Ji Hoon Ahn</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Signaling Network Research Center, School of Life Sciences and Biotechnology, Korea University, Seoul, 136-701, South Korea, 

	<sup>2</sup>School of Biological Sciences, Seoul National University, Seoul, 151-742, South Korea

</div>

<div><span style="font-style: italic">Early flowering and Curly leaves1&#45;1D</span> &#40;<span style="font-style: italic">ecl1&#45;1D</span>&#41; , which showed early flowering and curly leaf phenotype, was isolated by an activation tagging screen. Various approaches showed that earlying of <span style="font-style: italic">ecl1&#45;1D</span> was due to the upregulation of <span style="font-style: italic">Flowering locus T</span> &#40;<span style="font-style: italic">FT</span>&#41; . <span style="font-style: italic">ecl1&#45;1D co&#45;2</span> plants flowered earlier than <span style="font-style: italic">co&#45;2</span> plants and <span style="font-style: italic">ecl1&#45;1D 35S::CO</span> flowered earlier than its single overexpression plant suggesting that <span style="font-style: italic">ECL1</span> may not act within <span style="font-style: italic">CO</span> mediated&#45;photoperiod pathway. RT&#45;PCR analysis showed that activation of <span style="font-style: italic">ECL1</span> caused downregulation of <span style="font-style: italic">FLOWERING LOCUS C</span> &#40;<span style="font-style: italic">FLC</span>&#41; and <span style="font-style: italic">ecl1&#45;1D FRI FLC</span> showed early flowering compare with  flowering phenotype of <span style="font-style: italic">FRI FLC</span>. <span style="font-style: italic">ecl1&#45;1D</span> showed similar circadian rhythm of <span style="font-style: italic">FLC</span> mutant by analysis of <span style="font-style: italic">CHLOROPHYLL A&#47;B BINDING PROTEIN 2</span> &#40;<span style="font-style: italic">CAB2</span>&#41; expression and leaf movement. Our data suggest that <span style="font-style: italic">ECL1</span>, which exerts its effect in <span style="font-style: italic">FLC</span>, is a florial promoter.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The The Functional Characterization of OmLEJ and Its Homologs, OsLEJ and AtLEJ2: Those Genes Are Required for Stress Responses and Reproductive Development</div>

<div>

	<span style="text-decoration: underline" >
	Kyoung Shin Yoo</span>
	<sup>1</sup>, 

	<span  >
	Kwang Wook Jung</span>
	<sup>1</sup>, 

	<span  >
	Mei Hua Cui</span>
	<sup>1</sup>, 

	<span  >
	Yun Young Kim</span>
	<sup>1</sup>, 

	<span  >
	Jeong Sheop Shin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Life Sciences &amp;  Biotechnology, Korea University, Seoul, 136-701, Korea

</div>

<div>In previous study, we selected twenty candidate clones for functional analysis based on the results of microarray by fungal infection, insect infestation and wound treatment. One of them, of our interest, OmCDCP and its cultivated rice and Arabidopsis orthologs were used for functional analyses. Expression of OmCDCP and OsCDCP genes were much induced by wounding, brown plant&#45;hopper and rice blast fungus as well as exogenous JA treatment, in spite of their different intensity and expression pattern. One ortholog in Arabidopsis, AtCDCP2, is also characterized. The AtLEJ2 Protein is Localized in Chloroplast. ProAtCDCP2:GUS analysis revealed that the expression of AtCDCP2 is restricted in the pollen grains and endothecium of anther. AtCDCP2 ectopic expression caused developmental delay of floral organ, arrested anther cells on secondary cell wall biosynthesis, and reduced the accumulation of JA in the flower bud clusters, resulting in male&#45;sterility by anther indehiscence, and also those overexpression lines are much more tolerant to various osmotic stresses than wild type. Based on these data, we are trying to characterize the biological roles of novel genes in defense system or developmental process, including regulatory mechanism as well as the effects of gene itself. </div>

</div>





<div class="abs">

<div style="font-weight:bold">CIPK1 Interacts with an Arabidopsis Novel Nuclear Protein Similar to Human Activating Signal Cointegrator&#45;1</div>

<div>

	<span style="text-decoration: underline" >
	Yungyeong Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biology, Sejong University, Seoul 143-747, Korea

</div>

<div>CIPK1 interacts with a couple of calcineurin B&#45;like &#40;CBL&#41; calcium sensors such as CBL1 and CBL3. Therefore it is believed that CIPK1 is also involved in a variety of stress signal transduction pathways which use Ca2&#43; as a second messenger. However little is known about how CIPK1 transmits the signal to downstream components. In efforts to find the in vivo substrates of CIPK1, we have performed the yeast&#45;two hybrid screening and discovered a new CIPK1&#45;interacting protein &#40;named K26&#41; in Arabidopsis, which possesses a zinc&#45;finger motif found in human activating signal cointegrator&#45;1. Pull&#45;down assays verified formation of the complex between CIPK1 and K26 in vitro. Interestingly, K26 exclusively interacted with CIPK1, but not with other CIPK family members including CIPK2, 3, 5, 6, 7, 11, 12 and 23. Deletion analyses revealed that the nonkinase domain of CIPK1 &#40;282 to 451 amino acids&#41; is sufficient and required for the interaction with the K26 region between 147 and 308 amino acid residues. GFP fusion protein analyses indicated that K26 is mainly localized in the nucleus. Currently, we are analyzing a homozygous knock&#45;out Arabidopsis mutant that carries a single copy of T&#45;DNA in the K26 genomic locus.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Defense against &#34;Sclerotinia sclerotiorum&#34; in &#34;Arabidopsis&#34; is dependent on JA, SA, ethylene, and ABA signaling</div>

<div>

	<span style="text-decoration: underline" >
	Xiaomei Guo</span>
	<sup>1</sup>, 

	<span  >
	Henrik Stotz</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Oregon State University, Corvallis, OR, USA

</div>

<div>Genotypic differences in susceptibility of &#34;Arabidopsis thaliana&#34; to &#34;Sclerotinia sclerotiorum&#34; have not been reported due to the extreme susceptibility of this cruciferous plant.  To overcome this limitation, we have established inoculation conditions that enable evaluation of differences in susceptibility to &#34;S. sclerotiorum&#34; among &#34;Arabidopsis&#34; mutants and ecotypes.  Two &#34;coi1&#34; mutant alleles conferred hypersusceptibility to &#34;S. sclerotiorum&#34;.  The plant defensin gene PDF1.2 was no longer induced after challenging the &#34;coi1&#45;2&#34; mutant with &#34;S. sclerotiorum&#34;.  Hypersusceptibility of the &#34;coi1&#45;2&#34; mutant to &#34;S. sclerotiorum&#34; was not correlated with oxalate sensitivity.  The &#34;npr1&#34; and &#34;ein2&#34; mutants were also hypersusceptible to &#34;S. sclerotiorum&#34;.  Induction of PDF1.2 and the pathogenesis&#45;related gene PR1 was reduced in &#34;ein2&#34; and &#34;npr1&#34; mutants, respectively.  Actigard, a commercial formulation of the systemic acquired resistance inducer benzothiadiazole, reduced susceptibility to &#34;S. sclerotiorum&#34;.  Based on histochemical analysis of oxalate&#45;deficient and wild&#45;type strains of &#34;S. sclerotiorum&#34;, oxalate caused a decrease in hydrogen peroxide production, but no detectable changes in plant superoxide production or gene expression. The &#34;abi1&#34; and &#34;abi2&#34; mutants also conferred hypersusceptibility to &#34;S. sclerotiorum&#34; and they were more oxalate sensitive than wild type plants.  Hypersusceptibility of &#34;abi1&#34; and &#34;abi2&#34; mutants to &#34;S. sclerotiorum&#34; were correlated with oxalate sensitivity and wilting symptoms after inoculation with &#34;S. sclerotiorum&#34;.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">Structural and functional analysis of SGT1, a unique co&#45;chaperone of HSP90 required for plant disease resistance.</div>

<div>

	<span style="text-decoration: underline" >
	Marta Boter</span>
	<sup>1</sup>, 

	<span  >
	Beatrice Amigues</span>
	<sup>2</sup>, 

	<span  >
	Jack Peart</span>
	<sup>1</sup>, 

	<span  >
	Christian Breuer</span>
	<sup>1</sup>, 

	<span  >
	Yasuhiro Kadota</span>
	<sup>3</sup>, 

	<span  >
	Raphael Guerois</span>
	<sup>2</sup>, 

	<span  >
	Ken Shirasu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Sainsbury laboratory, John Innes Centre, Norwich, UK, 

	<sup>2</sup>D&#233;partement de Biologie Joliot-Curie CEA Saclay, Cedex, France, 

	<sup>3</sup>Plant Immunity Research Group, RIKEN Plant Science Center, Yokohama, Japan

</div>

<div>SGT1 and RAR1 are highly conserved eukaryotic proteins that interact with the molecular chaperone HSP90. In plants, SGT1, RAR1 and HSP90 are essential for disease resistance triggered by a number of resistance &#40;R&#41; proteins. To assess the mechanism by which SGT1 functions in disease resistance we screened a series of random point mutations in N. benthamiana for defects in Rx mediated resistance against Potato Virus X &#40;PVX&#41;. Loss of function and dominant negative causative mutations were distributed broadly over the CS and SGS domain indicating that both domains are involved in SGT1 function in R&#45;gene mediated resistance. Combination of NMR studies with random and site&#45;directed mutagenesis experiments revealed that RAR1 and HSP90 interact with opposite sites of the CS domain. Functional analysis of SGT1 mutants indicated that the interaction between SGT1 and HSP90 is required for SGT1 function in disease resistance triggered by Rx. Finally, we present evidence that SGT1, RAR1 and HSP90 can form a ternary complex.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Chemical genetic Interrogation of protein&#45;protein interaction</div>

<div>

	<span  >
	Yong-Jik Lee</span>
	<sup>1</sup>, 

	<span  >
	Augusta Jamin</span>
	<sup>1</sup>, 

	<span  >
	Thomas Girke</span>
	<sup>1</sup>, 

	<span  >
	David Carter</span>
	<sup>1</sup>, 

	<span  >
	Natasha Raikhel</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Zhenbiao Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Riverside, CA, USA

</div>

<div>The chemical genetics&#47;genomics approach uses small molecules to modify or disrupt the function of specific genes&#47;proteins. This is in contrast to classical genetics, in which mutations disrupt the function of genes. This approach is expected to overcome some limitations associated with genetic approaches, such as the inability to perturb biological processes instantaneously, transiently, and spatially.  Given the importance of protein&#45;protein interaction in protein function and regulation, we are interested in identifying chemicals that alter a specific protein&#45;protein interaction.   We are testing whether high throughput yeast two&#45;hybrid assay provides a useful method for this purpose.  As a proof of concept, we used this assay to screen chemical libraries for chemicals that disrupt the interaction between the ROP2 GTPase and a GAP protein.  Strategies to eliminate false positives and identification of several putative positive hits will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A Role of the C&#45;terminal of AtRGS1 in its Targeting to the Plasma Membrane</div>

<div>

	<span style="text-decoration: underline" >
	Guangzhen Hu</span>
	<sup>1</sup>, 

	<span  >
	Jirong Huang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Laboratory of Plant Molecular Genetics, Shanghai Institute of Plant Physiology and Ecology, Shanghai Institute for Biological Sciences, Shanghai 200032, China

</div>

<div>RGS&#40;regulators of G&#45;protein signaling&#41; proteins deactivate G&#45;protein signaling through accelerating GTPase activity of G&#945; subunits. There is only one protein containing RGS box in Arabidopsis thaliana&#65292;designated AtRGS1, which also has seven trans&#45;membrane domain at N&#45;terminal. AtRGS1 have been verified to be a plasma membrane protein. Here we present that GPA is unlikely involved in the plasma membrane targeting  of AtRGS1 and its C&#45;terminal play a role in this process.</div>

</div>

</div>
</div>

</body>
</html>
