<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>
<head>
<title>ICAR 2007 - Metabolism</title>
<link rel="Shortcut Icon" href="/favicon.ico">
<link rel="stylesheet" type="text/css" href="/css/main.css" />
<link rel="stylesheet" type="text/css" href="/css/search.css" />
<link rel="stylesheet" type="text/css" href="/css/base.css" />

<style type="text/css">
@import "/css/page/pagelevels.css";
</style>

<style type="text/css">
.abs {
 margin-top: 30px;
}
.abs div {
 margin: 10px;
}
</style>
</head>
<body>

<div id="container">

<script src="https://ssl.google-analytics.com/urchin.js" type="text/javascript"></script>
<script type="text/javascript">
_uacct = "UA-1945230-1";
urchinTracker();
</script>

 <div id="nav">	
  <div id="logo"><a href="/index.jsp"><img src="/i/logo2.gif" width="87" height="61" alt="tair"/></a></div>
 </div>

<div id="content" class="clearfix">

<div class="abs">

<div style="font-weight:bold">S&#45;acylation: Dynamic control of plant development and sigalling by lipid modification of proteins.</div>

<div>

	<span style="text-decoration: underline" >
	Piers Hemsley</span>
	<sup>1</sup>, 

	<span  >
	Laura Taylor</span>
	<sup>1</sup>, 

	<span  >
	Claire Grierson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Univeristy of Bristol, Bristol, United Kingdom

</div>

<div>S&#45;acylation, or palmitoylation, is the only dynamic and reversible lipid modification of proteins. It controls membrane &#47; lipid raft partitioning and sorting of proteins but also affects signalling complex formation, regulation of signal transduction in a similar manner to phosphorylation, affects protein stability and lifespan via ubiquitin mediated proteolysis, enables viral capsule release and acts as a gating mechanism for transmembrane channel proteins. S&#45;acylated proteins in plants include type I and II Rop small GTPases, hetrotrimeric G protein gamma subunits, ARA6 Rab GTPase, RIN4 involved in hypersensitive pathogen responses, calcium responsive CDPK&#47;CPKs, CBLs and the alpha&#45;subunits of tubulin. This indicates the varied roles that S&#45;acylation plays in protein function and regulation as well as the range of proteins modified, yet comparatively little is known about it. Prediction of S&#45;acylated proteins is hindered by the lack of a consensus motif for S&#45;acylation and until recently the enzymes responsible were unknown. Recent advances in the field have led to the identification of a family of S&#45;acyl transferases.<br/><br/>S&#45;acyl transferases are represented by 23 different genes in plants, each with differing spatial and temporal expression patterns. Here we present phenotypic data for mutants in S&#45;acyl transferases, with a detailed analysis of those involved in different aspects of root hair development. Plants defective in an S&#45;acyl transferase generally show distinct or partially overlapping phenotypes indicating that different subsets of targets exist for each S&#45;acyl transferase.<br/><br/>The first S&#45;acyl transferase to be identified in plants, TIP1 of Arabidopsis, will be discussed in detail. Data on whole plant mutant phenotypes, biochemical function, expression profiles, sub&#45;cellular localisation and hormone response have been combined to identify and test potential substrates for TIP1. We also present data obtained from proteomic analysis of the &#8216;&#8217;Tip1&#45;&#8216;&#8217; S&#45;acyl transferase mutants identifying substrates for TIP1.<br/> <br/><br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">THE ARABIDOPSIS HEAT STRESS INDUCED CO &#8211;CHAPERONES ROF1 AND ROF2 &#40;FKBPs &#41; ARE DEVEOPLMENTALLY REGULATED AND INTERACT WITH HSP90 &#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	Adina Breiman</span>
	<sup>1</sup>, 

	<span  >
	Dedi Meiri</span>
	<sup>1</sup>, 

	<span  >
	Odelia Farchi -Pisanti</span>
	<sup>1</sup>, 

	<span  >
	Keren Aviezer-Hagai</span>
	<sup>1</sup>, 

	<span  >
	Julia Skovorodnikova</span>
	<sup>1</sup>, 

	<span  >
	Nir Ohad</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dept of Plant Sciences,Tel Aviv University, Tel Aviv Israel

</div>

<div>The plant co&#45;chaperones FKBPs that belong  to the peptidyl prolyl cis&#45;trans isomerases &#40;PPIases&#41;are investigated due to their function in protein folding, signal transduction and chaperone activity. Arabidopsis possesses 22 FKBPs, present in all subcellular organelles.We are studying the large FKBPs ROF1 &#40;FKBP62&#41; and ROF2 &#40;FKBP65&#41;.They possess  three FKBP domains followed by TPR and calmodulin binding domains.Arabidopsis and wheat FKBPs interact with HSP90, calmodulin and dynein, participating in the heterocomplex trafficking machinery to the nucleus .The expression of ROF1 and ROF2  is organ specific and developmentally regulated . High expression  was observed in the vascular elements in roots,in hydatodes,trichomes and stomata,in stigma and sepals in young flowers, in connective tissues of siliques and  in mature seeds. Promoter analysis has revealed heat stress and ABA responsive elements.Both genes are induced at 37C but only ROF1 is expressed also in plants grown at 22C .ROF1 and ROF2 were assumed to interact with HSP90 similar to wheat FKBP73 and 77. We have shown that Rof1 coimmunoprecipitates  with HSP90.However , when the 5 amino acids defined as essential for interaction between the TPR and HSP90 were mutated , the interaction was abrogated. We have shown that ROF1 binds specifically to HSP90.1 whereas ROF2 did not. &#13;<br/>Microarray analysis in knockout mutants  reveals a high bias towards genes that are related to response to biotic and abiotic stress, both up and down regulated. We have isolated a T&#45;DNA knockout  mutant  of rof1 &#45;. An increase in expression of genes related to cell wall loosening enzymes and trehalose biosynthesis was revealed in the rof 1&#45; mutant by transcriptome analysis. Since no phenotype was observed under normal growth or heat stress conditions, we have produced a double rof1&#45;&#47;rof2&#45; mutant.&#13;<br/>We will test the hypothesis that ROF1 plays a role in the ABA regulated pathway associated with inhibition of germination and its capacity to interfere with the expression of cell wall loosening enzymes and  trehalose biosynthesis.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Understanding Arabidopsis peroxisomal proteomes</div>

<div>

	<span style="text-decoration: underline" >
	Jianping Hu</span>
	<sup>1</sup>, 

	<span  >
	Andreas Weber</span>
	<sup>1</sup>, 

	<span  >
	Laura Olsen</span>
	<sup>2</sup>, 

	<span  >
	Pingfang Yang</span>
	<sup>1</sup>, 

	<span  >
	Kalpana Shrestha</span>
	<sup>1</sup>, 

	<span  >
	Sigrun Reumann</span>
	<sup>1</sup>, 

	<span  >
	Sheng Quan</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Michigan State University, East Lansing, USA, 

	<sup>2</sup>University of Michigan, Ann Arbor, USA

</div>

<div>Plants contain a large number of structurally similar but functionally diverse peroxisomes that are crucial to various physiological and metabolic pathways in development and during stress responses. However, the protein composition and metabolic and regulatory networks associated with peroxisomes is still poorly understood. Using a combinatorial approach including proteomics, bioinformatics, cell biology, and reverse genetics, we are cataloging the membrane and matrix proteomes of three major peroxisomal variants: leaf peroxisomes from green leaves, glyoxysomes from germinating seedlings, and gerontosomes from senescent tissue. To date we have used mass spectrometry&#45;based shotgun proteomics to identify novel peroxisomal membrane and matrix proteins from mature, wounded, and senescent leaves. Several dozen novel candidate peroxisomal proteins have been identified by this approach. A subset of the novel proteins identified by both proteomic and bioinformatic analyses were subjected to subcelluar localization study using fluorescence microscopy of GFP&#45;fusion proteins; a number of them have been confirmed to be peroxisomal. Based on results from a series of physiological and biochemical assays conducted with sequence&#45;indexed T&#45;DNA insertion mutants of some of these genes, hypotheses of novel peroxisomal functions and pathways have been generated. In addition to providing the scientific community with a much&#45;needed comprehensive inventory of plant peroxisomal proteins, we will also disseminate a set of tagged reporter constructs for follow&#45;up studies. This work is funded by the NSF Arabidopsis 2010 program.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Dual localization of the Whirly 1 protein in plastids and in the nucleus</div>

<div>

	<span style="text-decoration: underline" >
	Karin Krupinska</span>
	<sup>1</sup>, 

	<span  >
	Ying Miao</span>
	<sup>1</sup>, 

	<span  >
	Evelyn Grabowski</span>
	<sup>1</sup>, 

	<span  >
	Isabell Kilbienski</span>
	<sup>1</sup>, 

	<span  >
	Maria Mulisch</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Institute of Botany, University of Kiel, Germany, 

	<sup>2</sup>Central Microscopy, University of Kiel, Germany

</div>

<div>The Whirly1 protein of Arabidopsis thaliana &#40;AtWHY1, gene locus At1g14410&#41; belongs to a small family of single stranded DNA binding proteins. It has been shown to bind to a specific DNA element called ERE &#40;elicitor response element&#41; which is present in the promoter of the PR10a gene of potato &#40;Desveaux et al. 2000&#41; as well as to the telomeric heptanucleoide TTTAGGG &#40;Yoo et a. 2007&#41;. A mutant with a T&#45;DNA insertion in the Atwhy1 gene was shown to possess longer telomeres &#40;Yoo et al.2007&#41; suggesting that AtWHY1 inhibites telomerase activity.&#13;<br/>When a GFP fusion protein of the complete AtWHY1 protein of Arabidopsis thaliana was analyzed for its subcellular location by transient transformation assays, it was detected exclusively in plastids &#40;Krause et al. 2005&#41;. When the mature AtWHY1 protein without the plastid target peptide was fused to GFP, the protein was however located in the nucleus. Immunological analyses with an antibody against the Whirly1 protein of barley revealed that in barley primary foliage leaves the protein was predominantly present  in chloroplasts as well as in the nucleus. When the WHY1 protein is located in the nucleus it is part of high molecular weight protein complexes. Bimolecular fluorescence complementation assays and yeast 2&#45;hybrid analysis showed that the AtWhy1 protein might form homooligomeres and colocalize in the nucleus when the plastid transit sequence was cleaved off. The high molecular weight complexes are extremely stable and do contain DNA. By DNAse treatment the monomeric form of ATWHY1 may be released from the complex. Immunological analyses indicated that during leaf senescence when chloroplasts were dismantled the proportion of WHY1 located in the nucleus increased.&#13;<br/>&#13;<br/>References:&#13;<br/>Desveaux D, Depr&#233;s C, Joyeux A, Subramanian R, Brisson N &#40;2000&#41;Plant Cell 12:1477&#45;1489&#13;<br/>Krause K, Kilbienski I, Mulisch M, R&#246;diger A, Sch&#228;fer A, Krupinska K &#40;2005&#41;FEBS Lett. 579:3707&#45;3712&#13;<br/>Yoo HH, Kwon C, Lee MM, Chung K &#40;2007&#41; The Plant Journal 49:442&#45;451&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">sec28 is an intracellular membrane trafficking mutant defective in the cell wall formation</div>

<div>

	<span style="text-decoration: underline" >
	Hugo  Zheng</span>
	<sup>1</sup>, 

	<span  >
	Gill Dean</span>
	<sup>2</sup>, 

	<span  >
	George Haughn</span>
	<sup>2</sup>, 

	<span  >
	Ljerka Kunst</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>McGill University, Montreal, Canada, 

	<sup>2</sup>University of British Columbia, Vancouver, Canada

</div>

<div>Intracellular membrane trafficking links the cellular organelles such as the endoplasmic reticulum &#40;ER&#41;, Golgi, and the plasma membrane &#40;PM&#41; in eukaryotic cells.  In plants, it plays crucial roles in the delivery of proteins, lipids and polysaccharides to vacuoles, PM and cell walls, which are among most biological significant and commercial important structures.  While proper sub&#45;cellular change in these structures is central to many aspects of plant developmental processes and cellular adaptation to environmental stresses, food, oil, cotton, wood and plant fibers rely on the formation of plant ER, vacuoles and cell walls.  It is therefore important to understand how intracellular membrane trafficking is regulated in plant cells.  <br/><br/>In Arabidopsis, forward genetics is a powerful approach, some of mutants such as gnom, knolle, keule and vac1, which were identified by virtue of their developmental phenotypes, have subsequently been found to have mutations in genes encoding regulators of plant membrane trafficking.  Systematic isolation of mutants defective in various membrane trafficking steps has, however, yet to be applied in Arabidopsis owing to the lack of common reporter markers and suitable screens.  We have recently developed a secretory GFP &#40;secGFP&#41;&#45;based visual screen and biochemical assay system in Arabidopsis for isolating potential mutants with perturbed membrane trafficking, aiming at identifying novel genes controlling plant specific membrane trafficking &#40;Zheng et al., 2004&#41;.  Based on the intracellular accumulation of secGFP in a compartment reminiscent of Golgi apparatus in a mutagenized secGFP transgenic line, we have recently identified sec28 as a potential post&#45;Golgi trafficking mutant.  Further biological and cellular characterization indicated that sec28 is associated with many physiological and developmental phenotypes, including defects in cell wall formation, cuticular wax secretion, and apical dominance.  The sec28 mutation has been mapped to a 300 kbp region on chromosome 1; the fine mapping of the mutation is currently in progress.<br/><br/>Zheng, H., et al. &#40;2004&#41;. Plant J. 37, 398&#45;414</div>

</div>





<div class="abs">

<div style="font-weight:bold">A Functional and structural characterization of plant lipid rafts as membrane signalling platforms</div>

<div>

	<span style="text-decoration: underline" >
	Sylvain Raffaele</span>
	<sup>1</sup>, 

	<span  >
	Fabienne Furt</span>
	<sup>1</sup>, 

	<span  >
	B&#233;atrice Satiat-Jeunemaitre</span>
	<sup>2</sup>, 

	<span  >
	Jean-Jacques Bessoule</span>
	<sup>1</sup>, 

	<span  >
	Jean-Pierre Carde</span>
	<sup>3</sup>, 

	<span  >
	S&#233;bastien Mongrand</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Lab de Biogen&#232;se Membranaire, Bordeaux, France, 

	<sup>2</sup>ISV, Gif-sur-Yvette, France, 

	<sup>3</sup>Lab de Physiologie &amp;  Biotechnologie V&#233;g&#233;tales, Villenave d?Ornon, France

</div>

<div>Evidences are accumulating suggesting that the plasma membrane &#40;PM&#41; of eukaryotic cells is not homogenous. Dynamic clustering of sphingolipids and sterols give rise to domains insoluble in non&#45;ionic detergents named lipid rafts. These years, a growing interest was devoted to their study in PM of plant cells. We showed that lipid rafts exist in PM isolated from tobacco leaves, BY2 cells, and <span style="font-style: italic">Medicago truncatula</span> roots. Similar domains were found in <span style="font-style: italic">Arabidopsis thaliana</span>. Nevertheless, little is known about the architecture of these structures, and their functional relevance. With aim to gain insights on these questions, we developed a strategy taking benefit from the different model plants Medicago, Nicotiana and Arabidopsis.&#13;<br/>In a first step, we carried out lipidomic and proteomic studies on plant lipid rafts. Interestingly, microdomains are able to recruit specific sets of PM proteins. Therefore, we investigated the association with microdomains of stress&#45;related enzymes enriched in rafts.&#13;<br/>Another striking result from proteomic studies is the enrichment in lipid rafts of a plant&#45;specific protein, hydrophilic although associated with PM, called Remorin &#40;REM&#41;. Remorins are a protein family present in various plant species: Potato REM1 is known to bind OGA and pectin, and its Arabidopsis homolog, AtDBP, forms filaments <span style="font-style: italic">in vitro</span>.&#13;<br/>To understand the basis of the association of REM with lipid rafts, we first performed in silico 3D modelling to propose a putative structure for AtDBP. Putative functional domains were tested using GFP fusions for their ability to bind the membrane. Then we adopted multiple microscopy strategies to study the localisation of REM1 <span style="font-style: italic">in situ</span>, and on isolated PM and rafts. In parallel, we exploited Arabidopsis large set of data to identify biological processes that may rely on REM. Wild type and over&#45;expressers were assayed for alteration in these processes. Results will be discussed in term of cell biology and putative function of Remorins.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The Kingdom&#45; and Tissue&#45;specific Anchorage of Plant RanGAP to the Nuclear Envelope Involves a Novel Family of Plant Nuclear Pore Associated Transmembrane Proteins</div>

<div>

	<span style="text-decoration: underline" >
	Xianfeng Morgan Xu</span>
	<sup>1</sup>, 

	<span  >
	Tea Meulia</span>
	<sup>2</sup>, 

	<span  >
	Iris Meier</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Biotechnology Center and Department of Plant Cellular and Molecular Biology, The Ohio State University, Columbus, Ohio 43210, USA, 

	<sup>2</sup>Molecular and Cellular Imaging Center, The Ohio State University/Ohio Agricultural Research and Development Center, Wooster, Ohio 44691, USA

</div>

<div>Ran is a multifunctional small GTPase of the Ras superfamily involved in nucleocytoplasmic transport, mitotic spindle assembly, cell cycle control, and nuclear envelope &#40;NE&#41; formation.  Its roles are accomplished by the asymmetric distribution of its GTP&#45; and GDP&#45; bound forms, enabled by the specific localization of Ran accessory proteins, the Ran GTPase&#45;activating protein RanGAP and the nucleotide exchange factor RCC1. Mammalian RanGAP1 is targeted to the NE during interphase and to the spindle and kinetochores during mitosis via a sumoylated C&#45;terminal domain and interaction with the nucleoporin Nup358&#47;RanBP2. In contrast, Arabidopsis RanGAP1 is associated with the NE during interphase and the cell plate during cytokinesis, mediated by an N&#45;terminal, plant&#45;specific domain &#40;WPP domain&#41;. Here, we have identified in a yeast two&#45;hybrid screen a novel plant&#45;specific protein that binds to the WPP domain. WPP&#45;domain Interacting Protein 1 &#40;WIP1&#41; and RanGAP1 interact in vivo and co&#45;localize at the NE and cell plate. WIP1 has two homologs in Arabidopsis &#40;WIP2 and WIP3&#41; with a similar domain organization of extended coiled&#45;coil domain and C&#45;terminal transmembrane domain. WIP1 co&#45;localizes with a nuclear pore marker and its nuclear pore association was confirmed by immunogold&#45;labeling. The targeting of WIP1 requires the transmembrane domain which is also sufficient for the NE &#40;presumably nuclear pore&#41; association. In a wip1&#47;wip2&#47;wip3 triple mutant, but not in single or double mutants, RanGAP1 is dislocated from the NE in undifferentiated root tip cells, while NE targeting in differentiated root cells and targeting to the cell plate remain intact. We propose that WIP family members are novel plant nuclear pore proteins involved in RanGAP1 NE anchoring in specific cell types. Our data support a separate evolution of RanGAP targeting mechanisms in different kingdoms.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification and characterization of systemic RNA interfering mutants in Arabidopsis thaliana by double strand luciferase</div>

<div>

	<span style="text-decoration: underline" >
	Nazim M Uddin</span>
	<sup>1</sup>, 

	<span  >
	Yeonggil Rim</span>
	<sup>1</sup>, 

	<span  >
	Chun-Lin Shi</span>
	<sup>1</sup>, 

	<span  >
	Juyeon Moon</span>
	<sup>1</sup>, 

	<span  >
	Jae-Yean Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Division of Applied Life Science, PMBBRC/EB-NCRC, Gyeongsang National University, Jinju 660-701, Korea.

</div>

<div>  Post&#45;transcriptional gene silencing &#40;PTGS&#41; is a plant RNA&#45;induced gene silencing  surveillance mechanism providing a defense against invasive nucleic acids such as transgenes, virus and transposons. Small RNAs generated from long dsRNA can be associated into the RNA&#45;induced silencing complex &#40;RISC&#41;, and then guide the cleavage of target mRNA at a single site in the region of complementarity. Previous studies showed that RNA&#45;induced silencing of endogenous and exogenous gene expression spreads locally and systemically, respectively. This mobile signal in systemic RNAi might be a complex consisted with small RNAs and proteins, and could traffic through plasmodesmata &#40;PD&#41; and phloem system in plants. However, little is known about this mobile signal, and the PD associated proteins functioning in this pathway. To isolate protein components functioning in this process, we developed a genetic mutant screening system in which the firefly luciferase &#40;Luc&#41; reporter was used. The transgenic plants carrying 35S&#45;Luc construct showed high luciferase activity, whilst the transformants containing both 35S&#45;Luc and pSuc2&#45;dsLuc showed no or little luciferase activity. These results suggested that the luciferase gene was silenced by pSuc2&#45;dsLuc and the silencing signal induced in the vascular cells was able to move into the neighboring cells through PD. This silencing line was treated with EMS, generating around 4000 M1 plants. Several candidates were found, showing significant luciferase recovery. Genetic Studies including mapping are being carried out to identify the candidate gene. So far, two mutants were roughly mapped on the long arm of chromosome 1 and 3, by using a series of SSLP and CAPS markers. Here, we present the current progress on the screening of siRNA movement mutants and their characterization.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Plant evolves a unique mechanism of endocytosis</div>

<div>

	<span style="text-decoration: underline" >
	Takashi Ueda</span>
	<sup>1</sup>, 

	<span  >
	Kazuo Ebine</span>
	<sup>1</sup>, 

	<span  >
	Tatsuaki Goh</span>
	<sup>1</sup>, 

	<span  >
	Mariko Sunada</span>
	<sup>1</sup>, 

	<span  >
	Wakana Uchida</span>
	<sup>2</sup>, 

	<span  >
	Tomoaki Nishiyama</span>
	<sup>3</sup>, 

	<span  >
	Mitsuyasu Hasebe</span>
	<sup>4</sup>, 

	<span  >
	Akihiko Nakano</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>University of Tokyo, Tokyo, Japan, 

	<sup>2</sup>RIKEN, Wako, Saitama, Japan, 

	<sup>3</sup>Kanazawa University, Kanazawa, Japan, 

	<sup>4</sup>National Institute for Basic Biology, Okazaki, Japan

</div>

<div>Crucial roles of endocytosis in various plant functions are emerging recently, but its molecular mechanism and physiological significance still remain largely unknown. Using Arabidopsis thaliana, we have been studying the molecular mechanism of endocytosis with a special focus on Rab5 GTPases.  Three Rab5 members, Ara7, Rha1 and Ara6, are encoded in the Arabidopsis genome, which are all involved in endocytosis.  Ara7 and Rha1 are orthologs of mammalian Rab5, and Ara6 is a plant&#45;unique type of Rab5 member.  Through genetic analysis, we have found that these two subgroups function antagonistically in various developmental stages, although they are all activated by the practically sole GEF, AtVps9a.  Bryophytes and lycophytes also have the Ara6&#45;type Rab5, thus this subgroup is well conserved among land plants.  These data indicate that land plants have evolved a quite unique mechanism for the regulation of endocytosis, which is essential for the plant life.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Isolation and characterization of a KN1 binding protein</div>

<div>

	<span style="text-decoration: underline" >
	Lijun Hwang</span>
	<sup>1</sup>, 

	<span  >
	Juyeon Moon</span>
	<sup>1</sup>, 

	<span  >
	Xiaoming Xu</span>
	<sup>1</sup>, 

	<span  >
	Xiong-Yan Chen</span>
	<sup>1</sup>, 

	<span  >
	Jae-Yean Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Division of Applied Life Science, PMBBRC/EB-NCRC, Gyeongsang National University, Jinju 660-701, Korea

</div>

<div>he intercellular trafficking of the macromolecules via plasmodesmata &#40;PD&#41; plays important roles for developmental and physiological process in plant. PD are intercelluar organelles consisted of specialized membrane channels between neighboring cells. However, little is known about PD&#45;associated and regulatory components involved in cell&#45;to&#45;cell trafficking of macromolecules. KNOTTED1&#40;KN1&#41;, a homeobox protein to function in SAM development in maize, is a well studied non&#45;cell&#45;autonomous protein &#40;NCAP&#41; that can traffic cell&#45;to&#45;cell through PD. To screen some trafficking pathway&#45;proteins involved in protein trafficking, we used KN1 protein as a bait for yeast two hybrid screening. We found a protein which can equally interact with KN1 and KNAT1. KBP1 shows very weak homology with a chick caldesmon &#40;24&#37; similarity in whole protein&#41; and includes myosin tail repeats. Blast search for its homologs disclosed only an Arabidopsis protein which has 55.2&#37; identity and 64.4&#37; similarity with the KBP1. Deletion assay revealed interacting domains in both KNAT1 and KBP1; N&#45;terminal region of KNAT1 and C&#45;terminal region corresponding exon 26 of KBP1. In silico analysis suggested that KBP1 and its homolog &#40;KBP2&#41; may be plant specific cytoskeletal binding proteins that may play either a role in intercellular protein trafficking of KNOX or a regulatory function of KNOX&#45;related transcription. Here, we present preliminary results obtained and on going research strategy to reveal the function of these proteins. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Proteomic analysis of the rice&#40;Oryza sativa&#41; cell wall</div>

<div>

	<span  >
	Xiong-Yan Chen</span>
	<sup>1</sup>, 

	<span  >
	Hye Sook Lee</span>
	<sup>2</sup>, 

	<span  >
	Yeonggil Rim</span>
	<sup>1</sup>, 

	<span  >
	Seonok Kwon</span>
	<sup>1</sup>, 

	<span  >
	Zee Yong Park</span>
	<sup>2</sup>, 

	<span  >
	Keun-Wu Lee</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Jae-Yean Kim</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Division of Applied Life Science, PMBBRC/EB-NCRC, Gyeongsang National University, Jinju, 660-701, Korea, 

	<sup>2</sup>Deaprtment of Life Science, Gwangju Institute of Science and Technology, Gwangju, 500-712, Korea, 

	<sup>3</sup>Division of Applied Life Science, PMBBRC/EB-NCRC, Gyeongsang National University, Jinju 660-701, Korea

</div>

<div>Plant cell wall proteins play crucial roles in cell wall structure and architecture, cell wall metabolism, cell enlargement, signaling, responses to biotic and abiotic stress, and many other physiological processes. Despite the importance of cell wall to the biology of plants, whose proteome is less well characterized than those of other subcellular compartments. Here we present a proteomic analysis aimed at generating a proteomic reference map for plant cell wall. Secreted soluble proteins, ionically &#40;weakly&#45;bound&#41; or tightly&#45;bound cell wall proteins from rice&#40;Oryza sativa&#41; callus were isolated from culture medium or extracted using CaCl2 containing buffer or SDS&#45;based buffer, respectively. Cell wall proteins from each fraction were separated by SDS&#45;PAGE. 2 fractions &#40;high and low molecular weight fractions&#41; were separated from polyacrylamide gel, in&#45;gel digested with trypsin and analyzed using multidimensional protein identification technology &#40;MudPIT&#41;. We have obtained about 500 soluble proteins, about 800 CaCl2 extracted proteins and about 1900 SDS&#45;solubilized proteins using 3 independent extraction methods mentioned above. Identified proteins include classical cell wall proteins, membrane proteins as well as proteins traditionally considered as cytosolic proteins. Much lower level of cytosolic contamination was observed in secreted or CaCl2 extracted proteins using intact callus rather than SDS extracted ones from purified cell wall fraction. To our knowledge, this is the first large&#45;scale &#40;&#62;2000 proteins&#41; rice cell wall proteomic analysis, allowing generating a systemic proteomic map for cell walls of rice. </div>

</div>





<div class="abs">

<div style="font-weight:bold">AtMinE1 is indispensable for chloroplast division in <span style="font-style: italic">Arabidopsis</span> and negatively places the FtsZ ring at the midchloroplast via a balance between the activities of AtMinE1 and ARC11&#47;AtMinD1</div>

<div>

	<span style="text-decoration: underline" >
	Makoto Fujiwara</span>
	<sup>1,2</sup>, 

	<span  >
	Haruki Hashimoto</span>
	<sup>2</sup>, 

	<span  >
	Tomoko Abe</span>
	<sup>1</sup>, 

	<span  >
	Shigeo Yoshida</span>
	<sup>1</sup>, 

	<span  >
	Naoki Sato</span>
	<sup>2</sup>, 

	<span  >
	Ryuuichi Itoh</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>RIKEN, Wako, Japan, 

	<sup>2</sup>University of Tokyo, Tokyo, Japan, 

	<sup>3</sup>University of the Ryukyus, Nishihara, Okinawa, Japan

</div>

<div>Chloroplast division comprises a coordinated and irreversible sequence of events that facilitate symmetric binary fission and that involve prokaryotic&#45;like stromal division factors such as tubulin&#45;like GTPase FtsZ and the division&#45;site regulator MinD. In <span style="font-style: italic">Arabidopsis</span>, a nuclear encoded prokaryotic <span style="font-style: italic">MinE</span> homologue, <span style="font-style: italic">AtMinE1</span>, has been isolated and characterized in a limited series of leaf tissues. However, the relationship between <span style="font-style: italic">AtMinE1</span> expression and chloroplast phenotype remains unclear. In our current study using comprehensive microscopic characterizations, we demonstrate that an <span style="font-style: italic">AtMinE1</span> gene&#45;knockout results in a severe inhibition of chloroplast division, whereas its overexpression causes varying degrees of heterogeneity as well as the division&#45;inhibition of chloroplasts per cell. In <span style="font-style: italic">AtMinE1</span> sense plants, dividing chloroplasts possess either single or multiple FtsZ rings located at random intervals and showing constriction depth, mainly along the chloroplast&#45;polarity axis. Quantitative examinations also revealed a preference for symmetric division among the binary&#45;dividing chloroplasts, suggesting that a sole potential division site exists in chloroplasts. The <span style="font-style: italic">AtMinE1</span> sense plants also displayed an equivalent chloroplast phenotypes to <span style="font-style: italic">arc11</span>, a loss&#45;of&#45;function mutant of <span style="font-style: italic">AtMinD1</span> which forms replicating mini&#45;chloroplasts. Furthermore, chloroplast expansion during leaf cell development was unexpectedly found to have an ability to inhibit chloroplast division in both <span style="font-style: italic">AtMinE1</span> sense and <span style="font-style: italic">arc11</span> plants. Our data thus demonstrate that the chloroplast division site placement involves a balance between the opposing activities of AtMinE1 and AtMinD1, which acts to prevent FtsZ ring formation anywhere outside of the midchloroplast. In addition, the imbalance caused by an AtMinE1&#45;dominance causes multiple, non&#45;synchronous division events at the single chloroplast level, as well as division&#45;inhibition mediated by chloroplast expansion.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Temperature&#45;sensitive allele of RSW4 links the putative plant separase to cell division and cell expansion in the root of Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Shuang Wu</span>
	<sup>1</sup>, 

	<span  >
	Wolf-Ruediger Scheible</span>
	<sup>2</sup>, 

	<span  >
	Tobias I. Baskin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Biology Department, University of Massachusetts, Amherst, Massachusetts, USA, 

	<sup>2</sup>Max-Planck-Institute for Molecular Plant Physiology, Golm, Germany

</div>

<div>         The temperature&#45;sensitive mutant, radially swollen4 &#40;rsw4&#41;, was isolated previously as root morphogenesis mutant, in which expansion becomes less anisotropic. Map&#45;based cloning revealed that a point mutation occurs in At4g22970. This gene encodes a homologue of separase, an essential protein in eukaryotes involved in separating replicated chromatids at the onset of anaphase. RSW4 is the single copy of separase in arabidopsis and T&#45;DNA insertion alleles are lethal. The connection between reduced separase function and root swelling remains unclear.&#13;<br/>&#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Here I report the confirmation of mitotic defects from reduced separase function by means of immuno&#45;fluorescence staining, TEM, and observations of histone&#45;GFP in living cells. The transition from metaphase to anaphase in the mutant at the restrictive temperature&#40;30&#730;C&#41; is severely affected. Chromosome bridges and un&#45;separated sister chromatids are common in the mutant. Although affected cells proceed into cytokinesis, new cell walls are often incomplete, characterized by cell wall stubs and gapped cell walls. With longer time at 30&#730;C, the meristem area of the mutant decreases, corresponding to a decreased rate of cell production.  &#13;<br/>&#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;To explain root swelling in rsw4, we first quantified the angular distribution of cortical microtubules in root epidermal cells, imaged with immuno&#45;fluorescence and in living cells expressing GFP&#45;tubulin. Cortical microtubules become disorganized and the extent of the disorganization correlates with the time course of root swelling. This suggests that separase could act at interphase on microtubule behavior. However, we also found that with time at 30&#730;C, cyclinB&#45;GUS staining is enhanced in the mutant suggesting the cell cycle prograssion is impaired. To explore the regulatory state of cells in rsw4 further, we have initiated a large analysis of mRNA using micro&#45;array and real time PCR methods. Our hypothesis is that orderly progression through M&#45;phase plays an essential role in the determination of cell and organ polarity. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">B2&#45;type cyclin&#45;dependent kinase is controlled by protein degradation</div>

<div>

	<span  >
	Sumiko Adachi</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Masaaki Umeda</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Graduate School of Biological Sciences, Nara Institute of Science and Technology, Ikoma, Nara 630-0101, JAPAN

</div>

<div>The eukaryotic cell cycle is controlled by cyclin&#45;dependent kinases &#40;CDKs&#41;. <span style="font-style: italic">Arabidopsis</span> CDKs have been classified into six types, among which the A&#45; and B&#45;type CDKs are assumed to be crucial for cell cycle progression. The B&#45;type CDKs &#40;CDKBs&#41; are plant&#45;specific CDKs that are expressed specifically from the late S&#45; to the M&#45;phase. While the oscillation of <span style="font-style: italic">CDKB</span> transcripts during the cell cycle has been well characterized, we have only limited information about the protein stability that may control its kinase activity in response to internal or external stimuli.<br/><br/>CDKBs are further classified into two subtypes, CDKB1 and CDKB2. Here we report that the expression of <span style="font-style: italic">Arabidopsis</span> CDKB2 is under the control of protein degradation machinery. We discovered that <span style="font-style: italic">Arabidopsis</span> CDKB2, but not the A&#45; and B1&#45;type CDKs, contains possible PEST sequences, which are known to target themselves for proteolytic degradation and therefore reduce their half&#45;lives. The &#946;&#45;glucuronidase &#40;GUS&#41; fused to the putative PEST motif of CDKB2 was unstable in tobacco Bright Yellow&#45;2 cells and <span style="font-style: italic">Arabidopsis</span> plants, and the proteasome inhibitor MG132 arrested its degradation. We propose that the kinase activity of CDKB2 is regulated not only at the transcriptional level, but also through proteasome&#45;mediated protein degradation. Such regulatory mechanisms have not been reported so far in yeast and animals, thus they may respond to plant&#45;specific signals in the checkpoint control.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Evolutionary and functional analysis of two recently duplicated <span style="font-style: italic">Arabidopsis</span> kinesin genes, <span style="font-style: italic">ATK1</span> and <span style="font-style: italic">ATK5</span>: roles in the reproductive development.</div>

<div>

	<span style="text-decoration: underline" >
	Li Quan</span>
	<sup>1</sup>, 

	<span  >
	Rong Xiao</span>
	<sup>1,2</sup>, 

	<span  >
	Wuxing Li</span>
	<sup>1,2</sup>, 

	<span  >
	Sung-Aeong Oh</span>
	<sup>3</sup>, 

	<span  >
	David Twell</span>
	<sup>3</sup>, 

	<span  >
	Hong Ma</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Department of Biology and the Huck Institutes of the Life Sciences, the Pennsylvania State University, University Park, PA 16802, USA, 

	<sup>2</sup>Plant Physiology Program, the Pennsylvania State University, University Park, PA 16802, USA, 

	<sup>3</sup>Department of Biology, University of Leicester, University Road, Leicester LE1 7RH, United Kingdom

</div>

<div>Gene duplication plays an important role in evolution, allowing functional and expression divergence. However, relatively few studies have been reported on functional divergence of recently duplicated genes. Here, we report that two kinesin genes, <span style="font-style: italic">ATK1</span> and <span style="font-style: italic">ATK5</span>, are paralogs of high overall sequence similarity derived from the most recent <span style="font-style: italic">Arabidopsis</span> genome&#45;wide duplication, which occured after the separation of <span style="font-style: italic">Arabidopsis</span> and poplar. Nevertheless, parts of the N&#45;terminal sequences of these proteins and their corresponding DNA sequences are significantly divergent. Additionally, <span style="font-style: italic">ATK1</span> has higher expression level than other close <span style="font-style: italic">ATK</span> homologs in anthers near the time of meiosis. Genetic study indicates that the <span style="font-style: italic">atk1</span> and <span style="font-style: italic">atk5</span> mutations cannot be transmitted through the gametophytes simultaneously, probably due to abnormal gametophyte mitosis. Moreover, although <span style="font-style: italic">ATK1</span> was previously found to be only required for male meiosis, our cytological experiments revealed that <span style="font-style: italic">ATK1</span> and <span style="font-style: italic">ATK5</span> together are important for both the male and female reproductive development. Therefore, our results suggest that the duplicate genes, <span style="font-style: italic">ATK1</span> and <span style="font-style: italic">ATK5</span>, have retained redundant functions in female meiosis and gametophyte development, but have experienced functional divergence in male meiosis. In male meiosis, the fact that <span style="font-style: italic">ATK1</span> seems to play a major role and has a greater extent of coding sequence divergence suggests that it may have undergone neofunctionalization. In gametophyte development, the fact that both <span style="font-style: italic">ATK1</span> and <span style="font-style: italic">ATK5</span> are needed suggest that they may both experienced subfunctionalization. The analysis of <span style="font-style: italic">ATK1</span> and <span style="font-style: italic">ATK5</span> provides a demonstration of functional evolution of duplicate genes and a scenario for maintenance of duplicated genes. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulation of Cell Divisions in Arabidopsis &#8211; Posttranslational Control of CDKs</div>

<div>

	<span style="text-decoration: underline" >
	Nico Dissmeyer</span>
	<sup>1,2</sup>, 

	<span  >
	Moritz K. Nowack</span>
	<sup>1,2</sup>, 

	<span  >
	Stefan Pusch</span>
	<sup>1,2</sup>, 

	<span  >
	Hilde Stals</span>
	<sup>3</sup>, 

	<span  >
	Dirk Inz&#233;</span>
	<sup>3</sup>, 

	<span  >
	Paul E. Grini</span>
	<sup>4</sup>, 

	<span  >
	Arp Schnittger</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, Cologne, Germany, 

	<sup>2</sup>University of Cologne, Department of Botany III, Cologne, Germany, 

	<sup>3</sup>Department of Plant Systems Biology, Flanders Interuniversity Institute for Biotechnology, Ghent University, Gent, Belgium, 

	<sup>4</sup>University of Oslo, Department of Molecular Biosciences, Oslo, Norway

</div>

<div>In general, the cell cycle control machinery appears to be highly conserved between the kingdoms: progression through the cell cycle is governed by cyclin&#45;dependent kinases &#40;CDKs&#41;. In Arabidopsis, one Cdc2&#43;&#47;Cdc28 homolog &#40;CDKA;1&#41; with the archetypical PSTAIRE hallmark is encoded and due to the specific life style of plants, peculiar mutant phenotypes were expected. In a mutant for cdka;1 that we have isolated, the haploid gametophyte was found to be strongly affected and pollen with only one instead of two sperm cells is produced. This dramatically disrupts flowering plants&#8217; key event of double fertilization. Based on this mutant, we explored the regulatory context of CDKA;1 in planta. In yeast two hybrid screens we have identified potential novel interactors and by mutating conserved phosphorylation&#45;sites we found that phosphorylation of a canonical Thr residue is required for high enzyme activity and is essential for its function. Interestingly, this crucial phosphorylation can be partially mimicked by a T&#62;D substitution and restore the primary defect of cdka;1 mutants. Although this variant displayed a dramatically reduced kinase activity with a compromised ability to bind substrates, homozygous mutant plants were recovered. The partially rescued plants, however, displayed various developmental abnormalities: even though flowers were formed, they were completely sterile as a result of the failure of the meiotic program, indicating that different requirements for CDKA;1 function are needed during the course of plant development. In BiFC assays, different mutant variants of CDKA;1 presented a differential substrate binding capability.&#13;<br/> &#13;<br/>Our data shows that the molecular mechanistics of CDK regulation is conserved between yeast, animals, and plants. However, the regulatory circuits controlling CDK activity appear to be strikingly different.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Sterol function in polar PIN2 auxin efflux carrier positioning during <span style="font-style: italic">Arabidopsis</span> root gravitropism</div>

<div>

	<span style="text-decoration: underline" >
	Shuzhen Men</span>
	<sup>1</sup>, 

	<span  >
	Yohann Boutt&#233;</span>
	<sup>1</sup>, 

	<span  >
	Thomas Moritz</span>
	<sup>1</sup>, 

	<span  >
	Markus Grebe</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Ume&#229; Plant Science Centre, Dept.  of Forest Genetics &amp;  Plant Physiology, SLU, Umea, Sweden

</div>

<div>Plant membrane sterols are crucial for diverse cellular processes during plant growth and development including cell polarity &#91;1, 2&#93;. The directional growth of plant roots along the gravity vector requires correct membrane sterol composition &#91;2&#93; and is mediated by polar auxin transport which involves the <span style="font-style: italic">Arabidopsis</span> PIN2 auxin efflux carrier &#91;3&#45;6&#93;. Previous studies suggested that polar membrane localisation of the auxin efflux carrier protein PIN1 in root tip cells requires correct sterol biosynthesis &#91;2&#93;, and that PIN2 co&#45;traffics with sterols along a common transport route &#91;7&#93;. However, a mechanistic link between sterol and PIN protein in a tropic growth process has not been provided. We have extended our previous studies and functionally analysed the dependence of polar PIN2 protein positioning on sterol composition, employing a combination of pharmacological and new genetic tools for modulation of sterol composition. We provide subcellular localization data for one component of the sterol biosynthesis machinery and employ cell biological approaches to demonstrate that sterol composition in meristematic cells just after cytokinesis is required for establishment of PIN2 polarity. Our results suggest that sterol biosynthesis in dividing cells and post&#45;cytokinetic cells is critical for polar PIN2 partitioning and root gravitropism. We further report more detailed results that post&#45;cytokinetic polarity acquisition of PIN2 requires sterol&#45;dependent endocytosis, providing a mechanism for sterol action on auxin efflux carrier positioning during root gravitropism. &#13;<br/>&#13;<br/>1. Fischer, Men, and Grebe &#40;2004&#41;. Curr. Opin. Plant Biol. 7, 670&#45;676.&#13;<br/>2. Willemsen et al. &#40;2003&#41;. Plant Cell 15, 612&#45;625.&#13;<br/>3. Luschnig et al. &#40;1998&#41;. Genes Dev. 12, 2175&#45;2187.&#13;<br/>4. M&#252;ller et al. &#40;1998&#41;. EMBO J. 17, 6903&#45;6911.&#13;<br/>5. Chen et al. &#40;1998&#41;. Proc. Natl. Acad. Sci. USA. 95, 15112&#45;15117.&#13;<br/>6. Utsuno et al. &#40;1998&#41;. Plant Cell Physiol. 39, 1111&#45;1118.&#13;<br/>7. Grebe et al. &#40;2003&#41;. Curr. Biol. 13, 1378&#45;1387.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Dissecting Plasma Membrane Localization, Vesicular Trafficking and Polar Growth Via Chemical Genomics</div>

<div>

	<span style="text-decoration: underline" >
	Glenn R Hicks</span>
	<sup>1</sup>, 

	<span  >
	N Chary Samboju</span>
	<sup>1</sup>, 

	<span  >
	Stephanie Robert</span>
	<sup>1</sup>, 

	<span  >
	Zhenbiao Yang</span>
	<sup>1</sup>, 

	<span  >
	Natasha V Raikhel</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Center for Plant Cell Biology &amp;  Dept of Botany and Plant Sciences, University of California, Riverside, CA, USA

</div>

<div>The trafficking of proteins to and from the plasma membrane &#40;PM&#41; is an essential process underlying polarized cell growth. The mechanisms and components involved in these processes are poorly understood especially in plants. We are using chemical genomics &#40;the use of chemicals to modify protein function&#41; to identify chemicals affecting protein targeting to the PM. To identify these reagents, a novel pollen&#45;based high throughput chemical screen was developed. The pollen tube is a polarized cell whose growth is strongly dependent upon targeting to the apical PM; thus, it is an excellent model for polar growth. However, genetic approaches would be difficult due to lethality. A 384&#45;well microtiter plate format is used to screen for chemicals that block tobacco pollen germination in vitro.  Among the inhibitors are effectors of polar PM targeting. These are identified via a secondary screen using tobacco pollen expressing GFP fused to ROP1&#45;Interacting Partner 1 &#40;GFP&#45;RIP1&#41;, a marker displaying focused tip localization. The screen is highly automated using the Atto Pathway confocal microscope permitting a rate of thousands of compounds per week. An initial screen of more than 20,000 compounds demonstrated clearly that the desired classes of compounds can be identified. Among the compounds characterized are those that cause depolarized &#40;isodiamteric&#41; growth and mis&#45;localization of GFP&#45;RIP1 which normally predicts the site of polar tip growth. Recently, several chemicals were examined with success against a panel of endomembrane compartment markers expressed in roots of Arabidopsis seedlings. Interestingly, we have found a chemical that appears to affect late endocytosis of several polar markers &#40;PIN2&#45;GFP, AUX1&#45;YFP&#41; specifically. This indicates that the compound may affect specific pathways and that the target&#40;s&#41; is probably expressed in gametes and sporophytes. Overall, we will use these bioactive chemicals directly to study endomembrane trafficking and as reagents to identify gene networks involved endomembrane trafficking and the control of polar growth.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The H subunit of eukaryotic translation initiation factor 3 contributes to the efficient translation of mRNAs with upstream open reading frames</div>

<div>

	<span  >
	Byung-Hoon Kim</span>
	<sup>1</sup>, 

	<span  >
	Bijoyita Roy</span>
	<sup>1</sup>, 

	<span  >
	Xue Cai</span>
	<sup>1</sup>, 

	<span  >
	Justin N Vaughn</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Albrecht G von Arnim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Tennessee, Knoxville, TN, USA

</div>

<div>Upstream open reading frames &#40;uORFs&#41; in 5&#8217; leader sequences often inhibit translation initiation and can play critical roles in signal&#45;dependent regulation of translation. Although over 30&#37; of mRNAs harbor at least one uAUG in Arabidopsis, factors required to overcome or to regulate them are elusive. Here, we demonstrate that the H subunit of eukaryotic translation initiation factor 3 &#40;eIF3&#41; contributes to the efficient translation of mRNAs with uORFs. eIF3h is considered one of seven non&#45;core subunits with potential regulatory roles in the multifunctional eIF3 complex. The <span style="font-style: italic">eif3h</span> mutant exhibits a pleiotropic phenotype including a hypersensitive sugar response. Transient and stable transgene expression of luciferase reporter constructs fused to different 5&#8217; leader sequences indicated that translation of certain genes, such as the sugar&#45;regulated mRNA AtbZip11, was compromised in the <span style="font-style: italic">eif3h</span> mutant background. Microarray comparisons of polysome loading in wild&#45;type and eif3h mutant seedlings revealed that eIF3h generally helps to maintain efficient polysome loading of mRNA species harboring multiple uORFs, independent of the mRNA transcript level. Functional classification by MapMan further showed that important regulatory gene classes that are enriched in uAUGs, such as transcriptional regulators and protein modifying enzymes, were translationally compromised in the eif3h mutant while other functional subgroups including ribosomal proteins maintained their polysome loading state. In addition, eIF3h boosted the polysome loading of mRNAs with long coding sequences. Taken together, eIF3h functions in translation initiation by overcoming the repressive effect of uORFs, which is particularly pronounced on mRNAs with a long main ORF.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Kinesin&#45;related proteins and a NIMA&#45;related kinase regulate epidermal cell morphogenesis in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Tatsuya Sakai</span>
	<sup>1</sup>, 

	<span  >
	Miki Nishioka</span>
	<sup>1</sup>, 

	<span  >
	Yukiko Uehara</span>
	<sup>1</sup>, 

	<span  >
	MIhoko Takahashi</span>
	<sup>1</sup>, 

	<span  >
	Noriko Fujisawa</span>
	<sup>1</sup>, 

	<span  >
	Kensuke Saji</span>
	<sup>2</sup>, 

	<span  >
	Motoaki Seki</span>
	<sup>1</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>1</sup>, 

	<span  >
	Kiyotaka Okada</span>
	<sup>2,1</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, 

	<sup>2</sup>Kyoto University

</div>

<div>The involvement of kinesin motor proteins in both cell tip growth and cell shape determination has been well characterized in various organisms. However, the functions of the kinesins during cell morphogenesis in higher plants remain largely unknown. Recent transcriptome analyses have suggested that <span style="font-style: italic">MRH2</span>, which encodes an armadillo repeat&#45;containing kinesin&#45;related protein, is involved in root&#45;hair morphogenesis in <span style="font-style: italic">Arabidopsis thaliana</span>, although the details of the underlying mechanisms have not been elucidated. In our current study, we have isolated a cell morphogenesis mutant of Arabidopsis root hairs, <span style="font-style: italic">antlers1&#45;1</span>, and demonstrate that <span style="font-style: italic">ANTLERS1</span> is allelic to <span style="font-style: italic">MRH2</span>.  For clarity, we have renamed the gene <span style="font-style: italic">Arabidopsis thaliana ARMADILLO REPEAT KINESIN1</span> &#40;<span style="font-style: italic">AtARK1</span>&#41;. The <span style="font-style: italic">AtARK1</span> gene has two homologs in the Arabidopsis genome, <span style="font-style: italic">AtARK2</span> and <span style="font-style: italic">AtARK3</span>, and our present results show that <span style="font-style: italic">AtARK2</span> is involved in cell morphogenesis in the root epidermis. We further reveal that a NIMA&#45;related protein kinase, AtNEK6, binds to the AtARK family proteins and has pleiotropic effects upon epidermal cell morphogenesis, suggesting that AtNEK6 is involved in cell morphogenesis in Arabidopsis via microtubule functions associated with these armadillo repeat kinesins.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Towards understanding translational control: Systematic analysis of the Arabidopsis cytosolic ribosome proteome provides detailed insights into its protein complement and their post&#45;translational modification</div>

<div>

	<span style="text-decoration: underline" >
	Adam Carroll</span>
	<sup>1,2</sup>, 

	<span  >
	Joshua Heazlewood</span>
	<sup>2</sup>, 

	<span  >
	Jun Ito</span>
	<sup>1,2</sup>, 

	<span  >
	Harvey Millar</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>ARC Centre of Excellence in Plant Energy Biology, Perth, Western Australia, 

	<sup>2</sup>University of Western Australia, Perth, Western Australia

</div>

<div>The synthesis of polypeptides by cytosolic ribosomes represents a major energy&#45;consuming process of the plant cell and an important point for the control of protein expression. Research aimed at understanding the genetic and molecular mechanisms underlying translation and its control will greatly benefit from accurate knowledge about ribosome structure such as the gene&#45;specific identities of proteins incorporated into ribosomes and the types and sites of covalent modifications on ribosomal proteins. We have used a combination of in silico approaches coupled to mass spectrometry analysis to advance the proteomic insight into Arabidopsis cytosolic ribosomal composition and covalent modifications. In silico digestion of all 409 ribosomal protein sequences in Arabidopsis defined gene&#45;specific peptides for each gene family. We then undertook an extensive MS&#47;MS survey of the cytosolic ribosome and used only high confidence matches to gene&#45;specific peptides for calling gene&#45;specific identifications. This has provided an in&#45;depth analysis of the protein composition based on 1413 high quality MS&#47;MS spectra matching to 776 peptide sequences from ribosomal proteins. These have identified peptides from 5 gene families of r&#45;proteins not identified previously, providing experimental data on 79 of the 80 different types of ribosomal subunits. We provide strong evidence for gene specific identification of 93 different ribosomal proteins from these 79 families. We also provided new information on 33 specific sites of co&#45; and post&#45;translational modification of r&#45;proteins in Arabidopsis by initiator methionine removal, N&#45;terminal acetylation, N&#45;terminal methylation, lysine N&#45;methylation and phosphorylation. This site specific modification data provides a wealth of resources for further assessment of the role of ribosome modification in influencing the control of translation in Arabidopsis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">An Arabidopsis chloroplast&#45;targeting MurE homolog has an essential role in chloroplast development.</div>

<div>

	<span style="text-decoration: underline" >
	Marlon Garcia</span>
	<sup>1</sup>, 

	<span  >
	Fumiyoshi Myouga</span>
	<sup>2</sup>, 

	<span  >
	Katsuaki Takechi</span>
	<sup>1</sup>, 

	<span  >
	Hiroshi Sato</span>
	<sup>3</sup>, 

	<span  >
	Kazuma Nabeshima</span>
	<sup>3</sup>, 

	<span  >
	Noriko Nagata</span>
	<sup>4</sup>, 

	<span  >
	Susumu Takio</span>
	<sup>5</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>2</sup>, 

	<span  >
	Hiroyoshi Takano</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Graduate School of Science and Technology Kumamoto University Kumamoto 860-8555, JAPAN, 

	<sup>2</sup>RIKEN Plant Science Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan, 

	<sup>3</sup>Faculty of Science, Kumamoto University, Kumamoto 860-8555, Japan, 

	<sup>4</sup>Faculty of Science, Women?s University, Mejirodai, Bunkyo-ku, Tokyo 112-8681, Japan., 

	<sup>5</sup>Center for Marine Environment Studies, Kumamoto University, Kumamoto 860-8555, Japan

</div>

<div>Enzyme encoded by bacterial MurE gene catalyzes the ATP&#45;dependent formation of UDP&#45;N&#45;acetylmuramic acid&#45;tripeptide in bacterial peptidoglycan biosynthesis. Genome of Arabidopsis thaliana contains one homologous gene of bacterial MurE &#40;AtMurE&#41;.  We identified one T&#45;DNA&#45;tagged and three Ds&#45;tagged mutant alleles of the AtMurE gene in A. thaliana. All four alleles gave the same pale&#45;green phenotype. Generated antisense mutants for the AtMurE gene in A. thaliana also showed the pale&#45;green phenotype as predicted. These results suggest that AtMurE is related with chloroplast biogenesis. In the mutant cells, development of thylakoid membranes and dispersion of plastid nucleoids, those which are normal developmental events of chloroplasts were inhibited. Under the normal conditions, in mature plants, amount of AtMurE transcripts was abundant in leaves and flowers, while no transcripts were detectable in roots and stems. Corroborating computer predictions, analysis of the GFP fusion proteins with the N terminus and full&#45;length of AtMurE suggests its plastid localization. To analyze functional relationship among MurE genes of cyanobacteria, moss and higher plant, complementation experiments for moss Physcomitrella patens &#40;Pp&#41; MurE knockout line with a small number of macrochloroplasts phenotype were carried out. While Anabaena MurE fused with N terminal region of PpMurE could complement the macrochloroplasts phenotype in P. patens, transformation of AtMurE gene could not show recovery of phenotype. This failure may be because of AtMurE functional divergence to bacterial and moss MurE.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Class XI myosin motors are required for optimal tip growth of root hairs</div>

<div>

	<span style="text-decoration: underline" >
	Eunsook Park</span>
	<sup>1</sup>, 

	<span  >
	Nilou Soltanian</span>
	<sup>1</sup>, 

	<span  >
	Andreas Nebenf&#252;hr</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Tennessee, Department of Biochemistry, Cellular and Molecular Biology Knoxville, TN 37996, USA Email: epark3@utk.edu http://www.bio.utk.edu/cellbiol

</div>

<div> Tip growth is the key growth mechanism on root hairs and pollen tubes. Many kinds of vesicles are trafficking toward &#40;or backward&#41; the apical dome of root hairs to supply membrane and cell wall material as well as energy for growing tips along the shaft of the hair with velocities around 2 to 3 &#181;m&#47;s for <span style="font-style: italic">Arabidopsis thaliana</span>. It suggested that fast movements along actin network are responsible for those trafficking. The relationship of tip growth and acto&#45;myosin cytoskeleton has been extensively studied with pollen tubes.<br/>To investigate the role of these organelle movements in the delivery of secretory vesicles to the root hair tip, we have isolated mutants in myosin genes. Arabidopsis genome encoded 13 class XI myosin genes. Mutations of the most conserved two genes resulted in reduced root hair elongation. Disruption of the <span style="font-style: italic">XI&#45;K</span> gene by T&#45;DNA insertion resulted in shorter root hairs with only 70&#37; the length of WT. Another mutant, <span style="font-style: italic">mya1</span>, did not show the shorter root hair phenotype by itself, however, in combination with <span style="font-style: italic">xi&#45;k</span> caused even shorter root hairs &#40;45&#37; of WT&#41;. It suggested that organelle movements in the root hairs of the mutants might be impaired. Interestingly, cytoplasmic streaming velocities in single and double mutant root hairs are within in the same range as in wild type when observed with DIC optics. This preliminary observation suggested that only the movements of specific organelles might be impaired in mutant cells. To identify which organelle movements and distributions are defect in mutants, we have transformed a number of fluorescent organelle markers, for example Golgi&#45;YFP, into the mutants and wild type. Preliminary evidence suggests that the distribution of YFP&#45;RabA4b&#45;labeled vesicles is abnormal in the mutants. A detailed observation and analysis of the distribution of several other organelles in WT and mutant root hairs might be able to give the clue for the role of organelle trafficking on tip growth of root hair cells. This work is supported by NSF grant MCB&#45;0416931.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">MIPS, the missing link between cell cycle and cell death.</div>

<div>

	<span style="text-decoration: underline" >
	Ping-Hong Meng</span>
	<sup>1</sup>, 

	<span  >
	Sophie Blanchet</span>
	<sup>1</sup>, 

	<span  >
	C&#233;cile Raynaud</span>
	<sup>1</sup>, 

	<span  >
	Catherine Bergounioux</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CNRS, Universit&#233; Paris-sud, IBP, France

</div>

<div>The Proliferating Cell Nuclear Antigen &#40;PCNA&#41; functions as a sliding clamp for the DNA polymerase, and is thus a key actor of DNA replication. It is also involved in DNA repair, maintenance of heterochromatic regions throughout replication, cell cycle regulation and Programmed Cell Death. We have precedently identified two Arabidopsis SET&#45;domain proteins that interact with PCNA: ATXR5 and ATXR6. ATXR5 possesses a double localisation in plastids and in nucleus and its subcellular localization is regulated via the presence&#47;absence of its chloroplast&#45;targeting sequence. ATXR6 is solely nuclear, up&#45;regulated during S&#45;phase, a direct target of the AtE2Fb transcription factor and is required for S&#45;phase progression. The over&#45;expression of either ATXR5 or ATXR6 causes male sterility due to the degeneration of defined cell types. A second round of two hybrid screen using ATXR5 as a bait has allowed the identification of ATMIPS as one of the AtXR5 partners. MIPS myo&#45;inositol phosphate synthase and IMP myo&#45;inositol monophosphatase catalyse reactions in the de novo inositol biosynthesis pathway. Scavenge inositol phosphates from the phosphotidylinositol signaling pathway and IP6 breakdown also provide inositol for intracellular pools. In this study, we have investigated the function of the AtMIPS1 gene in Arabidopsis. We confirmed the existing of interaction between AtMIPS1 and AtXR5&#47;6 in Yeast and in living plant cells. We found that two mutants Atmips alleles display a same cell death phenotype. This cell death phenotype was molecularly biochemicaly and cytologicaly analysed. Our data provide new insight to link cell cycle to cell death in plant.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A screen for interaction specificity between RAC&#47;ROP GTPases and their partners in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Tore Brembu</span>
	<sup>1</sup>, 

	<span  >
	Atle Bones</span>
	<sup>1</sup>, 

	<span  >
	Zhenbiao Yang</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Norwegian University of Science and Technology, Trondheim, Norway, 

	<sup>2</sup>University of California, Riverside, USA

</div>

<div>The RAC&#47;ROP family of small GTPases has emerged as master regulators of numerous cellular and developmental processes in plants, including polarised cell growth, hormone responses and plant defense. Previous screens for RAC&#47;ROP interacting partners have identified four proteins families regulating RAC&#47;ROP activity, as well as a number of effector protein families. Several of these protein families appear to be plant specific. The Arabidopsis RAC&#47;ROP family consists of 11 members, and a majority of the RAC&#47;ROP interacting proteins are also encoded by multi&#45;gene families. To date, no studies have been made to investigate whether individual plant RAC&#47;ROP GTPases show differences in binding preferences to their interacting partners. Identifying specific protein&#45;protein interactions will be of great help in elucidating the mechanisms of RAC&#47;ROP signaling pathways. In order to address this issue, we have used a high&#45;throughput protein interaction assay, based on liquid yeast two&#45;hybrid assays performed in 96&#45;well plates. Differences in binding specificity were observed, especially between type I and type II RAC&#47;ROPs. Selected interaction pairs were confirmed using FRET&#47;BiFC and coimmunoprecipitation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular Analysis of a Family of Arabidopsis Genes Related to Galacturonosyltransferases</div>

<div>

	<span style="text-decoration: underline" >
	yingzhen kong</span>
	<sup>1</sup>, 

	<span  >
	Michael Hahn</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Complex Carbohydrate Research Center, University of Georgia, 315 Riverbend Road, Athens, GA  30602-4712, USA, 

	<sup>2</sup>.  Department of Plant Biology, University of Georgia, Athens, GA  30602

</div>

<div>Pectin is a major component of the cell wall polysaccharides of all land plants and is made up of a range of galacturonic acid &#40;GalA&#41;&#45;containing polysaccharides.  Pectic polysaccharides play important roles in plant cell growth and development, and responses to pathogens.  A complete understanding of the role&#40;s&#41; of pectic polysaccharides in plant biology will require a complete understanding of their biosynthesis.  We are studying a gene family in Arabidopsis thaliana that was identified bioinformatically as being closely related to a group of 15 genes &#40;GAUT1&#45;15&#41;, one of which &#40;GAUT1&#41; has been shown to encode a functional galacturonosyltransferase. The proteins encoded by the ten GATL &#40;GAlacturonosylTransferase&#45;Like&#41; genes are smaller than those encoded by the GAUT genes and appear to lack a transmembrane domain, but do contain a signal sequence that suggests the targeting of GATL proteins into the endomembrane&#47;secretory pathways.  Expression studies &#40;RT&#45;PCR&#41; demonstrate that all ten GATL genes are transcribed, albeit to varying degrees, in Arabidopsis tissues. Promoter::b&#45;glucuronidase studies provided high resolution information about cell&#45;specific locations of GATL gene expression. Nine of the ten GATL genes are almost expressed in all major plant organs. However, GATL4 expression appears confined to pollen grains. Most of the GATL genes are expressed strongly in xylem cells in both stem and root sections. T&#45;DNA insertion mutants in several GATL genes display reduced GalA content in total cell wall material and two show abnormalities in seed coat mucilage, which is composed primarily of pectin. The sub&#45;cellular localization of several GATL proteins has been investigated by using green fluorescent protein &#40;GFP&#41; tagging, and the results provide evidence supporting Golgi localization for these proteins. These data suggest that GATL genes are involved in the biosynthesis of pectin by acting as galacturonosyltransferase.  &#91;This work is supported by grants from the NSF Plant Genome Program &#40;DBI&#45;0421683&#41; and DOE &#40;DE&#45;FG02&#45;96ER20220&#41;.&#93;</div>

</div>





<div class="abs">

<div style="font-weight:bold">Transgene&#45;induced Phytochrome A silencing and epiallele formation in 'Arabidopsis' 'thaliana'.</div>

<div>

	<span style="text-decoration: underline" >
	Scott Nicholson</span>
	<sup>1</sup>, 

	<span  >
	Rekha Chawla</span>
	<sup>2</sup>, 

	<span  >
	Vibha Srivastava</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Arkansas, 

	<sup>2</sup>Indiana University-Purdue University Indianapolis

</div>

<div>Strong silencing of Phytochrome A &#40;'PHYA'&#41; was found among a population of 'Arabidopsis' 'thaliana' transformed with a t&#45;DNA construct not intended to produce silencing. Two non&#45;transgenic epimutant lines were isolated from a transgenic line.  Several analyses were conducted upon the transgenic and epimutant lines to determine the basis of silencing.  Both transgenic and epimutant lines were screened for the transgene; the epiallele was found to have segregated away from the transgene while the silenced transgenic lines each contained several transgene loci.  'PHYA' transcript level was much reduced in transgenic and epimutant lines compared to wild type &#40;wt&#41;; the greatest reduction &#40;&#126;80&#37;&#41; was in the epimutants.  'PHYA' siRNA were detected in several of the transgenic lines, indicative of post&#45;transcriptional silencing &#40;PTGS&#41;.  Methylation analysis revealed promoter and coding sequence methylation in transgenic lines, also indicative of PTGS, while the epimutants displayed methylation only within the coding region, not in the promoter as typical in transcriptional silencing &#40;TGS&#41;.  Reduction of coding region methylation by crossing the epimutant with a CG methyltransferase&#45;deficient 'met1' line resulted in recovery of wild&#45;type progeny. Further, heterochromatic markers were found to be associated with 'phyA' exon 1 in the epimutant, but not with other regions of the gene. Nuclear run&#45;on analysis showed a lower level of 'PHYA' transcript in the epimutants than in wt, confirming that TGS was the cause of silencing in the epimutant.  As these findings deviate from previous descriptions of TGS, a novel method of TGS maintenance may have been discovered.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional characterization of the MAP65 family of microtubules associated proteins</div>

<div>

	<span style="text-decoration: underline" >
	Marie-Theres Hauser</span>
	<sup>1</sup>, 

	<span  >
	Roland Baumgartner</span>
	<sup>1</sup>, 

	<span  >
	Vera Vordermaier</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>BOKU-- University of Natural Resources and Applied Life Sciences Vienna

</div>

<div>The phylogenetically conserved family of MAP65s are non&#45;motor microtubule&#45;associated proteins that stabilize microtubules &#40;MT&#41; and regulate their dynamics. Lack of MAP65s causes cell division defects at the anaphase spindle, the central spindle midzone or the phragmoplast in yeast, C. elegans, Drosophila, human cell lines and plants, respectively. Their MT&#45;binding and &#45;bundling activity has been demonstrated for yeast &#40;Ase1&#41;, plant &#40;MAP65&#41; and human &#40;PRC1, protein regulating cytokinesis1&#41; homologs. At the N&#45;terminus, MAP65s have a dimerization domain and at the C&#45;terminus a conserved MT&#45;binding domain &#40;Smertenko et al., 2004&#41;. Phosphorylation of PRC1&#47;MAP65 at CDK &#40;cyclin dependent kinase&#41; sites of the human homolog and CDK&#47;MAPK &#40;mitogen&#45;activated protein kinase&#41; sites of the tobacco homolog appears to be important for suppressing MT&#45;bundling activity and progression through cytokinesis. <br/>The Arabidopsis genome codes for nine MAP65 proteins. We have previously shown that ple&#47;map65&#45;3 mutants form improper phragmoplasts and exhibit strong cytokinesis defects in roots &#40;M&#252;ller et al. 2004&#41;. Here we present the results of a reverses genetic approach to determine the function of the other MAP65 members. Since most of the MAP65s are co&#45;expressed during development we hypothesized that they can form homo&#45; and heteromers. Thus, the BiFC &#40;biomolecular fluorescence complementation&#41; system was exploited in transient protoplasts to determine the subcellular localization of homo&#45; and heteromerization and its effect on MT stabilization. The genetic analyses of single mutants show that most of the MAP65 are dispensable for normal development suggesting that the loss of one MAP65 can be substituted by its paralogs. To determine which MAP65s are functionally equivalent and able to replace each other, double mutant analyses have been initiated. The results of the genetic and cell biological analyses will be presented and discussed. <br/><br/>Smertenko et al.&#40;2004&#41;Plant Cell, 16, 2035&#45;2047<br/>M&#252;ller et al. &#40;2004&#41; Curr. Biol. 14, 412&#45;417<br/>This project was supported by the Austrian Science Fund &#40;FWF&#41; project No. P16410&#45;B12.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Interaction and functional characterization of ESCRT&#45;III complex components of Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Marie-Theres Hauser</span>
	<sup>1</sup>, 

	<span  >
	Verena Winter</span>
	<sup>1</sup>, 

	<span  >
	Nicole Schlager</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>BOKU-University of Natural Resources and Applied Life Sciences Vienna

</div>

<div>The ESCRT &#40;endosomal sorting complex required for transport&#41; machinery consists of three hetero&#45;oligomeric protein complexes involved in sorting of monoubiquitinylated membrane proteins into late endosomal compartments. The ESCRT machinery has been extensively studied in yeast and animal cells and bioinformatic analyses indicate that the components are also present in plants &#40;Winter and Huaser, 2006&#41;. Here we present in vivo protein&#45;protein interaction analyses using the yeast two hybrid &#40;Y2H&#41; and the bimolecular fluorescence complementation &#40;BiFC&#41; methods. ESCRT&#45;III components interact at vesicular structures in the cytoplasm of protoplasts and partially co&#45;localize with the endosomal marker FM4&#45;64. Moreover, we further show that the previously isolated mutant, hyade, which develops root specific cytokinesis defects is a plant specific homolog of an ESCRT&#45;III component. This phenotype is reminiscent to the recently identified elch mutant which turned out to be an Arabidopsis homolog of the ESCRT&#45;I component VPS23 &#40;Spitzer et al, 2006&#41;. hyade mutants stably transformed with a gHYA::GFP construct are complemented and exhibit a wildtype phenotype. gHYA::GFP localizes to similar cytoplasmic structures as the BiFC interactions primarily in the meristematic and cell expansion zone and weakly in the vasculature of mature roots. The link between the ESCRT&#45;machinery and cytokinesis will be discussed. &#13;<br/>&#13;<br/>Winter ans Hauser M&#45;T &#40;2006&#41; Trends Plant Sci 11, 115&#45;123&#13;<br/>M&#252;ller et al.,&#40;2002&#41; Plant Physiol 13, 312&#45;324&#13;<br/>Spitzer et al., &#40;2006&#41; Development. 133, 4679&#45;4689&#13;<br/>&#13;<br/>This project is supported by the Austrian Science Fund &#40;FWF&#41; project No. P16410&#45;B12.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis pri&#45;miRNA processing proteins HYL1 and DCL1 define a nuclear body distinct from the Cajal body</div>

<div>

	<span style="text-decoration: underline" >
	Liang Song</span>
	<sup>1</sup>, 

	<span  >
	Nina Fedoroff</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Plant Biology Program, The Pennsylvania State University, University Park, PA, USA, 

	<sup>2</sup>Biology Department and Huck Institute of the Life Sciences, The Pennsylvania State University, University Park, PA, USA

</div>

<div>Small regulatory miRNAs are encoded in long precursors and released from them during processing by cleavage within partially duplexed stem&#45;loop structures. In the present work, we investigated the role of the Arabidopsis nuclear RNA&#45;binding protein HYL1 and the nuclear RNAse III enzyme DCL1 in processing of pri&#45;miR171a.  The miR171a gene is complex, with multiple transcription start sites, as well as alternative splicing of exons and alternative polyadenlyation sites. Both HYL1 and DCL1 proteins are required for processing of the major pri&#45;miR171a, spliced and polyadenylated forms of which accumulate in plants homozygous for mutations in either gene, but not in wildtype plants. In transiently transfected Arabidopsis protoplasts, HYL1&#45;mCherry and YFP&#45;DCL1 fusion proteins co&#45;localize to small nuclear bodies similar to Cajal bodies, but lacking the Cajal body marker Atcoilin or U2B&#34;. The HYL1&#45;DCL1 bodies do not co&#45;localize well with SE bodies, nor do SE bodies co&#45;localize with AtCoilin or U2B&#34; labeled Cajal bodies. The distinct HYL1&#45; and DCL1&#45;containing nuclear bodies may be miRNA precursor processing sites.  Alternatively, they may be assembly and storage sites for the miRNA precursor processing machinery.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Variation and change in the cell wall glycome of <span style="font-style: italic">Arabidopsis thaliana</span> during growth, development, and differentiation</div>

<div>

	<span style="text-decoration: underline" >
	Michael G. Hahn</span>
	<sup>1</sup>, 

	<span  >
	Glenn Freshour</span>
	<sup>1</sup>, 

	<span  >
	Anathea Albert</span>
	<sup>1</sup>, 

	<span  >
	David Baldwin</span>
	<sup>1</sup>, 

	<span  >
	Ruth Davis</span>
	<sup>1</sup>, 

	<span  >
	Malcolm A. O'Neill</span>
	<sup>1</sup>, 

	<span  >
	Sivakumar Pattathil</span>
	<sup>1</sup>, 

	<span  >
	Sami Tuomivaara</span>
	<sup>1</sup>, 

	<span  >
	William S. York</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Georgia, Athens, GA, USA

</div>

<div>The cell walls of plants play a prominent role in determining the structure and shape of individual cells, and ultimately the morphology of the plant.  In addition, cell wall components play active roles in plant development and responses to the environment.  Chemical studies have given an overall picture of the structure and organization of the major wall polymers.  However, these analyses do not give complete information about wall structure and dynamics at the cellular and sub&#45;cellular levels.  Antibodies are specific and sensitive tools to define the composition of cell walls at the cellular level and to monitor changes in those walls as a function of plant growth and development.  An expanded set of over 100 monoclonal antibodies is now available that bind to diverse carbohydrate epitopes residing on all major cell wall polysaccharides in plants, including hemicelluloses, pectins, and arabinogalactans.  We have used antibodies generated in our laboratory &#40;CCRC series&#41;, and those generated by others &#40;MAC, JIM and LM series&#41;, for immunofluorescent and immunogold electron microscopic studies in wild&#45;type <span style="font-style: italic">Arabidopsis thaliana</span>.  Tissue sections examined were taken from leaves and flowers, and from different points along roots and stems.  The labeling patterns observed show a high degree of diversity in cell&#45; and tissue&#45;specific localizations of carbohydrate structures in cell walls and organelles.  In addition, complex patterns of appearance and disappearance of glyco structures in the walls of specific cell types were observed along developmental gradients in the cells and tissues examined.  These results provide new insights into the diversity and plasticity of the glycome of plant cell walls.  Furthermore, the observed localization patterns serve as a baseline for biochemical and mutagenic approaches toward understanding the functional roles of plant cell wall carbohydrate components in the biology of plant cells.  <span style="font-style: italic">&#91;Supported by grants from DOE &#40;DE&#45;FG02&#45;96ER20220 and DE&#45;FG02&#45;93ER20097&#41; and NSF &#40;DBI&#45;0421683 and RCN&#45;0090281&#41;.&#93;</span></div>

</div>





<div class="abs">

<div style="font-weight:bold">The mitotic checkpoint in Arabidopsis thaliana</div>

<div>

	<span  >
	Joseph Ramahi</span>
	<sup>1</sup>, 

	<span  >
	Jodi Stewart</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Simon Chan</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>UC Davis, Davis, California, USA

</div>

<div>The mitotic checkpoint &#40;also known as the spindle assembly checkpoint&#41; monitors the progress of mitosis, delaying the onset of anaphase until each chromosome is correctly attached to the spindle. Unattached kinetochores use the Mad1 and Bub3 proteins to bind the mitotic checkpoint components Mad2 and BubR1. A Mad2&#47;Bub3&#47;BubR1 complex is then released in a form that can inhibit the anaphase promoting complex cofactor Cdc20. <br/><br/>Although the mitotic checkpoint has been studied extensively in budding yeast, yeast kinetochores require sequence&#45;specific interactions with a 125bp centromere DNA and are thus very different from plant and animal kinetochores that assemble on 100s of kB of DNA and are epigenetically specified. Mouse mutants in Mad1, Mad2 and BubR1 are embryonic lethal, making it difficult to genetically analyze the mitotic checkpoint in an organism with a large, complex centromere.<br/><br/>In Arabidopsis thaliana, MAD1 and MAD2 are encoded by single genes. Interestingly, homozygous mad1 and mad2 T&#45;DNA mutants are viable and fertile, demonstrating that the mitotic checkpoint can be dispensable for normal development in a multicellular eukaryote &#40;bub3 and bubR1 mutants are also viable and fertile, but Arabidopsis has multiple homologs of these proteins&#41;. Viable mitotic checkpoint mutants present the opportunity to study how epigenetic changes in a complex centromere&#47;kinetochore affect the mechanism of chromosome segregation. The effects of mad1 and mad2 mutants on mitotic chromosome dynamics, sensitivity to microtubule destabilizing drugs, and genetic interactions with epigenetic mutants will be presented. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification and characterization of components of G beta&#47;gamma signaling in Arabidopsis thaliana: NDR gene family</div>

<div>

	<span  >
	yashwanti mudgil</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Alan Jones</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of North Carolina, Chapel Hill, USA

</div>

<div>In animals, the basic units of the heterotrimeric G&#45;protein signaling network include a cell&#45;surface receptor &#40;GPCR&#41;, a coupling complex comprised of alpha, beta, and gamma subunits &#40;G protein&#41;, and a downstream target &#40;effector&#41;. Sequential activation of these G protein components result in changes in cellular activities such as ion flux, protein trafficking, and gene expression. Plants also utilize signaling through G proteins but there are few candidates for the upstream GPCRs and the downstream effectors. In the current study, we are seeking downstream effectors of G protein signaling. Toward this, we performed a yeast three&#45;hybrid screen using Arabidopsis G beta1 and G gamma1&#47;or gamma2 subunits as baits to screen for physical interactors in an Arabidopsis cDNA expression library. We found and confirmed interaction between the G beta&#45;gamma2 dimer and a protein of unknown function designated NDR &#40;N&#45;Myc Down Regulated&#41; in animals. The sunflower NDR homolog, designated SF21, is expressed the highest in pollen and in the transmitting tissues of pistils, and it shows multiple alternative and organ&#45;specific splicing transcripts. Arabidopsis has three NDR proteins &#40;AtNDR1, 2 and 3&#41; which are highly similar, between them NDR2 and 3 are predominately expressed in pollen. All three members interact with G beta&#45;gamma2 in the yeast 3&#45;hybrid configuration. Over expression studies indicate its role in vegetative branching and flower development. Biochemical characterization and down regulation studies by micro RNA in wild type and other G protein component mutant background are in progress to further dissect out its function in G beta&#45;gamma signaling.</div>

</div>





<div class="abs">

<div style="font-weight:bold">GAP&#45;limited cycling in Arabidopsis G protein&#45;coupled sugar signaling</div>

<div>

	<span  >
	Christopher Johnston</span>
	<sup>1</sup>, 

	<span  >
	Philip Taylor</span>
	<sup>1</sup>, 

	<span  >
	Yajun Gao</span>
	<sup>2</sup>, 

	<span  >
	Adam Kimple</span>
	<sup>1</sup>, 

	<span  >
	Jin-Gui Chen</span>
	<sup>2</sup>, 

	<span  >
	David Siderovski</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Alan Jones</span>
	<sup>1</sup>, 

	<span  >
	Francis Willard</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of North Carolina, 

	<sup>2</sup>University of British Columbia

</div>

<div>Heterotrimeric G&#45;protein signaling is important for cell&#45;proliferative and glucose&#45;sensing pathways in Arabidopsis. AtRGS1 is a seven&#45;transmembrane, RGS&#45;domain containing protein that is a putative membrane receptor for D&#45;glucose. Using FRET, we show that D&#45;glucose alters the interaction between AtGPA1 and AtRGS1 in vivo. AtGPA1 is a unique G alpha subunit that is constitutively GTP bound given high spontaneous nucleotide exchange coupled with slow GTP hydrolysis. Analysis of a point mutation in AtRGS1 that abrogates GAP activity demonstrates that regulation of AtGPA1 GTP hydrolysis mediates sugar signal transduction during Arabidopsis development, in contrast to animals where nucleotide exchange is the limiting step.</div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">Arabidopsis</span> ABP30.6, a novel actin&#45;bundling protein, is phosphorylated by AtMAPK6&#42;</div>

<div>

	<span style="text-decoration: underline" >
	Yuan Li</span>
	<sup>1</sup>, 

	<span  >
	Yongdun Xie</span>
	<sup>1</sup>, 

	<span  >
	Lihui Zhang</span>
	<sup>1</sup>, 

	<span  >
	Guoqin Liu</span>
	<sup>1</sup>, 

	<span  >
	Dongtao Ren</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>China Agricultural University, College of Biological Sciences,Beijing 100094, China

</div>

<div>The actin cytoskeleton is a highly organized and dynamic structure present in all eukaryotic cells where it plays important role in many processes, including intracellular transport, cell growth, division and morphogenesis, or responses to environmental stimuli. In cells, the formation of higher&#45;order structures, such as bundles and cables, is regulated by specific families of actin&#45;binding proteins &#40;ABPs&#41;.  However, only a few ABPs with actin&#45;bundling activity were found in plant cells.  Here we reported the identification and characterization of ABP30.6, a novel actin&#45;bundling protein in <span style="font-style: italic">Arabidopsis</span>.  Low&#45;speed and high&#45;speed cosedimentation assays shown that the recombinant ABP30.6 was binding to F&#45;actin with high affinity &#40; <span style="font-style: italic">Kd</span> &#61;0.41 uM &#41; and inducing actin&#45;bundles formation.  Using truncated proteins, the N&#45;terminus of ABP30.6 protein, was shown to be required for actin bundles formation. The effect of ABP30.6 on depolymerization of pyrene&#45;labeled F&#45;actin indicated that, ABP30.6 stabilized actin filaments in a concentration&#45;dependent manner.  Interestingly, we found that ABP30.6 was strongly phosphorylated by activated mitogen&#45;activated protein kinase, AtMAPK6 in <span style="font-style: italic">in vitro</span> protein phosphorylation assay system.  The phosphorylated&#45;ABP30.6 shown a higher actin&#45;bundling activity.  Using transient transformation system, ABP30.6 was found to accumulate when co&#45;expressed ABP30.6 with the active mutant of AtMEK5, an upstream MAPK kinase of AtMAPK6.  The results suggested that phosphorylation of ABP30.6 may be required for fully activation of its function and ability of stabilization <span style="font-style: italic">in vivo</span>. &#13;<br/>&#42; Correspondence should be addressed to DT Ren, Email: ren&#64;cau.edu.cn&#13;<br/>This work was supported by the State Basic Research Program &#40;Grant No.2003CB114304&#41;, National Natural Science Foundation of China &#40;Grant No. 30421002, 30270064, 30370140&#41;and NCET 04&#45;0131 to DT Ren </div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic analysis of the protein import machinery of Arabidopsis plastids</div>

<div>

	<span style="text-decoration: underline" >
	Paul Jarvis</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Leicester, UK

</div>

<div>While plastids retain a fully&#45;functional genetic system, the plastid genome encodes less than 10&#37; of the proteins required to build a fully&#45;functional organelle. The majority of plastidic proteins are encoded in the nucleus and translated on free cytosolic ribosomes. They are synthesized in precursor form, each one bearing an amino&#45;terminal targeting signal, or transit peptide. The transit peptide directs the protein through a post&#45;translational targeting pathway, and is cleaved upon arrival inside the plastid. This targeting or import process is mediated by the coordinate action of two proteinaceous import machines, one in each of the envelope membranes. The import machinery of the outer envelope membrane is called the TOC complex, and that in the inner membrane is called the TIC complex. Over the last decade, several components of the TOC and TIC complexes have been identified using biochemical approaches and isolated pea chloroplasts. Interestingly, many of these components &#40;particularly receptor components of the TOC complex&#41; have been found to have multiple homologues in Arabidopsis. We have used genetic approaches to dissect the functional significance of these different TOC protein isoforms. Our results suggest that the different isoforms operate in different import pathways with distinct precursor recognition specificities; i.e., different import pathways exist for different precursor protein classes. The existence of such substrate&#45;specific import pathways might play a role in the differentiation of different plastid types, and act to prevent deleterious competition effects between abundant and non&#45;abundant precursors.</div>

</div>





<div class="abs">

<div style="font-weight:bold">An Allele&#45;Specific Suppressor of a Defective Brassinosteroid Receptor</div>

<div>

	<span  >
	Hua Jin</span>
	<sup>1</sup>, 

	<span  >
	Zhenyan Yan</span>
	<sup>1</sup>, 

	<span  >
	Kyoung-Hee Nam</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Jianming Li</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Michigan, Ann Arbor, MI, USA

</div>

<div>BRI1 is a leucine&#45;rich&#45;repeat receptor&#45;like kinase that functions as a cell surface receptor for brassinosteroids &#40;BRs&#41; and its mutations often lead to many developmental defects including dwarfism, delayed flowering, and reduced male fertility. The binding of active BRs to BRI1 triggers a phosphorylation&#45;mediated signaling cascade that regulates the amount, subcellular localization, and&#47;or DNA&#45;binding activities of several plant&#45;specific transcriptional factors. To identify additional components of the BR signaling pathway, we conducted a genetic screen for extragenic suppressors of the <span style="font-style: italic">bri1&#45;9</span> mutation that changes Ser662 to Phe in the newly defined BR&#45;binding domain and identified many wild&#45;type looking plants. One of them, named as <span style="font-style: italic">ebs1</span> for <span style="font-style: italic"><span style="text-decoration: underline">E</span>MS&#45;mutagenized <span style="text-decoration: underline">b</span>ri1 <span style="text-decoration: underline">s</span>uppressor <span style="text-decoration: underline">1</span></span>, exhibits near wild&#45;type morphology throughout its life cycle, including a large rosette, a long hypocotyl in the dark, and long inflorescence stems at maturity. However, the <span style="font-style: italic">ebs1</span> mutation fails to suppress the phenotypes of <span style="font-style: italic">bin2</span> and <span style="font-style: italic">det2</span> mutants defective in the intracellular signaling and biosynthesis of BRs, respectively, indicating that <span style="font-style: italic">ebs1</span> does not cause a constitutive activation of BR signaling. We suspected that the <span style="font-style: italic">ebs1</span> mutation might directly affect BR perception at the cell surface. Indeed, molecular and physiological assays showed that the <span style="font-style: italic">ebs1</span> mutation restored BR sensitivity to the <span style="font-style: italic">bri1&#45;9</span> mutants, while the double mutant analysis of <span style="font-style: italic">ebs1</span> and several other known <span style="font-style: italic">bri1</span> alleles revealed an allele&#45;specific interaction between <span style="font-style: italic">ebs1</span> and <span style="font-style: italic">bri1&#45;9</span>, implying that <span style="font-style: italic">ebs1</span> does not activates an alternative BR receptor but directly acts on bri1&#45;9 to restore its receptor function. The molecular identity of the <span style="font-style: italic">EBS1</span> gene and the underlying biochemical mechanism by which the <span style="font-style: italic">ebs1</span> mutation restores BR sensitivity to the defective BR receptor will be discussed at the meeting. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Plasmodesmata: Pathways for Intercellular Trafficking of Macromolecules&#13;<br/>&#13;<br/>&#13;<br/>&#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	William Lucas</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Davis, CA, USA

</div>

<div>Plasmodesmata &#40;PD&#41; establish a pathway for cell&#45;to&#45;cell communication within plant tissues and organs.  Recent studies have demonstrated that PD evolved the capacity to mediate the cell&#45;to&#45;cell transport of macromolecules, including proteins and RNA.  This gave rise to a system in which positional information could be delivered by non&#45;cell&#45;autonomously acting macromolecules, without passage into the extracellular milieu.  Initial insight into this unique function was gained through studies on viruses. Plant viruses encode non&#45;structural proteins, termed movement proteins that function in the transport of viral infectious material through PD, thereby allowing the spread of infectious nucleic acids. These findings led to the discovery that many plant proteins, including transcription factors, are able to use this PD &#8216;cytoplasmic highway&#8217; to traffic between cells. A number of these endogenous proteins have the capacity to potentiate the cell&#45;to&#45;cell transport of mRNA.  Interestingly, proteins that can gain entry into the plant vascular system, and the phloem in particular, are able to move over long distances within the body of the plant.  This special property of PD, to mediate the exchange of information molecules, led to the concept that higher plants function as supracellular organisms. Now that it has been unambiguously established that endogenous macromolecules have the capacity to function at a supracellular level, the challenge ahead is to identify the cellular components involved in both mediating and regulating this non&#45;cell&#45;autonomous protein translocation pathway. The focus of this talk will be on emerging principles of PD biology in areas of cell biology, developmental biology and inter&#45;organ signaling. &#40;Funding provided by NSF grant IBN 0444725 and DOE grant DE&#45;FG03&#45;94ER20134&#41;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Canonical SRP components can be bypassed for post&#45;translational protein targeting in chloroplasts</div>

<div>

	<span  >
	Tzvetelina Tzvetkova-Chevolleau</span>
	<sup>1</sup>, 

	<span  >
	Claire Hutin</span>
	<sup>1</sup>, 

	<span  >
	Laurent D. Noel</span>
	<sup>1</sup>, 

	<span  >
	Robyn Goforth</span>
	<sup>2</sup>, 

	<span  >
	Jean-Pierre Carde</span>
	<sup>3</sup>, 

	<span  >
	Stephano Caffarri</span>
	<sup>1</sup>, 

	<span  >
	Neil E. Hoffman</span>
	<sup>4</sup>, 

	<span  >
	Ralph Henry</span>
	<sup>2</sup>, 

	<span  >
	Michel Havaux</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Laurent Nussaume</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CEA, 

	<sup>2</sup>University of Arkansas, 

	<sup>3</sup>INRA Centre de Bordeaux, 

	<sup>4</sup>APHIS/USDA

</div>

<div>The chloroplast Signal Recognition Particle &#40;cpSRP&#41; and its receptor &#40;cpFtsY&#41;  targets proteins both co&#45;translationally and post&#45;translationally to the thylakoids. This dual function enables cpSRP to utilize its posttranslational activities for targeting a family of nuclear encoded light&#45;harvesting chlorophyll&#45;binding proteins &#40;LHCPs&#41;, the most abundant membrane proteins in plants. Previous in vitro experiments indicated an absolute requirement for all cpSRP pathway soluble components. In agreement, a cpFtsY mutant in Arabidopsis exhibits a severe chlorotic phenotype resulting from a massive loss of LHCPs. Surprisingly, a double cpftsy&#47; cpsrp54 mutant, recovers to a great extent from the chlorotic cpftsy phenotype. This establishes that in plants a new alternative pathway exists that can bypass cpSRP post&#45;translational targeting activities. Using a mutant form of cpSRP43 that is unable to assemble with cpSRP54, we complemented the cpSRP43&#45;deficient mutant and found that this subunit is required for the alternative pathway. Along with the ability of cpSRP43 alone to bind the ALB3 translocase required for LHCP integration, our results indicate that cpSRP43 has developed features to function independently of cpSRP54&#47;cpFtsY in targeting LHCPs to the thylakoid membranes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">MAIGO2 is involved in exit of seed storage proteins from the ER in &#34;Arabidopsis thaliana&#34;</div>

<div>

	<span style="text-decoration: underline" >
	Lixin Li</span>
	<sup>1</sup>, 

	<span  >
	Tomoo Shimada</span>
	<sup>1</sup>, 

	<span  >
	Hideyuki Takahashi</span>
	<sup>1</sup>, 

	<span  >
	Haruko Ueda</span>
	<sup>1</sup>, 

	<span  >
	Yoichiro Fukao</span>
	<sup>2</sup>, 

	<span  >
	Maki Kondo</span>
	<sup>3</sup>, 

	<span  >
	Mikio Nishimura</span>
	<sup>3</sup>, 

	<span  >
	Ikuko Hara-Nishimura</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Graduate School of Science, Kyoto University, Kyoto, Japan, 

	<sup>2</sup>Graduate School of Biological Sciences, Nara Institute of Science and Technology, Japan, 

	<sup>3</sup>Department of Cell Biology, National Institute for Basic Biology, Okazaki, Japan

</div>

<div>In higher plants, seed storage proteins are synthesized on the endoplasmic reticulum &#40;ER&#41; as precursors and then are transported to protein storage vacuoles, where they are processed into mature forms. We isolated and investigated Arabidopsis mutants &#40;&#34;maigo&#34;&#41; that abnormally accumulated the precursors of storage proteins in dry seeds &#40;1&#41;. &#34;maigo2&#34; mutant abnormally accumulated precursors of two major storage proteins, 2S albumin and 12S globulin, in dry seeds &#40;2&#41;. &#34;mag2&#34; seed cells contained many novel structures with an electron&#45;dense core composed of the precursors of 2S albumin. 12S globulins were localized in the matrix region of the structures together with ER chaperones, BiP and PDI, which were more abundant in &#34;mag2&#34; seeds. MAG2 protein had a RINT&#45;1&#47;TIP20 domain in the C&#45;terminal region. Some MAG2 molecules were peripherally associated with the ER membrane. MAG2 had an ability to bind to two ER&#45;localized t&#45;SNAREs &#40;At Sec20 and At Ufe1&#41;. Our findings suggest that MAG2 functions in the transport of storage protein precursors between the ER and the Golgi complex in plants.&#13;<br/>&#40;1&#41; Shimada et al. &#40;2006&#41; Plant Cell Physiol., 47&#40;9&#41;: 1187&#45;1194&#13;<br/>&#40;2&#41; Li et al., Plant Cell &#40;2006&#41; Dec;18 &#40;12&#41;: 3535&#45;3547. &#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">SWA2 encodes a nucleolar protein which is essential for female gametophyte development in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Na Li</span>
	<sup>1</sup>, 

	<span  >
	Li Yuan</span>
	<sup>1</sup>, 

	<span  >
	Jie Liu</span>
	<sup>1</sup>, 

	<span  >
	Weicai Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Key Laboratory of Molecular and Developmental Biology, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Datun Road, Beijing 100101, China

</div>

<div>Female gametogenesis is a fundamental step in sexual plant reproduction. One of the four meiotic products differentiates to form functional megaspore which undergoes three consecutive rounds of mitotic division to produce an eight&#45;nucleate embryo sac. Subsequently, cytokinesis occurs simultaneously in the coenocytic embryo sac to form a seven&#45;celled female gametophyte. Often the embryo sac development within a single pistil is coordinated and synchronous. Mechanisms controlling these division cycles and the coordinated development are not yet known. Here we report that SWA2 is essential for the progression of gametophytic division cycles in Arabidopsis thaliana. In swa2 mutant, the progression of the mitotic division cycles and the synchrony of female gametophyte development were impaired, causing arrest of mutant female gametophyte at two&#45;, four&#45; or eight&#45;nucleate stages. Delayed pollination test showed that some of these mutant ovules were able to develop into functional embryo sacs and could be fertilized. SWA2 encodes a nucleolar protein which can physically interact with a putative Arabidopsis NOC2 homologue in yeast two&#45;hybrid assay. SWA2 is expressed ubiquitously throughout the plant, at high level in meristems and gametophytes. These data suggest that SWA2 is essential for the mitotic progression of the female gametophyte.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Dissection of proapoptotic protein Bax&#45;induced plant cell death</div>

<div>

	<span style="text-decoration: underline" >
	Maki Kawai-Yamada</span>
	<sup>1</sup>, 

	<span  >
	Hirofumi Uchimiya</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The University of Tokto, Tokyo, Japan

</div>

<div>Reactive oxygen species &#40;ROS&#41; are generated under various biotic and abiotic stresses, and trigger cell death. Mammalian pro&#45;apoptotic gene Bax is known to cause cell death when expressed in yeast and plants, even if there is no homologous gene in these organisms. Bax is known to translocate to mitochondria during the initiation of apoptosis in animal cells. To determine the cellular localization of Bax in plant cells, we constructed Bax&#45;GFP plasmid and introduced into tobacco BY&#45;2 suspension cells by particle bombardment. Under confocal laser scanning microscopy, the fluorescent signal of Bax&#45;GFP was observed in punctuated structures that co&#45;localized with the fluorescent signals of Mito&#45;RFP. Furthermore, ROS generation was observed in transgenic Arabidopsis plants expressing mouse Bax, suggesting that Bax may cause oxidative stress&#45;induced plant cell death. Furthermore, we examined transgenic plants expressing both Bax and organelle&#45;targeted green fluorescent protein &#40;GFP&#41; to analyze the cellular events that occur during Bax&#45;induced plant cell death. The results indicated that Bax induced temporal and special cell death events at the organelle level. The mitochondria changed morphologically from being bacilli&#45;shaped to being round. The chloroplasts lost membrane function and their contents leaked out, followed by the disruption of the vacuole. Furthermore, ROS production was involved in triggering cell death. To compare Bax&#45;induced cell death and other ROS&#45;mediated plant cell death, Arabidopsis leaves expressing mitochondrial&#45;targeted GFP were treated with ROS&#45;inducing chemicals, such as hydrogen peroxide, paraquat and menadione. After treatment, mitochondria showed morphological changes from a bacillus&#45;like to a round shape. The size of mitochondria decreased by half compared with controls. Such cellular events may cause energy depletion, and resulted in cell death. These results suggest that the Bax&#45;induced plants cell death is a potentially useful heterologous system for studying the regulation of oxidative stress&#45;induced cell death.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The ECHIDNA protein is required for cell elongation and represents a novel component of the endocytic pathway in &#34;Arabidopsis&#34;</div>

<div>

	<span  >
	Jaesung Oh</span>
	<sup>1</sup>, 

	<span  >
	Robert Nilsson</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Delphine Gendre</span>
	<sup>1</sup>, 

	<span  >
	Julien Schmidt</span>
	<sup>1</sup>, 

	<span  >
	Errin Johnson</span>
	<sup>1</sup>, 

	<span  >
	Alan Marchant</span>
	<sup>1</sup>, 

	<span  >
	Malcom J Bennett</span>
	<sup>2</sup>, 

	<span  >
	Markus Grebe</span>
	<sup>1</sup>, 

	<span  >
	Rishikesh Bhalerao</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>UPSC, 

	<sup>2</sup>University of Nottingham

</div>

<div>We studied an &#34;Arabidopsis thaliana&#34; T&#45;DNA insertion mutant which exhibits a severe dwarf phenotype. Hypocotyl and root length in this mutant are reduced by 68&#37; and 44&#37;, respectively, compared to wild type &#40;WT&#41;. The mutant is also unable to form root hairs, together suggesting a defect in differentiation and cell elongation. In support of this, the localization of ROP proteins, which are integral in root hair initiation, are unchanged in the mutant, implying that the mutation acts downstream of the hair initiation pathway. Moreover, root hair formation in the mutant can be restored by the addition of the ethylene precursor ACC to the growth medium, indicating that the ethylene response pathway is intact. The gene, &#34;ECHIDNA&#34;, disrupted by this mutation encodes a protein with similarities to a yeast protein involved in vesicular trafficking. Interestingly, immunolocalization studies reveal that ECHIDNA partially co&#45;localizes with VHA&#45;a1 and ARF1, which reside in trans&#45;Golgi&#47;endocytic compartments. In addition, ECHIDNA associates with FM4&#45;64 labeled endosomes, while FM4&#45;64 internalization is altered in &#34;echidna&#34; roots. Taken together, these results suggest that ECHIDNA is possibly involved at multiple stages of the endocytic pathway. Furthermore, an immuno&#45;dotblot assay using a suite of antibodies directed against specific cell wall epitopes showed that some RG&#45;1 type pectins are more abundant in the mutant relative to WT plants, suggesting that ECHIDNA may function in the transport and&#47;or recycling of cell wall compounds, the absence of which impairs cell elongation and root hair formation. </div>

</div>





<div class="abs">

<div style="font-weight:bold">The &#34;SGR5&#34; and &#34;SGR9&#34; genes are synergistically involved in amyloplasts movement in endodermal cell of inflorescence stem.</div>

<div>

	<span style="text-decoration: underline" >
	Moritaka Nakamura</span>
	<sup>1</sup>, 

	<span  >
	Masao Tasaka</span>
	<sup>1</sup>, 

	<span  >
	Miyo Terao Morita</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Nara Institute of Science and Technology

</div>

<div>Our previous research using Arabidopsis &#34;shoot&#34; &#34;gravitropism&#34; &#40;&#34;sgr&#34;&#41; mutans have demonstrated that shoot endodermis is essential for shoot gravitropism, and that amyloplasts sedimentation in the endodermal cells is important in gravity sensing. However, molecular entities that control amyloplasts sedimentaion remain unclear.<br/> Each &#34;sgr5&#34; and &#34;sgr9&#34; single mutant exhibits weak gravitropic response in the inflorescence stem. Interestingly, the &#34;sgr5sgr9&#34; double mutant completely loses the response. The &#34;SGR9&#34; and &#34;SGR5&#34; genes encode a RING finger protein and a putative transcription factor, respectively. Both genes play their roles in gravitropism within the endodermis. <br/> Here, we analyzed the distribution and movement of amyloplasts in living endodermal cells of each single or double mutant by using vertical microscope system. In contrast to biased location of amyloplasts to the direction of gravity in wild&#45;type, amyloplasts were dispersed throughout cells in both single and double mutant. The movement of amyloplasts was severely restricted in double mutant, while that in each single mutant was similar to that in wild&#45;type. These results suggest that complete loss of gravitropic ability in &#34;sgr5sgr9&#34; double mutant due to restriction of amyloplasts movement in addtion to dispersed amyloplasts. <br/> We have previously reported that &#34;sgr2&#34; loses gravitropic ability as well as &#34;sgr5sgr9&#34;. Although amyloplasts movement is severely restricted in both &#34;sgr2&#34; and &#34;sgr5sgr9&#34;, distribution of amyloplasts in &#34;sgr2&#34; is obviously different from that in &#34;sgr5sgr9&#34;. Since SGR2 is involved in vacuolar function, impairment of vacuolar membrane dynamics is likely to affect amyloplasts distribution and movement in &#34;sgr2&#34; endodermis. Then, we observed vacuolar membrane dynamics in &#34;sgr5&#34; and &#34;sgr9&#34; single and the double mutant. Our results suggest that defect of amyloplasts movement in &#34;sgr5sgr9&#34; is unlikely to be due to impairment of vacuolar membrane dynamics. <br/> Taken together, &#34;SGR5&#34; and &#34;SGR9&#34; seem to be involved in distribution and movement of amyloplast via novel action that is independent of vacuolar function. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Existence of the cells without detectable plastids in the <span style="font-style: italic">crumpled leaf</span> mutant of <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Yasushi Yoshioka</span>
	<sup>1</sup>, 

	<span  >
	Yuling Chen</span>
	<sup>2</sup>, 

	<span  >
	Tomoya Asano</span>
	<sup>3</sup>, 

	<span  >
	Makoto Fujiwara</span>
	<sup>4</sup>, 

	<span  >
	Shigeo Yoshida</span>
	<sup>5</sup>, 

	<span  >
	Yasunori Machida</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Div. Biol. Sci., Grad. Sch. Sci., Nagoya Univ., Nagoya, Japan, 

	<sup>2</sup>Col.Life Sci., Hebei Norm. Univ., Shijiazhuang, China, 

	<sup>3</sup>Adv. Sci. Res. Center, Kanazawa Univ, Kanazawa, Japan, 

	<sup>4</sup>Dept. Life Sci., Grad. Sch. Arts Sci., Univ. Tokyo, Tokyo, Japan, 

	<sup>5</sup>Plant Science Center, RIKEN, Yokohama, Japan

</div>

<div>The plastid is an essential organelle to the plant cell.  Plant cells that lack plastids have not been found even in the plants of which plastid division is severely suppressed.  Absence of chloroplasts in stomatal guard cells was reported in <span style="font-style: italic">accumulation and replication of chloroplasts 6</span> &#40;<span style="font-style: italic">arc6</span>&#41; of <span style="font-style: italic">Arabidopsis thaliana</span> &#40;Robertson et al., J. Cell Sci. 108, 2937&#45;2944.1995&#41;. However the presence of plastids without chlorophyll has not been investigated in these cells.  We investigate whether a plastid division mutants of <span style="font-style: italic">A. thaliana</span>, <span style="font-style: italic">crumpled leaf</span> &#40;<span style="font-style: italic">crl</span>&#41;, contain the cells lacking plastids using a plastid&#45;targeted yellow fluorescent protein.  Our data indicated that leaf epidermis including stomata and cotyledon primordia of mature embryos contained the cell having no detectable platids in <span style="font-style: italic">crl</span>.  Staining of the  stomata and the cotyledon primorida of the mature embryos with 4',6&#45;diamidino&#45;2&#45;phenylindole &#40;DAPI&#41; also indicated the existence of the cells without detectable plastids DNA in <span style="font-style: italic">crl</span>. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Microtubule arrays at the cytokinesis&#45;interphase transition</div>

<div>

	<span  >
	Jessica Lucas</span>
	<sup>1,2</sup>, 

	<span style="text-decoration: underline" >
	Fred Sack</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University British Columbia, 

	<sup>2</sup>Ohio State University

</div>

<div>Plant microtubule arrays undergo major changes during the cell cycle, but transitions between the arrays are not well defined.  During cytokinesis, almost all microtubules are associated with the phragmoplast array.  But after cytokinesis microtubules are found primarily in the cell cortex.  At the transition from cytokinesis to interphase, existing data suggest that the first interphase microtubules form at the nuclear surface in suspension cells &#40;Yoneda and Hasezawa 2003, Eur J Cell Biol and references therein&#41;.  But relatively little is known about how the cortical array initiates in cells in tissues.&#13;<br/>&#13;<br/>To observe how microtubules repopulate the cortex during late cytokinesis, we visualized microtubules in living Arabidopsis leaf epidermal cells harboring a constitutively expressed alpha&#45;tubulin&#45;GFP construct.  The fine&#45;staging of cells late in cytokinesis was facilitated by the predictable polarity of division in this tissue.  Our results indicate that microtubule nucleation in the cell cortex plays a major role in the re&#45;establishment of the cortical array at the cytokinesis&#45;G1 transition.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The thylakoid membrane protein ABC2&#47;CYO1, a cotyledon&#45;specific chloroplast biogenesis factor of Arabidopsis thaliana, has protein disulfide isomerase activity.</div>

<div>

	<span style="text-decoration: underline" >
	Hiroshi Shimada</span>
	<sup>1</sup>, 

	<span  >
	Mariko Mochizuki</span>
	<sup>2</sup>, 

	<span  >
	Kan Ogura</span>
	<sup>1</sup>, 

	<span  >
	Yumiko Shirano</span>
	<sup>3</sup>, 

	<span  >
	Daisuke Shibata</span>
	<sup>4</sup>, 

	<span  >
	Ken-ichiro Takamiya</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Tokyo Institute of Technology, Yokohama, Japan, 

	<sup>2</sup>Tokyo Institute of Technology, Yokohama , Japan, 

	<sup>3</sup>Boyce Thompson Institute for Plant Research, N.Y., USA, 

	<sup>4</sup>Kazusa DNA Research Institute, Chiba, Japan

</div>

<div>The development of chloroplasts in cotyledons differs in several aspects from that in true leaves, but the cotyledon&#45;specific program of chloroplast biogenesis has not been clarified yet.   The <span style="font-style: italic">abc2&#47;cyo1</span> mutant in <span style="font-style: italic">Arabidopsis thaliana</span> exhibits albino cotyledons but normal green true leaves.   The <span style="font-style: italic">abc2&#47;cyo1</span> mutant plants developed abnormal chloroplasts under light but normal etioplasts under dark.   Because of its T&#45;DNA tag, we isolated the <span style="font-style: italic">ABC2&#47;CYO1</span> gene and verified it was responsible for the albino cotyledon phenotype by complementation.   The ABC2&#47;CYO1 protein has a C4&#45;type zinc finger domain similar to that in DnaJ of <span style="font-style: italic">E. coli</span>, and ABC2&#47;CYO1 appears to be the higher plant specific protein and is highly conserved among not only dicotyledonous but also monocotyledonous plants.   The <span style="font-style: italic">ABC2&#47;CYO1</span> gene was expressed in only young plants under light conditions, and the ABC2&#47;CYO1 protein localizes to the thylakoid membrane in chloroplast.   The <span style="font-style: italic">ABC2&#47;CYO1</span> mutation did not affect transcription of photosynthetic genes but decreased the amount of the photosynthetic proteins.   Recombinant ABC2&#47;CYO1 protein accelerates the reduction of a disulfide pair in insulin and renatures the RNase by oxidizing its dithiol pairs, indicating that the ABC2&#47;CYO1 protein has the protein disulfide isomerase activity.   These results suggest that ABC2&#47;CYO1 has a chaperone&#45;like activity that is crucial for rapid assembly of the thylakoid membrane proteins during chloroplast biogenesis in cotyledons. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Analysis of trans&#45;Golgi network &#40;TGN&#41; dynamics in plants</div>

<div>

	<span style="text-decoration: underline" >
	Tomohiro Uemura</span>
	<sup>1</sup>, 

	<span  >
	Keiko Shoda</span>
	<sup>2</sup>, 

	<span  >
	Takashi Ueda</span>
	<sup>1</sup>, 

	<span  >
	 Masa.H Sato</span>
	<sup>3</sup>, 

	<span  >
	Akihiko Nakano</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences, Graduate School of Science, University of Tokyo, 

	<sup>2</sup>Molecular Membrane Biology Laboratory, RIKEN Discovery Research Institute, 

	<sup>3</sup>Faculty of Human Environmental Sciences, Kyoto Prefectural University

</div>

<div>In all eucaryotic cells, the post&#45;Golgi organelles, such as the trans&#45;Golgi network &#40;TGN&#41;, endosomes, vacuoles and the plasma membrane, are connected by vesicular traffic. This complex network plays a critical role in several higher&#45;order functions. The TGN is one of the most important organelles for protein transport at the post&#45;Golgi network, and functions as a sorting spot that directs cargo proteins to a variety of post&#45;Golgi compartments. However, the TGN of plant cells has not been well understood yet. In order to elucidate the structure, function and dynamics of plant TGN, we focused on Syp41, the ortholog of Tlg2&#47;syntaxin16 which is localized to the TGN in yeast and mammalian cells, as a TGN marker. We established the transgenic plants expressing GFP&#45;Syp41 under the control of the native promoter. The observation by CLSM &#40;confocal laser scanning microscopy&#41; showed that the TGN is a dot&#45;like BFA &#40;brefeldin A&#41;&#45;sensitive and Wm &#40;wortmannin&#41;&#45;insensitive compartment, which is partially stained with FM4&#45;64. Next, we addressed the relations between the TGN and the Golgi apparatus by analyzing the transgenic plant expressing GFP&#45; or mRFP&#45;tagged Syp31 &#40;cis&#45;Golgi marker&#41; or ST &#40;sialyl transferase, trans&#45;Golgi marker&#41; with Syp41 tagged with GFP or mRFP. The fluorescence patterns indicated that TGN was mainly located at the trans&#45;side of Golgi apparatus, but some population was independent from the Golgi apparatus. Taken together, we suggest that TGN does not always behave together with the Golgi apparatus, and functions in endocytic pathways in addition to the biosynthetic pathways. We will also discuss dynamic movement between TGN and the Golgi apparatus.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">Topography of phytochrome A action</div>

<div>

	<span  >
	Jutta R&#246;sler</span>
	<sup>1</sup>, 

	<span  >
	Ilse Klein</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Mathias Zeidler</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University Giessen, Plant Physiology, Giessen, Germany

</div>

<div>Plants use the photoreceptor family of phytochromes to sense their light environment in the red&#47;far&#45;red region of the spectrum. Phytochrome A &#40;phyA&#41; is the primary photoreceptor that regulates early seedling development. phyA mediated seedling de&#45;etiolation is the critical developmental transition from heterotrophic to photoauxotrophic growth. Irradiation with high fluence rate far red provides a way to assess specifically the role of phyA in this process and was used to isolate phyA signaling intermediates. fhy1&#45;3 is an insertion mutant of a gene encoding a phyA signal transduction component. FHY1 is a small 24 kDa protein with little similarity to known proteins, besides a small conserved septin&#45;related domain at the C&#45;terminus, a nuclear localization signal &#40;NLS&#41; and a nuclear exclusion signal &#40;NES&#41;. The NLS and NES of FHY1 are indeed involved in its nuclear localization and exclusion &#40;Zeidler et al. 2004&#41;. Nuclear localization of FHY1 is needed for the execution of responses downstream of phyA. FHY1 has one homologous gene in Arabidopsis, FHL &#40;FHY1 Like&#41;, for which overlapping functionality with FHY1 has been demonstrated &#40;Zhou et al. 2005&#41;. We generated double fhl&#47;fhy1 mutant lines and analysed the physiological response in HIR &#40;high irradiance response&#41; and VLFR conditions. FHY1 and FHL both are specifically impaired in phyA signaling. The double knockout lines show a very strong phenotype, comparable to the phyA null mutant. This phenotype is not due to changes in phyA levels. In CFP:FHY1 and phyA:GFP cotransformed protoplasts light dependent intracellular distribution was analysed and co&#45;translocation as well as co&#45;localization of phyA and FHY1 could be observed. Phenotypical analysis of fhl&#47;fhy1 revealed novel cytoplasmic phyA functions.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Quantitative Proteomic Characterization of Detergent Resistant Domains in the Plasma membrane of Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Sylwia Bem</span>
	<sup>1</sup>, 

	<span  >
	Waltraud Schulze</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck-Institute for Molecular Plant Physiology, Golm, Germany

</div>

<div>The plasma membrane is a dynamic compartment with key functions in solute transport, cell shape and rigidity as well as communication between cells and the environment. In mammalian and yeast cells, the plasma membrane has clearly been shown to be compartmented into so&#45;called lipid rafts, which are defined by their resistance to treatment with non ionic detergent. The &#34;detergent resistant&#34; regions have unique protein and lipid composition rich in sterols and sphingolipids. In mammalian and yeast cells, it has been shown that these domains are nanometers to one micron in size and they form highly organized microdomains. There are indications that also protein composition and size of detergent resistant domains can change in response to internal and external signals. &#13;<br/>&#13;<br/>Using quantitative proteomic techniques, we characterized the proteome of the detergent resistant domains in Arabidopsis cell cultures. Cell cultures were metabolically labelled with heavy nitrogen and in labelled cell cultures the detergent resistant domains were disrupted by chemical treatment. Preparations of treated cells were then directly compared to preparations of unlabeled cells. &#13;<br/>&#13;<br/>We show that also in plants, similar detergent resistant plasma membrane microdomains are significantly enriched in signal transduction proteins like receptor like kinases, membrane ATPases, transporters, and GPI&#45;anchored proteins. Also, by specific disruption of &#34;detergent resistant domains&#34; by methyl&#45;&#223;&#45;cyclodextrin, we are able to define common contaminating proteins in the &#34;detergent resistant&#34; preparation. Lipid composition of the detergent resistant domain fraction also clearly differs from whole plasma membrane and changes upon methyl&#45;&#223;&#45;cyclodextrin treatment. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">GNOM acts during Arabidopsis stomatal development</div>

<div>

	<span style="text-decoration: underline" >
	Jie Le</span>
	<sup>1</sup>, 

	<span  >
	Steffen Vanneste</span>
	<sup>2</sup>, 

	<span  >
	Tom Beeckman</span>
	<sup>2</sup>, 

	<span  >
	Fred Sack</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Botany, University of British Columbia, 6270 University Blvd, Vancouver, BC V6T 1Z4, Canada, 

	<sup>2</sup>Department of Plant Systems Biology, VIB, Ghent University, B-9052 Ghent, Belgium

</div>

<div>    Arabidopsis GNOM&#47;EMB30 is a guanine&#45;nucleotide exchange factor &#40;GEF&#41; for the ADP&#45;ribosylation factor &#40;ARF&#41; family of small GTPases. Mutations in <span style="font-style: italic">GNOM</span> induce well&#45;studied defects in embryonic divisions, root growth, and vascular patterning. Some of these abnormalities are likely related to the function of GNOM in vesicular trafficking, especially in endosomal recycling of targets such as PIN proteins which are membrane&#45;associated auxin efflux carriers.<br/>    The role of GNOM in stomatal development has not been studied, except for brief mention of adjacent stomata in <span style="font-style: italic">gnom</span> alleles &#40;Mayer et al., 1993, Development&#41;. We found a range of stomatal defects in <span style="font-style: italic">gnom</span>, with phenotypic severity following an allelic series. These results suggest that stomatal development requires GNOM&#45;mediated vesicular trafficking and might involve PINs and polar auxin transport as well.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular analysis of cell death suppressor &#40;AtBI&#45;1&#41; interacting factors</div>

<div>

	<span style="text-decoration: underline" >
	Minoru Nagano</span>
	<sup>1</sup>, 

	<span  >
	Yuri Ihara-Ohori</span>
	<sup>1</sup>, 

	<span  >
	Hirofumi Uchimiya</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Institute of Molecular and Cellular Biosciences, The University of Tokyo, 

	<sup>2</sup>Iwate Biotechnology center

</div>

<div>In organisms including animals and plants, programmed cell death &#40;PCD&#41; is crucial for the maintenance of their lives and is highly regulated by various factors. Arabidopsis Bax Inhibitor&#45;1 &#40;AtBI&#45;1&#41; is proposed as one of such factors. BI&#45;1 was first identified from human cDNA library as a suppressor of yeast cell death&#45;induced by mammalian proapoptotic protein, Bax. Although no homologs of Bcl&#45;2 family proteins including Bax have been identified in plants, BI&#45;1 is widely conserved in not only animals but plants. BI&#45;1 is about 26 kDa protein with 7 transmembrane domains and is localized in the membrane of endoplasmic reticulum &#40;ER&#41;. AtBI&#45;1 is rapidly up&#45;regulated in plants during wounding or pathogen challenge and overexpression of AtBI&#45;1 suppresses H2O2&#45;, salicylic acid&#45;, elicitor&#45;, heat&#45; and cold&#45;induced cell death in plant cells, suggesting that AtBI&#45;1 plays an important role in responses to various environmental stresses. Moreover, there is a coiled&#45;coil region in the C&#45;terminus of AtBI&#45;1, in which AtBI&#45;1 interacts with calmodulin, and AtBI&#45;1 affects calcium homeostasis in cell death regulation. However, the mechanism of AtBI&#45;1&#45;mediated cell death suppression has not been elucidated.&#13;<br/>To investigate the function of AtBI&#45;1 in detail, we attempted to identify interacting factors of AtBI&#45;1 by the screening of Arabidopsis cDNA library. By using split&#45;ubiquitin yeast two&#45;hybrid system, Arabidopsis cytochrome b5 &#40;AtCb5&#41; was isolated as a candidate of such factor. Cb5 is known as a ubiquitous electron transfer protein found in animals, plants and yeasts and is localized in ER or mitochondrial outer membrane. Here, we show the interaction between AtBI&#45;1 and AtCb5, and the role of AtCb5 on the cell death suppression by AtBI&#45;1.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Roles of Membrane Traffic in Xylem Differentiation</div>

<div>

	<span style="text-decoration: underline" >
	Jun Ito</span>
	<sup>1</sup>, 

	<span  >
	Tomohiro Uemura</span>
	<sup>2</sup>, 

	<span  >
	Minoru Kubo</span>
	<sup>3</sup>, 

	<span  >
	Makiko Udagawa</span>
	<sup>4</sup>, 

	<span  >
	Taku Demura</span>
	<sup>4</sup>, 

	<span  >
	Hiroo Fukuda</span>
	<sup>2</sup>, 

	<span  >
	Takashi Ueda</span>
	<sup>2</sup>, 

	<span  >
	Akihiko Nakano</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>RIKEN Discovery Research Institute, Wako, Japan, 

	<sup>2</sup>Department of Biological Sciences, Graduate School of Science, The University of Tokyo, Hongo, Japan, 

	<sup>3</sup>Japan Science and Technology Agency, ERATO, Okazaki, Japan, 

	<sup>4</sup>Plant Science Center, RIKEN, Yokohama, Japan

</div>

<div>Establishment and maintenance of the polarity of vascular cells are essential for continuity of the vascular system, where membrane traffic should play crucial roles.  Membrane traffic is also expected to participate in critical steps of vasculature development such as tonoplast breakdown during xylem cell differentiation.  To understand the precise roles of membrane traffic in vasculature formation, we are studying the functions of Rab GTPases, key regulators in membrane traffic, especially with a focus on xylem cell differentiation.  Our expression analysis of all 57 RAB genes in Arabidopsis during xylem cell differentiation using a genechip and promoter:GUS transgenic plants revealed that 1&#41; several Rab GTPases, most of which belong to the endocytic Rab GTPases &#40;Rab5, Rab7, Rab11 and Rab18 subgroups&#41;, are expressed during xylem cell differentiation specifically, 2&#41; some members of the Rab11 group, which is known to regulate cell polarity in the mammalian system, are expressed at different stages of xylem differentiation, and 3&#41; some vacuole&#45;localizing Rab GTPases &#40;AtRab75, AtRab77, AtRab18&#45;2&#41; are strongly expressed just prior to the drastic change of tonoplast integrity in differentiating tracheary elements.  These results strongly suggest that several endocytic Rab GTPases are involved in the xylem formation.  We then carried out genetic analysis of Arabidopsis mutants, which may have a defect in the endocytic pathway.  Some rab7 multiple mutants showed excessive vein formation in their cotyledons and fragmentation of protein storage vacuoles in their dry seeds.  The results of the in&#45;depth analysis of these mutants will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional analysis of AtVamp727 in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Kazuo Ebine</span>
	<sup>1</sup>, 

	<span  >
	Yusuke Okatani</span>
	<sup>1</sup>, 

	<span  >
	Tatsuaki Goh</span>
	<sup>1</sup>, 

	<span  >
	Emi Ito</span>
	<sup>1</sup>, 

	<span  >
	Tomohiro Uemura</span>
	<sup>1</sup>, 

	<span  >
	Akihiko Nakano</span>
	<sup>1,2</sup>, 

	<span  >
	Takashi Ueda</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Tokyo, Tokyo, Japan, 

	<sup>2</sup>RIKEN, Wako, Saitama, Japan

</div>

<div>  While it is getting a consensus for plant scientists that the endocytic pathway plays very important roles in various plant functions of higher order, the molecular mechanism of endocytosis in plant cells still remains almost unknown.  We are studying the molecular mechanism of endocytosis and how endocytosis participates in morphogenesis, with a particular focus on SNARE and Rab proteins.  We previously examined all SNARE proteins for endosomal localization, and found that only one R&#45;SNARE, AtVamp727, is localized almost exclusively on the Rab5&#45;positive endosomes &#40;Ueda et al.,2004; Uemura et al., 2004&#41;.  This strongly suggests that AtVamp727 functions in the fusion events of endosomal membranes.  To reveal the precise function of this molecule, we carried out genetic and cell biological analysis, to understand the relationship between AtVamp727 and AtVam3&#47;Syp22.<br/>  AtVam3&#47;Syp22 is a Qa&#45;SNARE residing on the vacuolar membrane and the prevacuolar compartment&#47;late endosome, and is reported to form a stable <span style="font-style: italic">trans</span>&#45;SNARE complex with AtVti11 and Syp51, Qb&#45; and Qc&#45;SNARE respectively.  This <span style="font-style: italic">trans</span>&#45;SNARE complex should also contain another component, R&#45;SNARE, which is not identified to date.  The <span style="font-style: italic">atvam3&#47;syp22</span> mutant shows a wide spectrum of phenotypes including wavy leaves, semi&#45;dwarfism, excessive differentiation of idioblasts and late flowering.  These phenotypes should be caused by the defect in the endocytic pathway, because <span style="font-style: italic">atvam3&#47;syp22</span> mutation shows strong genetic interactions with rab5 mutations.  We found that the <span style="font-style: italic">AtVAM3&#47;SYP22</span> also shows interesting genetic interactions with <span style="font-style: italic">AtVAMP727</span>, suggesting that these two gene products function cooperatively.  Subcellular localizations of these SNARE proteins in the Arabidopsis plant also support this idea.  In this meeting, we will discuss the function of the possible SNARE complex consisting of AtVamp727 and AtVam3&#47;Syp22 in the vacuolar biogenesis and plant morphogenesis.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">AtAzg1 is a high&#45;affinity purine and cytokinin cell importer in Arabidopsis thaliana</div>

<div>

	<span  >
	Veronica G. Maurino</span>
	<sup>1</sup>, 

	<span  >
	Esther Grube</span>
	<sup>1</sup>, 

	<span  >
	Benjamin S. Schumacher</span>
	<sup>2</sup>, 

	<span  >
	Ondrej Novak</span>
	<sup>3</sup>, 

	<span  >
	Miroslav Strnad</span>
	<sup>3</sup>, 

	<span  >
	Ulf-Ingo Fl&#252;gge</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Marcelo Desimone</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Botanisches Institut, Universit&#228;t zu K&#246;ln, Gyrhofstr. 15, D-50931, K&#246;ln, Germany, 

	<sup>2</sup>Zentrum f&#252;r Molekularbiologie der Pflanzen, Universit&#228;t T&#252;bingen, Auf der Morgenstelle 1, D-72076 T&#252;bingen, Germany, 

	<sup>3</sup>Laboratory of Growth Regulators, Institute of Experimental Botany ASCR and Palacky University, Slechtitelu 11, 783 71 Olomouc, Czech Republic

</div>

<div>The mechanisms governing the transport of purines and the phytohormone cytokinins in plants are poorly understood. A high&#45;affinity purine transporter of Aspergillus nidulans has been recently identified as a member of a novel protein family with members in prokaryotes, fungi and plants. Two orthologous proteins are encoded in the A. thaliana genome. In this work, AtAzg1 &#40;Arabidopsis thaliana Aza&#45;guanine resistant&#41; was functionally expressed in a yeast mutant deficient in adenine uptake to study transport features. AtAzg1 mediated a H&#43;&#45;gradient&#45;dependent high&#45;affinity transport of adenine &#40;KM &#61; 1.62 &#181;M&#41; with a broad substrate specificity including adenine, hypoxathine, guanine, cytokinins and toxic purine analogs. In contrast, other structurally related compounds, such as pyrimidines, nucleosides, caffeine and degradation products of purines &#40;xanthine, uric acid, allantoin&#41; were not transported. Transient expression of AtAzg1&#45;GFP fusion proteins in cultured A. thaliana cells and in onion epidermal cells revealed that AtAzg1 is localized to the plasma membrane indicating a function as cell importer. Azg1 knock&#45;out mutants presented a conditional phenotype resistant to purine analogs and toxic concentrations of trans&#45;zeatin &#40;tZ&#41; and, in contrast to the wild&#45;type, azg1 knock&#45;out seedlings did not accumulate cytokinin conjugates after treatment with tZ. On the contrary, A. thaliana overexpressing lines showed hypersensitivity to purine analogs and tZ and presented uptake rates for adenine and tZ several times higher than the wild&#45;type. These results indicate that AtAzg1 functions as a purine and cytokinin importer in vivo. Promoter activity analysis showed that AtAzg1 gene expression is modulated by cytokinins suggesting a role for AtAzg1 in metabolism or signaling of cytokinins.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Study of ROPs and their regulators in root hair development</div>

<div>

	<span style="text-decoration: underline" >
	Xin Chen</span>
	<sup>1</sup>, 

	<span  >
	Matt Smallman</span>
	<sup>1</sup>, 

	<span  >
	Claire Grierson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Biological Sciences, University of Bristol, Bristol, UK

</div>

<div>ROPs are members of a plant specific subfamily of Rho small GTPases. The GTP binding GTPases are active, while the GDP binding forms are inactive. The switch between these two forms requires several different regulators, and they are very important molecular switches in eukaryote signal transduction. Among all these regulators, GDI1 &#40;Guanine nucleotide dissociation inhibitor&#41; was the first one found having a strong phenotype in root hair development in <span style="font-style: italic">Arabidopsis thaliana</span>. In <span style="font-style: italic">scn1&#45;1</span> mutant, where GDI1 can&#8217;t be fully translated, the root hair cell forms bulges and branches. The aim of my research is to study the affects of structural changes on the function of GDI1 by phenotyping GDI1 point mutated lines, and the interactions between ROPs and GDI1 in vitro and in vivo. Results of these experiments would reveal how ROPs, GDI1 and the interactions between them contribute to root hair development in <span style="font-style: italic">Arabidopsis thaliana</span>.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Coordination of transcriptional regulation and chromatin modification of Arabidopsis circadian clock genes</div>

<div>

	<span style="text-decoration: underline" >
	Hae-Ryong Song</span>
	<sup>1,2</sup>, 

	<span  >
	Ju-Hee Jeong</span>
	<sup>1,2</sup>, 

	<span  >
	Bosl Noh</span>
	<sup>3,2</sup>, 

	<span  >
	Yoo-Sun Noh</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences, Seoul National University, Seoul 151-742, Korea, 

	<sup>2</sup>Global Research Laboratory for Flowering at SNU and UW, Seoul National University, Seoul 151-742, Korea, 

	<sup>3</sup>Environmental Biotechnology National Core Research Center, Gyeongsang National University, Jinju 660-701, Korea

</div>

<div>Circadian clock genes are regulated through a transcriptional&#45;translational feedback loop. In Arabidopsis, LHY and CCA1 transcripts are highly expressed in the early morning. Translated LHY and CCA1 proteins repress the expression of TOC1 transcript which peaks in the evening. TOC1 protein elevates the expression of LHY and CCA1 mRNAs, forming a negative feedback loop that is believed to constitute the oscillatory mechanism of the clock. Recently the rhythmic oscillation of mouse clock genes, mPER1 and mPER2, was shown to be correlated with the regular alteration of chromatin structure through histone acetylation&#47;deacetylation. However, little is known about the chromatin modification&#45;mediated transcriptional regulation of Arabidopsis circadian clock genes. Here we propose a possibility that Arabidosis clock&#45;associated genes, LHY, CCA1, and TOC1 might be regulated by rhythmic histone modifications. Our results show that certain type of histone modifications either has positive or negative correlations with the expression of LHY, CCA1, and TOC1 transcripts. Therefore, the rhythmic transcription of these clock genes might depend on regular histone modifications within their chromatin and the fine&#45;tuning of the feedback loop comprising an oscillator in plants might be accomplished by an ordered modification of histones.   </div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular characterization of a vesicular transport system inside the chloroplast</div>

<div>

	<span style="text-decoration: underline" >
	Christel Garcia</span>
	<sup>1</sup>, 

	<span  >
	Henrik Aronsson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant and Environmental Sciences, Goteborg University, Goteborg, Sweden

</div>

<div>Thylakoid biogenesis and maintenance require the transfer of many components including galactolipids and carotenoids from the inner envelope where they originate to the thylakoid membrane. But the mechanism by which the thylakoid membrane is formed is still not well understood.&#13;<br/>A chloroplast localized vesicular transport system similar to the membrane transport between the endoplasmic reticulum and the Golgi apparatus has been proposed to mediate this transfer. Ultrastructural observations have shown the accumulation of vesicles in the aqueous stroma of the chloroplast at low temperatures &#40;Morr&#233; et al, 1991, Plant Physiol 97:1558&#45;1564&#41; and biochemical studies have demonstrated that the galactolipid transport to the thylakoid is strongly inhibited at the same temperatures &#40;Andersson et al, 2001, Plant Physiol 127:184&#45;193&#41;. The vesicles are also affected by inhibitors of the vesicular trafficking in the secretory pathway &#40;Westphal et al, 2001, FEBS Lett 506:257&#45;261&#41;. &#13;<br/>A recent study, using the Arabidopsis genome and web&#45;based localization prediction tools, identified putative chloroplast localized proteins with a high sequence similarity to components of the cytosolic vesicular trafficking &#40;Andersson and Sandelius, 2004, BMC Genomics 5:40&#41;. Experimental data must now be obtained to define if these proteins are involved in a chloroplast vesicular transport system.&#13;<br/>In this purpose, we have first determined the subcellular localization of these proteins using GFP fusion constructs and some of them were detected in the chloroplast. Available Arabidopsis T&#45;DNA insertion lines are currently being analyzed and abnormal phenotypes in accordance with a possible role of the affected proteins in chloroplast biogenesis have already been observed. Progress on different aspects of the project will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Overexpression in <span style="font-style: italic">At1g23540</span> gene suppresses inflorescence growth</div>

<div>

	<span style="text-decoration: underline" >
	Youngmin Park</span>
	<sup>1</sup>, 

	<span  >
	Indeok Hwang</span>
	<sup>1</sup>, 

	<span  >
	Hyeonsook Cheong</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biotechnology, Chosun University, Gwangju, Korea

</div>

<div>Genetic approach in mutants screening is an important tool for studying gene function in plant. We got phenotypically distinct plants in the activation tagged lines and isolated a mutant, <span style="font-style: italic">igi1</span> &#40;Inflorescence Growth Inhibitor 1&#41;. Segregation ratio of the F2 generation, TAIL&#45;PCR walking and genotyping PCR results indicated a single T&#45;DNA insertion at the 200bp upstream of the At1g23540 coding region in the BAC F5O8. We referred to At1g23540 gene as <span style="font-style: italic">IGI1</span> and confirmed overexpression of the gene by Real time PCR. The expression level for <span style="font-style: italic">IGI1</span> gene was increased approximately by 8,000 to 10,000 fold in <span style="font-style: italic">igi&#47;igi</span> homozygous line. <span style="font-style: italic">igi&#47;igi</span> homozygous line was sterile and severely defective in growth and differentiation. <span style="font-style: italic">igi&#47;IGI</span> heterozygous mutant showed smaller silique, bunchy stem and shortened inflorescence. We generated <span style="font-style: italic">IGI1</span> overexpression lines &#40;<span style="font-style: italic">IGI1&#45;OX</span>&#41; for recapitulation phenotypes of the <span style="font-style: italic">igi1</span> mutant. <span style="font-style: italic">IGI1&#45;OX</span> lines displayed similar phenotypes with <span style="font-style: italic">igi&#47;IGI</span> heterozygous mutant. These results indicated <span style="font-style: italic">IGI</span> gene overexpression is related to bunchy stems and shortening of inflorescences.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Expression profiles and intracellular dynamics of SYP1s; plasma&#45;membrane localized Qa&#45;SNARE family in &#8216;&#8216;Arabidopsis&#8217;&#8217;.</div>

<div>

	<span style="text-decoration: underline" >
	Kazuhiko Enami</span>
	<sup>1</sup>, 

	<span  >
	Tomohiro Uemura</span>
	<sup>2</sup>, 

	<span  >
	Masahiko Sato</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Kyoto Prefectural University, Kyoto, Japan, 

	<sup>2</sup>The University of Tokyo, Tokyo, Japan

</div>

<div>Membrane traffic from&#47;to PM is highly organized process within the cells, since plant cells need to respond rapidly to variety of environmental changes, including various stresses, phytohormones and fungal attacks. The numbers of SNARE &#40;Soluble NSF Attachment protein REceptor&#41; genes in &#8216;&#8216;Arabidopsis&#8217;&#8217; genome are much greater than that in any other model organisms. We have previously showed that many SNARE molecules are localized predominantly on the plasma membrane &#40;PM&#41; than the other intracellular compartments in the &#8216;&#8216;Arabidopsis&#8217;&#8217; cell. &#13;<br/>To elucidate physiological function of each PM&#45;localized SNARE, we analyzed spacio&#45;temporal expression profiling of 9 PM&#45;resident Qa&#45;SNAREs in &#8216;&#8216;Arabidopsis&#8217;&#8217; to generate transgenic plants expressing GFP&#45;fused Qa&#45;SNAREs under control of their authentic poromoters. The microscopic observations using transgenic plant lines depicted the differences in the expression patterns of 9 PMQa&#45;SNAREs. Developing root section, in particular, showed elaborated regulation of GFP&#45;fused transgenes. One SNARE molecule out of nine expressed constitutively throughout development, whereas expression of other SNAREs limited in the specific cell layer. Furtehrmore, SYP123 was exclusively expressed in the trichoblasts with elongating root hairs. Deletion of SYP123 expression caused an elongation defect in the root hairs.&#13;<br/>These results suggest that SYP123 functions in the elongation step of root hair, not in the bulging phase, and that, furthermore, this Qa&#45;SNARE may control the orientation of polarized vesicular transport by its precise localization on PM of root hair region.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Specific and redundant ARF&#45;GEF functions in membrane trafficking</div>

<div>

	<span style="text-decoration: underline" >
	Gerd Juergens</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Tuebingen, Tuebingen, Germany

</div>

<div>Membrane trafficking maintains the endomembrane organisation and mediates the passage of molecules involved in signaling, cell&#45;cell communication and response to abiotic or biotic stress. ARF GTPases are molecular switches that regulate vesicle formation at donor membranes. They are activated by ARF guanine&#45;nucleotide exchange factors &#40;ARF&#45;GEFs&#41;, which associate with specific membrane compartments and thus determine when and where ARFs become active. The Arabidopsis genome encodes several large ARF&#45;GEFs, which represent the only subfamily of ARF&#45;GEFs conserved between plants, animals and fungi. The best characterised member is GNOM, which is required for the recycling of the auxin&#45;efflux regulator PIN1 from endosomes to the basal plasma membrane. Recent studies on closely related GNOM&#45;LIKE 1 &#40;GNL1&#41; will be presented and the relative roles of GNOM and GNL1 will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Chloroplast protein targeting with emphasis on the role of Hsp93</div>

<div>

	<span  >
	Lars Sjogren</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Henrik Aronsson</span>
	<sup>1</sup>, 

	<span  >
	Adrian Clarke</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant and Environmental Sciences, G&#246;teborg University, Sweden

</div>

<div>Most chloroplast proteins are encoded in the nucleus and imported post&#45;translationally into chloroplasts. A protein complex in the chloroplast envelope, called the translocon, mediates import. Components of the translocon are called Toc or Tic, for translocon at the outer&#47;inner envelope membrane of chloroplasts. Many Toc and Tic components have been identified and characterised, but for most their function remains undefined. Proteins imported by the translocon have a transit peptide that targets them to the main receptors Toc33 and Toc159. Thereafter the incoming protein is transferred over the envelope membranes through the Toc75 and Tic110 channels. The transit peptide is removed upon transfer across the envelope membranes. &#13;<br/>&#13;<br/>Hsp93 &#40;also known as ClpC&#41; works in close proximity with Tic110 and the co&#45;chaperone Tic40 at the latter stage of import. Tic110 recruits Tic40 when a precursor is present in the channel. Tic40 then releases the precursor from Tic110 and stimulates ATP hydrolysis of Hsp93 which is suggested to be the motor for the translocation process. In the mitochondrial protein transport systems Hsp70 is the known motor. Although no connection has been established between the envelope associated Hsp70 and the TIC complex, a motor function for chloroplast Hsp70 cannot yet be excluded. In Arabidopsis two paralogs of Hsp93 exist &#40;Hsp93&#45;III and Hsp93&#45;V&#41; with Hsp93&#45;V being more highly expressed. Loss of the more abundant Hsp93&#45;V causes a chlorotic phenotype and less developed chloroplasts, but only at most a minor decrease in import rate. Thus, it appears the function of Hsp93&#45;V is not primarily that of a motor. Indeed, stromal Hsp93 plays an important role in protein degradation as part of the Clp protease, and so alternative roles for Hsp93 at the envelope need to be considered. We have postulated that Hsp93&#45;V performs a quality control function of the import and this will be further discussed.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional Identification of Sugar&#45;1&#45;P Kinases in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Ting Yang</span>
	<sup>1</sup>, 

	<span  >
	Maor Bar-Peled</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biochemisty and Molecular Biology, Complex Carbohydrate Research Center(CCRC), University of Georgia, Athens, Georgia, USA

</div>

<div>The salvage pathway is an essential metabolic process required for plant development specifically during seed germination, pollen growth and embryo development where storage carbohydrates are converted to simple sugars. These sugars need to be activated into nucleotide&#45;sugars in order to facilitate cell division and cell growth. The function of the salvage pathway is also essential during normal growth to recycle sugars generated from breakdown of diverse glycoconjugates &#40;glycoproteins, glycolipids, and small metabolite&#45;glycosides&#41;. A typical plant cell requires generating over 25 different types of nucleotide&#45;sugars. Sugar&#45;1&#45;P kinases are a group of enzymes that phosphorylate free sugars to sugar&#45;1&#45;P. Pyrophosphorylases &#40;PPases&#41; are a group of enzymes required to convert the sugar&#45;1&#45;P with nucleotides &#40;NTP&#41; such as ATP, GTP, TTP, CTP, or UTP to form the corresponding NDP&#45;sugar.&#13;<br/>Galactokinase &#40;GalK&#41; phosphorylates D&#45;galactose with ATP to D&#45;galactose&#45;1&#45;phosphate. Little is known about this and other sugar&#45;1&#45;P kinases, which are required to phosphorylate GlcA, GalA, Rha, Xyl, Ara, etc. Here we report the functional identification of many of these kinases at the biochemical and gene levels. We developed a new HPLC&#45;based assay in conjunction with NMR analysis. Current work involves the i&#41; determination of domains involved in sugar recognition, ii&#41; wall analysis of mutants in these kinase genes, iii&#41; determination of the expression and regulation of these kinase genes. The complete identification of all the sugar&#45;1&#45;P kinases is significant if we wish to understand the flux of carbon contributing to carbohydrate storage, wall synthesis, gene regulation in sugar synthesis and other regulatory mechanisms in salvage pathway. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A role of a small G protein Rop2 in ABA&#45;induced stomatal closing movement</div>

<div>

	<span style="text-decoration: underline" >
	Byeong Wook Jeon</span>
	<sup>1</sup>, 

	<span  >
	Jae-Ung Hwang</span>
	<sup>2</sup>, 

	<span  >
	Miyoung Lee</span>
	<sup>1</sup>, 

	<span  >
	Jiyoung Park</span>
	<sup>1</sup>, 

	<span  >
	Youngsook Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Division of Molecular Life Sciences, POSTECH, Pohang, 790-784, Korea, 

	<sup>2</sup>Center for Plant Cell Biology and Department of Botany and Plant Sciences, University of California, Riverside, CA, 92521-0124, USA,

</div>

<div>ABA induces stomatal closing, which results in reduction of water loss from plants. We previously suggested that ROP2 is important in guard cell signaling, based on different localization of active and inactive ROP2&#45;fused to green fluorescence protein &#40;GFP&#41;; CA&#45;ROP2&#45;GFP is localized predominantly at the plasma membrane whereas DN&#45;ROP2&#45;GFP is at the cytoplasm. Here, we investigated the roles of Rop2 in ABA&#45;induced stomatal closing movement. ABA induced translocation of Rop2 from the plasma membrane to cytoplasm in guard cells of Vicia faba expressing ROP2::GFP; the green fluorescence intensity at the plasma membrane decreased upon treatment with ABA. This result suggests that ABA induces inactivation of Rop2.  Stomata bordered by guard cells transformed with CA&#45;rop2 closed less than their neighbors bordered by non&#45;transformed guard cells upon ABA treatment. In contrast, stomata bordered by guard cells transformed with DN&#45;rop2 closed more than their controls in response to the same treatment. Mutant rop2 Arabidopsis thaliana guard cells showed phenotypes similar to these transformed V. faba guard cells.  In addition, CA&#45;rop2 mutant Arabidopsis plants were more sensitive than WT and DN&#45;rop2 mutants to drought stress. Thus, we suggest that ABA&#45;induced inactivation of Rop2 contributes positively to stomatal closing and water conservation during drought stress.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Control of Arabidopsis Hormone Signaling by the Ubiquitin&#47;26S Proteasome System</div>

<div>

	<span style="text-decoration: underline" >
	Richard Vierstra</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Wisconsin, Madison, (Wisconsin), USA

</div>

<div>Plants use a repertoire of methods to control the level and activity of their constituent proteins.  One route that plays a prominent role is selective protein breakdown by the ubiquitin &#40;Ub&#41;&#47;26S proteasome system &#40;UPS&#41;.  In this UPS system, proteins targeted for breakdown are tagged with multiple Ubs: these poly&#45;ubiquitinated species are then recognized and degraded by the 26S proteasome, an ATP&#45;dependent multicatalytic protease complex.  Remarkably, analyses of the near complete Arabidopsis thaliana genome identified over 1,500 genes, or approximately 5&#37; of the proteome, that are part of the UPS, making it one of the most elaborate regulatory mechanisms in plants.  Phylogenetic comparisons to rice also suggest that components of the UPS are rapidly evolving, possibly as a way to fine&#45;tune recognition of specific target classes.  Molecular genetic analyses have connected individual components of the UPS to almost all aspects of plant biology, with a particularly important role in hormone signaling.  For example, both the synthesis and the response to ethylene are strongly regulated by ubiquitination with two Ub ligases EBF1 and 2 playing key but distinct roles in targeting the ethylene&#45;response EIN3&#47;EIL1 transcription factors for turnover.  With respect to abscisic acid &#40;ABA&#41; signaling, regulated degradation of the ABI5 transcription factor is controlled by several factors, including the KEG RING Ub ligase, a family of RAD23 proteins that shuttle ubiquitinated proteins to the 26S proteasome, and the poly&#45;Ub receptor RPN10, which is a core subunit of the 26S proteasome.  Through such controls, the UPS is able to dampen signaling at low hormone concentrations, rapidly allow signaling when hormone levels rise, and then enhance recovery when hormone levels dissipate.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A dynamic scaffold: cytoskeletal organization of cellulose synthase</div>

<div>

	<span  >
	Alex Paredez</span>
	<sup>1</sup>, 

	<span  >
	Ryan Gutierrez</span>
	<sup>1</sup>, 

	<span  >
	Seth DeBold</span>
	<sup>1</sup>, 

	<span  >
	Viktor Kyryk</span>
	<sup>1</sup>, 

	<span  >
	Dorianne Moss</span>
	<sup>1</sup>, 

	<span  >
	Sydney Shaw</span>
	<sup>2</sup>, 

	<span  >
	John Sedbrook</span>
	<sup>3</sup>, 

	<span  >
	Chris Somerville</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	David Ehrhardt</span>
	<sup>4</sup>

</div>

<div>

	<sup>1</sup>Carnegie Institution, Stanford, CA, USA, 

	<sup>2</sup>Indiana University, Bloomington, IN,USA, 

	<sup>3</sup>Illinois State University, Normal, IL, USA, 

	<sup>4</sup>C

</div>

<div>The cortical array has long been hypothesized to regulate plant cell morphogenesis by guiding the oriented deposition of cellulose in the cell wall, thus  creating the material anisotropy that is responsible for directed expansion of the cell wall during turgor&#45;driven growth.  While electron microscopy and pharmacological evidence often supported this hypothesis, several studies had suggested that the relationship between the cytoskeleton and cellulose synthase was more complex than simple models predicted.  To shed new light on this question, we created a functional flourescent protein fusion to CESA6, an isozyme required to support rapid hypocotyl elongation.  This tool has permitted visualization of cellulose synthase complexes, revealing their patterns of organization and dynamic behavior as they move through the plasma membrane while creating microfibrils.  Co&#45;visualization with labeled microtubules reveals that CESA6 complexes track in a bidirectional fashion along individual elements of the cortical array, and that these trajectories reorient as the cortical array reorients.  However, in the absence of cortical microtubules, CESA6 complexes are still able to achieve an organized pattern of trajectories.  In addition to guiding CESA complexes in the plasma membrane, the cortical microtubule cytoskeleton also appears to interact with a cytosolic compartment containing CESA protein.  This compartment shows a novel form of tip&#45;tracking motility driven by microtubule dynamics.  These data suggest there may be a second role for the cortical cytoskeletal in regulating and organizing cellulose synthase.  These live cell studies help to reconcile the previous literature, and enable new investigations into the function of cortical cytoskeleton in organizing proteins and macromolecules in the cell membrane and cell wall.</div>

</div>


</div>
</div>

</body>
</html>
