
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>
<head>
<title>ICAR 2007 - Metabolism</title>
<link rel="Shortcut Icon" href="/favicon.ico">
<link rel="stylesheet" type="text/css" href="/css/main.css" />
<link rel="stylesheet" type="text/css" href="/css/search.css" />
<link rel="stylesheet" type="text/css" href="/css/base.css" />

<style type="text/css">
@import "/css/page/pagelevels.css";
</style>

<style type="text/css">
.abs {
 margin-top: 30px;
}
.abs div {
 margin: 10px;
}
</style>
</head>
<body>

<div id="container">

<script src="https://ssl.google-analytics.com/urchin.js" type="text/javascript"></script>
<script type="text/javascript">
_uacct = "UA-1945230-1";
urchinTracker();
</script>

 <div id="nav">	
  <div id="logo"><a href="/index.jsp"><img src="/i/logo2.gif" width="87" height="61" alt="tair"/></a></div>
 </div>

<div id="content" class="clearfix">

<div class="abs">

<div style="font-weight:bold">Feedback control of leaf polarity by a family of small leucine zipper proteins</div>

<div>

	<span style="text-decoration: underline" >
	Stephan Wenkel</span>
	<sup>1</sup>, 

	<span  >
	John Emery</span>
	<sup>1</sup>, 

	<span  >
	Kathy Barton</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Carnegie Institution fo Washington, Stanford, CA 94305, USA

</div>

<div>The plant specific class III homeodomain&#45;leucine zipper &#40;HD&#45;ZIPIII&#41; transcription factor proteins are master regulators for polar leaf development.  We report the identification of a novel family of plant proteins, the LITTLE ZIPPER &#40;ZPR&#41; proteins, and their role in establishing a negative feedback loop controlling leaf development.  HD&#45;ZIPIII proteins cause transcriptional activation of the ZPR genes.  The analysis of gain&#45;of&#45;function transgenic plants revealed that ZPR proteins specify abaxial development. ZPR proteins which consist primarily of a short stretch of leucine zipper physically interact with HD&#45;ZIPIII proteins and, when overexpressed, inhibit HD&#45;ZIPIII function.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cellular Dynamics in the Shoot Apical Meristem</div>

<div>

	<span style="text-decoration: underline" >
	Elliot Meyerowitz</span>
	<sup>1</sup>, 

	<span  >
	Vijay Chickarmane</span>
	<sup>1</sup>, 

	<span  >
	Sean Gordon</span>
	<sup>1</sup>, 

	<span  >
	Marcus Heisler</span>
	<sup>1</sup>, 

	<span  >
	Yuling Jiao</span>
	<sup>1</sup>, 

	<span  >
	Wuxing Li</span>
	<sup>1</sup>, 

	<span  >
	Zack Nimchuk</span>
	<sup>1</sup>, 

	<span  >
	Carolyn Ohno</span>
	<sup>1</sup>, 

	<span  >
	Xiang Qu</span>
	<sup>1</sup>, 

	<span  >
	Adrienne Roeder</span>
	<sup>1</sup>, 

	<span  >
	Kaoru Sugimoto</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>California Institute of Technology, Pasadena, California, USA

</div>

<div>The shoot apical meristems &#40;SAMs&#41; of a flowering plant are the sets of stem cells at the tip of each shoot, which ultimately provide the cells that make the stem, leaves, and flowers.  The cells of the SAM are highly dynamic, but at the same time there is a highly stereotyped spatial structure, with cells of different types &#40;or perhaps in different states&#41; characteristically in specific and predictable positions.  As cells divide their positions change, and therefore the stable pattern of gene expression is maintained despite the fact that the cells themselves do not remain in place.  This dynamism calls for new methods of observation, manipulation, and analysis.<br/><br/>To describe and manipulate the cells of the Arabidopsis SAM , we have developed a set of live&#45;imaging methods, a set of reporters for gene expression domains, and protein localization domains within cells of the SAM, and also a set of methods for changing the patterns of gene expression in the meristem while we are watching.  By combining these methods we have shown that cells in the meristem both differentiate from the stem cell state, and return to this state depending on their positions and on signaling between different cells &#8211; the stable patterns of gene expression found in unstable populations of cells are maintained by a series of feedbacks mediated by cell&#45;cell signaling.  Furthermore, using live imaging and reporter constructs we and others have found that the SAM has a private and highly controlled circulatory system for auxin, which serves as the morphogen that induces new leaf and floral primordia.  This circulatory system is based on movement within each SAM cell of a polar auxin transporter, with this movement determined by the auxin gradient itself.  This feedback system leads to successive peaks of auxin concentration in the precise pattern that is the phyllotactic spiral &#8211; the regular positions of new leaf or floral primordia.  This example seems to define a new class of developmental models, based on regulated and active transport of morphogens, rather than on diffusion or inhibitory cell&#45;cell interactions.  <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The <span style="font-style: italic">ASYMMETRIC LEAVES2</span> gene regulates adaxial cell proliferation involved in the development of symmetric and flat leaf laminae</div>

<div>

	<span style="text-decoration: underline" >
	Hidekazu Iwakawa</span>
	<sup>1</sup>, 

	<span  >
	Mayumi Iwasaki</span>
	<sup>1</sup>, 

	<span  >
	Hiroo Takahashi</span>
	<sup>2</sup>, 

	<span  >
	Takeshi Kobayashi</span>
	<sup>1,3</sup>, 

	<span  >
	Yasunori Machida</span>
	<sup>4</sup>, 

	<span  >
	Chiyoko Machida</span>
	<sup>1,3</sup>

</div>

<div>

	<sup>1</sup>Plant Biology Research Center, Chubu Univ., Kasugai, Japan, 

	<sup>2</sup>Grad. Sch. of Eng., Nagoya Univ., Nagoya, Japan, 

	<sup>3</sup>College of Biosci. and Biotech., Chubu Univ., Kasugai, Japan, 

	<sup>4</sup>Grad. Sch. of Sci., Nagoya Univ., Nagoya, Japan

</div>

<div>Leaves of angiosperms exhibit remarkable diversity in terms of both shape and complexity. Nonetheless, plants appear to exploit common mechanisms that are responsible for the establishment of three axes during leaf development. Leaves are generated as lateral organs from a shoot apical meristem and develop along the axes, proximal&#45;distal, adaxial&#45;abaxial and medial&#45;lateral axes. The <span style="font-style: italic">AS2</span>, a member of the <span style="font-style: italic">AS2&#47;LOB</span> family, and <span style="font-style: italic">AS1</span> play important roles to establish these axes. Here we report the patterns of expression of these genes and the importance of the sites of such expression in leaf development. Transcripts of both genes accumulated in entire leaf primordia at early stages but patterns of accumulation changed as leaves expanded. <span style="font-style: italic">AS2</span> and <span style="font-style: italic">AS1</span> transcripts were detected, respectively, in the adaxial domain and in the inner domain between the adaxial and abaxial domains of leaves. Ratios of numbers of adaxial cells to abaxial cells in cotyledons of corresponding mutants were larger than ratios in wild&#45;type cotyledons. Low level expression of <span style="font-style: italic">AS2</span> under the control of <span style="font-style: italic">AS1</span> promoter in <span style="font-style: italic">as2</span> restored an almost normal phenotype in some cases but also resulted in flatter leaves than those of wild&#45;type. Strong expression of the construct in wild&#45;type and <span style="font-style: italic">as2</span>, but not <span style="font-style: italic">as1</span>, resulted in the formation of upwardly curled leaves. Our results indicate that <span style="font-style: italic">AS2</span> represses cell proliferation in the adaxial domain in the presence of <span style="font-style: italic">AS1</span> and that adaxial expression of <span style="font-style: italic">AS2</span> at appropriate level is critical for development of flat and symmetrical laminae. Real&#45;time RT&#45;PCR analysis revealed that mutation of either <span style="font-style: italic">AS2</span> or <span style="font-style: italic">AS1</span> resulted in increased levels of transcripts of <span style="font-style: italic">ETT&#47;ARF3</span>, <span style="font-style: italic">KAN2</span> and <span style="font-style: italic">YAB5</span>. Thus, <span style="font-style: italic">AS2</span> and <span style="font-style: italic">AS1</span> might negatively regulate these genes in the adaxial domain, which might be related to the development of flat and expanded leaves.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Many Ways Regulating the Floral Repressor FLC in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Ilha Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Seoul National University

</div>

<div>Flowering is regulated by a complex genetic network sensing environmental cues and endogenous signals. FLOWERING LOCUS C &#40;FLC&#41;, a strong floral repressor, is one of central players in such a complex genetic network. To understand the regulatory mechanism of FLC, we screened early flowering mutants from late&#45;flowering winter annual plants after fast neutron mutagenesis. The map&#45;based gene cloning approach using the selected mutants revealed molecular mechanism of FLC regulation. The expression of FLC is regulated by diverse mechanisms as FRIGIDA&#45;mediated activation, autonomous pathway, vernalization, PAF1 complex mediated histone modification, and ATP&#45;dependent chromatin remodeling. Here, we present molecular evidence how FLC is transcriptionally activated and repressed by diverse mechanisms and how each mechanism interact one another.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Components of CLE signalling pathways</div>

<div>

	<span style="text-decoration: underline" >
	R&#252;diger Simon</span>
	<sup>1</sup>, 

	<span  >
	Andrea Bleckmann</span>
	<sup>1</sup>, 

	<span  >
	Ralf M&#252;ller</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institut f&#252;r Genetik, Heinrich-Heine Universit&#228;t, D&#252;sseldorf, Germany

</div>

<div>Stem cells are required to support the indeterminate growth style of plants. In shoot meristems, stem cell fate is decided at the populational level by an exchange of signals between the stem cell domain at the meristem tip and a small group of cells &#40;the organizing centre or OC&#41; underneath. Cells in the OC express the homeodomain transcription factor WUS, which is required for the induction and maintenance of stem cells at the tip. How WUS performs this function has not been characterized so far, but one of the &#40;direct or indirect&#41; target genes that depends on WUS is the stem cell specific CLV3 gene. CLV3 encodes a signaling molecule that belongs to the CLE peptide family. CLV3 acts via the CLV1 receptor kinase to downregulate WUS transcription. This negative feedback regulation shall ensure a constant stem cell population during meristem growth and development. <br/>We have studied this regulatory pathway in more detail, and identified new components that are essential for perception of CLE signals. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">Arabidopsis thaliana</span> trichomes as defense organs</div>

<div>

	<span style="text-decoration: underline" >
	Manli Davis</span>
	<sup>1</sup>, 

	<span  >
	Kengo Morohashi</span>
	<sup>1</sup>, 

	<span  >
	Erich Grotewold</span>
	<sup>1</sup>, 

	<span  >
	Rebecca Lamb</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Ohio State University, Columbus, OH, USA

</div>

<div>Control of gene transcription is of central importance in the development of any organism. Hierarchical arrangements &#40;networks&#41; of transcription factors &#40;TFs&#41; provide the information necessary to deploy genes with particular spatial and temporal patterns. It has been predicted that there are over 1,700 TFs encoded in the model plant species <span style="font-style: italic">Arabidopsis thaliana</span>. Many of these TFs are involved in various developmental processes but only a handful of them have known function. As part of our research on TFs, we are studying TFs involved in trichome initiation: <span style="font-style: italic">GLABROUS3 &#40;GL3&#41;&#47;ENHANCER OF GLABROUS3 &#40;EGL3&#41;, GLABROUS1 &#40;GL1&#41;</span> and <span style="font-style: italic">TRANSPARENT TESTA GLABRA1 &#40;TTG1&#41;</span>. These genes encode bHLH, MYB and WD repeat proteins respectively; <span style="font-style: italic">gl3 egl3</span> double mutants and <span style="font-style: italic">gl1</span> or <span style="font-style: italic">ttg1</span> single mutants are glabrous. These proteins have been shown to form a transcriptional complex necessary for trichome initiation. The identification of the targets of these genes will provide information about the developmental processes of trichome development and the regulatory network involved in this process. As a complement to this avenue of investigation, we are also comparing gene expression between <span style="font-style: italic">gl3 egl3, gl1</span> and <span style="font-style: italic">ttg1</span> mutants and wild type in order to identify other trichome&#45;enriched genes: 88 genes have significantly reduced expression in all three mutant backgrounds. Several of these genes had been previously identified as genes with high expression in trichomes. Identification of such genes will provide information on what function&#40;s&#41; the mature trichomes play in <span style="font-style: italic">Arabidopsis</span>. The proteins encoded by the trichome&#45;enriched genes are involved in biotic and&#47;or abiotic responses in plants, based on both bioinformatics and functional data. We conclude that Arabidopsis trichomes function as defense organs by serving as both a physical and a molecular barrier.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Analysis on the mutants with increased cell number and decreased cell size in leaves</div>

<div>

	<span style="text-decoration: underline" >
	Takeshi Usami</span>
	<sup>1</sup>, 

	<span  >
	Gorou Horiguchi</span>
	<sup>2</sup>, 

	<span  >
	Hirokazu Tsukaya</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>National Institute for Basic Biology, 

	<sup>2</sup>Graduate School of Science, The University of Tokyo

</div>

<div>During leaf development, cell proliferation at the base of leaf primordia and cell expansion at the tip takes place simultaneously and in a coordinated manner. Coordination of cell proliferation and cell expansion is essential for leaf development. It is known that a defect in cell proliferation in leaf primordia induces an increase in cell size in many mutants such as <span style="font-style: italic">angustifolia3</span> &#40;<span style="font-style: italic">an3</span>&#41; and <span style="font-style: italic">aintegumenta</span> &#40;<span style="font-style: italic">ant</span>&#41;. This phenomenon, called compensation syndrome, is thought to reflect the intrinsic mechanism that coordinates cell proliferation and cell expansion, and has been extensively studied in recent years. On the other hand, an increase in cell number caused by overexpression of <span style="font-style: italic">AN3</span> or <span style="font-style: italic">ANT</span> does not reduce cell size. To gain further insight into the coordination of cell proliferation and expansion, we screened for mutants that have increased cell number and decreased cell size in leaves. We isolated four such mutants and investigated them. Time course analysis of leaf development showed that decreased cell size in these mutants might not be the consequences of a prolonged cell proliferation period at the expense of cell expansion period. Genetic analysis showed that two of them were allelic to <span style="font-style: italic">squint</span> and <span style="font-style: italic">paused</span>, which showed accelerated heteroblasty. Another one also showed significantly accelerated heteroblasty. We then observed cellular changes associated with heteroblasty in wild type and found that the leaves at higher nodes contained larger number of smaller&#45;sized cells than those at lower nodes. To our knowledge, this is the novel cellular phenotype of heteroblasty in arabidopsis leaves. The leaves of the three mutants with accelerated heteroblasty must have characteristics of those at higher node of wild type. Unknown factor&#40;s&#41; that changes with heteroblasty might have an important role in regulating both the number and the size of leaf cells. Further investigation about this factor and the fourth mutant, which showed no significant acceleration in heteroblasty, will give us a clue to unravel the mechanism that coordinates cell proliferation and cell expansion during leaf development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">BLADE&#45;ON&#45;PETIOLE1 and 2 Control Arabidopsis Lateral Organ Fate Through Activation of LOB&#45;Domain Genes and Regulation of Organ Polarity Genes</div>

<div>

	<span style="text-decoration: underline" >
	CHAN MAN HA</span>
	<sup>1,2</sup>, 

	<span  >
	JI HYUNG JUN</span>
	<sup>1</sup>, 

	<span  >
	HONG GIL NAM</span>
	<sup>2</sup>, 

	<span  >
	JENNIFER FLETCHER</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>USDA/UC BERKELEY, PLANT GENE EXPRESSION CENTER, 800 BUCHANAN STREET, CA 94710, USA, 

	<sup>2</sup>DEPARTMENT OF LIFE SCIENCE, POSTECH, POHANG, KYUNGBUK 790-784, KOREA

</div>

<div>Continuous lateral organ formation is critical for higher plants to produce their characteristic architectures, but the regulatory pathways that specify organ cell fate are still poorly understood. &#13;<br/>Here, we present a novel function for the putative transcriptional regulators BLADE&#45;ON&#45;PETIOLE1 &#40;BOP1&#41; and BOP2 in the control of LATERAL ORGAN BOUNDARIES &#40;LOB&#41;&#45;domain &#40;LBD&#41; and adaxial&#45;abaxial polarity gene expression during Arabidopsis lateral organ development. 35S:BOP1 and 35S:BOP2 plants exhibit a very short and compact stature, hyponastic leaves, and downward&#45;orienting siliques. &#13;<br/>We show that three LBD genes, ASYMMETRIC LEAVES2 &#40;AS2&#41;, LOB and LBD36&#47;ASL1, are up&#45;regulated in 35S:BOP and down&#45;regulated in bop mutant plants. Ectopic activation of BOP1 or BOP2 also results in repression of class I knox gene expression. In addition, some 35S:BOP phenotypes are eliminated in the as2 and as1 mutant backgrounds. We demonstrate a role for BOP1 and BOP2 in the establishment of the adaxial&#45;abaxial polarity axis in the leaf petiole, where they regulate PHB and FIL expression and overlap in function with AS1 and AS2. Interestingly, we find that KANADI1 &#40;KAN1&#41; and KAN2 promote adaxial organ identity as well as abaxial organ identity. &#13;<br/>Our data indicate that BOP1 and BOP2 act at the lateral organ boundary to repress genes that promote meristem cell fate and to induce genes that promote lateral organ fate and polarity, thereby restricting the developmental potential of the organ&#45;forming cells and facilitating their differentiation.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">LACY</span>, an Arabidopsis Receptor&#45;like Cytoplasmic Kinase is Required for Proper Organ Growth</div>

<div>

	<span style="text-decoration: underline" >
	Carl-Erik Tornqvist</span>
	<sup>1</sup>, 

	<span  >
	Ronan O'Malley</span>
	<sup>1</sup>, 

	<span  >
	Anthony B. Bleecker</span>
	<sup>1</sup>, 

	<span  >
	Sara E. Patterson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Wisconsin, Madison, USA

</div>

<div>In an effort to elucidate the functions of Arabidopsis receptor&#45;like kinases &#40;RLKs&#41;, we have isolated homozygous T&#45;DNA lines with insertions in 150 different RLKs with unknown function. Of particular interest is a mutant we call <span style="font-style: italic">lacy</span>, with a single T&#45;DNA insertion, which displays asymmetric leaf morphology, small stature, increased axillary branching, and irregular trichome growth. <span style="font-style: italic">lacy</span> is a recessive mutation caused by a disruption in a Receptor&#45;like Cytoplasmic Kinase &#40;RLCK&#41;. RLCKs belong to the RLK family, but have neither a transmembrane nor an extracellular domain. To uncover the function of <span style="font-style: italic">LACY</span>, we have carried out phenotypic observations, analyzed gene expression, and looked at possible genetic interactions. We have conducted extensive phenotypic characterizations of <span style="font-style: italic">lacy</span>, including basic plant morphology, vasculature, and trichome formation. RT&#45;PCR analysis allowed us to determine if the <span style="font-style: italic">lacy</span> phenotype is associated with altered expression of hormone response genes and&#47;or other genes involved in growth and development. To link <span style="font-style: italic">LACY</span>&#8217;s function to current developmental pathway models, several marker lines have been crossed to <span style="font-style: italic">lacy</span>, harboring hormone, homeobox, meristem maintenance, and cell division gene promoter&#45;reporter constructs. Studies of genetic interactions between <span style="font-style: italic">LACY</span> and genes critical to growth and development will also be presented. Since RLKs have been implicated in plant functions other than growth and development, such as disease resistance and the self&#45;incompatibility response, the homozygous T&#45;DNA collection will also serve as a valuable resource for researchers interested in many aspects of plant science. Distribution to the ABRC of all homozygous lines is being completed, making this collection available to the entire Arabidopsis community.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Direct Interaction of AGL24 and SOC1 Integrates Flowering Signals in Arabidopsis</div>

<div>

	<span  >
	Chang Liu</span>
	<sup>1</sup>, 

	<span  >
	Hongyan Chen</span>
	<sup>1</sup>, 

	<span  >
	Hong Ling Er</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Hao Yu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National University of Singapore

</div>

<div>During the transition from vegetative to reproductive growth, the shoot meristem of flowering plants acquires the inflorescence identity to generate flowers rather than vegetative tissues. An important regulator that promotes the inflorescence identity in Arabidopsis is AGAMOUS&#45;LIKE 24 &#40;AGL24&#41;, a MADS&#45;box transcription factor. By using a functional estradiol&#45;inducible system in combination with microarray analysis, we identified AGL24&#45;induced genes including SUPPRESSOR OF OVEREXPRESSION OF CO 1 &#40;SOC1&#41;, a floral pathway integrator. Chromatin immunoprecipitation &#40;ChIP&#41; analysis of a functional AGL24&#45;6HA tagged line revealed in vivo binding of AGL24&#45;6HA to the regulatory region of SOC1. Mutagenesis of AGL24 binding site in SOC1 promoter resulted in the reduction of SOC1 expression. Our results show that SOC1 is one of the direct targets of AGL24, and that SOC1 expression is upregulated by AGL24 mainly at the floral transitional stage. ChIP analysis using a functional SOC1&#45;9myc tagged line and promoter deletion analysis also revealed in vivo binding of SOC1&#45;9myc to the regulatory regions of AGL24 and upregulation of AGL24 by SOC1. Furthermore, we found that like other flowering genetic pathways, the gibberellin pathway independently controls AGL24 and SOC1. These observations suggest that during floral transition a positive regulatory feedback loop conferred by direct transcriptional regulation between AGL24 and SOC1 integrates flowering signals. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Stamen and carpel development in Arabidopsis and Antirrhinum. How proteins evolution changes mechanisms of male and female organ development.</div>

<div>

	<span style="text-decoration: underline" >
	Chiara A. Airoldi</span>
	<sup>1</sup>, 

	<span  >
	Barry Causier</span>
	<sup>1</sup>, 

	<span  >
	Sara Bergonzi</span>
	<sup>1</sup>, 

	<span  >
	Brendan Davies</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centre for Plant Sciences University of Leeds, Leeds, United Kingdom

</div>

<div>The development of male and female floral organs has been one of the most studied processes in plant development; it has a key role for plant reproduction and therefore a high agronomical importance. To study stamen and carpel development we are using two plants that played a fundamental role in the study of floral organ development: Arabidopsis and Antirrhinum. In these two plants the evolution of genes that regulate stamen and carpel development has followed two different paths. In Arabidopsis, development of both male and female floral organs is determined by the presence of the MADS&#45;box transcription factor AGAMOUS &#40;AG&#41;. In Antirrhinum two genes, which are phylogenetically related to <span style="font-style: italic">AG</span>, are responsible for the same process: <span style="font-style: italic">FARINELLI &#40;FAR&#41;</span> and <span style="font-style: italic">PLENA &#40;PLE&#41;</span>. PLE and FAR differ from AG in their ability to determine the development of stamens and carpels. Indeed, during the course of evolution they have undergone a process of subfunctionalisation such that FAR is more responsible for the development of male organs and PLE is more involved in the formation of female organs. These differences can be observed when <span style="font-style: italic">PLE</span>, <span style="font-style: italic">FAR</span> and <span style="font-style: italic">AG</span> are overexpressed in Arabidopsis. Whereas <span style="font-style: italic">AG</span> overexpression converts sepals into carpelloid structures and petals into stamens, <span style="font-style: italic">FAR</span> overexpression only converts petals into stamens, and <span style="font-style: italic">PLE</span> mainly converts sepals into carpelloid structures. <span style="font-style: italic">AG</span>, <span style="font-style: italic">PLE</span> and <span style="font-style: italic">FAR</span> are all expressed in stamens and carpels, indicating that their different functions are driven by differences at the protein level. To identify if these protein differences are located in a specific domain of the protein, we overexpressed different domain fusions between FAR, AG and PLE in Arabidopsis. From an analysis of the flowers of these plants we demonstrated that the C terminal domain is responsible for the differential ability of AG PLE and FAR to induce female or male development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Repression of <span style="font-style: italic">AUXIN RESPONSE FACTOR10</span> by microRNA160 Is Critical for Plant Development</div>

<div>

	<span style="text-decoration: underline" >
	Hiroyuki Nonogaki</span>
	<sup>1</sup>, 

	<span  >
	Po-Pu Liu</span>
	<sup>1</sup>, 

	<span  >
	Taiowa Montgomery</span>
	<sup>2,3</sup>, 

	<span  >
	Noah Fahlgren</span>
	<sup>2,3</sup>, 

	<span  >
	Kristin Kasschau</span>
	<sup>2,3</sup>, 

	<span  >
	James Carrington</span>
	<sup>2,3</sup>

</div>

<div>

	<sup>1</sup>Department of Horticulture, Oregon State University, Corvallis, OR 97331, USA, 

	<sup>2</sup>Center for Genome Research and Biocomputing, Oregon State University, Corvallis, OR 97331, USA, 

	<sup>3</sup>Department of Botany and Plant Pathology, Oregon State University, Corvallis, OR 97331, USA

</div>

<div>AUXIN RESPONSE FACTORs &#40;ARFs&#41; are transcription factors involved in auxin signal transduction during many stages of plant growth development. <span style="font-style: italic">ARF10</span>, <span style="font-style: italic">ARF16</span> and <span style="font-style: italic">ARF17</span> are targeted by microRNA160 &#40;miR160&#41; in <span style="font-style: italic">Arabidopsis thaliana</span>. Here, we show that negative regulation of <span style="font-style: italic">ARF10</span> by miR160 is critical for normal plant development. Transgenic plants expressing a miR160&#45;resistant form of <span style="font-style: italic">ARF10</span>, which has silent mutations in the miRNA target site &#40;termed <span style="font-style: italic">mARF10</span>&#41;, exhibited developmental defects such as serrated leaves, curled stems, contorted flowers and twisted siliques. These phenotypes were not observed in wild&#45;type plants or plants transformed with the targeted <span style="font-style: italic">ARF10</span> gene. During seed germination and seedling establishment, <span style="font-style: italic">mARF10</span> mutant seeds and plants were hypersensitive to ABA in a dose&#45;dependent manner. ABA hypersensitivity was mimicked in wild&#45;type plants by exogenous auxin. In contrast, overexpression of <span style="font-style: italic">MIR160</span> &#40;<span style="font-style: italic">35S:MIR160</span>&#41; resulted in reduced sensitivity to ABA during germination. These results indicate that repression of <span style="font-style: italic">ARF10</span> by miR160 plays a critical role in seed germination and postembryonic developmental programs.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The peroxin ABSTINENCE BY MUTUAL CONSENT is required for gamete&#45;gamete recognition during pollen tube reception.</div>

<div>

	<span style="text-decoration: underline" >
	Aur&#233;lien Boisson-Dernier</span>
	<sup>1</sup>, 

	<span  >
	Sabine Frietsch</span>
	<sup>2</sup>, 

	<span  >
	Tae-Houn Kim</span>
	<sup>1</sup>, 

	<span  >
	Marie Beverley Dizon</span>
	<sup>1</sup>, 

	<span  >
	Julian Schroeder</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>UCSD, La Jolla, California, USA, 

	<sup>2</sup>UNR, Reno, Nevada, USA

</div>

<div>In animals and plants, fertilization relies on complex and specialized mechanisms that achieve the precise delivery of the male gamete to the female gamete and their subsequent union. In plants, the male gametophyte or pollen tube carries two sperm cells over a long distance through the maternal tissues to the female gametophyte or embryo sac. During this long assisted journey, a multitude of signal exchanges between the pollen tube, the maternal diploid tissues and the haploid embryo sac take place that culminate in pollen tube reception, the process through which the pollen tube release the sperm cells into the female gametophyte. Here, we report the isolation and characterization of the <span style="font-style: italic">Arabidopsis</span> mutant <span style="font-style: italic">abstinence by mutual consent</span> where pollen tube reception is impaired only when an <span style="font-style: italic">amc</span> pollen tube reaches an <span style="font-style: italic">amc</span> embryo sac leading to pollen tube overgrowth and the absence of sperm release. Moreover, <span style="font-style: italic">AMC</span> is strongly expressed in both male and female gametophytes during fertilization but strongly down&#45;regulated during subsequent embryo development. We further show that YFP&#45;AMC fusion localized to peroxisomes and that AMC functions as a peroxin that mediates protein import into peroxisomes. The identification of <span style="font-style: italic">AMC</span> as the first gene required for pollen tube reception with essential roles in both male and female gametophytes, points towards a key role for peroxisomes in gamete recognition and successful sperm release.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Analyses of CLV3 molecular mechanisms, and gain&#45;of&#45;function phenotypes of dodeca CLE peptides.</div>

<div>

	<span style="text-decoration: underline" >
	Shinichiro Sawa</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Tokyo, Tokyo, Japan

</div>

<div>12 amino acid MCLV3 peptide function in meristem size maintenance in Arabidopsis. MCLV3 treatment with wild type plants reduced SAM and RAM size. Reporter gene activities of CLV3::GUS, CLV3::GFP, and WUS::GUS were decreased when these reporter lines were treated with MCLV3 peptide. MCLV3 did not function in the clv1 and clv2 mutants, MCLV3 function in the CLV signaling pathway. We isolated mutants, which are resistant against to the MCLV3 peptides. These mutants can grow even on the agar medium including MCLV3 peptides. Here we will discuss about these mutant and causal genes. <br/>     On the other hand, dodeca CLE peptides are known to have a function in RAM size regulation. Using 26 chemically synthetic CLE peptides, which correspond to the predicted products of the 31 Arabidopsis CLE genes, we investigated the CLE peptide function in Arabidopsis SAM. Furthermore, we examined the CLE peptides in rice and moss, Physcomitrella patens. Some CLE peptides reduced colony size, cell elongation rate, and gametophore formation. Interestingly they did not affect meristematic nature in moss. moss may not utilize CLE peptides in the regulation of meristematic apical cell activity. Moss does not have multicellular meristem, and the moss might not need to utilize CLE peptides as intercellular signaling molecule to organize the meristematic apical cell architecture. Instead, CLE peptides may function in the cell elongation and gametophore formation in the moss. CLE9&#47;10 was most effective to reduce colony size, and the peptide sequence of CLE9&#47;10 and PpCLE1 has only one amino acid difference. These results may suggest that the CLE function between bryophyte and angiosperm is highly diverged, and the CLE function may be varied from as an intracellular signaling molecule in bryophyte to as an intercellular signaling molecule in angiosperm. This idea may be supported by the result that the putative CLE protein sequences of Arabidopsis and rice have typical signal sequence for secretion, but PpCLE1 of moss does not have. CLE protein may be evoloved as a secreted intercellular signaling molecule when plant acquired organized multicellular meristem during its evolution.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">HYL1 regulates Leaf Form through Coordination of MIR166 and MIR319 genes</div>

<div>

	<span  >
	Yanfei Mao</span>
	<sup>1</sup>, 

	<span  >
	Lin Yu</span>
	<sup>1</sup>, 

	<span  >
	Liguo Jia</span>
	<sup>1</sup>, 

	<span  >
	Zhongyuan Liu</span>
	<sup>1</sup>, 

	<span  >
	Guixuan Song</span>
	<sup>1</sup>, 

	<span  >
	Feijie Wu</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Yuke He</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Shanghai Institute of Plant Physiology &amp;  Ecology, Chinese Academy of Sciences, Shanghai, China

</div>

<div>During leaf differentiation from shoot apical meristem,, the three&#45;dimensional form of leaves is specified along proximodistal &#40;base&#45;to&#45;tip&#41;, dorsoventral &#40;top&#45;to bottom&#41; and mediolateral &#40;middle&#45;to&#45;margin&#41; planes. Null mutation of Arabidopsis HYPONASTIC LEAVES1, which participates in microRNA &#40;miRNA&#41; biogenesis, changed the three&#45;dimensional form of leaves as the hyl1 leaves become abnormally lengthy, narrow, sessile and transversely incurved.  Despite the recent progress in understanding of HYL1&#45;controled miRNA processing, it remains uncertain whether or how the shofar&#45;shaped leaves of hyl1 mutants is caused by impaired miRNA processing. To clarify the role of HYL1 in development of leaf form, we crossed hyl1 with the gain&#45;of&#45;function mutants of MIR166 and MIR319a, respectively, and made a series of double or triple mutants containing the mutant alleles of the miRNA&#45;targeted genes. Genetic analysis, in combination with expression analysis, showed that in hyl1 leaves the loss of polarity in the lateral area was caused by the impaired miR166 processing whereas unbalanced division of the distal cells and the marginal cells was related to the reduced accumulation of miR319, both contributed to the specification of the shofar&#45;shaped leaves. In rosette leaves of jba&#45;1d jaw&#45;1D double mutants, biogenesis of either miR166 or miR319 is enhanced when defects in polarity and cell division became severe, thus demonstrating mutual enhancive interactions between MIR166g and MIR319a genes. These results suggest that HYL1 regulate leaf form through coordination of MIR166 and MIR319 genes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">An Evolutionarily Conserved Mechanism Delimiting SHR movement Defines a Single Layer of Endodermis in Plants</div>

<div>

	<span style="text-decoration: underline" >
	Hongchang Cui</span>
	<sup>1</sup>, 

	<span  >
	Mitchell P. Levesque</span>
	<sup>1</sup>, 

	<span  >
	Teva Vernoux</span>
	<sup>1</sup>, 

	<span  >
	Jee W. Jung</span>
	<sup>1</sup>, 

	<span  >
	Alice J. Paquette</span>
	<sup>1</sup>, 

	<span  >
	Kimberly L. Gallagher</span>
	<sup>1</sup>, 

	<span  >
	Jean Y. Wang</span>
	<sup>1</sup>, 

	<span  >
	Ikram Blilou</span>
	<sup>2</sup>, 

	<span  >
	Ben Scheres</span>
	<sup>2</sup>, 

	<span  >
	Philip N. Benfey</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Duke University, Durham, NC, USA, 

	<sup>2</sup>Utrecht University, Padualaan, Utrecht, the Netherlands

</div>

<div>SHR and SCR are key regulators of radial patterning in the Arabidopsis root. Produced in the stele, the SHR protein subsequently moves into an adjacent cell layer where it controls SCR transcription and endodermis specification. SCR is required for the asymmetric cell division that gives rise to the cortex and endodermis. Despite of its ability to move between cells, SHR movement is limited to only one cell layer. There was indication that SCR plays a role in restricting SHR movement, but the underlying molecular mechanism has been unknown. <br/><br/>Using a ChIP&#45;PCR assay, we find that SCR directly regulates its own transcription through a positive feedback loop. SCR also directly controls the transcription of other known SHR targets, and loss of SCR abolishes the ability of SHR to regulate these genes. Moreover, microarray experiments indicate that SHR and SCR share a large set of downstream genes. Protein immunoprecipitation and yeast two&#45;hybrid assays show that SCR forms a heterodimer with SHR through the GRAS domain. Based on these results we proposed a model whereby SCR blocks SHR movement by sequestering it into the nucleus through protein&#45;protein interaction and by a SHR&#47;SCR dependent feedback loop for SCR transcription that presents SHR escape. Supporting this model, SHR becomes largely cytoplasmic in scr mutant, and reduction in SCR transcript level by RNAi causes ectopic movement of SHR and a multi&#45;layer endodermis phenotype.<br/><br/>Finally, we find that functional homologs of SHR and SCR in rice also interact with each other and have expression patterns analogous to those of their Arabidopsis counterparts. Since nearly all plant species examined so far have a single layer of endodermis, we suggest that the mechanism we have uncovered in this study by which SCR restricts SHR movement is likely to be evolutionarily conserved. <br/><br/>&#42; Note: this work has been accepted by <span style="font-style: italic">Science</span>. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of <span style="font-style: italic">NEVERSHED</span>, a vesicle trafficking gene essential for floral organ shedding in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Michelle Leslie</span>
	<sup>1</sup>, 

	<span  >
	Michael Lewis</span>
	<sup>1</sup>, 

	<span  >
	Sarah Liljegren</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of North Carolina at Chapel Hill, Chapel Hill, NC, USA

</div>

<div>Abscission is a specialized cell separation event that allows plants to shed mature organs, such as leaves, flowers and fruit.  Recent work has shown that <span style="font-style: italic">NEVERSHED</span> &#40;<span style="font-style: italic">NEV</span>&#41; and <span style="font-style: italic">INFLORESCENCE DEFICIENT IN ABSCISSION</span> &#40;<span style="font-style: italic">IDA</span>&#41; are each required for late stages of floral organ abscission in <span style="font-style: italic">Arabidopsis</span>.  IDA is the founding member of a new class of putative signaling ligands &#40;Butenko et al., 2003&#41;, and we have discovered that NEV is involved in vesicle trafficking.  Currently, I am taking genetic and molecular approaches to further define the roles of NEV and IDA, explore their relationship, and elucidate the pathways that control cell separation.  To identify additional components of the NEV pathway, we have carried out a suppressor screen for mutations that restore abscission in <span style="font-style: italic">nev</span> mutant flowers.  Interestingly, I have found that two allelic, recessive <span style="font-style: italic">nev</span> suppressors contain predicted loss&#45;of&#45;function mutations in <span style="font-style: italic">EVERSHED</span> &#40;<span style="font-style: italic">EVR</span>&#41;, a previously uncharacterized member of the leucine&#45;rich repeat receptor&#45;like kinase &#40;LRR&#45;RLK&#41; family.  Further genetic analysis has also revealed that neither mutant allele of <span style="font-style: italic">EVR</span> suppresses the floral organ shedding defects of the <span style="font-style: italic">ida</span> mutant, suggesting that <span style="font-style: italic">IDA</span> functions downstream of <span style="font-style: italic">NEV</span> and <span style="font-style: italic">EVR</span>, or that <span style="font-style: italic">IDA</span> acts in a separate pathway.  In addition to our genetic characterization of NEV, we have developed NEV&#45;specific antisera and <span style="font-style: italic">NEV&#45;GFP</span> transgenic lines that I am using to test the co&#45;localization of NEV with different endomembrane compartments.  Results thus far show that NEV shows near complete co&#45;localization with putative markers of recycling endosomal compartments and partial co&#45;localization with markers of the Golgi and early endosome.  Future experiments aimed at identifying the proteins that physically interact with NEV, EVR and IDA will further our understanding of the late stages of abscission.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Analysis of compensation reveals three different modes that modulate cell expansion in determinate organs</div>

<div>

	<span style="text-decoration: underline" >
	Ali Ferjani</span>
	<sup>1</sup>, 

	<span  >
	Naoko Ishikawa</span>
	<sup>2</sup>, 

	<span  >
	Tetsuya Hisanaga</span>
	<sup>3</sup>, 

	<span  >
	Ushio Fujikura</span>
	<sup>4</sup>, 

	<span  >
	Minoru Kubo</span>
	<sup>5</sup>, 

	<span  >
	Taku Demura</span>
	<sup>6</sup>, 

	<span  >
	Hiroo Fukuda</span>
	<sup>1</sup>, 

	<span  >
	Gorou Horiguchi</span>
	<sup>1</sup>, 

	<span  >
	Hirokazu Tsukaya</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Graduate School of Science, The University of Tokyo, 

	<sup>2</sup>National Institute for Basic Biology, 

	<sup>3</sup>Faculty of Sciences, The University of Tokyo, 

	<sup>4</sup>The Graduate University for Advanced Studies, 

	<sup>5</sup>JST, ERATO, 

	<sup>6</sup>RIKEN Plant Science Center

</div>

<div>Leaf size is determined by the number and the size of leaf cells. Therefore, cell division and expansion processes are fundamental for leaf&#45;size regulation. Organ&#45;wide cross&#45;talks between these two processes have been suggested, because inhibited cell division in leaves is compensated by increased cell size. This intriguing phenomenon that we named compensation has been frequently reported, however, it is not yet understood how, when and where it occurs. In this study, to answer these questions, we isolated five new <span style="font-style: italic">Arabidopsis thaliana</span> mutants &#40;<span style="font-style: italic">fugu1</span>&#45;<span style="font-style: italic">fugu5</span>&#41; that exhibit compensation. <span style="font-style: italic">fugu</span> mutants were characterized together with <span style="font-style: italic">an3</span>, <span style="font-style: italic">erecta</span>, and a <span style="font-style: italic">KRP2</span> o&#47;e, previously reported to exhibit compensation. Time&#45;course analyses of leaf development revealed that compensation in <span style="font-style: italic">fugu2&#45;1</span>, <span style="font-style: italic">fugu5&#45;1</span>, <span style="font-style: italic">an3&#45;4</span>, and <span style="font-style: italic">er&#45;102</span> mutants is induced post&#45;mitotically, indicating that cell enlargement is not caused by the uncoupling of cell division from cell growth. In each of the mutants, either the rate &#40;<span style="font-style: italic">fugu2&#45;1</span>, <span style="font-style: italic">er&#45;102</span>, <span style="font-style: italic">an3&#45;4</span>&#41; or duration &#40;<span style="font-style: italic">fugu5&#45;1</span>&#41; of cell expansion was selectively enhanced. Interestingly, DNA microarray analysis in <span style="font-style: italic">fugu2</span> and <span style="font-style: italic">fugu5</span> revealed that the expression level of substantially different gene sets was affected, confirming that compensation in these mutants is regulated differently. In contrast, we found that compensation in <span style="font-style: italic">KRP2</span> o&#47;e occurs during cell proliferation. Compensation did not occur in roots despite decreased cell numbers. Flow cytometric analyses revealed that increases in ploidy level are not always required to trigger compensation, suggesting that compensation is only partially mediated by ploidy&#45;dependent processes. Therefore, compensation indicates the presence of mechanisms that are specific to determinate organs and regulate their size by coordinating cell proliferation and expansion at least by three different modes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A Novel Transcriptional Cascade Regulating Expression of Heat Stress Proteins during Seed Development of Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Sachin Kotak</span>
	<sup>1</sup>, 

	<span  >
	Elizabeth Vierling</span>
	<sup>2</sup>, 

	<span  >
	Helmut B&#228;umlein</span>
	<sup>3</sup>, 

	<span  >
	Pascal von Koskull-D&#246;ring</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Molecular Biosciences, Biocenter N200/R306, Goethe University, D-60439 Frankfurt, Germany, 

	<sup>2</sup>Department of Biochemistry and Molecular Biophysics, University of Arizona, Tucson, Arizona 85721, 

	<sup>3</sup>Institute of Plant Genetics and Crop Plant Research, D-06466 Gatersleben, Germany

</div>

<div>Within the Arabidopsis thaliana family of 21 heat stress transcription factors &#40;Hsfs&#41;, HsfA9 is exclusively expressed in late stages of seed development. Here, we present evidence that developmental expression of HsfA9 is regulated by the seed&#45;specific transcription factor ABSCISIC ACID&#8211;INSENSITIVE3 &#40;ABI3&#41;. Intriguingly, ABI3 knockout lines lack detectable levels of HsfA9 transcript and protein, and further ectopic expression of ABI3 conferred the ability to accumulate HsfA9 in response to abscisic acid in transgenic plantlets. Consequently, the most abundant heat stress proteins &#40;Hsps&#41; in seeds &#40;Hsp17.4&#45;CI, Hsp17.7&#45;CII, and Hsp101&#41; were not detectable in the ABI3 knockout lines, but their expression could be detected in plants ectopically expressing HsfA9 in vegetative tissues. Furthermore, this seed&#45;specific transcription factor cascade was reconstructed in transient &#223;&#45;glucuronidase reporter assays in mesophyll protoplasts by showing that ABI3 could activate the HsfA9 promoter, whereas HsfA9 in turn was shown to be a potent activator on the promoters of Hsp genes. Thus, our study establishes a genetic framework in which HsfA9 operates as a specialized Hsf for the developmental expression of Hsp genes during seed maturation. </div>

</div>





<div class="abs">

<div style="font-weight:bold">THE <span style="font-style: italic">AGAMOUS</span> TARGET GENE <span style="font-style: italic">ATH1</span> IS REQUIRED FOR DEVELOPMENT OF THE BASAL REGION OF THE FLOWER .</div>

<div>

	<span style="text-decoration: underline" >
	Concepcion Gomez-Mena</span>
	<sup>1</sup>, 

	<span  >
	Robert Sablowski</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>John Innes Centre, Norwich, NR4 7UH UK

</div>

<div>Floral organs are initiated in the flanks of the floral meristems in concentric regions known as whorls. The identities of these primordia are specified by the combinatorial activity of 3 types of floral homeotic genes. These master genes must be central nodes in complex networks of genes that control the development of different floral organs. Despite many efforts, little is known about these gene networks functions.&#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In Arabidopsis, the floral homeotic gene <span style="font-style: italic">AG</span> specifies stamen and carpel identity. We have used an AG inducible system to identify genes activated by AG in early stages of flower development &#40;Gomez&#45;Mena et al 2005&#41;. Here, we report the functional characterization of one of these early targets of AG: <span style="font-style: italic">ATH1</span> &#40;At4g322980&#41;, which belongs to the TALE superfamily of homeodomain proteins that also includes BEL1. We have identified an insertional mutant in the <span style="font-style: italic">ATH1</span> gene that initially appeared to be defective in floral organ abscission. In wt plants, intact floral organs are shed soon after anthesis, whereas in the <span style="font-style: italic">ath1</span> mutant they remained attached to the developing fruit indefinitely. However, a closer look revealed multiple defects in the basal region of the <span style="font-style: italic">ath1</span> mutant flower, including not only a failure to develop the abscission zone, but also partial organ fusions and small or absent nectaries. Ectopic expression of ATH1 resulted in a compressed inflorescence that resembled the phenotype of mutants with defects in the synthesis or sensitivity to gibberellins. The <span style="font-style: italic">ath1</span> mutation modified the pattern of expression of a gibberellin 3&#45;oxidase &#8211; GUS reporter in young floral buds, supporting the idea that <span style="font-style: italic">ATH1</span> may control gibberellin biosynthesis. Together these results indicate that <span style="font-style: italic">ATH1</span> is required for the proper formation of multiple structures in the basal region of the flower, possibly through localized regulation of gibberellin biosynthesis.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic Dissection of Parent of Origin Effects in Seed Development</div>

<div>

	<span style="text-decoration: underline" >
	Reza Shirzadi</span>
	<sup>1</sup>, 

	<span  >
	Moritz K. Nowack</span>
	<sup>2</sup>, 

	<span  >
	Reidunn B. Aalen</span>
	<sup>1</sup>, 

	<span  >
	Arp Schnittger</span>
	<sup>2</sup>, 

	<span  >
	Paul E. Grini</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biosciences, University of Oslo, Oslo, Norway, 

	<sup>2</sup>University Group at the Max-Planck-Institute for Plant Breeding Research, Department of Botany III, University of Cologne, Cologne, Germany

</div>

<div>In the developing offspring of mammals the two parental genomes are differentially expressed due to epigenetic marks. A parent of origin dependent expression has also been observed in the offspring of flowering plants and mutations in the imprinting machinery lead to embryonic lethality, primarily affecting the development of the endosperm, a structure in the seed that nourishes the embryo analogous to the mammalian placenta. Seed development is the product of the double fertilization of the egg cell and the central cell by two sperm cells from the pollen and requires a coordinated interplay of the embryo, the endosperm and the maternal seed coat. We have used a mutant in the <span style="font-style: italic">Arabidopsis Cdc2&#47;CDC28</span> homologue, <span style="font-style: italic">CDKA;1</span> as a tool to dissect the involvement of parental gene programs in seed development. <span style="font-style: italic">cdka;1</span> mutant pollen fails to undergo the second pollen mitosis, resulting in pollen with a single sperm cell that exclusively fertilize the egg cell. Although not fertilized, the central cell in <span style="font-style: italic">cdka;1</span> ovules with fertilized egg cells is triggered to initiate endosperm proliferation. Surprisingly, viable seedlings can develop from seeds lacking a paternal contribution to the endosperm, given the mother is mutant for any of the <span style="font-style: italic">FIS</span> genes that encode Polycomb group &#40;PcG&#41; chromatin remodeling factors, thus suggesting that the action of the PcG FIS&#45;complex balances the contribution of the paternal genome. In order to identify further factors involved in crosstalk between the components of the seed and parent of origin specific regulation of seed development, we have generated microarray transcriptional profiles of seeds with uniparental endosperm. Our preliminary data identifies several regulated genes that could represent novel paternal specific transcripts &#40;downregulated&#41; or maternal targets of paternal repressors &#40;upregulated&#41;. </div>

</div>





<div class="abs">

<div style="font-weight:bold">MIKC&#42; MADS&#45;protein complexes regulate pollen maturation in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Wim Verelst</span>
	<sup>1</sup>, 

	<span  >
	David Twell</span>
	<sup>2</sup>, 

	<span  >
	Heinz Saedler</span>
	<sup>1</sup>, 

	<span  >
	Thomas Muenster</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, Cologne, Germany, 

	<sup>2</sup>University of Leicester, Leicester, United Kingdom

</div>

<div>Pollen grains represent the haploid male gametophyte in flowering plants, and play an essential role in sexual reproduction. However, the pathways that regulate pollen development remain largely uncharacterized. We found that five pollen&#45;specific MIKC&#42; MADS transcription factor complexes play a major role in regulating gene expression during the last stages of pollen development in <span style="font-style: italic">Arabidopsis thaliana</span>. The absence of these MADS&#45;complexes results in a partial arrest of the normal, developmentally regulated changes in the pollen transcriptome. Microarray experiments on mutant pollen, lacking all MIKC&#42; complexes, revealed that the expression of nearly 500 immature&#45;pollen&#45;specific genes fails to be downregulated before anthesis, while at least 250 mature&#45;pollen&#45;specific genes are not properly activated. Overall, the expression of more than 1000 genes is directly or indirectly regulated by the MIKC&#42; complexes in Arabidopsis pollen grains. These include important components of hormone pathways, the secretory pathway, cell wall synthesis, carbohydrate metabolism and the ubiquitin&#45;mediated proteolysis system, as well as 27 non&#45;MADS transcription factors. In addition, comparison of the pollen transcriptome of various single, double and triple MIKC&#42; mutants enabled us to visualize the functional redundancy between the five different MIKC&#42; complexes on the level of their target genes. Our data thus indicate that the MIKC&#42; complexes rank high in a regulatory network that controls pollen maturation. These datasets form a solid experimental basis for the systematic unravelling of the regulatory pathways that control pollen development in Arabidopsis, and provide a unique insight into the complexity of a pollen&#45;specific transcription factor network.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification and functional characterization of BIGPETALp and BIP, two interacting protein involved in the control of petal size in Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Emilie Varaud</span>
	<sup>1</sup>, 

	<span  >
	Florian Brioudes</span>
	<sup>1</sup>, 

	<span  >
	Judit Sz&#233;csi</span>
	<sup>1</sup>, 

	<span  >
	Karim Bordji</span>
	<sup>2</sup>, 

	<span  >
	Caroline Joly</span>
	<sup>1</sup>, 

	<span  >
	Philippe Vergne</span>
	<sup>1</sup>, 

	<span  >
	Mohammed Bendahmane</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Ecole Normale Sup&#233;rieure Lyon France, 

	<sup>2</sup>Universit&#233; Caen France

</div>

<div>Organ final size can be influenced by cell number or cell expansion or both. We identified a novel petal&#45;expressed bHLH encoding gene &#40;BIGPETAL or BPE&#41; which limits petal size by influencing post&#45;mitotic cell expansion in Arabidopsis thaliana. BPE is expressed via two proteins: a petal expressed protein &#40;BPEp&#41; and a ubiquitously expressed protein &#40;BPEub&#41;. Both proteins have identical N&#45;terminal domains &#40;containing the bHLH domain&#41; but have different C&#45;terminal domains. A yeast&#45;two hybrid screen was performed and permitted to identify a putative BPEp interacting protein we named BIP &#40;for BPEp Interacting Protein&#41;. Plants knockout for BIP have larger petals as a result of increased cell size, showing that like for BPEp, BIP interferes with post&#45;mitotic cell expansion. Our data suggests that BPEp and BIP must be part of a protein complex involved in the control of petal size by limiting cell expansion. The significance of this interaction in modulating and&#47;or participating to BPEp biological functions during petal morphogenesis will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">In silico identification of co&#45;transcribed core cell cycle regulators and transcription factors in Arabidopsis</div>

<div>

	<span  >
	Yixing Wang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Ming Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Oklahoma State University, Stillwater, OK, USA

</div>

<div>Regulatory networks involving transcription factors and core cell cycle regulators are expected to play crucial roles in plant growth and development. In this report, we describe the identification of a putative network involving five core cell cycle regulators and six transcription factors via a two&#45;step in silico screening strategy. We first screened the Arabidopsis microarray data &#40;AtGeneExpress: Expression Atlas of Arabidopsis Development&#41; in http:&#47;&#47;www.arabidopsis.org&#47; for core cell cycle regulators and transcription factors detected in young leaves and stage&#45;15 carpels but not in mature and senesced leaves and sepals, petals and stamens at floral stage 15. We then explored the correlations in transcriptional profile among the cell cycle genes and the transcription factor genes identified in the first screening using the AtGenExpress Visualization Tool. The identified core cell cycle regulators and transcription factors fall into two groups with matching transcriptional profiles. One such group consists of TARDY ASYNCHRONOUS MEIOSIS &#40;CYCA1;2&#41;, CURLY LEAF and a Forkhead&#45;associated domain transcription factor gene, and the other group consists of CYCB1;1, CYCB2;1, CDKB1;2, CDKB2;2, AINTEGUMENTA, a MYB gene, another Forkhead&#45;associated domain transcription factor gene, and a SCARECROW family gene. Results from Pearson&#8217;s correlation coefficient analysis are also consistent with the two groups of genes being co&#45;transcribed, respectively. Promoter analysis revealed a possible web of cross&#45; and self&#45;regulations among the identified genes.  Because some of the genes are known regulators of cell proliferation and all of them are only predominantly transcribed in young organs, we predict that these genes are specially involved in organ growth in Arabidopsis. </div>

</div>





<div class="abs">

<div style="font-weight:bold">NUCLEAR&#45;PORE ANCHOR, the Arabidopsis Homolog of Tpr&#47;Mlp1&#47;Mlp2&#47;Megator, Is Involved in mRNA Export, SUMO Homeostasis and Affects Diverse Aspects of Plant Development</div>

<div>

	<span style="text-decoration: underline" >
	Xianfeng Morgan Xu</span>
	<sup>1</sup>, 

	<span  >
	Annkatrin Rose</span>
	<sup>2</sup>, 

	<span  >
	Sivaramakrishnan Muthuswamy</span>
	<sup>1</sup>, 

	<span  >
	Sun Yong Jeong</span>
	<sup>3</sup>, 

	<span  >
	Sowmya Venkatakrishnan</span>
	<sup>1</sup>, 

	<span  >
	Qiao Zhao</span>
	<sup>1</sup>, 

	<span  >
	Iris Meier</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Biotechnology Center and Department of Plant Cellular and Molecular Biology, The Ohio State University, Columbus, OH 43210, USA., 

	<sup>2</sup>Department of Biology, Appalachian State University, Boone, NC 28608, USA., 

	<sup>3</sup>Department of Biochemistry and Biophysics, University of North Carolina at Chapel Hill, Chapel Hill, NC 27599, USA.

</div>

<div>Vertebrate Tpr and its yeast homologs Mlp1&#47;Mlp2 are long coiled&#45;coil proteins associated with the inner basket filaments of the nuclear pore. They are involved in mRNA export, telomere organization, spindle pole assembly, and unspliced RNA retention. We have identified a single gene in Arabidopsis thaliana, NUCLEAR PORE ANCHOR &#40;NUA&#41;, which encodes a protein of 237 kD with similarity to Tpr. Immunolocalization in Arabidopsis root cells demonstrates that NUA is located at the inner surface of the nuclear envelope in interphase and in the vicinity of the spindle in prometaphase. Four T&#45;DNA insertion lines were characterized in detail. They comprise an allelic series of increasing severity for several correlating phenotypes, such as early flowering under short days and long days, increased abundance of SUMO conjugates, altered expression of several flowering regulators, and nuclear accumulation of poly&#40;A&#41;&#43; RNA. Nua mutants phenocopy mutants of EARLY IN SHORT DAYS 4 &#40;ESD4&#41;, an Arabidopsis SUMO protease concentrated at the nuclear periphery. Nua esd4 double mutants resemble nua and esd4 single mutants, suggesting that the two proteins act in the same pathway or complex, supported by yeast two&#45;hybrid interaction. Together, our data indicate that NUA is a component of nuclear pore&#45;associated steps of sumoylation and mRNA export in plants, and that defects in these processes affect the signaling events of flowering time regulation and additional developmental processes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Non&#45;cell autonomous KNAT1 function for control of plant architecture</div>

<div>

	<span  >
	Yeonggil Rim</span>
	<sup>1</sup>, 

	<span  >
	Jin-Hee Jung</span>
	<sup>1</sup>, 

	<span  >
	Raju Datla</span>
	<sup>2</sup>, 

	<span  >
	David Jackson</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Jae-Yean Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Division of Applied Life Science, PMBBRC/EB-NCRC, Gyeongsang National University, Jinju 660-701, Korea, 

	<sup>2</sup>Plant Biotechnology Institute, National Research Council of Canada, 110 Gymnasium Place, Saskatoon, SK, Canada S7N 0W9, 

	<sup>3</sup>Cold Spring Habor Laboratory, 1 bungtown Rd, Cold Spring Harbor NY11724, USA

</div>

<div>How plant shape&#47;architecture is regulated is a central question in plant biology. Arabidopsis brevipedicellus &#40;bp&#41; mutants provide a wonderful system to study the control of plant architecture, especially related to the patterning of stems.  bp mutants show a phenotype with shorter internodes and pedicel lengths, downward pointing siliques, a compact inflorescence architecture, bends at nodes and an epidermal stripe of disorganized cells along the stem. BP encodes the KNAT1 homeodomain protein, a member of the KNOTTED1&#45;related HOMEOBOX &#40;KNOX&#41; proteins, which is the strongest Arabidopsis homolog of maize KNOTTED1 that can traffic cell&#45;to&#45;cell. KNAT1 is specifically expressed in the cortical cell layers of the inflorescence stem and pedicel, but its knock&#45;out alleles also display disruption in epidermal cell differentiation. This suggested that KNAT1 functions non&#45;cell autonomously either by its direct cell&#45;to&#45;cell trafficking or by the movement of some downstream factors. We carried out genetic complementation assay of bp mutations using a series of fusion constructs that express either cell&#45;autonomous or non&#45;cell autonomous KNAT1 fusion proteins. We here show the ability of cell&#45;to&#45;cell trafficking of KNAT1 is required for its biological function involved in the establishment of plant architecture.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Organ size, cell size and ploidy: how an increase in ploidy promotes cell and organ growth&#63;</div>

<div>

	<span style="text-decoration: underline" >
	Hirokazu Tsukaya</span>
	<sup>1</sup>, 

	<span  >
	Christian Breuer</span>
	<sup>2</sup>, 

	<span  >
	Keiko Sugimoto-Shirasu</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>The University of Tokyo, Tokyo, Japan, 

	<sup>2</sup>John Innes Centre, Norwich, UK

</div>

<div>Tetraploid plants generally have larger cells than diploid plants.<br/>Similarly, increasing genome size by endoreduplication has positive impacts<br/>on cell size in plants. To investigate these functional links between<br/>ploidy, cell size and organ size, we have generated series of tetraploid<br/>plants in several Arabidopsis mutant backgrounds. Mutations in the DNA<br/>topoisomerase VI complex arrest endoreduplication cycle prematurely, leading<br/>to the development of severely dwarf plants. In these mutants, many large<br/>cell types that normally endoreduplicate up to 32C are replaced with smaller<br/>cells &#40;Sugimoto&#45; Shirasu et al. 2002&#41;. We found that tetraploidisation in<br/>the mutant backgrounds doubles the nuclear DNA content and this partially<br/>rescues their cell and organ size phenotypes, suggesting that the mechanism<br/>that allows ploidy&#45;dependent cell enlargement is common between<br/>tetraploidisation and endoreduplication. To explore the genetic mechanism of<br/>this ploidy&#45;dependent cell enlargement, we are now in the process of making<br/>tetraploid plants from several other dwarf mutants that have specific<br/>defects in cell expansion. In addition, we have recently isolated several<br/>new mutants that have defects in the compensatory mechanism &#40;Fujikura et al.<br/>2007&#41; in which the reduction of cell number in leaf primordia triggers<br/>abnormal cell expansion&#65288;Tsukaya 2006&#41;.  We are now using these mutants to<br/>address whether this process involves ploidy&#45;dependent cell enlargement.<br/>Based on the results obtained from these experiments, we will discuss how<br/>ploidy may influence cell and organ size in plants.<br/><br/>References:<br/>Fujikura, U., Horiguchi, G., Tsukaya H &#40;2007&#41; Plant Cell Physiol. 48: 278&#45;286.<br/>Sugimoto&#45;Shirasu et al. &#40;2002&#41; Current Biol. 12: 1782&#45;1786.<br/>Tsukaya, H &#40;2006&#41; Ann. Rev. Plant Biol 57: 477&#8211;496<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The homeotic protein AGAMOUS controls late stamen development by regulation of jasmonic acid biosynthesis</div>

<div>

	<span  >
	Serena Lim Tze Soo</span>
	<sup>1</sup>, 

	<span  >
	Yu Hao</span>
	<sup>1,2</sup>, 

	<span  >
	Elliot M. Meyerowitz</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Toshiro Ito</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Temasek Life Sciences Laboratory, Singapore 117604, Singapore, 

	<sup>2</sup>National University of Singapore, Singapore 117543, Singapore, 

	<sup>3</sup>California Institute of Technology, Pasadena CA 91125, USA

</div>

<div>The Arabidopsis floral homeotic gene <span style="font-style: italic">AGAMOUS</span> &#40;<span style="font-style: italic">AG</span>&#41; plays a central role in reproductive organ  development. Although <span style="font-style: italic">AG</span> is the most studied example of a plant homeotic gene, the mechanisms by which a floral primordium responds to the genetic activities initiated by AG, and thereby develops stamens and carpels, are largely unknown. <span style="font-style: italic">AG</span> RNA is expressed in the organ primordia of floral 3rd and 4th whorls &#40;the positions of stamens and carpels, respectively&#41; from floral stage 3 until late in flower development, after all of the organs are fully formed. While early <span style="font-style: italic">AG</span> expression acts in specification of stamens and carpels, the role, if any, of continued <span style="font-style: italic">AG</span> expression in later flower development is unknown. <br/>In order to examine the timing of AG action and its possible late stage functions, we performed a series of time course experiments using a transgenic line with inducible AG activity, in an ag homozygous mutant background. We show that ectopic AG activity can transform organ primordia of as late as stage 6 floral buds into normal&#45;looking stamens. In contrast, activation of AG in floral buds after stage 7 results in 2nd and 3rd whorl organs that were stamen&#45;like filamentous petals. This suggests that AG can induce the gene activities necessary for filament elongation, but not for microsporogenesis, at late floral stages. We also show by using timed activation of AG that AG controls late&#45;stage stamen development, including anther morphogenesis and dehiscence, as well as filament formation and elongation. Furthermore, we show that AG controls maturation of stamens by regulating the transcription of a catalytic enzyme for the phytohormone jasmonic acid biosynthesis in late stamen development. Our results show that stamen identity and differentiation control by AG is achieved by the regulation of different transcriptional cascades in different floral stages, with organ specification cascades induced early, followed by activities necessary for filament formation and anther maturation.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The role of <span style="font-style: italic">YABBYs</span> in leaf patterning</div>

<div>

	<span style="text-decoration: underline" >
	Oliver Bonaccorso</span>
	<sup>1</sup>, 

	<span  >
	Melissa Stahle</span>
	<sup>1</sup>, 

	<span  >
	John Golz</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Genetics, University of Melbourne, Parkville, VIC 3010. Australia

</div>

<div>Patterning along the adaxial &#40;topside&#41;&#45;abaxial &#40;bottom side&#41; axis of a leaf requires the activity of several families of transcription factors that include the abaxially expressed YABBYs &#40;YABs&#41;. We present data showing that YABs also have a role in promoting adaxial cell identity. Using the Yeast two hybrid assay, we show that two closely related transcriptional co&#45;repressors LEUNIG &#40;LUG&#41; and LEUNIG&#45;HOMOLOGY &#40;LUH&#41; interact with three leaf expressed YABBYs &#8211; FILAMENTOUS FLOWER &#40;FIL&#41;, YAB3 and YAB5, suggesting these proteins might form a complex in planta. Consistent with this possibility, we find that lug enhances the leaf polarity defects of fil yab3 mutants. To further investigate the role of LUG and LUH in leaf development, we attempted to isolate lug luh double mutants. While our analysis indicates that the double mutants are embryonically lethal, we found that lug luh&#47;&#43; plants produce narrow leaves and occasionally abaxialised lotus or needle&#45;like leaves. These polarity defects become more frequent when either fil or fil and yab3 mutations are crossed into the lug luh&#47;&#43; background. Finally we also show that like fil yab3, lug luh&#47;&#43; enhances the polarity defects of kan1 kan2 double mutants.  &#13;<br/>The partial loss of adaxial cell identity in some lug luh&#47;&#43; leaves is incongruous with the reported abaxial promoting function of the YABs. To address this issue further we isolated yab2 and yab5 mutants and generated triple and quadruple combinations with fil and yab3.  Both fil yab3 yab5 and fil yab2 yab3 yab5 plants produce radially symmetric leaves lacking adaxial cell identity. These observations suggest that YABBYs promote adaxial identity and that this function is highly redundant.  Using microarrays we found that auxin response genes are up regulated following ectopic FIL expression as are KANADI1 and PHABULOSA.  Using cycloheximide we show that increased KAN1 expression is a direct result of FIL activity whereas changes in PHB expression are not. Based on these findings we propose a model in which YABs promote adaxial cell identity non cell&#45;autonomously and that this pathway is part of a system that allows leaf development to proceed independently of an adaxial promoting signal from the meristem.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Organogenesis of the perianth in Arabidopsis flowers is facilitated by the dual action of PETAL LOSS and AUX1</div>

<div>

	<span  >
	Edwin Lampugnani</span>
	<sup>1</sup>, 

	<span  >
	Aydin Kilinc</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	David Smyth</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Monash University, Melbourne, Vic., Australia

</div>

<div>We are interested in how organ primordia are initiated within flowers, and in identifying the genetic factors and pathways involved in this process. Flowers of mutants of the <span style="font-style: italic">PETAL LOSS</span> &#40;<span style="font-style: italic">PTL</span>&#41; gene of Arabidopsis show a progressive loss in the ability to generate petals, and the sepals are wider, closer together and sometimes fused. <span style="font-style: italic">PTL</span> encodes a trihelix transcription factor that is expressed between sepal primordia &#40;in the inter&#45;sepal zone&#41; during early stages of flower development. PTL may have a role in the suppression of growth between sepals, and a downstream effect that localises a signal required for petals to arise in the adjacent petal initiation zone. <br/><br/>To identify enhancers, particularly redundant factors involved in floral organ initiation, an EMS mutagenesis screen was conducted in <span style="font-style: italic">ptl</span> mutant background. We identified a recessive mutant &#40;<span style="font-style: italic">no petals</span>, <span style="font-style: italic">nop</span>&#41; lacking petals and nectaries &#40;which arise internal to the petals&#41;. We show that the <span style="font-style: italic">nop</span> mutant alone has normal flowers, but that it is incompletely dominant for petal initiation in the <span style="font-style: italic">ptl</span> homozygous background. Inversely, <span style="font-style: italic">ptl</span> shows incomplete dominance for petal initiation in the <span style="font-style: italic">nop</span> homozygous background. This reciprocal co&#45;dominance indicates that the two genes influence a common process in petal initiation. <br/><br/>The <span style="font-style: italic">NOP</span> gene was positionally cloned and identified as <span style="font-style: italic">AUX1</span>, an auxin influx carrier. We propose that auxin functions as a signal to promote the initiation of petals. Loss of AUX1 function alone apparently does not weaken the signal below an organ&#45;inducing threshold level, but the joint loss of PTL function results in distortions of growth in the inter&#45;sepal zone such that auxin levels are now insufficient to promote petal initiation in the adjacent regions.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The<span style="font-style: italic"> Jagged Lateral Organ</span> &#40;<span style="font-style: italic">JLO</span>&#41; gene regulates meristem maintenance and organ development by controlling auxin transport</div>

<div>

	<span style="text-decoration: underline" >
	Marina Bureau</span>
	<sup>1</sup>, 

	<span  >
	Lorenzo Borghi</span>
	<sup>2</sup>, 

	<span  >
	Madlen Rast</span>
	<sup>1</sup>, 

	<span  >
	R&#252;diger Simon</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institut f&#252;r Genetik, Heinrich-Heine Universit&#228;t, universit&#228;tstr.1, Duesseldorf, Germany, 

	<sup>2</sup>Swiss Federal Institut of Technology (ETH) Zurich, Institute of Plant Sciences, Universit&#228;tstr. 2, CH-8092 Z&#252;rich, Switzerland

</div>

<div>We have started to address the role of meristem&#45;to&#45;organ boundaries in the control of stem cell activities in meristems. To do this, we began with the analysis of the <span style="font-style: italic">Arabidopsis</span> <span style="font-style: italic">JLO</span> gene, a member of the LBD gene family of transcription factors. In <span style="font-style: italic">Arabidopsis</span>, LBD genes were shown to play a role in boundary establishment and communication between meristem and initiating lateral organs. RNA <span style="font-style: italic">in situ</span> hybridisation experiments showed that <span style="font-style: italic">JLO</span> is expressed in boundaries between meristem and organ primordia. Overexpression <span style="font-style: italic">JLO</span> causes formation of lobed leaves and meristem arrest, suggesting that<span style="font-style: italic">JLO</span> is involved in organ pattering and maintenance of stem cells. One role of <span style="font-style: italic">JLO</span> is to increase expression of meristem specific KNOX genes. Furthermore, we found that <span style="font-style: italic">JLO</span> reduces auxin transport by inhibiting the expression of the auxin export carrier PIN1.<span style="font-style: italic">JLO</span> loss&#45;of&#45;function mutants revealed embryo&#45;lethality when homozygous, demonstrating a prominent role of <span style="font-style: italic">JLO</span> in the early steps of development. We propose that <span style="font-style: italic">JLO</span> is required during the globular stage to control auxin dstribution and embryo patterning.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulatory proteins, DUO1 and DUO3, link cell cycle progression and cell specification in male germ line development</div>

<div>

	<span style="text-decoration: underline" >
	Lynette Brownfield</span>
	<sup>1</sup>, 

	<span  >
	Said Hafidh</span>
	<sup>1</sup>, 

	<span  >
	Anjusha Durbarry</span>
	<sup>1</sup>, 

	<span  >
	Anna Siderova</span>
	<sup>1</sup>, 

	<span  >
	Tony Wardle</span>
	<sup>1</sup>, 

	<span  >
	David Twell</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, University of Leicester, Leicester, UK.

</div>

<div>The post&#45;meiotic pathway of male gametophyte development in flowering plants that leads to the production of fertile male gametes or sperm cells is well documented in research that spans a century.  However the developmental mechanisms that integrate germ line cell divisions and gamete specification remain largely unknown. We have previously described a class of mutants, <span style="font-style: italic">duo pollen &#40;duo&#41;</span>, that specifically block generative cell cycle progression and provide compelling evidence for gametophytic control of germ line cell cycle progression &#40;Durbarry et al. &#40;2005&#41; Plant Physiol. 137, 297&#45;307&#41;.<br/><br/>In <span style="font-style: italic">duo1</span> pollen the generative cell fails at G2&#47;M phase of the cell cycle while the vegetative cell is unaffected.  DUO1 is a member of the R2R3 Myb transcription factor family that is expressed specifically in the male germ line &#40;generative and sperm cells&#41;, with the protein first detected shortly after pollen mitosis I.  We have used several marker lines to explore the role of DUO1 in pollen development.  We found that CyclinB1;1, is down regulated in <span style="font-style: italic">duo1</span> pollen highlighting the involvement of DUO1 in the decision of generative cells to enter mitosis.  Coupled with cell cycle progression, DUO1 also regulates sperm cell specification as several proteins expressed exclusively in the male germ cells are down regulated in <span style="font-style: italic">duo1</span> pollen.  <br/><br/>We have also identified DUO3, a novel protein with a single MYB&#45;like domain and acidic rich domains. In contrast to <span style="font-style: italic">DUO1</span>, <span style="font-style: italic">DUO3</span> is widely expressed in shoot and root meristems, vascular tissue and in generative and vegetative cells. Analysis of male germ line markers down regulated in <span style="font-style: italic">duo1</span> pollen reveals distinct patterns of male germ line&#45;specific transcription that depend on DUO3.  Thus, we conclude that DUO1 and DUO3 have essential and overlapping roles in male germ&#45;line development. These data provide the first molecular evidence for a regulatory link between cell cycle progression and the specification of functional sperm cells.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Comprehensive functional analyses of Arabidopsis transcription factors using the chimeric repressor gene silencing technology &#40;CRES&#45;T&#41;</div>

<div>

	<span  >
	Masahito Shikata</span>
	<sup>1,2</sup>, 

	<span  >
	Tomotsugu Koyama</span>
	<sup>1,2</sup>, 

	<span  >
	Nobutaka Mitsuda</span>
	<sup>1,2</sup>, 

	<span  >
	Kyoko Matsui</span>
	<sup>1</sup>, 

	<span  >
	Yoshimi Umemura</span>
	<sup>1,2</sup>, 

	<span  >
	Akira Iwase</span>
	<sup>1</sup>, 

	<span  >
	Miho Ikeda</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Masaru Ohme-Takagi</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Research Institute of Genome-based Biofactory, Advanced Industrial Science and Technology, Central 4, Tsukuba 305-8562, Japan, 

	<sup>2</sup>Core Research for Evolutional Science and Technology (CREST), Japan Science and Technology Agency (JST), Saitama 332-0012, Japan.

</div>

<div>It becomes evident that transcription regulation play an important role for the control of growth and development especially in plants. Therefore, identification of the function of transcription factors is an important subject for the control or improvement of plant traits. However, plant genes are frequently duplicated and many transcription factors form large family. This structural and functional redundancy often interferes with efforts to identify the functions of these factors. Even when gene&#45;knockout or antisense lines specific for a particular transcription factor can be isolated, such lines often fail to exhibit an informative phenotype that might provide some direct clue to the factor&#8217;s function, for example, a loss&#45;of&#45;function phenotype. To overcome such difficulties, we developed a novel gene silencing system, called Chimeric REpressor Gene&#45;SilencingTechnology &#40;CRES&#45;T&#41;, in which a chimeric repressor is produced by fusion of a transcription factor to the plant&#45;specific ERF associated Amphiphilic Repression &#40;EAR&#41; motif. We have shown that the chimeric repressors derived from various transcription factors dominantly suppress the expression of the respective target genes, and that the resultant transgenic plants exhibit loss&#45;of&#45;function phenotypes specific for the target genes. With the CRES&#45;T system, we are comprehensively analyzing function of Arabidopsis transcription factors. We describe the results of the functional analysis of the member of NAC, TCP, MADS and MYB family transcription factors on the regulation of plant growth and development, namely secondary wall synthesis, control of meristem development, flower formation and flavonoid biosynthesis. We also present that the CRES&#45;T system is a powerful tool for the manipulation of plant traits.</div>

</div>





<div class="abs">

<div style="font-weight:bold">NEW TRANSCRIPTION FACTORS AFFECTING DORMANCY IN ARABIDOPSIS SEEDS</div>

<div>

	<span style="text-decoration: underline" >
	JM Barrero</span>
	<sup>1</sup>, 

	<span  >
	AA Millar</span>
	<sup>1</sup>, 

	<span  >
	T Czechowski</span>
	<sup>2</sup>, 

	<span  >
	WR Scheible</span>
	<sup>2</sup>, 

	<span  >
	M Udvardi</span>
	<sup>2</sup>, 

	<span  >
	D Burgess</span>
	<sup>3</sup>, 

	<span  >
	RL Jones</span>
	<sup>3</sup>, 

	<span  >
	J Jacobsen</span>
	<sup>1</sup>, 

	<span  >
	F Gubler</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CSIRO Plant Industry, GPO Box 1600, Canberra ACT 2601, Australia, 

	<sup>2</sup>Max-Planck Institute of Molecular Plant Physiology, Am M&#252;hlenberg 1, 14476 Golm, Germany, 

	<sup>3</sup>Department of Plant and Microbial Biology, University of California, Berkeley, CA 94720-3102, USA

</div>

<div>Seed dormancy is a very important trait which maximizes the survival rate of the seed in nature, and whose control is essential in many crop species.  We have used transcription factor analysis to help identify new genes which are involved in the dormancy maintenance or release in <span style="font-style: italic">Arabidopsis thaliana</span>. RNA was isolated from imbibed dormant and non&#45;dormant C24 seeds and screened by qRT&#45;PCR for differentially expressed transcription factors.  We have identified 31 which are differentially expressed during the first few hours of imbibition.  After analyzing T&#45;DNA insertions lines for these genes, 4 of them displayed an increased or reduced dormancy level compared with wild type Columbia. RNAi lines have been generated to study the role of the selected genes in the high dormancy ecotype C24. Work is in progress to determine the spatial expression pattern of these genes in the seed tissues. One of the genes that we have identified encodes a RING finger protein which is involved in ABA biosynthesis during seed development, promoting the expression of <span style="font-style: italic">NCED</span> genes in the seed and generating a complete loss of dormancy when mutated. </div>

</div>





<div class="abs">

<div style="font-weight:bold">A chromatin link that couples cell division to Arabidopsis root epidermis cell fate through epigenetic changes of patterning genes</div>

<div>

	<span style="text-decoration: underline" >
	Elena Caro</span>
	<sup>1</sup>, 

	<span  >
	M Mar Castellano</span>
	<sup>1</sup>, 

	<span  >
	Crisanto Gutierrez</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centro de Biologia Molecular, CSIC-UAM, Cantoblanco, Madrid, Spain

</div>

<div>Cell proliferation and cell fate decisions are strictly coupled during plant embryogenesis and organogenesis. However, the molecular links that coordinate them are poorly understood. In the Arabidopsis root meristem epidermis, expression of the homeobox GLABRA2 &#40;GL2&#41; gene occurs in the atrichoblasts, determining that these cells will differentiate into non&#45;hair cells, whereas GL2 is repressed in trichoblasts, that form root hair cells. This decision requires signaling from the cortical cell layer, transcriptional regulation and a change in chromatin accessibility. We initially observed that CDT1, a protein that controls DNA replication at the G1&#47;S transition, activates GL2 expression, suggesting a link between cell division and cell fate decisions. <br/><br/>To define the molecular mechanism further, we identified a novel GL2&#45;Expression Modulator &#40;GEM&#41; protein as a CDT1&#45;interactor. GEM negatively affects both epidermal cell division and GL2 expression. Consistent with this, the level of GEM directly correlates with root hair density. GEM also affects the spatial pattern of GL2 expression. We found that GEM binds to the GL2 promoter by interacting with TTG1 &#40;TRANSPARENT TESTA GLABRA1&#41;, a WD40&#45;repeat protein involved in GL2&#45;mediated cell fate decision. Our data suggest that CDT1 and TTG1 compete in vivo for their binding to GEM. Moreover, GEM also binds to and negatively regulates the expression of CAPRICE &#40;CPC&#41;, involved in GL2 repression in trichoblasts.We hypothesized that GEM might regulate GL2 expression by controlling the histone epigenetic marks in its promoter. Indeed, we demonstrated that GEM mediates the acquisition and&#47;or maintenance of a correct histone H3 acetylation and H3K9 methylation at both GL2 and CPC promoters, specifically just upstream of the corresponding ORFs.<br/><br/>Our study provides direct evidence for GEM as a crucial component involved in root architecture, that couples cell division, fate and differentiation during Arabidopsis root development &#40;1&#41;. Further experiments defining this novel pathway that relies on GEM functions will be presented.<br/><br/>1. Caro et al. Nature &#40;2007, in press&#41;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Control of Floral Dorsoventral Asymmetry and Lateral Petal Morphogenesis in <span style="font-style: italic">Lotus japonicus</span></div>

<div>

	<span style="text-decoration: underline" >
	Yonghai Luo</span>
	<sup>1</sup>, 

	<span  >
	Da Luo</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Graduate School of the Chinese Academy of Sciences, Chinese Academy of Science., 

	<sup>2</sup>School of Life Science and Biotechnology, Shanghai Jiao Tong University

</div>

<div>Floral zygomorphy is a morphological novelty and considered to have evolved several times independently in flowering plants. It has been shown that in <span style="font-style: italic">Antirrhinum majus</span>, two TCP genes, <span style="font-style: italic">CYC&#47;DICH</span>, control both the floral DV asymmetry and petal organ asymmetry. To view a general Evo&#45;devo scenario of the origin of zygomorphic flowers, we investigate the relevant genetic mechanisms involved in <span style="font-style: italic">Lotus japonicus</span>, a model plant in <span style="font-style: italic">legumes</span>, with independent&#45;origin zygomorphic flowers. &#13;<br/>Here we isolated a key component of floral development, <span style="font-style: italic">KEW</span>, by map&#45;based cloning in <span style="font-style: italic">Lotus japonicus</span> and found it was sufficient and necessary for lateral petal development. Genetic data displays hierarchical epistatic relationships among dorsal, lateral and ventral identity and the ventral identity is a ground one. &#13;<br/>Furthermore, we show that <span style="font-style: italic">KEW</span> is likely involved in two processes, establishment and maintenance of floral DV asymmetry and morphogenesis of lateral petals. We characterize a putative global transcriptional inhibitory mechanism regulating the asymmetric expression of <span style="font-style: italic">KEW</span>, which might help to establish and maintain the floral DV asymmetry subsequently, and another local transcriptional regulation system mediates the morphogenesis of lateral petals. Our other relevant data confirm these conclusions and indicate a putative conserved, general regulation model of this kind of genes. &#13;<br/>In addition, we described the development of lateral petals based on the analysis of epidermal cell morphology. By doing that, we think that <span style="font-style: italic">KEW</span> is likely to communicate separable different genetic networks in different spatio&#45;temporal points during lateral petal development. We identified one kind of these responsive factors, namely <span style="font-style: italic">MOK</span> &#40;<span style="font-style: italic">modifier of KEW</span>&#41; and found that loss of its functions resulted in a semidominant phenotypes of <span style="font-style: italic">KEW1&#47;kew1</span>. Taken together, we integrated the genetic, molecular and morphological data to discuss a hypothetical process of lateral petal development. </div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis Dof transcription factor DAG1 is involved in PhyB&#45; and GA&#45; mediated seed germination</div>

<div>

	<span style="text-decoration: underline" >
	Stefano Gabriele</span>
	<sup>1</sup>, 

	<span  >
	Julie Martone</span>
	<sup>1</sup>, 

	<span  >
	Gianluca Ragone</span>
	<sup>2</sup>, 

	<span  >
	Luciana Corso</span>
	<sup>1</sup>, 

	<span  >
	Paolo Costantino</span>
	<sup>1</sup>, 

	<span  >
	Paola Vittorioso</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dept. Genetics and Molecular Biology, Univ. La Sapienza, Rome, (Italy), 

	<sup>2</sup>Istituto Dermatologico Italiano, Via dei Monti di Creta, Rome, (Italy)

</div>

<div>In Arabidopsis there is a clear relationship between light and gibberellin signaling with respect to seed germination. It has been demonstrated that phytochrome B acts on seed germination through transcriptional regulation of both GA biosynthetic genes &#40;<span style="font-style: italic">AtGA3ox1&#45;2</span>&#41; and GA deactivation genes &#40;<span style="font-style: italic">AtGA2ox2</span>&#41;. We have previously shown that inactivation of the DAG1 gene, encoding a zinc finger transcription factor belonging to the Dof family, considerably alters the germination potential of seeds. In fact, our data clearly indicated that the <span style="font-style: italic">dag1</span> seeds have a higher sensitivity to red light and an altered level&#47;sensitivity  to endogenous GAs. &#13;<br/>In order to unravel the role of DAG1 in PhyB&#45; and GA&#45; mediated seed germination, we produced double mutant of dag1 with respectively the <span style="font-style: italic">phyB</span> null mutant and with <span style="font-style: italic">rgl2</span>, the master negative regulator GA&#45;mediated seed germination.&#13;<br/>Morever we produced plants overexpressing PhyB in a <span style="font-style: italic">dag1</span> background compared to a Wt &#40;Ws&#41; background. Analyses of germination properties of both double mutants <span style="font-style: italic">dag1,phyB</span> and <span style="font-style: italic">dag1,rgl2</span> as well as of dag1 overexpressing PhyB transformants have been performed. &#13;<br/>Data arising from these analyses and a putative model of DAG1 function will be presented.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The DAG1 transcription factor from Arabidopsis thaliana: study of its transcriptional&#47;post&#45;transcriptional regulation</div>

<div>

	<span  >
	Julie Martone</span>
	<sup>1</sup>, 

	<span  >
	Patrizia Circelli</span>
	<sup>1</sup>, 

	<span  >
	Stefano Gabriele</span>
	<sup>1</sup>, 

	<span  >
	Annalisa Rizza</span>
	<sup>1</sup>, 

	<span  >
	Gianluca Ragone</span>
	<sup>2</sup>, 

	<span  >
	Paolo Costantino</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Paola Vittorioso</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dept. of Genetics and Molecular Biology, Univ. La Sapienza, Rome, (Italy), 

	<sup>2</sup>Istituto Dermatologico Italiano, Via dei Monti di Creta, Rome, (Italy)

</div>

<div>DAG1 &#40;Dof Affecting Germination 1&#41; is a DOF transcription factor of Arabidopsis thaliana involved in the control of seed dormancy and germination. In fact, seeds of the dag1 mutant show an increased germination potential compared to the Wt. Most interestingly, the pattern of segregation of the dag1 phenotype showed that the effect of the disruption of DAG1 is maternal and the gene is expressed only in the vasculature of the mother plant but not in the seed or at any stage of embryo development. This pointed to maternal control acting on the developmental fate &#45; dormancy vs. germination &#45; of the mature seed. A study on the DAG1:GFP cellular localization allowed us to reveal a potential action of this TF as a non&#45;cell autonomous protein, as it appears to be present in tissues where the DAG1 promoter is inactive and the mRNA is absent. Moreover, our data indicates that this phenomenon could be developmentally regulated. With respect to the maternal effect of DAG1 inactivation we decided to investigate whether the DAG1 protein could move to the seed. Genetic and molecular data will be presented on DAG1 localization and on its transcriptional and post&#45;transcriptional regulation.<br/>Moreover, to further characterize the role of DAG1 we performed an analysis of the dag1 transcriptome. We have identified several putative targets that are up&#47;down&#45;regulated in the dag1 mutant, and some of them seemed to be clearly related to DAG1.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The C&#45;terminus of INFLORESCENCE DEFICIENT IN ABSCISSION is sufficient to control floral abscission</div>

<div>

	<span  >
	Grethe-Elisabeth Stenvik</span>
	<sup>1</sup>, 

	<span  >
	Nora M. Tandstad</span>
	<sup>1</sup>, 

	<span  >
	Wenche Kristiansen</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Melinka A. Butenko</span>
	<sup>1</sup>, 

	<span  >
	Reidunn B. Aalen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biosciences, University of Oslo, Norway

</div>

<div>Abscission is a physiologically determined program of cell separation that enables plants to shed unwanted organs. The Arabidopsis floral organ abscission mutant <span style="font-style: italic">inflorescence deficient in abscission</span> &#40;<span style="font-style: italic">ida</span>&#41; &#40;Butenko et al. 2003, Plant Cell 15&#41; is completely deficient in abscission. The 77 amino acid IDA protein, which controls the final step of floral organ abscission, is a member of a novel group of putative ligands in plants. Both IDA and the IDA&#45;LIKE &#40;IDL&#41; proteins contain a predicted N&#45;terminal signal sequence and a conserved 12 aa motif &#40;PIP&#41; at the C&#45;terminus. Promoter:GUS expression studies of the <span style="font-style: italic">IDL</span> genes indicate that these genes may be involved in regulating various cell separation events such as root cap sloughing, stomata development and formation of hydathodes. A <span style="font-style: italic">35S:IDA</span> line constitutively overexpressing IDA exhibits earlier abscission of floral organs and ectopic abscission at the bases of the pedicel, branches of the inflorescence, and cauline leaves &#40;Stenvik et al. 2006, Plant Cell 18&#41;. Overexpression of <span style="font-style: italic">IDL</span> genes results in similar phenotypes, suggesting functional redundancy of the PIP domain. We have found the C&#45;terminal portion of the IDA protein, which contains the PIP domain, to be sufficient for IDA function. In addition the N&#45;terminal signal sequence of IDA must be present to induce the overexpression phenotypes seen in <span style="font-style: italic">35S:IDA</span> plants. Exogenous application of a synthetic peptide covering the C&#45;terminal portion of IDA will be used to see if early floral abscission can be induced in wild&#45;type plants. In addition, constructs where parts of the IDA and IDL proteins were swapped have been used to determine the level of complementation of the <span style="font-style: italic">ida</span> phenotype. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulation of BDL during early embryo development</div>

<div>

	<span style="text-decoration: underline" >
	Ive De Smet</span>
	<sup>1</sup>, 

	<span  >
	Steffen Lau</span>
	<sup>1</sup>, 

	<span  >
	Jasmin S. Ehrismann</span>
	<sup>1</sup>, 

	<span  >
	Marika  Kientz</span>
	<sup>1</sup>, 

	<span  >
	Gerd J&#252;rgens</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ZMBP Entwicklungsgenetik - Universitat Tubingen, Tuebingen, Germany

</div>

<div>The Arabidopsis zygote divides asymmetrically to give an apical and a basal daughter cell. Subsequently, auxin is transported from the basal into the apical daughter cell of the zygote &#91;1&#93;. The auxin signal is converted into a specific response by auxin&#45;promoted degradation of the Aux&#47;IAA protein BDL&#47;IAA12 that prevents auxin response factor &#40;ARF&#41; transcription factor MP&#47;ARF5 from regulating auxin&#45;responsive target genes &#91;2, 3, 4&#93;. The MP&#47;BDL&#45;dependent response in the apical cell lineage results in PIN1 expression and auxin efflux into the basally adjacent future hypophysis, thus initiating embryonic root formation &#91;4&#93;.&#13;<br/>To learn more about mechanisms underlying the initial difference between apical and basal lineages, we are focusing on the regulation of the BDL gene, which is expressed in the apical daughter cell of the zygote and its progeny. A yeast one&#45;hybrid screen has yielded candidate transcriptional regulators &#40;Regulator Of BDL or ROB&#41;, which are being characterized functionally. First, we validate the interaction with the BDL promoter using various approaches, such as yeast one&#45;hybrid assay, transactivation assay and DNA&#45;binding assay. Second, we describe the expression pattern in the developing embryo using mRNA in situ hybridization, PROB::GFP and PROB::ROB:GFP fusions. Third, we analyze the embryo phenotype upon altered expression levels of the ROBs in the wild&#45;type and bdl mutant background. Fourth, we investigate the BDL expression in knock&#45;out mutants of the ROBs. Taken together, these results provide a starting point to dissect the transcriptional control of BDL.&#13;<br/>&#13;<br/>1. Friml et al., 2003, Nature 426: 147&#45;153&#13;<br/>2. Weijers and J&#252;rgens, 2004, Curr Opin Plant Biol 7: 687&#45;693&#13;<br/>3. Dharmasiri et al., 2005, Dev Cell 9: 109&#45;119&#13;<br/>4. Weijers et al., 2006, Dev Cell 10: 265&#45;270&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">BET bromodomain protein GTE4 affects embryonic and post&#45;embryonic root development in &#34;Arabidopsis&#34;</div>

<div>

	<span style="text-decoration: underline" >
	Maria Maddalena Altamura</span>
	<sup>1</sup>, 

	<span  >
	Chiara C. Airoldi</span>
	<sup>2</sup>, 

	<span  >
	Giuseppina Falasca</span>
	<sup>1</sup>, 

	<span  >
	Federica Della Rovere</span>
	<sup>1</sup>, 

	<span  >
	Sandra Citterio</span>
	<sup>3</sup>, 

	<span  >
	Lucia Colombo</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Dipartimento di Biologia Vegetale, Sapienza Universit&#224; di Roma, Rome, Italy, 

	<sup>2</sup>Dipartimento di Biologia, Universit&#224; degli Studi di Milano, Milan, Italy, 

	<sup>3</sup>Dipartimento di Scienze dell'Ambiente e del Territorio, Universit&#224;  di Milano Bicocca, Milan, Italy

</div>

<div>The bromodomain was first discovered in the &#34;Drosophila&#34; Brahma protein &#40;Tamkun et al., 1992&#41; and is present in a broad range of chromatin&#45;modifying proteins &#40;Yang, 2004 and references therein&#41;. BET proteins form a separate group of bromodomain proteins which all share a unique additional domain named extra terminal &#40;ET&#41; domain &#40;Pandey et al., 2002&#41;. Plant BET proteins are distinguishable from the yeast and animal ones for the presence of only one bromodomain instead of two &#40;Pandey et al., 2002&#41;. In the &#34;Arabidopsis&#34; genome 12 BET encoding genes have been identified. Until now only two of these were characterized, &#34;IMB1&#34; and &#34;GTE6&#34;. IMB1 is a transcriptional activator and promotes seed germination regulating abscisic acid and phytochrome A &#40;Duque and Chua, 2003&#41;. GTE6 positively regulates the &#34;myb&#34;&#45;domain gene &#34;AS1&#34;, which is involved in leaf axis specification &#40;Chua et al., 2005&#41;. We are now investigating &#34;GTE4&#34; &#40;At1g06230&#41;  using an integrated analysis of molecular biology and histology on the homozygous knock&#45;out mutant. RT&#45;PCR experiments showed that &#34;GTE4&#34; is expressed in all the organs of the plant. Results about embryonic and post&#45;embryonic root development are presented. Anomalies in the development of the root pole cells are present in the mature embryo of the knock&#45;out mutant. A significant reduction in primary root length in comparison with the wild type was also observed during the first week after seed  germination. The histological analysis suggests that the abnormal root growth is due to an anomalous increase in size of the meristematic cells in the root apex. In conclusion, GTE4 seems to be involved in the regulation of root dome formation and development from the embryo to the seedling.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic control of male gametogenesis in Arabidopsis</div>

<div>

	<span  >
	Li-Qun Chen</span>
	<sup>1</sup>, 

	<span  >
	Xue-Qin Zhang</span>
	<sup>1</sup>, 

	<span  >
	Ke-Zhen Yang</span>
	<sup>1</sup>, 

	<span  >
	Xiao-Lei Liu</span>
	<sup>1</sup>, 

	<span  >
	Chuan Xia</span>
	<sup>1</sup>, 

	<span  >
	Yi Deng</span>
	<sup>1</sup>, 

	<span  >
	Wen-Qing Li</span>
	<sup>1</sup>, 

	<span  >
	Xiao-Yun Tan</span>
	<sup>1</sup>, 

	<span  >
	Wei Wang</span>
	<sup>1</sup>, 

	<span  >
	Dong-Jie Jia</span>
	<sup>1</sup>, 

	<span  >
	Xi Cao</span>
	<sup>1</sup>, 

	<span  >
	Xuan Ma</span>
	<sup>1</sup>, 

	<span  >
	Lu-Yuan He</span>
	<sup>1</sup>, 

	<span  >
	Wen Dui</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	De Ye</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>State Key Laboratory of Plant Physiology and Biochemistry, College of Biological Sciences, China Agricultural University,Beijing, China

</div>

<div>We are focusing on understanding the genetic mechanisms that control the cell differentiation and interaction in the plant male gametogenesis. As the first step, we largely isolate and characterize the mutants and corresponding genes that are involved in the male gametogenesis and pollen tube growth in Arabidopsis by phenotypic screens of the enhancer&#45;and gene&#45;trap dissociation&#40;Ds&#41; insertion lines in Arabidopsis thaliana ecotype Landsberg erecta &#40;Sundaresan et al., 1995, G &#38; D 13: 2108&#45;2117&#41;. So far, 107 male gametophytic mutants have been isolated and are being characterized. 89 out of 107 mutations have been successfully mapped on 5 different chromosomes using TAIL&#45;PCR. All mutations mainly or only affect the male gametophytic function. 15 genes that play key roles in controlling the pollen formation and pollen tube growth have been cloned and are being characterized. Molecular and functional characterization of the genes show that they may control the cell differentiation and interaction in male gametogenesis and pollen tube growth via taking part in signal transduction, transcription regulation and metabolism. &#40;The correspondence is addressed to De Ye, E&#45;mail: yede&#64;cau.edu.cn&#41;&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">&#34;DORNROESCHEN&#34; &#40;&#34;DRN&#34;&#41; and &#34;DRN&#45;LIKE&#34; control the change from radial to bilateral symmetry and act upstream of auxin in the &#34;Arabidopsis&#34; embryo.</div>

<div>

	<span style="text-decoration: underline" >
	John Chandler</span>
	<sup>1</sup>, 

	<span  >
	Melanie Cole</span>
	<sup>1</sup>, 

	<span  >
	Wolfgang Werr</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Cologne, Cologne, Germany

</div>

<div>Embryonic patterning is a fundamental process of plant development, leading to the formation of the SAM and RAM necessary for organogenesis and subsequent growth. A key aspect of this process is the transition from radial to bilateral symmetry. The AP2 domain&#45;encoding transcription factor &#34;DRN&#34; and its paralogue &#34;DRNL&#34; are expressed from very early globular stages and act redundantly to control cotyledon ontogeny and stereotypic hypophyseal cell divisions via interaction with proteins from the class III HD&#45;ZIP family such as PHAVOLUTA, and from the bHLH transcription factor family. &#13;<br/>The transition from radial to bilateral symmetry involves auxin signalling and response and is pre&#45;patterned by &#34;CUC&#34; gene expression. Both &#34;DRN&#34; and &#34;DRNL&#34; are implicated in this transition; genetic analyses show that &#34;DRN&#34; and &#34;DRNL&#34; function in different &#34;CUC&#34; pathways, a finding which addresses the redundancy within both gene families. A disruption in radial symmetry is reflected in single &#34;drn&#34; and &#34;drnl&#34; mutants which show mono&#45; or polycotyledony and initiate leaves with an altered phyllotaxis and &#34;drn drnl&#34; double mutants completely lack cotyledons, have radialised pin&#45;like embryos which initiate leaves directly and are disturbed in all subsequent developmental stages. &#34;drn drnl&#34; double mutants also show an asymmetry of &#34;STM&#34; and &#34;CUC&#34; gene expression, demonstrating a loss of positional information and placing &#34;DRN&#34; and &#34;DRNL&#34; gene functions in pathways leading to the establishment of bilateral symmetry. Polar localisation of PIN1 is important in establishing the RAM and cotyledon initiation and creating bilateral symmetry. Consistent with hypophyseal and cotyledon defects in the drn mutant, DR5::GFP and PIN1 expression are also altered, placing &#34;DRN&#34; function upstream of auxin transport and&#47;or response. &#34;DRN&#34; expression is also positively regulated by auxin: mutational analysis of several auxin response elements in upstream and downstream &#34;DRN&#34; regulatory regions reveals a differential contribution of elements to the tissue&#45;specific expression of &#34;DRN &#34;during embryo development.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The role of auxin in stamen development</div>

<div>

	<span  >
	Valentina Cecchetti</span>
	<sup>1</sup>, 

	<span  >
	Andrea Raganelli</span>
	<sup>2</sup>, 

	<span  >
	Giuseppina Falasca</span>
	<sup>3</sup>, 

	<span  >
	Maria Maddalena Altamura</span>
	<sup>3</sup>, 

	<span  >
	Paolo Costantino</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Maura Cardarelli</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Dipartimento di Genetica e Biologia Molecolare, Sapienza Universita' di Roma, Rome, Italy, 

	<sup>2</sup>IBPM-CNR, Sapienza Universita' di Roma, Rome, Italy, 

	<sup>3</sup>Dipartimento di Biologia Vegetale, Sapienza Universita' di Roma, Rome, Italy

</div>

<div>In angiosperms, the late phase of stamen development consists of three main processes &#45;filament elongation, pollen maturation and anther dehiscence &#45; and is coordinated with the development of the pistil. We previously demonstrated a role for auxin on stamen and pistil development, by means of the localised expression of <span style="font-style: italic">rolB</span>, an <span style="font-style: italic">Agrobacterium</span> oncogene that enhances the sensitivity to this hormone. Tobacco plants expressing <span style="font-style: italic">rolB</span> driven by the promoter of the gene <span style="font-style: italic">DMC1</span> display shorter filaments as compared to controls, a severe delay in anther dehiscence and alterations in male and female meiosis; these developmental alterations could be phenocopied by exogenous auxin &#40;1&#41;. To verify the role of auxin in stamen development in Arabidopsis flowers we analysed auxin accumulation and synthesis in the meiotic and postmeiotic stages of stamen development. The phenotype of <span style="font-style: italic">tir&#47;afb</span> mutants defective in auxin perception was analysed with respect to anther dehiscence, filament elongation and pollen maturation. Results demonstrate the role of auxin in controlling the timing of different processes in stamen development.<br/>In addition, we isolated a tobacco gene, <span style="font-style: italic">NtROX1</span>, acting downstream of <span style="font-style: italic">rolB</span>, overexpressed in <span style="font-style: italic">pDMC1:rolB</span> anthers. We demonstrated a role for <span style="font-style: italic">ROX1</span> in filament elongation, by controlling the balance between procambial cell proliferation and xylem differentiation during stamen development in tobacco flowers &#40;2&#41;. The sequence of <span style="font-style: italic">ROX1</span> shares conserved elements with three genes of Arabidopsis. We are currently analysing the expression pattern of these genes during stamen development in Arabidopsis and characterising monogenic mutants.<br/><br/>1.Cecchetti V., Marsilio S., Pomponi M., Altamura M.M., Pezzotti M., D&#8217;Angelo S., Tornielli G.B., Costantino P. and Cardarelli M. &#40;2004&#41; PLANT JOURNAL, 38, 512&#45;525<br/>2.Cecchetti V., Altamura M.M., Serino G., Pomponi M., Falasca G., Costantino P. and Cardarelli M. &#40;2007&#41; PLANT JOURNAL  49, 27&#45;37. <br/><br/><br/><br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The <span style="font-style: italic">Arabidopsis</span> <span style="font-style: italic">BEATNIK</span> gene acts as a repressor of auxin gradient&#45;directed polar root hair positioning</div>

<div>

	<span  >
	Yoshihisa Ikeda</span>
	<sup>1</sup>, 

	<span  >
	Karin Ljung</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Markus Grebe</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Ume&#229; Plant Science Centre,  Dept. of Forest Genetics &amp;  Plant Physiology, SLU,  SE-90183 Ume&#229;, Sweden

</div>

<div>The polarity of single cells is commonly coordinated within the plane of a single tissue layer. In animals, this phenomenon is referred to as planar polarity and has been highly successfully investigated e.g. during <span style="font-style: italic">Drosophila</span> wing hair positioning. We study plant&#45;specific planar polarity of root hair positioning in the <span style="font-style: italic">Arabidopsis</span> root epidermis, where hairs are strictly formed close to basal ends of hair&#45;forming cells &#91;1&#45;3&#93;. Hair position along the trichoblast is directed toward the maximum concentration of an auxin gradient in the root tip &#91;2,4,5&#93;. We recently demonstrated that shaping this gradient requires combined function of the auxin influx carrier <span style="font-style: italic">AUX1</span>, <span style="font-style: italic">ETHYLENE INSENSITIVE2</span> &#40;<span style="font-style: italic">EIN2</span>&#41;, and <span style="font-style: italic">GNOM</span> genes which converges prior to polar Rho&#45;of&#45;plant &#40;ROP&#41; recruitment to the hair initiation site. Moreover, local auxin gradients can coordinate planar root hair positioning &#91;5&#93;. In screens for mutants with altered hair placement, we have identified the recessive <span style="font-style: italic">beatnik</span> &#40;<span style="font-style: italic">btk</span>&#41; mutation. <span style="font-style: italic">btk</span> causes root&#45;specific phenotypes, including long hair and hyperpolar hair initiation at basal&#45;most ends of trichoblasts. We show that in <span style="font-style: italic">btk</span>, F&#45;actin prematurely accumulates in the hair bulge and ROPs localize to the hair initiation site at basal&#45;most ends of cells with enhanced polarity. The <span style="font-style: italic">btk</span> phenotype can be suppressed by <span style="font-style: italic">act2</span>, weak <span style="font-style: italic">gnom</span>, by <span style="font-style: italic">aux1</span> and <span style="font-style: italic">ein2</span> mutations, indicating that <span style="font-style: italic">BTK</span> acts as a repressor of coordinated polarity upstream of these genes. The auxin concentration gradient and auxin biosynthesis rates are enhanced in <span style="font-style: italic">btk</span> root tips, suggesting that <span style="font-style: italic">BTK</span> acts on planar polarity as a repressor of auxin gradient function. We report progress on  identification of the <span style="font-style: italic">btk</span> mutation and analysis of <span style="font-style: italic">btk</span> loss&#45;of&#45;function mosaics to study its local effect on planar polarity.&#13;<br/>&#13;<br/>1. Masucci &#38; Schiefelbein 1994. Plant Physiol. 106, 1335&#45;1346.&#13;<br/>2. Grebe et al. 2002. Curr. Biol. 12, 329&#45;334.&#13;<br/>3. Grebe 2004. Bioessays 26, 719&#45;729. &#13;<br/>4. Sabatini et al. 1999. Cell 99, 463&#45;472&#13;<br/>5. Fischer et al. 2006. Curr. Biol. 16, 2143&#45;2149.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A common factor involved in root and shoot meristem development</div>

<div>

	<span style="text-decoration: underline" >
	Anja van Dijken</span>
	<sup>1</sup>, 

	<span  >
	Ben Scheres</span>
	<sup>1</sup>, 

	<span  >
	Renze Heidstra</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Utrecht University, Dept. Molecular Genetics, Faculty of Sciences, Utrecht, The Netherlands

</div>

<div>Plant post&#45;embryonic development takes place in the meristems, where sets of stem cells self&#45;renew and produce daughter cells that differentiate giving rise to different organ structures. Stem cell maintenance is dependent on organizing cells within the stem cell niches in the shoot and root meristems. To identify genes involved in the root organizer &#40;or quiescent center, QC&#41; and stem cell specification and maintenance we screened an EMS mutant population generated in a transgenic Arabidopsis line expressing two independent fluorescent QC markers for altered expression patterns.&#13;<br/>Many isolated mutants displayed reduced expression of both QC markers in the early stages of seedling development. In several of these mutants columella stem cell identity was lost; suggesting disturbed signaling between QC and stem cells. Interestingly, the recessive mutant QC19&#45;3, displays both a root and shoot meristem phenotype. &#13;<br/>As early as globular embryo stages altered divisions in the root meristem occasionally occur. Root growth is also retarded; mutant root length is &#126;50&#37; reduced and displays shorter meristem size. QC marker expression is reduced and frequently re&#45;patterning of the root stem cell niche within the vasculature is observed indicating disturbed QC&#47;stem cell maintenance. In later stages occasionally the root splits into two root tips that continue to grow slowly. &#13;<br/>Mature embryos exhibit a reduced shoot apical meristem compared to wild type. After germination the QC19&#45;3 mutant shoot meristem appears arrested: seedlings up to 11 days old do not show any leaf primordia before retarded shoots are initiated, resembling the wus mutant phenotype. Mature mutant plants do form small, serrated, pale leaves and develop small fertile flowers. &#13;<br/>Introduction of markers and genetic interaction studies are in progress to determine the genes involved. In addition, a map based cloning approach is running to identify the gene responsible. Cloning the gene will shed light on its function in organizer&#47;stem cell specification and maintenance and whether it is a common factor involved in root and shoot meristem development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Isolation and characterization of novel transcriptional repressors of seed storage protein genes in &#34;Arabidopsis&#34;</div>

<div>

	<span style="text-decoration: underline" >
	Ming-Jun Gao</span>
	<sup>1</sup>, 

	<span  >
	Derek J Lydiate</span>
	<sup>1</sup>, 

	<span  >
	Dwayne D Hegedus</span>
	<sup>1</sup>, 

	<span  >
	Kevin Rozwadowski</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Molecular Genetics Section, Agriculture and Agri-Food Canada, Saskatoon Research Centre, Saskatoon, SK, Canada S7N 0X2

</div>

<div>Progression through embryo development to seed maturity and the transition to germination and vegetative growth is subject to tight regulation by the cooperation of many factors. These include transcriptional activators such as ABI3, FUS3, LEC1 and LEC2, transcriptional repressors such as PKL, HSI2 and HSL1, the competing effects of phytohormones including ABA and GA, and metabolite signals such as sugars. Given that the transcriptional activators regulating seed development also play central roles in regulating physiological response and development in the growing plant, effective means of repressing expression of seed&#45;specific genes exist to inhibit their ectopic expression and prevent negative pleiotropic effects. To date only a limited number of such repressors have been identified. &#34;PKL&#34; encodes a putative CHD3 chromatin remodelling factor and acts in conjunction with GA to repress embryonic identity during early stage of germination. HSI2 and HSL1 are B3&#45; and EAR&#45;domain containing transcriptional repressors of sugar&#45;inducible expression of seed maturation genes during seedling growth. Here, we report the isolation and characterization of two novel transcriptional repressors of seed storage protein &#40;SSP&#41; genes isolated using yeast one&#45;hybrid screening with putative negative cis&#45;acting elements present in the promoters of SSP gene. Although disruption of either of the two repressor genes did not significantly affect seedling growth, their inactivation dramatically resulted in the expression of SSP genes in seedlings and increased accumulation of SSPs in developing seeds. Their mode of action in regulating the transition from vegetative to reproductive growth and seed development will be discussed. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Overlapping and antagonistic roles of BELL TALE homeodomain proteins in shoot apical meristem phase identity</div>

<div>

	<span  >
	Bas Rutjens</span>
	<sup>1</sup>, 

	<span  >
	Evelien van Eck-Stouten</span>
	<sup>1</sup>, 

	<span  >
	Marco Brand</span>
	<sup>1</sup>, 

	<span  >
	Sjef Smeekens</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Marcel Proveniers</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Utrecht University, Utrecht, The Netherlands

</div>

<div>Floral induction is controlled by a plethora of genes acting in different pathways that either repress or promote floral transition at the shoot apical meristem &#40;SAM&#41;. During vegetative development high levels of floral repressors maintain the Arabidopsis SAM incompetent to respond to promoting factors. Among these repressors, <span style="font-style: italic">FLOWERING LOCUS C</span> &#40;<span style="font-style: italic">FLC</span>&#41; is the most prominent. The processes underlying down&#45;regulation of <span style="font-style: italic">FLC</span> in response to environmental and developmental signals have been elucidated in considerable detail. However, the basal induction of FLC and its up&#45;regulation by FRIGIDA &#40;FRI&#41; are less understood. Here we demonstrate that the BELL&#45;class proteins ARABIDOPSIS THALIANA HOMEOBOX1 &#40;ATH1&#41; and PENNYWISE &#40;PNY&#41; act redundantly as positive regulators of <span style="font-style: italic">FLC</span> expression. Plants that constitutively express ATH1 display a vernalization&#45;sensitive late flowering habit. Analysis of lines that differ in FRI and&#47;or FLC allele strength show that this late flowering is caused by up&#45;regulation of <span style="font-style: italic">FLC</span> as a result of synergism between ATH1 and FRI. Consistently, ath1 mutants flower early in short days &#40;SD&#41; and display attenuated <span style="font-style: italic">FLC</span> levels. Moreover, <span style="font-style: italic">ath1</span> mutations partially suppress FLC&#45;mediated late flowering of both a FRI&#45;expressing line and that of <span style="font-style: italic">fca&#45;1</span> and <span style="font-style: italic">fve&#45;1</span> autonomous pathway mutants. Like <span style="font-style: italic">ath1</span>, pny mutants are characterized by strongly reduced basal <span style="font-style: italic">FLC</span>' levels and an early flowering phenotype in SD. Also similar to <span style="font-style: italic">ath1</span>, <span style="font-style: italic">pny</span> partially suppresses late flowering in the presence of dominant FRI alleles. In contrast to the relatively weak effects of either single mutant, absence of both ATH1 and PNY almost fully impairs FRI&#45;mediated late flowering. Intriguingly, no role for the BELL protein POUND&#45;FOOLISH &#40;PNF&#41;, an established redundant partner of PNY required for the competence of the SAM to respond properly to floral inductive signals, was found in FLC&#45;mediated flowering time control. Taken together, this suggests that throughout plant development different BELL combinations dictate developmental phase identity by controlling key components of phase identity.</div>

</div>





<div class="abs">

<div style="font-weight:bold">BLADE&#45;ON&#45;PETIOLE is Required for Abscission Zone Formation</div>

<div>

	<span style="text-decoration: underline" >
	Sarah M. McKim</span>
	<sup>1</sup>, 

	<span  >
	Grethe-Elisabeth Stenvik</span>
	<sup>2</sup>, 

	<span  >
	Wenche Kristiansen</span>
	<sup>2</sup>, 

	<span  >
	Melinka A. Butenko</span>
	<sup>2</sup>, 

	<span  >
	Sung Ki Cho</span>
	<sup>3</sup>, 

	<span  >
	Shelley R. Hepworth</span>
	<sup>1</sup>, 

	<span  >
	Reidunn B. Aalen</span>
	<sup>2</sup>, 

	<span  >
	George W. Haughn</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Botany, University of British Columbia, Canada, 

	<sup>2</sup>Department of Molecular Biosciences, University of Oslo, Norway, 

	<sup>3</sup>Division of Biological Sciences, University of Missouri, USA

</div>

<div>The Arabidopsis &#34;BLADE&#45;ON&#45;PETIOLE&#34; &#40;&#34;BOP&#34;&#41; genes encode redundant transcription factors that are involved in leaf and flower development.  Loss&#45;of&#45;function &#34;bop1bop2&#34; mutants display a range of defects characterized in detail, including ectopic leaf blade growth and extra floral organs. Also, &#34;bop1bop2&#34; mutants show a loss of floral organ abscission although to date this has received little attention.  Abscission occurs along specialised cell files, called Abscission Zones &#40;AZs&#41;, which develop at the junctions between the leaving organs and main plant body.  We have thoroughly characterized the &#34;bop1bop2&#34; abscission phenotype to assess the role of &#34;BOP1&#47;BOP2&#34; in the known abscission developmental framework.  Microscopic and quantitative analyses of &#34;bop1bop2&#34; mutants show no differentiation of either floral AZs or vestigial cauline leaf AZs suggesting BOP proteins are essential to establish AZ cell identity.  In support of this hypothesis, BOP activity was required for precocious floral organ abscission and ectopic abscission at the pedicel and cauline leaves promoted by ectopic expression of the &#34;INFLORESCENCE DEFICIENT IN ABSCISSION&#34; &#40;&#34;IDA&#34;&#41;. The &#34;IDA&#34; gene acts late in the abscission program to trigger the final separation step if there is a differentiated AZ present &#40;1, 2&#41;.  Interestingly, expression of several abscission&#45;related markers, including &#34;IDA&#34;, were relatively unperturbed in &#34;bop1bop2&#34; mutants, indicating that these genes respond to positional cues independent of BOP activity.  We also discovered that BOP1&#47;BOP2 activity is essential for formation of nectary glands which normally develop at the receptacle proximal to AZs.  This suggests that &#34;BOP1&#47;BOP2&#34; genes are involved in determining multiple cell identities at the receptacle.  <br/>&#40;1&#41; Butenko et al. &#40;2003&#41; Plant Cell 15: 2296&#45;2307. &#40;2&#41; Stenvik et al. &#40;2006&#41; Plant Cell 18: 1467&#45;1476.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The BELL homeodomain proteins ATH1 and PNY are redundantly required for vegetative shoot apical meristem function in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Dongping Bao</span>
	<sup>1</sup>, 

	<span  >
	Bas Rutjens</span>
	<sup>1</sup>, 

	<span  >
	Evelien van Eck-Stouten</span>
	<sup>1</sup>, 

	<span  >
	Sjef Smeekens</span>
	<sup>1</sup>, 

	<span  >
	Marcel Proveniers</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Utrecht University, Utrecht, The Netherlands

</div>

<div>In animals, TALE homeodomain &#40;HD&#41; proteins play fundamental roles in development, in particular in the specification of body plan and pattern formation. In plants, pattern formation is an ongoing process that starts during embryogenesis and lasts the entire life span. Most of the plant adult body is formed post&#45;embryonically by continuous organogenic potential of the root and shoot apical meristems. Proper shoot apical meristem &#40;SAM&#41; function requires maintenance of a delicate balance between the depletion of stem cell daughters into developing primordia and proliferation of the central stem cell population. Mutations in the KNOX&#45;class TALE HD protein SHOOTMERISTEMLESS &#40;STM&#41; cause defective initiation and maintenance of the SAM and results in early developmental arrests. STM is able to physically interact with members of a distantly related second class of TALE HD transcription factors, the BELL class. In animals the formation of such heterodimers is often indispensable for TALE HD functionality. One of the STM&#45;associating BELL proteins, PENNYWISE &#40;PNY&#41; enhances meristem defects of weak to intermediate stm loss&#45;of&#45;function mutants. PNY itself, however, is not essential for meristem function, possibly due to redundancy. Here we show that the BELL&#45;class protein ARABIDOPSIS THALIANA HOMEOBOX1 &#40;ATH1&#41; fulfills a functional redundant role with PNY and STM during initiation and maintenance of the vegetative SAM. Like <span style="font-style: italic">PNY</span> and <span style="font-style: italic">STM</span>, <span style="font-style: italic">ATH1</span> is highly expressed in the vegetative SAM, but loss of ATH1 function seems not to be detrimental to meristem function. However, <span style="font-style: italic">ath1</span> mutations enhance weak and intermediate <span style="font-style: italic">stm</span> alleles. Moreover, combined loss of ATH1 and PNY results in a <span style="font-style: italic">stm</span> mutant phenocopy during the embryonic and vegetative phases, whereas generative development appears unaffected. Finally, we present data implying that both KNOX and BELL protein function are regulated through their subcellular distribution by a mechanism highly conserved in animals and plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Developmental Steps in Acquiring Competence for Shoot Development in Arabidopsis Tissue Culture</div>

<div>

	<span  >
	Ping Che</span>
	<sup>1</sup>, 

	<span  >
	Sonia Lall</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Stephen Howell</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Iowa State University, Ames IA, USA

</div>

<div>Arabidopsis shoots regenerate from root explants in tissue culture through a two&#45;step process requiring preincubation on an auxin&#45;rich callus induction medium &#40;CIM&#41; followed by incubation on a cytokinin&#45;rich shoot induction medium &#40;SIM&#41;.  During CIM preincubation, root explants acquire competence to respond to shoot induction signals.  During CIM preincubation, pericycle cells in root explants undergo cell divisions and dedifferentiate, losing the expression of a pericycle cell&#45;specific marker.  These cells acquire competence to form green callus after only one day CIM preincubation and to form shoots after 2&#45;3 days CIM preincubation.  Reversible DNA synthesis inhibitors and auxin response mutants &#40;<span style="font-style: italic">axr1</span> and <span style="font-style: italic">axr4</span>&#41; interfered with the acquisition of competence to form shoots.  Genes requiring CIM preincubation for upregulation on SIM were identified by microarray analysis and included <span style="font-style: italic">RESPONSE REGULATOR 15</span> &#40;<span style="font-style: italic">ARR15</span>&#41;, <span style="font-style: italic">POLYGALACTURONASE INHIBITING PROTEIN 2</span> &#40;<span style="font-style: italic">PGIP2</span>&#41; and <span style="font-style: italic">WUSCHEL</span> &#40;<span style="font-style: italic">WUS</span>&#41;.  These genes served as developmental markers for the acquisition of competence because the CIM preincubation requirements for <span style="font-style: italic">ARR15</span> and <span style="font-style: italic">PGIP2</span> upregulation correlated well with the acquisition of competence to form green callus, and the CIM preincubation requirements for <span style="font-style: italic">WUS</span> upregulation matched those for shoot formation. Unlike <span style="font-style: italic">ARR15</span>, another cytokinin inducible, A&#45;type ARR gene, <span style="font-style: italic">ARR5</span>, was upregulated on SIM, but the induction did not require CIM preincubation.  These findings indicate that competencies for various events associated with shoot regeneration are acquired progressively during CIM preincubation, and that a set of genes, normally upregulated on SIM, are repressed by a process that can be relieved by CIM preincubation.<br/><br/>This work was supported by the National Science Foundation &#40;IOB&#45;0236060&#41;.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">SEUSS and AINTEGUMENTA Regulate Carpel Margin and Ovule Development</div>

<div>

	<span style="text-decoration: underline" >
	Fang Bao</span>
	<sup>1</sup>, 

	<span  >
	Sridevi Azhakanandam</span>
	<sup>1</sup>, 

	<span  >
	Staci Nole-Wilson</span>
	<sup>1</sup>, 

	<span  >
	Huda Bhatti</span>
	<sup>1</sup>, 

	<span  >
	Robert Franks</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>North Carolina State University, Raleigh, (NC), USA

</div>

<div>In Arabidopsis, ovules arise from meristematic placental regions that reside on the adaxial margins of the carpel. Our work indicates that SEUSS and AINTEGUMENTA are required for the specification or early development of the placentae and for initiation of ovule primordia.  We show that the patterning genes PHABULOSA and CRABS CLAW are mis&#45;expressed in seu ant double mutant carpels and that seu ant mutants completely fail to initiate ovule primordia. As SEUSS and AINTEGUMENTA encode transcriptional regulators, we are working to determine if CRABS CLAW or PHABULOSA are direct targets of SEUSS regulation. Additionally, we are using genetic means to determine if mis&#45;regulation of these patterning genes in the seu ant double mutant is responsible for the failure to initiate ovule primordia. <br/><br/>We have also initiated genetic screens to identify novel regulators of ovule and carpel margin development. We have identified several suppressor of aintegumenta &#40;saint&#41; mutants that can partially rescue the integument growth defect of a hypomorphic loss&#45;of&#45;function ant&#45;3 allele. The ant&#45;3 saint double mutants display rescued integument and female gametophyte phenotypes and have a 4 to 8 fold increase in fertile seeds per pod relative to ant&#45;3.  Progress on map&#45;based cloning of one of the saint mutants &#40;saint49&#41; will be presented. In a second screen, we identified seuss modifier &#40;sum&#41; mutants that enhance the carpel and ovule defects of seu&#45;1 single mutants. One such mutant, sum63&#45;1, displays wild type ovule development, but acts as a synergistic enhancer of the seu&#45;1 ovule phenotype. The sum63 seu&#45;1 ovule phenotype is characterized by a strong loss of the outer integument, while inner integument and female gametophyte development are not disrupted.  The sum63 mutation maps to a portion of the genome with no previously identified ovule mutants and thus may represent a novel regulator of ovule development. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">TCP transcription factors control the morphology of shoot lateral organs via negative regulation of the expression of boundary&#45;specific genes in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Tomotsugu Koyama</span>
	<sup>1,2</sup>, 

	<span  >
	Masahiko Furutani</span>
	<sup>3</sup>, 

	<span  >
	 Masao Tasaka</span>
	<sup>3</sup>, 

	<span  >
	 Masaru  Ohme-Takagi</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup> National Institute of Advanced Industrial Science and Technology (AIST), 

	<sup>2</sup>Core Research for Evolutional Science and Technology (CREST), 

	<sup>3</sup>Nara Institute of Science and Technology

</div>

<div>Plants form shoot meristems in the so&#45;called boundary region and these meristems are necessary for normal morphogenesis of aerial parts of plants. However, the molecular mechanisms that regulate the formation of shoot meristems are not fully understood. We report here that expression of a chimeric repressor from TCP3 &#40;TCP3SRDX&#41;, a member of TEOSINTE BRANCHED1, CYCLOIDEA and PCF &#40;TCP&#41; transcription factors in Arabidopsis, resulted in various developmental defects that include the formation of ectopic shoots on cotyledons, the wavy surfaces and margins of shoot lateral organs, the irregular pattern of vasculature and lack of the specific differentiation of epidermal cells. Expression of TCP3SRDX induced ectopic expression of boundary&#45;specific genes, namely the CUP&#45;SHAPED COTYLEDON &#40;CUC&#41; genes, and suppressed the expression of miR164, whose product cleaves the transcripts of CUC genes. This abnormal phenotype was substantially reversed on the cuc1 mutant background. By contrast, to gain function of TCP3, we generated the miR319&#47;JAW&#45;resistant version of TCP3 &#40;mTCP3&#41;. Expression of mTCP3 suppressed the expression of CUC genes and resulted in the fusion of cotyledons and defects in formation of shoots. Our in situ hybridization assay showed that the TCP3 transcript was present in cotyledons, but not at the cotyledon boundary where CUC genes are functioning. In addition, we found that eight TCPs had functions similar to that of TCP3. Our results demonstrate that the TCP transcription factors play a pivotal role in the control of morphogenesis of shoot lateral organs by negatively regulating the expression of boundary&#45;specific genes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The use of Arabidopsis class 1 <span style="font-style: italic">KNOX</span> gene, <span style="font-style: italic">BP&#47;KNAT1</span> in <span style="font-style: italic">Agrobacterium</span>&#45;mediated transformation of the wild orchid species <span style="font-style: italic">Phalaenopsis amabilis</span><br/></div>

<div>

	<span style="text-decoration: underline" >
	Endang Semiarti</span>
	<sup>1,2</sup>, 

	<span  >
	Ari Indrianto</span>
	<sup>1</sup>, 

	<span  >
	Azis Purwantoro</span>
	<sup>3</sup>, 

	<span  >
	Sulastri Isminingsih</span>
	<sup>2</sup>, 

	<span  >
	Nilo Suseno</span>
	<sup>1</sup>, 

	<span  >
	Takaaki Ishikawa</span>
	<sup>4,5</sup>, 

	<span  >
	Yasushi Yoshioka</span>
	<sup>4</sup>, 

	<span  >
	Yasunori Machida</span>
	<sup>4</sup>, 

	<span  >
	Chiyoko Machida</span>
	<sup>5,6</sup>

</div>

<div>

	<sup>1</sup>Faculty of Biology, Gadjah Mada University, Yogyakarta, Indonesia, 

	<sup>2</sup>Center Study of Biotechnology, Gadjah Mada University, Yogyakarta, Indonesia, 

	<sup>3</sup>Faculty of Agriculture, Gadjah Mada University, Yogyakarta, Indonesia, 

	<sup>4</sup>Graduate School of Science, Nagoya University, Nagoya, Japan, 

	<sup>5</sup>Plant Biology Research Center, Chubu University, Kasugai, Japan, 

	<sup>6</sup>College of Bioscience and Biotechnology, Chubu University, Kasugai, Japan

</div>

<div><span style="font-style: italic">Phalaenopsis</span> hybrids constitute a major ornamental crop. An important parent species for many of these hybrids is <span style="font-style: italic">Phalaenopsis amabilis</span>. We developed a convenient method for the genetic modification of <span style="font-style: italic">P. amabilis</span> using <span style="font-style: italic">Agrobacterium tumefaciens</span>. The transformed intact protocorms, which are young orchid seedlings of <span style="font-style: italic">P. amabilis</span>, regenerated plants under the same conditions that showed the highest frequency of shooting. A kanamycin resistance gene under the control of the 35S promoter can be used as a selective marker. In addition, T&#45;DNA vectors containing the Arabidopsis class 1 <span style="font-style: italic">KNOX</span> gene, <span style="font-style: italic">BP&#47;KNAT1</span>, were successfully introduced into protocorms. Shoots were generated with an abnormal leaf shape that was easily distinguished from that of normal shoots, indicating that <span style="font-style: italic">BP&#47;KNAT1</span> can be used as a visible marker gene. Furthermore, the protocorms transformed with <span style="font-style: italic">BP&#47;KNAT1</span> produced multiple shoots. Both the presence and expression of the transgene in transformed plants were confirmed by molecular analysis.<br/>Key words: Agrobacterium, <span style="font-style: italic">KNOX</span>, multiple shoots, <span style="font-style: italic">Phalaenopsis amabilis</span>, transgenic orchid.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">IRONIC</span>, a new iron deficiency&#45;responsive mutant defective in root epidermal patterning</div>

<div>

	<span style="text-decoration: underline" >
	Paula Perry</span>
	<sup>1</sup>, 

	<span  >
	Wolfgang Schmidt</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>IPMB, Academia Sinica, Taipei, Taiwan

</div>

<div>Roots are the site of nutrient uptake, the supply of which is unevenly distributed in soils and varies with time. Hence, phenotypic plasticity of roots is essential for effective soil exploration. During growth, changes in root architecture and cell fate acquisition occur according to the appropriate environmental conditions. Thereby, post&#45;embryonic plant development is affected by environmental cues, overriding endogenous developmental programs to anticipate forthcoming conditions. Epidermal cells of Arabidopsis roots are particularly responsive to nutritional signals. Cell fate decisions are altered in the absence of some essential but immobile minerals, such as Fe and P. The resulting phenotype is characteristic for the respective condition. In response to Fe deficiency, wild&#45;type Arabidopsis plants form bifurchated root hairs, which results in a substantial increase in the absorptive surface area without a significant increase in the number root hairs formed. We have isolated a new mutant named <span style="font-style: italic">IRONIC &#40;<span style="text-decoration: underline">iron</span> def<span style="text-decoration: underline">ic</span>iency root hair defective&#41;</span>, which respond to Fe deficiency not only with the production of branched root hairs but also with a dramatic increase in the number of cells that form root hairs. Almost all of the epidermal cells in positions normally occupied by non&#45;hair cells are reprogrammed to develop into hairs when subjected to Fe&#45;deficient conditions. The phenotype seen is similar to that of the <span style="font-style: italic">werewolf &#40;wer&#41;</span> mutant, but in a nutrient specific manner. Deviations from the wild&#45;type phenotype were neither observed under other nutrient stresses that affects root epidermal patterning nor under control conditions. The mutant carries a GAL4&#45;GFP enhancer trap construct, enabling the visualization of the expression pattern and single cell&#45;type specific gene expression analysis using fluorescence&#45;activated cell sorting. The GFP expression in ironic is seen in epidermal cells independent of their position relative to underlying cortical cells and within the cortical cell layer. This expression pattern is not altered during Fe deficiency. Transcriptional profiling of the mutant has revealed genes with potential roles in cell fate changes in response to environmental signals.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A new view on the reduction of petals and stamens in cruciferous flowers.</div>

<div>

	<span  >
	Aleksey Penin</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Maria Logacheva</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Moscow State University, Moscow, Russia

</div>

<div>The studies on floral mutants of <span style="font-style: italic">Arabidopsis thaliana</span> and on wild type flowers of related species evidence that reduction of floral organs affects more frequently petals and stamens, not sepals and carpels. However, the genetic mechanism of this remains unclear.&#13;<br/>We report here the results of the studies on a new floral mutant of <span style="font-style: italic">A. thaliana</span> that is characterized by the reduction of petals and a part of stamens. Flower structure of this mutant is similar to those of some species of Lepidium thus it has been called <span style="font-style: italic">lepidium&#45;like</span> &#40;<span style="font-style: italic">lel</span>&#41;. &#13;<br/>In order to clarify the role of <span style="font-style: italic">LEL</span> in a genetic network controlling flower development we have constructed double mutants <span style="font-style: italic">ag lel</span>, <span style="font-style: italic">ap2&#45;1 lel</span>, <span style="font-style: italic">ap3&#45;1 lel</span>, <span style="font-style: italic">ap1&#45;20 lel</span>, <span style="font-style: italic">clv1&#45;1 lel</span>, ptl&#45;1 lel and analyzed the expression of genes involved in flower development using real time RT&#45;PCR. &#13;<br/>Double mutant <span style="font-style: italic">lel ag</span> shows indeterminate flowers with normal petals, suggesting that <span style="font-style: italic">LEL</span> is not a gene that acts specifically to promote petal development. In support of that gene expression analysis has revealed no alteration in the expression of A and B class genes. At the same time, <span style="font-style: italic">lel</span> shows altered expression of genes, controlling stem cell activity. Strong decrease in <span style="font-style: italic">WUS</span> expression suggests that <span style="font-style: italic">LEL</span> is a positive regulator of <span style="font-style: italic">WUS</span>. &#13;<br/>With regard to the fact that <span style="font-style: italic">lel</span> develop normal carpels and sepals its action on stem cell activity is confined to second and third floral whorls, i. e. the middle of the flower. &#13;<br/>The epistasis of <span style="font-style: italic">AG</span> over <span style="font-style: italic">LEL</span> indicates that <span style="font-style: italic">AG</span> acts at the earlier stages of flower development. Thus, the gene responsible for the formation of the meristem zone where petals and stamens develop is acting later than the gene responsible for carpel development. Floral morphology of <span style="font-style: italic">ap2&#45;1 lel</span>, <span style="font-style: italic">ap3&#45;1 lel</span>, <span style="font-style: italic">ap1&#45;20 lel</span>, <span style="font-style: italic">clv1&#45;1 lel</span> also supports the suggestion that during flower development the zones of sepal and carpel formation are activated before the zone which gives rise to petals and stamens and that the activity of stem cells in this zone is mediated by LEL. This provides a possible mechanism for the reduction of petals and stamens as the decrease of the stem cell activity.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The ACR4 receptor kinase is required for giant cell development in <span style="font-style: italic">Arabidopsis</span> sepals</div>

<div>

	<span style="text-decoration: underline" >
	Adrienne Roeder</span>
	<sup>1</sup>, 

	<span  >
	Vijay Chickarmane</span>
	<sup>1</sup>, 

	<span  >
	Carolyn Ohno</span>
	<sup>1</sup>, 

	<span  >
	Elliot Meyerowitz</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>California Institute of Technology, Pasadena, CA, USA

</div>

<div>One of the key questions in developmental biology is how cell division and cell expansion are regulated in conjunction with cell fate specification to produce specialized cells of the correct size and shape.  The outer epidermis of the <span style="font-style: italic">Arabidopsis</span> sepal contains highly elongated giant cells interspersed between smaller pavement cells, guard cells, and trichomes, making it a useful system for addressing this question.  Giant cells average 364 &#181;m &#40;&#177;145 s.d.&#41; in length, but the longest ones reach over half the length of the sepal.  Giant cells also have enlarged nuclei suggesting that they have undergone endoreduplication, a specialized cell cycle in which the DNA is replicated, but cell division does not occur.  To identify genes that are required for giant&#45;cell development, we have conducted an EMS mutant screen and we have isolated a number of mutants that lack giant cells as well as one mutant with almost entirely large cells.  We have focused on two allelic mutants, in which fully elongated giant cells are absent in the sepals.  We have observed, however, that smaller giant cells appear to develop in the sepals of these mutants.  The nuclei in the smaller giant cells of the mutant are not as large as wild&#45;type giant&#45;cell nuclei suggesting that they have undergone fewer endocycles.  On the molecular level, a giant cell&#45;specific marker is expressed in the smaller giant cells present in the mutant.  A second marker expressed specifically in the non&#45;giant pavement cells of wild&#45;type sepals is similarly excluded from the smaller giant cells of the mutant.  These results suggest that giant&#45;cell fate is specified, at least partially, in the mutant, but that the full expansion of the giant cells fails to occur.  Through positional cloning we have found that these mutations are in the <span style="font-style: italic">Arabidopsis CRINKLY4</span> gene, which encodes a receptor kinase involved in epidermal and ovule development &#40;Gifford et al., 2003; Watanabe et al., 2004; Cao et al., 2005&#41;, suggesting that a signaling pathway is involved in giant cell development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Differential requirement for the function of SRD2, an snRNA transcription activator, in various stages of plant development</div>

<div>

	<span style="text-decoration: underline" >
	Misato Ohtani</span>
	<sup>1</sup>, 

	<span  >
	Taku Demura</span>
	<sup>1</sup>, 

	<span  >
	Munetaka Sugiyama</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Plant Science Center, RIKEN, 

	<sup>2</sup>Bot. Garden., Grad. Sch. Sci., The Univ. Tokyo

</div>

<div>snRNA &#40;small nuclear RNA&#41; is a class of eukaryotic non&#45;coding RNAs, which have essential roles in pre&#45;mRNA splicing and rRNA processing. As these functions are fundamental to cell activities, the regulation of snRNA transcription should be a vital issue for all eukaryotes. Here we address developmental control of snRNA transcription and its significance through the analysis of arabidopsis <span style="font-style: italic">SRD2</span> gene, which encodes an activator of snRNA transcription. In young seedlings, a high level of <span style="font-style: italic">SRD2</span> expression was observed in shoot and root apical meristems, leaf primordia, and root stele tissues, where a large amount of snRNA accumulated. In grown&#45;up plants, <span style="font-style: italic">SRD2</span> was highly expressed in developing leaves and flowers as well as apical meristems. Mutations in the <span style="font-style: italic">SRD2</span> gene interfered with many but not all aspects of development in the regions that showed strong expression of <span style="font-style: italic">SRD2</span>. Of note, establishment of the fully active state of apical meristems in the seedling stage was very sensitive to the <span style="font-style: italic">srd2</span> mutation, while maintenance of the established meristems was substantially insensitive. These results demonstrated differential requirement for the SRD2 function in various stages of plant development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The roles of dsRNA binding proteins in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Shaun Curtin</span>
	<sup>1,2</sup>, 

	<span  >
	Neil Smith</span>
	<sup>1</sup>, 

	<span  >
	John Watson</span>
	<sup>1</sup>, 

	<span  >
	Paul Roffey</span>
	<sup>2</sup>, 

	<span  >
	Chris Blanchard</span>
	<sup>2</sup>, 

	<span  >
	Peter Waterhouse</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CSIRO Plant Industry, Canberra, (ACT), Australia, 

	<sup>2</sup>Charles Sturt University, Wagga Wagga, (NSW), Australia

</div>

<div>Double&#45;stranded binding proteins &#40;dsRBP&#41; are proteins that specifically bind to dsRNA. In humans and <span style="font-style: italic">Drosophila</span> they have been shown to be important components of the RNA silencing pathways of these organisms. We have obtained T&#45;DNA insertion lines for each of the five dsRBP homologs in <span style="font-style: italic">Arabidopsis</span> &#40;DRB1&#45;5&#41; and have investigated their roles in the miRNA, trans&#45;acting siRNA, DNA methylation, mobile silencing signal and viral defence pathways. A comprehensive analysis using double and triple DRB mutants was undertaken and the results from this study will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular mechanism of PSII biogenesis</div>

<div>

	<span style="text-decoration: underline" >
	Lixin Zhang</span>
	<sup>1</sup>, 

	<span  >
	Lianwei Peng</span>
	<sup>1</sup>, 

	<span  >
	Jinfang Ma</span>
	<sup>1</sup>, 

	<span  >
	Jinkui Guo</span>
	<sup>1</sup>, 

	<span  >
	Wei Chi</span>
	<sup>1</sup>, 

	<span  >
	Qingtao Lu</span>
	<sup>1</sup>, 

	<span  >
	Congming Lu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Botany, Beijing, CHINA

</div>

<div>Chloroplast development is largely dependent on the nuclear encoded proteins. To further unravel the regulation of chloroplast biogenesis, we screened 60000 T&#45;DNA tagged lines and isolated 90 Arabidopsis mutants exhibiting the phenotype with high chlorophyll fluorescence. A mutant with reduced PSII accumulation was subjected to further investigation. The leaves of the mutant were paler than those of the wild type, and growth was significantly reduced in the mutant. In addition, the wild&#45;type chloroplasts displayed well&#45;developed membrane systems composed of grana connected by the stroma lamellae, but the thylakoid membrane systems in lpa1 chloroplasts were disturbed, and the membrane spacing was not as clear. To identify the precise site of the lpa1 mutation, noninvasive fluorometric analyses were performed. In the lpa1 mutant, Fv&#47;Fm appeared to be significantly lower &#40;0.53&#41; than in wild&#45;type plants &#40;0.82&#41;, implying that the mutants have defects in energy transfer within PSII or a partial loss of PSII capacity. In vivo protein labeling experiments showed that synthesis of the D1 and D2 proteins was greatly reduced in the lpa1 mutant, while other plastid&#45;encoded proteins were translated at rates similar to the wild type. In addition, turnover rates of the PSII core proteins CP47, CP43, D1, and D2 were higher in lpa1 than in wild&#45;type plants. The newly synthesized PSII proteins were assembled into functional protein complexes, but the assembly was less efficient in the mutant. LPA1 encodes a chloroplast protein that contains two tetratricopeptide repeat domains and is an intrinsic membrane protein but not an integral subunit of PSII. Yeast two&#45;hybrid studies revealed that LPA1 interacts with D1 but not with D2, Cyt.b6, or Alb3. Thus, LPA1 appears to be an integral membrane chaperone that is required for efficient PSII assembly, probably through direct interaction with the PSII reaction center protein D1.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">NAC transcription factors NST1, NST2 and NST3 are key regulators of secondary wall formation in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Nobutaka Mitsuda</span>
	<sup>1</sup>, 

	<span  >
	Akira Iwase</span>
	<sup>1</sup>, 

	<span  >
	Hiroyuki Yamamoto</span>
	<sup>2</sup>, 

	<span  >
	Masato Yoshida</span>
	<sup>2</sup>, 

	<span  >
	Motoaki Seki</span>
	<sup>3</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>3</sup>, 

	<span  >
	Masaru Ohme-Takagi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Advanced Industrial Science and Technology (AIST), Tsukuba, JAPAN, 

	<sup>2</sup>Nagoya University, Nagoya, JAPAN, 

	<sup>3</sup>RIKEN Plant Science Center, Yokohama, JAPAN

</div>

<div>Wood is the largest terrestrial biomass and widely used in our life. However, the key factor that regulates wood formation remained still obscure. Wood is formed by successive addition of secondary xylem whose cells have heavily thickened secondary walls mainly composed of lignin and cellulose. In our comprehensive study of Arabidopsis transcription factors, we identified that NAC domain transcription factors, NST1 &#40;NAC Secondary wall Thickening promoting factor1&#41;, NST2 and NST3, are master regulators of secondary wall formation in <span style="font-style: italic">Arabidopsis</span>. <span style="font-style: italic">NST1</span> and <span style="font-style: italic">NST2</span> are expressed in anther and disruption of both genes resulted in indehiscent anther due to loss of secondary walls in anthers, which is required to generate tension for dehiscence. On the other hand, <span style="font-style: italic">NST1</span> and <span style="font-style: italic">NST3</span> are expressed in xylem of stem and hypocotyl, which has similar structural feature to a trunk of tree. The double mutant of these genes &#40;<span style="font-style: italic">nst1&#45;1 nst3&#45;1</span>&#41; cannot stand erect due to loss of secondary wall formation in fiber and secondary xylem of hypocotyls except vascular vessel. Conversely, over&#45;expression of <span style="font-style: italic">NST</span>s induced ectopic secondary wall formation in various aboveground tissues. Genes that are involved in the biosynthesis of the components of wood, namely lignin and cellulose, clearly downregulated in the knockout plants while they are upregulated in the plant over&#45;expressing <span style="font-style: italic">NST1</span>. We concluded that the NST transcription factors are master regulators of secondary wall formation in various tissues except vascular vessels. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Comparison of Arabidopsis growth analysis using traditional methods versus imaging techniques</div>

<div>

	<span style="text-decoration: underline" >
	Gonzalo Estavillo</span>
	<sup>1</sup>, 

	<span  >
	Daniel Harris-Pascal</span>
	<sup>1</sup>, 

	<span  >
	Robert Furbank</span>
	<sup>2</sup>, 

	<span  >
	Barry Pogson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The ARC Centre of Excellence in Plant Energy Biology, The Australian National University, Canberra, Australia, 

	<sup>2</sup>CSIRO Plant Industry , Canberra, Australia

</div>

<div>Growth analysis using simple primary data can be used to study plant morphology and function. The classical, destructive method involves tissue harvesting and estimation of different plant physical parameters such as leaf area and weight at regular intervals. An alternative method is the use of non invasive techniques based in image recording and software analyses that allows for periodical sampling in a non destructive way. In theory, this approach requires less starting material than the former, and the growth of the same individual can be documented through the life cycle or experimental conditions. It allows also for analysis of many other structural and morphological traits of a large number of plants. &#13;<br/>Although the intrinsic advantages of the imaging system, the results should be compared with those of traditional techniques to assess its utility in specific cases. Here we compare the results of plant growth analyses of wild type and mutant Arabidopsis plants performed with the traditional method versus morphological phenotyping with Scnalyzer &#40;Lemnatec&#41;.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">How Homeotic Protein AGAMOUS Controls Floral Meristem Determinacy in Arabidopsis&#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	Bo Sun</span>
	<sup>1</sup>, 

	<span  >
	Toshiro Ito</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Temasek Life Sciences Laboratory,Singapore,Singapore

</div>

<div>   In <span style="font-style: italic">Arabidopsis</span>, floral meristem activity ceases after formation of four different types of floral organs including sepals, petals, stamens and carpels. It is known that <span style="font-style: italic">AGAMOUS&#40;AG&#41;</span>, which encodes a MADS&#45;domain transcription factor, controls stamen and carpel development and also functions in terminating meristematic activity in flower buds. <span style="font-style: italic">WUSCHEL&#40;WUS&#41;</span>, which encodes a homeodomain transcription factor, is essential for maintenance of stem cells. Even though <span style="font-style: italic">WUS</span> is suggested to be suppressed in the downstream of <span style="font-style: italic">AG</span>, it is still a mystery how AG represses <span style="font-style: italic">WUS</span>. Thus, we started working on the transcriptional cascades from <span style="font-style: italic">AG</span> to <span style="font-style: italic">WUS</span> to understand how and in what timing meristematic activity is terminated.&#13;<br/>       <span style="font-style: italic">KNUCKLES&#40;KNU&#41;</span>, mutation of which leads to ectopic stamens and carpels inside developing gynoecia, is involved in floral meristem determinacy. KNU, a C2H2 zinc finger protein, has the C&#45;terminal EAR&#45;like repression motif, suggesting that <span style="font-style: italic">KNU</span> could be a transcriptional repressor. However, the relationships among <span style="font-style: italic">KNU</span>, <span style="font-style: italic">AG</span> and <span style="font-style: italic">WUS</span> are unclear.&#13;<br/>      We first analyzed <span style="font-style: italic">KNU</span> expression in transgenic plants with inducible AG activity and found that AG induces <span style="font-style: italic">KNU</span> within 48 hours. Our genetic data also showed that <span style="font-style: italic">ag</span> is epistatic to <span style="font-style: italic">knu</span>. Through GUS reporter analysis of <span style="font-style: italic">KNU</span> promoter, we confirmed that <span style="font-style: italic">KNU</span> expression is fully dependent on AG. All of these data suggest that <span style="font-style: italic">KNU</span> is a downstream gene of <span style="font-style: italic">AG</span>. Next, to test whether KNU represses <span style="font-style: italic">WUS</span>, we constructed the transgenic line with inducible <span style="font-style: italic">KNU</span> activity. After the continuous induction, we observed <span style="font-style: italic">wus&#45;like</span> flowers that lack gynoecium. We also observed prolonged <span style="font-style: italic">WUS</span> expression in <span style="font-style: italic">knu</span> mutant. These suggest KNU functions as a repressor of <span style="font-style: italic">WUS</span>. Furthermore, we observed that over&#45;expression of KNU could rescue the indeterminate phenotype of <span style="font-style: italic">ag&#45;1</span>, showing that <span style="font-style: italic">KNU</span>, a downstream gene of <span style="font-style: italic">AG</span>, plays a major function to terminate <span style="font-style: italic">WUS</span> expression in the floral meristem. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Expression studies and overexpression phenotypes indicate a role in cell separation processes for the &#34;IDL&#34; genes in &#34;Arabidopsis thaliana&#34;</div>

<div>

	<span style="text-decoration: underline" >
	Nora M. Tandstad</span>
	<sup>1</sup>, 

	<span  >
	Grethe-Elisabeth Stenvik</span>
	<sup>1</sup>, 

	<span  >
	Melinka A. Butenko</span>
	<sup>1</sup>, 

	<span  >
	Asbj&#248;rn Holmgren</span>
	<sup>1</sup>, 

	<span  >
	Reidunn B. Aalen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biosciences, University of Oslo, Oslo, Norway

</div>

<div>Small peptides can act as signaling molecules that coordinate development, growth and differentiation. Upon interaction with a receptor the ligand can trigger downstream pathways which induce cellular responses or regulation of gene expression. Recently, a novel group of putative ligands in plants, the IDA&#45;LIKE &#40;IDL&#41; proteins, were identified based on their similarities to IDA, a putative ligand involved in floral organ abscission &#40;Butenko et al. 2003, Plant Cell, 15: 2296&#45;307&#41;. Localization in the extracellular space is a necessity for being a ligand. The IDL proteins contain an N&#45;terminal signal peptide predicted to guide the proteins out of the cell. In consistence with this, our preliminary results indicated that IDL3&#45;GFP fusion protein in transgenic Arabidopsis was localized to the periphery of the cell. Histochemical analyses of promoter&#45;GUS constructs for the five &#8220;IDL&#8221; genes in transgenic Arabidopsis have been performed. Since IDA is involved in the cell separation process that induces the floral organs to be shed, it was interesting to find that the GUS expression of the the &#8220;IDL&#8221; genes in many cases was associated with sites of cell separation, such as AZs, valves of developing carpels and at the base of pedicels. Overexpression of the &#8220;IDL&#8221; genes resulted in similar phenotypes featured by early senescence in rosette and cauline leaves, premature floral organ abscission, and shedding of organs that are normally not shed in Arabidopsis; pedicels and cauline leaves. In constrast, plants overexpressing IDL1 lacking the signal peptide, were not phenotypically affected, confirming that the extracellular localization of these proteins is necessary in order to be functional. Similar overexpression phenotypes were also observed in plants overexpressing IDA &#40;Stenvik et al., 2006, Plant Cell, 18: 1467&#45;76&#41;. These data suggest some degree of functional redundancy between &#8220;IDA&#8221; and the &#8220;IDL&#8221; genes, and may indicate a possible role for the &#8220;IDL&#8221; genes in cell separation processes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cytokinins Determine Arabidopsis Root&#45;Meristem Size by Controlling Cell Differentiation</div>

<div>

	<span style="text-decoration: underline" >
	Raffaele Dello Ioio</span>
	<sup>1</sup>, 

	<span  >
	Francisco Scaglia Linhares</span>
	<sup>1</sup>, 

	<span  >
	Emanuele Scacchi</span>
	<sup>1</sup>, 

	<span  >
	Eva Casamitjana-Martinez</span>
	<sup>2</sup>, 

	<span  >
	Renze Heidstra</span>
	<sup>2</sup>, 

	<span  >
	Paolo Costantino</span>
	<sup>2</sup>, 

	<span  >
	Sabrina Sabatini</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University ?La Sapienza?, Rome, Italy, 

	<sup>2</sup>Utrecth University, Utrecht, The Netherlands

</div>

<div>Plant postembryonic development takes place in the<br/>meristems, where stem cells self&#45;renew and produce<br/>daughter cells that differentiate and give rise to different<br/>organ structures. For the maintenance of meristems,<br/>the rate of differentiation of daughter cells<br/>must equal the generation of new cells: How this is<br/>achieved is a central question in plant development.<br/>In the Arabidopsis root meristem, stem cells surround<br/>a small group of organizing cells, the quiescent center.<br/>Together they form a stem cell niche , whose<br/>position and activity depends on the combinatorial<br/>action of two sets of genes&#8212;PLETHORA1 &#40;PLT1&#41; and<br/>PLETHORA2 &#40;PLT2&#41;  and SCARECROW &#40;SCR&#41;<br/>and SHORTROOT &#40;SHR&#41;&#8212;as well as on polar auxin<br/>transport. In contrast, the mechanisms controlling<br/>meristematic cell differentiation remain unclear.<br/>Here, we report that cytokinins control the rate of meristematic<br/>cell differentiation and thus determine rootmeristem<br/>size via a two&#45;component receptor histidine<br/>kinase&#45;transcription factor signaling pathway. Analysis<br/>of the root meristems of cytokinin mutants, spatial<br/>cytokinin depletion, and exogenous cytokinin application<br/>indicates that cytokinins act in a restricted region<br/>of the root meristem, where they antagonize a noncell&#45;<br/>autonomous cell&#45;division signal, and we provide<br/>evidence that this signal is auxin.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">HAWAIIAN SKIRT</span> &#8211; an F&#45;box gene that regulates organ fusion and growth in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Zinnia H Gonzalez-Carranza</span>
	<sup>1</sup>, 

	<span  >
	Unchalee Rompa</span>
	<sup>1</sup>, 

	<span  >
	Janny L Peters</span>
	<sup>2</sup>, 

	<span  >
	Anuj Bhatt</span>
	<sup>3</sup>, 

	<span  >
	Carol Wagstaff</span>
	<sup>4</sup>, 

	<span  >
	Anthony Stead</span>
	<sup>5</sup>, 

	<span  >
	Jeremy A Roberts</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Biosciences, University of Nottingham, Sutton Bonington Campus, Loughbrough, United Kingdom., 

	<sup>2</sup>Faculty of Science, Radboud University Nijmegen,  Nijmegen, The Netherlands., 

	<sup>3</sup>Department of Plant Sciences, University of Oxford, Oxford, United Kingdom., 

	<sup>4</sup>School of Biological Sciences, University of Southampton, Southampton, United Kingdom., 

	<sup>5</sup>School of Biological Sciences, Royal Holloway, University of London,  Surrey, United Kingdom.

</div>

<div>The <span style="font-style: italic">hawaiian skirt</span> &#40;<span style="font-style: italic">hs</span>&#41; mutant of Arabidopsis was identified from a fast neutron&#45;mutagenised population of Col&#45;0 WT. The site of the mutation has been mapped to chromosome 3 and found to be the consequence of a 28bp deletion that introduces a premature amber termination codon into the ORF of an F&#45;box gene.  The principle phenotypic feature associated with the mutant is the failure of floral organs to abscise and this is due to the fusion of the sepals along the lower part of their margins.  By introgressing the abscission marker <span style="font-style: italic">ProPGAZAT:GUS</span> into the mutant it was possible to demonstrate that the differentiation of the abscission cells takes place but that cell separation is delayed in <span style="font-style: italic">hs</span>. Spatial and temporal characterisation of HS expression, using <span style="font-style: italic">ProHS:GUS</span> or <span style="font-style: italic">ProHS:GFP</span> fusions, has identified expression to take place at discrete sites throughout the plant. By comparing the <span style="font-style: italic">hs</span> mutant, Col&#45;0 WT, and <span style="font-style: italic">Pro35S:HS</span> ectopically expressing lines we have demonstrated that the silencing of <span style="font-style: italic">HS</span> results in plants with larger leaves, roots and seeds, whilst over&#45;expressing the gene produces the opposite effect. These results are discussed in the context of the role of <span style="font-style: italic">HS</span> during plant development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The pre&#45;replication factor AtMCM2, is involved in the maintenance of the columella root meristem initials</div>

<div>

	<span  >
	Di-An Ni</span>
	<sup>1</sup>, 

	<span  >
	Cecile Raynaud</span>
	<sup>1</sup>, 

	<span  >
	Rossangela Sozzani</span>
	<sup>2</sup>, 

	<span  >
	Rino Cella</span>
	<sup>2</sup>, 

	<span  >
	Sophie Blanchet</span>
	<sup>1</sup>, 

	<span  >
	Diego Albani</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Catherine Bergounioux</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CNRS, Universit&#233; Paris-sud, IBP, France, 

	<sup>2</sup>Depart. of genetics , University of Pavia, Pavia, Italy, 

	<sup>3</sup>Depart. of Botany , University of Sassari, Italy

</div>

<div>Initiation of DNA replication is a crucial decision point during cell proliferation and lies at the point of convergence of complex networks of signaling molecules that have evolved to specify when and where cells divide in an organism. This process is precisely regulated during G1 phase of the cell cycle by the pre&#45;replication complex &#40;pre&#45;RC&#41; &#40;Dresselhaus et al., 2006&#41;. Compared to budding yeast, animal and human, Arabidopsis possesses all the homologues of MCM proteins including two recently identified non&#45;MCM2&#45;7 proteins &#40;MCM8 and MCM9&#41;.The Arabidopsis AtMCM2 encodes a homologue of replication licensing factor MCM2. Arabidopsis mutant analysis indicated that AtMCM2 is embryo lethal. Transgenic Arabidopsis overexpressing AtMCM2 &#40;MCM2OE&#41; remain small in size and smaller cells are observed in MCM2OE plants. In addition, leaf DNA content analysis shows that endoreduplication is also inhibited, thus confirming AtMCM2 involvement in DNA replication. Moreover, in agreement with the behavior of almost all factors of the pre&#45;replication complex, AtMCM2 up and down regulation depends on E2F transcription factors. Expression studies show that AtMCM2 is developmentally regulated. Interestingly, during root development AtMCM2 expression is high in secondary root primordia and becomes later restricted to the central cylinder above the root quiescent center and excluded from the cortical cells of the dividing zone of root meristems. Down&#45;regulation of AtMCM2 by RNAi severely affects root growth. Conversely, over&#45;expression of AtMCM2 induces the formation additional stem cells in the root meristem. Moreover, over&#45;expression of MCM2 rescues the phenotype of plt mutants in which the root meristem stops functioning, by allowing the emergence of a lateral root close to the apical meristem. Therefore, the present work may open new prospects on the involvement of the pre&#45;RC in root stem cell specification.<br/><br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis MYB26&#47;MALE STERILITY35 gene controls anther dehiscence by regulating secondary thickening in the anther endothecium</div>

<div>

	<span style="text-decoration: underline" >
	Jie Song</span>
	<sup>1</sup>, 

	<span  >
	Caiyun Yang</span>
	<sup>1</sup>, 

	<span  >
	Zoe A Wilson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Sciences Division, University of Nottingham, Loughborough, LE12 5RD, UK

</div>

<div>Male sterility typically occurs due to the lack of viable pollen production, but can also be due to the failure of pollen release. Anther dehiscence is a two&#45;phase process involving lytic opening of the stomium and then retraction of the anther wall. During microspore maturation, cellulose and lignified thickenings are deposited in the anther endothecium &#40;Dawson et al., 1999&#41;. This cell layer is important in generating the forces required for dehiscence, initially by swelling and causing the stomium to rupture, then by desiccation causing differential shrinkage of thickened and unthickened parts of the cell wall, resulting in an outward bending force, retraction of the anther wall and anther opening.<br/><br/>MYB26&#47;MS35 acts as a key regulator of lignified secondary thickening in the anther endothecium and subsequent dehiscence. The ms35 mutant fails to produce secondary thickening in the endothecium &#40;Dawson et al., 1999&#41;. MYB26&#47;MS35 expression occurs early during endothecial development and is maximal during pollen mitosis I and the bicellular stage, suggesting that it may have a regulatory role in specifying early endothecial cell development.<br/><br/>Over&#45;expression of MYB26 results in ectopic secondary thickening and changes in expression in several genes linked to secondary thickening, including IRREGULAR XYLEM 1 &#40;IRX1&#41;, IRX3, IRX8, IRX12, NAC SECONDARY WALL PROMOTING FACTOR1 &#40;NST1&#41; and NST2. This indicates that MYB26 functions in a regulatory role in determining endothecial cell development and acts upstream of the lignin biosynthesis pathway &#40;Yang et al., 2007&#41;. Affymetrix microarray analysis using isolated anthers suggests that a number of additional genes are also involved in this pathway. Immunolocalisation of pectin and xylan has provided further evidence for the role of MYB26 in the regulation of secondary wall development in the anther.  These data and the potential role for MYB26 will be discussed.<br/><br/>References<br/>Dawson J, Sozen E, Vizir I, Van Waeyenberge S, Wilson ZA, Muligan BJ &#40;1999&#41; New Phytologist, 144: 213&#45;222<br/><br/>Yang C, Xu Z, Song J, Conner K, Barrena GV, Wilson ZA &#40;2007&#41; The Plant Cell. Feb 28</div>

</div>





<div class="abs">

<div style="font-weight:bold">Movement of FT protein from the vasculature to the apex is sufficient to induce flowering in <span style="font-style: italic">Arabidopsis thaliana</span>.</div>

<div>

	<span style="text-decoration: underline" >
	Johannes Mathieu</span>
	<sup>1</sup>, 

	<span  >
	Frank K&#252;ttner</span>
	<sup>1</sup>, 

	<span  >
	Norman Warthmann</span>
	<sup>1</sup>, 

	<span  >
	Markus Schmid</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Developmental Biology, Department of Molecular Biology, Spemannstrasse 37-39, D-72076 T&#252;bingen, Germany

</div>

<div>Correct timing of flowering is a crucial factor for reproductive success in plants, and the time at which the shoot apical meristem &#40;SAM&#41; switches fate from vegetative to reproductive growth is tightly regulated by intrinsic and extrinsic factors. An important environmental input is day length. More than 70 years ago grafting experiments provided evidence that plants produce in their leaves a signal that translocates to the SAM to induce flowering. The nature of this signal, termed &#8220;florigen&#8221;, has long eluded discovery. <br/>Recently, studies in <span style="font-style: italic">Arabidopsis</span> have shown that two genes, <span style="font-style: italic">CONSTANS &#40;CO&#41;</span> and <span style="font-style: italic">FLOWERING LOCUS T &#40;FT&#41;</span>, that are predominately expressed in leaves, are central for the plant to respond to inductive photoperiod. It was further shown that <span style="font-style: italic">FT</span> is the primary transcriptional target of <span style="font-style: italic">CO</span> in the leaf and that <span style="font-style: italic">FT</span> mRNA can be detected at the SAM after heat shock induced expression in the leaves. However, whether movement of the <span style="font-style: italic">FT</span> mRNA is necessary and &#47; or sufficient to induce flowering is unclear. <br/>Here we show that expression of a functional artificial miRNA targeting <span style="font-style: italic">FT</span>&#45;mRNA &#40;<span style="font-style: italic">amiR&#45;FT</span>&#41; under control of the <span style="font-style: italic">FD</span>&#45; &#40;SAM&#45;specific&#41; promotor does not suppress flowering. In contrast, plants expressing <span style="font-style: italic">amiR&#45;FT</span> from either the <span style="font-style: italic">35S&#45;</span> or the <span style="font-style: italic">SUC2</span>&#45;promotor mimic the late&#45;flowering of the <span style="font-style: italic">ft</span> mutant phenotype. Together, these results suggest that <span style="font-style: italic">FT</span> mRNA is required in the vasculature but might not be necessary at the apex to induce flowering. To assess the role of FT protein movement during floral transition, we developed a novel assay in which a large FT fusion protein is retained in the tissue where it is expressed. Upon exposure to a protease the mature FT protein is released and free to act. Using this assay we demonstrate that release of FT protein in the vasculature is sufficient to induce flowering. <br/>These results suggest an important role for the movement of FT protein in the induction of flowering by photoperiod.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification and analysis of functional domains of the bifunctional transcription factor, WUSCHEL</div>

<div>

	<span style="text-decoration: underline" >
	Miho Ikeda</span>
	<sup>1</sup>, 

	<span  >
	Masaru Ohme-Takagi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Advanced Industrial Science and Technology, Central 4, Higashi 1-1-1, Tsukuba, Ibaraki, Japan

</div>

<div><span style="font-style: italic">WUSCHEL</span> is known to be a multiple functional gene, which is involved in the maintenance of shoot apical meristem &#40;SAM&#41;, transition of developmental phase and phytohormone signaling. WUSCHEL is an activator of <span style="font-style: italic">AGAMOUS</span> but acts as a repressor for <span style="font-style: italic">ARR</span> genes. This suggests that WUSCHEL is a bifunctional transcription factor, which acts as activator or repressor depending on the target gene. But the functional mechanism of <span style="font-style: italic">WUSCHEL</span> gene was not fully analyzed. Transient expression assay showed that WUSCHEL, as it is, acted as a repressor and a series of mutation analysis revealed that WUSCHEL contained two independent repression domains. One is the EAR&#45;like motif that was previously reported as a repression domain conserved in various transcriptional active repressors, and the other was a novel domain that was conserved among <span style="font-style: italic">WUSCHEL</span>&#45;related homeobox genes and all of the known <span style="font-style: italic">WUSCHEL</span> ortholog genes. On the other hands, mutation analyses showed that WUSCHEL had an activation domain. Analysis of the transgenic plants that respectively expressed a series of mutated <span style="font-style: italic">WUSCHEL</span> gene revealed that the maintenance of SAM and the induction of embryonic gene expressions in germinated plants were regulated by the repression activity of WUSCHEL in addition to the regulation of <span style="font-style: italic">ARR</span> gene expressions. We also identified that the novel repression domain mainly regulated cytokinin signaling, maintenance of SAM and developmental phase transition &#40;embryonic to vegetative&#41;, while the EAR&#45;like domain has a supportive function of these physiological phenomena.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The <span style="font-style: italic">ASYMMETRIC LEAVES2</span> regulates leaf polarity by repressing <span style="font-style: italic">ETTIN</span> in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Mayumi Iwasaki</span>
	<sup>1</sup>, 

	<span  >
	Hidekazu Iwakawa</span>
	<sup>1</sup>, 

	<span  >
	Yoshihisa Ueno</span>
	<sup>2</sup>, 

	<span  >
	Hiroo Takahashi</span>
	<sup>3</sup>, 

	<span  >
	Shoko Kojima</span>
	<sup>1,4</sup>, 

	<span  >
	Takeshi Kobayashi</span>
	<sup>1,4</sup>, 

	<span  >
	Yasunori Machida</span>
	<sup>2</sup>, 

	<span  >
	Chiyoko Machida</span>
	<sup>1,4</sup>

</div>

<div>

	<sup>1</sup>Plant Biology Research Center, Chubu Univ., Kasugai, Japan, 

	<sup>2</sup>2 Grad. Sch. of Sci., Nagoya Univ., Nagoya, Japan, 

	<sup>3</sup>Grad. Sch. of Eng., Nagoya Univ., Nagoya, Japan, 

	<sup>4</sup>College of Biosci. and Biotech., Chubu Univ., Kasugai, Japan

</div>

<div>The <span style="font-style: italic">ASYMMETRIC LEAVES2</span> &#40;<span style="font-style: italic">AS2</span>&#41; and <span style="font-style: italic">ASYMMETRIC LEAVES1</span> &#40;<span style="font-style: italic">AS1</span>&#41; genes of <span style="font-style: italic">Arabidopsis thaliana</span> are required for symmetrical and flat lamina expansion. <span style="font-style: italic">AS2</span> encodes a plant specific protein that contains AS2&#47;LOB domain, and <span style="font-style: italic">AS1</span> encodes a myb &#40;SANT&#41; domain protein. AS2 and AS1 are thought to act as a transcriptional regulator of certain genes including class 1 <span style="font-style: italic">KNOX</span> genes. To identify the downstream target of <span style="font-style: italic">AS2</span>, microarray analysis of mRNA from shoot apices was performed. Expression levels of several genes involving abaxial identity were altered in <span style="font-style: italic">as1</span>, <span style="font-style: italic">as2</span>, and <span style="font-style: italic">pAS1::AS2</span> plants. We examined the expression of <span style="font-style: italic">ARF2</span>, <span style="font-style: italic">ETT&#47;ARF3</span>, <span style="font-style: italic">ARF4</span>, four <span style="font-style: italic">KANADI</span> genes, five <span style="font-style: italic">YABBY</span> genes and the class 1 <span style="font-style: italic">KNOX</span> genes by quantitative RT&#45;PCR, <span style="font-style: italic">ETT</span>, <span style="font-style: italic">KAN2</span>, <span style="font-style: italic">YABBY5</span>, as well as <span style="font-style: italic">BP</span>, were upregulated in <span style="font-style: italic">as1</span> and <span style="font-style: italic">as2</span> and downregulated in <span style="font-style: italic">pAS1::AS2</span> plants. AS2 and AS1 might regulate adaxial identity by repressing these genes. To assess whether AS2 acts directly regulates <span style="font-style: italic">ETT</span>, <span style="font-style: italic">KAN2</span>, <span style="font-style: italic">YABBY5</span>, and <span style="font-style: italic">BP</span>, we used glucocorticoid receptor &#40;GR&#41;&#45;mediated inducible system. After the treatment 35S:AS2&#45;GR plants with DEX, expression of <span style="font-style: italic">ETT</span> and <span style="font-style: italic">BP</span> were decreased, but those of <span style="font-style: italic">KAN2</span> and <span style="font-style: italic">YABBY5</span> were not changed within 12 hours. <span style="font-style: italic">ETT</span> is known to be a target of tasiR&#45;ARF. We report how AS2 regulates the <span style="font-style: italic">ETT</span> expression in the process of leaf development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">SCHIZORIZA is a regulator of the stem cell niche development.</div>

<div>

	<span style="text-decoration: underline" >
	Monica Pernas</span>
	<sup>1</sup>, 

	<span  >
	Eoin Ryan</span>
	<sup>1</sup>, 

	<span  >
	Paul Linstead</span>
	<sup>1</sup>, 

	<span  >
	Liam Dolan</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Cell and Developmental Biology. John Innes Centre, Norwich, UK.

</div>

<div>The root meristem contains a group of stem cells responsible for the generation of the different cell layers of the root during the life of the plant.  These stem cells or initials surround  a group of slowly dividing cells known as the quiescent centre &#40;QC&#41;. Together the stem cells and the QC constitute the stem cell niche of the root. The QC acts as a source of signals that position and maintain the stem cells during development. We have discovered that SCHIZORIZA &#40;SCZ&#41; is a putative transcription factor that is required for the maintenance of the stem cell niche in the arabidopsis root. The phenotype of the &#34;scz&#34; mutant and the pattern of SCZ gene expression support this role. The patterns of cell division in the stem cells niche are altered in &#34;scz&#34; mutants compared to wild type indicating a requirement for SCZ in the the maintenance of the stem cell niche organization. Furthermore SCZ positively regulates the differentiation of the columella initials given that these cells types do not develop in &#34;scz&#34; mutants. SCZ regulates also the activity of ground tissue initials since these cells undergo a periclinal division in &#34;scz&#34; mutants instead of an anticlinal division that occur in wild type. This results in the formation of extra layers of ground tissue with mixed endodermal and cortical identity in the &#34;scz&#34; mutant. To verify the role of SCZ in the function of the stem cell niche we show that SCZ is expressed in the QC, ground tissue initials, endodermis and cortex layers. SCZ is therefore an important regulator of the root stem cell nice and we are currently characterising its interaction with other genes that control the development of this structure.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Expression and function analyses of abscission related gene At3g14380</div>

<div>

	<span style="text-decoration: underline" >
	Li zhang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Nottingham

</div>

<div>Abscission is an important process during plant development. The shedding of plant organs, such as leaves, flowers and fruits, takes place at a predetermined position, called an abscission zone &#40;AZ&#41;.  Studies about abscission have been focusing on identifying abscission specific genes. Arabidopsis gene AT3G14380 is predicted to be up&#45;regulated during abscission based on microarray experiment. RT&#45;PCR showed that this gene was expressed in the abscising flower tissues. The expression of GUS was seen at the AZ and stigmatic surface in At3g14380: GUS transgenic plants. To identify the function of this gene, the construct with down regulated of At3g14380 expression was generated and transformed to wild type &#40;Arabidopsis Columbia&#41;.  Delayed abscission was observed in those transgenic plants compared with wild type. Petal break strength was measured for those plants and results showed that the mutant displayed delayed abscission in the agreement with phenotypic analysis. At3g14380 over&#45;expression construct driving by 35S promoter has been transformed to Arabidopsis. No aberrant phenotype was found in F1 transgenic plants. Analyses of homozygous lines are in the progress.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Defective of Tapetum Development and Function1 &#40;TDF1&#41; is essential for tapetal development and plays a vital role in tapetum gene regulation network in Arabidopsis</div>

<div>

	<span  >
	Hui Chen</span>
	<sup>1</sup>, 

	<span  >
	Jun Zhu</span>
	<sup>1</sup>, 

	<span  >
	Hui Li</span>
	<sup>1</sup>, 

	<span  >
	Yao Song</span>
	<sup>1</sup>, 

	<span  >
	Ju-Fang Gao</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Zhong-Nan Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Shanghai Normal University

</div>

<div>Tapetum plays important roles in anther development and is essential for pollen development by providing enzymes for callose dissolution, and materials for pollen wall formation in Arabidopsis. In this study, we reported the molecular analysis of a defective of tapetum development and function1 &#40;tdf1&#41; mutant which is essential for tapetum development and function. The tdf1 mutant shows a complete male sterile phenotype. It has two layers of tapetal cells suggesting that TDF1 negatively regulates tapetal periclinal division. TDF1 was isolated using a map&#45;based cloning strategy and was confirmed by genetic complementation. It encodes a R2R3 MYB transcription factor predominantly located to the nucleus, and highly expresses in meiocytes, tapetum, and microspores in anther development. Callose staining and RT&#45;PCR analysis suggest TDF1 regulate microspore release from tetrad and pollen wall formation. A preliminary regulation network containing nine regulation genes essential for tapetum formation, development and function has been constructed with TDF1 playing a key role in the network. The function investigation of TDF1 and the gene regulation network of tapetal development in this work should provide a roadmap for the investigation of molecular mechanism of tapetal development in plant.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Dek1&#45;a key player in a general model for epidermis formation in plants&#63;</div>

<div>

	<span style="text-decoration: underline" >
	Lene Olsen</span>
	<sup>1</sup>, 

	<span  >
	Qing Tian</span>
	<sup>2,3</sup>, 

	<span  >
	Stein Erik Lid</span>
	<sup>1</sup>, 

	<span  >
	Betty Lemmon</span>
	<sup>4</sup>, 

	<span  >
	Roy Brown</span>
	<sup>4</sup>, 

	<span  >
	Fred Gruis</span>
	<sup>2</sup>, 

	<span  >
	Kjetil Fosnes</span>
	<sup>1</sup>, 

	<span  >
	Marisa Otegui</span>
	<sup>5</sup>, 

	<span  >
	Odd-Arne Olsen</span>
	<sup>2,3</sup>, 

	<span  >
	Hilde-Gunn Opsahl-Sorteberg</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Norwegian University of Life Sciences,1432 Aas,Norway, 

	<sup>2</sup>Pioneer Hi-Bred International,A DuPont Company,Johnston,IA,USA, 

	<sup>3</sup>present address: Monsanto Company, Chesterfield, MO, USA, 

	<sup>4</sup>University of Louisiana,Lafayette,LA,USA, 

	<sup>5</sup>University of Wisconsin-Madison, Madison,WI,USA

</div>

<div>Cell fate specification and development of the maize endosperm aleurone layer, the endosperm epidermis, involve Dek1, Cr4 and Sal1. DEK1 &#40;240 kDa&#41;, consisting of a membrane spanning &#40;DEK1&#45;MEM&#41; domain and a cytosolic calpain&#45;like cystein proteinase &#40;DEK1&#45;CALP&#41;, is essential for position&#45; &#40;surface&#41; dependant formation of aleurone cells. In &#34;Arabidopsis thaliana&#34;, the dek1 lethal phenotype is complemented by the AtDek1 gene. In contrast, DEK1&#45;CALP is unable to complement the mutant. Interestingly, expression of DEK1&#45;MEM leads to a dominant negative phenotype similar to dek&#45;RNAi phenotypes, including lack of protoderm in the embryo and failure to form an apical meristem. These results suggest that Dek1 is activated by a ligand and that the calpain proteinase is regulated by the membrane domain, and that Dek1 is essential for epidermis formation throughout the plant. Embryo cytoskeleton studies show that despite a functional microtubule apparatus, the dek1 embryos have asymmetrical division planes, misplaced phragmoplasts and they appear tiered with protodermal defects. In addition to results from &#34;Arabidopsis thaliana&#34;, we will present data from maize  endosperm in vitro organ cultures that show that  Dek1, Cr4 and Sal1 co&#45;localizes to endosomes, possibly suggesting that these molecules may functionally interact and that a proper concentration of Dek1 and Cr4 in plasma membranes is maintained by internalization and degradation &#34;via&#34; traffic through Sal1&#45;positive endosomes.  Based on the effect of dek1 and cr4 mutations in a variety of epidermal cell types of both monocots and dicots, we propose that the Dek1 pathway described here is responsible for surface recognition, and hence epidermal cell fate specification in all plants. &#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Seed dormancy requires histone H2B monoubiquitination</div>

<div>

	<span style="text-decoration: underline" >
	Yongxiu Liu</span>
	<sup>1</sup>, 

	<span  >
	Kazumi Nakabayashi</span>
	<sup>1</sup>, 

	<span  >
	Maarten Koornneef</span>
	<sup>1</sup>, 

	<span  >
	Wim Soppe</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, Carl-von-Linne-weg 10, 50829 Cologne, Germany

</div>

<div>Seed dormancy has an adaptive role in nature by optimizing germination to the most suitable time, and a tight control of dormancy is very important in crop plants. Extensive genetic and physiological studies have identified the involvement of several factors, but the molecular mechanisms underlying this process are still largely unknown. With the molecular identification of the Arabidopsis HISTONE MONOUBIQUITINATION 1 &#40;HUB1&#41; gene, we have revealed a role for chromatin remodeling in the seed dormancy mechanism. The <span style="font-style: italic">hub1</span> mutant &#40;originally identified as <span style="font-style: italic">rdo4</span>&#41; has reduced dormancy levels and mild pleiotropic phenotypes. <span style="font-style: italic">HUB1</span> encodes a C3HC4 RING finger protein, which is required for monoubiquitination of histone H2B. The Arabidopsis genome contains one <span style="font-style: italic">HUB1</span> homologue, which we named <span style="font-style: italic">HUB2</span>. The <span style="font-style: italic">hub2</span> mutant also has reduced seed dormancy and is not redundant with <span style="font-style: italic">hub1</span>. The ubiquitinated form of histone H2B could not be detected in the <span style="font-style: italic">hub1</span> and <span style="font-style: italic">hub2</span> mutants. Furthermore, we found altered expression levels of dormancy related genes in the <span style="font-style: italic">hub1</span> mutant.&#13;<br/>In yeast and human, monoubiquitination of histone H2B is a prerequisite for histone H3 methylation at Lys&#45;4 and &#45;79 and is associated with actively transcribed genes. We are studying this relation in Arabidopsis and aim to identify the genes that are direct targets of HUB1. Furthermore, we are analyzing the relation between the HUB1 and HUB2 proteins and try to understand more about the role of histone H2B monoubiquitination on seed dormancy by analyzing other components that affect histone H2B monoubiquitination.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Development of tapetum&#45;specific organelles involved in the formation of pollen coats in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Sumie Ishiguro</span>
	<sup>1</sup>, 

	<span  >
	Toshiya Suzuki</span>
	<sup>1</sup>, 

	<span  >
	Yuka Nishimori</span>
	<sup>1</sup>, 

	<span  >
	Michika Sassa</span>
	<sup>1</sup>, 

	<span  >
	Kenzo Nakamura</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Grad. Sch. Bioagricultural Sci., Nagoya Univ., Nagoya, Japan

</div>

<div>The pollen coat, which is a surface component of pollen grains, consists mainly of proteins and lipidic compounds such as sterol esters and hydrocarbons.  To study how the pollen coat is produced, we identified and characterized a conditional male&#45;sterile mutant, <span style="font-style: italic">flaky pollen1&#45;1</span> &#40;<span style="font-style: italic">fkp1&#45;1</span>&#41;, whose pollen grains lack the pollen coat.  <span style="font-style: italic">FKP1</span> encodes 3&#45;hydroxy&#45;3&#45;methylglutaryl&#45;coenzyme A synthase, an enzyme of mevalonate pathway required for the biosynthesis of terpenoids including sterols.  The <span style="font-style: italic">fkp1&#45;1</span> possesses a T&#45;DNA insertion 550 bp upstream of the initiation codon.  RT&#45;PCR and promoter analyses revealed that the T&#45;DNA insertion results in knockdown of <span style="font-style: italic">FKP1</span> predominantly in tapetum.  The tapetal cells contain two classes of specific organelles, elaioplasts and tapetosomes.  The elaioplasts are plastids that accumulate sterol esters, whereas the tapetosomes derived from endoplasmic reticulum show a structural similarity to seed oil bodies, which contain vast amount of triglycerides.  These organelles are partially degraded immediately after the rupture of tapetal cells and become components of pollen coats.  Electron microscopy showed that the <span style="font-style: italic">fkp1&#45;1</span> mutation affected the development of both organelles, which caused the coatless phenotypes of <span style="font-style: italic">fkp1&#45;1</span> pollen grains.  These results suggest that not only the elaioplasts but also the tapetosomes, which contain less&#45;amount of sterols, require the mevalonate pathway for development.  In order to visualize the development of these organelles, we constructed a GFP&#45;fusion of FBP1, an elaioplast&#45;localized fibrillin&#45;like protein, and a GFP&#45;fusion of GRP17, a tapetosome&#45;specific oleosin protein.  When we expressed these proteins under their own promoters, many fluorescent particles are observed in developing tapetal cells.  Then only the GRP17&#45;fused GFP was deposited on the surface of pollen grains, which is consistent with the data from pollen coat proteome.  We concluded that the GFP fluorescence <span style="font-style: italic">in planta</span> reflects the development and degradation of two organelles as well as the fates of two proteins.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The <span style="font-style: italic">TEMPRANILLO</span> genes of the <span style="font-style: italic">Arabidopsis</span> RAV family act in the photoperiod pathway to directly repress <span style="font-style: italic">FT</span> expression</div>

<div>

	<span style="text-decoration: underline" >
	Cristina Castillejo</span>
	<sup>1</sup>, 

	<span  >
	Soraya Pelaz</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Laboratori de Genetica Molecular Vegetal, CSIC-IRTA. Barcelona, Spain, 

	<sup>2</sup>ICREA. Barcelona, Spain

</div>

<div>Floral induction is probably the most important process in plant development since it takes to the formation of the reproductive structures and, therefore, the species perpetuation. In the <span style="font-style: italic">Arabidopsis</span> photoperiod depending pathway, the first step of the floral induction is the activation of <span style="font-style: italic">FT</span> by CO. Activation of <span style="font-style: italic">FT</span> occurs specifically in the vascular bundles of leaves so it must travel to the shoot apex where flowering occurs. Once there, FT interacts with FD to activate the expression of the floral meristem identity genes, such as AP1.&#13;<br/>Our results suggest that two partially redundant transcription factors of the RAV family, TEMPRANILLO1 &#40;TEM1&#41; and TEMPRANILLO2 &#40;TEM2&#41; are involved in the regulation of <span style="font-style: italic">FT</span>. Silencing of these genes results in an early flowering phenotype and their constitutive expression extremely delays flowering as a consequence of <span style="font-style: italic">FT</span> repression. Moreover, the constitutive expression of <span style="font-style: italic">FT</span> completely rescues the <span style="font-style: italic">35S::TEM1</span> late flowering phenotype. In addition, the consensus RAV binding sequence is present in the FT 5&#8217;UTR and we found that TEM1 protein binds this sequence in vitro. These observations strongly suggest that TEMPRANILLO is a direct repressor of <span style="font-style: italic">FT</span> transcription. Furthermore, <span style="font-style: italic">FT</span> repression by TEM1 occurs both in the vascular bundles and in the shoot apex as revealed by the late flowering phenotype obtained when expressing <span style="font-style: italic">TEM1</span> under the control of specific phloem or shoot meristem promoters.&#13;<br/>Accordingly to the role of TEM in repressing <span style="font-style: italic">FT</span>, the expression of <span style="font-style: italic">TEM</span> and <span style="font-style: italic">FT</span> genes along development is antagonistic. Besides, <span style="font-style: italic">TEM</span> gene expression is regulated, like <span style="font-style: italic">CO</span>, by light and the circadian clock. We propose that TEM may be acting in the photoperiod pathway in parallel with CO to regulate <span style="font-style: italic">FT</span> and that the balance between CO and TEM activities would allow a sharp control of flowering time. In agreement with this, the progeny of the cross between early flowering <span style="font-style: italic">35S::CO</span> plants and the very late flowering <span style="font-style: italic">35S::TEM1</span> plants shows a wild type flowering time.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">HD9</span> homeobox gene is involved in Arabidopsis fruit development</div>

<div>

	<span style="text-decoration: underline" >
	Maida Romera-Branchat</span>
	<sup>1</sup>, 

	<span  >
	Soraya Pelaz</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Laboratori de Genetica Molecular Vegetal, CSIC-IRTA. Barcelona, Spain, 

	<sup>2</sup>ICREA. Barcelona, Spain

</div>

<div><span style="font-style: italic">Arabidopsis</span> fruits consist of a seedpod that encloses and protects the seeds as they mature and disperse them at maturity. The process that facilitates the dispersal of the fruit is known as dehiscence and depends on three different tissues: valve, replum and valve margin. The valves, or seed pod walls, encircle the developing seeds and join to the replum through a specialized stripe of cells that define the valve margin &#40;VM&#41;. Dehiscence occurs at the VM where the valves will separate from the replum. VM are divided into two layers, the separation layer at the replum side that permits the detachment of the valve from the replum through cell&#45;cell separation, and the lignified layer at the valve side of the margin that provides together with the lignified valve cells the necessary tension that helps the valves to detach. In the recent years, many factors involved in the pod&#45;shatter have been identified, but still some gaps exist in the complex genetic network that controls fruit development.&#13;<br/>In this line of work, we are studying a new homeobox transcription factor, <span style="font-style: italic">HD9</span>. The characterization of mutant lines and overexpressing plants suggests a role of <span style="font-style: italic">HD9</span> in the lignification pattern of the valve margin and a putative function in valve development. The <span style="font-style: italic">35S::HD9</span> fruits are small and show weak lignification and a wide replum whereas <span style="font-style: italic">hd9</span> mutant fruits display a narrow replum, thin valves and a not well defined valve margin. It will be interesting to determine the exact location of HD9 within the genetic network of fruit patterning, and for that, different crosses of <span style="font-style: italic">hd9</span> mutants and <span style="font-style: italic">35S::HD9</span> plants with mutants in known genes involved in fruit development are being performed. The unraveling of <span style="font-style: italic">HD9</span> function will give us further insight in the understanding of the complex process of fruit development, and should provide useful tools to improve important crop plants.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Structure and function of cell wall pectic arabinan and galactan during &#34;Arabidopsis thaliana&#34; seedling growth and development</div>

<div>

	<span style="text-decoration: underline" >
	Yves Verhertbruggen</span>
	<sup>1</sup>, 

	<span  >
	Susan E. Marcus</span>
	<sup>1</sup>, 

	<span  >
	J. Paul Knox</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Leeds, LS2 9JT, United Kingdom

</div>

<div>Plant cell walls are key cellular components underpinning plant growth and development. They are also highly complex structures with a great diversity of components and architectures across species. Cell wall polysaccharides occur in three major classes: cellulose, cross&#45;linking glycans &#40;or hemicelluloses&#41; and pectins. Pectins are a highly structurally diverse class that includes polymer domains such as homogalacturonans &#40;HG&#41;, rhamnogalacturonans &#40;RG&#41;, xylogalacturonans &#40;XGs&#41; and arabinogalactans. Although studies have suggested roles for pectic rhamnogalacturonan&#45;I &#40;RGI&#41; and its associated arabinan and galactan domains, in cell division, cell expansion, cell differentiation and in modifying the mechanical properties of plant cell walls, it is still unclear how the structures of these polymers is regulated &#34;in vivo&#34; and how they function during cell development. We have developed a series of monoclonal antibody probes to &#40;1&#8594;4&#41;&#45;beta&#45;galactan and &#40;1&#8594;5&#41;&#45;alpha&#45;arabinans that are structural features of RGI and have used these to track polymer occurrence in shoot organs and at the root surface of &#34;A. thaliana&#34; where they show dynamics in relation to root growth. Arabinans are shown to be a structurally complex class of polymers which displays extensive regulation in relation to cell development. At the surface of &#34;A. thaliana&#34; seedling roots, the occurrences of RGI galactan and arabinan epitopes are differently modulated in response to environmental, hormonal and other factors that reduce root growth including light, auxin and inhibitors of reactive oxygen species &#40;ROS&#41;. These antibody probes are being used for the analysis of RGI structures in response to mutations that impact upon cell walls and development in &#34;Arabidopsis&#34;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The MALE STERILITY1 protein regulates tapetal gene expression and pollen wall development</div>

<div>

	<span style="text-decoration: underline" >
	Caiyun Yang</span>
	<sup>1</sup>, 

	<span  >
	Jianping Lu</span>
	<sup>1</sup>, 

	<span  >
	Gema Vizcay Barrena</span>
	<sup>1</sup>, 

	<span  >
	Zoe A Wilson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Sciences Division, University of Nottingham, Lougborough, LE12 5RD, UK

</div>

<div>Pollen development is a fundamental stage in the higher plant life cycle that is pivotal for breeding and seed production. The control of pollen viability and release is of major commercial importance in the development of crops for hybrid seed production and selective breeding. One of the key tissues involved in pollen development is a specialised cell layer in the anther, the tapetum. The tapetum is metabolically very active with major secretory roles, including nutrition for the developing microspores, regulating callose breakdown and controlling pollen wall development. It has a highly regulated lifecycle with a close, reciprocal interaction with the developing microspores. Disruption of the tapetum is frequently seen in many male sterile plants.<br/><br/>We have shown that the Arabidopsis MS1 gene is critical to tapetal development&#47;tapetal programmed cell death &#40;Vizcay&#45;Barrena and Wilson, 2006&#41; and pollen wall formation. Orthologues of MS1 and the related genes linked to the MS1 complex have been identified in oilseed rape and rice indicating the conservation of this molecular pathway. The MS1 protein contains a leucine zipper followed by a C&#45; terminal PHD&#45;finger motif &#40;Wilson et al., 2001&#41;. PHD&#45;fingers are highly conserved motifs which have been associated with chromatin remodelling and gene regulation &#40;Asaland et al., 1995&#41;. We have shown that MS1 is nuclear localised and has significant effects on transcription during anther development. Microarray analysis of ms1 mutant buds indicates major gene changes compared to wt, particularly in genes associated with pollen wall development. RT&#45;PCR has shown that some of these are reciprocally regulated in MS1 over&#45;expressing lines &#40;35S::MS1&#41;. These data support the role of MS1 as a key transcription regulator of tapetal gene expression, particularly those associated with pollen wall development. The role of MS1 in tapetal development and pollen formation will be discussed.<br/><br/>References<br/>Asaland, R., et al. &#40;1995&#41;. Trends in Biotechnology 20, 56&#45;59.<br/>Vizcay&#45;Barrena, G., and Wilson, Z.A. &#40;2006&#41;. Journal Exp Bot 57, 2709&#45;2717.<br/>Wilson, Z.A., et al. &#40;2001&#41;. Plant Journal 28, 27&#45;39.</div>

</div>





<div class="abs">

<div style="font-weight:bold">SCL15 regulates floral transition through an autonomous&#45;independent pathway</div>

<div>

	<span style="text-decoration: underline" >
	Ming-Jun Gao</span>
	<sup>1</sup>, 

	<span  >
	Abdelali Hannoufa</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Molecular Genetics Section, Agriculture and Agri-Food Canada, Saskatoon Research Centre, 107 Science Place, Saskatoon, SK S7N 0X2, Canada

</div>

<div>The shoot apical meristem &#40;SAM&#41; gives rise to the aboveground portion of the plant, and the maintenance of SAM requires key signaling factors including WUSCHEL &#40;WUS&#41; and CLAVATA &#40;CLVs&#41;. Floral meristem &#40;FM&#41; is formed on the flanks of the SAM and produces flowers after the plant has been induced to transit from vegetative to reproductive development. The WUS&#47;CLV signaling pathway maintains the stem cells in the FM, and transition to flower development requires the activity of FM identity genes such as &#34;LEAFY&#34; &#40;&#34;LFY&#34;&#41; and &#34;APETALA1&#34; &#40;&#34;AP1&#34;&#41; and their antagonistic interaction with shoot identity gene &#34;TERMINAL FLOWER 1&#34; &#40;&#34;TER1&#34;&#41;. Compared to the mode of action of homeotic ABC genes, little is known about upstream events leading to the regulation of the SAM and FAM identity genes. We have isolated a &#34;Brassica napus&#34; auxin responsive SCARECROW&#45;like transcriptional activator, BnSCL1, based on its interaction with the histone deacetylase HDA19 &#40;Gao &#34;et al&#34;., 2004, Plant Mol Biol, 55: 417&#41;. Here, we report that SCL15, an ortholog of BnSCL1, regulates floral transition through an autonomous&#45;independent pathway. Disruption of SCL15 results in extremely delayed flowering and down&#45;regulation of expression of the FM identity genes &#40;&#34;LFY&#34;, &#34;WUS&#34; and &#34;UFO&#34;&#41;, their upstream floral pathway integrator &#34;FT&#34;, and their downstream homeotic genes &#40;&#34;AP1&#34;, &#34;AP3&#34;, &#34;PI&#34;, &#34;AG&#34; and &#34;SEP3&#34;&#41;. However, the expression of the homeotic floral organ identity gene &#34;AP2&#34; was not affected in the &#34;SCL15&#34; mutant. We demonstrate that SCL15 acts as a recruiting factor for HDA19 to regulate expression of reporter genes through recruitment of HDA19, and this repression is accompanied by an increase in histone deacetylation in the target promoter region, as determined by chromatin immunoprecipitation. We propose that SCL15 may function in recruiting HDA19 to repress a transcriptional repressor that is required to regulate the integrator &#34;FT&#34; and to control the transition from vegetative to flower development. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic analysis of the vascular patterning of the Arabidopsis root</div>

<div>

	<span style="text-decoration: underline" >
	Annelie Carlsbecker</span>
	<sup>1</sup>, 

	<span  >
	Ove Lindgren</span>
	<sup>2</sup>, 

	<span  >
	Jan Dettmer</span>
	<sup>2</sup>, 

	<span  >
	Christina Roberts</span>
	<sup>1</sup>, 

	<span  >
	Anne Honkanen</span>
	<sup>2</sup>, 

	<span  >
	Satu Lehesranta</span>
	<sup>2</sup>, 

	<span  >
	Siripong Thitamadee</span>
	<sup>2</sup>, 

	<span  >
	Yk&#228; Helariutta</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Dept. of Evolution, Genomics and Systematics, Evolutionary Biology Centre, Uppsala University, Uppsala, Sweden, 

	<sup>2</sup>Institute of Biotechnology, University of Helsinki, Finland

</div>

<div>The vasculature of plants is of immense importance as it provides paths for the transport of water, sugars, hormones and other signalling molecules through the phloem and xylem, in addition to providing physical support to the plant through the lignified cell walls of the xylem and fibres associated with the vasculature. Despite its importance, little is known about the regulation of the development of the tissues making up the vasculature. The influence of various hormones have been emphasized by several studies, but few regulatory factors have been identified. In order to identify novel components influencing the vascular patterning of the Arabidopsis root, we have performed a genetic screen for mutants miss&#45;expressing a phloem marker gene. This screen resulted in the identification of a set of novel mutants with patterning and&#47;or cell proliferation defects specific to the stele. We named these mutants <span style="font-style: italic">distorted root vascular pattern</span> &#40;<span style="font-style: italic">dva</span>&#41;. Collectively, these mutants have short primary roots, display a lack of <span style="font-style: italic">AtSUC2</span>::GFP expression at the root tip, accompanied by a delayed and distorted phloem development. In <span style="font-style: italic">dva1</span> and several of the other mutants we detected ectopic differentiation of xylem in the pericycle along the xylem axis, a phenotype associated with loss of pericycle markers. The phenotype of <span style="font-style: italic">dva1</span>, thus, indicates that the DVA1 protein is required for normal patterning of the root vasculature. We present a detailed characterisation of the <span style="font-style: italic">dva1</span> mutant.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">Two SET&#45;domain proteins involved in Arabidopsis reproductive development</div>

<div>

	<span  >
	Tage Thorstensen</span>
	<sup>1</sup>, 

	<span  >
	Vibeke  Alm</span>
	<sup>1</sup>, 

	<span  >
	Paul E. Grini</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Reidunn B. Aalen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biosciences, University of Oslo, Oslo, Norway

</div>

<div>The pattern of methylation of lysine residues on histone tails is a part of the &#8220;histone code&#8221; involved in the regulation of eukaryotic gene expression and chromatin structure. Histone metyltransferases &#40;HMTases&#41; contain a SET domain, and in Arabidopsis more than 30 genes encoding SET&#45;domain proteins have been identified that can be divided into the evolutionarily conserved subclasses Su&#40;VAR&#41;3&#45;9, E&#40;Z&#41;, TRX and ASH1 &#40;Baumbusch et al, 2001, Nucleic Acids Res 29, 4319&#8211;4333&#41;. Here we focus on two genes in the ASH1 subclass, ASHH2 and ASHR3. Promoter:GUS constructs and in situ hybridization studies show that ASHR3 is expressed in developing anthers, especially in the tapetum layer. Overexpression of ASHR3 results in anther degeneration and complete male sterility. Protein&#45;protein interaction studies revealed an interaction between ASHR3 and the bHLH transcription factor AMS, affecting pollen development. The ASHR3 protein is localized to euchromatin, and we suggest that AMS tether ASHR3 to target genes important for microspore and stamen development. ASHH2 is known to be involved in the control of flowering time. However, ashh2 mutants show pleiotropic developmental phenotypes, e.g. loss of apical dominance, dwarfism, occasional floral homeotic transformations and severely reduced fertility. Homozygous ashh2&#47;ashh2 plants are defective in both male and female meiosis. In addition, reciprocal crosses of heterozygous ashh2 to wild type plants indicate a requirement for ASHH2 in both male and female gametogenesis. Using in vitro assays we have demonstrated histone 3 lysine 36 HMTase activity for ASHH2. Moreover, in microarray transcriptional profiles, the expression levels of a number of transcription factors are affected in the ashh2 mutant, and we are currently verifying these as putative targets for ASHH2.</div>

</div>





<div class="abs">

<div style="font-weight:bold">&#34;AtBLH6&#34; Homeobox gene affects flowering time and floral organ development</div>

<div>

	<span style="text-decoration: underline" >
	Cristina Silva</span>
	<sup>1</sup>, 

	<span  >
	Soraya Pelaz</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Laboratori de Genetica Molecular Vegetal, CSIC-IRTA. Barcelona, Spain, 

	<sup>2</sup>ICREA. Barcelona, Spain

</div>

<div>The plant hormone auxin plays a crucial role in a wide variety of plant morphogenetic and physiological responses. Proper cellular responses to auxin gradients established by polar transport underlie organ patterning. Several studies established that apical&#45;basal patterning of the gynoecium, the female reproductive organ of flowering plants, involves both auxin responses and polar auxin transport.&#13;<br/>The genes &#8220;PINOID&#8221; &#40;&#8220;PID&#8221;&#41; and &#8220;PIN&#45;FORMED1&#8221; &#40;&#8220;PIN1&#8221;&#41; are both regulators of polar auxin transport involved in the establishment of an auxin gradient along the apical&#45;basal axis of gynoecium. This auxin gradient is recognised by different transcription factors, forming a genetic network that carry out the correct gynoecium development. Among these transcription factors, &#8220;ETTIN&#8221; &#40;&#8220;ETT&#8221;&#41; seems to be crucial recognising the auxin gradient. &#8220;ETT&#8221; encodes an auxin response factor &#40;ARF&#41; that binds to auxin response elements in the promoters of early auxin response genes and regulates their expression. Mutations in the &#8220;PID&#8221;, &#8220;PIN1&#8221;, and &#8220;ETT&#8221; genes cause the development of abnormal gynoecia, in which the boundary between ovary and gynophore is more distal than in the wild type.&#13;<br/>In this work, we focused on the study of &#8220;AtBLH6&#8221;, a member of the of the BELL family. This family was defined as a sub&#45;group of the Homeobox family when the first gene was identified, &#8220;BELL1&#8221; &#40;&#8220;BEL1&#8221;&#41; gene. &#8220;BEL1&#8221; is required for the normal ovule development since mutations in this gene affect the ovule identity. The constitutive expression of &#8220;AtBLH6&#8221; results in a delay in flowering time suggesting a role of this gene in the process of floral induction. Moreover, fruits of 35S::BLH6 plants presented several developmental defects that resemble those observed in the plants carrying mutations in the &#8220;PID&#8221;, &#8220;PIN1&#8221;, and &#8220;ETT&#8221; genes. This suggests a role for &#8220;AtBLH6&#8221; in the control of gynoecium patterning through a putative involvement in the processes of auxin signalling. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Cooperation between localized auxin biosynthesis and polar transport is required for organogenesis in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Youfa Cheng</span>
	<sup>1</sup>, 

	<span  >
	Xinhua Dai</span>
	<sup>1</sup>, 

	<span  >
	Yunde Zhao</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Section of Cell and Developmental Biology, University of California San Diego

</div>

<div>Auxin is essential for many aspects of plant development.  In the past few years, much progress has been made in auxin biosynthesis, polar auxin transport, and auxin signaling. However, the underlying mechanisms by which the distinct auxin pathways are integrated to control plant development are largely unknown.  Here we analyze interactions between polar auxin transport and YUC&#45;mediated localized auxin biosynthesis during plant development using Arabidopsis as a model system.  We demonstrate that both localized auxin biosynthesis and polar auxin transport are necessary for proper auxin distribution that is required for Arabidopsis organogenesis. We suggest a mechanism that integrates localized auxin biosynthesis and polar auxin transport to control auxin dynamics and developmental processes. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Role of the MNP subunits in COP9 signalosome assembly and activity and their functional interaction with CUL3&#45;based E3 ligases</div>

<div>

	<span style="text-decoration: underline" >
	Giuliana Gusmaroli</span>
	<sup>1</sup>, 

	<span  >
	Pablo Figueroa</span>
	<sup>2</sup>, 

	<span  >
	Giovanna Serino</span>
	<sup>3</sup>, 

	<span  >
	Xing Wang Deng</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Yale University, New Haven, (CT), USA, 

	<sup>2</sup>University of Delaware, Newark, (DE), USA, 

	<sup>3</sup>Universita' La Sapienza, Roma , ITALY

</div>

<div>The COP9 signalosome is an evolutionarily conserved multisubunit protein complex involved in the regulation of several developmental processes. Among its eight subunits, CSN5 and CSN6 contain a characteristic MPN domain and, in Arabidopsis, are encoded by families of two genes. Here we report the characterization of both MPN subunits using a series of single and double mutants within each gene family. Our results indicate that, while CSN6A and CSN6B retain mostly redundant functions, CSN5A and CSN5B have functionally diversified. Loss of CSN5A triggers severe pleiotropic developmental defects, whereas the complete depletion of either of the two MPN members results in CSN disassembly and instability of several subunits, along with the complete loss of CUL1, CUL3 and CUL4 de&#45;rubylation. Furthermore, we demonstrate that CSN interact with CUL3, in addition to CUL1 and CUL4, and that the lack of CSN activity differentially affects the stability of those three cullins. In fact, while CUL1 levels remain unchanged, CUL4 is up regulated and CUL3 is strongly down regulated. Interestingly, we also show that optimal CUL3 activity is required to maintain the cellular pool of CSN5, through a post&#45;transcriptional mechanism. In fact partial loss of CUL3 activity results in CSN5 accumulation and suppression of the pleiotropic phenotype of a weak <span style="font-style: italic">csn5</span> mutant. Our data suggest the existence of a reciprocal regulation between CUL3 and CSN5 accumulation. The present study thus completes the genetic analysis of all CSN subunits, and confirms the structural interdependency between PCI and MPN subunits in functional CSN complex formation. Finally, since all the lethal <span style="font-style: italic">csn</span> null mutants lack the entire CSN complex, we were interested in establishing if cullin de&#45;rubylation is the only CSN essential function. We generated transgenic <span style="font-style: italic">csn</span> mutants carrying a fully assembled COP9 signalosome complex specifically lacking the cullin de&#45;rubylation activity. The effect of this &#34;catalytically dead&#8221; complex on <span style="font-style: italic">csn</span> mutant viability will be presented. </div>

</div>





<div class="abs">

<div style="font-weight:bold">The Interaction of LATERAL ORGAN BOUNDARIES &#40;LOB&#41; with basic&#45;Helix&#45;Loop&#45;Helix Transcription Factor bHLH048 Inhibits DNA&#45;Binding Activity</div>

<div>

	<span style="text-decoration: underline" >
	Aman Husbands</span>
	<sup>1</sup>, 

	<span  >
	Elizabeth Bell</span>
	<sup>2</sup>, 

	<span  >
	Harley Smith</span>
	<sup>1</sup>, 

	<span  >
	Patricia Springer</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>UC Riverside, Riverside, CA, USA, 

	<sup>2</sup>John Innes Centre, Norwich, UK

</div>

<div>LATERAL ORGAN BOUNDARIES &#40;LOB&#41; is the founding member of the LOB DOMAIN &#40;LBD&#41; gene family, a novel plant&#45;specific family of transcription factors. This family is defined by the conserved LOB domain, a stretch of approximately 100 aa, shared by each member of the LBD family. Originally isolated from an enhancer trap screen, the function of LOB is still unknown, although there is some data suggesting roles in boundary specification and brassinosteroid signalling. To determine whether LOB had the ability to bind DNA, a Selection and Amplification Binding &#40;SAAB&#41; assay was performed and identified the nucleotide sequence GCGGCG as a cis&#45;element bound by the LOB domain. Electrophoretic Mobility Shift Assays &#40;EMSAs&#41; confirmed that this sequence is bound by the LOB domain of LOB, and by other LBD proteins, supporting the hypothesis that this family of transcription factors binds the same cis&#45;element. A yeast&#45;2&#45;hybrid screen identified bHLH048, a member of the basic&#45;helix&#45;loop&#45;helix &#40;bHLH&#41; family of transcription factors, as an interacting partner of LOB. This bHLH is predicted to be a non&#45;DNA binding protein, but preliminary results suggest that bHLH048 homodimers recognize a G&#45;box motif, the previously characterized binding site for the bHLHs. When complexed with LOB, bHLH048 appears to reduce the affinity of LOB for the LBD binding site. Related bHLH genes have been shown to be involved in brassinosteroid signalling, and preliminary data from the Springer lab shows that LOB also regulates BR responses. Interaction of these two transcription factors at the protein level, and the concomitant inhibition of DNA&#45;binding activity, may represent a control point whereby LOB&#8217;s downstream effect on brassinosteroid signalling is modulated by bHLH048.</div>

</div>





<div class="abs">

<div style="font-weight:bold">THE IMPACT OF PUTATIVE UBIQUITIN PROTEINS ON FLOWERING TIME IN ARABIDOPSIS</div>

<div>

	<span style="text-decoration: underline" >
	Shai Yerushalmi</span>
	<sup>1</sup>, 

	<span  >
	Rachel Green</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Hebrew University, Jerusalem, Israel

</div>

<div>A vast range of organisms from cyanobacteria to plants and mammals use their endogenous circadian rhythms to anticipate the changes in daily and annual environmental conditions and adapt their physiological and developmental states accordingly. In many plants, reproductive development is regulated by a photoperiodic pathway that integrates circadian rhythms and light signals. This research aims at identifying genes that are involved in the circadian&#45;regulation of flowering. Using a bioinformatics&#45;based approach we have identified a circadian&#45;regulated gene that encodes a putative ubiquitin fusion degradation &#40;UFD1&#41; protein. In yeast and animals, UFD1 is known to form a ternary complex together with NPL4 and CDC48&#47;P97 in the cytosol. Yeast UFD1 protein is required for ubiquitin&#45;dependent protein degradation in the endoplasmic reticulum associated degradation &#40;ERAD&#41; process. The function of UFD1 protein in plants is not yet known. In Arabidopsis, this gene is part of a family that contains three other UFD1&#45;like genes that share high homology and marked here as UFL1, UFL2 and UFL3. Mutations in UFD1, UFL1 and UFL2 significantly alter flowering time in light regime of long days but have only subtle effects on flowering time in short days, suggesting that this gene may have a role in photoperiodic regulation of flowering. Double mutants of ufd1 x ufl1 flower earlier than the wild type in long days but not earlier than ufd1 and ufl1, suggesting a functional redundancy with other proteins in the UFD1 family. Northern blot analysis shows that the expression of circadian&#45;regulated genes is not altered in ufd1 or ufl1 plants, suggesting that the protein is active downstream of the circadian oscillator. Our hypothesis is that UFD1 and the UFD1&#45;like proteins may participate in the degradation of a component that promotes flowering in long days but not in short days.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A heat shock transcription factor involved in <span style="font-style: italic">Arabidopsis</span> root stem cell patterning and maintenance</div>

<div>

	<span style="text-decoration: underline" >
	Colette A. ten Hove</span>
	<sup>1</sup>, 

	<span  >
	Wouter J. de Vries</span>
	<sup>1</sup>, 

	<span  >
	Ben Scheres</span>
	<sup>1</sup>, 

	<span  >
	Renze Heidstra</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Genetics, Utrecht University, Utrecht, The Netherlands

</div>

<div>In the heart of the root meristem of the plant <span style="font-style: italic">Arabidopsis thaliana</span> resides a group of stem cells that generate all the cells that make up the root and sustain growth. These stem cells surround a group of mitotically inactive cells called the quiescent center &#40;QC&#41; that is required for stem cell maintenance. Together the QC and stem cells form a so&#45;called stem cell niche. The combinatorial action of the plant specific auxin dependent AP2 transcription factors the PLETHORA &#40;PLT&#41; genes, in parallel with the GRAS family transcription factors SHORTROOT &#40;SHR&#41; and SCARECROW &#40;SCR&#41; specify the stem cell niche. To find missing links and additional genes involved in stem cell patterning and maintenance we have performed an EMS mutagenesis screen in a double fluorescent QC&#45;marker bearing background. The <span style="font-style: italic">schizoriza</span> mutant was identified because of its disturbed fluorescence QC&#45;marker expression and positional cloning revealed that the affected gene encoded a heat stress transcription factor. SCHIZORIZA appears to be a cell fate determinant as the <span style="font-style: italic">schizoriza</span> mutant shows a loss of QC and columella stem cells and misspecification of lateral root cap&#47;epidermis, cortex&#47;endodermis stem cells leading to major defects in root pattern formation. Until now not much was known about other functions of heat stress transcription factors besides their role in the heat stress response. Here we provide evidence for a role in development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Specification of Arabidopsis floral meristem identity by repressing flowering time genes</div>

<div>

	<span style="text-decoration: underline" >
	Chang Liu</span>
	<sup>1</sup>, 

	<span  >
	Jing Zhou</span>
	<sup>1</sup>, 

	<span  >
	Keren Bracha-Drori</span>
	<sup>2</sup>, 

	<span  >
	Shaul Yalovsky</span>
	<sup>2</sup>, 

	<span  >
	Toshiro Ito</span>
	<sup>1</sup>, 

	<span  >
	Hao Yu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences and Temasek Life Sciences Laboratory, National University of Singapore, Singapore 117543, Singapore., 

	<sup>2</sup>Department of Plant Sciences, Tel Aviv University, Tel Aviv 69978, Israel

</div>

<div>Flowering plants produce floral meristems in response to intrinsic and extrinsic flowering inductive signals. In Arabidopsis, the floral meristem identity genes LEAFY &#40;LFY&#41; and APETALA1 &#40;AP1&#41; are activated to play a pivotal role in specifying floral meristems during floral transition. We show here that the emerging floral meristems require AP1 to specify partly their floral identities by repressing directly a group of flowering time genes, including SHORT VEGETATIVE PHASE &#40;SVP&#41;, AGAMOUS&#45;LIKE 24 &#40;AGL24&#41; and SUPPRESSOR OF OVEREXPRESSION OF CO1 &#40;SOC1&#41;. In wild&#45;type plants, these flowering time genes are normally down&#45;regulated in emerging floral meristems. In the absence of AP1, these genes are ectopically expressed transforming floral meristems into shoot meristems. By posttranslational activation of an AP1&#45;GR fusion protein and chromatin immunoprecipitation assays, we further demonstrate the repression of these flowering time genes by induced AP1 activity and in vivo AP1 binding to the cis&#45;regulatory regions of these genes. These findings indicate that once AP1 is activated during the floral transition, it acts partly as a master repressor in floral meristems by directly suppressing the expression of flowering time genes, thus preventing the continuation of the shoot developmental program. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Transcriptional and post&#45;transcriptional regulation in the control of flowering by photoperiod&#13;<br/>&#13;<br/>&#13;<br/></div>

<div>

	<span  >
	Franziska Turck</span>
	<sup>1</sup>, 

	<span  >
	Fabio Fornara</span>
	<sup>1</sup>, 

	<span  >
	Seonghoe Jang</span>
	<sup>1</sup>, 

	<span  >
	Virginie Marchal</span>
	<sup>1</sup>, 

	<span  >
	Coral Vincent</span>
	<sup>1</sup>, 

	<span  >
	Antonis Giakountis</span>
	<sup>1</sup>, 

	<span  >
	Maria Albani</span>
	<sup>1</sup>, 

	<span  >
	Renhou Wang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	George Coupland</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, Cologne, Germany

</div>

<div>In many plants the transition to flowering is controlled by seasonal cues such as changing day length and temperature. In Arabidopsis a circadian&#45;clock regulated pathway that promotes flowering specifically in response to the longer day lengths of spring and early summer has been described. This pathway includes the GIGANTEA &#40;GI&#41;, CONSTANS &#40;CO&#41; and FT proteins, which act in the vascular tissue of the leaves to promote synthesis or transport of a systemic signal that triggers flower development at the shoot meristem. We have studied how circadian&#45;clock regulation and acute responses to light combine to activate this pathway in response to long days. We have analyzed the mechanisms regulating the precise timing of CO and FT gene expression during the day by studying the chromatin modifications on the FT gene, natural genetic variation for circadian phase in Arabidopsis and by identifying critical transcription factors through systematic misexpression, and these areas will be described. The importance in flowering control of acute light signalling in regulating protein stability by influencing the activity of the ubiquitin ligase COP1 will also be discussed. </div>

</div>





<div class="abs">

<div style="font-weight:bold">A systems approach to understand cell cycle regulation during leaf development</div>

<div>

	<span style="text-decoration: underline" >
	Gerrit Beemster</span>
	<sup>1,2</sup>, 

	<span  >
	Frederik Coppens</span>
	<sup>1,2</sup>, 

	<span  >
	Nubia Barbosa Eloy</span>
	<sup>1,2,3</sup>, 

	<span  >
	Steffen Vanneste</span>
	<sup>1,2</sup>, 

	<span  >
	Joanna Boruc</span>
	<sup>1,2</sup>, 

	<span  >
	Roeland Merks</span>
	<sup>1,2</sup>, 

	<span  >
	Tom Beeckman</span>
	<sup>1,2</sup>, 

	<span  >
	Lieven De Veylder</span>
	<sup>1,2</sup>, 

	<span  >
	Geert De Jaeger</span>
	<sup>1,2</sup>, 

	<span  >
	Dirk Inze</span>
	<sup>1,2</sup>, 

	<span  >
	Pierre Hilson</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>VIB, Ghent, Belgium, 

	<sup>2</sup>Ghent University, Ghent, Belgium, 

	<sup>3</sup>Universidade Federal do Rio de Janeiro; Rio de Janeiro, Brazil

</div>

<div>The molecular regulation of the plant cell cycle plays a crucial role in the regulation of the growth of multi&#45;cellular organs. The sequencing of the Arabidopsis genome has enabled us to identify 61 &#8220;core&#8221; cell cycle genes &#40;including CDKs, Cyclins and directly interacting proteins  &#40;Vandepoele et al., 2002; Plant Cell 14, 903 &#45; 916&#41;. Transcriptional analysis of genes that are specifically expressed in proliferating tissues during leaf development and in the root tip implicated several hundreds of additional genes, many still fully unknown, that also appear to regulate aspects of the cell cycle process &#40;Beemster et al., 2005; Plant Physiol. 138, 734&#45;743&#41;. We have functionally analyzed the function of a selection of these genes. The results indicate that the link between cell cycle regulation and whole organ rate is complex, due to the large numbers of genes involved, their non&#45;linear regulatory relationships and the lack of knowledge about the relationship between the cell cycle regulated events cell division and endoreduplication on the one hand and cell growth on the other.<br/>To unravel the nature of the regulatory of the cell cycle in the context of organ growth regulation we are taking a systems biology approach. To this end we have cloned the promotors of 51 of the core cell cycle genes and analyzed the expression pattern of during leaf and root development. These data allowed us to develop a &#8220;phenoclustering&#8221; approach to find genes with similar expression patterns as well as regulatory promoter elements that are shared between the members of such clusters.<br/>By means of matrix yeast&#45;2&#45;hybrid analysis, tandem affinity purification and split GFP analysis we are also constructing the cell cycle interactome and localisome. <br/>To integrate this wealth of data, we have developed a combination of top&#45;down and bottom&#45;up modeling approaches that can be used to identify regulatory relationships and to test their dynamic behavior.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">SHORT ROOT</span> regulates vascular tissue patterning in <span style="font-style: italic">Arabidopsis</span> roots.</div>

<div>

	<span style="text-decoration: underline" >
	Ji-Young Lee</span>
	<sup>1</sup>, 

	<span  >
	Annelie Carlsbecker</span>
	<sup>2</sup>, 

	<span  >
	Yka Helariutta</span>
	<sup>3</sup>, 

	<span  >
	Philip Benfey</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Duke University, 

	<sup>2</sup>Uppsala University, 

	<sup>3</sup>University of Helsinki

</div>

<div>Vascular tissues in <span style="font-style: italic">Arabidopsis</span> roots develop from initial cells above the quiescent center &#40;QC&#41; and form a bilaterally symmetric structure &#40;diarch&#41; constituted with xylem, phloem, and procambium. In wild type roots, the protoxylem, two outer xylem strands characterized by their spirally patterned secondary cell wall, differentiates earlier than the metaxylem, inner xylem strands with the reticulate secondary cell wall.  SHORT ROOT &#40;SHR&#41;, a GRAS family transcription factor, that was shown to regulate the asymmetric cell division of endodermis&#47;cortex initials and maintain the stem cell niche around the QC, is transcribed in the xylem and the procambium, an undifferentiated tissue in the boundary of xylem and phloem. Its proteins subsequently move out of those tissues into the outer tissues, QC and the endodermis, to play roles as described above.  However, genome&#45;wide gene expression profiling of <span style="font-style: italic">shr</span> mutant showed significant change in the expression of genes enriched in the stele, indicating an additional role of <span style="font-style: italic">SHR</span> played in the stele. To find out whether <span style="font-style: italic">SHR</span> is also involved in the specification of cell types in the stele, a systematic analysis was performed using stele cell&#45;type specific GFP marker lines in <span style="font-style: italic">shr</span> mutant. We found that the absence of <span style="font-style: italic">SHR</span> not only affects the proliferation of stele cells but also affects the cell type specification, xylem in particular.  In <span style="font-style: italic">shr</span>, the ectopic metaxylem forms instead of the protoxylem.  This ectopic metaxylem formation was found to happen via the activation of class III HD&#45;ZIP transcription factors in <span style="font-style: italic">shr</span>&#45;2.  In particular, <span style="font-style: italic">PHABULOSA</span> &#40;<span style="font-style: italic">PHB</span>&#41; was found to be critical for the proper protoxylem patterning via <span style="font-style: italic">SHR</span> regulation. This regulation seems to occur indirectly via the regulation of SHR on microRNA 165&#47;166. </div>

</div>





<div class="abs">

<div style="font-weight:bold">ROL5: a potential regulator of reactive oxygen species during cell wall formation</div>

<div>

	<span style="text-decoration: underline" >
	Ruth Maria Leiber</span>
	<sup>1</sup>, 

	<span  >
	Christoph Ringli</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Inst. of Plant Biology, Zurich, Switzerland

</div>

<div>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Cell walls are complex structures that confer shape to plant cells and ultimately to the whole plant. The basic structure of cell walls is composed of cellulose microfibrils, which are interconnected by hemicellulose and embedded in a pectin matrix. Numerous structural proteins also play a role in interconnecting the polysaccharides of the cell wall. How the process of wall development and &#8211;expansion and thus cell growth is controlled, still remains largely elusive.&#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In the model plant Arabidopsis thaliana the proteins of the LRX gene family encodes extracellular LRR&#45;extensin proteins that are good candidates for a regulatory function during cell wall development. The LRX1 protein is expressed in root hairs and strongly influences root hair cell wall formation. As a consequence, root hairs of the lrx1 mutant are miss&#45;shaped, swell, form bulges and often burst. &#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;To investigate the developmental process in which LRX1 is involved, we have performed a suppressor screen on the lrx1 mutant. In this screen we isolated a number of rol &#40;repressor of lrx1&#41; mutants that suppress the lrx1 root hair phenotype &#40;Diet A, et. al. Plant Cell 18, 1630&#45;41&#41;. rol5 represents a new suppressor mutant of lrx1. Although the encoded ROL5 protein is of unknown function, the characterization conducted so far revealed that ROL5 is a mitochondrial protein, which is predominately expressed in the elongation zone of roots. ROL5 is homologous to the yeast protein NCS6 and partially complements the function of NCS6 in yeast, where NCS6 is implicated in cell growth and morphology. Further the characterization indicates that ROL5 is involved in a process related to reactive oxygen species &#40;ROS&#41;. Rol5 mutant plants are hypersensitive to ROS, which are produced by the cells and have been shown to be important for the cell growth process. The characterization of ROL5 will shed light on the function of this protein during cell growth and may even provide new insights into the mechanisms of growth regulation in plants.&#13;<br/>&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Comparative and functional characterization of candidate regulators of wood and fiber differentiation in &#34;Arabidopsis thaliana&#34; and &#34;Populus trichocarpa&#34;</div>

<div>

	<span style="text-decoration: underline" >
	Eryang Li</span>
	<sup>1</sup>, 

	<span  >
	Juergen Ehlting</span>
	<sup>2</sup>, 

	<span  >
	Michael Friedmann</span>
	<sup>1</sup>, 

	<span  >
	Weiya Qiang</span>
	<sup>3</sup>, 

	<span  >
	Carl J. Douglas</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Botany, University of British Columbia, Vancouver BC, Canada V6T 1Z4, 

	<sup>2</sup>Institute for Plant Molecular Biology, Centre National de la Recherche Scientifique, 67000 Strasbourg, France, 

	<sup>3</sup>School of Life Science, Lanzhou University, Gansu, P. R. China

</div>

<div>In &#34;Arabidopsis&#34;, stages of interfascicular fiber differentiation can be identified along the axis of bolting stems. In &#34;Populus&#34;, different stages of xylem differentiation can be visualized along the axis of internodes development at the shoot tip, and different stages of cellular differentiation during secondary xylem formation can be dissected after re&#45;activation of the vascular cambium following winter dormancy. Using tissue samples representing different stages of xylem and fiber differentiation in &#34;Populus&#34; and &#34;Arabidopsis&#34;, we assayed global changes in gene transcription over the course of stem and xylem development and interfascicular fiber differentiation to gain insights into the developmental and regulatory events that control these patterns of xylem and fiber differentiation in both species, and to identify genes conserved in &#34;Arabidopsis&#34; and &#34;Populus&#34; that may regulate fiber and xylem differentiation including lignin deposition. &#13;<br/>These studies employed an &#34;Arabidopsis&#34; full genome longmer microarray &#40;&#62;26,000 genes&#41; and a poplar cDNA microarray.  A subset of  &#34;Arabidopsis&#34; transcription factors &#40;TF&#41; were identified that are associated with fiber development and&#47;or secondary wall formation and lignification in the root stele and other tissues, making them targets for functional studies. &#13;<br/>We have identified putative orthologues of many of the &#34;Arabidopsis&#34; candidates in Populus genome and many of these are differentially expressed on concert with xylem development, as assayed by global expression profiling of Populus cDNA microarray. Examining the functions of these Populus genes will allow greater insight into the potential roles played by conserved genes regulating secondary wall and wood formation, and provide a novel way to increase secondary wall differentiation and lignification. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Petunia LEAFY and UFO: a switch in the switch to flowering</div>

<div>

	<span style="text-decoration: underline" >
	Elske Kusters</span>
	<sup>1</sup>, 

	<span  >
	Rob Castel</span>
	<sup>1</sup>, 

	<span  >
	Alexandra Rebocho</span>
	<sup>1</sup>, 

	<span  >
	Ronald Koes</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Vrije Universiteit, Amsterdam, The Netherlands

</div>

<div>In flowering plants such as Petunia and Arabidopsis, the inflorescence meristem &#40;IM&#41; at the apex of an inflorescence shoot consists of a group of undifferentiated cells that repeatedly generates new floral meristems &#40;FMs&#41;, the initials from which flowers develop. FMs differ from IMs in their pattern of organogenesis and the identity of the organs that are formed. Petunia has a cymose inflorescence structure; at the time that the IM converts into an FM, a new IM is formed laterally.<br/><br/>Two Petunia meristem identity &#40;MI&#41; genes that specify the floral fate of the FM have been identified: ABBERANT LEAF AND FLOWER &#40;ALF&#41; and DOUBLE TOP &#40;DOT&#41;, which are the apparent orthologs of the Arabidopsis genes LEAFY &#40;LFY&#41; and UNUSUAL FLORAL ORGANS &#40;UFO&#41;, respectively. alf and dot mutants have a strong flower&#45;to&#45;shoot phenotype; both completely lack flowers and have a bushy phenotype due to the repetitive bifurcation of the IMs. In contrast to lfy mutants, alf and dot mutants are not late&#45;flowering. In situ hybridization shows that ALF is already expressed in the vegetative phase and later in the FM and all organs of the flower. DOT expression is first visible in the developing FM and later in a pentagonic domain on the adaxial side of the emerging sepal primordia.<br/><br/>Strikingly, when ALF or LFY is overexpressed in Petunia, no phenotype is observed, whereas overexpression of either of these genes in Arabidopsis leads to early flowering and the formation of single, terminal flowers. In contrast, ectopic expression of DOT or UFO in Petunia leads to a shoot&#45;to&#45;flower phenotype, comparable to overexpression of LFY or ALF in Arabidopsis, whereas overexpression of either of these F&#45;box genes in Arabidopsis does not lead to early flowering or changes in the inflorescence architecture.<br/><br/>Apparently, Petunia and Arabidopsis have evolved different ways of floral MI specification by these key MI genes. To examine whether the difference between the two species is caused by cis&#45; or trans&#45;acting factors, ALF, DOT, LFY and UFO promoter::GUS constructs were stably transformed to both Petunia and Arabidopsis. Results shed light on the divergent evolution of MI regulation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Biochemical Characterization of SPA proteins and Their Role In the Assembly of COP1 Complexes and Repression of Photomorphogenesis</div>

<div>

	<span style="text-decoration: underline" >
	Danmeng Zhu</span>
	<sup>1,2,3</sup>, 

	<span  >
	Ute Hoecker</span>
	<sup>4</sup>, 

	<span  >
	Jae-Hoon Lee</span>
	<sup>2</sup>, 

	<span  >
	Haiyang Wang</span>
	<sup>2</sup>, 

	<span  >
	Lijia Qu</span>
	<sup>1</sup>, 

	<span  >
	Xing Wang Deng</span>
	<sup>1,2,3</sup>

</div>

<div>

	<sup>1</sup>Peking-Yale Joint Center of Plant Molecular Genetics and Agrobiotechnology,Peking University, Beijing,China, 

	<sup>2</sup>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, 

	<sup>3</sup>National Institute of Biological Sciences,Beijing, China, 

	<sup>4</sup>Department of Plant Developmental and Molecular Biology, Institute fur Entwicklungsund,Molekularbiologie der Pflanzen, Heinrich-heine-Universitaet, D-40225 Duesseldorf,Germany

</div>

<div>COP1&#40;constitutive photomorphogenesis 1&#41; is one of the most important negative regulators in repressing photomorphogenesis, which functions as an E3 ligase to target the light promoting transcription factors for degradation via 26 proteosome. Unlike COP9 signalosome&#40;CSN&#41; and CDD complex&#40;COP10, DDB1 and DET1&#41;, the configuration of the COP1 complex has not been addressed so far, although COP1 was believed to act in concert with SPA1 in a 700KD complex.  Here, we report a family of four partially redundant SPA proteins work in concert with COP1 in vivo and raise the possibility that SPA proteins are likely to be components of a family of heterogeneous COP1&#45;SPA complexes. Furthermore, We characterized the biochemical property and expression patterns of the four endogenous SPA proteins by their individual epitope&#45;tagged stable transgenic lines and member&#45;specific antibodies. Our data indicated that SPA proteins are capable of both self association and heteromeric interactions in vitro and in vivo, which possibly results in their functional redundancy in the regulation of photomorphogenesis. While their distinct function was shown on the effect of COP1 accumulation by analysis of various triple mutants of SPA proteins. Moreover, like SPA1, SPA2 to SPA4 also contribute to promote COP1 mediated HY5 degradation. Additionally, other COP&#47;DET&#47;FUS proteins are also involved in the proper accumulation of the four SPA proteins. Thus, this study gave a complete view of the four SPA proteins&#8217; expression and regulatory behavior in the light control of development at the protein level. </div>

</div>





<div class="abs">

<div style="font-weight:bold">HERMIT regulates Petunia inflorescence architecture and plays part in floral transition.</div>

<div>

	<span style="text-decoration: underline" >
	R Castel</span>
	<sup>1</sup>, 

	<span  >
	E Kusters</span>
	<sup>1</sup>, 

	<span  >
	A Rebocho</span>
	<sup>1</sup>, 

	<span  >
	I Roobeek</span>
	<sup>1</sup>, 

	<span  >
	A Procissi</span>
	<sup>1</sup>, 

	<span  >
	R Koes</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Vrije Universiteit, Amsterdam, the Netherlands

</div>

<div>All flowering plants are built of very similar organ types, but the positions and numbers of these organs differ between species. This is most obvious in the inflorescences, which are the structures that bear the flowers. In some species the inflorescence consists of a single solitary flower, whereas in other species it consists of an infinite number of flowers arranged in different branching patterns. Two major types of multi&#45;flower inflorescences are cymose &#40;Petunia&#41; and racemose &#40;Arabidopsis&#41;. The molecular mechanisms that underlie these divergent branching patterns are still largely unknown.<br/><br/>We have identified three inflorescence branching loci in Petunia: EVERGREEN &#40;EVG, WUSCHEL&#45;related homeobox&#41;, EXTRAPETALS &#40;EXP, MADS&#45;box&#41; and, the latest, HERMIT &#40;HER&#41;. evg mutants have fasciated inflorescences, resulting in a bushy phenotype with few flowers.  Both exp and her mutants lack an inflorescence meristem and the multi&#45;flower cymose has transformed into a single, solitary flower. Transposon display and revertant analysis demonstrated that the her phenotype is caused by a dTph1 transposon insertion in the third exon of a class 1 KNOTTED1&#45;like homeobox gene. Judged by RT&#45;PCR, her is considered a null mutant. Phylogenetic analysis showed that HER is closely related to SHOOTMERISTEMLESS &#40;STM&#41; of Arabidopsis. Strong stm mutants completely lack a shoot apical meristem. The Petunia genome contains at least four other KNOX class 1 genes, one of which has no clear counterpart in Arabidopsis.<br/><br/>Like STM, HER is expressed throughout all meristems, and overexpression of STM and HER in Petunia or Arabidopsis gives identical phenotypes. Moreover, at least in Petunia, HER and STM are exchangeable, as the genomic fragments of both HER and STM can complement the hermit phenotype. Thus, in contrast to Arabidopsis, Petunia shoot apical meristem development is not dependent on HER&#47;STM. Instead, Petunia needs HER&#47;STM for inflorescence meristem development. <br/><br/>Although her mutants flower at the same time as wild type Petunias, double mutants with a late flowering Petunia mutant reveal an additional role for HER in floral transition. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Constitutive CKI1 activity modulates cytokinin signaling and regulates vascular tissue development in Arabidopsis</div>

<div>

	<span  >
	Hojin Ryu</span>
	<sup>1</sup>, 

	<span  >
	Jan Hej&#225;tko</span>
	<sup>2</sup>, 

	<span  >
	Gyung-Tae Kim</span>
	<sup>3</sup>, 

	<span  >
	Romana Dobe?ov&#225;</span>
	<sup>4</sup>, 

	<span  >
	P?emysl Sou?ek</span>
	<sup>4</sup>, 

	<span  >
	Petra Borkovcov&#225;</span>
	<sup>5</sup>, 

	<span  >
	Klaus Palme</span>
	<sup>6</sup>, 

	<span  >
	B?etislav Brzobohat&#253;</span>
	<sup>7</sup>, 

	<span style="text-decoration: underline" >
	Ildoo Hwang</span>
	<sup>8</sup>

</div>

<div>

	<sup>1</sup>POSTECH,Pohang,South Korea, 

	<sup>2</sup>Masaryk University,Brno,Czech Republic, 

	<sup>3</sup>Dong-A University, Busan, Korea, 

	<sup>4</sup>Masaryk University, Brno, Czech Republic, 

	<sup>5</sup>Masaryk University, , Brno, Czech Republic, 

	<sup>6</sup>Universit&#228;t Freiburg,  Freiburg, Germany, 

	<sup>7</sup>Masaryk University,  Brno, Czech Republic, 

	<sup>8</sup>POSTHCH, Pohang, South Korea

</div>

<div>Cytokinins play essential roles in cell division and vascular tissue development. While cytokinin signals are perceived by the histidine kinase receptors AHK2, AHK3, and AHK4&#47;CRE1&#47;WOL, recent genetic analyses indicate the existence of additional cytokinin signaling pathway in Arabidopsis. Here we show that histidine kinase CKI1 constitutively activates the cytokinin signaling pathway. CKI1 induces expression of cytokinin&#45;responsive type&#45;A ARRs and ARR2 phosphorylation in cytokinin&#45;deficient and wild&#45;type protoplasts and dominant negative CKI1 allele suppresses AHKs&#45;dependent activation of ARR6. Hormone&#45;regulated CKI1 expression was found in vascular tissues, root tip and shoot apex. Ectopic expression of CKI1 causes abnormal cell division and differentiation in the vascular bundles of inflorescence stems and in axillary meristems. CKI1 RNA interference and dominant negative CKI1 plants reveal cytokinin&#45;dependent aberrations in the inflorescence vascular tissue development. These results suggest that the hormone&#45;dependent transcriptional regulation of constitutive CKI1 activity modulates cytokinin signaling and regulates vascular tissue development in Arabidopsis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Signaling of Anther Cell Fate Determination by the EMS1 Receptor&#45;like Kinase in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Dazhong Zhao</span>
	<sup>1</sup>, 

	<span  >
	Xiaodong Liu</span>
	<sup>1</sup>, 

	<span  >
	Jian Huang</span>
	<sup>1</sup>, 

	<span  >
	Amy 	 Rymaszewski</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences, University of Wisconsin-Milwaukee, Milwaukee, WI 53211, USA

</div>

<div>Sexual reproduction requires specification of cells with distinct fates in plants and animals. In flowering plants, anthers are the male parts of flowers, which bear pollen for producing sperm. A mature anther is usually a four&#45;lobed structure. Each lobe contains five types of highly specialized cell layers, which are epidermis, endothecium, middle layer, tapetum and microsporocytes &#40;pollen mother cells&#41;. Among them, microsporocytes are reproductive cells that generate pollen via meiosis. The remaining somatic cells, particularly tapetum, are required for normal development and the release of pollen. However, very little is known about the molecular mechanisms governing the determination of somatic and reproductive cell fates. The Arabidopsis mutant, excess microsporocytes1 &#40;ems1&#41;, produces excess microsporocytes and lacks tapetal cells. The fact that the number of excess microsporocytes in the mutant is close to the sum of wild&#45;type microsporocytes and tapetal cells suggests that there is a trade off between somatic and reproductive cells. The EMS1 gene encodes a leucine&#45;rich repeat receptor&#45;like protein kinase &#40;LRR&#45;RLK&#41;, and its expression is associated with the differentiation of the microsporocytes and tapetal cells, indicating that EMS1 mediates signals that control cell fate determination during anther development. Identification of other signaling molecules in the EMS1 signal pathway is in progress. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular Mechanism of KNOX Gene Repression by the Arabidopsis ASYMMETRIC LEAVES1 Complex</div>

<div>

	<span style="text-decoration: underline" >
	Mengjuan Guo</span>
	<sup>1</sup>, 

	<span  >
	Julie Thomas</span>
	<sup>1</sup>, 

	<span  >
	Galen Collins</span>
	<sup>1</sup>, 

	<span  >
	Marja Timmermans</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Cold Spring Harbor Laboratory, 1 Bungtown Road, Cold Spring Harbor, NY 11724, USA

</div>

<div>Plant shoots maintain indeterminate growth resulting from the action of a population of stem cells in the shoot apical meristem &#40;SAM&#41;. Class I knox homeobox genes are among the key factors that specify indeterminacy in the SAM. Down&#45;regulation of the knox genes is required for leaf initiation and for the establishment of determinant organs. In Arabidopsis thaliana, the latter process is mediated by the myb domain protein ASYMMETRIC LEAVES1 &#40;AS1&#41;. We found that a fusion of the AS1 non&#45;myb domain to the DNA binding domain of LEAFY, a transcriptional factor required for floral induction, blocked the activation of LEAFY targets, suggesting AS1 is a transcriptional co&#45;repressor that acts directly at the knox targets. Consistent with this hypothesis, we recently showed that AS1 is part of a cellular memory system that also includes the DNA binding protein ASYMMETRIC LEAVES2 &#40;AS2&#41;, a predicted RNA binding protein RIK, and a homologue of the chromatin&#45;remodeling protein HIRA &#40;Phelps&#45;Durr et al., 2005&#41;. Using biochemical and genetic approaches we have defined the cis&#45; and trans&#45;acting factors that mediate the recruitment of the AS1 complex to the knox targets. Using chromatin immunoprecipitation &#40;ChIP&#41;, we have detected three sites in the knox gene BREVIPEDICELLUS &#40;BP&#41; that are bound to AS1 protein complex. Two sites are in the promoter region. Our promoter:reporter constructs resulted in distinct expression patterns in leaves of wild type and as1 mutant background, which verifies the significance of the two ChIP identified sites for AS1&#45;mediated BP silencing. Data from eletrophoretic mobility shift assay indicates that the interaction between AS1 and AS2 is required for their binding to both ChIP identified BP promoter sites, and we have identified the regulatory sequences mediating this binding. Together, our data suggests a molecular framework for maintaining determinacy during organogenesis. Interaction between AS1 and AS2 facilitates their binding to the knox targets and the recruitment of the chromatin&#45;remodeling protein HIRA, which maintains determinacy perhaps by forming a stable repressive chromatin state at the loci.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cell Patterning in the Arabidopsis Root Epidermis</div>

<div>

	<span style="text-decoration: underline" >
	John Schiefelbein</span>
	<sup>1</sup>, 

	<span  >
	Su-Hwan Kwak</span>
	<sup>1</sup>, 

	<span  >
	Christa Barron</span>
	<sup>1</sup>, 

	<span  >
	Yana Panciera</span>
	<sup>1</sup>, 

	<span  >
	Christine Bernhardt</span>
	<sup>1</sup>, 

	<span  >
	Yan Lin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Michigan, Ann Arbor, MI, USA

</div>

<div>A fundamental aspect of plant development is the specification of distinct cell types. In Arabidopsis, root hair and non&#45;hair cells of the root epidermis arise in a position&#45;dependent pattern, which implies that cell&#45;cell communication events play an important role in cell specification. Cellular, molecular, and genetic approaches have been used to define and analyze genes and their corresponding proteins involved in this process.  Some of these genes &#40;e.g. GL2, TTG, WER, GL3, and EGL3&#41; encode transcription factors important for non&#45;hair cell specification, whereas others &#40;e.g. CPC, TRY, and ETC1&#41; help to specify the hair cell type.  By studying the expression and regulatory interactions between these genes, we have found that transcriptional feedback loops acting within and between adjacent cells are important in establishing the cell type pattern.  Specifically, the WER MYB&#45;type protein, the GL3&#47;EGL3  bHLH&#45;type proteins, and the TTG WD&#45;protein appear to interact in a transcriptional complex to positively regulate the GL2, CPC, TRY, and ETC1 genes.  The GL2 homeodomain transcription factor is involved in regulating genes that generate the non&#45;hair cell type.  The CPC, TRY, and ETC1 proteins are structurally&#45;related small MYB transcription factors that appear to move between cells and inhibit the formation of the WER&#45;GL3&#47;EGL3&#45;TTG complex; this negative regulation represents a type of lateral inhibition mechanism.  In another regulatory loop, the GL3&#47;EGL3 proteins negatively affect their own genes&#8217; expression and move between adjacent cells.  The position&#45;dependent pattern relies on a leucine&#45;rich&#45;repeat receptor&#45;like kinase &#40;SCRAMBLED &#40;SCM&#41;&#41;, that appears to influence WER gene expression causing an unequal distribution of the transcriptional regulators in the N and H cell positions.  Current research is focused on understanding the localization and action of the SCM receptor protein and identifying new cell&#45;fate regulators using genomic and bioinformatics methods.  These studies are likely to provide new insights into the basic mechanisms of regulatory gene networks and cell&#45;type patterning during development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">MAB4, a NPH3&#45;like protein, is involved in polar auxin transport during <span style="font-style: italic">Arabidopsis</span> organogenesis</div>

<div>

	<span style="text-decoration: underline" >
	Masahiko Furutani</span>
	<sup>1</sup>, 

	<span  >
	Takahiro Kajiwara</span>
	<sup>1</sup>, 

	<span  >
	Takehide Kato</span>
	<sup>1</sup>, 

	<span  >
	Masao Tasaka</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Graduate School of Biological Sciences, NAIST, Ikoma, Nara, Japan

</div>

<div>In higher plants, aerial organs such as leaves and flowers are formed from the shoot apical meristem &#40;SAM&#41;. Organogenesis is mediated by the localized concentration of phytohormone auxin. The local auxin accumulation is established by PINOID &#40;PID&#41;, a Ser&#47;Thr kinase, through the control of cellular localization of PIN&#45;FORMED1 &#40;PIN1&#41; functioning as an auxin efflux carrier, in the SAM. To investigate molecular mechanisms by which auxin controls organogenesis, we analyzed the <span style="font-style: italic">macchi&#45;bou 4</span> &#40;<span style="font-style: italic">mab4</span>&#41; mutant identified as a <span style="font-style: italic">pinoid</span> &#40;<span style="font-style: italic">pid</span>&#41; enhancer mutant. While <span style="font-style: italic">mab4</span> and <span style="font-style: italic">pid</span> single mutants displayed relatively mild cotyledon phenotypes, <span style="font-style: italic">pid mab4</span> double mutants completely lacked cotyledons. <span style="font-style: italic">MAB4</span> encoded a novel protein, which belongs to the NON&#45;PHOTOTROPIC HYPOCOTYL 3 &#40;NPH3&#41; family supposed to function as a signal transducer. <span style="font-style: italic">MAB4</span> mRNA was detected in the protodermal cell layer of the embryo and the meristem L1 layer at the site of organ initiation. In <span style="font-style: italic">mab4</span> embryos, the abundance of PIN1:green florescent protein &#40;GFP&#41; was severely reduced on the plasma membrane of the protodermal cell layer. In <span style="font-style: italic">pid mab4</span> embryos, PIN1&#45;GFP polarity was completely reversed in the protodermal cell layer and auxin maxima were not established in the apex. In addition, subcellular localization analyses indicated that MAB4, localized in a subpopulation of endosomes and unidentified intracellular compartments, in part overlapped with PID in cultured <span style="font-style: italic">Arabidopsis</span> cells. These results suggested that <span style="font-style: italic">MAB4</span> is involved in polar auxin transport in organogenesis in concert with <span style="font-style: italic">PID</span>. </div>

</div>





<div class="abs">

<div style="font-weight:bold">A Subtilisin&#45;like Serine Protease essential for Mucilage release from Arabidopsis Seed Coats</div>

<div>

	<span style="text-decoration: underline" >
	Carsten Rautengarten</span>
	<sup>1</sup>, 

	<span  >
	Bj&#246;rn Usadel</span>
	<sup>2</sup>, 

	<span  >
	Lutz Neumetzler</span>
	<sup>2</sup>, 

	<span  >
	Thomas Altmann</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Potsdam, 

	<sup>2</sup>Max-Planck-Institute of Molecular Plant Physiology

</div>

<div>During Arabidopsis seed development large quantities of pectinaceous mucilage are deposited into the apoplast underneath the outer cell wall. After imbibing mature seeds this mucilage expands and breaks the outer cell wall encapsulating the whole seed. Mutants carrying loss&#45;of&#45;function alleles of AtSBT1.7 are characterized by the lack of mucilage release upon hydration. AtSBT1.7 encodes one of 56 Arabidopsis thaliana subtilisin&#45;like serine proteases &#40;subtilases&#41;. Analysis of the mutant seed coat epidermis revealed no visible structural differences compared to Col&#45;0 wild&#45;type seeds. However, weakening of the outer primary cell wall using Ca2&#43; chelators triggered mucilage release from the mutant&#8217;s seed coat. In contrast to mature wild&#45;type seeds the mutant primary cell wall did not rupture at the radial walls but opened at the chalazal end of the seed and was released in one piece. This was exactly the same case in immature wild&#45;type seeds collected from green siliques 6&#45;8 DAP when treated with Ca2&#43; chelators. Extrusion and solubility but not the deposition of mucilage was affected in atsbt1.7 mutants. In atsbt1.7 mucilage the neutral monosaccharide content was increased by 25&#37; compared to Col&#45;0 seeds whereas galacturonic acid contents, representing the backbone of mucilage, remained unchanged. AtSBT1.7 is preferentially expressed in the seed coat of developing seeds as revealed by RT&#45;PCR and promoter activity pattern analysis. This is consistent with its role in testa development. Moreover, atsbt1.7 mutants were strongly impaired in germination under water limiting conditions. Therefore, it is evident that AtSBT1.7 is indirectly involved in the rapid rupture of the outer wall of seeds for mucilage release upon imbibition. AtSBT1.7 may act via the accumulation and&#47;or activation of cell wall modifying enzymes necessary either for the loosening of the outer primary cell wall or to facilitate swelling of the outer mucilage.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Control of <span style="font-style: italic">AHP6</span>, a central player of vascular development in the <span style="font-style: italic">Arabidopsis</span> root</div>

<div>

	<span style="text-decoration: underline" >
	Ana Campilho</span>
	<sup>1</sup>, 

	<span  >
	Anthony Bishopp</span>
	<sup>1</sup>, 

	<span  >
	Hanna Help</span>
	<sup>1</sup>, 

	<span  >
	Yk&#228; Helariutta</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Biotechnology, University of Helsinki, Finland

</div>

<div>During vascular development in the <span style="font-style: italic">Arabidopsis</span> root, cytokinins promote pluripotent cell as well as phloem identity and inhibit protoxylem cell identity. Protoxylem cell fate is dependent on the localised inhibition of cytokinin signalling by AHP6, a pseudo&#45;phosphotransfer protein that acts to inhibit the phosphorelay associated with cytokinin signalling. <span style="font-style: italic">AHP6</span> is expressed specifically in both protoxylem cell files. Conversely, cytokinin signalling negatively regulates the spatial domain of <span style="font-style: italic">AHP6</span>expression. Consequently, a negative regulatory feed&#45;back loop operates where cytokinin signalling counteracts expression of its inhibitor facilitating protoxylem formation. The identity of either the negative regulatory &#40;cytokinin mediated&#41; or promotive factors which converge on <span style="font-style: italic">AHP6</span> is unknown. To identify and characterize upstream factors controlling <span style="font-style: italic">AHP6</span>, a forward genetic screen was performed to identify altered expression patterns of <span style="font-style: italic">pAHP6::GFP</span> within an EMS mutagenized line. A set of novel mutants was identified and the phenotypic description of these genetically interacting loci will be presented. Further functional and molecular characterization of those loci can reveal the basic genetic mechanisms underlying vascular development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">TORMOZ Like1 is essential for orientating cell divisions during early embryogenesis in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Nai-You Liu</span>
	<sup>1</sup>, 

	<span  >
	Hong-Ju Li</span>
	<sup>1</sup>, 

	<span  >
	Jie Liu</span>
	<sup>1</sup>, 

	<span  >
	Wei-Cai Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China

</div>

<div>Precise control of orientating cell divisions is critical for early embryogenesis in Arabidopsis. Here we report the characterization of an Arabidopsis Ds&#45;insertion mutant tormoz like1 &#40;tol1&#41;, which causes the early arrest of embryo development. Aberrant cell division patterns occur in the embryo proper, but not in the suspensor. The zygote of the mutant embryo in tol1 elongates and undergoes an asymmetric cell division normally, but further divisions of the apical cell are impaired. The apical cell divides transversely or obliquely but not longitudinally as that in wild type. In addition, the majority of the mutant embryos were arrested before the 16&#45;celled dermatogen stage. TOL1 encodes a nucleolus localized U3 SnoRNP&#45;associated WD&#45;40 protein which is highly conserved in almost all eukaryotic species. TOL1 is expressed early in the ovule primordial, the mature embryo sacs, young embryos, and tissues undergoing rapid growth including root tip and shoot apex. Genetic complementation experiment indicates that the mutant phenotype observed is indeed the result of a Ds insertion within TOL1. Together, our findings suggest an essential role of TOL1 to direct longitudinal cell divisions in Arabidopsis early embryo development. </div>

</div>





<div class="abs">

<div style="font-weight:bold">'ALCATRAZ&#45;INTERACTING PROTEIN1' encodes a lysine rich nuclear protein and is expressed ubiquitously in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Fang Wang</span>
	<sup>1</sup>, 

	<span  >
	Jie Liu</span>
	<sup>1</sup>, 

	<span  >
	Weicai Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China

</div>

<div>bHLH transcription factors often bind to specific DNA target sites as either homodimer or heterodimer. ALCATRAZ &#40;ALC&#41; is a member of the bHLH family which is critical for cell separation during fruit dehiscence &#40;Rajani and Sundaresan, 2001, Current Biology 11:1914&#45;1922&#41;. To investigate the molecular mechanism of ALC in regulating pod shattering and other developmental processes, a yeast two&#45;hybrid screen was performed using ALC as bait for Arabidopsis whole plant cDNA library. A gene named as AIP1 &#40;ALC Interacting Protein1&#41; was identified. AIP1 encodes a novel nuclear protein with a lysine rich domain.&#13;<br/>Deletion experiments showed that both the Lys&#45;rich domain and the C&#45;terminal of AIP1 are able to interact with ALC in yeast cells, but the latter has lower affinity, indicating that AIP1 interacts with ALC preferentially via its Lys&#45;rich domain. The in vivo interaction of ALC and AIP1 was further confirmed by bimolecular fluorescence complementation &#40;BIFC&#41;. RT&#45;PCR analysis and PAIP1::GUS reporter assay indicated that AIP1 and ALC have overlapping expression pattern, suggesting they likely function together in planta. RNA in situ hybridization analysis showed AIP1 transcripts were detected strongly in the mesocarp of the valve, not in epidermis and the separation zones. Later in development, AIP1 transcripts were mainly detected in vasculature. The investigation on the function of AIP1 in plant development is underway.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A semi&#45;dominant mutation in a ribosomal protein L10 gene suppresses the dwarf phenotype of the <span style="font-style: italic">acl5</span> mutant.</div>

<div>

	<span  >
	Akihiro Imai</span>
	<sup>1,2</sup>, 

	<span  >
	Mio Komura</span>
	<sup>1</sup>, 

	<span  >
	Eri Kawano</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Taku Takahashi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Okayama University, Okayama, Japan, 

	<sup>2</sup>National Institute for Basic Biology, Okazaki, Japan

</div>

<div>Disruption of the <span style="font-style: italic">Arabidopsis thaliana ACAULIS5 &#40;ACL5&#41;</span> gene, which encodes spermine synthase, results in a severely dwarfed phenotype. A previous study showed that <span style="font-style: italic">sac51&#45;d</span>, a dominant extragenic suppressor mutant of <span style="font-style: italic">acl5&#45;1</span>, has premature termination codon in an upstream open reading frame &#40;uORF&#41; of <span style="font-style: italic">SAC51</span>, which encodes a putative transcription factor, and suggested the involvement of its uORF&#45;dependent translational control in the <span style="font-style: italic">ACL5</span>&#45;mediated stem growth. Here we report the identification of a gene responsible for the second suppressor mutant of <span style="font-style: italic">acl5&#45;1, sac52&#45;d. SAC52</span> encodes a ribosomal protein L10 &#40;RPL10A&#41;, which is highly conserved among eukaryotes and thought to be a translational regulator. Transgenic insertion of a genomic fragment containing the semi&#45;dominant <span style="font-style: italic">sac52&#45;d</span> allele into <span style="font-style: italic">acl5&#45;1</span> mutants rescued the dwarf phenotype of <span style="font-style: italic">acl5&#45;1</span>. GUS reporter activity under the control of the SAC51 promoter with its uORFs was higher in <span style="font-style: italic">acl5&#45;1 sac52&#45;d</span> than that in <span style="font-style: italic">acl5&#45;1</span>, suggesting that the suppression of the <span style="font-style: italic">acl5&#45;1</span> phenotype by <span style="font-style: italic">sac52&#45;d</span> is attributable, at least in part, to the enhanced translation of <span style="font-style: italic">SAC51</span>. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Getting to the root of cell identity</div>

<div>

	<span style="text-decoration: underline" >
	Philip Benfey</span>
	<sup>1</sup>, 

	<span  >
	Siobhan Brady</span>
	<sup>1</sup>, 

	<span  >
	Hongchang Cui</span>
	<sup>1</sup>, 

	<span  >
	David Orlando</span>
	<sup>1</sup>, 

	<span  >
	Jose Dinneny</span>
	<sup>1</sup>, 

	<span  >
	Terri Long</span>
	<sup>1</sup>, 

	<span  >
	Twigg Richard</span>
	<sup>1</sup>, 

	<span  >
	Daniel Mace</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Duke University

</div>

<div>Central processes in development include creating distinctions between cells and producing coordination among different cells so that they function as units.While signaling and transcription are equally important for development, high throughput techniques for identifying the nodes and links in transcriptional networks have matured more rapidly. For plants the simplifying aspects of development in an organ such as the root, make it highly tractable for the application of these approaches. In an effort to identify the transcriptional networks that regulate cell identity in the root we are generating three datasets: 1&#41; global expression profiles; 2&#41; cellular localization of transcription factors; and 3&#41; transcription factor targets. To understand the role of transcriptional networks in development, each of these datasets needs to be at cell&#45;type specific resolution. To acquire global expression profiles we developed technology that uses sorted marked populations of cells with subsequent hybridization of the labeled RNA to microarrays. Employing this methodology, we now have data that cover most of the cell types in the root. We have also developed methods for identifying expression profiles along the developmental axis of the root. Combining these datasets gives us spatial and temporal expression patterns at high resolution. Analysis of these data provides insight into the regulatory basis of cell identity under standard growth conditions.  <br/>To investigate the response to environmental stimuli at the cellular level, we are profiling expression after cell sorting in response to a number of stresses. Many of the genes that are induced or repressed in response to these stresses in a tissue&#45;specific manner have not been previously characterized. We are also developing technology that will allow us to follow the effects of stimuli on hundreds of genes at cell&#45;type specific resolution over time. Initial results suggest that there is a core set of functions for each cell type overlaid with a set of cell&#45;type specific responses to different stimuli. <br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br/>This work is supported by grants from the NIH, NSF and DARPA. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Endoreduplication during leaf and root development is differentially regulated by the Anaphase&#45;Promoting Complex Activators CCS52A1 and CCS52A2 in Arabidopsis thaliana.</div>

<div>

	<span style="text-decoration: underline" >
	Marleen Vanstraelen</span>
	<sup>1</sup>, 

	<span  >
	Sylvie Tarayre</span>
	<sup>1</sup>, 

	<span  >
	Mikhail Baloban</span>
	<sup>1</sup>, 

	<span  >
	Eva Kondorosi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Science Institute (ISV)/CNRS, Gif-sur-Yvette, France

</div>

<div>Plant organs develop mostly post&#45;embryonically from persistent or newly formed meristems. After cell division arrests, cell differentiation starts which frequently involves endoreduplication and cell enlargement. CCS52A was previously shown to function in the transition from mitotis&#45;to&#45;endocycles during nodule development in M. truncatula. Arabidopsis has two CCS52A genes, CCS52A1 and CCS52A2. They share a high degree of homology and are expressed from late M&#45;phase to S&#45;phase in cell cultures. In plants, we found that the CCS52A genes are expressed in young differentiating tissues. CCS52A promoter&#45;driven &#61538;&#45;glucuronidase activity showed coloration in young developing leafs and in case of CCS52A1 staining was also observed near the root tip. In these tissues, we studied DNA ploidy levels in plants mutant for the CCS52A genes. Deletion of either CCS52A genes led to less endoreduplication cycles and small cell sizes in rozette leafs. In case of ccs52a1, normal leaf size was obtained by elevated cell numbers. Constitutive overexpression of the CCS52A genes triggered the opposite effect with higher ploidy levels and increased cell sizes. Reduced ploidy levels were also observed in roots of ccs52a1 plants. These roots grew faster than wild type roots due to bigger meristems with more dividing cells. Deletion of the CCS52A2 gene did not affect endoreduplication cycles in roots. However, roots of these plants were smaller with smaller meristems and less dividing cells. Together the data show that both CCS52A1 and CCS52A2 function in the transition from mitosis&#45;to&#45;endocycle during leaf development, whereas only CCS52A1 contributes to endoreduplication during root development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Investigation of HOX&#45;Domain Containing Transcription Factors of Arabidopsis and Cotton Involved in Trichome Development</div>

<div>

	<span style="text-decoration: underline" >
	Xue-Ying Guan</span>
	<sup>1</sup>, 

	<span  >
	Na Yu</span>
	<sup>1</sup>, 

	<span  >
	Xiao-Xia Shanggua</span>
	<sup>1</sup>, 

	<span  >
	Xiao-Ya Chen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Physiology and Ecology, Shanghai, China

</div>

<div>Arabidopsis trichomes are single&#45;celled epidermal hairs. Trichomes provide a valuable system for investigation of cell differentiation and cell&#45;to&#45;cell communication in plants. Several types of transcription factors that are involved in regulating trichome initiation and development have been identified, including a MYB protein GL1 &#40;AT3G27920&#41;, a bHLH protein GL3 &#40;AT5G41315&#41;, a WD40 domain&#45;containing protein TTG1 &#40;AT5G24520&#41;, and a HD&#45;ZIP III protein GL2 &#40;AT1G79840&#41;. Another three closely&#45;related MYB like proteins, TRY &#40;AT5G53200&#41;, CPC &#40;AT2G46410&#41; and ETC1 &#40;AT1G01380&#41;, are known as negative factors to suppress trichome formation in the neighbor cells. Cotton fibers are trichomes of Gossypium ovules &#40;seeds&#41;. Previously, we isolated a MYB gene, GaMYB2 &#40;AY626160&#41;, from G. arboreum. GaMYB2 is highly expressed in fiber cells at early developmental stages; when expressed in gl1 mutant plants of Arabidopsis, it rescued trichome development to a wild&#45;type level. We further analyzed three GL2&#45;like genes of cotton &#40;G. arboreum&#41;, namely GaHOX1, GaHOX2 and GaHOX3. These three genes show different expression patterns in cotton tissues, and they could rescue the Arabidopsis gl2 mutant phenotype either completely or in certain conditions. We found that the HD&#45;ZIP III proteins interact with each other to exert their functional activity.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The <span style="font-style: italic">Arabidopsis</span> gene <span style="font-style: italic">REVOLUTA</span> is required to function together with the 26S proteasome subunit genes for floral organ initiation and floral meristem maintenance</div>

<div>

	<span  >
	Zhenzhen Zhang</span>
	<sup>1</sup>, 

	<span  >
	Qihua Ling</span>
	<sup>1</sup>, 

	<span  >
	Limin Pi</span>
	<sup>1</sup>, 

	<span  >
	Hua Wang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Hai Huang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Shanghai Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences

</div>

<div>During flower development, cells in the central part of a flower primordium initially constitute the floral meristem, from which four whorls of floral organs develop sequentially. Here we report that the <span style="font-style: italic">REVOLUTA</span> &#40;<span style="font-style: italic">REV</span>&#41; and the 26S proteasome subunit genes play important roles in floral organ initiation. <span style="font-style: italic">REV</span> belongs to the plant&#45;specific HD&#45;ZIPIII gene family, participating in a variety of developmental processes. We previously demonstrated that the double mutant combination of <span style="font-style: italic">rev</span> and <span style="font-style: italic">ae3</span>, which carries a defective 26S proteasome subunit RPN8a, resulted in plants with a complete arrest of meristematic growth. We show in this study that plants carrying <span style="font-style: italic">rev</span> and <span style="font-style: italic">ae4</span>, which harbors another disrupted 26S proteasome subunit RPN2a, displayed some similar phenotypes to those of <span style="font-style: italic">rev ae3</span>, but its meristem defects appeared weaker. The reduced severity of the <span style="font-style: italic">rev ae4</span> double mutant allowed us to characterize its abnormalities in the reproductive growth in more detail. <span style="font-style: italic">rev ae4</span> has an abnormal inflorescence structure, with a slowly developing manner. While a few early flowers had their seeds set, the later flowers lacked the inner whorl floral organs, stamens and gynoecia, resembling those of the loss of <span style="font-style: italic">WUSCHEL</span> &#40;<span style="font-style: italic">WUS</span>&#41; function plants. During later reproductive stages, the inflorescence meristem in the double mutant produced filamentous structures, instead of flowers, around the reproductive apex. In situ hybridization experiments revealed that <span style="font-style: italic">WUS</span> expression was repressed in floral primordia of the rev ae4 double mutant, whereas <span style="font-style: italic">SHOOTMERISTEMLESS</span> and <span style="font-style: italic">APETALA1</span> appeared normally expressed despite the lack of any apparent floral organ development. These results suggest that <span style="font-style: italic">REV</span> and the 26S proteasome play a critical role in flower development by promoting floral organ initiation and sustaining meristematic capacity of the floral primordia.</div>

</div>





<div class="abs">

<div style="font-weight:bold">ASYMMETRIC LEAVES1&#47;2 and JAGGED are required for sepal and petal organogenesis by defining organs from their boundaries</div>

<div>

	<span style="text-decoration: underline" >
	Ben Xu</span>
	<sup>1</sup>, 

	<span  >
	Ziyu Li</span>
	<sup>2</sup>, 

	<span  >
	Yan Zhu</span>
	<sup>2</sup>, 

	<span  >
	Hua Wang</span>
	<sup>1</sup>, 

	<span  >
	Hong Ma</span>
	<sup>3</sup>, 

	<span  >
	Aiwu Dong</span>
	<sup>2</sup>, 

	<span  >
	Hai Huang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Laboratory of Plant Molecular Genetics, Shanghai Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, 300 Fenglin Road, Shanghai 200032, China, 

	<sup>2</sup>State Key Laboratory of Genetic Engineering, Department of Biochemistry, School of Life Sciences, Fudan University, Shanghai 200433, China, 

	<sup>3</sup>Department of Biology and the Huck Institute for the Life Sciences, The Pennsylvania State University, University Park, PA 16802, USA

</div>

<div>Boundary formation is crucial for organ development in multicellular eukaryotes.  In higher plants, boundaries between meristem and organ or between organs are regions that are slower in cell proliferation than their adjacent tissues.  In addition, boundaries coincide with the action domains of a set of boundary&#45;specifying genes.  Here we show that the Arabidopsis genes ASYMMETRIC LEAVES1, 2 &#40;AS1 and AS2&#41; and JAGGED &#40;JAG&#41; are critical for sepal and petal organogenesis by suppressing genes involved in boundary formation in the initiated perianth.   Loss&#45;of&#45;function as1&#47;as2 jag double mutants produced extremely tiny sepals and petals due to a disrupted cell proliferation.  These abnormal phenotypes are accompanied by ectopic expression of the boundary&#45;specifying genes PETAL LOSS &#40;PTL&#41;, CUP&#45;SHAPED COTYLEDONS1 and 2 &#40;CUC1 and CUC2&#41; in the organs.  Together, our results reveal a previously unrecognized fundamental regulation, by which AS1&#47;AS2&#47;JAG act to define organs from their surrounding boundaries through suppressing boundary genes for normal organogenesis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The <span style="font-style: italic">CAPRICE&#45;like Myb3 &#40;CPL3&#41;</span> is involved in the determination of epidermal cell fate in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Rumi Tominaga</span>
	<sup>1</sup>, 

	<span  >
	Mineko Iwata</span>
	<sup>1</sup>, 

	<span  >
	Ryosuke Sano</span>
	<sup>1</sup>, 

	<span  >
	Kiyotaka Okada</span>
	<sup>1</sup>, 

	<span  >
	Takuji Wada</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN, Yokohama, Kanagawa, Japan

</div>

<div><span style="font-style: italic">CAPRICE</span> &#40;<span style="font-style: italic">CPC</span>&#41; encodes a small protein with a R3 MYB motif and promotes root&#45;hair cell differentiation in <span style="font-style: italic">Arabidopsis</span> &#40;Wada et al., 1997&#41;. We showed that the CPC protein moves from hairless cells to neighboring hair&#45;forming cells and represses the expression of the homeodomain&#45;leucine zipper gene <span style="font-style: italic">GLABRA2</span>  &#40;<span style="font-style: italic">GL2</span>&#41; &#40;Wada et al., 2002&#41;. We found four additional <span style="font-style: italic">CPC</span>&#45;like Myb sequences including <span style="font-style: italic">TRIPTYCHON</span> &#40;Schellmann et al., 2002&#41; in <span style="font-style: italic">Arabidopsis</span>. Plant lines that overexpress these four genes had a reduction in the number of trichomes and an increase in the number of root&#45;hairs, which is similar to a plant with an additional copy of <span style="font-style: italic">CPC</span> &#40;<span style="font-style: italic">35S:CPC</span>&#41;. We also made double and triple mutants to clarify the function of each gene. These results suggest that the <span style="font-style: italic">CPC</span>&#45;like R3 MYB genes cooperatively regulate epidermal cell differentiation, particularly in structures that arise from epidermal cell proliferation. Unlike CPC, CPC&#45;like Myb GFP fusion proteins did not move from cell to cell. Promoter&#45;GUS analyses indicated that these <span style="font-style: italic">CPC</span> homologs were specifically expressed in epidermal cells, including stomate guard cells. Notably, the <span style="font-style: italic">CAPRICE&#45;like Myb3</span> &#40;<span style="font-style: italic">CPL3</span>&#41; gene alone has pleiotropic effects on stomate guard cell distribution, flowering development and epidermal cell size through the regulation of endoreduplication.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Role of <span style="font-style: italic">TTG2</span> in genetic network of epidermal cell differentiation in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Tetsuya Ishida</span>
	<sup>1</sup>, 

	<span  >
	Sayoko Hattori</span>
	<sup>1</sup>, 

	<span  >
	Kiyotaka Okada</span>
	<sup>1,2</sup>, 

	<span  >
	Takuji Wada</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN PSC, Yokohama, Kanagawa, Japan, 

	<sup>2</sup>Graduate School of Science, Kyoto Univ., Kyoto, Japan

</div>

<div> Root epidermal cells differentiate into either root hair cell or hair&#45;less cell in <span style="font-style: italic">Arabidopsis</span>. In current model, a transcriptional complex consisting of WER &#40;R2R3&#45;MYB transcription factor&#41;, GL3&#47;EGL3 &#40;bHLH transcription factor&#41; and TTG1 &#40;WD40 repeat protein&#41; promotes <span style="font-style: italic">GL2</span> &#40;encoding an HD&#45;Zip transcription factor&#41; expression, which results in differentiation into hair&#45;less cell. CPC &#40;R3&#45;MYB protein&#41; inhibits formation of the WER&#45;GL3&#47;EGL3&#45;TTG1 complex to repress <span style="font-style: italic">GL2</span> expression, resulting in differentiation into root hair cell.&#13;<br/> <span style="font-style: italic">TTG2</span>, encoding a WRKY transcription factor, regulates trichome development. <span style="font-style: italic">TTG2</span> is expressed in atrichoblasts of roots as well as in trichomes. We found that <span style="font-style: italic">TTG2</span> expression in roots was directly promoted by the WER&#45;GL3&#47;EGL3&#45;TTG1 complex and was repressed by <span style="font-style: italic">CPC</span>. Transgenic Arabidopsis expressing chimeric TTG2 protein fused with transcription repressor domain &#40;TTG2:SRDX&#41; formed root hairs from all epidermal cells of roots. In <span style="font-style: italic">TTG2:SRDX</span> lines, <span style="font-style: italic">GL2</span> expression were repressed in roots, suggesting a possibility that <span style="font-style: italic">TTG2</span> promotes <span style="font-style: italic">GL2</span> expression.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Delayed&#45;fluorescence, a potential universal tool for the measurement of circadian rhythms in plants.</div>

<div>

	<span  >
	Peter Gould</span>
	<sup>1</sup>, 

	<span  >
	James Hartwell</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Anthony Hall</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Liverpool

</div>

<div>The endogenous 24 hour oscillator, the circadian clock, plays an important role in plants, enhancing performance and increasing vegetative yield. Much of our current understanding of the mechanism and function of the circadian clock in plants has come from the development of Arabidopsis thaliana as a model circadian organism. Key to this rapid progress has been the development of robust circadian markers, specifically clock driven promoter luciferase fusions. Studies of the clock in crop species and non&#45;model organisms are currently hindered by the absence of a simple high&#45;throughput universal assay for, clock function, accuracy and robustness. Delayed&#45;fluorescence &#40;DF&#41; is a fundamental process occurring in all photosynthetic organisms. It is luminescence produced post&#45;illumination as photosynthetic reaction centres decay to a ground state. Here, we report that the amount of DF oscillates with an approximately 24h period and in Arabidopsis is under the control of the circadian clock. Thus, delayed&#45;fluorescence provides a simple clock output that potentially allows the clock to be assayed in&#45;vivo in any photosynthetic organism. This simple high&#45;throughput, non&#45;transgenic assay could be integrated into crop breeding programmes, allowing the selection of plants that have robust and accurate clocks. It can also be used to rapidly characterise the role and function of Arabidopsis mutants in the circadian system.</div>

</div>





<div class="abs">

<div style="font-weight:bold">FT RNA Act as a Systemic Floral Initiator</div>

<div>

	<span  >
	Kuan-Ju Lu</span>
	<sup>1,2</sup>, 

	<span  >
	Yu-Chen Liu</span>
	<sup>1</sup>, 

	<span  >
	Nien-Chen Huang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Tien-Shin Yu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant and Microbial Biology, Academia Sinica, Taipei, Taiwan, 

	<sup>2</sup>Department of Life Science, National Central University, Taiwan

</div>

<div>The idea that FT RNA or protein may act as florigen&#45;like molecules to regulate floral initiation is one of the most exciting discoveries in recent plant biology. However, the original paper described FT RNA is a florigen has been retracted recently. Therefore, whether FT RNA is a mobile floral initiator remains to be further investigated. From classic physiological experiments, it has been established that florigen is transported through phloem translocation stream. To test whether FT RNA is localized in the phloem sap, RNA were extracted from the phloem extrudes of excised Brassica oleraceae inflorescences. RT&#45;PCR analysis indicated that the transcript of FT is detected in Broccoli phloem sap. To further examine the long&#45;distance movement of FT RNA, the Arabidopsis grafting experiments were next conducted. Our results indicated that the FT RNA moves long&#45;distance to the scion apices, in addition, the floral initiation of scion tissues are accelerated. Interestingly, the long&#45;distance movement of FT&#45;GFP chimera RNA is dramatically impaired after the insertion of GFP in the 3&#8217; end of FT, suggesting the movement of FT RNA is determined by the intact structure of RNA. Consistent with this finding, the results performed with tomato grafting indicate that tomato SFT full&#45;length RNA, an ortholog of FT, moves long&#45;distance to the scion apices. Taken together, our results support the hypothesis that FT RNA may act as a systemic floral initiator in diverse plant species.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identifying potential target proteins for degradation regulated by the <span style="font-style: italic">Hawaiian Skirt</span> gene &#40;<span style="font-style: italic">HWS</span>&#41;</div>

<div>

	<span style="text-decoration: underline" >
	Xuebin Zhang</span>
	<sup>1</sup>, 

	<span  >
	Zinnia H Gonzalez-Carranza</span>
	<sup>1</sup>, 

	<span  >
	Jeremy A Roberts</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Biological Sciences, University of Nottingham, Sutton Bonington Campus, Loughborough, United Kingdom.

</div>

<div>The ubiquitin &#40;Ub&#41;&#47; 26S proteasome pathway is probably the most studied process to degrade proteins in the cell. A hierarchical cascade of three types of enzymes are involved in the turnover of targets which include; ubiquitin&#45;activating &#40;E1&#41;, ubiquitin&#45;conjugating &#40;E2&#41; and ubiquitin ligase &#40;E3&#41; enzyme. Cells identify the proteins that must be degraded by attaching an ubiquitin chain. The canonical E3 is the complex Skp1&#45;cullin&#45;F&#45;box protein &#40;SCF&#41; and is composed of four subunits: Cullin, SKP1, RBX1 and an F&#45;box protein. &#13;<br/>Recently a novel F&#45;box protein from Arabidopsis has been identified; the <span style="font-style: italic">Hawaiian skirt</span> gene &#40;<span style="font-style: italic">HWS</span>, alias <span style="font-style: italic">HS</span>&#41; by the mapping of its mutant <span style="font-style: italic">hws&#45;1</span>; which was identified because its floral organs do not shed; the sepals remain fused at their base retaining the rest of the floral organs inside. The characterization of the mutant and comparison with over expressor lines suggest a role of the <span style="font-style: italic">HWS</span> gene in plant growth &#40;Gonz&#225;lez&#45;Carranza <span style="font-style: italic">et al</span>, in press&#41;&#13;<br/>The protein of HWS contains an F&#45;box domain located 40&#45;85 amino acids downstream of the N&#45;terminus, and a Kelch&#45;2 domain in its C&#45;terminus. To test the hypothesis that <span style="font-style: italic">HWS</span> is the component from an SCF complex and is involved in the ubiquitin proteasome pathway; an anther specific Y&#45;2&#45;H library using both a full transcript and a truncated version without the F&#45;box domain from the HWS protein as baits has been screened.&#13;<br/>Current work, including an analysis of the proteome in wild type and hws material, is focused on identifying both potential SCF<span style="font-style: italic">hws</span> components and potential substrate&#40;s&#41; for HWS degradation.&#13;<br/>&#13;<br/>&#91;1&#93; Gonzalez Carranza ZH, Rompa U, Peters JL, Bhatt AM, Wagstaff C, Stead AD, Roberts JA &#40;2007&#41; <span style="font-style: italic">HAWAIIAN SKIRT</span> &#8211; an F&#45;box gene that regulates organ fusion and growth in Arabidopsis. Plant Physiology &#40;In Press&#41;.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">&#34;Arabidopsis&#34; homologs of components of SWR1 complex regulate flowering and plant development</div>

<div>

	<span style="text-decoration: underline" >
	Kyuha Choi</span>
	<sup>1</sup>, 

	<span  >
	Chulmin Park</span>
	<sup>1</sup>, 

	<span  >
	Jungeun Lee</span>
	<sup>1</sup>, 

	<span  >
	Mijin Oh</span>
	<sup>1</sup>, 

	<span  >
	Bosl Noh</span>
	<sup>2,3</sup>

</div>

<div>

	<sup>1</sup>National Research Laboratory of Plant Developmental Genetics, Department of Biological Sciences, Seoul National University, Seoul, 151-742, Korea., 

	<sup>2</sup>Global Research Laboratory for Flowering at SNU and UW, Seoul, 151-742, Korea., 

	<sup>3</sup>Environmental Biotechnology National Core Research Center, Gyeongsang National University, Jinju 660-701, Korea.

</div>

<div>The SWR1 complex &#40;SWR1C&#41; in yeast catalyzes the replacement of nucleosomal H2A with the H2AZ variant, which ensures full activation of underlying genes. We compared the phenotype of mutants in the homologs of SWR1C components in &#34;Arabidopsis thaliana&#34;. Mutations in &#34;Arabidopsis SWC6 &#40;AtSWC6&#41;&#34;, &#34;SUPPRESSOR OF FRIGIDA3 &#40;SUF3&#41;&#34; and &#34;PHOTOPERIOD INDEPENDENT EARLY FLOWERING1 &#40;PIE1&#41;&#34;, homologs of &#34;SWC6&#34;, &#34;ARP6&#34; and &#34;SWR1&#34;, respectively, caused similar developmental defects, including leaf serration, weak apical dominance, flowers with extra petals, and early flowering by reduction in expression of &#34;FLOWERING LOCUS C &#40;FLC&#41;&#34;, a strong floral repressor. Chromatin immunoprecipitation assays showed that AtSWC6 and SUF3 bind to the proximal region of the &#34;FLC&#34; promoter, and protoplast transfection assays showed that AtSWC6 colocalizes with SUF3. Protein interaction analyses suggested the formation of a complex between PIE1, SUF3, AtSWC6, and AtSWC2. In addition, H2AZ, a substrate of SWR1C, interacts with both PIE1 and AtSWC2. Finally, knockdown of the H2AZ genes by RNA interference or artificial micro RNA caused a phenotype similar to that of atswc6 or suf3. Our results strongly support the presence of SWR1C&#45;like complex in Arabidopsis that ensures proper development, including floral repression through full activation of &#34;FLC&#34;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A subfamily of transcription factors control root hair development downstream of RHD6&#47;RSL1.</div>

<div>

	<span style="text-decoration: underline" >
	Keke Yi</span>
	<sup>1</sup>, 

	<span  >
	Beno&#238;t Menand</span>
	<sup>1</sup>, 

	<span  >
	Liam Dolan</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Cell and Developmental Biology, John Innes Centre, Norwich, UK

</div>

<div>The development of root hairs involves the establishment of the root hair cell identity, followed by the initiation of the root hair and its elongation by tip growth. Although the established root hair cell identity is well characterized at the genetic level, the mechanism controlling the initiation and tip growth remains to be elucidated. Previously, in our lab, we isolated the <span style="font-style: italic">RHD6 &#40;<span style="text-decoration: underline">R</span>OOT <span style="text-decoration: underline">H</span>AIR <span style="text-decoration: underline">D</span>EFECTIVE 6&#41;</span> and <span style="font-style: italic">RSL1 &#40;<span style="text-decoration: underline">R</span>HD <span style="text-decoration: underline">S</span>IX&#45;<span style="text-decoration: underline">L</span>IKE1&#41;</span> genes &#40;a pair of duplicated genes&#41;, which participate in the root hair initiation &#40;Menand et al., accepted&#41;. These two genes are specifically expressed in the trichoblast in the meristem and elongation zone. We performed microarray transcription profiling analysis for the <span style="font-style: italic">rhd6&#47;rsl1</span> double mutants to identify target of RHD6&#47;RSL1, and found a subfamily of transcription factors, which might function downstream of RHD6 and RSL1. The expression of this subfamily of genes is significantly repressed in the <span style="font-style: italic">rhd6&#47;rsl1</span> double mutant background. Using whole gene GFP fusions we showed that each of these genes are specifically expressed in the trichoblast at the stage where root hairs initiate. Therefore we named these genes <span style="font-style: italic">RHS &#40;<span style="text-decoration: underline">R</span>OOT <span style="text-decoration: underline">H</span>AIR <span style="text-decoration: underline">S</span>PECIFIC&#41;</span> 1 to 4. When <span style="font-style: italic">RHS1, RHS2</span> or <span style="font-style: italic">RHS3</span> was expressed ectopically in the <span style="font-style: italic">rhd6&#47;rsl1</span> double mutant background, the root hair deficient phenotype of rhd6&#47;rsl1 was partially rescued. This suggests that this subfamily of genes are also involved in root hair development. <span style="font-style: italic">rhs1</span> loss of function mutants have a short root hair phenotype compared with wild type. While root hair density is decreased in mutants that lack <span style="font-style: italic">RHS3</span> function.. No distinct root hair phenotype can be detected in mutants that lack either <span style="font-style: italic">RHS2</span> or <span style="font-style: italic">RHS4</span> function. The double, triple and quadruple mutants are being generated. These genetic resources will be used for future microarray transcription profiling analysis, which will help us to define a transcriptional regulating network for the process of root hair initiation and tip growth.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular Genetic analysis of cell division effects in leaf morphogenesis</div>

<div>

	<span  >
	Sang Eun Jun</span>
	<sup>1</sup>, 

	<span  >
	Kiu-Hyung Cho</span>
	<sup>1</sup>, 

	<span  >
	Yoon-Ah Byun</span>
	<sup>1</sup>, 

	<span  >
	Masaaki Umeda</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Gyung-Tae Kim</span>
	<sup>1,3</sup>

</div>

<div>

	<sup>1</sup>Division of Molecular Biotechnology, Dong-A University, Busan 604-714, South Korea, 

	<sup>2</sup>Graduate School of Biological Sciences, Nara Institute of Science and Technology, Nara 630-0101, Japan, 

	<sup>3</sup>Environmental Biotechnology Research Center, Gyeongsang National University, Jinju 660-701, South Korea

</div>

<div>As plants grow, cell division plays an important role in which plants undergo normal developmental process and form appropriate shape. There are 7 inhibitors of cyclin&#45;dependent kinase &#40;ICK&#47;KRP&#41; which inhibit CDK activity by tight interaction with cyclin&#45;CDK complexes in Arabidopsis. To investigate how KRPs affect leaf morphology, we construct transgenic plants overexpressing KRPs under the control of the CaMV 35S promoter. &#13;<br/>In our study, common phenotypes of each transgenic plant which overexpresses KRPs, such as reduced size of whole plant, reduced size and serration of leaves and floral organs, and production of small siliques with fewer seeds, were characterized. &#13;<br/>Transgenic plants overexpressing KRP1 and KRP5 show similar serration phenotypes. However, transgenic plants overexpressing KRP1 &#40;Group I&#41; have round leaves whereas those of KRP5 &#40;Group II&#41; show narrow leaves. Interestingly, the phenotype of bifurcate leaves with serration was observed only in transgenic plants overexpressing Group III KRPs. In addition, rolling up phenotype of leaves, and rough and curvature phenotype on leaf surface were observed in transgenic plants overexpressing KRP6.&#13;<br/>Taken together, we will discuss about the roles of KRPs and cell division in leaf morphogenesis from our results of genetic and anatomical analysis of each transgenic plant overexpressing these KRPs.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Isolation and Characterization of A Male Sterile Mutant Which Has Defect in Exine Patterning of Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Guan Yuefeng</span>
	<sup>1</sup>, 

	<span  >
	Zhang Hongxia</span>
	<sup>1</sup>, 

	<span  >
	Yang Zhongnan</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>National Key Laboratory of Plant Molecular Genetics, Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, People's Republic of China., 

	<sup>2</sup>Shanghai Normal University,Shanghai,People's Republic of China.

</div>

<div>A novel mutant of Arabidopsis thaliana was isolated from an activation tagging mutant pool. The homozygous mutant exhibited dramatically reduced fertility with otherwise normal vegetative development. Megagametogenesis and male meiosis of mutant is not affected. Whereas, inhibited stamen extension and aborted pollen development was observed after microspore release. SEM &#40;Scanning Electron Microscopy&#41; analysis revealed that sporopollenin deposited randomly on surface of most of the pollen grains, resulting a spotted exine pattern instead of reticulate pattern. &#13;<br/>To identify the corresponding gene, we performed thermal asymmetric&#13;<br/>interlaced PCR &#40;TAIL&#45;PCR&#41;&#41; to obtain the genomic flanking sequences adjacent to T&#45;DNA. Sequencing of flanking sequence indicated that the T&#45;DNA inserted in the last intron of a gene encoding a putative transmembrane protein. GFP fusion and tissue specific RT&#45;PCR indicated that the gene was expressed high specifically in reproductive organ of Arabidopsis, especially in anther. Our results provide further insight into the molecular basis of plant male reproductive development.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Axis&#45;dependent gene expression in the lateral organ formation</div>

<div>

	<span style="text-decoration: underline" >
	Kiyotaka Okada</span>
	<sup>1</sup>, 

	<span  >
	Koichi Toyokura</span>
	<sup>2</sup>, 

	<span  >
	Keiro Watanabe</span>
	<sup>2</sup>, 

	<span  >
	Toshiaki Tameshige</span>
	<sup>2</sup>, 

	<span  >
	Yuhei Tsuchida</span>
	<sup>2</sup>, 

	<span  >
	Ryuji Tsugeki</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>National Institute for Basic Biology, Myodaiji, Okazaki, Aichi, Japan, 

	<sup>2</sup>Graduate School of Science, Kyoto University, Kyoto, Japan

</div>

<div>In order to unveil the signaling cascade which lead region&#45;specific gene expression responsible for axis&#45;dependent organ formation, we are examining expression pattern of three genes working in the early stages of lateral organ development. Lateral organs, such as leaves, young floral buds and floral organs develop from primordia formed at the fixed position in the peripheral region of shoot apical meristem, and are considered to follow three axes; basal&#45;apical, central&#45;marginal, and adaxial&#45;abaxial.  The direction of each axis could be fixed in relation to the position of the meristem. FILAMENTOUS FLOWER &#40;FIL&#41;, a member of YABBY family, is responsible to the formation of abaxial side tissue and is expressed in the abaxial side of lateral organ primordia,  Promoter analysis of FIL showed that the region&#45;specific expression is based on two discrete cis elements: one is responsible to promote expression at both abaxial and adaxial sides, and the other is to repress at adaxial side.  On the contrary, PHABULOSA &#40;PHB&#41;, a member of homeobox family, is required for formation of the adaxial side tissue, and is expressed at the adaxial side. Recent researches propose that the region&#45;specific expression of PHB is controlled by microRNA which is expressed at the abaxial side. These observations suggest that the two sides in lateral organ primordia mutually control specific gene expression. However, double&#45;staining analysis of FIL&#45; and PHB&#45;expressing regions showed the two regions are partially overlapped, not supporting the simple mutual&#45;repression model. The third gene, PRESSED FLOWER &#40;PRS&#41;, a member of homeobox&#47;WOX family, is expressed at the marginal region of lateral organ primordia. But the FIL&#45;PHB overlapping region covers the PRS expressing region. Based on these observations and other ongoing experiments, we are trying to present a model of axes&#45;dependent gene expression and lateral organ development. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Low germination of brassinosteroids&#45;deficient mutant can be overcome by the treatment of the reducing reagent, dithiothreitol</div>

<div>

	<span  >
	Sun Young Kim</span>
	<sup>1</sup>, 

	<span  >
	Beg Hab Kim</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Kyoung Hee Nam</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Division of Biological Science, Sookmyung Women?s University, Seoul, 140-742, Korea

</div>

<div>Several environmental factors including light, temperature, and moisture and plant hormones, especially gibberellin &#40;GA&#41;, abscisic acid &#40;ABA&#41;, ethylene, and brassinosteroids &#40;BR&#41; affect seed germination. BR has been known to rescue the germination of GA&#45;deficient and GA response mutants seeds, implicating that BR probably acts downstream of GA. BR&#45;deficient mutant, det2, also has low germination phenotype in addition to dark green dwarfed stature. Det2 seeds germinated only 50&#37; after 3 to 4 days, when the wild type seeds germinated almost completely. We found treatment of the reducing reagent, dithiothreitol &#40;DTT&#41;, to the media induced the increased germination in det2, which is not substituted by treatment of GA. To investigate whether the effect of DTT on the stimulation of seed germination would be specific to det2 mutation, we observed the germination rate of other mutants involved in the metabolism and signaling of GA and ABA. Each mutant exhibited its characteristic early or late germination phenotype, and DTT treatment was not induced the difference of the seed germination. We also examined the germination aspects in brassinazole &#40;BRZ&#41;&#45;treated wild type seeds on the DTT&#45;containing media. In contrast to det2, BRZ&#45;treated wild type seeds showed normal germination compared to the untreated seeds and DTT did not affect germination. This suggests that exogenously induced BR&#45;deficiency may not exert the same influence on the reduced germination as shown in det2, in which continuous low levels of BR were kept in seeds throughout the formation of the mature seeds during embryogenesis. We further wanted to know the genes differentially regulated in germinating seeds of wild type and det2 with or without DTT and performed the full screening of differentially expressed genes using ACP&#45;based PCR method. Several genes were fished out as their expression patterns were fluctuating and were analyzed. </div>

</div>





<div class="abs">

<div style="font-weight:bold">PLETHORA proteins as dose&#45;dependent master regulators of Arabidopsis root development</div>

<div>

	<span style="text-decoration: underline" >
	Hugo Hofhuis</span>
	<sup>1</sup>, 

	<span  >
	Carla Galinha</span>
	<sup>2</sup>, 

	<span  >
	Marijn Luijten</span>
	<sup>1</sup>, 

	<span  >
	Viola Willemsen</span>
	<sup>1</sup>, 

	<span  >
	Ikram Blilou</span>
	<sup>1</sup>, 

	<span  >
	Renze Heidstra</span>
	<sup>1</sup>, 

	<span  >
	Ben Scheres</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>utrecht university, 

	<sup>2</sup>university of Oxfort

</div>

<div>Factors with a graded distribution can program fields of cells in a dose&#45;dependent manner, but no evidence has hitherto surfaced for such mechanisms in plants. In the Arabidopsis root, two &#34;PLETHORA &#40;PLT&#41;&#34; genes, encoding AP2&#45;domain transcription factors, have been shown to maintain the activity of stem cells. Here, we show that a clade of four &#34;PLT&#34; homologues is necessary for root formation. Promoter activity and protein fusions of PLT homologues display gradient distributions with maxima in the stem cell area. PLT activities are largely additive, dosage dependent and their functions in separate expression domains can be uncoupled: high levels of PLT activity promote stem cell identity and maintenance; lower levels promote mitotic activity of stem cell daughters; and further reduction in levels is required for cell differentiation. Our findings indicate that PLT protein dosage is translated into distinct cellular responses in plant development.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">Control of flowering time by the SPL3&#47;miR156 pathway</div>

<div>

	<span style="text-decoration: underline" >
	Jia-Wei Wang</span>
	<sup>1</sup>, 

	<span  >
	Benjamin Czech</span>
	<sup>1</sup>, 

	<span  >
	Detlef Weigel</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Developmental Biology, Tuebingen, Germany

</div>

<div>A drastic developmental transition is the switch from vegetative to reproductive development. The initiation of flowering requires an endogenous developmental program to specify the floral identity of the new structures that arise at the shoot apex. Genetic analysis has identified four main signals and corresponding pathways that control flowering time in Arabidopsis: the photoperiod, autonomous, vernalization, and gibberellin pathways. Overexpressing of one Arabidopsis microRNA &#40;miRNA&#41;, miR156, leads to a plant with delayed flowering and shortened plastochrons &#40;the interval between formation of primordia at the shoot apex&#41; &#40;Schwab et al., 2005&#41;. miR156 targets a group of genes encoding plant&#45;specific transcription factors, named squamosa promoter binding protein&#45;like &#40;SPL&#41; proteins. Overexpression of a miR156&#45;resistant form of SPL3 &#40;rSPL3&#41; accelerates flowering in both long day and short day condition. Using mis&#45;expression, promoter GUS &#40;&#946;&#45;glucuronidase&#41; reporter fusions, and genetic analyses, we show here that miR156 regulates SPL3 levels in leaves and that SPL3 promotes flowering in a FT &#40;FLOWERING LOCUS T&#41;&#45;dependent manner. We will discuss the position of SPL&#47;miR156 in the current framework of flower signal integration.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Stem cell and meristem action in Arabidopsis: a numbers game&#63;</div>

<div>

	<span style="text-decoration: underline" >
	Ben Scheres</span>
	<sup>1</sup>, 

	<span  >
	Carla Galinha</span>
	<sup>1</sup>, 

	<span  >
	Hugo Hofhuis</span>
	<sup>1</sup>, 

	<span  >
	Marijn Luijten</span>
	<sup>1</sup>, 

	<span  >
	Renze Heidstra</span>
	<sup>1</sup>, 

	<span  >
	Veronica Grieneisen</span>
	<sup>1</sup>, 

	<span  >
	Paulien Hogeweg</span>
	<sup>1</sup>, 

	<span  >
	Jian Xu</span>
	<sup>1</sup>, 

	<span  >
	Stan Maree</span>
	<sup>1</sup>, 

	<span  >
	Ikram Blilou</span>
	<sup>1</sup>, 

	<span  >
	Viola Willemsen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dpt of Biology, Utrecht University

</div>

<div>Recent data suggest that transport&#45;dependent auxin maxima are important for development not only in the root but also during embryogenesis and in shoot&#45;derived organs. It now becomes an important question how auxin as a patterning cue induces specific downstream pathways to mediate diverse effects. The Arabidopsis PLETHORA1 and PLETHORA2 genes encode transcription factors required for stem cell specification and can ectopically induce root identity. PLT expression is auxin&#45;inducible, depends on auxin response factors and follows auxin accumulation patterns during embryogenesis and in post&#45;embryonic root development. PLT genes translate auxin accumulation into region&#45; and cell type specification patterns, and interact with the SHORTROOT&#45;SCARECROW pathway that plays a role in patterning the root stem cells and in providing mitotic potential to the stem cell daughters that form the proximal meristem. This interaction involves the conserved RETINOBLASTOMA&#45;RELATED pocket protein, and we are investigating the interaction between the RBR pathway and patterning genes. &#13;<br/>Mutations in new PLT genes reveal that the PLT gene clade extensively regulates expression of the PIN facilitators of polar auxin transport in the root and this contributes to a specific auxin transport route that maintains stem cells at the appropriate position. We are currently investigating the role of graded PLT expression in this control, using genetic and genomic approaches. We are also addressing the properties of the PLT&#45;PIN feedback loop by computational modelling.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Contribution of KNAT2 and KNAT6 in the inflorescence</div>

<div>

	<span style="text-decoration: underline" >
	Laura Ragni</span>
	<sup>1</sup>, 

	<span  >
	Marcus G&#252;nl</span>
	<sup>1</sup>, 

	<span  >
	V&#233;ronique  Pautot</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Laboratoire de Biologie Cellulaire, IJPB, Institut National de la Recherche Agronomique,  Route de St Cyr, 78026 Versailles cedex,  France

</div>

<div>The aerial part of the plant is produced by the shoot apical meristem &#40;SAM&#41;, a group of cells established during embryogenesis. The SAM has two mains functions: the maintenance of a population of stem cells and the production of organs. The KNOTTED&#45;like from <span style="font-style: italic">Arabidopsis thaliana</span> &#40;KNAT&#41; class I family of transcription factors plays a crucial role in controlling meristem activity. The KNAT proteins belong to the three amino acid loop extension &#40;TALE&#41; homeodomain superclass, and form heterodimers with other TALE proteins belonging to the BELL &#40;BEL1&#45;like&#41; family. KNAT class I comprises four members: SHOOT MERISTEMLESS, is required for SAM initiation during embryogenesis and its maintenance during post&#45;embryonic development, whereas BREVIPEDICELLUS&#47;KNAT1 has a primary role in internode development. BP also interacts with PNY, a BELL member, to regulate inflorescence patterning. We previously showed that KNAT6 contributes with STM to SAM and organ boundary maintenance. KNAT2 does not have such a role.<br/>To investigate the role of KNAT6 and KNAT2 in the inflorescence, we examined their interactions with BP&#47;KNAT1 and PENNYWISE.<br/><br/><span style="font-style: italic">L. Ragni and M. G&#252;nl are supported by European Marie&#45;Curie &#40;FP6&#41; program</span><br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification of the <span style="font-style: italic">cis</span>&#45;acting sequences required for KNOX regulation of <span style="font-style: italic">LATERAL ORGAN BOUNDARIES</span></div>

<div>

	<span style="text-decoration: underline" >
	Amanda Mangeon</span>
	<sup>1</sup>, 

	<span  >
	Bin Shuai</span>
	<sup>1</sup>, 

	<span  >
	Patricia S. Springer</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Botany and Plant Sciences, Center for Plant Cell Biology, University of California, Riverside

</div>

<div><span style="font-style: italic">LATERAL ORGAN BOUNDARIES</span> &#40;<span style="font-style: italic">LOB</span>&#41; is the founding member of the plant&#45;specific <span style="font-style: italic">LBD</span> gene family of hypothetical transcription factors. <span style="font-style: italic">LOB</span> was isolated from an enhancer&#45;trap screen based on its expression pattern in the boundaries of lateral organs, including the adaxial boundaries between the shoot apical meristem and leaf primordia. Class I <span style="font-style: italic">KNOX</span> transcription factors are expressed in the meristem and excluded from leaves. The <span style="font-style: italic">KNOX</span> genes <span style="font-style: italic">BREVIPEDICELLUS</span> &#40;<span style="font-style: italic">BP</span>&#41; and <span style="font-style: italic">SHOOT MERISTEMLESS</span> &#40;<span style="font-style: italic">STM</span>&#41; are required for meristem maintenance and are responsible for the repression of leaf&#45;expressed genes, such as <span style="font-style: italic">ASYMMETRIC LEAVES1</span> &#40;<span style="font-style: italic">AS1</span>&#41; and <span style="font-style: italic">AS2</span>, in the meristem. In turn, <span style="font-style: italic">AS1</span> and <span style="font-style: italic">AS2</span> function to repress <span style="font-style: italic">KNOX</span> gene expression in leaves, indicating an antagonistic relationship between genes expressed in these adjacent domains.  As <span style="font-style: italic">LOB</span> is expressed at the boundary between these domains, we are interested in whether <span style="font-style: italic">LOB</span> might function to mediate communication between the meristem and organ primordia.  <span style="font-style: italic">LOB</span> expression is positively regulated by both <span style="font-style: italic">KNOX</span> transcription factors and by <span style="font-style: italic">AS1</span>&#47;<span style="font-style: italic">AS2</span>, consistent with this idea. Using promoter deletion analysis we have mapped the regulatory regions responsible for <span style="font-style: italic">LOB</span> expression pattern and for <span style="font-style: italic">KNOX</span> regulation of <span style="font-style: italic">LOB</span>. These results will be presented.&#13;<br/>&#13;<br/>Supported by National Science Foundation and CAPES &#40;Brazil&#41;</div>

</div>





<div class="abs">

<div style="font-weight:bold">Natural variation of flowering&#45;time gene &#34;CONSTANS&#34;</div>

<div>

	<span style="text-decoration: underline" >
	Yasushi Kobayashi</span>
	<sup>1</sup>, 

	<span  >
	Detlef Weigel</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biology, Max Planck Institute for Developmental Biology, D-72076 T&#252;bingen, Germany, 

	<sup>2</sup>Salk Institute, La Jolla, CA 92037, USA

</div>

<div>Plants normally flower in response to seasonal cues, such as day&#45;length and temperature. &#34;Arabidopsis thaliana&#34; has a broad range of habitat in the Northern hemisphere, and extensive flowering&#45;time variation has been observed among natural populations. &#34;FRIGIDA&#34; and &#34;FLC&#34; genes have been shown to act as major determinants controlling flowering&#45;time variation. The large effect of the epistatic &#34;FRIGIDA&#45;FLC&#34; module suggests that potentially significant other genetic variation could be easily overlooked. To gain a clue of potentially hidden genetic variation, we have combined a sequence&#45;based reverse quantitative genetic approach and a QTL mapping&#45;based forward quantitative genetic approach. We will present the functional analysis of natural variation on flowering&#45;time gene &#34;CONSTANS&#34;, which we expect to have significant effects after exposure to vernalization, which reduces the action of the &#34;FRIGIDA&#45;FLC&#34; induced flowering delay.<br/><br/>References:<br/>1. J.R. Stinchcombe et al., &#40;2004&#41; &#34;Proc. Natl. Acad. USA&#34; 101, 4712&#45;4717.<br/>2. A.L. Caicedo et al., &#40;2004&#41; &#34;Proc. Natl. Acad. USA&#34; 101, 15670&#45;15675.<br/>3. C. Shindo et al., &#40;2005&#41; &#34;Plant Physiol.&#34; 138, 1163&#45;1173.<br/>4. J.D. Werner et al., &#40;2005&#41; &#34;Genetics&#34; 170, 1197&#45;1207.<br/>5. J. Lempe et al., &#40;2005&#41; &#34;PLoS Genet.&#34; 1, 109&#45;118.<br/>6. V. Le Corre, &#40;2005&#41; &#34;Mol. Ecol.&#34; 14, 4181&#45;4192.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic evidence for a role of cation&#47;H&#43; transporters in male fertility</div>

<div>

	<span style="text-decoration: underline" >
	Yongxian Lu</span>
	<sup>1</sup>, 

	<span  >
	Xiyan Li</span>
	<sup>1</sup>, 

	<span  >
	Senthilkumar Padmanaban</span>
	<sup>1</sup>, 

	<span  >
	Heven Sze</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Cell Biology and Molecular Genetics, University of Maryland, College Park, MD 20742, USA

</div>

<div>Pollen development and tube growth are critical for male fertility and seed production, though little is known about the molecular and cellular bases governing these processes. A family of Arabidopsis thaliana CHX genes was found to be preferentially expressed in pollen, but their specific role in pollen biology is unknown &#40;Sze et al, 2004&#41;. AtCHXs are thought to be K&#43;&#40;Na&#43;&#41;&#47;H&#43; exchangers with a role in osmoregulation and K&#43; homeostasis in vegetative tissues &#40;Padmanaban et al. 2007&#41;.  &#13;<br/>A reverse genetics approach was used to determine the function of two related genes CHX&#45;A and CHX&#45;B. Single T&#45;DNA insertion mutants &#40;aa or bb&#41; showed similar in vitro pollen germination and tube growth to that of wild&#45;type. However, no double mutants &#40;aabb&#41; were recovered in over 300 progeny plants from the AaBb parent.  Reciprocal crossing of Aabb with wild&#45;type &#40;AABB&#41; plants suggested a block in male&#45;specific transmission when both A and B genes are defective. When wild&#45;type pistil was pollinated with a limited number of pollen grains from Aabb plants, seed set was reduced about half of that from AAbb plants.  These results demonstrate that CHX&#45;A and CHX&#45;B perform similar functions, and that either A or B is essential for male fertility.  The cellular roles of both genes in reproduction are under investigation.  &#13;<br/>&#13;<br/>Reference:&#13;<br/>Sze H, Padmanaban S, Cellier F, Honys D, Cheng NH, Bock KW, Conejero G, Li XY, Twell D, Ward JM. &#40;2004&#41; Expression pattern of a novel AtCHX gene family highlight potential roles in osmotic adjustment and K&#43; homeostasis in pollen development. Plant Physiology 136: 2532&#45;2546&#13;<br/>Padmanaban S, Chanroj S, Kwak JM, Li XY, Ward JM, Sze H &#40;2007&#41; Participation of Endomembrane Cation&#47;H Exchanger AtCHX20 in Osmoregulation of Guard Cells. Plant Physiology &#40;In press&#41;&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">An Arabidopsis PEL may play a role in male gametophyte development</div>

<div>

	<span style="text-decoration: underline" >
	Si-Qi Wang</span>
	<sup>1</sup>, 

	<span  >
	Wei-Cai Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>INSTITUTE OF GENETICS AND DEVELOPMENTAL BIOLOGY, CHINESE ACADEMY OF SCIENCES, Beijing, China

</div>

<div>Pectin is a main component of plant cell wall and is partly responsible for the maintenance of the structural integrity of cell walls. Pectate lyases &#40;PELs&#41; catalysis the cleavage of the de&#45;esterified pectin and cause its degradation. Plant PELs presumably participate in the developmental progress such as in pollen tube initial germination and penetration into the style, fruit&#45;ripening, and lateral roots emergence. Here we report an Arabidopsis transposon mutant line kd468, in which a gene coding for a putative PEL is disrupted. This line displayed non&#45;Mendelian segregation of KanR marker gene. The ratio of KanR to KanS is 1.16, instead of the Mendelian 3:1 segregation. Reciprocal crosses indicate that Ds transmission through the male gametophyte is severally affected, while the transmission through the female gametophyte is almost normal. Pollen development seems normal when stained with Alexander and DAPI compared with that of wild type. Molecular analyses show that a single Ds is inserted at the ORF of the PEL gene. Complementation experiment and detailed phenotypic study are ongoing. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification of downstream genes of Arabidopsis <span style="font-style: italic">CUP&#45;SHAPED COTYLEDON</span> genes</div>

<div>

	<span style="text-decoration: underline" >
	Mitsuhiro Aida</span>
	<sup>1</sup>, 

	<span  >
	Yuka Tsubakimoto</span>
	<sup>1</sup>, 

	<span  >
	Ayano Kariya</span>
	<sup>1</sup>, 

	<span  >
	Ken'ichiro Hibara</span>
	<sup>1</sup>, 

	<span  >
	Masao Tasaka</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Nara Institute of Science and Technology (NAIST), Nara, Japan

</div>

<div>The Arabidopsis <span style="font-style: italic">CUP&#45;SHAPED COTYLEDON</span> genes <span style="font-style: italic">CUC1</span>, <span style="font-style: italic">CUC2</span>, and <span style="font-style: italic">CUC3</span> encode transcriptional activators of the NAC family and are redundantly required for shoot meristem formation and organ separation. Double mutant combinations in any two of these genes cause severely fused cotyledons and lack of an embryonic shoot meristem. Conversely, ectopic overexpression of <span style="font-style: italic">CUC1</span> induces ectopic shoot meristem formation on the surface of cotyledons. To identify downstream genes of <span style="font-style: italic">CUC1</span> and <span style="font-style: italic">CUC2</span>, we carried out microarray experiments. We compared expression profiles in RNA extracted form wild&#45;type and <span style="font-style: italic">cuc1 cuc2</span> double mutant embryos at the torpedo stage and selected candidate genes that were downregulated in <span style="font-style: italic">cuc1 cuc2</span>. In addition, another set of microarray experiments identified genes ectopically activated in cotyledons of plants ectpically expressing <span style="font-style: italic">CUC1</span>. We are now analyzing expression patterns of these candidate downstream genes in wild type and <span style="font-style: italic">cuc1 cuc2</span> embryos by <span style="font-style: italic">in situ</span> hybridization. The results from these experiments will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">'LFR'&#40;text in italics&#41; point mutant affects leaf and flower development in Arabidopsis</div>

<div>

	<span  >
	Zhijuan Wang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Sujuan Cui</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Laboratory of Molecular and Cell Biology, Hebei Normal University, Shijiazhuang, China

</div>

<div>Leaf and flower, as major aerial organs of plant, play important roles in plant individual development. During screening SALK T&#45;DNA insertion mutant lines of Arabidopsis, one novel mutant line was named 'lfr'&#40;text in italics&#41;, which shows both leaf and flower mutant phenotype. 'lfr'&#40;text in italics&#41;has curling&#45;up young leaves. When the leaf matures, it can expand. The number of petal in 'lfr'&#40;text in italics&#41;flower increases and stamen reduces. The early flowers are infertility, for the filament is too short to pollinate. The latter flowers can produce seeds but less than wild type. Besides these mutant phenotypes, the cotyledon vein pattern, bolting time and angle of secondary bolts are also different from those of wild type. 'LFR'&#40;text in italics&#41;gene may function in many processes in plant aerial organ development.&#13;<br/>'lfr'&#40;text in italics&#41;mutant phenotype was shown not linked with inserted T&#45;DNA by TAIL&#45;PCR and Southern blot. Genetics cross analysis indicated the mutant phenotype may be controlled by one recessive locus. To clone the locus, we used the map&#45;based cloning strategy using SSLP and CAPS markers. Among 2012 recombinant chromosomes, we mapped the locus between two markers in chromosome &#61472;3. Sequencing of DNA and RT&#45;PCR product showed a G to A transversion in the first base of the third intron of gene 'LFR'&#40;text in italics&#41;. The transversion causes a 67 bp deletion of the 'LFR'&#40;text in italics&#41; mRNA, which results in an early stop codon. To further identify the linkage between LFR and mutant phenotype, SALK T&#45;DNA inserted line was screened but we didn&#8217;t get null mutant, but '35S::LFR'&#40;text in italics&#41; cDNA and 'LFR::LFR'&#40;text in italics&#41;  transgenic plants can rescue the phenotype completely. &#13;<br/>As annotation, 'LFR'&#40;text in italics&#41; is one unknown gene, and LFR protein contains 3 putative protein&#45;protein interaction domains which involved in cell signaling or cellular architecture in many eukaryotic. We suppose that LFR may interact with other proteins or might be an integrator to involve in so many processes. Ongoing or further works will be done to explain the mechanism of LFR functions.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular and physiological characterization of TF71, a member of the <span style="font-style: italic">Arabidopsis</span> AP2&#47;EREBP transcription factor family</div>

<div>

	<span style="text-decoration: underline" >
	Mohammad Mehrnia</span>
	<sup>1</sup>, 

	<span  >
	Erwan Michard</span>
	<sup>1,2</sup>, 

	<span  >
	Maria Ines Zanor</span>
	<sup>3</sup>, 

	<span  >
	Bernd Moeller-Roeber</span>
	<sup>1,3</sup>

</div>

<div>

	<sup>1</sup>University of Potsdam, Institute of Biochemistry and Biology, 14476 Potsdam-Golm, Germany, 

	<sup>2</sup>Present address: Centro de Biologia do Desenvolvimento, Instituto Gulbenkian de, Ci&#234;ncia, 2780-156 Oeiras, Portugal, 

	<sup>3</sup>Max Planck Institute of Molecular Plant Physiology, 14476 Potsdam-Golm, Germany

</div>

<div>AP2&#47;EREBP transcription factors are known to be involved in many aspects of cellular and developmental processes as well as various responses to environmental stimuli. To date, most of the members of the superfamily have yet to be analysed. This study investigates the function of one member &#40;named <span style="font-style: italic">TF71</span>&#41; of the <span style="font-style: italic">Arabidopsis</span> AP2&#47;EREBP family. <span style="font-style: italic">TF71</span> gene expression was analysed using promoter&#45;reporter &#40;&#223;&#45;glucuronidase&#41; fusions. Expression was detected in root tips, at positions of lateral root formation, in various flower tissues, in pedicels and the abscission zone of siliques, and in stipules. In leaves, GUS activity was generally only marginal, but strongly, within 6 hours, induced by wounding. <span style="font-style: italic">TF71</span> expression was also found to be induced by auxin treatment, and by the addition of phosphate to phosphate&#45;starved seedlings. <span style="font-style: italic">TF71</span> expression is remarkably strong during the S phase of the cell cycle. In addition, strong <span style="font-style: italic">TF71</span> activity was observed during callus formation. Ectopic expression of <span style="font-style: italic">TF71</span> under the control of the CaMV35 S promoter strongly reduces apical dominance and leads to a reduction of overall plant size. Our data indicate that TF71 is a transcription factor particularly important during cell proliferation. To identify direct target genes of TF71 we have generated transgenic plants expressing it under the control of a chemically inducible promoter. Transcriptome profiles will be obtained from plants shortly after treatment with the chemical inducer. The data obtained will be presented and the function of identified target genes discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A novel function of ORC1 in transcriptional regulation</div>

<div>

	<span style="text-decoration: underline" >
	Mar&#237;a de la Paz S&#225;nchez</span>
	<sup>1</sup>, 

	<span  >
	Crisanto Gutierrez</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centro de Biolog&#237;a Molecular Severo Ochoa, CSIC-UAM, Cantoblanco, 28049 Madrid, Spain

</div>

<div>Initiation of DNA replication is tightly regulated in eukaryotic cells. During this process, the origin recognition complex &#40;ORC&#41; is required for initiation of DNA replication at replication origins in all eukaryotes. The ORC complex is formed by six proteins that bind to DNA as part of the pre&#45;replicative complex &#40;preRC&#41;, the first step to assemble the DNA replication machinery. Some DNA binding proteins frequently play multiple roles in different cellular functions, including transcriptional regulation, one example being ORC1, the large subunit of the origin recognition complex. ORC1 has been involved in gene silencing in yeast and mammalian cells, process where ORC1 is required to recruit Sir proteins to silencers; in addition, in mammalian cells ORC1 is able to bind a histone acetyl transferase &#40;HBO1&#41;. <br/>One of our interests is to define the roles of Arabidopsis ORC1 in DNA replication and transcriptional regulation. Arabidopsis have two ORC1 genes &#40;A and B&#41;, both of which show high amino acid homology, and conserve the typical domains of ORC proteins that participate in ORC complex formation and interact with DNA. By pull down assays we found that ORC1 interacts with histone proteins and with a Histone Acetyl Transferase &#40;HAT&#41; homologue of human HBO1. Here we will focus in discussing the physiological function of those associations and its implication in transcriptional regulation.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">E2F regulates FASCIATA1, a chromatin assembly gene whose loss switches on the endocycle and activates gene expression by changing the epigenetic status</div>

<div>

	<span  >
	Elena Ramirez-Parra</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Crisanto Gutierrez</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centro de Biologia Molecular Severo Ochoa, CSIC-UAM, Cantoblanco, 28049 Madrid, Spain

</div>

<div>Chromatin reorganization is facilitated by histone chaperones, whose function is crucial for genome integrity. Deposition of histones H3&#47;H4 can occur either in association with DNA replication during the S&#45;phase of the cell cycle or independently of the replication process. These processes depend on the chromatin assembly factor 1 &#40;CAF&#45;1&#41; or the HIRA chaperones, respectively. CAF&#45;1 is a heterotrimeric complex which in Arabidopsis is composed of the products encoded by the FASCIATA1 &#40;FAS1&#41;, FASCIATA2 &#40;FAS2&#41; and MULTICOPY SUPPRESOR OF IRA1 &#40;MSI1&#41; genes. We have found that FAS1, which encodes the large subunit of CAF&#45;1, is a target of the E2F family of transcription factors. <br/>Depletion of CAF&#45;1 in human cells leads to cell death whereas in Arabidopsis, where it is involved in heterochromatin compaction and homologous recombination, plants are viable. The mechanism that makes the lack of CAF&#45;1 activity compatible with development is not known. Loss of FAS1 results in inhibition of mitotic progression, formation of organs with much fewer but larger cells, and a precocious and systemic switch to the endocycle program. A selective up&#45;regulation of the expression of a subset of genes, including those involved in activation of the G2 DNA damage checkpoint, notably RAD51, BRCA1 and PARP1, also occurs upon FAS1 loss. This depends on the introduction of activation epigenetic marks just upstream of the corresponding ORFs. These data suggest that a correct chromatin assembly during S&#45;phase is required to prevent unscheduled changes in the epigenetic marks of target genes. Interestingly, the same set of G2 checkpoint genes is detected upon treatment of wild type plants with DNA damaging treatments. <br/>Together, our data support a model in which defects in chromatin assembly during S&#45;phase and the DNA damage signaling share part of a pathway, which ultimately lead to mitotic arrest and triggers the endocycle program. In such way, this might be a bypass mechanism that makes development compatible with cell division arrest induced by DNA damage stress.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional Characterization of <span style="font-style: italic">STICHEL</span> gene in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Cho-Chun Huang</span>
	<sup>1</sup>, 

	<span  >
	Martin Huelskamp</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department III of Botanical Institute, University of Cologne, Germany

</div>

<div>The <span style="font-style: italic">STICHEL</span> &#40;<span style="font-style: italic">STI</span>&#41; gene plays an important role in the regulation of branch number of the unicellular trichomes in Arabidopsis &#40;Ilgenfritz et al., 2003&#41;. GFP&#45;STI protein fusions in trichome cell indicate that the localization of STI in the cortex and that it is concentrated at the tip of trichomes. Mutation analysis with the conserved residue of the ATPase domain showed that the domain is essential for STICHEL protein &#40;STI&#41; to regulate trichome branching. Interestingly, yeast two hybrid assay and BiFC analysis showed that STI interacts with itself and might act as oligomers. Progress on this work will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional analysis of a rice gene homologous to the Arabidopsis FVE</div>

<div>

	<span style="text-decoration: underline" >
	Hyo Young Park</span>
	<sup>1</sup>, 

	<span  >
	Il-Sun Baek</span>
	<sup>1</sup>, 

	<span  >
	Yun-Hee Jang</span>
	<sup>1</sup>, 

	<span  >
	Soon-Kap Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Life Sciences and biotechnology, Korea university

</div>

<div> A full length rice cDNA clone &#40;AK111794, OsFVE&#41; showing significant similarity to the Arabidopsis FVE gene was isolated from a Japanese rice cultivar.  The FVE is a chromatin remodeling complex protein which belongs to the autonomous pathway of flowering time control. The amino acid sequence identity of the OsFVE clone with FVE1 &#40;MIS4&#41; and FVE2 &#40;MIS5&#41; are 75&#37; and 69&#37;, respectively.  The OsFVE gene is composed of fourteen introns and fifteen exons.  RT&#45;PCR analysis indicated that the OsFVE mRNA was detectable in all tissues examined, but showed higher expression in mature leaves than in leaves of young rice seedlings.&#13;<br/>We made transgenic Arabidopsis fve&#45;3 mutant plants expressing the OsFVE cDNA driven by 35S promoter in order to examine if the OsFVE could rescue the late flowering phenotype of fve&#45;3 mutant.  The T1 transgenic Arabidopsis plants showed a wide spectrum of flowering times from one similar to that of Col&#45;0 to the other showing later flowering time than the fve&#45;3 mutants.  About one third of total T1 lines showed partially&#45;rescued phenotype.  We examined the expression patterns of flowering time genes such as FLC and SOC1 in the T2 lines showing partially&#45;rescued phenotype.  Intermittent cold treatment delays the flowering of Col&#45;0 but not that of fve&#45;3 plants.  This phenotype was not rescued in the T2 transgenic lines.  This is consistent with observation that the FLC mRNA level of the T2 transgenic lines were not changed by intermittent cold treatment.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Organ formation and gene expression patterning in ovule development</div>

<div>

	<span style="text-decoration: underline" >
	Charles Gasser</span>
	<sup>1</sup>, 

	<span  >
	Ryan Brown</span>
	<sup>1</sup>, 

	<span  >
	Thomas Gallagher</span>
	<sup>1</sup>, 

	<span  >
	Dior Kelley</span>
	<sup>1</sup>, 

	<span  >
	Mona Monfared</span>
	<sup>1</sup>, 

	<span  >
	Christina Passerini</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Davis, CA, USA

</div>

<div>We are studying the genetic and molecular mechanisms underlying the process of ovule morphogenesis in Arabidopsis. We have focused in particular on initiation and laminar growth of integuments, the lateral structures developing on the flanks of an ovule primordium. Classical genetic analysis has identified a number of genes necessary for normal ovule morphogenesis.  The majority of such genes identified to date encode putative transcription factors. Several such factors belong to families that are associated with ab&#47;adaxial polarity determination in the primary lateral organs of seed plants. <span style="font-style: italic">INNER NO OUTER</span> &#40;<span style="font-style: italic">INO</span>&#41; plays an essential role in the asymmetric growth of the outer integument and encodes an ovule&#45;specific member of the YABBY family of transcription factors. <span style="font-style: italic">ABERRANT TESTA SHAPE</span> &#40;<span style="font-style: italic">ATS</span>&#41; encodes a member of the KANADI protein family and is required for the separation of the inner and outer integuments and for growth of the inner integument. Other KANADI proteins appear to be required for outer integument development. We propose a model in which the activity of these genes in integuments parallels the action of gene family members in primary lateral organ development.  In this model, the spatial confinement of expression of these abaxial factors is essential to their function and results in part from interactions with yet to be identified adaxial factors. Studies on the regulation of <span style="font-style: italic">INO</span> have revealed the presence of specific regulatory elements in the promoter sequence, and have allowed the identification of transcription factors that may be involved in <span style="font-style: italic">INO</span> expression. Identification of ovule regulatory genes in Arabidopsis has enabled the isolation of orthologous genes from other species with divergent ovule morphologies, helping to illuminate the developmental basis of this diversification. Supported by grants from U. S. National Science Foundation &#40;IBN&#45;0419531, MCB&#45;0517104&#41;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulation of Flowering Time by a Protein Arginine Methyltransferase, AtPRMT10</div>

<div>

	<span style="text-decoration: underline" >
	Falong Lu</span>
	<sup>1,2</sup>, 

	<span  >
	Lifang Niu</span>
	<sup>1,2</sup>, 

	<span  >
	YanXi Pei</span>
	<sup>1,3</sup>, 

	<span  >
	ChunYan Liu</span>
	<sup>1</sup>, 

	<span  >
	XiaoFeng Cao</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>State Key Laboratory of Plant Genomics and Center for Plant Gene Research, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, 100101, China, 

	<sup>2</sup>Graduate School, Chinese Academy of Sciences, Beijing, 100039, China, 

	<sup>3</sup>College of Life Science and Technology, Shanxi University, Taiyuan, China, 030006.

</div>

<div>Histone methylation is involved in modulation of chromatin structure, gene expression, heterochromatin formation, imprinting, genome stability and other important cellular processes. In plants, it has been reported that histone H3 methyltransferases play important roles in gene silencing and developmental regulation. However, little histone H4 methyltransferase has been identified and roles of histone H4 methylation in plant development remain to be elucidated. Here, we purified a histone methyltransferases from cauliflower by a conventional biochemical approach. PHRMT10, a dimerized plant specific protein arginine methyltransferase, was identified to methylate histone H4 specifically <span style="font-style: italic">in vivo</span>. <span style="font-style: italic">Arabidopsis thaliana</span> protein arginine methyltransferase 10 &#40;AtPRMT10&#41;, the <span style="font-style: italic">Arabidopsis</span> homolog of PHRMT10, was demonstrated to be a type I protein arginine methyltransferase which preferentially methylate histone H4 at R3 <span style="font-style: italic">in vitro</span>. Genetic disruption of AtPRMT10 resulted in late flowering by the up&#45;regulated <span style="font-style: italic">FLOWERING LOCUS C</span> &#40;<span style="font-style: italic">FLC</span>&#41; transcriptional level, indicating that wild&#45;type function of <span style="font-style: italic">AtPRMT10</span> is required for promoting flowering in <span style="font-style: italic">Arabidopsis</span> through repressing the expression of <span style="font-style: italic">FLC</span>. This work adds an extra layer of complexity in flowering time regulation and also sheds light on the importance of arginine methylation in plant development. </div>

</div>





<div class="abs">

<div style="font-weight:bold">The use of FRUITFULL versions with modified transcriptional activity reveals <br/>the possible role of FUL in meristem maintenance<br/></div>

<div>

	<span style="text-decoration: underline" >
	Vicente Balanza</span>
	<sup>1</sup>, 

	<span  >
	Shussei Sato</span>
	<sup>2</sup>, 

	<span  >
	Martin Yanofsky</span>
	<sup>2</sup>, 

	<span  >
	Cristina Ferrandiz</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Instituto de Biologia Molecular y Celular de Plantas IBMCP (CSIC-UPV), Valencia, Spain, 

	<sup>2</sup>Department of Biology and Center for Molecular Genetics, University of California San Diego, La Jolla, CA 92093-0116, USA

</div>

<div>FRUITFULL is an Arabidopsis MADS box gene involved in multiple aspects of plant development. FUL controls flowering time and cauline leaf development, acts redundantly with APETALA1 and CAULIFLOWER in the specification of floral meristem identity and plays a major role in carpel and fruit development. To gain further insights on FUL mechanisms of action and to identify FUL molecular targets, we have generated new alleles of FUL with modified transcriptional activity by traductional fusion of the FUL coding sequence to the VP16 activation domain under the control of FUL regulatory sequences. The phenotypes of the transgenic plants have an effect on different aspects of plant development. The FUL:VP16 plants show altered shoot apical meristem phenotypes. FUL:VP16 lines, after initiating between 10 and 30 normal flowers, form a terminal structure at the shoot apex consisting of a spiral arrangement of &#62;15 stamens and carpels. FUL:VP16 also show weak floral meristem indeterminacy and strongly enhances <span style="font-style: italic">agamous</span>, <span style="font-style: italic">clavata</span> or <span style="font-style: italic">crabs claw</span> indeterminacy phenotypes. Recently generated FUL:EAR plants show phenotype in the shoot apical meristem too. FUL:EAR shoot apical meristems often terminate producing 4&#45;5 flowers and a terminal carpel. A possible role of FUL in the regulation of meristem maintenance under the view of current models will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Involvement of the Histone Acetyltransferase AtHAC1 in the Regulation of Flowering Time via Repression of <span style="font-style: italic">FLC</span> in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Weiwei Deng</span>
	<sup>1</sup>, 

	<span  >
	Chunyan Liu</span>
	<sup>1</sup>, 

	<span  >
	Yanxi Pei</span>
	<sup>2</sup>, 

	<span  >
	Xian Deng</span>
	<sup>1</sup>, 

	<span  >
	Lifang Niu</span>
	<sup>1</sup>, 

	<span  >
	Xiaofeng Cao</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>State Key Laboratory of Plant Genomics and Center for Plant Gene Research, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China, 

	<sup>2</sup>College of Life Science and Technology, Shanxi University, Taiyuan, China

</div>

<div>Histone acetylation is an important post&#45;translational modification correlated with gene activation. In <span style="font-style: italic">Arabidopsis</span>, the histone acetyltransferase AtHAC1 is homologous to animal p300&#47;CBP &#40;CREB&#45;binding protein&#41; proteins, which are the main histone acetyltransferases participating in many physiological processes, including proliferation, differentiation and apoptosis. The functions of p300&#47;CBP in animals are well characterized, whereas little is known about the roles of AtHAC1 in developmental control in <span style="font-style: italic">Arabidopsis</span>. Lesions in <span style="font-style: italic">AtHAC1</span> caused pleiotropic developmental defects, including delayed flowering, a shortened primary root and partially reduced fertility. Analysis of the molecular basis of late&#45;flowering in <span style="font-style: italic">hac1</span> mutants showed that the <span style="font-style: italic">hac1</span> plants respond normally to day&#45;length, gibberellin acid &#40;GA&#41; treatment and vernalization. Furthermore, the expression level of the flowering repressor <span style="font-style: italic">FLOWERING</span> <span style="font-style: italic">LOCUS</span> <span style="font-style: italic">C</span> &#40;<span style="font-style: italic">FLC</span>&#41; is increased in <span style="font-style: italic">hac1</span> mutants, indicating that the late&#45;flowering phenotype of <span style="font-style: italic">hac1</span> mutants is mediated by <span style="font-style: italic">FLC</span>. Since histone acetylation is usually associated with the activation of gene expression, histone modifications of <span style="font-style: italic">FLC</span> chromatin are not affected by mutations in <span style="font-style: italic">HAC1</span> and expression levels of all known autonomous pathway genes are unchanged in <span style="font-style: italic">hac1</span> plants, we propose that HAC1 affects flowering time by epigenetic modification of factors upstream of <span style="font-style: italic">FLC</span>. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Mutations in the Type II protein arginine methyltransferase AtPRMT5 result in pleiotropic developmental defects in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	LiFang Niu</span>
	<sup>1,2</sup>, 

	<span  >
	YanXi Pei</span>
	<sup>1,3</sup>, 

	<span  >
	FaLong Lu</span>
	<sup>1,2</sup>, 

	<span  >
	ChunYan Liu</span>
	<sup>1</sup>, 

	<span  >
	JiXian Zhai</span>
	<sup>1,2</sup>, 

	<span  >
	XiangFeng Kong</span>
	<sup>1,2</sup>, 

	<span  >
	XiaoFeng Cao</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>State Key Laboratory of Plant Genomics and Center for Plant Gene Research, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China 100101, 

	<sup>2</sup>Graduate School, Chinese Academy of Sciences, Beijing, China, 100039, 

	<sup>3</sup>College of Life Science and Technology, Shanxi University, Taiyuan, China, 030006

</div>

<div>Human <span style="font-style: italic">protein arginine methyltransferase 5</span> encodes a type II protein arginine methyltransferase and its orthologs in animals and yeast have been shown to play important roles in the regulation of RNA processing, signal transduction and gene expression. However, PRMT5 homologs in higher plants have not yet been reported and biological significance of these proteins in plant development remains elusive. Here, using conventional biochemical approach, we purified a plant histone arginine methyltransferase from cauliflower that was nearly identical to AtPRMT5, an <span style="font-style: italic">Arabidopsis</span> homolog of human PRMT5. AtPRMT5 symmetrically di&#45;methylated histone H4R3, H2A and myelin basic protein &#40;MBP&#41; <span style="font-style: italic">in vitro</span>. Mutations in <span style="font-style: italic">AtPRMT5</span> caused pleiotropic developmental defects, including growth retardation, dark green and curled leaves, and <span style="font-style: italic">FLC</span>&#45;dependent delayed flowering. This is the first time demonstration that the type II protein arginine methyltransferase is involved in repression of vegetative growth and <span style="font-style: italic">FLC</span>&#45;dependent flowering time in <span style="font-style: italic">Arabidopsis</span>.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Monitoring root elongation and lateral root generation in <span style="font-style: italic">Arabidopsis</span> via regulating <span style="font-style: italic">GNOM</span> expression by an inducible anti&#45;sense system</div>

<div>

	<span style="text-decoration: underline" >
	Jingzhe Guo</span>
	<sup>1</sup>, 

	<span  >
	Jun Wei</span>
	<sup>1</sup>, 

	<span  >
	Mengxiang Sun</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Key Laboratory of MOE for Plant Developmental Biology, College of Life Sciences, Wuhan University, Wuhan 430072, China

</div>

<div>The <span style="font-style: italic">Arabidopsis GNOM</span> gene encodes an ARF GDP&#47;GTP exchange factor. Studies of strong <span style="font-style: italic">gnom</span> mutants show that GNOM is involved in embryonic axis formation and polar localization of the auxin efflux carrier PIN1. Subsequent analysis of weak <span style="font-style: italic">gnom</span> mutants also reveals its important role in post&#45;embryogenesis. We established a <span style="font-style: italic">GNOM</span> inducible anti&#45;sense system to further study the role of GNOM during plant growth. It enables to suppress <span style="font-style: italic">GNOM</span> expression in specific tissues and at specific stage of plant development. Thus it offers a unique opportunity to seek how auxin transport and polar distribution could regulate root generation and development in terms of their normal pattern and level are temporally and spatially interrupted.&#13;<br/>Our data show that root elongation of induced seedlings was obviously less than that of control, and the main reason for it is that the root cell elongation in root differentiation zone was greatly inhibited, but not the meristem activity was reduced. Such inducible phenomenon is found dose&#45;dependent. When grew on plates containing increasing concentration of NAA, inhibition of root elongation in induced seedlings was getting severer, but this was not the case for 2, 4&#45;D. This confirms that GNOM does regulate auxin efflux transport. However, when inducer was removed from medium, normal root elongation was observed again. The cell elongation can restore similar to wide type after withdrawing the inducer. Thus&#65292;root elongation was reversibly controlled by regulating <span style="font-style: italic">GNOM</span> expression and interrupting the auxin distribution temporally and spatially. Lateral roots formation was also greatly inhibited when <span style="font-style: italic">GNOM</span> was down regulated. However, there was almost no emergence of lateral roots after withdrawing the inducer. The lateral roots could only generate again in the presence of exogenous NAA.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Downregulation of <span style="font-style: italic">RETINOBLASTOMA&#45;RELATED PROTEIN &#40;RBR&#41;</span> expression in <span style="font-style: italic">A. thaliana</span> leads to enhanced cell division and defects in cell specification</div>

<div>

	<span style="text-decoration: underline" >
	Lorenzo Borghi</span>
	<sup>1</sup>, 

	<span  >
	Ruben Gutzat</span>
	<sup>1</sup>, 

	<span  >
	Johannes F&#252;tterer</span>
	<sup>1</sup>, 

	<span  >
	Wilhelm Gruissem</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Sciences, ETH Z&#252;rich, Switzerland

</div>

<div>In animal cells, the RETINOBLASTOMA protein &#40;pRB&#41; plays a key role in controlling the transition from G1 to S phase by modifying the activity of E2F&#47;DP transcription factor complexes. In <span style="font-style: italic">A. thaliana</span>, RBR seems to perform a similar function &#40;1,2,3&#41;. Here we report that a constitutive down regulation of <span style="font-style: italic">RBR</span> expression via RNA interference &#40;RNAi&#41; or co&#45;suppression leads to enhanced cell division during embryogenesis and altered post embryonic development.  Cloning of an RNAi inducible system against <span style="font-style: italic">RBR</span>, based on the pOp&#47;LhG4 binary system &#40;4&#41;, gave us a tool to perturb <span style="font-style: italic">RBR</span> expression levels at later stages of plant development. Interestingly, newly formed organs after RNAi induction are characterized by disturbed control of cell division and organ growth and also show defects in certain differentiation pathways. Our preliminary results are presented here.&#13;<br/>&#13;<br/>&#13;<br/>1. J Virol. 2007 Apr;81&#40;8&#41;:4177&#45;85&#13;<br/>2. Plant Physiol. 2006 Jan;140&#40;1&#41;:67&#45;80&#13;<br/>3. Cell. 2005 Dec 29;123&#40;7&#41;:1337&#45;49&#13;<br/>4. Plant J. 2005 Mar;41&#40;6&#41;:899&#45;918&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Activation of the TAS3&#45;derived tasiRNA pathway in the root system of Arabidopsis thaliana.</div>

<div>

	<span style="text-decoration: underline" >
	Alexis Maizel</span>
	<sup>1</sup>, 

	<span  >
	Martin Crespi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CNRS Plant Science Institute, Gif-sur-Yvette, France

</div>

<div>Plant and animals use small RNAs &#40;microRNAs and siRNAs&#41; as guide for post&#45;transcriptional and epigenetic regulation. In plants, miRNAs and trans&#45;acting siRNA &#40;tasiRNA&#41; result from different biogenesis pathways but both interact with target transcripts to direct their cleavage. Three ta&#45;siRNA gene families &#40;TAS1, TAS2, and TAS3&#41; are known in Arabidopsis thaliana. TAS gene transcripts are cleaved by miRNAs; the cleavage products are copied into dsRNA by RDR6, and diced into tasiRNAs by DCL4. Biogenesis of TAS3&#45;derived tasiRNAs involves miR390 and they target mRNAs encoding AUXIN RESPONSE FACTORs ARF3&#47;ETTIN and ARF4. Specific degradation of ARF3 in the leaves by TAS3&#45;derived tasiRNA is critical for leaf development and phase transition. In a screen for large non&#45;protein coding RNAs in Arabidopsis using a dedicated micro&#45;array, we identified that TAS3 gene is highly expressed in root tissues. Using reporter constructs for TAS3 and miR390 loci as well as analysis of the accumulation of their derived RNAs, we have characterized the expression pattern of the TAS3 pathway during root development. We present evidence that this pathway can be linked to the architecture of the root system.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular mechanism in the control of floral dorsoventral and organ internal asymmetries during zygomorphic flower development in <span style="font-style: italic">Lotus japonicus</span></div>

<div>

	<span style="text-decoration: underline" >
	Jiechen Wang</span>
	<sup>1</sup>, 

	<span  >
	Zhigang Cai</span>
	<sup>1</sup>, 

	<span  >
	Yonghai Luo</span>
	<sup>1</sup>, 

	<span  >
	Shilei Xu</span>
	<sup>1</sup>, 

	<span  >
	Lin Weng</span>
	<sup>1</sup>, 

	<span  >
	Jun Yang</span>
	<sup>1</sup>, 

	<span  >
	Luo Da</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Key Laboratory of Plant Molecular Genetics, Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Graduate School of the Chinese Academy of Sciences, Chinese Academy of Sciences, Shanghai, China

</div>

<div>   In a model legume <span style="font-style: italic">Lotus japonicus</span> the zygomorphic flower consists of three types of petals: a dorsal bilateral&#45;symmetric petal, two asymmetric lateral petals and two asymmetric ventral petals. Two types of asymmetry in the flower could be identified: the floral dorsoventral asymmetry along the plan of the flower and the organ asymmetry in the plan of petals. This raises the questions about the genetic factors in the control of these two types of asymmetry and the mechanism coupling them to form handedness symmetry between a pair of lateral or ventral petals. It has been showed that in a distinct related model plant, <span style="font-style: italic">A. majus</span>, development of zygomorphy is controlled by two <span style="font-style: italic">TCP</span> genes, <span style="font-style: italic">CYC</span> and <span style="font-style: italic">DICH</span>. Our previous work showed that a <span style="font-style: italic">CYC</span> homologue, <span style="font-style: italic">LjCYC2</span>, plays a key role in the development of zygomorphy in <span style="font-style: italic">Lotus japonicus</span>, although the evolution of zygomorphy should be independent among different species. Furthermore, a legume specific factor, <span style="font-style: italic">KEW1</span>, was identified, which plays a key role in the control of lateral petal identity.&#13;<br/>&#13;<br/>   This work has been investigating the function of <span style="font-style: italic">LjCYC1</span>, a close homologue of <span style="font-style: italic">LjCYC2</span> in Lotus japonicus, and the interaction between <span style="font-style: italic">LjCYC1&#47;LjCYC2</span> and <span style="font-style: italic">KEW</span>. We demonstrate that <span style="font-style: italic">LjCYC1</span> and <span style="font-style: italic">LjCYC2</span> play a redundant role in the control of the dorsal identity and regulate cell differentiation during petal development, and the function of <span style="font-style: italic">KEW1</span> was involved in the coupling of floral dorsoventral asymmetry with the organic asymmetry during zygomorphic development. Furthermore, knock&#45;down of the two <span style="font-style: italic">LjCYC</span> genes in <span style="font-style: italic">kew1</span> background revealed a default stage when the dorsal and lateral activities being abolished, indicating that there exit other genetic factors in the control of organic asymmetry during floral zygomorphic development in <span style="font-style: italic">Lotus japonicus.</span>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Evidence that plant miRNPs ribonucleoproteic complexes are associated with translated mRNAs in polysomes</div>

<div>

	<span style="text-decoration: underline" >
	Elodie Lanet</span>
	<sup>1</sup>, 

	<span  >
	Etienne Delannoy</span>
	<sup>1</sup>, 

	<span  >
	Rodnay Sormani</span>
	<sup>1</sup>, 

	<span  >
	Christophe Robaglia</span>
	<sup>1</sup>, 

	<span  >
	Patrice Cr&#233;t&#233;</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Laboratoire de Genetique et Biophysique des Plantes, Marseille, France

</div>

<div>MicroRNAs &#40;miRNAs&#41; regulate gene expression post&#45;transcriptionally by a conserved mechanism from plants to animals called RNA silencing. The common hypothesis is that most animal miRNAs affect gene expression by blocking mRNAs translation while most plant miRNAs trigger mRNAs cleavage prior to translation. However this view is certainly far from being complete. Therefore we are studying the link between the translational machinery and plant miRNAs.  <br/>Results: Cytoplasmic extract fractionation on sucrose gradient reveals that although miRNAs are found in light fractions from the top of the gradient, some miRNAs are linked with heavy fractions of polysomes. The level of co&#45;sedimentation varies for different miRNAs, and miR168 is strongly associated with polysomes. In contrast, miR168&#42;, the non&#45;functional partner of the miRNA in the duplex miR&#47;miR&#42;, is only found within light, untranslated, fractions. Interestingly, miR173, which triggers cleavage of the <span style="font-style: italic">TAS2</span> non coding RNA, is also weakly associated with polysomes. We conclude that the association of miRNAs to polysomes is dependent on the function of regulatory RNA and is linked to the presence of a translatable mRNA target. <br/>In addition, protein analysis reveals that Argonaute 1 &#40;AGO1&#41; protein is in part associated with polysomal fractions and that miR168 association with polysomes is lost in <span style="font-style: italic">ago1&#45;4</span> mutants. So we conclude that miRNAs are present in a ribonucleoprotein complex associated with polysomes. Further we show that association of miR168 with polysomes is sensitive to microccocal nuclease, suggesting that miRNAs association with polyribosomes is mRNA mediated. <br/>In conclusion this work shows that plant miRNAs, like animal miRNAs, are associated with polysomes both in <span style="font-style: italic">Arabidopsis thaliana</span> cell culture and plants. This association depends on a ribonucleoproteic complex called miRNP, containing at least intact miRNA and AGO1. Further work will address the effect of miRNAs on mRNA translation and provide a better understanding of the spatio&#45;temporal relationship between RNA cleavage and translational regulation.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The RETINOBLASTOMA&#45;RELATED PROTEIN &#40;RBR&#41; interacts with sucrose and auxin signalling to control cell division and cell differentiation.</div>

<div>

	<span style="text-decoration: underline" >
	Yec'han Laizet</span>
	<sup>1</sup>, 

	<span  >
	Ruben Gutzat</span>
	<sup>1</sup>, 

	<span  >
	Johannes F&#252;tterer</span>
	<sup>1</sup>, 

	<span  >
	Wilhelm Gruissem</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ETH - Institute of Plant Sciences, Z&#252;rich, Switzerland

</div>

<div>Plant organogenesis in shoot and root meristems relies on a tightly regulated balance between cell division and cell differentiation. One of the key regulators of this process is the Retinoblastoma&#45;related protein &#40;RBR&#41;, acting at least in part through the DP&#47;E2F pathway to regulate cell cycle activity &#40;reviewed in 1&#41;.<br/>Arabidopsis plants carrying a constitutive <span style="font-style: italic">RBR</span> loss&#45;of&#45;function allele are gametophytic lethal. To investigate the role of RBR during post&#45;embryonic development, we constructed a conditional <span style="font-style: italic">RBR</span> loss&#45;of&#45;function allele, using a CRE&#45;lox system inducible by heat shock &#40;2&#41;. Excision of <span style="font-style: italic">RBR</span> after induction is detectable through the transcriptional activation of a green fluorescent protein &#40;GFP&#41; reporter.<br/>In plants induced 3 days after germination, loss of <span style="font-style: italic">RBR</span> triggers defects in cell division and differentiation in leaves: epidermal cells do not acquire the typical jig&#45;saw shape; trichomes and stomata hardly differentiate. Reiterative heat shocks arrest organ production, and previously initiated leaves show additional cell division. <br/>Growth signals like sucrose and hormones are known to play a role in cell division control. In <span style="font-style: italic">rbr&#45;</span> roots, the hormone auxin together with sucrose triggers a strong cell hyper&#45;proliferation. Moreover, these roots do not show the typical differentiation pattern but rather develop into undifferentiated callus tissue. A similar hyperproliferation is observed in <span style="font-style: italic">RBR</span> co&#45;supression line grown on sucrose and gene expression analysis reveal that genes involved in cell cycle regulation are deregulated.<br/>Our results confirm the role of RBR on cell division control, but also show that organ development requires RBR for a proper cell differentiation.<br/><br/><span style="font-style: italic">1. Plant Molecular Biology &#40;2006&#41; 60:947&#8211;961.</span><br/><span style="font-style: italic">2. Cell. 2005 Dec 29;123&#40;7&#41;:1337&#45;49.</span><br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Senescence associated members of the ALA gene family in Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Tilbert Kosmehl</span>
	<sup>1</sup>, 

	<span  >
	Alexis Kasaras</span>
	<sup>1</sup>, 

	<span  >
	Christine Rausch</span>
	<sup>1</sup>, 

	<span  >
	Reinhard Kunze</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Freie Universit&#228;t Berlin, Institut f&#252;r Biologie / Angewandte Genetik, Albrecht-Thaer-Weg 6, 14195 Berlin, Germany

</div>

<div>Senescence is the genetically programmed last phase of leaf development and responsible for the recycling of nutrients before the cells die. It is a highly regulated process that involves complex morphological and metabolic changes and extensive alterations in gene expression. The mobilization of metabolites and nutrients out of senescing tissues into juvenile, storage or reproductive organs is mostly dependent on the activity of transmembrane transporters. However, to date only very few senescence&#45;associated transporters &#40;SATPs&#41; have been reported. By a genome&#45;wide transcriptional analysis of Arabidopsis thaliana developmental leaf senescence many novel, previously unrecognized, SATPs were identified.<br/>Among these are three putative aminophospholipid translocases that show increased transcript levels in rosette leaves during senescence &#40;ALA1, ALA10 and ALA11&#41;. They belong to the P4&#45;type ATPases, a new subfamily of the P&#45;type ATPase ion pump superfamily with 12 members. ALA10 and ALA11 are highly similar &#40;86&#37; identity&#41;, whereas ALA1 is more distantly related &#40;33&#37; identity&#41;.<br/>T&#45;DNA insertion lines for ALA1, 10 and 11 were identified and, to recognize overlapping or redundant functions, double and triple mutant lines were generated. Mutant lines ALA1 and 11 exhibit weakly reduced growth and enter senescence slightly earlier compared to wild type, whereas the ALA10 mutant line shows no obvious phenotype. The ALA10&#47;11 double mutant line starts to senesce earlier than wild type and shows a stunted growth. Remarkably, in the ALA1&#47;10&#47;11 triple mutant line premature senescence is even more pronounced and accompanied by developmental abnormalities. In all mutant lines cold treatment promotes precocious senescence and reduced growth, whereas these symptoms disappear when plants grow at elevated temperatures. These characteristics are consistent with a role of the ALA proteins in membrane reorganization by transport of phospholipids or modification of the lipid bilayer composition during development and stress response.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Overexpression of <span style="font-style: italic">INFLORESCENCE DEFICIENT IN ABSCISSION</span> Activates Ectopic Abscission</div>

<div>

	<span style="text-decoration: underline" >
	Grethe-Elisabeth Stenvik</span>
	<sup>1</sup>, 

	<span  >
	Melinka A. Butenko</span>
	<sup>1</sup>, 

	<span  >
	Sarah M. McKim</span>
	<sup>2</sup>, 

	<span  >
	George W. Haughn</span>
	<sup>2</sup>, 

	<span  >
	Reidunn B. Aalen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biosciences, University of Oslo, Norway, 

	<sup>2</sup>Department of Botany, University of British Columbia, Canada

</div>

<div>Abscission is an active process that enables plants to shed unwanted organs. Because the purpose of the flower is to facilitate pollination, it often is abscised after fertilization. We have identified a floral organ abscission mutant in <span style="font-style: italic">Arabidopsis</span>, <span style="font-style: italic">inflorescence deficient in abscission</span> &#40;<span style="font-style: italic">ida</span>&#41; which shows a complete lack of floral organ abscission. The <span style="font-style: italic">IDA</span> gene encodes a small protein with an N&#45;terminal signal peptide, suggesting that the IDA protein is a ligand of an unknown receptor involved in controlling the final step of cell separation of floral organs &#40;Butenko et al, 2003 Plant Cell 15&#41;. Arabidopsis <span style="font-style: italic">35S:IDA</span> lines constitutively overexpressing IDA exhibit earlier abscission of floral organs, which shows that the abscission zones are responsive to IDA soon after the opening of the flowers &#40;Stenvik et al, 2006 Plant Cell 18&#41;. In addition, ectopic abscission was observed at the bases of the pedicel, branches of the inflorescence, and cauline leaves. In regions at the bases of the organs lost in the <span style="font-style: italic">35S:IDA</span> lines differentiated abscission zone &#40;AZ&#41; cells were present in wild&#45;type plants. By transforming the floral abscission deficient mutant <span style="font-style: italic">bop1&#47;bop2</span>, which does not develop a floral or cauline leaf AZ, with the <span style="font-style: italic">35S:IDA</span> construct we show that differentiation of AZs is crucial for IDA function. Scanning electron microscopy of <span style="font-style: italic">35S:IDA</span> indicated a spread of middle lamella degradation from preformed abscission zone cells to neighboring cells. In addition, a white substance, identified as arabinogalactan, was secreted in large amounts at the sites of abscission and a transcript encoding an arabinogalactan protein &#40;AGP24&#41; was upregulated in the <span style="font-style: italic">35S:IDA</span> lines. We suggest that the restricted expression pattern of IDA to the floral AZ precludes abscission of non&#45;floral organs in Arabidopsis and that plant species that shed organs other than petals, sepals, and anthers have a different expression pattern of putative IDA orthologs.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Manipulating cell cycle components during male gametogenesis using pollen cell&#45;specific molecular tools</div>

<div>

	<span style="text-decoration: underline" >
	Said Hafidh and David Twell Department of Biology, Univers</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>UNIVERSITY OF LEICESTER

</div>

<div>Double fertilization is one of the defining characteristics of angiosperms and depends upon the production of twin sperm cells. The formation of two sperm cells is controlled by two division events that are essential for correct cell fate specification and succesful fertilization. We are interested in identifying core cell cycle genes involved in the control of cell division during male gametogenesis. However, lack of tools for manipulating gene expression during pollen development has been a hinderance for the study of gene function in the male germline. <br/><br/>RNA interference &#40;RNAi&#41; is currently been used as an approach for studying gene function. This pathway is conserved in eukaryotes mediating sequence specific transcriptional silencing and degradation of cytoplasmic RNA through short intefering &#40;si&#41; RNAs. We showed that some of the core components involved, <span style="font-style: italic">Argounate</span>&#45;like &#40;AGO&#8217;s&#41; and <span style="font-style: italic">Dicer</span>&#45;like &#40;DCL&#41; genes, are expressed throughout pollen development.<br/>We have developed pollen cell&#45;specific molecular tools for the production of hairpin double stranded RNA to down regulate target genes, and pollen cell&#45;specific over&#45;expression vectors. <br/><br/>The progression through G2&#47;M transition is regulated by different classes of B&#45;type cyclins in combination with CDKA. To identify the B&#45;type cyclins potentially involved in sperm cell formation, we have applied the tools we have generated to manipulate expression of <span style="font-style: italic">Cyclin B1</span> family during pollen development. Our data support an essential role of AtCycB1 family for the completion of pollen mitosis I and pollen mitosis II. Moreover, using <span style="font-style: italic">AtCycB1;1</span> translational&#45;reporter fusion construct, we could show cycling of CYCB1;1 protein during the two pollen cell division events, highlighting the existence of anaphase promoting complex activity in the male gametophyte.<br/>We further illustrate that <span style="font-style: italic">AtDUO1</span>, an R2R3&#45;MYB transcription factor regulating cell fate specification, is also required for <span style="font-style: italic">AtCycB1;1</span> expression in the generative cell. Therefore, we demonstrate an important link between cell fate specification and core cell cycle components regulating G2&#47;M transition in the plant male germ line.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification and Characterization of the Arabidopsis Orthologs of Nuclear Transport Factor 2, the Nuclear Import Factor of Ran</div>

<div>

	<span style="text-decoration: underline" >
	Qiao Zhao</span>
	<sup>1</sup>, 

	<span  >
	Sara Leung</span>
	<sup>2</sup>, 

	<span  >
	Anita H. Corbett</span>
	<sup>2</sup>, 

	<span  >
	Iris Meier</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Cellular and Molecular Biology and Plant Biotechnology Center, The Ohio State University, Columbus, OH, USA, 

	<sup>2</sup>Department of Biochemistry, Emory University School of Medicine, Atlanta, GA, USA

</div>

<div>Ran is a multifunctional small GTPase that is involved in nucleocytoplasmic transport, mitotic spindle assembly, and nuclear envelope formation. Nuclear import of Ran relies on a small RanGDP&#45;binding protein, Nuclear Transport Factor 2 &#40;NTF2&#41;. Three proteins are expressed in Arabidopsis &#40;Arabidopsis thaliana&#41; that show significant sequence similarity to human and yeast &#40;Saccharomyces cerevisiae&#41; NTF2. Here, we demonstrate that two of them, AtNTF2a and AtNTF2b, can functionally replace the essential NTF2 gene in yeast. Consistent with this finding, both AtNTF2a and AtNTF2b interact with yeast and Arabidopsis Ran. The third NTF2&#45;related protein, AtNTL, does not functionally replace NTF2 in yeast. Similar to yeast NTF2&#45;green fluorescent protein &#40;GFP&#41;, AtNTF2a&#45;GFP and AtNTF2b&#45;GFP accumulate at the nuclear rim. We find that a putative Arabidopsis FG&#45;repeat nucleoporin &#40;NUP&#45;FG&#41; can interact with NTF2a as shown by co&#45;IP in Nicotiana benthamiana. A GFP&#45;NUP&#45;FG fusion protein is located at the nuclear rim in transgenic Arabidopsis. We are exploring the function of NUP&#45;FG in targeting NTF2 to the Arabidopsis nuclear pore.<br/><br/>AtNTF2a E38K and E91K mutants, which fail to bind Ran, are not functional in yeast, indicating conservation of the requirement for these key amino acids in plants and yeast. AtNTF2a overexpression, but not AtNTF2aE38K overexpression, blocks nuclear import of a plant transcription factor in Nicotiana benthamiana leaves, indicating that excess AtNTF2a disrupts nuclear import in a Ran&#45;binding dependent manner. On the basis of these results, we propose that AtNTF2a and AtNTF2b function in Ran import in Arabidopsis and that nuclear import of Ran is functionally conserved in plants.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Analysis of the &#8217;&#8217;BRC1&#8217;&#8217; gene promoter</div>

<div>

	<span style="text-decoration: underline" >
	Jos&#233; Antonio Aguilar-Mart&#237;nez</span>
	<sup>1</sup>, 

	<span  >
	Pilar Cubas</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dpto. Gen&#233;tica Molecular de Plantas, Centro Nacional de Biotecnolog&#237;a-CSIC, UAM, Madrid, Spain

</div>

<div>Plant branching patterns depend mainly on the control of axillary meristem development. Axillary meristems form axillary buds that can grow out to generate new shoots or become arrested, depending on endogenous and environmental factors. We have shown that the Arabidopsis TCP transcription factor &#8217;&#8217;BRANCHED1&#8217;&#8217; &#40;&#8217;&#8217;BRC1&#8217;&#8217;&#41; controls axillary bud development and acts as an integrator of branching signals within axillary buds &#40;1&#41;. We have also shown that &#8217;&#8217;BRC1&#8217;&#8217; transcription is affected by developmental and external signals &#40;1&#41;. To identify important genomic regions responsible for the regulation of &#8217;&#8217;BRC1&#8217;&#8217; we have carried out a promoter deletion analysis and a phylogenetic comparison of 5&#180; flanking regions of the &#8217;&#8217;BRC1&#8217;&#8217; gene in several Brassicaceae species. Results obtained in this study have been used as a starting point to perform a one&#45;hybrid screening to look for upstream regulators of &#8217;&#8217;BRC1&#8217;&#8217;. <br/>1 Aguilar&#45;Mart&#237;nez et al. 2007 Plant Cell 19:458&#45;472<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A Cre&#47;loxP&#45;Mediated Cell Autonomous Gene Excision System to Produce Marker Gene Free Transgenic Soybean Plants</div>

<div>

	<span style="text-decoration: underline" >
	Zhongsen Li</span>
	<sup>1</sup>, 

	<span  >
	Aiqiu Xing</span>
	<sup>1</sup>, 

	<span  >
	Bryan P. Moon</span>
	<sup>1</sup>, 

	<span  >
	Susan A. Burgoyne</span>
	<sup>1</sup>, 

	<span  >
	Anthony D. Guida</span>
	<sup>1</sup>, 

	<span  >
	Huiling Liang</span>
	<sup>1</sup>, 

	<span  >
	Catharina Lee</span>
	<sup>1</sup>, 

	<span  >
	Cheryl S. Caster</span>
	<sup>1</sup>, 

	<span  >
	Theodore M. Klein</span>
	<sup>1</sup>, 

	<span  >
	Joanne E. Barton</span>
	<sup>1</sup>, 

	<span  >
	Saverio C. Falco</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Pioneer, a DuPont Company, Wilmington, Delaware, USA

</div>

<div>Marker&#45;gene&#45;free transgenic soybean plants were produced by isolating a developmentally regulated embryo&#45;specific gene promoter app1 from Arabidopsis and developing a cell autonomous gene excision system using the P1 bacteriophage Cre&#47;loxP recombination system. To accomplish this, the Cre recombinase gene was placed under control of the app1 promoter and, together with a selectable marker gene &#40;hygromycin phosphotransferase&#41;, were cloned between two loxP recombination sites. This entire sequence was then placed between a constitutive promoter and a coding region for either &#946;&#45;glucuronidase &#40;Gus&#41; or glyphosate acetyltransferase &#40;Gat&#41;. Gene excision would remove the entire sequence bewteen the two loxP sites and bring the Gus or Gat coding region to the constitutive promoter for expression. Using this system marker gene excision occurred in over 30&#37; of the stable transgenic events as indicated by the activation of the Gus reporter gene or the Gat gene in separate experiments. Transgenic plants that have 1 or 2 copies of a functional excision&#45;activated gat transgene and are free of any marker gene were obtained in T0 or T1 generation. This demonstrates the feasibility of using developmentally controlled promoters to mediate marker excision in soybean.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A role for LMI2, a MYB transcription factor, in the Meristem Identity Transition</div>

<div>

	<span style="text-decoration: underline" >
	Jennifer Pastore</span>
	<sup>1</sup>, 

	<span  >
	Natasha Chavdaroff</span>
	<sup>1</sup>, 

	<span  >
	Doris Wagner</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Pennsylvania, Philadelphia, U.S.A.

</div>

<div>Plants go through distinct developmental phases throughout their life cycle.  The transition from vegetative to reproductive phase, known as the meristem identity transition, is vital to species survival and important for our economical and agricultural resources.   In <span style="font-style: italic">Arabidopsis thaliana</span>, this transition is initiated by the plant specific transcription factor LEAFY &#40;LFY&#41;. We have shown that LFY directly activates two other meristem identity regulators APETALA1 &#40;AP1&#41; and CAULIFLOWER &#40;CAL&#41;.  Using a microarray approach combined with chromatin immunoprecipitation &#40;ChIP&#41; and RT&#45;PCR, our lab has identified five additional direct targets of LFY that are candidate meristem identity regulators.  One of these targets is a MYB transcription factor known as MYB17 &#40;At3g61250&#41;.  MYB transcription factors have been identified in all major eukaryotic organisms.  In animals, their function is to control cell differentiation and proliferation.  In plants, their role is more diverse, with roles in processes including secondary metabolism, cellular morphogenesis, signal transduction in growth, abiotic stress and pathogen defense.   <br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Is MYB17 involved in the meristem identity transition&#63; To address this question we performed phenotypic analyses using two T&#45;DNA insertion lines.  Mutations in MYB17 cause the strongest meristem identity phenotype of all the newly identified LFY targets.  The severity of the single mutant phenotype is similar to that of AP1 loss&#45;of&#45;function mutants. We named this meristem identity regulator LMI2 &#40;<span style="text-decoration: underline">L</span>ate <span style="text-decoration: underline">M</span>eristem <span style="text-decoration: underline">I</span>dentity 2&#41;.  Preliminary expression analysis places LMI2 upstream of AP1.  Phenotypic analysis of <span style="font-style: italic">ap1lmi2</span> double mutants indicates both genes likely act in the same pathway.  These and additional findings that will be presented demonstrate that LMI2 plays an important role in the meristem identity pathway.<br/><br/><br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The NAC transcription factor GORGON enhances the pid&#45;8 phenotype in Arabidopsis<br/></div>

<div>

	<span style="text-decoration: underline" >
	Wei Li</span>
	<sup>1</sup>, 

	<span  >
	Hicham Zegzouti</span>
	<sup>1</sup>, 

	<span  >
	Mingtang Xie</span>
	<sup>2</sup>, 

	<span  >
	Roman Deniskin</span>
	<sup>1</sup>, 

	<span  >
	Sioux K. Christensen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University Of California-Los Angeles, Los Angeles, CA, USA, 

	<sup>2</sup>University Of California, Riverside, CA, USA

</div>

<div>Auxin is an essential phytohormone that regulates numerous mitogenic and morphogenic processes during plant development. PINOID &#40;PID&#41;, a previously identified plant specific serine&#47;threonine kinase, has been shown to regulate auxin signaling. In Arabidopsis, among the phenotypes associated with the strong allele of pinoid &#40;pid&#45;9&#41; is a pin&#45;like inflorescence characterized by the absence of buds and flowers, and the weak allele pid&#45;8 produces sterile flowers and other various auxin signaling or transport related abnormalities rather than pins. To identify other genes affecting the establishment or maintenance of polar auxin flow, we generated an activation tagged insertion lines in the pid&#45;8 background to isolate pinoid enhancer mutants. Transformed plants were screened for the pin&#45;like inflorescence characteristic of pid strong alleles phenotype. Gorgon &#40;gon&#45;ID&#41; was identified as a dominant enhancer of the pid&#45;8 phenotype that produces both pin and sterile flowers. Also, this mutant produced a thickened and often bent or twisted stems, reduced fertility, and delayed bolting and senescence. Sequences flanking the T&#45;DNA insertion have been cloned using plasmid rescue, and the ORF upstream of the insertion is identified as a gene that encodes a NAC transcription factor family of proteins. The GON gene was showed by RT&#45;PCR to be overexpressed in the gon&#45;1D mutant. The resulting phenotypes have been recapitulated by overexpressing the GON gene in pid&#45;8. Here, we present a histological examination of the mutant and the expression pattern of GON. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Role of <span style="font-style: italic">SVP</span> in the control of flowering time by ambient temperature in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Jeong Hwan Lee</span>
	<sup>1</sup>, 

	<span  >
	Seong Jeon Yoo</span>
	<sup>1</sup>, 

	<span  >
	Soo Hyun Park</span>
	<sup>1</sup>, 

	<span  >
	Ildoo Hwang</span>
	<sup>2</sup>, 

	<span  >
	Jong Seob Lee</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Plant Signaling Network Research Center, School of Life Sciences and Biotechnology, Korea University, Seoul, 136-701, Korea, 

	<sup>2</sup>Division of Molecular and Life Sciences, Pohang University of Science and Technology, Pohang 790-784, Korea, 

	<sup>3</sup>School of Biological Sciences, Seoul National University, Seoul, 151-742, Korea

</div>

<div>Plants must perceive and rapidly respond to changes in ambient temperature for their successful reproduction. Here we demonstrate that <span style="font-style: italic">Arabidopsis SHORT VEGETATIVE PHASE &#40;SVP&#41;</span> plays an important role in the response of plants to ambient temperature changes. The loss of <span style="font-style: italic">SV</span>P function elicited insensitivity to ambient temperature changes. <span style="font-style: italic">SVP</span> mediates the temperature&#45;dependent functions of <span style="font-style: italic">FCA</span> and <span style="font-style: italic">FV</span>E within the thermosensory pathway. SVP controls flowering time by negatively regulating the expression of a floral integrator, <span style="font-style: italic">FLOWERING LOCUS T &#40;FT&#41;</span>, via direct binding to the CArG motifs in the <span style="font-style: italic">FT</span> sequence. We propose that this is one of the molecular mechanisms that modulate flowering time under fluctuating temperature conditions.</div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">WEREWOLF</span>, a Regulator of Root Hair Pattern Formation, Controls Flowering Time through the Regulation of <span style="font-style: italic">FT</span></div>

<div>

	<span style="text-decoration: underline" >
	Eunjoo Seo</span>
	<sup>1</sup>, 

	<span  >
	Jihyeon Yu</span>
	<sup>1</sup>, 

	<span  >
	Kook Hui Ryu</span>
	<sup>2</sup>, 

	<span  >
	Myeong Min Lee</span>
	<sup>2</sup>, 

	<span  >
	Ilha Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Biological Sciences, Seoul National University, Seoul 151-742, Korea, 

	<sup>2</sup>Department of Biology, Yonsei University, Sinchon 134, Seoul 120-749, Korea

</div>

<div>In <span style="font-style: italic">Arabidopsis</span>, <span style="font-style: italic">FT</span> integrates floral stimuli from the photoperiod, the vernalization, and autonomous pathways to trigger the floral induction. Since <span style="font-style: italic">FT</span> performs a great important role for flowering, <span style="font-style: italic">FT</span> is strictly controlled by many genes: <span style="font-style: italic">CO</span>, <span style="font-style: italic">FLC</span>, <span style="font-style: italic">TFL2</span>, and <span style="font-style: italic">EBS</span>. This study demonstrates that the additional factor, <span style="font-style: italic">WEREWOLF</span>&#40;<span style="font-style: italic">WER</span>&#41; known as a regulator of root hair patterning, is involved in <span style="font-style: italic">FT</span> regulation. <span style="font-style: italic">wer&#45;1</span> mutants flower late in long days but not in short days, and they show weak sensitivity to vernalization in long days. It suggests that <span style="font-style: italic">WER</span> controls flowering time through the photoperiod pathway. Real&#45; time PCR and double mutant analysis show that <span style="font-style: italic">WER</span> modulates <span style="font-style: italic">FT</span> expression levels independent of <span style="font-style: italic">CO</span> and <span style="font-style: italic">FLC</span>; besides, <span style="font-style: italic">WER</span> affects on not transcription but stability of <span style="font-style: italic">FT</span> mRNA. However, <span style="font-style: italic">WER</span> transcripts were observed in epidermis in which <span style="font-style: italic">FT</span> mRNA does not exist. In conclusion, we insist that <span style="font-style: italic">WER</span> regulates <span style="font-style: italic">FT</span> mRNA stability in a non&#45;cell autonomous manner.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Targets of <span style="font-style: italic">CINCINNATA</span>, a gene involved in leaf morphogenesis in <span style="font-style: italic">Antirrhinum</span></div>

<div>

	<span style="text-decoration: underline" >
	Pooja Aggarwal</span>
	<sup>1</sup>, 

	<span  >
	Utpal Nath</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Indian Institute of Science, Bangalore (Karnataka), India

</div>

<div>Although much is known about how plant organs are patterned at the meristem by the action of developmental genes in specific domains, we are only beginning to understand how organs such as leaves, once initiated, grow to their final shape and size.  Since cell migration is absent from the plant, cell proliferation is likely to play a more important role in plant organogenesis than it does in animal.  The recently identified TCP family of transcription factors has a key role in controlling organ morphogenesis through regulation of cell division.  For instance, it has been shown in <span style="font-style: italic">Antirrhinum</span> &#40;Nath <span style="font-style: italic">et al.</span>, <span style="font-style: italic">Science</span> 299:1404&#45;1407, 2003&#41; that the <span style="font-style: italic">TCP</span> gene <span style="font-style: italic">CINCINNATA &#40;CIN&#41;</span> controls leaf shape and surface curvature by regulating cell proliferation in the young leaf.  It has been proposed that <span style="font-style: italic">CIN</span> regulates cell proliferation by altering the expression of the genes involved in cell division.  However, the molecular targets of <span style="font-style: italic">CIN</span> are largely unknown.&#13;<br/>&#13;<br/>As a first step to identify the direct targets of <span style="font-style: italic">CIN</span>, we have determined the consensus DNA binding sites of the CIN protein using Random Binding Site Selection Assay &#40;RBSS&#41;.  The consensus site, GTGGTCCC, is identical to that determined for <span style="font-style: italic">TCP4</span>, the <span style="font-style: italic">Arabidopsis</span> orthologue for <span style="font-style: italic">CIN</span> &#40;our unpublished results&#41;.  We, then, have carried out differential display PCR &#40;DD&#45;PCR&#41; to identify the transcripts differentially expressed between the wild type and the <span style="font-style: italic">cin</span> mutant.  We have got the differentially displayed bands cloned and sequenced to know the identity of the genes.  To further identify the genes differentially expressed in the <span style="font-style: italic">cin</span> mutant, we have carried out micro&#45;array analysis using 44K <span style="font-style: italic">Arabidopsis</span> chips as well as 13K custom&#45;made <span style="font-style: italic">Antirrhinum</span> chips.  Since majority of the orthologous genes between <span style="font-style: italic">Antirrhinum</span> and <span style="font-style: italic">Arabidopsis</span> have considerable sequence homology, and because the consensus binding sequence of CIN and TCP4 are identical, it is likely that the <span style="font-style: italic">Arabidopsis</span> chip can be used for <span style="font-style: italic">Antirrhinum</span> probe with appropriately modified hybridization conditions.  Using these bio&#45;chemical approaches, we have identified several direct targets of <span style="font-style: italic">CIN</span>.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Expression activation of a gene encoding an unknown protein with a DUF1723 domain affected silique development in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Liqing Ma</span>
	<sup>1</sup>, 

	<span  >
	Aili Li</span>
	<sup>1</sup>, 

	<span  >
	Xiaomei Tan</span>
	<sup>1</sup>, 

	<span  >
	Hanzi Guo</span>
	<sup>1</sup>, 

	<span  >
	Xiang Wang</span>
	<sup>1</sup>, 

	<span  >
	Bo Wei</span>
	<sup>1</sup>, 

	<span  >
	Yuanfang Zhu</span>
	<sup>1</sup>, 

	<span  >
	Charles H. Leseberg</span>
	<sup>2</sup>, 

	<span  >
	Long Mao</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Institute of Crop Sciences, Chinese Academy of Agricultural Sciences and National Key Facility for Crop Gene Resources and Genetic Improvement ( NFCRI), Beijing 100081, China, 

	<sup>2</sup>Department of Biological Sciences, Northern Illinois University, DeKalb, IL 601115, USA

</div>

<div>Insertion of a T&#45;DNA construct at the promoter region of a gene encoding an unknown protein caused shortened stature with smaller, partially fertile siliques and curly cauline leaves in Arabidopsis plants. The gene contained a DUF1723 domain that is conserved in the rice genome. Genotypic analysis showed that heterozygous plants exhibited the most significant phenotypes whereas only a portion of homozygous transgenic plants displayed mutant phenotypes. The gene&#8217;s expression, as measured by RT&#45;PCR, was significantly up&#45;regulated in plants displaying a mutant phenotype when compared to the control. This suggests that the over expression of the unknown gene may be responsible for the observed phenotypes. As expected, preliminary analysis of loss&#45;of&#45;function plants from the ABRC did not find obvious phenotypic changes. Over&#45;expression experiments showed that only 5 out of 30 transgenic plants displayed similar phenotypes to the plant originally discovered. We deduce that the expression of the gene is negatively regulated, maybe in a stage&#45;specific manner. Results from promoter analysis using the GUS reporter gene, interacting proteins from yeast two&#45;hybrid screening and further genetic analysis will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Comparative interactome analysis of tomato MADS&#45;box proteins</div>

<div>

	<span style="text-decoration: underline" >
	Charles H. Leseberg</span>
	<sup>1</sup>, 

	<span  >
	Christie L. Eissler</span>
	<sup>1</sup>, 

	<span  >
	Xiang Wang</span>
	<sup>2</sup>, 

	<span  >
	Hanzi Guo</span>
	<sup>2</sup>, 

	<span  >
	Liqing Ma</span>
	<sup>2</sup>, 

	<span  >
	Bo Wei</span>
	<sup>2</sup>, 

	<span  >
	Aili Li</span>
	<sup>2</sup>, 

	<span  >
	Long Mao</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences, Northern Illinois University, DeKalb, IL 601115, USA, 

	<sup>2</sup>Institute of Crop Sciences, Chinese Academy of Agricultural Sciences and National Key Facility for Crop Gene Resources and Genetic Improvement ( NFCRI), Beijing 100081, China

</div>

<div>MADS&#45;box genes are key transcription factors for plant development. The MIKCc subfamily of MADS&#45;box genes has arisen due to its diversification in land plants. Large&#45;scale protein interaction data is limited to Arabidopsis and petunia. We conducted a comprehensive protein&#45;protein interaction study in tomato &#40;Solanum lycopersicum&#41;, a model plant for fruit abscission, development and ripening. Approximately half of the tomato MIKCc subfamily representing the subclades APETALA1&#47;FRUITFUL, APETALA3&#47;PISTILLATA, AGAMOUS, SEPALATTA, STMADS11, and SUPPRESSOR OF CONSTANS1 was included. A comparative analysis between existing MADS&#45;domain protein networks and other species including Arabidopsis was conducted. In addition, we tested potential higher order interaction complexes as shown to exist in Arabidopsis while attempting to find new complexes amongst the tomato MADS&#45;domain proteins. Details will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A genetic understanding of aging and senescence of leaf organ in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Jin Hee Kim</span>
	<sup>1</sup>, 

	<span  >
	Hyo Jung Kim</span>
	<sup>1</sup>, 

	<span  >
	In Chul Lee</span>
	<sup>1</sup>, 

	<span  >
	Seung Hee Choi</span>
	<sup>1</sup>, 

	<span  >
	Hong Gil Nam</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Division of Molecular Life Sciences and National Core Research Center for Systems Bio-Dynamics, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea., 

	<sup>2</sup>The I-BIO graduate program, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea.

</div>

<div>Leaf development ends with senescence, a process consisting of deterioration events that ultimately led to death. Since leaf senescence is well coordinated developmental process, it is well expected there are many genes that influence leaf senescence. However, in spite of the biological and practical importance, the genetic mechanisms of senescence are largely unknown. Through forward and reverse genetic approach, we have identified several delayed leaf senescence mutant. From chemical &#45; mutantgenized pool of Arabidopsis, we found ORE1, a transcription factor and ORE12, a cytokinin receptor. And to identify upstream or downstream component of ORE12, we screened EMS&#45;mutagenized suppressors of the ore12&#45;1 mutant. We have also screened delayed leaf senescence mutants from activation tagging liens to identify the negative regulators of leaf senescence. The recent progress on characterization of mutants and identification of the mutated genes is presented. From reverse genetic approach, RAV1 that is up regulated in an early senescence stage and RPK1, receptor&#45;like kinase were isolated. The overexpressor lines exhibit early senescence symptoms, revealing a possible positive regulator of leaf senescence. Some of our results suggest that unique mechanisms for leaf senescence control leaf longevity by hormone, stress, and ROS. Although we are still seeing only a glimpse of the regulatory genes to occur during leaf senescence, we anticipate that we will be able to reconcile or organize general mechanisms relevant to leaf senescence in near future. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulation of plant development by chemicals, identified through the chemical genetic approach.</div>

<div>

	<span style="text-decoration: underline" >
	Junyoung Kim</span>
	<sup>1</sup>, 

	<span  >
	Suna Jeong</span>
	<sup>1</sup>, 

	<span  >
	Jaeki Min</span>
	<sup>2</sup>, 

	<span  >
	Young Tae Chang</span>
	<sup>2</sup>, 

	<span  >
	Hong Gil Nam</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Division of Molecular Life Sciences, POSTECH, Pohang, Kyungbuk, 790-784, Republic of Korea., 

	<sup>2</sup>Department of Chemistry, New York University, New York, New York 10003, USA., 

	<sup>3</sup>Division of Molecular Life Sciences, POSTECH, Pohang, Kyungbuk, 790-784, Republic of Korea. National Core Research Center for Systems Bio-Dynamics and The I-BIO graduate program, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea.

</div>

<div>  Chemical genetics is to use small molecules that alter the protein &#13;<br/>functions to which they bind. Among various chemical libraries, some ligands could inactivate or activate certain protein functions. Until now, there are many reports that various ligands modulate functions of proteins with binding their partner molecules. Thus, using chemical genetic strategy, we want to identify new small molecules that act like conditional mutagen regulating the protein functions. Thereafter, these chemicals will be used to study the functional mechanism by identifying the protein targets of the chemicals. In this study, we have attempted to probe chemicals which can regulate a variety of plant phenotype such as seed germination, growth, development and aging through a chemical library &#45; whole plant screening system using 24 well plate. Also we are focusing on finding chemicals that affect flowering time in Arabidopsis. A flowering key regulator gene, GIGANTEA&#40;GI&#41; identified from our laboratory is screened using GI promoter and luciferase fused transgenic plant.&#13;<br/>&#13;<br/>  Among about 60,000 chemical libraries obtained from department of chemistry at NYU and Korea Chemical Bank, we found 48 chemicals that affected plant growth and development. Plump3 is one of the them which affect plant development. Now, we are trying to describe action mode of plump3 and analyze hit chemical inducing different transcription pattern of GI pro::luciferase. Finally, we would like to elucidate plump3 function by identifying its binding partner in Arabidopsis. Also, we hope to regulate flowering time using identified small molecule through the chemical genetic screening. &#13;<br/> </div>

</div>





<div class="abs">

<div style="font-weight:bold">Mutations in Arabidopsis homologs of proteins of the SWR1 complex accelerate flowering time</div>

<div>

	<span  >
	Ana Lazaro</span>
	<sup>1</sup>, 

	<span  >
	M. Angeles G&#243;mez-Zambrano</span>
	<sup>1</sup>, 

	<span  >
	Leticia L&#243;pez-Gonz&#225;lez</span>
	<sup>1</sup>, 

	<span  >
	Manuel Pi&#241;eiro</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Jose A. Jarillo</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Instituto Nacional de Investigaci&#243;n y Tecnolog&#237;a Agraria y Alimentaria, Department of Biotechnology, Madrid, Spain

</div>

<div>Our interests is focused on the analysis of the molecular mechanisms involved in the regulation of flowering time and, particularly, on those factors required for the repression of flowering until plants are in optimal environmental conditions or reach the appropriate developmental stage to flower. Initially, we characterized <span style="font-style: italic">early in short days</span> &#40;<span style="font-style: italic">esd1</span>&#41; mutations, which cause early flowering and several vegetative and reproductive developmental defects. <span style="font-style: italic">esd1</span> abolishes the <span style="font-style: italic">FLC</span>&#45;mediated late flowering phenotype of plants carrying active alleles of <span style="font-style: italic">FRI</span> and of mutants of the autonomous pathway &#40;Martin&#45;Trillo et al., 2006&#41;. We found that <span style="font-style: italic">ESD1</span> is required for the expression of <span style="font-style: italic">FLC</span> and other members of the <span style="font-style: italic">FLC</span>&#45;like&#47;<span style="font-style: italic">MAF</span> gene family to levels that inhibit flowering.  The <span style="font-style: italic">ESD1</span> locus encodes ARP6, a homolog of the actin&#45;related protein family that is part of the SWR1 chromatin&#45;remodeling complex in yeast, which catalyzes the exchange of histone H2A for the histone H2A.Z variant in nucleosome arrays ensurimg full activation of underlying genes. Now, we have isolated Arabidopsis mutants in homologs of others SWR1C components. Mutations in <span style="font-style: italic">AtSWC6</span> cause similar developmental defects as <span style="font-style: italic">esd1</span>, including serrated leaves, weak apical dominance, flowers with altered number and size of organs and early flowering by reduction in expression of <span style="font-style: italic">FLC</span> and other <span style="font-style: italic">MAF</span> genes. <span style="font-style: italic">swc6</span> suppresses late flowering phenotype of <span style="font-style: italic">FRI</span> and of <span style="font-style: italic">fve</span> and <span style="font-style: italic">fca</span>  mutant plants. Protein interaction analyses suggest the formation of a complex between ESD1 and AtSWC6. In addition, we will present evidence that <span style="font-style: italic">AtSWC6</span> is needed to achieve both the levels of H3 acetylation and H3&#45;K4 methylation required for high <span style="font-style: italic">FLC</span> expression, on the same manner as ESD1 does. These observations are consistent with the presence of a SWR1C&#45;like complex in Arabidopsis that may regulate plant development controlling gene expression, being required for modulation of chromatin structure. &#13;<br/>&#13;<br/>Martin&#45;Trillo et al. &#40;2006&#41;. Development, 13, 1241&#45;1252.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Study on &#34;AtMYB23&#34; in the &#34;Arabidopsis&#34; Root Epidermal Cell Fate Specification</div>

<div>

	<span style="text-decoration: underline" >
	Yeon Hee Kang</span>
	<sup>1</sup>, 

	<span  >
	Kook Hui Ryu</span>
	<sup>1</sup>, 

	<span  >
	John Schiefelbein</span>
	<sup>2</sup>, 

	<span  >
	Myeong Min Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, Yonsei University, Sichon 134, Seoul, Korea, 

	<sup>2</sup>Department of Molecular Cellular and Developmental Biology, University of Michigan, Ann Arbor, MI 48109, USA

</div>

<div>AtMYB23 is a member of the MYB transcription factor family. It has been suggested that several MYB&#45;related transcription factors are involved in cell fate specification in &#34;Arabidopsis&#34;. The &#34;WER&#34; gene is expressed in non&#45;hair cells in the root and is proposed as a master regulator in root epidermal cell fate pattering. A WER homologue, the &#34;GL1&#34; gene is expressed in developing trichomes in the leaves and regulates in trichome development. AtMYB23 protein shows high similarity in amino acid sequence with WER and GL1 protein. &#34;AtMYB23&#34; is expressed both in non&#45;hair cells of the root epidermis and in trichomes of the leaves. It has been reported that &#34;AtMYB23&#34; is involved in the trichome morphogenesis as well aw its initiation. However, its function in root epidermal cell fate specification is not clear yet. To clarify the role of &#34;AtMYB23&#34; in the root, we analyzed two T&#45;DNA insertion mutant lines. The position&#45;dependent cell fate specification was not significantly disrupted in these mutants. However, mutation in &#34;AtMYB23&#34; gene affects the expression of some genes which are involved in this cell fate specification. Based on our results, &#34;AtMYB23&#34; may not be critical in the cell fate specification in the &#34;Arabidopsis&#34; root epidermis, but may regulate the efficiency of this patterning process.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional analysis of TEBICHI in meristem maintenance and organ development in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Soichi Inagaki</span>
	<sup>1</sup>, 

	<span  >
	Kenzo Nakamura</span>
	<sup>1</sup>, 

	<span  >
	Atsushi Morikami</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Graduate School of Bioagricultural Sciences, Nagoya University, Nagoya, Japan, 

	<sup>2</sup>Department of Agrobiological Resources, Meijo University, Nagoya, Japan

</div>

<div>TEBICHI &#40;TEB&#41; of <span style="font-style: italic">Arabidopsis thaliana</span> is homologous to <span style="font-style: italic">Drosophila</span> MUS308 and mammalian POLQ with DNA helicase and DNA polymerase domains. Loss&#45;of&#45;function mutant, <span style="font-style: italic">teb</span>, shows various morphological defects such as short roots, asymmetric leaves, and fasciated stems, that are accompanied with defects in cell division patterns and cell differentiation in embryo and meristems. In <span style="font-style: italic">teb</span> mutants, response to DNA&#45;damage is constitutively activated, and cells expressing <span style="font-style: italic">cyclinB1;1:GUS</span> are accumulated in meristems, suggesting that DNA damage accumulated in <span style="font-style: italic">teb</span> mutants activates cell cycle checkpoint, leading to a defect in G2&#47;M cell cycle progression. To further analyze the function of TEB, we analyzed the genetic interaction of <span style="font-style: italic">TEB</span> with <span style="font-style: italic">ATR</span>, which is involved in cellular responses to defects in DNA replication. We observed that accumulation of cells expressing <span style="font-style: italic">cyclinB1;1:GUS</span> in <span style="font-style: italic">teb</span> was rather suppressed by <span style="font-style: italic">atr</span> mutation, suggesting that the ATR&#45;dependent G2&#47;M cell cycle checkpoint is activated in <span style="font-style: italic">teb</span>. However, <span style="font-style: italic">atr</span> mutation enhanced morphological phenotypes of <span style="font-style: italic">teb</span> mutants such as defects of growth of roots and aerial parts, as well as abnormal leaf shapes. These results suggest that defects which occur in association with DNA replication, rather than defects in cell cycle progression, are important in causing the morphological defects in <span style="font-style: italic">teb</span> mutants. <span style="font-style: italic">teb atr</span> double mutants frequently developed needle&#45;like leaves, suggesting that <span style="font-style: italic">teb atr</span> is defective in adaxial&#45;abaxial polarity of leaves. Histochemical analysis of expression of abaxial&#45;specific <span style="font-style: italic">FILAMENTOUS FLOWER</span> showed that leaves of <span style="font-style: italic">teb</span> and <span style="font-style: italic">teb atr</span> were partially abaxialized, suggesting that TEB is involved in the establishment of adaxial&#45;abaxial polarity of leaves. Altogether, our results suggest that proper replication of genome in the S&#45;phase is critical for the proper expression of developmentally regulated genes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">CHICORY1&#40;CHI1&#41;,  A  C2H2 ZINC FINGER PROTEIN OF ARABIDOPSIS,  IS INVOLVED IN DEVELOPMENT</div>

<div>

	<span style="text-decoration: underline" >
	Kyung Sook Chung</span>
	<sup>1</sup>, 

	<span  >
	Joonki Kim</span>
	<sup>1</sup>, 

	<span  >
	Jong Seob Lee</span>
	<sup>2</sup>, 

	<span  >
	Ji Hoon Ahn</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Signaling Network Research Center, School of Life Sciences and Biotechnology, Korea University, Seoul, 136-701, Korea., 

	<sup>2</sup>Crop Functional Genomics Center, School of Biological Sciences, Seoul National University, Seoul 152-742, Korea

</div>

<div>From activation tagging screening, we isolated a dominant mutant, which has a serrated leaves and named <span style="font-style: italic">chicory1&#45;1D &#40;chi1&#45;1D&#41;</span>. In addition,  <span style="font-style: italic">chi1&#45;1D</span> showed pleiotropic phenotypes such as, reduced height, delayed growth, thick stems, open floral buds, decreased silique size, and increased indehiscent fruits. The 35S enhancers of cauliflower mosaic virus &#40;CaMV&#41; were inserted into  the promoter region of the <span style="font-style: italic">CHI1</span> gene and resulted in gain&#45;of&#45;function of <span style="font-style: italic">CHI1</span> gene. <span style="font-style: italic">CHI1</span> encodes a novel type of C2H2 zinc&#45;finger protein and has a putative nuclear localization signal. CHI1:GFP fusion protein was localized in nuclear in the transgenic root epidermal cells. Loss&#45;of&#45;function mutants of <span style="font-style: italic">CHI1</span>, <span style="font-style: italic">chi1&#45;2</span> and <span style="font-style: italic">chi1&#45;3</span> were decreased a silique size and the number of seeds. these were due to decreased seed set in proximal regions of the silique. After fertilization, the fruit size increases dramatically and <span style="font-style: italic">FUL</span> gene was known to be important to cell expansion after fertilization in fruit development. The expression of <span style="font-style: italic">FUL</span> was up&#45;regulated in <span style="font-style: italic">chi1&#45;2</span>, but was down&#45;regulated in <span style="font-style: italic">chi1&#45;1D</span>. It shows the possibility that <span style="font-style: italic">CHI1</span> may function as a negative regulator of <span style="font-style: italic">FUL</span>.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Role of polar auxin transport in orchestrating plant root growth and patterning</div>

<div>

	<span style="text-decoration: underline" >
	Ari Pekka Mahonen</span>
	<sup>1</sup>, 

	<span  >
	Veronica Grieneisen</span>
	<sup>1</sup>, 

	<span  >
	Jian Xu</span>
	<sup>1</sup>, 

	<span  >
	Paulien Hogeweg</span>
	<sup>1</sup>, 

	<span  >
	Ben Scheres</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Utrecht University, Utrecht, The Netherlands

</div>

<div>The phytohormone auxin has been implicated in patterning of the root meristem. From embryogenesis onwards, an auxin response maximum is formed and maintained in the distal region of the root. The auxin maximum is achieved by PIN auxin efflux facilitators, which collectively control auxin distribution and form auxin reflux loop to regulate cell division and cell expansion in the primary root. The PLETHORA1 &#40;PLT1&#41; and PLT2 genes encode double AP2&#45;domain transcription factors expressed in the embryonic root pole and later in the root stem cell region. Auxin induces transcription of PLT genes, which are redundantly required for stem cell maintenance in the root meristem. Furthermore, PIN proteins have an important role in restricting the expression domain of PLT genes. Conversely, PLT genes are required for PIN gene transcription to stabilize the auxin maximum at the distal root tip. These data reveal an interaction network of auxin transport facilitators and root fate determinants that control patterning and growth of the root primordium. In order to dissect this complex interaction network, we have developed in silico system to model auxin transport. The auxin flux pattern in our model yields to stable accumulation of auxin in a distal maximum, without local auxin production or decay. From the maximum auxin concentration decreases in graded manner. Both the auxin maximum and the gradient are robust towards parameter settings, root growth and noise. Next, we will use genetic tools to manipulate PIN localization in cell&#45;type&#45;specific manner. These local interruptions of the auxin transport route should result in altered auxin accumulation in the root meristem and in the in silico model, and consequently, in global repatterning in the root meristem.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification of SUF4 protein complex in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Hyun-Ju Hwang</span>
	<sup>1</sup>, 

	<span  >
	Kyuha Choi</span>
	<sup>1</sup>, 

	<span  >
	Sanghee Kim</span>
	<sup>2</sup>, 

	<span  >
	Youngmi Lee</span>
	<sup>1</sup>, 

	<span  >
	Ilha Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences, Seoul National University, Korea, 

	<sup>2</sup>Laboratory of Plant Molecular Biology, Rockefeller University, New York, USA.

</div>

<div>We previously reported that <span style="font-style: italic">PPRESSOR OF FRIGIDA4</span>encoding a C2H2&#45;type zinc finger protein, represses flowering by transcriptional activation of Arabidopsis <span style="font-style: italic">FLOWERING LOCUS C</span> &#40;2006, The Plant Cell, 18, 2985&#45;2998&#41;. <span style="font-style: italic">FLC</span> encodes a MADS box transcription factor to act as a floral repressor. <span style="font-style: italic">FRI</span>, encoding a coiled&#45;coil protein, is an <span style="font-style: italic">FLC</span> activator. Our results showed that SUF4 interacts with FRI and FRIGIDA&#45;LIKE1 &#40;FRL1&#41; and binds to the <span style="font-style: italic">FLC</span> promoter in vivo. Interestingly, LUMINIDEPENDENS &#40;LD&#41;, a FLC repressor, interacts with SUF4. <span style="font-style: italic">LD</span> encodes a homeodomain protein and functions as a floral activator. In addition, our transient gene expression results demonstrated that direct physical interactions of SUF4 with FRI, FRL1, and LD. In this study, to verify the <span style="font-style: italic">FLC</span>regulatory complex containing SUF4 in <span style="font-style: italic">Arabidopsis</span>, we first predicted approximate size of this complex in diverse background by gel filtration. Also, we generated the epitope tagged transgenic plants and double mutants among them for immunoprecipitation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A role for the BAH&#45;PHD containing protein SHORT LIFE &#40;SHL&#41; in the regulation of the floral transition in Arabidopsis</div>

<div>

	<span  >
	Leticia L&#243;pez-Gonz&#225;lez</span>
	<sup>1</sup>, 

	<span  >
	Laura Narro-Diego</span>
	<sup>1</sup>, 

	<span  >
	Jos&#233; Miguel Mart&#237;nez-Zapater</span>
	<sup>2</sup>, 

	<span  >
	Jos&#233; A. Jarillo</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Manuel Pi&#241;eiro</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Instituto Nacional de Investigaci&#243;n y Tecnolog&#237;a Agraria y Alimentaria (INIA) Dpto. Biotecnolog&#237;a, Edificio Z. Carretera de La Coru&#241;a km 7. 28040 Madrid, Spain, 

	<sup>2</sup>Dpto. Gen&#233;tica Molecular de Plantas. Centro Nacional de Biotecnolog&#237;a-CSIC. C/ Darwin 3, Cantoblanco. 28049 Madrid, Spain

</div>

<div>We are interested in understanding chromatin remodeling mechanisms involved in the repression of the floral transition, one of the most dramatic phase changes in plant development. Previously, we have identified <span style="font-style: italic">EARLY BOLTING IN SHORT DAYS</span> &#40;<span style="font-style: italic">EBS</span>&#41;, a locus involved in the repression of flowering and the regulation of other developmental processes in Arabidopsis &#40;G&#243;mez&#45;Mena et al, 2001&#41;. <span style="font-style: italic">EBS</span> is required to repress the expression of the floral integrator <span style="font-style: italic">FT</span> under non&#45;inductive photoperiodic conditions short days and encodes a nuclear protein with a BAH and a PHD domain &#40;Pi&#241;eiro et al, 2003&#41;. Both types of domains are thought to mediate protein&#45;protein interactions and are present in chromatin remodeling factors. Proteins with the same modular architecture as EBS have been found in Arabidopsis and other plant species, but not in other organisms, suggesting that these proteins are part of a plant&#45;specific family of transcriptional regulators. By analysing loss&#45;of&#45;function alleles we show that another member of this family in Arabidopsis, <span style="font-style: italic">SHL</span>, also has a role in the repression of flowering. Moreover, double mutant analyses suggest that <span style="font-style: italic">SHL</span> has partially overlapping functions with <span style="font-style: italic">EBS</span> in the control of the floral transition in Arabidopsis. Progress in the genetic and molecular analysis currently underway in our laboratory to understand the role of <span style="font-style: italic">SHL</span> in the control of flowering time will be presented. In addition, we will discuss natural variation data with implications in the evolution of this family of plant specific proteins likely involved in chromatin remodeling processes.&#13;<br/>&#13;<br/>G&#243;mez&#45;Mena, C., et al. &#40;2001&#41; Plant Cell 13, 1011&#45;1024.&#13;<br/>Pi&#241;eiro, M., et al. &#40;2003&#41; Plant Cell 15, 1552&#45;1562.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Rice, a Model System for Studying Plant Architecture</div>

<div>

	<span style="text-decoration: underline" >
	Jiayang Li</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing 100101, China

</div>

<div>Rice is an ideal system for studying plant architecture of cereal crops. Rice plant architecture, a collection of the important agronomic traits that determine its grain production, is mainly affected by factors including tillering &#40;tiller number and tiller angle&#41;, plant height and panicle morphology. To elucidate the molecular mechanisms that control rice plant architecture, we have identified several related mutants and isolated the corresponding genes. Among them, the rice mutant lazy1 &#40;la1&#41;, originally reported as lazy &#40;la&#41; in 1938, is a classical mutant for studying the rice tiller angle due to its extremely spreading growth phenotype. We have now cloned the LA1 gene through a map&#45;based approach and show that LA1 is a novel gene that plays an important role as a negative regulator of polar auxin transport &#40;PAT&#41;. Loss&#45;of&#45;function of LA1 enhances PAT greatly and thus alters the endogenous IAA distribution in shoots, leading to the reduced gravitropism and therefore the tiller&#45;spreading phenotype of rice plants. We also found that the tiller angle of LA1 RNAi transgenic plants showed a correlation to endogenous LA1 expression levels, implying a potential application of LA1 to modify rice tiller angle in molecular breeding. In addition, we have identified and characterized the DWARF27 &#40;D27&#41; gene that controls rice tiller number and the SMALL PANICLE &#40;SP&#41; gene that regulates the panicle size, respectively. Further functional studies on these essential regulators will provide a molecular basis for developing elite rice and other cereal crops in the future.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Promoter Analysis of <span style="font-style: italic">PRESSED FLOWER</span> &#40;<span style="font-style: italic">PRS</span>&#41; Necessary for Development of the Marginal Domain of Lateral Organs in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Miyuki Nakata</span>
	<sup>1</sup>, 

	<span  >
	Shunji Funaki</span>
	<sup>1</sup>, 

	<span  >
	Noritaka Matsumoto</span>
	<sup>1</sup>, 

	<span  >
	Ryuji Tsugeki</span>
	<sup>1</sup>, 

	<span  >
	Kiyotaka Okada</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Department of Botany, Graduate School of Science, Kyoto University, Kyoto, Japan, 

	<sup>2</sup>National Institute for Basic Biology, Aichi, Japan

</div>

<div>Lateral organs of vascular plants, including leaves and floral organs, develop various tissues in a position&#45;dependent manner. For example, in marginal domain of lateral organs, especially leaves and sepals, margin&#45;specific epidermal cells differentiate and form the boundary between adaxial and abaxial epidermal cells. Some of <span style="font-style: italic">WUSCHEL</span>&#45;related homeobox genes, Arabidopsis <span style="font-style: italic">PRESSED FLOWER</span> &#40;<span style="font-style: italic">PRS</span>&#41; and maize <span style="font-style: italic">NARROW SHEATH1</span> &#40;<span style="font-style: italic">NS1</span>&#41;&#47;<span style="font-style: italic">NS2</span> are specifically expressed in marginal domain of lateral organ primordia. <span style="font-style: italic">PRS</span> and <span style="font-style: italic">NS1</span>&#47;<span style="font-style: italic">NS2</span> are necessary for formation of the marginal domain &#40;1&#45;3&#41;. These suggest that the margin&#45;specific expression of <span style="font-style: italic">PRS</span> and <span style="font-style: italic">NS1</span>&#47;<span style="font-style: italic">NS2</span> is an important step for development of marginal domain of lateral organs.<br/>To investigate the mechanism of the margin&#45;specific expression, we carried out promoter analysis of <span style="font-style: italic">PRS</span>. The 838 bp upstream region of <span style="font-style: italic">PRS</span> was sufficient for the margin&#45;specific expression in lateral organs at the early developmental stage. We further dissected the 838&#45;bp promoter and identified the 140&#45;bp region necessary for the early margin&#45;specific expression of lateral organ primordia. We are screening for the transcription factors which specifically bind to the 140&#45;bp region by yeast one&#45;hybrid system. <br/>References:<br/>1. Scanlon et al., Development, 122 &#40;1996&#41;<br/>2. Nardmann et al., Development, 131 &#40;2004&#41;<br/>3. Matsumoto and Okada, Genes &#38; Dev., 15&#40;24&#41; &#40;2001&#41;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The effect of flowering on vegetative phase change in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Matthew R. Willmann</span>
	<sup>1</sup>, 

	<span  >
	R. Scott Poethig</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Science Institute, Department of Biology, University of Pennsylvania, Philadelphia, PA, USA

</div>

<div>Shoot development in most flowering plants is characterized by three primary postembryonic phases:  juvenile vegetative, adult vegetative, and reproductive.  These phases are distinguished by differences in vegetative characteristics, reproductive competence, phytochemical production, and disease and insect resistance.  It is believed that a series of temporally overlapping pathways regulate the transitions between phases.  Under long day conditions, Arabidopsis flower primordia were already visible in most young Columbia wild&#45;type seedlings when the fifth leaf was only 1 mm long.  Therefore, cell division, differentiation, and expansion within most leaves on a shoot are occurring under the context of floral induction and are likely regulated by this process.  In attempt to distinguish the effects of the juvenile&#45;to&#45;adult transition on vegetative growth and development from those of the reproductive onset, we characterized vegetative phase change under different photoperiods and in early &#40;&#34;FRI; flc&#45;3&#34;&#41; and late&#45;flowering &#40;&#34;FRI; FLC&#34;&#41; genotypes of the flowering time repressors FRI and FLC.  Short&#45;day conditions, which inhibit flowering, also delayed vegetative phase change in all genotypes.  Under both short and long&#45;day light regimes, the timing of the onset of vegetative phase change in these genotypes was statistically the same as wild&#45;type Columbia &#40;&#34;fri; FLC&#34;&#41;.  The length of the transition zone, however, was dependent on flowering time.  The early flowering genotype had a shorter transition zone than wild type.  Plants with delayed flowering &#40;&#34;FRI; FLC&#34; plants, as well as all genotypes grown under short days&#41; had a much longer transition zone.  Preliminary data suggests that the FLC targets SOC1 and FT may mediate these effects of flowering on vegetative development.  This work indicates a role for flowering in the acceleration of adult vegetative characteristics in Arabidopsis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Gene expression profiling of stomatal precursors</div>

<div>

	<span  >
	Nalini Odapalli</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Jeanette Nadeau</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, University of Central Florida, Orlando, FL 32816-2368, USA

</div>

<div>Stomata are structures that act as valves to control water loss and gas exchange through the plant epidermis.  They are non&#45;randomly spaced through cell&#45;cell signaling that controls the position, frequency, and polarity of asymmetric divisions of stem cell&#45;like precursors that form stomata.  In leaves, mutation of the TOO MANY MOUTHS &#40;TMM&#41; LRR receptor&#45;like protein causes patterning defects that lead to overabundant and clustered stomata.  In contrast, the <span style="font-style: italic">tmm</span> mutation prevents development of stomata in other organs such as the inflorescence stem.  In <span style="font-style: italic">tmm</span> stems, stomatal precursors &#40;meristemoids&#41; are produced but stop dividing and differentiate as epidermal pavement cells.  Also in contrast to leaves, stem stomata form in a morphogenetic gradient so that mature stomata and guard mother cells are spatially separated from meristemoids.  We chose to exploit these characteristics to identify genes that may play a role in stomatal stem cell maintenance and biogenesis using microarray gene expression profiling.  The apical region of the inflorescence stem tip containing only meristemoids &#40;but no stomata&#41; was harvested from both mutant and wild&#45;type plants.  Resulting RNA was hybridized to the Affymetrix ATH1 genechip.  A Bayesian statistical method &#40;RACE&#41; following robust multichip averaging was used to identify 352 genes that showed significant changes in expression level between mutant and wild&#45;type tissues.  No other known regulators of stomatal development, with the exception of <span style="font-style: italic">TMM</span> itself, were differentially expressed.  Although a large proportion of these genes are probable components of signal transduction cascades, no functional class was statistically over&#45;represented in the dataset.  11 genes were also differentially regulated by the YODA MAPKKK that controls stomatal development &#40;Bergmann et al. &#40;2004&#41; Science 304:1494&#41;.  Analysis of insertional mutants in candidate genes is currently underway and stomatal phenotypes have been observed in several, validating this approach to identification of new regulators of stomatal biogenesis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification of STM protein interactors involved in flowering</div>

<div>

	<span style="text-decoration: underline" >
	Ricardo Junqueira</span>
	<sup>1</sup>, 

	<span  >
	Siddhartha Kanrar</span>
	<sup>1</sup>, 

	<span  >
	Harley M. S. Smith</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Riverside, CA, USA

</div>

<div>In higher plants, shoot development is characterized by the continuous production of organs initiated on the flanks of the shoot apical meristem &#40;SAM&#41;. The activity of the SAM is differentially regulated throughout plant development and the identity of initiated lateral organs corresponds to the plant developmental phase.  The homeodomain transcription factor SHOOTMERISTEMLESS &#40;STM&#41; is essential for meristem maintenance.  In addition, the weak allele <span style="font-style: italic">stm&#45;10</span> displays a late flowering phenotype with defects in floral specification during inflorescence development, suggesting that STM is involved in promotion of flowering. To gain further insights into the role of STM, a yeast two&#45;hybrid screen was performed in order to identify proteins involved in flowering  that associate with this homeodomain protein.  The results will be discussed.&#13;<br/>Supported by NSF Grant IOB&#45;0615774 and CAPES&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Unique, Shared, and Redundant Roles for the Arabidopsis&#13;<br/>SWI&#47;SNF Chromatin Remodeling ATPases BRAHMA&#13;<br/>and SPLAYED</div>

<div>

	<span  >
	Staver Bezhani</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Cara Winter</span>
	<sup>1</sup>, 

	<span  >
	Steve Hershman</span>
	<sup>1</sup>, 

	<span  >
	John D. Wagner</span>
	<sup>1</sup>, 

	<span  >
	John F. Kennedy</span>
	<sup>1</sup>, 

	<span  >
	Chang Seob Kwon</span>
	<sup>1</sup>, 

	<span  >
	Jennifer Pfluger</span>
	<sup>1</sup>, 

	<span  >
	Yanhui Su</span>
	<sup>1</sup>, 

	<span  >
	Doris Wagner</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Pennsylvania

</div>

<div>SWI&#47;SNF chromatin remodeling ATPases are conserved in the animal and plant kingdom and regulate transcriptional programs in response to endogenous and exogenous cues. In contrast with their metazoan orthologs, null mutants in two Arabidopsis thaliana SWI&#47;SNF ATPases, BRAHMA &#40;BRM&#41; and SPLAYED &#40;SYD&#41;, are viable, facilitating investigation of their role in the organism. Previous analyses revealed that syd and brm null mutants exhibit both similar and distinct developmental defects, yet the functional relationship between the two closely related ATPases is not understood. Another central question is whether these proteins act as general or specific transcriptional regulators. Using global expression studies we find overlapping functions for the two SWI&#47;SNF ATPases. This partial diversification may have allowed expansion of the SWI&#47;SNF ATPase regulatory repertoire, while preserving essential ancestral functions. Moreover, only a small fraction of all genes depends on SYD or BRM for expression, indicating that these SWI&#47;SNF ATPases exhibit remarkable regulatory specificity. Our studies provide a conceptual framework for understanding the role of SWI&#47;SNF chromatin remodeling in regulation of Arabidopsis development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of embryo and endosperm mutants caused by Ds insertion into genes encoding putative leucine&#45;rich repeat&#45;containing receptor kinases</div>

<div>

	<span style="text-decoration: underline" >
	Tian-Ying Yu</span>
	<sup>1</sup>, 

	<span  >
	Jie Liu</span>
	<sup>1</sup>, 

	<span  >
	Wei-Cai Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Key Laboratory of Molecular and Developmental Biology, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Datun Road, Beijing 100101, China

</div>

<div>In flowering plant, double fertilization gives rise to a diploid zygote and a triploid primary endosperm cell that develop into embryo and endosperm respectively. To investigate the genetic control of embryo and endosperm development, we screened a Ds gene trap collection for mutants defective in embryo and endosperm development. One of the mutants was isolated that showed 27.82&#37; seed abortion. Embryo development was blocked at either the zygotic stage &#40;97&#47;1136&#61;8.54&#37;&#41; or one&#45;celled embryo stage &#40;180&#47;1136&#61;15.85&#37;&#41;. Molecular analysis showed that the Ds element was inserted into a gene encoding for a putative leucine&#45;rich repeat receptor&#45;like kinase &#40;LRR&#45;RLK&#41;. GFP fusion experiment showed that the gene product is localized in plasma membrane. Expression analysis revealed by RT&#45;PCR indicated that the gene is expressed highly in siliques between fertilization and 4&#45;8 cell embryos. The expression level is very low in inflorescence, and no expression was detected in other tissues. These data suggested that the gene is expressed in embryo and&#47;or endosperm. In situ experiment further confirmed that the gene is expressed specific in embryo and endosperm. Further molecular study on its function in embryo and endosperm development is being carried out. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The flowering pathway integrator SOC1 directly regulates <span style="font-style: italic">LFY</span> by interacting with AGL24</div>

<div>

	<span style="text-decoration: underline" >
	Jungeun Lee</span>
	<sup>1</sup>, 

	<span  >
	Mijin Oh</span>
	<sup>1</sup>, 

	<span  >
	Ilha Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences, Seoul National University, Seoul, Korea

</div>

<div>The transition to flowering is controlled by complex genetic networks in response to environmental and endogenous signals in Arabidopsis. The genetic and molecular analyses of Arabidopsis revealed that the photoperiod pathway, autonomous&#47;vernalization pathway and GA pathways can act distinctly to promote flowering, converging on common downstream integrators; <span style="font-style: italic">FLOWERING LOCUS T </span>&#40;<span style="font-style: italic">FT</span>&#41;, <span style="font-style: italic">SUPPRESSOR of OVEREXPRESSION of CONSTANS</span> &#40;<span style="font-style: italic">SOC1</span>&#41; and <span style="font-style: italic">LEAFY</span> &#40;<span style="font-style: italic">LFY</span>&#41;. These regulators integrate inputs from the multiple flowering pathways and make plants decide when to flower by inducing floral meristem identity genes at the shoot apex.&#13;<br/> The previous studies have proposed that the MADS&#45;box transcription factor, SOC1 might regulate <span style="font-style: italic">LFY</span> expression and another MADS&#45;box transcription factor AGL24 would act together with SOC1 or mediate the signal from SOC1 to LFY. However, there are still little data to support these hypotheses. In this study, we present empirical evidences to verify the hypotheses. Chromatin Immunoprecipitation assay shows SOC1 directly binds to the <span style="font-style: italic">LFY</span> promoter, and transient expression assay shows AGL24 acts as interacting partner of SOC1. In addition, through the suppressor mutagenesis of the transgenic lines overexpressing SOC1 protein, we found that MADS domain is important for SOC1 function and contributes to make SOC1&#45;AGL24 complex.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of Mutants Defective in Female Gametogenesis and Embryogenesis in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Man Liu</span>
	<sup>1</sup>, 

	<span  >
	Naiyou Liu</span>
	<sup>1</sup>, 

	<span  >
	Li Yuan</span>
	<sup>1</sup>, 

	<span  >
	Dongqiao Shi</span>
	<sup>1</sup>, 

	<span  >
	Jie Liu</span>
	<sup>1</sup>, 

	<span  >
	Weicai Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Key Laboratory of Molecular and Developmental Biology, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Datun Road, Beijing 100101, China

</div>

<div>Female gametophyte development and embryogenesis are essential for plant sexual reproduction. However, little is known about the relationship between these two subsequent stages and the functions of essential genes in these developmental stages. We identified two Arabidopsis thaliana mutants, sgt13018 and 77B, which are defective in these steps. In sgt13018, both female gametogenesis and embryogenesis were impaired. The female gametophyte development was paused at FG5 stage, but they eventually could develop into mature embryo sacs and attracted pollen tubes to enter them, and the mutant embryos were mainly arrested at zygote or 1&#45;cell stage. Molecular analysis indicates that sgt13018 encodes a putative U5 snRNP&#45;specific protein, and is predominantly expressed in the young organs. In 77B, a gene, which encodes a putative DEAD&#47;DEAH box helicase protein, was disrupted with a Ds insert. Lose of 77B gene function cause arrest of embryo sacs at four&#45;, five&#45; or eight&#45;nucleate stage within the same pistil. The gene is expressed ubiquitously throughout the plant. Together, these data suggest that the essential genes, sgt13010 and 77B, play an important role in the regulation of sexual reproduction.</div>

</div>





<div class="abs">

<div style="font-weight:bold">DAU encodes a membrane protein that is critical to pollen maturation in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Li Yuan</span>
	<sup>1</sup>, 

	<span  >
	Jie Liu</span>
	<sup>1</sup>, 

	<span  >
	Weicai Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Key Laboratory of Molecular and Developmental Biology, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Datun Road, Beijing 100101, China

</div>

<div>In plants, pollens go through a dehydration process after pollen mitosis II to protect the male gametes after pollen release from the anther. Proper timing and initiation of the dehydration processes are critical to pollen viability and fertility. Here we report the characterization of an Arabidopsis mutant, dayu &#40;dau&#41; which displayed premature pollen germination before anther dehiscence, leading to male sterility. Compared to wild&#45;type, mutant pollen grains displayed ultrastructural changes typical of germinating pollen and germinated within anthers prior to dehiscence if water was supplied. DAU was cloned by transposon tagging and encodes a putative membrane protein. DAU&#45;DsRed2 fusion experiment showed that DAU indeed is localized in the endomembrane system of male gametophytes. RNA in situ hybridization showed that DAU is transiently expressed in male gametophytes from pollen mitosis I on till anther dehiscence, no expression could be detected in pollen grains after dispersed onto the stigma. These data imply that DAU plays a critical role in controlling the initiation of pollen maturation process in plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis eIF3e is regulated by the COP9 signalosome and impacts development and protein translation.</div>

<div>

	<span  >
	Avital Yahalom</span>
	<sup>1</sup>, 

	<span  >
	Tae-Houn Kim</span>
	<sup>2</sup>, 

	<span  >
	Albrecht G. von Arnim</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Daniel A. Chamovitz</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Tel Aviv University, 

	<sup>2</sup>University of Tennessee

</div>

<div>The roles of individual eIF3 subunits are largely unclear, though some are essential, while others are thought to have regulatory roles. The &#8220;e&#8221; subunit, also known as Int&#45;6, is a candidate for a regulatory subunit as it is not essential for translation initiation in yeasts. eIF3e associates with the COP9 signalosome and localizes in certain tissues to the nucleus. To further elucidate the roles of eIF3e, we have taken a genetic approach using Arabidopsis as a model system.  Overexpression of eIF3e results in defects similar to mutations in the COP9 signalosome. eIF3e protein, but not transcript, over&#45;accumulates in csn mutants, and eIF3e is degraded in a proteasome&#45;dependent fashion.  Over&#45;expression of eIF3e leads to accumulation of ubiquinylated proteins.  In vitro and in vivo assays suggest that excess eIF3e inhibits translation. We conclude that CSN maintains a precise regulation of eIF3e levels which is necessary for normal development in Arabidopsis. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Towards the understanding of the function of CLV3 and CLE19 peptides</div>

<div>

	<span style="text-decoration: underline" >
	Chun-Ming Liu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Botany, Chinese Academy of Sciences

</div>

<div>Over the past ten years studies in Arabidopsis provided concrete evidence showing that stem cells are indeed present in higher plants. Stem cells, positioned in the central zone of the shoot apical meristem &#40;SAM&#41;, are the source of totipotent cells that serve as the founders of all newly formed aboveground organs during the plant&#8217;s life cycle. The maintenance of a constant pool of stem cells requires tightly regulated machinery to sustain a dynamic equilibrium between cells in an undifferentiated state and cells that are destined to differentiate into various tissues and organs. At the molecular level, the balance is controlled by a feedback regulation loop consisting of a WUS homeobox transcription factor and a CLV1&#47;CLV2&#47;CLV3 receptor complex. WUS, which is expressed in the L3 layer of the central zone in the SAM organizing center, promotes the stem cell identity. CLV3 encodes a putative extracellular protein with 96 amino acids and is mainly expressed in the L1 and L2 layers of the central zone of the SAM, above the CLV1 domain. We have showed previously that the conserved C&#45;terminal CLE &#65357;otif is the functional cue of CLV3 and CLE19.  Most likely the endogenious CLV3 consisits of 12 animo acids &#40;RTVPhSGPhDPLHH&#41;, with hydroxyl groups attached to two of the three proline residues &#91;Kondo, 2006&#93;. Using molecular genetic tools we have further studied the function of CLV3 and several other CLE genes.  The results showed that most of these CLE genes are functionally redundent.</div>

</div>





<div class="abs">

<div style="font-weight:bold">INCREASED LEVEL OF POLYPLOIDY 1&#40;ILP1&#41; encodes a novel transcriptional repressor that controls endoreduplication level in Arabidopsis.</div>

<div>

	<span  >
	Takeshi Yoshizumi</span>
	<sup>1</sup>, 

	<span  >
	Yuko Tsumoto</span>
	<sup>1</sup>, 

	<span  >
	Tomoko Takiguchi</span>
	<sup>1</sup>, 

	<span  >
	Noriko Nagata</span>
	<sup>2</sup>, 

	<span  >
	Yoshiharu Yamamoto</span>
	<sup>3</sup>, 

	<span  >
	Mika Kawashima</span>
	<sup>1</sup>, 

	<span  >
	Takanari Ichikawa</span>
	<sup>1</sup>, 

	<span  >
	Miki Nakazawa</span>
	<sup>1</sup>, 

	<span  >
	Naoki Yamamoto</span>
	<sup>4</sup>, 

	<span style="text-decoration: underline" >
	Minami Matsui</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Functional Genomics Research Team, PSC, RIKEN, Yokohama, Japan, 

	<sup>2</sup>Department of Chemical Biological Sciences, Japan Women?s University, Tokyo, Japan, 

	<sup>3</sup>Center for Gene Research, Nagoya University, Nagoya, Japan, 

	<sup>4</sup>Graduate School of Human Environmental Science, Ochanomizu University, Tokyo, Japan

</div>

<div> In plants most organs are composed of cells of various ploidy levels and endoreduplication is a key process in the plant&#8217;s developmental plan. <br/> We have isolated 'ilp1&#45;1D' as a dominant mutant in which polyploidy level is increased in the seedlings. 'ilp1&#45;1D' showed correlation between increased cell volume with polyploidy. 'ILP1' gene encodes for a novel nuclear protein and this protein functions as a transcriptional repressor in vivo. <br/> The expression of all the member of 'CYCA2' family was reduced in an 'ILP1' over&#45;expressing line. This indicates that ILP1 is a repressor that control 'CYCA2' expression. T&#45;DNA insertion mutants of 'CYCA2;1' has increased ploidy level. Here we demonstrate that ILP1 regulates endoreduplication through control of 'CYCA2' expression in 'Arabidopsis'.<br/> ILP1 has homologs not only in plants but also in insects and mammals, and the mouse ortholog of the ILP1 also repressed 'cyclin A2' expression in mouse NIH3T3 cells.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The Development and Evolution of Leaves</div>

<div>

	<span style="text-decoration: underline" >
	Neelima Sinha</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Davis, USA

</div>

<div>The Class I Knotted&#45;like homeobox &#40;KNOX 1&#41; genes are highly expressed in the shoot apical meristem but not expressed in the emerging leaf primordium in tobacco, maize, or Arabidopsis.  In tomato, KNOX1 expression &#40;LeT6, TKN1&#41; is seen in the early leaf primordium &#40;Chen et al. 1997; Hareven et al. 1996&#41;.  It is worth noting that tomato has compound leaves while the other organisms thus far tested have simple leaves.  We have analyzed compound leaf producing shoot apices in clades with independently derived compound leaves and shown that with one exception &#40;a derived clade in the Fabaceae&#41; compound leaves always show expression of KNOX genes &#40;Bharathan et al., 2002&#41;.  In the derived pea clade the LFY&#47;FLO gene regulates this function of generating leaf complexity.  While KNOX genes appear to be important for generating leaf complexity &#40;except in a derived clade in the Fabaceae&#41; we find that other genes like PHANTASTICA might play a role in determining the form of the compound leaf generated.  Transgenic plants overexpressing antisense PHAN suggest that PHAN, by modulating dorsiventrality, has a role in regulating the number of leaflets and their placement in a compound leaf.  The role of other key regulatory genes is also being explored in the context of normal and &#34;unusual&#34; leaf morphologies seen in nature.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Steroid binging protein in photomorphogenesis and BR signaling</div>

<div>

	<span style="text-decoration: underline" >
	Hong-Wei Xue</span>
	<sup>1</sup>, 

	<span  >
	Li Song</span>
	<sup>1</sup>, 

	<span  >
	Qiu-Ming Shi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Key Laboratory of Plant Molecular Genetics, Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Science, Chinese Academy of Sciences, China

</div>

<div>Steroid&#45;binding proteins &#40;SBPs&#41; are essential for growth and development of both animals and higher plants. Most known animal SBPs not only regulate the concentrations of general available steroids and hence control theirs metabolic clearance rates, but also function in steroid signal transduction through interacting with specific receptors on the plasma membrane of target cells, thus directly involve in regulation of cell growth and differentiation &#40;Rosner et al., 1999; Breuner and Orchinik, 2002&#41;. Four SBP coding genes have been annotated in Arabidopsis and MSBP1, which could bind progesterone, 5&#45;dihydrotestosterone, 24&#45;epi&#45;brassinolide &#40;24&#45;eBL&#41;, and stigmasterol with different affinities in vitro, and whose expression is regulated by light, serves as a negative regulator of cell elongation in Arabidopsis and plays negative roles in BR signaling. Our further studies indicate that expression of MSBP1 was regulated by key factors HY5&#47;HYH and involve in plant photomorphogenesis. In addition, MSBP1 suppress BR signaling through interactions with BAK1.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Prediction and characterization of genes exclusively expressed in the female gametophyte of <span style="font-style: italic">Arabidopsis thaliana</span>.</div>

<div>

	<span style="text-decoration: underline" >
	V&#237;ctor P&#233;rez-Espa&#241;a</span>
	<sup>1</sup>, 

	<span  >
	Nidia S&#225;nchez-Le&#243;n</span>
	<sup>1</sup>, 

	<span  >
	Mario Arteaga-V&#225;zquez</span>
	<sup>1</sup>, 

	<span  >
	Marcelina Garc&#237;a-Aguilar</span>
	<sup>1</sup>, 

	<span  >
	Octavio Mart&#237;nez-de la Vega</span>
	<sup>1</sup>, 

	<span  >
	Kan Nobuta</span>
	<sup>2</sup>, 

	<span  >
	Kalyan Vemaraju</span>
	<sup>2</sup>, 

	<span  >
	Blake Meyers</span>
	<sup>2</sup>, 

	<span  >
	Jean-Philippe Vielle-Calzada</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Laboratory of Reproductive Development and Apomixis, Langebio and Unidad Irapuato, Cinvestav. Campus Guanajuato, Irapuato M&#233;xico, 

	<sup>2</sup>Delaware Biotechnology Institute, University of Delaware Newark. DE, USA

</div>

<div>The female gametophyte of <span style="font-style: italic">Arabidopsis thaliana</span> is contained within the ovule and composed of seven cells: 3 antipodals, 2 synergids, the egg cell, and a binucleated central cell. We have used a combination of publically available microarray data and Massively Parallel Signature Sequencing &#40;MPSS&#41; information to identify candidate genes exclusively expressed in the female gametophyte. Initially, the paterrns of expression of MPSS&#45;detected genes that were active in wild&#45;type but not <span style="font-style: italic">nozzle&#47;sporocyteless</span> ovules were identified in ATH1 microarray experiments conducted by Twell. A group of 13 genes, not detected in other MPSS collections or in the overall microarray data available, were selected for genetic and molecular characterization. Some of these weakly expressed genes encode transcription factors or signaling proteins. Conventional and real&#45;time reverse&#45;transcription polymerase chain reaction  &#40;RT&#45;PCR&#41; confirmed that 4 are expressed in wild&#45;type but not in ovules lacking the female gametophyte. A genetic screen identified 2 insertional lines showing semi&#45;sterile phenotypes, suggesting a role at the gametophytic level. Whole&#45;mount in situ hybridization is being conducted to determine the pattern of mRNA localization of these genes and confirm unambiguously their gametophytic nature.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis DNA ligase I exerts a maternal gametophytic effect on seed development</div>

<div>

	<span style="text-decoration: underline" >
	Jing Li</span>
	<sup>1</sup>, 

	<span  >
	S&#233;bastien Andreuzza</span>
	<sup>1,2</sup>, 

	<span  >
	Anne-Elisabeth Guitton</span>
	<sup>1</sup>, 

	<span  >
	Jean-Emmanuel Faure</span>
	<sup>1,3</sup>, 

	<span  >
	Fr&#233;d&#233;ric Berger</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Temasek LifeScience Laboratory, 1 Research Link, National University of Singapore, 117604 Singapore, 

	<sup>2</sup>Ecole Normale Sup&#233;rieure de Lyon, 46 All&#233;e d?Italie, F-69364 Lyon cedex 07, France, 

	<sup>3</sup>Current address: European Commission, Directorate General for Research

</div>

<div>Flowering plant reproduction is characterized by a predominant maternal involvement in embryogenesis and production of seeds. However, only a limited number of maternal effect genes have been identified. Most of these genes are implicated in a single genetic pathway and the associated maternal effects are the consequence of imprinting mechanisms resulting in the sole expression of the maternal allele during seed development. Other mechanisms linked to maternal effects have been suspected but never demonstrated in a few other cases of non&#45;imprinted genes.<br/>In this study, we characterize a new mutant displaying maternal effect on seed development. The maternal effect results from loss&#45;of&#45;function of the gene encoding the Arabidopsis thaliana DNA ligase I &#40;AtLIG1&#41;. Genetic analyses show that atlig1 mutants display a complex array of phenotypes, causing some degree of lethality during male and female gametogenesis, embryogenesis and endosperm development. Maternal effects associated with atlig1 inheritance from the ovule primarily cause defects in endosperm development. Interactions between atlig1 and mutations affecting the maintenance DNA methyltransferase MET1 indicate that AtLIG1 may play a role in parental genomic imprinting.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">At5PTase13 is involved in root gravitropism through modulating vesicle trafficking</div>

<div>

	<span style="text-decoration: underline" >
	Yuan Wang</span>
	<sup>1</sup>, 

	<span  >
	Wen-Hui Lin</span>
	<sup>1</sup>, 

	<span  >
	Xu Chen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Physiology &amp;  Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China

</div>

<div>Inositol polyphosphate 5&#45;phosphatases &#40;5PTases&#41; is involved in phosphatidylinositol metabolism and affect various aspects of plant growth and development. Through molecular and genetic approaches, we are trying to study the physiological functions of the At5PTase13, which has been proved to regulate the homeostasis of auxin. The knockout mutant At5pt13 showed elevated sensitivity to gravistimulation in the root gravitropic response. Treatment with polar auxin transport inhibitor 1&#45;N&#45;naphthylphthalamic acid &#40;NPA&#41; indicated that At5PTase13 also altered auxin transport, which is closely related to gravitropism. Observations of the membrane&#45;selective dye FM4&#45;64 revealed that At5PTase13 negatively regulated vesicle trafficking and At5pt13 was less sensitive, comparing with wild type, to the inhibition of brefeldin A &#40;BFA&#41; on vesicle cycling. Further studies showed that BFA had less effect on seedling growth in At5pt13. These results suggest that At5PTase13 may have an influence on auxin transport through regulating vesicle trafficking and subsequently involve in root gravitropism.</div>

</div>





<div class="abs">

<div style="font-weight:bold">An Arabidopsis ARM repeat protein, ARCP1, is necessary for the anther dehiscence</div>

<div>

	<span style="text-decoration: underline" >
	Yu Mei</span>
	<sup>1</sup>, 

	<span  >
	Hong-Wei Xue</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Physiology &amp;  Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China

</div>

<div>Armadillo &#40;Arm&#41; repeat proteins are a large family of proteins containing tandem copies of an approximately 40 amino acid long sequence motif. Animal Arm repeat proteins function in various processes, including intracellular signalling and cytoskeletonal regulation. Over 100 Arm repeat proteins are predicted in Arabidopsis, but only few of them have been well characterized. Our study focused on ARCP1, an Arabidopsis Arm repeat protein which contains sixteen Arm repeats at the N terminus and a C2 domain at the C terminus. Studies through semi&#45;quantitative RT&#45;PCR and promoter&#45;GUS gene fusion showed that ARCP1 was constitutively expressed in various tissues with the strongest expression in roots, stigma and pollen grains. A T&#45;DNA knock&#45;out mutant of ARCP1 was further characterized and analyzed to examine its physiological functions. The homozygous mutant displayed a male sterile phenotype. Further analysis through light microscopy and scanning electron microscopy revealed that the sterility was caused by the defect of the mutant in the right time of anther dehiscence. Exogenous MeJA and expression of only the C2 domain could not rescue the sterile phenotype, suggesting that the multiple Arm repeats containing in ARCP1 were of great importance to its physiological functions and that the defect in dehiscence of anther at the correct time may be a result of MeJA insensitivity.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Investigation of possible DME&#45;downstream pathway genes using gene chip microarray</div>

<div>

	<span style="text-decoration: underline" >
	Hyonhwa Ohr</span>
	<sup>1</sup>, 

	<span  >
	Jean Choi</span>
	<sup>1</sup>, 

	<span  >
	Robert Fischer</span>
	<sup>2</sup>, 

	<span  >
	Yeonhee Choi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences, Seoul National University, Seoul, 151-742, Republic of Korea, 

	<sup>2</sup>Department of Plant and Microbiology, University of California, Berkeley, CA94720, USA

</div>

<div>DME &#40;DEMETER&#41; DNA glycosylase is an active demethylase excising 5&#45;methylcytosin and required for the maternal allele expression of the imprinted genes such as MEDEA, FWA and Fertilization&#45;Independent seed2 &#40;FIS2&#41;. Temporal and spatial expression of DME in the central cell of the female gametophyte, not in pollen, establishes gene imprinting by hypomethylating the maternal allele of the imprinted genes. While the biological relationship between DME and the target genes are well investigated, it is still unknown whether there are other genes regulated by DME. To gain insights of the possible DME&#45;downstream pathway genes, we generated transgenic plants showing ectopic DME expression under the CaMV 35S promoter in pollen.  In this study, we analyzed the microarray gene profile from pollen and stamen tissues comparing CaMV::DME transgenic plants to wild type.  We found only 91 genes were induced by ectopic DME expression in male tissues. Some of the possible DME target genes are involved in cell cycle and transcriptional regulation.  Semi&#45;quantitative RT&#45;PCR was performed to confirm and narrow&#45;down the candidate genes of gene chip analysis. The possible candidate genes in this study will be useful for the further research on DME DNA glycosylase regulatory mechanism. </div>

</div>





<div class="abs">

<div style="font-weight:bold">When partners don't talk: the molecular basis of cell&#45;cell communication during double fertilization</div>

<div>

	<span  >
	Juan-Miguel Escobar-Restrepo</span>
	<sup>1</sup>, 

	<span  >
	Norbert Huck</span>
	<sup>1</sup>, 

	<span  >
	Sharon Kessler</span>
	<sup>1</sup>, 

	<span  >
	Valeria Gagliardini</span>
	<sup>1</sup>, 

	<span  >
	Jacqueline Gheyselinck</span>
	<sup>1</sup>, 

	<span  >
	Wei-Cai Yang</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Ueli Grossniklaus</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Z&#252;rich, Z&#252;rich, Switzerland, 

	<sup>2</sup>Chinese Academy of Sciences, Beijing, China

</div>

<div>Research in our laboratory focuses on the developmental genetics of plant reproduction.  We have used genetic and molecular approaches to identify genes controlling the underlying developmental processes using Arabidopsis thaliana and Zea mays as model systems. Our studies have shown that both genetic and epigenetic mechanisms play a key role in plant reproduction. In this presentation I will focus on cell&#45;cell interactions during double fertilization. We have isolated a female gametophytic mutant, feronia, which disrupts double fertilization: in feronia mutant embryo sacs the pollen tubes, even if wild&#45;type, are unable to release the sperm cells to effect fertilization. This phenotype suggests that the female gametophyte plays a crucial role in pollen tube reception and, thus, controls the behavior of the male gametophyte &#40;Huck et al., 2003, Development 130:2149&#41;.  The feronia and sirene mutants, which display similar phenotypes &#40;Rotman et al., 2003, Curr Biol. 13:432&#41;, define novel signaling processes between the male and female gametophytes in the process of double fertilization. I will report on the molecular characerization of FERONIA, which is expressed at high levels in the synergid cells through which the pollen tube enters to effect fertilization. The molecular nature and subcellular localization of the FERONIA protein is consistent with its proposed function in a signaling process. Interestingly, some interspecific crosses result in phenotypes that are very similar to those observed in the feronia and sirene mutants. The evolutionary implications of these findings will be discussed. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Role of the Arabidopsis Retinoblastoma&#45;related &#40;RBR&#41; protein in cell specification</div>

<div>

	<span style="text-decoration: underline" >
	Wilhelm Gruissem</span>
	<sup>1</sup>, 

	<span  >
	Lorenzo Borghi</span>
	<sup>1</sup>, 

	<span  >
	Ueli Grossniklaus</span>
	<sup>2</sup>, 

	<span  >
	Ruben Gutzat</span>
	<sup>1</sup>, 

	<span  >
	Amal Johnson</span>
	<sup>2</sup>, 

	<span  >
	Johannes F&#252;tterer</span>
	<sup>1</sup>, 

	<span  >
	Yec'han Laizet</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ETH Zurich, Switzerland, 

	<sup>2</sup>University of Zurich, Switzerland

</div>

<div>The mammalian Retinoblastoma gene &#40;&#34;RB&#34;&#41; is best known for its role in G1&#47;S control and tumor formation, but other functions of the RB protein are still poorly understood. We have previously shown that loss of function of the Arabidopsis RB&#45;related &#40;RBR&#41; protein is gametophytically lethal and results in supernumerary nuclei &#40;1&#41;. The female gametophyte develops from the megaspore and consists of six haploid cells and one homo&#45;diploid cell that represent four different cell types, thus making it uniquely suited for dissecting the role of RBR in cell specification. Using markers for the egg cell, synergids, central cell and antipodal cells we could show that all cell types in rbr female gametophytes become mis&#45;specified. These results suggest that the maternal RBR contribution to the megaspore is not sufficient to properly connect the three ensuing miotic divisions to specification processes, or to direct specification during cellularization. To further test the hypothesis that RBR is required for cell specification, we constructed conditional alleles to perturb the RBR pathway during other developmental processes. Targeting RNAi&#45;induced downregulation of &#34;RBR&#34; expression to the root meristematic region produced supernumerary stem cells, suggesting that specification of cells exiting the stem cell was impaired &#40;2&#41;. Loss of a complementing &#34;RBRlox&#34; allele during leaf development does not block cell division, but &#34;rbr&#47;rbr&#34; sectors do not specify cells for stomata and trichome development. Similar results were obtained when RNAi&#45;induced downregulation of &#34;RBR&#34; expression was targeted to the shoot apical region during leaf primordia development. Together, the results support a model that RBR functions at a central node, which connects mitotic activity to cell specification processes. <br/><br/>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Ebel C, Mariconti L, Gruissem W. &#40;2004&#41; Plant retinoblastoma homologues control nuclear proliferation in the female gametophyte. Nature 429: 776&#45;780<br/>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wildwater M et al. &#40;2005&#41; The &#34;RETINOBLASTOMA&#45;RELATED&#34; gene regulates stem cell maintenance in Arabidopsis roots. Cell 123: 1337&#45;1349<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Function of DNA&#45;repair genes in meiosis</div>

<div>

	<span style="text-decoration: underline" >
	Hong Ma</span>
	<sup>1</sup>, 

	<span  >
	Wuxing Li</span>
	<sup>1</sup>, 

	<span  >
	Rong Xiao</span>
	<sup>1</sup>, 

	<span  >
	Asela Wijeratne</span>
	<sup>1</sup>, 

	<span  >
	Wei Zhang</span>
	<sup>1</sup>, 

	<span  >
	Changbin Chen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Pennsylvania State University

</div>

<div>Plants face a variety of stresses, including DNA&#45;damaging radiation and chemicals. We have studied the function of Arabidopsis homologs of DNA repair genes, including the members of the RAD51 gene family. We found that some of these genes are involved in DNA repair during mitotic growth, whereas others genes have no detectable effects on the growth of plants in response to DNA&#45;damaging agents. Furthermore, many of these genes have critical functions during meiosis. We present results from detailed analysis of the meiotic functions of these RAD51 homologs.  In addition, we have performed genetic analysis to test for possible interactions among members of this gene family.  We have also identified novel meiotic genes using expression profiling. Two of these genes are homologs of known DNA repair genes. Our studies support the idea that homologs of DNA repair genes play essential functions during meiosis to ensure proper chromosome segregation, and that they also seem to play a role in DNA repair during mitotic growth.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Investigating the Role of <span style="font-style: italic">Arabidopsis thaliana</span> Genes At1G14120 and At1G14130 in Adventitious Rooting.</div>

<div>

	<span style="text-decoration: underline" >
	Colm Nolan</span>
	<sup>1</sup>, 

	<span  >
	Thomas Gallagher</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Biology and Environmental Science, University College Dublin, Dublin, Ireland.

</div>

<div>Adventitious rooting is a complex process, the control mechanisms of which are still poorly understood. It is mediated by several factors, both light and hormones &#40;especially auxin&#41; play a central role and may interact with other endogenous factors or environmental stimuli. Arabidopsis genes At1g14120 and At1g14130 show sequence homology to Apple &#40;<span style="font-style: italic">Malus domestica</span>&#41; ARRO&#45;1 &#40;adventitious rooting related oxygenase&#41;, which is up&#45;regulated during auxin&#45;induced adventitious rooting, encoding a dioxygenase activity.<br/>Full length gene nucleotide sequences under the control of the cauliflower mosaic virus 35S promoter have been expressed in lines of transgenic Arabidopsis plants to cause constitutive and ectopic gene expression. For underexpressors&#47;knockouts, hairpin loop&#45;forming RNAi constructs have similarly been engineered, to promote  cleavage by the plant cell of both the transgene&#45;derived and the endogenous messenger RNA, reducing&#47;obliterating endogenous gene expression. A gene knockout line &#40;by T&#45;DNA insertion&#41; for AT1G14130 has been obtained and the gene sequence disruption verified. Real time PCR will be carried out with short PCR products and comparisons made between different tissues of the plant which will complement results from microarray experiments. Absolute mRNA copy number in overexpressors&#47; knockouts and RNAi plants can also be estimated using this method. In gene promoter studies, At1G14120 and At1G14130 full length promoter fragments have been generated with an in&#45;frame fusion to the GUS reporter gene, to colorimetically display potential spatial and temporal patterning in gene expression. Cell and plant level gene product transportation and localisation is also being examined by the expression of  the gene nucleotide sequences fused in&#45;frame to the EYFP reporter gene.<br/></div>

</div>


</div>
</div>

</body>
</html>
