<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>
<head>
<title>ICAR 2007 - Metabolism</title>
<link rel="Shortcut Icon" href="/favicon.ico">
<link rel="stylesheet" type="text/css" href="/css/main.css" />
<link rel="stylesheet" type="text/css" href="/css/search.css" />
<link rel="stylesheet" type="text/css" href="/css/base.css" />

<style type="text/css">
@import "/css/page/pagelevels.css";
</style>

<style type="text/css">
.abs {
 margin-top: 30px;
}
.abs div {
 margin: 10px;
}
</style>
</head>
<body>

<div id="container">

<script src="https://ssl.google-analytics.com/urchin.js" type="text/javascript"></script>
<script type="text/javascript">
_uacct = "UA-1945230-1";
urchinTracker();
</script>

 <div id="nav">	
  <div id="logo"><a href="/index.jsp"><img src="/i/logo2.gif" width="87" height="61" alt="tair"/></a></div>
 </div>

<div id="content" class="clearfix">

<div class="abs">

<div style="font-weight:bold">REM18 and REM53: two direct targets of the ovule identity complex of Arabidopsis &#13;<br/>Luis Matias&#45;Hernandez1, Raffaella Battaglia1 , Vittoria Brambilla1, Marco Rubes1 , Martin M. Kater2, and Lucia Colombo1&#13;<br/>1Dipartimento di Biologia, Universit</div>

<div>

	<span style="text-decoration: underline" >
	Luis Matias-Hernandez</span>
	<sup>1</sup>, 

	<span  >
	Lucia Colombo</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Milano

</div>

<div>Three genes responsible for ovule identity determination in Arabidopsis thaliana have been identified which are SEEDSTICK &#40;STK&#41;, SHATTERPROOF1 &#40;SHP1&#41; and SHATTERPROOF2 &#40;SHP2&#41; &#40;Pinyopich et al., 2003&#41;, all of them belonging to MADS&#45;box transcription factors family. STK is the only one specifically expressed in the ovule, while SHP1 and SHP2 are also involved in carpel formation. STK, SHP1, and SHP2 form a multimeric complex together with SEPALLATA &#40;SEP&#41; MADS&#45;box proteins to regulate ovule development &#40;Favaro et al., 2003&#41;.&#13;<br/>To identify targets of this ovule identity complex we have done microarray analysis using RNA isolated through Laser Capture Microdisection &#40;LCM&#41; from ovules primordia. Once a small subset of transcription factor genes were identified, based on their expression profile and having in their regulatory regions multiple MADS&#45;box binding sites, Chromatin Immunoprecipitation &#40;ChIP&#41; experiments were performed to confirm the in vivo targets of the ovule identity complex.&#13;<br/>Two novel genes REM18 &#40;At5g18000&#41;and REM53 &#40;At3g53310&#41; have been demonstrated to be targets using ChIP techniques, and we have further confirmed these results through in situ hybridization analyses. REM18 and REM53 belongs to REproductive Meristem &#40;REM&#41; family &#40;Franco&#45;Zorrilla et al., 2002&#41;, an almost unknown family of transcription factors unique in plants. Both genes are developmentally regulated, firstly expression is localized into few cells in the shoot apical meristem, secondly REM18 and REM53 expression expand to the whole reproductive meristem, and finally is restricted to the ovule in later stages of flower development. In the triple mutant stk&#45;shp1&#45;shp2 expression of REM18 and REM53 is not detected inside the ovule , confirming that these two novel genes are the first targets of the ovule identity complex.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Chromosomal context of transgenes affects RNA&#45;directed transcriptional gene silencing and DNA methylation.</div>

<div>

	<span  >
	Markus Kuhlmann</span>
	<sup>1</sup>, 

	<span  >
	Katja Richert-Poeggeler</span>
	<sup>1</sup>, 

	<span  >
	Ute Fischer</span>
	<sup>1</sup>, 

	<span  >
	Ales Pecinka</span>
	<sup>2</sup>, 

	<span  >
	Renate Schmidt</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Michael Florian Mette</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Leibniz Institute of Plant Genetics and Crop Plant Research, 06466 Gatersleben, Germany, 

	<sup>2</sup>present address: Gregor Mendel Institute of Molecular Plant Biology, 1030 Vienna, Austria

</div>

<div>RNA&#45;directed transcriptional gene silencing of transgenes in plants provides a versatile experimental system for the study of epigenetic gene regulation. In such a setup, transcription of a promoter inverted repeat provides an RNA signal that can trigger transcriptional gene silencing and methylation of unlinked homologous promoters in trans. In order to determine the contribution of chromosomal localisation to gene inactivation, 26 single copy target transgenes integrated at defined positions of the Arabidopsis thaliana genome were combined by crosses with a silencer transgene containing a transcribed pNOS inverted repeat. Indeed, the activity and the degree of silencing of a pNOS&#45;NPTII reporter gene present in the target transgenes was found to vary for the different chromosomal integration sites. In absence of the silencer, NPTII transcript levels deviated at maximum 10&#45;fold among the target transgenes. When challenged by the silencer transgene providing the pNOS RNA signal, reduction of the NPTII expression in the F1 generation varied more than 100&#45;fold ranging from no reduction to reduction to less than 1 per cent of the non&#45;silenced level. Generally, silencing was correlated with proportional levels of DNA methylation in the pNOS region, but for one target transgene, substantial DNA methylation without adequate silencing was observed. Silencing and methylation were progressive through sexual reproduction, but differences between target transgenes were maintained to the F3 generation. The efficiency of silencing of target transgenes appeared to be mainly determined by their flanking sequences, with repeats promoting and functional genes diminishing the response to the silencing signal. Current results from further analysis will be presented. The work is supported by German Research Foundation grant SFB 648 C4.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis Genome Annotation &#45; Challenges and Opportunities</div>

<div>

	<span style="text-decoration: underline" >
	Eva Huala</span>
	<sup>1</sup>, 

	<span  >
	David Swarbreck</span>
	<sup>1</sup>, 

	<span  >
	Chris Wilks</span>
	<sup>1</sup>, 

	<span  >
	Robert Muller</span>
	<sup>1</sup>, 

	<span  >
	Philippe Lamesch</span>
	<sup>1</sup>, 

	<span  >
	Tanya Berardini</span>
	<sup>1</sup>, 

	<span  >
	Hartmut Foerster</span>
	<sup>1</sup>, 

	<span  >
	Donghui Li</span>
	<sup>1</sup>, 

	<span  >
	Vanessa Swing</span>
	<sup>1</sup>, 

	<span  >
	Christophe Tissier</span>
	<sup>1</sup>, 

	<span  >
	Peifen Zhang</span>
	<sup>1</sup>, 

	<span  >
	Seung Rhee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Carnegie Institution, Stanford, CA, USA

</div>

<div>In March 2007 TAIR released a new version of the Arabidopsis genome annotation, TAIR7, incorporating community submissions directly to TAIR and new cDNAs and ESTs submitted to GenBank since the previous TAIR6 release.  The new release includes 681 new genes bringing the total gene set to 32,041 genes, of which 26,819 are protein&#45;coding, 3889 are pseudogenes or transposable elements and 434 are ncRNAs.  The release also contains updates to 9755 genes, including 784 updates to protein sequences  and addition of 1003 new splice variants as well approximately 10,700 updates to UTRs.  A total of 34 gene merges and 41 gene splits were also carried out.  TAIR7 has been released to GenBank and can be accessed from the NCBI Plant Genomes section as well as through TAIR.  Current progress and future plans for maintaining and updating the Arabidopsis genome annotation will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">THE FOX HUNTING SYSTEM: AN ALTERNATIVE GAIN&#45;OF&#45;FUNCTION GENE HUNTING TECHNIQUE.</div>

<div>

	<span style="text-decoration: underline" >
	Takanari Ichikawa</span>
	<sup>1</sup>, 

	<span  >
	Miki Nakazawa</span>
	<sup>1</sup>, 

	<span  >
	Mika Kawashima</span>
	<sup>1</sup>, 

	<span  >
	Iizumi Haruko</span>
	<sup>1</sup>, 

	<span  >
	Kuroda Hirofumi</span>
	<sup>1</sup>, 

	<span  >
	Kondou Youichi</span>
	<sup>1</sup>, 

	<span  >
	Akie Ishikawa</span>
	<sup>1</sup>, 

	<span  >
	Seki Motoaki</span>
	<sup>1</sup>, 

	<span  >
	Miki Fujita</span>
	<sup>2</sup>, 

	<span  >
	Mieko Higuchi</span>
	<sup>1</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>1</sup>, 

	<span  >
	Minami Matsui</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN PSC, 

	<sup>2</sup>CREST JST

</div>

<div>We have developed a novel gain&#45;of&#45;function system that we have named the FOX hunting system &#40;Full&#45;length cDNA over&#45;expressing gene hunting system&#41;. We used normalized full&#45;length cDNA and introduced each cDNA into Arabidopsis by in planta transformation. About 10,000 independent Arabidopsis full&#45;length cDNAs were expressed independently under the CaMV 35S promoter in Arabidopsis. Each transgenic Arabidopsis contained on average 2.6 cDNA clones and was monitored under various categories such as morphological changes, fertility and leaf color. We found 1,487 possible morphological mutants from 15,547 transformants. When 115 pale green T1 mutants were analysed, 59 lines represented the mutant phenotypes in more than 50&#37; of the T2 progeny. Characterization of two leaf color mutants revealed the significance of this approach. We will also introduce 6 other mutant lines that showed the visible phenotypes upon retransformation of the wild type plants with the rescured full&#45;length cDNAs. Applications of this system for functional genomics will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The nucleolus is involved in mRNA surveillance in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Peter Shaw</span>
	<sup>1</sup>, 

	<span  >
	Ali Pendle</span>
	<sup>1</sup>, 

	<span  >
	Olga Koroleva</span>
	<sup>1</sup>, 

	<span  >
	Peter McKeown</span>
	<sup>1</sup>, 

	<span  >
	Dominika Lewandowska</span>
	<sup>2</sup>, 

	<span  >
	Sang Hyon Kim</span>
	<sup>2</sup>, 

	<span  >
	John Brown</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>John Innes Centre, Norwich, UK, 

	<sup>2</sup>Scottish Crop Research Institute, Dundee, Scotland

</div>

<div>The nucleolus is the site of rDNA transcription, rRNA processing and ribosomal subunit assembly, but is also involved in a range of other RNA&#47;RNP functions, cell cycle control and stress responses. To investigate the wider functions of the nucleolus in plants, we carried out a proteomic analysis of Arabidopsis nucleoli, in which we identified 217 proteins of which about 70&#37; were also present in the human nucleolus. The proteins include many expected nucleolar proteins involved in ribosome biogenesis, ribosomal proteins, snoRNP core proteins, DEAD box helicases, RNA&#47;DNA&#47;chromatin interacting proteins, splicing and translation factors and RNA transport factors. In addition, proteins with unknown function were identified, some of which were plant&#45;specific proteins and some of which were found in both the Arabidopsis and human nucleolar proteomes. <br/>Full&#45;length cDNAs for 76 of the identified proteins were expressed as GFP&#45;protein fusions in Arabidopsis cells. Nearly 90&#37; of the protein fusions are clearly nucleolar or associated with the nucleolus. Many also show labelling of other nuclear bodies. Surprisingly, the Arabidopsis nucleolar proteome contained components of the exon junction complex &#40;EJC&#41;, which links transcription and splicing with mRNA export, surveillance and decay. GFP&#45;fusions for these and a number of other EJC proteins confirmed an association with the nucleolus. <br/>Evidence for a role for the plant nucleolus in NMD was obtained from an examination of nucleolar mRNAs. Poly A&#43; mRNA libraries were made from purified nucleoli, as well as purified nuclei and whole cell extracts. Full&#45;length sequences were obtained from randomly selected clones from each library. This showed that the nucleolar library was highly enriched in aberrant or mis&#45;spliced mRNAs. Furthermore, GFP fusions to upf3 and upf2, proteins known to be associated with nonsense&#45;mediated decay &#40;NMD&#41;, were clearly localised to the nucleolus. Taken together these results show that spliced or aberrant mRNAs move through the nucleolus, either as part of the mRNA export process or in mRNA surveillance and NMD.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">GREENPHYL: A comparative genomic platform for Rice and Arabidopsis.</div>

<div>

	<span style="text-decoration: underline" >
	Conte MG</span>
	<sup>1</sup>, 

	<span  >
	Gaillard S</span>
	<sup>1</sup>, 

	<span  >
	Guiderdoni E</span>
	<sup>1</sup>, 

	<span  >
	P&#233;rin C</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CIRAD

</div>

<div>Gene ortholog identification has become a major objective for mining the increasing amount of sequence data generated by full or partial genome sequencing projects. Comparative and functional genomics urgently need reliable ways to identify orthologs to shorten gene function inference and conservation&#47;divergence of genetic pathways between species. <br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We developed a two&#45;step strategy for ortholog prediction between <span style="font-style: italic">Oryza Sativa</span> and <span style="font-style: italic">Arabidopsis thaliana</span> using transcription factors genes as a golden test case. We first evaluated the feasibility of accurate gene family clustering of <span style="font-style: italic">Arabidopsis thaliana</span> and <span style="font-style: italic">Oryza sativa</span> genes using TribeMCL software. We then developed GreenPhyl, an optimized phylogenomic pipeline which can predict ortholog and paralog relationships based on gene family clustering. The outcomes of our studies show that GreenPhyl achieved high accuracy level in predicting ortholog&#47;paralog relationships. We then demonstrated that a phylogenomics approach is suitable for plants and outperform the similarity methods for ortholog prediction. <br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;GreenPhylDB &#40;http:&#47;&#47;greenphyl.cines.Fr&#41;, a database dedicated for <span style="font-style: italic">Oryza sativa</span> and <span style="font-style: italic">Arabidopsis thaliana</span> gene family classification and ortholog predictions, has been developed in order to store the results of the GreenPhyl pipeline. Until now, GreenPhylDB contains 5800 manually annotated plant gene families with cross references &#40;Uniprot, KEGG, Interpro, TAIR, TIGR, OrygenesDB&#41;. Currently 4000 of these plant families were phylogenetically analysed. In addition, we have developed a set of tools around GreenPhylDB to retrieve as exhaustively as possible information related to queries with several starting points: TAIR or TIGR ID, sequence, keywords, Interpro, KEGG, family name, gene name...<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Dual Autonomous Elements for the MITE mPing</div>

<div>

	<span style="text-decoration: underline" >
	Guojun Yang</span>
	<sup>1</sup>, 

	<span  >
	Feng Zhang</span>
	<sup>1</sup>, 

	<span  >
	C. Nathan Hancock</span>
	<sup>1</sup>, 

	<span  >
	Susan Wessler</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Georgia

</div>

<div>The first active miniature inverted repeat transposable element &#40;MITE&#41;, mPing, was discovered by computer&#45;assisted analysis of rice genome sequence.  The mPing element is mobile in rice cell culture and in a few rice strains where it has amplified to over 1000 copies during recent domestication. However, determination of the transposase source and characterization of the mechanism of transposition have been hampered by the high copy number of mPing and the presence of several candidate autonomous elements in the rice genome.  Here we report that mPing is active in Arabidopsis thaliana where its transposition is catalyzed by three sources of transposase from rice: the autonomous Ping and Pong elements and by a cDNA derived from a Ping transcript.  In addition to transposase, the product of a second element&#45;encoded ORF of unknown function is also required for mPing transposition. Excision of mPing in A. thaliana is usually precise and transposed copies usually insert into unlinked sites in the genome that are preferentially in or near genes.  As such, this will be a valuable assay system for the dissection of MITE transposition and a potentially powerful new tagging system for gene discovery in eukaryotes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">CHROMATIN CHARTING: Global mapping and characterization of epigenetic control mechanisms.</div>

<div>

	<span style="text-decoration: underline" >
	Eric Lam</span>
	<sup>1</sup>, 

	<span  >
	Faye Rosin</span>
	<sup>1</sup>, 

	<span  >
	Naohide Watanabe</span>
	<sup>1</sup>, 

	<span  >
	Naohiro Kato</span>
	<sup>1</sup>, 

	<span  >
	Juana-Mari Arroyo</span>
	<sup>2</sup>, 

	<span  >
	Jean-Luc Cacas</span>
	<sup>1</sup>, 

	<span  >
	Joe Simorowski</span>
	<sup>2</sup>, 

	<span  >
	Umamaheswari Ramu</span>
	<sup>2</sup>, 

	<span  >
	Matthew Vaughn</span>
	<sup>2</sup>, 

	<span  >
	Yuda Fang</span>
	<sup>2</sup>, 

	<span  >
	Bruce May</span>
	<sup>2</sup>, 

	<span  >
	Richard McCombie</span>
	<sup>2</sup>, 

	<span  >
	David Spector</span>
	<sup>2</sup>, 

	<span  >
	Robert Martienssen</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Rutgers University, New Brunswick, NJ, USA, 

	<sup>2</sup>Cold Spring Harbor laboratory, Cold Spring Harbor, NY, USA

</div>

<div>In the phenomena known as position effect &#40;PE&#41;, integration location is hypothesized to affect the expression of transgenes. To globally characterize PEs in the genome, which is likely mediated by epigenetic mechanisms, we generated 611 Ds&#45;tagged lines of Arabidopsis, designated CCT lines. The Ds&#45;tagging cassette contains a CaMV 35S::luciferase gene which allows monitoring of PEs, and a lac operator repeat array that allows the visual tracking of the tagged loci by expressing fusions of fluorescent proteins with LacI within the nuclei of live plants. By inserting the same 35S::Luc cassette into various loci within the genome together with the pNos::NPTII selection marker for kanamycin resistance, we seek to quantify and assess the chromosomal location contribution to gene expression that is superimposed on cis&#45;element&#45;specific controls on the global level &#40;i.e. the &#34;transcription potential&#34; for a particular location within the genome&#41;.  Our first gene expression potential map with 277 CCT lines dispersed over all five chromosomes revealed that over 80&#37; of the insertions have gene expression levels that are within 2&#45;fold of the average activity.  Thus, for the majority of locations within the arabidopsis genome, we conclude that the activity of a particular transgene is essentially constant and PE is not a factor.  The remaining 15 to 20&#37; of the insertion loci, however, exhibit significant enhancement or suppression of one or both of our marker genes. We designate the latter class as Position Effect Loci &#40;PEL&#41;.  Focusing on a 100 kb region on the top arm of Chr. 2, we found that insertions into a 50 kb &#34;heterochromatin island&#34; adjacent to the Nucleolar Organizing Region &#40;NOR&#41; are silenced more than 10 fold in their associated gene activities, with root tissues exhibiting much more extensive silencing. Hypermethylation at CpG sites appears to be an important contributor for this PE silencing and is correlated with physical constraint of the silenced locus to the nucleolus.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">Antagonists of RNA&#45;directed transcriptional gene silencing &#40;TGS&#41; and DNA methylation in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Katja Richert-Poeggeler</span>
	<sup>1</sup>, 

	<span  >
	Quanan Hu</span>
	<sup>2</sup>, 

	<span  >
	Markus Kuhlmann</span>
	<sup>1</sup>, 

	<span  >
	Renate Schmidt</span>
	<sup>1</sup>, 

	<span  >
	M. Florian Mette</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Leibniz Institute of Plant Genetics and Crop Plant Research, D-06466 Gatersleben, Germany, 

	<sup>2</sup>Botanical Institute, University of Basel, CH-4056 Basel, Switzerland

</div>

<div>RNA&#45;directed TGS and DNA methylation, a branch of RNAi operating in the cell nucleus, has been extensively studied in plants. Work has mainly focused on the analysis of molecular features of the silenced state such as DNA methylation patterns as well as on the identification of RNAs and proteins involved in its establishment and propagation. A two component transgene system in <span style="font-style: italic">Arabidopsis</span> has proven particularly powerful in the genetic analysis of RNA&#45;directed TGS and DNA methylation. The &#8220;silencer&#8221; transgene provides dsRNA homologous to a particular gene promoter, whereas the &#8220;target&#8221; transgene contains a reporter gene under the control of the same promoter which undergoes RNA&#45;directed TGS and DNA methylation in the presence of the &#8220;silencer&#8221;. &#13;<br/>Recent studies &#40;Zhu et al., 2007 and references therein&#41; have revealed that DNA methylation in <span style="font-style: italic">Arabidopsis</span> can be actively removed by plant enzymes like the DNA glycosylase&#47;lyase ROS1. We are introgressing a ros1 &#8220;loss of function&#8221; mutation into our transgene system to analyse its possible involvement. Two different &#8220;targets&#8221; are being used. At a &#8220;target&#8221; resistant to RNA&#45;directed DNA methylation that acquires only little promoter methylation in presence of the &#8220;silencer&#8221;, we will test whether the ros1 mutation will lead to increased methylation levels. This would indicate a role in counteracting de novo DNA methylation. At a &#8220;target&#8221; acquiring high promoter methylation in presence of the silencer, we will analyse whether the inheritance of DNA methylation after segregation of the &#8220;silencer&#8221; will be increased by the ros1 mutation. This would indicate a role in counteracting maintenance DNA methylation.&#13;<br/>Furthermore we are setting up a new transgene system involving a conditional negative selectable marker for the acquisition of novel mutations enhancing RNA&#45;directed transcriptional gene silencing.&#13;<br/>The work is supported by MK&#45;LSA project 3555A&#47;1203T.&#13;<br/>&#13;<br/>References&#13;<br/>&#91;1&#93; Zhu, J. et al. 2007. Current Biology 17: 54&#45;59.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular Analyses of the Plant WNK Gene Family and Evidence for their Regulation of Flowering Time in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Yingxiang Wang</span>
	<sup>1</sup>, 

	<span  >
	Huiyan Wan</span>
	<sup>1</sup>, 

	<span  >
	Hong Liao</span>
	<sup>1</sup>, 

	<span  >
	Chuxiong Zhuang</span>
	<sup>2</sup>, 

	<span  >
	Hong Ma</span>
	<sup>3</sup>, 

	<span  >
	Xiaolong Yan</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Lab of Plant Nutritional Genetics, Root Biology Center, South China Agricultural University, Guangzhou 510642, China, 

	<sup>2</sup> Lab of Genetic Engineering, College of Life Science, South China Agricultural University, Guangzhou 510642, China, 

	<sup>3</sup>Department of Biology and Huck Institutes of the Life Sciences, Pennsylvania State University, University Park, PA 16802, USA

</div>

<div>The WNK &#40;with no lysine kinase&#41; protein kinase gene family was recently found as a novel gene family possibly involved in regulating the ion permeability of epithelia in mammals. However, WNK&#45;like genes in plants have not been well characterized and their functions are largely unknown. We have previously identified a novel gene encoding a WNK homolog from soybean root through suppression subtraction hybridization &#40;SSH&#41;. By searching the NCBI and TAIR databases, we found that there are 10 WNK genes in the Arabidopsis genome, designated as AtWNK1&#45;10, respectively. The predicted sizes Arabidopsis WNK proteins range from 492 to 701 amino acids and they share 59&#37; to 91&#37; amino acid identities in the conserved kinase domain. Phylogenetic analysis of Arabidopsis and rice WNK genes suggests that the most recent common ancestor of monocots and eudicots had 4 to 8 WNK genes, and that the WNK gene family may have experienced some duplication events, as well as possible gene losses. Semi&#45;quantitative reverse transcription polymerase chain reaction &#40;RT&#45;PCR&#41; experiments revealed that all Arabidopsis WNK genes except AtWNK6 are expressed in various organs from the seedling to flowering plant. T&#45;DNA knockout mutations in the AtWNK2, AtWNK5 and AtWNK8 genes in different phylogenetic clades cause early flowering under long day conditions, suggesting that these genes may regulate flowering time. In contrast, a T&#45;DNA knockout wnk1 mutant exhibited a distinct phenotype, including severe growth retardation, rounded leaf shape, much delayed flowering time and somewhat shorter and thicker siliques compared with the wild type. Our results suggest that the Arabidopsis WNK gene family may be involved in regulating flowering time and other processes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic Redundancy and Recessive Antimorphism: Insights From Genetic Analyses Of The Ribonucleotide Reductase Small Subunit Mutants tso2.</div>

<div>

	<span style="text-decoration: underline" >
	Zhongchi Liu</span>
	<sup>1</sup>, 

	<span  >
	Chunxin Wang</span>
	<sup>1</sup>, 

	<span  >
	Connie Yoon</span>
	<sup>1</sup>, 

	<span  >
	Boyana Grigorova</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Maryland

</div>

<div>Genetic redundancy often hinders our ability to decipher gene function. For example, T&#45;DNA insertional mutations in Arabidopsis often fail to cause a mutant phenotype due to multiple copies of similar genes in the genome. We observed and characterized a phenomenon coiled &#34;recessive antimorphism&#34;, which is currently poorly understood and may provide novel insights in the study of functionally redundant genes. <br/><br/>Ribonucleotide reductase &#40;RNR&#41; converts NDPs into dNDPs for DNA synthesis and repair and thus is essential for growth and development.  RNR is a tetrameric protein complex consisting of two large subunits &#40;R1&#41; and two small subunits &#40;R2&#41;.  In Arabidopsis, the R2 small subunit is encoded by three functionally redundant genes: TSO2, RNR2A and RNR2B &#40;Wang and Liu, 2006 Plant Cell&#41;. While tso2&#45;5, a T&#45;DNA insertion near the N&#45;terminus of TSO2 protein, exhibited no abnormality, four independently isolated tso2 missense mutations &#40;tso2&#45;1, 2, 3, 4&#41; caused severe developmental defects and a reduction of dNTP level, suggesting that the tso2 missense mutations might be antimorphic.  While previously reported antimorphic alleles are always dominant, the four tso2 missense alleles are completely recessive, raising the possibility that these tso2 alleles are recessive&#45;antimorphic. We hypothesize that the antimorphic &#40;poisonous&#41; effect of TSO2 mutant proteins depends on to their ratio to the wild type R2 proteins encoded by TSO2, RNR2A and RNR2B. To test this hypothesis, the missense TSO2&#45;1 mutant gene under 35S promoter as well as TSO2 native promoter was introduced into wild type Arabidopsis to test if the TSO2&#45;1 mutant gene could induce tso2&#45;like mutant phenotype.  Our data support that the TSO2&#45;1 mutant gene  exhibits antimorphic effect.  However, the phenomenon is more complex and will be described in more detail.<br/><br/>Our study of recessive antimorphism provides novel insights in the study of functionally redundant genes. It is likely that missense&#45;based recessive antimorphic alleles are more effective in eliminating gene function by interfering with wild type proteins produced by other gene family members. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Evolutionary Genomics of self&#45;incompatibility in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Kentaro Shimizu</span>
	<sup>1</sup>, 

	<span  >
	Michael Purugganan</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>University of Zurich, Institute of Plant Biology, 

	<sup>2</sup>North Carolina State University

</div>

<div> While millions species are living on the earth with astonishing diversity, little is known about two fundamental questions regarding molecular basis of biodiversity. Which genes are responsible for adaptation and for speciation&#63; What evolutionary processes affect allele frequency of these genes&#63; The integration of molecular genetics, ecology and evolutionary genomics are critical to answer the questions. In particular, genomics bridges field ecology and laboratory molecular genetics in two ways. First, the isolation of ecologically&#45; and evolutionary&#45;relevant genes is facilitated by genomic tools. Second, genome&#45;wide polymorphism is essential to deduce natural selection and population history affecting allele frequency of these genes.<br/>As a case study using a model species Arabidopsis thaliana, we will talk about the evolution of selfing, which is one of the most prevalent evolutionary trends in flowering plants. In 1876, Charles Darwin proposed the reproductive assurance model to explain the prevalence of self&#45;pollination in plants, suggesting that selfing can be evolutionarily advantageous when pollinators or mates are scares in spite of inbreeding depression. In the selfing plant A. thaliana, pseudogenes at the SCR&#47;SP11 and SRK self&#45;incompatibility loci are shown to underlie the evolution of selfing. We found lowest nucleotide diversity at SCR in the genomic region. This suggests the positive selection on selfing and supports the Darwin's hypothesis. Parallel disruption of those genes and the timing will be discussed in conjunction with its population structure.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of Ubiquitin&#45;Specific Proteases in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Yanfen Liu</span>
	<sup>1</sup>, 

	<span  >
	Feng Wang</span>
	<sup>1</sup>, 

	<span  >
	Huiyong Zhang</span>
	<sup>1</sup>, 

	<span  >
	Hang He</span>
	<sup>1</sup>, 

	<span  >
	Xing Wang Deng</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute of Biological Sciences, Beijing, China

</div>

<div>Ubiquitin&#45;Specific Proteases &#40;UBPs&#41; are highly conserved family of proteins in eukaryotes that play critical roles in cellular protein de&#45;ubiquitination. A total of 27 UBPs are present in the <span style="font-style: italic">Arabidopsis thaliana</span> genome, which can be further grouped into 14 subfamilies based on their sequence similarity. Microarray data provide detailed expression profiles of those <span style="font-style: italic">UBP</span>s and an analysis of 38 available T&#45;DNA insertion lines in 25 specific <span style="font-style: italic">UBP</span>s covering all 14 subfamilies result in that only mutants in 2 subfamilies exhibited observable phenotypes. One of the subfamilies, <span style="font-style: italic">AtUBP15</span> subfamily, whose members also possess a single zinc&#45;finger domain, contains 5 individual genes, mutations in which leading to 3 distinct phenotype consequences. <span style="font-style: italic">AtUBP15</span> mutants display narrower, serrated and flat rosette leaves partially due to lack of cell proliferation, as well as other phenotype such as early flowering, apical dominance losing, short or sometimes unfertile siliques, while the overexpression lines result in opposite phenotype. Both <span style="font-style: italic">in vitro</span> and <span style="font-style: italic">in vivo</span> assays confirm it is a <span style="font-style: italic">bona fide</span> de&#45;ubiqutinating enzyme and Cysteine447 is essential for this activity. Further transcriptome analysis show expression level of 804 genes are miss&#45;regulated in <span style="font-style: italic">Atubp15&#45;1</span>, including down regulation of a MADS box &#40;<span style="font-style: italic">MAF5</span>&#41; gene. Phenotype of double mutant <span style="font-style: italic">Atubp15</span> <span style="font-style: italic">Atubp16</span> is more severe than that of <span style="font-style: italic">AtUBP15</span> mutant, whereas <span style="font-style: italic">Atubp16</span> <span style="font-style: italic">Atbubp17</span> is comparable with <span style="font-style: italic">AtUBP15</span> mutant, implying <span style="font-style: italic">AtUBP15</span> and <span style="font-style: italic">AtUBP16</span> but not <span style="font-style: italic">AtUBP17</span> involve in the same pathway. Intriguingly, mutation of another subfamily member <span style="font-style: italic">AtUBP19</span> leads to embryo lethality. Distinct <span style="font-style: italic">AtUBP</span>s genes, even within closely related subfamilies, function in different while sometimes overlapped developmental pathways to regulate protein de&#45;ubiquitination.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis Whole&#45;Genome Transcriptome Analysis under Drought, Cold, High&#45;Salinity, and ABA Treatment Conditions using Tiling Array and 454 Sequencing Technology</div>

<div>

	<span style="text-decoration: underline" >
	Motoaki Seki</span>
	<sup>1</sup>, 

	<span  >
	Akihiro Matsui</span>
	<sup>1</sup>, 

	<span  >
	Jong-Myong Kim</span>
	<sup>1</sup>, 

	<span  >
	Junko Ishida</span>
	<sup>1</sup>, 

	<span  >
	Maiko Nakajima</span>
	<sup>1</sup>, 

	<span  >
	Taeko Morosawa</span>
	<sup>1</sup>, 

	<span  >
	Makiko Kawashima</span>
	<sup>1</sup>, 

	<span  >
	Masakazu Satou</span>
	<sup>1</sup>, 

	<span  >
	Taiko Kim To</span>
	<sup>1</sup>, 

	<span  >
	Yukio Kurihara</span>
	<sup>1</sup>, 

	<span  >
	Eli Kaminuma</span>
	<sup>2</sup>, 

	<span  >
	Takaho Endo</span>
	<sup>2</sup>, 

	<span  >
	Yoshiki Mochizuki</span>
	<sup>2</sup>, 

	<span  >
	Norio Kobayashi</span>
	<sup>2</sup>, 

	<span  >
	Tetsuro Toyoda</span>
	<sup>2</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, 

	<sup>2</sup>RIKEN Genomic Sciences Center

</div>

<div>Plants respond and adapt to drought, cold and high&#45;salinity stresses in order to survive.Many stress&#45;regulated genes have been identified by the expression profiling studies using the DNA microarrays. However, we think that novel antisense RNAs, non&#45;coding RNAs, small RNAs and chromatin remodeling mechanisms have additional functions in the regulation of plant responses to the stresses.&#13;<br/>  Recently, whole&#45;genome tiling array and 454 sequencing technology have become powerful tools for the analysis of the whole transcriptome, including analyses of sense&#45;antisense transcripts, non&#45;coding RNAs, small RNAs and chromatin remodeling. Now we are also applying Arabidopsis Affymetrix tiling arrays and the 454 sequencing technology   to study the whole transcriptome under drought, cold, high&#45;salinity stress, and ABA treatment conditions. The tiling array experiments showed that many novel stress&#45;responsive transcriptional units and antisense RNAs exist in the &#34;Arabidopsis&#34; genome. Large&#45;scale analyses of the small RNAs by the 454 sequencing technology showed that the expression of several novel small RNAs are regulated by the stress. Functional analyses of the stress&#45;responsive functional RNAs are in progress.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular and Functional Characterization of VPS23&#45;3 &#40;ELCH homolog&#41; in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Channa Keshavaiah</span>
	<sup>1</sup>, 

	<span  >
	Mojgan Shahriari</span>
	<sup>1</sup>, 

	<span  >
	Aneta Sabovljevic</span>
	<sup>1</sup>, 

	<span  >
	Martin Huelskamp</span>
	<sup>1</sup>, 

	<span  >
	Swen Schellmann</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Cologne, Cologne, Germany

</div>

<div>Recently the <span style="font-style: italic">Arabidopsis</span> <span style="font-style: italic">ELCH</span> gene, a homolog of Vps23&#47; TSG101 and the key component of plant ESCRT&#45;I complex, has been functionally characterized &#40;Spitzer et al, Development, 2006&#41;. The <span style="font-style: italic">elch</span> mutant shows multiple nuclei in various cell types, indicating a role in cytokinesis. VPS28 and VPS37 are the other known components of ESCRT&#45;I. In addition to the ELCH homolog VPS23&#45;2 we have identified another, <span style="font-style: italic">Arabidopsis</span>&#45;specific homolog VPS23&#45;3 encoded by At2g38830. Interestingly VPS23&#45;3 showed a lesser degree of homology &#40;47&#37;&#41; to <span style="font-style: italic">Arabidopsis</span> ELCH compared to the homology of ELCH to <span style="font-style: italic">Oryza sativa</span> ELCH &#40;66&#37;&#41; and VPS23&#45;2 &#40;72&#37;&#41;. Similar to ELCH, VPS23&#45;3 is ubiquitously expressed, localized on endosomes and binds to ubiquitin with its N&#45;terminal UEV domain. However, VPS23&#45;3 interacts only with VPS28 whereas ELCH interacts with VPS37 in yeast&#45;two hybrid and bi&#45;molecular fluorescence complementation &#40;Bi&#45;FC&#41; assays. The <span style="font-style: italic">vps23&#45;3</span> mutant shows an embryonically lethal phenotype. This indicates that it is not simply redundant to ELCH. We are currently performing a genetic and functional characterization of this novel gene in the context of the ESCRT pathway in plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Divergent evolution of recent duplicate gene revealed by genetic incompatibility between Arabidopsis natural populations</div>

<div>

	<span  >
	David Bikard</span>
	<sup>1</sup>, 

	<span  >
	Dhaval Patel</span>
	<sup>2</sup>, 

	<span  >
	Claire Le Mett&#233;</span>
	<sup>1</sup>, 

	<span  >
	Christine Camilleri</span>
	<sup>1</sup>, 

	<span  >
	Malcolm Bennett</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Olivier Loudet</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>INRA, SGAP, Versailles, France, 

	<sup>2</sup>Univ. of Nottingham, UK

</div>

<div> While generating sets of Recombinant Inbred Lines &#40;RILs&#41;, many researchers have witnessed that genetically unlinked loci may not always segregate independently: it is actually quite frequent among Arabidopsis thaliana RIL sets that one homozygous allelic combination at two independent locus is rare or totally absent in the descendance of a specific cross &#40;which, by the way, explains an important part of the segregation distortion almost inevitably inherited in such material&#41;. We have fine&#45;mapped such a two&#45;locus interaction for which a combination of the Col&#45;0 allele at the bottom of chrom. 1 and the Cvi allele at the top of chrom. 5 causes arrested embryo development at the globular stage. In one intermediate heterozygous combination, the seed developed normally but primary root growth is significantly reduced. We have identified that the mechanism for such a genetic conflict results from the divergent evolution of a duplicate gene pair: the histidinol&#45;phosphate amino&#45;transferase gene is essential for histidine biosynthesis and is present in two almost&#45;identical copies in the reference genome of Col&#45;0. We show that Col&#45;0 and Cvi retain alternate efficient copies of the amino&#45;transferase: the chrom. 1 gene is not expressed in Col&#45;0 while it is in Cvi; the chrom. 5 gene is completely deleted in Cvi, while it is present and expressed in Col&#45;0. Quantitative complementation tests by crossing to different mutants clearly show how the different alleles at the two duplicate genes interact to qualitatively control embryo development and quantitatively limit primary root growth. We are now analysing the extent of natural variation at those duplicate genes to characterize further intra&#45;specific evolution of essential recent duplicate genes. Interestingly, the same genetic incompatibility is seen between the same loci when crossing Cvi to Ler. Although both genes are expressed in Ler, the chrom. 1 gene contains a premature stop codon. Our study has revealed that there is strong selective pressure to retain the function of only one copy of the histidinol&#45;phosphate amino&#45;transferase gene.</div>

</div>





<div class="abs">

<div style="font-weight:bold">From Chlamydomonas to Arabidopsis: a bZIP history</div>

<div>

	<span style="text-decoration: underline" >
	Luiz Gustavo Guedes Correa</span>
	<sup>1</sup>, 

	<span  >
	Renato Vicentini</span>
	<sup>2</sup>, 

	<span  >
	Diego Ria&#241;o-Pachon</span>
	<sup>1</sup>, 

	<span  >
	Bernd Mueller-Roeber</span>
	<sup>1,3</sup>,
        
        <span  >
	Michel Vincentz</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Universit&#228;t Potsdam, 

	<sup>2</sup>Universidade Estadual de Campinas, 

	<sup>3</sup>MPI Molecular Plant Physiology

</div>

<div>Growth and development of all organisms depend on proper regulation of gene expression. Differential gene expression is the result of the control of transcription initiation rates by transcription regulation factors. bZIP transcription factors were described in all eukaryotes. Their conserved domain is constituted by a basic amino acid rich region that binds to DNA bases, and a leucine zipper, responsible for protein dimerization. Genetic, molecular and biochemical analysis indicate that plant bZIP factors are important regulators of specific processes such as light sensing, sugar signaling, osmotic control and the control of the carbon&#47;nitrogen balance. Identification of Possible Groups of Orthologous genes &#40;PoGOs&#41; from different evolutionary lineages allows rationalizing functional studies. Within this context, the present work aims at a better understanding of the evolution and function of bZIP transcription factors in plants. We identified a non&#45;redundant set of 77 bZIP factors in <span style="font-style: italic">Arabidopsis thaliana</span>, 93 in <span style="font-style: italic">Oryza sativa</span>, 89 in <span style="font-style: italic">Populus trichocarpa</span>, 7 in <span style="font-style: italic">Chlamydomonas reinhardtii</span> and 8 in <span style="font-style: italic">Ostreococcus tauri</span> genomes, respectively. The phylogenetic analysis of <span style="font-style: italic">Oryza</span>, <span style="font-style: italic">Arabidopsis</span> and <span style="font-style: italic">Populus</span> sets of bZIP transcription factors allowed the identification of 13 groups of homologues genes. A more detailed analysis led to the identification of 36 PoGOs, that possibly correspond to 36 ancestral functions in angiosperms, two exclusive monocot PoGOs and a possible Group of Paralogous genes in <span style="font-style: italic">Arabidopsis</span>. Inclusion of Algae, Bryophyte and Gymnosperm bZIP sequences in the analysis shed new light on the origin of the different PoGOs. Based on the results presented here, we propose an updated bZIP classification and a model of their evolution. This approach is revealing new perspectives on bZIP functional studies.</div>

</div>





<div class="abs">

<div style="font-weight:bold">An RT&#45;PCR&#45;based knockout screening system for rapid identification of transcription factors controlling nutrient uptake in Arabidopsis</div>

<div>

	<span  >
	Akinori Suzuki</span>
	<sup>1</sup>, 

	<span  >
	Takashi Kuromori</span>
	<sup>1</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>1</sup>, 

	<span  >
	Kazuki Saito</span>
	<sup>1,2</sup>, 

	<span style="text-decoration: underline" >
	Hideki Takahashi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, Yokohama, Japan, 

	<sup>2</sup>Chiba University, Chiba, Japan

</div>

<div>Uptake of nutrients from the soil environment requires active ion transport systems in roots. During the past decade, numbers of transporters facilitating acquisition of major nutrients such as nitrate, ammonium, sulfate and phosphate have been identified from Arabidopsis. Among the isoforms of these functional nutrient uptake facilitators, high&#45;affinity&#45;type transporters in particular draw attention as they are regulated responding to nutrient stress and ideally localized in cell&#45;types such as epidermis and root hairs having direct contact to the soil environment. &#13;<br/>In general, uptake of nutrients correlates with the transcript levels of transporters in roots, emphasizing significance of their transcriptional regulation. However, only few transcription factors are known to participate in adjustment of this fundamental biological process. This perhaps originates from the difficulties to find clear visible phenotypes of plant nutrient responses or to measure observable changes of nutrient uptake activities in a large&#45;scale experiment of mutant screening. In this study, we focused on transcription factors and established an RT&#45;PCR&#45;based knockout screening system to isolate mutants defective in regulating nitrate and sulfate transporters in Arabidopsis roots. All available candidates of mutants containing transposon insertions in transcription factors were collected from RIKEN collection &#40;www.rarge.gsc.riken.jp&#41;. Using RNAs from this mutant collection, we performed quantitative real&#45;time RT&#45;PCR of NRT2;1 nitrate transporter and SULTR1;2 sulfate transporter. Up to the present, approximately 300 homozygous insertion lines were tested, allowing us to identify a knockout of MADS&#45;box transcription factor controlling the sucrose deficiency response of NRT2;1. In addition, we isolated a candidate of sulfur limitation response&#45;less mutant from the screening using SULTR1;2 as a profiler gene. This screening system may further be utilized as a versatile tool to isolate mutants of transcription factors by applying any of the transcriptionally regulated genes as profilers.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulatory network construction in Arabidopsis using genome&#45;wide gene expression QTLs</div>

<div>

	<span style="text-decoration: underline" >
	Joost J. B. Keurentjes</span>
	<sup>1,2</sup>, 

	<span  >
	Jingyuan Fu</span>
	<sup>3</sup>, 

	<span  >
	Ritsert C. Jansen</span>
	<sup>3</sup>, 

	<span  >
	Inez R. Terpstra</span>
	<sup>4</sup>, 

	<span  >
	Guido van den Ackerveken</span>
	<sup>4</sup>, 

	<span  >
	L. Basten Snoek</span>
	<sup>5</sup>, 

	<span  >
	Anton J. M. Peeters</span>
	<sup>5</sup>, 

	<span  >
	Dick Vreugdenhil</span>
	<sup>2</sup>, 

	<span  >
	Maarten Koornneef</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Laboratory of Genetics, Wageningen University, NL-6703 BD Wageningen, The Netherlands, 

	<sup>2</sup>Laboratory of Plant Physiology, Wageningen University, NL-6703 BD Wageningen,The Netherlands, 

	<sup>3</sup>Groningen Bioinformatics Centre, Groningen Biomolecular Sciences and Biotechnology Institute, University of Groningen, NL-9751 NN Haren,The Netherlands, 

	<sup>4</sup>Molecular Genetics Group, Department of Biology, Utrecht University, NL-3584 CH Utrecht, The Netherlands, 

	<sup>5</sup>Laboratory of Plant Ecophysiology, Institute of Environmental Biology, Utrecht University, NL-3584 CA Utrecht, The Netherlands

</div>

<div>In analogy to classical traits, quantitative genetic variation is often observed for transcript levels of genes. Jansen and Nap therefore introduced the concept of genetical genomics, where Quantitative Trait Locus &#40;QTL&#41; analysis is applied to levels of transcript abundance, identifying genomic loci controlling the observed variation in expression &#40;eQTLs&#41;. A logical next step would be the construction of genetic regulatory networks but the main reason holding back the identification of gene by gene regulation has been the lack of a reliable identification of candidate regulators. Consequently, the molecular dissection of quantitative trait regulation is still in its infancy and would greatly benefit from approaches reducing the number of candidate genes in a QTL support interval. We therefore developed a novel approach for the assignment of maximum likelihood regulators by combining QTL analysis of gene expression profiling and iterative Group Analysis &#40;iGA&#41; of functionally related genes. To apply the concept of genetical genomics to higher plants we analyzed genome&#45;wide gene expression variation in a large and well&#45;studied Recombinant Inbred Line &#40;RIL&#41; population of <span style="font-style: italic">Arabidopsis thaliana</span>. We show that for many genes the variation in transcript level can be explained by genetic factors. By integrating current knowledge of the genetics of a specific trait, we demonstrate the construction of genetic regulatory networks, which can serve to form hypotheses about as yet unknown regulatory steps. </div>

</div>





<div class="abs">

<div style="font-weight:bold">PlantAFAWE: Automatic Functional Annotation in a distributed Web Service Environment</div>

<div>

	<span style="text-decoration: underline" >
	Anika Joecker</span>
	<sup>1</sup>, 

	<span  >
	Moritz Schoen</span>
	<sup>1</sup>, 

	<span  >
	Heiko Schoof</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck institute for plant breeding research

</div>

<div>A huge amount of high throughput functional genomics data is &#40;becoming&#41; available. This provides a data basis for intelligent filters and an automatic functional annotation. Currently, several methods for automatic protein function prediction are in use, for example homology detection, structure prediction and comparison and phylogenetic tree prediction. However, each of these methods has limited prediction accuracy and the comparison of the results and decisions about significance of hits are tasks of the manual annotator.<br/>In the last years a few hybrid approaches were published, which use machine learning algorithms for automatic prediction of ontology terms. They all have an increased accuracy, but in all cases only one feature is assigned and furthermore the tools can only handle the functional prediction of data from one organism and this data is not always up to date. Another problem is that in some cases the automatic functional prediction is not able to give a significant result. In this case combination of individually insignificant data could give clues to the function of a protein. But at the moment there is no program able to display these data.<br/>These problems motivate to implement an automatic functional annotation system with an intuitive web interface. This will integrate multiple inputs and evaluate correlations also in available functional genomics data to improve prediction. All available analysis results can be graphically and tabularly displayed and are iteratively combined by a rule&#45;based system. Terms from well&#45;known nomenclatures for protein function description and a human readable description are assigned to each protein. The program runs analysis tools for function prediction through web services in a distributed system. This improves the scalability, accessibility, maintainability, efficiency and simplifies the process. Another advantage is that the used data is always up to date. This also means that, even in cases where the automatic machine learning algorithm does not output significant results, the tool will still be highly useful for manual annotation by summarizing in one user interface all available data on a given protein.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Comprehensive analysis of the Arabidopsis GATA factors reveals light&#45; and clock&#45;regulation and organ&#45;specific patterns of gene expression.</div>

<div>

	<span style="text-decoration: underline" >
	Iain Manfield</span>
	<sup>1</sup>, 

	<span  >
	Toby Joyce</span>
	<sup>1</sup>, 

	<span  >
	Paul Devlin</span>
	<sup>2</sup>, 

	<span  >
	Chih-hung Jen</span>
	<sup>1</sup>, 

	<span  >
	David Westhead</span>
	<sup>1</sup>, 

	<span  >
	Phil Gilmartin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Leeds, Leeds, U.K., 

	<sup>2</sup>School of Biological Sciences, Royal Holloway University of London, U.K.

</div>

<div>Comprehensive expression analysis of the 29 GATA genes in Arabidopsis across a range of organs has identified genes which are light up&#45;regulated, clock&#45;regulated and expressed only in green tissues and these genes are therefore candidates to play roles in light signalling. Other genes are light down&#45;regulated and with evening peaks of expression. Therefore, there is a complex pattern of up and down&#45;regulation by light and clock regulation of many GATA genes, representing diverse regulatory inputs. In contrast, some genes show organ&#45;specific expression, in roots or siliques, but with no evidence of light regulation. Our results also reveal a small group of genes with expression patterns similar to &#34;HAN&#34;, suggesting that there may be additional GATA genes with related roles.&#13;<br/>&#13;<br/>Correlation of expression between genes across many microarray experiments can suggest similar regulation; this analysis can be performed using the Arabidopsis Co&#45;expression Tool &#40;www.arabidopsis.leeds.ac.uk&#47;ACT&#41;. Co&#45;expression of certain GATA genes with well&#45;characterised &#8220;guide genes&#8221; is being used for function prediction and aiding design of experiments for analysis of mutants.&#13;<br/>&#13;<br/>Determination of gene structures and expression patterns for all GATA genes has provided information additional to comparison of coding sequences. Together with analysis of gene pairs using the ACT Scatter Plot tool, these results reveal subtle differences in expression for duplicated genes, suggesting redundancy and divergence of functions.&#13;<br/>&#13;<br/>Microarray analysis of two GATA mutants which did not show a visible defect has identified mis&#45;regulation of a number of salt&#45; and cold&#45;responsive genes. In addition, promoter:GUS reporter constructs for these genes show expression in guard cells and in vascular tissue. We will present results demonstrating the salt&#45;sensitivity of mutants and exploring the positions of GATA genes in abiotic stress&#45;response pathways.&#13;<br/>&#13;<br/>Manfield et al., 2007 Plant Physiology 143:941&#45;958&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis Co&#45;expression Tool &#40;ACT&#41;; Web&#45;based tools to identify co&#45;regulated sets of genes and predict gene function.</div>

<div>

	<span style="text-decoration: underline" >
	Iain Manfield</span>
	<sup>1</sup>, 

	<span  >
	Chris Needham</span>
	<sup>2</sup>, 

	<span  >
	James Bradford</span>
	<sup>3</sup>, 

	<span  >
	Chih-hung Jen</span>
	<sup>3</sup>, 

	<span  >
	John Pinney</span>
	<sup>3</sup>, 

	<span  >
	Ioannis Michalopoulos</span>
	<sup>3</sup>, 

	<span  >
	Andy Bulpitt</span>
	<sup>2</sup>, 

	<span  >
	David Westhead</span>
	<sup>3</sup>, 

	<span  >
	Phil Gilmartin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centre for Plant Sciences, University of Leeds, Leeds, UK, 

	<sup>2</sup>School of Computing, University of Leeds, Leeds, U.K., 

	<sup>3</sup>Institute of Molecular and Cellular Biology, University of Leeds, Leeds, U.K.

</div>

<div>Large amounts of microarray data from diverse experiments contain information beyond the original purpose of each experiment and analysis of this data en masse can identify genes regulated in the same manner. This analysis is provided by the Arabidopsis Co&#45;expression Tool &#40;www.arabidopsis.leeds.ac.uk&#47;ACT&#41; which ranks 22000 genes across a large microarray dataset according to how closely their expression follows a gene of interest. We will illustrate how ACT elucidates gene regulatory mechanisms with examples of genes encoding components of the circadian clock and by predicting functions for poorly&#45;characterised transcription factor genes.&#13;<br/>&#13;<br/>Corroboration of pre&#45;calculated co&#45;expression lists can be performed by calculating user&#45;defined lists using only the arrays or experiments deemed relevant. GO and word&#45;counting tools measuring the over&#45;representation of themes in the annotations of co&#45;expressed genes can suggest biological significance. Novel tools dissect co&#45;expression lists into groups of genes likely to be coherently regulated.&#13;<br/>&#45; CLIQUE FINDER identifies the genes consistently co&#45;expressed with each other affording objectivity to determination of cut&#45;offs, giving sets of genes for further characterisation &#40;e.g. promoter motif prediction&#41;.&#13;<br/>&#45; SCATTER PLOT displays the correlation r&#45;values for nearly 22 000 genes versus two query genes and can reveal clusters of outlying genes. Highlighting well&#45;characterised genes on these plots can be used to infer common properties for co&#45;expressed genes. &#13;<br/> &#13;<br/>Beyond co&#45;expression analysis, we are using Bayesian inference methods to predict genetic regulatory networks from microarray data, without specific perturbation experiments. This reproduces aspects of the regulation of well&#45;characterised genes and predicts positions of less well&#45;characterised genes within experimentally&#45;identified regulatory networks.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic Diversity and Population Structure of <span style="font-style: italic">Arabidopsis thaliana</span> in the U.S. and World&#45;wide Collections</div>

<div>

	<span style="text-decoration: underline" >
	Yan Li</span>
	<sup>1</sup>, 

	<span  >
	Luz Rivero</span>
	<sup>2</sup>, 

	<span  >
	Joy Bergelson</span>
	<sup>3</sup>, 

	<span  >
	Magnus Nordborg</span>
	<sup>4</sup>, 

	<span  >
	Diane Byers</span>
	<sup>5</sup>, 

	<span  >
	Eric Holub</span>
	<sup>6</sup>, 

	<span  >
	Randy Scholl</span>
	<sup>2</sup>, 

	<span  >
	Justin Borevitz</span>
	<sup>7,8</sup>

</div>

<div>

	<sup>1</sup>Department of Ecology and Evolution, University of Chicago, 

	<sup>2</sup>Arabidopsis Biological Resource Center, Ohio State University, 

	<sup>3</sup>Department of Ecology and Evolution, Committee on Genetics, University of Chicago, 

	<sup>4</sup>Molecular and Computational Biology, University of Southern California, 

	<sup>5</sup>Department of Biological Sciences, Illinois State University, 

	<sup>6</sup>Warwick-HRI, University of Warwick, United Kingdom, 

	<sup>7</sup>Department of Ecology and Evolution, 

	<sup>8</sup>Committee on Genetics, University of Chicago

</div>

<div>Natural variation in <span style="font-style: italic">Arabidopsis thaliana</span> can be exploited to identify new alleles of known genes, new quantitative trait loci &#40;QTLs&#41;, perform association mapping studies, and discover the ecological and evolutionary forces driving local adaptation and shaping the genetic diversity. We genotyped over 5000 lines for 149 common SNPs to determine the relatedness and population structure, including &#126;800 world&#45;wide samples from the Arabidopsis Biological Resource Center &#40;ABRC&#41;, &#126; 3500 lines from Europe, and over 1600 lines collected from different locations in the U.S. Midwest across different times of year. We observed different levels of genetic diversity and population structure in both global and regional samples. Certain common genotypes were identified across multiple collection locations. Clones with identical genotypes at 149 common SNPs have been identified and will be removed to reduce the redundancy in these world&#45;wide collections including stock center lines. Genetic distance between regions will be compared with each other and against the geographic distance to test for the presence of isolation&#45;by&#45;distance. The possible ancestral origins of the U.S. Midwest populations will be inferred via model&#45;based structure analysis with world&#45;wide collections. A HapMap panel of 384 strains with maximal genetic diversity and controlled population structure is being selected for high density genotyping with a custom 250kSNP&#47;tiling array &#40;AtSNPtile1&#41;, which will be used for whole genome association mapping. The high marker density, large population size, and species wide diversity will allow powerful LD mapping studies to reveal common variation important for adaptation. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Evolutionary Dynamics in Glucosinolate Biosynthesis</div>

<div>

	<span style="text-decoration: underline" >
	Juergen Kroymann</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Chemical Ecology

</div>

<div>Plants synthesize an immense number of secondary metabolites, so called because their significance for processes of basic growth and development is not immediately evident. More than 200,000 known secondary metabolites provide an increasingly exploited reservoir for the generation of pharmaceutically active agents, and many more await discovery. Classic hypotheses that seek to explain the generation of this vast metabolic diversity propose a stepwise and reciprocal process of adaptation and counteradaptation between plants and their natural enemies, moulded by mutual selection. <br/><br/>Glucosinolates are a major component in the m&#233;lange of secondary metabolites in cruciferous plants, and they contribute to resistance against herbivorous insects and other enemies. More than 120 glucosinolates are known which share a chemical core structure, but differ in their amino acid&#45;derived side chain. Methylthioalkylmalate synthases &#40;MAM&#41; encoded by a small gene family in Arabidopsis thaliana and related plants control an early step in glucosinolate biosynthesis and, therefore, are central to the diversification of glucosinolate metabolism. In the MAM gene cluster, gene duplication, neofunctionalization and positive selection provide the evolutionary basis for biochemical alteration. These processes occur repeatedly in the history of this gene family, indicating their fundamental importance for the generation of plant metabolic diversity both within and among species. However, different processes, including multiple gene deletion events, genetic interchange between tandemly arranged MAM gene family members, and balancing selection, explain how glucosinolate diversity is maintained within Arabidopsis thaliana.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">QTL analysis of developmental trade&#45;offs: plastochron and pathogen response in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Marco Todesco</span>
	<sup>1</sup>, 

	<span  >
	Sureshkumar Balasubramanian</span>
	<sup>1</sup>, 

	<span  >
	Sridevi Sureshkumar</span>
	<sup>1</sup>, 

	<span  >
	Chris Schwartz</span>
	<sup>2</sup>, 

	<span  >
	Petra Epple</span>
	<sup>3</sup>, 

	<span  >
	Jeff Dangl</span>
	<sup>3</sup>, 

	<span  >
	Norman Warthman</span>
	<sup>1</sup>, 

	<span  >
	Julin Maloof</span>
	<sup>4</sup>, 

	<span  >
	Justin Borevitz</span>
	<sup>4</sup>, 

	<span  >
	Joanne Chory</span>
	<sup>4,5</sup>, 

	<span  >
	Detlef Weigel</span>
	<sup>1,4</sup>

</div>

<div>

	<sup>1</sup>Max-Planck-Institute for Developmental Biology, Tuebingen, Germany, 

	<sup>2</sup>University of Wisconsin, Madison, WI, USA, 

	<sup>3</sup>University of North Carolina, Chapel Hill, NC, USA, 

	<sup>4</sup>Salk Institute for Biological Sciences, La Jolla, CA, USA, 

	<sup>5</sup>Howard Hughes Medical Institute, La Jolla, CA, USA

</div>

<div>Plants modulate their growth and development based on environmental cues. The shoot apical meristem &#40;SAM&#41; generates leaves in a particular spatial arrangement &#40;phyllotaxis&#41; at defined time intervals &#40;plastochron length&#41;. Environmental cues such as light, temperature and presence of pathogens may affect both traits although underlying mechanisms are currently unknown. Here we report co&#45;localisation of a QTL for plastochron length with a natural variant in a pathogen response gene, suggesting a built&#45;in trade&#45;off between immunity and growth rates. <br/>We screened 24 accessions of <span style="font-style: italic">Arabidopsis thaliana</span> in three different growth conditions to reveal extensive variation in plastochron length. Exploiting this natural variation, we performed QTL analysis in two different temperature conditions using a newly developed Recombinant Inbred Lines &#40;RIL&#41; population derived from Col&#45;0 &#40;Columbia&#41; and Est&#45;1 &#40;Estland&#41;. The QTL analysis revealed a major temperature&#45;sensitive QTL for plastochron lenght in Chromosome 4, and a co&#45;localising major QTL for leaf lesioning, a trait peculiar to Est&#45;1 plants. Using an Heterogenous Inbred Family &#40;HIF48&#41; we fine mapped the lesioning phenotype to a 12 kb region. Expression analysis suggests that the lesioning is associated with constitutive activation of pathogen response pathways; we are in the process of testing whether this response is functional. Sequence analysis of the genes present in the QTL interval reveals substantial natural variation not only between Est&#45;1 and Col&#45;0, but also among other wild strains. Transgenic analysis through complementation and amiRNA&#45;mediated silencing is currently underway to confirm the causality for the QTL. The apparent co&#45;localisation of a plastochron QTL and potential variation in pathogen response has important implications for adaptation to the environment.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Poly &#40;A&#41; &#45;dependent RNA degradation in plants</div>

<div>

	<span style="text-decoration: underline" >
	Sarah Holec</span>
	<sup>1</sup>, 

	<span  >
	Heike Lange</span>
	<sup>1</sup>, 

	<span  >
	Jean Canaday</span>
	<sup>1</sup>, 

	<span  >
	Dominique Gagliardi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>IBMP, Strasbourg, France

</div>

<div>Eukaryotic mRNAs require 3' polyadenylation for their stability and translation. In contrast, addition of short poly&#40;A&#41; tails mark bacterial transcripts for degradation. Polyadenylation also triggers RNA degradation in chloroplasts and in mitochondria of several organisms including plants. Indeed, we have shown that in Arabidopsis thaliana mitochondria, polyadenylated mRNAs and rRNAs are degraded by the 3'&#45;5' exoribonuclease mtPNPase &#40;PolyNucleotide Phosphorylase&#41;. In absence of mtPNPase, polyadenylated maturation by&#45;products of tRNA and rRNA synthesis, and aberrant transcripts generated from intergenic regions accumulate to sometimes extraordinary high levels. These results suggest that the mitochondrial transcriptome is mostly defined by posttranscriptional RNA surveillance involving polyadenylation and PNPase &#40;1&#41;.<br/>In the course of this study, we noticed that some nuclear non&#45;coding RNAs were also polyadenylated, suggesting that a degradative mechanism involving oligoadenylation may also occur in the nucleus of plants. In yeast, it was recently shown that polyadenylation could trigger degradation of nuclear non&#45;coding RNAs &#40;2&#45;4&#41;. This novel pathway of degradation involves the TRAMP polyadenylation complex and Rrp6p, a RNase associated with the exosome. The exosome is a multiprotein complex that participates in RNA maturation, degradation and quality control &#40;5&#41;. <br/>We identified three genes exhibiting significant homology to Rrp6p in the genome of Arabidopsis thaliana. We determined the subcellular localization of these Rrp6p&#45;like proteins using GFP fusion proteins and found that AtRRP6A and B are targeted to the nucleus whereas AtRRP6C is cytoplasmic. Only Atrrp6b mutant plants accumulate a maturation by&#45;product of nuclear rRNA synthesis suggesting different substrate specificity between AtRRP6B and A. Interestingly, this rRNA maturation by&#45;product accumulates as polyadenylated species. These preliminary results suggest that AtRRP6B is indeed involved in the degradation of polyadenylated transcripts in the plant nucleus.<br/>1. Holec S et al&#40;2006&#41; Mol. Cell. Biol<br/>2. Vanacova S et al&#40;2005&#41; Plos biology <br/>3. Wyers F et al&#40;2005&#41; Cell<br/>4. LaCava J et al&#40;2005&#41; Cell<br/>5. Houseley J et al&#40;2006&#41; Nature reviews</div>

</div>





<div class="abs">

<div style="font-weight:bold">&#34;RBR1&#34; and &#34;MSI1&#34; activate maternal expression of methylation&#45;dependent imprinted genes in &#34;Arabidopsis&#34;</div>

<div>

	<span style="text-decoration: underline" >
	Pauline E Jullien</span>
	<sup>1</sup>, 

	<span  >
	Assaf Mosquna</span>
	<sup>2</sup>, 

	<span  >
	Nir Ohad</span>
	<sup>2</sup>, 

	<span  >
	Fr&#233;d&#233;ric Berger</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Temasek Lifesciences Laboratory, Singapore, Singapore, 

	<sup>2</sup>Tel Aviv University, Tel Aviv, Israel

</div>

<div>Genomic imprinting involves the epigenetic inactivation of one allele of a gene and the activation of the other one depending on their parental origin, resulting in monoallelic expression of this gene. In &#34;Arabidopsis&#34;, &#34;FIS2&#34; and &#34;FWA&#34; genes are known to be expressed from their maternal allele only. Their paternal allele is silenced by the DNA methyltransferase MET1, which maintains methylation on CpG site through cell division. The activation of their maternal allele requires the action of &#34;DEMETER&#34; responsible for removal of DNA methylation on the maternal allele causing its expression. Here we show that in addition to &#34;DEMETER&#34;, &#34;RBR1&#34; &#40;&#34;RETINOBLASTOMA RELATED 1&#34;&#41; and &#34;MSI1&#34; &#40;&#34;MULTICOPY SUPPRESSOR OF IRA1&#34;&#41; are required for the maternal expression of FIS2 and FWA. However they are not required for the maternal expression of &#34;MEA&#34;, imprinted in a DNA methylation independent manner. MSI1 and RBR1 are known to interact together in other organisms and to play a role in regulation of cell cycle progression. This interaction is also conserved in &#34;Arabidopsis&#34;. We further show that &#34;MSI1&#34; is required to repress &#34;MET1&#34; expression. We propose that &#34;RBR1&#34; and &#34;MSI1&#34; activate &#34;FIS2&#34; and &#34;FWA&#34; maternal allele through repression of &#34;MET1&#34; expression therefore linking cell cycle regulation to DNA methylation and genomic imprinting.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Positional cloning of the TUMOROUS SHOOT DEVELOPMENT2 gene reveals a mutation in a putative methyltransferase that is necessary for cell adhesion and co&#45;ordinated plant development</div>

<div>

	<span style="text-decoration: underline" >
	Eva Krupkov&#225;</span>
	<sup>1</sup>, 

	<span  >
	Thomas Schm&#252;lling</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Biology/Applied Genetics, Free University of Berlin, Berlin, Germany

</div>

<div>Mutations in the TUMOROUS SHOOT DEVELOPMENT2 &#40;TSD2&#41; gene reduce cell adhesion and cause non&#45;coordinated shoot development and tumor&#45;like growth in vitro. tsd2 mutants exhibited increased activity of axial meristems, reduced root growth and enhanced de&#45;etiolation. Expression analysis of shoot meristem marker genes KNAT1:GUS and KNAT2:GUS indicated enlarged expression domains in the mutant background. Soil&#45;grown tsd2 mutants were dwarfed but showed an overall morphology similar to wild type. The TSD2 gene was identified by map&#45;based cloning. It encodes a novel 684 amino acid long polypeptide containing a single membrane&#45;spanning domain in the N&#45;terminal part, and S&#45;adenosyl&#45;L&#45;methionine binding and methyltransferase domains in the C&#45;terminal part. Expression of a TSD2:GUS reporter gene was detected mainly in meristems and young tissues. A GFP&#45;tagged TSD2 protein localized to the Golgi apparatus. The cell adhesion defects indicated altered pectin properties and we hypothesize that TSD2 acts as a pectin methyltransferase. </div>

</div>





<div class="abs">

<div style="font-weight:bold">PolIV, Central proteins in sRNA production</div>

<div>

	<span style="text-decoration: underline" >
	Rebecca A. Mosher</span>
	<sup>1</sup>, 

	<span  >
	David Studholme</span>
	<sup>1</sup>, 

	<span  >
	Frank Schwach</span>
	<sup>1</sup>, 

	<span  >
	David C. Baulcombe</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Sainsbury Laboratory, Norwich, UK

</div>

<div>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Most endogenous sRNAs in plant cells are derived through DNA&#45;dependent RNA polymerase IVa &#40;PolIVa&#41;, <span style="font-style: italic">RNA&#45;dependent RNA polymerase 2</span> &#40;<span style="font-style: italic">RDR2</span>&#41;, and <span style="font-style: italic">Dicer&#45;like 3</span> &#40;<span style="font-style: italic">DCL3</span>&#41;. At some loci, these small RNAs act via <span style="font-style: italic">Argonaute 4</span> &#40;<span style="font-style: italic">AGO4</span>&#41; and a second isoform of PolIV &#40;PolIVb&#41; to direct DNA methylation. Many targets of RNA&#45;directed DNA methylation &#40;RdDM&#41; are transposable elements or repetitive DNA, though developmental genes such as <span style="font-style: italic">SUPERMAN</span>, <span style="font-style: italic">FWA</span>, and <span style="font-style: italic">MEDEA</span> are also regulated by RdDM.<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In order to define and characterize endogenous sRNA&#45;generating loci, we sequenced sRNA libraries from wildtype and PolIV mutants &#40;<span style="font-style: italic">nrpd1a&#45;4</span> and <span style="font-style: italic">nrpd1b&#45;1</span>&#41;. 3838 <span style="font-style: italic">bona fide</span> sRNA loci &#40;containing &#8805;4 sRNA matches with &#8805;1 unique match&#41; were identified, and 94&#37; require PolIVa. PolIVa&#45;dependent loci are highly enriched for all classes of transposable elements and concentrate in the pericentromere. All tested loci requiring PolIVa also require <span style="font-style: italic">RDR2</span> and <span style="font-style: italic">DCL3</span>, but show varying degrees of dependence on PolIVb. Non&#45;LTR retrotransposons require both PolIV isoforms, while LTR&#45;retrotransposons only require PolIVa. The inherent ability of LTRs to sustain siRNA production indicates that PolIVb may play a role in maintaining siRNA production at loci without such repeats. RdDM of PolIVa&#45;dependent loci was assessed with bisulfite sequencing. In addition to the expected loss of asymmetric methylation, gain of methylation was detected in nrpd1a. Both changes required PolIVb, though sRNA accumulation at these loci does not. <br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;This work highlights the extensive and diverse role of PolIV. PolIVa&#45;dependent loci comprise more than 1&#37; of the Arabidopsis genome, including 550 unique regions. PolIVb likely functions at each of these loci, directing various DNA modifications and&#47;or maintaining sRNA production.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cosima &#8211; The Community Semantic Interrogation Machine</div>

<div>

	<span style="text-decoration: underline" >
	Andreas Groscurth</span>
	<sup>1</sup>, 

	<span  >
	Heiko Schoof</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max-Planck-Institute for Plant Breeding Research

</div>

<div>In the present internet one can find biological knowledge mostly stored in relational databases, accessible through web services and presented in a human readable from. The idea of the semantic web is to bring structure to the meaning of information and thereby to enable the integration of the huge amount of distributed biological data. <br/>The key feature of the semantic web are ontologies, which allow to capture existing knowledge in a systematic and standardized way and describe terms and their relationships. The usual way to create such ontologies is to hire highly trained experts. This approach faces the issues of  shortage of available person&#45;hours and the either not deep or not broad enough knowledge of the experts to cover the full scope of information. This makes ontology creation expensive and not flexible enough to keep track of the fast changing knowledge in biology. Also text mining approaches failed due to the impossibility to extract the full semantic meaning of publications without a preexisting ontology.<br/>One emerging approach is engaging a wider community. The Cosima application provides a mechanism for rapid and inexpensive knowledge mining by learning an ontology from the interaction with the community. It represents an open, parallel and mainly decentralized system, which builds ontologies from the knowledge of many, who do not need to be specially trained in knowledge or ontology engineering. The system provides an intuitive web front end based on so&#45;called &#8220;chatterbots&#8221; in which the biologist can easily interact with the system and contribute his or her knowledge into the ontology. <br/>In an open platform where anyone can contribute, a validation systems needs to check the consistency of the ontology. But also this can be addressed by a community approach. Every statement which the system has learned will be checked again by asking others to confirm it. In this way, disputed statements can be highlighted for expert review or contradicting statements removed from the ontology. </div>

</div>





<div class="abs">

<div style="font-weight:bold">QTL ANALYSES OF GROWTH RELATED TRAITS UNDER POTASSIUM &#38; PHOSPHATE STARVATION REGIMES &#13;<br/>Department of Plant Breeding and Genetics, Max Planck Institute for Plant Breeding</div>

<div>

	<span style="text-decoration: underline" >
	Hugues Barbier</span>
	<sup>1</sup>, 

	<span  >
	Maarten Koornneef</span>
	<sup>1</sup>, 

	<span  >
	Barbara Eilts</span>
	<sup>1</sup>, 

	<span  >
	Matthieu Reymond</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute

</div>

<div>Plants in natural environment experience abiotic starvation at each step of their development, which can have an important effect on overall plant performance and fitness &#40;growth and reproduction&#41;. The purpose of this study is to detect the chromosomal regions &#40;Quantitative Trait Loci &#45; QTL&#41; involved in the responses of plant performance under phosphate and potassium starvation regimes.&#13;<br/>Growth&#45;related traits &#40;seed size, germination, rosette expansion rate, shoot and root weight, root length&#41; were studied during vegetative phase. Natural variation of growth was analyzed using a set of 145 Recombinant Inbred Lines &#40;RIL&#41; of Arabidopsis thaliana derived from Landsberg erecta &#40;Ler, Poland&#41; and Kashmir &#40;Kas&#45;2, Kashmir&#41; ecotypes. This RIL population was grown in a hydroponic system in order to monitor the amount of nutrients available for the plants. 3 different regimes were used in this study: Control &#40;&#8220;C&#8221;&#41;, Phosphate starvation &#40;&#8220;&#45;P&#8221;: &#45;4.7 Fold&#41; and Potassium starvation &#40;&#8220;&#45;K&#8221;: &#45; 3.4 Fold&#41;.&#13;<br/>In order to detect QTLs, we used a mixed model developed under SAS software &#40;y&#61; Markers &#43; Blocks &#43; Traits &#43; Treatments &#43; Markers&#42;Treatments &#43; Markers&#42;Traits &#43; Markers&#42;Traits&#42;Treatments&#41;. This model takes into account the variance between replicates &#40;Blocks&#41; and allows us to analyze jointly the different traits and treatments. For each of the growth&#45;related traits, common QTL were detected whatever the level of nutrients. These QTL have been considered as constitutive QTLs for this trait. Another set of QTL were specific to phosphate and potassium starvation and have been considered as starvations QTLs. Finally, QTL were detected specifically to a particular nutrient level and been considered as adaptive QTLs. We found also some growth QTL common whatever the traits and the starvations and have been considered as Master QTLs. &#13;<br/>The results obtained describe the genetic architecture of growth in response to different nutrient levels. Molecular characterization of the major QTL described will be performed, allowing us to better understanding GxE interactions in plant performance.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">TBR and TBLs &#8211; a Novel Gene Family required for Secondary Cellulose Biosynthesis</div>

<div>

	<span style="text-decoration: underline" >
	Volker Bischoff</span>
	<sup>1</sup>, 

	<span  >
	Ana-Silvia Nita</span>
	<sup>1</sup>, 

	<span  >
	Wolf-R&#252;diger Scheible</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max-Planck-Institute of Molecular Plant Physiology, Potsdam, Germany

</div>

<div>The EMS&#45;induced trichome birefringence &#40;tbr&#41; mutant of Arabidopsis is deficient in secondary cellulose deposition in trichomes and to some extent in the vasculature, causing a lack of trichome birefringence under polarized light &#40;Potikha et al., 1995&#41;. Additional phenotypes of the mutant include &#40;i&#41; reduced trichome density, &#40;ii&#41; callose deposition in subsidiary cells at the base of trichomes and in the vasculature system, and &#40;iii&#41; a variable but general reduction in growth.  We identified TBR by map&#45;based cloning and found this gene encoding a protein of unknown function with the predicted topology of a type II membrane protein. TBR has several homologs in the Arabidopsis genome, and several of these TBR&#45;like &#40;TBL&#41; genes, show co&#45;expression with secondary cellulose synthases, e.g. CESA7.  Consistently, T&#45;DNA insertion and RNAi lines of TBL2 show reduced levels of cellulose in inflorescence stems, and GC cell wall sugar patterns typical for secondary cell wall mutants. We present a novel and non&#45;described family of genes, involved in secondary cell wall synthesis and will give direct evidence of these genes being situated near the plasma membrane, confirming their function in cellulose synthesis.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Sorting sequences into the MapMan ontology: Using an automated pipeline to facilitate genome wide comparisons</div>

<div>

	<span style="text-decoration: underline" >
	Bjoern Usadel</span>
	<sup>1</sup>, 

	<span  >
	Axel Nagel</span>
	<sup>2</sup>, 

	<span  >
	Yves Gibon</span>
	<sup>1</sup>, 

	<span  >
	Doerte Lodka</span>
	<sup>1</sup>, 

	<span  >
	Oliver Thimm</span>
	<sup>1</sup>, 

	<span  >
	Wolfgang Lein</span>
	<sup>1</sup>, 

	<span  >
	Mark Stitt</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max-Planck Institute of Molecular Plant Physiology, Golm, Germany, 

	<sup>2</sup>RZPD Deutsches Ressourcenzentrum f&#252;r Genomforschung, Berlin, Germany

</div>

<div>The recent advances in high&#45;throughput sequencing and genomic technologies have made it possible to sequence full genomes or to obtain near full genome&#45;scale EST clusters and compare them to the sequenced model plants Arabidopsis and rice. Furthermore, massively parallel sequencing of ESTs can be used to get an insight into the expression of genes within different tissues or under different conditions.<br/>However, often when dealing with such high throughput data, or with EST sequencing projects it is difficult to classify these ESTs for tasks such as enrichment analysis or genome wide comparison of metabolic pathways.<br/>Here, we present a pipeline for automatic classification of sequences using the MapMan ontology. The pipeline uses a combination of machine learning and expert based rules for classification into the more than 1000 different MapMan classes and sub&#45;classes. It is currently available as a web&#45;based tool, and will be equipped with a BioMOBY interface for integration into bioinformatics toolkits.<br/>The resulting classes can than be used for over representation analysis and&#47;or comparisons using Arabidopsis as a background model.</div>

</div>





<div class="abs">

<div style="font-weight:bold">TILLING strategy to identify change of function mutations in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Georg Strompen</span>
	<sup>1</sup>, 

	<span  >
	Nadja H&#228;drich</span>
	<sup>2</sup>, 

	<span  >
	John Lunn</span>
	<sup>2</sup>, 

	<span  >
	Thomas Altmann</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Potsdam, Institute of Biochemistry and Biology, Department of Genetics, Potsdam, Germany, 

	<sup>2</sup>Max-Planck-Institute of Molecular Plant Physiology, Golm, Germany

</div>

<div>The GABI&#45;TILL project Arabidopsis use the TILLING &#40;Targeting Induced Local Lesions IN Genomes&#41; technology for the analysis of gene functions. The utility of TILLING will be enhanced by technical improvements and exploration of novel applications first tested in the Arabidopsis reference species. The value of mutants identified by TILLING will be demonstrated by the identification of mutant alleles of lead genes in the crop species like sugar beet and barley. These alleles will allow researchers to determine the importance of the analysed genes for traits of agronomic and commercial importance.Supported by international co&#45;operation, we have implemented the TILLING technology and applies it now on a routine basis. An Arabidopsis EMS mutant collection &#40;40.500 M1 plants&#41; composed of approx. 7,500 M2 plants has been established, DNA has been extracted and pooled. A novel 2D pooling strategy was implemented that improved the mutant detection efficiency and reduced the number of necessary reactions. TILLING screens were performed for several genes of interest in &#8220;proof of concept studies&#8221; and hitherto 134 mutations were found in 11 genes. The134 mutations were detected and 80 mutations were checked and confirmed by sequencing. This includes SPS2 &#40;major isoform of sucrose phosphate synthase in leaves&#41; and APS1 &#40;catalytic subunit of ADP glucose pyrophosphorylase&#41; genes which encode key regulatory enzymes of starch and sucrose synthesis, respectively. The aim is to generate a set of novel alleles of APS1 and SPS2. We are particularly interested in new forms of the enzymes with altered kinetic and regulatory properties &#8211; substrate affinity, sensitivity to allosteric regulators or altered post&#45;translational regulation.&#13;<br/>To demonstrate the power of TILLING further &#34;proof of concept&#34; studies have been initiated. The mutant collections will be increased up to 10.000 M2 plants serving as the basis for a TILLING service carried out for interested users. For further information about the TILLING Service Arabidopsis visit our homepage: http:&#47;&#47;www.gabi&#45;till.de.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genes encoding defense signaling proteins in plants are more conserved than those encoding recognition proteins</div>

<div>

	<span style="text-decoration: underline" >
	Katherine Caldwell</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Davis

</div>

<div>The close interplay between pathogen effectors, their host targets, and the guard proteins that monitor these targets provides various opportunities for opposing cycles of selection acting on plant and pathogen to achieve or abrogate resistance, respectively.  Selection had previously been shown to be acting to maintain diversity in both plant proteins involved in pathogen recognition and some of the pathogen effectors they guard against.  Here we provide the first analysis of the signatures of selection at genes encoding defense signal transduction proteins in plants, which are putative targets of pathogens.  We show that in contrast to recognition genes, the majority of signaling components appear to be either evolving neutrally or under purifying selection.  This is consistent with these proteins not being targets of effectors and&#47;or indicative of functional constraints on their evolution.  However, there was significant evidence of selection maintaining diversity at the NPR1, PAD4, and EDS1 loci.  Differences in the signatures of selection observed may reflect the numbers of effectors that target a particular protein, presence or absence of a cognate guard protein, as well as interactions with other plant proteins. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Web services for Arabidopsis data integration</div>

<div>

	<span style="text-decoration: underline" >
	Chris Town</span>
	<sup>1</sup>, 

	<span  >
	Dirk Haase</span>
	<sup>2</sup>, 

	<span  >
	Hank Wu</span>
	<sup>1</sup>, 

	<span  >
	Heiko Schoof</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>The Institute for Genomic Research, Rockville MD USA, 

	<sup>2</sup>Max Planck Institute for Plant Breeding Research, Cologne, Germany

</div>

<div>The ever increasing amount and complexity of Arabidopsis genomic data presents a growing problem that should be of concern to both the user and the bioinformatic community. Exploiting the full potential of these large and diverse datasets is currently hindered by the limited mechanisms of availability and the lack of integration. Web services provide a means whereby data residing at many different locations can be seamlessly integrated to provide the user with richer data sets. Unlike web pages that are idiosyncratic in their layout and content, and must be visited by a researcher one at a time, web services &#40;in this case BioMOBY&#41; provide data in well structured and agreed formats, document their availability through a central registry and can be combined to provide richer views of the data. Under a demonstration project encouraged by the Multinational Arabidopsis Steering Committee &#40;MASC&#41;, our goal was to facilitate the deployment of web services from 16 data providers &#40;eight each in Germany and in the USA&#41; that vary both in the kinds of data that they host, and in the level of expertise at the sites. Training workshops for developers have been held both at the Max Planck Institute for Plant Breeding Research in Cologne and The Institute for Genomic Research in Rockville MD. These have both provided the attendees with the knowledge needed to deploy web services and have resulted in new services. In addition, the extensive &#8220;how to&#8221; documentation developed in conjunction with these workshops is freely available to the community &#40;http:&#47;&#47;bioinfo.mpiz&#45;koeln.mpg.de&#47;araws&#41;. Besides the new individual services, project bioinformaticians have developed &#8220;aggregator&#8221; web clients that can simultaneously invoke services from multiple locations in search of a particular data type &#40;e.g. literature or images&#41;. The growing number of web services can also be combined into custom workflows using clients such as Taverna. Some example applications and tools will be highlighted and mechanisms to encourage a much more widespread use of web services discussed.<br/>Work supported by the Deutsche Forschungsgemeinschaft and the National Science Foundation.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Completing the Expression Catalog of the Arabidopsis Transcriptome by Quantitative Real Time PCR</div>

<div>

	<span style="text-decoration: underline" >
	YONGLI XIAO</span>
	<sup>1</sup>, 

	<span  >
	Beverly Underwood</span>
	<sup>1</sup>, 

	<span  >
	Julia Redman</span>
	<sup>1</sup>, 

	<span  >
	Wei Wang</span>
	<sup>1</sup>, 

	<span  >
	Hui Quan</span>
	<sup>1</sup>, 

	<span  >
	Hank Wu</span>
	<sup>1</sup>, 

	<span  >
	William Moskal</span>
	<sup>1</sup>, 

	<span  >
	Erin Monaghan</span>
	<sup>1</sup>, 

	<span  >
	CHRIS TOWN</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Institute for Genomic Research, A Division of the J. Craig Venter Institute, 9712 Medical Center Drive, Rockville, MD 20850, USA

</div>

<div>Five sequential rounds of whole genome annotation of Arabidopsis at TIGR and one at TAIR produced a dataset that contained structural and functional annotation for 26,751 protein coding genes of which over 8,000 have both molecular function and biological process annotated as &#34;unknown&#34;. Potential roles for many of these genes can be inferred by examining the correlation of their expression patterns with known genes and pathways. Publicly available datasets from Affymetrix ATH1 expression array combined with massively parallel signature sequencing have provided statistically significant expression values over diverse tissues for just over 22,000 distinct protein&#45;coding genes. Because many of the genes of unknown function are expressed at levels that are too low to be effectively profiled by hybridization&#45;based methods, our current NSF 2010 project is to generate expression profiles by quantitative real time PCR for all of the 4,000&#43; Arabidopsis genes for which expression data are unavailable. To date, we have performed quantitative real time PCR on over 1,000 genes that either lack reliable expression data from or are not represented on the ATH1 array using cDNAs from leaf, root and T87 cell culture and seedlings treated with IAA, SA and salt. Over 90&#37; of the genes were expressed in one of our current cDNA populations and &#126;40&#37; of them showed differential expressions in at least 2 out of 6 conditions. In addition, we have developed a high throughput pipeline to generate promoter&#45;reporter constructs and transgenic Arabidopsis plants for 1,000 low&#45;expressing genes of unknown function. So far, promoters from 812 genes have been cloned into a GFP reporter construct, 184 have been transferred into Agrobacterium and 82 have been transformed into Arabidopsis. Most of the transgenic plants examined to date show GFP expression localized to small regions of tissues and cell types. Both the qPCR and reporter construct data can be found at http:&#47;&#47;www.tigr.org&#47;tdb&#47;e2k1&#47;ath1&#47;qpcr&#47;index.shtml</div>

</div>





<div class="abs">

<div style="font-weight:bold">Second Generation Arabidopsis Genomics &#13;<br/>with Second Generation Sequencing instruments&#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	William McCombie</span>
	<sup>1</sup>, 

	<span  >
	Robert Martienssen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Cold Spring Harbor Laboratory

</div>

<div>The sequence of Arabidopsis thaliana &#40;Columbia&#41; was completed in 2000.  A major breakthrough in DNA sequencing instrumentation has led to new instruments which are capable of generating significant coverage of the Arabidopsis genome in a period of as little as 3 days.  These instruments will have significant impact on the next generation of studies of the Arabidopsis genome and transcriptome.  We are working on a number of applications for these technologies in Arabidopsis.  One of these is adding to the sequence of the original Columbia accession.  While that sequence is among the highest quality sequences of a eukaryotic organism, there remain areas of the genome that are not complete.  We will attempt to fill in the gaps in the Columbia accession reference sequence.  In addition, the instruments have significant capabilities for sequencing other accessions of Arabidopsis and comparing them to Columbia.  We have done this with Landsburg erecta.  The resulting sequence, while interesting for showing the differences between the accessions is also very important because it generates a high density single nucleotide polymorphism map between them.  This demonstrates the ease at which such maps can be generated between any two accessions of Arabidopsis.  The resulting data should have a significant impact on polymorphism based mapping.  In addition, we have sequenced several Landsburg erecta lines with transposon insertions showing that sequencing the genome can be carried out to determine the site of transposon insertions.  Lastly, we present data showing that the next generation sequencers coupled with genome&#45;wide bisulphite treatment can be used to determine the methylation state of the genome under varying conditions.  The studies in aggregate show the significant impact of the new technologies on studies of the Arabidopsis genome, it&#8217;s evolution, correlation between structural and functional elements among different accessions of Arabidopsis and the impact of methylation on gene expression.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A high resolution root expression map predicts cellular function and reveals novel expression patterns</div>

<div>

	<span style="text-decoration: underline" >
	Siobhan Brady</span>
	<sup>1</sup>, 

	<span  >
	David Orlando</span>
	<sup>1</sup>, 

	<span  >
	Ji-Young Lee</span>
	<sup>1</sup>, 

	<span  >
	Jean Wang</span>
	<sup>1</sup>, 

	<span  >
	Daniel Mace</span>
	<sup>1</sup>, 

	<span  >
	Uwe Ohler</span>
	<sup>1</sup>, 

	<span  >
	Philip Benfey</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Duke University, Durham, NC, USA

</div>

<div>Transcriptional programs that regulate the development of multicellular organs are exquisitely controlled in both space and time.  Elucidating the regulatory networks that underlie development is essential to understand the acquisition of cell and tissue identity within an organ.  A previous root expression map profiled 5 radial tissues and 3 developmental stages.  We present a more detailed root expression map with profiles of nearly all cell types &#40;15&#41; in the root and 13 developmental stages.  <br/><br/>First, we identified transcriptional signatures of individual cell types and their associated biological processes. In many cases, these processes predict novel cellular function.  To deal with this magnitude of high&#45;resolution data, we then developed an algorithm that identifies dominant expression patterns and asked how expression defines root development.  51 groups of co&#45;regulated genes were identified that correlate with individual cell&#45;types, spatially related cell types, and intriguingly, to cell types which are spatially and ontologically unrelated.  Analysis of these groups suggests that there are transcriptional and biological connections between disparate cell types. The 40 dominant developmental patterns demonstrate that at the transcriptional level, transcriptional programs occur in a gradient along the longitudinal axis, and do not strictly correlate to previously defined meristematic, elongation and maturation zones.  Furthermore, many of these programs show expression oscillations along the root longitudinal axis.  We also examined the extent of between&#45;root variation and found that while some groups exhibit reproducible expression, others exhibit potential dynamic responses.  To combine these cell type and developmental expression profiles, a method was developed that proved accurate in identifying relative peaks of high expression in both space and time.  <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">There and back again: From Genomics to Proteomics in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Katja Baerenfaller</span>
	<sup>1</sup>, 

	<span  >
	Jonas Grossmann</span>
	<sup>1</sup>, 

	<span  >
	Franz Roos</span>
	<sup>1</sup>, 

	<span  >
	Wilhelm Gruissem</span>
	<sup>1</sup>, 

	<span  >
	Sacha Baginsky</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ETH Zuerich

</div>

<div>Mass spectrometry based proteomics has become widely accessible due to recent technical and conceptual advancements and the availability of gene and genome sequence databases. As a result, efforts of identifying and quantifying full proteomes are underway, which will complement the genomics approaches. Furthermore, the exploitation of the proteome data will improve the annotation of the genomes and their gene models. One such effort is the Arabidopsis thaliana Model Organism Proteomics &#40;AtMOP&#41; project. AtMOP forms a part of the Model Organism Proteomics &#40;MOP&#41; project, in which the first comprehensive maps of the proteomes of Drosophila, C. elegans and Arabidopsis are currently being generated.<br/>Up to date, we have identified more than 10&#8217;000 expressed proteins in Arabidopsis cell culture cells and differentiated tissues using standard database searches. In order to increase proteome coverage and to identify proteins, which are not represented in protein databases we developed a specialised analysis pipeline for high&#45;throughput mass spectrometry data &#40;reviewed in Baginsky and Gruissem, 2006&#41;. Subjecting our data to this pipeline has already lead to the identification of several alternatively spliced proteins, proteins containing SNPs or posttranslational modifications, or proteins that are not correctly predicted by current gene models.<br/>The data produced in AtMOP are integrated into SBEAMS and PeptideAtlas &#40;Desiere F, et al. 2005&#41;, in which all the identified and validated peptide sequences are mapped to the Arabidopsis genome and such allows the integration of protein data with genomic data. Through collaborations both with PPAP and TAIR, the data will be incorporated into protein and genome databases, and such disseminated to the scientific community.<br/><br/>References<br/>Baginsky S. and Gruissem W. &#40;2006&#41; Journal of Experimental Botany 57, 1485&#45;91.<br/>Desiere F, et al. &#40;2005&#41; Genome Biology 6, R9.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Transposon&#45;Induced Genome Rearrangements in Maize and Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Thomas Peterson</span>
	<sup>1</sup>, 

	<span  >
	Chuanhe Yu</span>
	<sup>1</sup>, 

	<span  >
	Lakshminarasimhan Krishnaswamy</span>
	<sup>1</sup>, 

	<span  >
	Jianbo Zhang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Iowa State University, Ames, Iowa, USA

</div>

<div>Since their discovery by McClintock, transposable elements have been associated with the generation of a variety of genome rearrangements, including deletions, direct and inverted duplications, and translocations.  In addition to providing dispersed sequence homologies for ectopic recombination, transposons can induce genome rearrangements through alternative transposition reactions that utilize the termini of different elements.  Transposition reactions involving transposon termini in direct orientation can generate deletions and inverted duplications.  In addition, pairs of Ac termini in reversed orientation can undergo transposition reactions resulting in inversions, deletions, and translocations.  In each of these cases, the rearrangement breakpoints are bounded by the characteristic footprint or target site duplications typical of Ac transposition reactions. These results show how alternative transposition reactions could contribute significantly to genome evolution by generating chromosome rearrangements, and by creating new genes through shuffling of coding and regulatory sequences &#40;Zhang, Zhang and Peterson, 2006&#41;.  These alternative transposition reactions were first observed in natural maize variants, and have now been reproduced in transgenic maize and Arabidopsis plants.  The system utilizes transgene constructs containing maize Ac termini in direct or reversed orientation.  The action of Ac transposase on the Ac termini generates a variety of rearrangements, including deletions, inversions and translocations.  In these rearrangements, one endpoint is at the Ac termini, and the other endpoint is at another genomic site.  This transposition&#45;based system provides an alternative to the cre&#45;lox system for genome modifications.  The current state of the project will be presented.  To view an animation of the alternative transposition model, see http:&#47;&#47;jzhang.public.iastate.edu&#47;Transposition.html.  <br/>This research is supported by NSF awards 0450243 to T. Peterson and J. Zhang, and 0450215 to D. Weber.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">PRX34</span> GENE IS INVOLVED IN LIGNIFICATION AT THE YOUNG STEMS OF ARABIDOPSIS</div>

<div>

	<span style="text-decoration: underline" >
	Mohammad MIR DERIKVAND</span>
	<sup>1</sup>, 

	<span  >
	Nathalie CAULET</span>
	<sup>1</sup>, 

	<span  >
	Brigitte POLLET</span>
	<sup>2</sup>, 

	<span  >
	Val&#233;rie MECHIN</span>
	<sup>2</sup>, 

	<span  >
	Dominique BUFFARD</span>
	<sup>1</sup>, 

	<span  >
	Catherine LAPIERRE</span>
	<sup>2</sup>, 

	<span  >
	Lise JOUANIN</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Biologie Cellulaire, INRA, Route de Saint Cyr, 78026 Versailles, France, 

	<sup>2</sup>Chimie Biologique, INRA-INA PG, 78850 Thiverval-Grignon, France

</div>

<div>The plant peroxidases have generally the various roles in the plants, but it is supposed that each particular gene has the specific activity&#40;ies&#41;. Dehydrogenative polymerization of lignin precursors &#40;monolignols&#41; is one of the proposed roles for plant peroxidases and laccases. Indeed, Lignins as the major phenolic components of the secondary cell wall are indispensable to plants.&#13;<br/>In this study, four peroxidases were identified among N&#45;glycosylated proteins isolated from Arabidopsis mature stems using affinity chromatography on Concanavalin A Sepharose and 2D electrophoresis. Two of the corresponding genes &#40;<span style="font-style: italic">PRX33</span>: At3g49110 and <span style="font-style: italic">PRX34</span>: At3g49120&#41; are located in tandem on chromosome 3. By RT&#45;PCR analysis, the <span style="font-style: italic">PRX34</span> gene was shown to be highly expressed in stems whereas PRX33 expression was low. &#13;<br/>Null and over&#45;expressing mutants of each gene were identified in the Versailles and SALK collections and characterized. The total peroxidase activity was reduced by 20&#37; in the knockout <span style="font-style: italic">prx34</span> mutant whereas no reduction was observed in the knockout <span style="font-style: italic">prx33</span> mutant. Chromatographic purification of peroxidase activity on CM&#45;sepharose led to two peroxidase&#45;containing peaks. One peak was greatly reduced in <span style="font-style: italic">prx34</span> and not in <span style="font-style: italic">prx33</span>. In addition, this peak was increased in a PRX34 overexpressing line and in the <span style="font-style: italic">prx34</span> line complemented with the <span style="font-style: italic">PRX34</span> cDNA under the control of the CaMV35S promoter.&#13;<br/>In the <span style="font-style: italic">prx34</span> null mutant, lignin deposition was reduced in fibers of young stems as shown by Wiesner staining. Klason and thioacidolyis analysis showed decreased lignin content in young stems of this mutant but not in mature stems. Purified PRX34 was able to oxidize coniferyl alcohol <span style="font-style: italic">in vitro</span> and produce oligomers. &#13;<br/>Taken together, these results suggest that PRX34 is involved in lignin polymerization whereas PRX33 is not involved in this process. However, PRX34 is only one of the oxidases responsible for the polymerization of monolignols, and other peroxidases need to be identified.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of interaction of ARIADNE E3 ligases with the unusual cap binding protein nCBP</div>

<div>

	<span style="text-decoration: underline" >
	Christina Mladek</span>
	<sup>1</sup>, 

	<span  >
	Suzanne Cathrin Burey</span>
	<sup>1</sup>, 

	<span  >
	Juan-Antonio Torres Acosta</span>
	<sup>1</sup>, 

	<span  >
	Marie-Theres Hauser</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>BOKU - University of Natural Resources and Applied Life Sciences Vienna

</div>

<div>ARIADNE &#40;ARI&#41; proteins belong to the class of &#8220;RING between RING fingers&#8221; &#40;RBR&#41; domain proteins with E3 ligase activity &#40;Eisenhaber et al., 2007&#41;. The Arabidopsis genome codes for 16 ARI genes that are subdivided into three subclasses. E3 ligase activity of AtARI8 &#40;subclass B&#41; has been shown to depend on the ubiquitin&#45;conjugating enzymes &#40;UBC&#41; 8, 10 and 11 &#40;Mladek et al. 2003&#41;. Here we present protein&#45;protein interaction results for AtARI representatives of each subclass showing that they are able to from homo&#45; and heteromers in yeast to hybrid assays. Whereas homomerization was only detected for AtARI12 &#40;subclass B&#41;, AtARI1 &#40;subclass A&#41; forms heteromers with both AtARI12 and AtARI15 &#40;subclass C&#41;. <br/>Furthermore, the novel RNA cap binding protein, nCBP, has been identified in yeast two hybrid assays as strong interactor of AtARI12 and AtARI15, but also weakly interacts with AtARI2 &#40;subclass A&#41;. The complex formation of AtARI12 with nCBP was further confirmed by coimmuno&#45;precipitation and &#45;localization. AtARI12 and nCBP can be detected in similar speckles in the cytoplasm and nucleus in Arabidopsis cell cultures. We will present the results of our analyses to test the hypothesis that nCBP is a candidate substrate of the AtARI E3 ligases and propose a model on the role of AtARIs as post&#45;translational regulators of nCBP during plant development and stress responses.  <br/><br/>Eisenhaber et al. &#40;2007&#41; Genome Biol 8, 209<br/>Mladek et al. &#40;2003&#41;. Plant Physiol 131, 27&#45;40<br/><br/>This project was supported by the Austrian Science Fund &#40;FWF&#41; project No. P17888 and the GENome Research in AUstria &#40;GEN&#45;AU&#41; project GZ 200.140&#47;1&#45;VI&#47;12006</div>

</div>





<div class="abs">

<div style="font-weight:bold">Global analysis of the Arabidopsis transcriptome data suggests methylation is a key factor that determines regulation of gene expression.</div>

<div>

	<span  >
	Felipe Aceituno</span>
	<sup>1</sup>, 

	<span  >
	Nick Moseyko</span>
	<sup>2</sup>, 

	<span  >
	Seung Rhee</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Rodrigo Gutierrez</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Departmento de Gen&#233;tica Molecular y Microbiolog&#237;a. P. Universidad Cat&#243;lica de Chile., 

	<sup>2</sup>Department of Plant Biology, Carnegie Institution of Washington, 260 Panama St, Stanford, CA 94305, USA.

</div>

<div>DNA microarray technology is the most widely used approach for monitoring genome&#45;wide gene expression changes. For Arabidopsis, there are &#62;1,800 microarray hybridizations that correspond to &#62;470 different experimental conditions in ATH1 Affymetrix gene chips alone. The large amount of public data accumulated, offers a unique opportunity to infer general principles that govern regulation of gene expression. &#13;<br/>The ATH1 hybridizations were normalized and filtered to eliminate low&#45;quality hybridizations. We then classified and compared control and treatment hybridizations. We determined differential gene expression between control and treatment hybridizations using the RankProducts method. As expected, the experimental conditions represented in our data set behave differently in terms of the nature and number of genes affected. Interestingly, organ was the variable that changes expression of the largest number of genes, on average ten&#45;fold more than any other experimental variable. Our data indicates the Arabidopsis transcriptome is robust to most perturbations and it is established during development.&#13;<br/>We defined &#8220;gene responsiveness&#8221; as the number of comparisons &#40;control vs treatment&#41; in which a gene changed its expression significantly. Genes with the highest and lowest responsiveness defined hypervariable and housekeeping genes. In an effort to identify basic principles that explain regulation of gene expression, we contrasted several structural features in these two groups of genes. Notably, transcript region methylation clearly distinguished housekeeping from hypervariable genes. Moreover, transcript region methylation inversely correlated &#40;R2&#61;0.8&#41; with gene responsiveness on a genome&#45;wide scale &#40;based on two available methylome datasets &#91;Zhang et al 2006; Zilberman et al 2007&#93;&#41;. This data suggests that methylation in the transcribed region of an Arabidopsis gene is a key determinant of the capacity of this gene to respond to internal or external cues.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">VirtualPlant: A Software Platform to Support Sysytems Biology Research in the Post&#45;Genomic Era</div>

<div>

	<span style="text-decoration: underline" >
	Manpreet Katari</span>
	<sup>1</sup>, 

	<span  >
	Steve Nowicki</span>
	<sup>1</sup>, 

	<span  >
	Chris Poultney</span>
	<sup>2</sup>, 

	<span  >
	Varuni Pravhakar</span>
	<sup>1</sup>, 

	<span  >
	Theresa Colombo</span>
	<sup>1</sup>, 

	<span  >
	Dennis Shasha</span>
	<sup>2</sup>, 

	<span  >
	Gloria Coruzzi</span>
	<sup>1</sup>, 

	<span  >
	Rodrigo Guti&#233;rrez</span>
	<sup>1,3</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, New York University, New York, USA, 

	<sup>2</sup>Courant Institute of Mathematical Sciences, New York University, New York, USA, 

	<sup>3</sup>Department of Molecular Genetics &amp;  Microbiology. P. Universidad Cat&#243;lica de Chile, Santiago, Chile

</div>

<div>Our long term goal is to understand how internal and external perturbations affect processes and networks controlling plant growth and development. In this project, we start with data integration of the known relationships among genes, proteins and molecules &#40;extracted from public databases and&#47;or generated with predictive algorithms&#41; as well as experimental measurements under many different treatments. We go beyond data integration to conceptual integration by using novel visualization techniques to render the multivariate information in visual formats that facilitate extraction of biological concepts. We also use mathematical and statistical methods to help summarize the data. We implement and combine these approaches in a system we term &#34;VirtualPlant&#34;. Whereas our project relates specifically to Arabidopsis, the data structures, algorithms, and visualization tools are designed in a species&#45;independent way. Thus the informatic, math, statistic and visualization tools that we develop can be used to model the cellular and physiological responses of any organism for which genomic data is available.<br/><br/>We have implemented a proto&#45;type that is already being actively and effectively used &#40;http:&#47;&#47;www.virtualplant.org&#41;.  This tool is being used by biologists and computer scientist alike for the purpose it was designed for &#45; to support the analysis of original genomic data generated by the researchers themselves. We have found that working with experimental biologists, even from very early stages of software development, to be the most effective way to generate real solutions to the problems encountered by researchers in the laboratory.<br/><br/>http:&#47;&#47;www.virtualplant.org&#47;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The GABI&#45;Kat Arabidopsis thaliana T&#45;DNA Mutagenised Population for Flanking Sequence Tag&#45;Based Reverse Genetics</div>

<div>

	<span style="text-decoration: underline" >
	Yong Li</span>
	<sup>1</sup>, 

	<span  >
	Mario G. Rosso</span>
	<sup>1</sup>, 

	<span  >
	Prisca Viehoever</span>
	<sup>1</sup>, 

	<span  >
	Heinz Saedler</span>
	<sup>2</sup>, 

	<span  >
	Bernd Weisshaar</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Bielefeld University, Institute for Genome Research, Bielefeld, Germany, 

	<sup>2</sup>Max Planck Institute for Plant Breeding Research, Koeln, Germany

</div>

<div>The GABI&#45;Kat population of T&#45;DNA mutagenised Arabidopsis thaliana lines with sequence&#45;characterised insertion sites is used extensively for efficient progress in plant functional genomics. Here we provide details about the establishment of the material, discuss results from quality control studies, and inform about the progress of donation of confirmed lines to NASC as well as the projected end of GABI&#45;Kat services in 2008.<br/>T&#45;DNA insertion mutants were created in the accession Columbia &#40;Col&#45;0&#41;. DNA from leaves of T1 plants was extracted and used for the production of flanking sequence tags &#40;FSTs&#41;. About 109,700 FSTs that &#34;hit&#34; regions on the A.th pseudochromosomes were submitted to EMBL&#47;GenBank&#47;DDBJ and are available from the GABI&#45;Kat website &#40;http:&#47;&#47;www.gabi&#45;kat.de&#41;. This website also gives access to information about which insertion is present in which line, and about the predicted location of insertions relative to a given gene. The GABI&#45;Kat database has recently been updated to contain additional information such as segregation data, gene&#45;specific primers and confirmation sequences. This information not only helps users to evaluate the usefulness of the mutant lines, but also covers a big part of the molecular characterisation of the insertion alleles. In addition to directly serving the community with confirmed insertion mutants, we have started in 2005 to transfer confirmed GABI&#45;Kat lines to the Nottingham Arabidopsis Stock Centre &#40;NASC&#41;. As of February 2007, 4,293 confirmed lines represented by sets of T3 seed have been delivered to NASC. We will continue to transfer confirmed lines to NASC until the projected end of the project in December 2008. The last user order will be accepted in June 2008 to be able to deliver within the duration of the project. In parallel to user orders, we will confirm the &#34;most valuable GABI&#45;Kat insertions&#34;, i.e. insertions that are unique to GABI&#45;Kat and those in which we have an CDSi&#45;hit while other populations have alleles with a much lower chance of causing a null mutation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Comparative Proteomics of Circadian and Flowering Proteins Between Arabidopsis and Rice</div>

<div>

	<span style="text-decoration: underline" >
	Duangdao Wichadakul</span>
	<sup>1</sup>, 

	<span  >
	Prasit Palittapongarnpim</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>BIOTEC Central Research Unit, National Center for Genetic Engineering and Biotechnology,  Pathumthani, Thailand, 

	<sup>2</sup>Department of Microbiology, Faculty of Science, Mahidol University, Bangkok, Thailand

</div>

<div>Comparative genomics and proteomics reveal the conservation and diversification between genes and proteins among organisms. They shed light on their evolution and diversity. With the available Arabidopsis and rice proteomes, we deploy comparative proteomics to investigate the conservation and differentiation of circadian and flowering proteins between Arabidopsis and rice, the models organisms for the long day and short day plants. Steps in comparative proteomics includes: &#40;1&#41; compile all protein sequences of Arabidopsis and rice from TAIR1 and TIGR Rice Genome Annotation Database2 &#91;1&#93;, &#40;2&#41; compile Arabidopsis gene ontology &#91;2&#93; from TAIR and GO terms and IDs from Gene Ontology3&#91;3&#93;, &#40;3&#41; extract GO IDs and terms that are related to circadian rhythms, flowering &#40;including flowering pathways such as vernalization&#41;, photoperiodism, and some other biological processes such as phosphorylation and response to cold, &#40;4&#41; extract all protein sequences in Arabidopsis that correspond to these GO IDs and terms, &#40;5&#41; PSI&#45;BLAST &#91;4&#93; the extracted protein sequences to all proteins in rice. We analyzed and categorized the hit results based on the lowest e&#45;values with the cutoff lower than 1e&#45;40, the conserved domains and domain organizations using InterPro &#91;5&#93; and IntroProScan &#91;6&#93;, and Gene Ontology. Our results confirm most results of comparative genomics studies in &#91;7, 8&#93; with several additional findings including the ortholog of Arabidopsis FRIGIDA protein in rice. Interestingly, InterProScan finds no hits domains for GI, ELF3, and EMF1. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">ChIP on chip&#45;based identification of target genes for key transcription factors involved in seed development of Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Urs Haehnel</span>
	<sup>1</sup>, 

	<span  >
	Gudrun Moenke</span>
	<sup>1</sup>, 

	<span  >
	Astrid Vorwieger</span>
	<sup>1</sup>, 

	<span  >
	Michaela Mohr</span>
	<sup>1</sup>, 

	<span  >
	Prisca Viehoever</span>
	<sup>2</sup>, 

	<span  >
	Linh My Tran</span>
	<sup>1</sup>, 

	<span  >
	Jens Tiedemann</span>
	<sup>1</sup>, 

	<span  >
	Annegret Tewes</span>
	<sup>1</sup>, 

	<span  >
	Andreas Czihal</span>
	<sup>1</sup>, 

	<span  >
	Bernd Weisshaar</span>
	<sup>2</sup>, 

	<span  >
	Ivo Grosse</span>
	<sup>1</sup>, 

	<span  >
	Helmut Baeumlein</span>
	<sup>1</sup>, 

	<span  >
	Udo Conrad</span>
	<sup>1</sup>, 

	<span  >
	Lothar Altschmied</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Leibniz-Institute of Plant Genetics and Crop Plant Research (IPK), Gatersleben, Germany, 

	<sup>2</sup>Institute of Genome Research, University Bielefeld, Bielefeld, Germany

</div>

<div>Seed formation is of utmost importance for human nutrition and animal feed and will provide the basic compounds for a bio&#45;based economy and CO2&#45;neutral energy production in the future. The development of seeds is a complex process including the storage of reserve compounds, the acquisition of desiccation tolerance, and the development of dormancy. Several transcription factors &#40;TF&#41;, namely LEC1, LEC2, FUS3, ABI3, MYB44, MYB77, ET1, and ET2, have been identified as key regulators in seed development. The underlying regulatory network is only partially understood, because many of the primary target genes for these TF are still unknown. Using chromatin immuno&#45;precipitation &#40;ChIP&#41; coupled with the hybridization to promoter arrays &#40;chip&#41; we aim to identify TF target genes in vivo at a genome wide scale.<br/><br/>Towards that goal transgenic Arabidopsis lines with estradiol&#45; &#40;Zuo et al. 2000, Plant J 24:265&#45;273&#41;, resp. dexamethasone&#45;inducible &#40;Aoyama and Chua 1997, Plant J 11:605&#45;612&#41; expression have been created for most of the TF mentioned above and antisera were raised against TF domains expressed in E. coli. Protocols for chromatin immuno&#45;precipitation were adapted to developing seeds of Arabidopsis and an array containing &#62;10.000 promoter fragments of Arabidopsis was constructed in cooperation with the SAP project &#40;www.psb.ugent.be&#47;SAP&#41;. Novel motif&#45;finding algorithms based on variable order Markov models and variable order Bayesian trees were developed &#40;Grau et al. 2006, Nucl. Acids Res. 34:W529&#45;533&#41; to identify potential TF binding sites in non&#45;aligned target promoters. Initial results for ChIP on chip experiments with ABI3 and LEC1 will be discussed.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Formation of apomeiotic embryosacs in the &#34;dyad&#34; mutant of Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Maruthachalam Ravi</span>
	<sup>1</sup>, 

	<span  >
	Mohan Prem Anand Marimuthu</span>
	<sup>1</sup>, 

	<span  >
	Imran Siddiqi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centre for Cellular and Molecular Biology(CCMB), Hyderabad, Andhra Pradesh, India 500 007

</div>

<div>The &#34;dyad&#34; mutant was previously isolated in an EMS mutagenesis screen for genes that affect reproductive development in &#34;Arabidopsis&#34; &#40;Siddiqi et al., 2000&#41;. DYAD is required for reductional chromosomal segregation and for female meiotic progression &#40;Agashe et al., 2002&#41;. The megaspore mother cell in &#34;dyad&#47;swi1&#34; undergoes an equational meiosis I and arrests at a two celled stage giving rise to two diploid cells leading to female sterility.  However, a variable low frequency of seed set was observed in &#34;dyad&#34; plants. Chromosome analysis of those progenies revealed that majority of them were triploid. Through various genetic approaches we show that the triploids arise by the fusion of a diploid female gamete with a haploid male gamete.  SSR marker analysis indicates that the &#34;dyad&#34; diploid female gametes lack recombination during their formation.  Mounting evidence suggests that apomixis involves temporal and spatial deregulation of genes involved in the sexual pathway. Our results provide a proof of principle demonstration of the formation of unreduced embryo sacs resembling apomeiosis due to a mutation of a single gene involved in the sexual pathway. We also provide evidence for sexual polyploidization mediated by &#34;dyad&#34; diploid gametes and obtained plants upto the heptaploid level. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Analysis of gene expression modified by compatible virus infection: A comparative study between <span style="font-style: italic">Arabidopsis thaliana</span> and <span style="font-style: italic">Vitis vinifera</span></div>

<div>

	<span style="text-decoration: underline" >
	Andrea Vega</span>
	<sup>1</sup>, 

	<span  >
	Carmen Espinoza</span>
	<sup>1</sup>, 

	<span  >
	Consuelo Medina</span>
	<sup>1</sup>, 

	<span  >
	Patricio Arce-Johnson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Departamento de Gen&#233;tica Molecular y Microbiolog&#237;a. Facultad de Ciencias Biol&#243;gicas. Pontificia Universidad Cat&#243;lica de Chile. Santiago de Chile, Chile

</div>

<div>In viral compatible interactions, pathogens spread through all plant tissues causing disease without resistance response associated. However, susceptible hosts are not passive against viruses and they can set up a defense response. Systemic viral infection causes several changes in plant gene expression, underlying the symptom development and also controlling virus levels. Molecular, cellular and physiological changes induced by viruses are part of the host defense response, but are not enough to stop viral replication and dissemination. With the aim of study the changes in gene expression trigger by compatible virus infection in diverse host plants, we compared the transcript profiles of two different models: <span style="font-style: italic">Arabidopsis thaliana</span> infected with TMV&#45;Cg and <span style="font-style: italic">Vitis vinifera</span> infected with GLRaV&#45;3. Changes in gene expression were evaluated using cDNA microarrays slides from the Arabidopsis Functional Genomics Consortium &#40;AFGC&#41;. Complementary, transcript profiling of virus&#45;infected grapevine was studied with the <span style="font-style: italic">Vitis vinifera</span> GeneChip&#174; from Affymetrix, validating the use of a heterologous microarrays system. We describe the identification of genes that are differentially expressed between infected plants and their corresponding healthy state in both models. Analysis of these genes permits to visualize the metabolic and cellular changes occurring during viral infection in <span style="font-style: italic">Arabidopsis</span> and <span style="font-style: italic">Vitis vinifera</span>, comparing the main processes affected. The most relevant changes in gene expression are associated with various biological functions including processes of translation, proteolysis, as well as modifications, targeting and folding of proteins. On the other hand, genes associated with leaf senescence and endomembrane system are significantly affected during infection in both models. Thus, the main biological and molecular processes trigger by viral compatible disease could be similar in different host plants.<br/>Acknowledgements: Fondecyt 1040789, Innova 05CTE01&#45;03, Conicyt</div>

</div>





<div class="abs">

<div style="font-weight:bold">Data&#45;mining from 600 oligomer arrays</div>

<div>

	<span style="text-decoration: underline" >
	Atle M. Bones</span>
	<sup>1</sup>, 

	<span  >
	Tommy J&#248;rstad</span>
	<sup>1</sup>, 

	<span  >
	Per Winge</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Norwegian university of Science and Technology

</div>

<div>We have performed some 600 microarray experiments using printed oligomer arrays covering probes for 25000 Arabidopsis genes. Samples from various mutants, treatments, conditions and developmental stages have been used. This set of data has been used for characterization of the transcriptome of Arabidopsis. In this presentation we will present results which we think will have general interest for the large number of scientists presently performing microarray studies. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Predicting functional modules root hairs that drive tip growth</div>

<div>

	<span style="text-decoration: underline" >
	Gordon Breen</span>
	<sup>1</sup>, 

	<span  >
	Claire Grierson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Bristol

</div>

<div>Root hairs are an ideal cell type to investigate polarized cell growth in eukaryotes. Unidimensional plant cell development requires the coordination of an array of cellular processes that need to work in unison to create a straight, precisely dimensioned cellular outgrowth to facilitate a variety of functions. By using a range of microarray datasets and applying Bayesian statistics it has been possible to confidently identify over 150 Arabidopsis thaliana genes involved in root hair morphogenesis. These include cell wall modifiers, G protein inhibitors, activators and effectors, cytoskeleton assemblers, vesicle transporters and modifiers, and many genes yet to be analysed. A network analysis using the coexpression profile of these genes has identified two highly linked modules connected by two separate pathways. Both modules contain genes with different associated functions suggesting each module &#40;or molecular machine&#41; has a separate role in tip growth. Integrating protein interaction data and phenotype mapping into this network will enable a detailed look at how these genes could interact to bring about tip growth.</div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="text-decoration: underline">AGRIS and ReIN</span>: A systems biology framework to establish and link <span style="font-style: italic">cis</span>&#45;regulatory networks and to identify direct targets for transcription factors in <span style="font-style: italic">Arabidopsis thaliana</span> using service oriented architecture.</div>

<div>

	<span style="text-decoration: underline" >
	Saranyan K. Palaniswamy</span>
	<sup>1</sup>, 

	<span  >
	Rebecca S. Lamb</span>
	<sup>1</sup>, 

	<span  >
	Ramana V. Davuluri</span>
	<sup>1</sup>, 

	<span  >
	Erich Grotewold</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Ohio State University, Ohio, USA

</div>

<div>Genome&#45;wide gene regulatory networks govern the phenotypic states of different cell&#45;types, tissues and developmental stages in eukaryotic organisms. The gene regulatory networks converge at the level of transcription, where the DNA&#45;binding transcription factors recognize <span style="font-style: italic">cis</span>&#45;regulatory elements in the promoter regions of target genes. This fact underscores that it is vital to establish the architecture of plant promoters to understand gene expression. The identification of direct target genes for selected TFs is serving as a platform in establishing genome&#45;wide regulatory networks to explain the expression of all Arabidopsis genes, simultaneously validating an Arabidopsis <span style="font-style: italic">cis</span>&#45;regulatory element map developed in parallel. In order to integrate this information into a web&#45;based knowledgebase, we have developed AGRIS &#40;Arabidopsis Gene Regulatory Information Server&#41;, which provides the scientific community with a platform to establish regulatory networks. AGRIS currently houses three linked databases: AtcisDB,AtTFDB and AtRegNet. AtTFDB contains more than 1,750&#43; Arabidopsis TFs and their sequences &#40;protein and DNA&#41; grouped into 50&#43; families with information on available mutants for the corresponding genes. AtcisDB consists of 25,000&#43; promoter sequences of annotated Arabidopsis genes with a description of putative cis&#45;regulatory elements, both predicted and experimentally validated. AtRegNet links in direct interactions between several hundred genes and the TFs that control their expression. The current release of AtRegNet contains a total of 350&#43; direct TF&#47;target gene interactions. ReIN is a web&#45;based software tool for creating, visualizing and identifying regulatory networks with a multi&#45;tier architecture based on J2EE technology. AGRIS has also been implemented in Service Oriented Architecture &#40;SOA&#41;. SOA provides a standard way of publishing applications and data sources over the internet, enabling mass dissemination of knowledge. AGRIS can be accessed at <span style="text-decoration: underline">http:&#47;&#47;arabidopsis.med.ohio&#45;state.edu.</span></div>

</div>





<div class="abs">

<div style="font-weight:bold">A novel gene, &#40;&#34;SEN10&#34;&#41;, regulates senescence in Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Hamad Siddiqui</span>
	<sup>1</sup>, 

	<span  >
	Salma Balazadeh</span>
	<sup>2</sup>, 

	<span  >
	Barbara K&#246;hler</span>
	<sup>1</sup>, 

	<span  >
	Bernd Mueller-Roeber</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>University of Potsdam, Potsdam, Germany, 

	<sup>2</sup>MPI of Molecular Plant Physiology, Potsdam, Germany, 

	<sup>3</sup>University of Potsdam &amp;  MPI of Molecular Plant Physiology, Potsdam, Germany

</div>

<div>Leaf senescence is a unique developmental process which involves a transition from a functional photosynthetic organ to an actively degenerating and nutrient&#45;recycling tissue. Senescence in plants is a highly regulated process and depends on the expression of many genes. To identify transcription factors &#40;TFs&#41; induced during natural leaf senescence in Arabidopsis thaliana we performed quantitative real&#45;time PCR &#40;qRT&#45;PCR&#41;, allowing to analyse the transcript abundance of more than 1600 TF genes. Several TF genes were found to exhibit enhanced or reduced expression, respectively, in senescent Arabidopsis leaves. One of the TF genes selected for further studies was &#40;&#34;SEN10&#34;&#41;. The SEN10 protein was observed to be localised in the cell nucleus when fused to green fluorescence protein &#40;GFP&#41; reporter in a transient expression assay in Arabidopsis thaliana protoplast. Using qRT&#45;PCR, &#40;&#34;SEN10&#34;&#41; was found to be strongly expressed in fully expanded and 20&#37; senescent leaves, whereas expression was low to moderate in juvenile leaves. Promoter&#45;reporter &#40;GUS&#41; studies confirmed senescence&#45;dependent expression of the &#40;&#34;SEN10&#34;&#41; gene. Constitutive over&#45;expression of &#40;&#34;SEN10&#34;&#41; under the control of the Cauliflower Mosaic Virus 35S promoter caused precocious senescence while RNAi lines exhibited a significant delay in leaf senescence. &#40;&#34;SEN10&#34;&#41; transcript abundance was not significantly altered in WRKY53 knock&#45;out lines, indicating that it functions independent of the WRKY53&#45;mediated senescence signalling pathway. Promoter&#45;GUS studies revealed a strong induction of &#40;&#34;SEN10&#34;&#41; upon wounding of rosette leaves. Wound&#45;induced &#40;&#34;SEN10&#34;&#41; expression was confirmed by qRT&#45;PCR, showing that expression was highest two hours after applying the wound stimulus. Recent experimental data will be presented.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis QTLs affecting RNA editing in mitochondria</div>

<div>

	<span  >
	Stephane Bentolila</span>
	<sup>1</sup>, 

	<span  >
	Leah Elliott</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Maureen Hanson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Cornell University, Ithaca, NY, USA

</div>

<div>Transcripts of both the Arabidopsis chloroplast and mitochondrial genomes undergo C&#45;to&#45;U editing. Editing usually results in changes in the encoded amino acid and can also create start and stop codons. Editing is likely to be essential for transcripts to encode functional mitochondrial proteins. Editing of mitochondrial Cs occurs at an astonishing level: more than 1 in every 15 mitochondrial Cs are edited in Arabidopsis, resulting in about 450 C targets of editing. How can so many different targets of editing be recognized by the editing apparatus, which must remain selective, avoiding undesirable C&#45;to&#45;U changes&#63; Features of the cis&#45;acting elements required for mitochondrial editing have been discerned by introducing exogenous transcripts carrying C editing targets into electroporated mitochondria or into mitochondrial extracts. However, no trans&#45;factor involved in plant mitochondrial RNA editing has yet been identified. <br/>     We have taken a genetic approach in order to identify trans&#45;acting components of the mitochondrial editing apparatus. A pilot study with the 39 C targets of editing in ccb206 transcripts revealed an editing efficiency polymorphism between 'Arabidopsis thaliana' accessions Ler and Col. The C at position 406 in 'ccb206' is edited in Ler at an average of 23&#37; and in Col at 51&#37;; this polymorphism is controlled by a major QTL on chromosome 4 &#40;Bentolila, Chateigner&#45;Boutin, Hanson, 2005, Pl. Phys. 139:2006&#41;. We now report that 362 C&#45;to&#45;U editing sites in 33 mitochondrial genes have been surveyed with RNA extracted from rosette leaves. We detected 67 new editing events in leaves that were not observed in a prior report of mitochondrial editing in Arabidopsis suspension cultures. Furthermore, 37 of the 441 C&#45;to&#45;U editing events reported in Arabidopsis suspension cultures were not observed in leaves. Forty editing sites that are polymorphic in extent of editing were detected between Col and Ler. QTL mapping with recombinant inbred lines has detected 12 major QTLs for 11 of the 13 editing traits analyzed, indicating that map&#45;based cloning of major mitochondrial editing factors should be possible. &#40;Supported by NSF MCB 0344007 to MRH. and an NSF BTI&#47;Cornell PGRP summer internship to LEE.&#41;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Cloning and characterization of a TIR&#45;binding protein in maize Mutator transposon.</div>

<div>

	<span style="text-decoration: underline" >
	Yew Lee</span>
	<sup>1</sup>, 

	<span  >
	Jin-Woo Jung</span>
	<sup>1</sup>, 

	<span  >
	Soo-Hwan Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Life Science, Yonsei University, Wonju-Si 220-710, Korea

</div>

<div>The Mutator family &#40;Mu&#41; of maize transposable elements is the most efficient gene&#45;tagging systems in plants. All Mu elements share highly conserved &#126;210 bp of terminal inverted regions &#40;TIRs&#41; where Mutator transposase, MURA, binds a 32 bp region in the TIR of Mu1. The internal DNA of different members of the Mu system is non&#45;homologous both in the size and sequence. Mutator shows &#8220;cut &#38; paste&#8221; excision activity in the somatic cells while it switches to replicative transposition activity in the germinal cells and gametophytes. Another characteristic activity of the Mutator system is the fact that it is tightly regulated in the maize life cycle so that events are restricted to terminally differentiating cells. In an effort to understand host contribution to the regulation of Mutator activities, we cloned two TIR&#45;binding proteins that bind to 13 bp Mu1 TIR sequence &#40;CGGGAACGGTAAA, previously designated as Site I by Zhao and Sundaresan&#41;. Sequence analysis reveals that one of the binding proteins, St1&#45;65, contains a nuclear targeting signal and Zn&#45;Cys signature which is a Cys&#45;rich motif that is involved in zinc&#45;dependent DNA binding activity. Moreover, a gel shift analysis using yeast&#45;expressed protein as a probe shows that St1&#45;65 specifically binds to Site I in vitro. More biochemical analyses of binding activities and their in vivo meaning in regulation of Mutator activities will be discussed. &#13;<br/>&#13;<br/>&#42; This work was supported by the Korea Research Foundation Grant funded by the Korean Government &#40;MOEHRD, Basic Research Promotion Fund&#41;&#40;KRF&#45;2006&#45;521&#45;C00171&#41;.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis 1&#45;Aminocyclopropane&#45;1&#45;Carboxylic Acid Synthase &#40;ACS&#41; Gene Family</div>

<div>

	<span style="text-decoration: underline" >
	Atsunari Tsuchisaka</span>
	<sup>1</sup>, 

	<span  >
	Takeshi Yamagami</span>
	<sup>1</sup>, 

	<span  >
	Jose M. Alonso</span>
	<sup>2</sup>, 

	<span  >
	Joseph R. Ecker</span>
	<sup>3</sup>, 

	<span  >
	Athanasios Theologis</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Gene Expression Center, 800 Buchanan St., Albany, CA 94710, U.S.A., 

	<sup>2</sup>Department of Genetics, North Carolina State University, Raleigh, NC 27695, U.S.A., 

	<sup>3</sup>Salk Institute for Biological Studies, La Jolla, CA 92037, U.S.A.

</div>

<div>1&#45;AMINOCYCLOPROPANE&#45;1&#45;CARBOXYLATE SYNTHASE &#40;ACS&#41; catalyzes the rate&#45;limiting step in the ethylene biosynthetic pathway in plants. The Arabidopsis genome encodes nine ACS polypeptides that form eight functional and one non&#45;functional homodimers. Why are there so many ACS isozymes in Arabidopsis&#63; We hypothesize that each member of the <span style="font-style: italic">ACS</span> gene family may have a distinct biological function. Here, we report various approaches for elucidating the role of each isozyme. Firstly, biochemical characterization of the various ACS polypeptides revealed that they are biochemically diverse. We think that the biochemical diversity defines a distinct biological function of each isozyme, which in turn defines its tissue specific expression. Second, functional intermolecular complementation experiments in <span style="font-style: italic">E</span>.<span style="font-style: italic">coli</span> show that all isozymes can form heterodimers. However, functional heterodimers are detected only among members that belong to the same phylogenetic branch. We propose that functional heterodimerization enhances the biochemical diversity of the <span style="font-style: italic">ACS</span> gene family; the non&#45;functional heterodimers may have a regulatory role. Thirdly, analysis of promoter&#45;GUS fusions reveals unique and overlapping expression patterns during plant development. This raises the prospect that functional ACS heterodimers may be formed <span style="font-style: italic">in planta</span>. Lastly, we have identified T&#45;DNA insertion lines for seven &#40;<span style="font-style: italic">ACS1</span>, <span style="font-style: italic">ACS2</span>, <span style="font-style: italic">ACS4</span>, <span style="font-style: italic">ACS5</span>, <span style="font-style: italic">ACS6</span>, <span style="font-style: italic">ACS7</span> and <span style="font-style: italic">ACS9</span>&#41; among the nine <span style="font-style: italic">ACS</span> genes; <span style="font-style: italic">double</span> and higher order mutations were constructed. The analysis reveals that ethylene produced by specific ACS isozymes play a central role in determining flowering time. The data suggest that isozyme specific ethylene production regulates the relative ratio of repressors&#47;activators responsible for flowering time. These observations provide molecular insight into the unique and overlapping functions of the <span style="font-style: italic">ACS</span> gene family members in <span style="font-style: italic">Arabidopsis</span>. </div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis Biological Resource Center &#8211; Current Collections and Activities</div>

<div>

	<span style="text-decoration: underline" >
	Randy Scholl</span>
	<sup>1</sup>, 

	<span  >
	Luz Rivero</span>
	<sup>1</sup>, 

	<span  >
	Deborah Crist</span>
	<sup>1</sup>, 

	<span  >
	Emma Knee</span>
	<sup>1</sup>, 

	<span  >
	James Mann</span>
	<sup>1</sup>, 

	<span  >
	Natalie Case</span>
	<sup>1</sup>, 

	<span  >
	Zhen Zhang</span>
	<sup>1</sup>, 

	<span  >
	Garret Posey</span>
	<sup>1</sup>, 

	<span  >
	Pamela Vivian</span>
	<sup>1</sup>, 

	<span  >
	Christopher Calhoun</span>
	<sup>1</sup>, 

	<span  >
	Sarah Pfingsten</span>
	<sup>1</sup>, 

	<span  >
	Kanika Johar</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Ohio State University, Columbus, (OH), USA

</div>

<div>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The Arabidopsis Biological Resource Center &#40;ABRC&#41; collects, preserves and distributes seed and DNA stocks of Arabidopsis.  ABRC stock information is accessible through TAIR &#40;http:&#47;&#47;arabidopsis.org&#41;.<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The seed stock collection includes: A&#41; 10,000 purified T&#45;DNA lines from J. Ecker, B&#41; 1,018 purified T&#45;DNA insertion lines from different researchers, C&#41; 53,084 SAIL T&#45;DNA lines from Syngenta, D&#41; 918 homozygous SAIL T&#45;DNA lines, E&#41; 2,800 lines from GABI&#45;Kat representing 404 T&#45;DNA insertions, F&#41; 4,574 insertion lines from John Innes Centre, G&#41; 626 RNAi lines from AGRIKOLA representing 66 genes, H&#41; 2,500 mutant lines, I&#41; 15 recombinant inbred populations, J&#41; 1,500 natural accessions &#40;ecotypes&#41;, K&#41; 95 other species, and L&#41; miscellaneous transgenic lines.  <br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The T&#45;DNA lines of the SALK, SAIL and Wisconsin collections provide insertions in 25,000&#43; different Arabidopsis genes.  Diverse sets of RI populations have been received from groups associated with the following researchers: A. Lloyd, J. Borevitz, C. Schwartz and L. Comai.<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The collection of DNA stocks includes: A&#41; 15,146 sequence&#45;validated open Reading Frame &#40;ORF&#41; clones from SSP&#47;J. Ecker, B&#41; 2,950 ORF clones from C. Town, C&#41; 2,086 Gateway&#8482; expression clones from S. P. Dinesh Kumar, D&#41; 61 multifunctional vectors that utilize the Gateway&#8482; system, E&#41; 1,000 expression clones from S. Clouse, and F&#41; 20,000 GST entry clones from Agrikola.  The present ORF and cDNA collection represents 13,500&#43; genes.  The SSP ORF collection was formatted to plates at ABRC, transferred to Gateway&#8482; vectors, and validated by end sequencing at SALK; 1,899 of these clones have been received. <br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;During the past year, ABRC distributed 85,000&#43; seed and DNA stocks to researchers worldwide.  Distributions of T&#45;DNA lines contribute to the very high numbers of seed stocks being sent, and the ORF clones represent 50&#37; of the DNA stocks being shipped.  <br/><br/>ABRC is supported by the National Science Foundation.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Acquisition and Distribution of Stocks for Genomics and Phenomics by ABRC</div>

<div>

	<span style="text-decoration: underline" >
	Randy Scholl</span>
	<sup>1</sup>, 

	<span  >
	Emma Knee</span>
	<sup>1</sup>, 

	<span  >
	Luz Rivero</span>
	<sup>1</sup>, 

	<span  >
	Deborah Crist</span>
	<sup>1</sup>, 

	<span  >
	Zhen Zhang</span>
	<sup>1</sup>, 

	<span  >
	Natalie Case</span>
	<sup>1</sup>, 

	<span  >
	James Mann</span>
	<sup>1</sup>, 

	<span  >
	Garret Posey</span>
	<sup>1</sup>, 

	<span  >
	Pamela Vivian</span>
	<sup>1</sup>, 

	<span  >
	Christopher Calhoun</span>
	<sup>1</sup>, 

	<span  >
	Sarah Pfingsten</span>
	<sup>1</sup>, 

	<span  >
	Kanika Johar</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Ohio State University, Columbus, (OH), USA

</div>

<div>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The Arabidopsis Biological Resource Center &#40;ABRC&#41; acquires, preserves, maintains, and distributes many stocks relevant to genome exploration and functional genomics.  Several flank&#45;tagged insertion collections are available from ABRC, specifically: the T&#45;DNA lines from a&#41; Salk Institute, J. Ecker lab, b&#41; Syngenta Biotechnology, including the lines originally associated with MTAs, c&#41; the Wisconsin Ds&#45;Lox population, d&#41; GABI&#45;Kat lines.  RNAi lines from the AGRIKOLA consortium are also distributed.<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Developing initiatives related to insertion lines and their application to phenomics include: a&#41; receipt of 10,000 lines to date of the confirmed, homozygous collection from J. Ecker; b&#41; incorporation of 2,000 homozygous T&#45;DNA insertion lines from misc. donors, and c&#41; efforts to make these lines available for large phenotypic studies.<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;21,222 full&#45;length open reading frame &#40;ORF&#41; cDNA clones from SSP and Salk &#40;J. Ecker&#41;, TIGR &#40;C. Town&#41;, Peking&#47;Yale and J. Callis are available.  These represent ca. 13,500 loci.  GST entry clones from the AGRIKOLA project are available and RNAi constructs will be received shortly.  We have formatted 11,000 of the SSP&#47; SALK and 1,153 of the Peking&#47;Yale clones into plates.  TIGR and AGRIKOLA entry clones were also received in plate format.  We plan to distribute similar large collections in this format in addition to providing individual clones.<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Clones of the Salk and SSP collections are presently being received in Gateway&#8482; Entry vectors.  Eventually, ca, 11,000 clones from these collections will be available as sequence&#45;confirmed entry clones.  We are receiving many ORFs cloned into Gateway&#8482; expression vectors, 3,243 of these &#40;from S. P. Dinesh Kumar and other donors&#41; are currently in house.  Versatile Gateway&#8482; and Uni&#45;vector destination vectors for various expression applications in plants, bacteria, insects, mammals and yeast are also available.<br/><br/>ABRC is supported by the National Science Foundation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulation of the <span style="font-style: italic">Arabidopsis TRANSPARENT TESTA 10</span> gene involved in seed coat flavonoid oxidation</div>

<div>

	<span style="text-decoration: underline" >
	Przemyslaw Bidzinski</span>
	<sup>1</sup>, 

	<span  >
	Lucille Pourcel</span>
	<sup>1</sup>, 

	<span  >
	Jean-Marc Routaboul</span>
	<sup>1</sup>, 

	<span  >
	Lo&#239;c Lepiniec</span>
	<sup>1</sup>, 

	<span  >
	Isabelle Debeaujon</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Seed Biology Laboratory (UMR 204 INRA-AgroParisTech) ; INRA, Jean-Pierre Bourgin Institute, route de Sait-Cyr, 78026 Versailles, France, e-mail: isabelle.debeaujon@versailles.inra.fr

</div>

<div>The <span style="font-style: italic">TRANSPARENT TESTA 10 &#40;TT10&#41;</span> gene is involved in developmental seed coat browning by triggering flavonoid oxidation. <span style="font-style: italic">TT10</span> &#40;At5g48100&#41; encodes a laccase&#45;type polyphenoloxidase &#40;AtLAC15&#41;. Mature <span style="font-style: italic">tt10</span> mutant seeds contained more soluble proanthocyanidins &#40;PAs&#41; than the wild type &#40;WT&#41;, and more epicatechin monomers. Flavonol analysis revealed that quercetin&#45;3&#45;O&#45;rhamnoside &#40;QR&#41; monomers were more abundant and QR dimers were reduced in <span style="font-style: italic">tt10</span> compared with WT. The <span style="font-style: italic">TT10</span> gene is expressed mainly in the testa during seed development. Detailed analysis with GUS and GFP fusions revealed that promoter activity co&#45;localized with two seed coat layers where flavonols and tannins are accumulated, respectively &#40;Pourcel et al., 2005, Plant Cell 17 : 2966&#45;2980&#41;.<br/>5&#8217;&#45;end promoter dissections have been realized and the corresponding <span style="font-style: italic">in planta</span> patterns of GUS expression are being analyzed. Promoter regions for silique and anther expression have been identified. Currently, detailed analysis is being made on seed sections. To progress further in promoter activity understanding, site&#45;directed mutagenesis and gain&#45;of&#45;function experiments will be used to determine which cis&#45;elements are involved in gene expression and particularly in the two&#45;cell layer pattern. Promoter activity is also studied in three regulatory mutant backgrounds <span style="font-style: italic">tt1</span> &#40;a zinc finger protein&#41;, <span style="font-style: italic">tt16&#47;abs</span> &#40;MADS&#41; and <span style="font-style: italic">ttg2</span> &#40;WRKY&#41;. Apart from developmental regulation, <span style="font-style: italic">TT10</span> can also be induced by various stress conditions. For instance, we found out that it is highly induced by drought stress and in response to jasmonic acid. The <span style="font-style: italic">TT10</span> level of expression varies according to the accession and is influenced by UTRs &#40;Pourcel et al., 2005&#41;. Both aspects are currently the subject of detailed analysis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Small RNA&#45;Directed Epigenetic Natural Variation Between Arabidopsis Ecotypes</div>

<div>

	<span style="text-decoration: underline" >
	Jixian Zhai</span>
	<sup>1,2</sup>, 

	<span  >
	Jun Liu</span>
	<sup>3</sup>, 

	<span  >
	Pingchuan Li</span>
	<sup>1,2</sup>, 

	<span  >
	Blake Meyers</span>
	<sup>4,5</sup>, 

	<span  >
	Xuemei Chen</span>
	<sup>6</sup>, 

	<span  >
	Xiaofeng Cao</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>State Key Laboratory of Plant Genomics and National Center for Plant Gene Research, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing 100101, P.R. China, 

	<sup>2</sup>Graduate University of the Chinese Academy of Sciences, Yuquan Road, Beijing 100039, P.R. China, 

	<sup>3</sup>School of Life Science, Shanghai University, Shangda Road 99, Shanghai 200436, P.R. China, 

	<sup>4</sup>Delaware Biotechnology Institute, 

	<sup>5</sup>Department of Plant and Soil Sciences, University of Delaware, Newark, DE 19711, USA., 

	<sup>6</sup>Department of Botany and Plant Sciences, Institute of Integrative Genome Research, University of California, Riverside, Riverside, California 92508, USA.

</div>

<div>Progress in epigenetics has revealed mechanisms which can heritably regulate gene function  independent of genetic alternations1. Nevertheless, little is known about the role of epigenetics in evolution. This is due in part to scant data on epigenetic variation among natural populations2. In plants, small interfering RNA &#40;siRNA&#41; is involved in both the initiation and maintenance of gene silencing by directing heterochromatic modification and&#47;or DNA methylation which could further influence gene expression3. Here we report that, in the model plant Arabidopsis thaliana, a cluster of &#126;24 nt siRNAs found at high levels in the ecotype Landsberg <span style="font-style: italic">erecta</span> &#40;L<span style="font-style: italic">er</span>&#41; could direct DNA methylation and heterochromatinization at the promoter of <span style="font-style: italic">FLOWERING LOCUS C &#40;FLC&#41;</span>, a major repressor of flowering4, whereas the same region in ecotype Columbia &#40;Col&#41; with almost identical DNA sequence, generates a set of low abundance siRNAs that do not direct these activities. DNA methylation of the Ler allele is hypersensitive to mutants in silencing pathway. A comparison of L<span style="font-style: italic">er</span> and Col small RNAs identified at least 70 loci matched by significant level of &#126;24 nt siRNAs found specifically in L<span style="font-style: italic">er</span> but not Col.  These data reveal that there could be substantial epigenetic differences between two closely related Arabidopsis ecotypes, suggesting epigenetic variation may play an important role in the evolution of these and other lineages. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular and functional characterization of a putative <span style="font-style: italic">AtCLC</span> Gene Family in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Qundan Lv</span>
	<sup>1</sup>, 

	<span  >
	Renjie Tang</span>
	<sup>1</sup>, 

	<span  >
	Huiqiong Zheng</span>
	<sup>1</sup>, 

	<span  >
	Hongxia Zhang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Shanghai Institute of Plant Physiology and Ecology,SIBS,CAS,Shanghai , 200032, P.R.China

</div>

<div>Chloride channels &#40;CLC&#41; are passive anion channels that play a variety of roles in turgor maintenance, stomatal movement, nutrient transport, metal tolerance as well as cellular signal perception and transduction. In Arabidopsis thaliana, seven candidate genes encoding putative chloride channels were cloned. The predicted protein of <span style="font-style: italic">AtCLC</span> genes after revision consists of 8 to 10 transmembrane spans and two conserved CBS domains at the carboxyl end, which are proposed to have regulatory roles. Among these seven genes, CLC&#45;c and CLC&#45;d could suppress, with different efficacy, the MnCl2&#45;sensive phenotype of a yeast mutant that is deficient in <span style="font-style: italic">GEF1</span>, the sole chloride channel homologue in yeast <span style="font-style: italic">Saccharomyces cerevisiae</span>. RT&#45;PCR analysis of each transcript during seed germination and in different organs indicates that this gene family was ubiquitously expressed in <span style="font-style: italic">Arabidopsis</span>, some of which, however, exhibit tissue preference. Promoter&#45;GUS analysis in transgenic <span style="font-style: italic">Arabidopsis</span> showed that <span style="font-style: italic">AtCLC</span> was differentially distributed in young seedlings. <span style="font-style: italic">AtCLC&#45;a,b,c,d,.g</span> were strongly expressed in vascular tissues but not in root tip, while <span style="font-style: italic">AtCLC&#45;e,f</span> expressions were dominantly localized in leaf. Recently AtCLC&#45;a has been reported to be a nitrate&#47;proton anitporter mediating nitrate accumulation in plant vacuoles &#40;De Angeli A <span style="font-style: italic">et al</span>, 2006&#41;, which stimulate us to reconsider and dig out substrate specificity of the remaining members of this gene family. We are also isolating and characterizing clc knock&#45;out mutants to address the functions of AtCLC family <span style="font-style: italic">in vivo</span>, coupled with genetic, physiological and biochemical investigations.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis Pseudo Response Regulators PRR9, PRR7, and PRR5 are Involved in the Circadian Phase Transition of the Transcriptome</div>

<div>

	<span style="text-decoration: underline" >
	Norihito Nakamichi</span>
	<sup>1</sup>, 

	<span  >
	Miyako Kusano</span>
	<sup>2</sup>, 

	<span  >
	Atsushi Fukushima</span>
	<sup>2</sup>, 

	<span  >
	Masanori Kita</span>
	<sup>3</sup>, 

	<span  >
	Shogo Ito</span>
	<sup>3</sup>, 

	<span  >
	Takayuki Thoge</span>
	<sup>2</sup>, 

	<span  >
	Takafumi Yamashino</span>
	<sup>3</sup>, 

	<span  >
	Kazuki Saito</span>
	<sup>2,4</sup>, 

	<span  >
	Hitoshi Sakakibara</span>
	<sup>2</sup>, 

	<span  >
	Takeshi Mizuno</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Graduate School of Science, Nagoya University, 

	<sup>2</sup>RIKEN Plant Science Center, 

	<sup>3</sup>School of Agriculture, Nagoya University, 

	<sup>4</sup>Graduate School of Pharmaceutical Sciences, Chiba University

</div>

<div>Arabidopsis Pseudo Response Regulator &#40;PRR&#41; genes are involved in the circadian clock mechanism. A prr9&#45;10 prr7&#45;11 prr5&#45;11 triple mutant was arrhythmic and showed pleiotropic phenotypes. It was consistent with the evidence that a myriad of genes are regulated under the circadian clock. But it has remained largely unknown how PRRs regulate output events of the circadian clock. To address this issue, we performed GeneChip analysis of prr9&#45;10 prr7&#45;11 prr5&#45;11 under light and dark &#40;LD&#41; cycles and under 18 days of constant light conditions &#40;18dLL&#41; as &#8216;steady state&#8217; of the circadian rhythm. The comparative GeneChip analyses suggested two consequences. &#40;1&#41; prr9&#45;10 prr7&#45;11 prr5&#45;11 mutants stall circadian phase of many genes at late day phase, and &#40;2&#41; the expression levels of these genes were highly associated with PRR expression levels. In addition, novel physiological relevance, which was induced by the stalled circadian rhythm during day phase, was observed in prr9&#45;10 prr7&#45;11 prr5&#45;11. For example, GeneChip and GC&#45;TOF&#47;MS analyses revealed that target genes and metabolites of DREB1A &#40;the day&#45;phased gene&#41; were substantially upregulated in prr9&#45;10 prr7&#45;11 prr5&#45;11, even under conditions free from cold or drought stress, and prr9&#45;10 prr7&#45;11 prr5&#45;11 exhibited a phenotype of drought tolerance.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of Ds&#45;tagged mutants for nuclear&#45;encoded chloroplast proteins involved in plastid development and oxidative stress responses.</div>

<div>

	<span style="text-decoration: underline" >
	Fumiyoshi Myouga</span>
	<sup>1</sup>, 

	<span  >
	Haruko Iizumi</span>
	<sup>1</sup>, 

	<span  >
	Takashi Kuromori</span>
	<sup>1</sup>, 

	<span  >
	Reiko Motohashi</span>
	<sup>2</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, 

	<sup>2</sup>Shizuoka University

</div>

<div>  Most of the chloroplast proteins are nuclear&#45;encoded and function in development and environmental responses of chloroplast.  For the functional analysis of the nuclear&#45;encoded chloroplast proteins, we systematically collected their Ds or T&#45;DNA tagged mutant lines in Arabidopsis.  Putative 2,090 chloroplast protein genes were identified by using 4 prediction softwares.  Base on databases of tagged mutant lines, such as RIKEN, NASC, Wisconsin, CSHL and SALK, we collected 3,416 tagged&#45;lines disrupted by Ds transposon or T&#45;DNA that encode 1,374 chloroplast proteins.  From these mutant resources, we have collected 702 of RIKEN Ds tagged&#45;lines and 88 of CSHL Ds tagged&#45;lines.  By screening the 790 Ds tagged&#45;lines, we identified 652 homozygotes with no obvious phenotype, 67 mutants showing abnormal seedling and 66 mutants of which homozygotes were not obtained.  To determine whether the mutations with visible phenotypes are caused by the Ds insertions, we examined mutant alleles existed in other tagged&#45;lines.  Among 48 genes of 67 mutants showing abnormal phenotypes, 21 genes existed several alleles with similar phenotypes, suggesting that their important roles in chloroplast development.<br/>  The chloroplasts play important roles not only in photosynthesis but also in abiotic stress responses such as drought, salinity, freezing.  Our collection of homozygous mutants is a powerful tool for the screening of chloroplast mutants with abnormal stress responses.  By screening of 577 homozygous mutants of RIKEN Ds tag&#45;lines, we isolated several mutant candidates for stress&#45;related chloroplast proteins.  We obtained 42 lines with different capabilities to survive under severe oxidative or osmotic stress conditions on agar plate medium.  Among them, a lot of mutants showed abnormal responses to Paraquart and Alloxan, producers of reactive oxygen species &#40;ROS&#41; .  We are analyzing these mutants to identify nuclear&#45; encoded chloroplast proteins involved in anti&#45;oxidative defense reponses.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Parental imprinting, dosage compensation and dosage effects are observed at a gene expression level in response to changes in genome dosage in <span style="font-style: italic">Arabidopsis thaliana</span>.</div>

<div>

	<span style="text-decoration: underline" >
	Rachel Clifton</span>
	<sup>1</sup>, 

	<span  >
	Olivier Garnier</span>
	<sup>1</sup>, 

	<span  >
	Pjotr Prins</span>
	<sup>2</sup>, 

	<span  >
	Charles Spillane</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University College Cork, Cork, Ireland, 

	<sup>2</sup>Wageningen University, Wageningen, The Netherlands

</div>

<div>A range of epigenetic phenomena appear to be involved in regulating gene expression in plants, derived from observations of dosage compensation, dosage effects and parent&#45;of&#45;origin effects following genome dosage increases. Using the model plant Arabidopsis, this study investigates the transcriptomic consequences of increasing genome dosage by examining gene expression in diploid, maternal and paternal excess triploid and tetraploid plants. Whilst the majority of the Arabidopsis seedling transcriptome is unaffected by changes in the quantity of each parental genome contribution or by whole genome dosage increases, analysis of the fraction of genes differentially expressed in response to altered dosage provides insight into the mechanisms that may be involved. A series of expression responses are observed implying multiple mechanisms may be involved in reprogramming gene expression following an increase in whole genome dose. The function, chromosomal location and expression response of the dosage responsive genome complement of <span style="font-style: italic">Arabidopsis thaliana</span> is explored and its implications discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Constructing a complex RIL population derived from intercrossing 8 Arabidopsis accessions</div>

<div>

	<span style="text-decoration: underline" >
	Xueqing Huang</span>
	<sup>1</sup>, 

	<span  >
	Maria-Joao Paulo</span>
	<sup>1</sup>, 

	<span  >
	Sigi Effgen</span>
	<sup>1</sup>, 

	<span  >
	Maarten Koornneef</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, D-50829, Cologne, Germany

</div>

<div>The genetic control of the majority of biological traits differing between natural accessions is complex. Analysis of quantitative trait loci &#40;QTL&#41; affecting these complex traits is often pursued in single&#45;cross experiments which can only reveal genomic regions that are polymorphic between the parents of the population. Considering several populations derived from diverse parental materials increases the probability that a QTL will be polymorphic in at least one population and consequently increases the power of QTL detection and the discovery of epistatically interacting QTL. Here, we reported that a complex cross population was created from a set of 8 accessions of Arabidopsis thaliana &#40;Col, Kyo, Cvi, Sha, Eri, Ler and C24&#41;. The population consists of 12 subpopulations each of 100 F5 lines each, having per sub population four different parents. A total of 108 microsatellite markers with 3&#45;7 different alleles among the 8 parents and SNP based markers for 206 loci were used to construct a framework linkage map for the complex population. This map provides a starting point for genetic dissection of complex traits. The current effort is towards to use this complex population to map multiple developmental traits such as flowering time, branching pattern, leaf shape for which phenotypic data were collected in the F5 generation. The population as a whole will provide a novel resource for QTL mapping in Arabidopsis, which will allow a higher detection power and detection of  QTL with complex epistatic interactions that can result from interactions between specific alleles derived from more than two parents. </div>

</div>





<div class="abs">

<div style="font-weight:bold">The use of Arabidopsis natural variation for gene function analysis</div>

<div>

	<span style="text-decoration: underline" >
	Maarten Koornneef</span>
	<sup>1</sup>, 

	<span  >
	Xuexing Huang</span>
	<sup>1</sup>, 

	<span  >
	Wim Soppe</span>
	<sup>1</sup>, 

	<span  >
	Matthieu Reymond</span>
	<sup>1</sup>, 

	<span  >
	Joost Keurentjes</span>
	<sup>2</sup>, 

	<span  >
	Leonie Bentsink</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, Cologne, Germany, 

	<sup>2</sup>Wageningen University, The Netherlands, 

	<sup>3</sup>Utrecht University, The Netherlands

</div>

<div>The functional analysis of genes in Arabidopsis has followed the routes of both reverse genetics &#40;from DNA to mutant phenotype&#41; and forward genetics &#40;from variant phenotype to DNA&#41;. For the latter approach in addition to using induced mutants, natural variation is a resource of increasing importance. Arabidopsis thaliana accessions have been collected in a wide range of habitats in the Northern hemisphere. Because these accessions come from different habitats, it is assumed that selection for adaptation to these local environments has occurred and provides genetic variation of responses to environmental factors.&#13;<br/>Natural genetic variation is most effectively studied using immortal mapping populations. QTL and genetic networks involved in the variation of multiple traits can be revealed by studying such populations. Moreover, co&#45;location of QTL for different traits can indicate functions for genes that are not expected. Especially the combination of &#8216;omics&#8217; with QTL analysis provides large data sets for such systems biology. Natural variation is also a useful resource for studying traits that are not accessible in mutant screens in the common laboratory strains because the phenotypes of interest cannot be observed using these genetic backgrounds.. This is illustrated by the study of flowering time, seed dormancy and plant performance QTL. Furthermore epistatic interactions are common when studying the genetics of complex traits. Such interactions often result in novel phenotypes that occur in the progeny of accession crosses. Mapping populations derived from inter&#45;crossing multiple accessions may detect as much as possible such interactions. Map based cloning approach is commonly used to clone the gene&#40;s&#41; underlying the effect of a single QTL that has been Mendelized by backcrossing. The presence of candidate gene&#40;s&#41; in the region of interest and the use of loss of function mutants helps identifying the relevant gene&#40;s&#41; as has been demonstrated for seed dormancy QTL&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis gateway compatible full&#45;length cDNA library</div>

<div>

	<span style="text-decoration: underline" >
	Su Jeoung Suh</span>
	<sup>1</sup>, 

	<span  >
	Sunghwa Choe</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Seoul National University, Seoul, Republic of Korea

</div>

<div>A term plant hormone refers to signal molecules that regulate various processes such as growth, cell division, and stress responses. Thus exogenous application of hormones to plants should accompany induction of rarely&#45;expressed genes. Concerning hormone metabolism and growth regulatory processes, great deal of metabolic and signal transduction pathways is already studied, but there are steps still waiting to be elucidated. To systematically study the function of the &#8220;unknown&#8221; genes in growth and development, we are isolating full length cDNA clones from Gateway&#45;compatible library. RNAs were isolated separately from each different condition, such as auxin, giberellin, cytokinin, salicylic acid, brassinolide, and callus, and mixed together to make cDNA. Adapter&#45;ligated cDNAs were size fractionated using column to remove small fragment. Gateway system was used for cloning the cDNA for the purpose of sub&#45;cloning into various expression vectors. As pioneering screening, we sequenced 100 clones and isolated more than 50&#37; full length cDNAs. Genome annotation analysis of the 100 sequences shows a similar distribution pattern to Arabidopsis whole genome analysis, indicating that the library represent the normalized cDNA clones reflecting the whole genome complexity. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Transcriptional regulation of proanthocyanidin biosynthesis in Arabidopsis seed</div>

<div>

	<span style="text-decoration: underline" >
	Christian DUBOS</span>
	<sup>1</sup>, 

	<span  >
	Jos&#233; LE GOURRIEREC</span>
	<sup>1</sup>, 

	<span  >
	Antoine BAUDRY</span>
	<sup>1</sup>, 

	<span  >
	Jean-Marc ROUTABOUL</span>
	<sup>1</sup>, 

	<span  >
	Isabelle DEBEAUJON</span>
	<sup>1</sup>, 

	<span  >
	Lo&#239;c LEPINIEC</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>INRA Versailles

</div>

<div>In <span style="font-style: italic">Arabidopsis thaliana</span>, proanthocyanidins &#40;PAs; <span style="font-style: italic">syn</span>. condensed tannins&#41; accumulate specifically in the innermost integumentary layer of the seed coat &#40;endothelium&#41;, giving the mature seed its brown colour after oxidation. Metabolomic analysis has shown that PA accumulation is also tightly regulated at the spatio&#45;temporal level. The biosynthesis of PAs is controlled through the specific expression of structural genes such as <span style="font-style: italic">BAN</span> &#40;anthocyanidin reductase&#41;. We have previously shown that TT2 &#40;MYB&#41;, TT8 &#40;bHLH&#41;, and TTG1 &#40;WDR&#41; form a ternary protein complex that directly controls <span style="font-style: italic">BAN</span> and <span style="font-style: italic">TT8</span> expression at the transcriptional level, in a self&#45;activated feedback loop. In this complex, TT2 and TT8 are required for DNA binding <span style="font-style: italic">in vivo</span>, whereas TTG1 appears to act as a co&#45;factor. <br/>To progress further in our understanding of the transcriptional regulation of PA biosynthesis, we are now focusing on the identification of the <span style="font-style: italic">cis</span>&#45;elements on the <span style="font-style: italic">BAN</span> and <span style="font-style: italic">TT8</span> promoters &#40;or other putative target genes&#41;, that are necessary for the binding of the TT2&#45;TT8&#45;TTG1 complex <span style="font-style: italic">in vitro</span> &#40;EMSA&#41; and <span style="font-style: italic">in vivo</span> &#40;yeast and <span style="font-style: italic">in planta</span>&#41;. The second aspect of this work deals with the negative regulation of TT2&#45;TT8&#45;TTG1 complex activity and the possible involvement of other MYBs in this mechanism, like it is the case in some other known MYB&#45;bHLH&#45;WDR regulatory complexes. Finally, the <span style="font-style: italic">TT2</span> and <span style="font-style: italic">TT8</span> promoters are studied in order to identify the <span style="font-style: italic">cis</span>&#45;elements involved in their spatial and temporal activation specificity &#40;promoter dissection and site&#45;directed mutagenesis&#41;. This study should allow us to determine which <span style="font-style: italic">cis</span>&#45;elements are of particular interest to trigger and control the self&#45;activated loop, and to identify new regulators involved in the transcriptional regulation of <span style="font-style: italic">TT2</span> and <span style="font-style: italic">TT8</span> &#40;yeast one&#45;hybrid screening&#41;.<br/>The importance of these regulatory mechanisms in the strong and specific induction of proanthocyanidin biosynthesis in Arabidopsis seed coat will be discussed.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Current status of plant resource project in RIKEN BRC</div>

<div>

	<span style="text-decoration: underline" >
	Masatomo Kobayashi</span>
	<sup>1</sup>, 

	<span  >
	Hiroshi Abe</span>
	<sup>1</sup>, 

	<span  >
	Satoshi Iuchi</span>
	<sup>1</sup>, 

	<span  >
	Toshihiro Kobayashi</span>
	<sup>1</sup>, 

	<span  >
	Takuro Tamura</span>
	<sup>2</sup>, 

	<span  >
	Satoshi Oota</span>
	<sup>1</sup>, 

	<span  >
	Kaoru Fukami-Kobayashi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN BRC, Tsukuba, Japan, 

	<sup>2</sup>BITS Co.,Ltd., Tokyo, Japan

</div>

<div>RIKEN BioResource Center &#40;BRC&#41; was established in 2001 to preserve and distribute biological materials produced in Japan. Experimental Plant Division of RIKEN BRC conducts a project on the resources of model plants including <span style="font-style: italic">Arabidopsis thaliana</span>. We have distributed approximately 23,000 materials to the world research community during these five years. The number of laboratories that have received our materials is nearly 1,000. The major resources are as follows:&#13;<br/>&#40;1&#41; RIKEN Arabidopsis Transposon&#45;tagged Mutant &#40;RATM&#41; lines &#40;&#34;KO&#34;&#45;plants, ca. 16,000 lines&#41;&#13;<br/>&#40;2&#41; RIKEN Arabidopsis Full&#45;Length cDNA &#40;RAFL&#41; clones &#40;ca. 240,000 clones&#41;&#13;<br/>&#13;<br/>Now we are preparing homozygous seeds of RATM lines. Up to now, homozygous seed stock has been prepared for 1,100 lines. We also preparing a new cDNA database, &#34;SABRE&#34; &#40;Systematic consolidation of Arabidopsis and other Botanical REsource&#41;. With this system, you can search not only RAFL clones but also homologous cDNA clones of non&#45;Arabidopsis plants such as <span style="font-style: italic">Physcomitrella patens</span> and <span style="font-style: italic">Populus nigra</span> by using AGI code. If you have an interest, please visit our website &#40;http:&#47;&#47;www.brc.riken.jp&#47;lab&#47;epd&#47;Eng&#47;&#41;. Comments and questions are welcome &#40;plant&#64;brc.riken.jp&#41;.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The leucine&#45;rich repeat receptor like protein family in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Guodong Wang</span>
	<sup>1</sup>, 

	<span  >
	Ursula Ellendorff</span>
	<sup>2</sup>, 

	<span  >
	Chun-Ming Liu</span>
	<sup>1</sup>, 

	<span  >
	Pierre J.G.M. de Wit</span>
	<sup>2</sup>, 

	<span  >
	Gerco C. Angenent</span>
	<sup>1</sup>, 

	<span  >
	Bart P.H.J. Thomma</span>
	<sup>2</sup>, 

	<span  >
	Martijn Fiers</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Research International, B.V., Business Unit of Bioscience, P.O. Box 16, 6700 AA Wageningen, The Netherlands, 

	<sup>2</sup>Laboratory of Phytopathology, Wageningen University, Binnenhaven 5, 6709 PD Wageningen, The Netherlands

</div>

<div>The &#34;Clavata2&#34; &#40;&#34;CLV2&#34;&#41; gene in Arabidopsis encodes a leucine&#45;rich repeat receptor&#45;like protein &#40;LRR&#45;RLP&#41; that assembles into a functional CLV1&#47;CLV2 complex and is involved in restricting the number of stem cells in shoot apical meristem. The complex perceives the CLV3 signal and transduces it to the stem cell organizing centre to repress the expression of the stem cell&#45;promoting WUS transcription factor. LRR&#45;RLPs contain distinct domains: a predicted signal peptide, a variable number of extracellular leucine&#45;rich repeats, a transmembrane domain and a short cytoplasmic tail that lacks any apparent signalling domain. Based on sequence and structural characteristics of CLV2 and TOO MANY MOUTHS &#40;TMM&#41;, 57 LRR receptor&#45;like proteins can be identified in the Arabidopsis genome. To investigate the function of these genes, we have identified homozygous T&#45;DNA insertion lines for each gene. For most of the mutant lines, no obvious phenotype was found with respect to plant growth and development, susceptibility to pathogens, and sensitivity to various stress responses, suggesting that most of them are functionally redundant. However, knockout of &#34;AtRLP41&#34; leads to early leaf senescence with exogenous ABA. Loss&#45;of&#45;function mutation in &#34;AtRLP10&#34; &#40;&#34;CLV2&#34;&#41; had pleiotropic effects with subtle changes in carpel number, more rosette leaves, shorter plants and failed to respond to the CLV3p&#47;CLE19p peptide. &#34;tmm&#45;1&#34; and T&#45;DNA insertion mutant &#34;AtRLP17&#34; &#40;&#34;TMM&#34;&#41; both display enhanced sensitivity to ABA. In addition, to determine the functional domains of CLV2, we have made several constructs in which different domains of CLV2 were deleted. Our data shows that the deletion of the island region does not affect the CLV2 function.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Evolution of miRNAs from random sequences</div>

<div>

	<span style="text-decoration: underline" >
	Felipe Fenselau de Felippes</span>
	<sup>1</sup>, 

	<span  >
	Tobias Dezulian</span>
	<sup>2</sup>, 

	<span  >
	Korbinian Schneeberger</span>
	<sup>1</sup>, 

	<span  >
	Michael Schr&#246;der</span>
	<sup>2</sup>, 

	<span  >
	Daniel H. Huson</span>
	<sup>2</sup>, 

	<span  >
	Detlef Weigel</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Developmental Biology, T&#252;bingen, Germany, 

	<sup>2</sup>Center for Bioinformatics, T&#252;bingen University, T&#252;bingen, Germany

</div>

<div>Plant microRNAs &#40;miRNAs&#41; are produced from precursors that contain self&#45;complementary foldbacks. These precursors are processed by DICER&#45;LIKE1 &#40;DCL1&#41;, generating the mature miRNAs that are incorporated into the RISC, a protein complex that regulates miRNA target genes. In plants, some young miRNA genes arose by inverted duplication of what later might become target genes of the miRNAs. However, analysis of the evolutionary origin of young miRNAs suggests that this is not the only path to the emergence of new miRNAs. <span style="font-style: italic">In silico</span> folding of a typical plant genome results in hundreds of thousands of potential foldback sequences. To test if such random foldbacks could give origin to new miRNAs genes, we previously used data from massively parallel signature sequencing &#40;MPSS&#41; to identify new potential <span style="font-style: italic">Arabidopsis thaliana</span> miRNA genes that are not conserved in other genomes. These potential miRNA precursors were assayed for both their ability to generate a miRNA and potential effects on target mRNAs. Analysis of these potential miRNAs together with database entries of miRNAs that are private to <span style="font-style: italic">A. thaliana</span> &#40;i.e., not found in poplar or rice&#41; suggest that while some arose from a sequence that either has self&#45;complementarity by chance or that represents a highly degenerate inverted duplication. We propose that miRNAs can arise spontaneously from foldback sequences captured by transcriptional regulatory sequences. Subsequent stabilization through co&#45;evolution with potential targets may lead to subsequent fixation of a small subset.<br/><br/>Acknowledgment: Max Planck Society, DAAD. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis AtbZIP9 protein fused to the VP16 transcriptional activation domain alters leaf and vascular development</div>

<div>

	<span  >
	Amanda Bortolini Silveira</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Luiz Gustavo Guedes Corr&#234;a</span>
	<sup>2</sup>, 

	<span  >
	Luciane Gauer</span>
	<sup>1</sup>, 

	<span  >
	Juarez Tomaz</span>
	<sup>1</sup>, 

	<span  >
	Michel Vincentz</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centro de Biologia Molecular e Engenharia Gen&#233;tica-UNICAMP, Campinas (SP), Brazil, 

	<sup>2</sup>Institut f&#252;r Biochemie und Biologie, Universit&#228;t Potsdam, Potsdam, Germany

</div>

<div>Transcriptional regulatory factors play an important role in controlling growth and development. bZIPs transcriptional factors have been reported to act in several different plant&#45;specific processes such as organ development, cell elongation, defense mechanism, hormones and sucrose signalization, light response, control of nitrogen&#47;carbon balance, flowering and the unfolded protein response. The Arabidopsis Group C bZIP transcriptional regulatory factors includes four members that are homologous to the maize <span style="font-style: italic">Opaque&#45;2</span> regulatory locus. These four Arabidopsis bZIP were organized into three groups of orthologues each of which possibly representing an ancestral angiosperm function. To better define the evolution of group C functions we initiated the characterization of <span style="font-style: italic">AtbZIP9</span>, a single Arabidopsis gene corresponding to one of the three group C ancestral functions. We showed that <span style="font-style: italic">AtbZIP9</span> expression is restricted to phloem cells and that its mRNA accumulation is repressed by glucose and induced by abscisic acid and cytokinin. Reverse genetic approaches such as RNAi, knockout and superexpression suggest that post&#45;transcriptional regulation and&#47;or functional redundancy may act on AtbZIP9. To overcome the redundancy aspect, Arabidopsis transgenic lines expressing a fusion between the <span style="font-style: italic">AtbZIP9</span> cDNA and the VP16 transcriptional activation domain were produced. Transformants displayed leaf developmental defects, as well as metabolic and physiologic modifications, such as phenolic compound accumulation in mesophyl cells, early cell death and senescence symptoms. <span style="font-style: italic">AtbZIP9</span> can also be involved in the control of leaf and root vascular system development. We suspect that the observed alteration of leaf morphology and physiology possibly reflects consequences of changes in phloem transport properties, due to defects in vascular cylinder cell differentiation and organization. &#13;<br/>&#13;<br/>Supported by FAPESP and CNPq&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genome&#45;wide expression profiling of the <span style="font-style: italic">Arabidopsis</span> female gametophyte identifies families of small, secreted proteins</div>

<div>

	<span style="text-decoration: underline" >
	Matthew Jones-Rhoades</span>
	<sup>1</sup>, 

	<span  >
	Justin Borevitz</span>
	<sup>2</sup>, 

	<span  >
	Daphne Preuss</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Genetics and Cell Biology, The University of Chicago, Chicago, IL, USA, 

	<sup>2</sup>Department of Ecology and Evolution, The University of Chicago, Chicago, IL, USA

</div>

<div>        The female gametophyte of flowering plants, the embryo sac, develops within the sporophytic tissue of the ovule.  While embryo sac expressed genes are known to be required at multiple stages of the fertilization process, the set of embryo sac expressed genes has remained poorly defined.  In particular, the genes responsible for mediating intracellular communication between the embryo sac and the male gametophyte, the pollen grain, is unknown.&#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;We used high throughput cDNA sequencing and whole genome tiling arrays to compare gene expression in wild type ovules to that in <span style="font-style: italic">dif1</span> ovules, which entirely lack embryo sacs, and <span style="font-style: italic">myb98</span> ovules, which are impaired in pollen tube attraction.  We identified over 400 genes that are down regulated in <span style="font-style: italic">dif1</span> ovules.  75&#37; of these embryo sac dependent genes were predicted to encode for secreted proteins, and over 60 &#37; belonged to multigenic families.  Our results define a large number of candidate intracellular signaling molecules that may act during embryo sac development or during fertilization, less than half of which are represented on the widely used ATH1 expression array.  In particular, 39 out of 40 genes encoding Domain of Unknown Function 784 &#40;DUF784&#41; domains require the synergid specific transcription factor MYB98 for expression in the embryo sac, implicating the DUF784 gene family as mediators of late stages of embryo sac development or interactions with pollen tubes.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">EasyGO: Gene Ontology&#45;based annotation and functional enrichment analysis system for agronomical species<br/></div>

<div>

	<span style="text-decoration: underline" >
	Xin Zhou</span>
	<sup>1</sup>, 

	<span  >
	Zhen Su</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>College of Biological Sciences, China Agricultural University

</div>

<div>Background: Interpretation of microarray experimental results has always been a difficult task. Recently a handful of tools have been developed for this need, but nearly none of them is designed to support agronomical species.<br/>Results: We present EasyGO, a web server to perform Gene Ontology&#45;based functional interpretation on groups of genes or microarray probe sets. EasyGO makes special contribution to agronomical research community by supporting Affymetrix GeneChips of both crops and farm animals, and is strengthened in result visualization and user interaction. Currently it supports 11 crops, 3 farm animals, and model plant Arabidopsis. We demonstrated EasyGO&#8217;s ability to uncover hidden knowledge by analyzing a group of probe sets sharing common expression profile.<br/>Conclusion: EasyGO is a good tool to help biologists to discover enriched biological facts that can provide solution or hints to original problem. It is freely available for all users at http:&#47;&#47;bioinformatics.cau.edu.cn&#47;easygo&#47;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Exploiting the natural genetic variation in recombinant inbred lines: two accessions, multiple traits.</div>

<div>

	<span style="text-decoration: underline" >
	Christian Hardtke</span>
	<sup>1</sup>, 

	<span  >
	Richard Sibout</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Lausanne, Lausanne, Switzerland

</div>

<div> Arabidopsis wild accessions are an increasingly popular resource to identify allelic variants and even novel genes of interest for a given developmental process. In our lab, we exploit this natural genetic variation to identify factors that modulate growth rate and thereby contribute to morphological differences between accessions. Growth is most easily monitored in the root, where growth measurements are essentially one&#45;dimensional. In a survey of primary root growth rate in a sample of 44 accessions, we identified one line, Uk&#45;1, in which root growth is markedly reduced. Molecular genetic analysis of this line revealed that this phenotype is largely due to loss&#45;of&#45;function of one locus, named BREVIS RADIX &#40;BRX&#41;. Gene expression analyses and physiological experiments suggest that BRX plays a role in the interaction of the auxin and brassinosteroid pathways. Both hormones have been implicated in vascular differentiation, and therefore it was not surprising that in another survey of natural variation in secondary growth of the hypocotyl vasculature, Uk&#45;1 was among the lines with a high phloem to xylem ratio. This phenotype has also been observed in certain brassinosteroid signaling mutants. Indeed, vascular growth is decreased in Uk&#45;1 plants, both in the hypocotyl and root, when compared to transgenic Uk&#45;1 plants complemented by a 35S:BRX transgene. Surprisingly though, the xylem to phloem ratio in the hypocotyl is not influenced by BRX, a result that is also confirmed by QTL analysis of a recombinant inbred line population derived from a Uk&#45;1 x Sav&#45;0 cross. Rather, this trait mapped to the same large effect QTL that explains the difference in flowering time between Uk&#45;1 and Sav&#45;0. This finding prompted us to revisit our secondary growth survey of natural accessions. Indeed, nearly all natural variation in secondary growth of the vasculature could be explained by natural variation in flowering time. The implications of this result for screening secondary growth mutants and further details will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A mutant&#45;based phenome analysis and collection of mutant&#45;phenotype information</div>

<div>

	<span style="text-decoration: underline" >
	Takashi Kuromori</span>
	<sup>1</sup>, 

	<span  >
	Asako Kamiya</span>
	<sup>1</sup>, 

	<span  >
	Takuya Ito</span>
	<sup>2</sup>, 

	<span  >
	Takuji Wada</span>
	<sup>1</sup>, 

	<span  >
	Takashi Hirayama</span>
	<sup>2,3</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, Yokohama, Japan, 

	<sup>2</sup>RIKEN Discovery Research Institute, Tsukuba, Japan, 

	<sup>3</sup>Yokohama City University, Yokohama, Japan

</div>

<div>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;By the availability of a variety of research resources, it is now able to investigate mutant lines for almost every gene in <span style="font-style: italic">Arabidopsis</span>. <span style="font-style: italic">Arabidopsis</span> is, then, not only a model plant for plant research, but also a model species in which it is possible to carry out saturation mutagenesis, and totally analyze each gene and mutant in an organism. One of the future goals of phenome project is to collect knockout&#45;mutant phenotypes for each <span style="font-style: italic">Arabidopsis</span> gene. To make a total phenome database, we are collecting phenotype information by two activities, a mutant research activity and a publication research activity.&#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In mutant research activity, we are setting three categories of measurement to see various phenotypes, such as physical, chemical or biological method. We selected 4,000 transposon&#45;tagged lines with a transposon insertion in gene&#45;coding regions and systematically observed the visible phenotype as a first step of phenome analysis. Totally about 200 clear visible phenotypes were classified into 43 categories of morphological phenotypes. Phenotypic images have been entered into a searchable database &#40;http:&#47;&#47;rarge.gsc.riken.jp&#47;phenome&#47;&#41; &#40;1, 2&#41;. In parallel, we have been selecting homozygous transposon&#45;tagged plants, which would be useful resources to see other phenotypes, such as chemically or biologically measured phenotypes.&#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In publication research activity, we are collecting mutant phenotype information in published reports of journals. We are focusing the information of single&#45;gene mutants, since we referred to Dr. Meinke's work &#40;3&#41;. To date we obtained the information about 1,400 genes. Two activities will be combined to make a comprehensive phenotype database.&#13;<br/>&#13;<br/>References&#13;<br/>1. Kuromori T. et al. &#40;2004&#41; Plant J. 37, 897&#45;905.&#13;<br/>2. Kuromori T. et al. &#40;2006&#41; Plant J. 47, 640&#45;651.&#13;<br/>3. Meinke D. et al. &#40;2003&#41; Plant Physiol. 47, 409&#45;418.</div>

</div>





<div class="abs">

<div style="font-weight:bold">PlantTFDB: The Plant Transcription Factor Database</div>

<div>

	<span  >
	Xin Chen</span>
	<sup>1</sup>, 

	<span  >
	An-Yuan Guo</span>
	<sup>1</sup>, 

	<span  >
	Ge Gao</span>
	<sup>1</sup>, 

	<span  >
	Qi-Hui Zhu</span>
	<sup>1</sup>, 

	<span  >
	Ying-Fu Zhong</span>
	<sup>1</sup>, 

	<span  >
	He Zhang</span>
	<sup>1</sup>, 

	<span  >
	Xiao-Cuan Liu</span>
	<sup>1</sup>, 

	<span  >
	Xiaocheng Gu</span>
	<sup>1</sup>, 

	<span  >
	Kun He</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Jingchu Luo</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Center for Bioinformatics, Peking University, Beijing 100871, China

</div>

<div>PlantTFDB: The Plant Transcription Factor Database<br/><br/>Xin Chen, An&#45;Yuan Guo, Ge Gao, Qi&#45;Hui Zhu, Ying&#45;Fu Zhong, He Zhang, Xiao&#45;Cuan Liu, Xiaocheng Gu, Kun He, Jingchu Luo. Center for Bioinformatics, Peking University, Beijing 100871, China<br/><br/>We implemented the following approach to make genome&#45;wide identification of transcription factors &#40;TFs&#41; and constructed the plant transcription factor database PlantTFDB &#40;http:&#47;&#47;planttfdb.cbi.pku.edu.cn&#41;. First, we collected a list of plant TF families from literature and compiled HMM profiles of the DNA binding domains for all these families with the information retrieved from Pfam and obtained from our own analysis. Second, we made HMMER search against the protein sequence of plant genomes to identify putative TFs. We chose seed sequences to make BLASTP search for some families without HMM profiles. Finally, we checked the predicted results manually to reduce false positives and redundant hits.<br/>To provide comprehensive information for the putative TFs, we made extensive annotations at both family and gene levels. A brief introduction and key references were presented for each family. Structure and functional information as well as the target sequence were also provided if available. For each identified TF, general description as well as gene structure, putative domains and PDB hits were provided. Cross&#45;references to various databases such as GenBank, UniGene, UniProt, Prosite, Pfam and TransFac were linked to each entry. In addition, PlantTFDB has a simple interface to allow users to search the database by IDs or free texts, to make sequence similarity search using BLAST, and to download all TF sequences. <br/>Currently, PlantTFDB contains TFs identified from the genome sequences of three model organisms, Arabidopsis, rice and poplar. We are adding two other genomes, the green alga Chlamydomonas reinhardtii and the moss Physcomitrella patens. We are also working on the available EST sequences from 17 plant species including crops &#40;maize, barley, wheat, etc&#41;, fruits &#40;apple, orange, grape, etc&#41;, trees &#40;pine, spruce, etc&#41; and other economically important plants &#40;cotton, potato, soybean, etc&#41;.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The plant specific BPC&#47;BBR family of GAGA&#45;repeat Binding proteins</div>

<div>

	<span style="text-decoration: underline" >
	Dierk Wanke</span>
	<sup>1</sup>, 

	<span  >
	Mareike Hohenstatt</span>
	<sup>1</sup>, 

	<span  >
	Kenneth W. Berendzen</span>
	<sup>1</sup>, 

	<span  >
	Sabine Hummel</span>
	<sup>1</sup>, 

	<span  >
	Joachim Kilian</span>
	<sup>1</sup>, 

	<span  >
	&#220;ner Kolukisaoglu</span>
	<sup>2</sup>, 

	<span  >
	Klaus Harter</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ZMBP-Pflanzenphysiologie, Universit&#228;t T&#252;bingen, Auf der Morgenstelle 1, 72076 T&#252;bingen, 

	<sup>2</sup>CELISCA (Center for Life Science Automation), Friedrich-Barnewitz-Strasse 8, D-18119 Rostock

</div>

<div>BPC&#47;BBR proteins comprise a novel class of transcription factors that are confined to the plant kingdom. They have recently been identified as essential key&#45;regulators of homeotic gene expression in barley and Arabidopsis.<br/><br/>BPC&#47;BBR&#45;proteins have been identified due to their specific binding to a conserved element with its simple sequence repeat consensus of GA&#47;TC dinucleotides. These proteins have properties of animal GAGA&#45;binding factors, but they exhibit no sequence homologies to Trl and Psq of Drosophila, which encode functionally analogous proteins.<br/><br/>So far three distinct regions could be identified common to most BPC&#47;BBR proteins: An N&#45;terminal putative activation domain, a NLS and a highly conserved domain at its C&#45;terminus, which mediates DNA&#45;binding.<br/><br/>By structural means, the protein family can be subdivided into two groups which differ by their N&#45;terminal domain. Similarly, phylogenetic analysis based solely on the DNA&#45;binding domain sequence strongly supports the division into two distinct groups.<br/><br/>The basic DNA&#45;binding domain, a 90 amino acid region, is structured as a typical zinc&#45;finger&#45;like motif putatively comprising two &#223;&#45;sheets followed by an a&#45;helix. A full genome target site analysis of putative binding motifs suggest a role in regulating other transcriptional regulators or auxin signalling related genes.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genome&#45;wide analysis of the Tetraspanin gene family in Arabidopsis: phylogeny, gene structure and contribution of gene duplication events</div>

<div>

	<span style="text-decoration: underline" >
	&#211;scar Oliveira</span>
	<sup>1</sup>, 

	<span  >
	C&#225;tia Barbeta</span>
	<sup>1</sup>, 

	<span  >
	Teresa Lino - Neto</span>
	<sup>1</sup>, 

	<span  >
	Rui Tavares</span>
	<sup>1</sup>, 

	<span  >
	Klaus Palme</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Departament of Biology,University of Minho, Campus de Gualtar, 4710-057 Braga, Portugal, 

	<sup>2</sup>Faculty of Biology II, University of Freiburg, Sch&#228;nzlestr. 1, Freiburg, Germany

</div>

<div>Tetraspanin genes were first described in animals and are common to all multicellular organisms. They encode membrane glycoproteins with four conserved trans&#45;membrane domains. Specific, highly conserved amino acid residues influence the molecular interactions crucial for the proteins&#8217; function &#40;Levy and Shoham, 2005&#41;. Tetraspanins are necessary for correct cellular signalling, as well as cell migration, adhesion, proliferation, recognition and fusion &#40;Martin et al, 2005&#41;. <br/>Although the majority of animal tetraspanins have been studied, only two tetraspanin genes have been characterized in Arabidopsis. These studies focus have focused on the mutant phenotypes for the TORNADO2 and the TOM2A genes. <br/>Here, we report a genome&#45;wide analysis of tetraspanins in Arabidopsis. We have identified 21 tetraspanin genes in publicly&#45;available databases. Subsequent phylogenetic analysis reveals a complex evolution of tetraspanins in Arabidopsis, inferred from the complex arrangement of the subfamilies. A comparison among individual gene structures also supports a complex model for tetraspanin evolution, with several introns and exons forming conserved structural patterns.<br/>We also evaluated the contribution of different duplication events in the Arabidopsis genome to the expansion of the tetraspanin family. The results from genome distribution of Arabidopsis tetraspanin genes support the existence of segmental and tandem duplication events for some genes. Despite this, our analysis suggests that after duplication some extensive rearrangement, and probable divergent evolution, might have occurred. <br/><br/><br/>Aknowledgments:<br/><br/>O. Oliveira and C. Barbeta were supported by FCT &#40;grants ref. SFRH &#47; BD&#47; 19005 &#47; 2004 and SFRH&#47;BD&#47;12081&#47;2003, respectively&#41;.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular and functional characterization of Nucleosome Assembly Protein 1&#40;NAP1&#41; family proteins in plant</div>

<div>

	<span style="text-decoration: underline" >
	Yan ZHU</span>
	<sup>1</sup>, 

	<span  >
	Ziqiang LIU</span>
	<sup>1</sup>, 

	<span  >
	Juan GAO</span>
	<sup>1</sup>, 

	<span  >
	Yu YU</span>
	<sup>1</sup>, 

	<span  >
	Wen-Hui SHEN</span>
	<sup>2</sup>, 

	<span  >
	Aiwu DONG</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biochemistry, School of Life Sciences, Fudan University, Shanghai, China, 

	<sup>2</sup>Institut de Biologie Mol&#233;culaire des Plantes, Centre National de la Recherche Scientifique,Strasbourg, France

</div>

<div>Plant growth is an iterative process of cell proliferation and differentiation in response to intrinsic genetic program and environmental signals; therefore, accurate duplication of genome in cell cycle as well as dynamic regulation on chromatin conformation plays critical biological roles. Current knowledge considers nucleosome as the fundamental unit of chromatin in structure and function, and several factors involved in nucleosome assembly have been identified, such as Nucleosome Assembly Protein 1 &#40;NAP1&#41;. NAP1 is highly conserved in eukaryotes, while there is only one copy of NAP1 gene in yeast, plant genomes evolve out NAP1 family of multiple genes encoding NAP1 homologues and NAP1&#45;Related Proteins &#40;NRP&#41;. Our studies show that plant NAP1 and NRP proteins bind histones <span style="font-style: italic">in vitro</span> and <span style="font-style: italic">in vivo</span>, and present expression patterns related to proliferating state, consistent with their biological roles in nucleosome assembly. Microarray analysis shows that both NAP1 and NRP modulate gene expression in genome scale, however, with overlap in low level. Furthermore, <span style="font-style: italic">Arabidopsis nrp</span> double mutant is defective in root growth with cell cycle arrest in root tip after germination; additionally, its genome is more sensitive to genotoxic stress compared with that of wild&#45;type, indicating NRP functions in chromatic duplication and structural maintenance. Our recent work shows that <span style="font-style: italic">nap1</span> triple mutant is insensitive to ABA treatment during seed germination, implying plant NAP1 proteins are involved in phytohormone regulation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">An ORFeome&#45;based analysis of F&#45;box protein genes in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Hirofumi Kuroda</span>
	<sup>1</sup>, 

	<span  >
	Youko Horii</span>
	<sup>1</sup>, 

	<span  >
	Tomoyuki Takase</span>
	<sup>2</sup>, 

	<span  >
	Tomohiro Kiyosue</span>
	<sup>2</sup>, 

	<span  >
	Minami Matusi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN, Yokohama, Japan, 

	<sup>2</sup>Kagawa Univ. Takamatsu, Japan

</div>

<div>F&#45;box protein acts as a receptor for target protein of ubiquitination in SCF complex.  We found 568 F&#45;box protein genes in Arabidopsis genome &#40;Kuroda et al. 2002&#41;, however functions of most of the F&#45;box proteins are still unclear. To carry out large&#45;scale functional analysis, we constructed ORFeome resource of F&#45;box proteins of Arabidopsis in recombination type cloning vector. The cloning was carried out by RT&#45;PCR cloning using total RNAs from various tissues to clone novel splicing variant as well as predicted ORFs. All cloned PCR fragments were sequenced. We cloned 612 kinds of cDNAs from 489 genes. Among the 489 genes, 103 genes had more than two kinds of ORFs that were candidates for alternative splicing variants. We characterized the splicing variants by the analysis of their sequences, structures, expression patterns and subcellular localizations. We identified changing of domain organization between splicing variants, non&#45;canonical intron having nucleotide repeat at both splice sites and tissue&#45;specific splicing event. We also will present our on going studies using cloned ORFs.</div>

</div>





<div class="abs">

<div style="font-weight:bold">OSRs, a family of MscS&#45;like Proteins, involve in Arabidopsis responses to osmotic stress</div>

<div>

	<span  >
	Yi-Fang Chen</span>
	<sup>1</sup>, 

	<span  >
	Hong Li</span>
	<sup>1</sup>, 

	<span  >
	Xin-Jie Han</span>
	<sup>1</sup>, 

	<span  >
	Jie Pang</span>
	<sup>1</sup>, 

	<span  >
	You-Han Kong</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Wei-Hua Wu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>State Key Laboratory of Plant Physiology and Biochemistry, College of Biological Sciences, National Plant Gene Research Centre (Beijing), China Agricultural University, Beijing 100094, China

</div>

<div>Mechanosensitive &#40;MS&#41; ion channels were originally identified as essential components as the osmolyte efflux system activated under hyperosmotic shock in bacteria, and were found to also exist eukaryotes, including both the plants and animals.  However, there functions remain unclear so far in higher plants.  We isolated a low phosphate tolerant mutant lpt1, and it turned out that LPT1 gene encoding a protein containing the same motif with MscS and named as AtOSR1 &#40;Osmotic Stress Responsive&#41;.  Blast with all noted genes in Arabidopsis genome, other 9 genes were founded to encode the same motif with MscS, and named as AtOSR2, 3, 4, 5, 6, 7, 8, 9 and 10, respectively.  Transcription of most AtOSRs in Arabidopsis plants was increased by osmotic stress induction.  Furthermore, tolerance of a MscS&#45;deficient bacterial mutant line to osmotic&#45;shock can be partially rescued by 9 AtOSRs respectively.&#13;<br/>AtOSR1 overexpression Arabidopsis lines showed greater tolerance low Pi stress, increased Pi uptake rate and Pi content compared with wild&#45;type plants.  More interestingly, tolerance to low Pi stress for wild&#45;type plants was induced by an osmotic stress treatment, which suggests that osmotic stress&#45;induced expression of AtOSR1 may increase plant tolerance to low Pi stress. The AtOSR2 knockout mutant &#40;atosr2&#41; was more sensitive to drought stress and exhibited an increased water loss rate and decreased stomatal aperture, indicating that AtOSR2 may play an important role in plant response to drought.  Additional experimental results further showed that function of AtOSR2 may somehow related to mitochondria. The AtOSR9 knockout mutant &#40;atosr9&#41; had the similar phenotype with atosr2.  In addition, AtOSR4 and AtOSR7 may involve in plant responses to water stress, and AtOSR8 may function in embryonic development.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Analysis of the pollen transcriptome changes during pollen germination and pollen tube growth in Arabidopsis</div>

<div>

	<span  >
	Wen-Zheng Zhang</span>
	<sup>1</sup>, 

	<span  >
	Yi Wang</span>
	<sup>1</sup>, 

	<span  >
	Lian-Fen Song</span>
	<sup>1</sup>, 

	<span  >
	Jun-Jie Zou</span>
	<sup>1</sup>, 

	<span  >
	Zhen Su</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Wei-Hua Wu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>State Key Laboratory of Plant Physiology and Biochemistry, College of Biological Sciences, National Plant Gene Research Centre (Beijing), China Agricultural University, Beijing 100094, China

</div>

<div>Pollen germination as well as following pollen tube growth is an essential process for the reproduction of flowering plants.  In addition, a germinating pollen with tip&#45;growth characteristics, provides an ideal model system for the study of cell growth and morphogenesis.  As an essential step towards a detailed understanding of this important process, the objective of this study was to comprehensively analyze the transcriptome changes during pollen germination and tube growth.  This study, for the first time, analyzed the changes in transcriptome from desiccated mature pollen grains to hydrated pollen grains and to pollen tubes of Arabidopsis thaliana by using Affymetrix Arabidopsis ATH1 Genome Arrays.  The number of expressed genes increased significantly from the desiccated mature pollen to the hydrated pollen and to the growing pollen tubes, which is consistent with the result that pollen germination and tube growth in vitro was significantly inhibited by transcriptional inhibitor.  The results showed that expression of the genes related to signal transduction, cellular transport, and subcellular localization were significantly changed and the genes of several families, such as CaM&#47;CML, CHX and Hsp, showed the most significant changes during pollen germination and tube growth.  The presented results demonstrate that the overall transcription of genes, in both the number of expressed genes and the levels of transcription, was increased.  Furthermore, the expression of numerous novel transcripts may be indicative of roles for these newly expressed genes in pollen germination and pollen tube growth.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Phenome Analysis of the Arabidopsis GRAS Genes</div>

<div>

	<span  >
	Mi-Hyun Lee</span>
	<sup>1</sup>, 

	<span  >
	Sang-Kee Song</span>
	<sup>2</sup>, 

	<span  >
	Bohye Kim</span>
	<sup>1</sup>, 

	<span  >
	Jeong-Ok Heo</span>
	<sup>1</sup>, 

	<span  >
	Miran Kim</span>
	<sup>1</sup>, 

	<span  >
	Dong Gwan Kim</span>
	<sup>1</sup>, 

	<span  >
	Sung Ho Sohn</span>
	<sup>1</sup>, 

	<span  >
	Shin Ae Lee</span>
	<sup>1</sup>, 

	<span  >
	Nan-Ie Yu</span>
	<sup>1</sup>, 

	<span  >
	Chae Eun Lim</span>
	<sup>1</sup>, 

	<span  >
	Myeong Min Lee</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Jun Lim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Bioscience and Biotechnology, Konkuk University, 

	<sup>2</sup>Department of Biology, Yonsei University

</div>

<div>GRAS proteins belong to a plant&#45;specific transcription factor family. Currently, 33 GRAS members including an expressed pseudogene have been identified in the Arabidopsis genome. To provide a comprehensive evaluation of the Arabidopsis GRAS family, we conducted a large&#45;scale analysis of the GRAS genes. With a reverse genetic approach, we constructed a &#8220;phenome&#45;ready unimutant collection&#8221;. Of this collection, we focused on loss&#45;of&#45;function mutations in 23 novel GRAS members. Under standard conditions, homozygous mutants have no obvious morphological phenotypes as compared to those of wild&#45;type plants. Expression analysis of the members using quantitative real&#45;time and semi&#45;quantitative RT&#45;PCR, microarray data mining, and promoter::GUS analysis revealed their tissue&#45;specific expression patterns. Our analysis on protein&#45;protein interaction and subcellular localization of individual members indicated their physiological roles as transcription regulators. Our phenome&#45;ready unimutant collection of the GRAS genes will be a useful resource to better understand individual GRAS members that play diverse roles in plant growth and development.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Pollen development and function; integration of transcriptomic and proteomic data</div>

<div>

	<span style="text-decoration: underline" >
	David Honys</span>
	<sup>1,2</sup>, 

	<span  >
	David Twell</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Institute of Experimental Botany ASCR vvi, Prague, Czech Republic, 

	<sup>2</sup>Charles University in Prague, Prague, Czech Republic, 

	<sup>3</sup>University of Leicester, Leicester, United Kingdom

</div>

<div>Male gametophyte development in higher plants is a complex process that requires the coordinated participation of various cell and tissue types and their associated specific gene expression patterns. The male gametophytic life cycle can be divided into a developmental phase leading to the formation of mature pollen grains, and a functional or progamic phase, beginning with the impact of the grains on the stigma surface and ending at double fertilisation. Pollen ontogeny is also an excellent model in which to dissect the cellular networks that control cell growth, polarity, cellular differentiation and cell signaling. Pollen transcriptomic studies have provided an extensive genome&#45;wide view of gene expression during male reproductive cell development in Arabidopsis These studies have revealed at least two successive global gene expression programs and the identity of a large number of male gametophyte&#45;specific genes and putative transcriptional regulators. Transcriptome analysis has also revealed the striking overrepresentation of expressed genes associated with cell wall metabolism, cytoskeleton and signaling in preparation for pollen germination and tube growth during the progamic phase. Although transcriptomic analyses provide valuable information about global and specific gene expression patterns it does not measure the influence of posttranscriptional control of gene expression. In male gametophyte, posttranscriptional and translational control mechanisms have already been demonstrated to be of great importance during pollen maturation and the progamic phase. Recently, the first studies characterizing the pollen proteome are emerging. Although currently available proteomic data are incomplete these data provides the first opportunity to overcome some of the limitations of transcriptomic data and already offers some interesting insights.<br/><br/>Presented work was supported by MSMT &#40;LC06004&#41; and GACR &#40;522&#47;06&#47;0896, 525&#47;06&#47;0864&#41;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Transcriptional regulation of the &#180;&#180;Arabidopsis thaliana&#180;&#180; diacylglycerolkinase gene &#180;&#180;AtDGK7&#180;&#180; in the root</div>

<div>

	<span style="text-decoration: underline" >
	Jonas Krebs</span>
	<sup>1</sup>, 

	<span  >
	Fernando Arana-Ceballos</span>
	<sup>1</sup>, 

	<span  >
	Miroslaw Kwasniewski</span>
	<sup>1,2,3</sup>, 

	<span  >
	Bernd Mueller-Roeber</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>University of Potsdam, Institute of Biochemistry and Biology, Karl-Liebknecht-Str. 24-25, Haus 20, 14476 Potsdam-Golm, Germany, 

	<sup>2</sup>Max-Planck Institute of Molecular Plant Physiology, Am Muehlenberg 1,, 

	<sup>3</sup>Department of Genetics, University of Silesia, Jagiellonska 28, 40-032 Katowice, Poland

</div>

<div>Plants employ various types of signalling molecules to coordinate developmental and physiological processes. Recently, phosphatidic acid &#40;PA&#41; has been identified to play an important role as one of such factors. PA is produced through two different pathways, one including the enzyme diacylglycerol kinase &#40;DGK&#41;. The &#180;&#180;AtDGK7&#180;&#180; gene, one of seven members of the &#180;&#180;Arabidopsis thaliana&#180;&#180; DGK family, exhibits a highly specific expression pattern in roots, where it is activated early during root development. Here we report the investigation of the transcriptional regulation of &#180;&#180;AtDGK7&#180;&#180; including the identification of functionally relevant cis and trans regulatory elements. To identify cis elements of the &#180;&#180;AtDGK7&#180;&#180; promoter contributing to the highly specific and strong expression in roots, promoter deletion studies were performed, using &#223;&#45;glucuronidase as reporter. One of the deletion constructs &#45; containing a motif known to be recognized by Zn&#45;finger transcription factors &#45; was shown to be necessary for &#180;&#180;AtDGK7&#180;&#180; expression. By using public available databases of &#180;&#180;A. thaliana&#180;&#180; gene expression data, we identified one Zn&#45;finger transcription factor gene exhibiting a highly overlapping pattern of expression in the root. This expression was confirmed in transgenic plants transformed with a promoter&#45;GFP fusion. The function of the candidate transcription factor was analysed further by analysing over&#45;expression and loss&#45;of&#45;function mutant lines. Details will be presented.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Global analysis of natural target genes of NMD in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Masato Yoine</span>
	<sup>1</sup>, 

	<span  >
	Maya Akahori</span>
	<sup>1</sup>, 

	<span  >
	Koichi Hori</span>
	<sup>2,3</sup>, 

	<span  >
	Yuichiro Watanabe</span>
	<sup>2</sup>, 

	<span  >
	Kenzo Nakamura</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Grad. Schl. Bioagric. Sci., Nagoya Univ., 

	<sup>2</sup>Dep. of Life Sci., Tokyo Univ., 

	<sup>3</sup>Life Sci., College Sci., Rikkyo Univ.

</div>

<div>Nonsense&#45;mediated mRNA decay &#40;NMD&#41; is an mRNA surveillance mechanism that recognizes mRNAs containing premature termination codons &#40;PTCs&#41; and targets them for degradation. In addition to mutation&#45;derived PTCs, various forms of natural transcripts, either with or without PTC, are targets of NMD, and 4&#37; to 10&#37; of total genes in yeast, <span style="font-style: italic">Drosophila</span>, and human are estimated to be subjected to post&#45;transcriptional regulation by NMD. Although major factors of NMD, namely UPF1, UPF2, and UPF3, are highly conserved among eukaryotes, the mechanisms to discriminate PTCs from natural termination codons are not evolutionarily conserved, and target genes of NMD are different among various eukaryotes. In <span style="font-style: italic">Arabidopsis</span>, missense mutant of <span style="font-style: italic">UPF1, lba1&#47;atupf1&#45;1</span>, exhibits pleiotropic phenotypes such as anomalies in growth, flowering time, sugar signaling, and seed size &#40;1,2&#41;, and null mutant of <span style="font-style: italic">UPF1</span> is seedling lethal &#40;2&#41;. The T&#45;DNA insersion line of <span style="font-style: italic">UPF3, atupf3&#45;1</span>, also exhibits morphological aberration &#40;3&#41;. These observations indicate that NMD plays important roles in plant growth and development. We identified putative natural target genes of NMD in Arabidopsis by comparing transcript levels in Col&#45;0, <span style="font-style: italic">lba1</span> and <span style="font-style: italic">atupf3&#45;1</span> using Affymetrix ATH1 microarray. Approximately 2&#37; of total genes analyzed showed more than 1.5&#45;fold enhancement of transcript levels in both <span style="font-style: italic">lba1</span> and <span style="font-style: italic">atupf3&#45;1</span> compared to Col&#45;0. These putative targets of NMD did not show significant bias towards genes with particular function, while transcripts that are up&#45;regulated in both mutants were enriched with specific structural features. For example, 23&#37; of these genes contained PTC&#45;containing transcript that is generated by alternative splicing. Nearly 40&#37; of genes contained intron in the 5&#8217;&#45;UTR region of the gene, and 50&#37; of genes contained uORF for longer than 10 amino acids in the 5&#8217;&#45;UTR. Experiments with cordycepin suggested that increased levels of target transcripts in <span style="font-style: italic">lba1</span> was due to enhanced mRNA stability. &#40;1&#41; Plant J. 47; 49 &#40;2006&#41;, &#40;2&#41; Plant Cell Physiol. 47; 572 &#40;2006&#41;; &#40;3&#41; Plant J. 43; 530 &#40;2005&#41;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Florigen in rice</div>

<div>

	<span style="text-decoration: underline" >
	Ko Shimamoto</span>
	<sup>1</sup>, 

	<span  >
	Shojiro Tamaki</span>
	<sup>1</sup>, 

	<span  >
	Shoichi Matsuo</span>
	<sup>1</sup>, 

	<span  >
	Hann Lin Wong</span>
	<sup>1</sup>, 

	<span  >
	Shuji Yokoi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Nara Institute of Science and Technology

</div>

<div>Rice is a model of short day plants &#40;SDP&#41; for the molecular genetic study of the photoperiodic regulation of flowering. Rice has a highly conserved genetic pathway for flower induction in SD consisting of OsGI, Hd1, and Hd3a genes. In LD, Hd1suppresses Hd3a expression while in SD Hd1 activates Hd3a expression. Therefore, this dual role of Hd1 in the regulation of Hd3a expression is a molecular basis of the photoperiodic regulation of flowering in rice &#40;Izawa et al., 2002, Hayama et al., 2003&#41;. &#13;<br/>  We are examining the possibility that Hd3a encodes florigen, which is considered to be induced in leaf and transported to the shoot apex and causes flowering. The florigen hypothesis was postulated 70 years ago but its nature is not known. Results of the analysis on transgenic rice expressing the Hd3a:GFP gene fused with three different promoters suggest that Hd3a protein is florigen.  &#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis Chromodomain&#45;containing Protein LIKE HETEROCHROMATIN PROTEIN 1 Colocalizes with Histone H3 Lysine27 Trimethylation</div>

<div>

	<span  >
	Xiaoyu Zhang</span>
	<sup>1</sup>, 

	<span  >
	Sophie Germann</span>
	<sup>2</sup>, 

	<span  >
	Bartlomiej J. Blus</span>
	<sup>3</sup>, 

	<span  >
	Sepideh Khorasanizadeh</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Val&#233;rie Gaudin</span>
	<sup>2</sup>, 

	<span  >
	Steven E. Jacobsen</span>
	<sup>1,4</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular, Cell and Developmental Biology, University of California, Los Angeles, Los Angeles, CA 90095, USA, 

	<sup>2</sup>Laboratoire de Biologie Cellulaire, IJPB, INRA, route de St Cyr, 78026 Versailles, France, 

	<sup>3</sup>Department of Biochemistry and Molecular Genetics, University of Virginia Health System, Charlottesville, VA 22908, USA, 

	<sup>4</sup>Howard Hughes Medical Institute, University of California, Los Angeles, Los Angeles, CA 90095, USA

</div>

<div>The HP1 and POLYCOMB families of proteins are required for the establishment and maintenance of silent chromatin states mediated by histone H3K9 and H3K27 methylation in animals; however, little is known about these proteins in plants. By DamID&#45;chip, we found that the Arabidopsis chromodomain&#45;containing protein LHP1 localizes to chromatin associated with H3K27me3 genome&#45;wide, and its chromodomain binds H3K27me3 with high affinity. These results suggest that LHP1 shares similar functions with POLYCOMB.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Chemical genetic interrogation of natural variation</div>

<div>

	<span  >
	Yang Zhao</span>
	<sup>1</sup>, 

	<span  >
	Tszfung  Chow</span>
	<sup>1</sup>, 

	<span  >
	Rachel Puckrin</span>
	<sup>1</sup>, 

	<span  >
	Simon Alfred</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Sean Cutler</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>University of Toronto, 

	<sup>2</sup>University of California, Riverside

</div>

<div>Natural sequence variation in drug metabolism and target genes can cause inter&#45;individual or pharmacogenetic variation. Given the extensive metabolic variation documented between Arabidopsis accessions we reasoned that Arabidopsis could be used to identify natural drug sensitivity alleles, which in turn could facilitate the characterization of compounds discovered in chemical genetic screens. To test this, we subjected several Arabidopsis accessions to chemical genetic screens and discovered 12 polymorphic&#45;hit molecules that induce accession&#45;selective effects. The natural variation approach revealed that hypostatin, a new inhibitor of cell expansion, requires glucosylation for its bioactivity. Map&#45;based cloning and population scans identified a prevalent SNP in HYR1, a UDP&#45;glycosyl transferase &#40;UGT&#41;, which explains most cases of natural resistance to hypostatin. Additionally, we show that HYR1 potentiates sensitivity to multiple xenobiotics. HYR1 is a member of the UGT&#45;superfamily, which are pharmacogenetic factors in humans; thus natural UGT sequence&#45;variation can modulate xenobiotic sensitivity across biological kingdoms. Our results demonstrate that natural variation that can be exploited to inform the biology of new small molecules. <br/>In addition to describing our natural variation work, in our workshop talk we will provide an update on community&#45;targeted chemical genomic resources developed in our lab.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Mitotic Epigenetic inheritance by meristemic DNA demethylation in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Minhee Kim</span>
	<sup>1</sup>, 

	<span  >
	Hyonhwa Ohr</span>
	<sup>1</sup>, 

	<span  >
	Yeonhee Choi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences, Seoul National Universtiy

</div>

<div>DEMETER, a DNA glycosylase domain&#45;containing protein which functions as a DNA demethylase, shows highly specific expression during plant development. In plant reproductive stage, DME is temporarily expressed in central cell within the embryo sac. In vegetative stage, DME expression is restricted to the region including shoot apical meristem and root apical meristem. Based on this specific DME expression, DNA hypomethylation has been induced to the region where DME is expressed by generating antisense MET1 under the DME promoter. Since the expression of antisense MET1 is dependent on DME promoter, MET1 is expected to be knocked&#45;down in the restricted area including meristemic region and central cell of the female gametophyte, but not in elsewhere. However, global DNA hypomethylation and expression of silent genes and transposons have been detected where DME promoter is inactive such as leaf tissues and pollen. These phenomena imply DNA methylation&#47;demethylation is epigenetically inherited from meristem to the whole plant tissue throughout mitosis.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">microRNA biogenesis and function in Arabidopsis</div>

<div>

	<span  >
	Bin Yu</span>
	<sup>1</sup>, 

	<span  >
	Zhiyong Yang</span>
	<sup>1</sup>, 

	<span  >
	Julien Curaba</span>
	<sup>1</sup>, 

	<span  >
	YunJu Kim</span>
	<sup>1</sup>, 

	<span  >
	Vanitharani Ramachandran</span>
	<sup>1</sup>, 

	<span  >
	Binglian Zheng</span>
	<sup>1</sup>, 

	<span  >
	Theresa Dinh</span>
	<sup>1</sup>, 

	<span  >
	Lijuan Ji</span>
	<sup>1</sup>, 

	<span  >
	Liu Bi</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Xuemei Chen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Riverside, CA, USA

</div>

<div>We are interested in how microRNAs &#40;miRNAs&#41; are produced, modified, and degraded in plants, and how miRNAs act in developmental processes. We are using a combination of genetic and biochemical approaches to study miRNA metabolism and function. We have previously demonstrated that plant miRNAs are methylated on the 2&#8217; OH of the 3&#8217; terminal ribose by the methyltransferase protein HEN1. The methyl group protects miRNAs from an uridylation activity and an exonuclease activity that target the 3&#8217; OH. From a genetic suppressor screen using the weak hen1&#45;2 allele, we isolated mutations in the largest and second largest subunits of pol IVa that restore miRNA methylation in the hen1&#45;2 background. I will present evidence that HEN1 and pol IVa have antagonistic activities in miRNA biogenesis and that one role of methylation is to protect miRNAs from the activities of pol IVa.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Screening of upstream regulator of DEMETER, a DNA glycosylase domain&#45;containing protein, in Arabidopsis by Activation tagging mutagenesis</div>

<div>

	<span style="text-decoration: underline" >
	Jee-woong LEE</span>
	<sup>1</sup>, 

	<span  >
	Geuntae Park</span>
	<sup>1</sup>, 

	<span  >
	Yeonhee Choi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biological Sciences, Seoul National University, Seoul, South KOREA

</div>

<div>DEMETER &#40;DME&#41; is a DNA glycosylase domain protein that controls maternal expression of imprinted genes such as MEDEA &#40;MEA&#41;, Fertilization&#45;Independent Seed 2 &#40;FIS2&#41; and FWA in the central cell of the female gametophyte. DME is required for seed viability. Restricted DME expression in the central cell before fertilization is essential for the activation of maternally expressed&#45;imprinted genes, thus provides the establishment of the imprinting in the endosperm after fertilization. An active DNA glycosylase domain of DME, normally associated with DNA repair, replaces methylated cytosine to unmethylated cytosine. Ectopic DME expression in pollen or endosperm activates expression of the normally silenced paternal MEA and FWA allele. &#13;<br/>  To detect the upstream regulators of DME, activation tagging mutagenesis has been performed by inserting T&#45;DNA of pSKI015 vector into DME::GUS homozygous plants. Activation tagging can induces insertional mutation like other conventional T&#45;DNA as well. In DME::GUS control plants, GUS gene is expressed only in the central cell nucleus within the ovules, but not in pollen grain or seeds. If T&#45;DNA causes recessive insertional mutation and is inserted in the DME activator, transgenic plants will show the decreased 50&#37; T2 ovule expression compared to the 100&#37; T2 ovule in T1 DME:GUS homozygous control plants. If T&#45;DNA is inserted into DME repressor and causes recessive mutation, we expect to see ectopic 50&#37; T2 pollen expression in T1 plants. About 3,000 lines have been screened so far. Five lines of which showed the 50&#37; T2 ovule expression have been selected for the further research. This screen will tell us how DME&#45;restricted expression and genomic imprinting are regulated during gametogenesis. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">MAPPING GENOTYPE INTO PHENOTYPE: NATURAL, INDUCED AND EPIGENETIC VARIATION IN ARABIDOPSIS GENOMES</div>

<div>

	<span style="text-decoration: underline" >
	Joseph R. Ecker</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Salk Institute for Biological  Studies, La Jolla, CA  USA

</div>

<div>A precise and comprehensive understanding of DNA sequence and epigenetic variation in natural populations of a species will be essential for elucidating the basis of phenotypic variation. We have applied  oligonucleotide tiling microarrays and next&#45;generation ultra&#45;high throughput DNA sequencing methods to identify DNA sequence and epigenetic variation in wild strains &#40;accessions&#41; of <span style="font-style: italic">Arabidopsis thaliana</span> that were chosen for maximal genetic diversity. The entire Arabidopsis Genome Initiative &#40;AGI&#41; reference genome sequence of <span style="font-style: italic">A. thaliana</span> Col&#45;0 was tiled on high&#45;density tiling microarrays at single base resolution using over a billion 25 base oligonucleotides. The arrays were hybridized with genomic DNA from 19 accessions, isothermally amplified DNA to minimize ascertainment biases. In addition, whole genome sequencing using a Solexa&#47;Illumina Genetic Analyzer allowed more complete sequence coverage of the genomes of several strains &#40;Col&#45;0, Ler&#45;1, Cvi&#45;0, etc.&#41;, allowing independent assessment of the quality of the AGI Sanger and Perlegen array&#45;based genome sequences. The degree of polymorphism and types of genes that harbor major effect polymorphisms in natural populations will be described. We are also using next&#45;generation sequencing technology to develop methods for creating large populations of T&#45;DNA insertion mutants in diverse genetic backgrounds as well as for determining    the extent of epigenetic diversity in wild accessions.  A proposal  for cost effective re&#45;sequencing of 1,001 Arabidopsis genomes will be outlined. When coupled with community&#45;wide phenotypic screens for traits of interest, this database would allow realization of genome&#45;wide association mapping studies and provide  an unprecedented resource for understanding the genetic&#47;epigenetic basis of phenotypic variation in plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Transcriptional  analysis of the differentiated ovule and female gametophyte of <span style="font-style: italic">Arabidopsis thaliana</span>  by Massively Parallel Signature Sequencing &#40;MPSS&#41;.</div>

<div>

	<span style="text-decoration: underline" >
	Nidia S&#225;nchez-Le&#243;n</span>
	<sup>1</sup>, 

	<span  >
	Mario Arteaga-V&#225;zquez</span>
	<sup>1</sup>, 

	<span  >
	Vianey Olmedo-Monfil</span>
	<sup>1</sup>, 

	<span  >
	Marcelina Garc&#237;a-Aguilar</span>
	<sup>1</sup>, 

	<span  >
	Vicenta Garc&#237;a-Campayo</span>
	<sup>1</sup>, 

	<span  >
	V&#237;ctor P&#233;rez-Espa&#241;a</span>
	<sup>1</sup>, 

	<span  >
	Javier Mendiola-Soto</span>
	<sup>1</sup>, 

	<span  >
	Ana Dorantes-Acosta</span>
	<sup>1</sup>, 

	<span  >
	Mario Arteaga-S&#225;nchez</span>
	<sup>1</sup>, 

	<span  >
	Octavio Mart&#237;nez-de la Vega</span>
	<sup>1</sup>, 

	<span  >
	Kan Nobuta</span>
	<sup>2</sup>, 

	<span  >
	Kalyan Vemaraju</span>
	<sup>2</sup>, 

	<span  >
	Blake Meyers</span>
	<sup>2</sup>, 

	<span  >
	Jean-Philippe Vielle-Calzada</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Laboratory of Reproductive Development and Apomixis, Langebio and Unidad Irapuato, Cinvestav. Campus Guanajuato, Irapuato M&#233;xico, 

	<sup>2</sup>Delaware Biotechnology Institute, University of Delaware Newark. DE, USA.

</div>

<div>The female gametophyte of <span style="font-style: italic">Arabidopsis thaliana</span> is contained within the ovule and composed of seven cells: 3 antipodals, 2 synergids, the egg cell, and a binucleated central cell. We have developed a microaspirator&#45;based strategy for ovule harvesting that allows the isolation of &#126;12 &#956;g of total RNA in 2 hours. The technology was used to generate large collections of 21 nt mRNA&#45;derived or non&#45;coding RNA&#40;ncRNA&#41;&#45;derived Massively Parallel Signature Sequencing &#40;MPSS&#41; tags. In wild&#45;type ovules, a total of 9,649 genes were detected at significant and reliable expression levels; 1,373 are candidate to be preferentially expressed in the differentiated female gametophyte. To validate these results, we genetically screened 450 T&#45;DNA lines corresponding to insertions in the same number of gene candidate to be specifically expressed in the female gametophyte. The identification of 20 new mutations &#40;&#126;4.4&#37; of lines screened&#41; suggests that our bioinformatic strategy was successful on identifying genes acting at the gametophytic level. Additional reverse&#45;transcription PCR experiments &#40;RT&#45;PCR&#41; and large&#45;scale GUS translational fusions confirmed that many of these genes are specifically expressed in cells of the female gametophyte. The overall results are important to globally analyze the transcriptional universe present in the ovule prior to double fertilization.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetics and Genomics of DNA methylation in Arabidopsis</div>

<div>

	<span  >
	Xiaoyu Zhang</span>
	<sup>1</sup>, 

	<span  >
	Lianna Johnson</span>
	<sup>1</sup>, 

	<span  >
	Ian Henderson</span>
	<sup>1</sup>, 

	<span  >
	Peng Peng</span>
	<sup>1</sup>, 

	<span  >
	Israel Ausin</span>
	<sup>1</sup>, 

	<span  >
	Magnolia Bostick</span>
	<sup>1</sup>, 

	<span  >
	Julie Law</span>
	<sup>1</sup>, 

	<span  >
	Carey Li</span>
	<sup>1</sup>, 

	<span  >
	Yana Bernatavichute</span>
	<sup>1</sup>, 

	<span  >
	Suhua Feng</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Steve Jacobsen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>HHMI/University of California-Los Angeles, Los Angeles, CA, USA

</div>

<div>Our laboratory studies epigenetic gene regulation using <span style="font-style: italic">Arabidopsis thaliana</span> as a model genetic system.  We utilize epigenetic mutations at the <span style="font-style: italic">SUPERMAN</span> and <span style="font-style: italic">FWA genes</span>, which are caused by heritable changes in DNA methylation and chromatin structure of these loci.  These epigenetic mutations allow us to follow the methylation state of the genes by easily scorable morphological phenotypes, and allow for straightforward genetic analysis of the underlying epigenetic silencing.  Using suppressor mutant screening approaches, we are characterizing genes that are required for the <span style="text-decoration: underline">maintenance</span> of DNA methylation and silencing.  These include the DNA methyltransferase CHROMOMETHYLASE3, which appears to be the major enzyme controlling CpNpG methylation in the genome, and KRYPTONITE, a lysine 9 histone H3 protein methyltransferase gene, which is also required for the maintenance of CpNpG methylation.  A third gene, ARGONAUTE4, encodes a type of protein normally found in RNA interference pathways, suggesting that small RNAs target chromatin modifications. We are also screening for mutants specifically blocked in the <span style="text-decoration: underline">initiation</span> of DNA methylation.  This has thus far resulted in the identification of seven genes, including the de novo DNA methyltransferase enzyme DRM2 &#40;the ortholog of mammalian Dnmt3&#41;.  We also discovered that several RNA silencing genes including DICER&#45;LIKE3, RNA DEPENDENT RNA POLYMERASE2, ARGONAUTE4, NRPD1a, NRPD1b, and DRD1 are required to establish DNA methylation, again showing that small RNAs are at the heart of DNA methylation control.  Current work is directed toward understanding the mechanism of action of these proteins in DNA methylation control.  We have also begun genomic studies of DNA methylation, as well as the histone modifications and small RNAs, which are important in DNA methylation control.  These studies are allowing a much broader study of the function of epigenetic modification in wild type, and a much more detailed analysis of mutants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The pam74 mutant is caused by an insertion in a receptor&#45;like kinase gene</div>

<div>

	<span style="text-decoration: underline" >
	Faisal Saeed Awan</span>
	<sup>1</sup>, 

	<span  >
	Anja Schneider</span>
	<sup>2</sup>, 

	<span  >
	Dario Leister</span>
	<sup>2</sup>, 

	<span  >
	Iftikhar Ahmad Khan</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centre of Agricultural Biochemistry and Biotechnology (CABB), University of Agriculture Faisalabad, Pakistan, 

	<sup>2</sup>Ludwig-Maximilians Universitat, Department Biologie I: Botanik, Munich. Germany

</div>

<div>Eight photosynthetic Arabidopsis mutants were screened for co&#45;segregation of a photosynthetic phenotype with the T&#45;DNA insertion. These mutants were selected from out of around 80 photosynthetic Arabidopsis mutants, which were previously identified by the Leister group. These mutants showed a difference in the quantum efficiency of PS11&#40;PSII&#61;&#40;&#61510;Fm&#8217;&#45;Ft&#41;&#47;Fm&#8217;&#41;, therefore they were called pam &#40;photosynthesis affected mutant&#41;. One of these mutants, pam74 was used for further studies, because the T&#45;DNA insertion co&#45;segregated with the mutant phenotype. The homozygous mutant plants either died or turned pale green and showed a stunted growth &#40;loss of function&#41; on MS plates. The mutants could only grow under greenhouse conditions in the heterozygous state. The affected gene in pam74 was identified. Sequence analysis of the flanking regions indicated that a gene has been disrupted coding for a secretory pathway protein and with a tentative function as a receptor&#45;like kinase &#40;RLK&#47;Pelle&#41;, belonging to the LRR&#45;Ia subfamily. We confirmed that the mutant phenotype is caused by disruption of the receptor&#45;like kinase gene by complementing the mutant phenotype with the wild&#45;type gene. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Using Arabidopsis to find new roles for peroxins and peroxisomes</div>

<div>

	<span style="text-decoration: underline" >
	Bonnie Bartel</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Rice University

</div>

<div>Plant peroxisomes carry out diverse functions, including fatty acid beta&#45;oxidation, hydrogen peroxide decomposition, certain steps in photorespiration, and the conversion of indole&#45;3&#45;butyric acid &#40;IBA&#41; to the auxin indole&#45;3&#45;acetic acid.  These single membrane bound organelles contain no DNA; all peroxisomal matrix proteins must be post&#45;translationally imported.  Proteins required for matrix protein import are known as peroxins.  More than 20 yeast peroxins have been identified through genetic screens, and human homologs for many of these peroxins underlie the peroxisome biogenesis disorders.  Although many peroxins playing roles in matrix protein import have been identified, the process is still incompletely understood, and not all proteins involved in matrix protein import in yeast and mammals have identified homologs in Arabidopsis.  We found that IBA resistance provides a facile screen for peroxisome defects in Arabidopsis, and have positionally cloned 12 genes that when disrupted lead to IBA response defects.  We will report on the insights derived from genetic, physiological, and cell biological analyses of these mutants.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">Short silencing RNA networks</div>

<div>

	<span style="text-decoration: underline" >
	David Baulcombe</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Sainsbury Laboratory, John Innes Centre, Norwich, UK

</div>

<div>RNA silencing was first implicated as a defense system against plant viruses. However, from genetic analysis and through the use of high throughput sequencing, it is now clear that this process pervades many aspects of regulation at the RNA and DNA level. I will propose that the short silencing RNAs serve roles as negative switches, as mediators of systems robustness through positive and negative feedback and as initiators of epigenetic mechanisms. Current understanding of silencing RNA mechanisms reveals the potential for complex interaction networks in which the silencing RNAs determine the specificity of the interactions between initiators, nodes and endpoints. &#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic and molecular characterization of inter&#45;subspecific hybrid sterility and wide compatibility in rice</div>

<div>

	<span style="text-decoration: underline" >
	Qifa Zhang</span>
	<sup>1</sup>, 

	<span  >
	Jiongjiong Chen</span>
	<sup>1</sup>, 

	<span  >
	Jihua Ding</span>
	<sup>1</sup>, 

	<span  >
	Yidan Ouyang</span>
	<sup>1</sup>, 

	<span  >
	Jiangyi Yang</span>
	<sup>1</sup>, 

	<span  >
	Huihui Yu</span>
	<sup>1</sup>, 

	<span  >
	Hongyi Du</span>
	<sup>1</sup>, 

	<span  >
	Yunhe Jiang</span>
	<sup>1</sup>, 

	<span  >
	Shuqing Qiu</span>
	<sup>1</sup>, 

	<span  >
	Xiang Song</span>
	<sup>1</sup>, 

	<span  >
	Kede Liu</span>
	<sup>1</sup>, 

	<span  >
	Xianghua Li</span>
	<sup>1</sup>, 

	<span  >
	Caiguo Xu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Key Laboratory of Crop Genetic Improvement, Huazhong Agricultural University, Wuhan, China

</div>

<div>The Asian cultivated rice &#40;Oryza sativa L.&#41; consists of several ecological groups, among which indica and japonica are the two major ones, often referred to as two subspecies. Indica varieties are widely cultivated in tropical and subtropical areas, and japonica cultivars are grown in temperate regions. Although hybrids between the two subspecies frequently show strong heterosis, reduced fertility occurs in most of the hybrids ranging from partial to complete sterility, which has hindered utilization of the inter&#45;subspecific heterosis. Wide compatibility varieties comprise a special germplasm class that produce hybrids with normal fertility when crossed with both indica and japonica subspecies. We have been conducting a systematic study on the genetic and molecular basis of the I&#45;J hybrid sterility and wide compatibility. The results of the cytological study showed that both male and female gamete abortion, and reduced affinity between the uniting gametes all occur in the hybrids, resulting in greatly reduced fertility. QTL analysis identified and confirmed two major genetic loci, S5 and f5, specifying embryo&#45;sac and pollen abortion, respectively. A tri&#45;allelic system was found at each of the loci: an indica allele Si, a japonica allele Sj and a neutral allele &#40;or wide compatibility gene&#41; Sn, such that an individual heterozygous for Si&#47;Sj would show reduced fertility, and all the other genotypes produce normal fertility. Map&#45;based cloning has been conducted to isolate these genes. The gene for S5 has been characterized as encoding an aspartic protease. Current progresses of this work will be presented in the meeting.</div>

</div>


</div>
</div>

</body>
</html>
