
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>
<head>
<title>ICAR 2007 - Metabolism</title>
<link rel="Shortcut Icon" href="/favicon.ico">
<link rel="stylesheet" type="text/css" href="/css/main.css" />
<link rel="stylesheet" type="text/css" href="/css/search.css" />
<link rel="stylesheet" type="text/css" href="/css/base.css" />

<style type="text/css">
@import "/css/page/pagelevels.css";
</style>

<style type="text/css">
.abs {
 margin-top: 30px;
}
.abs div {
 margin: 10px;
}
</style>
</head>
<body>

<div id="container">

<script src="https://ssl.google-analytics.com/urchin.js" type="text/javascript"></script>
<script type="text/javascript">
_uacct = "UA-1945230-1";
urchinTracker();
</script>

 <div id="nav">	
  <div id="logo"><a href="/index.jsp"><img src="/i/logo2.gif" width="87" height="61" alt="tair"/></a></div>
 </div>

<div id="content" class="clearfix">

<div class="abs">

<div style="font-weight:bold">Arabidopsis 1&#45;aminocyclopropane&#45;1&#45;carboxylate Synthase Gene AtACS7 Is a System 2 ACS Gene&#42;</div>

<div>

	<span  >
	Shuang Wu</span>
	<sup>1</sup>, 

	<span  >
	Xuenu Tang</span>
	<sup>1</sup>, 

	<span  >
	Li Chang</span>
	<sup>1</sup>, 

	<span  >
	Ming-Che Shih</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Ning Ning Wang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Biology and Ecology, Nankai University, Tianjin 300071, P. R. China, 

	<sup>2</sup>114 Biology Building, Department of Biological Sciences and Roy J. Carver Center for Comparative Genomics, University of Iowa, Iowa City, Iowa 52242, USA

</div>

<div>1&#45;Aminocyclopropane&#45;1&#45;carboxylate synthase &#40;ACC synthase, ACS&#41; &#40;S&#45;adenosyl&#45;L&#45;methionine methylthioadenosine&#45;lyase; EC 4.4.1.14&#41;, the key enzyme for ethylene biosynthesis, is subjected to positive or negative feedback regulation by the hormone itself.  Here we report, aided by promoter&#45;GUS &#40;&#946;&#45;glucuronidase&#41; reporter approach, that among the multigene family of Arabidopsis ACC synthases, AtACS5 and AtACS7 exhibited active expression in similar localizations but quite different responses to ethylene. The AtACS5::GUS and AtACS7::GUS fusions were found to express in company during the entire life cycle of transgenic Arabidopsis. However, treating the 2&#45;week&#45;old light&#45;grown seedlings with exogenous ethylene greatly increased the GUS activity driven by AtACS7 promoter but exhibited no effect on the expression level of AtACS5::GUS transgene.  The highest expression level of AtACS5::GUS fusion was detected in the 3&#45;d&#45;old etiolated seedlings, and the developmental expression profile of AtACS5 was not affected by the etr1&#45;1 mutation. In contrast, the expression of AtACS7 during the entire life cycle of Arabidopsis, such as in 3&#45;d&#45;old etiolated seedlings, older rosette leaves, fully opened flowers, young and old&#45;yellowing siliques, was dramatically suppressed in the ethylene&#45;insensitive mutant etr1&#45;1. These results strongly suggested that AtACS7 is subjected to the positive feedback regulation by ethylene and plays an important role in catalyzing the system 2 ethylene production while AtACS5 mainly responds to developmental stimuli.&#13;<br/>&#13;<br/>&#42;This work was financially supported by a grant from the Natural Science Foundation of China &#40;NSFC, Project No. 30470174&#41; to Ning Ning Wang and the research funding for the undergraduate students of Nankai University to Shuang Wu.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Evolution effect of ABA on antioxidant enzyme  and lipid  peroxidation in Arabidopsis thaliana  seedling under salt stress&#13;<br/>&#13;<br/></div>

<div>

	<span  >
	Shahram Rashidi</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Rozbeh Farhoudi</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Shahid Chamran University, Ahwaz,Iran, 

	<sup>2</sup>Agronomy Dep., Faculty of Agriculture, Islamic Azad University, Shoushtar Branch, Iran

</div>

<div>&#13;<br/>&#13;<br/>An experiment was carried out to study Arabidopsis thaliana ABA&#45;seed priming effects on antioxidant enzyme content and lipid proxidation of seedlings under salt stress.  Twenty five &#40;25&#41; seeds were planted in each Petri dish &#40;7cm in diameter&#41;. Salt stress was applied at four levels including: 0.3&#40;control&#41; 8, 16 and 24ds&#47;m&#45;1. Sodium chloride &#40;NaCl&#41; was used to apply salt stress. Distilled water was used for control Petri dishes. In order to prime, seeds were soaked in 10&#956;mol ABA solution for 12hrs. The experiment was carried out in terms of factorial &#8211; completely random design with 8 treatments at four replications. Salt stress condition significantly decreased catalase content and increase peroxidase and superoxid dismotase content of seedling tissues but this increase was higher in ABA&#45;priming seedling. ABA application decreased lipid proxidation of seedling tissue. Data of this study showed positive correlation between antioxidant enzyme content of Arabidopsis seedling and decrease of lipid proxidation of Arabidopsis tissue.&#13;<br/>&#13;<br/>Key word: Arabidopsis thaliana, salt stress, ABA, antioxidant enzyme. &#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A novel Arabidopsis MAPK phosphatase, MKP5, regulates ABA&#45;signaling and lipid mobilization in seedling establishment</div>

<div>

	<span style="text-decoration: underline" >
	Montserrat Pages</span>
	<sup>1</sup>, 

	<span  >
	Victoria Lumbreras</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CSIC. Barcelona.Spain

</div>

<div>The phytohormone abscisic acid &#40;ABA&#41; plays an essential role in adaptative stress responses. Although genetic analyses have identified numerous ABA&#45;response mutants, the regulators of ABA signal transduction remain to be fully elucidated. Here, we report the identification and characterization of a novel MAPK phosphatase &#40;MKP5&#41; from Arabidopsis involved in developmental and stress responses. The mkp5 gene is expressed at very low levels in vegetative tissues, but is markedly induced during seedling establishment. Seeds expressing an MKP5&#45;GFP fusion protein show phenotypic alterations, including ABA and salt hypersensitivity and a severe growth arrest after germination. This developmental arrest is a result from inefficient lipid reserve mobilization and is rescued by providing a source of exogenous sugar in the growth medium. This phenotype is not observed after overexpression of the normal MKP5 protein, suggesting that it arises from dominant negative effects of the MKP5&#45;GFP protein, confirmed by a hypomorphic mkp5 allel. We also find that the constitutive MKP5 overexpression results in ABA and salt hypersensitivity and slight enhanced drought tolerance. Molecular analyses reveal upregulation of ABA signaling genes in those plants. The spatial expression pattern and protein MKP5 accumulation is consistent with their functional role. MKP5&#45;GFP shows specific accumulation in tissues involved in water homeostasis, hydathodes and guard cells. Taken together, these data suggest that MKP5 is a regulator of ABA signaling pathway and present new evidence implicating a link by MAPK activities in the control of plant development and survival under stress conditions.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The FORKED1 gene is expressed in preprocambium and responds to auxin&#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	Hongwei Hou</span>
	<sup>1</sup>, 

	<span  >
	Elizabeth Schultz</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, University of Lethbridge, Alberta, Canada

</div>

<div>We have previously described plants mutant for FORKED1 &#40;FKD1&#41;, which have reduced leaf response to auxin and show lack of distal vein meeting in leaves. Our analysis of fkd1&#45;1 has allowed us to propose a model whereby FKD1 is necessary for the auxin response that directs vein patterning. We have now identified FKD1 through map&#45;based cloning. Several SALK lines, all corresponding to insertions in a single gene, had fkd1&#45;1&#45;like phenotypes and did not complement fkd1&#45;1. Transformation of fkd1&#45;1 plants with the wild type FKD1 gene resulted in a reversion to the wild type phenotype, indicating that we have indeed identified the FKD1 gene. FKD1 encodes a hypothetical protein with a pleckstrin homology &#40;PH&#41; domain, suggesting the involvement of FKD1 in IP3 signal transduction. We fused the region 1.8kb upstream of FKD1 to the reporter gene &#946; &#8211;glucuronidase &#40;PFKD1::GUS&#41; and compared expression to PAthB8::GUS, a pre&#45;procambial marker and to PDR5::GUS, a marker of auxin response. In leaves, PFKD1::GUS and PAthB8::GUS are expressed at the same time in the tip. Whereas AthB8::GUS expression in secondary veins progresses primarily acropetally, PFKD1::GUS progresses basipetally. To test if FKD1 is responsive to changes in auxin, we treated seedlings with auxin transport inhibitors NPA or NOA or with the synthetic auxin 2, 4&#45;D. In all cases, PFKD1::GUS expands coincident with PDR5::GUS in leaves. During embryogenesis, PFKD1::GUS expression is first evident in late torpedo stage at the cotyledon tip and progresses basipetally, and then in secondary vein loops. This expression is similar to that of DR5::GUS, although unlike PDR5::GUS, PFKD1::GUS is not expressed in the embryonic root. In contrast, PAthB8::GUS begins earlier, in heart stage embryos, in cells anticipating veins in root and shoot. In roots of seedlings, PFKD1::GUS is expressed in vascular tissue and in vascular tissue precursors. As in leaves, treatment with NPA, NOA or 2, 4&#45;D alters PFKD1::GUS root expression. The results indicate that FKD1 is a pre&#45;procambial cell marker, whose expression is related to auxin response.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Multiple phosphate starvation responses are modulated by GA signaling in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Xiangdong Fu</span>
	<sup>1</sup>, 

	<span  >
	Caifu Jiang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing 100101, P R China

</div>

<div>Phosphate &#40;Pi&#41; is an essential macronutrient for plant growth, development and reproduction. Although the total amount of phosphorus in the soil maybe high, Pi is unavailable for plants uptake because of its low mobility. Plants have developed a number of adaptive physiological and biochemical strategies to cope with low Pi availability. Here we report that phosphate starvation induced responses are in part dependent on a family of nuclear growth&#45;repressing DELLA proteins, the central components in GA signaling in Arabidopsis. We show that Pi deficiency promotes the GFP&#45;tagged DELLA &#40;GFP&#45;RGA&#41; accumulation in root nuclei but does not delay GA&#45;mediated 26S proteasome&#45;dependent DELLAs destruction. We show that Pi starvation induced multiple responses including root architecture, plants growth, anthocyanin induction and late flowering can be repressed by exogenous GA treatment. Lacking quadruple DELLA proteins in Arabidopsis gai&#45;t6 rga&#45;t2 rgl1&#45;1 rgl2&#45;1 mutant, also results in Pi starvation insensitivity. Whereas, enhanced DELLA functions, such as ga1&#45;3 mutant, cause the hypersensitive responses under Pi starvation, suggesting that DELLA&#45;mediated signaling is essential for Pi starvation responses in plant. Furthermore, these changes during phosphate starvation are paralleled by alterations of bioactive GA level by increases of GA&#45;2ox2 transcripts and decreases of both GA&#45;20ox1 and GA3ox1 transcripts. In addition, we investigate the role of GA in formation of root hairs. GA&#45;deficient ga1&#45;3 mutant was significant decreases in the length of root hairs but not frequency. However, it can be restored by exogenous GA treatment or lacking DELLA functions, suggesting that appropriate GA level is essential for root hairs development. These results demonstrate that down&#45;regulation of active GA by low Pi availability is one of adaptively significant mechanism via which DELLAs modulating multiple phosphate starvation responses.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Roles of RAV2 and ERF11 Arabidopsis Transcriptions Factors in Response to ABA and Glucose.</div>

<div>

	<span  >
	Luis De Luna</span>
	<sup>1</sup>, 

	<span  >
	Patricia Le&#243;n</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Angel Guevara-Garcia</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Departamento de Biolog&#237;a Molecular de Plantas, Instituto de

</div>

<div>&#34;Arabidopsis&#34; dedicates over 6&#37; of its genome to code for more than 1700 transcription factors, but the elucidation of the specific roles that each of them perform in different plant processes is incipient. RAV2 is a singular transcription factor that contains both an AP2 and a B3 binding domains, involved in flower development and abscisic acid &#40;ABA&#41; responses, respectively. Although RAV2 could be considered to regulate a wider range of genes, neither its target genes nor physiological functions are currently known. ERF11  is member of a large family of transcriptional regulators with important functions in a variety of plant processes related to growth development and responses to environmental stimuli. Interestingly, ERF11 contains an ERF&#45;associated repression motif &#40;EAR&#41;, which functions as a repression domain. Recently, ERF4, another member of the small subfamily of ERF repressors, has been involved in modulate ABA responses. However, no information is available with respect to the physiological roles and downstream targets of ERF repressors in plants. Both &#34;RAV2&#34; and &#34;ERF11&#34; genes were identified to be induced in a stress&#47;defense transcriptome of &#34;Arabidopsis&#34;.<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ABA is a plant hormone that mediates myriad physiological responses to the environment such as drought, chilling, salinity, etc., and plays fundamental roles in seed and vegetative development. Glucose plays an important role as carbon source or plant growth and development, but also as signal to regulate metabolism, differentiation and stress responses. The co&#45;regulation of several plant genes by both glucose and ABA has been reported, but the information about the molecular basis of that interaction is fragmentary.<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;In this study, the participation of &#34;RAV2&#34; and &#34;ERF11&#34; in ABA and glucose responses is investigated. To reach that goal, we compare the phenotype of &#34;A. thaliana&#34; transgenic lines with high &#40;over&#45;express&#41; and low &#40;RNAi&#41; levels of expression of both genes, growing in ABA y glucose. Moreover, our study is complemented with an analysis of expression of &#34;RAV2&#34; and &#34;ERF11&#34; genes in response to ABA and glucose on wild&#45;type plants in different tissues and developmental stages.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Systemic and Intracellular Responses to High Light&#45;Induced Oxidative Stress</div>

<div>

	<span  >
	Dawar Hussain</span>
	<sup>1</sup>, 

	<span  >
	Pip Wilson</span>
	<sup>1</sup>, 

	<span  >
	Jan Bart Rossel</span>
	<sup>1</sup>, 

	<span  >
	Osman Mewett</span>
	<sup>1</sup>, 

	<span  >
	Kate Howell</span>
	<sup>2</sup>, 

	<span  >
	Jim Whelan</span>
	<sup>2</sup>, 

	<span  >
	Kemal Kazan</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Barry Pogson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ARC Centre of Excellence in Plant Energy Biology, Australian National University, 

	<sup>2</sup>ARC Centre of Excellence in Plant Energy Biology, University of Western Australia, 

	<sup>3</sup>CSIRO Plant Industry, Australia

</div>

<div>Since plants are unable to avoid oxidative damage caused by excess light they employ a repertoire of photoprotective measures which include minimizing light absorption, minimizing ROS over&#45;accumulation, and repairing damaged proteins, lipids and photosystems. A large suite of genes are induced by excess light and many encode proteins that function in the aforementioned processes. We have been investigating the function of the transcription factor ZAT10 in modulating responses to oxidative stress. ZAT10 participates in the regulation of reactive oxygen species homeostasis, photosynthetic rates and multiple antioxidative processes in different organelles. Additionally, we have used ZAT10 promoter:LUCIFERASE fusions to study systemic acquired acclimation &#40;SAA&#41;. SAA is a novel systemic signal reported by Karpinski and colleagues &#40;Karpinski et al., 1999&#41; that resulted in the induction of ascorbate peroxidise 2 &#40;APX2&#41; gene expression in shaded leaves of a plant partially exposed to excess light. We have demonstrated that this signal can move substantial distances, namely from rosette to cauline leaves, and that several hundred genes are induced within 30 minutes of exposure to high light. Analysis of the arrays by clustering to stress and hormone responsive experiments and testing the induction of the systemic response in hormonal and other signalling mutants has provided insight into which hormones and processes may participate in the systemic signalling of oxidative stress.&#13;<br/>&#13;<br/>Karpinski, S., Reynolds, H., Karpinska, B., Wingsle, G., Creissen, G., and Mullineaux, P. &#40;1999&#41;. Systemic signaling and acclimation in response to excess excitation energy in Arabidopsis. Science 284, 654&#45;657.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Brassinosteroids Regulate Organ Boundary Formation and Organ Separation in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Joshua Gendron</span>
	<sup>1,2</sup>, 

	<span  >
	Nathan Gendron</span>
	<sup>2</sup>, 

	<span  >
	Yu Sun</span>
	<sup>2</sup>, 

	<span  >
	Srinivas Gampala</span>
	<sup>2</sup>, 

	<span  >
	Dongmei Cao</span>
	<sup>3</sup>, 

	<span  >
	Kang Chong</span>
	<sup>3</sup>, 

	<span  >
	Zhi-Yong Wang</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Stanford University, 

	<sup>2</sup>Carnegie Institution, Dept. of Plant Biology, 

	<sup>3</sup>Institute of Botany, Chinese Academy of Science

</div>

<div>Much is known about how brassinosteroids &#40;BRs&#41; are made, perceived, and transduced as a signal, and it is well established that BRs play a major role in cell expansion. However, little is known about whether or how BRs affect development outside of this role in cell expansion.  In this work we show that BRs are involved in establishment of organ boundaries.  Organ fusion phenotypes were first noticed in the <span style="font-style: italic">bzr1&#45;1D</span> mutant, which contains a gain of function mutation in a transcription factor that acts as a positive regulator of BR signaling.  <span style="font-style: italic">bzr1&#45;1D</span> shows kinking of the main stem towards the lateral branches, and scanning electron microscopy revealed an abnormal fusion of the cauline leaf to the stem.  Furthermore, fusions are observed in the <span style="font-style: italic">bzr1&#45;1D</span> mutant between floral organs within and between whorls.  Specifically, <span style="font-style: italic">bzr1&#45;1D</span> contains stamen&#45;stamen and stamen&#45;carpel fusions, the latter resulting in bent siliques.  Also the <span style="font-style: italic">bzr1&#45;1D</span> mutant shows fused cotyledons similar to those seen in <span style="font-style: italic">cuc</span> and <span style="font-style: italic">pin</span> mutants.  Brassinolide &#40;BL&#41; treatment of wildtype plants leads to stamen&#45;stamen fusion, and decreased BR biosynthesis or signaling leads to the appearance of extra valves and replums. A <span style="font-style: italic">DWF4p</span>::GUS transgenic line shows that the BR biosynthesis gene, <span style="font-style: italic">DWF4</span>, is expressed in newly forming organ primordia.  A GFP tagged wildtype BZR1 protein accumulates in the nuclei of cells of newly forming organ primordia, and the mutant BZR1 protein accumulates highly in all areas of the meristem and organ primordia, suggesting that restricted localization of BZR1 is required for proper organ separation.  Using genetics, semi&#45;quantitative PCR, and chromatin immunoprecipitation we are investigating the interactions between BRs and the auxin and cup&#45;shaped&#45;cotyledon pathways, which both regulate organ fusion.  These results demonstrate that BRs not only determine plant size through regulation of cell expansion but are also involved in developmental patterning and organ separation. </div>

</div>





<div class="abs">

<div style="font-weight:bold">SBI1 regulates BRI1 level to control BR response</div>

<div>

	<span style="text-decoration: underline" >
	Guang Wu</span>
	<sup>1,2,3</sup>, 

	<span  >
	Yuji Kamiya</span>
	<sup>3</sup>, 

	<span  >
	Joanne Chory</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Shangdong Agricultural University, Taian, PRC, 

	<sup>2</sup>Salk Institute , Jolla,  USA, 

	<sup>3</sup>RIKEN, Yokohama, Japan

</div>

<div>Brassinosteroids &#40;BRs&#41;play important roles in plant growth and development. Over the past decade, studies have been focusing on finding new components in on BR signaling pathway. As such, the several key components in BR signaling pathway have been identified, including receptor BRI1 and co&#45;receptor BAK1, and downstream components BKI1, BIN2, BES1 and BZR1. BRs are perceived by BRI1, which inactivates a negative regulator, BIN2, a GSK3&#45;like kinase, which then dephosphorylates and activates a family of plant&#45;specific transcription factors &#40;such as BES1 and BZR1&#41; to regulate the expression of many BR responsive genes. The loss of function <span style="font-style: italic">bri1</span> mutant is similar to the strongest BR biosynthetic mutant, indicating that BRI1 is the master switch for BR function. However, we know little about how this master switch, BRI1, is regulated.  <br/> We used a genetic approach to identify regulators of BRI1 using an unstable <span style="font-style: italic">bri1</span> mutant, a weak <span style="font-style: italic">bri1</span> mutant allele that accumulates less bri1 protein. We screened for suppressors of this <span style="font-style: italic">bri1</span> mutant and found a BR signaling specific mutant &#40;<span style="font-style: italic">sbi1</span>, suppressor of <span style="font-style: italic">bri1</span>&#41; that accumulates a high level of BRI1 protein. <span style="font-style: italic">sbi1</span> does not suppress other <span style="font-style: italic">bri1</span> alleles in which the protein level is not altered. Nevertheless, the <span style="font-style: italic">sbi1</span> single mutant has an elevated BRI1 protein level and has an enhanced response to BL and reduced sensitivity to BR inhibitors. The increased accumulation of BRI1 protein may explain the basis for its suppression of <span style="font-style: italic">bri1</span> and the enhanced growth in <span style="font-style: italic">sbi1</span> single mutant. Interestingly, the expression of <span style="font-style: italic">BRI1::bri1</span> in the <span style="font-style: italic">bri1</span> mutant had similar phenotype as <span style="font-style: italic">sbi1&#47;bri1</span> double mutant, suggesting that the BRI1&#47;bri1 protein accumulation in <span style="font-style: italic">sbi1</span> single mutant and <span style="font-style: italic">sbi1&#47;bri1</span> double mutant is responsible for their respective enhanced response to BL compared to wild type and <span style="font-style: italic">bri1</span>  respectively. Furthermore, the treatment of <span style="font-style: italic">bri1</span> mutant with a protease inhibitor leads to a similar accumulation of BRI1 protein as in the <span style="font-style: italic">sbi1&#47;bri1</span> double mutant. Therefore,SBI1 plays an important role in BR signaling by regulating BRI1 levels.<br/><br/><br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Studies on the role of phytohormones and glucosinolates on sphinganine&#45;analog mycotoxins&#45;induced programmed cell death in Arabidopsis and tomato</div>

<div>

	<span  >
	Jiansheng Wang</span>
	<sup>1</sup>, 

	<span  >
	Chengguo Jia</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Qiaomei Wang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Horticulture, Zhejiang University, The State Agriculture Ministry,Laboratory of Horticultural Plant Growth, Development &amp;  Biotechnology, Hangzhou 310029, China

</div>

<div>Fumonisin and AAL toxin are two common sphinganine&#45;analog mycotoxins &#40;SAMs&#41; produced by <span style="font-style: italic">Fusarium moniliforme</span> in maize and <span style="font-style: italic">Alternaria alternata</span> f. sp. <span style="font-style: italic">Lycopersici</span> in tomato, respectively. SAMs induce apoptosis&#45;like programmed cell death &#40;PCD&#41; in plants. We have established model systems to study plant PCD in Arabidopsis and susceptible lines of tomato &#40;<span style="font-style: italic">asc&#47;asc</span>&#41; using FB1 and AAL toxin. The effects of brassinosteroids &#40;BRs&#41;, salicylic acid &#40;SA&#41;, and jasmonate &#40;JA&#41; on SAM&#45;induced PCD were investigated in Arabidopsis and tomato by hormone application and hormone biosynthesis and response mutant experiments. Our results showed that the role of SA in SAM&#45;induced PCD is distinct between Arabidopsis and tomato, SA accelerated FB1&#45;induced lesion formation in Arabidopsis leaves, while suppressed AAL toxin&#45;induced PCD in tomato &#40;<span style="font-style: italic">asc&#47;asc</span>&#41; leaves. Brassinolide &#40;BL&#41; could decrease the susceptibility of both Arabidopsis and tomato to SAM with an unknown mechanism; the cross&#45;talk of JA&#47;ETH and SA in SAM&#45;induced PCD was also discussed.&#13;<br/>&#13;<br/>Glucosinolates are a kind of sulfur&#45; and nitrogen&#45;containing secondary metabolites that have diverse biological functions. In <span style="font-style: italic">Arabidopsis thaliana</span>, 23 different glucosinolates have been identified, which makes Arabidopsis a model system to study the biosynthesis, metabolism and biological functions of glucosinolates. The hormone regulation of glucosinolates in Arabidopsis ecotypes &#40;Col, Ler&#41; and glucosinolate mutants &#40;<span style="font-style: italic">Tu3</span>, <span style="font-style: italic">gcc8</span>&#41; differing in glucosinolate composition, contents, and their roles in FB1&#45;induced PCD in Arabidopsis were studied, and the results indicated that the biosynthesis of individual glucosinolates was under fine regulation of different phytohormones, the cross&#45;talk of phytohormone and glucosinolate occured at the level of biosynthesis and functions in the resistance of Arabidopsis to FB1.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Crosstalk between Auxin and Brassinosteroids is via AuxIAA and BES1&#47;BZR1&#45;mediated regulation of the <span style="font-style: italic">DWF4</span> gene</div>

<div>

	<span  >
	Ok Sun Lee</span>
	<sup>1</sup>, 

	<span  >
	Su Youn Jang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Sunghwa Choe</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Seoul National University

</div>

<div>DWARF4 catalyzes a flux&#45;determining step in the Brassinosteroid biosynthetic pathways and its expression is subjected to a feedback&#45;downregulation by BR treatment. However the molecular mechanisms for this regulation has remained unclear. To elucidate this, DWF4&#45;driven GUS expression was studied after various genetic and hormone treatments. <span style="font-style: italic">DWF4::GUS</span> expression was up&#45;regulated in the BR&#45;deficient dwarf mutants, whereas the change was not noticeable or even weaker in <span style="font-style: italic">axr2</span> and <span style="font-style: italic">axr6</span> background. In addition, the <span style="font-style: italic">DWF4::GUS</span> expression was down&#45;regulated in the brassinazole&#45;resistant mutant <span style="font-style: italic">bzr1&#45;D</span>. In contrast, <span style="font-style: italic">DWF4::GUS</span> expression was significantly up&#45;regulated by 2,4&#45;D treatment, especially in the <span style="font-style: italic">bes1&#45;D</span> mutant background, suggesting that auxin may have provided additional room for BES1 action. Interestingly, the auxin&#45;mediated up&#45;regulation of the <span style="font-style: italic">DWF4::GUS</span> expression was independent of both BRI1 and BIN2 functions, suggesting that auxin directly acts on the <span style="font-style: italic">DWF4</span> promoter. Chromatin Immunoprecipitation &#40;ChIP&#41; analysis revealed that BES1 directly binds to the <span style="font-style: italic">DWF4</span> promoter regardless of BL and 2,4,&#45;D treatments. However, BZR1 interaction to <span style="font-style: italic">DWF4</span> promoter was abolished by auxin, suggesting that auxin&#45;mediated increase of <span style="font-style: italic">DWF4</span> be due to de&#45;repression mechanisms. Collectively, it is obvious that <span style="font-style: italic">DWF4</span> promoter serves as a focal point in a cross talk between BR and auxin, and this regulation is mediated by transcriptional activator BES1.</div>

</div>





<div class="abs">

<div style="font-weight:bold">KEG, a Novel RING E3&#47;Kinase&#47;Ankyrin Protein Essential for Arabidopsis Growth and Development is Involved in Abscisic Acid Signaling</div>

<div>

	<span style="text-decoration: underline" >
	Sophia Stone</span>
	<sup>1</sup>, 

	<span  >
	Luis Williams</span>
	<sup>2</sup>, 

	<span  >
	Lisa Farmer</span>
	<sup>3</sup>, 

	<span  >
	Richard Vierstra</span>
	<sup>3</sup>, 

	<span  >
	Judy Callis</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Dalhousie University, Halifax, Nova Scotia, Canada, 

	<sup>2</sup>University of California-Davis, Davis, California, USA, 

	<sup>3</sup>University of Wisconsin, Madison, Wisconsin, USA

</div>

<div>Ubiquitin&#45;dependent protein degradation regulates numerous eukaryotic cellular processes by selectively targeting proteins for degradation by the 26S proteasome. Recent studies have highlighted the role of ubiquitination, specifically E3 ligases, in plant growth and development, hormone signaling and photomorphogenesis. The Arabidopsis proteome contains 469 RING&#45;type E3 ligases, which are further classified into 30 different sub&#45;groups based on domain organization. Of particular interest is the KEG protein, which belongs to the RING&#45;ankyrin sub&#45;group. In addition to the RING&#45;HCa domain and nine ankyrin repeats, KEG also contains a serine&#47;threonine kinase domain as well as a twelve previously unidentified HERC2&#45;like repeats. Both the RING and kinase domains were functional in <span style="font-style: italic">in vitro</span> ubiquitination and phosphorylation assays, respectively. To determine the function of KEG we analyzed four independent T&#45;DNA insertional lines. <span style="font-style: italic">keg</span> seedlings exhibit characteristics of post&#45;germinative growth arrest, such as inhibited cotyledon greening and expansion, indicative of elevated abscisic acid &#40;ABA&#41; signaling, a major phytohormone that plays a key role in plant development and survival under unfavorable conditions. As expected, <span style="font-style: italic">keg</span> seedlings are hypersensitive to exogenous sugars and <span style="font-style: italic">keg</span> roots are extremely sensitive to the inhibitory effects of ABA on root growth. The observations that KEG accumulates high levels of ABSCISIC ACID&#45;INSENSITIVE 5 &#40;ABI5&#41; without exogenous ABA and that loss of ABI5 rescues the growth&#45;arrest phenotype of <span style="font-style: italic">keg</span> seedlings point to a central role for KEG in ABA signaling. Future studies include functional characterization of the remaining RING&#45;ankyrin proteins and other novel plant specific RING E3 ligases.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The <span style="font-style: italic">sua</span> mutant is an allele specific suppressor of <span style="font-style: italic">abi3&#45;5</span></div>

<div>

	<span style="text-decoration: underline" >
	Matteo Sugliani</span>
	<sup>1</sup>, 

	<span  >
	Maarten Koornneef</span>
	<sup>1</sup>, 

	<span  >
	Wim Soppe</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research

</div>

<div>The <span style="font-style: italic">ABSCISIC ACID INSENSITIVE 3</span> &#40;<span style="font-style: italic">ABI3</span>&#41; transcription factor has an essential role during seed maturation for the acquisition of desiccation tolerance and dormancy. Arabidopsis plants carrying <span style="font-style: italic">abi3</span> mutant alleles, including <span style="font-style: italic">abi3&#45;5</span>, are disturbed in seed maturation processes and yield seeds with reduced longevity. In a suppressor mutagenesis screen of the <span style="font-style: italic">abi3&#45;5</span> mutant we have obtained the <span style="font-style: italic">suppressor of abi3&#45;5</span> &#40;<span style="font-style: italic">sua</span>&#41; mutant, which reverts all of the <span style="font-style: italic">abi3&#45;5</span> mutant phenotypes. Map based cloning revealed that <span style="font-style: italic">SUA</span> encodes an RNA binding protein. In the wild type background the <span style="font-style: italic">sua</span> mutation causes a higher sensitivity to abscisic acid during germination, which also correlates with higher dormancy. Interestingly, <span style="font-style: italic">sua</span> only reverts the phenotype of the <span style="font-style: italic">abi3&#45;5</span> allele but does not affect other <span style="font-style: italic">abi3</span> alleles that we tested. We are currently analyzing the relationship between <span style="font-style: italic">sua</span> and <span style="font-style: italic">abi3&#45;5</span> and are determining the role of <span style="font-style: italic">SUA</span> during seed maturation. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Investigating the role of ETA2&#47;CAND1 in regulating SCF complex activity</div>

<div>

	<span style="text-decoration: underline" >
	Wenjing Zhang</span>
	<sup>1</sup>, 

	<span  >
	Hironori Ito</span>
	<sup>1</sup>, 

	<span  >
	William M. Gray</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Minnesota, TwinCities, MN, USA

</div>

<div>The <span style="font-style: italic">eta2&#45;1</span> mutant was identified in a genetic screen for enhancers of the <span style="font-style: italic">tir1&#45;1</span> auxin &#40;eta&#41; response defect. <span style="font-style: italic">eta2&#45;1</span> plants exhibit several phenotypes related to impaired auxin response. Molecular studies found that these phenotypes are the result of reduced SCF&#40;TIR1&#41; activity in <span style="font-style: italic">eta2&#45;1</span> mutants. Isolation of the ETA2 gene revealed that it encodes an Arabidopsis ortholog of human CAND1 &#40;Cullin&#45;Associated and Neddylation&#45;Dissociated&#41;.  Biochemical studies with mammalian cell lines suggest that CAND1 acts as a negative regulator of SCF function by sequestering unmodified CUL1 away from SKP1 and the F&#45;box protein, thus preventing assembly of the SCF complex.  In contrast, we find that the <span style="font-style: italic">eta2&#45;1</span> mutation diminishes the ability of CAND1 to interact with CUL1, demonstrating that the interaction between these two proteins is required for SCF activity and that CAND1 positively regulates SCF function.  These paradoxical findings have been explained by a model invoking CAND1 regulation of a dynamic cycle of assembly and disassembly of the SCF complex in vivo, through association and dissociation with CUL1. Double mutant analysis with the <span style="font-style: italic">axr6&#45;2</span> and <span style="font-style: italic">axr6&#45;3</span> alleles of CUL1 reveals additional insight into the interactions between CAND1 and CUL1.  Whereas <span style="font-style: italic">eta2&#45;1</span> and <span style="font-style: italic">axr6&#45;3</span> interact synergistically, <span style="font-style: italic">eta2&#45;1</span> <span style="font-style: italic">axr6&#45;2</span> double mutants show mutual suppression of <span style="font-style: italic">eta2&#45;1</span> and <span style="font-style: italic">axr6&#45;2</span>. Although the <span style="font-style: italic">eta2&#45;1</span> mutation itself is recessive, its suppression of <span style="font-style: italic">axr6&#45;2</span> is dominant, indicating a heightened sensitivity to CAND1 dosage level. This genetic study, together with co&#45;immunoprecipitation experiments suggests that the <span style="font-style: italic">axr6&#45;2</span> mutation may inhibit dissociation of CUL1 from the CAND1&#45;CUL1 complex. The result that the <span style="font-style: italic">eta2&#45;1</span> mutation suppresses the rub&#45;modification defect in <span style="font-style: italic">axr6&#45;2</span> implies that in order to be modified by rub CUL1 needs to be dissociated from CAND1 first.  Gel&#45;filtration experiments show some surprising differences in the fractionation patterns of SCF subunits between <span style="font-style: italic">eta2&#45;1</span> mutants and wild&#45;type, shedding new light on CAND1&#8217;s function. Molecular studies examining the effects of these mutations on SCF homeostasis are underway.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cross&#45;talk between SA&#45; and JA&#45;dependent defense pathways coincides with a cellular increase in glutathione</div>

<div>

	<span style="text-decoration: underline" >
	Annemart Koornneef</span>
	<sup>1</sup>, 

	<span  >
	Tita Ritsema</span>
	<sup>1</sup>, 

	<span  >
	Floor C. Den Otter</span>
	<sup>1</sup>, 

	<span  >
	Adriaan Verhage</span>
	<sup>1</sup>, 

	<span  >
	L.C. Van Loon</span>
	<sup>1</sup>, 

	<span  >
	Corn&#233; M.J. Pieterse</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant-Microbe Interactions, Institute of Environmental Biology, Utrecht University, P.O.Box 800.84, 3508 TB Utrecht, the Netherlands

</div>

<div>The signaling hormones salicylic acid &#40;SA&#41; and jasmonic acid &#40;JA&#41; are important molecules that mediate inducible defense mechanisms in Arabidopsis. They each activate a differential set of defense&#45;related genes, which are effective against a variety of pathogens and&#47;or insects. &#13;<br/>SA has an antagonistic effect on JA signaling, a process called pathway cross&#45;talk, which allows the plant to fine&#45;tune its defense response. Analysis of mutant <span style="font-style: italic">npr1&#45;1</span>, which is impaired in SA&#45;dependent systemic acquired resistance &#40;SAR&#41;, revealed that the down&#45;regulation of JA&#45;responsive gene expression requires the regulatory protein NPR1.&#13;<br/>Cross&#45;talk was studied by applying the chemicals SA and methyl jasmonate &#40;MeJA&#41;. Simultaneous treatment with these chemicals within hours results in down&#45;regulation of JA&#45;responsive <span style="font-style: italic">PDF1.2</span> expression, and lasts up to several days.&#13;<br/>However, in nature, Arabidopsis might trigger SA&#45; and JA&#45;dependent defenses at different time&#45;points. Therefore, we applied SA at several time&#45;intervals before application of MeJA, and monitored <span style="font-style: italic">PDF1.2</span> expression. SA appeared to have a &#8216;window of opportunity&#8217; to suppress <span style="font-style: italic">PDF1.2</span>, which ranges between 0 and 30 hours before MeJA application. SA treatment more than 30 hours before MeJA did not trigger cross&#45;talk, even though SA&#45;responsive <span style="font-style: italic">PR&#45;1</span> expression was still detectable. Thus, SA triggers a transient response that mediates suppression <span style="font-style: italic">PDF1.2</span> gene expression. &#13;<br/>Since induction of SAR and activation of NPR1 require a change in the redox state of the cell, we hypothesized that the redox state might be involved in this transient response. Therefore, we monitored the total levels of glutathione, a major redox buffer in plant tissue, upon application of SA. SA triggered a transient increase in glutathione levels, which returned to baseline after approximately 30 hours. Interestingly, this transient glutathione peak correlated with the time&#45;frame in which SA was capable of suppressing <span style="font-style: italic">PDF1.2</span>.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Cross&#45;talk between salicylate and jasmonate signaling in Arabidopsis does not require inhibition of octadecanoid biosynthesis</div>

<div>

	<span style="text-decoration: underline" >
	Antonio L&#233;on Reyes</span>
	<sup>1</sup>, 

	<span  >
	Elvira S. De Lange</span>
	<sup>1</sup>, 

	<span  >
	Annemart Koornneef</span>
	<sup>1</sup>, 

	<span  >
	Floor C. Den Otter</span>
	<sup>1</sup>, 

	<span  >
	Ruth G. Joosten</span>
	<sup>1</sup>, 

	<span  >
	Tita Ritsema</span>
	<sup>1</sup>, 

	<span  >
	Corn&#233; M.J. Pieterse</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant-Microbe Interactions, Institute of Environmental Biology, Utrecht University, P.O. Box 800.84, 3508 TB Utrecht, the Netherlands

</div>

<div>Salicylic Acid &#40;SA&#41; and jasmonates &#40;JA&#41; are plant hormones implicated in disease resistance. Interaction between hormone signaling pathways plays a role in determining the final outcome of defense responses in plants. In <span style="font-style: italic">Arabidopsis thaliana</span>, endogenously accumulating SA antagonizes JA&#45;dependent defenses by prioritizing SA&#45;dependent resistance over JA&#45;dependent defense. In this study, by looking at the expression of the JA&#45;responsive maker genes <span style="font-style: italic">PDF1.2</span> and <span style="font-style: italic">VSP2</span>, and the SA&#45;responsive marker gene <span style="font-style: italic">PR&#45;1</span>, we show that SA is capable of suppressing the JA response that is triggered by an array of JA&#45;inducing pathogens, insects and wounding.  Furthermore, we tested whether SA&#45;mediated inhibition of JA biosynthesis plays a role in cross&#45;talk between these two hormones. SA antagonized the expression of <span style="font-style: italic">PDF1.2</span> in the same time frame as it suppressed genes encoding enzymes of the JA biosynthesis pathway, such as <span style="font-style: italic">AOS</span>, <span style="font-style: italic">OPR3</span>, <span style="font-style: italic">LOX2</span> and <span style="font-style: italic">AOC3</span>. If the antagonistic effect of SA on JA signaling functions through inhibition of JA biosynthesis, then JA&#45;biosynthesis mutants should no longer display cross&#45;talk. Mutant <span style="font-style: italic">aos&#47;dde2</span> appeared to be a suitable mutant to test this hypothesis. Mutant <span style="font-style: italic">aos&#47;dde2</span> was incapable of expressing JA&#45;responsive genes such as <span style="font-style: italic">PDF1.2</span> upon infection by the JA&#45;inducing pathogen <span style="font-style: italic">Alternaria brassicicola</span>, whereas exogenous application of methyl jasmonate &#40;MeJA&#41; resulted in wild&#45;type levels of <span style="font-style: italic">PDF1.2</span> transcripts.  Pharmacological assays showed that cross&#45;talk between SA and JA is still intact in the <span style="font-style: italic">aos&#47;dde2</span> mutant. This demonstrates that down&#45;regulation of JA biosynthesis is not necessary for SA&#45;mediated suppression of JA responses. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Comparative investigation of the glucosinolate&#45;myrosinase defense system in Arabidospis hypocotyls and cell suspensions cultures</div>

<div>

	<span  >
	Sophie Alvarez</span>
	<sup>1</sup>, 

	<span  >
	Yan He</span>
	<sup>1</sup>, 

	<span  >
	Emily Wang</span>
	<sup>1</sup>, 

	<span  >
	Xiu-Feng Yan</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Sixue Chen</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>University of Florida, Gainesville, FL32605, USA, 

	<sup>2</sup>Northeast Forestry University, Harbin 150080, China

</div>

<div>Glucosinolates are secondary metabolites, derived from amino acids, found in many plants of the order Capparales in which Arabidospsis thaliana belongs. Upon hydrolysis by myrosinases, they produce a variety of biologically active compounds, such as isothiocyanates which showed fungicidal, nematocidal and herbicidal effects. In this study, the Glucosinolate&#45;Myrosinase Defense System &#40;GMDS&#41; was studied in Arabidopsis suspensions cells and intact tissues treated with methyl jasmonate &#40;MeJA&#41;, a signaling compound induced by herbivore attack and wounding to initiate plant defense mechanisms. In order to test whether the GMDS in suspension cells works similarly to that in planta, hypocotyls from 10 days old seedlings, where the suspension cells were derived, were used for the study at the same time. Among the 16 different glucosinolates identified in hypocotyls, the levels of indole glucosinolates were induced by MeJA as expected. In addition, we found that the levels of aliphatic glucosinolates, especially, 4&#45;methylsulfinylbutyl and 8&#45;methylsulfinyloctyl&#45;glucosinolates, were significantly increased by MeJA treatment. We identified 8 glucosinolates in suspension cells. They showed different profiles and responses when treated with MeJA. The transcripts of several genes involved in the biosynthesis of different glucosinolates were analyzed and found to be induced in both suspension cells and hypocotyls after MeJA treatment. Myrosinase levels and activities were also monitored in the two systems. To systematically investigate changes in metabolic pathways and connect MeJA signaling and the GMDS, we have initiated proteomic analysis of MeJA signaling and metabolic network in the two systems. Recent results will be presented and discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Phosphorylation&#8211;dependent Nucleocytoplasmic Shuttling of BZR1 is Essential in Arabidopsis Brassinosteroid Signaling</div>

<div>

	<span  >
	Hojin Ryu</span>
	<sup>1</sup>, 

	<span  >
	Kangmin Kim</span>
	<sup>1</sup>, 

	<span  >
	Hyunwoo Cho</span>
	<sup>1</sup>, 

	<span  >
	Junghyuk Park</span>
	<sup>1</sup>, 

	<span  >
	Sunghwa Choe</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Ildoo Hwang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>POSTECH, Pohang, Korea, 

	<sup>2</sup>Seoul National University, Seoul , Korea

</div>

<div>Phytohormone brassinosteriods &#40;BRs&#41; play critical roles in plant growth and development. BR exerts its action by modulating the phosphorylation status of two key transcriptional factors, BES1 and BZR1, upon perception of BR by BRI1&#47;BAK1 receptors at the plasma membrane. However, a regulatory mechanism mediating BR signals from the plasma membrane&#45;associated BR perception to expression of BR target genes in the nucleus remains unknown. We here show that phosphorylation by BIN2 directly induces the nuclear export of BZR1; in the cytoplasm, BR&#45;activated BSU1 mediates their dephosphorylation and subsequent nuclear translocation. Two distinct potential phosphorylation domains in BZR1 were required for the nuclear export induced by BIN2. Transgenic Arabidopsis plants overexpressing BZR1 with mutations in these phosphorylation sites, displays constitutive BR response phenotypes. Taken together, we propose that the phosphorylation&#45;dependent spatial redistribution of BZR1 is essential for proper BR signaling.</div>

</div>





<div class="abs">

<div style="font-weight:bold">THE IMPACT OF TEMPORAL AND SPATIAL REGULATION OF THE IPT GENE EXPRESSION BY VARIOUS PROMOTERS ON LEAF SENESCENCE AND WATER STRESS TOLERANCE</div>

<div>

	<span  >
	Idit Banner</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Shimon Gepstein</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Technion-Israel Institute of Technology,Haifa,ISRAEL

</div>

<div>A most dramatic transgenic intervention leading to a delay senescence phenotype concerns the manipulation of levels of endogenous cytokinins. An effective current approach is the autoregulation of cytokinins synthesis, based on a specific promoter of a senescence gene &#45; SAG12. Surprisingly, this autoregulatory system did not show delay of leaf senescence in Arabidopsis. Two promising strategies for delaying senescence in tobacco and Arabidopsis plants are presented. The autoregulated biosynthesis of cytokinins, mediated by the expression of a isopentyltransferase &#40;IPT&#41; gene driven by an inducible early&#45;senescence&#47;late maturation promoter &#40;SARK, senescence&#45;associated receptor kinase&#41;, resulted in the remarkable delay of senescence of tobacco plants. Additional approach is the estrogen&#45;inducible XVE system of the IPT gene for the production cytokinins in Arabidopsis transgenic plants. These transgenic plants displayed altered seedlings development and delayed leaf senescence. &#13;<br/>Most surprisingly, the tobacco transgenic plants also displayed water stress tolerance as reflected by vigorous growth after a severe drought &#40;18 days without watering&#41;. The transgenic plants retained photosynthetic activity during drought and did not show chlorophyll degradation. After rewatering, the plants recovered photosynthetic activity and active growth. Our results showed that plants expressing SARK::IPT, cytokinins were synthesized in the whole plant and transported to the bottom.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Proteomic and genomic studies of the brassinosteroid signal transduction pathway</div>

<div>

	<span  >
	Wenqiang Tang</span>
	<sup>1</sup>, 

	<span  >
	Yu Sun</span>
	<sup>1</sup>, 

	<span  >
	Srinivas S. Gampala</span>
	<sup>1</sup>, 

	<span  >
	Zhiping Deng</span>
	<sup>1</sup>, 

	<span  >
	Joshua M. Gendron</span>
	<sup>1</sup>, 

	<span  >
	Tae-Wuk Kim</span>
	<sup>1</sup>, 

	<span  >
	Ming-Yi Bai</span>
	<sup>2</sup>, 

	<span  >
	Sheng-Wei Zhu</span>
	<sup>2</sup>, 

	<span  >
	Juan A Oses-Prieto</span>
	<sup>3</sup>, 

	<span  >
	Shenheng Guan</span>
	<sup>3</sup>, 

	<span  >
	Alma L. Burlingame</span>
	<sup>3</sup>, 

	<span  >
	Kang Chong</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Zhi-Yong Wang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Carnegie Institution, Stanford, CA, USA, 

	<sup>2</sup>Key Laboratory of Photosynthesis and Environmental Molecular Physiology, Institute of Botany, the Chinese Academy of Sciences, Beijing 100093, China, 

	<sup>3</sup>Mass Spectrometry Facility, Department of Pharmaceutical Chemistry, University of California, San Francisco CA 94143

</div>

<div>The phytohormone Brassinosteroid &#40;BR&#41; plays essential roles in multiple developmental processes. BR is perceived by the receptor kinase BRI1 at the cell surface, and downstream BR signal transduction involves a receptor&#45;like kinase &#40;BAK1&#41;, a GSK3&#45;like kinase &#40;BIN2&#41;, a phosphatase &#40;BSU1&#41;, and transcription factors &#40;BZR1 and BZR2&#47;BES1&#41;. BR binding to the extracellular domain of BRI1 induces BRI1&#45;BAK1 dimerization and receptor kinase activation, leading to dephosphorylation of BZR1 and BZR2&#47;BES1 possibly by inhibiting BIN2 or activating BSU1. When BR levels are low, BIN2 phosphorylates BZR1 and BZR2&#47;BES1 to inhibit their nuclear accumulation and DNA binding. Our studies of BZR1&#45;interacting proteins demonstrate that 14&#45;3&#45;3 proteins specifically bind to the BIN2&#45;phosphorylated BZR1 to increase its cytoplasmic retention. Mutations of a BIN2&#45;phosphorylation site in BZR1 that abolish 14&#45;3&#45;3 binding increase nuclear localization of BZR1 and BR responses. Genetic and genomic studies of target genes of BZR1 have identified new molecular links between BR signaling and developmental responses. Using proteomic approaches, we have identified over 70 BR&#45;regulated proteins, including known BR signaling components as well as novel proteins potentially involved in BR signal transduction or downstream responses. Some of the BR&#45;regulated proteins interact directly with known BR&#45;signaling proteins. Overexpression of a novel BR&#45;induced protein suppresses the phenotypes of the BR&#45;deficient <span style="font-style: italic">det2</span> mutant, demonstrating its function in BR responses. Our genomic and proteomic studies have begun to fill the gaps in the BR signal transduction pathway and elucidate the molecular networks that link BR signaling to various cellular and developmental processes. A model of the BR signaling pathway will be presented. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Ethylene&#45;Induced Opposite Redistributions of Calcium and Auxin are Essential Components in the Development of Tomato Petiolar Epinastic Curvature</div>

<div>

	<span style="text-decoration: underline" >
	Soo-Hwan Kim</span>
	<sup>1</sup>, 

	<span  >
	Yew Lee</span>
	<sup>1</sup>, 

	<span  >
	Jin-Woo Jung</span>
	<sup>1</sup>, 

	<span  >
	June-Seung Lee</span>
	<sup>2</sup>, 

	<span  >
	Seong-Ki Kim</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Department of Life Science, Yonsei University, Wonju-Si 220-710, Korea, 

	<sup>2</sup>Department of Life Science, Ewha Womans University, Seoul 120-750, Korea, 

	<sup>3</sup>Department of Life Science, Chung-Ang University, Seoul 156-756, Korea

</div>

<div>Calcium has been suggested as an important mediator of gravity signaling transduction within the root cap statocyte. In a horizontally&#45;placed root, it is redistributed in the direction of the gravity vector &#40;i.e. it moves downward&#41; and is closely correlated with auxin downward movement. However, the involvement of calcium in the regulation of ethylene&#45; induced epinasty and auxin movement is not known. In this report, we examined the involvement of calcium in lateral auxin transport during ethylene&#45;induced epinasty in an effort to understand the relationship among calcium, auxin, and ethylene. Ethylene&#45;induced epinasty was further stimulated by exogenously applied Ca2&#43;, the calcium effect being the strongest among divalent cations tested. Pretreatment with NPA, an auxin transport inhibitor, negated the promotive effect of calcium ions on the petiolar epinasty. Ethylene caused redistribution&#47; differential accumulation of 45Ca2&#43; toward the morphologically lower &#40;abaxial&#41; side of the leaf petioles, an effect opposite to that of 14C&#45;IAA redistribution. Verapamil, a Ca2&#43; channel blocker, inhibited ethylene&#45;induced epinasty, as well as the redistribution of 14C&#45;IAA and 45Ca2&#43;. When the petiole was inverted in the presence or absence of ethylene, the direction of 45Ca2&#43; differential accumulation was still toward the morphologically abaxial side of the petiole during epinastic movement regardless of gravitational direction. These results suggest that gravity&#45;insensitive, ethylene&#45;induced Ca2&#43; redistribution and accumulation toward the abaxial side are closely coupled to the adaxial auxin redistribution&#47; accumulation and, in turn, to the petiolar epinasty.&#13;<br/>&#13;<br/>&#42; This work was supported by the Korea Research Foundation Grant funded by the Korean Government &#40;MOEHRD, Basic Research Promotion Fund&#41;&#40;KRF&#45;2006&#45;311&#45;C00149&#41;.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulation of the Elongation and Gravitropic Responses of Arabidopsis Roots by Brassinolide and IAA</div>

<div>

	<span  >
	Tae-Wuk Kim</span>
	<sup>1</sup>, 

	<span  >
	Seong-Ki Kim</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Soo Chul Chang</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Department of Life Science, Chung-Ang University, Seoul, Korea, 

	<sup>2</sup>University College, Yonsei University, Seoul, Korea

</div>

<div>Interactive effects of brassinolide &#40;BL&#41; with IAA on root gravitropic response and root elongation were investigated in &#34;Arabidopsis thaliana&#34;.  At various concentration exogenously applied BL increased gravitropic curvature but showed optimal curve in root elongation.  The stimulatory effect of BL was evident at later stage of the gravitropic response.  However, &#34;BRI1&#45;GFP&#34; plants, which carry two copies of BRI1, exhibited a greater gravitropic curvature response and showed more root elongation.  In contrast, the two parameters in &#34;bri1&#45;301&#34; plants were less than those for wild&#45;type &#40;Col&#45;0&#41; plants.  When the mutant plants were treated with various concentrations of IAA, the gravitropic response of the &#34;bri1&#45;301&#34; plants was increased in a dose&#45;dependent manner, attaining the same curvature response as &#34;BRI1&#45;GFP&#34; plants.  In wild&#45;type and &#34;BRI1&#45;GFP&#34; plants, it changed only slightly.  When the plants were treated with NPA or auxin&#45;transport mutants were tested, the gravitropic responses to BL dose were greatly affected.  The BL&#45;related mutants exhibited differential gene expression of CYP79B2, a IAA biosynthetic enzyme.  Taken together, BL seems to affect the gravitropic response at least in part by modulating endogenous IAA level.  The patterns of root elongation for the BL&#45;related mutant plants were not appreciably changed by treatment with different concentrations of IAA.  Also, a result that ethylene reduced the elongation indicates that BL effect on the elongation is mediated by ethylene.  Taken together, our results indicate that BL interacts with IAA in interdependent and partially additive ways in the processes of root gravitropic response and root elongation, respectively.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A nuclear protein interaction cascade mediating coordinated regulation of Arabidopsis seedling development by plant hormone gibberellins and light signal</div>

<div>

	<span  >
	Suhua Feng</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Cristina Martinez</span>
	<sup>1</sup>, 

	<span  >
	Yu Wang</span>
	<sup>2</sup>, 

	<span  >
	Liying Chen</span>
	<sup>3</sup>, 

	<span  >
	Giuliana Gusmaroli</span>
	<sup>1</sup>, 

	<span  >
	Juan M. Iglesias-Pedraz</span>
	<sup>4</sup>, 

	<span  >
	Feng Wang</span>
	<sup>3</sup>, 

	<span  >
	Lu Yu</span>
	<sup>3</sup>, 

	<span  >
	Xiangdong Fu</span>
	<sup>5</sup>, 

	<span  >
	Liu-Min Fan</span>
	<sup>2</sup>, 

	<span  >
	Xing Wang Deng</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular, Cellular and Developmental Biology, Yale University, New Haven, USA, 

	<sup>2</sup>Peking-Yale Joint Center for Plant Molecular Genetics and Agrobiotechnology, and National Laboratory for Protein Engineering and Plant Genetic Engineering, College of Life Sciences, Peking University, Beijing, China, 

	<sup>3</sup>National Institute of Biological Sciences, Zhongguancun Life Science Park, Beijing China, 

	<sup>4</sup>Centro Nacional de Biotecnologia-CSIC, Madrid, Spain, 

	<sup>5</sup>Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing, China

</div>

<div>Gibberellins &#40;GAs&#41; are plant hormones implicated in different aspects of plant growth and development. DELLA proteins are repressors of GA responses and their stability is controlled by the ubiquitin&#45;26S proteasome pathway. However, it is not well understood how GA triggers the degradation of DELLA proteins within a cell, and how the GA signal is transduced into DELLA&#180;s regulation of gene expression. Here, we characterize a nuclear protein&#45;protein interaction cascade mediating transduction of GA signals to the activity regulation of a light&#45;responsive transcription factor in Arabidopsis. In the absence of GA, nuclear localized DELLA proteins accumulate to higher level, interact with phytochrome&#45;interacting factor 3 &#40;PIF3, a bHLH&#45;type transcription factor&#41;, and prevent PIF3 from binding to its target promoters, therefore abrogating PIF3&#45;mediated light control of hypocotyl elongation of Arabidopsis seedlings. In the presence of GA, three largely redundant GA receptors &#40;GID1 proteins&#41; elevate their direct interaction with DELLA proteins in the nucleus, trigger DELLA&#180;s ubiquitination and proteasome&#45;mediated degradation, and thus release the negative effect of DELLA proteins on PIF3. Therefore, DELLA proteins integrate the regulatory effect of GA and light on gene expression and plant development through modulating the activity of some light&#45;responsive transcription factors like PIF3.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Early Events in Gibberellin Signaling</div>

<div>

	<span style="text-decoration: underline" >
	Tai-ping Sun</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Duke University, Durham, NC, USA

</div>

<div>Plant hormone gibberellin &#40;GA&#41;&#45;induced growth and development is modulated by DELLA proteins, which are major repressors of GA signaling.  Recent studies demonstrated that GA, upon binding to its receptor, de&#45;represses its signaling pathway by targeting DELLA for rapid degradation, via the ubiquitin&#45;proteasome pathway.  The nuclear&#45;localized DELLA proteins may function as transcriptional regulators, which control target gene expression via interaction with other transcription factors. Our recent microarray analysis has help to identify putative DELLA targets in Arabidopsis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">AtGL5, a cell wall protein in <span style="font-style: italic">Arabidopsis</span>, may regulates flowering and cell growth</div>

<div>

	<span  >
	Zhang Shengchun</span>
	<sup>1</sup>, 

	<span  >
	Yang Chengwei</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Wang Xiaojing</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>College of Life Sciences, South China Normal University, Guangdong Key Lab of Biotechnology for Plant Development, Guangzhou 510631, P.R. China

</div>

<div><span style="font-style: italic">AtGL5</span> gene is a GA regulated gene in <span style="font-style: italic">Arabidopsis</span>. The analysis of <span style="font-style: italic">gl5</span> mutant showed an earlier flowering phenotype. Spatial localization of the &#946;&#45;glucuronidase &#40;GUS&#41; gene product driven by <span style="font-style: italic">AtGL5</span> promoter was examined and GUS expression was tested by histochemical staining in T2 progeny of the transgenic plants. In 8&#45;day old seedlings GUS expression was first detected in cotyledons, elongation part of the root, lateral root and root hairs. Stronger GUS expression was appeared in the shoot meristem. During reproductive development of the plants, stronger GUS expression was found in primary and secondary inflorescence meristems emerging at the base of the cauling leaf. Branching sites of the flower from the inflorescence stem and rosettle leaves had also a strong expression. Moreover, GUS expression could be detected in the ovary and in the vascular of the siliques. To determine the subcellular localization of AtGL5 protein, we constructed AtGL5&#45;EGFP fusion protein that was expressed under the regulation of the cauliflower mosaic virus &#40;CaMV&#41; 35S promoter. When it was transiently expressed in onion epidermal cells, the transformed cells showed fluorescence of green fluorescent protein &#40;GFP&#41; in cell wall and no change under hypotonic solution. Whereas the control GFP protein was uniformly localized in the nucleus and cytoplasm and moves from the cell periphery to the central part of the cell volume in plasmolysis cells. Permanent expression of fusion protein in the cells of root hair in <span style="font-style: italic">Arabidopsis</span> obtained the same result, demonstrating that AtGL5 is localized in cell wall or the apoplasm of the cell. In addition, the expression of <span style="font-style: italic">AtGL5</span> was up&#45;regulated by NAA, SA and PAC and down&#45;regulated by GA3. Further investigation of the <span style="font-style: italic">AtGL5</span> function will be still elucidative. This work was supported by grants from National Natural Science Foundation of China 30570165.</div>

</div>





<div class="abs">

<div style="font-weight:bold">PINs reveal differential auxin efflux activities in Arabidopsis root hair cells</div>

<div>

	<span style="text-decoration: underline" >
	Sang Ho Lee</span>
	<sup>1</sup>, 

	<span  >
	Misuk Cho</span>
	<sup>1</sup>, 

	<span  >
	Ok Ran Lee</span>
	<sup>1</sup>, 

	<span  >
	Hyung-Taeg Cho</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, Chungnam National University, Daejeon 305-764, Korea

</div>

<div>Auxin is the unique phytohormone that moves cell&#45;to&#45;cell with polarity. Intercellular transport of auxin is mediated by influx and efflux carriers in the plasma membrane. We have previously demonstrated that the protein kinase PINOID &#40;PID&#41; and an auxin efflux carrier PIN3 positively regulate auxin efflux using the auxin&#45;sensitive Arabidopsis root hair cell system &#40;Plant Cell 18: 1604&#45;1616&#41;. Here, we have adopted the root hair system to compare the auxin efflux activities among different Arabidopsis PINs. Arabidopsis has eight PIN members whose distinctive primary structures imply diversity in spatial structures and thus molecular activities. Our assay result of the PIN activities demonstrated that different PINs have differential activities in facilitating auxin efflux in Arabidopsis root hair cells. We will be presenting the data for changes in root parameters and subcelluar PIN&#45;GFP localizations. Pharmacological approaches, regarding auxin&#45;transporting and protein kinase activities, also revealed distinctive responses of PINs to the chemicals. Our study suggests that the root hair cell system provides a quantitative tool to study auxin carrier protein molecules. &#13;<br/>This work was supported by grants from the MOST&#47;KOSEF Environmental Biotechnology National Core Research Center &#40;R15&#45;2003&#45;012&#45;02003&#45;0&#41;, and the Korea Research Foundation &#40;KRF&#45;2004&#45;041&#45;C00366&#41;. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">AtPGP4 Facilitates Auxin Efflux in Arabidopsis Root Hair Cells and Tobacco Cells</div>

<div>

	<span style="text-decoration: underline" >
	Misuk Cho</span>
	<sup>1</sup>, 

	<span  >
	Sang Ho Lee</span>
	<sup>1</sup>, 

	<span  >
	Hyung-Taeg Cho</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, Chungnam National University, Daejeon 305-764, Korea

</div>

<div>Formation of local auxin maxima through polar auxin movement has been implicated in plant growth and development. The polar transport of auxin is achieved by cooperation of polarly localized influx and efflux carriers particularly such as AUX1 &#40;and LAXs&#41; and PINs. On the other hand, albeit less polarized in their subcellular localization, some ABC transporters of multidrug resistance P&#45;glycoprotein &#40;MDR&#47;PGP&#41; subfamily, are involved in cellular auxin transport as well. AtPGP1 and AtPGP19 showed auxin efflux activities in plant cells whereas AtPGP4 was reported to catalyze auxin influx in animal and yeast cells. However, the phenotype of root hairs, whose elongation is enhanced by elevated intracellular auxin levels, implied an efflux activity of AtPGP4. We expressed AtPGP4&#45;YFP &#40;PGP4ox&#41; using the root hair cell&#45;specific Arabidopsis expansin7 promoter. PGP4ox developed shorter root hairs than control, which was expected from lowered auxin levels in the root hair cell by the auxin efflux activity of PGP4. The root hair phenotype of PGP4ox was similar to that of PGP1ox but opposite to that of AUX1ox. The efflux activity of AtPGP4 was further confirmed by using the tobacco BY&#45;2 suspension cell system. AtPGP4 in tobacco cells showed a NPA&#45;sensitive auxin efflux activity. The localization of PGP4&#45;YFP in the plasma membrane is non&#45;polar in Arabidopsis root hair cells and tobacco cells. A brefeldin&#45;A &#40;BFA&#41; treatment caused internalization of PGP4&#45;YFP as did PINs. These results suggest that AtPGP4 mediates auxin efflux in plant cells, differently from heterologous systems, and its endocytotic cycling could be mediated by BFA&#45;sensitive trafficking machinery. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of a novel transcriptional regulator gene AtSAT32: involvement in salinity response and gibberellin hormonal regulation of Arabidopsis root length</div>

<div>

	<span style="text-decoration: underline" >
	Min Young Park</span>
	<sup>1</sup>, 

	<span  >
	Hee-Seok Koh</span>
	<sup>1</sup>, 

	<span  >
	Sung-Ju Ahn</span>
	<sup>1</sup>, 

	<span  >
	Cheol Soo Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Biotechnology and Agricultural Plant Stress Research Center, Chonnam National University, Kwangju 500-757, Korea

</div>

<div>We conducted a genetic yeast screen to identify salt tolerance &#40;SAT&#41; genes in maize kernel cDNA library. During the screening, we identified a maize clone &#40;SAT32&#41; that seemed to confer elevated salt tolerance in comparison to control cells. SAT32 cDNA encodes a 48&#45;kDa protein which is 40 &#37; identical to the Arabidopsis unknown protein &#40;AtSAT32&#41;. To further examine salinity tolerance in Arabidopsis, we functionally characterized the AtSAT32 gene and found that dehydration as well as gibberellin &#40;GA&#41; induced AtSAT32 expression. Constitutive expression of AtSAT32 in Arabidopsis led to elevated salt tolerance in transgenic lines. Interestingly, the AtSAT32 null mutant exhibited insensitivity to GA. Our results suggest that AtSAT32 is involved in salt stress tolerance as well as in mediating the control of plant root length by GA hormone.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis Salt Tolerance 2 &#40;AtSAT2&#41; gene encode homologous GA MYB&#45;binding protein involved in abscisic acid signal transduction</div>

<div>

	<span style="text-decoration: underline" >
	Gah-Hyun Lim</span>
	<sup>1</sup>, 

	<span  >
	Cheol Soo Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Biotechnology and Agricultural Plant Stress Research Center, Chonnam National University, Kwangju 500-757, Korea

</div>

<div>We conducted a genetic yeast screen to identify salt tolerance &#40;SAT&#41; genes in maize kernel cDNA library. During the screening, we identified a maize clone &#40;SAT2&#41; that seemed to confer elevated salt tolerance in comparison to control cells. SAT2 cDNA encodes a 70&#45;kDa protein which is 67 &#37; identical to the Arabidopsis GA MYB&#45;binding protein &#40;AtSAT2&#41;. To further examine salinity tolerance in Arabidopsis, we functionally characterized the AtSAT2 gene and found that dehydration as well as abscisic acid &#40;ABA&#41; induced AtSAT2 expression. Constitutive expression of AtSAT2 in Arabidopsis led to elevated salt tolerance in transgenic lines. Interestingly, plants overexpressing AtSAT2 exhibited insensitivity to ABA. Our results suggest that AtSAT2 is involved in osmotic stress tolerance as well as in ABA signaling in Arabidopsis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional characterisation of PIN5 auxin transport protein</div>

<div>

	<span style="text-decoration: underline" >
	Jozef Mravec</span>
	<sup>1</sup>, 

	<span  >
	Markus Geisler</span>
	<sup>2</sup>, 

	<span  >
	Petr Skupa</span>
	<sup>3</sup>, 

	<span  >
	Vassilena Gaykova</span>
	<sup>1</sup>, 

	<span  >
	Christian Luschnig</span>
	<sup>4</sup>, 

	<span  >
	Jan Petrasek</span>
	<sup>3</sup>, 

	<span  >
	Eva Zazimalova</span>
	<sup>3</sup>, 

	<span  >
	Jiri Friml</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Center for Plant Molecular Biology (ZMBP), Auf der Morgenstelle 3, University T&#252;bingen, D-72076 T&#252;bingen, Germany, 

	<sup>2</sup>Z&#252;rich-Basel Plant Science Center, University of Zurich, Institute of Plant Biology, Molecular Plant Physiology, CH-8008 Zurich, Switzerland, 

	<sup>3</sup>Institute of Experimental Botany, ASCR, Rozvojov&#225; 135, 165 02 Praha 6, Czech Republic, 

	<sup>4</sup>Institute for Applied Genetics and Cell Biology, University of Natural Resources and Applied Life Sciences BOKU, Muthgasse 18, A-1190 Wien

</div>

<div>Development of plant body strongly depends on intercellular transport and distribution of hormone auxin. According to the chemiosmotic hypothesis auxin transport requires auxin influx and efflux carriers located on the plasma membrane of  the transporting cells. In Arabidopsis,  several proteins have already been shown to be directly involved in cellular auxin export or import. Best characterised is the plant&#45;specific family of PIN auxin efflux components. PIN proteins have been shown to determine rate and direction of intracellular auxin flow. Arabidopsis PIN gene family consists of 8 members, of which  PIN5, PIN6 and PIN8 have not been functionally characterised yet. PIN5 is the most distant member lacking the typical central hydrophilic loop and his role in auxin efflux has not been demonstrated. Here we present results of reverse genetic analysis aimed to characterise developmental role of PIN5. Full knock&#45;out mutant in PIN5 gene is deffective in cell elongation and  partially resistant to exogenous auxins and auxin transport inhibitors. On the other hand, overexpression of PIN5 causes phenotypes resebling mutants with increased auxin content. Both the loss&#45;of&#45;function and gain&#45;of&#45;function data are consistent with  the role of PIN5 in auxin transport&#45;related process. This data along with localisation studies and functional characterisation of PIN5 in heterologous systems will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis ROP9 Small GTPase positively regulate auxin responses</div>

<div>

	<span style="text-decoration: underline" >
	Fan Yang</span>
	<sup>1</sup>, 

	<span  >
	Zhenbiao Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Center for Plant Cell Biology, and Department of Botany and Plant Sciences, University of California, Riverside, California 92521, USA

</div>

<div>Auxin is an important plant hormone that modulates a myriad of plant growth and developmental processes. Recent studies have established an auxin signaling pathway that involves auxin activated degradation of transcription&#45;repressive IAA&#47;AUX proteins directly through auxin stabilizing the interaction between IAA&#47;AUX and the cyto&#45;&#47;nucleoplasmic auxin rceptors, which are the TIR1&#47;AFBs F&#45;box proteins that are a component of the SCF E3 ubiquitin ligase complex.  However, it is not clear whether other signaling mechanisms are also functionally important in auxin signaling. In this reporter, we demonstrate that the plasma membrane&#45;localized ROP9 small GTPase is an important positive regulator of auxin signaling using a combination of genetic, molecular, and biochemical analyses. <span style="font-style: italic">rop9&#45;2</span> knockout mutant seedlings exhibit reduced lateral root formation, shorter hypocotyls and roots as well as reduced responses to auxin in primary root elongation and lateral root initiation. In contrast, transgenic lines expressing a constitutively active mutant of <span style="font-style: italic">ROP9</span> are enhanced in auxin responses. By using a promoter::GUS fusion, we show that ROP9 expression is up&#45;regulated by exogenous auxin in hypocotyls and root tips. We demonstrate that ROP9 mediates auxin&#45;responsive gene expression. <span style="font-style: italic">DR5::GUS</span> expression is reduced in <span style="font-style: italic">rop9&#45;2</span> mutant but promoted by the constitutive active form of <span style="font-style: italic">ROP9</span>. In addition, ROP9 modulates auxin&#45;activated IAA degradation. The auxin sensitive turnover of IAA17&#45;LUC is reduced in <span style="font-style: italic">rop9&#45;2</span> mutant but enhanced by constitutively active form of ROP9. Importantly, exogenous auxin promotes ROP9 activation within 10 minutes after auxin application. These results show that ROP9 is a positive regulator of auxin signaling leading to IAA&#47;AUX degradation. We are currently interested in whether and how ROP9 signaling is connected to the TIR1&#45;IAA&#45;ARF auxin signaling pathway.  &#13;<br/>&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Impairment in the mRNA degradation causes abnormal response to both abscisic acid and salicylic acid in Arabidopsis.</div>

<div>

	<span style="text-decoration: underline" >
	Takashi Hirayama</span>
	<sup>1,2</sup>, 

	<span  >
	Yoshihiro Narusaka</span>
	<sup>3</sup>, 

	<span  >
	Michiko Yasuda</span>
	<sup>1</sup>, 

	<span  >
	Mari Narusaka</span>
	<sup>3</sup>, 

	<span  >
	Shimpei Hayashi</span>
	<sup>1,2</sup>, 

	<span  >
	Nobutaka Kitahata</span>
	<sup>4</sup>, 

	<span  >
	Takashi Kuromori</span>
	<sup>5</sup>, 

	<span  >
	Tadao Asami</span>
	<sup>4</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>5</sup>

</div>

<div>

	<sup>1</sup>RIKEN, Wako, Japan, 

	<sup>2</sup>Yokohama City Univ., Yokohama, Japan, 

	<sup>3</sup>RIBS Okayama, Okayama, Japan, 

	<sup>4</sup>Univ. of Tokyo, Tokyo, Japan, 

	<sup>5</sup>RIKEN PSC, Yokohama, Japan

</div>

<div>In eukaryotic cells, the stability of each mRNA is tightly regulated. Once mRNA enters the degradation pathway, its polyA is firstly removed for further degradation. Eukaryotes usually have three polyA removal activities. At the moment, it is obscure that they have distinct functions or not. In plants, disruption mutation of polyA specific ribonuclease &#40;PARN&#41; causes lethality, indicating its importance in plant systems. However, its physiological role has not been elucidated. By analyzing the <span style="font-style: italic">ABA hypersensitive germination2</span> mutants, we showed that the reduction of Arabidopsis PARN &#40;AtPARN&#47;AHG2&#41; activity causes multiple effects on the plant physiology including dwarf phenotype, higher accumulation of endogenous ABA in seeds and stressed plants, and constitutive activation of the SA response pathway &#40;1&#41;. We further confirmed the enhanced SA response in the <span style="font-style: italic">ahg2&#45;1</span> mutant by showing its higher endogenous SA level and enhanced resistance to pathogens. In addition, several SA defective mutations partially suppress the dwarf phenotype of <span style="font-style: italic">ahg2&#45;1</span>. These results suggest that PARN in plant is deeply involved in both biotic and abiotic stress responses. To address the molecular basis for AtPARN&#47;AHG2 in hormone response, we are trying to identify the target genes by microarray experiments and genetic approaches, eg. isolating suppressor mutants for <span style="font-style: italic">ahg2&#45;1</span>. Interestingly, some putative suppressor mutants show normal root growth, ABA insensitivity, and normal SA levels, although any ABA insensitive mutations or SA insensitive mutations could not give such effect as far as we examined. The function encoded by these suppressor loci will be discussed. <br/><br/>&#40;1&#41; Nishimura et al., Plant J., 44, 972&#45;984, 2005. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">SOR12 is an essential component of cytokinin&#45;mediated delay of leaf senescence</div>

<div>

	<span style="text-decoration: underline" >
	In Chul Lee</span>
	<sup>1</sup>, 

	<span  >
	Hyo Jung Kim</span>
	<sup>1</sup>, 

	<span  >
	Seung Hee Choi</span>
	<sup>1</sup>, 

	<span  >
	Hong Gil Nam</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>POSTECH, Pohang, South Korea

</div>

<div>Senescence is a sequence of biochemical and physiological events that constitute the final stage of development. Senescence is now clearly regarded as a genetically programmed and evolutionally acquired developmental process. However, in spite of the biological and practical importance, genetic mechanism of senescence has been very limited. &#13;<br/>Previously, we reported that ore12&#45;1 has increased leaf longevity due to a missense mutation in AHK3, a sensor histidine kinase cytokinin receptor, and suggested that cytokinins exert their anti&#45;senescing effect specifically and positively through AHK3 to control senescence &#40;Kim et al., 2006&#41;. To identify signaling components downstream of AHK3, we have undertaken a systematic genetic screening in an ore12&#45;1 allele through ethyl methanesulfonate &#40;EMS&#41;&#45;mutagenesis. One suppressor named sor12 &#40;suppressor of ore12&#45;1&#41; was identified and showed complete suppression of the ore12&#45;1 senescence phenotypes. sor12 ore12&#45;1 double mutants exhibited accelerated senescence symptoms in age&#45;dependent leaf senescence as well as in dark&#45;induced senescence. Furthermore, sor12 ore12&#45;1 dramatically reduced the sensitivity of the plant to cytokinins in delaying leaf senescence and in inducing cytokinin&#45;responsive genes, although these mutants still showed normal sensitivity to cytokinins in other responses, such as shoot induction and hypocotyl elongation inhibition. &#13;<br/>Therefore, we suggest that SOR12 plays a major role in controlling cytokinin&#45;mediated leaf senescence as a downstream component of AHK3. The identification of the mutated genes is underway and will be reported soon.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional study of <span style="font-style: italic">ETO1</span> gene family in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Kevin Long-Chi WANG</span>
	<sup>1</sup>, 

	<span  >
	Liang-Ching WANG</span>
	<sup>1</sup>, 

	<span  >
	Chia-Jeng WU</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant and Microbial Biology, Academia Sinica, Taipei, TAIWAN

</div>

<div>Phytohormones play an essential role in modulation of plant development and adaptation to the constantly changing environment. Ethylene is a gaseous phytohormone and has versatile function in plant physiology. Ethylene biosynthesis is highly regulated by developmental and environmental signals. The immediate precursor of ethylene, 1&#45;aminocyclopropane&#45;1&#45;carboxylic acid &#40;ACC&#41;, was converted by ACC synthase &#40;ACS&#41; from S&#45;adenosylmethionine. ACS catalyzes the rate&#45;limiting step of ethylene biosynthetic pathway and is encoded by gene family in all plant species. Previous studies showed that mutations in <span style="font-style: italic">ETO1</span> &#40;<span style="font-style: italic">ETHYLENE OVERPRODUCER1</span>&#41; resulted in a higher ethylene emanation, which was suppressed by a null mutation of <span style="font-style: italic">ACS5</span>, indicating ETO1 is a negative regulator in ethylene biosynthesis. <br/><br/>There are two ETO1 paralogs in <span style="font-style: italic">Arabidopsis thaliana</span>, EOL1 and EOL2 &#40;ETO1&#45;LIKE&#41;. ETO1 and EOL proteins share 50 and 75&#37; of identity at the N terminal BTB &#40;Broad&#45;Complex, Tramtrack, Bric&#45;&#225;&#45;brac&#41; domain and the C terminal TPR &#40;Tetratricopeptide Repeat&#41; domain, respectively. While the TPR domain of ETO1 is essential for the interaction with ACS5, the BTB domain has been shown to interact with CULLIN3 in the SCF &#40;Skp1&#45;Cul1&#45;F box&#41; ubiquitin E3 ligase. The working model is that ETO1 negatively regulates ethylene biosynthesis by interacting with ACS, specifically ACS5 and ACS9, to promote protein degradation dependent on the 26S proteasome. The observation that recessive mutations in <span style="font-style: italic">ETO1</span> alone resulted in a constitutive triple response &#40;ctr&#41; phenotype in etiolated seedlings suggested distinct function of <span style="font-style: italic">EOL</span> genes. <br/><br/>We have identified homozygous T&#45;DNA insertional mutants of <span style="font-style: italic">ETO1</span> and <span style="font-style: italic">EOL</span> genes for functional analysis. In addition to single mutants, we have also generated double and triple mutants. Null mutants of individual <span style="font-style: italic">EOL</span> did not show an obvious ctr phenotype or a higher ethylene emanation. However, double &#40;<span style="font-style: italic">eto1&#45;4 eol1&#45;1</span> and <span style="font-style: italic">eto1&#45;13 eol2&#45;1</span>&#41; and triple &#40;<span style="font-style: italic">eto1&#45;13 eol1&#45;4 eol2&#45;1</span>&#41; mutants produced a higher level of ethylene than wild type did. Recent results of phenotypical analysis using these mutants will be presented.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Role of farnesylation of adenylate isopentenyltransferase &#40;AtIPT3&#41; in control of enzyme targeting and cytokinin levels in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Kl&#225;ra Hoyerov&#225;</span>
	<sup>1</sup>, 

	<span  >
	Arnaud Galichet</span>
	<sup>2</sup>, 

	<span  >
	Miroslav Kam&#237;nek</span>
	<sup>1</sup>, 

	<span  >
	Wilhelm Gruissem</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Institute of Experimental Botany ASCR, Prague, Czech Republic, 

	<sup>2</sup>Institute of Plant Sciences, ETH Zurich, Switzerland

</div>

<div>Posttranslational prenylation of proteins facilitates their interactions with some other proteins and cell membranes. A number of prenylated plant proteins are involved in regulation of processes controlling plant development, signaling and responses to environmental stimuli. We report that plant isopentenyl transferase &#40;AtIPT3&#41; that catalyzes biosynthesis of isoprenoid cytokinins, is a farnesylated protein as it was confirmed by &#34;in vitro&#34; and &#34;in vivo&#34; farnesylation assays. Green fluorescent protein fused to AtIPT3 and expressed in Arabidopsis plants was localized in nucleus while it was accumulated in plastids when expressed in Arabidopsis &#34;era1&#45;2&#34; mutant lacking functional protein farnesyl transferase &#40;PFT&#41;. Ectopic expression of &#34;AtIPT3&#34; in Arabidopsis resulted in 4&#45;fold increase in cytokinin levels. The increase was accompanied with preferential &#40;11&#45;fold&#41; accumulation of isopentyl&#45;type cytokinins namely of isopentenyladenine 7&#45;glucoside. Expression of AtIPT3, in which acceptor cysteine was replaced with serine, significantly reduced cytokinin accumulation. This modification evidently cancelled the enzyme catalytic activity as indicates the lack of cytokinin accumulation in yeast and &#34;E. coli&#34; overexpressing AtIPT3 and modified AtIPT3 with serine. Evidently, the acceptor cysteine is essential not only for attachement of farnesyl moiety but also for AtIPT3 catalytic activity. Surprisingly, lack of PFT activity in the Arabidopsis &#34;era1&#45;2&#34; mutant enhanced cytokinin levels indicating that farnesylation of AtIPT3 may slightly reduce the enzyme catalytic activity.&#13;<br/>&#34;Acknowledgements&#34;: Research was supported by the ETH Zurich, Switzerland and by the agency of the Academy of Sciences of the Czech Republic &#40;A600380507&#41;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulation of 'BRX' expression by hormone pathways: evidence for 'BRX' as a putative target of MP</div>

<div>

	<span style="text-decoration: underline" >
	Karen Osmont</span>
	<sup>1</sup>, 

	<span  >
	Naden Krogan</span>
	<sup>2</sup>, 

	<span  >
	Thomas Berleth</span>
	<sup>2</sup>, 

	<span  >
	Christian Hardtke</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Lausanne, Switzerland, 

	<sup>2</sup>University of Toronto, Canada

</div>

<div>The novel growth regulator 'BREVIS RADIX &#40;BRX&#41;' has been proposed to function in a feedback loop that connects the auxin and brassinosteroid pathways to modulate root system growth and architecture. Consistent with the idea of a feedback loop, 'BRX' expression is regulated by both hormones and particularly responsive to auxin. Prototypical auxin response elements are present in the 'BRX' promoter, suggesting that the involvement of auxin response factors &#40;ARFs&#41; in 'BRX' regulation. A candidate ARF for this is 'MONOPTEROS &#40;MP&#41;', because 'BRX' and 'MP' expression patterns are strikingly similar. Expression of both genes is observed very early in embryogenesis, throughout the embryo, and becomes gradually restricted to the incipient vasculature. Further, post&#45;embryonically, the expression of both genes is specific to the vasculature. Double mutant analyses of 'brx' with the redundant ARFs 'MP' and 'NPH4' and provide preliminary molecular and biochemical evidence that MP may indeed regulate 'BRX'. Details about this interaction as well as 'BRX' auto&#45;regulation are reported.</div>

</div>





<div class="abs">

<div style="font-weight:bold">First insights into the functions of CUL3&#45;based ubiquitin ligases in Arabidopsis<br/><br/></div>

<div>

	<span style="text-decoration: underline" >
	Esther Lechner</span>
	<sup>1</sup>, 

	<span  >
	Alexis Thomann</span>
	<sup>1</sup>, 

	<span  >
	Eva Dumbliauskas</span>
	<sup>1</sup>, 

	<span  >
	Maureen Hansen</span>
	<sup>2</sup>, 

	<span  >
	Joseph Kieber</span>
	<sup>2</sup>, 

	<span  >
	Pascal Genschik</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>IBMP, Strasbourg, France, 

	<sup>2</sup>University of North Carolina, department of Biology, Chapel Hill, NC, USA

</div>

<div>Regulation of protein stability through the ubiquitin proteasome system &#40;UPS&#41; is now considered as a major mechanism underlying many cellular and organismal processes, such as cell division, DNA repair, developmental pathways, important parts of immune defence and in plants, light and phytohormone signal transduction. Degradation via the UPS is a two&#45;step process: the protein is first tagged by covalent attachment of ubiquitin and subsequently degraded by a multicatalytic protease complex called the 26S proteasome. Conjugation of ubiquitin to the protein is achieved through an enzymatic cascade involving the sequential action of three enzymes: E1, E2 and E3. The E3 enzymes play the most important role by bringing the specificity to the system. Several hundred different E3s have been identified; among them Cullin &#40;CUL&#41;&#45;dependent ubiquitin ligases are the most intensively studied. The CUL3&#45;BTB ligases belong to this class of enzymes. In contrast to budding and fission yeast, CUL3 is an essential gene in metazoans. Arabidopsis thaliana encodes two related CUL3 genes, called CUL3A and CUL3B. These two genes are expressed in reproductive tissues and shows largely overlapping expression patterns suggesting possible functional redundancy. The disruption of both the CUL3A and CUL3B genes reduces gametophytic transmission and causes embryo lethality. Arrest of embryogenesis occurs at multiple stages of embryo development, but predominantly at the heart stage. This result demonstrates that CUL3 genes play an essential role during embryogenesis like in other metazoans. In order to understand the role of CUL3 during postembryonic development, we engineered a hypomorphic cul3a&#47;cul3b double mutant. We will present here the phenotypical characterization of the cul3a&#47;cul3b hypomorphic double mutant and give the first insight about the role of CUL3 in the regulation of ethylene biosynthesis.<br/></div>

</div>



<div class="abs">

<div style="font-weight:bold">Bestatin, an Inhibitor of Aminopeptidases, Provides a Chemical Genetics Approach to Dissect Jasmonate Signaling in Arabidopsis</div>

<div>

	<span  >
	Wenguang Zheng</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Chuanyou Li</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>State Key Laboratory of Plant Genomics and Center for Plant Gene Research, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing 100101, China

</div>

<div>Bestatin, a potent inhibitor of some aminopeptidases, was shown previously to be a powerful inducer of wound response genes in tomato. Here, we present several lines of evidence showing that bestatin specifically activates jasmonic acid &#40;JA&#41; signaling in plants. First, bestatin specifically activates the expression of JA&#45;inducible genes in tomato &#40;Lycopersicon esculentum&#41; and Arabidopsis &#40;Arabidopsis thaliana&#41;. Second, the induction of JA responsive genes by bestatin requires the COI1&#45;dependent JA signaling pathway, but does not depend strictly on JA biosynthesis. Third, microarray analysis using Arabidopsis whole genome chip demonstrates that the gene expression profile of bestatin&#45;treated plants is similar to that of JA&#45;treated plants. Fourth, bestatin promotes a series of JA&#45;related developmental phenotypes. Taken together, the unique action mode of bestatin in regulating JA&#45;signaled processes leads us to the hypothesis that bestatin exerts its effects through the modulation of some key regulators in JA signaling. We have employed bestatin as an experimental tool to dissect JA signaling through a chemical genetic screening, which yielded a collection of Arabidopsis bestatin&#45;resistant &#40;ber&#41; mutants that are insensitive to the inhibitory effects of bestatin on root elongation. Further characterization efforts demonstrate that some ber mutants are defective in various JA&#45;induced responses, which allowed us to classify the ber mutants into three phenotypic groups: JA&#45;insensitive ber mutants, JA&#45;hypersensitive ber mutants, and mutants insensitive to bestatin but showing normal response to JA. Genetic and phenotypic analyses of the ber mutants with altered JA responses indicate that we have identified several novel loci involved in JA signaling. &#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A High throughput over&#45;expression pipeline in Arabidopsis to discover genes important in phytohormone biosynthesis and regulation</div>

<div>

	<span style="text-decoration: underline" >
	Kenneth Feldmann</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Ceres, Inc., Thousand Oaks, (CA), USA

</div>

<div>Ceres has over&#45;expressed 20,000 plant genes in Arabidopsis. The primary transformants have been screened for alterations in phenotype under standard growth conditions. The progeny have been screened in a variety of stress conditions. An analysis of the morphological phenotypes uncovered a number of lines corresponding to genes known to affect hormone biosynthesis and regulation pathways as well as many other genes giving equivalent phenotypes and which may also affect phytohormones. This pipeline will be discussed in general along with details of several interesting over&#45;expression lines.</div>

</div>





<div class="abs">

<div style="font-weight:bold">High&#45;sensitive and High&#45;throughput Plant Hormone Analysis by LC&#45;MS&#47;MS</div>

<div>

	<span  >
	Yusuke Jikumaru</span>
	<sup>1</sup>, 

	<span  >
	Atsushi Hanada</span>
	<sup>1</sup>, 

	<span  >
	Eiji Nambara</span>
	<sup>1</sup>, 

	<span  >
	Shinjiro Yamaguchi</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Yuji Kamiya</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, Yokohama, Japan, 

	<sup>2</sup>RIKEN

</div>

<div>Plant hormones interact each other and regulate many aspects of plant growth and development. Therefore it is important to analyze all the hormones from the same plant materials. We used to use GC&#45;MS for hormone analysis. However, this method is not necessary suitable for high&#45;throughput and comprehensive hormone analysis. We have recently introduced a LC&#45;ESI&#45;MS&#47;MS system, which does not require derivatization of hormones. Using both GC&#45;MS and LC&#45;MS we have performed comprehensive hormone analysis. We could identify 16 different GAs by a simple LC&#45;MS from Arabidopsis. &#40;Varbanova M. et al. Plant Cell 2007&#41; ABA and PA, DPA and ABA&#45;GE were analyzed from submerged rice shoots &#40;Saika H. et al. Plant Cell Physiol. 2007&#41;. In a special case, we could quantify ABA even from a single seed of Arabidopsis cyp707a1&#47;a2&#47;a3. Analysis of cytokinins by LC&#45;MS is already established in RIKEN PSC. We are currently analyzing auxins, jasmonic acid and salicylic acid. Although brassinosteroids are still difficult to analyze with high&#45;sensitivity, recently we could identify castasterone from tomato fruit by LC&#45;MS &#40;ca. 1g fresh weight&#41;. This method is at least 20 times more sensitive than a conventional GC&#45;MS. We will also report the present situation of the hormone analysis platform of RIKEN PSC. For collaborative research on hormone analysis, contact Yuji Kamiya, ykamiya&#64;postman.riken.jp</div>

</div>





<div class="abs">

<div style="font-weight:bold">Light Mediates Brassinosteroid Sensitivity via Activation of MSBP1 by HY5 and HYH</div>

<div>

	<span style="text-decoration: underline" >
	Qiu-Ming Shi</span>
	<sup>1</sup>, 

	<span  >
	Song Li</span>
	<sup>1</sup>, 

	<span  >
	Hong-Wei Xue</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Physiology &amp; Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China

</div>

<div>Membrane Steroid Binding Protein 1 &#40;MSBP1&#41; which can bind brassinolide in vitro is known as a negative regulator of brassinosteroid &#40;BR&#41; signal transduction and cell expansion. Interestingly, the promoter&#45;GUS assay showed the expression of MSBP1 in hypocotyls is induced by various lights. To further study the role of MSBP1 in light signaling, relationships of MSBP1 to COP1 and HY5&#47;HYH were analyzed. Results showed that HY5 and HYH could directly bind to the promoter region of MSBP1. Further Western blot assay shows that the expression of MSBP1 in HY5, HYH and double mutant was repressed. The groundwork show that MSBP1 may play as a bridge between Light and brassinosteroid signal, the activation of MSBP1 by HY5 and HYH deals a great impact on the alternation of BR sensitivity and photomorphogenesis in dark&#45;light transition.</div>

</div>


</div>
</div>

</body>
</html>
