<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>
<head>
<title>ICAR 2007 - Metabolism</title>
<link rel="Shortcut Icon" href="/favicon.ico">
<link rel="stylesheet" type="text/css" href="/css/main.css" />
<link rel="stylesheet" type="text/css" href="/css/search.css" />
<link rel="stylesheet" type="text/css" href="/css/base.css" />

<style type="text/css">
@import "/css/page/pagelevels.css";
</style>

<style type="text/css">
.abs {
 margin-top: 30px;
}
.abs div {
 margin: 10px;
}
</style>
</head>
<body>

<div id="container">

<script src="https://ssl.google-analytics.com/urchin.js" type="text/javascript"></script>
<script type="text/javascript">
_uacct = "UA-1945230-1";
urchinTracker();
</script>

 <div id="nav">	
  <div id="logo"><a href="/index.jsp"><img src="/i/logo2.gif" width="87" height="61" alt="tair"/></a></div>
 </div>

<div id="content" class="clearfix">

<div class="abs">

<div style="font-weight:bold">Structural Prediction and Study of Uridine Diphosphate Glycosyltransferases</div>

<div>

	<span style="text-decoration: underline" >
	Xiaoqiang Wang</span>
	<sup>1</sup>, 

	<span  >
	Hui Shao</span>
	<sup>1</sup>, 

	<span  >
	Zhenzhan Chang</span>
	<sup>1</sup>, 

	<span  >
	Lenong Li</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Biology Division, Samuel Roberts Noble Foundation, Ardmore, OK 73401, USA

</div>

<div>   Glycosylation plays very important roles in chemical diversity of plant secondary metabolites and  control their bioactivities and functions. Plants contain a large number of uridine diphosphate glycosyltransferases &#40;UGTs&#41; involved in glycosylation of secondary metabolites, e.g. over 100 putative UGT genes have been identified in Arabidopsis thaliana. It&#8217;s a challenging task to characterize the function of each enzyme. We are using structure&#45;based approaches, including crystallography and computer prediction&#47;modeling, to explore the structure&#45;function relations of these complex and divergent enzymes.&#13;<br/>   We are developing a predictive server based on plant UGT structures determined recently, including structures of Medicago truncatula UGT71G1 and grape VvGT1. With this predictive tool, we may identify putative key residues involved in substrate binding for a new UGT, and generate a predicted three dimensional structural model, providing a structural basis to analyze the putative substrate binding pocket and predict the substrates and functions. We are utilizing this approach to study several Arabidopsis UGTs involved in flavonoid metabolism and hormone homeostasis. Crystallographic study of these Arabidoposis UGTs is also in progress. The cDNAs of the target UGTs were cloned into E. coli expression vectors pET28a or pGEX&#45;2T containing an His&#45;tag or GST&#45;tag with a thrombin cleavage site. Proteins were expressed in E. coli BL21&#40;DE3&#41; and purified with Ni2&#43;&#45;NTA or glutathione&#45;agaroses. The N&#45;terminal His&#45; or GST&#45;tags were cleaved with biotinylated thrombin. The proteins were further purified on Resource Q and Superdex&#45;200 gel filtration columns. Crystallization screening towards structure determination was carried out. These studies help to identify and confirm the key residues for enzyme functions and substrate specificities, and reveal the detailed interactions between enzymes and substrates.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Induction of glucosinolates biosynthesis in Arabidopsis cell culture by carbon nanotubes &#40;CNT&#41; and thidiazuron &#40;TDZ&#41;</div>

<div>

	<span style="text-decoration: underline" >
	Maged Fouad</span>
	<sup>1</sup>, 

	<span  >
	Mohammad Jabasini</span>
	<sup>1</sup>, 

	<span  >
	Noritada Kaji</span>
	<sup>1,2</sup>, 

	<span  >
	Kazuyoshi Terasaka</span>
	<sup>3</sup>, 

	<span  >
	Manabu Tokeshi</span>
	<sup>1,2</sup>, 

	<span  >
	Hajime Mizukami</span>
	<sup>3</sup>, 

	<span  >
	Yoshinobu BABA</span>
	<sup>1,2,4,5</sup>

</div>

<div>

	<sup>1</sup>Department of Applied Chemistry, Graduate School of Engineering, Nagoya University, Nagoya, Japan, 

	<sup>2</sup>MEXT Innovative Research Center for Preventive Medical Engineering, Nagoya University, Nagoya, Japan, 

	<sup>3</sup>Department of Pharmacognosy, Graduate School of Pharmaceutical sciences, Nagoya City University, Nagoya, Japan, 

	<sup>4</sup>Plasma Nanotechnology Research Center, Nagoya University, Nagoya, Japan, 

	<sup>5</sup>National Institute of Advanced Industrial Science and Technology, Takamatsu, Japan

</div>

<div>Elicitors are molecules that stimulate defense or stress&#45;induced responses in plants. They trigger signal cascades that activate several defense responses such as reinforcement of cell walls, induction of pathogenesis&#45;related proteins or synthesis of phytochemicals. The enhancement of the content of phytochemicals as well as detection of novel ones upon elicitation suggests that elicitation may be a functional tool of technology for phytochemicals production. CNT and TDZ were found to act as abiotic elicitors when added to Arabidopsis thaliana callus culture. Quantitative estimation of total glucosinolates was achieved using microchip capillary electrophoresis and showed about 5.5 &#40;0.082 mM&#41; and 4 &#40;0.06mM&#41; folds increase in case of cultures containing CNT and TDZ respectively compared to normal cultures &#40;0.015mM&#41;. Glucosinolates contents in cell suspension cultures, containing TDZ, were found to be low &#40;&#8776;0.007mM&#41; compared to normal cultures. This may be due to high activity of myrosinase enzyme in suspension culture compared to callus culture. Glucosinolates qualitative analysis was achieved using microchip capillary electrophoresis and MALDI&#45;TOF&#45;MS.  Seven different glucosinolates were identified in both normal cultures and cultures containing TDZ, while ten different glucosinolates were identified in cultures containing CNT. This study suggests CNT to be a valuable technology tool that can affect plant cell system biology and elicit the plant defense network leading to production of valuable phytochemicals.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Jasmonates, fast&#45;acting master switches between growth and defence responses in Arabidopsis</div>

<div>

	<span  >
	Laurens Pauwels</span>
	<sup>1</sup>, 

	<span  >
	Kris Morreel</span>
	<sup>1</sup>, 

	<span  >
	Wout Boerjan</span>
	<sup>1</sup>, 

	<span  >
	Dirk Inz&#233;</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Alain Goossens</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>VIB Department of Plant Systems Biology, UGent, Gent, Belgium

</div>

<div>Jasmonates &#40;JAs&#41; are plant&#45;specific signalling molecules that steer a diverse set of cellular processes, of which their ability to invoke defence responses is the most renown. To further unravel the pleiotropic functions of JAs, the transcriptome of an Arabidopsis thaliana cell suspension culture treated with methyl jasmonate &#40;MeJA&#41; was monitored over time with microarrays. Additionally, cycloheximide &#40;CHX&#41; was used to distinguish primary and secondary MeJA response genes. Early MeJA response genes were predominantly involved in transcriptional regulation and JA biosynthesis, and were superinduced in the presence of CHX. In a second transcriptional wave, MeJA co&#45;ordinately upregulated the expression of genes involved in monolignol biosynthesis and several supporting or related pathways and processes. Simultaneously, MeJA repressed the expression of genes peaking in the M phase of the cell cycle. Targeted metabolite profiling and flow cytometric analysis confirmed the transcriptome data, showing a MeJA&#45;mediated increased flux through the monolignol pathway and a G2 cell cycle arrest, respectively. When comparing the microarray dataset from MeJA elicited suspension cells with other, publicly&#45;available datasets reflecting the responses of various Arabidopsis tissues to JA raises, a context dependency of the JA transcriptional response was observed.<br/>The MeJA inducible oligolignol biosynthesis model system and the clearly defined gene platform that have been established here are now being exploited to characterize hitherto unknown biochemical and molecular aspects of Arabidopsis phenylpropanoid metabolism. Furthermore, the potential role of the primary MeJA response genes, for instance members of different families of putative transcription factors, in the steering of the dual MeJA effect on growth and defence is being investigated.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Metabolites in Motion &#8211; Tracking The Dynamic Metabolome with 1H NMR</div>

<div>

	<span style="text-decoration: underline" >
	Jane Ward</span>
	<sup>1</sup>, 

	<span  >
	Aimee Galster</span>
	<sup>1</sup>, 

	<span  >
	Sonia Miller</span>
	<sup>1</sup>, 

	<span  >
	Delia-Irina Corol</span>
	<sup>1</sup>, 

	<span  >
	John Baker</span>
	<sup>1</sup>, 

	<span  >
	Michael Beale</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The National Centre For Plant and Microbial Metabolomics, Rothamsted Research, Harpenden, Herts, UK

</div>

<div>The model plant Arabidopsis thaliana is small, has a rapid growth cycle and the availability of collections of ecotypes, knockout mutants, and transgenic lines make this plant an ideal system for large&#45;scale metabolomic analysis in the context of gene function and whole system analysis.  &#13;<br/>&#13;<br/>Experimental protocols for the uniform growth, harvesting and NMR analysis of Arabidopsis have been extensively researched and developed, under GARNet. This work has now been extended to a high&#45;throughput screening operation on a hyphenated 600MHz instrument utilising NMR&#45;MS and SPE&#45;NMR to further refine and deconvolute the initial spectral fingerprints. In addition we have &#40;using NMR data from the screening of Arabidopsis mutants&#41; developed a fully interactive novel software tool to automate the interpretation of metabolomics results from fingerprint data in terms of true metabolite identities. This approach has clearly shown that 1H NMR is an informative and robust analytical technique in the field of metabolomics.&#13;<br/>&#13;<br/>In order to examine Arabidopsis in a further dimension, we have carried out metabolomics studies designed to superceed the single snapshot approach. These range from basic studies on the diurnal variation in metabolites, over light&#45;dark growth cycles, to experiments tracking metabolites through the entire Arabidopsis life cycle. Data from these experiments show that &#91;1H&#93;&#45;NMR provides a comprehensive fingerprint that reports rapidly on the status of the more abundant metabolites in the plant. PCA and alternative methods of data analysis demonstrate the utility of the technique in these large scale dynamic experiments and show how the cycling of individual metabolites can be tracked through time&#45;course dataset.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Classification of mutants by metabolite fingerprinting&#45; the HiMeT project</div>

<div>

	<span style="text-decoration: underline" >
	Michael Beale</span>
	<sup>1</sup>, 

	<span  >
	Jane Ward</span>
	<sup>1</sup>, 

	<span  >
	Delia-Irina Corol</span>
	<sup>1</sup>, 

	<span  >
	John Baker</span>
	<sup>1</sup>, 

	<span  >
	Yudong Cai</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Centre for Plant and Microbial Metabolomics, Rothamsted Research, Harpenden,Herts, UK

</div>

<div>The HiMeT project was conceived to test the ability of metabolite fingerprinting &#40;datasets generated from unpurified plant extracts&#41; to phenotype plants. The aim was to utilise a collection of previously well characterised Arabidopsis mutants, selected to represent a spectrum of biochemical pathway knockouts, as a base set, to test the hypothesis that comparison of plants grown in controlled environment would allow focus on metabolite signatures specific for the mutation. In this project we used rapid fingerprinting techniques such as NMR and direct infusion ESI&#45;MS, underpinned by classical metabolite analysis by hyphenated&#45;MS to form a database of mutant specific chemical signatures arising from multivariate comparison of mutants with wild&#45;type. The ability to use these signatures to predict phenotype was tested using &#8216;unknown&#8217; SM mutants selected from the ATIDB transposon insertion database.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A Systems Approach to Nitrogen Networks and the &#34;VirtualPlant&#34;</div>

<div>

	<span style="text-decoration: underline" >
	G Coruzzi</span>
	<sup>1</sup>, 

	<span  >
	D Shasha</span>
	<sup>2</sup>, 

	<span  >
	M Katari</span>
	<sup>1</sup>, 

	<span  >
	M Gifford</span>
	<sup>1</sup>, 

	<span  >
	K Birnbaum</span>
	<sup>3</sup>, 

	<span  >
	C Poultney</span>
	<sup>2</sup>, 

	<span  >
	R Gutierrez</span>
	<sup>4,1</sup>

</div>

<div>

	<sup>1</sup>NYU, Biology, NY, NY, 

	<sup>2</sup>NYU, Courant, NY, NY, 

	<sup>3</sup>NYU, Comparative Functional Genomics, NY, NY, 

	<sup>4</sup>P. Universidad Catolica de Chile, Santiago, Chile

</div>

<div>Our goal is to understand how internal and external perturbations affect gene networks linking plant metabolism with development. We seek to explain how changes in molecular networks evoke responses and to predict network states under untested conditions or in response to gene modifications. This systems approach should enable researchers to test biotechnological strategies for gene modification in silico, prior to testing in transgenic plants. Our systems approach starts with the integration of Arabidopsis genomic data into a &#8220;multinetwork&#8221; where the &#8220;edges&#8221; connecting gene &#8220;nodes&#8221; are supported by multiple data&#47;evidence including: metabolic pathway connections, protein:protein and protein:DNA interactions, microarray data, microRNA:target datasets, &#38; literature&#45;based interactions. This multinetwork currently contains 7,000 nodes and 230,000 interactions. As proof&#45;of&#45;principle, we used this Arabidopsis multinetwork to identify gene networks transcriptionally controlled by light, carbon and nitrogen signals using microarray data from specific organs or cell&#45;types. In selected cases, the networks identified in wild&#45;type plants have been validated using microarray and other data from Arabidopsis mutants. To support this type of plant systems biology approach, we have implemented a set of data integration, analysis and visualization tools into a system called &#34;VirtualPlant&#8221; &#40;www.virtualplant.org&#41;.  This platform encompasses visualization techniques that render the multivariate genomic information in visual formats that facilitate the extraction of biological concepts and enable a &#8220;Systems Biology&#8221; view of the genomic data. While VirtualPlant relates to Arabidopsis, the data structures, algorithms, and visualization tools developed are designed in a species&#45;independent fashion. Thus, with the proper data uploads, the system can be used to visualize and model the molecular basis and underlying genomic responses in any organism for which genomic data is available. Funding: NIH, NSF and DOE.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Metabolic engineering of terpene biosynthesis in Arabidopsis and consequences for plant&#45;environment communication</div>

<div>

	<span style="text-decoration: underline" >
	Harro Bouwmeester</span>
	<sup>1,2</sup>, 

	<span  >
	Iris Kappers</span>
	<sup>3,2</sup>, 

	<span  >
	Asaph Aharoni</span>
	<sup>4,2</sup>, 

	<span  >
	Carolien Ruyter-Spira</span>
	<sup>1</sup>, 

	<span  >
	Zhongkui Sun</span>
	<sup>2</sup>, 

	<span  >
	Tatsiana Charnikhova</span>
	<sup>1</sup>, 

	<span  >
	Catarina Cardoso</span>
	<sup>1</sup>, 

	<span  >
	Marcel Dicke</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Laboratory for Plant Physiology, Arboretumlaan 4, 6703 BD  Wageningen, the Netherlands, 

	<sup>2</sup>Plant Research International, P.O. Box 16, 6700 AA  Wageningen, the Netherlands, 

	<sup>3</sup>Laboratory of Entomology, P.O. Box 8031, 6700 EH  Wageningen, the Netherlands, 

	<sup>4</sup>Weizmann Institute of Science, P.O. Box 26, Rehovot 76100, Israel

</div>

<div>Terpenoids are a class of secondary metabolites that play an important role in the communication of plants with their environment. In the past few years rapid progress was achieved in terpenoid metabolic engineering in plants, including Arabidopsis. These engineering experiments have demonstrated that the presence of terpenoid precursors in sub&#45;cellular compartments is not as strictly separated as previously thought and that multi&#45;step pathway engineering, even across cell compartments is feasible &#91;1&#45;4&#93;. Hence it is now becoming feasible to create transgenic plants producing terpenoids on demand. With such engineered plants fascinating results have been obtained that show that insect behaviour is strongly influenced by terpenoids. The nature of these effects varies with terpenoid compound and insect species, and for example concerns repellency of aphids and thrips &#91;1,2&#93;, but also attraction of predatory mites, the natural enemies of spider mites &#91;3&#93;. However, also rhizosphere communication with other types of organisms such as parasitic plants and arbuscular mycorrhizal &#40;AM&#41; fungi is mediated by terpenoids. Also here Arabidopsis, although not a host of AM fungi, turns out to be a suitable model. For the future, we foresee great progress in the engineering of terpenoid production in Arabidopsis. We expect that such transgenic plants will increase our understanding of the biological relevance of these secondary metabolites in the communication of plants with a multitude of other organisms but may also lead to commercial applications.&#13;<br/>1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aharoni, A., <span style="font-style: italic">et al.</span> &#40;2003&#41; Plant Cell 15, 2866&#45;2884&#13;<br/>2&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Aharoni, A., <span style="font-style: italic">et al.</span> &#40;2005&#41; Trends Plant Sci. 10, 594&#45;602&#13;<br/>3&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Kappers, I.F., <span style="font-style: italic">et al.</span> &#40;2005&#41; Science 309, 2070&#45;2072&#13;<br/>4&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Wu, S., <span style="font-style: italic">et al.</span> &#40;2006&#41; Nature Biotechnol. 24, 1441&#45;1447</div>

</div>





<div class="abs">

<div style="font-weight:bold">Carbon availability and stress response. Do different DAHP synthase isoforms serve different demands&#63;</div>

<div>

	<span  >
	Sven Martin</span>
	<sup>1</sup>, 

	<span  >
	Sabine Preiskowski</span>
	<sup>1</sup>, 

	<span  >
	Anika Wiese</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Ingar Janzik</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Institute Phytosphere (ICG 3), Research Centre J&#252;lich, 52425 J&#252;lich, Germany, 

	<sup>2</sup>Heinrich Heine University D&#252;sseldorf, Germany

</div>

<div>The DAHP synthase &#40;DAHPS&#41; catalyses the first step of the shikimate pathway, a key pathway for the synthesis of aromatic primary and secondary plant metabolites. The substrates for the DAHPS are intermediates of the oxidative or reductive pentosephosphate cycle and glycolysis and hence depend directly on the availability of carbon in primary metabolism. On the other hand many plant reactions to environmental stress include the induction of aromatic secondary metabolites, derived from shikimate pathway products. Thus carbon flux through shikimate pathway depends on two main influences: the substrate availability and the drain into secondary metabolism. This led to the hypothesis that the DAHPS might be regulated by these two different signals independently&#13;<br/>To test this hypothesis, we first supplied Arabidopsis leaves with external glucose. Glucose turnover is known to act as a direct signal for many genes in response to increased carbon availability. By isoform specific Real&#45;time&#45;PCR we were able to show, that the supply of external glucose &#8211; but not sorbitol &#45; led to a strong increase in the DAHPS transcripts of two of the three existing isoforms. This was independent from the analysed ecotype. Further more we could demonstrate, that this induction was not regulated by the HXK1 mediated signalling cascade, what is in agreement with the regulation of other enzymes relevant for aromatic secondary metabolism, like PAL and CHS. But in contrast we could also demonstrate that the stress specific induction of aromatic secondary metabolites, resembling a drain of shikimate pathway products, only was preceded by the induction of one of the three isoforms. These results indicate that an increased carbon flux into primary aromatic metabolism, triggered by an increase in substrate availability might be mediated by another isoform, than a carbon efflux, which is forced by an increase in aromatic secondary metabolites. So that the different isoforms of the DAHPS are regulated by different signals and serve different demands, depending on the activation stimulus.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic Characterization and Functional Analysis of the Arabidopsis Mutants Disrupted in Endoplasmic Reticulum to Chloroplast Lipid Trafficking</div>

<div>

	<span style="text-decoration: underline" >
	Changcheng Xu</span>
	<sup>1</sup>, 

	<span  >
	Koichiro Awai</span>
	<sup>1</sup>, 

	<span  >
	Jilian Fan</span>
	<sup>1</sup>, 

	<span  >
	Binbin Lu</span>
	<sup>1</sup>, 

	<span  >
	Adams Cornish</span>
	<sup>1</sup>, 

	<span  >
	Christoph Benning</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biochemistry and Molecular Biology, Michigan State University, East Lansing, MI 48824, USA

</div>

<div>The photosynthetic &#40;thylakoid&#41; membrane of plants is one of the most extensive biological cell membrane systems found in nature. It harbors the photosynthetic apparatus which his essential to life on earth as carbon dioxide is fixed and atmospheric oxygen released by photosynthesis. Lipid biosynthetic enzymes of different subcellular compartments participate in the biogenesis of the thylakoid membrane system. This process requires the extensive exchange of lipid precursors between the chloroplast and the endoplasmic reticulum &#40;ER&#41;. The underlying lipid trafficking phenomena are not yet understood at the mechanistic level, but genetic mutants of the model plant <span style="font-style: italic">Arabidopsis thaliana</span> with disruptions in lipid trafficking between the ER and the chloroplast have recently become available. The TGD1&#45;TGD3 proteins are similar to components of multipartite ABC transporters as found in bacteria. They are localized in the inner chloroplast envelope membrane. The <span style="font-style: italic">tgd1&#45;1</span> mutant shows an increase in the level of phosphatidic acid. The rate of phosphatidic acid incorporation into galactolipids by isolated <span style="font-style: italic">tgd1&#45;1</span> chloroplast is decreased. The TGD2 protein specifically binds phosphatidic acid. The TGD3 protein has ATPase activity in vitro. Based on this evidence, we hypothesize that the respective complex transfers phosphatidic acid from the outer envelope membrane to the phosphatidic acid phosphatase at the inside of the inner envelope membrame, thereby providing the ER&#45;derived DAG precursor of galactoglycerolipid biosynthesis. A fourth protein, TGD4, is ER&#45;associated. We hypothesize that it mediates the direct interaction of ER and the outer chloroplast envelope membrane and&#47;or facilitates the transfer of lipids between the membrane systems.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Analysis of S&#45;Adenosyl&#45;L&#45;Homocystein Hydrolase Gene Expression Reveals a Role of Cytokinin in Promoting Transmethylation Reactions</div>

<div>

	<span  >
	Chun-Hong Li</span>
	<sup>1</sup>, 

	<span  >
	Shi-Min Jiang</span>
	<sup>1</sup>, 

	<span  >
	Ling-Jian Wang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Xiao-Ya Chen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Key Laboratory of Plant Molecular Genetics, Institute of Plant Physiology and Ecology, SIBS, Chinese Academy of Sciences, 300 Fenglin Road, Shanghai 200032, China

</div>

<div>S&#45;Adenosyl&#45;L&#45;homocystein hydrolase &#40;SAHH&#41;, catalyzing the conversion of S&#45;adenosyl&#45;L&#45;homocysteine into adenosine and L&#45;homocystein, is a key enzyme for the maintenance of cellular transmethylation potent. SAHH has been shown to be a cytokinin&#45;binding protein, however, the effect of cytokinin on transmethylation reactions has been unclear. In <span style="font-style: italic">Arabidopsis thaliana</span>, there are two genes encoding SAHH, <span style="font-style: italic">AtSAHH1</span> and <span style="font-style: italic">AtSAHH2</span>, which show non&#45;overlapping expression patterns. A T&#45;DNA insertion mutant of <span style="font-style: italic">AtSAHH1</span> &#40;<span style="font-style: italic">sahh1&#45;1</span>&#41; and the RNA interference &#40;RNAi&#41; plants &#40;<span style="font-style: italic">dsAtSAHH2</span>&#41; exhibited different degrees of phenotypic changes, including higher chlorophyll contents, delayed flowering and senescence, and bushy architecture. We found that expression of a cytokinin synthesis gene, <span style="font-style: italic">AtIPT7</span>, was up&#45;regulated. On the other hand, cytokinins positively regulate the transmethylation pathway genes, including <span style="font-style: italic">AtSAHH1</span> and <span style="font-style: italic">AtADK1</span> &#40;for adenosine kinase&#41;, and this regulation involves the cytokinin signaling pathway, but not the purine structure. Unlike adenine and adenosine which are SAHH inhibitors, the adenine&#45;type cytokinins have no effect on SAHH activity at protein level. Changing of endogenous cytokinin levels by overexpressing the cytokinin synthesis or degradation genes resulted in alterations of DNA methylation status in the <span style="font-style: italic">sahh1&#45;1</span> background, suggesting that cytokinins affect DNA methylation under metabolism&#45;stress conditions. These data demonstrate a cross&#45;talk between the SAM&#45;dependent transmethylation cycle and the cytokinin signaling.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Omics&#45;based identification of R2R3&#45;Myb transcription factors regulating aliphatic glucosinolate biosynthesis</div>

<div>

	<span style="text-decoration: underline" >
	Masami Yokota Hirai</span>
	<sup>1</sup>, 

	<span  >
	Kenjiro Sugiyama</span>
	<sup>2</sup>, 

	<span  >
	Yuji Sawada</span>
	<sup>1</sup>, 

	<span  >
	Takayuki Tohge</span>
	<sup>1</sup>, 

	<span  >
	Takeshi Obayashi</span>
	<sup>3,4</sup>, 

	<span  >
	Akane Suzuki</span>
	<sup>1</sup>, 

	<span  >
	Ryoichi Araki</span>
	<sup>1,5</sup>, 

	<span  >
	Nozomu Sakurai</span>
	<sup>2</sup>, 

	<span  >
	Hideyuki Suzuki</span>
	<sup>2</sup>, 

	<span  >
	Koh Aoki</span>
	<sup>2</sup>, 

	<span  >
	Hideki Goda</span>
	<sup>1</sup>, 

	<span  >
	Osamu Ishizaki Nishizawa</span>
	<sup>1,5</sup>, 

	<span  >
	Daisuke Shibata</span>
	<sup>2</sup>, 

	<span  >
	Kazuki Saito</span>
	<sup>1,3</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, Yokohama, Japan, 

	<sup>2</sup>Kazusa DNA Research Institute, Kisarazu, Japan, 

	<sup>3</sup>Chiba University, Chiba, Japan, 

	<sup>4</sup>CREST, Japan Science and Technology Agency, Saitama, Japan, 

	<sup>5</sup>Kirin Brewery Co., Ltd., Yokohama, Japan

</div>

<div>Understanding plant metabolism as an integrated system is essential for metabolic engineering aimed at the effective production of compounds useful to human life and the global environment. By integrating transcriptomics and metabolomics, we elucidated global regulation of the transcriptome and metabolome of Arabidopsis under nutritional stress conditions, and presented a strategy to identify novel gene functions with an example of sulfotransferase genes involved in the biosynthesis of glucosinolates &#40;GSLs&#41;, bioactive secondary metabolites &#91;1, 2&#93;. In this presentation we report the discovery of two R2R3&#45;Myb transcription factors that positively control the biosynthesis of  aliphatic GSLs &#91;3&#93;. Combined transcriptome coexpression analysis of publicly&#45;available condition&#45;independent data, and the condition&#45;specific &#40;i.e., sulfur&#45;deficiency&#41; data identified <span style="font-style: italic">Myb28</span> and <span style="font-style: italic">Myb29</span> as candidate transcription factor genes specifically involved in the regulation of aliphatic GSL production. Analysis of a knockout mutant and ectopic expression of the gene demonstrated that <span style="font-style: italic">Myb28</span> is a positive regulator for basal&#45;level production of aliphatic GSLs. <span style="font-style: italic">Myb29</span> presumably plays an accessory function for methyl jasmonate&#45;mediated induction of a set of aliphatic GSL biosynthetic genes. Overexpression of <span style="font-style: italic">Myb28</span> in Arabidopsis cultured suspension cells, which don't normally synthesize GSLs, resulted in the production of large amounts GSLs, suggesting the possibility of efficient industrial production of GSLs by manipulation of these transcription factors. A working model for regulation of GSL production involving these genes, renamed <span style="font-style: italic">Production of Methionine&#45;derived Glucosinolate</span> &#40;<span style="font-style: italic">PMG</span>&#41; <span style="font-style: italic">1</span> and <span style="font-style: italic">2</span>, are discussed.&#13;<br/>&#91;1&#93;Hirai et al. &#40;2004&#41; PNAS 101:10205&#13;<br/>&#91;2&#93;Hirai et al. &#40;2005&#41; JBC 280:25590&#13;<br/>&#91;3&#93;Hirai et al. &#40;2007&#41; PNAS 104:6478</div>

</div>





<div class="abs">

<div style="font-weight:bold">Vitamin B6 metabolism in plants: All is not as it had seemed.</div>

<div>

	<span style="text-decoration: underline" >
	Teresa Fitzpatrick</span>
	<sup>1</sup>, 

	<span  >
	Marina Tambasco-Studart</span>
	<sup>1</sup>, 

	<span  >
	Olca Titiz</span>
	<sup>1</sup>, 

	<span  >
	Thomas Raschle</span>
	<sup>1</sup>, 

	<span  >
	Maja Raschke</span>
	<sup>1</sup>, 

	<span  >
	Nikolaus Amrhein</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Science, ETH Zurich, Switzerland

</div>

<div>Vitamin B6 is an essential metabolite in all organisms. It can act as a coenzyme for numerous metabolic enzymes and has recently been shown to be a potent antioxidant. Its 'de novo' biosynthesis occurs only in bacteria, fungi and plants, making it an essential nutrient in the human diet. Despite its paramount importance, the pathway has classically only been investigated in 'Escherichia coli' where it is synthesized from the condensation of 1&#45;deoxy&#45;D&#45;xylulose&#45;5&#45;phosphate &#40;a precursor in the non&#45;mevalonate pathway of isoprenoid biosynthesis&#41; and 4&#45;hydroxy&#45;threonine phosphate catalyzed by the concerted action of PdxA and PdxJ. It was tacitly assumed that the vitamin is synthesized in the same way in plants. However, recent studies have implicated that in the majority of organisms &#40;including plants&#41; that are capable of producing vitamin B6, a pathway distinctive and exclusive from 'E. coli' exists. This pathway is characterized by the presence of two genes, namely 'PDX1' and 'PDX2'. We have identified the functional 'PDX1' and 'PDX2' homologs in 'Arabidopsis thaliana' and will report on the biochemistry and physiology of vitamin B6 biosynthesis in plants. Specific topics that will be addressed will include the chemistry behind the synthesis of the vitamin, biochemical and structural aspects of the PDX1 and PDX2 proteins involved in the novel pathway, their essentiality for plant viability and the relationship of vitamin B6 biosynthesis to oxidative stress.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification and Characterization of Pyruvate Decarboxylase &#40;pdc&#41; Gene Family Members in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Songqing Ye</span>
	<sup>1</sup>, 

	<span  >
	Jerry Cohen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Minnesota, Saint Paul, US

</div>

<div>The indole&#45;3&#45;pyruvic acid &#40;IPA&#41; pathway has long been proposed to be one of the major tryptophan&#45;dependent pathways in auxin biosynthesis.  However, the details of this pathway have not been clearly defined and no gene involved in this pathway has been fully described in plants. We have found that there are six genes in Arabidopsis having high sequence similarity to the ipdc &#40;EC 4.1.1.74&#41; or pdc &#40;pyruvate decarboxylase, EC 4.1.1.1&#41; from microbes.  I have identified the Atpdc2 is a functional pdc gene yielding a protein product with the predicted biochemical function.  PDC converts, by decarboxylation, pruvate into acetaldehyde and then alcohol dehydrogenase &#40;ADH, EC 1.1.1.1&#41; reduces acetaldehyde into ethanol.  This set of reactions shunts the main glycolytic pathway into ethanol fermentation, instead of entry into the tricarboxylic acid cycle, so it is very important for plants as they respond to anaerobic stress such as flooding.  Most interesting, this active enzyme does not have the classical bi&#45;functional activity &#40;pdc&#47;ipdc&#41; found with all known bacterial enzymes and it shows around 40&#37; of its activity with only a single cofactor TPP &#40;Thiamine Pyrophosphate&#41; or Mg2&#43; or without any cofactor supplied as compared to the rate obtained with a combination of both cofactors.  <br/><br/>We have designed a new highly specific GC&#45;MS method to analyze plant indole pyruvate decarboxylase &#40;IPDC&#41; activity.  We found that this method works well for the IPDC from bacteria, but as yet has not revealed plant IPDC activity from any of the genes cloned to date.  In addition, antibodies against the whole PDC family and to specific protein sequences have been generated independently such that the native proteins can be immunopurified from extracts prepared from plant tissues.  Hopefully, their roles in auxin biology as well as the kinetics of their turnover can be determined.<br/><br/>This work was supported, in part, by a grant from the U.S. National Science Foundation, Plant Genome Program &#40;DBI&#45;0606666&#41;.   <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A natural fusion gene encoding uridine kinase and uracil phosphoribosyltransferase in Arabidopsis and rice.</div>

<div>

	<span  >
	M.R. Islam</span>
	<sup>1</sup>, 

	<span  >
	Kang S.W.</span>
	<sup>1</sup>, 

	<span  >
	Kim H.</span>
	<sup>1</sup>, 

	<span  >
	Y.M. Jeong</span>
	<sup>1</sup>, 

	<span  >
	B.W. Kim</span>
	<sup>2</sup>, 

	<span  >
	D.K. Kim</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	J.S. Kim</span>
	<sup>2</sup>, 

	<span  >
	S.G. Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Seoul National University, Seoul, Republic of Korea, 

	<sup>2</sup>Cheju National University, Jeju, Republic of Korea

</div>

<div>Uridine kinase &#40;UK&#41; and uracil phosphoribosyltransferase &#40;UPRT&#41; are enzymes catalyzing the formation of uridine 5&#8217;&#45;monophosphate from uridine and adenosine 5&#8217;&#45;triphosphate or uracil and 5&#45;phosphoribosyl&#45;1&#45;pyrophosphate in pyrimidine salvage pathway, respectively. The genes for UK or UPRT were reported as separate genes in bacteria or yeasts. Here we report the analysis of a gene with dual domains for UK and UPRT from Arabidopsis &#40;AtUK&#47;UPRT1&#41; and rice &#40;OsUK&#47;UPRT1&#41;. Sequence analysis was revealed that AtUK&#47;UPRT1 and OsUK&#47;UPRT1 are encoded as a molecular mass of approximately 53 kDa and 51kDa, respectively. The predicted amino acid sequences of AtUK&#47;UPRT1 and OsUK&#47;UPRT1 are similar to the two proteins for UK and UPRT from bacteria and yeasts. Amino&#45;terminal region is similar to UDK of Escherichia coli and many bacteria and contains an ATP&#47;GTP&#45;binding site motif A called P&#45;loop whereas the carboxyl&#45;terminal is similar to UPP of E. coli and contains signature&#45;binding motif for a uracil and a PRPP. Expression of AtUK&#47;UPRT1 in a upp and udk mutant of E. coli led to growth inhibition effect with 5&#45;fluorouracil &#40;5&#45;FU&#41; or 5&#45;fluorouridine &#40;5&#45;FD&#41;. Identical results were obtained when the UK and UPRT domains were separated into the upp&#45;udk and upp mutants. We analyzed an Arabidopsis mutant of AtUK&#47;UPRT1 which its growth was not affected whereas wild&#45;type Arabidopsis showed drastic growth retardation with treatment of 5&#45;FU or 5&#45;FD. These results suggest that the AtUK&#47;UPRT1 and OsUK&#47;UPRT1 products can use uracil or uridine as a substrate and would be natural fusion proteins of a UK and a UPRT enzyme.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Chloroplast NADPH thioredoxin reductase: <br/>a novel thiol redox regulator of amino acid biosynthesis in chloroplasts&#63;<br/><br/></div>

<div>

	<span style="text-decoration: underline" >
	Saijaliisa Kangasj&#228;rvi</span>
	<sup>1</sup>, 

	<span  >
	Anna Lepist&#246;</span>
	<sup>1</sup>, 

	<span  >
	Kati H&#228;nnik&#228;inen</span>
	<sup>1</sup>, 

	<span  >
	Briitta Ruokamo</span>
	<sup>1</sup>, 

	<span  >
	G?nter Brader</span>
	<sup>2</sup>, 

	<span  >
	Nina Sipari</span>
	<sup>3</sup>, 

	<span  >
	Markku Kein&#228;nen</span>
	<sup>3</sup>, 

	<span  >
	Eevi Rintam&#228;ki</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, University of Turku, FI-20014 Turku Finland, 

	<sup>2</sup>Faculty of Biosciences, Department of Biological and Environmental Sciences, Genetics, FI-00014 University of Helsinki, Finland, 

	<sup>3</sup>Faculty of Biosciences, University of Joensuu, FI-80101 Joensuu, Finland

</div>

<div>A novel chloroplastic thiol redox regulator, an NADPH&#45;dependent thioredoxin reductase &#40;NTRC&#41;, was recently described in <span style="font-style: italic">Arapidopsis thaliana</span>. To gain insights into the physiological significance of NTRC, we have characterized T&#45;DNA insertion mutant lines of the <span style="font-style: italic">NTRC</span> gene. Homozygous <span style="font-style: italic">ntrc</span> plants showed reduced growth and a pale green phenotype when grown under moderate light intensity. Biochemical analysis revealed that these characteristics were not connected with deficiencies in the accumulation or function of photosynthetic membrane protein complexes, or with impaired carbon assimilation capacity of Rubisco. Rather, <span style="font-style: italic">ntrc</span> plants showed distinct developmental disorders: mesophyll cells were small and possessed lowered amount of chloroplasts, the number of stomata was increased, and senescence was delayed. Interestingly, all these characteristics were more pronounced under short day conditions &#40;8&#47;16 hour light rhythm&#41;. The developmental deficiencies correlated with imbalances in basic metabolism: the profile of plastid&#45;derived amino acids was altered, and the auxin content of ten&#45;day&#45;old <span style="font-style: italic">ntrc</span> plants was diminished. Accordingly, growth of <span style="font-style: italic">ntrc</span> plants on a medium supplemented with auxin, aromatic amino acids or isoleucine restored the cell size and the number of chloroplasts per cell in <span style="font-style: italic">ntrc</span> leaves. Interestingly, <span style="font-style: italic">Arabidopsis</span> belongs to the family of Brassicas, the members of which possess an additional tryptophan&#45;dependent pathway for auxin biosynthesis that takes place in chloroplasts. Altogether, our results suggest a novel function for NTRC in the photoperiod&#45;dependent modulation of plant development via thiol redox regulation of amino acid biosynthesis in chloroplasts. Yeast two&#45;hybrid analysis and thiol&#45;redox labelling experiments to demonstrate the target proteins of NTRC are underway. </div>

</div>





<div class="abs">

<div style="font-weight:bold">A single MYB domain protein, AtMYBL2, acts as a negative regulator in flavonoid biosynthesis</div>

<div>

	<span style="text-decoration: underline" >
	Kyoko Matsui</span>
	<sup>1,2</sup>, 

	<span  >
	Masaru Ohme-Takagi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Research Institute of Genome-based Biofactory, National Institute of Advanced Industrial Science and Technology (AIST), 

	<sup>2</sup>GreeenSogna, Inc. http://greensogna.com

</div>

<div>A protein complex composed of MYB and bHLH transcription factors associated with a WD40 repeat protein has been shown to regulate flavonoid biosynthesis in plants. A similar regulatory complex specifies epidermal cell fate. In Arabidopsis, CPC, TRY and ETC1, which are single MYB domain transcription factors and lack transactivation domain, were found to be negative regulator for epidermal cell differentiation. However a single&#45;MYB domain&#45;type transcription factor that is involved in the flavonoid biosynthesis has not been identified. Recently we showed that the PAP1 chimeric repressor, in which the EAR&#45;motif repression domain was fused to PAP1 &#40;35S:PAP1SRDX&#41;, inhibited the accumulation of proanthocyanidin and anthocyanin. The seeds of 35S::PAP1SRDX plants exhibited light yellow due to suppression of the accumulation of proantocyanidin. To isolate a single&#45;MYB domain&#45;type transcription factor that is involved in the regulation of flavonioid biosynthesis, we prepared transgenic plants that expressed the chimeric repressor for a single&#45;MYB domain protein and found that the expression of the chimeric AtMYBL2 repressor induced pale&#45;color in seeds. The ectopic expression of AtMYBL2 &#40;35S:AtMYBL2&#41; also suppressed accumulation of proantocyanidin in the seeds. In addition, accumulation of anthocyanine in leaves of both  35S::AtMYBL2 and 35S::AtMYBL2SRDX plants were lower than those of wild type plants, indicating that AtMYBl2 acts as transcriptional repressor.  Transient assay revealed that AtMYBL2 acted as a transcriptional repressor and the six amino acids at the carboxy&#45;terminal region sufficiently confer the repression activity, which is a novel repression domain different from known EAR&#45;motif. The 35S:AtMYBL2&#916;C transgenic plants and AtMYBL2 knockout lines exhibited an increase of anthocyanin accumulation in rosette leaves. We will discuss a possible role of AtMYBL2 on the regulation of flavonoid biosynthesis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Global Measurement of Plant Protein Turnover</div>

<div>

	<span style="text-decoration: underline" >
	Wen-Ping Chen</span>
	<sup>1</sup>, 

	<span  >
	Xiaoyuan Yang</span>
	<sup>2</sup>, 

	<span  >
	Jerry Cohen</span>
	<sup>1</sup>, 

	<span  >
	William Gray</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Department of Horticultural Science, University of Minnesota, St. Paul, MN 55108, USA, 

	<sup>2</sup>Department of Plant Biological Sciences, University of Minnesota, St. Paul, MN 55108, USA

</div>

<div>An increasing body of evidence has revealed that protein turnover is critical to the cellular regulatory processes that allow cells to not only rapidly respond to intracellular signal molecules but also to changing environmental conditions. In our lab, we are developing isotopic labeling strategies coupling with high&#45;throughput mass spectrometry that will enable the measurement of dynamic protein turnover for plant proteins with the ultimate application for use as a proteome&#45;wide fingerprinting technique. These procedures should result in sensitive measures of protein turnover profiles which would give us a powerful new insight into protein metabolism in plants and its importance on plant growth, development and adaptation to changing environmental conditions. &#13;<br/> &#13;<br/>So far, we have been working on the construction of plant growth systems where labeling experiments using Arabidopsis with 13CO2 and 15N&#45;nitrogen sources will be performed. As part of experiments to optimize the systems, once plants are grown in these systems in the presence of labeled primary nutrient sources &#40;15N nitrate or ammonia, 13CO2&#41;, the enrichment in individual amino acids will be determined by GC&#45;MS analysis of the free amino acids as well as protein hydrolysates. In addition, enrichment in a series of targeted proteins will be monitored by their isolation using immuno&#45;affinity columns and analysis by MALDI&#45;TOF or LC&#45;MS&#45;MS. The developed methods will be then used to investigate the kinetics of protein turnover for a set of proteins, for example those in the auxin biosynthetic and signaling pathways, by examining changes in label loss after a pre&#45;labeling period. Labeling at high enrichments will be compared to subtle modifications to determine which methods provide the best overall biological and statistical significance. Such experiments with known proteins should greatly help define the suitability of each method for further protein turnover studies on larger scale. If successful, we will have established methods with good general applicability and, importantly, with limited artifacts.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Strategy for plant metabolic systems analysis: glucosinolate and amino acid biosynthesis in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Yuji Sawada</span>
	<sup>1</sup>, 

	<span  >
	Kazuki Saito</span>
	<sup>1,2</sup>, 

	<span  >
	Masami Yokota-Hirai</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama, Kanagawa, Japan, 

	<sup>2</sup>Graduate School of Pharmaceutical Sciences, Chiba University, Chiba, Japan

</div>

<div>Integration of transcriptomics and metabolomics enabled us to identify the co&#45;expressed genes and co&#45;accumulated metabolites in <span style="font-style: italic">Arabidopsis</span> under sulfur deficiency condition &#91;1&#93;.  The remarkable co&#45;expression patterns were very similar to the condition&#45;independent co&#45;expression patterns clarified by using publicly&#45;available Arabidopsis DNA array data sets.  We assumed that the co&#45;expressed genes are involved in the same metabolic pathway.  In this study, co&#45;expression analysis was applied to a group of genes encoding Methionine&#45;derived glucosinolate &#40;MET&#45;GSL&#41; biosynthesis enzymes and their transcriptional factors.  The candidate genes, which were thought to encode the enzymes catalyzing the sequential four steps in MET side&#45;chain elongation &#91;methylthioalkylmalate synthase &#40;MAM&#41;, methylthioalkylmalate isomerase &#40;MAM&#45;I&#41;, methylthioalkylmalate dehydrogenase &#40;MAM&#45;D&#41; and methionine&#45;analog aminotransferase &#40;MAAT&#41;&#93;, were successfully predicted based on co&#45;expression pattern with known MET&#45;GLS biosynthesis genes.  Interestingly, the candidate genes of MET side&#45;chain elongation enzymes &#40;MET&#45;ELONG&#41; are homologs of the genes coding for leucine biosynthesis enzymes &#40;isopropylmalate synthase, isopropylmalate isomerase, isopropylmalate dehydrogenase, and branched chain aminotransferase&#41;.  The predicted gene functions were confirmed by the changes in metabolite profiles, i.e., glucosinolate and amino acid profiles, of T&#45;DNA insertion mutants of corresponding genes.  That is, in the MET&#45;ELONG gene knock&#45;out mutants, the levels of MET&#45;GSLs with elongated side&#45;chain markedly decreased.  Using a similar strategy, we have also identified the two genes encoding Myb transcription factors that positively control the biosynthesis of MET&#45;GSL &#91;2&#93;.  These results suggest that a metabolic system, MET&#45;GSL biosynthesis and its regulation, is constituted of these and other candidate genes we found.  Further experiments on these genes are being conducted to confirm our prediction.&#13;<br/>&#91;1&#93; Hirai et al. &#40;2005&#41; JBC 280:25590 &#13;<br/>&#91;2&#93; Hirai et al. &#40;2007&#41; PNAS 104:6478</div>

</div>





<div class="abs">

<div style="font-weight:bold">Metabolite profiling of the plant <span style="font-style: italic">RCD1&#45;SRO</span> gene family: amino acids and TCA cycle intermediates</div>

<div>

	<span style="text-decoration: underline" >
	Nina Sipari</span>
	<sup>1</sup>, 

	<span  >
	Tiina Blomster</span>
	<sup>2</sup>, 

	<span  >
	Markku Kein&#228;nen</span>
	<sup>1</sup>, 

	<span  >
	Jaakko Kangasj&#228;rvi</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>University of Joensuu, Joensuu, Finland, 

	<sup>2</sup>University of Helsinki, Helsinki, Finland

</div>

<div>The <span style="font-style: italic">rcd1&#45;1 &#40;radical&#45;induced cell death1&#41;</span> mutant of <span style="font-style: italic">Arabidopsis</span> has been shown to be defective in the containment of programmed cell death and in the signalling of several plant hormones. The RCD1 protein function is yet unknown, but according to a yeast two&#45;hybrid analysis, it may include interactions with several stress&#45;related transcription factors. <span style="font-style: italic">RCD1</span> belongs to a novel gene family with 5 unknown genes encoding proteins distinctively similar to RCD1 &#40;SRO1&#45;SRO5; SIMILAR TO RCD&#45;ONE 1&#45;5&#41;. Interestingly, a conserved domain of ADP&#45;ribosylation has been assigned to all the RCD1&#45;SRO proteins. RCD1 appears to have partially overlapping functions with at least SRO1, because <span style="font-style: italic">rcd1&#45;sro1</span> double mutant plants have a severely stunted phenotype even in control conditions. The role of markedly increased SRO3 expression in <span style="font-style: italic">rcd1</span> will also be further studied.  &#13;<br/>In addition to global gene expression with microarrays, we have accomplished metabolite profiling of the T&#45;DNA insertion mutants of the &#34;RCD1&#45;SRO&#34; gene family, <span style="font-style: italic">rcd1&#45;1</span>, and <span style="font-style: italic">Col</span> wildtype, both after stress treatments and in control condition by HPLC&#45;MSn and GC&#45;MS. Recent HPLC&#45;MSn and GC&#45;MS analysis have revealed differences between <span style="font-style: italic">Col</span> and <span style="font-style: italic">rcd1&#45;1</span> in the diurnal rhythm of some amino acids and their TCA cycle intermediates, respectively. The observed metabolic changes are discussed in respect to the latest results obtained from microarrays.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">Understanding the complex machinery responsible for cuticle assembly in plants</div>

<div>

	<span style="text-decoration: underline" >
	David Panikashvili</span>
	<sup>1</sup>, 

	<span  >
	Sigal Savaldi-Goldstein</span>
	<sup>2</sup>, 

	<span  >
	Tali Mandel</span>
	<sup>1</sup>, 

	<span  >
	Tamar Yifhar</span>
	<sup>1</sup>, 

	<span  >
	Rochus Franke</span>
	<sup>3</sup>, 

	<span  >
	Ren&#233; H&#246;fer</span>
	<sup>3</sup>, 

	<span  >
	Lukas Schreiber</span>
	<sup>3</sup>, 

	<span  >
	Joanne Chory</span>
	<sup>2</sup>, 

	<span  >
	Asaph Aharoni</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Weizmann Institute of Science, Rehovot, Israel, 

	<sup>2</sup>The Salk Institute, La Jolla, CA, USA, 

	<sup>3</sup>Institute of Cellular and Molecular Botany, Bonn, Germany

</div>

<div>The aerial organs of plants are covered with a continuous layer overlaying the outermost cell walls of the epidermis, termed the cuticle. Cutin, the main component of the cuticle, is composed of various interesterified hydroxy&#45; and epoxy&#45; C16&#45;18 &#40;un&#41;saturated fatty acids. It is embedded with intracuticular and covered with epicuticular waxes consisting largely of derivatives of very long chain fatty acids. Metabolic routes involved in cuticle assembly may be divided into biosynthetic and transport pathways. The transport pathway comprises the extrusion of cutin monomers from their site of biosynthesis to the extracellular space through the plasma membrane and cell wall. The molecular mechanisms of this process are poorly understood. Here, we report on the isolation and characterization of the &#34;desperado&#34; mutant in &#34;Arabidopsis&#34;, which exhibits an array of severe surface defects including postgenital organ fusion and epidermal ruptures suggesting a malfunctional cuticle. Transmission Electron Microscopy revealed unusual cytoplasmatic inclusions in leaf and stem epidermal cells and disappearance of the cuticle in fusion areas. Loss of &#34;DESPERADO&#34; function resulted in a striking phenotype, characterized by altered morphology of stomatal cells. &#34;DESPERADO&#34; encodes a putative ABC transporter and is localized to the plasma membrane. Chemical analysis showed that total cutin monomers load was reduced in the desperado knockout line. &#34;DESPERADO&#34; expression is induced by abscisic acid &#40;ABA&#41;, salt and mechanical wounding. Consistent with this observation, the &#34;desperado&#34; mutant is susceptible to salt and shows a reduction in root branching, a phenomenon associated with increased ABA and salty environment. Thus, we provide evidence that &#34;DESPERADO&#34; is required for the export of cutin monomers to the surface through the plasma membrane of epidermis cells. Yet, it is possible that &#34;DES&#34; might interact with a non&#45;specific lipid transfer protein that could carry out cutin transport through the hydrophobic cell wall.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Seeing in the dark: Integrating metabolite, transcript and enzyme measurements during diurnal cycles and prolonged darkness</div>

<div>

	<span style="text-decoration: underline" >
	Bjoern Usadel</span>
	<sup>1</sup>, 

	<span  >
	Yves Gibon</span>
	<sup>1</sup>, 

	<span  >
	Axel Nagel</span>
	<sup>2</sup>, 

	<span  >
	Oliver Blaesing</span>
	<sup>1</sup>, 

	<span  >
	Jan Hannemann</span>
	<sup>1</sup>, 

	<span  >
	Peter Krueger</span>
	<sup>1</sup>, 

	<span  >
	Dirk Walther</span>
	<sup>1</sup>, 

	<span  >
	Mark Stitt</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max-Planck Institute of Molecular Plant Physiology, Golm, Germany, 

	<sup>2</sup>RZPD Deutsches Ressourcenzentrum f&#252;r Genomforschung, Berlin, Germany

</div>

<div>To investigate the changes of the metabolism in Arabidopsis plants during diurnal cycles and after transfer into darkness, metabolite and transcript levels as well as enzymatic activities were measured. The data were analyzed both globally as well as by focusing on potential key pathways within the data set. <br/>The MapMan software family was used for data analysis in order to<br/>a&#41; normalize microarray data and to statistically evaluate simple experiment designs;<br/>b&#41; visualize data in the context of pre&#45;existing biological knowledge using transcript, metabolite and protein&#47;enzyme profiling experiments;<br/>c&#41; compress large experiments down to a simple 1&#45;2 page display by using summary statistics on functional categories using statistics such as pathway enrichment analysis;<br/>d&#41; investigate into the data in the light of other experiments.<br/><br/>Using this approach, for several pathways such as nitrogen metabolism or phospholipid biosynthesis, an agreement at all three functional levels was found. However, for some pathways where only metabolites and transcripts were measured,  discrepancies were sometimes evident, indicative of possible  posttranscriptional modification of enzymes. <br/><br/>Furthermore, a global view of the experiments was obtained by using correlative approaches between metabolites and transcripts. Many metabolite&#45;transcript correlations could be identified, and for several, these could be explained by investigating the respective pathway context. Interestingly, most metabolite&#45;transcript correlations could be identified for sugars, indicating the pivotal role of sugars during the investigated  conditions.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Monoacylglycerols are Components of Root Waxes and can be Produced in the Aerial Cuticle by Ectopic Expression of a Suberin&#45;associated Acyltransferase</div>

<div>

	<span style="text-decoration: underline" >
	Yonghua Li</span>
	<sup>1</sup>, 

	<span  >
	Fred Beisson</span>
	<sup>1</sup>, 

	<span  >
	John Ohlrogge</span>
	<sup>1</sup>, 

	<span  >
	Mike Pollard</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Biology, Michigan State University, East Lansing, MI, USA

</div>

<div>The interface between plants and the environment is provided for aerial organs by epicuticular waxes that have been extensively studied. By contrast, little is known about the nature, biosynthesis and role of the waxes at the root&#45;rhizosphere interface. Waxes isolated by rapid immersion of Arabidopsis roots in organic solvents were rich in saturated C18&#45;C22 alkyl esters of p&#45;hydroxycinnamic acids, but also contained significant amounts of both alpha&#45; and beta&#45;isomers of monoacylglycerols &#40;MAGs&#41; with C22 and C24 saturated acyl groups, and the corresponding free fatty acids &#40;FFAs&#41;. The production of these compounds in root waxes was positively correlated to the expression of GPAT5, a gene encoding an acyltransferase previously shown to be involved in suberin synthesis. This suggests a direct metabolic relationship between suberin and some root waxes. Furthermore, when ectopically expressed in Arabidopsis, GPAT5 produced very long chain saturated MAGs and FFAs as novel components of cuticular waxes. The crystal morphology of stem waxes was altered and the load of total stem wax compounds was doubled while the major components of wild&#45;type waxes decreased. These results indicate that GPAT5 functions in vivo as an acyltransferase to a glycerol&#45;containing acceptor and has access to the same pool of acyl intermediates and&#47;or may be targeted to the same membrane domain as that of wax synthesis in aerial organs. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of Arabidopsis acyltransferase mutants affected in cutin synthesis</div>

<div>

	<span style="text-decoration: underline" >
	Fred Beisson</span>
	<sup>1</sup>, 

	<span  >
	Yonghua Li</span>
	<sup>1</sup>, 

	<span  >
	Abraham Koo</span>
	<sup>1</sup>, 

	<span  >
	John Ohlrogge</span>
	<sup>1</sup>, 

	<span  >
	Mike Pollard</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Biology, Michigan State University, East Lansing, MI, USA 48824

</div>

<div>Cutin and suberin are the two major cell wall&#45;associated lipid polymers of plants. Cutin is a fatty acid&#45; and glycerol&#45;based polyester that constitutes the insoluble polymer matrix of the epidermal cuticle. The cutin polymer is embedded and covered with waxes.&#13;<br/>Major issues of cutin research include the identification of biosynthetic enzymes &#40;only 3 genes known so far&#41;, the intra&#45; or extra&#45;cellular site&#40;s&#41; of polymerization of monomers, the mechanism of transport of cutin monomers, oligomers or polymers through the cell wall. &#13;<br/>We have obtained Arabidopsis cutin mutants by an approach combining microarray analysis of the transcriptome of Arabidopsis stem epidermis and reverse genetics. Cutin content was measured by a GC&#47;MS method developed in our lab that allows routine analysis of Arabidopsis cutin. Single T&#45;DNA insertional knock&#45;outs for the acyltransferases GPAT4 and GPAT8 had no changes in cutin load and cuticle permeability. However, double KOs gpat4,gpat8 showed:&#13;<br/>&#45;a 60&#37;&#45;70&#37; reduction in cutin in leaves and stems&#13;<br/>&#45;normal amounts of cuticular waxes and intracellular lipids&#13;<br/>&#45;the replacement of the normal electron&#45;dense cuticle by a thicker but less compact structure&#13;<br/>&#45;a steep increase in water loss&#13;<br/>&#45;increase susceptibility to infection by the fungus Alternaria brassicicola .&#13;<br/>These results demonstrate that the acyltransferases GPAT4 and GPAT8 are functionally redundant and essential for cutin synthesis.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Glyco&#45;activation of a pro&#45;drug in vivo</div>

<div>

	<span style="text-decoration: underline" >
	Yang Zhao</span>
	<sup>1</sup>, 

	<span  >
	Tszfung Chow</span>
	<sup>1</sup>, 

	<span  >
	Rachel Puckrin</span>
	<sup>1</sup>, 

	<span  >
	Simon Alfred</span>
	<sup>1</sup>, 

	<span  >
	Sean Cutler</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Department of Cell and Systems Biology, University of Toronto, 

	<sup>2</sup>Center for Plant Cell Biology, University of California, Riverside

</div>

<div>Sequence variation in human drug metabolism and target genes can cause pharmacogenetic variation. Given the extensive metabolic variation documented between Arabidopsis accessions, we reasoned that Arabidopsis would be a powerful system for identifying natural alleles that modulate drug sensitivity, which in turn could be used to unravel the mechanism of action of compounds discovered in chemical genetic screens. Toward this goal, we subjected several Arabidopsis accessions to chemical genetic screens for inhibitors of cell expansion and discovered 12 molecules that induce strong strain&#45;selective effects. The natural variation approach revealed that hypostatin, a new inhibitor of etiolated hypocotyl cell expansion, requires in vivo glycosylation for bioactivty.  Many Arabidopsis strains are resistant to hypostatin due to mutations in HYR1, which encodes a UGT that glycosylates hypostatin, thus activating it. We additionally show that a prevalent SNP is sufficient for hypostatin resistance and, like other pharmacogenetic factors, HYR1 modulates sensitivity to multiple xenobiotics. Our results show that pharmacogenetic variation can be uncovered efficiently with small molecule screens and that natural alleles can yield mechanistic insights for new bioactive compounds. Because HYR1 is a member of the UGT&#45;superfamily of enzymes that are pharmacogenetic factors in humans, our results demonstrate that intra&#45;specific variation in UGT&#45;function modulates xenobiotic sensitivity across biological kingdoms.</div>

</div>





<div class="abs">

<div style="font-weight:bold">New properties of the glucosinolate&#45;myrosinase defense system discovered by vacuolar proteomics</div>

<div>

	<span style="text-decoration: underline" >
	Sixue Chen</span>
	<sup>1</sup>, 

	<span  >
	Jean-Emmanuel Sarry</span>
	<sup>2</sup>, 

	<span  >
	Richard Collum</span>
	<sup>2</sup>, 

	<span  >
	Emily Wang</span>
	<sup>1</sup>, 

	<span  >
	Chao-Xing Yuan</span>
	<sup>2</sup>, 

	<span  >
	Philip A. Rea</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>University of Florida, Gainesville, FL32605, USA, 

	<sup>2</sup>University of Pennsylvania, Philadelphia, PA19104, USA

</div>

<div>Processes that converge on the vacuole, one of the largest organelles known, include pH and regulatory Ca2&#43; homeostasis, nutrient storage, xenobiotic sequestration and processing, volume regulation, and macromolecule turnover and salvage. If we are to understand how plant cells work at the systems level, and if the latter are to be engineered for enhanced nutritional quality and for the provision of environmentally friendly source of fine chemicals and pharmaceuticals, such as glucosinolates, it will be critical that researchers have ready access to a vacuolar proteomic toolbox.  Knowing this, we have refined techniques for the purification of &#8220;proteomics&#45;grade&#8221; intact vacuoles inclusive of their luminal protein complement from the model plant Arabidopsis thaliana. We have clearly resolved 270&#45;300 polypeptides from the vacuolar lumen of Arabidopsis by 2&#45;dimensional gel electrophoresis and identified relatively abundant proteins by MALDI&#45;TOF&#45;MS and ESI&#45;LCQ&#45;MS. Among the proteins are proteins which have been established to have a vacuolar localization, for instance aspartic proteinase, cysteine protease, leucine aminopeptidase, mannosidase, vegetative storage protein VSP1, and others, such as SOUL&#45;like proteins which might be expected to have a vacuolar localization, but which were not known to be such, and several proteins of unknown functions. &#13;<br/>&#13;<br/>It is interesting to note that myrosinase TGG1, TGG2, myrosinase associate proteins &#40;MyAP&#41; and myrosinase binding proteins &#40;MBP&#41; were all identified in the vacuoles isolated from rosette leaves. TGG1 are present as several glycosylated forms. At early developmental stage, TGG1 was more abundant than TGG2, which was only present as a small spot on 2D gels. In fully expanded leaves, both TGG1 and TGG2 levels increased. Concurrently, MyAP became a dominant spot on the 2D gel map. The co&#45;localization of myrosinases with interacting proteins and the concurrent expression during development imply important regulatory mechanism of the myrosinase&#45;glucosinolate system in plant development and interactions with environment factors. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Engineering of &#180;&#180;Arabidopsis thaliana&#180;&#180; FMOs increases production of the cancer&#45;preventive isothiocyanates</div>

<div>

	<span style="text-decoration: underline" >
	Jing Li</span>
	<sup>1</sup>, 

	<span  >
	Bjarne Gram Hansen</span>
	<sup>1</sup>, 

	<span  >
	Daniel J.Kliebenstein</span>
	<sup>2</sup>, 

	<span  >
	Barbara Anne Halkier</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Copenhagen University, Copenhagen, Denmark, 

	<sup>2</sup>University of California-Davis, Davis,USA

</div>

<div>A high dietary intake of cruciferous vegetables has been associated with a reduction in cancers, and isothiocyanates &#40;ITCs&#41; derived from the natural products glucosinolates &#40;GSLs&#41; are considered to be responsible for the cancer&#45;preventive action. Epidemiological studies have shown that ITCs derived from methylsulfinylalkyl GSLs particularly sulforaphane&#40;the isothiocyanate hydrolysis product from methionine&#45;derived 4&#45;methylsulfinylbutyl GSL&#41;has been identified as a very potent cancer&#45;preventive agent. Here we report identification of a cluster of 5 &#180;&#180;Arabidopsis&#180;&#180; flavin&#45;monooxygenase &#40;FMO&#41; genes, FMO&#180;&#180;GS&#45;OX1&#45;5&#180;&#180;, encoding for enzymes responsible for the S&#45;oxygenation of methylthioalkyl into methylsulfinylalkyl GSLs. This is evidenced by biochemical characterization of recombinant proteins and analyses of the GSL profile in FMO over&#45;expression lines and FMO knock&#45;out mutants. As an example, FMO&#180;&#180;GS&#45;OX1&#180;&#180; over&#45;expression lines show almost complete conversion of methylthioalkyl into methylsulfinylalkyl GSLs, with a &#126; 5 fold increase in 4&#45;methylsulfinylbutyl GSL in seeds. Our finding represents the first identification of S&#45;oxygenating FMOs in plants and provides an important molecular tool for breeding of Brassica vegetable crops with increased levels of these important GSLs. This has implications for production of functional foods enriched with cancer&#45;preventive ITCs, including the well&#45;known sulforaphane. &#13;<br/>&#13;<br/>B.G., Kliebenstein, D, and Halkier, B. A, The Plant Journal &#40;in press&#41;. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis acyl&#45;CoA binding proteins ACBP4 and ACBP5 are localized in the cytosol</div>

<div>

	<span  >
	Shi Xiao</span>
	<sup>1</sup>, 

	<span  >
	Hong-Ye Li</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Mee-Len Chye</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Botany, The University of Hong Kong, Pokfulam, Hong Kong

</div>

<div>Arabidopsis acyl&#45;CoA binding proteins &#40;ACBPs&#41; are encoded by a gene family consisting of six members. They range in size from 10.4 kDa to 73.1 kDa and show conservation at the acyl&#45;CoA binding domain. We have demonstrated that they display varying affinities to acyl&#45;CoA esters and show different subcellular localizations, suggesting that they do not have redundant roles in plant lipid metabolism. While ACBP1 and ACBP2 are membrane&#45;associated proteins, ACBP3 is targeted extracellularly. Here we report that ACBP4 and ACBP5 are localized in the cytosol. Each of ACBP4 and ACBP5 was translationally&#45;fused to an autofluorescent protein for particle gun bombardment of onion epidermal cells and tobacco BY&#45;2 cells, and for agroinfiltration of tobacco leaves. Observations by confocal laser scanning microscopy revealed that ACBP4::DsRed, ACBP5::DsRed and ACBP5::GFP fusions were present predominantly in the cytosol. To further confirm the localizations of ACBP4 and ACBP5, stable transgenic plants expressing eGFP::ACBP4, ACBP5::eGFP and ACBP5::Red were generated. Confocal microscopy and western blot analyses using subcellular protein fractions from transgenic Arabidopsis revealed that the ACBP4 and ACBP5 fusions occur in the cytosol. Hence, other than the well&#45;studied 10&#45;kDa cytosolic ACBP that is prevalent in eukaryotes, we have identified two more cytosolic ACBPs in Arabidopsis. ACBP4 and ACBP5, which binds oleoyl&#45;CoA ester, could participate in its transfer from the plastid &#40;the site of de novo fatty acid biosynthesis&#41; to the endoplasmic reticulum for the eukaryotic pathway in lipid biosynthesis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification and characterization of mono&#45;ADP&#45;ribosylation activity in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Hai Wang</span>
	<sup>1</sup>, 

	<span  >
	Kaiming Cao</span>
	<sup>1</sup>, 

	<span  >
	Xiaochun Ge</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biochemistry and Molecular Biology, School of Life Sciences, Fudan University, Shanghai, 200433, China

</div>

<div>ADP&#45;ribosylation is a novel post&#45;translational modification involved in diverse cellular processes such as signal transduction, cell differentiation and protein trafficking. It could be divided into two types: mono&#45;ADP&#45;ribosylation and poly&#45;ADP&#45;ribosylation. While mono&#45;ADP&#45;ribosylation is catalyzed by mono&#45;ADP&#45;ribosyltransferases &#40;mADPRTs&#41; which transfer only a single ADP&#45;ribose residue from &#946;NAD&#43; to a specific amino acid of the acceptor protein, poly&#45;ADP&#45;ribosylation is catalyzed by poly&#40;ADP&#45;ribose&#41; polymerases &#40;PARPs&#41; which transfer multiple ADP&#45;riboses to target proteins. Interestingly, both mono&#45; and poly&#45;ADP ribosylation could be reversed, either by enzyme activities which could hydrolyze the protein&#45;ADP&#45;ribose linkage or by those that degrade poly&#45;ADP&#45;ribose, suggesting the existence of ADP&#45;ribosylation&#47;de&#45;ADP&#45;ribosylation loops as a regulatory mechanism for protein substrates of these reactions.&#13;<br/>&#13;<br/>Although PARPs have been extensively studied in various organisms &#40;including plants&#41; and have been shown to affect stress tolerance, cell death and energy homeostasis, the characterization of mono&#45;ADP&#45;ribosyltransferase activity has only been reported in bacteria, fungi, and the animal kingdom. However, in a trial to study mono&#45;ADP&#45;ribosylation activity in <span style="font-style: italic">Arabidopsis thaliana</span>, we identified a protein with molecular mass of 32&#45;KDa which is likely the protein substrate of endogenous mono&#45;ADP&#45;ribosylation reaction. The modified amino acid of the protein was further investigated by means of characterizing the chemical stability of the ADP&#45;ribosyl linkage. After treatment of the &#91;32P&#93;ADP&#45;ribosylated 32&#45;KDa protein with NaOH or HCOOH for 2h, the 32&#45;KDa band disappeared, as opposed to treatment with HgCl2 &#40;which act on ADP&#45;ribosylated cysteine&#41; or NH2OH &#40;specific to ADP&#45;ribosylated arginine&#41;, both of which had no significant effect on the 32&#45;KDa band. These results indicate that the ADP&#45;ribose residue is most likely linked to serine or threonine residues. Further investigation is needed to characterize the identity of this protein as well as the enzyme which catalyzes this reaction.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Recent Advances in Plant Cell Wall Biology</div>

<div>

	<span style="text-decoration: underline" >
	geoffrey fincher</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Australian Centre for Plant Functional Genomics, University of Adelaide, Waite Campus, Glen Osmond, SA 5064, Australia.

</div>

<div>Cell walls are central determinants of plant form, growth and development, and change in response to environmental and pathogen&#45;induced stresses. Walls play important roles in the quality of plant&#45;based foods for both human and animal consumption, and in the production of fibres during pulp and paper manufacture. In the future, wall remnants in crop residues could be used as a source of renewable fuel. Cellulose, a range of non&#45;cellulosic polysaccharides, proteins and phenolic compounds are the major components of plant cell walls. They form covalently and non&#45;covalently linked molecular networks that have the strength, flexibility and porosity necessary for cellular function. The fine structures and sizes of wall polysaccharides can be altered following deposition into the wall, to accommodate changing physicochemical requirements as cells grow and differentiate. The chemical structures of most wall polysaccharides have been defined in detail, but the enzymes involved in their synthesis and re&#45;modelling remain largely undefined. While there have been real advances in our understanding of cellulose biosynthesis in plants, with few exceptions the identities and modes of action of polysaccharide synthases and other glycosyltransferases that participate in the biosynthesis of the non&#45;cellulosic wall polysaccharides are not known. Emerging functional genomics, molecular genetics and X&#45;ray crystallographic technologies are allowing us to re&#45;examine the central questions related to wall biosynthesis. For example, the availability of the rice, Populus and Arabidopsis genome sequences, various mutant populations, high density genetic maps for cereals, high throughput genome and transcript analysis systems, extensive publicly available genomics resources, and an increasing array of analytical systems for the definition of candidate gene function, permit a broad, non&#45;biased approach to the description of wall biosynthesis in plants. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Metabolomics&#45;based functional genomics in Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Kazuki Saito</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, Yokohama, Japan, 

	<sup>2</sup>Graduate School of Pharmaceutical Sciences, Chiba University, Chiba, Japan

</div>

<div>Metabolomics is a rapidly&#45;emerging sector of post&#45;genome research. The metabolome &#40;a set of all metabolites of an organism&#41; represents not only the ultimate phenotype of cells by the perturbation of gene expression and the modulation of protein functions caused by the environment or mutations, but the metabolome can also feed back on  gene expression and protein function. Therefore, metabolomics plays a key role for understanding cellular systems. The completion of the whole genome sequence of Arabidopsis thaliana has made it possible to discover the genes involved in metabolism in a high throughput manner by determining gene&#45;to&#45;metabolite correlation through the comprehensive analysis of metabolite accumulation and gene expression. In this seminar I will present the aims, rationale and some examples of the identifying gene function of Arabidopsis by such analysis. In silico co&#45;expression analysis of genes involved in flavonoid metabolism in Arabidopsis was performed using a publicly available transcriptome database of DNA microarrays. We inferred a co&#45;expression framework model of the genes involved in the pathways of flavonoids, suggesting specific functions and co&#45;regulation of the genes of pathway enzymes and transcription factors. Changes in flavonoid profiles of wild&#45;type plants and T&#45;DNA insertion mutants of the delimited genes led to the confirmation of gene function &#40;Yonekura&#45;Sakakibara et al., J Biol Chem, doi:10.1074&#47;jbc.M611498200 &#40;2007&#41;&#41;. We also applied this strategy to glucosinolate biosynthetic pathway for identification of MYB transcription factors crucial for aliphatic glucosinolate production &#40;Hirai et al., PNAS, 104, 6478 &#40;2007&#41;&#41;. These results suggest that the functional genomics approach by integration of metabolome with transcriptome provides an efficient way of identifying novel gene functions involved in plant metabolism.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional analysis of rice CYP85</div>

<div>

	<span style="text-decoration: underline" >
	BoKyung Kim</span>
	<sup>1</sup>, 

	<span  >
	Sunghwa Choe</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of biological sciences, College of Natural Sciences, Seoul National University, Seoul 151-747, Korea

</div>

<div>Brassinosteroids &#40;BRs&#41; collectively refer to steroidal plant hormones that are essential for growth and development of plants. It has been proposed that BRs are synthesized via two parallel pathways, the early and the late C&#45;6 oxidation pathways according to the order of C&#45;6 oxidation status. One is the early C&#45;6 oxidation pathway, in which oxidation at C&#45;6 occurs before the introduction of vicinal hydroxyls at C&#45;22 and C&#45;23 of the side chain. The other is the late C&#45;6 oxidation pathway in which C&#45;6 is oxidized after the introduction of hydroxyls on the side chain. The C&#45;6 oxidation of BR intermediates is catalyzed by the enzymes encoded by Cytochrome P450 85 &#40;CYP85&#41; genes. Arabidopsis CYP85 enzymes have been shown to catalyze C&#45;6 oxidation of 6&#45;deoxo intermediates. Interestingly, Arabidopsis CYP85A2 was also found to mediate the ultimate step which is Baeyer&#45;Villiger type oxidation of C&#45;6. However, the functions of rice CYP85 protein are still unknown. Therefore, we aimed to understand the function through feeding experiments with a yeast strain that is heterologously expressing the rice CYP85 gene. Feeding tests following GC&#45;MS based analyses revealed that both Arabidopsis CYP85A2 and rice CYP85 metabolize 6&#45;deoxo&#45;BRs into Castasterone. Further experiments with Castasterone will confirm whether OsCYP85 mediates the ultimate step or not.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis AtOPT3 oligopeptide transporter plays a role in iron homeostasis</div>

<div>

	<span  >
	Minviluz G. Stacey</span>
	<sup>1</sup>, 

	<span  >
	Ami Patel</span>
	<sup>1</sup>, 

	<span  >
	Sharon Pike</span>
	<sup>1</sup>, 

	<span  >
	Elizabeth E. Rogers</span>
	<sup>1</sup>, 

	<span  >
	Walter Gassmann</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Gary Stacey</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Missouri, Columbia, MO, USA

</div>

<div>The Arabidopsis AtOPT3 gene encodes a predicted transmembrane protein belonging to the oligopeptide transporter &#40;OPT&#41; family, a largely uncharacterized family of transporter proteins involved in the transport of short peptides or modified peptides. A null mutation in AtOPT3 &#40;opt3&#45;1&#41; resulted in embryo&#45;lethality, indicating an essential role for AtOPT3 in embryo development.  A second AtOPT3 mutant line, opt3&#45;2, harboring a T&#45;DNA insertion in the 5&#8217; UTR of AtOPT3, was recently identified. The T&#45;DNA insertion in the AtOPT3 promoter resulted in reduced but sufficient AtOPT3 expression to allow embryo formation in opt3&#45;2 homozygous seeds.   Phenotypic analyses of opt3&#45;2 plants revealed three interesting loss&#45;of function phenotypes associated with iron metabolism.  First, reduced AtOPT3 expression resulted in the constitutive expression of root iron&#45;deficiency responses in opt3&#45;2 plants regardless of exogenous supply of iron.  Second, deregulation of root iron uptake processes in opt3&#45;2 roots resulted in the accumulation of very high levels of iron in opt3&#45;2 tissues. Hyperaccumulation of iron in opt3&#45;2 resulted in the formation of brown necrotic areas in opt3&#45;2 leaves and was more pronounced during the seed filling stage. Third, reduced AtOPT3 expression resulted in decreased accumulation of iron in opt3&#45;2 seeds.  The reduced accumulation of iron in opt3&#45;2 seeds is especially noteworthy considering the excessively high levels of accumulated iron in other opt3&#45;2 tissues.  AtOPT3, therefore, plays a critical role in two important aspects of iron metabolism, namely, maintenance of whole&#45;plant iron homeostasis and iron nutrition of developing seeds.  We propose that AtOPT3 transports a peptide&#47;modified peptide iron chelator or Fe&#45;chelator complex to cells or organs that are involved in the mobilization of iron to seeds and in shoot&#45;to&#45;root signaling of iron&#45;deficiency responses.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional Analysis of a Peroxisomal Acyl&#45;Coenzyme A Synthetase&#13;<br/>Protein Family from <span style="font-style: italic">Arabidopsis thaliana</span> Uncovers Enzymes&#13;<br/>Participating in Jasmonic Acid Biosynthesis&#13;<br/></div>

<div>

	<span  >
	Lucie Kienow</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Katja Schneider</span>
	<sup>1</sup>, 

	<span  >
	Michael Bartsch</span>
	<sup>1</sup>, 

	<span  >
	Otto Miersch</span>
	<sup>2</sup>, 

	<span  >
	Claus Wasternack</span>
	<sup>2</sup>, 

	<span  >
	Erich Kombrink</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, Cologne, Germany, 

	<sup>2</sup>Leibniz-Institute of Plant Biochemistry, Halle, Germany

</div>

<div>The <span style="font-style: italic">Arabidopsis</span> genome contains a large number of carboxylic acid&#45;activating enzymes, including nine long&#45;chain fatty acyl&#45;CoA synthetases &#40;LACS&#41;, four 4&#45;coumarate:CoA ligases &#40;4CL&#41;, and nine proteins closely related to 4CLs with unknown functions &#40;4CLL&#41;. We systematically examined the function of the latter group by applying an extensive substrate screen to recombinant 4CLL enzymes. This analysis uncovered the activation of fatty acids of variable chain length as the common feature of all active members of this protein family, thereby defining a new group of fatty acyl&#45;synthetases, which is distinct from the known LACS. Significantly, four family members also displayed high activity towards different biosynthetic precursors of jasmonic acid &#40;JA&#41;, including 12&#45;oxo&#45;phytodienoic acid &#40;OPDA&#41;, dinor&#45;OPDA, 3&#45;oxo&#45;2&#40;2&#8217;&#45;&#91;<span style="font-style: italic">Z</span>&#93;&#45;pentenyl&#41;cyclopentane&#45;1&#45;octanoic acid &#40;OPC&#45;8&#41; and OPC&#45;6, but not OPC&#45;4 and JA. Detailed analysis of the <span style="font-style: italic">in vitro</span> substrate specificity uncovered substantial differences for the individual enzymes acting on JA precursors, identifying one unique OPC&#45;8:CoA ligase &#40;At1g20510&#41;. To further understand the <span style="font-style: italic">in vivo</span> function of At1g20510, an <span style="font-style: italic">Arabidopsis</span> insertion mutant for the gene was analyzed by transcript and jasmonate profiling.  We found that JA accumulation after wounding was reduced in the mutant whereas the precursors OPC&#45;8, OPC&#45;6 and OPC&#45;4 accumulated to enhanced levels. The analysis of <span style="font-style: italic">At1g20510</span>&#45;promoter::GUS reporter lines showed the induction of <span style="font-style: italic">At1g20510</span> expression after wounding or treatment with <span style="font-style: italic">Pseudomonas syringae</span>. Collectively, the results demonstrate that OPC&#45;8:CoA ligase catalyzes an essential step in JA biosynthesis by initiating the beta&#45;oxidative chain shortening of its precursors.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Isolation and characterization of novel phosphatidic acid phosphatases in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Yuki Nakamura</span>
	<sup>1</sup>, 

	<span  >
	Mami Tsuchiya</span>
	<sup>1</sup>, 

	<span  >
	Ryota Koizumi</span>
	<sup>1</sup>, 

	<span  >
	Hiroyuki Ohta</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology

</div>

<div>Phosphatidic acid phosphatase &#40;PAP&#41; hydrolyzes phosphatidic acid &#40;PA&#41; to yield diacylglycerol &#40;DAG&#41;. Since PAP in chloroplast mediates crucial step for synthesizing indispensable galactolipids, its activity was well&#45;characterized to date. In addition, recent identification of TGD1, a potent ER&#45;to&#45;chloroplast lipid transporter, emphasized the importance of PA as a central metabolite &#40;Xu et al., 2003, 2005&#41;. Despite the importance of PAP, however, its phylogenetic origin was unknown.&#13;<br/>Although Arabidopsis has four lipid phosphate phosphatases &#40;LPP1&#126;4&#41; &#40;Katagiri et al., 2005&#41; as mammalian LPP homologues, they are not predicted to be localized at chloroplast. Since no mammalian LPP homologues are identified in cyanobacteria, a possible ancestor of chloroplast, we hypothesized that there is a different type of LPP in cyanobacteria and Arabidopsis. Based on a distinct strategy, we identified a strong candidate for cyanobacterial PAP and its five homologues &#40;LPPbeta, LPPgamma, LPPdelta, LPPepsilon1 and LPPepsilon2&#41; in Arabidopsis which were distinct from LPP1&#126;4. Protein sequence analysis showed that these homologues are distant from known mammalian and Yeast LPP but close to cyanobacteria and other prokaryotes, suggesting that LPP may be categorized as either prokaryotic or eukaryotic subfamily. Localization analysis by specific antibodies showed that three of them &#40;LPPgamma, LPPepsilon1 and LPPepsilon2&#41; were localized to the chloroplast. They possessed significant PAP activity&#12288;and LPPgamma showed enzymatic properties mostly similar to that of native PAP activity in Arabidopsis chloroplasts. Expression study by RT&#45;PCR and GUS staining showed that LPPgamma was primarily expressed in shoot. Knock out mutants of LPPepsilon1, LPPepsilon2 and their double knock out, showed no detectable changes in lipid composition whereas single knock out mutant of LPPgamma was not isolated both by T&#45;DNA insertion and RNAi gene suppression, suggesting that complete loss of LPPgamma might be lethal. Latest results regarding the novel PAP will be presented.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A new class of membrane proteins as potential amino acid exporters&#47;facilitators</div>

<div>

	<span style="text-decoration: underline" >
	Friederike Ladwig</span>
	<sup>1</sup>, 

	<span  >
	Axel Hirner</span>
	<sup>1</sup>, 

	<span  >
	Wolfgang Koch</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ZMBP Plant Physiology, Tuebingen, Germany

</div>

<div>Nitrogen is one of the limiting nutrients in plants. In the phloem, nitrogen can be transported as amino acids from the site of biosynthesis to the sink. Concerning the cellular transport at the plasma membrane, all so far characterized amino acid transporters function as importers. The export mechanism out of the cell is still unclear. We suppose that a new class of membrane proteins, the N21&#45;proteins, act as potential exporters&#47;facilitators. These proteins are plasmamembrane localized. Initial experiments show that amino acids can be transported in both directions independently of pH and membrane potential, probably in a concentration&#45;dependant manner. Some of the proteins tested localize to specific regions in roots and developing seeds which supports the idea of a function as selective amino acid filters. A knockout mutant for one of the N21 genes displays a visible phenotype in developing seeds. Analytical analysis of amino acids in knockout plants in comparison to wildtype exhibits an altered amino acid composition. Proteins of the large N21 gene family are good candidates for amino acid facilitators&#47;exporters and they could function as counterparts for the great number of characterized import proteins.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of two ammonium transporters in a salt&#45;resistant green alga, <span style="font-style: italic">Dunaliella viridus</span></div>

<div>

	<span style="text-decoration: underline" >
	Ting Song</span>
	<sup>1</sup>, 

	<span  >
	Rentao Song</span>
	<sup>2</sup>, 

	<span  >
	Zhengkai Xu</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Shanghai Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China, 

	<sup>2</sup>School of Life Science, Shanghai University, Shanghai, China

</div>

<div>Ammonium and nitrate are primary nitrogen sources for plant. Nevertheless, ammonium is used preferentially over nitrate by most species. So ammonium transport is a key process in nitrogen metabolism. To elucidate the role of ammonium transporters in N nutrition of a salt&#45;resistant green alga, <span style="font-style: italic">Dunaliella viridus</span>, two ammonium transporter genes were isolated from cDNA libraries of <span style="font-style: italic">D.viridus</span> treated by different NaCl concentrations. Each cDNA has a long 3&#8217;&#45;UTR sequence, almost 1Kb. DvAMT1;1 has only 35.9&#37; amino acid identity with DvAMT1;2, which also appears in Chlamydomonas. When expressed in yeast mutants defective in ammonium uptake, however, only the latter complemented the defective phenotype. To provide a direct evidence for plasma membrane location for the two transporters, subcellular localization in Chalamydomonas has been done. Real Time PCR showed similar expression patterns for <span style="font-style: italic">DvAMT1;1</span> and <span style="font-style: italic">DvAMT1;2</span>, both of which were induced by low nitrogen and inhibited by high nitrogen, especially NH4&#43;. Different from ammonium transporters identified in higher plant, expression of these two ammonium transporter genes were not diurnal regulated.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Isolation of rice cDNAs for higher accumulation of pigments from rice FOX lines</div>

<div>

	<span style="text-decoration: underline" >
	Youichi Kondou</span>
	<sup>1</sup>, 

	<span  >
	Mika Kawashima</span>
	<sup>1</sup>, 

	<span  >
	Takanari Ichikawa</span>
	<sup>1</sup>, 

	<span  >
	Akie Ishikawa</span>
	<sup>1</sup>, 

	<span  >
	Yukako Hasegawa</span>
	<sup>1</sup>, 

	<span  >
	Yuko Makita</span>
	<sup>1</sup>, 

	<span  >
	Hirohiko Hirochika</span>
	<sup>2</sup>, 

	<span  >
	Minami Matsui</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN, Yokohama, Japan, 

	<sup>2</sup>NIAS, Tsukuba, Japan

</div>

<div>FOX hunting system is novel gain&#45;of&#45;function technique.  In this system, random over&#45;expression of a normalized plant&#8217;s full&#45;length cDNA library cause dominant gain&#45;of&#45;function mutations and it is expected that plant acquire useful phenotype.  We are establishing rice FOX lines using 13,000 non&#45;redundant rice full&#45;length cDNAs to introduce them into 'Arabidopsis' plants.  At present, we are screening these lines with various traits for exploration of useful rice genes.  In this study, we introduce screening strategy for isolation of mutants of higher accumulated pigments in rosette leaves.  We already isolated some mutants by screening in T1 generation from about 6,000&#45;7,000 lines.  Rice cDNAs isolated from these mutants encoded stress related genes, some enzymes, unknown proteins and so on.  Some of these rice cDNAs have, upon overexpression in 'Arabidopsis', re&#45;exhibited higher accumulation of pigments.  At present, we introduce these cDNAs into rice &#40;Nipponbare&#41;.<br/>  In addition, we analyzed microarray experiments by using these 'Arabidopsis' overexpressors to re&#45;isolate useful genes for higher accumulation of pigments from 'Arabidopsis' genome, and some commonly increased genes in these overexpressor were isolated as candidates.  And the categorization of gene ontology by using data sets of these microarray analyses should appear some pathway for regulation of pigments in 'Arabidopsis'.  We would discuss transcription factors, which should regulate these pathways.<br/>  This research is supported by the Special coordination funds for Promoting Science and Technology in Japan entitled Rapid identification of useful traits using rice full&#45;length cDNAs.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Investigating putative functions of peroxisomal proteins through microarray analysis</div>

<div>

	<span style="text-decoration: underline" >
	John D Bussell</span>
	<sup>1</sup>, 

	<span  >
	Itsara Pracharoenwattana</span>
	<sup>1</sup>, 

	<span  >
	Andy AG Wiszniewski</span>
	<sup>1</sup>, 

	<span  >
	Wenxu Zhou</span>
	<sup>1</sup>, 

	<span  >
	Steven M Smith</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ARC Centre of Excellence in Plant Energy Biology, University of Western Australia, 35 Stirling Highway, Crawley WA 6009, Australia

</div>

<div>Transcription of genes involved in specific pathways is often tightly regulated. For example using a time course microarray experiment, Smith et al. &#40;Plant Physiol 136, 2687&#45;2699, 2004&#41; demonstrated coordinated expression of several starch degradation enzymes during the leaf diurnal cycle, leading to the discovery of a new enzyme required for starch metabolism. <br/><br/>We used this same data set as a tool to probe peroxisomal function during the diurnal cycle &#40;12 h dark, 12 h light&#41;. Beginning with experimentally determined &#40;SUBA, http:&#47;&#47;www.plantenergy.uwa.edu.au&#47;applications&#47;suba&#47;index.php&#41; and bioinformatically&#45;predicted &#40;ARAPEROX, http:&#47;&#47;www.araperox.uni&#45;goettingen.de&#47;&#41; peroxisome proteins, we extracted microarray data for transcripts of genes involved in peroxisomal pathways. <br/><br/>Enzymes of photorespiration, including HPR, SHMT1 GGT1 and AGT are coordinately expressed. Their transcript levels vary approximately 2 fold with the maximum 4 h into the light period and the minimum 1 h into dark period. In contrast core beta&#45;oxidation genes &#40;LACS6, ACX4, MFP2, KAT2&#41; and some apparently associated transcripts &#40;CHY1&#41; experience a maximum during the dark period and reduced expression in the light. This observation probably reflects changing metabolic requirement and cycling of resources during the diurnal cycle. This analysis allowed us to assign PMDH2 to the photorespiratory gene set and PMDH1 to the beta&#45;oxidation set. Some transcripts encoding enzymes of unknown function have similar expression profiles to those of beta&#45;oxidation, implying a role in fatty acid metabolism. Other genes potentially associated with beta&#45;oxidation exhibit different expression patterns with maxima in the light period, including AIM1&#47;MFP1 and KAT5, and those involved in JA biosynthesis &#40;At4g05160, At5g63380, OPR3&#41;.<br/><br/>We have thus identified transcripts of peroxisomal proteins that are co&#45;expressed with photorespiration, B&#45;oxidation and other known peroxisomal pathways. This analysis provides new gene targets that are now the subject of genetic and functional studies.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Systems approach to understand integration of the isoprenoid pathway into the cellular gene network.</div>

<div>

	<span style="text-decoration: underline" >
	Eva  Vranov&#225;</span>
	<sup>1</sup>, 

	<span  >
	Marta Closa</span>
	<sup>2</sup>, 

	<span  >
	Anja  Wille</span>
	<sup>3</sup>, 

	<span  >
	Albert Ferrer</span>
	<sup>2</sup>, 

	<span  >
	Wilhelm Gruissem</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Sciences, ETH Zurich, 8092 Zurich, Switzerland, 

	<sup>2</sup>Faculty of Pharmacy, University of Barcelona, 08028 Barcelona, Spain, 

	<sup>3</sup>Reverse Engineering Group, ETH Zurich, 8092 Zurich, Switzerland

</div>

<div>Plant isoprenoids represent the largest group of biologically active metabolites with at least 30 000 chemical compounds. Plant isoprenoids have essential functions in physiological and biochemical processes &#40;e.g., photosynthesis, respiration, membrane fluidity, pathogen defense, key plant hormones &#45;cytokinins, brassinolides, gibberellic acid, abscisic acid; protein prenylation&#41;. Many plant isoprenoids also have substantial commercial, pharmacological and agricultural value &#40;e.g., essential oils, anticancer drug taxol, antimalarial drug artemisinin, polymers in rubber, antibiotics, carotenoid and tocopherol antioxidants, repellents&#41;. Understanding the regulation of the plant isoprenoid biosynthetic pathway and its integration into the cellular gene network is therefore not only of scientific but also of commercial interest. <br/> In our laboratory, we are using systems approaches to understand the isoprenoid pathway regulation as well as pathway integration into the cellular genetic network. To this end, using variety of publicly available genome&#45;wide gene expression data sets, we modeled the isoprenoid pathway network using sparse Gaussian graphical modeling &#40;Wille et al., 2004&#41;. This analysis revealed some novel unexpected dependencies between the genes in the pathway that can be further investigated. <br/>Another approach that we are undertaking to understand the pathway integration into the cellular network is the molecular analysis of responses upon systematic perturbation of the isoprenoid pathway using Arabidopsis loss&#45;of&#45;function mutants. This approach, as well as preliminary results will be discussed in more details at the meeting. <br/><br/>Wille A., Zimmermann P., Vranov&#225; E., Bleuler S., F&#252;rholz A., Hennig L., Laule O., Prel&#237;c A., von Rohr P., Thiele L., Zitzler E., Gruissem W. and B&#252;hlmann P. &#40;2004&#41; Sparse graphical gaussian modeling for genetic regulatory network inference. Genome Biology 5: R92.1&#45;R92.13<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Using metabolomics to decipher functions of Arabidopsis genes in the context of metabolic and regulatory networks</div>

<div>

	<span style="text-decoration: underline" >
	Basil Nikolau</span>
	<sup>1</sup>, 

	<span  >
	Julie Dickerson</span>
	<sup>1</sup>, 

	<span  >
	Philip Dixon</span>
	<sup>1</sup>, 

	<span  >
	Oliver Fiehn</span>
	<sup>2</sup>, 

	<span  >
	Bernd Markus Lange</span>
	<sup>3</sup>, 

	<span  >
	Seung Yon Rhee</span>
	<sup>4</sup>, 

	<span  >
	Vladimir Shulaev</span>
	<sup>5</sup>, 

	<span  >
	Lloyd Sumner</span>
	<sup>6</sup>, 

	<span  >
	Ruth Welti</span>
	<sup>7</sup>, 

	<span  >
	Eve Syrkin Wurtele</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Iowa State University, 

	<sup>2</sup>University of California, Davis, 

	<sup>3</sup>Washington State University, 

	<sup>4</sup>Carnegie Institution, Stanford, 

	<sup>5</sup>Virginia Bioinformatics Institute, 

	<sup>6</sup>The Samuel Roberts Noble Foundation, 

	<sup>7</sup>Kansas State University

</div>

<div>A multi&#45;institutional consortium of labs &#40;www.plantmetabolomics.org&#41; is developing metabolomics as a new functional genomics tool for elucidating the functions of Arabidopsis genes that are currently annotated as having an &#8220;unknown function&#8221;.  Approximately one third of the Arabidopsis genes are so annotated.  The consortium utilizes five distinct analytical platforms that couple different separations methods &#40;GC, LC, and CE&#41; to mass&#45;spectroscopic detection systems.  These analytical platforms are used in both &#8220;non&#45;targeted&#8221; and targeted metabolomics analyses, which in combination detect approximately 2,000 metabolites, of which 700 are chemically defined. The consortium is applying these platforms to reveal changes in the metabolome associated with knockout mutations in genes of unknown function and comparing these to similar mutants in genes of known functions.  We will discuss initial data generated from a small set of exemplar mutants.  These data indicate that metabolomics can reveal metabolic changes in mutants that are otherwise &#8220;silent&#8221; in phenotype.  These data are being interpreted via two strategies: 1&#41; as a &#8220;fingerprint&#8221; of the metabolic consequence of each mutation, which can be used to functionally cluster genes; and 2&#41; by mapping metabolite changes on metabolic and regulatory network maps, such as AraCyc and MetNetDB, to identify specific functions that are affected by each mutation.  Thus, metabolomics, in combination with other &#8220;&#45;omics&#8221; technologies promises to be a new resource for determining the function of Arabidopsis genes.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Metals, Mutants and Mayhem</div>

<div>

	<span style="text-decoration: underline" >
	Mary Lou Guerinot</span>
	<sup>1</sup>, 

	<span  >
	Suna Kim</span>
	<sup>1</sup>, 

	<span  >
	Joe Morrissey</span>
	<sup>1</sup>, 

	<span  >
	Tracy Punshon</span>
	<sup>1</sup>, 

	<span  >
	Ivan Baxter</span>
	<sup>2</sup>, 

	<span  >
	Brett Lahner</span>
	<sup>2</sup>, 

	<span  >
	Antonio Lanzirotti</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Dartmouth College, Hanover, NH USA, 

	<sup>2</sup>Purdue University, West Lafayette, IN USA, 

	<sup>3</sup>University of Chicago, Chicago, IL USA

</div>

<div> We are using ICP&#45;MS &#91;inductively coupled plasma&#45;mass spectroscopy&#93; as a tool to determine the biological significance of connections between a plant's genome and its elemental profile or &#8220;ionome.&#8221;  We are screening loss of function mutants in each of the predicted ORFs in the Arabidopsis genome and have already identified a number of proteins that dramatically alter a plant&#8217;s elemental composition, including changes in P, Ca, K, Mg &#40;macronutrients&#41;; Cu, Fe, Zn, Mn, Co, Ni, Se, Mo, I &#40;micronutrients of significance to plant and human health&#41;; Na, As, and Cd &#40;minerals causing agricultural or environmental problems&#41;.  We have also been applying our high&#45;throughput ICP&#45;MS screening, in combination with powerful genetic mapping tools such as DNA microarray&#45;based genotyping, to identify loci that drive the natural variation we observe in the plant's elemental profile using accessions of both Arabidopsis and rice.  Our ionomics project is managed and disseminated using the Purdue Ionomics Information Management System &#40;PiiMS; www.purdue.edu&#47;dp&#47;ionomics&#41;, which currently contains publicly accessible ionomic information on over 62,000 Arabidopsis shoot samples.<br/><br/>While the ability to quantify the amount of particular metals present in various plant tissues via ICP&#45;MS has proved very informative, we would also like to be able to see where the metals are distributed within various plant organs.  We are using X&#45;ray fluorescence microtomography to determine, in vivo, the spatial distribution of metals in Arabidopsis and rice seed.  Examination of various mutants is shedding light on which transporters influence the distribution of important nutrients such as iron in the seed.  Such information should aid the development of nutrient&#45;rich seed, impacting not only human health, but leading to agronomic benefits such as increased seedling vigor, crop yields and resistance to disease.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">DIFFERENTIAL ROLES OF ARABIDOPSIS PIP2 AQUAPORINS</div>

<div>

	<span  >
	Olivier Da Ines</span>
	<sup>1</sup>, 

	<span  >
	Birgit Geist</span>
	<sup>1</sup>, 

	<span  >
	Michael Mader</span>
	<sup>1</sup>, 

	<span  >
	Gerhard Welzl</span>
	<sup>1</sup>, 

	<span  >
	Ulrich Hammes</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Tony R. Schaeffner</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>GSF Research Center for Environment and Health - Biochemical Plant Pathology (OD, BG, ARS), Stem Cell Research (MM), Developmental Genetics (GW), 85764 Neuherberg/M&#252;nchen, Germany, 

	<sup>2</sup>FAU Erlangen, Molecular Plant Physiology,  91058 Erlangen, Germany

</div>

<div>The Arabidopsis major intrinsic protein &#40;MIP&#41; family consists of 35 members. Several MIPs facilitate the flow of small uncharged molecules across the plasma membrane &#40;PIP&#41; or tonoplast &#40;TIP&#41;. Due to the frequently observed water permeability they are named aquaporins and thought to be involved in plant water relations. PIPs are subdivided into 5 PIP1 and 8 PIP2, constituting highly homologous subfamilies. We focused on the PIP2 members asking whether they might exhibit differential functions. Promoter&#45;GUS constructs showed gene&#45;specific, discernable patterns. Most PIP genes were expressed in the region of the vasculature in roots and&#47;or aerial parts, however, individual members were expressed in root exodermis &#40;PIP2;4&#41; or more uniformly in the leaves &#40;PIP2;5, PIP2;7&#41;.&#13;<br/>Except for PIP2;7 knock&#45;outs for all PIP2 genes could be obtained. They did not show visible phenotypes under normal growth conditions. Using a custom&#45;made DNA array &#40;500 gene&#45;specific targets&#41; focusing on membrane proteins we discovered only minor, but distinct changes for the pip2 knock&#45;outs. A PCA of the expression data corroborated the differential responses. Furthermore, the individual PIP2 genes were analyzed for co&#45;expressing genes at bbc.botany.utoronto.ca and www.arabidopsis.leeds.ac.uk&#47;act&#47;index.php. Only PIP2;1 and PIP2;2 showed significant correlations with other PIPs, however most PIP2 genes were also correlated to TIP isoforms, which may indicate an interaction of plasma membrane and tonoplast permeabilities. Although there were a few functional categories enriched among the co&#45;expressed genes in all cases &#40;transport, cell wall&#41;, the affected genes were different. A third approach was based on the stress&#45;responsive expression patterns of the individual PIP2 genes from publicly available array data. PCA analyses revealed three distinct groups of PIP2 members.&#13;<br/>Based on these analyses, we propose a differential, and mostly non&#45;redundant function for the closely related PIP2 genes.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Co&#45;expression of two sorghum desaturases in transgenic &#34;Arabidopsis&#34; plants results in the accumulation of unusual terminal double bond&#45;containing fatty acids &#34;in planta&#34;</div>

<div>

	<span style="text-decoration: underline" >
	Zhiqiang Pan</span>
	<sup>1</sup>, 

	<span  >
	Scott R. Baerson</span>
	<sup>1</sup>, 

	<span  >
	Agnes M. Rimando</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>USDA-ARS-NPURU, University of Mississippi, University, MS 38677

</div>

<div>Polyunsaturated fatty acids play important roles as structural components of membrane glycerolipids, and as precursors of signalling molecules in eukaryotic cells. Previously, we isolated two cDNAs encoding fatty acid desaturases from &#34;Sorghum bicolor&#34;, designated &#34;SbDES2&#34; and &#34;SbDES3&#34;, which consecutively convert 16:1 to 16:3 FAs &#40;16:3&#916;9,12,15&#41; possessing a terminal double bond  when co&#45;expressed in &#34;S. cereviseae&#34; &#40;Pan et al, 2007&#41;. To study the functionality of these enzymes when overexpressed &#34;in planta&#34;, a binary vector &#40;pCBDA1&#41; was constructed for  co&#45;expression of the &#34;SbDES2&#34; and &#34;SbDES3&#34; under the direction of the CaMV 35S promoter. The mutant &#34;fad7&#34; line of &#34;Arabidopsis thaliana&#34; &#40;L.&#41; Heynh., deficient in hexadecatrienoic acid &#40;16:3&#916;7,10,13&#41;, was transformed with recombinant &#34;A. tumefaciens&#34; strains harboring pCBDA1, using the &#8216;floral dip&#8217; procedure. The expression of both the &#34;SbDES2&#34; and &#34;SbDES3&#34; in transgenic &#34;Arabidopsis&#34; lines was confirmed by RT&#45;PCR. Significantly, the accumulation of 16:3&#916;9,12,15 fatty acid was detected in multiple lines co&#45;expressing the two desaturases.  Molecular and biochemical characterization of these transformants will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Looking beyond Arabidopsis and angiosperms for biomass crop improvement genes</div>

<div>

	<span style="text-decoration: underline" >
	Clint Chapple</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Purdue University, West Lafayette, Indiana, USA

</div>

<div>One of the most significant barriers to the efficient conversion of biomass to biofuels is the lignin polymer. The biosynthesis of lignin is now relatively well understood in flowering plants, and can be readily manipulated. We have now begun investigations into lignin monomer &#40;monolignols&#41; biosynthesis in lycophytes to determine whether these modern relatives of the earliest tracheophytes might provide new tools for the modification of lignin in biomass crops. Fossil records show that the lycophyte clade arose 400 million years ago, 150&#45;200 million years earlier than the angiosperms. Our interest in the genus <span style="font-style: italic">Selaginella</span> in particular was piqued by the observation that it deposits syringyl lignin, a type of lignin often regarded as being restricted to angiosperms. To gain insight into the evolution of syringyl lignin biosynthesis in <span style="font-style: italic">Selaginella</span>, we cloned candidates for the <span style="font-style: italic">S. moellendorffii</span> homologs of the three phenylpropanoid cytochrome P450&#45;dependent monooxygenases that play essential roles in determining lignin composition in angiosperms. Although genetic approaches and transformation are not currently available in <span style="font-style: italic">S. moellendorffii</span>, complementation experiments in Arabidopsis and enzyme kinetic assays indicate that this <span style="font-style: italic">S. moellendorffii</span> P450 is a ferulate 5&#45;hydroxylase &#40;F5H&#41; that can functionally replace its angiosperm counterpart in syringyl monolignol biosynthesis. Phylogenetic analysis suggests that the identified <span style="font-style: italic">Selaginella</span> F5H is very divergent from all known plant P450s in sequence and appears to have evolved independently from angiosperm F5Hs. This project is thus providing insight into the evolution of phenylpropanoid metabolism in vascular plants, while simultaneously augmenting our lignin modification toolbox. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Global analysis of lipid metabolism regulation in Brassica napus developing seeds</div>

<div>

	<span style="text-decoration: underline" >
	Guo-Zhang Wu</span>
	<sup>1</sup>, 

	<span  >
	Ya Niu</span>
	<sup>1</sup>, 

	<span  >
	Rui Ye</span>
	<sup>1</sup>, 

	<span  >
	Wen-Hui Lin</span>
	<sup>1</sup>, 

	<span  >
	Qiu-Ming Shi</span>
	<sup>1</sup>, 

	<span  >
	Xiao-Dong Xu</span>
	<sup>2</sup>, 

	<span  >
	Rao Li</span>
	<sup>2</sup>, 

	<span  >
	Yu-Guang Du</span>
	<sup>3</sup>, 

	<span  >
	Hong-Wei Xue</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Physiology &amp;  Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China, 

	<sup>2</sup>United Gene Holdings, LTD. , Shanghai, China, 

	<sup>3</sup>The Nature Products and Glycoconjugate Research Group, Dalian Institute of Chemical Physics, Chinese Academy of Sciences, Dalian, Liaoning, China

</div>

<div>Multiple high&#45;throughput genomic approaches were performed to study the gene expression profiles during Brassica napus seed development and fatty acid &#40;FA&#41; metabolism, as well as the relevant regulation. Serial Analysis of Gene Expression &#40;SAGE&#41; using seed materials obtained a total of 68,716 tags, of which 23,895 were unique and 503 tags were functionally identified, and further revealed the transcriptome of approximately 35,000 transcripts in B. napus developing seeds. Further, &#126;22,000 independent ESTs were obtained by large&#45;scale sequencing using immature embryos at different stages, and 8462 uni&#45;ESTs and 3355 full&#45;length cDNAs were identified respectively, resulting in the systemic identification of B. napus FA biosynthesis&#45;related genes. Gene expression profiles were further studied employing cDNA chip hybridization to reveal the global regulatory network of FA metabolism in developing seeds. Together with the analysis on FA amounts and composition, it was shown that 17&#45;21 days after pollination &#40;DAP&#41; was a crucial stage for transition of seed to sink tissue. High expressions of FA biosynthesis&#45;related genes and transition of FA components are mainly at stages 21 DAP or 21&#45;25 DAP respectively. Compared to Arabidopsis, more critical roles of starch metabolism are detected for B. napus seed FA metabolism and storage components accumulation. Crucial effects of starch metabolism, oxidative pentose phosphate pathway &#40;OPPP&#41;, photosynthesis, other regulators and different carbon flux model compared to Arabidopsis in FA metabolism were discussed.</div>

</div>


</div>
</div>

</body>
</html>
