
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>
<head>
<title>ICAR 2007 - Metabolism</title>
<link rel="Shortcut Icon" href="/favicon.ico">
<link rel="stylesheet" type="text/css" href="/css/main.css" />
<link rel="stylesheet" type="text/css" href="/css/search.css" />
<link rel="stylesheet" type="text/css" href="/css/base.css" />

<style type="text/css">
@import "/css/page/pagelevels.css";
</style>

<style type="text/css">
.abs {
 margin-top: 30px;
}
.abs div {
 margin: 10px;
}
</style>
</head>
<body>

<div id="container">

<script src="https://ssl.google-analytics.com/urchin.js" type="text/javascript"></script>
<script type="text/javascript">
_uacct = "UA-1945230-1";
urchinTracker();
</script>

 <div id="nav">	
  <div id="logo"><a href="/index.jsp"><img src="/i/logo2.gif" width="87" height="61" alt="tair"/></a></div>
 </div>

<div id="content" class="clearfix">

<div class="abs">

<div style="font-weight:bold">Transcriptomic and proteomic profiling reveals new insights into salt stress responses in Arabidopsis root</div>

<div>

	<span style="text-decoration: underline" >
	YQ Jiang</span>
	<sup>1</sup>, 

	<span  >
	MK Deyholos</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Alberta, Edmonton, Canada

</div>

<div>Salt stress is a major abiotic stress limiting the productivity and the geographical distribution of many plants. Roots are an attractive system for genomic and post&#45;genomic studies of NaCl responses, due to their primary importance to agriculture and relative structural and biochemical simplicity. We used Qiagen Operon 70&#45;mer oligonucleotide microarrays representing 23,686 Arabidopsis genes to identify the transcripts in 18d&#45;old Arabidopsis roots that changed in relative abundance following 6 h, 24 h, or 48 h of hydroponic exposure to 150 mM NaCl. We verified 15 selected genes by qRT&#45;PCR. Enrichment analysis identified groups of structurally or functionally related genes whose members were statistically over&#45;represented among up&#45; or down&#45;regulated transcripts. Our results are consistent with generally observed stress response themes, and highlight potentially important roles for underappreciated gene families, including several groups of transporters, signalling molecules and transcription factors. As the mRNA level is not correlated well with protein level as a result of alternative splicing and post&#45;translational modifications, we also carried out a comparative proteomic analysis of Arabidopsis roots treated with 150 mM NaCl for 6 and 48 h using two&#45;dimensional electrophoresis and LC&#45;MS&#47;MS. We detected &#126;1,000 protein spots reproducibly on each gel. In contrast to the transcriptomic analyses, the greatest differences between control and treated samples were detected 48 h after NaCl treatment. Mass spectrometry analysis allowed the identification of 86 differentially expressed proteins, including well&#45;known and novel salt&#45;responsive proteins. The identified proteins are involved in several processes, i.e. ROS scavenging, signal transduction, protein translation, processing and degradation, primary and hormone metabolisms, and metabolism of amino acids. We chose a few novel salt&#45;responsive genes for further characterization. In conclusion, our study provides new insights into salt stress responses in Arabidopsis root and will facilitate mapping of regulatory networks and extend our ability to improve salt tolerance in plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cyclic Electron Transport Protects against Moderate Heat Stress in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Ru Zhang</span>
	<sup>1</sup>, 

	<span  >
	Thomas Sharkey</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Botany, University of Wisconsin-Madison, Madison, ( WI ), USA

</div>

<div>Photosynthesis is among the most heat&#45;sensitive functions in plants.  However, the mechanism by which moderate heat stress reduces photosynthetic capacity is unclear. Photosynthesis uses both linear and cyclic electron flows to convert light energy into the transthylakoid proton motive force, composed of the proton gradient and the electric field. Heat stress frequently causes increased cyclic electron transport. Here we used two <span style="font-style: italic">Arabidopsis thaliana</span> mutants, each impaired in one of the two pathways of cyclic electron flow &#45;&#45; <span style="font-style: italic">crr2</span> &#40;<span style="font-style: italic"><span style="text-decoration: underline">c</span>hloro<span style="text-decoration: underline">r</span>espiratory <span style="text-decoration: underline">r</span>eduction</span>, deficient in the chloroplast NAD&#40;P&#41;H dehydrogenase dependant cyclic flow from NADPH to plastoquinone&#41; and <span style="font-style: italic">pgr5</span> &#40;<span style="font-style: italic"><span style="text-decoration: underline">p</span>roton <span style="text-decoration: underline">g</span>radient <span style="text-decoration: underline">r</span>egulation</span>, deficient in the antimycin A sensitive cyclic flow from ferredoxin to plastoquinone&#41;. Heat stress experiments were done in light with intact leaves by switching leaf temperature from 23oC to 40oC in 2 minutes and electrochromic shift was measured to monitor transthylakoid proton fluxes during photosynthesis. The electrochromic shift results from an electric field effect on carotenoid absorbance bands at 518 nm. Our result indicated that these two cyclic&#45;electron&#45;transport mutants were more sensitive to heat stress and had less ability to recover from heat damage than wildtype, especially <span style="font-style: italic">pgr5</span>. What&#8217;s more, the electrochromic shift data showed that the transthylakoid electrical gradient was significantly reduced in <span style="font-style: italic">pgr5</span> mutants. We propose that cyclic electron transport protects against moderate heat stress in <span style="font-style: italic">Arabidopsis thaliana</span>.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Effect of DDB1A Overexpression on DET1 Genetics and Biochemistry</div>

<div>

	<span style="text-decoration: underline" >
	Yu Zhang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Manitoba, Winnipeg, Manitoba, Canada

</div>

<div>Yu Zhang and Dana F. Schroeder<br/>Department of Botany, University of Manitoba, Winnipeg, MB<br/>R3T 2N2    Canada<br/>                              Abstract<br/>Light is an important regulator of <span style="font-style: italic">Arabidopsis</span> growth. Dark&#45;grown wildtype <span style="font-style: italic">Arabidopsis</span> seedlings exhibit elongated hypocotyls and small closed cotyledons, whereas light&#45;grown seedlings develop photomorphogenically, exhibiting short hypocotyls and expanded green cotyledons. De&#45;Etiolated 1 &#40;DET1&#41; is an important negative regulator of <span style="font-style: italic">Arabidopsis</span> photomorphogenesis. In loss&#45;of&#45;function <span style="font-style: italic">DET1</span> mutants, dark&#45;grown seedlings look photomorphogenic: they have short hypocotyls and open cotyledons. Transformation of <span style="font-style: italic">det1</span> mutants with epitope&#45;tagged DET1 can partially rescue the <span style="font-style: italic">det1</span> mutant phenotype. UV&#45;Damaged DNA Binding protein 1 &#40;DDB1&#41; is a component of the 350 kD DET1 complex in plant cells. <span style="font-style: italic">DDB1A</span>, one of two <span style="font-style: italic">DDB1</span> genes in the <span style="font-style: italic">Arabidopsis</span> genome, has been found to genetically interact with <span style="font-style: italic">DET1</span>. <br/>To further study the biochemical and genetic interaction between DET1 and DDB1A, we generated <span style="font-style: italic">Arabidopsis</span> lines with overexpressed DDB1A&#45;3HA in wildtype, <span style="font-style: italic">det1</span>, and Myc&#45;DET1 and GFP&#45;DET1 rescued genetic backgrounds. Overexpression of DDB1A&#45;3HA in wildtype and <span style="font-style: italic">det1</span> backgrounds did not result in significant phenotypic changes. However combination of DDB1A&#45;3HA and Myc&#45;DET1 overexpression resulted in decreased rescue of dark&#45;grown hypocotyl and adult height and rosette width phenotypes. This result is consistent with the decreased levels of Myc&#45;DET1 detected in the double overexpressor in both dark&#45;grown seedlings and adults. Interestingly, the GFP&#45;DET1 DDB1A&#45;3HA double overexpressor exhibits increased rescue of dark hypocotyl and light chlorophyll phenotypes relative to GFP&#45;DET1 alone, despite the fact that GFP&#45;DET1 also decreases in this background. Variation in DDB1A&#45;3HA levels and complex formation may account for this difference. Overall, DDB1A overexpression reduces levels of epitope&#45;tagged DET1 and modulates rescue of <span style="font-style: italic">det1</span> mutants by the DET1&#45;DDB1 complex.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification of ecotype diversity between different accessions of Arabidopsis using CAPS and SSLP molecular markers</div>

<div>

	<span style="text-decoration: underline" >
	RUDRA NAIK</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>UAS,DHARWAD,KARNATAKA,INDIA

</div>

<div>&#13;<br/>           Arabidopsis is a small flowering plant adapted to various ecological niches of the world. It is a member of the mustard &#40;Brassicaceae&#41; family, which includes cultivated spp like cabbage and raddish. This genus is of great interest since it contains Thale Cress &#40;Arabidopsis thaliana&#41;, one of the model organisms used for studying plant biology and the first plant to have its entire genome sequenced. Arabidopsis is not of major economic significance, but it offers important advantages for basic research in genetics and molecular biology since the changes in the plant are easily observed, making it a very useful model. Molecular markers are available that are polymorphic for Arabidopsis. &#13;<br/>DNA was isolated from four accessions of Arabidopsis collected from the Netherlands &#40;Wageningen, Landsberg.  Rhenen, and Oosterberg&#41; and used for CAPS and SSLP mapping. Wageningen accessions were found to be different by the SSLP marker while the others were identical. CAPS revealed the presence of one Hind III restriction site in Landsberg, but absent in all the other accessions studied.  In this experiment, the resolving power of genetic markers is determined by the level of polymorphism detected, which is primarily affected by the mutation rate at the genomic sites involved.  The 30bp difference detected between Wageningen accessions could be due to DNA polymerase slippage during DNA replication creating some mismatches by using the primers.&#13;<br/>                    There is a genetic variation in the selected Arabidopsis accessions as revealed by CAPS and SSLP markers. The Wageningen accessions though collected from the same ecological niche are different accessions, which may be due to ecotype selection over some alleles as revealed by the SSLP. All the accessions have no Hind III except Landsberg SSLP and CAPS are essential marker tools in studying the genetic diversity in Arabidopsis.&#13;<br/></div>

</div>






<div class="abs">

<div style="font-weight:bold">Boron transcriptome analysis &#8211; profiles of boron nutrition&#45;regulated genes and role of <span style="font-style: italic">WRKY6</span> in regulation</div>

<div>

	<span style="text-decoration: underline" >
	Ichiro Kasajima</span>
	<sup>1,2,3</sup>, 

	<span  >
	Yoko Ide</span>
	<sup>1,2,3</sup>, 

	<span  >
	Masami Yokota Hirai</span>
	<sup>4</sup>, 

	<span  >
	Toru Fujiwara</span>
	<sup>1,2,5</sup>

</div>

<div>

	<sup>1</sup>Department of Biotechnology, Graduate School of Agricultural and Life Sciences, The University of Tokyo, 

	<sup>2</sup>Biotechnology Research Center, The University of Tokyo, 

	<sup>3</sup>JSPS Research Fellow, 

	<sup>4</sup>RIKEN Plant Science Center, 

	<sup>5</sup>Solution Oriented Research for Science and Technology, Japan Science and Technology Agency

</div>

<div>Boron &#40;B&#41; is an essential nutrient for plants but is toxic in excess.  We performed transcriptome analysis under conditions of both low B &#65288;0.3 &#956;M borate&#65289;and high B &#65288;3,000 &#956;M borate&#65289;in <span style="font-style: italic">Arabidopsis thaliana</span> using Affymetrix 22K array.  Induction by high B was also confirmed for nine genes in an independent experiment.  Transcripts of <span style="font-style: italic">At1g03770</span> &#40;transcription factor&#41; and <span style="font-style: italic">At5g57340</span> were found to be elevated in roots, whereas <span style="font-style: italic">At2g04040</span>, <span style="font-style: italic">At2g04050</span>, <span style="font-style: italic">At2g04070</span> &#40;all encoding multidrug and toxic compound extrusion transporters&#41;, <span style="font-style: italic">At1g32870</span> &#40;transcription factor&#41;, <span style="font-style: italic">At5g51440</span> &#40;heat shock protein&#45;like&#41;, <span style="font-style: italic">At2g41730</span> and <span style="font-style: italic">At2g21640</span> are induced in the shoots.  The accumulation of both <span style="font-style: italic">At2g04050</span> and <span style="font-style: italic">At5g51440</span> transcripts was further found to be elevated by more than 100&#45;fold after two days of high B treatment.  Although the roles of these genes in high B conditions remain unknown, our findings represent the first identification of high B&#45;induced genes.  We also identified <span style="font-style: italic">WRKY6</span> as a low&#45;B induced gene.  Its transcript is upregulated in root tips under low B condition.  This induction was confirmed with the <span style="font-style: italic">WRKY6</span> promoter&#45;GUS plants, suggesting that regulation is at the transcriptional level.  To elucidate target genes of <span style="font-style: italic">WRKY6</span>, we performed transcriptome analysis with 8&#45;mm root tip samples of <span style="font-style: italic">wrky6&#45;3</span>, a mutant carrying T&#45;DNA in <span style="font-style: italic">WRKY6</span>.  We identified several genes whose mRNA accumulations are affected in the root tips of <span style="font-style: italic">wrky6&#45;3</span>.  Preliminary experiments suggested that root elongation of <span style="font-style: italic">wrky6&#45;3</span> mutant plants are reduced only under low&#45;B conditions.  The downstream genes may be involved in maintenance of root elongation under low&#45;B conditions.</div>

</div>





<div class="abs">

<div style="font-weight:bold">ER stress activates the processing and relocation to the nucleus of ER membrane&#45;bound transcription factors</div>

<div>

	<span style="text-decoration: underline" >
	Jianxiang Liu</span>
	<sup>1</sup>, 

	<span  >
	Renu Srivastava</span>
	<sup>1</sup>, 

	<span  >
	Ping Che</span>
	<sup>1</sup>, 

	<span  >
	Stephen Howell</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Sciences Institute, Iowa State University, Ames IA 50011, USA

</div>

<div>ER stress in eukaryotic cells results from the disruption of processes leading to the folding or modification of secreted proteins. This form of stress is perceived and acted upon by sensor&#47;transducers located in the ER. We have identified three ER&#45;resident bZIP transcription factors in Arabidopsis that are candidates for ER stress sensors&#47;transducers. Those candidates are type II proteins with a cytosol&#45;facing b&#45;ZIP domain, a transmembrane segment, and a lumen&#45;facing canonical site&#45;1 protease &#40;S1P&#41; cleavage site.  We found that one of the candidate sensor&#47;transducers, AtbZIP17, is activated by salt stress while another, AtbZIP28, is activated in the typical manner of an unfolded protein response &#40;UPR&#41; triggered by stress agents, such as tunicamycin or DTT. Following different ER stresses, the transcription factors are processed, releasing their N&#45;terminal b&#45;ZIP domains, which are then translocated to the nucleus.  Both responses require the function of a subtilisin&#45;like serine protease &#40;subtilase&#41;, AtS1P.  AtS1P is a Golgi&#45;localized protease, and it is thought that following stress, the bZIP sensor&#47;transducers are translocated from the ER to the Golgi where they are processed by AtS1P and other proteases.  We have demonstrated processing of epitope&#45;tagged forms of AtbZIP17 and AtbZIP28 in vitro using recombinant AtS1P. In Arabidopsis, AtbZIP17 activation directly or indirectly up&#45;regulated the expression of many salt stress response genes, including the homeodomain transcription factor ATHB&#45;7. Expression in transgenic seedlings of a truncated form of AtbZIP28, containing only the N&#45;terminal cytosol&#45;facing domain, up&#45;regulated the expression of several genes associated with UPR, such as BiP3. Thus, two different types of ER stress are sensed by two different sensor&#47;transducers, which activate different sets of genes that act to mitigate the corresponding stress.<br/><br/>This work supported by the NSF 2010 program &#40;IBN0420015&#41;.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">HARDY</span>, an Arabidopsis gene conferring drought and salt tolerance in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Shital Dixit</span>
	<sup>1</sup>, 

	<span  >
	Raffaella Greco</span>
	<sup>2,1</sup>, 

	<span  >
	Asaph Aharoni</span>
	<sup>3</sup>, 

	<span  >
	Jelle Hiemstra</span>
	<sup>1</sup>, 

	<span  >
	Nayelli Marsch-Martinez</span>
	<sup>1</sup>, 

	<span  >
	Andy Pereira</span>
	<sup>4</sup>

</div>

<div>

	<sup>1</sup>Plant Research International,Wageningen University, The Netherlands, 

	<sup>2</sup>Universit&#224; degli Studi di Milano, Italy, 

	<sup>3</sup>Weizmann Institute of Science, Israel, 

	<sup>4</sup>VBI, Virginia Tech, Blacksburg, USA

</div>

<div>Fresh water scarcity is one of the principal global problems of this century, and plants account for around 65&#37; of global fresh water use. Water scarcity often gives rise to drought and salinity which can cause severe and irreversible damage to crop yields. Identification and characterization of genes and mechanisms for tolerance to multiple stress factors is therefore desirable. &#13;<br/>In a phenotypic screen of an Arabidopsis <span style="font-style: italic">En&#47;I&#40;Spm&#41;</span>  transposon activation tagged mutant collection, we identified a semi&#45;dominant mutant <span style="font-style: italic">hardy &#40;hrd&#45;D&#41;</span> with the remarkable feature of having roots with enhanced strength, branching and extra starch&#45;storing cortical cells. Overexpression of <span style="font-style: italic">HRD</span> produced dark green leaves that were smaller and thicker with more chloroplast bearing mesophyll cells. A quantitative root&#45;pull assay showed that the <span style="font-style: italic">hrd&#45;D</span> mutant has 20&#45;50 &#37; more root&#45;strength compared to wild&#45;type.&#13;<br/>When tested for drought tolerance, the <span style="font-style: italic">hrd&#45;D</span>  mutant showed drought tolerance by surviving a prolonged period of progressive drought compared to the wild type &#40;9 vs. 12 days&#41;.  To eliminate the effect of reduced growth and size of the mutant in the drought tolerance assay, we developed an inducible <span style="font-style: italic">HRD</span> overexpression genotype &#40;HRD&#45;GR, glucocorticoid receptor fusion&#41; and showed that steroid DEX induced HRD&#45;GR plants, which did not show a small&#45;plant phenotype, were also drought tolerant. In a salt tolerance assay <span style="font-style: italic">hrd&#45;D</span> gave tolerance to as high as 300mM of NaCl and could reach full maturity in contrast to the wild&#45;type. The salinity tolerance of the HRD overexpressors is accompanied by lower accumulation of Na&#43; ions. HRD overexpression lines also displayed a significant tolerance to a root&#45;infecting pathogen &#40;Verticillium&#41; suggesting a general stress tolerance mechanism is active due to <span style="font-style: italic">HRD</span> overexpression.  Expression of <span style="font-style: italic">HRD</span> is predominantly found in developing seed and mature pollen, suggesting that <span style="font-style: italic">HRD</span> might play a role in priming seeds for desiccation tolerance.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">ENGINEERING STOMATAL ACTIVITY FOR DROUGHT TOLERANCE IN PLANTS</div>

<div>

	<span style="text-decoration: underline" >
	Massimo Gabiati</span>
	<sup>1</sup>, 

	<span  >
	Eleonora Cominelli</span>
	<sup>1</sup>, 

	<span  >
	Laura Simoni</span>
	<sup>1</sup>, 

	<span  >
	Priscilla Francia</span>
	<sup>1</sup>, 

	<span  >
	Chiara Tonelli</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Milan, Milan, Italy

</div>

<div>Land plants lose over 95&#37; of their water via transpiration through stomatal pores, distributed on the surface of leaves and stems. The opening and closing of the pore is mediated by turgor&#45;driven volume changes of two surrounding guard cells. Engineering of stomatal responses in mutant or transgenic plants represents a valuable tool to design new crops with a more sustainable water use and opens new possibilities to improve plant survival and productivity during drought. We employed large scale genomic screens to identify guard cell&#45;specific mutations and promoters in Arabidopsis. Reverse genetic screen of T&#45;DNA mutagenized lines, allowed the identification of a null allele of the guard cell&#45;specific transcription factor AtMYB60. Analysis of stomatal movements in wild type and mutant plants revealed that the atmyb60&#45;1 mutation results in the constitutive reduction of stomatal aperture, and thus in reduced transpirational water loss during drought. Microarray analysis of gene expression indicated that a limited number of stress&#45;related gene is altered in the atmyb60&#45;1 mutant. We also analyzed the GUS expression pattern of approximately 20,000 Arabidopsis gene trap lines, and identified 5 lines in which the reporter gene was specifically expressed in guard cells. We established the genomic insertion sites of the gene trap element  and investigated the expression profile of the tagged genes. Our findings indicate that the gene trap lines isolated in the screen provide valuable marker lines to study pathways that are unique to guard cells, a powerful tool for reverse genetics analyses of guard cell&#45;specific genes, and a precious source of guard cell&#45;specific regulatory regions.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The expression of senescence&#45;enhanced genes in response to oxidative stress&#13;<br/>&#13;<br/>&#13;<br/>&#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	SAEID NAVABPOUR</span>
	<sup>1</sup>, 

	<span  >
	VICKY BUCHANAN-WOLLASTON</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Dept. of Agronomy &amp; Plant Breeding, University of Agricultural Siences and Natural Resources, Gorgan, IRAN, 

	<sup>2</sup>Horticultural Research Internatunal, Wellesbourne, UK

</div>

<div>Senescence is a genetically regulated oxidative process that involves a degradation of cellular structures and the mobilization of the products of degradation to other parts of the plant. Environmental stress is a major limiting factor in plant productivity. Much of the injury to the plant caused by stress exposure is associated with oxidative damage at the cellular level. The role of Reactive Oxygen Species &#40;ROS&#41; during abiotic stress has become a subject of considerable interest given that ROS have been implicated in processes leading to plant stress acclimation. High concentrations of ROS lead to phytotoxicity, whereas relatively low levels can be used for acclamatory signalling. Therefore, ROS are not simply toxic by&#45;products of metabolism but also can function as signalling molecules. Thus, the controlled modulation of ROS levels in plants is extremely important.&#13;<br/>The metallothionein gene, &#34;LSC54&#34;, is expressed at high levels during leaf senescence in &#34;Brassica napus&#34; and &#34;Arabidopsis thaliana&#34;. The promoter region of this gene has been cloned, fused to GUS gene and transformed into &#34;Arabidopsis&#34;.&#13;<br/>The aim of this project was to analyse the regulation of gene expression in stress induced and developmental senescence. For this purpose transgenic &#34;Arabidopsis&#34; plants containing the &#34;LSC54&#34; promoter: GUS fusion were treated with numerous chemicals known to generate ROS. Silver nitrate was the most consistent treatment, resulting in strongly elevated levels of &#34;LSC54&#34; expression. Treatment with 3&#45;Amino&#45;Triazole &#40;catalase inhibitor&#41; also gave measurable and consistent increases in GUS expression. The role of silver nitrate &#40;AgNO3&#41; in &#34;LSC54&#34; expression was investigated by treatment with several Reactive Oxygen Species &#40;ROS&#41; scavengers combined with silver nitrate treatment. It was found that &#34;LSC54&#34; expression is induced by a combination of ROS. Northern hybridisation experiments showed that many senescence&#45;enhanced genes are induced by AgNO3 and 3&#45;AT treatment, in many cases the levels of expression are comparable to those seen during senescence.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Transcriptome changes in response to magnesium deficiency and restoration in Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Christian Hermans</span>
	<sup>1</sup>, 

	<span  >
	Adrian Craciun</span>
	<sup>1</sup>, 

	<span  >
	Nathalie Verbruggen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Universit&#233; Libre de Bruxelles, Brussels, Belgium

</div>

<div>Magnesium plays a key role in plant metabolism. In addition to being a central atom of the chlorophyll molecule, Mg is also essential for the functioning of many enzymes, including RNA polymerases, ATPases, protein kinases and phosphatases. Despite the wide recognition of Mg deficiency in several soil types, relatively little is known about Mg deficiency in plants. The first physiological analysis of Mg deficient Arabidopsis was recently reported &#91;1&#93;, however proper knowledge of the molecular mechanisms underpinning Mg homeostasis is currently scarce. One dramatic effect during the early response to Mg deficiency is sucrose and starch accumulation in young source leaves, before any noticeable effect on photosynthetic activity. A later effect of Mg deficiency is a reduction in plant growth, with the shoot biomass being more affected than root.&#13;<br/>We were primarily interested in the identification of genes involved in rapid responses to external Mg deprivation. The roots and the most recently expanded leaves of the rosette constituted the material for our transcriptomic analysis. A cDNA&#45;AFLP approach &#40;covering on estimation 5&#37; of the transcriptome&#41; was chosen for a pilot experiment, in order to decide about the sampling time for further microarray studies. Several time points following the removal of Mg from the nutrient solution, were analyzed in order to increase the temporal resolution and thus to discriminate between early and late Mg response gene. In this analysis, microarray hybridizations were done using Affymetrix AtH1 chips on three independent biological replicates. We also analyzed the transcriptomic response to Mg resupply &#40;re&#45;addition of Mg in the nutrient solution of 7 days deficient plants&#41;. The analysis of microarrays data generated will provide information for a long&#45;term goal: uncovering the relationship of specific downstream events to the signaling pathways that control Mg deficiency responses in Arabidopis. &#13;<br/>&#91;1&#93;  Hermans and Verbruggen &#40;2005&#41; J. Exp. Bot. 56: 1983&#45;1989.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Reverse genetics approach on the identification of putative salt and osmotic stress determinants in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Sara Laranjeira</span>
	<sup>1</sup>, 

	<span  >
	Herl&#226;nder Azevedo</span>
	<sup>1</sup>, 

	<span  >
	Vitor Amorim-Silva</span>
	<sup>1</sup>, 

	<span  >
	Rui Tavares</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Biology Dep. , Minho University, Braga, Portugal

</div>

<div>In the face of a global scarcity of water resources and the increased salinization of soil and water, abiotic stress is already a major limiting factor in plant growth and will soon become even more severe as desertification covers more and more of the world&#8217;s terrestrial area. Drought and salinity are already widespread in many regions, and are expected to cause serious salinization of more than 50&#37; of all arable lands by the year 2050. The major progress in the identification of salt&#47;osmo&#45;tolerance determinants is due primarily to the use of Arabidopsis as a molecular genetic model system. Functional screening in the Arabidopsis biological model makes use of phenotype&#45;centred forward and reverse genetic approaches, usually using loss&#45;of&#45;function mutant analysis. Based on a reverse genetics approach, the expression analysis of salt and osmotic stressed Arabidopsis plants made available through the Affymetrix microarray database &#40;NASC&#41;, led to the identification of genes of unknown function. Through loss&#45;of&#45;function mutant strategy we are searching for salt&#47;osmotic stress related phenotypes. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification of putative oxidative stress&#45;associated genes in Arabidopsis by reverse genetics</div>

<div>

	<span style="text-decoration: underline" >
	Herl&#226;nder Azevedo</span>
	<sup>1</sup>, 

	<span  >
	Vitor Amorim-Silva</span>
	<sup>1</sup>, 

	<span  >
	Sara Laranjeira</span>
	<sup>1</sup>, 

	<span  >
	Rui Tavares</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Biology Dep., Minho University, Braga, Portugal

</div>

<div>With the onset of aerobic life, cells have had to cope with the by&#45;products of oxygen metabolism: reactive oxygen species &#40;ROS&#41;. These species result from the transfer of energy or electrons to ground state oxygen, being extremely toxic to the cell by easily reacting with proteins, lipids and nucleic acids. Under standard conditions, the production of ROS in cells is maintained in homeostatic levels, with many abiotic stresses disrupting this condition and enhancing the production of ROS. The major progress in the identification of determinants involved in the homeostasis of ROS is due primarily to the use of Arabidopsis as a molecular genetic model system. Functional screening in the Arabidopsis biological model makes use of phenotype&#45;centred forward and reverse genetic approaches, usually using loss&#45;of&#45;function mutant analysis. Based on a reverse genetics approach, the expression analysis of oxidative stressed Arabidopsis plants, available through the Affymetrix microarray database &#40;NASC&#41;, led to the identification of putative oxidative stress&#45;related genes. Through loss&#45;of&#45;function mutants we are currently screening these genes for phenotype.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The molecular basis of vernalization</div>

<div>

	<span style="text-decoration: underline" >
	Caroline Dean</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The John Innes Centre, Norwich, UK

</div>

<div>At a certain stage in their life&#45;cycle plants flower and undergo the transition from vegetative to reproductive development. The correct timing of flowering is crucial for reproductive success so plants integrate multiple environmental and endogenous signals. The Dean laboratory is studying the importance of prolonged cold or winter for flowering, a process known as vernalization. The need for vernalization ensures plants over winter in a vegetative form and flower in the favourable conditions of spring. Vernalization requirement has been bred into many crop species to extend their geographical range and variation in vernalization is a key parameter in adaptation of plants to different climates. &#13;<br/>&#13;<br/>We have used a molecular genetic analysis in <span style="font-style: italic">Arabidopsis thaliana</span> to identify genes involved in determining both the need for vernalization and the ability to vernalize. The pathways we study share a common downstream target, <span style="font-style: italic">FLC</span>, a gene encoding a MADS box repressor of flowering. The talk will describe our current understanding of these pathways, how <span style="font-style: italic">FLC</span> chromatin regulation has been identified as a central mechanism and how the functioning of the pathways has changed as Arabidopsis has adapted to different climates. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">MLO function&#40;s&#41; in plants: Towards the genetic control of the &#34;root curling&#34; phenotype associated with <span style="font-style: italic">Atmlo4</span> and <span style="font-style: italic">Atmlo11</span> mutants</div>

<div>

	<span style="text-decoration: underline" >
	Noir Sandra</span>
	<sup>1</sup>, 

	<span  >
	Alan M. Jones</span>
	<sup>2</sup>, 

	<span  >
	Zhongying Chen</span>
	<sup>2</sup>, 

	<span  >
	Ralph Panstruga</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research -Dept. Plant Microbe Interactions- Carl-von-Linn&#233; Weg 10, D-50829 Cologne - Germany, 

	<sup>2</sup>The University of North Carolina -Dept. of Biology- CB# 3280 Coker Hall, Chapel Hill, North Carolina 27599-3280 - USA

</div>

<div>MLO proteins constitute a plant&#45;specific family of seven&#45;transmembrane domain proteins comprising approximately 10&#45;15 members per plant species. Barley Mildew&#45;resistant Locus O &#40;MLO&#41; protein is the &#34;founder&#34; of this protein family, and based on amino acid similarity, 15 <span style="font-style: italic">MLO</span> homologs have been identified in <span style="font-style: italic">Arabidopsis thaliana</span>. To date, barley MLO and its functional Arabidopsis ortholog, <span style="font-style: italic">At</span>MLO2, are the only members for which functional data are available: they are described as negative regulators of defence responses against appropriate powdery mildew fungi. However, each <span style="font-style: italic">AtMLO</span> gene has a unique expression pattern and is differentially regulated, and they can be grouped in at least four distinct phylogenetic clades. These data indicate putative functional diversification within the protein family. &#13;<br/>To elucidate function&#40;s&#41; of MLO proteins in plants, we characterized <span style="font-style: italic">Atmlo4</span> and <span style="font-style: italic">Atmlo11</span> mutant root growth behaviour since roots are the predominant organ in which these genes are expressed. We observed that <span style="font-style: italic">Atmlo4</span> and <span style="font-style: italic">Atmlo11</span> mutants exhibit a dramatic circling root growth pattern under in vitro culture conditions. We propose to explore the biological mechanisms associated with MLO function by identifying potential regulators implicated in the &#34;root curling&#34; phenotype observed in <span style="font-style: italic">Atmlo</span> mutants. &#13;<br/>As a first step, we identified <span style="font-style: italic">Arabidopsis</span> mutants that suppress the &#34;root curling&#34; phenotype using a chemical re&#45;mutagenesis approach. We identified and characterized nine suppressor mutants. A map&#45;based cloning of selected suppressing genes is on&#45;going and an update on our progress will be provided. &#13;<br/>In addition, we took a directed chemical&#45;genetic approach. By screening chemical libraries, we found that compounds described as auxin transport, vesicle trafficking&#45;inhibitors, and flavonoids reverse the &#34;root curling&#34; phenotype of <span style="font-style: italic">Atmlo</span> mutants.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Unwinding the biological clock with systems biology</div>

<div>

	<span style="text-decoration: underline" >
	Andrew Millar</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centre for Systems Biology at Edinburgh, University of Edinburgh, Edinburgh, UK

</div>

<div>The 24&#45;hour circadian clock is a regulatory module with complex inputs from the environment. It controls many 24&#45;hour rhythms in plants and other eukaryotes, including organ growth, photosynthetic capacity and the photoperiodic response. The clock mechanisms in all taxa include complex gene circuits with interlocking negative feedback loops. Genome&#45;wide, at least 15&#37; of RNAs are targets for circadian regulation &#40;Edwards et al., Plant Cell 2006&#41;. Mis&#45;timing of circadian rhythms under light&#45;dark cycles can cause 50&#37; deficit in the growth of Arabidopsis in the lab &#40;Dodd et al., Science 2005&#41;, so correct temporal regulation can significantly enhance plant growth rate. &#13;<br/>&#13;<br/>We use systems biology approaches to understand the clock mechanism. To develop systems&#45;level models, we combine timeseries of molecular data and luciferase &#40;LUC&#41; reporter gene imaging, with analysis of plant clock mutants, statistical parameter estimation and global parameter searches. The resulting differential equation models of the clock and the photoperiod sensor allowed us to predict the properties of unidentified clock components. One gene corresponding to a predicted clock component was identified by experiment, validating its location in the model &#40;Locke et al., Mol. Syst. Biol. 2005 and 2006&#41;, though additional predictions remain to be validated. The models are clearly incomplete but they facilitate reasoning about the clock network and aid experimental design. The challenges are now mathematical simplification with more biological realism. For realism, we need to incorporate additional molecular components and processes and also to expand the range of rhythmic phenomena that the models can match. &#13;<br/>&#13;<br/>The analysis of clock models is helping to address why all circadian clocks have multi&#45;loop circuits. In this area, we have focussed on 1. flexibility in the evolution of deterministic regulation &#40;Rand et al., Interface 2004; J. Theor. Biol. 2006&#41;, 2. flexibility in phase and waveform in the face of daily or seasonal changes in photoperiod, 3. temperature compensation &#45; the remarkable constancy of clock speed across a range of temperatures. I will discuss new experimental and mathematical results on these aspects of clock function.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The drought tolerant Arabidopsis &#34;alx8&#34; mutant has altered leaf morphology as well as changes in stress response signalling pathways under normal growth conditions and during abiotic stress.</div>

<div>

	<span style="text-decoration: underline" >
	Pip Wilson</span>
	<sup>1</sup>, 

	<span  >
	Gonzalo Estavillo</span>
	<sup>1</sup>, 

	<span  >
	Katie Fields</span>
	<sup>2</sup>, 

	<span  >
	Peter Crisp</span>
	<sup>1</sup>, 

	<span  >
	Daniel Harris-Pascal</span>
	<sup>1</sup>, 

	<span  >
	Barry Pogson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The ARC Centre for Plant Energy Biology, The Australian National University, Canberra, Australia, 

	<sup>2</sup>Department of Plant and Animal Sciences, University of Sheffield, Sheffield, UK

</div>

<div>The development of tolerance to abiotic stress requires a combination of physiological, morphological and molecular responses. These responses are regulated by complex signalling pathways, which often coalesce or antagonise each other. In this study, we use the Arabidopsis &#34;alx8&#34; mutant to examine the relationship between the development of stress tolerance and the pathways that regulate it.<br/><br/>The &#34;alx8&#34; mutant has increased drought tolerance, elevated ABA, and constitutive expression of the antioxidant enzyme, APX2 &#40;Rossel &#34;et al.&#34;, 2006&#41;. The drought tolerance of alx8 seems to be due to a combination of physiological, morphological and molecular changes both under normal growth conditions and stress treatment. For example, alx8 has changes in stomatal function under both normal and high light stress conditions and altered leaf and stomatal morphology. There are also changes in stress response signalling compounds, such as ABA, and microarray studies have revealed significantly altered gene expression. The relationship between these changes and the drought tolerance is currently being investigated.<br/><br/>The &#34;alx8&#34; gene has been identified by positional cloning; the details of which will be discussed at the conference. Interestingly, despite the ALX8 protein being a key player in regulation of multiple stress response pathways, not all of the expected down&#45;stream targets have altered expression. Furthermore, the altered leaf morphology and gene expression indicate a role for this protein under normal growth conditions. Hence, the &#34;alx8&#34; mutant may be very useful in elucidating the regulation of the various components of stress responses and the development of stress tolerance.<br/><br/>Reference: Rossel &#34;et al.&#34;, 2006, Plant, Cell and Environment 29&#40;2&#41;: 269&#45;281<br/><br/>Research supported by Meat and Livestock Association of Australia and the ARC Centre for Excellence in Plant Energy Biology.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of Lipid&#45;related Novel Stress&#47;Defense Genes &#8212; from Model Higher Plant towards Crops</div>

<div>

	<span style="text-decoration: underline" >
	Shilan Zhao</span>
	<sup>1</sup>, 

	<span  >
	Zheng Qing Fu</span>
	<sup>2</sup>, 

	<span  >
	Da Luo</span>
	<sup>1</sup>, 

	<span  >
	Wei Ma</span>
	<sup>1</sup>, 

	<span  >
	James R. Alfano</span>
	<sup>2</sup>, 

	<span  >
	Donald F. Becker</span>
	<sup>3</sup>, 

	<span  >
	Qi Cheng</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>School of Life Science and Biotechnology, Shanghai Jiao Tong University, Shanghai 200030, China, 

	<sup>2</sup>Plant Science Initiative and Department of Plant Pathology, University of Nebraska, Lincoln, Nebraska 68588, USA, 

	<sup>3</sup>Department of Biochemistry, Redox Biology Center, University of Nebraska, Lincoln, Nebraska 68588, USA

</div>

<div>Plant productivity is significantly impacted by metabolic activities and environmental stress. Our research is to characterize 4 long chain fatty omega&#45;alcohol oxidases homologues &#40;AtFAOs&#41; found in model higher plant Arabidopsis in concert with 8 Arabidopsis long chain omega&#45;hydroxylases. In some industrial yeast, omega&#45;oxidation pathways in lipid metabolism have been characterized but in higher plants the omega&#45;oxidation pathway has been overlooked. We identified long chain fatty alcohol oxidases in higher plant and have initiated the characterization of all 4 AtFAOs in Arabidopsis. Some pioneering studies have already shown that the omega&#45;hydroxylase could play important roles in response to pathogenic stress but the detail mechanism is yet to be elucidated. Recent microarray and experimental data increasingly suggest that AtFAO genes may be regulated by several abiotic or biotic stresses, including cold stress and pathogenic attack. We hypothesize that the AtFAOs may participate in such defense network in concert with other defense candidate proteins, resembling omega&#45;oxidation pathways in yeast. Such pathway&#40;s&#41; play extensive roles in higher plants in response to the environmental stress. One of our long term goal is to study the biochemical and biophysical aspects of these novel proteins and relevant metabolic&#47;signaling pathway&#40;s&#41; in Arabidopsis and&#47;or oil crops. This will help contribute to better understanding of higher plant lipid metabolism, defense mechanisms and explore more ways of crop engineering. </div>

</div>





<div class="abs">

<div style="font-weight:bold">STO &#40;Salt Tolerance protein&#41; participates in the light signalling pathway in plants.</div>

<div>

	<span style="text-decoration: underline" >
	Marta Rodriguez-Franco</span>
	<sup>1</sup>, 

	<span  >
	Martin Indorf</span>
	<sup>1</sup>, 

	<span  >
	Katrin Marquardt</span>
	<sup>1</sup>, 

	<span  >
	Felipe Sarmiento</span>
	<sup>1</sup>, 

	<span  >
	Huili Yan</span>
	<sup>1</sup>, 

	<span  >
	Eija Schulze</span>
	<sup>1</sup>, 

	<span  >
	Gunther Neuhaus</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Cell Biology, University of Freiburg, Schaenzlestrasse 1, 79104 Freiburg, Germany

</div>

<div>The Salt Tolerance protein &#40;STO&#41;, and its homologue STH, of Arabidopsis were identified as proteins conferring salt tolerance to yeast cells. In order to uncover their function, we isolated <span style="font-style: italic">STO</span> and <span style="font-style: italic">STH</span> T&#45;DNA insertion lines and generated RNAi and overexpressor Arabidopsis plants. Here we present data of these lines indicating that STO acts as a negative regulator in phytochrome and blue light signalling. Transcription analysis of <span style="font-style: italic">STO</span> uncovered a light and circadian dependent regulation of gene expression and analysis of light regulated genes revealed that STO is involved in the regulation of <span style="font-style: italic">CHS</span> expression during de&#45;etiolation. In addition, we could show that CONSTITUTIVE PHOTOMORPHOGENESIS 1 &#40;COP1&#41; represses the transcription of <span style="font-style: italic">STO</span> and contributes to the destabilisation of the protein in etiolated seedlings. Microscopic analysis revealed that the STO:eGFP and STH:eGFP fusion protein are located in the nucleus, and STO:eGFP accumulates in a light&#45;dependent manner. Colocalisation experiments of these two proteins with COP1 in transient transformation assays will be presented and the role in the light signalling pathway will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">HSP90 regulates heat shock response that is responsible for heat adaptation</div>

<div>

	<span style="text-decoration: underline" >
	Kenji Yamada</span>
	<sup>1</sup>, 

	<span  >
	Yoichiro Fukao</span>
	<sup>2</sup>, 

	<span  >
	Makoto Hayashi</span>
	<sup>1</sup>, 

	<span  >
	Mitsue Fukazawa</span>
	<sup>1</sup>, 

	<span  >
	Iku Suzuki</span>
	<sup>1</sup>, 

	<span  >
	Mikio Nishimura</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute for Basic Biology, Aichi, Japan, 

	<sup>2</sup>Nara Institute of Science and Technology, Nara, Japan

</div>

<div>Plant survival requires the ability to adapt to heat. When plants are subjected to heat shock &#40;a slightly higher temperature than the germinating temperature&#41;, the expression of various genes is induced, and the plants become tolerant to much higher temperatures. The molecular mechanisms responsible for the induction of such heat tolerance remain poorly understood. Here, we found that HSP90 negatively regulates heat&#45;inducible genes and that transiently inhibiting HSP90 in Arabidopsis seedlings induces heat&#45;inducible genes and heat adaptation. Moreover, in transgenic Arabidopsis, heat shock reduced the activity of exogenously expressed glucocorticoid receptor &#40;GR&#41;. As GR activity depends on HSP90, this suggests that heat shock reduces HSP90 activity in vivo. Microarray analysis revealed that many of the genes that are upregulated by both heat shock and HSP90 inhibitors are involved in protein folding and degradation, suggesting that the activation of a protein maintenance system is a crucial part of this response. A motif searching revealed that most of these genes have heat shock response element &#40;HSE&#41;&#45;like motifs in their promoters, which suggests that heat shock transcription factor &#40;HSF&#41; is involved in the response to HSP90 inhibition. Several HSF genes are expressed constitutively in Arabidopsis in the absence of heat shock, including AtHsfA1d and AtHsfA4c, whose constitutive expression is unchanged by heat shock or HSP90 inhibitor treatment. Recombinant AtHsfA1d protein recognizes the HSE motif, indicating that AtHsfA1d is involved in the heat shock response. Thus, it appears that in the absence of heat shock, HSP90 actively suppresses HSF function. Upon heat shock, HSP90 is transiently inactivated, which leads to HSF activation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Oxidative damage to respiratory function in plants: understanding the mechanisms behind selective protein damage in mitochondria during environmental oxidative stress</div>

<div>

	<span style="text-decoration: underline" >
	A. Harvey Millar</span>
	<sup>1</sup>, 

	<span  >
	Alison M. Winger</span>
	<sup>1</sup>, 

	<span  >
	Yew-Foon Tan</span>
	<sup>1</sup>, 

	<span  >
	Nicolas L. Taylor</span>
	<sup>1</sup>, 

	<span  >
	Adam Carroll</span>
	<sup>1</sup>, 

	<span  >
	Joshua L. Heazlewood</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Univeristy of Western Australia, Perth, WA, Australia

</div>

<div>Respiration is a fundamental process in plant cell function and is also a site of damage during environmental stress leading to loss of energy generation and restriction of both carbon and nitrogen metabolism. Analysis of the changing role of the plant mitochondrial compartment under oxidative stress is providing new insights into the nature and mechanisms of oxidative stress and damage to plant cellular function. We have been analysing the mitochondrial proteome &#40;1&#41; and have now defined links between lipid peroxidation, protein damage and protease activation using proteomic approaches &#40;2&#45;4&#41;. We have coupled this to evidence of metabolic changes &#40;using GC&#45;MS profiling&#41; to identify the particular mitochondrial processes that are most susceptible to damage and degradation during stress induced by oxidative conditions in Arabidopsis. We show that a series of cumulative factors: differential susceptibility of proteins to lipid peroxidation modification due to surface reactive amino acid moieties; affinity of specific proteins for transition metals that then catalyse localised ROS formation and lipid peroxidation reactions; and the presence of proteases that selectively cleave oxidatively&#45;modified proteins, play integrated roles in the nature and extent of damage.  We are using this to build evidence for elements in retrograde signaling to the nucleus of mitochondrial oxidative stress status. This provides a model system for understanding the mechanism of selective protein damage within a plant organelle and provides tools to consider engineering stress tolerance in respiratory energy generation in plants. &#13;<br/>&#13;<br/>1. Heazlewood et al &#40;2004&#41; Plant Cell 16:241&#45;256.&#13;<br/>2. Taylor et al &#40;2005&#41; Molecular and Cellular Proteomics 4:1122&#45;1133.&#13;<br/>3. Winger et al &#40;2005&#41; Biochemical Journal 387:865&#45;870.&#13;<br/>4. Millar et al &#40;2005&#41; Trends in Plant Science 10:36&#45;43.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular analysis of <span style="font-style: italic">FT</span> orthologs defines distinct mechanisms for controlling photoperiodic flowering in <span style="font-style: italic">Arabidopsis</span> and <span style="font-style: italic">Pharbitis</span></div>

<div>

	<span style="text-decoration: underline" >
	Ryosuke Hayama</span>
	<sup>1</sup>, 

	<span  >
	Bhavna Agashe</span>
	<sup>1</sup>, 

	<span  >
	Rod King</span>
	<sup>2</sup>, 

	<span  >
	George Coupland</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, 

	<sup>2</sup>Commonwealth Scientific and Industrial Research Organization

</div>

<div>The photoperiodic response of flowering is mediated by the interaction between environmental light signals and the circadian clock. Recent molecular studies revealed that in the long&#45;day plant <span style="font-style: italic">Arabidopsis</span> the length of the day is measured by a mechanism that involves the post&#45;transcriptional activation of a clock&#45;controlled gene <span style="font-style: italic">CONSTANS</span> &#40;<span style="font-style: italic">CO</span>&#41; by light. Under LDs <span style="font-style: italic">CO</span> transcripts begin to rise in the middle of the day, and expression of <span style="font-style: italic">CO</span> while the plants are exposed to light induces the protein resulting in activation of transcription of the <span style="font-style: italic">FLOWERING LOCUS T</span> &#40;<span style="font-style: italic">FT</span>&#41; gene, and flowering occurs. In contrast under SDs CO activation does not occur because the transcripts of <span style="font-style: italic">CO</span> only rises in the night and therefore <span style="font-style: italic">FT</span> is not induced.&#13;<br/>Here we illustrate at the molecular level a distinct mechanism for controlling the photoperiodic response of flowering in the short&#45;day plant <span style="font-style: italic">Pharbitis</span>, which measures the length of the night. We isolated the ortholog of <span style="font-style: italic">FT</span> &#40;<span style="font-style: italic">PhFT</span>&#41; from <span style="font-style: italic">Pharbitis</span>. The short&#45;day induction of the <span style="font-style: italic">PhFT</span> mRNA expression, as well as the early&#45;flowering phenotype of transgenic <span style="font-style: italic">Arabidopsis</span> and <span style="font-style: italic">Pharbitis</span> plants that overexpresses the gene demonstrated the participation of <span style="font-style: italic">PhFT</span> in the photoperiodic response of flowering in <span style="font-style: italic">Pharbitis</span>. <span style="font-style: italic">PhFT</span> transcript abundance rises during the night and shows a circadian rhythm in constant dark, and this rhythm is set by light off at dusk. Exposure to continuous light dampened this rhythmicity and suppressed <span style="font-style: italic">PhFT</span> expression. We propose that in <span style="font-style: italic">Pharbitis</span> a light&#45;sensitive circadian clock set by dusk is dedicated to measure the length of the night. This clock generates a rhythm and if this rhythm oscillates sufficiently long in the dark <span style="font-style: italic">PhFT</span> is activated and flowering occurs.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification and Characterization of Transcription Factors involved in Salt and Osmotic Stress Response</div>

<div>

	<span style="text-decoration: underline" >
	Felix Lippold</span>
	<sup>1</sup>, 

	<span  >
	Magdalena Musialak</span>
	<sup>1</sup>, 

	<span  >
	Diego Sanchez</span>
	<sup>1</sup>, 

	<span  >
	Franziska Schwabe</span>
	<sup>1</sup>, 

	<span  >
	Tomasz Kobylko</span>
	<sup>1</sup>, 

	<span  >
	Rosa Morcuende</span>
	<sup>2</sup>, 

	<span  >
	Dirk Hincha</span>
	<sup>1</sup>, 

	<span  >
	Wolf R&#252;diger Scheible</span>
	<sup>1</sup>, 

	<span  >
	Michael Udvardi</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Molecular Plant Physiology, Wissenschaftspark Golm, Am M&#252;hlenberg 1, Potsdam-Golm, 14476, Germany, 

	<sup>2</sup>Instituto de Recursos Naturales y Agrobiolog&#237;a de Salamanca, Consejo Superior de Investigaciones Cient&#237;ficas, 37008 Salamanca, Spain, 

	<sup>3</sup>Samuel Roberts Noble Foundation, 2510 Sam Noble Pky., Ardmore, OK 73401, USA

</div>

<div>Transcription factors &#40;TFs&#41; are master&#45;control proteins in all living cells. They often exhibit sequence&#45;specific DNA binding and are capable of activating or repressing transcription of multiple target genes. In this way, they control or influence many biological processes, including cell cycle progression, metabolism, growth and development, and responses to the environment.&#13;<br/>To identify TFs that have an essential regulatory impact in response to salt stress, we used a qRT&#45;PCR based platform to screen almost all Arabidopsis thaliana TFs &#40;Czechowski et al., 2004&#41;. Arabidopsis thaliana seedlings have been grown for nine days in a liquid culture system and subsequently stressed with 100 mM NaCl respectivly 200mM Mannitol for three hours. Material from three biological replicates has been used to screen approximately 2000 TFs. Among them, twenty candidate genes displaying an altered gene expression with respect to salt stress or osmotic treatment were selected. To confirm the results additional experiments have been carried out including a hydroponic growth system and time course experiments. Six out of this 20 TFs showed an significant expression change combined with an interesting expression pattern. They have been selected for further analysis using a reverse genetic approach. Overexpressors were analyzed on a physiological and molecular level. The results from these experiments revealed potential regulators of early salt stress response whose impact on gene expression is now examined on a genome wide level. &#13;<br/>&#13;<br/>Czechowski, T. et al. &#40;2004&#41; &#34;Real&#45;time RT&#45;PCR profiling of over 1400 Arabidopsis transcription factors: unprecedented sensitivity reveals novel root&#45; and shoot&#45;specific genes&#34;, The Plant Journal 38, 366&#45;379.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">HY5 interacting B&#45;box proteins are members of a novel class of transcription factors and regulate plant growth and development.</div>

<div>

	<span style="text-decoration: underline" >
	Magnus Holm</span>
	<sup>1</sup>, 

	<span  >
	Henrik Johansson</span>
	<sup>1</sup>, 

	<span  >
	Chamari Hettiarachchi</span>
	<sup>1</sup>, 

	<span  >
	Sourav Datta</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Gothenburg University, CMB-Molecular Biology

</div>

<div>Light is an important factor for plant growth and development and a dramatic example of light signaling can be seen during seedling de&#45;etiolation. This process is induced by wavelength specific photoreceptors and entails a dramatic transcriptional reprogramming. We study a group of Arabidopsis proteins that were identified through their interactions with two key regulators of light signaling, COP1 and HY5. COP1, an E3 ubiquitin ligase, represses light dependent development by targeting positive factors such as the transcription factor HY5 for degradation in the dark. Our genetic and molecular analysis of the HY5 and&#47;or COP1 interacting proteins indicate that they are positive downstream regulators of light signaling and one of our goals is to characterize how light regulates these proteins and how they in turn regulate light dependent development. Furthermore, five of the proteins have tandem repeated Zn2&#43; binding B&#45;boxes. The B&#45;box is generally regarded as a protein&#45;protein interaction domain and is often found in the so&#45;called RBCC proteins, which also contain RING finger and Coiled&#45;Coil domains. RBCC proteins such as the tumor supressor PML in humans and LIN&#45;41, which is involved in developmental timing in C. elegans, are important regulators in animals. On the other hand, no RBCC proteins exist in Arabidopsis. Our findings that B&#45;box containing proteins interact with the RING finger and Coiled&#45;Coil containing COP1 protein is therefore interesting. However, our recent results, which strongly suggest that the B&#45;box proteins constitute a novel class of DNA binding transcription factors, are remarkable and have important implications for the interpretation of RBCC protein function. Taken together, we have genetic, functional and cell biological data suggesting that these B&#45;box proteins are transcription factors that positively regulate HY5 activity in light and are currently investigating if the B&#45;box domain, previously considered a protein&#45;protein interaction domain, is a novel DNA binding domain. The characterization of this novel B&#45;box function is an important goal for our research. </div>

</div>





<div class="abs">

<div style="font-weight:bold">COP9 Signalosome Interacts with RNA Processing Factors in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Shiori AKI</span>
	<sup>1</sup>, 

	<span  >
	Atsuhiro OKA</span>
	<sup>1</sup>, 

	<span  >
	Tomohiko TSUGE</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ICR Kyoto University, Kyoto, JAPAN

</div>

<div>The COP9 signalosome &#40;CSN&#41; is a nuclear protein complex essential for plant and animal development, regulating proteasome&#45;mediated proteolysis. To characterize novel mechanisms of CSN, CSN subunit 1 &#40;CSN1&#41; binding proteins were isolated and analyzed in <span style="font-style: italic">Arabidopsis</span>.<br/>SAP130, one of the identified CSN1 binding proteins, is a component of the SF3b complex which is involved in mRNA splicing. In <span style="font-style: italic">Arabidopsis</span>, AtSAP130 was coded by two genes &#40;<span style="font-style: italic">AtSAP130a</span> and <span style="font-style: italic">AtSAP130b</span>&#41; which both map on chromosome 3. The mRNAs for both genes were detected in all tested organs. The expression level of <span style="font-style: italic">AtSAP130b</span> was found to be significantly higher than that of <span style="font-style: italic">AtSAP130a</span> in all tissues. Pull down experiments <span style="font-style: italic">in vitro</span> revealed that AtSAP130 directly binds AtCSN1 at the N terminal portion of AtCSN1. Here we discuss the possible roles of CSN in post&#45;transcriptional regulation in <span style="font-style: italic">Arabidopsis</span>.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">ZEITLUPE is a circadian photoreceptor stabilized by a blue&#45;light&#45;enhanced protein&#45;protein interaction</div>

<div>

	<span  >
	Woe-Yeon Kim</span>
	<sup>1</sup>, 

	<span  >
	Sumire Fujiwara</span>
	<sup>1</sup>, 

	<span  >
	Sung-Suk Suh</span>
	<sup>1</sup>, 

	<span  >
	Jeongsik Kim</span>
	<sup>2</sup>, 

	<span  >
	Yumi Kim</span>
	<sup>2</sup>, 

	<span  >
	Linqu Han</span>
	<sup>1</sup>, 

	<span  >
	Hong Gil Nam</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	David E. Somers</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Cellular and Molecular Biology/Plant Biotechnology Center, Ohio State University, Columbus, OH 43210, USA, 

	<sup>2</sup>Department of Life Science, Pohang University of Science and Technology, Pohang, Kyungbuk, 790-784, South Korea

</div>

<div>Appropriate regulation of rhythmic cycling of key elements of the clock is essential to maintain robust oscillations. Cyclic oscillation of ZEITLUPE &#40;ZTL&#41; is required to sustain normal circadian cycling via the proteasome&#45;dependent degradation of TOC1, a key element of the central oscillator in plants. ZTL is an F&#45;box protein and its cyclic oscillation is post&#45;transcriptionally regulated by the proteasome. We have identified a factor which regulates ZTL protein abundance. In its absence, the normal four&#45;fold diurnal cycling in ZTL is post&#45;transcriptionally eliminated, resulting in constitutively low ZTL accumulation. Their interaction occurs through the N&#45;terminal ZTL LOV domain, which is necessary and sufficient for the interaction. Further, this interaction is strongly and specifically enhanced by blue light, via the N&#45;terminal flavin&#45;binding LIGHT, OXYGEN OR VOLTAGE &#40;LOV&#41; domain of ZTL. Mutations within the LOV domain that greatly diminish the interactions also lead to strongly reduced ZTL levels. A C82A transition in the LOV domain, implicated in the flavin&#45;dependent photochemistry, eliminates blue&#45;light enhanced binding. These data establish ZTL as a blue&#45;light photoreceptor, by which light absorption facilitates its own stability via a blue&#45;light enhanced protein&#45;protein interaction. Circadian control of message levels of the ZTL&#45;interacting protein, and subsequent cycling of the protein, confers a post&#45;translational rhythm on ZTL protein abundance, defining a novel mechanism to establish and sustain circadian oscillations. Additional progress on this mechanism will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Transposase&#45;Derived Proteins FHY3 and FAR1 From Arabidopsis Modulate phyA Signaling Homeostasis Through Direct Activation of <span style="font-style: italic">FHY1</span> and <span style="font-style: italic">FHL</span> Expression</div>

<div>

	<span  >
	Rongcheng Lin</span>
	<sup>1</sup>, 

	<span  >
	Lei Ding</span>
	<sup>1</sup>, 

	<span  >
	Claudio Casola</span>
	<sup>2</sup>, 

	<span  >
	Cedric Feschotte</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Haiyang Wang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Boyce Thompson Institute for Plant Research, Cornell University, Ithaca, NY, USA, 

	<sup>2</sup>Department of Biology, University of Texas, Arlington, TX, USA

</div>

<div>Phytochrome A &#40;phyA&#41; is the primary photoreceptor for mediating various far&#45;red light induced responses in higher plants. Following photoactivation, phyA translocates into and accumulates in the nucleus in a process dependent on two homologous proteins FHY1 and FHL. Once in the nucleus, phyA interacts with a set of transcription factors to regulate phyA&#45;responsive gene expression. In this study, we demonstrate that Arabidopsis <span style="font-style: italic">FHY3</span> and <span style="font-style: italic">FAR1</span>, which encode two homologous proteins related to <span style="font-style: italic">Mutator</span>&#45;like transposases, act redundantly to modulate phyA nuclear accumulation through directly binding to and activating the expression of <span style="font-style: italic">FHY1</span> and <span style="font-style: italic">FHL</span>, with FHY3 playing a more predominant role. We show that FHY3 and FAR1 bind directly to the upstream DNA regions of <span style="font-style: italic">FHY1</span> and <span style="font-style: italic">FHL</span> via a zinc finger&#45;like motif most likely acquired from an ancestral transposase. In addition, reporter assays in yeast and Arabidopsis protoplasts indicate that FHY3 possesses an intrinsic capability to mediate transcriptional activation of an adjacent gene. Transcriptional activation is separable from DNA&#45;binding activity, but requires the presence of two residues highly conserved in <span style="font-style: italic">Mutator</span>&#45;like transposases. Constitutive expression of <span style="font-style: italic">FHY1</span> driven by the 35S promoter can largely suppress the mutant phenotypes of <span style="font-style: italic">fhy3&#45;4</span>, <span style="font-style: italic">far1&#45;2</span>, and the <span style="font-style: italic">fhy3 far1</span> double mutant, supporting that <span style="font-style: italic">FHY1</span> acts downstream of <span style="font-style: italic">FHY3</span> and <span style="font-style: italic">FAR1</span>. Finally, we also show that the expression of <span style="font-style: italic">FHY3</span> and <span style="font-style: italic">FAR1</span> is subject to a negative feedback regulation by phyA signaling. Together these results suggest that FHY3 and FAR1 define a novel class of transcription factors that were co&#45;opted from an ancient <span style="font-style: italic">Mutator</span>&#45;like transposase to modulate phyA signaling homeostasis in higher plants. </div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis DAY NEUTRAL FLOWERING gene and its role in the control of flowering time.</div>

<div>

	<span  >
	Karl Morris</span>
	<sup>1</sup>, 

	<span  >
	Sarah Thornber</span>
	<sup>1</sup>, 

	<span  >
	Lesley Codrai</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Stephen Jackson</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Warwick University, Warwick, UK

</div>

<div>Mutation of the Arabidopsis DAY NEUTRAL FLOWERING &#40;DNF&#41; gene causes the mutant plants to flower as early in 8h short days &#40;SD&#41; as they would if grown in 16h long days &#40;LD&#41;, the dnf mutant is therefore essentially aphotoperiodic. The reason for this is because expression of the FT gene is induced in SD as it is in LD. Analysis of co,dnf double mutants indicates that an active CONSTANS &#40;CO&#41; gene is required for early flowering of the dnf mutant, furthermore CO expression has been shown to be altered in the mutant. This places DNF above CO in the photoperiodic pathway. The DNF gene is expressed transiently at a specific time of the day when its role is to repress CO transcription, in the mutant this does not occur resulting in increased CO expression earlier in the day. The effect of the dnf mutation on CO expression is correlated to the observed shift in critical nightlength for flowering in the mutant.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterisation of dracula &#40;no avoidance of shade&#41; and icarus &#40;extreme avoidance of shade&#41; mutants in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Paul Devlin</span>
	<sup>1</sup>, 

	<span  >
	Xuewen Wang</span>
	<sup>1</sup>, 

	<span  >
	Irma Roig Villanova</span>
	<sup>2</sup>, 

	<span  >
	Jaime Martinez Garcia</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Royal Holloway University of London, Egham, UK, 

	<sup>2</sup>Institut de Biologia Molecular de Barcelona y Consorci CSIC-IRTA, Barcelona, Spain

</div>

<div>Plants are capable of detecting red&#45;depleted light reflected from neighbouring vegetation which induces a classical, shade avoidance response, including an increase in elongation growth, a decrease in the degree of branching and a shortening of the time to flowering. The phytochrome photoreceptors involved in the perception of vegetative shade have been well characterised but we know little about the signal transduction events downstream of these. &#13;<br/>We used a transcriptomic approach to identify a number of genes differentially expressed under simulated shade in Arabidopsis thaliana and have used this knowledge to generate a shade responsive marker for a high throughput, non&#45;invasive, real&#45;time bioluminescent assay. &#13;<br/>We have carried out a mutant screen using this marker and have identified several mutants in which the response to shade is defective. These are divided into two classes: dracula &#40;no avoidance of shade&#41; and icarus &#40;extreme avoidance of shade&#41;. In many cases, these mutants show additional, novel defects in light signalling. &#13;<br/>We have mapped the approximate location of a number of these mutations. Curiously, one of these mutations is the result of a novel mutation in one of the phytochrome photoreceptors that appears to result in a constitutive non&#45;shade&#45;avoiding phenotype with respect to our marker.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Constitutive over&#45;expression of the MYB&#45;related transcription factor PHR1 results in accumulation of high levels of phosphate in shoots of Arabidopsis.</div>

<div>

	<span style="text-decoration: underline" >
	Tom Hamborg Nielsen</span>
	<sup>1</sup>, 

	<span  >
	Lena Nilsson</span>
	<sup>1</sup>, 

	<span  >
	Renate M&#252;ller</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Dept. Plant Biology, University of Copenhagen, Copenhagen, Denmark, 

	<sup>2</sup>Dept. Agricultural Sciences, University of Copenhagen, Copenhagen, Denmark

</div>

<div>Plants have evolved a number of adaptive strategies to cope with fluctuations in phosphate supply. The current knowledge of the transcriptional regulation of the phosphate starvation response in plants is limited.  However, the MYB&#45;related transcription factor PHR1 &#40;At4g28610&#41; is known to be involved in the P&#45;starvation response &#40;Rubio et al., 2001, Gene Dev 15: 2122&#45;2133&#41;. In this study we characterise a T&#45;tagged knock&#45;out phr1&#45;mutant, and a series of transgenic plant lines which over&#45;express PHR1 in wild type and phr1 mutant background. &#13;<br/>    The knock&#45;out mutant has an altered phosphate allocation between root and shoot, accumulates less sugar and starch than P&#45;starved wildtype and has a lower AGPase activity. The mutant is impaired in induction of a subset of phosphate&#45;starvation induced genes including a transporter gene &#40;PHT1&#45;7&#41;, a ribonuclease gene &#40;RNS1&#41; and several genes involved in anthocyanin biosynthesis. Over&#45;expression of PHR1 in the phr1 mutant rescues the responsiveness to P&#45;starvation and leads to wild type levels of sugars and starch during phosphate starvation. This confirms the involvement of PHR1 in adjusting carbon metabolism. Furthermore, over&#45;expression of PHR1 in both wild type and phr1 mutant leads to strongly increased phosphate content irrespective of P&#45;regime. &#13;<br/>    This study emphasises the importance of PHR1 in regulation of phosphate starvation responses, and shows that targeting a key regulatory element in the phosphate starvation regulatory network may represent a useful approach for molecular breeding of plants toward more efficient phosphate uptake and assimilation.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic engineering of farnesylation for plant drought tolerance and yield protection</div>

<div>

	<span  >
	Jiangxin Wan</span>
	<sup>1</sup>, 

	<span  >
	Yang Wang</span>
	<sup>1</sup>, 

	<span  >
	Jifeng Ying</span>
	<sup>1</sup>, 

	<span  >
	Monika Kuzma</span>
	<sup>1</sup>, 

	<span  >
	Michelle Beaith</span>
	<sup>1</sup>, 

	<span  >
	Maryse Chalifloux</span>
	<sup>1</sup>, 

	<span  >
	Angela Sample</span>
	<sup>1</sup>, 

	<span  >
	Charlene McArthur</span>
	<sup>1</sup>, 

	<span  >
	Tina Uchacz</span>
	<sup>1</sup>, 

	<span  >
	Carlene Sarvas</span>
	<sup>1</sup>, 

	<span  >
	David Dennis</span>
	<sup>1</sup>, 

	<span  >
	Peter McCourt</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Yafan Huang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Performance Plants, Inc., 

	<sup>2</sup>University of Toronto

</div>

<div>Protecting crop yield under drought stress is a major challenge for modern agriculture. One biotechnological target for improving plant drought tolerance is the genetic manipulation of the stress response to the hormone abscisic acid &#40;ABA&#41;. Previous genetic studies have implicated the involvement of the b&#45;subunit of the Arabidopsis farnesyltransferase &#40;ERA1&#41; in the regulation of ABA sensing and drought tolerance. Here we show that molecular manipulation of protein farnesylation in Arabidopsis, through down&#45;regulation of either the a&#45; or b&#45;subunit of farnesyltransferase, enhances the plant&#8217;s response to ABA and drought tolerance. To test the effectiveness of tailoring farnesylation for drought tolerance and yield protection in a crop plant, transgenic Brassica napus carrying down&#45;regulation constructs of either the a&#45; or b&#45;subunit of farnesyltransferase were examined. In comparison with the non&#45;transgenic control, the transgenic canola showed enhanced ABA sensitivity, as well as significant reduction of stomatal conductance and water transpiration under drought stress conditions. The transgenic plants were more resistant to water deficit&#45;induced seed abortion during flowering. Results from multiple years and multiple locations of field trial studies suggest that with adequate water, the transgenic canola plants produced the same amount of seed as the parental control. However, under moderate drought stress conditions at flowering, the seed yields of the transgenic canola were significantly higher than the control.  These results represent a successful demonstration of engineered drought tolerance and yield protection in a crop plant under laboratory and field conditions.  Using protein farnesyltransferase as an effective target, we have also made significant progresses on engineering drought tolerance in other important crop species.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Mapping of QTL for Pieris resistance and for defense&#45;related traits in Arabidopsis Da&#40;1&#41;&#45;12 &#215; Ei&#45;2</div>

<div>

	<span style="text-decoration: underline" >
	Marina Pfalz</span>
	<sup>1</sup>, 

	<span  >
	Heiko Vogel</span>
	<sup>1</sup>, 

	<span  >
	Juergen Kroymann</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Chemical Ecology

</div>

<div>Natural genetic variation among Arabidopsis accessions has been successfully exploited to map and clone QTL &#40;quantitative trait loci&#41; for many quantitative traits, including plant resistance against generalist insect herbivores such as Spodoptera exigua or Trichoplusia ni. Typically, plant damage caused by generalist insect herbivores is negatively correlated with increasing glucosinolate concentration or with increasing myrosinase activity, and resistance QTL co&#45;localize with glucosinolate or myrosinase QTL. These results demonstrate that the glucosinolate&#45;myrosinase system plays a major role in the defense of cruciferous plants against generalist insect herbivores. <br/><br/>Less clear, however, is whether and how cruciferous plants defend themselves against insects that have specialized on crucifers, such as Plutella xylostella and Pieris brassicae. These insects possess counteradaptations that help avoiding deleterious effects arising from toxic glucosinolate hydrolysis products. Nonetheless, highly replicated insect feeding trials on 16 Arabidopsis accessions revealed substantial genetic variation within Arabidopsis for resistance against specialist herbivores. We are now using a recombinant inbred line &#40;RIL&#41; population developed from a cross between the Arabidopsis accessions Da&#40;1&#41;&#45;12 and Ei&#45;2 to map QTL controlling resistance against crucifer specialists, and to investigate a potential impact of the glucosinolate&#45;myrosinase system on Pieris brassicae herbivory.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional analysis of dominant GAF&#45;domain tyrosine mutants of Arabidopsis phytochromes in transgenic plants</div>

<div>

	<span  >
	Yi-shin Su</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	J. Clark Lagarias</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Davis, (CA), USA

</div>

<div>Light sensing by phytochromes, a family of biliprotein photoreceptors that are widely distributed nature, exploits the reversible photoisomerization of their covalently bound linear tetrapyrrole &#40;bilin&#41; prosthetic groups. Initially undertaken to examine the biological activity of a recently identified class of highly fluorescent, poorly photoactive phytochrome mutants in transgenic plants, the present investigation led to the unexpected discovery of constitutively active phytochrome alleles that possess mutations in a conserved GAF domain tyrosine &#40;YGAF&#41; residue. Most pronounced gain&#45;of&#45;function activities were observed for the Y276H allele of Arabidopsis phyB &#40;YHB&#41; whose expression conferred dominant constitutively photomorphogenic &#40;COP&#41; phenotypes as well as constitutive, light&#45;insensitive phyB signaling activities to both dark&#45; and light&#45;grown transgenic plants. YHB&#45;mediated COP development paralleled constitutive nuclear localization of the YHB protein and expression of light&#45;regulated genes in darkness &#45; both of which are consistent with its light&#45;independent activation. Moreover, the COP phenotype was suppressed in bilin&#45;deficient genetic backgrounds, indicating that the YHB allele encodes a bilin&#45;dependent regulator of photomorphogenesis. Phenotypic analysis of transgenic plants expressing YQB, YIB and YRB alleles further revealed that the signaling activity of phyB critically depends on the amino acid at the YGAF position. By comparison with YHB, the COP phenotype conferred by the Y242H allele of phyA &#40;YHA&#41; was less pronounced, and a dominant&#45;negative response to YHA expression was observed in wild&#45;type genetic backgrounds. Taken together, these results implicate participation of this conserved YGAF residue in the transduction of light&#45;driven bilin chromophore isomerization to phytochrome&#45;mediated regulation of plant growth and development. Dominant, constitutively active phytochrome alleles are of potential agronomic significance since their introduction into any transformable crop plant species represents a practical approach to suppress deleterious responses to light quality in field environments.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Function of de&#45;phosphorylated NPH3 in phototropic response.</div>

<div>

	<span style="text-decoration: underline" >
	Tomoko Tsuchida-Mayama</span>
	<sup>1</sup>, 

	<span  >
	Michiharu Nakano</span>
	<sup>1</sup>, 

	<span  >
	Yukiko Uehara</span>
	<sup>1</sup>, 

	<span  >
	Noriko Fujisawa</span>
	<sup>1</sup>, 

	<span  >
	Kiyotaka Okada</span>
	<sup>2</sup>, 

	<span  >
	Tatsuya Sakai</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, Yokohama, Japan, 

	<sup>2</sup>Kyoto University, Kyoto, Japan

</div>

<div>The phototropic response of <span style="font-style: italic">Arabidopsis thaliana</span> seedlings is initiated by the blue&#45;light photoreceptors, phototropin &#40;phot&#41; 1 and phot2, and by their signal transducers, NONPHOTOTROPIC HYPOCOTYL &#40;NPH&#41; 3 and ROOT PHOTOTROPISM &#40;RPT&#41; 2. Although previous studies have indicated that phot1 contains blue&#45;light&#45;activated autophosphorylation activity and binds to NPH3 and RPT2, its biochemical function during the induction of the phototropic response remains unclear. In this report, we show that the NPH3 protein is phosphorylated under dark conditions and is de&#45;phosphorylated under blue light, and that phot1 is necessary for blue&#45;light&#45;dependent de&#45;phosphorylation of NPH3 but not phot2 under any fluence rates of blue light that we tested. Since seedlings of phot1 mutant showed that partially phototropism under high fluence rate of blue light, de&#45;phosphorylation of NPH3 is not essential to phototropism at least in phot2 signaling pathway. The phosphorylation sites of NPH3 were predicted using prediction server, NetPhos 2.0 &#40;http:&#47;&#47;www.cbs.dtu.dk&#47;services&#47;NetPhos&#47;&#41;. The functional significance of each of the predicted phosphorylation sites in NPH3 was assessed by site&#45;directed mutagenesis of each specific Ser or Thr to Ala followed by of analysis of NPH3 phoshorylation state.  Four candidates of NPH3 phosphorylation sites were identified. We are also testing abilities of the construct, altered or deleted phosphorylated region, to rescue the null <span style="font-style: italic">nph3</span> &#40;SALK_110039&#41; mutant. We will discuss function of de&#45;phosphorylated NPH3 in phototropic response here.</div>

</div>





<div class="abs">

<div style="font-weight:bold">SCABP8&#47;CBL10, a Putative Calcium Sensor, Interacts with the Protein Kinase SOS2 to Protect <span style="font-style: italic">Arabidopsis</span> Shoots from Salt Stress</div>

<div>

	<span style="text-decoration: underline" >
	Ruidang Quan</span>
	<sup>1</sup>, 

	<span  >
	Huixin Lin</span>
	<sup>1</sup>, 

	<span  >
	Imelda Mendoza</span>
	<sup>2</sup>, 

	<span  >
	Yuguo Zhang</span>
	<sup>1,3</sup>, 

	<span  >
	Wanhong Cao</span>
	<sup>1</sup>, 

	<span  >
	Yongqing Yang</span>
	<sup>1</sup>, 

	<span  >
	Mei Shang</span>
	<sup>1</sup>, 

	<span  >
	Shouyi Chen</span>
	<sup>3</sup>, 

	<span  >
	Jos&#233; M. Pardo</span>
	<sup>2</sup>, 

	<span  >
	Yan Guo</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute of Biological Sciences, Beijing, 7 Science Park Road, Zhongguancun Life Science Park, Beijing 102206, P.R. China, 

	<sup>2</sup>Instituto de Recursos Naturales y Agrobiolog&#237;a, Consejo Superior de Investigaciones Cient&#237;ficas, Sevilla 41012, Spain, 

	<sup>3</sup>Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Chaoyang District, Beijing 100101, China

</div>

<div>The SOS &#40;Salt Overly Sensitive&#41; pathway plays an essential role in conferring salt tolerance in <span style="font-style: italic">Arabidopsis</span>. Under salt stress, the calcium sensor SOS3 activates SOS2, a protein kinase, which in turn positively regulates the activity of a plasma membrane sodium&#47;proton antiporter, SOS1. Here we show that SOS3 acts primarily in roots under salt stress. By contrast, the SOS3 homologue, SCABP8&#47;CBL10 &#40;SOS3 like Calcium Binding Protein 8, and Calcineurin B&#45;Like 10&#41;, functions mainly in the shoot response to salt toxicity. While root growth is reduced in the <span style="font-style: italic">sos3</span> mutants in the presence of NaCl, the salt sensitivity of <span style="font-style: italic">scabp8</span> is more prominent in the shoot tissues. SCABP8 is further shown to bind calcium, interact with SOS2 both in <span style="font-style: italic">vitro</span> and <span style="font-style: italic">in vivo</span>, recruit SOS2 to the plasma membrane, enhance SOS2 activity in a calcium dependent manner, and activate SOS1 in yeast. In addition, the <span style="font-style: italic">sos3 scabp8</span> and <span style="font-style: italic">sos2 scabp8</span> double mutants display a phenotype similar to <span style="font-style: italic">sos2</span>, which is more sensitive to salt than either the <span style="font-style: italic">sos3</span> or <span style="font-style: italic">scabp8</span> single mutant. Over&#45;expression of <span style="font-style: italic">SCABP8</span> in the <span style="font-style: italic">sos3</span> mutant partially rescues the <span style="font-style: italic">sos3</span> salt sensitive phenotype. However, over&#45;expression of SOS3 in the <span style="font-style: italic">scabp8</span> mutant fails to complement <span style="font-style: italic">scabp8</span>. These results suggest that SCABP8 and SOS3 are only partially redundant in their function and each play additional and unique roles in the plant salt stress response.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Alterations in the plasma membrane microdomains during cold acclimation in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span  >
	Anzu Minami</span>
	<sup>1</sup>, 

	<span  >
	Akari Furuto</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Matsuo Uemura</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>The 21st Century Center of Excellence (COE) Program, Iwate University, Morioka, Japan, 

	<sup>2</sup>Cryobiosystem Research Center, Iwate University, Morioka, Japan

</div>

<div><span style="font-style: italic">Arabidopsis thaliana</span> increases freezing tolerance upon exposure to low, non&#45;freezing temperatures, which is known as cold acclimation. Cold acclimation results in substantial changes in compositions, functions and structure of the plasma membrane. Among these changes, a decrease in the proportion of sphingolipids &#40;i.e., glucocerebrosides&#41; and dymanic changes in protein composition have been widely reported occurring in many cold&#45;acclimated plant species including both herbaceous and woody plants. Recently, it has emerged that the plasma membrane contains microdomains that are enriched in sphingolipids and sterols and have specific protein compositions. The microdomains can be practically isolated as low&#45;density detergent&#45;resistant membrane fraction &#40;DRM&#41; from the plasma membrane. Although DRM is thought to play important roles in plasma&#45;membrane mediated biological processes, DRM functions in environmental stress adaptation are largely unknown. To investigate whether DRM&#45;associated proteins involve in cold&#45;acclimation&#45;induced enhancement of freezing tolerance of plants, we isolated DRM and compared DRM&#45;associated protein profiles before and after cold acclimation in <span style="font-style: italic">Arabidopsis thaliana</span> seedlings. When separated with SDS&#45;PAGE and 2D&#45;SDS&#45;PAGE, DRM&#45;associated proteins showed quantitative changes during cold acclimation, suggesting a possible involvement in cold acclimation process in <span style="font-style: italic">Arabidopsis thaliana</span>. In addition, DRM lipid composition seems to respond to cold treatment, which is different from the responses of the plasma membrane as a whole. We will present a progress report of identification of cold&#45;responsive DRM&#45;associated proteins using mass spectrometry and discuss the molecular aspect of the DRM&#45;associated proteins in plant cold acclimation. &#40;Supported by grants &#40;to M.U. and A.M.&#41; and the 21st Century COE program &#40;to Iwate University&#41; from MEXT, Japan&#41;</div>

</div>





<div class="abs">

<div style="font-weight:bold">Physiological activities of the N&#45;terminal photosensory and the C&#45;terminal kinase domains of phototropin 2 in transgenic Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Sam-Geun Kong</span>
	<sup>1,2</sup>, 

	<span  >
	Toshinori Kinoshita</span>
	<sup>3</sup>, 

	<span  >
	Ken-ichiro Shimazaki</span>
	<sup>3</sup>, 

	<span  >
	Nobuyoshi Mochizuki</span>
	<sup>1</sup>, 

	<span  >
	Tomomi Suzuki</span>
	<sup>1</sup>, 

	<span  >
	Akira Nagatani</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Kyoto University, Kyoto, Japan, 

	<sup>2</sup>National Institute for Basic Biology, Okazaki, Japan, 

	<sup>3</sup>Kyushu University, Fukuoka, Japan

</div>

<div>Phototropins &#40;phot1 and phot2&#41; are membrane&#45;associated blue light photoreceptors that mediate various physiological responses such as phototropism, chloroplast relocation, and stomatal opening in plants. We have recently reported that phot2 associates with the Golgi apparatus in a light&#45;dependent manner and its C&#45;terminal kinase domain is essential for the localization &#40;Kong et al., 2006&#41;. In this study, we further analyzed biological activities of the N&#45;terminal photosensory and C&#45;terminal kinase domains of phot2 in transgenic Arabidopsis. For this purpose, these domains were fused to green fluorescent protein &#40;GFP&#41; and ectopically expressed in the wild type and a <span style="font-style: italic">phot1phot2</span> double mutant of Arabidopsis. Consequently, the kinase domain fused to GFP &#40;P2CG&#41; was localized to the plasma membrane and the Golgi apparatus, whereas the photosensory domain fused to GFP &#40;P2NG&#41; was uniformly localized in the cytosol. Hence, the kinase domain but not the photosensory domain was responsible for the membrane association. Interestingly, the P2CG plants exhibited constitutive blue light responses even in dark conditions. Namely, stomata were open and chloroplasts were in the avoidance position. By contrast, P2CG, with a mutation that abolishes the kinase activity &#40;P2C&#91;D720&#47;N&#93;G&#41;, failed to exhibit these activities. However, it inhibited the blue light&#45;induced stomatal opening in the wild type background. Hence, P2CG appears to be correctly localized to the functional sites in the cell and trigger the light signal transduction through its kinase activity. By contrast to P2CG, P2NG failed to affect the phot2 responses except that P2NG partly inhibited the phototropic response caused by the endogenous phototropins.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Phytochromes and cryptochromes regulate the differential growth of Arabidopsis hypocotyls in both PGP19&#45;dependent and independent manners</div>

<div>

	<span style="text-decoration: underline" >
	Akitomo Nagashima</span>
	<sup>1</sup>, 

	<span  >
	Genki Suzuki</span>
	<sup>1</sup>, 

	<span  >
	Kensuke Saji</span>
	<sup>2</sup>, 

	<span  >
	Toshiko Furukawa</span>
	<sup>3</sup>, 

	<span  >
	Tomokazu Koshiba</span>
	<sup>3</sup>, 

	<span  >
	Masayo Sekimoto</span>
	<sup>4</sup>, 

	<span  >
	Shozo Fujioka</span>
	<sup>4</sup>, 

	<span  >
	Takeshi Kuroha</span>
	<sup>1</sup>, 

	<span  >
	Mikiko Kojima</span>
	<sup>1</sup>, 

	<span  >
	Hitoshi Sakakibara</span>
	<sup>1</sup>, 

	<span  >
	Noriko Fujisawa</span>
	<sup>1</sup>, 

	<span  >
	Yukiko Uehara</span>
	<sup>1</sup>, 

	<span  >
	Kiyotaka Okada</span>
	<sup>1,2</sup>, 

	<span  >
	Tatsuya Sakai</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, 

	<sup>2</sup>Department of Botany, Graduate School of Science, Kyoto University, 

	<sup>3</sup>Department of Biological Sciences, Tokyo Metropolitan University, 

	<sup>4</sup>RIKEN Discovery Research Institute

</div>

<div>Photoreceptors, phytochromes and cryptochromes regulate hypocotyl growth under specific conditions by suppressing negative&#45;gravitropism, modulating phototropism and inhibiting elongation.  Although those effects seem to be partially caused via the regulation of the phytohormone auxin, the molecular mechanisms underlying this process are still poorly understood.  In our present study, we demonstrate that the flabby mutation enhances the phytochrome&#45;inducible hypocotyl bending in Arabidopsis.  The FLABBY gene encodes the ABC&#45;type auxin transporter, PGP19, and its expression is suppressed by the activation of phytochromes and cryptochromes.  These results therefore reveal that phytochromes and cryptochromes have at least two effects upon the tropic responses of the hypocotyls in Arabidopsis; one is the enhancement of hypocotyl bending through the suppression of PGP19, and another is a hypocotyl bending&#45;inducing mechanism, which is independent of PGP19.  The auxin polar transport assay and the DR5::GUS expression analysis suggest that phytochromes inhibit the basipetal auxin transport and induce the asymmetric distribution of auxin in hypocotyls.  Further, activations of phytochromes and cryptochromes decrease auxin content in the aerial portion of the seedlings.  These results thus suggest that the control of auxin transport and accumulation by phytochromes and cryptochromes is a critical component of the regulation of hypocotyl growth in response to light.</div>

</div>





<div class="abs">

<div style="font-weight:bold">GIGANTEA and regulation of flowering by day length in Arabidopsis and ryegrass</div>

<div>

	<span style="text-decoration: underline" >
	Jo Putterill</span>
	<sup>1</sup>, 

	<span  >
	Karine David</span>
	<sup>1</sup>, 

	<span  >
	Moyra Black</span>
	<sup>1</sup>, 

	<span  >
	Vic Arcus</span>
	<sup>1</sup>, 

	<span  >
	Milan Gagic</span>
	<sup>2</sup>, 

	<span  >
	Igor Kardailsky</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>University of Auckland, Auckland, New Zealand, 

	<sup>2</sup>AgResearch, Palmerston North, New Zealand

</div>

<div>Major seasonal cues such as changing day length and winter cold help to synchronise the reproduction of many plants to favourable seasons of the year and enable outcrossing. Summer annual ecotypes of Arabidopsis germinate and flower rapidly in spring, while winter annuals grow vegetatively over the winter and need an extended period of cold &#40;vernalisation&#41; in order to be able to respond to the long day signal in spring. Molecular&#45;genetic analysis has revealed the photoperiod flowering time pathway which regulates flowering in response to day length in Arabidopsis. Components of this pathway that promote flowering such as GI and CO act to upregulate the floral integrator gene FT.  These genes are conserved in other evolutionarily&#45;diverse plants including rice although their regulation can be different. Relatively little is known about the molecular and biochemical role of the flowering&#45;time protein GI, which has pleiotropic effects and is a very large &#40;127kD&#41;, nuclear, plant&#45;specific protein with no domains of known biochemical function. Here we report on our work on tracking Arabidopsis GI in plants by immunoblotting which shows that GI protein oscillates through the day&#47;night cycle, accumulating during the day and that it is subjected to degradation by the 26S proteasome at night. We present recent work on analysis of the effect of photoreceptor mutations on GI accumulation. We also report on progress on biophysical characterization of the GI protein including initial Cryo EM pictures and present results on the identification and   characterization, of flowering time genes from the important forage crop ryegrass which is induced to flower by vernalisation followed by longer daylengths in spring. Interestingly, over expression of a rye grass FT gene in Arabidopsis rescues an ft mutation and leads to an increase in the abundance of the endogenous gene supporting the idea of positive feedback regulation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Dual targeting of mungbean &#40;Vigna radiate L.&#41; MLTI107 encoding 2&#45;Cys Prx to mitochondria and chloroplasts</div>

<div>

	<span  >
	Eunsook Chung</span>
	<sup>1</sup>, 

	<span  >
	Jee-Eun Heo</span>
	<sup>1</sup>, 

	<span  >
	Chang-Woo Cho</span>
	<sup>1</sup>, 

	<span  >
	Hyun-A So</span>
	<sup>1</sup>, 

	<span  >
	Bo-Hyun Yun</span>
	<sup>1</sup>, 

	<span  >
	Jee-Sook Gang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Yao Ran</span>
	<sup>1</sup>, 

	<span  >
	Kyoungmi Kim</span>
	<sup>1</sup>, 

	<span  >
	Jai-Heon Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dong-A University, Busan, Republic of Korea

</div>

<div>We isolated low temperature inducible genes using suppression subtractive hybridization &#40;SSH&#41; method and were able to obtain to clone MLT107 gene encoding peroxiredoxin and aminotransferase. The full&#45;length cDNA of MLT107 is 1,049 bp with an open reading frame &#40;ORF&#41; consisting of 261 amino acid &#40;aa&#41;. Genomic southern blot confirmed that mungbean genome has two copies of MLT107 gene. Northern blot analysis was also carried out for the gene expression during ABA, NaCl, drought, wounding and H2O2 stresses. The expression of MLT107 gene significantly decreased by ABA, NaCl and drought stress, but wounding and H2O2 stress significantly induced MLT107 gene expression. Especially, H2O2 strongly induced the MLT107 gene expression. The expression of MLT107 gene during low temperature stress started to increase in 3 h after treatment, and than slightly decreased and again increased at 24 h. Using GFP fusion vector, smGFP&#45;MLT107 was targeted both to mitochondria and chloroplast. However, it was mostly targeted to mitochondria and partially targeted to chloroplast. For the functional analysis of MLT107, MLT107 recombinant protein was heterologously expressed in E.coli. The MLT107 recombinant cells showed enhanced antioxidant activity compared to that of vector control cells.  &#13;<br/>&#13;<br/> &#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional characterization of soybean SLTI114 gene encoding matrix metalloproteinase protein induced by abiotic stresses</div>

<div>

	<span style="text-decoration: underline" >
	Chang-Woo Cho</span>
	<sup>1</sup>, 

	<span  >
	Eunsook Chung</span>
	<sup>1</sup>, 

	<span  >
	Kyoungmi Kim</span>
	<sup>1</sup>, 

	<span  >
	Hyun-A So</span>
	<sup>1</sup>, 

	<span  >
	Boh-Hyun Yun</span>
	<sup>1</sup>, 

	<span  >
	Jee-Sook Kang</span>
	<sup>1</sup>, 

	<span  >
	Yao Ran</span>
	<sup>1</sup>, 

	<span  >
	Jai-Heon Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dong-A University, Busan, Republic of Korea

</div>

<div>Matrix metalloproteinase proteins &#40;MMPs&#41; are involved in remodeling of plant extracellular matrix in association with plant growth, development, and possibly defense processes. A novel soybean &#40;Glycine max&#41; metalloprotein gene, SLTI114 was identified. The complete cDNA sequence of SLTI114 comprised of 1,443 bp with an open reading frame of 1,179 bp which encodes 43.2 kDa polypeptide consisting of 393 amino acid residues. The nascent SLTI114 polypeptide contained N&#45;terminal signal peptide with a central hydrophobic core between amino acids Asp29 and Ser30, and predicted cleavage site between amino acids Asp153 and Val154. The deduced SLTI114 polypeptide is a pre&#45;pro&#45;enzyme that has all of the hallmark motif characteristic of matix metalloprotainases. To confirm the expression of the SLTI114 gene at the transcriptional level, northern blot analysis was also carried out using the mRNA prepared from developing soybean cotyledons and the soybean leaves exposed to various stresses and hormone. The expression of SLTI114 was induced by LT &#40;5&#8451;&#41;, NaCl, wounding stresses and ABA. SLTI114 protein expressed in E. coli cells showed protease activity in zymography assay. Further in vivo function of SLTI114 was investigated in STI114 overexpressing Arabidopsis plants.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Cloning and Characterization of SLTI98 encoding Ribosomal Protein Genes S6 from Soybean &#40;Glycine max&#41;</div>

<div>

	<span style="text-decoration: underline" >
	Jee-Sook Gang</span>
	<sup>1</sup>, 

	<span  >
	Eunsook Chung</span>
	<sup>1</sup>, 

	<span  >
	Kyoungmi Kim</span>
	<sup>1</sup>, 

	<span  >
	Chang-Woo Cho</span>
	<sup>1</sup>, 

	<span  >
	Jee-Eun Heo</span>
	<sup>1</sup>, 

	<span  >
	Bo Hyun Yun</span>
	<sup>1</sup>, 

	<span  >
	Hyun-A So</span>
	<sup>1</sup>, 

	<span  >
	Yao Ran</span>
	<sup>1</sup>, 

	<span  >
	 Jai-Heon Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dong-A University, Busan, Republic of Korea

</div>

<div>In an attempt to better understand translational control during abiotic stresses, we isolated and characterized a stress inducible gene designated as SLTI98 encoding ribosomal protein S6 in soybean. The derived amino acid sequence of SLTI98 showed the highest identity of 93&#37; with ribosomal protein S6 from Medicago truncatula &#40;ABD32373&#41;. The size of the full&#45;length genomic clone of SLTI98 is 2,701 bp containing 6 exons and 5 introns, of which structure is similar to that of Arabidopsis ribosomal protein S6. Genomic southern blot analysis confirmed that soybean genome has two copies of SLTI98 gene. RNA expression of SLTI98 was mildly induced by salt stress, ABA and wounding stress, but not by dehydration stress. According to localization study using GFP fusion expression system, we were able to confirm that SLTI98&#45;smGFP was restricted mostly to nucleus and partly to cytoplasm. The present study implies that the nuclear SLTI98, ribosomal protein S6 play an important role in translational control during abiotic stresses. </div>

</div>





<div class="abs">

<div style="font-weight:bold">N&#45;terminal 21 amino acid region of soybean &#40;Glycine max&#41; SLTI25 encoding Ribosomal Protein Genes S6 is necessary but not sufficient for chloroplast targeting</div>

<div>

	<span style="text-decoration: underline" >
	Hyun-A So</span>
	<sup>1</sup>, 

	<span  >
	Jee-Sook Gang</span>
	<sup>1</sup>, 

	<span  >
	Eunsook Chung</span>
	<sup>1</sup>, 

	<span  >
	Kyoungmi Kim</span>
	<sup>1</sup>, 

	<span  >
	Chang-Woo Cho</span>
	<sup>1</sup>, 

	<span  >
	Jee-Eun Heo</span>
	<sup>1</sup>, 

	<span  >
	Bo Hyun Yun</span>
	<sup>1</sup>, 

	<span  >
	Yao Ran</span>
	<sup>1</sup>, 

	<span  >
	Jai-Heon Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dong-A University, Busan, Republic of Korea

</div>

<div>The suppression subtractive hybridization &#40;SSH&#41; was used to isolate low temperature stress inducible genes from soybean. Functional Analysis of stress inducible genes SLTI25 showed high homology with genes encoding Ribosomal protein S13. The full&#45;length with intron of SLTI25 is 1,858 bp containing 5 exon and 6 intron. The full&#45;length cDNA of SLTI25 is 742bp contained an open reading frame &#40;ORF&#41; consisted 151 amino acid &#40;aa&#41;. The derived amino acid sequence showed the highest identity of 95&#37; with ribosomal protein S13 from P. ginseng &#40;BAA96366&#41;. SLTI25 gene showed one copy number in the soybean genome using Southern blot analysis. The expression of SLTI25 gene during low temperature stress, salt, wounding, ABA, and drought showed various induction. Wounding is early induction and LT, salt, ABA are late induction. Cellular localization of SLTI25 &#40;full amino acids&#41; is chloroplast, 21 amino acids at N&#45;terminal is cytosol, and the reminder 130 amino acids is cytosol. &#13;<br/>  &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Dissecting P&#45;type H&#43;&#45;mediated proton extrusion in Arabidopsis</div>

<div>

	<span  >
	Wenfeng Li</span>
	<sup>1</sup>, 

	<span  >
	Simonetta Santi</span>
	<sup>2</sup>, 

	<span  >
	Chris Tan</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Wolfgang Schmidt</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>IPMB, Academia Sinica, Taipei, Taiwan, 

	<sup>2</sup>University of Udine, Italy

</div>

<div>As part of a concerted response to iron &#40;Fe&#41; deficiency, comprising developmental, physiological and metabolic reactions, the acquisition of sparingly soluble iron compounds is aided by P&#45;type ATPase&#45;mediated extrusion of protons across the plasma membrane of apical rhizodermic cells. This response is critical for Fe uptake but understudied in Arabidopsis because the response is not very pronounced in the lab strains commonly used. Screening of 96 natural accessions for enhanced proton extrusion revealed several lines that showed substantially higher acidification capacity when compared to Col&#45;0. Four ATPase genes, <span style="font-style: italic">AHA2, AHA3, AHA4</span>, and <span style="font-style: italic">AHA7</span> appeared to be particularly responsive to Fe deficiency. Accessions with higher proton extrusion activity showed a generally higher induction of <span style="font-style: italic">AHA2</span> and <span style="font-style: italic">AHA7</span> in response to Fe deficiency. In addition, other key genes of the Fe deficiency response such as <span style="font-style: italic">IRT1</span> and <span style="font-style: italic">FRO2</span> showed higher message levels in strong acidifying lines. Time&#45;course experiments revealed that full induction levels were reached earlier in active lines. Light affected the expression of all Fe&#45;responsive genes. The effects of light and Fe deficiency on <span style="font-style: italic">IRT1</span> and <span style="font-style: italic">AHA</span> were additive and revealed strict co&#45;regulation of <span style="font-style: italic">AHA7</span>, but not of <span style="font-style: italic">AHA2</span>, with <span style="font-style: italic">IRT1</span>. Transcripts of all Fe&#45;regulated genes were lower at high external pH. Regulation of iron&#45;responsive genes was impaired in <span style="font-style: italic">aha7</span> mutants, indicating a regulatory function of AHA7 in iron homeostasis.&#13;<br/>In order to generate RILs with high proton extrusion activity in response to Fe deficiency, we have developed a high&#45;throughput system to screen for highly active individuals. Crosses from parents that have been selected from the initial screening showed a more pronounced response when compared to Col&#45;0. Backcrosses with high proton extrusion capacity exhibited an increased frequency of root hairs, indicating a close linkage of root hair formation and acidification capacity. The role of ATPase&#45;mediated proton extrusion in the regulation of physiological and developmental responses to Fe deficiency is discussed.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Elicitor and UV light related transcription factor &#40;OgElr&#41; derived from Wild Rice &#40;Oryza grandiglumis&#41; gave resistance against UV&#45;B radiation</div>

<div>

	<span style="text-decoration: underline" >
	Eun-Hee Jeon</span>
	<sup>1</sup>, 

	<span  >
	Jung-Hun Pak</span>
	<sup>1</sup>, 

	<span  >
	Mi-Jin Kim</span>
	<sup>1</sup>, 

	<span  >
	Hye-Jeong Kim</span>
	<sup>1</sup>, 

	<span  >
	Ki-Jung Lee</span>
	<sup>1</sup>, 

	<span  >
	Young-Soo Chung</span>
	<sup>1</sup>, 

	<span  >
	Kyung-Ho Kang</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Dept. of Genetic Engineering, Dong-A University, Busan, Korea, 

	<sup>2</sup>National Crop Experiment Station, Suwon, Korea

</div>

<div>One of the wild rice species Oryza grandiglumis is tetraploid&#40;2n&#61;48&#41; and has CCDD genome. It has been known to own fungal resistance against sheath blight, rice blast, bacterial leaf blight and insect resistance against brown plant hopper&#40;Nilaparvata lugens&#41;.&#13;<br/>The multiple techniques were used to identify the genes differentially expressed at wounding and fungal elicitor treated wild rice &#40;Oryza grandiglumis&#41;. This technique is a combination of suppression subtractive hybridization&#40;SSH&#41; and GenomeWalker PCR. Here we report the gene cloning and expression characterization of OgElr gene. Whole sequences of OgElr gene from Oryza grandiglumis &#40;CCDD, 2n&#61;48&#41; were obtained by GenomeWalker method. OgElr contains 447 nucleotides and 148 amino acids. The cDNA has 96&#37; sequence homology to Oryza sativa elicitor and UV light related transcription factor &#40;Elr&#41;. The OgElr gene induced by wounding, fungal elicitor, jasmonic acid &#40;JA&#41; and salicylic acid &#40;SA&#41;, protein phosphatase inhibitors cantharidin &#40;CN&#41; and endotall&#40;EN&#41; as well as UV&#45;B. To identify in vivo function of OgElr gene, the gene was transformed into Arabidopsis thaliana. The OgElr transgenic plants were exposed to UV&#45;B for 4 hours and UV&#45;C for 2 hours and a root&#45;bending assay was carried out. The survival rate of OgElr transgenic plants under UV exposure greatly increased. And also root growth was less inhibited by UV&#45;B exposure in OgElr transgenic plants compared to the wild type. Our results suggest that OgElr gene may involve in tolerant response of plant against UV irradiation.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">rcd3</span> &#8211; a novel guard cell membrane transport mutant</div>

<div>

	<span  >
	Triin Kollist</span>
	<sup>1</sup>, 

	<span  >
	Hannes Kollist</span>
	<sup>1,2</sup>, 

	<span  >
	Heino Moldau</span>
	<sup>2</sup>, 

	<span  >
	Yong-Fei Wang</span>
	<sup>3</sup>, 

	<span  >
	Radhika Desikan</span>
	<sup>4</sup>, 

	<span  >
	Julian Schroeder</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Jaakko Kangasj&#228;rvi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Helsinki, Helsinki, Finland, 

	<sup>2</sup>University of Tartu, Tartu, Estonia, 

	<sup>3</sup>University of California, San Diego, (CA), USA, 

	<sup>4</sup>University of the West of England, Bristol, UK

</div>

<div>Ozone &#40;O3&#41; is a predominant air pollutant, which has also been used as a tool to induce the formation of reactive oxygen species &#40;ROS&#41; in the apoplast, and this way identify components and processes regulated by apoplastic ROS. We have previously isolated a series of O3&#45;sensitive <span style="font-style: italic">rcd</span> &#40;for radical induced cell death&#41; mutants. Here we describe an O3&#45;sensitive mutant <span style="font-style: italic">rcd3</span>, the phenotype of which is caused by a change in stomatal function. <span style="font-style: italic">rcd3</span> has constitutively higher stomatal conductance than the Col&#45;0 wild type and O3&#45;induced stomatal closure is largely suppressed in <span style="font-style: italic">rcd3</span>. The number of stomata in the abaxial side of the leaf is not altered. Detailed analysis of stomatal responses immediately after the onset of O3&#45;treatment revealed that O3 induces a rapid stomatal closure and subsequent reopening within 30 minutes in several mutants with altered ozone sensitivity or stomatal function, and in several ecotypes including Col&#45;0 and Ler. This fast stomatal response, which is induced by even as short as 60&#45;180 sec O3&#45;pulse, is most likely an indication of the activity of ROS in the guard cell apoplast. During the reopening the guard cells loose responsiveness to the ROS&#45;pulse and gain it again after 90 minutes. In the abscisic acid &#40;ABA&#41; insensitive mutants <span style="font-style: italic">abi1, abi2</span>, and <span style="font-style: italic">ost1</span>, and in <span style="font-style: italic">rcd3</span>, the transient closure was absent. The phenotype of <span style="font-style: italic">rcd3</span> is not caused by altered ABA content since a six hour treatment with O3 caused similar ABA induction in both Col&#45;0 and <span style="font-style: italic">rcd3</span>. However, guard cells of <span style="font-style: italic">rcd3</span> are insensitive to ABA, nitric oxide, and hydrogen peroxide, and their responses to light, humidity and CO2 are altered. Map&#45;based cloning of <span style="font-style: italic">rcd3</span> revealed a plasma membrane transport protein which is a novel component in membrane transport processes involved in stomatal closure.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Plant tolerance to drought and salinity: modulation of transcription factors</div>

<div>

	<span style="text-decoration: underline" >
	Chiara Tonelli</span>
	<sup>1</sup>, 

	<span  >
	Eleonora Cominelli</span>
	<sup>1</sup>, 

	<span  >
	Domenico Allegra</span>
	<sup>1</sup>, 

	<span  >
	Massimo Galbiati</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dip Scienze Biomolecolari e Biotecnologie, University of Milan Italy

</div>

<div>Plant growth and productivity are greatly affected by abiotic stresses such as drought, high salinity, and low temperature. In the signal transduction network from perception of stress signals to stress&#45;responsive gene expression, a crucial role is played by transcription factors. Here we report the characterization of two R2R3MYB genes of Arabidopsis thaliana. AtMYB60 is specifically expressed in stomata guard cells and its expression is negatively modulated during drought. A null mutation in AtMYB60 results in  the constitutive reduction of stomatal opening and in decreased wilting under water stress conditions. Transcript levels of a limited number of genes are altered in the mutant and some of them are known to be involved in plant response to stress. AtMYB60 promoter sequence have been dissected and the minimal region for guard cell expression identified.&#13;<br/>AtMYB90 is up&#45;regulated in response to drought, salt and ABA treatments. Transgenic plants overexpressing AtMYB90 showed enhanced salt tolerance compared to wild&#45;type while plants, in which AtMYB90 was silenced through antisense approach, resulted more sensitive to salt treatment. &#13;<br/>Our data indicate that modulation of AtMYB60 and AtMYB90 opens new possibilities to engineer plant responses to improve survival during drought and salt stress. &#13;<br/>&#13;<br/>References&#13;<br/>E Cominelli, M Galbiati, A Vavasseur, L Conti, T Sala, M Vuylsteke, N Leonhardt, SL Dellaporta, C Tonelli.. A guard cell&#45;specific MYB transcription factor regulates stomatal movements and plant drought tolerance. Current Biology, 15,1196&#45;1200, 2005. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic architecture of a Genotype&#42;Environment interaction using <span style="font-style: italic">Arabidopsis thaliana</span> natural variation</div>

<div>

	<span style="text-decoration: underline" >
	Anna Ihnatowicz</span>
	<sup>1</sup>, 

	<span  >
	Sigi Effgen</span>
	<sup>1</sup>, 

	<span  >
	Maarten Koornneef</span>
	<sup>1</sup>, 

	<span  >
	Matthieu Reymond</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, Cologne, Germany

</div>

<div><span style="font-style: italic">Arabidopsis thaliana</span> natural accessions have been collected in a wide range of habitats. It is assumed that selection for adaptation to the local environments has occurred, providing genetic variation in responses to environmental factors. Consequently, analysis of natural variation is a powerful tool to identify which allelic variants are adaptive under specific environmental conditions. A set of Arabidopsis accessions originating from different locations, grown together at several light and temperature conditions, has shown contrasting responses of traits such as growth, flowering time, pigmentation and photosynthetic performance. Phenotypic variation for some of these traits was characterized using spectroscopic analysis and by measuring chlorophyll content.<br/>Interestingly, under particular environmental conditions &#40;low temperatures&#41; one high&#45;altitude accession Hodja&#45;Obi&#45;Garm &#40;Hog&#45;collected around 1800m&#41; showed a chlorotic phenotype linked with a large reduction in photosynthetic performance. When Hog plants were grown at 4&#176;C, the fraction of QA, the primary electron acceptor of PSII, present in the reduced state was significantly increased, whereas Fv&#47;Fm and photosynthetic yield were reduced. These results indicate photoinhibitory effects.<br/>An advanced&#45;backcrossed population has been developed using the Hog accession and Ler as a non&#45;chlorotic recurrent parental line. The aim of this study is to introduce only the gene&#40;s&#41; from Hog responsible for the chlorotic phenotype into the genetic background of Ler, creating near&#45;isogenic lines. The obtained near&#45;isogenic lines &#40;BC2F2&#41; are being further genotyped by using tilling array and illumina techniques. To further explore the functional relationship between genotype and phenotype, differences in gene expression between parental and near&#45;isogenic lines grown at different temperatures are being examined. The mapping strategy combined with gene chip experiments and further physiological characterization of the observed phenotype will enable us to select candidate genes. The molecular basis of the underlying G&#42;E interaction will be further studied.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Regulation of boron transport in Arabidopsis thaliana</div>

<div>

	<span  >
	Junpei Takano</span>
	<sup>1</sup>, 

	<span  >
	Kyoko Miwa</span>
	<sup>1</sup>, 

	<span  >
	Mayuki Tanaka</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Toru Fujiwara</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>University of Tokyo, 

	<sup>2</sup>University of Tokyo, SORST, JST

</div>

<div>Boron is an essential element in plants.  We have been working on identification and characterization of boron transporters&#47;channels.  A member of MIP family, NIP5;1, is involved in boron uptake in roots and BOR1, an efflux transporter of boron, is required for efficient boron transport from roots to shoots.  Genes homologous to these transporters are differentially expressed and their roles in B transport have been elucidated.  Moreover, accumulation of these transporters are regulated in response to B conditions and some members show polar localization patterns.  Our studies suggest that these transporters function coordinately to transport boron from soil solutions to sites in plants&#47;cells where boron is necessary.  We have also been successful in generating transgenic Arabidopsis that are tolerant to boron stress.  We will discuss an updated model of overall mechanisms of boron transport and their regulation in Arabidopsis.&#13;<br/>&#13;<br/>Takano et al., Nature 2002&#13;<br/>Takano et al., PNAS 2005&#13;<br/>Takano et al., Plant Cell 2006&#13;<br/>Miwa and Takano et al., Plant Journal 2006&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Chloroplast Signal Modifies the Light Signal Transduction</div>

<div>

	<span style="text-decoration: underline" >
	Aurora Pi&#241;as Fern&#225;ndez</span>
	<sup>1</sup>, 

	<span  >
	&#197;sa Strand</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Ume&#229; University, Ume&#229;, Sweden

</div>

<div>Phytochrome A &#40;PHYA&#41; mediates the far&#45;red block&#45;of&#45;greening response and mutants in the phyA gene can survive a far&#45;red block&#45;of&#45;greening treatment. We have shown that the gun5 mutant can also survive this treatment indicating that chloroplast retrograde signal mediated by intermediates of chlorophyll synthesis is involved in the far&#45;red block&#45;of&#45;greening response. Thus, we used two mutants impaired in the chlorophyll biosynthesis pathway, gun5 and crd, in order to identify intermediates implicated in this response. The gun5 mutant has a mutation in the H&#45;subunit of Mg&#45;chelatase and it has a reduced amount of Mg&#45;protoporphyrin IX &#40;Mg&#45;ProtoIX&#41;. The crd mutant has reduced cyclase activity and as a result it accumulates large amounts of Mg&#45;ProtoIX&#45;monomethyl ester &#40;Mg&#45;ProtoIXMe&#41; under stress conditions. In contrast to the gun5 mutant, the crd mutant demonstrated the same phenotype as wild type seedlings. Biochemical analysis of Arabidopsis and barley demonstrated that the chlorophyll biosynthesis intermediate, Mg&#45;ProtoIXMe, accumulates in response to a far&#45;red block&#45;of&#45;greening treatment. The gun5 and phyA mutants both accumulated lower amounts of Mg&#45;ProtoIXMe compared to the crd mutant or wild type seedlings. Our observation is in contrast to the hypothesis that the far&#45;red block&#45;of&#45;greening response is an effect of photo&#45;oxidative damage in cells caused by an excess of protochlorophyllide accumulation driven by PHYA. At the expression level, we found that LHCB2 and RBCS transcript levels were negatively correlated with the kinetics of Mg&#45;ProtoIXMe accumulation in wild&#45;type seedlings. Transcript levels of LHCB2 and RBCS were maintained in the gun5 and phyA mutants. Our results suggest that PHYA regulates chlorophyll biosyntheses by controlling cyclase activity and that whereas PHYA initiates the far&#45;red block&#45;of&#45;greening response, the component necessary to fulfil this response is Mg&#45;ProtoIXMe. In summary, we describe an interaction between light signalling components and plastid signals that are necessary to fine&#45;tune the response of seedlings to environmental cues.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Proteomic analysis of the response of Arabidopsis proteins to polycyclic aromatic hydrocarbons stress</div>

<div>

	<span style="text-decoration: underline" >
	Hong Liu</span>
	<sup>1</sup>, 

	<span  >
	Yuejun Yang</span>
	<sup>2</sup>, 

	<span  >
	Bo Cui</span>
	<sup>1</sup>, 

	<span  >
	Yuanbei Ye</span>
	<sup>1</sup>, 

	<span  >
	Xuanming Liu</span>
	<sup>2</sup>, 

	<span  >
	Chentao Lin</span>
	<sup>2</sup>, 

	<span  >
	Yanhe Huang</span>
	<sup>1</sup>, 

	<span  >
	Zonghua Wang*</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Fujian Agriculture and Forestry University, Fuzhou,P.R.China, 

	<sup>2</sup>Hunan University, Changsha, P.R.China

</div>

<div>Polycyclic aromatic hydrocarbons &#40;PAHs&#41; are of serious environmental issues because they cause many health problems in humans including cancer and mutation . Plants are important in removing PAHs from the atmosphere; yet, information on biochemistry and molecular biology of PAH stress responses in plants is lacking. In order to study the difference at the protein expression level, a proteome approach based on two&#45;dimensional gel electrophoresis &#40;2&#45;DE&#41; was used to compare the protein patterns of the phenanthrene&#45;treated plants and control. Thirty of these 53 of protein spots which were differentially expressed in phenanthrene&#45;treated plants were identified by Matrix Assisted Laser Desorption&#47;Ionization&#45;time of Flight Mass Spectrometry&#45;Mass Spectrometry &#40;MALDI&#45;TOF&#45;TOF&#45;MS&#41; and database analysis. Among the differentially expressed 30 proteins, 25 were up&#45;regulated and 5 were down&#45;regulated respectively when treated with 1 mM phenanthrene. According to the function, these proteins can be classified into seven categories: metabolism &#47;cellular component&#40;26.6&#37;&#41;, energy&#40;16.6&#37;&#41;, cellular communication &#47; signal transduction&#40;3.3&#37;&#41;, protein fate&#40;3.3&#37;&#41;, defense stress and detoxification&#40;23.3&#37;&#41;, miscellaneous function&#40;3.3&#37;&#41;, and unclassified protein&#40;23.3&#37;&#41;. It indicated that photosynthetic processes are strongly inhibited and oxidative stress signaling is induced.&#13;<br/>&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Proteome analysis of the cold stress response in Arabidopsis thaliana</div>

<div>

	<span  >
	Andrea Matros</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Hans-Peter Mock</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Genetics and Crop Plant Research

</div>

<div>Environmental influences such as high light, drought, high or low temperature and salinity affect growth and yield of crop plants by leading to altered gene and protein expression, metabolic changes, and growth retardation &#91;1&#93;. How such environmental stimuli are perceived and trigger the complex defensive and adaptive signalling networks, and how these events result in resistance&#47;tolerance is of major practical interest. A complementary proteome study was started to analyse the response of Arabidopsis plants exposed to cold stress &#40;6&#176;C&#41;. We first carried out a comparison of protein patterns by using DIGE technology &#91;2&#93;. Proteins displaying significant changes in abundance included RNA&#45;binding protein CP29, a glycin&#45;rich protein, carbohydrate metabolism&#45;associated proteins, dehydrins and low&#45;temperature&#45;induced protein 78. Taking advantage of recent developments in proteome analysis samples from these studies were also analysed by a label free quantification method using LC&#45;based separation combined with ESI&#45;Q&#45;TOF MS. This novel approach confirmed earlier results obtained by using DIGE technology but also lead to the identification of a number of additional proteins with changed expression pattern after cold stress exposure. In addition we have started to investigate fractions of phosphorylated proteins from these experiments in order to evaluate regulatory circuits of cold stress responses in Arabidopsis. Many of the proteins identified have been described previously in the context of cold stress responses &#91;3, 4&#93;, indicating the validity of our complementary approach for further in depth studies. &#13;<br/>&#13;<br/>&#13;<br/>&#91;1&#93; Seki, M., Narusaka, M., Ishida, J., Plant Journal, 2002, 31, 279&#45;292. &#91;2&#93; Amme, S., Matros, A., Schlesier, B., Mock, H.&#45;P., J. Exp. Bot., 2006, 57 &#40;7&#41;, 1537&#45;1546.&#91;3&#93; Kreps, J. A., Wu, Y., Chang, H. S., Zhu, T., Wang, X., Harper, J. F., Plant Pysiol., 2002, 130, 2129&#45;2141&#13;<br/>&#91;4&#93; Kawamura, Y., Uemura, M., Plant Journal, 2003, 36, 141&#45;154&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Two seven&#45;transmembrane domain MLO proteins function in modulating asymmetrical root growth on horizontal surfaces</div>

<div>

	<span  >
	Zhongying Chen</span>
	<sup>1</sup>, 

	<span  >
	Andreas Hartmann</span>
	<sup>2</sup>, 

	<span  >
	Sandra  Noir</span>
	<sup>2</sup>, 

	<span  >
	Ralph Panstruga</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Alan Jones</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The University of North Carolina at Chapel Hill, Chapel Hill, USA, 

	<sup>2</sup>Max-Planck Institute for Plant Breeding Research, Cologne, Germany

</div>

<div>Mildew resistance locus O &#40;MLO&#41; proteins comprise the largest plant&#45;specific seven&#45;transmembrane domain protein family and may operate together with the heterotrimeric G&#45;protein complex in plant signaling. Among the 15 Arabidopsis <span style="font-style: italic">MLO</span> genes, <span style="font-style: italic">MLO4</span>, <span style="font-style: italic">MLO11</span> and <span style="font-style: italic">MLO14</span>, which are closely&#45;related, have overlapping expression in root tips. Relative to wild type plants, loss&#45;of&#45;function alleles of <span style="font-style: italic">mlo4</span> or <span style="font-style: italic">mlo11</span>, but not of <span style="font-style: italic">mlo14</span>, exhibited a dramatically enhanced asymmetrical root growth, manifested as a tight circling pattern on horizontal surfaces. Quantitative end&#45;point analyses of this type of cumulative asymmetrical growth revealed different sample distribution pattern of <span style="font-style: italic">mlo11mlo14</span> double mutants compared with <span style="font-style: italic">mlo11</span> single mutant. Therefore, these three MLO proteins are involved in regulating root curling. The auxin transporter inhibitor NPA abolishes the exaggerated root curling phenotypes of <span style="font-style: italic">mlo4</span> and <span style="font-style: italic">mlo11</span> mutants, suggesting defects of these mutants in modulating differential auxin distribution or downstream auxin signaling when grow on horizontal surfaces. Root curling phenotypes of <span style="font-style: italic">mlo4</span> and <span style="font-style: italic">mlo11</span> mutants were less pronounced under darkness, suggesting that these MLO proteins are mainly involved in light&#45;regulation of this asymmetrical root growth. Null mutations in the G&#45;alpha &#40;<span style="font-style: italic">gpa1</span>&#41; and G&#45;beta &#40;<span style="font-style: italic">agb1</span>&#41; subunits of the Arabidopsis G&#45;protein complex also exhibited light&#45;dependent alteration in root curling. However, <span style="font-style: italic">agb1mlo4</span>, <span style="font-style: italic">gpa1mlo4</span>, <span style="font-style: italic">agb1mlo11</span> and <span style="font-style: italic">gpa1mlo11</span> double mutants exhibited combined root curling phenotypes, suggesting that MLO4 and MLO11 likely function independently or downstream of heterotrimeric G&#45;proteins in modulating light&#45;regulated asymmetrical root growth.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Mutation of an enzyme involved in the <span style="font-style: italic">de novo</span> synthesis of NAD causes early leaf senescence in <span style="font-style: italic">Arabidopsis thaliana</span></div>

<div>

	<span  >
	Jos Schippers</span>
	<sup>1</sup>, 

	<span  >
	Adriano Nunes-Nesi</span>
	<sup>2</sup>, 

	<span  >
	Roxana Apetrei</span>
	<sup>1</sup>, 

	<span  >
	Alisdair Fernie</span>
	<sup>2</sup>, 

	<span  >
	Jacques Hille</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Paul Dijkwel</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Groningen, Groningen, Netherlands, 

	<sup>2</sup>Max Planck Institute for Molecular Plant Physiology, Golm, Germany

</div>

<div>Leaf senescence is a process of enigmatic beauty that plays its role during the season of change. The rational behind the process is a genetic program aimed at the efficient reuse of valuable nutrients and minerals. In Arabidopsis the senescence program is executed in an age&#45;dependent way. By studying mutants with an altered onset of leaf senescence, we aim to reveal the mechanisms that underlie plant ageing.  In the presented study we characterized the <span style="font-style: italic">old5</span> &#40;<span style="font-style: italic">onset of leaf death5</span>&#41; mutant. <span style="font-style: italic">OLD5</span> encodes a quinolinate synthase that is involved in the de novo synthesis of Nicotinamide adenine dinucleotide &#40;NAD&#41;. Studies on NAD mainly relate to its role in redox biochemistry and energetic metabolism. Recently, the molecule gained a renewed interest with the discovery that nicotinamide cofactors play important roles in cell signaling in animals, yeast and plants. For instance, NAD is used as a substrate for post&#45;translational modification of proteins. Interestingly, the yeast Silent Information Regulater 2 &#40;Sir2&#41; protein has been shown to be an NAD&#45;dependent deacetylase involved in the control of lifespan. Moreover NAD metabolism has been proposed as the mechanism behind calorie restriction. <br/>The <span style="font-style: italic">old5</span> mutant provides an excellent opportunity to answer the question whether NAD metabolism also plays a role in plant ageing. A comprehensive study including results of a physiological and genetical characterization of the mutant are presented as well as results of protein interaction analysis and a broad metabolic profiling of the mutant.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A functional genomics approach reveals TCP21 as a new component of the Arabidopsis clock transcriptional network</div>

<div>

	<span style="text-decoration: underline" >
	Jose Pruneda-Paz</span>
	<sup>1</sup>, 

	<span  >
	Ghislain Breton</span>
	<sup>1</sup>, 

	<span  >
	Steve Kay</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Scripps Research Institute, La Jolla, (CA), USA

</div>

<div>Two MYB transcription factors &#40;CCA1&#47;LHY&#41; are considered key components in the current model of Arabidopsis circadian network. Although it has been well documented that CCA1&#47;LHY regulate the expression of several core clock genes, the molecular mechanisms governing CCA1&#47;LHY transcription remains poorly understood. In an attempt to identify new members of the Arabidopsis clock transcriptional network, we have generated a library of circadian&#45;regulated transcription factors. The library was screened against different core clock gene promoters in a yeast one&#45;hybrid system. Here we present the identification of TCP21, a class I TCP transcription factor, as a new regulator of CCA1 expression. TCP21 binds to a TCP class I binding site present in CCA1 promoter and represses the expression of a CCA1promoter::LUC&#43; reporter in transient assays. TCP21mRNA expression peaks four hours before the trough in CCA1 expression, which is consistent with TCP21 functioning as a repressor of CCA1 promoter activity. In addition, low amplitude rhythmic CCA1 mRNA is observed in TCP21 over&#45;expressing transgenic lines. Although CCA1 and LHY are co&#45;expressed, we were not able to identify any TCP21 binding activity to the LHY promoter. This finding suggests that CCA1 and LHY might not be regulated by the same mechanism. A similarity of clock architecture among different organisms is that their main components are part of feedback loops. Interestingly, a CCA1 binding site &#40;CBS&#41; is present in TCP21 promoter region. Specific binding of CCA1 to this element was observed in gel shift assays and elevated TCP21 mRNA levels were found &#8220;in planta&#8221; in a CCA1 over&#45;expression background. In conclusion, using a novel alternative approach we have identified TCP21 as a transcription factor directly regulating CCA1 promoter activity. We propose a transcriptional feedback loop, were CCA1 is inducing the expression of TCP21 that constitutes the negative arm of the loop.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The role of cell identity in plant salt stress response</div>

<div>

	<span style="text-decoration: underline" >
	Jose Dinneny</span>
	<sup>1</sup>, 

	<span  >
	Jean Wang</span>
	<sup>1</sup>, 

	<span  >
	Dan Mace</span>
	<sup>1</sup>, 

	<span  >
	Solomon Pointer</span>
	<sup>1</sup>, 

	<span  >
	Christa Barron</span>
	<sup>2</sup>, 

	<span  >
	John Schiefelbein</span>
	<sup>2</sup>, 

	<span  >
	Philip Benfey</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Duke University, Durham, NC, USA, 

	<sup>2</sup>University of Michigan, Ann Arbor, MI, USA

</div>

<div>The survival of an organism is dependent on its ability to respond appropriately to environmental changes.  In organs, where different cell&#45; and tissue&#45;types perform unique biological functions, cell&#45;identity is likely to determine aspects of differential stimulus perception and response.  Most studies however, have analyzed the transcriptional impact of environmental stimuli on single&#45;cells, whole organs, or organ systems.  Here we show that, in Arabidopsis roots, the majority of the response to a well characterized environmental stress &#40;high salinity&#41; is cell&#45;type specific resulting in the differential regulation of particular biological functions in subsets of tissue layers.  Our analysis is based on three microarray data sets we have generated that explore transcriptional changes spatially among 6 tissue layers and 4 longitudinal regions or temporally along 5 time points after salt treatment.  Analysis of cis&#45;element enrichment in the promoters of salt responsive genes shows that known stress regulatory elements such as the ABRE and DRE likely regulate semi&#45;ubiquitous responses to salt occurring in multiple cell types whereas the W&#45;box may control stele specific responses.  We functionally test the role of cell&#45;identity in regulating salt responses in the epidermis and reveal cell&#45;autonomous and non&#45;autonomous effects when cell identity is altered.  Together, these data reveal previously unknown spatial and temporal aspects of a stress response and demonstrate the important role cell&#45;identity plays in guiding changes in transcriptional states.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Microtubule dynamics in Zea mays roots in response to drought stress</div>

<div>

	<span style="text-decoration: underline" >
	Jiansheng Liang</span>
	<sup>1</sup>, 

	<span  >
	Bing L&#252;</span>
	<sup>1</sup>, 

	<span  >
	Feng Chen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>College of Bioscience and Biotechnology, Key Laboratory of Crop Genetics and Physiology of Jiangsu Province, Yangzhou University, Yangzhou, People?s Republic of China, 225009

</div>

<div>Microtubules play important roles in plant physiological responses to environmental stress, including drought stress, etc. When plants are exposed to drought stress, dynamic changes of microtubules occur. Plenty of evidence has showed that plants have developed complicated mechanisms to adapt to these environmental stress conditions. Abscisic acid &#40;ABA&#41; accumulates significantly in plants in response to drought conditions, which has been considered a major mechanism for plants to enhance drought tolerance. However, little has been known whether the dynamic changes of microtubule are correlated to ABA accumulation under a drought condition. In the present study, we have focused on the role of microtubules in the induction of abscisic acid biosynthesis as a response of Zea mays to drought stress. The functions of microtubules in response to drought stress were investigated by immuno&#45;fluorescence localization, enzyme&#45;linked immuno&#45;sorbent assay and reverse transcription&#45;polymerase chain reaction. The microtubule&#45;destablizing &#40;oryzalin&#41; and &#8211;stablizing reagents &#40;taxol&#41; were used in combination with osmotic stress treatment. Results indicated that microtubule dynamics correlated with the ABA accumulation under osmotic stress. Microtubule&#45;destablizing and &#8211;stabilizing reagent treatments also led to significant changes of ABA accumulation. Our results suggest that the dynamic arrangement of microtubules is involved in the accumulation of ABA under osmotic stress. A possible mechanism of plant response to drought stress through microtubules is proposed that provides an explanation for the increased abscisic acid biosynthesis under drought stress.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Intracellular Localization of Integrin&#45;like Protein and Its Roles in Osmotic Stress&#45;induced ABA Biosynthesis in Zea Mays</div>

<div>

	<span style="text-decoration: underline" >
	Bing L&#252;</span>
	<sup>1</sup>, 

	<span  >
	Juan Wang</span>
	<sup>1</sup>, 

	<span  >
	Jiansheng Liang</span>
	<sup>1</sup>, 

	<span  >
	Zhonghua Gong</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>College of Bioscience and Biotechnology, Key Laboratory of Crop Genetics and Physiology of Jiangsu Province, Yangzhou University, Yangzhou, People?s Republic of China, 225009

</div>

<div>Plants have evolved many mechanisms to cope with adverse environmental stresses. Abscisic acid &#40;ABA&#41; accumulates significantly in plant cells in response to drought conditions, which has been considered as a major mechanism for plants to enhance drought tolerance. In this study, we explore the possible mechanisms how plant cells perceive the osmotic stress, and as a consequence, induce cells to biosynthesize ABA. Western Blotting and immuno&#45;fluorescence localization experiment, using an anti&#45;human Integrin &#946;1 polyantibody, showed that there existed a protein in roots of Zea mays, which was similar to integrin protein of animals and it mainly localized in insoluble fraction of plant cells. Treatment with GRGDS, a synthetic pentapeptide containing RGD domain, which interacted specifically with integrin protein and thus blocked the cell wall&#45;plasma membrane interaction, significantly inhibited osmotic stress induced ABA biosynthesis in cells, but not in protoplasts, and the RGD analog, which did not contain the RGD domain, had no effects on them. Our results showed that a strong interaction existed between cell wall and plasma membrane and this interaction was largely mediated by integrin&#45;like proteins. A great difference in responses to osmotic stress in induction of ABA biosynthesis was observed between protoplasts and cells, implying that cell wall and&#47;or cell wall&#45;plasma membrane interaction play important roles in perceiving osmotic stress. Accordingly, we concluded that cell wall and&#47;or cell wall&#45;plasma membrane interaction mediated by integrin&#45;like protein played important roles in osmotic stress&#45; induced ABA biosynthesis in Zea mays.</div>

</div>





<div class="abs">

<div style="font-weight:bold">YUCCA genes are required for shade&#45;induced increases in auxin signaling</div>

<div>

	<span  >
	Melissa Pytlak</span>
	<sup>1</sup>, 

	<span  >
	Kazunari Nozue</span>
	<sup>1</sup>, 

	<span  >
	Youfa Cheng</span>
	<sup>2</sup>, 

	<span  >
	Michael Covington</span>
	<sup>1</sup>, 

	<span  >
	Stacey Harmer</span>
	<sup>1</sup>, 

	<span  >
	Yunde Zhao</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Julin Maloof</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Section of Plant Biology, University of California, Davis, CA, USA, 

	<sup>2</sup>Division of Biological Sciences, University of California, San Diego, CA, USA

</div>

<div>Because plants depend on light for energy they have developed the ability to detect adjacent plants and alter their growth to compete for light. Neighbor detection can lead to the shade&#45;avoidance syndrome, a suite of traits that includes increased stem and petiole elongation, early flowering, and changes in resource allocation. Work by others has established that auxin is involved in shade&#45;induced increases in cell elongation, but how shade alters auxin signaling is unclear. One model suggests that shade increases lateral transport of auxin; other data suggests that shade increases auxin biosynthesis. To examine shade&#47;auxin interactions in real&#45;time, we have used the synthetic auxin responsive promoter DR5 to drive expression of firefly luciferase &#40;LUC&#41;. We observed a strong increase in DR5:LUC bioluminescence after treating plants with end&#45;of&#45;day far red &#40;EOD&#45;FR&#41; to induce the shade&#45;avoidance syndrome. Auxin transport inhibitors did not prevent the shade&#45;induced increase in DR5 expression, consistent with shade increasing auxin biosynthesis rather than changing transport. To test this idea further we examined extant microarray data and found three putative auxin biosynthesis genes, YUCCA5, 8, and 9, that are all induced by shade treatment. Analysis of a quintuple mutant that removes function of <span style="font-style: italic">YUCCA5, 8,</span> and <span style="font-style: italic">9</span>, along with two other <span style="font-style: italic">YUCCA</span> genes in the same clade, shows that these genes are required for shade&#45;induced increases in auxin signaling. In addition, we find that although inhibition of auxin transport does not prevent increased DR5:LUC bioluminescence, it does prevent increases in cell elongation. Together our data suggests that shade causes an increase in auxin biosynthesis by increasing <span style="font-style: italic">YUCCA</span> expression, and that auxin transport is required to bring the newly synthesized auxin to target tissues.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular evolution of clock genes in Arabidopsis</div>

<div>

	<span  >
	Aziz Jamai</span>
	<sup>1</sup>, 

	<span  >
	Katherine R Amato</span>
	<sup>1</sup>, 

	<span  >
	Kathryn E. Hacker</span>
	<sup>1</sup>, 

	<span  >
	Mark Borsuk</span>
	<sup>1</sup>, 

	<span  >
	Mark A. McPeek</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	C. Robertson McClung</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dartmouth College, Hanover, (NH), USA

</div>

<div>The circadian clock is an endogenous timing mechanism with a period of &#126;24 hr.  It serves to synchronize an organism with the externally imposed cycle associated with the daily rotation of the earth on its axis.   One of the central premises of the study of circadian rhythms is that a functional circadian clock enhances organismal fitness and recent studies &#40;e.g., Dodd et al. 2005 <span style="font-style: italic">Science</span> 309:630&#41; have provided experimental proof that this is so.  It is abundantly clear that there is considerable natural variation in clock function among Arabidopsis accessions &#40;e.g., Michael et al. 2003 <span style="font-style: italic">Science</span> 302:1049&#41;.  Presumably some of this natural variation arises from mutational differences in clock genes among accessions and, given that clocks enhance fitness, one might hypothesize that these sequence variations might be subject to selection.  We have tested this hypothesis through sequence analysis of <span style="font-style: italic">PSEUDO&#45;RESPONSE REGULATOR 7</span> &#40;<span style="font-style: italic">PRR7</span>; At5g02810&#41; from &#62;100 accessions.  There is a significant excess of replacement mutations relative to synonymous mutations &#40;20 and 5, respectively; p&#60;0.005&#41;, consistent with diversifying selection at the <span style="font-style: italic">PRR7</span> locus.  This cannot be attributed to selection occurring at adjacent loci, as one flanking gene &#40;<span style="font-style: italic">DNA TOPOISOMERASE</span>; At5g02820&#41; shows excess synonymous mutations &#40;6 replacement and 31 synonymous; p&#60;0.001&#41; and the other flanking gene &#40;<span style="font-style: italic">PROTEIN KINASE</span>; At5g02800&#41; shows roughly equal numbers of replacement and synonymous mutations &#40;10 and 14, respectively; p&#62;0.5&#41;, consistent with purifying and neutral selection, respectively.  This suggests selection for multiple allelic variants at the <span style="font-style: italic">PRR7</span> locus, which is consistent with distinct 'PRR7<span style="font-style: italic"> alleles contributing differentially to organismal fitness in different environments.  Indeed, two </span>PRR7<span style="font-style: italic"> clades show distinct spatial distributions within Europe.  We are currently testing whether these different alleles are functionally equivalent through complementation of mutants defective in </span>PRR7<span style="font-style: italic"> activity.  This work is supported by a grant from the National Science Foundation &#40;MCB&#45;0343887&#41; to CRM.</div>

</div>





<div class="abs">

<div style="font-weight:bold">SDIR1 Is A Novel RING Finger E3 Ligase That Positively Regulates Stress&#45;responsive ABA Signaling in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Qi Xie</span>
	<sup>1</sup>, 

	<span  >
	Yiyue Zhang</span>
	<sup>1</sup>, 

	<span  >
	Chengwei Yang</span>
	<sup>2</sup>, 

	<span  >
	Yin Li</span>
	<sup>2</sup>, 

	<span  >
	Nuoyan Zheng</span>
	<sup>2</sup>, 

	<span  >
	Hao Chen</span>
	<sup>1</sup>, 

	<span  >
	Qingzhen Zhao</span>
	<sup>1</sup>, 

	<span  >
	Ting Gao</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Genetics and Developmental Biology, Chinese Academy of Science, Beijing 100101, China, 

	<sup>2</sup>Zhongshan University, Guangzhou 501275, China

</div>

<div>Ubiquitination plays an important role in plant hormone signal transduction. Here we show that the novel RING finger E3 ligase, Arabidopsis SDIR1 &#40;Salt and Drought Induced RING finger 1&#41;, is involved in abscisic acid &#40;ABA&#41;&#45;related stress signal transduction. The expression of SDIR1 was detected in all tissues of Arabidopsis by RT&#45;PCR. SDIR1 expression is upregulated by drought and salt stress, but not by ABA. Transgenic Arabidopsis plants carrying the SDIR1 promoter fused to the GUS gene confirmed the results, in particular, strong induction of GUS expression was found in stomatal guard cells and leaf mesophyll cells under drought stress. The GFP&#45;SDIR1 fusion protein is co&#45;localized with intracellular membranes. We confirmed that SDIR1 is an E3 ubiquitin ligase and that the RING finger conservation region is required for its activity. Overexpression of SDIR1 leads to ABA hypersensitivity and ABA&#45;associated phenotypes, such as salt hypersensitivity in germination, enhanced ABA&#45;induced stomata closing, and enhanced drought tolerance. The expression levels of a number of key ABA and stress marker genes are altered both in SDIR1&#45;overexpression and sdir1&#45;1 mutant plants. Cross complementation experiments showed that the ABI5, ABF3 and ABF4 genes can rescue the ABA insensitive phenotype of the sdir1&#45;1 mutant, whereas SDIR1 could not rescue the abi5&#45;1 mutant. This suggests that SDIR1 acts upstream of those bZIP family genes. Our results indicate that SDIR1 is a positive regulator of ABA signaling.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Role of an auxilin&#45;like J&#45;domain protein, JAC1, on the regulation of actin filaments during chloroplast avoidance response</div>

<div>

	<span style="text-decoration: underline" >
	Noriyuki Suetsugu</span>
	<sup>1</sup>, 

	<span  >
	Noboru Yamada</span>
	<sup>2</sup>, 

	<span  >
	Akeo Kadota</span>
	<sup>2</sup>, 

	<span  >
	Masamitsu Wada</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute for Basic Biology, Okazaki, Japan, 

	<sup>2</sup>Tokyo Metropolitan University, Hachioji, Japan

</div>

<div>Chloroplasts relocate in plant cells according to the ambient light conditions. Chloroplasts accumulate in weak light to capture light efficiently &#40;accumulation response&#41; and they escape from strong light to avoid photodamage &#40;avoidance response&#41;. In <span style="font-style: italic">Arabidopsis thaliana</span>, the accumulation response is mediated by two phototropins &#40;phot1 and phot2&#41; that act redundantly and the avoidance response is mediated by phot2 alone. Dynamic organization of short actin filaments on chloroplasts &#40;cp&#45;actins&#41; by blue light mediates chloroplast photorelocation movement. Using a new screening method, we recently isolated a mutant, <span style="font-style: italic">jac1</span> &#40;<span style="font-style: italic">J</span>&#45;domain protein required for chloroplast <span style="font-style: italic">ac</span>cumulation response 1&#41;, which lacks the accumulation response but retains the avoidance response &#40;Suetsugu et al., 2005, Plant Physiol.&#41;. <span style="font-style: italic">JAC1</span> gene encodes a C&#45;terminal J&#45;domain protein similar to clathrin uncoating factor auxilin at its C&#45;terminus. In this study, we analyzed in detail chloroplast avoidance movement and cp&#45;actin dynamics during the avoidance in petiole cells of wild type and <span style="font-style: italic">jac1</span> mutant. </div>

</div>





<div class="abs">

<div style="font-weight:bold">REGULATORY GENE NETWORK IN DROUGHT AND ABA RESPONSES</div>

<div>

	<span style="text-decoration: underline" >
	Kazuo Shinozaki</span>
	<sup>1</sup>, 

	<span  >
	Kazuko Yamaguchi-Shinozaki</span>
	<sup>2</sup>, 

	<span  >
	Motoaki Seki</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN Plant Science Center, Yokohama, Japan, 

	<sup>2</sup>The University of Tokyo, Tokyo &amp;  Japan International Research Center for Agricultural Sciences (JIRCAS), Tsukuba, Japan

</div>

<div>Drought stress induces a variety of genes of which products function in drought stress tolerance and response in plants. Many stress&#45;inducible genes have been used to improve stress tolerance by gene transfer.  In this meeting, we present our recent studies on regulatory networks in drought and ABA responses. We have identified complex regulatory systems in stress&#45;responsive gene expression:  ABA&#45;dependent and ABA&#45;independent systems. In one of the ABA&#45;independent pathways, a cis&#45;acting element &#40;DRE&#47;CRT&#41; and its binding proteins, DREB2s, are important cis&#45; and trans&#45;acting elements in drought&#45;responsive gene expression, respectively. DREB2 is also involved in heat stress response. In the ABA&#45;dependent pathways, bZIP transcription factors &#40;AREB&#47;ABF&#41; are involved in the major process. Protein phosphorylation is important for the activation of AREB proteins. The MYB&#47;MYC and NAC transcription factors are involved in ABA&#45;responsive gene expression and jasmonic acid responses. <br/>    In the ABA&#45;dependent pathway, stress&#45;inducible NCED3 is mainly involved in the ABA biosynthesis during drought stress. We analyzed metabolic profiles regulated by ABA using T&#45;DNA tagged mutant and with GC&#45;MS and LC&#45;MS. We discuss the function of CYP707A3 in the regulation of ABA metabolism during stress responses. We also report the functions of SnRK2 protein kinases in drought and ABA responses using mutants and transgenic overexpressors. In addtion, recent progress in transcriptome analysis for novel transcripts including noncoding RNAs, miRNAs, and sense&#45;antisense RNAs will be presented based on tiling array analysis and 454 sequencing of small RNAs.<br/><br/>Umezawa et al. Curr Opin Biotech 17:113&#45;122 &#40;2006&#41;<br/>Yamaguchi&#45;Shinozaki and Shinozaki. Ann Rev Plant Bio 57:781&#45;803 &#40;2006&#41; <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">MPK3&#47;6 Protein Accumulation Primes <span style="font-style: italic">Arabidopsis</span> for Enhanced Stress Resistance</div>

<div>

	<span style="text-decoration: underline" >
	Gerold Beckers</span>
	<sup>1</sup>, 

	<span  >
	Yidong Liu</span>
	<sup>2</sup>, 

	<span  >
	Shuqun Zhang</span>
	<sup>2</sup>, 

	<span  >
	Uwe Conrath</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant Biochemistry &amp;  Molecular Biology Group, Department of Plant Physiology, RWTH Aachen University, Aachen, 52056, Germany., 

	<sup>2</sup>Department of Biochemistry, 371G Life Sciences Center, University of Missouri-Columbia, Columbia, MO 65211, USA.

</div>

<div>In plants, induced resistance to biotic and abiotic stress is associated with an enhanced capacity for more rapid and intense activation of defense responses, a phenomenon called &#8220;priming&#8221;. The mechanism of priming and the cellular components that mediate priming remained obscure. We show that in Arabidopsis priming is associated with the accumulation of transcripts and protein of mitogen&#45;activated protein kinase 3 and 6 &#40;MPK3&#47;6&#41;. MPK3&#47;6 accumulation is associated neither with dual phosphorylation of the MPK3&#47;6 TEY activation motif nor with MPK3&#47;6 enzyme activity. However, upon exposure to stress, activation site phosphorylation and MPK3 activity are induced to high levels in primed plants, and this leads to boosted defense gene expression and disease resistance. Furthermore, <span style="font-style: italic">mpk3</span> and <span style="font-style: italic">mpk6</span> mutants cannot be primed for induced resistance. Thus, we provide the first evidence that provision of a latent signaling component is a mechanism for priming plants to better fend off stress. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of three Arabidopsis AP2&#47;EREBP family transcription factors that were involved in ABA sensitivity, freeze and salt tolerance</div>

<div>

	<span style="text-decoration: underline" >
	Wenqian Mei</span>
	<sup>1</sup>, 

	<span  >
	Pei Han</span>
	<sup>1</sup>, 

	<span  >
	Xiang Jin</span>
	<sup>1</sup>, 

	<span  >
	Huimin Tan</span>
	<sup>1</sup>, 

	<span  >
	Yuxian Zhu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Laboratory of Protein Engineering and Plant Genetic Engineering,College of Life Science,Peking University,Beijing,China

</div>

<div>AP2&#47;EREBP transcription factors &#40;TFs&#41; play very important roles in plant development, hormonal regulation and stress response. Upon genome&#45;wide cDNA cloning, phylogenetic and expression pattern analyses of this plant specific TF family, we found that three of the members including At1g71450, At1g50680 and At5g13910 were likely involved in ABA, cold and salt responses. Complementary DNAs containing putative full length ORFs of these three TFs were obtained and fused individually to the GAL4 DNA&#45;binding domains. All three genes functioned effectively as trans&#45;activators using yeast one&#45;hybrid assay. RT&#45;PCR experiments showed that At1g71450 gene was induced by ABA and low temperature, At1g50680 gene was responsive to quite a few stress conditions especially to freezing temperature and At5g13910 gene was induced by high salt treatment, drought and ethylene. By searching the ABRC T&#45;DNA insertion mutant stocks, we obtained knockout lines for these TFs. Homozygous ko1 &#40;At1g71450&#41; plants showed hypersensitive response to ABA during seed germination and also in stomata movement. Homozygous ko2 &#40;At1g50680&#41; plants showed a significant reduction in plant freezing tolerance compared to the wild type after chilling treatment.Homozygous ko3 &#40;At5g13910&#41; were less tolerant to high salinity than wild type plants. Our data suggest At1g71450 is a negative regulator in ABA signaling while At1g50680 and At5g13910 are positive regulators in cold and salt stress responses&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional characterization of AtATM1, AtATM2 and AtATM3, a subfamily of Arabidopsis half&#45;molecule ABC transporters implicated in iron homeostasis</div>

<div>

	<span style="text-decoration: underline" >
	Sixue Chen</span>
	<sup>1</sup>, 

	<span  >
	Roc&#237;o S&#225;nchez-Fern&#225;ndez</span>
	<sup>2</sup>, 

	<span  >
	Elise Lyver</span>
	<sup>2</sup>, 

	<span  >
	Andrew Dancis</span>
	<sup>2</sup>, 

	<span  >
	Philip A. Rea</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>University of Florida, Gainesville, FL32610, USA, 

	<sup>2</sup>University of Pennsylvania, Philadelphia, PA19104, USA

</div>

<div>Arabidopsis AtATM1, AtAATM2 and AtATM3 are half&#45;molecule ATP&#45;binding cassette &#40;ABC&#41; proteins that are homologous to the yeast ATM1 &#40;ScATM1&#41; protein. The ScATM1 localizes to the inner mitochondrial membrane and is implicated in the export of mitochondrially synthesized iron&#45;sulfur &#40;Fe&#47;S&#41; clusters to the cytosol for the assembly of redox&#45;active Fe&#47;S proteins. Yeast ATM1&#45;deficient &#40;atm1&#41; mutants grow very slowly, are respiration&#45;deficient, accumulate toxic levels of iron in the mitochondria and show enhanced high&#45;affinity iron uptake. Of the three AtATMs, AtATM3 bears the closest functional resemblance to ScATM1. AtATM3 is not only able to complement the growth phenotype of yeast atm1 mutants but is also able to suppress the high&#45;affinity iron uptake associated with the mutation of ScATM1, abrogate intramitochondrial iron hyperaccumulation, and restore mitochondrial respiratory function and cytochrome c levels. By comparison the functional resemblance of AtATM1 and AtATM2 to ScATM1 is weaker. AtATM1 only partially suppresses the atm1 phenotype, exerts little effect on high&#45;affinity iron uptake, and only partially alleviates mitochondrial iron hyperaccumulation and cytochrome c levels. AtATM2 exerts little suppressive action on the yeast atm1 phenotype but instead is toxic when expressed in this system. These differences between AtATM3 and AtATM1 are maintained after exchanging their target peptides and these proteins as well as AtATM2 colocalize with the mitochondrial fluor MitoTracker Red when expressed in yeast as GFP fusions. Of the three AtATM::GFP fusions, those derived from AtATM3 exhibit the most pronounced mitochondrial localization versus AtATM1::GFP and AtATM2::GFP fusions when ectopically expressed in Arabidopsis. While its toxicity when heterologously expressed in yeast except when fused with GFP precluded the functional analysis of native AtATM2, a common function, mitochondrial export of Fe&#47;S clusters or their precursors for the assembly of cytosolic Fe&#47;S proteins, is inferred for AtATM3 and AtATM1.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Great promoting effect of high irradiance from germination on flowering in Arabidopsis thaliana &#8211; a process of photo&#45;acclimation</div>

<div>

	<span style="text-decoration: underline" >
	Shubhangi Moharekar</span>
	<sup>1</sup>, 

	<span  >
	Sanjay Moharekar</span>
	<sup>1</sup>, 

	<span  >
	Ryouichi Tanaka</span>
	<sup>1</sup>, 

	<span  >
	Ayumi Tanaka</span>
	<sup>1</sup>, 

	<span  >
	Toshihiko Hara</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Low Temperature Science, Hokkaido University, Sapporo, Japan

</div>

<div>Arabidopsis thaliana &#40;L.&#41;, ch1&#45;1 &#40;chlorophyll b&#45;less mutant&#41;, gi&#45;1 &#40;GI deficient mutant&#41;, cry2&#45;1 &#40;blue&#45;light&#45;photoreceptor CRY2 deficient mutant&#41; and Columbia &#40;Col&#41; &#40;wild ecotype&#41; were grown under broad range of irradiance from the beginning of germination  and the effect of light intensity on the survival, development and flowering was studied. Under low and moderate light intensities &#40;&#60; 300 &#181;mol m&#45;2s&#45;1&#41;, flowering time and plant size at flowering showed great variations among ch1&#45;1, gi&#45;1, cry2&#45;1 and Col, whereas under higher light intensities &#40;&#62; 500 &#181;mol m&#45;2s&#45;1&#41;, these were almost the same. These results suggest that under high light intensities, development and flowering of ch1&#45;1, gi&#45;1, cry2&#45;1 and Col converge to almost the same state. Flowering time was negatively correlated with light intensity and under higher light intensities, acclimation in A. thaliana is associated with a decrease in chlorophyll &#40;Chl&#41; content and increases in xanthophyll cycle pool and membrane&#45;bound APX activity &#40;EC 1.11.1.11&#41;, suggest that an increase in oxidative stress induces earlier flowering. The plants of gi&#45;1 and cry2&#45;1 survived but Col and ch1&#45;1 died under 1000 &#181;mol m&#45;2s&#45;1, suggest that mutants deficient in GI or CRY2 are more photo&#45;stress&#45;tolerant than Col and Chl b&#45;less mutant. In this study, we demonstrate that high light intensity exerts on plants of Arabidopsis raised from germination till flowering, a promoting effect on development and flowering time involving modulation of the photosynthetic apparatus, and this promoting effect seems to be independent of the functions of flower&#45;inducing GI or CRY2 gene. This can be regarded as photo&#45;acclimation of A. thaliana for survival and reproduction under high light conditions. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic dissection of the plant circadian clock</div>

<div>

	<span style="text-decoration: underline" >
	Seth Davis</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, Cologne, Germany

</div>

<div>Proper plant growth and development requires a robust detection of the diurnal environment. This occurs through a coupling mechanism of light detection and the circadian clock. We recently took a genetic approach to confirm the core of the circadian oscillator, which is comprised of the morning&#45;acting genes CCA1 and LHY that work with the evening&#45;acting gene TOC1 &#40;Ding, Genetics 2007a&#41;. From this core, we have further characterized elements that work at the dawn and dusk boundaries. For dawn, we genetically isolated the TIC locus required for proper detection of the dark to light transition. The cloning of this gene provides an understanding that TIC regulates dawn sensation through LHY regulation &#40;Ding, Plant Cell 2007b&#41;. We have further found that TIC responds through post&#45;transcriptional processes that include phosphorylation and ubiquitin, suggesting that dawn detection is molecularly signaled through regulated proteolysis. To further our understanding of dusk perception, we are working on the ELF3 and ELF4 loci required for detection of the light to dark transition. Various molecular&#45;physiological analyses have been use to place the ELF3 and ELF4 proteins within a model framework of the clock &#40;McWatters, Plant Phys. 2007&#41;. We have also uncovered an allelic series of elf3 and elf4 mutations. Characterization of these mutations at physiological and biochemical levels is assisting our understanding of the protein function of these pioneer proteins. The current model from these collective efforts is that a multiple&#45;protein transcriptional&#45;regulator complex forms to regulate initiation of CCA1 and LHY. The mode of this mechanism of action will be discussed. Our collective use of genetics and protein analyses is enhancing our understanding of the biochemical mechanisms plants use to detect the boundaries ever present every day.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Correlation analysis and QTL identification for some photosynthetic factors &#13;<br/>in rice under different environment &#45;watered conditions&#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	Songping Hu</span>
	<sup>1,2</sup>, 

	<span  >
	Lijun Luo</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup> College of Resource and Environmental Science, Jishou university, Hunan province, China, 

	<sup>2</sup> Shanghai Agrobiological Gene Center, Shanghai , China.

</div>

<div>Methods: &#13;<br/>A total of 195 RIL population developed between the paddy rice Zhenshan97B and upland rice IRAT109 and their parents. All materials were planted randomly with three replications in the drought screen facility in Shanghai , China &#13;<br/>The facility was set up in a rainproof greenhouse including two islands separated by 1.8 m depth canal. When starting stress, the sprinkler irrigation was stopped while the drip irrigation was still working and underground water was drained out through the canal. A water gradient from island edge to center was formed and drought treatment would also be built naturally. The plants in the center were in well &#45;watered condition while the plants near edge were in the stress. &#13;<br/> Phenotyping and genotyping&#13;<br/>A total of 213 microsatellite markers were used to genotype the population. A linkage map was constructed by using MapMaker&#47;Exp V3.0 &#40;Lincoln et al., 1992&#41; to span 1825.0 cM of genome size with an average distance of 8.6 cM between adjacent markers. &#13;<br/>Results: &#13;<br/>The relationship between CC and PR:  &#13;<br/>There was significant positive correlation&#40;r&#61;0.1857&#42;&#42;&#41; between CC and PR of rice RILs&#8217; leaves in well&#45;watered condition, while not significant in drought&#40;r&#61;0.0766&#41;, indicating that the enhancing function of high CC to PR could be exerted only in well&#45;watered condition.&#13;<br/>The main QTLs: A total of 13 QTLs related to CC were detected and located on chromosomes 1,2,3,4,5,6 and 10; The phenotypic variations explained by single QTL ranged from 3.54&#37; to 14.53&#37;. Seven QTLs were found inwell&#45;watered condition accounting totally 56.19&#37; phenotypic variation. Six QTLs were detected in drought explaining totally 47.39&#37; phenotypic variation. Six QTLs were detected in both two conditions. &#13;<br/>Four QTLs, of which three in stress and one inwell&#45;watered related to PR were located on chromosome 2, 10 and 11 respectively. qPR10 was detected in both conditions. The phenotypic variance explained by 3 QTLs of PR in stress was 11.03&#37; together, while in well&#45;watered it was 2.91&#37;.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Cryptochrome signaling to the plant clock associates with inhibition of COP1&#45;mediated ELF3 ubiquitination</div>

<div>

	<span  >
	Jae-Woong Yu</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Vicente Rubio</span>
	<sup>2,3</sup>, 

	<span  >
	Sulan Bai</span>
	<sup>2</sup>, 

	<span  >
	Na-Yeoun Lee</span>
	<sup>1</sup>, 

	<span  >
	James A. Sullivan</span>
	<sup>2</sup>, 

	<span  >
	Sang-Sook Kim</span>
	<sup>1</sup>, 

	<span  >
	Sun-Young Lee</span>
	<sup>1</sup>, 

	<span  >
	Ilha Lee</span>
	<sup>4</sup>, 

	<span  >
	Nam-Chon Paek</span>
	<sup>1</sup>, 

	<span  >
	Xing Wang Deng</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Science and Research Institute for Agriculture &amp;  Life Sciences, Seoul National University, Seoul, Korea, 

	<sup>2</sup>Department of Molecular, Cellular, and Developmental Biology, Yale University, New Haven, CT, USA, 

	<sup>3</sup>Department of Plant Molecular Genetics, Centro Nacional de Biotecnolog&#237;a-CSIC, Madrid, Spain, 

	<sup>4</sup>Department of Biological Sciences, Seoul National University, Seoul, Korea

</div>

<div>Seasonal changes in daylength are perceived by plant photoreceptors and transmitted to the circadian clock to modulate developmental responses, such as flowering time. Cryptochromes and clock&#45;associated ELF3 regulate this process although a link between them is unclear. Here, we show that <span style="font-style: italic">COP1</span> mediates daylength signaling from <span style="font-style: italic">CRY2</span> to <span style="font-style: italic">ELF3</span> within the photoperiod flowering pathway. We find that ELF3 is ubiquitinated by COP1 <span style="font-style: italic">in vitro</span> and that this activity decreases in the presence of the constitutively active C&#45;termini of CRY1 &#40;CCT1&#41; and CRY2 &#40;CCT2&#41;. Accordingly, rhythmic accumulation of ELF3 protein is altered in both <span style="font-style: italic">cop1&#45;4</span> mutants and plants overexpressing CCT2. Remarkably, mutation of COP1 overcomes both the late flowering phenotype and the increased period length of <span style="font-style: italic">LHY</span> rhythmic accumulation caused by ELF3 overexpression. Therefore, we propose that ELF3 ubiquitination by COP1 is essential for ELF3 function to modulate light signal input from cryptochromes to the circadian clock.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Universal Stress Potein &#40;USP&#41;&#45;A possible link between, stress, early flowering and accelerated senescence</div>

<div>

	<span  >
	Natalia Edison</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Shimon Gepstein</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Technion-Israel Institute of Technology,Haifa,ISRAEL

</div>

<div>Universal Stress Proteins have been described widely in prokaryotes, where their production is stimulated by starvation or large variety of stresses. In eukaryotes very little is known about these proteins and their function has not been revealed. Some genes with homology to USP family have been briefly described in plants and in Arabidopsis USP gene family has been identified.&#13;<br/>We have identified USP gene that was preferentially expressed during senescence.The USP transcript levels were found to be the highest during the initiation of flowering ,a stage known to be related to the induction  of senescence.Stress&#45;induced senescence displayed dramatic up&#45;regulation of the USP gene with synergistic effect of  dark and leaf detachment.Periodic rhythm, with peak in the afternoon, of USP transcript levels has been demonstrated. Addition of ACC, the precursor of ethylene, caused an up&#45;regulation of the USP transcript, whereas addition of cytokinins and ozone caused down&#45;regulation of the expression of the USP gene. Over expression of the USP gene   exhibited significant acceleration of life cycle. Sub&#45;cellular immunolocalization  analysis suggest chloroplast localization of the USP.  Taken together, the results  raise the possibility that the USP gene &#40;At3g62550&#41; serves as a link between stress,early flowering and  premature senescence.&#13;<br/> </div>

</div>





<div class="abs">

<div style="font-weight:bold">Photoreceptor&#45;interacting proteins as immediate facilities for environmental light information processing</div>

<div>

	<span  >
	Jong Sang Ryu</span>
	<sup>1</sup>, 

	<span  >
	Sung Hyun Hong</span>
	<sup>1</sup>, 

	<span  >
	Hyunmo Choi</span>
	<sup>1</sup>, 

	<span  >
	Tomonao Matsushita</span>
	<sup>2</sup>, 

	<span  >
	Akira Nagatani</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Hong Gil Nam</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Division of Molecular Life Sciences and National Core Research Center for Systems Bio-Dynamics, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea, 

	<sup>2</sup>Laboratory of Plant Physiology, Graduate School of Science, Kyoto University, Sakyo-Ku, Kyoto 606-8502, Japan, 

	<sup>3</sup>Division of Molecular Life Sciences and National Core Research Center for Systems Bio-Dynamics and The I-BIO Graduate Program, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea.

</div>

<div>Plants, being photosynthetic and sessile, exhibit particularly plastic development and growth, depending on the environmental light information. The quality, intensity, duration, and direction of the environmental light provide plants with information not only on the ambient light condition but also on other elements in their environments such as neighboring plants and seasonal changes. Plants perceive light information using specialized photoreceptors, the red &#40;R&#41; and far&#45;red &#40;FR&#41; light&#45;absorbing phytochromes and the blue&#47;UV&#45;A&#45;absorbing cryptochromes and phototropins that ensure optimal responses to their ever&#45;changing environmental conditions. To transmit the responsive light information properly, plants have evolved a set of signaling components that assess and relay the quantitative and qualitative natures of the incident signals to the changes of gene expression that control responses in growth and development. Since protein&#45;protein interactions are fundamental to all cellular signaling processes, we are attempting identification and functional assignment of the proteins that interact with photoreceptors with the aim to understand the very early steps in light signaling. We have identified several putative phyB&#45;interacting proteins, including PAPP5. We will describe how the spatial distribution of PAPP5 is affected by interaction with phyB and in turn affects light signaling. We will also describe the role of a potential cry&#45;interacting transcription factor.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Root tip contact with low&#45;phosphate media reprogrammes plant root architecture</div>

<div>

	<span  >
	Sergio Svistoonoff</span>
	<sup>1,2</sup>, 

	<span  >
	Audrey Creff</span>
	<sup>1</sup>, 

	<span  >
	Matthieu Reymond</span>
	<sup>1,3</sup>, 

	<span  >
	C&#233;cile Sigoillot-Claude</span>
	<sup>1</sup>, 

	<span  >
	Lilian Ricaud</span>
	<sup>1</sup>, 

	<span  >
	Aline Blanchet</span>
	<sup>1</sup>, 

	<span  >
	Laurent Nussaume</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Thierry Desnos</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CEA, 

	<sup>2</sup>IRD, 

	<sup>3</sup>Max Planck Institute for Plant Breeding Research

</div>

<div>In plants, the roots are able to sense soil nutrients availability. In order to acquire heterogeneously distributed water and minerals, they optimize their root architecture. One poorly understood plant response to soil phosphate &#40;Pi&#41;&#45;deficiency is a reduction in primary root growth with an increase in the number and length of lateral roots. Here we show that physical contact of the Arabidopsis thaliana primary root tip with low&#45;Pi medium is necessary and sufficient to arrest root growth. We further show that loss&#45;of&#45;function mutations in LPR1 and its close paralogue LPR2 strongly reduce this inhibition. LPR1 was previously mapped as a major quantitative trait locus &#40;QTL&#41;; the molecular origin of this QTL is explained by the differential allelic expression of LPR1 in the root cap. These results provide strong evidences for the involvement of the root cap in sensing and&#47;or responding to nutrient deficiency. LPR1 and LPR2 encode multicopper oxidases &#40;MCO&#41;, highlighting the essential role of MCO for plant development.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Involvement of Arabidopsis HOS15 in histone deacetylation and cold tolerance</div>

<div>

	<span style="text-decoration: underline" >
	Jianhua Zhu</span>
	<sup>1</sup>, 

	<span  >
	Jaecheol Jeong</span>
	<sup>2</sup>, 

	<span  >
	Yanmei Zhu</span>
	<sup>3</sup>, 

	<span  >
	Irina Sokolchik</span>
	<sup>3</sup>, 

	<span  >
	Saori Miyazaki</span>
	<sup>4</sup>, 

	<span  >
	Jian-Kang Zhu</span>
	<sup>5</sup>, 

	<span  >
	Paul M. Hasegawa</span>
	<sup>3</sup>, 

	<span  >
	Hans J. Bohnert</span>
	<sup>4</sup>, 

	<span  >
	Huazhong Shi</span>
	<sup>6</sup>, 

	<span  >
	Dae-Jin Yun</span>
	<sup>2</sup>, 

	<span  >
	Ray A. Bressan</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>Purdue University, West Lafayette, IN; University of California, Riverside, CA, USA, 

	<sup>2</sup>Gyeongsang National University, Jinju 660-701, Korea, 

	<sup>3</sup>Purdue University, West Lafayette, IN, USA, 

	<sup>4</sup>University of Illinois, Urbana, IL, USA, 

	<sup>5</sup>University of California, Riverside, CA, USA, 

	<sup>6</sup>Texas Tech University, Lubbock, TX, USA

</div>

<div>Histone modification in chromatin is one of the key control points in gene regulation in eukaryotic cells. Protein complexes composed of histone acetyltransferase or deacetylase, WD40 repeat protein and many other components have been implicated in this process.  The WD40 repeat protein in these complexes functions as a bridge through protein&#45;protein interactions to link histones in chromatin with other proteins in the complex including histone modifying enzymes.  Here, we report the identification and functional characterization of HOS15, a WD40 repeat protein crucial for repression of abiotic stress&#45;responsive genes through histone deacetylation in Arabidopsis.  The HOS15 gene was identified by forward genetic screening for mutations that alter the expression of abiotic stress&#45;responsive genes. The mutation in the HOS15 gene renders the mutant hypersensitive to freezing temperatures and causes late flowering. HOS15 shares high sequence similarity with human transducin&#45;beta like proteins &#40;TBL&#41;, a component of a repressor protein complex involved in histone deacetylation.  HOS15 is localized in the nucleus and specifically interacts with histone H4.  The level of acetylated histone H4 is higher in the hos15 mutant than in wild type plants.  Moreover, the RD29A promoter in the hos15 mutant is associated with a substantially higher level of acetylated histone H4 than in wild type under cold stress conditions. This is consistent with enhanced induction of the RD29A gene in the hos15 mutant.  Our results suggest a critical role for gene activation&#47;repression by histone acetylation&#47;deacetylation in plant acclimation to cold stress.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Constitutive Expression of <span style="font-style: italic">ThIPK2</span>, an Inositol Polyphosphate kinase, from <span style="font-style: italic">Thellungiella halophila</span>, Enhances Multiple Abiotic Stress Tolerance, and Alters Seed Oil Fatty Acid Composition in Transgenic <span style="font-style: italic">Brassica napus</span> L.</div>

<div>

	<span style="text-decoration: underline" >
	Jinqi Zhu</span>
	<sup>1</sup>, 

	<span  >
	Jiantao Zhang</span>
	<sup>1</sup>, 

	<span  >
	Qiuqing Wang</span>
	<sup>1</sup>, 

	<span  >
	Lei Yang</span>
	<sup>1</sup>, 

	<span  >
	Renjie Tang</span>
	<sup>1</sup>, 

	<span  >
	Hongxia Zhang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Shanghai Institute of Plant Physiology and Ecology,SIBS,CAS,Shanghai , 200032, P.R.China

</div>

<div>It has been reported that the inositol polyphosphate 6&#45;&#47;3&#45;kinase &#40;<span style="font-style: italic">AtIPK2&#946;</span>&#41; in Arabidopsis plays a possible role as a transcriptional control mediator in higher plants. And some evidence suggested that <span style="font-style: italic">AtIPK2&#946;</span> plays an essential role in abiotic stresses, such as salt, drought and ABA.&#13;<br/><span style="font-style: italic">Thellungiella halophila</span> &#40;Salt Cress&#41;, a halophyte and cryophyte&#13;<br/>Arabidopsis relative model plant, shows 90&#45;95&#37; identity with Arabidopsis genomic DNA and higher salt tolerant ability than Arabidopsis. To further study the homogenous gene&#8217;s functions and characterizations, we first isolated an inositol polyphosphate kinase gene &#40;<span style="font-style: italic">ThIPK2</span>&#41; was from <span style="font-style: italic">Thellungiella halophila</span>, and investigated its functions in yeast and <span style="font-style: italic">Brassica napus</span> L. cv. Huyou No. 1 &#40;canola&#41;. The results shows that <span style="font-style: italic">ThIPK2</span> shares 85&#37; amino acid identity &#40;91&#37; similarity&#41; with Arabidopsis <span style="font-style: italic">AtIPK2&#946;</span> &#40;GenBank accession no. AY147936&#41;. Transient expression of a ThIPK2&#45;green fluorescent protein &#40;GFP&#41; fusion protein in onion &#40;Alliumcepa&#41; cells indicated that the ThIPK2&#45;GFP fusion protein was localized to both nucleus and plasma membrane. Expression of <span style="font-style: italic">ThIPK2</span> in <span style="font-style: italic">Saccharomyces arg82&#916;&#47;Ipk2&#916;</span>, a mutant lacking Ipk2 activity, rescued the mutant&#8217;s normal growth at 37&#176;C and growth under high salt or osmotic stress condition. Constitutive expression of <span style="font-style: italic">ThIPK2</span> in canola enhanced tolerance to salt, dehydration and oxidative stress of transgenic plants. Furthermore, expression of <span style="font-style: italic">ThIPK2</span> increased the transcripts of stress responsive genes and altered fatty acid constituents of transgenic canola, resulting in an increased level of oleic acid &#40;C18:1&#41; and a decreased level of linoleic acid &#40;C18:2&#41; in transgenic seeds. Taken together, the results in this study implicated a mutual correlation between <span style="font-style: italic">ThIPK2</span> and abiotic&#45;related defense responses, and a possible regulatory role of <span style="font-style: italic">ThIPK2</span> in fatty acid metabolic syntheses in canola.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">ZBF3&#47;CAM7 plays an important role as transcriptional regulator in Arabidopsis seedling development</div>

<div>

	<span  >
	Ritu Kushwaha</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Sudip Chattopadhyay</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Centre for Plant Genome Research

</div>

<div>Calmodulin &#40;CaM&#41; plays multiple regulatory roles in eukaryotes as Ca&#43;&#43; sensor, however its direct function as transcriptional regulator is still unknown. Additionally, the physiological functions of CaM remains largely unknown in plants. In this study, we demonstrate that CAM7 is a unique transcriptional regulator of light signalling that directly binds to the promoters of light inducible genes and promotes photomorphogenesis. The mutational, transgenic and physiological studies illustrate the concerted function of CAM7 and HY5 in Arabidopsis seedling development. </div>

</div>





<div class="abs">

<div style="font-weight:bold">DIG5, a novel plastidic protein, regulates auxin polar transport and lateral root development in <span style="font-style: italic">Aidopsis thaliana</span></div>

<div>

	<span style="text-decoration: underline" >
	Wei Liu</span>
	<sup>1,2</sup>, 

	<span  >
	Ruigang Wang</span>
	<sup>1</sup>, 

	<span  >
	Liming Xiong</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Donald Danforth Plant Science Center,St. Louis,Missouri 63132,USA, 

	<sup>2</sup>High-Tech Research Center,Shandong Academy of Agricultural Sciences,Jinan 250100,China

</div>

<div>Environmental conditions such as nutrients, water, and soil mechanical properties have dramatic impacts on pastembryonic root development of higher plants. However, the molecular mechanisms underlying root development are largely unknown. To understand the mechanism involved in root adaptation to water deficit, we initiated a genetic analysis of root response to osmotic stress and abscisic acid &#40;ABA&#41; in Arabidopsis &#40;<span style="font-style: italic">Plant Physiol</span> 142:1065&#41;. One of the loci defined in the study, <span style="font-style: italic">DIG5</span> &#40;for Drought Inhibition of lateral root Growth 5&#41; was studied in detail. The <span style="font-style: italic">dig5</span> mutants have significantly fewer lateral roots than the wild type either with or without ABA treatments. Their leaves are pale green and have irregular venation patterning. Analyses of hormonal responses in the <span style="font-style: italic">dig5</span> mutant found the seedlings are able to respond to auxin in promoting lateral root growth but are less sensitive to ethylene in inducing apical hook formation. Assays of auxin transport in the mutant found that both acropetal and basipetal auxin transport are impaired. Map&#45;based cloning and molecular complementation revealed that <span style="font-style: italic">DIG5</span> encodes a novel protein that shows no overall sequence homology to other proteins but contains a catalytic domain found in nucleotide deaminases that may participate in RNA editing. Sequence profiling of all major RNA editing sites in the chloroplast and mitochondrial transcriptomes, however, failed to find any changes in RNA editing in the mutant. We also were unable to detect the deaminase activity of the recombinant full&#45;length or truncated DIG5 proteins. Nonetheless, HPLC analysis of the mutant root extracts revealed that <span style="font-style: italic">dig5</span> accumulates several&#45;fold flavonoids including kaempferol, naringenin, and dihydroflavone. We propose that the DIG5 enzyme catalyzes a key step in a novel metabolic pathway whose blockage causes its substrates to flow to the flavonoid pathway. As a result, the accumulation of flavonoids impairs auxin polar transport and root development. Supported by USDA&#45;ARS &#40;grant&#35;2004&#45;02111 to LX&#41;.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification and Phenotyping of T&#45;DNA Insertion lines of AtGLRs in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Lai-Hua Liu</span>
	<sup>1</sup>, 

	<span  >
	Pia Walch-Liu</span>
	<sup>2</sup>, 

	<span  >
	Xiao-Qing Qu</span>
	<sup>1</sup>, 

	<span  >
	Feng-Qiu Cao</span>
	<sup>1</sup>, 

	<span  >
	Wei-Hong Wang</span>
	<sup>1</sup>, 

	<span  >
	 Brian Forde</span>
	<sup>2</sup>, 

	<span  >
	Julia Davies</span>
	<sup>3</sup>, 

	<span  >
	Mark Tester</span>
	<sup>4</sup>

</div>

<div>

	<sup>1</sup>College of Resources and Environmental Sciences, China Agricultural University, 100094 Beijing, China, 

	<sup>2</sup>Biological Sciences, Lancaster University, Bailrigg, Lancaster LA1 4YQ, United Kingdom, 

	<sup>3</sup>Department of Plant Sciences, University of Cambridge, Downing Street, CB2 3EA, United Kingdom, 

	<sup>4</sup>Australian Centre for Plant Functional Genomics and The University of Adelaide, Glen Osmond, Adelaide SA 5064, Australia

</div>

<div>Eight years ago, 20 genes encoding AtGLRs were identified and were predicted to be glutamate receptor&#45;like ion channels based on sequence similarity to animal iGluRs. However, a clear understanding of the biological or physiological function of any member of the AtGLR family is still lacking. &#13;<br/>Using a reverse genetic approach, by PCR&#45;based detection and gene expression analysis, we identified and isolated homozygous T&#45;DNA insertion lines of the 20 AtGLR genes from &#8220;Salk&#8221; and &#8220;GABI&#8221; seed sources. To understand potential roles in the plant physiology of these genes, we performed phenotype&#45;screening assays under a variety of growth conditions, including treatments with different nitrogen sources, sugar, amino acids, plant hormones, micro&#45;nutrients, low temperature. Several treatments revealed different growth phenotypes in the mutant lines compared with wildtype &#40;wt&#41; plants. Among all of the 20 Atglr mutants, only Atglr3.2 mutants displayed a reduced growth rate when treated with a combination of ammonium and no sugar in the medium. Aluminium, zinc and cold inhibited root growth to a greater extent in several Atglr lines than in controls. Atglr1.3, Atglr3.2 and Atglr3.4 mutants were more sensitive to NaCl compared to the other lines and wt. No Atglr line showed a distinct growth phenotype with IAA or ABA application compared with the wt. More detailed experimental data will be discussed in the conference. To further confirm and characterize observed plant phenotypes caused by T&#45;DNA insertions in the Atglr mutants, approaches such as histochemical analysis, pharmacological tests, heterologous expression for a survey of the channel function, studies of protein localization and promoter activity as well as functional complementation in plants are being taken.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Interaction between Arabidopsis HSF&#45;A1a and HSF&#45;A1b in vivo</div>

<div>

	<span style="text-decoration: underline" >
	Ming Li</span>
	<sup>1</sup>, 

	<span  >
	Fritz Schoeffl</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Zentrum f&#252;r Molekularbiologie der Pflanzen ? Allgemeine Genetik, Universit&#228;t T&#252;bingen, auf der Morgensstelle 28, 72076 T&#252;bingen, Germany

</div>

<div>The class A Heat Shock Factors HSF&#45;A1a and HSF&#45;A1b have been identified as early response regulators activating the transcription of several target genes upon heat stress in Arabidopsis thaliana &#40;Busch et al., 2005, Plant J., 41, 1&#45;14&#41;. The analysis of gene knock out mutations suggests that both HSF may interact in vivo and that they can functionally replace each other &#40;Lohmann et al., 2004 Mol Gen Genomics, 271, 11&#45;21&#41;. Using bimolecular fluorescence complementation we tried to investigate whether these HSF can interact in vivo. Arabidopsis thaliana &#40;Columbia&#41; cell culture protoplasts were transformed with suitable gene fusion constructs between full length HSF and C&#45; or respectively N&#45;terminal parts of the yellow fluorescence protein. Confocal microscopic imaging and fluorescence activated cell sorting  &#40;FACS&#41; analysis indicate that HSF&#45;A1a can interact in a homomeric and with HSF&#45;A1b in a heteromeric combination. Interaction can also be observed between truncated versions of HSF, representing only the oligomerization domains &#40;OD: hydrophobic repeat region&#41;. These interactions are localized in the cytoplasm, with no preference for the nucleus. However, no homo&#45; and heteromeric combinations were observed when the oligomerization domain HSF&#45;A1a was deleted. Our data indicate that these two structurally and functionally related HSF can interact in vivo, possibly through hetero&#45;oligomerization. This interaction may be instrumental in stress sensing and signalling in the very early phase of the heat shock response.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Identification of CO2 and ABA Signaling Components and Novel Membrane Transduction Mechanisms in Guard Cells.</div>

<div>

	<span style="text-decoration: underline" >
	Julian I. Schroeder</span>
	<sup>1</sup>, 

	<span  >
	Yong-Fei Wang</span>
	<sup>1</sup>, 

	<span  >
	Tae-Houn Kim</span>
	<sup>1</sup>, 

	<span  >
	Yingzhen Yang</span>
	<sup>1</sup>, 

	<span  >
	Robert S. Siegel</span>
	<sup>1</sup>, 

	<span  >
	Maria Israelsson</span>
	<sup>1</sup>, 

	<span  >
	Erwin Grill</span>
	<sup>2</sup>, 

	<span  >
	Jaakko Kangasjarvi</span>
	<sup>3</sup>, 

	<span  >
	Hannes Kollist</span>
	<sup>3</sup>, 

	<span  >
	Yoshiyuki Murata</span>
	<sup>4</sup>, 

	<span  >
	Izumi Mori</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Div. of Biol. Sciences, University of California, San Diego, La Jolla, CA, USA, 

	<sup>2</sup>T.U. Munich, Germany, 

	<sup>3</sup>U. Helsinki, Finland, 

	<sup>4</sup>Okayama U., Okayama, Japan

</div>

<div>Guard cells have been developed as a model system for dissecting ion channel functions and early signal transduction mechanisms. Genetic loss&#45;of&#45;function mutants in Ca2&#43;  sensors that impair Ca2&#43;&#45;  and abscisic acid&#45;regulated  stomatal movements have been lacking.  In addition, a Ca2&#43;&#45;independent pathway functions in the abscisic acid &#40;ABA&#41; response. We have recently identified two calcium&#45;dependent protein kinases &#40;CDPKs&#41; that function in abscisic acid &#40;ABA&#41; and Ca2&#43; regulation of guard cell anion and Ca2&#43;&#45;permeable channels and stomatal closing &#40;I. Mori et al., 2006 PLoS Biol.&#41;. Furthermore, several independent CO2 and abscisic acid signal transduction analyses suggest a new model for how plant cells can achieve specificity in calcium signaling through &#8220;priming&#8221; and &#8220;de&#45;priming&#8221; of Ca2&#43; sensitive mechanisms &#40;J. Young et al., 2006 PNAS&#41;.  Further evidence that correlates with this &#8220;Ca2&#43; sensitivity priming&#8221; hypothesis will be presented. In addition, evidence for a parallel pathway that functions in the ABA signaling  network will also be presented. A central target of ABA signaling is the activation of S&#45;type anion channels in guard cells. Identification of a new membrane protein&#45;encoding gene that is essential for mediating this response will be presented.&#13;<br/>Genetic, genomic and signal transduction analyses in a number of laboratories indicate that genetic redundancies and robustness exist within the abscisic acid signal transduction network. To address this complexity we have pursued genomic guard cell expression approaches &#40;e.g. Leonhardt et al., 2004 Pl Cell; Mori et al., 2006 PLoS Biol.&#41;.  More recently we have developed a chemical genetics approach that allows high&#45;throughput screening for molecules and mutants that affect ABA signal transduction. Progress at isolating a small molecule that blocks ABA responses and identification of mutants in ABA signaling that are insensitive to this compound will be presented.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Transgenerational memeory of stress in plants</div>

<div>

	<span  >
	Jean Molinier</span>
	<sup>1</sup>, 

	<span  >
	Gerhard Ries</span>
	<sup>2</sup>, 

	<span  >
	Cyril Zipfel</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Barbara Hohn</span>
	<sup>4</sup>

</div>

<div>

	<sup>1</sup>Institut de Biologie Moleculaire des Plantes, Strasbourg, France, 

	<sup>2</sup>BioMedinvestor AG, Basel, Switzerland, 

	<sup>3</sup>The Sainsbury Laboratory, John Innes Centre, Norwich, UK, 

	<sup>4</sup>Friedrich Miescher Institute, Basel, Switzerland

</div>

<div>Plants are constantly exposed to stresses from the environment. Both biotic and abiotic stresses can result in tolerance to conditions such as unbalanced water supply, light stress, extreme temperatures and pathogens. Influences such as these have also been documented to result in elevated rates of genomic changes such as transposition or homologous recombination. However, it has not been analysed whether the impact of these influences could also be inherited. Experiments using <span style="font-style: italic">Arabidopsis thaliana</span> plants transgenic for markers for genomic changes, however, showed that exposure of these plants to UV&#45;B or to the bacterial ellicitor flagellin  exhibited changes of homologous recombination far exceeding those of unexposed plants. Quite unexpectedly, however, the progeny of these plants still exhibited elevated recombination rates, although they themselves were not influenced by the agents their parents were exposed to. This change in the recombination frequency must be due to an epigenetic modification of an unknown locus or of unknown loci; it was the whole population that reacted to the challenge. If a mutation would have caused the described effect only a very small number of plants would have shown the changed behavior. The change was not located at the genomic position of the transgene used to monitor recombination, since plants lacking the transgene transmitted their experience of being treated to an untreated crossing partner containing the recombination monitoring transgene. This resulted in increaesed recombination frequencies in the progeny of these plants. These experiments may point to the importance of epigentic changes of plant chromatin in permitting evolutionarily important flexibilities useful for adaptation. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Defects in Chloroplast Purine Biosynthesis alter Metal Homeostasis in Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Yan He</span>
	<sup>1</sup>, 

	<span  >
	Alberto Maurer</span>
	<sup>1</sup>, 

	<span  >
	Elizabeth Rogers</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biochemistry and Bond Life Sciences Center, University of Missouri, Columbia, MO 65211, United States

</div>

<div>    Iron is essential for various proper biological functions of most living organisms. It is also important for hormone perception, enzyme activities, signaling, membrane transport and osmoregulation. However, excessive iron is toxic since iron can catalyze the formation of reactive oxygen species. The chloroplast is the major site for the use and storage of iron and other metals in plants. In a screen for altered iron homeostasis in a mutagenized population of Arabidopsis thaliana, we identified two new allelic lines, called SP46 and CS557, with mutations in AtATase2, a gene encoding a chloroplast&#45;localized enzyme that catalyzes the first step in purine biosynthesis. Mutations in ATase2 confer variegated chlorosis and stunted growth, especially under high light growth conditions. Chlorophyll concentration and ATP concentration in leaves of those mutants are lower than wild type. Additionally, SP46 and CS557 display a delayed induction of ferric&#45;chelate reductase activity when compared with wild type; ferric chelate reductase is a key activity in iron acquisition. The SP46 mutant also over&#45;accumulates some metals, including manganese, cobalt, and cadmium, while underaccumulating others &#40;zinc and copper&#41;; overall iron levels in SP46 are unchanged. We are currently testing the hypothesis that because photosynthesis in these mutants is impaired, they need less iron than wild type plants. Therefore, it takes longer under iron deficient conditions for the SP46 and CS557 mutants to become iron deficient and upregulate their ferric chelate reductase activity. These mutants provide the first evidence linking disruption of chloroplast function with general metal homeostasis in Arabidopsis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The roles of microRNAs in Arabidopsis thaliana under various abiotic stress conditions</div>

<div>

	<span style="text-decoration: underline" >
	Hwa Jung Lee</span>
	<sup>1</sup>, 

	<span  >
	Yeon Ok Kim</span>
	<sup>1</sup>, 

	<span  >
	Won Yong Kim</span>
	<sup>1</sup>, 

	<span  >
	Hunseung Kang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Chonnam National University, Gwangju, Korea

</div>

<div>Micro RNAs &#40;miRNAs&#41; are 20 to 22&#45;nucleotide&#45;long RNAs that regulate the expression of a large number of genes by mRNA cleavage or translational repression of target mRNAs. Although the importance of miRNAs in plants is becoming more evident, our understanding on the importance and function of miRNAs in the growth, development, and stress responses of plants are severely limited. To better understand the functions of miRNAs in the responses of plants to various stresses, we examined the transgenic Arabidopsis plants that overexpress miR393, miR397, or miR406 under the control of cauliflower mosaic virus 35S promoter. Under normal growth conditions, no significant changes in growth pattern were observed between the genotypes. However, the 35S::miR393 and 35S::miR397 transgenic plants displayed promoted germination compared with the wild&#45;type plants under glucose or abscisic acid treatment. The 35S::miR406 plants survived better than the wild&#45;type plants under salt stress conditions. The expression of putative target genes was analyzed in the wild&#45;type and transgenic plants. The biogenesis and functions of natural miRNAs and artificial miRNAs were compared. These results suggest that these miRNAs play roles in the responses of plants to various environmental stimuli. &#91;Supported by KRF&#93;</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional characterization of DEAD&#45;box RNA helicases in Arabidopsis thaliana under various abiotic stress conditions</div>

<div>

	<span style="text-decoration: underline" >
	Kyung Ae Kim</span>
	<sup>1</sup>, 

	<span  >
	Chul Min Park</span>
	<sup>1</sup>, 

	<span  >
	Hunseung Kang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Chonnam National University, Gwangju, Korea

</div>

<div>Although DEAD&#45;box RNA helicases have been implicated to play roles during stress adaptation processes, the functions of RNA helicases in the stress responses of plants are poorly understood. To understand the functions of DEAD&#45;box RNA helcases in Arabidopsis thaliana in responses to environmental stimuli, the stress&#45;related expression patterns, nucleic acid&#45;binding properties, and functional roles of the three RNA helicases, AtRH9, AtRH14, and AtRH25, were investigated. The transcript levels of the three RNA helicases were up regulated in response to cold stress, whereas their transcript levels were down regulated by salt or drought stress. The RNA helicases have different nucleic acid&#45;binding specificity, and contain an RNA helicase activity in vitro. Phenotypic analysis of the transgenic plants revealed that constitutive overexpression of AtRH9, AtRH14, or AtRH25 resulted in retarded germination compared with the wild&#45;type plants under salt or dehydration stress. AtRH25 contributed to enhance freezing tolerance in Arabidopsis plants. In situ poly&#40;A&#41; hybridization analysis indicated that the export of mRNAs was impaired in the loss&#45;of&#45;function mutants of RNA helicases at low temperatures. AtRH14 and AtRH25 complemented the cold sensitivity of BX04 mutant E. coli cells, had a transcriptional anti&#45;termination activity in RL211 E. coli cells, and contained nucleic acid&#45;melting activity in vitro, demonstrating that AtRH14 and AtRH25 exhibit an RNA chaperone activity during the cold adaptation process. The present results demonstrate that AtRH9, AtRH14, and AtRH25 impact on germination and growth of Arabidopsis plants under various stress conditions, and strongly imply that AtRH14 and AtRH25 exhibit an RNA chaperone activity during the cold adaptation process. &#91;Supported by KOSEF&#93;</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cold shock domain proteins and zinc finger&#45;containing RNA&#45;binding proteins play roles as an RNA chaperon during the cold adaptation process</div>

<div>

	<span style="text-decoration: underline" >
	Won Yong Kim</span>
	<sup>1</sup>, 

	<span  >
	Tae Rin Oh</span>
	<sup>1</sup>, 

	<span  >
	Su Jung Park</span>
	<sup>1</sup>, 

	<span  >
	Yeon Ok Kim</span>
	<sup>1</sup>, 

	<span  >
	Hunseung Kang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Chonnam National University, Gwangju, Korea

</div>

<div>Despite the fact that cold shock domain proteins &#40;CSDPs&#41; and zinc finger&#45;containing RNA&#45;binding proteins &#40;GRPs&#41; have been implicated to play a role during the cold adaptation process, their importance and function in plants are largely unknown. To get better insight into the functional roles of CSDPs and GRPs in the cold response, four CSDPs &#40;CSDP1 to CSDP4&#41; and three GRPs &#40;AtRZ&#45;1a to 1c&#41; from Arabidopsis thaliana were investigated. AtRZ&#45;1a confers cold and freezing tolerance in Arabidopsis plants. Heterologous expression of CSDP1, CSDP3, AtRZ&#45;1a, or AtRZ&#45;1b complemented the cold sensitivity of BX04 mutant Escherichia coli, By contrast, CSDP2, CSDP4, and AtRZ&#45;1c had very weak ability. The CSDPs and GRPs that showed complementation ability at low temperatures had DNA melting activity, RNA melting activity, and transcription anti&#45;termination activity. By contrast, CSDP2, CSDP4, and AtRZ&#45;1c did not have these activities. To understand the importance of the N&#45;terminal RRM and CSD, and the C&#45;terminal glycine&#45;rich region containing CCHC&#45;type zinc fingers, a domain swapping experiment between the N&#45;terminal and C&#45;terminal domains was performed, and the RNA chaperone activity of the chimeric proteins was analyzed. Together, these results strongly imply that CSDPs and GRPs exhibit an RNA chaperone activity during the cold adaptation process. &#91;Supported by KOSEF and KRF&#93;</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Genetic Basis of Singlet Oxygen&#45;Mediated Signaling of Stress Responses</div>

<div>

	<span style="text-decoration: underline" >
	Klaus Apel</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ETH-Zurich Switzerland

</div>

<div>The evolution of aerobic metabolic processes such as respiration and photosynthesis unavoidably leads to the production of reactive oxygen species &#40;ROS&#41; in mitochondria, chloroplasts and peroxisomes. A common feature among the different ROS types is their capacity to cause oxidative damage by inactivating e.g. proteins, nucleic acids and lipids. These cytotoxic properties explain the evolution of complex arrays of ROS scavengers. In plants chloroplasts and peroxisomes are the major sites of ROS production. Various abiotic stress conditions may limit the ability of a plant to use light energy for photosynthesis. Under such stress conditions hyper&#45;reduction of the photosynthetic electron transport chain and photoinhibition of photosynthesis may occur even at moderate light intensities, often causing damages that have been interpreted as unavoidable consequences of injuries inflicted upon plants by toxic levels of ROS. However, this paradigm needs to be modified. Stress responses triggered by ROS are not only due to physicochemical damages but may also be caused by the activation of genetically determined stress response programs. We&#8217;ll present results of our work showing that ROS may act as signals during stress whose specificities seem to depend on the chemical identity of a given ROS and its intracellular site of generation.</div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">endodermal&#45;amyloplast less 1</span> is a novel allele of <span style="font-style: italic">SHORT&#45;ROOT</span></div>

<div>

	<span style="text-decoration: underline" >
	Miyo T. Morita</span>
	<sup>1</sup>, 

	<span  >
	Moritaka Nakamura</span>
	<sup>1</sup>, 

	<span  >
	Chieko Saito</span>
	<sup>2</sup>, 

	<span  >
	Masao Tasaka</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Nara Institute of Science and Technology, Ikoma, Japan, 

	<sup>2</sup>RIKEN, Wako, Japan

</div>

<div><span style="font-style: italic">endodermal amyloplastless 1 &#40;eal1&#41;</span> is a unique mutant that completely lacks gravitropism in inflorescence stems and exhibits reduced gravitropism in hypocotyls, whereas its roots showed normal gravitropism. Previously, it was suggested that differentiation or development of amyloplasts in shoot statocytes &#40;endodermal cells&#41; is affected by the <span style="font-style: italic">eal1</span> mutation. Here, we have identified <span style="font-style: italic">EAL1</span> as a <span style="font-style: italic">SHORT&#45;ROOT &#40;SHR&#41;</span> allele based on map position. Three nucleotides in the <span style="font-style: italic">SHR</span> coding region were deleted in the <span style="font-style: italic">eal1</span> mutant, resulting in the deletion of just one amino acid. The protein encoded by the novel allele of <span style="font-style: italic">SHR</span> appears to have retained its function as a transcription factor since the endodermal cell layer was formed both in roots and in shoots of <span style="font-style: italic">eal1</span>. <span style="font-style: italic">SCARECROW &#40;SCR&#41;</span> promoter activity monitored by reporter protein expression was significantly decreased in <span style="font-style: italic">eal1</span>, suggesting that the activity of SHR lacking one amino acid is reduced. In addition, transcription level of <span style="font-style: italic">SHOOT GRAVITROPISM 5 &#40;SGR5&#41;</span>, which is mainly expressed in the endodermis of inflorescence stems, was markedly decreased.  Together with the presence of abnormal endodermal amyloplasts in <span style="font-style: italic">eal1</span>, these results strongly suggest that the endodermis observed in <span style="font-style: italic">eal1</span> is not sufficiently differentiated to execute shoot gravitropism.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis <span style="font-style: italic">abi2&#45;1</span> Mutant Demonstrates an Enhanced Beta&#45;Aminobutyric Acid&#45;Mediated Acquired Thermotolerance and Root Growth Inhibition</div>

<div>

	<span style="text-decoration: underline" >
	Laurent Zimmerli</span>
	<sup>1</sup>, 

	<span  >
	Bi-Huei Hou</span>
	<sup>2</sup>, 

	<span  >
	Chia-Hong Tsai</span>
	<sup>1</sup>, 

	<span  >
	Gabor Jakab</span>
	<sup>3</sup>, 

	<span  >
	Brigitte Mauch-Mani</span>
	<sup>4</sup>, 

	<span  >
	Shauna Somerville</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Department of Life Science, Institute of Plant Biology, National Taiwan University, Taipei, Taiwan., 

	<sup>2</sup>Department of Plant Biology, Carnegie Institute, Stanford, CA 94305, U.S.A., 

	<sup>3</sup>Department of Plant Physiology, Institute of Biology, University of Pecs, H-7601 Pecs, P. O. Box 266, Hungary., 

	<sup>4</sup>Department of Science, Laboratory of Molecular and Cellular Biology, University of Neuch&#226;tel, 2009 Neuch&#226;tel, Switzerland.

</div>

<div>The non&#45;protein amino acid beta&#45;aminobutyric acid &#40;BABA&#41; primes Arabidopsis plants to respond more quickly and strongly to pathogen and osmotic stress. Here, we report that BABA also significantly enhanced acquired, but not basal, thermotolerance in Arabidopsis. An Arabidopsis mutant deficient in heat shock protein 101, a protein essential for thermotolerance, did not demonstrate a BABA&#45;enhanced thermotolerance. Microarrays were used to analyze the global gene expression patterns of water&#45; and BABA&#45;treated Arabidopsis. BABA&#45;treated plants demonstrated elevated transcript levels for several transcription factors and DNA binding proteins regulating abscisic acid &#40;ABA&#41; and ethylene signalings. To test the role of ABA signaling in BABA mode of action, we evaluated the BABA response of ABA insensitive mutants <span style="font-style: italic">abi1&#45;1</span> and <span style="font-style: italic">abi2&#45;1</span>. BABA&#45;enhanced thermotolerance was partially compromised in <span style="font-style: italic">abi1&#45;1</span>, but was augmented in <span style="font-style: italic">abi2&#45;1</span>. Like ABA, BABA inhibited root growth in wild type and the level of inhibition was roughly additive in roots treated with both compounds. Root growth of both <span style="font-style: italic">abi1&#45;1</span> and <span style="font-style: italic">abi2&#45;1</span> was also inhibited by BABA, with <span style="font-style: italic">abi2&#45;1</span> roots being highly sensitive. Unexpectedly, <span style="font-style: italic">abi1&#45;1</span> and <span style="font-style: italic">abi2&#45;1</span> root growth was inhibited more strongly by combined ABA and BABA treatments than by BABA alone. Our results together with previously published data suggest that BABA is a general enhancer of plant stress resistance. In addition, our data illustrate cross&#45;talk between BABA and ABA signaling cascades. Specifically, the BABA&#45;mediated accumulation of ABA transcription factors without concomitant activation of a downstream ABA response could represent one component of the BABA&#45;primed state in Arabidopsis.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular mechanism of chloroplast photorelocation movement</div>

<div>

	<span style="text-decoration: underline" >
	Masamitsu Wada</span>
	<sup>1</sup>, 

	<span  >
	Noriyuki Suetsugu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute for Basic Biology, Okazaki, Japan

</div>

<div>Chloroplast photorelocation movement is essential for sessile plants. Chloroplasts gather at an area irradiated with weak light to maximize photosynthesis &#40;accumulation response&#41;. They move away from an area irradiated with strong light to minimize damage of the photosynthetic apparatus &#40;avoidance response&#41;. The process of these chloroplast movements can be divided into three parts; photoperception, signal transduction, and the movement of chloroplasts.  The mechanisms are not yet fully understood, but we present&#12288;in this symposium new findings on the parts of photoperception and chloroplasts movement. <br/>&#12288;&#12288;Phototropins &#40;phot1 and phot2&#41; are membrane&#45;associated blue light photoreceptors that function not only in chloroplast relocation, but also in phototropism, stomatal opening, and leaf flattening in Arabidopsis. The molecular structure of phototropin is well conserved for the distinct biochemical roles; two LOV domains in its N&#45;terminal half as a photosensory domain and a Ser&#47;Thr kinase domain in its C&#45;terminal half as a signal transducer. Here, we studied the structural roles of the N&#45;terminus of phot2 &#40;P2N&#41; in vivo and in vitro. Immunoprecipitation and size exclusion chromatography suggested that both phot1 and phot2 exist as multimeric forms. Domain analysis further suggested that the predicted &#945;&#45;helix &#40;from Thr240 to Arg280&#41; was necessary to phot2 tetramer formation. <br/>    For the movement of chloroplasts we found the involvement of short actin filaments on chloroplasts &#40;cp&#45;actins&#41;. Phototropins regulate the localization and the amount of cp&#45;actins to determine the direction and the motility of moving chloroplasts. CHUP1 and newly identified KAC are necessary for cp&#45;actin generation or maintenance. Because, the lack of cp&#45;actin in <span style="font-style: italic">chup1</span> and <span style="font-style: italic">kac</span> mutants chloroplasts moved rapidly via cytoplasmic streaming and cannot change their position in response to light irradiation. Furthermore, <span style="font-style: italic">jac1</span> mutant deficient in the accumulation response was also impaired in cp&#45;actin regulation under the strong blue light, resulting in aberrant avoidance movement. Thus, phototropins and JAC1 mediate chloroplast photorelocation movement via the regulation of cp&#45;actin dependent on CHUP1 and KAC.<br/><br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Iron Deficiency in Arabidopsis Roots: High&#45;resolution Transcriptional Profiling to Decipher Regulatory Networks</div>

<div>

	<span style="text-decoration: underline" >
	Terri Long</span>
	<sup>1</sup>, 

	<span  >
	Jean Wang</span>
	<sup>1</sup>, 

	<span  >
	Joel Burill</span>
	<sup>1</sup>, 

	<span  >
	Philip Benfey</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Duke University

</div>

<div>Plant roots undergo a host of developmental alterations in response to environmental perturbations. When <span style="font-style: italic">Arabidopsis thaliana</span> is exposed to iron deficient conditions &#40;&#45;Fe&#41;, we find that roots and root hairs respond by growing in a wavy growth pattern. Our objectives are to determine the spatio&#45;temporal global transcriptional responses to &#45;Fe, to characterize how these responses may lead to developmental alterations, and explore how root transcriptional regulatory networks control these responses. &#13;<br/>&#13;<br/>As alterations in root development can be seen within the first 24 hours of exposure to &#45;Fe, we performed time&#45;course microarray analysis at 0, 3, 6, 12, and 24 hours exposure to &#45;Fe to determine how the transcriptome responds to &#45;Fe over time. We found that over 600 genes are differentially expressed during this time and that these genes are most responsive at 24 hours &#45;Fe. To determine the spatial responses to &#45;Fe, we have performed microarray analysis of 4 major longitudinal growth stages of the root after 24 hours &#45;Fe, and found that two oldest stages of growth are the most responsive. We have also performed microarray analysis of 6 of the major radial zones of the root at 24 hours &#45;Fe, and found distinct patterns of gene expression within specific cell&#45;types. &#13;<br/>&#13;<br/>Results will be presented on the transcriptome&#8217;s spatio&#45;temporal responses to &#45;Fe, with emphasis on patterns of gene expression, enriched <span style="font-style: italic">cis</span> elements and, particularly, transcription factor &#40;TF&#41; responses. Based on our microarray analyses we have selected TFs that respond to &#45;Fe, which may play roles in iron sensing, for further characterization. We will present results on how several TF insertion mutants respond to &#45;Fe, and whether we have uncovered new key regulators of the &#45;Fe response. Future studies include determining how possible regulators of the &#45;Fe response control expression of downstream targets.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Analysis of transcriptome for <span style="font-style: italic">Deschampsia Antarctica</span> Desv. from King George island in the Antarctic.</div>

<div>

	<span style="text-decoration: underline" >
	Hyoungseok Lee</span>
	<sup>1</sup>, 

	<span  >
	Hyun Hee Cho</span>
	<sup>1</sup>, 

	<span  >
	Yoo Kyung Lee</span>
	<sup>1</sup>, 

	<span  >
	Il-Chan Kim</span>
	<sup>1</sup>, 

	<span  >
	Joung Han Yim</span>
	<sup>1</sup>, 

	<span  >
	Hong Kum Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Polar BioCenter, Korea polar Research Institute (KOPRI), Song Do Techno Park, Incheon, 406-840 Korea

</div>

<div><span style="font-style: italic">Deschampsia anatarctica</span> Desv. is the only monocot that thrives in the rough conditions of the Antarctic region and represents an invaluable resource for identifying genes associated with tolerance against various environmental pressures. In order to identify genes differentially regulated between greenhouse&#45;grown and Antarctic field&#45;grown plants, we have initiated a detailed analysis of its gene expression. Two different cDNA libraries were constructed with these plants. A total of 2,112 cDNA clones were sequenced and grouped into 1,199 non&#45;redundant &#40;nr&#41; ESTs consisting of 242 consensus and 957 singleton sequences. Using similarity searches against several public databases we constructed a functional classification of the ESTs various categories such as stress&#45;responsive genes, genes related to carbon and nitrogen metabolisms. Real&#45;time PCR analysis of <span style="font-style: italic">DaWRAB1</span>, <span style="font-style: italic">DaCOR413</span>, <span style="font-style: italic">DaCOR2</span>, and <span style="font-style: italic">DaCORU</span>, showed a distinct regulation pattern in different environments indicating that they are involved in responses for specific environmental factors. The significance of these results during the cold&#45;acclimation process will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional analysis of Arabidopsis transcriptional factor <span style="font-style: italic">AtHsfA6a</span> in heat stress tolerance&#13;<br/></div>

<div>

	<span  >
	Xu Chao</span>
	<sup>1,2</sup>, 

	<span  >
	Dong-Ping Wang</span>
	<sup>1</sup>, 

	<span  >
	Xia Zhang</span>
	<sup>1</sup>, 

	<span  >
	Zhen-Hui Gong</span>
	<sup>2</sup>, 

	<span  >
	Yun-liu Fan</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Jun Zhao</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Biotechnology Research Institute , National Key Facility for Crop Gene Resources and Genetic Improvement ,Chinese Academy of Agricultural Sciences ,Beijing 100081, China., 

	<sup>2</sup>College of Horticulture, Northwest Agricultual and Foresty University, Yangling 712100, China;

</div>

<div>To dissect the molecular mechanism underling plant tolerance to heat, a genomic fragment containing the ORF of putative Arabidopsis heat shock transcriptional factor <span style="font-style: italic">AtHsfA6a</span> was cloned by PCR amplification, and homozygous transgenic Arabidopsis lines containing either over&#45;expression &#40;OE&#41; or antisense &#40;AS&#41; constructs of <span style="font-style: italic">AtHsfA6a</span> were generated through Agrobacterium&#45;mediated genetic transformation. The performance of OE, AS and wild type &#40;WT&#41; plants under heat stress was analyzed. The results showed that the survival rate of OE plants &#40;86&#65285;&#41; was much higher than that of WT &#40;59&#65285;&#41; after treatment under 43&#8451; for 2 hours, in contrast to that of AS plants of which only 48&#37; survived. Measurement of electric conductivity revealed that the level of electrolyte leakage after treated under 43&#8451; for half an hour was significantly reduced in OE plants comparing to that in WT, whereas that in AS plants was largely increased. Gene expression analysis indicated that <span style="font-style: italic">AtHsfA6a</span> was induced by heat stress and <span style="font-style: italic">Hsp70</span> was the downstream target up&#45;regulated by <span style="font-style: italic">AtHsfA6a</span>. These data suggests that <span style="font-style: italic">AtHsfA6a</span> functions to enhance heat stress tolerance through activating the expression of <span style="font-style: italic">Hsp70</span>.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cytochrome P450s as reporters for circadian&#45;regulated pathways</div>

<div>

	<span style="text-decoration: underline" >
	Yinghong Pan</span>
	<sup>1</sup>, 

	<span  >
	Matthew E. Hudson</span>
	<sup>1</sup>, 

	<span  >
	Todd Michael</span>
	<sup>2</sup>, 

	<span  >
	Joanne Chory</span>
	<sup>2</sup>, 

	<span  >
	Mary A. Schuler</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Illinois, Urbana, Illinois, USA, 

	<sup>2</sup>The Salk Institute, La Jolla, California, USA

</div>

<div>          Cytochrome P450s play important roles in the synthesis of diverse secondary compounds in 'Arabidopsis thaliana'. Comparing four datasets that included seedlings harvested over a 2&#45;day period of constant conditions after growth with varying photo and thermocycles, a total of 98 P450 loci were circadian&#45;regulated for at least one of the four different conditions. Here, we further describe the different circadian&#45;regulated pathways using, as reporters, individual P450 loci that are likely to be rate&#45;limiting in secondary metabolic pathways. Using RT&#45;PCR analysis, we have confirmed circadian regulation for P450s involved in phenylpropanoid, carotenoid, oxylipin, glucosinolate and brassinosteroid biosynthesis. Using bioinformatics approaches, we have identified conserved motifs in the promoters of different circadian&#45;regulated pathways by defining their over&#45;representation compared to all promoters in the 'Arabidopsis genome'. Analysis of these promoters will provide us with the opportunity to functionally characterize novel promoter motifs that might be important in circadian regulation of cytochrome P450s.</div>

</div>





<div class="abs">

<div style="font-weight:bold">RCI1A negatively regulates constitutive freezing tolerance and cold acclimation in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Julio Salinas</span>
	<sup>1,2</sup>, 

	<span  >
	Rosa L&#243;pez-Cobollo</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Departamento de Biotecnolog&#237;a, INIA, Madrid, Spain, 

	<sup>2</sup>Departamento de Biolog&#237;a de Plantas, CIB-CSIC, Madrid, Spain

</div>

<div>Low temperatures constitute one of the most influential environmental factors on plant development and survival. In the case of cultivated species, they also limit their growing season and yield, originating important economical losses. Plants exhibit a large diversity of responses to low temperatures, and in some cases have evolved adaptive mechanisms allowing them to survive to low &#40;0&#45;10&#186;C&#41; and freezing &#40;&#60; 0&#186;C&#41; temperatures. The central element in the adaptation to freezing temperatures is the process of cold acclimation, an adaptive response by means of which many plants from temperate regions are able to increase their freezing tolerance after being exposed some days to low&#45;nonfreezing temperatures. Understanding the molecular mechanisms that control cold acclimation is important from both basic and practical points of view. <span style="font-style: italic">RCI1A</span> is an Arabidopsis gene whose expression is induced in response to low temperature and encodes a 14&#45;3&#45;3 protein. We have identified a mutant, <span style="font-style: italic">rci1a&#45;1</span>, which does not show cold induction of <span style="font-style: italic">RCI1A</span>. The characterization of <span style="font-style: italic">rci1a&#45;1</span> revealed that it has increased capacity to tolerate freezing before and after cold acclimation, indicating that RCI1A negatively regulates constitutive freezing tolerance and cold acclimation response. In order to understand these phenotypes at molecular level we are analyzing the transcriptome of the <span style="font-style: italic">rci1a&#45;1</span> mutant under both control and cold conditions. On the basis of these results, the role of RCI1A in freezing tolerance and cold acclimation will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">STH2, a B&#45;box protein in Arabidopsis, activates transcription and positively regulates light mediated development.</div>

<div>

	<span style="text-decoration: underline" >
	Sourav Datta</span>
	<sup>1</sup>, 

	<span  >
	Chamari Hettiarachchi</span>
	<sup>2</sup>, 

	<span  >
	Henrik Johansson</span>
	<sup>1</sup>, 

	<span  >
	Verena Lorenz</span>
	<sup>1</sup>, 

	<span  >
	Magnus Holm</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>CMB-Molecular Biology, Gothenburg University, Sweden, 

	<sup>2</sup>Department of Chemisrty,Colombo University, Sri Lanka

</div>

<div>COP1 and HY5 are two major regulators of light signaling in plants. Proteins interacting with either could therefore be important regulators of light dependent development. Using COP1 or HY5 as bait in yeast two&#45;hybrid screens several putative regulators of light signaling, all containing B&#45;boxes, were identified. One of them STH2 was found to interact with HY5. Mapping studies confirmed that the bZIP domain of HY5 and the B&#45;boxes in STH2 are important for this interaction. STH2 and HY5 when co&#45;expressed in onion epidermal cells show FRET. Furthermore, although STH2 is uniformly nuclear by itself, it localizes to speckles when co&#45;expressed with unfused COP1.We identified two independent T&#45;DNA insertion lines in STH2. Northern blots confirmed that both sth2&#45;1 and sth2&#45;2 are null alleles. Both alleles are hyposensitive to red and blue light. Double mutants between sth2 and hy5 show an additive effect on hypocotyl elongation in light whereas sth2 partially suppresses the hypocotyl phenotype of dark grown cop1 alleles. The sth2 mutant, like hy5, shows enhanced number of lateral roots and accumulates less anthocyanin. Furthermore, it can suppress the reduced number of lateral roots and high anthocyanin levels of the cop1 alleles. Interestingly we found that STH2 can activate transcription. Transient transfection assays in protoplasts using LUC reporter driven by the Chalcone Isomerase promoter show that the B&#45;boxes in STH2 and a functional G&#45;Box element in the promoter are required for this activity. In conclusion we have identified STH2, a B&#45;box protein in Arabidopsis, as a positive regulator of photomorphogeneis and for the first time report that the B&#45;box domain plays a direct role in activating transcription in plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">PHOSPHATE STARVATION SIGNALLING IN ARABIDOPSIS</div>

<div>

	<span  >
	Regla Bustos</span>
	<sup>1</sup>, 

	<span  >
	Jsose Manuel Franco-Zorrilla</span>
	<sup>1</sup>, 

	<span  >
	Pablo Catarecha</span>
	<sup>1</sup>, 

	<span  >
	Gabriel Castrillo</span>
	<sup>1</sup>, 

	<span  >
	Isabel Mateos</span>
	<sup>1</sup>, 

	<span  >
	Mabel Puga</span>
	<sup>1</sup>, 

	<span  >
	Vicente Rubio</span>
	<sup>1</sup>, 

	<span  >
	Antonio Leyva</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Javier Paz-Ares</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Centro Nacional de Biotecnologia-CSIC, Madrid, Spain

</div>

<div>Plants have evolved adaptive responses to cope with growth under phosphate &#40;Pi&#41; limiting conditions. This rescue system is under the control of a highly elaborated regulatory mechanism, which, besides Pi, is modulated by sugars, cytokinins and other unknown signal&#40;s&#41; mediating long distance systemic repression. We have found that transcription of Pi starvation&#45;induced genes is repressed by arsenate, potentially reflecting the existence of a savage system to protect plants from arsenate, particularly in Pi poor soils. One component of the Pi starvation regulatory system is transcription factor PHR1, which recognises an imperfect palindromic motif &#40;GNATATNC, P1BS&#41; that is over&#45;represented in the promoter region of Pi starvation induced genes. We have shown that PHR1&#47;P1BS acts as key integrator in Pi starvation signalling. Thus, a minimal promoter containing a multimerised P1BS motif is specifically induced by Pi starvation and is responsive to the stimulatory effect of sugars and the inhibitory effects of cytokinins, long distance repression signals and arsenate. Control of Pi starvation responses also involves the participation of Pi starvation responsive miRNAs &#40;miR399&#41; and other noncoding RNAs, namely the IPS1 family members. Functional characterisation of IPS1 has disclosed a novel mechanism of inhibition of miRNA activity, in which a non&#45;coding RNA sequesters a miRNA and reduces its effective levels. We coin the term target mimicry to define this riboregulatory mechanism of miRNA activity&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Toward physiological function of AtFtsH4&#45;mitochondrial ATP dependent metalloprotease from <span style="font-style: italic">Arabidopsis</span>: mutation in the <span style="font-style: italic">AtFtsH4</span> gene leads to altered morphology, elevated level of ROS and accumulation of oxidized proteins.</div>

<div>

	<span  >
	Marta Gibala</span>
	<sup>1</sup>, 

	<span  >
	Wataru Sakamoto</span>
	<sup>2</sup>, 

	<span  >
	Alicja Antonczak</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Hanna Janska</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biotechnology, University of Wroclaw, Przybyszewskiego 63/77, Wroclaw, Poland, 

	<sup>2</sup>Research Institute for Bioresources, Okayama University, Kurashiki, Japan

</div>

<div>AtFtsH4 is one out of four <span style="font-style: italic">Arabidopsis</span> mitochondrial ATP&#45;dependent metalloproteases, known to possess proteolytic and chaperone&#45;like activities. AtFtsH4 is anchored to the inner mitochondrial membrane with the catalytic site exposed to the intermembrane space. Here we demonstrate morphological and molecular consequences of AtFtsH4 loss by studying two independent homozygous lines with T&#45;DNA insertion in the <span style="font-style: italic">AtFtsH4</span> gene. &#13;<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Comparative studies of the wild type and <span style="font-style: italic">ftsh4</span> mutant growth were performed under short &#40;SD&#41; and long day &#40;LD&#41; conditions at 22&#176;C or 30&#176;C. We examined phenotypic changes at different stages: germination, early seedling growth on agar plates and soil&#45;based growth over the entire life of plants. A lack of serious defects was observed under LD conditions at 22&#176;C. However, phenotypic differences were observed under SD photoperiod at normal temperature as well as during continual moderate heat stress &#40;30&#176;C&#41; under LD conditions. Interesting, most of the striking phenotypic changes &#40;asymmetric leaves, late bolting, shorter inflorescence&#41; were similar between both treatments and were detected at the same time of development: at the end of vegetative phase and during flowering. We found a correlation between morphological abnormalities of rosette and cauline leaves and the elevated level of reactive oxygen species. Furthermore, we noticed a substantially higher amount of two mitochondrial membrane proteins &#40;prohibitin and porin&#41; in the mutant compared with the wild type. These proteins are known to be targets of reactive oxygen species in plant mitochondria. We are now testing hypothesis that AtFtsH4 protease is involved in degradation of oxidatively damaged proteins prior to bolting. Taken together, our data suggest that under LD conditions AtFtsH4 is required for a normal growth and development at temperature only slightly above the optimal growth for <span style="font-style: italic">Arabidopsis</span>.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Isolation and Characterization of an Arabidopsis Temperature&#45;dependent Mutant lls</div>

<div>

	<span style="text-decoration: underline" >
	Xiaofang Li</span>
	<sup>1</sup>, 

	<span  >
	Xinghua Shao</span>
	<sup>1</sup>, 

	<span  >
	Bin Yao</span>
	<sup>1</sup>, 

	<span  >
	Yue Sun</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The School of Life Science, East China Normal University

</div>

<div>It is well known that temperature controls a number of aspects of plant development. However, the mechanism of which is still unknown. We isolated a long life span &#40;lls&#41; mutant from Arabidopsis thaliana L. through a T&#45;DNA tagging approach. The phenotype of lls plant includes loss of apical dominance, reduced plant size and dwarfism, dark&#45;green  and curved leaves,  severe late flowering, growing slowly  and long life span at 22&#45;26&#186;C. However, these phenotypes lost at 28&#186;C or higher than 28&#186;C, suggesting that the lls mutant is temperature dependent. Many mutant phenotypes at room temperature indicate that lls may be involved in certain phytohormones metabolism, transport or signalling. Using a reporter gene driven by an auxin&#45; responsive promoter, we found that the expression pattern of auxin response element was not altered in lls. The auxin sensitivity and transport did not change either. While the sensitivity of lls plants to cytokinin was reduced. Physiological assay indicated that sugar content in lls was higher than that in wild&#45;type by at least 3 times. These results suggest that the lls may connect with temperature and sugar and phytohormone metabolism, transport or signaling to control plant development. Genetic analysis demonstrates that lls is a recessive mutant and segregates with a T&#45;DNA insertion, which indicates that LLS gene could not be cloned by iPCR approach. We have finely mapped LLS between two markers on chromosome4 by mapped&#45;clone method and the cloning of LLS will provide us more valuable information on how this gene controls plant development related with temperature. </div>

</div>





<div class="abs">

<div style="font-weight:bold">HIGH LEVEL OF ARABIDOPSIS <span style="font-style: italic">EXPANSIN</span> EXPRESSION IN TRANSGENIC PLANTS CAUSES ENHANCED HORMONE OR SALT STRESS SENSITIVITY</div>

<div>

	<span style="text-decoration: underline" >
	YeRim Kwon</span>
	<sup>1</sup>, 

	<span  >
	Hee-Jin Lee</span>
	<sup>2</sup>, 

	<span  >
	Eun-Ji Koh</span>
	<sup>1</sup>, 

	<span  >
	Jee Eun Oh</span>
	<sup>1</sup>, 

	<span  >
	Kyoung-Heon Kim</span>
	<sup>2</sup>, 

	<span  >
	Suk-Whan Hong</span>
	<sup>3</sup>, 

	<span  >
	Hojoung Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Division of Biotechnology, College of Life Sciences and Biotechnology, Korea University, Seoul, Republic of Korea., 

	<sup>2</sup>Division of Food Science, College of Life Sciences and Biotechnology, Korea University, Seoul, Republic of Korea., 

	<sup>3</sup>Division of Applied Plant Science, College of Agriculture and Life Sciences, Chonnam National University, Gwangju, Korea.

</div>

<div>  Cellulose is the most abundant and renewable natural resource which can be obtained from less costly producible plant lignocellulosic materials. A number of wall&#45;modifying proteins were discovered to have crucial roles in the molecular basis of cell wall expansion. Expansin, one of cell wall loosening proteins, appears to permit the microfibril matrix network to slide in growing plant cell walls, thereby enabling the wall to expand. Hence, as one strategy of improving cellulase performance leading to an increase in sugar yields, we intended to express expansins in various protein expression systems such as <span style="font-style: italic">E.coli</span> or yeast. Although we obtained a fare amount of expansins in these systems, there was no activity detected in terms of cell wall extension. This led us to establish plant expression systems for the production of higher amount of expansins. However, expansins are involved in the cell expansion process, it may cause harmful defects in plant growth and development when tried to over&#45;express these proteins <span style="font-style: italic">in planta</span> Hence, to scrutinize possible impacts of expansin over&#45;expression in plant growth and development, we generated two different types of transgenic plants constitutively expressing Arabidopsis <span style="font-style: italic">EXP3</span> or <span style="font-style: italic">EXP&#45;&#946;1</span> under the control of <span style="font-style: italic">35S&#45;CaMV</span> promoter. Here, we report that transgenic plants showing high level of <span style="font-style: italic">EXP&#45;&#946;1</span> expression exhibit enhanced sensitivity to salt stress. Possible reasons for this phenomenon were explored. &#13;<br/>&#13;<br/>This work was supported by a grant from the Rural Development Administration &#40;Biogreen21; 2007 to Hojoung Lee; grant &#35;2005040103468302&#41;. </div>

</div>





<div class="abs">

<div style="font-weight:bold">The Functional Characterization of Novel Arabidopsis CBS&#45;DOMAIN CONTAINING PROTEIN1 Gene That Acts on Ethylene &#38; Jasmonate Biosynthetic Pathways during Anther Dehiscence, also Is Required for Biotic and Abiotic Stress Responses.</div>

<div>

	<span style="text-decoration: underline" >
	Kwang Wook Jung</span>
	<sup>1</sup>, 

	<span  >
	Kyoung Shin Yoo</span>
	<sup>1</sup>, 

	<span  >
	Mei Hua Cui</span>
	<sup>1</sup>, 

	<span  >
	Yun Young Kim</span>
	<sup>1</sup>, 

	<span  >
	Jeong Sheop Shin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Life Sciences and Biotechnology, Korea University, Seoul 136-701, Korea

</div>

<div>The programmed cell death &#40;PCD&#41; process is an integral part of development and the defense response, but the underlying mechanisms of its executors and regulators remain largely unknown in planta. We report the identification a novel gene, CBS&#45;DOMAIN CONTAINING PROTEIN1 &#40;AtCDCP1&#41;, which is believed to be involved in determining the timing of JA and ET biosynthesis. The loss&#45;of&#45;function mutant of AtCDCP1 had well&#45;developed anther cells and showed an increased accumulation of JA and ET in its flower bud clusters. Conversely, overexpression of AtCDCP1 caused a delay in the development of the floral organs, arrested anther cells on secondary cell&#45;wall biosynthesis in endothecium, and a reduction in the accumulation of JA and ET in the flower, resulting in male sterility through anther indehiscence. Moreover, those overexpression lines are much more tolerant to salt and mannitol stresses. Based on these data, we are trying to characterize the biological roles of novel genes in defense system or developmental process, including regulatory mechanism as well as the effects of gene itself. Subsequently, these data will be applicable to the molecular breeding of cultivated rice.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional characterization of At398 &#40;unknown protein&#41; which is homology to Om398, that Induced by Mechanical Wounding Based on cDNA Microarray Data in Wild Rice</div>

<div>

	<span style="text-decoration: underline" >
	Mei Hua Cui</span>
	<sup>1</sup>, 

	<span  >
	Yun Young Kim</span>
	<sup>1</sup>, 

	<span  >
	Kwang Wook Jung</span>
	<sup>1</sup>, 

	<span  >
	Kyoung Shin Yoo</span>
	<sup>1</sup>, 

	<span  >
	Jeong Sheop Shin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Graduate School of Life Sciences and Biotechnology, Korea University, Seoul 136-701, Korea

</div>

<div>In this study, O. minuta &#40;BBCC&#41; of wild rice species, which has been known to have useful resistance&#45;related genes against brown planthopper &#40;BPH&#41;, bacterial blight and green leafhopper &#40;GLH&#41;, etc, was used to screen and identify the differentially expressed genes induced by insect feeding and wounding. And we found Om398 was strongly induced by wounding and insect feeding compared to controls. Om398 protein showed sequence homology to At398 in Arabidopsis. And these proteins have no reported homology with any known animal protein. To identify Om398 functions, first we focus on the At398 in Arabidopsis. Through screening now we have three At398&#45;overexpression lines and one knock out line in Arabidopsis. On germination assays under various stress conditions, At398&#45;overexpressed Arabidopsis plants showed salt, ABA, mannitol and glucose stress tolerance. With the ProAt398:GUS lines, we identified that At398 specially expressed in the senescence leaves and open flowers. Based on these data, we have determined the biological functions in defense system, including regulatory mechanism as well as the effects of gene itself. Moreover, we will define the components of regulatory networks that mediate defense signaling using the yeast hybrid system. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of GIGANTEA expression patterns in the nucleus</div>

<div>

	<span style="text-decoration: underline" >
	Yumi Kim</span>
	<sup>1</sup>, 

	<span  >
	Jeongsik Kim</span>
	<sup>1</sup>, 

	<span  >
	Miji Yeom</span>
	<sup>1</sup>, 

	<span  >
	Hong Gil Nam</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Division of Molecular Life Sciences and National Core Research Center for Systems Bio-Dynamics, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea, 

	<sup>2</sup>The I-BIO graduate program, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea.

</div>

<div>Circadian regulation in plant is essential for adaptation to environment. <span style="font-style: italic">GIGANTEA</span> &#40;<span style="font-style: italic">GI</span>&#41; was isolated through forward genetics as a circadian regulator. GI, as a nuclear protein, is post&#45;translationally regulated by light&#47;dark via 26S proteasome, and required for maintaining circadian amplitude and proper period length. Several circadian regulators including GI express in the nucleus and make distinctive nuclear bodies. However, little is known about the roles of nuclear bodies in circadian regulation. Here we describe nuclear expression patterns of GI using GI:GFP fusion protein and their regulation. GI has three typical nucleus expression patterns; dispersed in the nucleus, small and numerous nuclear bodies and large and hollow nuclear bodies. Three nucleus expression patterns have distinct portions during a day. Our results suggest that proper nuclear expression of GI might be another mechanism to explain its function in circadian and photoperiodic flowering regulation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">An essential role of FIONA1 for circadian clock function in Arabidopsis.</div>

<div>

	<span style="text-decoration: underline" >
	Jeongsik Kim</span>
	<sup>1</sup>, 

	<span  >
	Yumi Kim</span>
	<sup>1</sup>, 

	<span  >
	Miji Yeom</span>
	<sup>1</sup>, 

	<span  >
	Hong Gil Nam</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Division of Molecular Life Sciences and National Core Research Center for Systems Bio-Dynamics, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea., 

	<sup>2</sup>The I-BIO graduate program, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea.

</div>

<div>Summary&#13;<br/>The circadian clock is a time&#45;keeping mechanism to support and give organisms to prepare the metabolic and physiological processes with daily cycles of light and dark. These daily information processes also determine proper flowering time by recognizing the day&#45;length in higher plants. In order to facilitate to isolate putative clock components, we isolated mutants of the circadian clock by screening early flowering phenotypes. One of these mutants, fiona1 exhibited extremely early flowering both in long day and short day. The fio1 mutation affects the essential photoperiodic genes, CO and FT in Arabidopsis, and also lengthens the period of multiple output rhythms of the circadian clock in constant light free&#45;running condition including leaf movement and expression of various clock controlled genes. The 'FIO1' gene acts independently of light&#45; and temperature&#45;dependent pathway. Furthermore, transient expression of FIO1 gene in LL free&#45;running condition lengthened the period length of wild type plants. Collectively, these results suggest that FIO1 plays an essential role for sustaining the master oscillator activity. Map&#45;based cloning of FIO1 identified a novel nuclear protein which is well conserved in other prokaryotes and eukaryotes, implicating a common and dispensable role for this protein in diverse organisms.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A <span style="font-style: italic">Dunaliella viridis</span> trehalose&#45;6&#45;phosphate synthase  increases abiotic stress tolerance in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Xiangzong Meng</span>
	<sup>1</sup>, 

	<span  >
	Saifan Luo</span>
	<sup>2</sup>, 

	<span  >
	Nan Zhang</span>
	<sup>2</sup>, 

	<span  >
	Zhengkai Xu</span>
	<sup>1,2</sup>, 

	<span  >
	Rentao Song</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Science, Shanghai, China., 

	<sup>2</sup>School of Life Sciences, Shanghai University, Shanghai, China.

</div>

<div>In many organisms, trehalose protects against harsh environmental stresses, such as freezing, drought, heat, and salt, probably by stabilizing protein structures and lipid membranes. The biosynthesis of trehalose consists of two enzymatic steps catalyzed by trehalose&#45;6&#45;phosphate synthase &#40;TPS&#41; and trehalose&#45;6&#45;phosphate phosphatase &#40;TPP&#41;. In this study, a cDNA &#40;designated as <span style="font-style: italic">DvTPS1</span>&#41;, encoding a TPS homologue, was isolated from the unicellular green alga <span style="font-style: italic">Dunaliella viridis</span>, which is highly tolerant to salt and cold. Protein&#45;sequence comparison revealed that DvTPS1 belongs to class&#8545; type of TPS, which has a N&#45;terminal TPS domain and an extended C&#45;termial TPP domain. RT&#45;PCR showed that <span style="font-style: italic">DvTPS1</span> mRNA was constitutively expressed in <span style="font-style: italic">D. viridis</span>. The heterologous expression of <span style="font-style: italic">DvTPS1</span> in the yeast mutant <span style="font-style: italic">ena1</span> suppressed Na&#43; hypersensitivity and demonstrated the salt&#45;resistant function of DvTPS1. Transgenic <span style="font-style: italic">Arabidopsis</span> overexpressing <span style="font-style: italic">DvTPS1</span> under the control of the CaMV 35 S promoter showed increased tolerance to salt, cold and drought stresses, without growth inhibition or visible phenotypic alterations. This study documents a useful strategy for improving abiotic stress tolerance of plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">An <span style="font-style: italic">Arabidopsis</span> mutant <span style="font-style: italic">uvt34</span> exhibits enhanced tolerance to UV&#45;B stress</div>

<div>

	<span style="text-decoration: underline" >
	SeungWon Choi</span>
	<sup>1</sup>, 

	<span  >
	YeRim Kwon</span>
	<sup>1</sup>, 

	<span  >
	Suk-Whan Hong</span>
	<sup>2</sup>, 

	<span  >
	Hojoung Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Division of Biotechnology, College of Life Sciences and Biotechnology, Korea university, Seoul, Republic of Korea, 

	<sup>2</sup>Division of Applied Plant Science, College of Agriculture and Life Sciences, Chonnam National University, Gwangju, Republic of Korea

</div>

<div>One of the severe abiotic stresses encountered by plants is solar radiation which includes UV light. However, plants should cope with this type of stress because of the photosynthetic reaction. It is reported that UV radiation causes harmful effects on cellular metabolisms via damaging DNAs, proteins and lipids, leading to cell death. One of the prime candidates for cellular damage by UV stress includes reactive oxygen species &#40;ROS&#41; which are known to be increased in the cell exposed to UV radiation. Studies with UV sensitive mutants have aided in our understanding on the mechanisms by which plants protect themselves against UV&#45;induced cellular damage. Some mutants have defects in the biosynthetic pathway of phenolics, flavonoids or sinapates, which are known to help plants by serving as sunscreen. Up to date, four <span style="font-style: italic">Arabidopsis</span> mutants have been identified, <span style="font-style: italic">uvt1</span>, <span style="font-style: italic">uvi1</span>, <span style="font-style: italic">rcd1&#45;2</span> and <span style="font-style: italic">uvi4</span> that are tolerant to UV&#45;B. To enhance our understanding in the UV tolerance mechanisms, we have isolated a new UV&#45;B tolerant mutant, <span style="font-style: italic">uvt34</span> &#40;<span style="font-style: italic">UV&#45;B tolerant 34</span>&#41;, of <span style="font-style: italic">Arabidopsis</span> &#40;EMS mutant&#41;. The <span style="font-style: italic">uvt34</span> is a recessive mutant and exhibit tolerance to high UV&#45;B radiation which is lethal level to wild&#45;type plants. Possible reasons for this enhanced tolerance in the <span style="font-style: italic">uvt34</span> were explored. &#13;<br/>&#13;<br/>This work was supported by the Korea Science and Engineering Foundation &#40;KOSEF&#41; grant funded by the Korea government &#40;MOST&#41; &#40;to Hojoung Lee, Grant &#35;R11&#45;2001&#45;092&#45;04004&#45;0&#41;</div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of Physiological Function of <span style="font-style: italic">Arabidopsis Methionine Sulfoxide Reductase 4</span> and Identification of its Signal Transduction Pathway</div>

<div>

	<span style="text-decoration: underline" >
	Jee Eun Oh</span>
	<sup>1</sup>, 

	<span  >
	Eun-Ji Koh</span>
	<sup>1</sup>, 

	<span  >
	YeRim Kwon</span>
	<sup>1</sup>, 

	<span  >
	Ki Deok Kim</span>
	<sup>1</sup>, 

	<span  >
	Suk-Whan Hong</span>
	<sup>2</sup>, 

	<span  >
	Hojoung Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Korea University, Seoul, Republic of Korea, 

	<sup>2</sup>Chonnam National University, Gwangju, Republic of Korea

</div>

<div>Owing to its cellular respiration and metabolic processes in aerobic environments, living organism is susceptible to abundant oxidative stresses, which result in accumulation of damaged proteins, cellular senescence and fundamentally even more catastrophic event, death. To protect themselves against both biotic and abiotic stresses, organisms have developed extensive repair mechanisms for improperly modified proteins. Methionine sulfoxide reductase &#40;MSR&#41; is an enzyme to restore oxidized free and&#47;or bound methionine residues. In <span style="font-style: italic">Arabidopsis</span>, two types of methionine sulfoxide reductase, MsrA and MsrB, and their function in control of oxidized methionine residues are now characterized. In our former study, we depicted molecular regulation of <span style="font-style: italic">MSR4</span> gene induction in abiotic stress conditions. Thus, we generated overexpressors and mutants &#8211; including point mutation and antisense &#8211; to understand the physiological role of <span style="font-style: italic">MSR4</span>. Interestingly we observed formation of large siliques in the overexpressor lines under stress conditions. Moreover, among the transgenic plants, we were able to distinguish lines that are specifically responsive to high salt condition. These were found to have amino acid alterations on the conserved region of MSRs from various organisms. Thus, we suppose this observation may be the result of dominant negative mutation.  We also scrutinized the relationship between the control of <span style="font-style: italic">MSR4</span> and the proteasome&#45;mediated degradation because, in order to maintain &#34;protein homeostasis&#34;, removal of oxidized proteins and rapid turnover of repairing enzymes must be carefully regulated at all times. Fine adjustment of gene expression will be a better, feasible and reliable way to improve stress tolerance of plants and thus to approach to the application of GM crops and phyto&#45;products. &#13;<br/>&#13;<br/>This work was supported by the Korea Science and Engineering Foundation &#40;KOSEF&#41; grant funded by the Korea government &#40;MOST&#41; &#40;to Hojoung Lee, Grant &#35;R11&#45;2001&#45;092&#45;04004&#45;0&#41;</div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular architecture of circadian clock system in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Miji Yeom</span>
	<sup>1</sup>, 

	<span  >
	Jeongsik Kim</span>
	<sup>1</sup>, 

	<span  >
	Yumi Kim</span>
	<sup>1</sup>, 

	<span  >
	Hong Gil Nam</span>
	<sup>1,2</sup>

</div>

<div>

	<sup>1</sup>Division of Molecular Life Sciences and National Core Research Center for Systems Bio-Dynamics, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea, 

	<sup>2</sup>The I-BIO graduate program, POSTECH, Pohang, Kyungbuk 790-784, Republic of Korea.

</div>

<div>Circadian clocks give organisms the ability to anticipate environmental changes that arise due to the rotation of the Earth on its axis. In <span style="font-style: italic">Arabidopsis</span>, many aspects of physiology, metabolism and development are controlled by circadian clock. Also, a large part of the transcriptome shows circadian regulation. The core components of circadian clock such as <span style="font-style: italic">TOC1</span>, <span style="font-style: italic">CCA1</span>,and <span style="font-style: italic">LHY</span> were suggested to make a transcriptional feedback loop. Recently, it is proposed that molecular architecture of circadian clock is composed of well&#45;coordinated multiple loops. However, previous reports using the mutant plants are insufficient to explain the well&#45;coordinated multiple circuits. Therefore, we would introduce the inducible gene expression system which elevates the specific clock gene mRNA at specific circadian time&#40;CT&#41;. We would analyze the response kinetics among the other clock components mRNA expression in time series and find out the closely or weakly regulations in circadian clock system.  And then, we would make mathematical model using these data. Through this approach, we could describe accurate the molecular architecture of circadian clock.  </div>

</div>





<div class="abs">

<div style="font-weight:bold">The barley ERF&#45;type transcription factor HvRAF confers enhanced pathogen resistance and salt tolerance in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span style="text-decoration: underline" >
	Jinwook Jung</span>
	<sup>1</sup>, 

	<span  >
	So Youn Won</span>
	<sup>2</sup>, 

	<span  >
	Seok Cheol Suh</span>
	<sup>3</sup>, 

	<span  >
	Youn Soo Cheong</span>
	<sup>1</sup>, 

	<span  >
	Sang Ryoung Park</span>
	<sup>1</sup>, 

	<span  >
	HyeRan Kim</span>
	<sup>4</sup>, 

	<span  >
	Rod Wing</span>
	<sup>4</sup>, 

	<span  >
	Yeonhwa Jeong</span>
	<sup>1</sup>, 

	<span  >
	Ingyu Hwang</span>
	<sup>1</sup>, 

	<span  >
	Minkyun Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Agrigultrual Biotechnology, Seoul National University, Seoul, Repulic of Korea, 

	<sup>2</sup>GMO Safety Division, National Institute of Agricultural Biotechnology, Rural Development Administration, Suwon, Republic of Korea, 

	<sup>3</sup>Cell and Genetics Division, National Institute of Agricultural Biotechnology, Rural Development Administration, Suwon, Republic of Korea, 

	<sup>4</sup>Arizon Genomics Institute, University of Arizona, Tucson, USA

</div>

<div>We isolated <span style="font-style: italic">HvRAF</span> &#40;<span style="font-style: italic">Hordeum vulgare</span> Root Abundant Factor&#41;, a cDNA encoding a novel ERF&#45;type transcription factor from young seedlings of barley. In addition to the most highly conserved AP2&#47;ERF DNA&#45;binding domain, the encoded protein contained an N&#45;terminal MCGGAIL signature sequence, a putative nuclear localization sequence, and a C&#45;terminal acidic transcription activation domain containing a novel mammalian hemopexin domain signature&#45;like sequence. RNA blot analyses revealed that <span style="font-style: italic">HvRAF</span> transcripts were more abundant in roots than in leaves. <span style="font-style: italic">HvRAF</span> expression was induced in barley seedlings by various treatment regimes such as SA, ethephon, MeJA, cellulase, and methyl viologen. In a subcellular localization assay, the HvRAF&#45;GFP fusion protein was targeted to the nucleus. The fusion protein of HvRAF with the GAL4 DNA&#45;binding domain strongly activated transcription in yeast. Various deletion mutants of HvRAF indicated that the transactivating activity was localized to the acidic domain of the C&#45;terminal region, and that the hemopexin domain signature&#45;like sequence was important for the activity. Overexpression of the C&#45;terminus region of the HvRAF conferred enhanced cadmium tolerance to yeast. Furthermore, overexpression of the HvRAF gene in <span style="font-style: italic">Arabidopsis</span> plants induced the activation of various stress&#45;responsive genes. Furthermore, the transgenic <span style="font-style: italic">Arabidopsis</span> plants showed enhanced resistance to <span style="font-style: italic">Ralstonia solanacearum</span> strain GMI1000, as well as seed germination and root growth tolerance to high salinity. These results collectively indicate that HvRAF is a transcription factor that plays dual regulatory roles in response to biotic and abiotic stresses in plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional Characterization of Stress&#45;Related Putative Caleosin &#40;SRC&#41; that Plays on Biotic and Abiotic Stress Response</div>

<div>

	<span style="text-decoration: underline" >
	Yun Young Kim</span>
	<sup>1</sup>, 

	<span  >
	Mei Hua Cui</span>
	<sup>1</sup>, 

	<span  >
	Kwang Wook Jung</span>
	<sup>1</sup>, 

	<span  >
	Kyoung Shin Yoo</span>
	<sup>1</sup>, 

	<span  >
	Jeong Sheop Shin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>School of Life Sciences and Biotechnology, Korea University, Seoul 136-701, Korea

</div>

<div>With cDNA microarray experiments, we have previously identified a group of genes differentially expressed in wild rice by fungal infection, insect infestation and wound treatment. Based on expression profiling, Northern and Southern analysis, OmSRC &#40;Oryza minuta Stress&#45;Related Putative Caleosin&#41; was selected for the further study. Caleosins, which contain a single Ca2&#43;&#45;binding EF&#45;hand, have been identified in a number of plant species. In Arabidopsis, until now, it was only reported that caleosins play on seed embryo development. However, our studies demonstrated that SRC protein could act on stress response. To intensive study, we have also characterized ARABIDOPSIS STRESS&#45;RELATED PUTATIVE CALEOSIN1 &#40;AtSRC1&#41; as a Arabidopsis ortholog of OmSRC. Interestingly, strong expression of ProAtSRC1:GUS was detected in all tissues, including leaves, flower and stem, along the veins. On germination assays under various osmotic stresses condition, AtSRC1&#45;overexpressed Arabidopsis plants show multiple abotic&#45;stress tolerance. Moreover, we confirmed that AtSRC1 gene is required to restrict to spread of necrotrophic fungal pathogen Botrytis cinerea. Based on these data, we have determined the biological functions of novel AtSRC1 in defense system, including regulatory mechanism as well as the effects of gene itself. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Effects of Phosphorus Signaling on Auxin Transport as Related to Root Development</div>

<div>

	<span  >
	Zhou Guoquan</span>
	<sup>1</sup>, 

	<span  >
	Wang Jinxiang</span>
	<sup>1</sup>, 

	<span  >
	Yan Xiaolong</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Liao Hong</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Root Biology Center, South China Agricultural University, Guangzhou 510642, China

</div>

<div>Phosphorus &#40;P&#41; is an essential nutrient element for plant growth, and it is well established that P availability regulates root architecture. Low P availability inhibits primary root growth and promotes lateral root growth of <span style="font-style: italic">Arabidopsis</span>, and induces a more branching root system. The interactions between P availability and auxin biosynthesis as well as auxin signaling were documented. However it remains unclear that the effects of spatial availability of phosphorus on auxin transport as related to root development. In the present study, we elucidated the interactions between the spatial P availability and auxin transport in <span style="font-style: italic">Arabidopsis</span> through stratified P treatment approach. Four treatments, namely LP&#47;LP, LP&#47;HP, HP&#47;LP and HP&#47;HP are employed. We found that primary root growth could be rescued in two stratified P treatments, LP&#47;HP and HP&#47;LP; and the lateral root proliferation in the high P patch was induced in these two stratified P treatments. More auxin accumulation and cell division activities of the lateral root primordia in the high P patch were found as indicated by the expressions of the cell cycle marker gene <span style="font-style: italic">CycB1::GUS</span> and auxin reporter gene <span style="font-style: italic">DR5::GUS</span>, but the effects were decreased after the primordia emerged, implying that the effects of spatial P availability on cell division and auxin transport is dependent on root development stage. We conclude that P availability has a close relationship with auxin distribution, and P availability affects root development through altering auxin transport or&#47;and synthesis.&#13;<br/>&#13;<br/>We thank Drs Tom J. Guilfoyle &#40;University of Missouri &#41; and Tom Beeckman &#40;Univerisity of Gent&#41; for providing seeds of transgenic lines of Arabidopsis with <span style="font-style: italic">DR5::GUS</span> and <span style="font-style: italic">CycB1::GUS</span>. This work is supported by the National Natural Science Foundation of China &#40;30571111 and 30600380&#41;.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Cooperation and Functional Diversification of Two <span style="font-style: italic">Phospholipase A1</span> Genes for Jasmonate Biosynthesis</div>

<div>

	<span style="text-decoration: underline" >
	Youbong Hyun</span>
	<sup>1</sup>, 

	<span  >
	Sungwook Choi</span>
	<sup>1</sup>, 

	<span  >
	Hyun-Ju Hwang</span>
	<sup>1</sup>, 

	<span  >
	Jihyeon Yu</span>
	<sup>1</sup>, 

	<span  >
	Sang-Jip Nam</span>
	<sup>2</sup>, 

	<span  >
	Jaeyoung Ko</span>
	<sup>2</sup>, 

	<span  >
	Ju-Young Park</span>
	<sup>3</sup>, 

	<span  >
	Hyoung Yool Lee</span>
	<sup>4</sup>, 

	<span  >
	Stephen Beungtae Ryu</span>
	<sup>4</sup>, 

	<span  >
	Yong-Hwan Lee</span>
	<sup>3</sup>, 

	<span  >
	Yoo-Sun Noh</span>
	<sup>5</sup>, 

	<span  >
	Sunghwa Choe</span>
	<sup>1</sup>, 

	<span  >
	Heonjoong Kang</span>
	<sup>2</sup>, 

	<span  >
	Ilha Lee</span>
	<sup>1,5</sup>

</div>

<div>

	<sup>1</sup>National Research Laboratory of Plant Developmental Genetics, Department of Biological Sciences, Seoul National University, Seoul 151-742, Korea, 

	<sup>2</sup>Center for Marine Natural Products and Drug Discovery, School of Earth and Environmental Sciences, Seoul National University, NS80, Seoul 151-747, Korea, 

	<sup>3</sup>Department of Agricultural Biotechnology and Center for Agricultural Biomaterials, Seoul National University, Seoul 151-921, Korea, 

	<sup>4</sup>Bio-Evaluation Center, Korea Research Institute of Bioscience and Biotechnology, Ochang, Chungbuk 363-883, Korea, 

	<sup>5</sup>Global Research Laboratory for Flowering at SNU and UW, Seoul 151-742, Korea

</div>

<div>Jasmonic acid &#40;JA&#41; plays pivotal roles in diverse plant biological processes including wound response. The release of a&#45;linolenic acid is the committed step for JA biosynthesis, but the mechanism of this process remains elusive. We report here that two genes encoding chloroplast&#45;targeted phospholipase A1, <span style="font-style: italic">DONGLE</span> &#40;<span style="font-style: italic">DGL</span>&#41; and <span style="font-style: italic">DEFECTIVE IN ANTHER DEHISCENCE1</span> &#40;<span style="font-style: italic">DAD1</span>&#41;, are both necessary and sufficient for a&#45;linolenic acid production. These two genes, despite being functionally redundant, show different induction kinetics following wounding, indicating temporally separated roles in the wound response &#8211; <span style="font-style: italic">DGL</span> for the early phase and <span style="font-style: italic">DAD1</span> for the late phase. Moreover, <span style="font-style: italic">DGL</span> has a specific role in maintaining basal JA content under normal conditions, which regulates vegetative tissue growth and is required for a rapid JA burst after wounding. Our results indicate that PLD produces a wound&#45;inducible signal for the transcriptional activation of <span style="font-style: italic">DGL</span> and <span style="font-style: italic">DAD1</span>, but that it does not provide the precursor of a&#45;linolenic acid.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Comprehensive analysis of ABA&#45;regulated metabolome and transcriptome in drought response of Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Kaoru Urano</span>
	<sup>1</sup>, 

	<span  >
	Kyonoshin Maruyama</span>
	<sup>2</sup>, 

	<span  >
	Migiwa Takeda</span>
	<sup>3</sup>, 

	<span  >
	Nozomu Sakurai</span>
	<sup>3</sup>, 

	<span  >
	Hideyuki Suzuki</span>
	<sup>3</sup>, 

	<span  >
	Kazuki Saito</span>
	<sup>3,4,5</sup>, 

	<span  >
	Daisuke Shibata</span>
	<sup>3</sup>, 

	<span  >
	Kazuko Yamaguchi-Shinozaki</span>
	<sup>2,6</sup>

</div>

<div>

	<sup>1</sup>RIKEN PSC,Tsukuba,Japan, 

	<sup>2</sup>JIRCAS,Tsukuba,Japan, 

	<sup>3</sup>Kazusa DNA Res. Inst.,Kisarazu,Japan, 

	<sup>4</sup>Chiba Univ.,Chiba,Japan, 

	<sup>5</sup>RIKEN PSC,Yokohama,Japan, 

	<sup>6</sup>The Univ. of Tokyo,Tokyo,Japan

</div>

<div>ABA is an important phytohormone involved in drought stress response in higher plants. 9&#45;CIS&#45;EPOXYCAROTENOID DIOXYGENASE &#40;NCED&#41; is a key enzyme for ABA biosynthesis. Drought&#45;inducible Arabidopsis NCED3 is a key gene for ABA accumulation under drought stress. To understand ABA&#45;regulated metabolic network in response to drought stress, we analyzed metabolome and transcriptome using Arabidopsis wild&#45;type plants and NCED3 knockout mutant &#40;nc3&#45;2&#41; under drought stress. Of 200 metabolites were detected by GC&#45;TOF&#45;MS &#40;gas chromatography time&#45;of&#45;flight mass spectrometry&#41; and CE&#45;MS &#40;capillary electrophoresis mass spectrometry&#41;, 64 metabolites were found to increase in Arabidopsis wild&#45;type plants in response to drought stress. Of these 64 metabolites, 37 metabolites less increased in nc3&#45;2. Accumulation of amino acids and organic acids depended on ABA accumulation under drought stress. The transcript levels of many drought&#45;inducible genes were also altered in nc3&#45;2 under stress conditions. These genes have the ABA responsive elements &#40;ABREs&#41; in their promoter regions. ABRE dependent gene expression is a major regulation system under drought stress. Integrated analysis of metabolome and transcriptome revealed the ABA dependent transcriptional regulation of branch chain amino acids, saccharopine, proline and polyamine biosynthesis. Our study provides the endogenous ABA functions to control the metabolic network in response to drought stress. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Responses of sexual reproduction pattern and leaf traits of <span style="font-style: italic">Arabidopsis thaliana</span> to elevated temperature from global warming</div>

<div>

	<span style="text-decoration: underline" >
	Jin Biao</span>
	<sup>1</sup>, 

	<span  >
	JIANG Ke-zhen</span>
	<sup>1</sup>, 

	<span  >
	WANG Jing</span>
	<sup>1</sup>, 

	<span  >
	NI Cheng-yang</span>
	<sup>1</sup>, 

	<span  >
	QI Yi-ming</span>
	<sup>1</sup>, 

	<span  >
	TENG Nian-jun</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>College of Horticulture and Plant Protection, Yangzhou University, Yangzhou 225009, China, 

	<sup>2</sup>College of Horticulture, Nanjing Agricultural University, Nanjing 210095, China

</div>

<div>It is predicted that the mean global temperature will increase 2&#45;5&#8451; during the next 50&#45;100 years according to the report of IPCC Fourth Assessment Report &#40;2007&#41;. The issue of global warming has been appealed to many people and governments in the world. How future plants will respond to this global change&#63; &#13;<br/>In the present study, <span style="font-style: italic">Arabidopsis thaliana</span>, will be continuously grown under the temperature of 23&#8451;&#47;18&#8451; &#40;day&#47;night temperature&#41;&#40;ambient temperature&#41; and 28&#8451;&#47;23&#8451; &#40;elevated day&#47;night temperature by 5&#8451;&#41;&#40;elevated temperature&#41; for multi&#45;generations. The changes in various reproductive characteristics of an individual generation will be investigated and their overall changes in multi&#45;generation will be further investigated. In addition, the changes in physiological, biochemical, and cytological traits of leaves of <span style="font-style: italic">Arabidopsis</span> grown under two different temperature conditions will be compared. The main results of the first generation are listed as follows: compared with ambient temperature, elevated temperature significantly advanced the flowering time and  shortened the life cycle of <span style="font-style: italic">Arabidopsis</span>, and resulted in less flowers, less fruits, less seeds, less reproductive organ biomass, less vegetative organ biomass, and less total biomass as well. In addition, seed weight, seed number per silique, silique length, and the mass allocated to reproduction were significantly decreased, whereas seed germination rate was significantly increased. Furthermore, elevated temperature also had a negative effect on leaf growth and development. The content of chlorophyll in leaf cell and the numbers of grana thylakoid, grana thylakoid lamelle and stroma thylakoid in the chloroplast were decreased under elevated temperature. All these results suggest that global warming in the near future will result in possible consequences of plant growth and development. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A protein&#45;protein interaction network model for salt stress in Arabidopsis thaliana</div>

<div>

	<span  >
	Xingguo Lan</span>
	<sup>1</sup>, 

	<span  >
	Yu-zhe Nie</span>
	<sup>1</sup>, 

	<span  >
	Guang Li</span>
	<sup>1</sup>, 

	<span  >
	Tieliu Shi</span>
	<sup>2</sup>, 

	<span  >
	Qijiang Xu</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Yuhua Li</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>College of Life Science, Northeast Forestry University, Harbin, China, 

	<sup>2</sup>Bioinformation Center, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China

</div>

<div>Protein&#45;protein interactions play key roles in protein function and the structural organization of a cell. We constructed a protein&#45;protein interaction network using a naive Bayes classifier for integrating disparate data types in predicting Arabidopsis thaliana protein&#45;protein interactions. These methods included gene expression method, shared biological function method, protein domain method, phylogenetic profile method&#65292;gene fusion method&#65292;gene neighbor method and GO annotation method. Here, we collected 2645 non&#45;redundant salt related proteins through consulting drastic database&#65288;http:&#47;&#47;www.drastic.org.uk&#47;&#65289;, GO database&#65288;http:&#47;&#47;www.geneontology.org&#65289; and texts mining. And a protein&#45;protein interaction network for salt stress in Arabidopsis thaliana was constructed with 2645 proteins. In the network, we found that 898 proteins formed 1778 interaction pairs. We performed analysis that showed a dynamical explanation for the SOS pathway and MAPK pathway. Furthermore, relationships of salt induced proteins were discussed. This protein&#45;protein interaction network model provides a framework for exploring molecular mechanism in plant salt stress response. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Continuous mechanical impedence enhances ethylene response rather than ethylene production in Arabidopsis roots.</div>

<div>

	<span style="text-decoration: underline" >
	Takashi Okamoto</span>
	<sup>1</sup>, 

	<span  >
	Abidur Rahman</span>
	<sup>2</sup>, 

	<span  >
	Yutaka Oono</span>
	<sup>3</sup>, 

	<span  >
	Seiji Tsurumi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Center for Supports to Research and Education Activities Isotope Division, Kobe University, 

	<sup>2</sup>Cryobiosystem Research Center, Faculty of Agriculture, Iwate University, 

	<sup>3</sup>Department of ion-beam-applied biology, Japan Atomic Energy Research Institute

</div>

<div>We investigated the role of hormones in regulating the growth and morphology of roots during mechanical impedance by developing a new growing system and using the model plant Arabidopsis thaliana.    The Arabidopsis seedlings grown horizontally on a dialysis membrane&#45;covered agar plate encountered adequate mechanical impedance as the roots show a characteristic ethylene phenotype; 0.5&#45;fold reduction in root growth, 1.2&#45;fold increase in root diameter, 0.5&#45;fold decrease in cell elongation and ectopic root hair formation.    The root phenotype characterization of various mutants having altered response to ethylene biosynthesis or signaling, the chemical inhibitor studies, gene expression analyses and the ethylene production assay, altogether suggested that enhanced ethylene response plays a primary role in changing the root morphology and development during mechanical impedance.    We also provided evidence that ethylene signaling rather than ethylene synthesis plays a pivotal role in enhancing the ethylene response.    Taken together, our results provide a mechanistic explanation of the role of ethylene in changing the root morphology during mechanical impedance.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Natural variation in genes determining high&#45;latitude adaptation in Arabidopsis thaliana.</div>

<div>

	<span style="text-decoration: underline" >
	AM Lewandowska-Sabat</span>
	<sup>1</sup>, 

	<span  >
	S Fjellheim</span>
	<sup>1</sup>, 

	<span  >
	R Nestestog</span>
	<sup>1</sup>, 

	<span  >
	JE Olsen</span>
	<sup>1</sup>, 

	<span  >
	OA Rognli</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Norwegian University of Life Sciences P.O. Box 5003, 1432 Aas, Norway

</div>

<div>Vernalization &#40;exposure to prolonged cold&#41; accelerate flowering by suppressing expression of the floral repressor FLC. Analyses of vernalization responses help to reveal how these responses are influencing flowering and local adaptation in Arabidopsis. The objective of this study is to understand the effect of vernalization on flowering in natural populations of A. thaliana at high latitudes. A. thaliana populations have been collected from range of high&#45;latitude and high&#45;altitude locations in Norway. These locations have unique combinations of photoperiod, light quality and temperature found nowhere else where A. thaliana naturally occurs. Flowering time after 5 different vernalization treatments &#40;0, 3, 6, 9, 12 wks of vernalization&#41; was scored in 27 &#40;0, 3, 6, 9 wks&#41; or 36 &#40;12 wks&#41; populations of A. thaliana using 5&#45;12 lines per population. Sequence analysis of flowering pathway genes such as PHYC, FLC, CRY1 and CRY2 has been performed in 15&#45;25 populations. Variation for flowering time among populations and among lines within populations was high. Statistical analysis revealed significant clinal variation in flowering time regressed against temperature, precipitation, altitude and latitude. No genetic variation in CRY1&#47;2 was found, and SNPs in PHYC did not show any geographic&#47;phenotypic pattern. However, analysis of the FLC gene in 25 populations revealed two main genetic clusters among Norwegian populations. Northern populations cluster together and flower significantly later than Southern ones. However the bootstrapping values of the phylogenetic tree are low and in addition not all late or rapid flowering populations are clustered together. This can be due to population structure, outcrossing or recent introduction of these accessions in Norway. Further genetic and phenotypic screening confirming the local adaptation and elucidating the origin of Norwegian populations are necessary. Consequently, phenotypic screening for responses to 5 different photoperiods of 10 selected populations is currently being conducted. Extreme individuals will be crossed with standard Col&#45;0 and used to establish QTL mapping populations.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Changes in DNA base sequences in the mutant of arabidopsis thaliana induced by low&#45;energy Ar&#43; implantation</div>

<div>

	<span style="text-decoration: underline" >
	Min Xu</span>
	<sup>1</sup>, 

	<span  >
	 Bian Po</span>
	<sup>1</sup>, 

	<span  >
	Yuejin Wu</span>
	<sup>1</sup>, 

	<span  >
	Zengliang Yu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Key Laboratory of Ion Beam Bio-engineering of Institute of Plasma Physics,Chinese Academy of Sciences, Hefei 230031, China

</div>

<div>As ion beams are a type of high linear energy transfer ionizing radiation, they can deposit high energy on a target as opposed to low LET raditations such as electron beams, r&#45;rays and x&#45;rays.It has been suggested that high LET radiation causes clustered damage on DNA produced several novel mutants, in rice &#40;semi dwarf,early maturity and high yield mutants&#41; with agron ion.They also indicated thatsubstantial DNA alterations might be involved in profucing these mutants. To reveal the mutation effect of low&#45;energy ion implantation on Arabidopsis thaliana.Tc243, a stable male&#45;sterility mutant, derived from the seeds irrdiated by 30KeV Ar&#43;  with the dose of 1.5&#215;1017 ions&#47;cm2, was used for genetic mapping and DNA sequence analysis.The results indicated that repetitive insertion of &#8220;TTATA&#8221; mutation in &#43;624bp was happened in the AUXIN RESISTANT 1 gene, encoding one of a subunit of a heterodimeric RUB&#45;activating enzyme of the jasmonic singaling pathway. The possibility reason resulted in the changes in DNA sequences may be double stand break.The mechanisms of low&#45;energy bio&#45;effect on DNA sequences were discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cloning and characterisation of <span style="font-style: italic">dracula</span> I2&#45;164, an Arabidopsis mutant showing no avoidance of shade</div>

<div>

	<span style="text-decoration: underline" >
	Xuewen Wang</span>
	<sup>1</sup>, 

	<span  >
	Irma Roig-Villanova</span>
	<sup>2</sup>, 

	<span  >
	Jaime Martinez-Garcia</span>
	<sup>2</sup>, 

	<span  >
	Paul Devlin</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Royal Holloway University of London, 

	<sup>2</sup>Institut de Biologia Molecular de Barcelona

</div>

<div>The shade avoidance response is a response to competition from neighbouring plants. Plants are able to detect light that has been reflected from a neighbouring plant by perceiving the change in light quality. Leaves absorb strongly in the red and blue wavelengths due to absorption by chlorophyll but they reflect far&#45;red wavelengths. Plants are able to monitor the red:far red &#40;R:FR&#41; ratio of incident light and can interpret a reduction in red:far red ratio as evidence of a neighbour growing closely alongside that may overtop it in future. Low R:FR ratio triggers a pronounced elongation growth response; reduction in branching; and reduction in leaf area to prevent future shading. The photoreceptors detecting this change in R:FR ratio are the phytochrome family of R and FR&#45;responsive photoreceptors.&#13;<br/>We have carried out a high throughput mutant screen using a shade responsive bioluminescent marker and have identified several shade avoidance mutants. One such mutant, <span style="font-style: italic">dracula</span> I2&#45;164, constitutively shows no avoidance of shade in this screen. <span style="font-style: italic">Dracula</span> I2&#45;164 is a novel phytochrome A mutant. In addition to showing a long hypocotyl in far red light, this mutant also shows a long hypocotyl in red light. Physiological analysis suggests that this red light phenotype is the result of interference with phyB signalling. The nature of the mutation points to a possible disruption of normal signal transduction leading us to predict that <span style="font-style: italic">dracula</span> I2&#45;164 could yield important clues in understanding phytochrome signalling.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Screening of rice genes for salinity or high&#45;temperature tolerance using Arabidopsis expression library of rice full&#45;length cDNAs</div>

<div>

	<span style="text-decoration: underline" >
	Kenji Oda</span>
	<sup>1</sup>, 

	<span  >
	Naoki Yokotani</span>
	<sup>1</sup>, 

	<span  >
	Naka Fujimoto</span>
	<sup>1</sup>, 

	<span  >
	Tsutomu Saitou</span>
	<sup>1</sup>, 

	<span  >
	Takanari Ichikawa</span>
	<sup>2</sup>, 

	<span  >
	Youichi Kondo</span>
	<sup>2</sup>, 

	<span  >
	Minami Matsui</span>
	<sup>2</sup>, 

	<span  >
	Hirohiko Hirochika</span>
	<sup>3</sup>, 

	<span  >
	Masaki Iwabuchi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Research Institute for Biological Sciences, Okayama, Japan, 

	<sup>2</sup>RIKEN Plant Science Center, Yokohama, Japan, 

	<sup>3</sup>National Institute of Agrobiological Sciences, Tsukuba, Japan

</div>

<div>To identify useful genes of rice, we started large&#45;scale functional analysis of rice genes using a gene screening procedure named &#8216;full&#45;length cDNA over&#45;expressor gene &#40;FOX&#41; hunting system&#8217;. About 13,000 full&#45;length cDNAs of rice were introduced into Arabidopsis by Agrobacterium&#45;mediated floral dip transformation method. More than 20,000 of transgenic Arabidopsis were generated and seeds of each line were harvested separately. Using this expression library, we tried to identify rice genes related to salinity or high&#45;temperature tolerance. To isolate salinity tolerant lines, T2 seeds were sowed on agar medium containing 1&#47;2 x MS salt and 150 mM of sodium chloride. Germination and growth were assessed visually for two weeks. About 200 tolerant lines were selected and 52 lines showed better germination and growth even on agar medium with 180 mM of sodium chloride. For isolation of thermotolerant lines, 4 day&#45;old T2 plants were treated transiently at 42 oC. Following 10 days after heat treatment, survivability was checked. Three lines were selected as thermotolerant lines. Rice cDNAs introduced in about 80 &#37; of selected lines were amplified by genomic PCR. To confirm direct relation between these cDNAs and the phenotype, re&#45;introduction of these cDNAs into Arabidopsis is in progress. This work is supported by Special Coordination Funds for Promoting Science and Technology in Japan.</div>

</div>





<div class="abs">

<div style="font-weight:bold">AKT1 Regulation and Pottasium Uptake in <span style="font-style: italic">Arabidopsis</span></div>

<div>

	<span  >
	Jiang Xu</span>
	<sup>1</sup>, 

	<span  >
	Hao-Dong Li</span>
	<sup>1</sup>, 

	<span  >
	Li-Qing Chen</span>
	<sup>1</sup>, 

	<span  >
	Yi Wang</span>
	<sup>1</sup>, 

	<span  >
	Li-Li Liu</span>
	<sup>1</sup>, 

	<span  >
	Liu He</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Wei-Hua Wu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>China Agricultural University, Beijing 100094, China

</div>

<div>Potassium is an essential mineral element for plant growth and development. Although it is known that plants absorb and transport K&#43; through the membrane transporters, it remains unclear how these transporters are regulated.  Here we show that a protein kinase CIPK23, encoded by the <span style="font-style: italic">LKS1</span> gene cloned from the low&#45;K&#43; sensitive <span style="font-style: italic">Arabidopsis</span> mutant <span style="font-style: italic">lks1</span>, significantly regulates K&#43;&#45;uptake particularly under low&#45;K&#43; conditions.  Lesion of <span style="font-style: italic">LKS1</span> significantly reduced K&#43;&#45;uptake and caused leaf chlorosis and growth inhibition, whereas overexpression of <span style="font-style: italic">LKS1</span> significantly enhanced K&#43;&#45;uptake and low&#45;K&#43; tolerance.  It was further demonstrated that CIPK23 positively regulated K&#43; transporter AKT1 and two calcineurin B&#45;like proteins CBL1 and CBL9 were upstream positive regulators of CIPK23.  The activation of AKT1 by CIPK23 and CBL1 or CBL9 was further confirmed by electrophysiological recordings using Xenopus oocyte expression system and also in root cell protoplasts. The AKT1&#45;mediated and CIPK23&#45; and CBL1&#47;CBL9&#45;regulated K&#43;&#45;uptake pathway in <span style="font-style: italic">Arabidopsis</span> under low&#45;K&#43; stress will be discussed.</div>

</div>





<div class="abs">

<div style="font-weight:bold">TOR kinase pathway in plants: Linking stress to growth signals and adaptation of the plant.</div>

<div>

	<span style="text-decoration: underline" >
	D.P.S. Verma</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Ohio State University, Columbus, OH, USA

</div>

<div>Plants grow under a variety of environmental conditions facing both abiotic and biotic stresses as well as nutrition limitations.  Multiplexing of these diverse signals is essential for a plant to optimize its growth. TOR &#40;Target Of Rapamycin&#41; kinase is recognized as a central regulator for nutrition and stress signals in yeast and animal cells, however our understanding of the role of this kinase in plants is very limited.  We have recently cloned all major players of this pathway from Arabidopsis, including pTOR, pRapTOR, pGbetaL, pS6K1, pS6K2 and demonstrated that this pathway is operative in plants &#40;Mahfouz et a., 2006&#41;.  In addition, we have demonstrated a direct link of ribosomal protein S6 &#40;S6p&#41; phosphorylation by pS6K and the possible control of ribosome biogenesis and protein synthesis. This occurs via binding of a histone deacetylase with S6p. The fact that G&#61538;L homolog protein that interacts with TOR is induced by auxin, suggests a possible link of TOR with growth signaling. We observed that the ineffectiveness of rapamycin to inhibit pTOR in plants is apparently due to the inability of plant FKBP12 to interact with rapamycin and thus fail to form a stable complex with pTOR.  We have shown that if pFKBP12 is replaced by human FKBP12, a stable complex can be formed with pTOR &#40;Mahfouz et al., 2006&#41;.  We are developing a stable line of Arabidopsis expressing human FKBP12 and would like to use this line to carry out chemical genomics studies followed by the isolation of suppressors of the TOR kinase pathway in plants. This novel approach will allow us to dissect the entire TOR kinase pathway and may facilitate the linking of stress signals to the growth signal pathways, a fundamental problem in plant development and adaptation to a given environment.  This study is likely to have a major impact on our understanding of plant growth and development, particularly under stress. This may lead to our understanding of the adaptation and speciation through epigenetic changes that are accelerated under stress conditions.&#13;<br/>&#13;<br/></div>

</div>


</div>
</div>

</body>
</html>
