
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">

<html>
<head>
<title>ICAR 2007 - Metabolism</title>
<link rel="Shortcut Icon" href="/favicon.ico">
<link rel="stylesheet" type="text/css" href="/css/main.css" />
<link rel="stylesheet" type="text/css" href="/css/search.css" />
<link rel="stylesheet" type="text/css" href="/css/base.css" />

<style type="text/css">
@import "/css/page/pagelevels.css";
</style>

<style type="text/css">
.abs {
 margin-top: 30px;
}
.abs div {
 margin: 10px;
}
</style>
</head>
<body>

<div id="container">

<script src="https://ssl.google-analytics.com/urchin.js" type="text/javascript"></script>
<script type="text/javascript">
_uacct = "UA-1945230-1";
urchinTracker();
</script>

 <div id="nav">	
  <div id="logo"><a href="/index.jsp"><img src="/i/logo2.gif" width="87" height="61" alt="tair"/></a></div>
 </div>

<div id="content" class="clearfix">

<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">Arabidopsis</span>&#8212;<span style="font-style: italic">Botrytis</span> interactions: using a model plant system to reveal diversity of a plant pathogen</div>

<div>

	<span style="text-decoration: underline" >
	Heather Rowe</span>
	<sup>1</sup>, 

	<span  >
	Dan Kliebenstein</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Davis, (CA), USA

</div>

<div><span style="font-style: italic">Arabidopsis</span> genetic resources can be used to gain understanding of plant pathogen diversity and infection mechanisms.  The molecular bases of host plant interactions with necrotrophic fungal pathogens such as <span style="font-style: italic">Botrytis cinerea</span> are unknown relative to the well&#45;described avr&#47;R interactions that mediate <span style="font-style: italic">Arabidopsis</span> relations with biotrophic pathogens.  We use a combined approach to investigate naturally variable molecular mechanisms controlling <span style="font-style: italic">Arabidopsis&#8212;Botrytis</span> interactions: <span style="font-style: italic">Arabidopsis</span> mutants with identified genetic lesions in defense&#45;related pathways allow direct tests of the effects of candidate genes, while natural variation among Arabidopsis accessions can be exploited to identify novel components of the plant&#45;pathogen interaction via QTL mapping.  <span style="font-style: italic">B. cinerea</span> is genetically diverse and infects numerous plant hosts.  We measure intra&#45;specific diversity of <span style="font-style: italic">B. cinerea</span> pathogenesis mechanisms by testing multiple <span style="font-style: italic">Botrytis</span> isolates on both <span style="font-style: italic">Arabidopsis</span> mutants and near&#45;isogenic lines differing at QTL influencing <span style="font-style: italic">Arabidopsis</span> susceptibility to <span style="font-style: italic">Botrytis</span> infection.  <span style="font-style: italic">B. cinerea</span> isolates differed in their inhibition by polygalacturonase&#45;inhibiting proteins &#40;PGIPs&#41; and their ability to induce the phytoalexin camalexin in planta. Additionally, isolates varied for increases in virulence on <span style="font-style: italic">Arabidopsis</span> mutants deficient in JA&#45;mediated signaling, suggesting that some isolates are detected by a JA&#45;independent signal transduction network.  Tests of 2 QTL using near&#45;isogenic lines showed that one QTL influenced lesion growth on <span style="font-style: italic">Arabidopsis</span> leaves for 90&#37; of isolates tested, while a second QTL influenced camalexin production for only 60&#37; of isolates tested.  These data indicate that studies of plant defenses using single pathogen isolates may not produce results broadly informative of the mechanisms of plant defense.  Measuring diversity among <span style="font-style: italic">B. cinerea</span> isolates may allow us to understand how this pathogen can have such a broad host range, while identifying conserved elements of the host&#45;pathogen interaction to assist control efforts.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Nuclear activity of MLA immune receptors links isolate&#45;specific and basal resistance responses</div>

<div>

	<span style="text-decoration: underline" >
	Qian-Hua Shen</span>
	<sup>1</sup>, 

	<span  >
	Yusuke Saijo</span>
	<sup>1</sup>, 

	<span  >
	Stefan Mauch</span>
	<sup>1</sup>, 

	<span  >
	Christoph Biskup</span>
	<sup>2</sup>, 

	<span  >
	St&#233;phane Bieri</span>
	<sup>3</sup>, 

	<span  >
	Beat Keller</span>
	<sup>3</sup>, 

	<span  >
	Hikaru Seki</span>
	<sup>1</sup>, 

	<span  >
	Bekir &#220;lker</span>
	<sup>1</sup>, 

	<span  >
	Imre Somssich</span>
	<sup>1</sup>, 

	<span  >
	Paul Schulze-Lefert</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max Planck Institute for Plant Breeding Research, K&#246;ln, Germany, 

	<sup>2</sup>Friedrich-Schiller-University of Jena, Jena, Germany, 

	<sup>3</sup>University of Z&#252;rich, Z&#252;rich, Switzerland

</div>

<div>Plant immune responses are triggered by pattern recognition receptors that detect conserved pathogen&#45;associated molecular patterns &#40;PAMPs&#41; or by resistance &#40;R&#41; proteins recognizing isolate&#45;specific pathogen effectors.  The polymorphic barley MLA R locus encodes allelic intracellular receptors containing an N&#45;terminal coiled&#45;coil &#40;CC&#41; structure, a central nucleotide binding &#40;NB&#41; site, and a leucine&#45;rich repeat &#40;LRR&#41; region. MLA receptors share &#62;90&#37; sequence identity but recognize isolate&#45;specific <span style="font-style: italic">B. graminis</span> f sp <span style="font-style: italic">hordei</span> effector. We show that in barley MLA R proteins function in the nucleus to confer resistance against the powdery mildew fungus. The recognition of the fungal AVRA10 effector by MLA10 induces nuclear associations between receptor and the barley transcription factors. The identified WRKY proteins act as repressors of PAMP&#45;triggered basal defence in barley. Two <span style="font-style: italic">Arabidopsis</span> WRKY TFs, <span style="font-style: italic">AtWRKY18&#47;40</span> that are the closest homologs of <span style="font-style: italic">HvWRKY1&#47;2</span>, also function as repressors of basal defence in <span style="font-style: italic">Arabidopsis</span>. Genome&#45;wide gene expression profiling data obtained upon inoculated with virulent <span style="font-style: italic">P. syringae</span> DC3000 in <span style="font-style: italic">Arabidopsis</span> reveals that 21 out of 23 up&#45;regulated genes are PAMP&#45;responsive genes, including the 6&#45;fold up&#45;regulated <span style="font-style: italic">SID2</span>. MLA appears to interfere with the WRKY repressor function, thereby de&#45;repressing PAMP&#45;triggered basal defence. Our findings imply a mechanism by which the MLA NB&#45;LRR immune receptors integrate distinct pathogen signals, and suggest that the MLA immune receptors affect the transcription machinery to activate an efficient immune response.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Study of ABA&#45;seed priming effects on germination and ionic distribution in Arabidopsis plant under salt stress</div>

<div>

	<span style="text-decoration: underline" >
	rozbeh Farhoudi</span>
	<sup>1</sup>, 

	<span  >
	Adel Modhej</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Agronomy Dep., Faculty of Agriculture, Islamic Azad University, Shoushtar Branch, Iran

</div>

<div>&#13;<br/>&#13;<br/>An experiment was carried out to study Arabidopsis thaliana ABA&#45;seed priming effects on seed germination and ionic distribution in seedlings under salt stress.  Twenty five &#40;25&#41; seeds were planted in each Petri dish &#40;7cm in diameter&#41;. Salt stress was applied at four levels including: 4, 8, 12 and 16ds&#47;m. Sodium chloride &#40;NaCl&#41;&#47;calcium chloride &#40;CaCl2&#41; mixture at 3:1 ratio was used to apply salt stress. Distilled water was used for control Petri dishes. In order to prime, seeds were soaked in 10&#956;mol ABA solution for 12hrs. The experiment was carried out in terms of factorial &#8211; completely random design with 8 treatments at four replications. 8&#45;16ds&#47;m salt stress significantly decreased germination rate, vigor index and coleoptiles length, but increased mean germination duration. However, coleorhizae length was not affected by salt stress conditions except at 16ds&#47;m level. ABA&#45;priming of stressed seeds increased germination rate and seed vigor index, decreased mean germination duration, whereas did not significantly affect coleoptiles and coleorhizae length. Salt stress condition significantly decreased K&#43; and Ca2&#43; and increased Na&#43;   content of seedling tissues.  ABA&#45;priming increased K&#43; and Ca2&#43;   resulting from salt stress, whereas had no effect on Na&#43;   content of seedlings. ABA application decreased Na&#43; level and increased K&#43; and Ca2&#43; content of seedling tissues.&#13;<br/>&#13;<br/>Keywords:  Arabidopsis, salt stress, ABA, seed priming </div>

</div>





<div class="abs">

<div style="font-weight:bold">The plant innate immune system, role of phospholipids and oxylipins.</div>

<div>

	<span style="text-decoration: underline" >
	Olga Kourtchenko</span>
	<sup>1</sup>, 

	<span  >
	Mats X Andersson</span>
	<sup>1</sup>, 

	<span  >
	Cornelia G&#246;bel</span>
	<sup>2</sup>, 

	<span  >
	Mats Hamberg</span>
	<sup>3</sup>, 

	<span  >
	Jefferey L Dangl</span>
	<sup>4</sup>, 

	<span  >
	David Mackey</span>
	<sup>5</sup>, 

	<span  >
	Ivo Feussner</span>
	<sup>2</sup>, 

	<span  >
	Mats Ellerstr&#246;m</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>G&#246;teborg University, Gothenburg, Sweden, 

	<sup>2</sup>Universit&#228;t G&#246;ttingen, G&#246;ttingen, Germany, 

	<sup>3</sup>Karolinska Institutet, Stockholm, Sweden, 

	<sup>4</sup>University of North Carolina, Chapel Hill, USA, 

	<sup>5</sup>The Ohio State University, Columbus, USA

</div>

<div>Bacterial pathogens deliver type III effector proteins into the plant cell during the course of infection. On susceptible hosts, type III effectors contribute to virulence, but in the case of resistant host plants, they betray the pathogen to the plant&#8217;s immune system. These proteins are functionally termed avirulence &#40;Avr&#41; proteins. Recognition induces a complex suite of cellular and molecular events comprising the plant&#8217;s inducible defense response. As recognition of the bacterial Avr proteins occurs inside host cells, the response can be elicited by &#34;in planta&#34; expression of bacterial type III effectors. Genes encoding two such effector molecules, AvrRpm1 and AvrRpt2 of &#34;Pseudomonas syringae&#34; were introduced as chemically inducible constructs in wild type Arabidopsis and in the corresponding resistance&#45;gene mutants, &#34;rpm1&#34; and &#34;rps2&#34;. We demonstrate that recognition of either of the two Avr proteins induces biphasic accumulation of phosphatidic acid &#40;PA&#41; via the sequential activation of phosphoinositide specific phospholipase C and D. Inhibition of phospholipases blocks the response, and feeding PA and phospholipase D directly to leaf tissue causes cell death and defense&#45;gene activation. These results indicate that PA is an integral part of the disease resistance signaling. Avr&#45;protein&#45;elicited defense response also includes the induction of 9&#45; and 13&#45;lipoxygenase &#40;LOX&#41;&#45;dependent oxylipin synthesis. The 13&#45;LOX products 12&#45;oxo&#45;phytodienoic acid &#40;OPDA&#41; and dinor&#45;oxo&#45;phytodienoic acid &#40;dinor&#45;OPDA&#41; are the most prominent oxylipins accumulated. Interestingly, the majority of OPDA and dinor&#45;OPDA &#40;&#62;90 &#37;&#41; are esterified to glycerolipids and, in particular, to a novel galactolipid. The structure of this compound was determined and given the trivial name Arabidopside E. We will discuss possible roles of Arabidopside E in the context of plant immune responses.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Genetic analysis of suppressors of <span style="font-style: italic">rar1</span></div>

<div>

	<span style="text-decoration: underline" >
	Yijian He</span>
	<sup>1</sup>, 

	<span  >
	David Hubert</span>
	<sup>1</sup>, 

	<span  >
	Pablo Tornero</span>
	<sup>2</sup>, 

	<span  >
	Jeff Dangl</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, University of North Carolina at Chapel Hill, Chapel Hill, NC, United States, 

	<sup>2</sup>IBMCP, Universidad Politecnica de Valencia, Valencia, Spain

</div>

<div>Disease resistance in plants is controlled by a gene&#45;for&#45;gene mechanism. Products of plant disease resistance &#40;R&#41; genes specifically recognize, directly or indirectly, avirulence &#40;Avr&#41; proteins encoded y pathogens. Most R genes encode NBS&#45;LRR proteins that have some domain similarity to animal NOD&#47;CARD proteins. NBS&#45;LRR&#45;mediated recognition triggers plant defense responses. Mutational analysis in Arabidopsis has been used to identify many components required for R gene function. RAR1 is required for hypersensitive cell death &#40;HR&#41; and disease resistance mediated by many NBS&#45;LRR proteins, even those that do not genetically require RAR1. To find additional genetic elements involved in RAR1 function in NBS&#45;LRR signaling, we isolated disease resistance following infection with bacteria Pseudomonas syringae pathovar tomato strain DC3000 &#40;avrPphB&#41;. This strain triggers the RPS5 NBS&#45;LRR protein. While disease resistance mediated by RPS5, RPM1 and RPS2 is diminished in rar1 plants, it is recovered in all five mutants. Three of these suppressors also recover RPM1&#45;mdeiated HR, while the other two mutants do not. However, none of these five suppressor mutants can recover RPS5&#45;mediated HR in a rar1 background. These finding support previous findings that show the roles of RAR1 in R gene&#45;mediated disease resistance and HR are genetically separable. Genetic analysis demonstrates that these five suppressors belong to three loci which are localized on Chromosome I, II and V. In addition, the non&#45;allelic noncomplementation between Chromosome II and V loci suggests that the corresponding protein products interact with each other.</div>

</div>





<div class="abs">

<div style="font-weight:bold">MOS7 is essential for plant innate immunity</div>

<div>

	<span style="text-decoration: underline" >
	Xin Li</span>
	<sup>1</sup>, 

	<span  >
	Yu Ti Cheng</span>
	<sup>1</sup>, 

	<span  >
	Dongling Bi</span>
	<sup>2</sup>, 

	<span  >
	Yuelin Zhang</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>University of British Columbia, 

	<sup>2</sup>National Institute of Biological Sciences

</div>

<div>Immunity against microbial pathogen infections in plants is a dynamic process. One of the most effective disease resistance mechanisms is mediated by Resistance proteins &#40;R proteins&#41;, which play a central role in recognizing pathogens and initiating downstream defense cascades. In Arabidopsis, a dominant mutant, <span style="font-style: italic">snc1</span>, was previously identified that constitutively expresses pathogenesis&#45;related &#40;<span style="font-style: italic">PR</span> genes and resistance against both bacterial and oomycete pathogens. A point mutation in one of the <span style="font-style: italic">RPP4</span> homologs rendered this TIR&#45;NB&#45;LRR&#45;type R protein constitutively active without interaction with pathogens. In screens for suppressors of the constitutive defence responses in <span style="font-style: italic">snc1</span>, a number of loss&#45;of function mutants were identified. The mutants restore wild&#45;type morphology either completely or partially. We have recently cloned nine of the mutations &#40;<span style="font-style: italic">mos1</span> to <span style="font-style: italic">mos9</span>&#41; using map&#45;based approaches. These suppressors suggest a complicated signaling network downstream of R&#45;protein activation that involves RNA processing, nucleocytoplasmic trafficking, and protein modification, etc.&#13;<br/>&#13;<br/>Here we report the analysis of <span style="font-style: italic">mos7</span>. A recessive mutation in <span style="font-style: italic">mos7&#45;1</span> completely suppressed all the autoimmune phenotypes of <span style="font-style: italic">snc1</span>. In <span style="font-style: italic">mos7&#45;1</span> single mutant, basal defense against virulent pathogens along with several R protein&#45;mediated defense are attenuated. <span style="font-style: italic">mos7&#45;1</span> also exhibit defects in systemic acquired resistance &#40;SAR&#41;, a key systemic innate immunity pathway in plants. MOS7 localizes to the nuclear rim, consistent with its homology with Nup88. In Drosophila, the Nup88&#45;Nup214 complex was shown to modulate NFkappaB activation, which is essential for activating animal innate immunity pathways. We are currently investigating whether MOS7 is required for NPR1 nuclear translocation, which is a key event in SAR.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Repression of defense and cell death by the <span style="font-style: italic">BON1&#47;CPN1</span> family likely involves multiple <span style="font-style: italic">R</span> genes in Arabidopsis</div>

<div>

	<span  >
	Yongqing Li</span>
	<sup>1</sup>, 

	<span  >
	Huijun Yang</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Jian Hua</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Cornell Universtiy, Ithaca, NY 14853, USA

</div>

<div>The Arabidopsis <span style="font-style: italic">BON1&#47;CPN1</span> gene is a negative regulator of defense responses. It belongs to an evolutionarily conserved copine gene family encoding two calcium&#45;dependent phospholipid binding C2 domains and a VWA domain. The loss of <span style="font-style: italic">BON1</span> function leads to constitutive defense responses apparently through activating an accession&#45;specific TIR&#45;NBS&#45;LRR type of disease resistance &#40;<span style="font-style: italic">R</span>&#41; gene <span style="font-style: italic">SNC1</span>. To investigate the intriguing repression of defense responses by <span style="font-style: italic">BON1</span>, we set out to identify the cellular processes <span style="font-style: italic">BON1</span> regulates. <br/>We found that <span style="font-style: italic">BON1</span> functions together with <span style="font-style: italic">BAP1</span> and its homolog <span style="font-style: italic">BAP2</span> that both encode samll proteins with one C2 domain. <span style="font-style: italic">BAP1</span> and <span style="font-style: italic">BAP2</span> appear to be a general inhibiotors of programmed cell death across kingdoms, as their overexpression suppresses <span style="font-style: italic">R</span>&#45; and mamalian apoptotic <span style="font-style: italic">Bax</span>&#45;induced programed cell death &#40;PCD&#41; in plants as well as hydrogen peroxide&#45;induced PCD in yeasts. Co&#45;expression of <span style="font-style: italic">BON1</span> is required in some cases for the inhition of PCD by <span style="font-style: italic">BAP1</span> and <span style="font-style: italic">BAP2</span>, suggesting the involvement of both <span style="font-style: italic">BON1</span> and <span style="font-style: italic">BAP1&#47;BAP2</span> in PCD modulation. <br/>In addition, we identified potential target genes of <span style="font-style: italic">BON1</span> other than <span style="font-style: italic">SNC1</span>. <span style="font-style: italic">BON1</span> has an overlapping function with its homolog <span style="font-style: italic">BON3</span> in Arabidopsis and the <span style="font-style: italic">bon1bon3</span> double mutant is seedling lethal in the Col accession but wild&#45;type looking in the Ws accession. Using recombinant inbred lines, we mapped three major QTLs responsible for this natural variation. Interestingly, they all coincided with clusters of NBS&#45;LRR type of <span style="font-style: italic">R</span> genes. It is likely that some of these <span style="font-style: italic">R</span> genes become activated in <span style="font-style: italic">bon1bon3</span>, leading to lethality. <br/>We propose that <span style="font-style: italic">BON1</span> is involved in modulating PCD and the <span style="font-style: italic">BON1</span> gene family is guarded by multiple <span style="font-style: italic">R</span> genes. </div>

</div>





<div class="abs">

<div style="font-weight:bold">A Dual Regulation Mechanism for Disease Resistance and Susceptibility through Modulation of Salicylic Acid and Auxin Signaling by A GH3 in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Zuhua He</span>
	<sup>1</sup>, 

	<span  >
	Zhong Zhang</span>
	<sup>1</sup>, 

	<span  >
	Qun Li</span>
	<sup>1</sup>, 

	<span  >
	Paul E. Staswick</span>
	<sup>2</sup>, 

	<span  >
	Muyang Wang</span>
	<sup>1</sup>, 

	<span  >
	Ying Zhu</span>
	<sup>1</sup>, 

	<span  >
	Zhimiao Li</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Key Laboratory of Plant Molecular Genetics, Institute of Plant Physiology and Ecology, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences, Shanghai, China, 

	<sup>2</sup>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska, USA

</div>

<div>Salicylic acid &#40;SA&#41; plays a central role in plant disease resistance, and emerging evidence indicates that auxin, an essential growth hormone, is involved in plant disease susceptibility. GH3.5, a member of the GH3 family of early auxin&#45;responsive genes in Arabidopsis, encodes a protein possessing an in vitro adenylation activity on both IAA and SA and playing a role in modulating IAA homeostasis. Here we show that GH3.5 acts as a bifunctional modulator in both the SA&#45;mediated disease resistance and auxin&#45;elicited disease susceptibility. Overexpression of the GH3.5 gene in an activation&#45;tagged mutant gh3.5&#45;1D leads to elevated accumulation of SA and increased expression of PR&#45;1 in local and systemic tissues in response to avirulent pathogens. By contrast, two T&#45;DNA insertional mutations of GH3.5 partially compromised the systemic acquired resistance &#40;SAR&#41; associated with the diminished PR&#45;1 expression in systemic tissues. The gh3.5&#45;1D mutant accumulated high levels of free IAA after pathogen infection and impaired multiple R gene&#45;mediated resistance, which was also observed in the GH3.6 activation&#45;tagged mutant dfl1&#45;D that impacted the auxin pathway, indicating an important role of GH3.5&#47;GH3.6 in disease susceptibility. Microarray and northern blot analysis showed that the SA and auxin pathways were simultaneously augmented in gh3.5&#45;1D upon infection with an avirulent pathogen. The SA pathway was amplified by GH3.5 through inducing SA&#45;responsive genes, basal defense components, and multiple R gene homologs; while the auxin pathway was de&#45;repressed through up&#45;regulating IAA biosynthesis and transport genes and down&#45;regulating auxin repressor genes, resulting in altered metabolism and transport of various nutrients that might favor pathogen infection. Taken together, our data revealed novel regulatory functions of GH3.5 in the plant&#8211;pathogen interaction.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Cadmium stress alters gene expression of Arabidopsis seedlings</div>

<div>

	<span style="text-decoration: underline" >
	Wan Liu</span>
	<sup>1</sup>, 

	<span  >
	YS Yang</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Institute of Applied Ecology, Chinese Academy of Sciences, 

	<sup>2</sup>Cardiff University

</div>

<div>Cadmium &#40;Cd&#41; is a non essential element, and is a widespread environmental pollutant. Exposure&#13;<br/>to Cd can result in a variety of adverse health effects in plant and humans. In this study, Arabidopsis seedlings were used as bioindicator of Cd pollution in the range of 0&#45;6.0 mg L&#45;1 for 18 days and gene expression patterns were used to link increased Cd exposure with progressive biological effects. Seven genes known to be involved in cell division and DNA mismatch repair &#40;MMR&#41; were investigated by RT&#45;PCR and normalized according to 18S rRNA gene expression. Expression of proliferating cell nuclear antigen 2 &#40;PCNA2&#41; or MutS3 homolog &#40;MSH 3&#41; genes in shoots was induced by exposure to Cd of 0.75 ppm or 1.5 ppm Cd significantly, but was repressed by other Cd concentration, respectively; whereas exposure to 0.75&#45; 6 ppm of Cd obviously decreased expression of PCNA1, MutL1 homolog &#40;MLH1&#41; and MSH6 independent of any observable health effects, including survival, gross morphology and height of shoots. This work demonstrated that specific gene expression changes could serve as useful molecular biomarkers of exposure and related health effects.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Ferredoxin plays an essential role in plant innate immunity and fruit ripening</div>

<div>

	<span style="text-decoration: underline" >
	Teng-Yung FENG</span>
	<sup>1</sup>, 

	<span  >
	Hsiang-En HUANG</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of plant and Microbial Biology, Academia Sinica, Taipei, Taiwan

</div>

<div>There are many isoforms of ferredoxin proteins in plant cells. In this communication, we would like to present that ferredoxin&#40;Fd&#41;,is not only involving in electron flow of photosystem I, but also highly linked to plant biotic&#47;abiotic stress monitoring. We found that that Fd1&#45;medaited disease resistance is established through the regulation of evolution of active oxygen species &#40;AOS&#41;. The disease immunity of various transgenic plants was enhanced by the over&#45;expression of Fd1 protein&#40;note 1&#41;. The &#34;fd1&#34; gene was transferred to wild&#45;type and resistance&#45;defective Arabidopsis to reveal the mechanism of Fd1&#45;mediated resistance. The wild&#45;type plant with elevated amount of Fd1 behaves more resistant to bacterial pathogen infection. However, &#34;fd&#34;&#45;transgenic plant becomes less tolerant under abiotic stress &#40;light and heat&#41;. Under the experimental conditions the &#34;fd1&#34;&#45;transgenic Arabidopsis plants are more sensitive to the treatment of heat, salicylic acid &#40;SA&#41;, harpin or &#34;Pseudomonas syringae&#34;. As the non&#45;transgenic Arabidopsis plants were shifted into dark condition, both the transcripts of Fd1 and Fd2 are quickly diminished in a similar fashion, although Fd2 is in a slower way. Under this condition, heat treatment would slow down this decline phenomenon. In tomato plants elevating Fd3 content artificially would speed up fruit ripening process in terms of pigment formation. In contrast, the increment of Fd1 would slow down this ripening process. These findings would help us to understand how the contents of different Fd proteins in plant could monitor some important stress physiology,even including fruit ripening.&#13;<br/>&#13;<br/>Note 1: reference cited&#13;<br/>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;in rice: Plant Sci. 2001, 160:1035; &#13;<br/>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;in tobacco: Plant Mol. Biol. 2003, 51:913;&#13;<br/>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;in orchid: Transgenic Res. 2004, 13:567; &#13;<br/>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;in Arabidopsis: Mol. Plant Pathol. 2007, 8:129.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Role of the Arabidopsis MYB transcription factor AtMYB30 in the control of disease resistance and hypersensitive cell death</div>

<div>

	<span  >
	Sol&#232;ne Froidure</span>
	<sup>1</sup>, 

	<span  >
	Carine Huard</span>
	<sup>1</sup>, 

	<span  >
	Amandine Leger</span>
	<sup>1</sup>, 

	<span  >
	Dominique Roby</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Susana Rivas</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Laboratoire des Interactions Plantes-Microorganismes, UMR CNRS/INRA 2594/441, BP 52627, 31326 Castanet Tolosan, FRANCE.

</div>

<div>The Hypersensitive Response &#40;HR&#41;, characterized by a rapid and localized cell death at the inoculation site, is one of the most efficient resistance reactions to pathogen attack in plants. We previously found AtMYB30 as specifically, rapidly and transiently expressed during incompatible interactions between Arabidopsis and bacterial pathogens. AtMYB30 presents homology to plant MYB transcription factors which play a central role in the regulation of a variety of developmental and metabolic responses. We also demonstrated that AtMYB30 is a positive regulator of the hypersensitive cell death. Results from a transcriptome analysis, together with recent molecular, genetic and biochemical studies, show that putative AtMYB30 target genes are involved in the lipid biosynthesis pathway leading to the production of very long chain fatty acids &#40;VLCFAs&#41;, suggesting a role of this pathway in the control of the HR and plant defence responses&#13;<br/>New strategies aimed at &#40;i&#41; studying the subcellular localization of AtMYB30, &#40;ii&#41; characterizing posttranslational modifications within the protein, and &#40;iii&#41; identifying proteins that may interact and work together with AtMYB30 in the initiation of the HR will be presented. &#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Dissection of the signaling pathway of RPW8&#45;mediated broad&#45;spectrum disease resistance</div>

<div>

	<span  >
	Wenming Wang</span>
	<sup>1</sup>, 

	<span  >
	Xiaohua Yang</span>
	<sup>1</sup>, 

	<span  >
	Undral Orgil</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Shunyuan Xiao</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Maryland Biotechnology Institute

</div>

<div>The structurally unique resistance &#40;R&#41; gene RPW8 confers broad&#45;spectrum resistance in Arabidopsis to powdery mildew pathogens. To understand the molecular basis of RPW8&#45;mediated resistance, we first established that RPW8 activates hypersensitive cell death and defense via a highly conserved signaling pathway also recruited by the TIR&#45;NBS&#45;LRR &#40;Toll&#45;interleukin receptor nucleotide&#45;binding&#45;site and leucine&#45;rich&#45;repeat&#41; R genes. More recently, we identified one isoform of 14&#45;3&#45;3 &#40;GF14a&#41; and a protein phosphatase type 2C &#40;designated DAPP1 for defense&#45;associated protein phosphatase type 2C 1&#41;, each of which belongs to a highly conserved gene family, as putative RPW8&#45;interacting proteins in the yeast&#45;two&#45;hybrid system. Our data showed that transcriptional up&#45;regulation of GF14a or down&#45;regulation of DAPP1 in Arabidopsis resulted in constitutive expression of PR genes, spontaneous HR&#45;like cell death and enhanced resistance against powdery mildew &#40;Erysiphe cichoracearum&#41; and downy mildew &#40;Hyaloperonospora parasitica&#41;. Genetic analysis indicated that GF14a and DAPP1 may play a positive and a negative regulatory role, respectively, in a conserved salicylic acid&#45;dependent basal defense pathway that is also required for function of RPW8 and TIR&#45;NBS&#45;LRR R genes. Interestingly, we found that expression of GF14a was highly induced in DAPP1&#45;silenced plants, suggesting that DAPP1 may also function as a transcriptional repressor of GF14a. Transient expression of hemagglutinin &#40;HA&#41;&#45;tagged DAPP1 in Nicotiana benthamiana and subsequent enzymatic assay demonstrated that DAPP1 is a biologically active phosphatase. By using the bimolecular fluorescence complementation &#40;BiFC&#41; method, we found that both GF14a and DAPP1 appeared to interact with RPW8 in vivo. We thus hypothesize that GF14a may bind to phosphorylated RPW8 to enhance its resistance function, whereas DAPP1 may dephosphorylate RPW8 to down&#45;regulate its function. Both GF14a and DAPP1 may be promising targets for engineering broad&#45;spectrum disease resistance in crop plants.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Salicylic Acid Antagonizes the Auxin Signaling Pathway</div>

<div>

	<span style="text-decoration: underline" >
	Dong Wang</span>
	<sup>1,2</sup>, 

	<span  >
	Karolina Mukhtar</span>
	<sup>1</sup>, 

	<span  >
	Xinnian Dong</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Duke University, Durham, NC, USA, 

	<sup>2</sup>Stanford University, Stanford, CA, USA

</div>

<div>The phytohormone auxin regulates almost every aspect of plant development.  A growing body of evidence indicates that during pathogen infection, auxin homeostasis is perturbed.  Many plant pathogens either produce auxin themselves or manipulate host auxin biosynthesis to interfere with the host normal developmental processes. Disruption of bacterial auxin synthesis genes compromised virulence, suggesting that manipulation of auxin signaling be taken by pathogenic microbes as a virulence strategy.  In response, plants likely evolved mechanisms to regain control over auxin signaling.  Auxin molecules are recognized by the auxin receptor TIR1 and related proteins; these auxin&#45;bound F&#45;box proteins then target the Aux&#47;IAA family of transcriptional repressors of auxin response for degradation.  Bacterial flagellin has been shown to trigger rapid degradation of auxin receptor transcripts via a microRNA, resulting in down regulation of the auxin signaling pathway and an increase in basal resistance to the pathogen. Recognition of flagellin and other pathogenesis&#45;associated molecular patterns often leads to long&#45;lasting and broad&#45;spectrum resistance mediated by the defense signaling molecule salicylic acid &#40;SA&#41;.  Here, we demonstrate that activation of the SA pathway strongly desensitize plants&#8217; responses to both exogenous and endogenous auxin. Furthermore, we provide evidence that SA exerts its inhibitory effect on auxin by stabilizing the Aux&#47;IAA repressor proteins.  This antagonism between a major defense signal and a key growth hormone may enable the plant to adjust global homeostasis in response to a changing environment.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Toward isolation and characterization of the protein complex containing the Arabidopsis disease resistance protein RPS2</div>

<div>

	<span style="text-decoration: underline" >
	Yiping Qi</span>
	<sup>1</sup>, 

	<span  >
	Fumiaki Katagiri</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Minnesota-Twin city, USA

</div>

<div>The Arabidopsis resistance &#40;R&#41; gene RPS2 encodes a NB&#45;LRR protein which recognizes the avirulence protein AvrRpt2 from the bacterial pathogen Pseudomonas syringae. Accumulating evidence suggests that NB&#45;LRR proteins function in a large protein complex which may change dynamically after activation. Identification of unknown members of the RPS2 protein complex will provide insights into the mechanisms controlling early events in RPS2&#45;mediated gene&#45;for&#45;gene resistance. &#13;<br/>To facilitate purification of R protein complexes, we devised a tripartite sequence termed the HPB tag.  This tag, based on the concept in Zhong et al. &#40;2003&#41;, consists of one copy of the HA epitope tag, a Prescission&#174; protease recognition site, and the biotinylation site of the Arabidopsis 3&#45;methylcrotonyl&#45;CoA carboxylase.  The biotinylation sequence facilitates high affinity purification using a streptavidin matrix, while the HA tag allows protein visualization using a Western Blot.  We have demonstrated that the C&#45;terminal RPS2&#45;HPB fusion protein is functional, is biotinylated in planta, and can be purified from plant extracts using a streptavidin matrix.  Protein fractions purified from RPS2&#45;HPB&#45;expressing plants were compared to those from untranformed plants using SDS&#45;PAGE and silver staining.  Several unique protein bands were identified from RPS2&#45;HPB&#45;expressing plants.  We are currently identifying the RPS2&#45;HPB&#45;specific proteins using mass&#45;spectrometry&#45;based methods.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Over&#45;expression of wild rice &#40;Oryza grandigulmis&#41; OgPAE1 conferred fungal   resistance against Botrytis cinerea infection</div>

<div>

	<span style="text-decoration: underline" >
	Jung-Hun Pak</span>
	<sup>1</sup>, 

	<span  >
	Eun-Hee Jeon</span>
	<sup>1</sup>, 

	<span  >
	Mi-Jin Kim</span>
	<sup>1</sup>, 

	<span  >
	Hye-Jeong Kim</span>
	<sup>1</sup>, 

	<span  >
	Ki-Jung Lee</span>
	<sup>1</sup>, 

	<span  >
	Young-Soo Chung</span>
	<sup>1</sup>, 

	<span  >
	Kyung-Ho Kang</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Dept. of Genetic Engineering, Dong-A University, Busan, Korea, 

	<sup>2</sup>National Crop Experiment Station, Suwon, Korea

</div>

<div>Oryza grandiglumis &#40;CCDD, 2n&#61;48&#41;, one of the wild rice species, has been known to possess fungal resistance against sheath blight, rice blast, bacterial leaf blight and insect resistance against brown plant hopper&#40;Nilaparvata lugens&#41;. Here we report the cDNA cloning , expression characterization and resistance against Botrytis cinerea of OgPAE1 gene. OgPAE1 contains 714bp of nucleotides and 237 amino acids. The highest homology was observed with OsPAE1 mRNA for alpha 5 subunit of 20S proteasome from Oryza sativa &#40;japonica cultivar&#45;group&#41;. The 20S proteasome is the proteolytic complex that is involved in removing abnormal proteins and other various biological functions. In this study, we observed resistance of OgPAE1 gene against Botrytis cinerea. The OgPAE1 gene was induced by wounding, yeast extract, jasmonic acid &#40;JA&#41; and salicylic acid &#40;SA&#41;, protein phosphatase inhibitors cantharidin &#40;CN&#41; as well as endotall &#40;EN&#41;. To identify in vivo function of gene OgPAE1, the gene was transformed into Arabidopsis thaliana and high concentration of Botrytis cinerea &#40;5&#215;105&#47;mL&#41; was inoculated on the OgPAE1 transgenic plants. The growth of Botrytis cinerea was suppressed in OgPAE1 transgenic plants but most of control plants destroyed completely. Average necrosis sizes of transgenic plants were 3.8 through 6.4 mm2, whereas that of control was 11.2 mm2.  Our results suggest that OgPAE1 may contribute to partial resistance against fungal diseases, even though clear in vivo function of the gene has not been reported yet from any plant species.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A Pseudomonas syringae Effector Inactivates MAPKs to Suppress PAMP&#45;Induced Immunity</div>

<div>

	<span  >
	Jie Zhang</span>
	<sup>1,2</sup>, 

	<span  >
	Feng Shao</span>
	<sup>2</sup>, 

	<span  >
	Yan Li</span>
	<sup>2</sup>, 

	<span  >
	Haitao Cui</span>
	<sup>2</sup>, 

	<span  >
	Linjie Chen</span>
	<sup>2</sup>, 

	<span  >
	Hongtao Li</span>
	<sup>2</sup>, 

	<span  >
	Yan Zou</span>
	<sup>2</sup>, 

	<span  >
	Chengzu Long</span>
	<sup>2</sup>, 

	<span  >
	Lefu Lan</span>
	<sup>3</sup>, 

	<span  >
	Jijie Chai</span>
	<sup>2</sup>, 

	<span  >
	She Chen</span>
	<sup>2</sup>, 

	<span  >
	Xiaoyan Tang</span>
	<sup>3</sup>, 

	<span style="text-decoration: underline" >
	Jianmin Zhou</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Institute of plant physiology and ecology,Shanghai,China, 

	<sup>2</sup>National institute of biological sciences, Beijing,China, 

	<sup>3</sup>Department of Plant Pathology, Kansas State University,USA

</div>

<div>Pseudomonas syringae effector proteins suppress plant immunity to promote parasitism. However, their biochemical function and virulence targets in the plant remain largely unknown. Here we show that HopAI1, an effector widely conserved in both plant and animal bacterial pathogens, inhibits the Arabidopsis mitogenactivated protein kinases &#40;MAPKs&#41; stimulated by bacterial flagellar peptide flg22, consequently suppressing two independent downstream defenses &#45; one leads to cell wall reinforcement through the AtrbohD&#45;mediated oxidative burst, the other activates the transcription of a large number of genes. The MAPKs MPK3 and MPK6 interact with HopAI1 in vitro and in vivo, indicating that they are direct targets of HopAI1. HopAI1 inactivates the MAPKs by removing the phosphate group from phosphothreonine through a unique phosphothreonine lyase activity, uncovering a novel mechanism by which the bacterium overcomes host innate immunity.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Virulence function of Pseudomonas syringae effector AvrB on Arabidopsis</div>

<div>

	<span  >
	Haitao Cui</span>
	<sup>1</sup>, 

	<span  >
	Yujing Wang</span>
	<sup>1</sup>, 

	<span  >
	Xiaoyan Tang</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	 Jian-Min Zhou</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute of Biological Sciences, Beijing,China, 

	<sup>2</sup>Department of Plant Pathology, Kansas State University, Manhattan, KS

</div>

<div>Pathogenic bacterial effectors suppress pathogen&#45;associated molecular pattern &#40;PAMP&#41;&#45;triggered host immunity &#40;PTI&#41; to promote parasitism. Previously we show that the Pseudomonas syringae effector AvrB enhances virulence on Arabidopsi in a RAR1&#45; and jasmonate pathway&#45;dependent manner. RAR1 is known to be a HSP90 cochaperone that stabilizes many resistance proteins in effector&#45;triggered immunity &#40;ETI&#41;. However, rar1 mutants exhibit an enhanced cell wall defense response to flg22, indicating that RAR1 is a negative regulator of basal defense. Co&#45;immunoprecipitation experiments indicated that AvrB and RAR1 are in the same protein complex. Experiments are underway to test the hypothesis that RAR1 also stabilizes some negative regulators in plant immunity. These negative regulators may be exploited by AvrB to enhance plant susceptibility. Progress will be presented.</div>

</div>





<div class="abs">

<div style="font-weight:bold"><span style="font-style: italic">Pseudomonas syringae</span> effector AvrPto blocks innate immunity by targeting receptor kinases</div>

<div>

	<span  >
	Tingting Xiang</span>
	<sup>1</sup>, 

	<span  >
	Yan Zou</span>
	<sup>2,1</sup>, 

	<span  >
	Na Zong</span>
	<sup>1</sup>, 

	<span  >
	Yong Wu</span>
	<sup>1</sup>, 

	<span  >
	Jie Zhang</span>
	<sup>1</sup>, 

	<span  >
	Weiman Xing</span>
	<sup>1</sup>, 

	<span  >
	Yan Li</span>
	<sup>1</sup>, 

	<span  >
	Xiaoyan Tang</span>
	<sup>3</sup>, 

	<span  >
	Lihuang Zhu</span>
	<sup>2</sup>, 

	<span  >
	Jijie Chai</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Jian-min Zhou</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute of Biological Sciences, 

	<sup>2</sup>Institute of Genetics, Chinese Academy of Sciences, 

	<sup>3</sup>Department of Plant Pathology, Kansas State University

</div>

<div>Receptor kinases perceive Pathogen Associated Molecular Patterns &#40;PAMPs&#41; to initiate plant innate immunity. PAMP&#45;induced immunity is often suppressed by bacterial effectors, resulting in pathogen propagation. To counteract, plants have evolved disease resistance genes to detect the effectors and trigger defenses. The <span style="font-style: italic">Pseudomonas syringae</span> effector AvrPto assists bacterial infection in susceptible plants through an unknown mechanism but triggers disease resistance in tomato plants carrying the cytoplasmic protein kinase Pto. Here we show that AvrPto binds multiple Arabidopsis kinases both <span style="font-style: italic">in vitro</span> and <span style="font-style: italic">in vivo</span> and block innate immune responses as a consequence. Structural study of the Pto&#45;AvrPto complex and mutational analysis of receptor kinase&#45;AvrPto interaction indicated that Pto and the kinase domain of the receptor kinases are structurally related and bind AvrPto in a competitive manner. The results support a model in which Pto is evolved to &#8220;deceive&#8221; AvrPto by mimicking the receptor kinases. While the binding of AvrPto to the receptor kinases blocks plant immune responses, the binding to Pto &#8220;betrays&#8221; the bacterium and triggers disease resistance.    </div>

</div>





<div class="abs">

<div style="font-weight:bold">Turnip crinkle virus resistance in Arabidopsis requires CRT1, a new member of the GHKL ATPase family</div>

<div>

	<span style="text-decoration: underline" >
	Hong-Gu Kang</span>
	<sup>1</sup>, 

	<span  >
	Joseph Kuhl</span>
	<sup>1</sup>, 

	<span  >
	Pradeep Kachroo</span>
	<sup>2</sup>, 

	<span  >
	Daniel F. Klessig</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Boyce Thompson Institute for Plant Research, Ithaca, New York, USA, 

	<sup>2</sup>University of Kentucky, Lexington, Kentucky, USA

</div>

<div>HRT is a CC&#45;NBS&#45;LRR&#45;type resistance &#40;R&#41; protein that is required for resistance to turnip crinkle virus &#40;TCV&#41; in Arabidopsis. To gain insights into HRT&#45;mediated signaling, a genetic screen was performed to identify mutants compromised for recognition of the TCV avirulence factor, which is encoded by an endogenously expressed TCV coat protein &#40;CP&#41; transgene. One of the mutants identified, crt1 &#40;compromised recognition of TCV&#41;, carries a prematurely terminated novel ATPase protein, in which a recently recognized GHKL ATPase motif termed a &#8216;Bergerat fold&#8217; has been deleted. Following TCV infection, crt1 plants developed a spreading hypersensitive response &#40;HR&#41; and failed to prevent viral replication and spread to the systemic portions of the plants. Systemic viral movement and disease symptoms were even more pronounced when two closely related CRT1 homologs were partially silenced. In addition, silencing CRT1 and its homologs in the Col&#45;0 background led to delayed HR against avirulent Pseudomonas syringae carrying avrRpt2 and the crt1 mutation suppressed spontaneous cell death in the ssi4 mutant, which contains a gain&#45;of&#45;function mutation in a TIR&#45;NBS&#45;LRR type R gene. In summary, this novel CRT1 family appears to play an important function in activating defenses against pathogen infection.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Abscisic acid plays an important role in plant disease susceptibility</div>

<div>

	<span style="text-decoration: underline" >
	Jun Fan</span>
	<sup>1</sup>, 

	<span  >
	Chris Lamb</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Disease and Stress Biology, John Innes Centre, Norwich, U.K.

</div>

<div>We have used activation tagging to investigate the complex signaling networks underlying disease resistance in Arabidopsis. From a screen of &#126; 8,000 lines, we identified a <span style="font-style: italic">constitutive disease suceptibility</span> &#40;<span style="font-style: italic">cds2</span>&#41; mutant showing enhanced susceptibility to virulent <span style="font-style: italic">Pseudomonas syringae</span> pv. <span style="font-style: italic">maculicola</span> ES4326 &#40;<span style="font-style: italic">Psm</span>&#41; and the equivalent strain carring avirulence gene <span style="font-style: italic">avrRpm1</span> &#40;<span style="font-style: italic">Psm&#45;avrRpm1</span>&#41;.  Systemic acquired resistance and systemic induction of PR protein transcripts were also compromised in <span style="font-style: italic">cds2</span> plants. The T&#45;DNA insertion adjacent to <span style="font-style: italic">AtNCED5</span>, one of the six genes encoding the abscisic acid biosynthetic enzyme 9&#45;cis&#45;epoxycarotinoid dioxygenase, led to a marked increase in transcript level and &#126;2 fold enhanced abscisic acid levels. Over expression of <span style="font-style: italic">AtNCED</span> on either constitutive or inducible promoters recreated the enhanced disease susceptibility phenotype. These findings prompted further investigation of the role of abscisic acid in disease susceptibility. Several <span style="font-style: italic">AtNCED</span> genes are induced by pathogen and abscisic acid accumulates strongly in a compatible interaction. Furthermore, exogenous abscisic acid increases bacterial growth and the abscisic acid biosynthetic mutant <span style="font-style: italic">aba3</span> shows reduced susceptibility to virulent <span style="font-style: italic">P.syringae</span> pv. <span style="font-style: italic">tomato</span> DC3000. Finally, abscisic acid synergises with jasmonic acid and exhibits a complex antagonistic relationship with salicylic acid. Our findings provide genetic evidence that abscisic acid has an important role in disease susceptibility.</div>

</div>





<div class="abs">

<div style="font-weight:bold">TRILATERAL TIES: PLANT GROWTH PROMOTION BY INTERACTION OF SOIL BACTERIA AND AMOEBA</div>

<div>

	<span style="text-decoration: underline" >
	Tony R. Schaeffner</span>
	<sup>1</sup>, 

	<span  >
	Kristin Krome</span>
	<sup>2</sup>, 

	<span  >
	Katja Rosenberg</span>
	<sup>2</sup>, 

	<span  >
	Birgit Geist</span>
	<sup>1</sup>, 

	<span  >
	Stefan Scheu</span>
	<sup>2</sup>, 

	<span  >
	Michael Bonkowski</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>GSF Research Center for Environment and Health - Institute of Biochemical Plant Pathology, 85764 Neuherberg/M&#252;nchen, Germany, 

	<sup>2</sup>Darmstadt University of Technology - Institute of Zoology - Rhizosphere Ecology Group, 64287 Darmstadt, Germany

</div>

<div>Increased plant growth due to soil protozoa&#45;soil bacteria interactions in the rhizosphere is well documented and generally has been assigned to nutrient release from consumed bacterial biomass. However, protozoa are highly selective grazers on bacteria and more recent investigations indicate that protozoa&#45;induced changes in bacterial community composition, rather than nutrient effects may be additionally responsible for plant growth promotion in presence of protozoa. &#13;<br/>In order to gain a better understanding of the underlying mechanisms in protozoa&#45;bacteria&#45;plant interactions, we established a sand&#45;litter system with Arabidopsis thaliana grown either completely sterile, in presence of soil bacteria or bacteria plus the amoeba Acanthamoeba castellanii, an abundant protozoan predator of bacteria. Compared to sterile treatments, plant rosette diameter, shoot biomass and seed production increased significantly in presence of bacteria, but was further enhanced in presence of amoebae. Measurements of plant&#45;available nitrogen in soil indicated that the plant growth promoting effect of amoebae cannot be exclusively explained by an enhanced nutrient availability. However, analyses of the bacterial communities demonstrated shifts in bacterial species composition in presence of amoebae. &#13;<br/>Analyses of the plant transcriptome revealed  that a number of genes involved in secondary metabolism, which were up&#45;regulated by soil bacteria, were down&#45;regulated in presence of amoebae. In addition, several growth associated genes were up&#45;regulated under the latter conditions. We hypothesize that strong grazing&#45;induced shifts in bacterial community composition most likely ameliorated the plant defense&#45;response to bacteria by down&#45;regulation of plant defense genes and genes involved in the production of plant secondary metabolites, resulting in a plant growth promoting effect of protozoa.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Chip&#45;based cloning of &#34;Suppressor of SNI1 3&#34; &#40;&#34;SSN3&#34;&#41;</div>

<div>

	<span style="text-decoration: underline" >
	Shui Wang</span>
	<sup>1</sup>, 

	<span  >
	Wendy E. Durrant</span>
	<sup>1</sup>, 

	<span  >
	Xinnian Dong</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Biology, Duke University, Box 90338, Durham, NC 27708, USA

</div>

<div>The Arabidopsis &#34;Nonexpresser of PR genes 1&#34; &#40;&#34;NPR1&#34;&#41; is a key positive regulator of salicylic acid &#40;SA&#41;. The &#34;npr1&#34; mutant abolishes the SA&#45;induced expression of &#34;Pathogenesis&#45;related&#34; &#40;&#34;PR&#34;&#41; genes. In the &#34;npr1&#45;1&#34; background, the &#34;sni1&#34; &#40;&#34;suppressor of npr1&#45;1, inducible 1&#34;&#41; mutant restored the inducibility of the &#34;PR&#34; genes. This indicates that an SA&#45;dependent and &#34;NPR1&#34;&#45;independent pathway is involved in the expression of PR genes. To identify those genes belong to this pathway, a genetic screen was conducted for &#34;Suppressors of SNI1&#34; &#40;&#34;SSN&#34;&#41; genes. The &#34;SSN1&#34; &#40;&#34;RAD51D&#34;&#41; and &#34;SSN2&#34; have successively been cloned. Here, we report the cloning of &#34;SSN3&#34; gene. The &#34;ssn&#34; mutants were generated by fast neutron bombardment and the &#34;ssn3&#34; locus has 6 alleles: 6C&#45;1, 9F&#45;1, 13F&#45;1, 13G&#45;1, 17A&#45;1 and 21B&#45;1. Using Arabidopsis Tiling Array &#40;Affymetrix&#41;, a deletion &#40;more than 10 kb&#41; was found in 9F&#45;1 allele. In addition, mutations were identified in other alleles &#40;6C&#45;1, 13F&#45;1, 13G&#45;1 and 21B&#45;1&#41; by sequencing their genomic DNA. So far, the &#34;ssn3&#34; mutants have been complemented by the &#34;SSN3&#34; cDNA driven by the &#34;35S&#34; promoter. According to our data, SSN1, SSN2 and SSN3 may form a chromatin remodeling complex to sense the SA signal and to promote the expression of &#34;PR&#34; genes. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Signaling crosstalk between PAMP&#45;triggered immunity and UVB&#47;sucrose stress responses</div>

<div>

	<span style="text-decoration: underline" >
	Yusuke Saijo</span>
	<sup>1</sup>, 

	<span  >
	Nico Tintor</span>
	<sup>1</sup>, 

	<span  >
	Francesca Ceron</span>
	<sup>1</sup>, 

	<span  >
	Hye Sup Yun</span>
	<sup>1</sup>, 

	<span  >
	Silke Robatzek</span>
	<sup>1</sup>, 

	<span  >
	Paul Schulze-Lefert</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max-Planck Institute for Plant Breeding Research, Cologne, Germany

</div>

<div>Plants recognize encounters with microbes by detection of molecular patterns that are conserved in many microbial species, designated pathogen&#45;associated molecular patterns &#40;PAMPs&#41;. PAMP perception triggers immune responses that restrict microbe invasion and growth. Bacterial flagellin and elongation factor Tu &#40;EF&#45;Tu&#41; act as effective elicitors of PAMP&#45;triggered immune responses in Arabidopsis. However, constitutive stimulation of PAMP&#45;signaling leads to growth retardation, suggesting that defense activation is achieved at the cost of growth&#45;related processes. We show that UVB&#45; and sucrose&#45;induced flavonoid accumulation, a characteristic response to these abiotic stresses, is abolished in young seedlings in the presence of PAMPs. PAMP&#45;signaling activation diminishes UVB&#45; and sucrose&#45;induced elevation in the mRNA levels of &#34;chalcone synthase&#34; &#40;&#34;chs&#34;&#41;, encoding for a key enzyme in the flavonoid biosynthetic pathway. Accordingly, CHS steady&#45;state levels do not increase upon exposure to PAMPs and UVB&#47;sucrose. The two PAMPs Flagellin and EF&#45;Tu confer similar repression of abiotic stress&#45;induced flavonoid accumulation that is dependent on the presence of cognate PAMP receptors FLS2 and EFR, respectively. Thus, it seems likely that PAMP&#45;mediated suppression of flavonoid accumulation occurs downstream of a presumed convergence point for multiple PAMP receptor&#45;initiated signaling pathways. Furthermore, the ease and robustness of flavonoid visualization and quantification in Arabidopsis seedlings provide an ideal system for mutant screens. We present the preliminary characterization of mutant plants defective in PAMP&#45;signaling and its crosstalk with UVB&#47;sucrose&#45;mediated control of flavonoid metabolism.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis SNI1 Complex Regulates both Gene Transcription and DNA Recombination During the Defence Response</div>

<div>

	<span style="text-decoration: underline" >
	Junqi Song</span>
	<sup>1</sup>, 

	<span  >
	Wendy Durrant</span>
	<sup>1</sup>, 

	<span  >
	Ek Han Tan</span>
	<sup>1</sup>, 

	<span  >
	Xinnian Dong</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Duke University, Durham, NC 27708, USA

</div>

<div>Systemic acquired resistance &#40;SAR&#41; is a general defence mechanism that confers long&#45;lasting and broad&#45;spectrum plant disease resistance that occurs after an initial infection.  Induction of SAR involves salicylic acid &#40;SA&#41;&#45;mediated activation of pathogenesis&#45;related &#40;PR&#41; genes. SNI1 &#40;suppressor of NPR1, inducible&#41; functions as a negative regulator of SAR required to dampen the basal PR gene expression. A suppressor of SNI1, RAD51D has been recently cloned that is involved in SA&#45;dependent NPR1&#45;indepent PR gene expression. Here we report the identification and characterization of another SNI1 suppressor, SSN2.  Map&#45;based cloning of SSN2 revealed that it encodes a novel protein containing a SWIM domain which is predicted to have DNA&#45;binding and protein&#45;protein interaction functions in different contexts. Physical interaction between SSN2, SNI1 and RAD51D indicate that these proteins can form a complex that plays a dual role in regulating both PR gene transcription and DNA recombination during the defense response. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Complex regulatory pathways control the basal defense response</div>

<div>

	<span style="text-decoration: underline" >
	Yiji Xia</span>
	<sup>1</sup>, 

	<span  >
	Huifen Zhu</span>
	<sup>1</sup>, 

	<span  >
	Yongdun Xie</span>
	<sup>1,2</sup>, 

	<span  >
	Shengbing Wang</span>
	<sup>1</sup>, 

	<span  >
	Xiaochun Ge</span>
	<sup>1,3</sup>, 

	<span  >
	Guojing Li</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Danforth Plant Science Center, 

	<sup>2</sup>China Agricultural University, 

	<sup>3</sup>Fudan University

</div>

<div>The basal defense response&#47;innate immune response can be trigged when host cells recognize microbe&#45;associated molecular patterns &#40;MAMPs&#41; and is delicately regulated through interplay of both positive and negative pathways.  The regulatory components that control basal resistance remain largely unknown.  Using functional genomics and reverse genetic approaches, we have identified several Arabidopsis genes that play positive or negative roles in basal immunity.  AtNUDT7, a Nudix protein that hydrolyzes nucleotide derivatives including NADH and ADPR, was found to be a negative regulator of basal immunity.  NUDT7 modulates two distinct branches of the defense response pathway and functions to prevent excessive cellular stimulation. The nudt7 mutation leads to perturbation of cellular redox homeostasis which potentiates an amplified defense response when provoked by biotic and other stresses. We have also found that heterotrimeric G protein play a positive role in activation of the basal defense response.  In addition, a nuclear gene encoding a transcriptional regulator that acts in chloroplast has been found to be involved in the defense response.  The transcription factor likely enhances expression of some plastid genes encoding ribosomal proteins. The study may shed light on how the photosynthetic organelle plays a role in disease resistance.  Our findings further reveal the complexity of the mechanisms that regulate basal disease resistance.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Transcriptional regulators in rhizobacteria&#45;induced systemic resistance &#40;ISR&#41;</div>

<div>

	<span style="text-decoration: underline" >
	Sjoerd Van der Ent</span>
	<sup>1,2</sup>, 

	<span  >
	Maria J. Pozo</span>
	<sup>1</sup>, 

	<span  >
	Bas W.M. Verhagen</span>
	<sup>1</sup>, 

	<span  >
	L.C. Van Loon</span>
	<sup>1</sup>, 

	<span  >
	Jurriaan Ton</span>
	<sup>1</sup>, 

	<span  >
	Corn&#233; M.J. Pieterse</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Plant-Microbe Interactions, Institute of Environmental Biology, Utrecht University, P.O.Box 800.84, 3508 TB Utrecht, the Netherlands, 

	<sup>2</sup>Center for Biosystems Genomics, the Netherlands

</div>

<div>Plants possess inducible resistance mechanisms through which they can regulate their defense response to pathogen attack. Colonization of <span style="font-style: italic">Arabidopsis thaliana</span> roots by non&#45;pathogenic <span style="font-style: italic">Pseudomonas fluorescens</span> WCS417r bacteria triggers a jasmonate&#45; and ethylene&#45;dependent induced systemic resistance &#40;ISR&#41; that is effective against a broad range of foliar pathogens. In the roots, the transcriptional activity of a large number of genes, is altered upon colonization by WCS417r. Bioassays revealed that one of the up&#45;regulated genes, the transcription factor &#40;TF&#41; <span style="font-style: italic">AtMYB72</span>, is essential for activation of ISR. Two independent <span style="font-style: italic">myb72</span> knockout mutants were incapable of mounting WCS417r&#45;mediated ISR against different pathogens. Constitutive expression of <span style="font-style: italic">AtMYB72</span> did not result in elevated resistance of the transformants, which demonstrates that AtMYB72 is required, yet not sufficient for ISR. Yeast two&#45;hybrid analysis revealed that AtMYB72 interacts in vitro with EIN3&#45;LIKE 3 &#40;EIL3&#41;, a TF that belongs to a family of TFs that play a role in ethylene signaling. Whether EIL3 is a parallel signaling component required for the onset of ISR is currently under investigation. &#13;<br/>&#13;<br/>Micro&#45;array studies revealed that ISR&#45;expressing leaves are primed for augmented expression of predominantly jasmonate&#45; and ethylene&#45;responsive genes. Promoter analysis of the primed genes showed that the ISR&#45;primed genes are enriched for a AtMYC2 binding motif, suggesting a regulatory role for this TF in ISR. Further indications for the involvement of AtMYC2 in ISR arose from a qPCR&#45;based TF profiling, which demonstrated that the systemic expression of several TFs, amongst which <span style="font-style: italic">AtMYC2</span>, is directly activated upon colonization of the roots by ISR&#45;inducing  WCS417r bacteria. Bioassays using <span style="font-style: italic">AtMYC2</span> knockout mutants <span style="font-style: italic">jin1&#45;1</span> and <span style="font-style: italic">jin1&#45;2</span> demonstrated that these <span style="font-style: italic">myc2</span> mutants were not able to mount WCS417r&#45;mediated systemic resistance, indicating that AtMYC2 is indeed required for ISR.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">The Arabidopsis MAP kinase 4 substrate, MKS1, Contribute to Basal Resistance and Exist in Complex with WRKY33 in planta</div>

<div>

	<span style="text-decoration: underline" >
	Jin-Long Qiu</span>
	<sup>1</sup>, 

	<span  >
	Klaus Petersen</span>
	<sup>1</sup>, 

	<span  >
	Berthe Katrine Friil</span>
	<sup>1</sup>, 

	<span  >
	Juri Lutje</span>
	<sup>1</sup>, 

	<span  >
	John Mundy</span>
	<sup>1</sup>, 

	<span  >
	Morten Petersen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Molecular Biology, Copenhagen University, Copenhagen Biocenter, Ole Maaloes Vej 5, 2200 Copenhagen N, Denmark

</div>

<div>Recently, we identified MKS1 as a MPK4 substrate and analyses of transgenic plants and genome&#45;wide transcript profiling indicated that MKS1 is required for full SA&#45;dependent resistance in mpk4 mutants, and that overexpression of MKS1 in wild&#45;type plants is sufficient to activate SA&#45;dependent resistance &#40;Andreasson et al., 2005&#41;. In addition, the transcription factors WRKY25 and WRKY33 were found to interact with MKS1 in yeast suggesting that these two WRKY factors could function downstream of MPK4 in controlling the expression of various defense genes &#40;Andreasson et al., 2005&#41;. Here we demonstrate that MKS1 also play a pivotal role in basal resistance. Loss of MKS1 leads to enhanced growth of virulent strains of P. syringae and PR1 accumulation is severely repressed in the initial phases of infection on mks1 mutants. In addition, we find that MKS1 interacts with WRKY33 in vivo suggesting that MPK4 may control gene expression through WRKY33 via MKS1. PR1 accumulates to wild&#45;type levels in wrky33 mutants infected with both virulent and avirulent strains of P. syringae. However, comparative microarray analysis of wrky33 and wild type shows that other defense genes fails to accumulate in wrky33 after SA treatment but also after virulent and avirulent infections. Thus WRKY33 is required for induction of only a small subset of SA &#45;regulated defense genes. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Molecular mechanisms underlying the activation of NBS&#45;LRR proteins</div>

<div>

	<span style="text-decoration: underline" >
	Roger Innes</span>
	<sup>1</sup>, 

	<span  >
	Jules Ade</span>
	<sup>1</sup>, 

	<span  >
	Brody DeYoung</span>
	<sup>1</sup>, 

	<span  >
	Thomas Burke</span>
	<sup>1</sup>, 

	<span  >
	Tom Ashfield</span>
	<sup>1</sup>, 

	<span  >
	Lauren Galloway</span>
	<sup>1</sup>, 

	<span  >
	Michelle Metcalf</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Indiana University, Bloomington, Indiana, USA

</div>

<div>This talk will focus on the molecular mechanisms by which plant disease resistance &#40;R&#41; proteins mediate pathogen recognition. In Arabidopsis, the <span style="font-style: italic">RPM1</span>gene mediates resistance to <span style="font-style: italic">Pseudomonas syringae</span> strains that express either AvrB or AvrRpm1, while in soybean recognition of these two proteins is mediated by distinct, but closely linked R genes, <span style="font-style: italic">Rpg1&#45;b</span> and <span style="font-style: italic">Rpg1&#45;r</span>. With the help of several collaborators, we have cloned <span style="font-style: italic">RPM1</span> and <span style="font-style: italic">Rpg1&#45;b</span> and expect to soon clone <span style="font-style: italic">Rpg1&#45;r</span>. Phylogenetic analyses revealed that <span style="font-style: italic">RPM1</span> and <span style="font-style: italic">Rpg1&#45;b</span> are not orthologous and evolved the ability to detect AvrB independently.  We were thus interested in determining whether the recognition mechanisms employed by RPM1 and Rpg1&#45;b were the same or different. RPM1 has been shown to detect both AvrB and AvrRpm1 indirectly, via monitoring the status of a second Arabidopsis protein, RIN4. In addition, it has been shown that a third <span style="font-style: italic">P. syringae</span> protein AvrRpt2 can block RPM1&#45;mediated resistance by proteolytically degrading RIN4. To test whether the soybean Rpg1&#45;b protein employs a recognition mechanism similar to that of RPM1, we identified soybean orthologues of <span style="font-style: italic">RIN4</span>, tested their interaction with AvrB, determined whether they were substrates of AvrRpt2, and assessed whether Rpg1&#45;b function was suppressed by AvrRpt2.  Our data indicate that Rpg1&#45;b likely detects AvrB by a mechanism very similar to RPM1, even though it evolved this ability independently.  We are also investigating how the Arabidopsis RPS5 protein is activated by the <span style="font-style: italic">P. syringae</span> effector protein AvrPphB.  We have previously shown that AvrPphB functions as a protease to cleave the Arabidopsis protein kinase PBS1 and that this cleavage is required to activate RPS5.  We are now investigating how this cleavage event triggers RPS5 signaling. <br/><br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Penetration Resistance: The First Line of Defense Against Invasive Pathogens.</div>

<div>

	<span style="text-decoration: underline" >
	Shauna Somerville</span>
	<sup>1</sup>, 

	<span  >
	Monica Stein</span>
	<sup>1</sup>, 

	<span  >
	Matt Humphry</span>
	<sup>1</sup>, 

	<span  >
	Laurent Zimmerli</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Carnegie Institution, Stanford, CA, U.S.A.

</div>

<div>A majority of host plants are resistant to the majority of pathogens in their environment. The underlying mechanisms of this form of disease resistance are poorly studied. Working collaboratively, the Paul Schulze&#45;Lefert lab &#40;Cologne, Germany&#41;, Hans Thordal&#45;Christensen &#40;Copenhagen, Denmark&#41; lab and our lab identified mutants of the model plant <span style="font-style: italic">Arabidopsis thaliana</span> that were compromised in their resistance to the powdery mildew fungus, <span style="font-style: italic">Blumeria graminis</span> f. sp. <span style="font-style: italic">hordei</span>, which is pathogenic only on barley. Several Arabidopsis genes, not previously implicated in plant resistance mechanisms, have been identified by this approach, including <span style="font-style: italic">PEN1</span> &#40;&#61;<span style="font-style: italic">SYP121</span>&#41;, a syntaxin and functional homologue of the barley <span style="font-style: italic">ROR2</span> gene; <span style="font-style: italic">PEN2</span>, a glycosyl hydrolase; and <span style="font-style: italic">PEN3</span> &#40;&#61;<span style="font-style: italic">PDR8</span>&#41;, an ABC transporter. In addition, <span style="font-style: italic">MLO2</span> was identified as the functional homologue of the barley <span style="font-style: italic">MLO</span> gene, which confers penetration resistance when inactivated. Several of these mutants permit enhanced penetration success by additional pathgens including <span style="font-style: italic">Erysiphe pisi</span>, a pathogen of pea, and <span style="font-style: italic">Phytophthora infestans</span>, the potato late blight pathogen. The plasma membrane localized MLO, PEN1 and PEN3 proteins accumulate at penetration sites in inoculated plant tissues. These observations suggest that plants mount effective, broad spectrum and multi&#45;genic defenses designed to block pathogen invasion across the host cell wall and entry into cells. Over evolutionary time, virulent species, such as <span style="font-style: italic">Golovinomyces cichoracearum</span>, a pathogen of Arabidopsis, must have acquired the ability to avoid these defenses by stealth or to be insensitive to them.<br/><br/>Current Addresses: Matt Humphry, Max Planck Institute for Plant Breeding, Cologne, Germany; Laurent Zimmerli, Taiwan National University, Taipei, Taiwan</div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional Analysis of Arabidopsis WRKY38 and WRKY62 Transcription Factors in Plant Defense Responses</div>

<div>

	<span  >
	Kang-Chang Kim</span>
	<sup>1</sup>, 

	<span  >
	Baofang Fan</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Zhixiang Chen</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Purdue University, West Lafayette, IN 47907 USA

</div>

<div>Arabidopsis WRKY38 and WRKY62 encode two structurally similar Type III WRKY transcription factors.  WRKY38 and WRKY62 are induced by salicylic acid &#40;SA&#41; in an NPR1&#45;dependent manner.  The two WRKY genes also exhibit SA&#45; and NPR1&#45;dependent induction by the bacterial pathogen Pseudomonas syringae.  WRKY38 and WRKY63 function as transcriptional activators in plant cells.  Both loss&#45;of&#45;function mutants and overexpression lines for WRKY38 and WRKY62 have been generated for functional analysis of their roles in plant defense responses.  WRKY38 and WRKY62 interact with a histone deacetylase in both yeast and plant cells.  Expression of the gene encoding the histone deacetylase is also induced by P. syringae and the stability of its transcripts in pathogen&#45;infected plants is dependent on SA and NPR1.  T&#45;DNA insertion mutant and overexpression lines for the histone deacetylase gene have also been generated.  Characterization of the mutants and overexpression lines for the genes encoding WRKY38, WRKY62 and their interacting histone deacetylase partner will be described.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Positioning Arabidopsis EDS1 in pathogen and oxidative stress responses</div>

<div>

	<span style="text-decoration: underline" >
	Jane Parker</span>
	<sup>1</sup>, 

	<span  >
	Lennart Wirthmueller</span>
	<sup>1</sup>, 

	<span  >
	Steffen Rietz</span>
	<sup>1</sup>, 

	<span  >
	Michael Bartsch</span>
	<sup>1</sup>, 

	<span  >
	Enrico Gobbato</span>
	<sup>1</sup>, 

	<span  >
	Ana Garcia</span>
	<sup>1</sup>, 

	<span  >
	Christine Foyer</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>Max-Planck Institute for Plant Breeding Research, Cologne, Germany, 

	<sup>2</sup>School of Agriculture, University of Newcastle, Newcastle upon Tyne, UK

</div>

<div>Our interest is in the control of Arabidopsis defences once pathogens have overcome resistance at the cell surface &#40;normally triggered by PAMPs&#41; and started to colonize host cells. Low level post&#45;invasive &#40;basal&#41; resistance to biotrophic and hemi&#45;biotrophic pathogens depends on the EDS1 family of nucleo&#45;cytoplasmic proteins &#40;&#45; EDS1, PAD4 and SAG101&#41;. These regulators restrict pathogen growth in the absence of cell death and generate signals leading to the priming of systemic defences in uninfected parts of the plant. They also serve as essential components of localised cell death and resistance triggered by intracellular TIR&#45;NB&#45;LRR type immune receptors. In all of these functions, EDS1 controls the production of salicylic acid &#40;SA&#41; and other signal intermediates as well as the fine balance between SA and jasmonic acid&#47;ethylene pathway activation. Several pieces of evidence point to an EDS1&#47;PAD4 activity in transducing ROS&#45;derived signals in pathogen and photo&#45;oxidative stress responses. Characterisation of additional components of the EDS1 pathway &#40;eg. FMO1, a flavin&#45;dependent monooxygenase&#41; and of EDS1&#45;PAD4 dual over expression lines reinforce a connection between EDS1 and the regulation of ROS homeostasis and signalling that impacts significantly on plant development. In order to position EDS1 in TIR&#45;NB&#45;LRR triggered defence and cell death, we have examined RPS4, a prototypical TIR type NB&#45;LRR receptor that recognizes the secreted bacterial effector, AvrRps4. RPS4 protein accumulates to low levels as endomembrane&#45;associated and nuclear pools in unchallenged cells. Its amounts in either compartment do not change substantially after pathogen challenge. Our data suggest that RPS4 interception of AvrRps4 occurs indirectly and outside the nucleus but that a nuclear &#8216;activated&#8217; RPS4 pool is needed to trigger resistance. We present further evidence that EDS1 is an indispensible intermediate between RPS4 activation and RPS4&#45;mediated transcriptional reprogramming in the nucleus.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A microarray assisted screen to identify mutants that compromise nonhost resistance</div>

<div>

	<span style="text-decoration: underline" >
	Li Kang</span>
	<sup>1</sup>, 

	<span  >
	Keri Wang</span>
	<sup>1</sup>, 

	<span  >
	Kirankumar Mysore</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Samuel Roberts Noble Foundation, Ardmore, OK USA

</div>

<div>Transgenic <span style="font-style: italic">Arabidopsis</span> lines overexpressing a fatty acid amide hydrolase <span style="font-style: italic">&#40;FAAH&#41;</span> were more susceptible to bacterial pathogens and lost resistance against nonhost pathogens. Uninoculated and mock inoculated <span style="font-style: italic">AtFAAH</span> overexpressors had lower amounts of both free and conjugated salicylic acid &#40;SA&#41; when compared with the wild&#45;type. Surprisingly, <span style="font-style: italic">AtFAAH</span> overexpressors accumulated significantly higher amounts of free SA, 12 hours after inoculation with bacteria, while the total SA &#40;free &#43; conjugated&#41; was still low. Gene expression studies revealed that transcripts of a number of disease resistance <span style="font-style: italic">&#40;R&#41;</span> genes as well as genes involved in SA biosynthesis and signaling were considerably lower in <span style="font-style: italic">AtFAAH</span> overexpressors compared to those in wild&#45;type plants. To further dissect the signal pathway mediated by AtFAAH in <span style="font-style: italic">Arabidopsis</span>, 130 genes that were expressed most differently between Col&#45;0 and <span style="font-style: italic">AtFAAH</span> overexpressor were further characterized. We obtained T&#45;DNA knockout lines for these 130 genes and screened them for compromised resistance against nonhost pathogen <span style="font-style: italic">Pseudomonas syringae</span> pv. <span style="font-style: italic">tabaci</span>. Five mutants that exhibited enhanced susceptibility to a nonhost pathogen were </div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis EDR1, EDR2 and EDR3 regulate plant defense responses to powdery mildew</div>

<div>

	<span  >
	Haozhen Nie</span>
	<sup>1</sup>, 

	<span  >
	Huirong Pan</span>
	<sup>1</sup>, 

	<span  >
	Yiping Wang</span>
	<sup>1</sup>, 

	<span  >
	Yingying Wu</span>
	<sup>1</sup>, 

	<span  >
	Chunpeng Yao</span>
	<sup>1</sup>, 

	<span  >
	Ting Zhao</span>
	<sup>1</sup>, 

	<span  >
	Roger Innes</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Dingzhong Tang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing China 100101, 

	<sup>2</sup>Department of Biology, Indiana University, Bloomington, IN, USA 47405

</div>

<div>The edr1, edr2, and edr3 mutants of Arabidopsis display enhanced disease resistance to the powdery mildew pathogen Erysiphe cichoracearum. Leaves from edr mutant plants form dramatic lesions upon infection with E. cichoracearum. Powdery mildew resistance mediated by edr1, edr2 and edr3 is SA dependent, but JA and ethylene independent. Among them, EDR1 encodes a CTR1&#45;like protein, consisting of an N&#45;terminal putative regulatory domain and a C&#45;terminal kinase domain. EDR1 displays kinase activity in vitro. EDR2 encodes a novel protein, consisting of a putative pleckstrin homology &#40;PH&#41; domain and a StAR &#40;Steroidogenic Acute Regulatory protein&#41;&#45;related lipid&#45;transfer &#40;START&#41; domain. The PH and START domains are implicated in lipid binding, suggesting that EDR2 may regulate defense responses through lipid signaling. EDR3 encodes a dynamin like protein &#40;DRP1E&#41;, consisting of a N&#45;terminal GTPase and a C&#45;terminal GTPase effector domain. Analysis of EDR3&#45;GFP fusion proteins revealed that EDR3 is at least partially localized to mitochondria, suggesting that EDR3 may regulate programmed cell death via modification of mitochondrial function. The edr3 phenotype suggests a mechanistic link between salicylic acid signaling, mitochondria and programmed cell death in plants. Currently, we are performing molecular and biochemical analyses on the EDR1, EDR2, EDR3 and hope to identify other components in these pathways using genetic and molecular approaches. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Tzs, a host range factor, is required for <span style="font-style: italic">Agrobacterium tumefaciens</span> growth under infection conditions</div>

<div>

	<span style="text-decoration: underline" >
	Hau-Hsuan Hwang</span>
	<sup>1,2</sup>, 

	<span  >
	Yu-Chen Liao</span>
	<sup>1</sup>, 

	<span  >
	Yi-Chun Chen</span>
	<sup>1</sup>, 

	<span  >
	Erh-Min Lai</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institute of Plant and Microbial Biology, Academia Sinica, Taipei, Taiwan, 

	<sup>2</sup>Department of Life Sciences, National Chung-Hsing University, Taichung, Taiwan

</div>

<div><span style="font-style: italic">Agrobacterium tumefaciens</span> is an organism capable of trans&#45;kingdom DNA transfer, transforming mainly plants but also other eukaryotic species, from fungi to human cells. Genetic transformation by <span style="font-style: italic">A. tumefaciens</span>, which in plants causes neoplastic growths called &#8220;crown gall&#8221;, results from the transfer and integration of a specific DNA fragment &#40;transferred DNA or T&#45;DNA&#41; from the bacterium into the plant genome. Here, we characterized a Ti&#45;plasmid encoded gene, <span style="font-style: italic">tzs</span> &#40;trans&#45;zeatin synthesizing&#41;, that is responsible for the synthesis of a plant hormone cytokinin in <span style="font-style: italic">A. tumefaciens</span> when bacteria were induced by a phenolic compound acetosyringone &#40;AS&#41;. To determine the role&#40;s&#41; of <span style="font-style: italic">tzs</span> in <span style="font-style: italic">A. tumefaciens</span> virulence, <span style="font-style: italic">tzs</span> deletion mutants &#40;&#9651;<span style="font-style: italic">tzs</span>&#45;277 and &#9651;<span style="font-style: italic">tzs</span>&#45;278&#41; were generated and characterized. Quantitative tumor assays demonstrated that <span style="font-style: italic">tzs</span> mutants decreased their ability to cause tumors on <span style="font-style: italic">Arabidopsis</span> roots and potato tubers. Additionally, <span style="font-style: italic">tzs</span> mutants reduce transient transformation efficiency in <span style="font-style: italic">Arabidopsis</span> roots, suggesting that Tzs is likely involved in step&#40;s&#41; prior to T&#45;DNA integrations. Interestingly, the <span style="font-style: italic">tzs</span> mutants are impaired in cell viability and&#47;or growth in both AS&#45;induced and infection conditions. Tzs protein is constitutively expressed and is up regulated by AS either in rich or minimal media. Tumor assays on various plant species were also tested to determine if Tzs is a host range factor. The <span style="font-style: italic">tzs</span> mutants were able to enhance transformation efficiency on green pepper and cowpea stems, reduce transformation efficiency on white radish, but not on other plant species tested. In summary, <span style="font-style: italic">tzs</span> may be required for <span style="font-style: italic">A. tumefaciens</span> growth under infection conditions and may play different roles when <span style="font-style: italic">A. tumefaciens</span> infects different kinds of plants.      &#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Little and large pathogen effectors in plant resistance and susceptibility</div>

<div>

	<span style="text-decoration: underline" >
	Jonathan Jones</span>
	<sup>1</sup>, 

	<span  >
	Kee Sohn</span>
	<sup>1</sup>, 

	<span  >
	Alexandre Robert-Seilaniantz</span>
	<sup>1</sup>, 

	<span  >
	Lionel Navarro</span>
	<sup>1</sup>, 

	<span  >
	Rajendra Bari</span>
	<sup>1</sup>, 

	<span  >
	Rita Lei</span>
	<sup>1</sup>, 

	<span  >
	Alejandra Rougon</span>
	<sup>1</sup>, 

	<span  >
	Adnane Nemri</span>
	<sup>1</sup>, 

	<span  >
	Georgina Fabro</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>The Sainsbury Laboratory, JIC, Colney Lane, Norwich NR4 7UH, UK

</div>

<div>Plant pathogens use small molecules and also proteins to render their hosts susceptible.  Several pathogens either make plant hormones, or perturb host hormone signalling networks by other means.  To counteract pathogen activation of auxin signalling, plants induce microRNA miR393 that targets the TIR1 auxin receptor, to attenuate auxin sensitivity.  Furthermore, gibberellin made by fungal necrotrophic pathogens leads to attenuation of the defence response to necrotrophs by interference with jasmonic acid signalling.&#13;<br/>In addition, many bacteria and other pathogens use a specialized secretion system to deliver proteins into host cells that interfere with host defence.  We have taken advantage of the bacterial T3SS secretion system to investigate effectors from filamentous pathogens such as oomycetes.  Recent data on oomycete effector functions will be reported.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Recognition of Pathogen Effectors and Activation of Disease Resistance Signaling Pathways in Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Brian Staskawicz</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of California, Berkeley, USA

</div>

<div>A common feature of most phytopathogens is the ability to directly deliver effector proteins to the plant host to either modulate or suppress host defense signalling. In our laboratory, we are studying effector proteins from both bacterial and oomycete pathogens with the aim of understanding both pathogen virulence during a compatible interaction and effector recognition during an incompatible interaction. Specifically, we have focused on studying the activation of the RPS2 disease resistance signalling pathway after the recognition of the cognate Type Three Secretion Effector protein, AvrRPT2. We will present our most recent model describing the molecular events controlling AvrRPT2 effector recognition and the subsequent activation of the RPS2 disease resistance signalling pathway. In addition, we will describe our progress in the identification and characterization of effector proteins from Hyaloperonospora parasitica and our ability to deliver oomycete effector proteins to Arabidopsis thaliana via the Pseudomonas syringae TTSS. Finally, we will present data to demonstrate that plants have evolved a common mechanism to inhibit growth of pathogens with varied lifestyles. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Insights about the role of CDR1 in the complex reiterative signal networks underlying disease resistance</div>

<div>

	<span style="text-decoration: underline" >
	Maria C. Romero-Puertas</span>
	<sup>1</sup>, 

	<span  >
	Chris J. Lamb</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Disease and Stress Biology Department, John Innes Center, Norwich NR4 7UH, UK

</div>

<div>From a screen of &#126;5,000 lines with a T&#45;DNA carrying cauliflower mosaic virus 35S enhancer, we identified Constitutive Disease Resistance 1 &#40;CDR1&#41; encoding an apoplastic aspartic protease, over&#8211;expression of which causes dwarfing and resistance to virulent Pseudomonas syringae.  These phenotypes reflect salicylic acid&#45;dependent activation of micro oxidative bursts and various defence&#45;related genes.  Antisense CDR1 plants were compromised for resistance to avirulent P. syringae and more susceptible to virulent strains than wild type.  The CDR1 protein accumulates in the apoplast in response to pathogen attack.  Induction of CDR1 generates a small mobile signal, and CDR1 action is blocked by the protease inhibitor pepstatin and by mutations in the protease active sites.  Thus CDR1 mediates a peptide signal system involved in the activation of inducible resistance mechanisms. We have explored how CDR1 is integrated within the disease resistance signal network to position CDR1 action stage relative to other key players such as small molecule signals like ROS, salicylic acid and signal transduction proteins. Thus, transgenic plants containing CDR1 under the control of the dexamethasone inducible TA promoter &#40;TApr::CDR1&#41; have been crossed with the following key mutant&#47;transgenic phenotypes: <span style="font-style: italic">npr1&#45;1</span>, <span style="font-style: italic">eds1&#45;1</span>, <span style="font-style: italic">pad4&#45;5</span>, <span style="font-style: italic">dir1&#45;1</span>, <span style="font-style: italic">rboh D</span>, <span style="font-style: italic">rboh F</span>, <span style="font-style: italic">rboh D&#47;F</span> and <span style="font-style: italic">sid 2&#45;2</span>. As final goal, ROS production, cell death and gene induction have been analyzed to determine if RBOH D&#47;F, NPR1, EDS, DIR, PAD4 and SA are required for both, local and systemic effects of CDR1.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Hormonal control of cell death and defense responses in the Arabidopsis lesion mimic mutant <span style="font-style: italic">vad1</span></div>

<div>

	<span style="text-decoration: underline" >
	Olivier Bouchez</span>
	<sup>1</sup>, 

	<span  >
	Carine Huard</span>
	<sup>1</sup>, 

	<span  >
	S&#233;verine Lorrain</span>
	<sup>2</sup>, 

	<span  >
	Dominique Roby</span>
	<sup>1</sup>, 

	<span  >
	Claudine Balagu&#233;</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>LIPM, Cactanet-Tolosan, France, 

	<sup>2</sup>University of Lausanne, Switzerland

</div>

<div>The lesion mimic mutant <span style="font-style: italic">vad1</span> &#40;<span style="font-style: italic">Vascular Associated Death</span>&#41; exhibits propagative HR&#45;like lesions along the vascular system, presents expression of defense&#45;related genes and increased resistance to different strains of <span style="font-style: italic">Pseudomonas syringae</span> pv <span style="font-style: italic">tomato</span>. <span style="font-style: italic">VAD1</span> encodes a novel putative membrane&#45;associated protein containing a GRAM domain &#40;a lipid&#45; or protein&#45;binding signaling domain&#41;. VAD1 might constitute a new potential component of the signaling pathways leading to HR&#47;resistance &#40;Lorrain et al, 2004&#41;. &#13;<br/>To elucidate the function of VAD1, we performed overexpression tests in the heterologous system <span style="font-style: italic">Nicotiana benthamiana</span>. Results indicate that VAD1 overexpression leads to a delay in HR symptoms appearance.&#13;<br/>The placement of this gene within defense signaling pathways was also addressed. We previously demonstrated that crosses of <span style="font-style: italic">vad1</span> with mutants affected in the SA pathway &#40;<span style="font-style: italic">NahG, sid1</span>&#41; drastically alter the vad1&#45;associated phenotypes &#40;Lorrain et al, 2004&#41;. These results, confirmed by the analysis of the double mutant <span style="font-style: italic">vad1&#47;sid2</span>, indicate that SA is a key component in the <span style="font-style: italic">vad1</span>&#45;associated phenotypes. We demonstrate here that ethylene &#40;ET&#41;, an other defense signal, is also an essential component in the <span style="font-style: italic">vad1</span>&#45;associated phenotypes. Analysis of the progeny from crosses between <span style="font-style: italic">vad1</span> plants and ET biosynthesis and signaling mutants revealed that the <span style="font-style: italic">vad1</span> cell death and defense phenotypes are dependent on ET biosynthesis and signaling pathways. These results suggest that VAD1 could act as an integrative node in hormonal signalling, with ET acting in concert with SA, as a positive regulator of cell death propagation.&#13;<br/>In order to identify new actors of cell death and defense pathways, search for suppressors of <span style="font-style: italic">vad1</span>&#45;associated phenotypes was initiated. Preliminary results will be presented on the characterization of identified mutants. &#40;Lorrain et al., 2004, Plant Cell, 16, 2217&#45;2232&#41;.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">S&#45;Nitrosoglutathione Reductase Affords Protection against Pathogens in Arabidopsis, Both Locally and Systemically</div>

<div>

	<span style="text-decoration: underline" >
	M. Carme Espunya</span>
	<sup>1</sup>, 

	<span  >
	Christine Rust&#233;rucci</span>
	<sup>2</sup>, 

	<span  >
	Maykelis D&#237;az</span>
	<sup>1</sup>, 

	<span  >
	Matthieu Chabannes</span>
	<sup>3</sup>, 

	<span  >
	M. Carmen Mart&#237;nez</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Universitat Aut&#242;noma de Barcelona, 

	<sup>2</sup>Universit&#233; Jules Verne-Picardie Sciences, 

	<sup>3</sup>John Innes Center

</div>

<div>Nitric oxide and S&#45;nitrosothiols &#40;SNOs&#41; are widespread signaling molecules that regulate immunity in animals and plants. Levels of SNOs &#8220;in vivo&#8221; are controlled by nitric oxide synthesis &#40;which in plants is achieved by different routes&#41; and by S&#45;nitrosoglutathione turnover, which is mainly performed by the S&#45;nitrosoglutathione reductase &#40;GSNOR&#41;. GSNOR is encoded by a single&#45;copy gene in Arabidopsis. We report here that transgenic plants with decreased amounts of GSNOR&#40;using antisense strategy&#41; show enhanced basal resistance against &#8220;Peronospora parasitica Noco2&#8221; &#40;oomycete&#41;, which correlates with higher levels of intracellular SNOs and constitutive activation of the pathogenesis related gene, PR&#45;1. Moreover, systemic acquired resistance is impaired in plants overexpressing GSNOR and enhanced in the antisense plants, and this correlates with changes in the SNO content both in local and systemic leaves.We also show that GSNOR is localized in the phloem and, thus, could regulate systemic acquired resistance signal transport through the vascular system. Our data corroborate the data from other authors that GSNOR controls SNO &#8220;in vivo&#8221; levels, and shows that SNO content positively influences plant basal resistance and resistance&#45;gene&#45;mediated resistance as well. These data highlight GSNOR as an important and widely utilized component of resistance protein signaling networks conserved in animals and plants &#40;1&#41;.<br/><br/>&#40;1&#41; Rust&#233;rucci &#34;et al&#34;. &#40;2007&#41; Plant Physiol. 143:1282&#45;1292<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Arabidopsis TGA1 and TGA4 are required for NPR1&#45;independent Disease Resistance</div>

<div>

	<span style="text-decoration: underline" >
	Jinman Liu</span>
	<sup>1</sup>, 

	<span  >
	Yu Ti Cheng</span>
	<sup>2</sup>, 

	<span  >
	Dongling Bi</span>
	<sup>1</sup>, 

	<span  >
	Xin Li</span>
	<sup>2</sup>, 

	<span  >
	Yuelin Zhang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute of Biological Science, Beijing, China, 

	<sup>2</sup>Michael Smith Labs, University of British Columbia, Canada

</div>

<div>Systemic acquired resistance &#40;SAR&#41; is a secondary response following the R protein&#45;mediated HR and it provides general systemic resistance against a broad&#45;spectrum of pathogens. Salicylic acid &#40;SA&#41; is a necessary and sufficient signal to induce the SAR response, which includes the up&#45;regulation of PR &#40;pathogenesis&#45;related&#41; genes. NPR1 is required for SA&#45;induced PR gene expression and pathogen resistance. NPR1 interacts with a subgroup of TGA transcription factors, TGA2, TGA5 and TGA6.  We previously showed that induction of PR gene expression and pathogen resistance by the SA analog INA is blocked in the tga2&#45;1 tga5&#45;1 tga6&#45;1 triple knockout mutant, indicating that TGA2, TGA5 and TGA6 are essential for SAR like NPR1. &#13;<br/>The Arabidopsis genome contains seven additional TGA transcription factors in addition to TGA2, TGA5 and TGA6. Sequence analysis indicates that TGA1 is closely related to TGA4. We found that the tga1 tga4 double mutant plants were more susceptible to the virulent bacterial pathogen Pseudomonas syringae pv maculicola ES4326 &#40;P.s.m. ES4326&#41;, although the SA&#45;induced PR gene expression is not affected in the double mutant. To determine whether TGA1 and TGA4 function in the NPR1&#45;independent resistance pathway, we crossed the tga1 tga4 double mutant with the snc1 npr1 double mutant in which NPR1&#45;independent resistance against P.s.m. ES4326 is constitutively activated. snc1 contains a gain&#45;of&#45;function mutation in a TIR&#45;NBS&#45;LRR&#45;type of R&#45;protein. In the snc1 npr1 tga1 tga4 quadruple mutant, the constitutive resistance to P.s.m. ES4326 in snc1 npr1 is blocked by the tga1 tga4 double mutation, suggesting that TGA1 and TGA4 are required for the NPR1&#45;independent resistance. We have also created the tga1 tga4 npr1 triple mutant. The triple mutant is more susceptible to P.s.m. ES4326 than the tga1 tga4 and npr1 alone, and displayed enhanced susceptibility to the oomycete pathogen Paranospora parasitica EMWA1 compared to the tga1 tga4 and npr1 mutants, further suggesting that TGA1 and TGA4 is important for the regulation of NPR1&#45;independent resistance.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Negative Regulation of Pathogen Resistance by a Replication Factor Subunit&#45;like Protein in Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Shitou Xia</span>
	<sup>1</sup>, 

	<span  >
	JinGui Chen</span>
	<sup>2</sup>, 

	<span  >
	Xin Li</span>
	<sup>3</sup>, 

	<span  >
	Yuelin Zhang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute of Biological Science, Beijing, China, 

	<sup>2</sup>Department of Botany, University of British Columbia, Canada, 

	<sup>3</sup>Michael Smith Labs, University of British Columbia, Canada

</div>

<div>Systemic acquired resistance &#40;SAR&#41; is a plant immune response induced after a local infection by necrotizing pathogens. Expression of SAR in plants involves up&#45;regulation of many Pathogenesis&#45;Related &#40;PR&#41; genes, which work in concert to confer resistance to a broad spectrum of pathogens. Salicylic acid &#40;SA&#41; is an essential signal for the onset of SAR. Because constitutive activation of SAR is detrimental to plants, expression of PR genes must be tightly regulated. In an EMS mutagenesis screen to look for negative regulators of SAR, we found a mutant hypersensitive to INA, an analogue of SA. This mutant has enhanced induction of PR genes and display enhanced resistance against Peronospora parasitica &#40;P.p.&#41; Noco2, an oomycete pathogen. The gene was identified by map&#45;based cloning and found to encode a replication factor subunit&#45;like protein &#40;RFL&#41; and the mutant was named rfl&#45;1. rfl&#45;1 mutant plants are smaller than wild type plants with narrower leaves and petals. In the epidermis of cotyledons and true leaves, there are less cells. In the root elongation zone, the number of cortex cells is also lower, indicating that the mutated RFL slows down the cell proliferation and RFL plays important roles in DNA replication. Transforming wild type RFL gene into rfl&#45;1 revert the mutant phenotypes of rfl&#45;1 to wild type, suggesting that the rfl&#45;1 mutation is responsible for the mutant phenotypes observed. As RFL genes may be involved in the replication&#45;coupled chromatin assembling, we propose that chromatin assembly and remodeling are required for the negative regulation of PR genes. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Screening for resistance to bacterial and fungal pathogens in Rice&#45;Arabidopsis FOX lines overexpressing full&#45;length rice cDNA</div>

<div>

	<span style="text-decoration: underline" >
	Masaki Mori</span>
	<sup>1</sup>, 

	<span  >
	Joseph Gogo Dubouzet</span>
	<sup>1</sup>, 

	<span  >
	Satoru Maeda</span>
	<sup>1</sup>, 

	<span  >
	Miki Otake</span>
	<sup>1</sup>, 

	<span  >
	Takanari Ichikawa</span>
	<sup>2</sup>, 

	<span  >
	Youichi Kondo</span>
	<sup>2</sup>, 

	<span  >
	Minami Matsui</span>
	<sup>2</sup>, 

	<span  >
	Kenji Oda</span>
	<sup>3</sup>, 

	<span  >
	Hiroshi Takatsuji</span>
	<sup>1</sup>, 

	<span  >
	Hirohiko Hirochika</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>National Institute of Agrobiological Sciences, Tsukuba, Japan, 

	<sup>2</sup>RIKEN PSC, Yokohama, Japan, 

	<sup>3</sup>RIBS, Okayama, Japan

</div>

<div>About 13,000 full&#45;length cDNAs of rice were mixed and ligated to an expression vector cassette. They were introduced into Arabidopsis by Agrobacterium&#45;mediated floral dip transformation to generate Rice&#45;Arabidopsis FOX &#40;full&#45;length cDNA over&#45;expressor&#41; lines. Three&#45;week&#45;old T2 seedlings of approximately 20,000 FOX lines were screened for resistance to a bacterial pathogen, <span style="font-style: italic">Pseudomonas syringae</span> pv <span style="font-style: italic">tomato</span> DC3000 &#40;Pst3000&#41;, by dip inoculation. About 100 resistant lines were selected after 2&#45;3 rounds of screening and the rice cDNAs inserted in their genomes were identified. Some of these genes that passed further evaluation have been re&#45;introduced into Arabidopsis and rice for overexpression to confirm the disease resistance phenotypes. At present, overexpression of three transgenic Arabidopsis &#40;Columbia&#41; lines expressing rice cDNAs exhibited repeatable resistance to Pst3000. Two of the rice cDNAs have, upon overexpression in rice &#40;Nipponbare&#41;, provided resistance to <span style="font-style: italic">Xanthomonas oryzae oryzae</span>, the pathogen that causes Rice Blight, in T0 generation. Overexpression of one cDNA that enabled resistance to <span style="font-style: italic">Xanthomonas</span> in rice had marked pleiotropic effects that included weak root regeneration, low fertility, and large grain. Seed progeny of these resistant lines will be re&#45;screened for resistance to <span style="font-style: italic">Xanthomonas</span> and other rice pathogens.&#13;<br/>In addition, approximately 12,000 FOX lines were screened for resistance to the fungal pathogen <span style="font-style: italic">Colletotrichum higginsianum</span>, whose infection mechanisms in Arabidopsis are similar to those of <span style="font-style: italic">Magnaporthe grisea</span> in rice. Both of these fungi form appressoria and penetration pegs. About 30 resistant lines were selected after 2&#45;3 rounds of screening and the rice cDNAs inserted in their genomes were identified. The selected lines are being subjected to other requisite analytical processes similar to that applied to the Pst3000&#45;resistant lines.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">A negative effect of RPM1 on the AvrRpt2&#47;RPS2&#45;mediated gene&#45;for&#45;gene resistance</div>

<div>

	<span style="text-decoration: underline" >
	Tack-Min Kwon</span>
	<sup>1</sup>, 

	<span  >
	Soon-Jae Jeong</span>
	<sup>1</sup>, 

	<span  >
	Young-Byung Yi</span>
	<sup>1</sup>, 

	<span  >
	Jaesung Nam</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Faculty of Molecular Biotechnology, Dong-A University, Busan 604-714, Korea

</div>

<div>The TTSS effecter, AvrRpt2, targets and eliminates RIN4 that is not only a positive regulator for basal defense but also a negative regulator for RPS2&#45;mediated resistance. Therefore, if P. syringae expressing avrRpt2 infects host plant lacking resistance gene RPS2, AvrRpt2&#45;mediated elimination of RIN4 suppresses a basal defense and results in a hospital environment for propagation of pathogen. However, when RPS2 is present in the host plants, the elimination of RIN4 triggers RPS2&#45;mediated effective defenses including hypersensitive response &#40;HR&#41; and lead to resistance against pathogens. Kinetics of RPS2&#45;mediated HR reveals that AvrRpt2&#45;mediated elimination of RIN4 occurs in 3 &#45; 5 hr post infiltration of P. syringae &#40;avrRpt2&#41;, which sequentially destabilize RPM1 and eliminate RPM1 in 12 &#45; 20 hrs independent on RPS2. When RPS2 is present, the elimination of RPM1 is tightly linked with RPS2&#45;mediated HR time point. Interestingly enough, RPS2&#45;mediated HR is accelerated in rpm1 mutant, suggesting that RPM1 may function as a negative regulator for AvrRpt2&#47;RPS2&#45;mediated gene&#45;for&#45;gene resistance.  We will discuss that what kinds of plant factors involve in AvrRpt2&#45;mediated RPM1 elimination and what their functions are in the plant defense mechanism.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Durable broad&#45;spectrum powdery mildew resistance in crops and cereals: What can we learn from Arabidopsis&#63;</div>

<div>

	<span  >
	Chiara Consonni</span>
	<sup>1</sup>, 

	<span  >
	Matt Humphry</span>
	<sup>1</sup>, 

	<span  >
	Sandra Noir</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Ralph Panstruga</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max-Planck Institute for Plant Breeding Research, Cologne, Germany

</div>

<div>Recessively inherited loss&#45;of&#45;function alleles &#40;mlo&#41; of the barley Mlo gene confer resistance that is effective against all known isolates of the barley powdery mildew fungus, Blumeria graminis f. sp. hordei. In susceptible Mlo wild type plants, the fungus potentially manipulates the protein encoded by this gene for plant cell invasion. Although Mlo homologs are found in all flowering plants examined to date it was unclear until recently whether mlo resistance represents a species&#45;specific phenomenon restricted to barley. Our results demonstrate that mlo&#45;based powdery mildew resistance can be induced in the model plant Arabidopsis thaliana by inactivation of a particular of the 15 Mlo homologs. These data demonstrate that mlo resistance is effective in both major clades of flowering plants, suggesting that the role of MLO proteins for colonization by powdery mildews is ancient and evolutionarily conserved. Besides conferring powdery mildew resistance, lack of Mlo results in an early senescence&#45;like pleiotropic phenotype in both barley and Arabidopsis. The genetic and molecular tool&#45;box available for the dicotyledonous reference species may enable to unravel the molecular basis of mlo resistance. In addition, it will help in designing strategies to uncouple the undesired pleiotropic phenotypes from the desired resistance trait. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Perception and Signaling of Chitooligosaccharides in Arabidopsis</div>

<div>

	<span  >
	Jinrong Wan</span>
	<sup>1</sup>, 

	<span  >
	Xuecheng Zhang</span>
	<sup>1</sup>, 

	<span  >
	Steve Clough</span>
	<sup>2</sup>, 

	<span  >
	Katrina Ramonell</span>
	<sup>3</sup>, 

	<span  >
	Sung-Yong Kim</span>
	<sup>1</sup>, 

	<span  >
	Minviluz G. Stacey</span>
	<sup>1</sup>, 

	<span  >
	Shuqun Zhang</span>
	<sup>1</sup>, 

	<span  >
	Geon Hui Son</span>
	<sup>4</sup>, 

	<span  >
	Jong Chan Hong</span>
	<sup>4</sup>, 

	<span style="text-decoration: underline" >
	Gary Stacey</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>University of Missouri, Columbia, MO, USA, 

	<sup>2</sup>USDA-ARS, University of Illinois, Urbana, IL, USA, 

	<sup>3</sup>University of Alabama, Tuscaloosa, AL, USA, 

	<sup>4</sup>Gyeongsang National University, Jinju, Korea

</div>

<div>Chitin is a polymer of N&#45;acetyl&#45;D&#45;glucosamine. It is found in the cell walls of plant pathogenic fungi and is recognized by the plant as a pathogen&#45;associated&#45;molecular pattern &#40;PAMP&#41;, which triggers a general defense response. To study chitin signaling and its role in plant defense, we took advantage of the model plant Arabidopsis thaliana. In Arabidopsis, there are five LysM RLKs which have an extracellular LysM motif&#45;containing domain, a transmembrane domain and an intracellular serine&#47;threonine kinase domain. Mutation in one of them blocked the induction of almost all chitin&#45;responsive genes by chitin, suggesting that this LysM RLK may play a critical role in perceiving chitin signal. The mutation also leads to more susceptibility to fungal pathogens, suggesting a connection between chitin signaling and plant defense. Further studies demonstrate that a MAPK cascade and several WRKY transcription factors appear to be involved in the chitin signaling pathway to mediate gene regulation and defense.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Galactinol as a novel priming component on induced systemic resistance elicited by Pseudomonas chlororaphis O6</div>

<div>

	<span style="text-decoration: underline" >
	Song Mi Cho</span>
	<sup>1</sup>, 

	<span  >
	Mi Seong Kim</span>
	<sup>1</sup>, 

	<span  >
	Young Cheol Kim</span>
	<sup>1</sup>, 

	<span  >
	Kwang Yeol Yang</span>
	<sup>1</sup>, 

	<span  >
	Baik Ho Cho</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Agricultural Plant Stress Research Center, Biotechnology Research Institute, Department of Plant Biotechnology, College of Agriculture and Life Sciences, Chonnam National University, Gwangju 500-757, Republic of Korea

</div>

<div>Root colonization by Pseudomonas chlororaphis O6 elicited induced systemic resistance &#40;ISR&#41; in cucumber against Corynespora cassiicola. To understand O6&#45;mediated ISR, a suppressive subtractive hybridization method was undertaken and led to isolation of several distinct genes including a cucumber galactinol synthase &#40;CsGolS1&#41; gene. The transcriptional level of the CsGolS1 and the resulting galactinol content showed an increase 12 h earlier under O6 treatment than in water control plants only after challenge with C. cassiicola, while no difference detected on the plants without pathogen challenge. To examine the role of galactinol on ISR, we employed Arabidopsis mutants at the AtGolS1 gene, an ortholog of the CsGolS1 gene, and AtGolS1&#45;overexpressing transgenic Arabidopsis. The transcriptional level of the AtGolS1 was also primed by showing an increase 12 h earlier under O6 treatment than in water control plants after challenge with Botrytis cinerea on Arabidopsis. The AtGolS1 mutant plant compromised ISR against B. cinerea, while the AtGolS1 overexpressor showed constitutive ISR against B. cinerea and Erwinia carotovora coupling to increased accumulation of galactinol content. Pharmaceutical application of 5 mM galactinol was sufficient to elicit ISR in Arabidopsis Col&#45;0, npr1&#45;1, etr1&#45;1, and ctr1&#45;1 plants against B. cinerea, but did not in the jar1&#45;1 mutant that was validated to detect transcriptional expression of two JA indicator genes, AtVSP and AtPDF1.2. Taken together, our results indicate that primed accumulation of galactinol in the cucumber and Arabidopsis when P. chlororaphis O6 colonized on the root can play a critical role in the JA&#45;associated ISR against fungal and bacterial pathogens.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Induced systemic drought tolerance elicited by Pseudomonas chlororaphis O6 in Arabidopsis thaliana</div>

<div>

	<span style="text-decoration: underline" >
	Song Mi Cho</span>
	<sup>1</sup>, 

	<span  >
	Kwang Hyun Min</span>
	<sup>1</sup>, 

	<span  >
	Song Hee Han</span>
	<sup>1</sup>, 

	<span  >
	Beom Ryong Kang</span>
	<sup>1</sup>, 

	<span  >
	Kwang Yeol Yang</span>
	<sup>1</sup>, 

	<span  >
	Baik Ho Cho</span>
	<sup>1</sup>, 

	<span  >
	Young Cheol Kim</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Biotechnology and Environmental-friendly Agriculture Research Center, College of Agriculture and Life Sciences, Chonnam National University, Gwangju 500-757, Korea

</div>

<div>Root&#45;associated bacteria have been reported as possible inoculants for eliciting enhancement of plant growth and induced systemic resistance in several plant species. In this study, here we report that root colonization of Pseudomonas chlororaphis O6 elicit tolerance referred to as induced systemic tolerance &#40;IST&#41; to abiotic stresses such as drought and salt stress in Arabidospis thaliana. To identify the genes that are involved in IST we performed microarray analysis using the 22,800&#45;gene Affymetrix GeneChip probe array with RNA extracted from leaf tissue. Different clusters of genes were expressed or repressed by growth of the plant with and without root colonization by P chlororaphis O6 and with and without water holding. Root colonization in watered plants increased genes associated with defense, response to activated oxygen species and auxin&#45;responses, and jasmonic acid signaling but decreased transcription factors associated with ethylene and ABA signaling. Withholding water from colonized plants caused changes in expression of more genes. Analysis of two gene clusters that showed different regulation analysis showed down&#45;regulation of genes involved in pectin catabolism and up&#45;regulation of genes for sugar metabolism, ACC synthase and a transcription factor for ABA, ATHB&#45;7. Clusters of genes that were up&#45;regulated in the P. chlororaphis O6&#45;water&#45;stressed plants were observed as being down&#45;regulated in comparison with the drought&#45;stressed non&#45;colonized plants and vice versa. Other authenticated stress and defense responsive genes also were regulated by P. chlororaphis O6 root colonization with some enhanced response under drought stress. These findings confirm that root colonization alters gene regulation in a systemic manner in IST. We speculate that jasmonic acid as well as ethylene and salicylic acid regulated genes were important in the systemic induction of the both abiotic and biotic stress by P. chlororaphis O6 root colonization.</div>

</div>





<div class="abs">

<div style="font-weight:bold">A phage&#45;display approach for high throughput screening of cDNA libraries expressed during plant&#45;pathogen interactions</div>

<div>

	<span  >
	Ines Arrieta-Aguirre</span>
	<sup>1</sup>, 

	<span  >
	Florencia Lucca</span>
	<sup>1</sup>, 

	<span  >
	Keith A. Charlton</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Susana Garcia-Sanchez</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>NEIKER, 

	<sup>2</sup>Haptogen Ltd

</div>

<div>Arabidopsis response to microorganisms involves a large number of plant proteins. Although genome sequence is fully available, many of the microbial&#45;response proteins remain unknown because there is not experimental evidence for their function.  The physical affinities of these proteins for pathogen molecules can be the key to evidence their role in the plant&#45;pathogen interactions. The phage&#45;display technology allows the phage&#45;mediated expression of a large number of proteins and their selection on the bases of the physical affinity to a ligand molecule. In this work we have applied the phage&#45;display technology to selection of a large number of proteins related to plant&#45;pathogen interactions. The strategy has been proven to be a efficient tool to select proteins from &#34;Allium sativum&#34; that bind microbial living cells and has lead to the identification of a putative defensin&#45;like protein &#40;AS2&#45;3P&#41; able to recognize &#34;Pseudomonas aeruginosa&#34; cells with high affinity.&#13;<br/>We constructed cDNA libraries from &#34;Arabidopsis thaliana&#34; after infection with  &#34;P. syringae&#34; pv. &#34;tomato&#34; DC3000 and &#34;P. aeruginosa&#34; PA14, an opportunistic pathogen of humans which is also able to infect Arabidopsis. The cDNA&#45;encoded genes were expressed as functional proteins fused to the capsid of a viral bacteriophage. These phage&#45;displayed proteins and their corresponding genes were rescued by the ability of phagemic particles to bind microbial cells as a ligand, in a so&#45;called &#8220;biopanning&#8221; selection. The construction of a library of 10&#42;6 clones in combination with microarray hibridization allowed to test the capacity of an equivalent number of proteins for pathogen interaction. The value of the phage&#45;display technology to genome&#45;wide exploration of plant defence mechanisms is discussed.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Aspartic protease modulation of stress signalling in Arabidopsis</div>

<div>

	<span style="text-decoration: underline" >
	Matthieu Chabannes</span>
	<sup>1</sup>, 

	<span  >
	Yiji Xia</span>
	<sup>2</sup>, 

	<span  >
	Xinwei Chen</span>
	<sup>1</sup>, 

	<span  >
	Chris Lamb</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>John Innes Centre, Norwich Research Park, Colney, Norwich NR4 7UH, UK, 

	<sup>2</sup>Donald Danforth Plant Science Center 975 North Warson Road St. Louis, Missouri 63132, USA.

</div>

<div>We have used activation tagging with T&#45;DNA carrying cauliflower mosaic virus 35S enhancers to investigate the complex signalling networks underlying disease resistance in Arabidopsis. From the screen of &#126; 5000 lines we have identified <span style="font-style: italic">CDR1</span> &#40;constitutive disease resistance&#41; that encodes an apoplastic aspartate protease involved in local peptide signalling in basal disease resistance. Here we describe the characterisation of a second activation tagged allele <span style="font-style: italic">CDS1&#45;D</span> which shows enhanced disease susceptibility. Remarkably, <span style="font-style: italic">CDS1</span> &#40;constitutive disease susceptibility&#41; encodes a second apoplastic aspartic protease closely related to CDR1. Over&#45;expression of <span style="font-style: italic">CDS1</span> causes some drastic effects on the plant phenotype, which are totally abolished when point mutations are introduced into the protease active site indicating that the phenotypes require CDS1 protease activity. The <span style="font-style: italic">CDS1&#45;D</span> activation tagged line exhibits enhanced disease susceptibility to virulent <span style="font-style: italic">P. syringae</span> and is also compromised for localised resistance to isogenic avirulent strains. Recent experiments show that over&#45;expression of <span style="font-style: italic">CDS1</span> compromises also the resistance to the biotrophic fungus <span style="font-style: italic">Peronospora parasitica</span> and to the necrotrophic fungus <span style="font-style: italic">Fusarium graminearum</span>. On the contrary <span style="font-style: italic">CDS1&#45;D</span> shows an increase of resistance to chewing insects and interestingly increased tolerance to different abiotic stresses. Finally, we have shown that <span style="font-style: italic">CDS1</span> over expression leads to a potentiation of abscisic &#40;ABA&#41; and jasmonic &#40;JA&#41; acid accumulation in response to biotic and abiotic stresses. Taken together these emerging data suggest a model involving proteinase&#45;mediated regulation of mobile peptide signal molecules in the modulation of biotic and abiotic stress responses.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Suppression of Dicer&#45;like 4 by the P6 protein of <span style="font-style: italic">Cauliflower mosaic virus</span>: relevance of sub&#45;cellular localization and link to symptom induction.</div>

<div>

	<span style="text-decoration: underline" >
	Gabrielle Haas</span>
	<sup>1</sup>, 

	<span  >
	Guillaume Moissiard</span>
	<sup>1</sup>, 

	<span  >
	Christophe Himber</span>
	<sup>1</sup>, 

	<span  >
	Mario Keller</span>
	<sup>1</sup>, 

	<span  >
	Olivier Voinnet</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Institut de Biologie Mol&#233;culaire des Plantes, ULP/CNRS UPR2357, 12 rue du G&#233;n&#233;ral Zimmer, 67084 Strasbourg cedex, France

</div>

<div>RNA silencing is a mechanism of gene regulation with antiviral roles in plants and insects. A common counter&#45;defensive strategy of viruses against RNA silencing is the production of viral factors referred to as &#171;silencing suppressor&#187;. Although RNA silencing and its suppression have been well documented for plant RNA viruses, less data are available for DNA viruses. <br/> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span style="font-style: italic">Cauliflower mosaic virus</span> &#40;CaMV&#41; has a DNA genome &#40;8kbp&#41; which is replicated by reverse transcription of a pregenomic RNA &#40;35S RNA&#41;, a feature of the pararetrovirus supergroup. This 35S RNA is polycistronic and encode for all the viral proteins. Recently, it has been shown that the <span style="font-style: italic">Arabidopsis</span> silencing machinery targets an extensively base&#45;paired region of the 35S RNA &#40;referred to as &#8216;leader&#8217;&#41;, which is hierarchically processed by the four <span style="font-style: italic">Arabidopsis</span> Dicer&#45;like proteins &#40;DCLs&#41; to produce populations of viral siRNAs &#40;vsRNAs&#41;. Although it remains unclear if these molecules are effectively used to guide viral transcript degradation, it has been shown that Dicing is one limiting factor of 35S RNA accumulation in infected cells. It was thus expected that CaMV would deploy a suppressor activity to limit siRNA biogenesis in order to preserve intact 35S RNA for viral replication.<br/>The P6 protein of CaMV is a pathogenicity factor, which is a common feature of viral silencing suppressors. To test P6 ability to suppress silencing, we introduced its ORF in several silencing reporter systems available in <span style="font-style: italic">Arabidopsis</span>. Molecular analyses of the resulting transformants and of infected plants suggest that P6 has a strong and specific inhibitory effect on the DCL4 activity, the primary antiviral dicer of plants. Transgenic plants expressing various P6 alleles, defective for translational activation, dsRNA binding, nucleo&#45;cytoplasmic trafficking, or viroplasms formation, have been generated and the analyses of those plants now allow us to define which features of P6 are involved &#40;or not&#41; in suppression of RNA silencing in <span style="font-style: italic">Arabidopsis</span>. <br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Investigating roles of RAR1 and SGT1 in TIR&#45;NB&#45;LRR immune receptor function</div>

<div>

	<span style="text-decoration: underline" >
	Shigeyuki Betsuyaku</span>
	<sup>1</sup>, 

	<span  >
	Lennart Wirthmueller</span>
	<sup>1</sup>, 

	<span  >
	Jane Parker</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Max-Planck Institute for Plant Breeding Research, Koeln. Germany

</div>

<div>Animal and plant immune receptors &#40;R protein&#41; of the NOD&#47;NB&#45;LRR type are assembled inside the cell as structurally constrained proteins that recognize specific pathogen effectors. <span style="font-style: italic">RAR1</span> &#40;<span style="font-style: italic">Required for</span> Ml&#45;a12 <span style="font-style: italic">conditioned resistance</span>&#41; and <span style="font-style: italic">SGT1</span> &#40;<span style="font-style: italic">Suppressor of G2 allele of</span> skp1&#41; are positive regulators of plant immune responses mediated by R proteins. RAR1 and SGT1 have molecular features of cochaperones and are required for accumulation of Coiled&#45;coil &#40;CC&#41;&#45;NB&#45;LRR type receptors most likely through their cooperation with HSP90 during assembly. Another major class of plant NB&#45;LRR receptors has N&#45;terminal homology &#40;the TIR domain&#41; to mammalian Toll&#45;like receptors. The roles of RAR1 and SGT1 in function of plant TIR&#45;NB&#45;LRR immune receptor signalling remain unclear. In this study, we are investigating the functions of RAR1 and SGT1 in the assembly, activities and subcellular localizations of Arabidopsis TIR&#45;NB&#45;LRR proteins using genetic and biochemical approaches. We find that transcript levels of <span style="font-style: italic">EDS1</span> &#40;<span style="font-style: italic">Enhanced Disease Susceptibility1</span>&#41;, a key positive regulator of TIR&#45;NB&#45;LRR mediated and basal defence, are reduced in both <span style="font-style: italic">rar1</span> and <span style="font-style: italic">sgt1b</span> mutants, resulting in depleted EDS1 protein accumulation in the absence of pathogen. This correlates with defects in basal resistance of the <span style="font-style: italic">rar1</span> and <span style="font-style: italic">sgt1b</span> mutants. We are now analyzing how accumulation of TIR&#45;NB&#45;LRR proteins is affected in these mutants to establish more precisely where RAR1 and SGT1 operate in the TIR&#45;NB&#45;LRR triggered plant immune response. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Gene expression in Brassica napus in response to infection by Sclerotinia sclerotiorum and identification of WRKY transcription factors</div>

<div>

	<span style="text-decoration: underline" >
	Jianwei Zhao</span>
	<sup>1</sup>, 

	<span  >
	Roger Rimmer</span>
	<sup>1</sup>, 

	<span  >
	Lone Buchwaldt</span>
	<sup>1</sup>, 

	<span  >
	Dwayne Hegedus</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Agriculture and Agri-Food Canada, Saskatoon, Canada

</div>

<div>Sclerotinia sclerotiorum is a pathogenic fungus that can infect a wide range of plant species, including Arabidopsis thaliana and several important crop species. The temporal response of a partially resistant genotype of Brassica napus, Zhong You 821, to this pathogen was examined using a Brassica gene oligo&#45;array representing 15,000 unique genes.  RNA was extracted from inoculated stem tissues at 6, 12, 24, 48 and 72 h post&#45;infection and used for micro&#45;array analysis. A total of 7,744 genes were found to be significantly differentially expressed across times. The largest proportion of these genes was differentially expressed at 48 h post&#45;infection, while a smaller group was induced at the earlier times. Hierarchical clustering analysis revealed a temporal expression pattern across times.  Exploiting the genome synteny between B. napus and A. thaliana together with A. thaliana gene annotation, the function of proteins encoded by these genes were categorized into pathogenesis&#45;related proteins, protein kinase pathways, secondary metabolism and maintenance of cell wall integrity, abiotic stress, transcription factors as well as some proteins with unknown function. In particular, we found several genes encoding WRKY transcription proteins that were highly induced at the early stages of sclerotinia infection.  Three homologous WRKY genes were isolated from B. napus and over&#45;expressed in A. thaliana. These transgenic lines are being screened for phenotypic changes in response to pathogen infection.</div>

</div>





<div class="abs">

<div style="font-weight:bold">The <span style="font-style: italic">Pseudomonas syringae</span> type III effector HopU1 ADP&#45;ribosylates RNA&#45;binding proteins and suppresses plant immunity</div>

<div>

	<span style="text-decoration: underline" >
	Zheng Qing Fu</span>
	<sup>1</sup>, 

	<span  >
	Ming Guo</span>
	<sup>1</sup>, 

	<span  >
	Byeong-ryool Jeong</span>
	<sup>1</sup>, 

	<span  >
	Fang Tian</span>
	<sup>2</sup>, 

	<span  >
	Thomas Elthon</span>
	<sup>2</sup>, 

	<span  >
	Ronald Cerny</span>
	<sup>3</sup>, 

	<span  >
	Dorothee Staiger</span>
	<sup>4</sup>, 

	<span  >
	James Alfano</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Pathology, University of Nebraska, Lincoln, NE, USA, 

	<sup>2</sup>School of Biological Sciences, University of Nebraska, Lincoln, NE, USA, 

	<sup>3</sup>Department of Chemistry, University of Nebraska, Lincoln, NE, USA, 

	<sup>4</sup>Molecular Cell Physiology, University of Bielefeld, Bielefeld, Germany

</div>

<div><span style="font-style: italic">Pseudomonas syringae</span> is a bacterial plant pathogen and its pathogenicity depends on the type III protein secretion system &#40;T3SS&#41; and the type III effectors it injects into host cells. The predicted product of the <span style="font-style: italic">hopU1</span> effector gene carried a potential mono&#45;ADP&#45;ribosyltransferases &#40;ARTs&#41; active site. These enzymes have been well characterized in animal pathogens and are also found in mammals but have not been demonstrated to be present in plants or plant pathogens. RT&#45;PCR indicated that <span style="font-style: italic">hopU1</span> was expressed in <span style="font-style: italic">P. syringae</span> and translocation assays indicated that HopU1 was injected into plant cells by the T3SS. HopU1 inhibited several outputs of plant innate immunity. Partially&#45;purified HopU1&#45;His ADP&#45;ribosylated poly&#45;L&#45;arginine, which indicates it is an arginine specific ART. We found that HopU1 ADP&#45;ribosylated several proteins in plant extracts and this activity required its putative ART active site. Using two dimensional gel electrophoresis and mass spectrometry, we identified the <span style="font-style: italic">in vitro</span> substrates of HopU1&#45;His to be three chloroplast RNA&#45;binding proteins and two glycine&#45;rich RNA&#45;binding proteins &#40;GR&#45;RBPs&#41; in <span style="font-style: italic">Arabidopsis</span> extracts. <span style="font-style: italic">Arabidopsis</span> T&#45;DNA knockout plants of one of the GR&#45;RBPs, AtGRP7, were more susceptible to <span style="font-style: italic">P. syringae</span> than wild type plants, indicating that this substrate plays a role in innate immunity. When co&#45;expressed in plants, HopU1 was capable of ADP&#45;ribosylating AtGRP7 indicating HopU1 could modify AtGRP7 <span style="font-style: italic">in planta</span>. Two arginine residues within AtGRP7&#8217;s RNA&#45;binding domain were required for it to be ADP&#45;ribosylated by HopU1&#45;His, suggesting that ADP&#45;ribosylation interferes with AtGRP7&#8217;s ability to bind RNA. Our results suggest a novel strategy employed by a bacterial pathogen where ADP&#45;ribosylation of plant RNA&#45;binding proteins results in posttranscriptional inhibition of host innate immunity.</div>

</div>





<div class="abs">

<div style="font-weight:bold">'Ascophyllum nodosum' extract induces resistance in 'Arabidopsis thaliana' against 'Pseudomonas syringae' pv 'tomato' DC3000</div>

<div>

	<span  >
	Sowmyalakshmi Subramanian</span>
	<sup>1</sup>, 

	<span  >
	Bruce A. Gray</span>
	<sup>1</sup>, 

	<span  >
	Rudra P. Singh</span>
	<sup>2</sup>, 

	<span  >
	Simon D. Hankins</span>
	<sup>3</sup>, 

	<span  >
	Alan T. Critchley</span>
	<sup>3</sup>, 

	<span  >
	D. Mark Hodges</span>
	<sup>4</sup>, 

	<span style="text-decoration: underline" >
	Balakrishnan Prithiviraj</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Nova Scotia Agricultural College, Truro, NS, Canada B2N 5E3, 

	<sup>2</sup>Agriculture and Agri-Food Canada, 850 Lincoln Rd., Fredericton, NB Canada, E3B 4Z7, 

	<sup>3</sup>Acadian Seaplants Limited, 30 Brown Avenue, Dartmouth, NS , Canada B3B 1X8, 

	<sup>4</sup>Agriculture and Agri-Food Canada, 32 Main St. Kentville, NS, Canada  B4N 1J5

</div>

<div>Experiments were conducted to study the effect of extracts of a brown seaweed 'Ascophyllum nodosum' on the induction of resistance in 'Arabidopsis thaliana' against a bacterial pathogen 'Pseudomonas syringae' pv. 'tomato' DC3000 &#40;Pst&#41;. 'A. thaliana' was treated with 'A. nodosum' extracts by root irrigation followed by inoculation of the leaves of the treated plants with Pst. 'A. nodosum' extracts significantly reduced the area of leaf infected and subsequent disease intensity &#40;DI&#41;. This reduction in the DI correlated with a decline in the bacterial titer in the leaf apoplast. Surprisingly, addition of seaweed extracts to the bacterial culture medium enhanced the growth of Pst, suggesting that 'A. nodosum' induced systemic disease resistance in 'A. thaliana'. PR1 &#40;Pathogen related protein 1&#41; gene, a marker gene associated with induced resistance was upregulated by seaweed extract treatments. Taken together the results suggest that the extracts of 'A. nodosum' elicit systemic acquired resistance in 'A. thaliana'.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of a Chinese cabbage cyclophilin with antifungal activity</div>

<div>

	<span style="text-decoration: underline" >
	Mi Rim Shin</span>
	<sup>1</sup>, 

	<span  >
	Jin Ho Park</span>
	<sup>1</sup>, 

	<span  >
	Ho Byoung Chae</span>
	<sup>1</sup>, 

	<span  >
	Deok Ho LEE</span>
	<sup>1</sup>, 

	<span  >
	Kyun Oh Lee</span>
	<sup>1</sup>, 

	<span  >
	Woe-Yeon Kim</span>
	<sup>1</sup>, 

	<span  >
	Sang Yeol Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Environmental Biotechnology National Core Research Center,PMBBRC &amp;  Division of Applied Life Science(BK21), Gyeonsang National University, Jinju 660-701 Korea

</div>

<div>An antifungal protein that inhibits the growth of filamentous fungal pathogens was isolated from Chinese cabbage &#40;Brassica campestris L. ssp. pekinensis&#41; by affinity chromatography on Affi&#45;gel blue gel and ion exchange chromatography on CM&#45;Sepharose. The N&#45;terminal amino acid sequence of the protein was highly homologous to that of plant cyclophilins and consequently the protein was denoted as C&#45;CyP. To understand the antifungal activity of C&#45;CyP, we isolated a cDNA encoding its gene from a Chinese cabbage leaf cDNA library. The Chinese cabbage genome bears more than one C&#45;CyP gene copy and C&#45;CyP mRNA is highly expressed in all tissues except the seeds. Recombinant C&#45;CyP catalyzed the cis&#8211;trans inter&#45;conversion of the Ala&#8211;Pro bond of the substrate, which indicates this protein has peptidyl&#45;prolyl cis&#8211;trans isomerase activity. It also inhibited the growth of several fungal pathogens.</div>

</div>





<div class="abs">

<div style="font-weight:bold">HSP90, SGT1, and RAR1 form a ternary chaperone complex to regulate resistance&#45;protein dependent plant immunity</div>

<div>

	<span style="text-decoration: underline" >
	Ken Shirasu</span>
	<sup>1,2</sup>, 

	<span  >
	Marta Boter</span>
	<sup>1</sup>, 

	<span  >
	Beatrice Amigues</span>
	<sup>3</sup>, 

	<span  >
	Jack Peart</span>
	<sup>1</sup>, 

	<span  >
	Christian Breuer</span>
	<sup>1</sup>, 

	<span  >
	Yasuhiro Kadota</span>
	<sup>2</sup>, 

	<span  >
	Catarina Casais</span>
	<sup>1</sup>, 

	<span  >
	Francoise Ochsenbein</span>
	<sup>3</sup>, 

	<span  >
	Raphael Guerois</span>
	<sup>3</sup>

</div>

<div>

	<sup>1</sup>The Sainsbury Laboratory, John Innes Centre, Colney Lane, Norwich NR4 7UH, UK, 

	<sup>2</sup>RIKEN Plant Science Center, Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama, 230-0045 Japan, 

	<sup>3</sup>SBFM-DBJC, CEA Saclay, 91191 Gif sur Yvette Cedex, FRANCE

</div>

<div>SGT1 and RAR1 are highly conserved eukaryotic proteins that interact with HSP90, a molecular chaperone. In plants, SGT1, RAR1 and HSP90 are essential for disease resistance triggered by a number of resistance &#40;R&#41; proteins that contain NB &#40;nucleotide binding site&#41; and LRR &#40;Leucine Rich Repeat&#41;. SGT1 and HSP90 also play a crucial role in Nod&#45;like receptors &#40;NLRs&#41;&#45;mediated innate immunity responses in mammals. Here we present structural and functional characterization of plant SGT1. Random mutagenesis of SGT1 revealed that its CS and SGS domains are essential for disease resistance. NMR&#45;based interaction surface mapping of the CS domain combined with mutational analyses showed that the CHORD&#45;II domain of RAR1 and the N&#45;terminal domain of HSP90 interact with opposite sides of the CS domain. Functional analysis of the CS mutations indicated that the interaction between SGT1 and HSP90 is required for disease resistance triggered by Rx, a potato R protein. Biochemical reconstitution experiments revealed that RAR1 functions to enhance SGT1&#45;HSP90 interaction by promoting ternary complex formation.</div>

</div>





<div class="abs">

<div style="font-weight:bold">LRA5 &#8211; an EPIGENETIC component of ARABIDOPSIS disease resistance&#13;<br/>&#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	Albor Dobon</span>
	<sup>1</sup>, 

	<span  >
	Brande B.H. Wulff</span>
	<sup>1</sup>, 

	<span  >
	Jefferey L. Dangl</span>
	<sup>2</sup>

</div>

<div>

	<sup>1</sup>IBMCP (UPV-CSIC), Valencia, Spain, 

	<sup>2</sup>University of North Carolina, Chapell Hill, (North Carolina), USA

</div>

<div>The Arabidopsis mutant LRA5 was isolated in a screen for loss of resistance to the model bacterial pathogen Pseudomonas syringae pv tomato DC3000.  Intriguingly, further characterisation of the mutant has revealed it to be impaired in several resistances &#40;including basal; some, but not all gene&#45;for&#45;gene; and SAR&#41;. The most prominent feature however, is that LRA5 lacks non&#45;host resistance against several isolates of Pseudomonas. &#13;<br/>&#13;<br/>Another important characteristic of this mutant was discovered when it was shown by genetics to be a dominant gain&#45;of&#45;function mutation. Analysis of the transcriptome of non&#45;challenged LRA5 plants has revealed that the basal expression of defence genes was greatly reduced in LRA5 plants, suggesting that LRA5 operates at a nodal point in defence signalling.&#13;<br/>&#13;<br/>LRA5 has been mapped to a &#126;11.5 kb interval on chromosome II containing three known genes: two transcription factors &#40;TF1 and TF2&#41; and a gene without homology to any gene with a known function &#40;UNKNOWN&#41;. None of these three genes have previously been implicated in disease resistance. Sequence analysis of the LRA5 mapping interval did not reveal any nucleotide change, indicating that LRA5 is an epigenetic mutant. Furthermore, LRA5 is genetically meta&#45;stable &#40;1 revertant per 3,000 mutant chromosomes&#41;, which is another hallmark of epigenetic mutants.  &#13;<br/>&#13;<br/>In a second round of mutagenesis of LRA5 we isolated six suppressors, three of which were defined by genetics to be intragenic. One of these carries a mutation in the UNKNOWN, suggesting this gene may be LRA5. However, qRT&#45;PCR analysis has so far not revealed a significant change in the transcript level of this gene in mutant plants, whereas, surprisingly, transcript levels of TF1 are significantly down in LRA5 compared to wild&#45;type.&#13;<br/>&#13;<br/>To identify LRA5 we are currently complementing the above analyses with the phenotyping of transgenic plants &#40;RNAi, T&#45;DNA and overexpression&#41; as well as comparative methylation analysis of the mapping interval.&#13;<br/>&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Characterization of hypersensitive mutant &#40;stop1&#41; to proton&#45;rhizotoxicity</div>

<div>

	<span style="text-decoration: underline" >
	Satoshi Iuchi</span>
	<sup>1</sup>, 

	<span  >
	Hiroyuki Koyama</span>
	<sup>2</sup>, 

	<span  >
	Atsuko Iuchi</span>
	<sup>1</sup>, 

	<span  >
	Yasufumi Kobayashi</span>
	<sup>2</sup>, 

	<span  >
	Sadako Kitabayashi</span>
	<sup>1</sup>, 

	<span  >
	Yuriko Kobayashi</span>
	<sup>2</sup>, 

	<span  >
	Takashi Ikka</span>
	<sup>2</sup>, 

	<span  >
	Takashi Hirayama</span>
	<sup>3</sup>, 

	<span  >
	Kazuo Shinozaki</span>
	<sup>4</sup>, 

	<span  >
	Masatomo Kobayashi</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>RIKEN BRC, 

	<sup>2</sup>Gifu University, 

	<sup>3</sup>Yokohama City University, 

	<sup>4</sup>RIKEN PSC

</div>

<div> Acid soil represents a limiting factor of crop production. Plant growth on acid soil is limited mostly because of the rhizotoxicities of ions such as aluminum &#40;Al3&#43;&#41;. Although proton &#40;H&#43;&#41; could be the major rhizotoxicant in some soil types, molecular mechanisms of plant tolerance against H&#43; rhizotoxicity have not been identified yet. &#13;<br/>  We have isolated a mutant that is hypersensitive to H&#43; rhizotoxicity from EMS mutagenized seed stock of Arabidopsis thaliana by root bending assay system. The isolated mutant, stop1 &#40;Sensitive TO Proton rhizotoxicity&#41;, was defect in root growth under low pH conditions. This phenotype of stop1 was as attributed to a missense mutation at the zinc&#45;finger domain in a predicted Cys2&#47;His2 type zinc&#45;finger protein, designated as STOP1. The STOP1 belongs to a functionally unidentified subfamily of zinc&#45;finger proteins in Arabidopsis. &#13;<br/> The treatments of cadmium, copper, lanthanum, manganese and sodium chloride gave no significant difference in root growth between stop1 mutant and wild type, however, stop1 mutant show hypersensitivity to Al3&#43; treatment. The stop1 mutant lacked the induction of the AtALMT1 gene that encodes a malate transporter, and was unable to induce malate exudation by Al3&#43; treatment. The AtALMT1 knockout mutant shows hypersensitivity to Al3&#43;, whereas the stop1 mutant is hypersensitive to both Al3&#43; and low pH treatment. These results suggest that STOP1 play a key role in both Al3&#43; tolerance and low pH tolerance.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">&#34;Arabidopsis thaliana&#34; homeobox gene, &#34;ATHB12&#34; is induced by&#13;<br/>Beet Severe Curly Top Geminivirus infection&#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	Jungan Park</span>
	<sup>1</sup>, 

	<span  >
	Kun-sup Lee</span>
	<sup>1</sup>, 

	<span  >
	Hyunsik Hwang</span>
	<sup>2</sup>, 

	<span  >
	Yong Rhee</span>
	<sup>1</sup>, 

	<span  >
	 Choong-Ill  Cheon</span>
	<sup>3</sup>, 

	<span  >
	Chungkyoon Auh</span>
	<sup>4</sup>, 

	<span  >
	Sukchan Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dept. of Genetic Engineering, Sungkyunkwan University, Suwon, South Korea, 

	<sup>2</sup>Dept. of Molecular Genetics, Plant Biotechnology Center, and Program in Molecular, Cellular and Developmental Biology, The Ohio State University, Columbus, Ohio, USA, 

	<sup>3</sup>Dept. of Biological Science, Sookmyung Womens University, Seoul, South Korea, 

	<sup>4</sup>Dept. of Life Science, Mokpo National University, Mokpo, Korea

</div>

<div>&#34;ATHB12&#34; is an &#34;Arabidopsis thaliana&#34; homeobox gene, and a member of the homeodomain&#45;leucin zipper &#40;HD&#45;Zip&#41; family. It has been generally reported that &#34;ATHB12&#34; was induced by abscisic acid &#40;ABA&#41; and water stress. Moreover, recent findings strongly suggest that &#34;ATHB12&#34; expression is also induced by the infection of beet severe curly top geminivirus &#40;BSCTV&#41; to &#34;Arabidopsis&#34;. BSCTV develops an array of morphological abnormalities such as leaf curling, stunting, and callus&#45;like structure on the infected &#34;Arabidopsis&#34;. However, on molecular level, the direct correlation was not yet established between patho&#45;morphogenesis and HD&#45;Zip gene in BSCTV infected &#34;Arabidopsis&#34;. At this time, regulation&#47;expression of &#34;ATHB12&#34; gene under the environmental &#40;biotic &#38; abiotic&#41; stress was studied with an array of transgenic &#34;Arabidopsis&#34; with over&#47;under&#45;expressed &#34;ATHB12&#34; gene. The real&#45;time quantitative RT&#45;PCR results indicated that expression of HD&#45;Zip protein members as well as cell cycle&#45; and hormone&#45;related genes were regulated by BSCTV infection.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Differential expression of cell wall metabolism&#45;, cell cycle&#45; and hormone response&#45; related genes in the interactions between &#34;Arabidopsis&#34; and geminivirus</div>

<div>

	<span  >
	Jungan Park</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Yuchul Jung</span>
	<sup>1</sup>, 

	<span  >
	Seungmi Kim</span>
	<sup>1</sup>, 

	<span  >
	Jongbum Park</span>
	<sup>2</sup>, 

	<span  >
	Kenn Buckley</span>
	<sup>3</sup>, 

	<span  >
	Keith R. Davis</span>
	<sup>4</sup>, 

	<span  >
	Sukchan Lee</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Dept. of Genetic Engineering, Sungkyunkwan University, Suwon, South Korea, 

	<sup>2</sup>Dept. of Life Science, Silla University, Busan, South Korea, 

	<sup>3</sup>Dept. of Molecular Genetics, Plant Biotechnology Center, and Program in Molecular, Cellular and Developmental Biology, The Ohio State University, Columbus, Ohio, USA, 

	<sup>4</sup>Mitchell Memorial Cancer Center, KY, USA

</div>

<div>Beet severe curly top virus &#40;BSCTV&#41; has the monopartite genome of single&#45;stranded DNA as a member of the geminivirus. BSCTV is transmitted by leafhopper, and can infect broad host plants of dicot. BSCTV causes systemic symptoms such as stunted growth, callus formation, curling and the accumulation of anthocyanin pigments in &#34;Arabidopsis thaliana&#34;. Particularly, BSCTV induces abnormal cell divisions on tips of influorescent stems and roots in &#34;Arabidopsis&#34;. The internal structures of the BSCTV infected or BSCTV C4 ORF expressing transgenic &#34;Arabidopsis&#34; showed that cell divisions were induced on phloem cells and its surrounding cortex cells. To elucidate basis of the symptom development caused by BSCTV, real&#45;time quantitative RT&#45;PCR and DNA microarray were carried out to analyze the transcription profile involved in symptom development. Altered gene expressions such as hormone metabolism&#45;, cell wall biosynthesis&#45; and cell cycle&#45; related genes shown altered cell structures caused by BSCTV infection or C4 expression may play a critical role on typical symptom development by the interaction between &#34;Arabidopsis&#34; and BSCTV. </div>

</div>





<div class="abs">

<div style="font-weight:bold">Functional analysis of the wound&#45;inducible Arabidopsis blue copper&#45;binding protein gene promoter in transgenic tobacco plants</div>

<div>

	<span  >
	Jin An Jeong</span>
	<sup>1</sup>, 

	<span  >
	Kwang Hyun Min</span>
	<sup>1</sup>, 

	<span  >
	Eun Kyoung Jang</span>
	<sup>1</sup>, 

	<span  >
	Jin A Jeong</span>
	<sup>1</sup>, 

	<span  >
	Young Cheol Kim</span>
	<sup>1</sup>, 

	<span  >
	Baik Ho Cho</span>
	<sup>1</sup>, 

	<span style="text-decoration: underline" >
	Kwang Yeol Yang</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Department of Plant Biotechnology, College of Agriculture and Life Sciences, Chonnam National University, Gwangju 500-757, Korea

</div>

<div>The plant wound response has to be a rapid and coordinated process in the local region of injury, since wound sites are known to be the main entry points for many kinds of plant pathogens such as fungi, bacteria and viruses. The promoters of the genes that are induced by wounding could be valuable in regulating the expression of heterologous beneficial genes. The transcriptional regulatory region &#40;from &#45;1294 to &#43;247&#41; of a wound&#45;inducible Arabidopsis gene encoding a blue copper&#45;binding protein was fused to the &#946;&#8211;glucuronidase gene in plant transformation vector. Stable integration of the single Arabidopsis BCB&#47;GUS gene into tobacco genome resulted in strongly induced GUS activity upon wounding in the homozygous plants. The BCB promoter was activated at the site of infection with bacterial, fungal and viral pathogens. It was also activated after treatment with salicylic acid and ultraviolet. The cis&#45;acting elements of the BCB promoter region were analyzed by online programs. The promoter analysis revealed several potential regulatory elements that are related to biotic and abiotic stress regulation. These data suggest that, although the BCB promoter activity is reduced in the heterologous host, its pathogen&#45;responsiveness could be useful in driving the expression of transgenes to promote disease resistance in plant.</div>

</div>





<div class="abs">

<div style="font-weight:bold">Involvement of lipase in R gene&#45;induced increase in salicylic acid</div>

<div>

	<span  >
	Hyoung-Yool Lee</span>
	<sup>1</sup>, 

	<span  >
	Hong Gil Nam</span>
	<sup>2</sup>, 

	<span  >
	Inhwan Hwang</span>
	<sup>2</sup>, 

	<span style="text-decoration: underline" >
	Stephen B. Ryu</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Korea Research Institute of Bioscience and Biotechnology, Korea, 

	<sup>2</sup>POSTECH, Korea

</div>

<div>The establishment of local host defense response in plants is R gene&#45;dependent and associated with the elevated levels of salicylic acid &#40;SA&#41;. SA is a necessary and sufficient plant hormone for local host defense response. In spite of functional importance of SA, cellular mechanism of R gene&#45;induced SA increase remains to be much unknown. Here we report that lipase A and its lipid products are involved in upstream signaling between gene&#45;for&#45;gene interaction and SA increase in plant host defense response. The lipase A mutant showed the defect in R gene&#45;induced expression of ICS1, a key enzyme in SA biosynthesis, and PR genes. </div>

</div>





<div class="abs">

<div style="font-weight:bold">A BRASSINOLIDE&#45;INDEPENDENT ROLE FOR THE BRI1&#45;ASSOCIATED RECEPTOR KINASE 1 &#40;BAK1&#41; IN PLANT CELL DEATH CONTROL AND IMMUNITY TO NECROTROPHIC FUNGAL INFECTION</div>

<div>

	<span style="text-decoration: underline" >
	Thorsten Nuernberger</span>
	<sup>1</sup>, 

	<span  >
	Birgit Kemmerling</span>
	<sup>1</sup>, 

	<span  >
	Anne Schwedt</span>
	<sup>1</sup>, 

	<span  >
	Sara Mazotta</span>
	<sup>1</sup>, 

	<span  >
	Patricia Rodriguez</span>
	<sup>1</sup>, 

	<span  >
	Sandra Postel</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>ZMBP Plant Biochemistry, University of Tuebingen, Germany

</div>

<div>Programmed cell death &#40;PCD&#41; is a common host response to microbial infection. In plants, PCD has been associated with immunity to biotrophic pathogens, but has also been shown to promote disease upon infection by necrotrophic pathogens. Therefore, execution and extent of plant cell suicidal programs must be strictly controlled. We have demonstrated that the Arabidopsis thaliana Brassinosteroid Insensitive 1 &#40;BRI1&#41;&#45;Associated receptor Kinase 1 &#40;BAK1&#41;, which operates as a co&#45;receptor of BRI1 in brassinolide&#45;dependent plant growth control, also regulates the containment of microbial infection&#45;induced cell death. BAK1&#45;deficient plants develop spreading necrosis upon microbial infection, which is accompanied by production of reactive oxygen intermediates and which results in enhanced susceptibility to necrotrophic fungal pathogens. Exogenous application of BL rescues growth defects of bak1 mutants, but fails to restore immunity to fungal infection. Moreover, BL&#45;insensitive and deficient mutants do not exhibit enhanced susceptibility to fungal infections. Together, these findings suggest that plant steroid hormone signalling is not required for the containment of infection&#45;induced PCD. We propose a novel, BL&#45;independent function of BAK1 in plant cell death control that is distinct from its BL&#45;dependent role in plant development.&#13;<br/>&#13;<br/>He, P. et al. Cell 125, 563&#45;575 &#40;2006&#41;.&#13;<br/>N&#252;rnberger, T., Kemmerling, B. Trends Plant Sci. 11, 519&#45;522 &#40;2006&#41;. Qutob et al. Plant Cell 18, 3721&#45;3744 &#40;2006&#41;.&#13;<br/>Kemmerling et al. Curr. Biol. in press &#40;2007&#41;.&#13;<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">&#34;MtERF1&#34;, a regulator of rhizobial symbiosis in &#34;Medicago truncatula&#34;</div>

<div>

	<span style="text-decoration: underline" >
	Tatiana Verni&#233;</span>
	<sup>1</sup>, 

	<span  >
	Sandra Moreau</span>
	<sup>1</sup>, 

	<span  >
	Fran&#231;oise De Billy</span>
	<sup>1</sup>, 

	<span  >
	Fabienne Vailleau</span>
	<sup>2</sup>, 

	<span  >
	Marie-Fran&#231;oise Jardinaud</span>
	<sup>2</sup>, 

	<span  >
	Giles Oldroyd</span>
	<sup>3</sup>, 

	<span  >
	Christian Rogers</span>
	<sup>3</sup>, 

	<span  >
	Pascal Gamas</span>
	<sup>1</sup>, 

	<span  >
	Andreas Niebel</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Laboratoire des Interactions Plantes Micro-organismes, INRA-CNRS, Castanet Tolosan, France, 

	<sup>2</sup>Symbiosis and Plant Pathologies, INP-ENSAT, Castanet-Tolosan, France, 

	<sup>3</sup>John Innes Center, Norwich, UK

</div>

<div>&#34;Arabidopsis&#34; has been successfully used as a model plant to study plant&#45;pathogen interactions during the last years leading to the functional characterisation of key regulators during this type of interaction. Unfortunately species of the Brassicaceae are unable to undergo mutualistic interactions such as mycorrhizal or nitrogen&#45;fixing symbioses. Leguminous plants for which various genetic and genomic tools have been recently developed open the unique opportunity to study molecular dialogues between symbiotic bacteria of the Rhizobiaceae family and the plant host. <br/>Using &#34;Medicago truncatula&#34; as a model plant, we performed transcriptomic approaches to get better insights into the way legumes control symbiotic rhizobial infection and nodule development. Among a set of genes up&#45;regulated during nodulation, we identified &#34;MtERF1&#34;, a transcription factor &#40;TF&#41; belonging to the ERF &#40;Ethylene Response Factor&#41; subfamily of the AP2&#47;EREBP family, known to play a role in biotic stresses.<br/>Expression studies by RT&#45;QPCR showed an induction in roots 3 days post &#34;Sinorhizobium meliloti&#34; inoculation and a maximal expression in young nodules. We localized &#34;MtERF1&#34; expression in dividing cells of young nodule primordia and in the infection zone of mature nodules by &#34;in situ&#34; hybridisation and promoter&#45;Gus fusions. We also analysed &#34;MtERF1&#34; expression in the context of various symbiotic bacterial and plant mutants.<br/>&#34;MtERF1&#34; appears as a regulator of symbiosis as suggested by the phenotype of transgenic roots expressing RNAi or over&#45;expression constructs, and more recently of a deletion mutant. &#34;MtERF1&#34; may also be a regulator of pathogenic infections, since it is up&#45;regulated by &#34;Ralstonia solanacearum&#34;, a bacterial root pathogen. We are now looking for MtERF1 targets by transcriptomic approaches and preliminary results suggest that &#34;MtERF1&#34; could play a role in the cytokinin signalling pathway.<br/></div>

</div>





<div class="abs">

<div style="font-weight:bold">Does Rhizobium root nodule symbiosis involve a legume specific toolbox&#63;&#13;<br/></div>

<div>

	<span style="text-decoration: underline" >
	Ton Bisseling</span>
	<sup>1</sup>

</div>

<div>

	<sup>1</sup>Research School Experimental Plant Sciences, lab of Molecular Biology Wageningen University,  Wageningen The Netherlands

</div>

<div>Rhizobium root nodule symbiosis is a unique property of leguminous plants. The formation of these nitrogen fixing root nodules is set in motion by lipoolichosaccharides that are secreted by the rhizobia. The perception and transduction of Nod factors requires a rather small set of genes that are specifically involved in this symbiosis &#40;and in part also in the arbuscular mycorrhizal symbiosis&#41;. Nod factor signaling occurs at the onset of the interaction, but most likely also at 2 additional steps during the infection process as well as in the growing root nodule. A similar set of Nod factor signaling genes has been identified by forward genetics in the legumes Medicago, Lotus and pea. This set of legume Nod factor signaling genes provides the means to compare legumes and non&#45;legumes by which a better understanding of the evolutionary origin of legume nodulation can be obtained.&#13;<br/></div>

</div>


</div>
</div>

</body>
</html>
