<title>jacob-1976-aabdo.html</title><p>

<h2>BIOCHEMICAL GENETICS OF ACID PHOSPHATASE ISOZYMES IN ARABIDOPSIS THALIANA (L.) HEYNH.</h2><p>

M. Jacobs and F. Schwind<p>

(Laboratory of Plant Genetics, Free University of Brussels,
Sint-Genesius-Rode, Belgium).<p>

Acid phosphatase (orthophosphiric monoesterphosporhydrolase E.C.
3.1.3.2.) is a very extensively studied enzyme and there are several
plant species as maize, rice, potato, wheat, peanuts and cotton within
which varying number of molecular forms have been described.  However,
despite numerous investigations on the physico-chemical
characteristics of these enzymes, there are few reports about the
genetical basis of this hetrogeneity. BROWN and ALLARD (1969) have
first described a control system of three phosphatase isozymes in
maize by three codominant alleles.  EFRON (1971) has particularly
studied the pollen and scutellar isozymes specified by the AP1 system
in the same species. ENDO (1970) has studied the inheritance of
isozymic differences in rice cultivars.<p>

The purpose of this paper is to combine a genetical and biochemical
study of acid phosphatase isozymes in Arabidopsis thaliana.  The paper
deals with the natural and induced polymorphism observed in the
species, the study of the genetic system which controls this
polymorphism as its biochemical properties, and changes in isozyme
patterns occurring at defined stages of development. This report
includes some alrady published results (JACOBS and SCHWIND, 1973,
1975)<p>

Numerous geographical races of Arabidopsis originating from the
collection of Prof. F. LAIBACH (cf. ROBBELEN, 1965) as well as from
various botanical gardens were screened for isozyme variations. Growth
conditions of the plants, preparation of extracts, starch gel and
polyacrylamide gel electrophoresis, staining procedures and assay for
enzyme activity were used as previously described (JACOBS and SCHWIND,
1975).<p>

Gel filtration chromatography.  A column of 2.5 cm x 90 (Pharmacia
K25) was packed with G.200 Sephadex suspended in 0.01 M Tris-HC1
buffer at pH 7.4.  Sample volumes of 2-5 ml were applied to the column
and were eluted at 4*degrees*C with the same buffer at a flow rate of
18-20 ml/hour. Fractions of 2.5 ml were collected and assayed for
enzyme activity at 400 nm and protein at 280 nm.<p>

Ion exchange chromatography. 5-10 ml Arabidopsis extract was eluted
from a G25 fine column with a 0.00 1 M Tris-HC1, pH 7.4 buffer. The
first 25 ml eluted from the column were collected and applied to a
column (1.5 x 30 cm) of DEAE-cellulose (Whatman DE 52) equilibrated
with 0.001 M Tris-HC1 buffer, pH 7.4. After washing the column with
the same buffer until no more ultra-violet absorbing material was
eluted, acid phosphatase was eluted with a gradient of increasing NaCl
concentration (0.001 M to 0.25 M). Fractions (5m1) were collected and
analysed as before.<p>

Elution method.  After polyacrylamide gel electrophoresis, strips cut
on the side of the gel were stained for acid phosphatase activity.  As
soon as bands appeared, the strips were replaced into the gel and the
corresponding zones were cut out and squashed by pushing through a
syringe without needle.  2 ml acetate buffer 1M, pH 4.8 was then
added, the mixture homogenized with a Vortex, allowed to stand
overnight at 4*degrees*C and then centrifuged for one hour at 20.000
rpm.  The supernatant is kept at 4*degrees*C, and 0,5 ml used for
enzyme assay.<p>

RESULTS and DISCUSSION<p>

1.  Enzyme polymorphism<p>

Three electrophoretically distinct groups of acid phosphatase (AP1,
AP2, and AP3) have been found in Arabidopsis.  The characteristics of
the AP1 set have been principally identified after starch gel
electrophoresis. AP2 and AP2 sets have been clearly resolved by the
use of flat bed gel acrylamide electrophoresis. These groups differ in
their relative migration rates towards the anode and in their levels
of activity.<p>

The AP1 pattern shows four bands (numbered from 1 to 4 starting with
the least anodal) and contains about 70% of the activity of acid
phosphatase present in the leaves. The intensity of the AP1 bands is
graded so that band 3 had the highest activity, bands 1 and 2 a lower
comparable activity, and band 4 is only slightly active. Three types
of electrophoretic variants of acid phosphatase were discovered among
115 geographical races. We have designated these types as A (slow) I
(intermediate) and F (fast) in order of electrophoretical mobility
with respect to the anode (Figure 1).  Each electrophoretic variant
contains a set of four bands which in comparison to each other are
displaced in a parallel fashon from S to I and from I to F towards the
anode. The value of this displacement is twice the distance between
two successive bands so that the patterns overlap between S and I
types and I and F types.<p>

<a href="/ais/images/vol13/vol13-pg48-fig1.gif">Click here for figure</a><p>

Figure 1: Schematic representation of the three acid phosphatase
systems AP1, AP2, and AP3 in Arabidopsis thaliana.  In each system,
the F/F, I/I or S/S genotypes indicate respectively a fast,
intermediate or slow variant. + indicates the anode, O the origin.<p>

A second acid phosphatase group (AP2) has been identified on
polyacrylamide gel. This group which migrates more rapidly towards the
anode than the AP1 bands consists of two isozymes, and three distinct
phenotypes have been recognized: a slow migrating type (S), an
intermediate type (I) represented in only two races and a fast type
(F) (Figure 1).  No correlations could be found between the variation
observed within the AP1 and AP2 systems.<p>

A third acid phosphatase group is represented by the most anodal band.
This form is the only one among the acid phosphatase isozymes which
does not show activity when stained in a buffer of pH 6.5 or above.
One slow variant has been identified in a few geographical races
(Figure 1).<p>

Table 1 shows the distribution of the AP1, AP2, and AP3 phenotypes
among 115 different geographical races spreading from Sweden to
Portugal and from Siberia to England.<p>

2.  Genetics of the acid phosphatase variants.<p>

a.  Electrophoretic variants.<p>

In the case of the AP1 group, the pattern of the three possible
heterozygous hybrids (S/I, I/F, S/F) shows both group of isozymes,
each of which comes from the homozygous parents.  On a zymogram of a
cross between an S and an I type (Figure 2), 6 bands are present
instead the expected eight parental bands which can be explained by
overlapping between bands 3S and II and between bands 4S and Sl.  A
hybrid between F type and S type displays 8 bands on the gel as there
is no overlapping between the two parental sets (Figure 3).<p>

<a href="/ais/images/vol13/vol13-pg59-fig2.gif">Click here for figure</a><p>

Figure 2: Schematic picture of a AP1 S/I heterozygote in comparison
with the S/S and I/I homozygotes. The hybrid displays six bands.<p>

Hybrid enzymes with a electrophoretic mobility differing from those of
the parental bands do not appear in heterozygotes.  It seems unlikely
that the isozymes are composed of monomers which could randomly
associate to generate the active enzymes.  However, the distribution
of activity between the bands of the heterozygote does not correspond
to that expected from the addition of the parental bands. This pattern
can be obtained with mixtures in a l : l ratio of the F and S types
(see Figure 3).<p>

<a href="/ais/images/vol13/vol13-pg60-fig3.gif">Click here for figure</a><p>

Figure 3: Schematic picture of a AP1 S/F heterozygote in comparison
with a 1:1 mixture of extracts from S/S and F/F types and the parents
S/S and F/F.  Activity levels are indicated by the density of the
dots.<p>

The particular distribution of activity in the heterozygote can be
considered as indication of a dimeric constitution for the enzyme,
with a restriction in the dimerization process to subunits belonging
to the same corresponding bands in the three migration types S, I and
F.  Table 2 gives an example of a possible subunit composition of AP1
isozymes in the case of two heterozygotes S/I and S/F.<p>

The intensity distribution of the bands in heterozygotes suggests that
each subunit retains its characteristic activity in the heterodimer.
Thus here the combinations 3SS 3II or 3S 3F give the densest bands,
the bands IS IF, IS II, 2S 2F and 2S 2I are less dense and the
bands 4I 4S, 4S 4F are the least intense.<p>

<a href="/ais/images/vol13/vol13-pg61-tab1.gif">Click here for table</a><p>

Table 1: Distribution of acid phosphatases phenotypes of Arabidopsis
among 115 geographical races (%)<p>

<a href="/ais/images/vol13/vol13-pg61-tab2.gif">Click here for table</a><p>

Table 2: Proposed subunit composition in AP1 homozygotes and
heterozygotes.<p>

In the F2 generation, all three possible types - the two parental and
their corresponding heterozygote - were found in progenies in
proportions showing a close agreement with the expected 1:2:1 ratio
(cf. table 3) which is consistent with the hypothesis that the AP1
variants are controlled by three alleles at one locus (AP1S, AP1I)
acting without dominance.<p>

In the AP2 system, the hetrozygous condition is characterized by the
presence of both parental bands (Figure 4).  Segregations for AP2
phenotypes in crosses between the two main types S and F lead to the
conclusion that the two patterns are coded by two allelic, co-dominant
genes (AP2S, AP2F) (Table 3).  There is no evidence for the formation
of hybrid enzymes.<p>

<a href="/ais/images/vol13/vol13-pg62-tab3.gif">Click here for table</a><p>

Table 3: Segregation observed in the progeny of crosses made to
determine the inheritance of AP1, AP2, and AP3 variations.<p>

Polymorphism of AP3 is caused by two alleles S and F of the AP3 locus
which determine the primary structure of the polypeptides A and F.
Presumably, these aggregate to form enzymatically active dimers
immediately after they have been synthetised.  Heterozygotes display a
three band pattern composed of the parental homodimer bands and a
heterodimer hybrid band (Figure 4). Phenotypes resulting from the
crosses between the variants are exactly predictable on the basis that
the alleles are codominantly expressed (Table 3).<p>

<a href="/ais/images/vol13/vol13-pg62-fig4.gif">Click here for figure</a><p>

Figure 4: Schematic representation of various combinations of AP2 and
AP3 genotypes after acrylamide gel electrophoresis.<p>

Linkage data between the AP1, AP2, and AP3 loci shows that close
linkage between the loci is ruled out (M. JACOBS and SCHWIND 1975).
AP1-AP2 loci and AP1-AP3 loci appear to segregate independently, but a
possibility of linkage exists between AP2 and AP3.  Results of test
crosses shall be presented.<p>

b.  Activity variant.<p>

An X-ray induced morphological mutant of the race Antwerpen (96) -
normally with AP1I/AP1I genotype-appeared to be a "null-activity"
mutant of the AP1 system. This lack of activity concerned ll four AP1
isozymes, AP2 and AP3 remaining normally active.  Hybrids between 96
and a race Wil-2 (80) with a genotype AP1I/AP1I shows an intermediate
activity as compared to the activity of the parents, noticeable in the
staining intensity of the bands on the starch gel as in the
spectrophotometric data presented in Figure 5.<p>

<a href="/ais/images/vol13/vol13-pg63-fig5.gif">Click here for figure</a><p>

Figure 5: Phosphatase activity determined in extracts from a F/F plant
with a normal activity (+++), from the null activity mutant 96 (open
circles) and their hybrid (closed circles)q.  15 micro l extract was
added to 1 m1 0.01M p-nitrophenyl-phosphate solution in acetate buffer
0.01 M, pH 4.8. The reaction was stopped after 5 min. incubation by
addition of 200 micro l NAOH 1M.<p>

The segregation classes in a F2 population of 138 plants obtained by
crossing 96 and 80 comprised 37 plants with a normal activity, 69 I
plants with an intermediate activity and 32 "null-activity" plants.
The chi squ. value for this F2 segregation indicates that the theoretical
hypothesis of a 1-2-1 segregation may fit with the observed data. This
means that one gene is involved in the control of AP1 activity.
Analysis of test crosses using heterozygotes for the electrophoretic
types (F and I) and for activity (N and O) should allow to decide if
the charge of the enzyme molecules and the activities are specified by
the same gene allelic to the other active genes (AP1*degree*) or by
two different genes.<p>

3. Biochemical characteristics of acid phosphatase isozymes.<p>

a. Non-allelic ixozymes coded by Ap1, and AP2 and AP3 genes.<p>

Genetic analysis has shown that AP1, AP2 and AP3 systems are coded by
three non-allelic genes.  In this case we normally expect to find
three different protein groups charcterised by their own physical and
biochemical characteristics.  The molecular weight of the acid
phosphatase was estimated by Sephadex G-200 gel filtration using a
series of proteins of known molecular weight as standard. The elution
pattern, represented in Figure 6, gives a first estimation of the
molecular weight of acid phosphatases, namely 105,000 daltons for AP1,
95.000 daltons for AP3 and 90.000 daltons for AP2.  The different acid
phosphatase systems were determined after electrophoresis of the
concentrated fractions obtained from the Sephadex column.  An overlap
exists between the fractions containing the three phosphatase systems,
due to the relatively small differences in molecular weight.
Phosphatases comparable in molecular weight (in the range of 100.000
daltons) have been mentioned by several authors (FILBURN, 1973, DI
PIETRO and ZENGERLE, 1967, KUBICZ, 1973, UEHARA et al., 1974).  Acid
phosphatase of high molecular weight such as repressible phosphatase
of Neurosporo crassa (JACOBS and BROWN, 1971), and the violet coloured
acid phosphatase of sweet potato (UEHARA et al., 1974) are composed to
two subunits.  A dimeric structure of the Arabidopsis AP3 system is
indicated by the formation of a hybrid band in the heterozygote and
the AP1 isozymes probably behave also as dimer on the basis of the
relative distribution of intensity among bands of the three possible
hybrid types.<p>

<a href="/ais/images/vol13/vol13-pg64-fig6.gif">Click here for figure</a><p>

Figure 6: Gel filtration of an Arabidopsis extract on Sephadex G-200.
Fractions (2.5 ml) were collected and assayed for protein (280 ml)
(----) and acid phosphatase activity (400 m micro (- - - ). The column
was calibrated with 5 standard proteins to estimate the molecular
weight of the phosphatases.<p>

The optimal pH was determined by eluting the different phosphatases
out of an acrylamide gel.  Aliquots were incubated in a substrate
solution (0.01 M p-nitrophenyl phosphate) in acetate buffer adjusted
to different pH values. Figure 7a ahows that AP1, AP2, and AP3 groups
have very close but distinct optima respectively 4.1, 4.0, and 3.75.<p>

A thermal inactivation study of the three phosphatase systems show
that AP1 and AP2 present closely similar thermal inhibition and that
AP3 is relatively a more temperature-sensitive enzyme (Figure 7b).<p>

These results indicate thus that the three phosphatase systems are
constituted by different proteins.<p>

<a href="/ais/images/vol13/vol13-pg65-fig7a.gif">Click here for figure</a><p>

Figure 7a: Effect of pH on the relative activity of AP1, AP2, and AP3
enzymes. Assay mixture: 0.01 M p-nitrophenyl-phosphate in 0.02 M
acetate buffer within the pH range 3-5.<p>

<a href="/ais/images/vol13/vol13-pg65-fig7b.gif">Click here for figure</a><p>

Figure 7b: Effect of temperature on the activity of AP1, AP2, AP3
enzymes. Assay mixture: 0.01M p-nitrophenylphosphate, in 0.01M acetate
buffer, pH 4.9, incubated for 15 min. at the indicated temperature.
The results are expressed relatively to the control value determined
at 25*degrees*C.<p>

b.  Allelic isozymes coded by the alleles S, I and F of the AP1 gene.<p>

As pointed out by the genetic study, the AP1 isozymes are controlled
by alleles of the same gene. These three isozymes should thus present
very similar biochemical characteristics, although they differ in
their migration rate in electrophoresis.<p>

Results of gel filtration on a G-200 column using extracts obtained
from F/F, I/I, or S/S plants show that the three AP1 types are
characterized by the same molecular weight (+/- 105.000 daltons).
Substrate specificity determination indicate that no significant
differences exist in the two AP1 types (S and I) considered in Table
4.  However, if AP1 activity is compared with the activity of a crude
extract, it appears that the AP1 group was almost inactive towards
nucleoside phosphates.<p>

<a href="/ais/images/vol13/vol13-pg66-tab4.gif">Click here for table</a><p>

Table 4: Substrate specificity of AP1S and AP1I enzymes compared to a
total extract. The assay mixture contained 3 ml of 10 mM substrate in
acetate buffer 0.0 5M, pH 4.8 and ).15 ml enzyme extract.  After
incubation for 45 min. at 30*degrees*C, 3 ml TCA 10% were adjoined.
After centrifugation, 2.5 ml supernatant aliquots were used to
determine PO4 coloremetrically according to FISKE-SUBARROW, 1925.
Enzyme activity was expressd as a percent of that for
p-nitrophenylphosphate (=100).  The partially purified AP1S and AP1I
enzymes were obtained by elution from polyacrylamide gels (see
Methods).<p>

After elution of an Arabidopsis extract by a linear NaCl gradient
(0.01M - 0.25M) applied to a DEAE cellulose column, one peak
corresponding to the AP1 system is observed (Figure 8). By comparing
elution patterns of an extract of S, I, and F type, we notice only one
peak in each case.  However, the order of elution of the isozyme
groups was as might be expected from their electrophoretical
behaviour, i.e., S followed by I followed by F. The NaCl molarity
coresponding to the maximum activity was characteristic of each type.<p>

<a href="/ais/images/vol13/vol13-pg67-fig8.gif">Click here for figure</a><p>

Figure 8: Elution pattern of an Arabidopsis extract of a S/S, I/I and
F/F type from a column of DEAE cellulose.  Acid phosphatase activity
was measured at 400 millimicrons.  Starch gel electrophoresis shows that
the activity peak includes only the AP1 isoymes.<p>

It was thus confirmed that AP1 allelic isozyme series share very
similar properties but differ in their net charge.<p>

c.  AP1 gene - AP1 isozyme relationships<p>

We are now faced with the problem of why in homozygous condition each
allele of the AP1 gene controls a set of four bands in most of the
plant organs.  To obtain more critical evidence that a single gene was
involved in the formation of different isozymes, we have firstly tried
to induce a mutation in this gene.  Seeds of the race Wilna (which
gives a API pattern) were treated by ethyl methanesulfonate (80mM-6h)
and analysed in the M2 generation.  In one case out of 1500 we found a
shift in the electrophoretic pattern from I to F and the occurrence of
F/I hybrids. The mutant showed all the morphological and physiological
characteristics of the race Wilna used in this experiment which
excludes a possible contamination. The spacing between the isozyme
bands in the set is not altered.<p>

The lack of AP1 activity which characterizes the previously mentioned
"null" mutant 96 indicates that a single mutation at the AP1 locus
eliminates all the bands simultaneously as reported for E. coli
alkaline phosphatase (SIGNER et al., 1961).  This confirms the
assumption that the same structural gene is responsible for the
synthesis of the four AP1 molecular forms.<p>

The occurrence of multiple forms of enzymes coded by one single gene
may be due to epigenetic changes of the same primary sequence as:
subunits aggregation, presence of different amount of ligands, action
of peptidase, artifacts due to the extraction method or the
electrophoretic conditions, and the presence of a protein in different
confirmations.<p>

The existence of aggregates is denied by the fact that after gel
filtration on G 200, the four AP1 bands are located in the same
fraction. That the enzyme forms were homogenous with respect to
molecular weight was confirmed by the calculation of retardation
coefficients using 10 and 15% starch gel concentration according to
SMITHIES, 1962.<p>

Secondary modifications of the enzyme molecule could arise for
examples from cleavage of peptide bonds by proteases as reported by
GAZITH et al. (1968) or yeast hexokinase and by GARDNER et al. (1971)
for phytochrome.  No difference in migration pattern and rate was
observable for an Arabidopsis extract kept at 4*degrees*C for a few
days compared with a fresh extract after starch gel electophoresis.
Moreover adding bovine serum albumine up to 40 mg/ml final
concentration in the extraction buffer did not change the pattern or
migration rate. BSA was added in large amount in order to protect the
phosphatase against peptidase activity liberated during the
extraction.<p>

If a defined band of the AP1 system such as the most intense, was cut
out after electrophoresis on starch gel and re-run under the same
conditions it again migrates as a single band and with its
characteristic migration rate.<p>

The possibility that the heterogeneity of enzymes should be due to
"artifactual" interactions during the extraction and/or the migration
was thus not confirmed. This question of possible artifacts is
important in relation to the pattern described by GROVER, 1975 for
acid phosphatase in Arabidopsis.<p>

Another possibility may be that different bands in a set differ by the
amount of charge groups conjugated to the protein, i.e.,
polysaccharide residues.  Prostatic acid phosphatase has been shown to
be a glycoprotein with differing numbers of sialic acid residues
(SMITH and WHITBY, 1971).  However, Arabidopsis extract incubated with
neuraminidase does not show any conversion of acidic forms of acid
phosphatase to more basic forms. Therefore the association of various
numbers of sialic acid residues with Arabidopsis acid phosphatase does
not account for the hetrogeneity of the AP1 enzyme.<p>

Finally, the existence of the same enzyme in various conformations has
also been suggested as a possible explanation for multiple enzyme
forms determined by a single allele.  (KITTO et al. 1966, PAHLICH,
1972).<p>

Elution of an Arabidopsis extract by a linear NaCl gradient applied to
a DEAE cellulose column followed by electrophoresis of the fractions
of the activity peak revealed a progressive shift-band 1 to band 4
according to the fractions. The AP1 system is thus formed by four
molecules differing in net charge.<p>

A similar situation where a single allele determines the appearance of
several electrophoretically distinct isozymes have been reported for
red cell acid phosphatase (FISCHER and HARRIS 1971) and esterase E4 in
maize (HARRIS 1968).<p>

4.  Organ-specific patterns.<p>

Zymogram for the AP1 isozymes have been obtained from various organs
during the life cycle of Arabidopsis.  While the activity in the AP1
system was present in all tissues in the same 4 bands pattern,
quantitative variation among the bands was noticed. Bands 1 and 2 are
very weak in root extract in contract to band 3 which is particularly
intense in flower homogenates.<p>

Seed extracts are characterized by a very distinct pattern comprising
8 to 9 anodal bands and 2 cathodal bands. Seeds hydrated for 24 hours
show a similar pattern to dry seed extracts but a progressive
reduction of the number and activity of these seed bands was observed
40-50 hours after sowing on moistened filter paper. These changes
progressed until a typical leaf pattern was reached with the emergence
of the cotyledons. Emergence was accompanied by an increase in
activity of a defined band which corresponds to band 3 of the leaf
isozyme set (Figure 9).  The seed enzymes produced in the presence of
AP1F, AP1I, or AP1S alleles exhibit electrophoretic mobilities which
do not parallel those exhibited by the leaf extracts of the same
genotypes.  This lack of parallelism may suggest the absence of
genetic relationships between the two isozyme systems.<p>

<a href="/ais/images/vol13/vol13-pg70-fig9.gif">Click here for figure</a><p>

Figure 9: Schematic representation of the acid phosphatase pattern in
seeds during germination.  Starch gel electrophoresis was performed
24, 40, 50 and 72 hrs after sowing.<p>

There is, however, a correspondence in migration rate between the leaf
bands and some seed bands.  A possible explanation for such a
situation may consist in the co-expression of two different systems:
one characterizing the mature seeds and one already present in the
cotyledons but with low activity before germination.<p>

To confirm the existence of two different systems, we have followed
the activity changes in germinating seeds in the presence or absence
of an inhibitor of protein synthesis, cycloheximide (CHI).  Seed
samples incubated in a mineral solution containing 0.5 migro g/ml were
tested spectrophotometrically for phosphatase activity using
p-nitrophenyl phosphate as substrate. The extracts were also subjected
to starch gel electrophoresis.<p>

Results described in Table 5 show that the de novo synthesis of the
"leaf" enzyme was seriously inhibited by the presence of CHI.
Although germination was not affected, the seedling development was
retarded. The "seed" enzymes were not affected by the inhibitor
suggesting that seed phosphatase activity should be the result of
activation of pre-existing inactive enzymes.<p>

<a href="/ais/images/vol13/vol13-pg71-tab5.gif">Click here for table</a><p>

Table 5: Changes in phosphatase activity in Arabidopsis seeds during
germination in the presence or absence of cycloheximide (0.5
microg/ml).  One AP unit is defined in the assay as the amount of
enzyme that liberates 1 microM p-nitrophenol.  Protein was determined
according to LOWRY 1951.<p>

SUMMARY<p>

1.  The primary purpose of this investigation was to etablish the
genetic relationship between the various acid phosphatase enzyme types
and to determine the nature of the differences which are responsible
for their differential migration rate in electrophoresis.<p>

2.  Among geographical races, 3 acid phosphatases were found, AP1,
AP2, and AP3.  The alleles AP1S, AP1I, and AP1F specify each a series
of isozymes with different rates of anodal migration in starch gel
electrophoresis at pH 8.3. The third band of each series is the most
intense after staining. The AP2 system included 3 alleles, S, I, and F
and each allele specifies 2 bands in each homozygote.  The AP3 system
is characterized by 2 alleles S and F, each allele coding for 1 band.
A "null" activity mutant for AP1 has been characterized.<p>

3.  The relative activity distribution among the bands in
heterozygotes of the AP1 system has been interpreted as due to the
formation of hybrid enzymes with preferential association of monomers.
AP3 enzymes should be constituted by two subunits on the basis of the
formation of a hybrid enzyme in a heterozygote AP2S/AP3F.<p>

4.  Evidence obtained from crosses between races of different AP1
constitution and from a mutation shifting an I type to a F type, leads
to the conclusion that all the genes responsible for the production of
different series are allelic (AP1S, AP1I, AP1F).<p>

5.  It was shown by biochemical studies that the differential
electrophoretic migration rates are due to variation in charge rather
than size.<p>

6.  A single structural locus (AP1) appears to be involved in the
appearance of four distinct enzyme forms. They repreent probably
conformational stable isomers of a single polypeptide chain although
the activity of a peptidase operating after synthesis of the primary
chain of the enzyme could not be excluded.<p>

References:<p>

BROWN, A.H. and R. H. Allard, Crop Sci. 9, 72-75 (1969)<p>

DI PIETRO, D.L. and S. Zengerle: J. Biol. Chem. 2112, 3397-8 (1967)<p>

ENDO, I., B.B. Shahi, and Chiang Pai: Japan, J. Genet. 46, 147-52
(1971)<p>

EFRON, Y.: Genetics 65, 575-87 (1970)<p>

FILBURN, C.R., Arch. Biochem. Biophys. 159, 683-693 (1973)<p>

FISHER, R.A. and H. Harris: Ann. Hum. Genet. 34, 439-48 (1971)<p>

FISKE, C.H. and Y. Subarrow, J. Biol. Chem 66: 375 (1925)<p>

GARDNER, G., C.S. Pike, H.V. Rice and W.R. Briggs: Plant Physiol. 48,
686-93 (1971)<p>

GAZITH, J. I.T. Schulze, R.H. Goodling, F.C. Womack, and S.P.
Colowick: Ann. N.Y. Acad. Sci. 151, 307 (1968)<p>

GROVER, N.S. and O.R. Byrne: Biochem. Genetics 13, 527-31 (1975)<p>

HARRIS, J.W.: Genetics 60, 5-186-187 (1968)<p>

JACOBS, M. and F. Schwind: Pl. Sci. Letter 1, 95-104 (1973)<p>

JACOBS, M. and F. Schwind: Isozymes IV Genetics and Evolution, San
Francisco, p. 349-369 (1975)<p>

JACOBS, M.M., J.F. Nye and D.M. Brown: J. Biol. Chem. 246, 1419-25<p>

KITTO, G.B., P.M. Wasseman and N.D. Kaplan: Proc. Natl. Acad. Sci.
U.S.A. 55, 573 (1966)<p>

KUBICZ, A.: Acta Bioch. Polonica 20, 223-29 (1973)<p>

LOWRY, O.H., N.L. Rose Brough, A.L. Farr and R.J. Randall, J. Biol.
Chem. 193, 256 (1961)<p>

PAHLICH, E.: Planta 104, 78-88 (1972)<p>

SIGNER, E.A. TORRIANI and J. LEWINTHAL: Cold Spring Harbor Symp. 25,
31 (1961)<p>

SMITH, J.K. and L.G. Whitby: Biochem. Biophys. Acta 151, 607-618
(1968)<p>

SMITHIES, O.: Arch. Biochem. Biophys. Suppl. 1, 125-131 (1962)<p>

UEHARA, K., S. Fujimoto and F. Taniguhi: J. Biochem. 75, 627-38 (1974)<p>

