<TITLE>How to map using the Lister and Dean RI Lines</TITLE>
<!-- Changed by: Dr Mike Arnold,  6-Sep-1996 -->
<body bgcolor=#ffffff>

<center><H2>How to map using the Lister and Dean RI Lines</H2></center>
<hr>

<BLOCKQUOTE>

Take me to the <A HREF="All_chroms.html">latest RI map</A>. <P>


The Recombinant Inbred (RI) lines were generated from a cross 
between the <I>Arabidopsis</I> ecotypes Columbia and Landsberg <I>erecta</I> 
(Lister and Dean, 1993). 300 lines were generated to be used for 
mapping. A large number of markers have been mapped using 
100 of these lines. Mapping a new marker using these 100 will 
give an accurate map position relative to >450 other markers 
however a rough map position can be determined with a much 
smaller number of these lines (20 - 30, see below). For very fine 
mapping the additional 200 lines can be used but markers in the 
area of interest would then need to be scored on the 200 lines.  <P>
</BLOCKQUOTE>


<BLOCKQUOTE>
One can only directly map something on these lines if it is 
polymorphic between Columbia and Landsberg <I>erecta</I> 
and its segregation is therefore scorable in the RI lines. 
QuantitativeTrait Loci (QTL), ie. loci affecting quantitative traits, 
can also be mapped as they are segregating in the RI population, 
although these may not be obviously polymorphic in the parental 
ecotypes.  <P>
</BLOCKQUOTE>


<BLOCKQUOTE>
<H3>Examples of markers, loci or traits that have been 
mapped.</H3>

<H4>i) RFLP markers</H4>
For example, cosmid (Nam <I>et al</I>, 1989) or lambda 
(Chang et al, 1988)  clones, T-DNA or transposable element 
flanking sequences, cDNAs, RAPDs, ARMS markers, CAPS 
markers, random bits of DNA etc... which show an RFLP 
between Columbia and Landsberg erecta. <P>

Larger probes (ie. cosmid or lambda clones) clearly increase the 
chances of finding an RFLP, however if many restriction 
fragments hybridize to the probe scoring of the polymorphism 
may be difficult, in addition the bands which are polymorphic 
may be cross-hybridizing sequences that map to different 
positions in the genome. Sequences flanking T-DNA/transposon 
inserts or cDNAs (ie 0.1 - 3kb) may be too small to show RFLPs 
at a reasonable frequency so many enzymes may need to be 
tested.  <P>

Enzymes that have been most useful for finding RFLPs between 
Columbia and Landsberg erecta are BclI, BglII, BstEII, CfoI, 
DdeI, DraI, EcoRI, EcoRV, HindIII, SacI and XbaI.  Other 
restriction enzymes which could be tested are the six-base cutters 
with AT-rich recognition sequences (as the <I>Arabidopsis</I> genome 
is AT-rich). Alternatively restriction enzymes with four-base 
recognition sequences can also be tested. Use of small fragments 
to isolate larger fragments (from cosmid or lambda libraries) 
should also help in finding an RFLP. <P> 

<H4>ii) Phenotypic differences</H4>
Two examples are:-<br>
<H5>a) The erecta mutation.</H5>
 Landsberg <I>erecta</I> carries the <I>erecta</I> 
mutation and therefore has a short and erect stature, Columbia has 
the wild-type gene and is therefore taller and less compact. This 
phenotype is segregating in the RI population and was scored in 
the RI lines, and the data compared to the segregation data of the 
67 RFLP markers. This resulted in a map position of the 
<I>ERECTA</I> gene relative to the RFLP markers.  <P>

<H5>b)  Resistance/susceptibility to a fungal pathogen.</H5>
The downy mildew <I>Peronospora parasitica</I> infects <I>Arabidopsis</I>. For the 
isolate No-Co, there is a differential resistance response, with 
Columbia showing the sensitive phenotype (sporulation, no 
hypersensitive response (HR)) and Landsberg <I>erecta</I>, showing 
the resistant phenotype (HR/cell necrosis). This trait segregated as 
a single locus with the Ler allele dominant for resistance. In order 
to map the locus (RPP5)  the RI lines were infected with the No-
Co isolate and the response, either resistant (L) or sensitive(C), 
scored (see below).  This data was compared to the segregation 
data of the 67 RFLP markers enabling the map position of RPP5 
to be determined (Parker <I>et al</I>, 1993). <P>

<H4>iii) Biochemical differences </H4><br>
The length and structure of the side chains of several 
glucosinolates vary between Columbia and Landsberg <I>erecta</I>. 
These differences can be detected using gas chromatography. 
Glucosinolate profiles of the RI lines were carried out and using 
this data the map positions of genes affecting the glucosinolate 
pathway were determined  (Magrath <I>et al</I> 1994). <P>

<H4>iv) Quantitative Trait Loci </H4><br>
A wide range of quantitative traits are segregating in the RI lines, 
including stature, rosette size, leaf number and shape, and 
flowering time. These traits were not necessarily different in the 
parental ecotypes but result from the many new combinations of 
alleles in the Columbia and Landsberg <I>erecta</I> genomes that have 
been brought together in the RI lines. These measurements can be 
used in conjunction with the segregation data for the RFLP 
markers to position the multiple loci influencing these traits (C. 
Lister and C. Dean, unpublished results; D. Marshall and M. 
Kearsey, unpublished results).  <P>
<H5>NOTE: concerning markers g4715-a and g4715-b:</H5>
The g4715 cosmid probe
detects two mappable polymorphisms, only one of which was reported
earlier as "g4715" on chromosome I. However, the Dean/Lister data
includes both polymorphisms. "g4715-a" is identical to the original
"g4715" marker on I. "g4715-b", the newer marker, is on chromosome V.
The image file displaying the g4715 polymorphism shows "g4715-a". <P>

<H3>Steps to Mapping a Locus</H3> <P>

<H4>1. Identify an RFLP</H4>
Identify an RFLP or other polymorphic feature in the parental 
ecotypes (see above) or a quantitative trait which can be scored in 
the RI lines. <P>

<H4>2. Sow out the RI lines.</H4> 
Plants to be used for DNA preps can be 
grown either in the glasshouse and tissue culture (for leaves) or in 
liquid media in flasks (for mainly root material). Plants that are 
grown for phenotypic or biochemical examination may require 
special growing conditions or treatments to reveal/accentuate the 
differences between Columbia and Landsberg erecta (ie 
inoculation with fungus, different light regimes, etc..).  <P>

</BLOCKQUOTE>
<BLOCKQUOTE>

If only using 20-30 RI lines for mapping the 
following lines have been selected as having the 
highest frequency of recombination over the five 
chromosomes and therefore should be the most 
informative for mapping purposes:-<br>
33 115 190 191 194 217 231 232 238 245 263 267 
283 284 288 295 302 332 356 370 (=20 lines)<br> 
4 5 13 35 37 242 303 358 367 377 (=30 lines) <P>

<H4>3. Score the lines for your locus</H4> 

i) RFLP markers. Harvest plant material (leaves or roots), 
make DNA. Do restriction digests, Southern blots, and 
hybridization experiments or do PCR reactions and run gels. 
Score (see below). <P>

ii) Phenotypic difference. Score differential phenotype in the 
lines. <P>

iii) Biochemical difference. Carry out the biochemical assay and 
score. <P>

(iv) QTL Count or measure the trait under examination, usually 
recorded as a mean from 5-10 individuals/RI line.) <P>

In all the above experiments do not forget to include the Columbia 
and Landsberg erecta parents at  the same time, as controls. <P>

</BLOCKQUOTE>

<BLOCKQUOTE>
<H4>4. Submit the data to NASC.</H4>
RI mapping is conducted using Mapmaker (Lander <I>et al</I> 1987) and so the data
has to conform to the Mapmaker format. We have designed a <A HREF="http://nasc.nott.ac.uk/RI_submit.html" target="_top">special form</A> (URL: http://nasc.nott.ac.uk/RI_submit.html) for data submission to help you send in the data in the correct format. We recommend that you use this form for data submission and that you fill in the information for each line. It has been designed to allow us to automatically enter your data and so circumvents any potential problems with typing errors.
<BR>
<BR>
However, we can also receive data by email to <a href="mailto:arabidopsis@nottingham.ac.uk">arabidopsis@nottingham.ac.uk</a>.
Label your message RIDATA. If sending your information by email please adhere
strictly to the following format, indicating the numbers of the RI lines
used and including the data in (RI) numerical order. <BR>
<BR>
<BLOCKQUOTE>
<H5>Note: concerning plant order:</H5>
The 101 plants in the Dean/Lister
population are presented in the following order. Each symbol
corresponds to one an RI line (for example, "4" corresponds to
CL4, which is Nottingham strain N1900. <P>


<PRE>
4      5      13     14     17     19A    19B    25     29     30
32     33     34     35     36     37     46     52     53     54
59     62     67     68     71     79     84     90     107    113
115    123    125    131    160    161    166    167    173    175
177    179    180    181    182    188    190    191    193    194
199    209    214    217    231    232    235    237    238    240
242    245    253    257    259    263    264    266    267    279
283    284    288    295    296    297    302    303    311    321
332    342    345    349    350    351    356    358    359    363
367    370    377    378    386    390    394    395    397    398
400
</PRE>

</BLOCKQUOTE>
Enter your scores for the RI lines data into a Text file (eg. MacExcel,
or another text file) as follows:-<BR>
A = like Columbia parent<BR>
B = like Landsberg parent<BR>
H = heterozygote (if possible to score)<BR>
- = unscorable or no score<BR>
<BR>
Marker names should start with a * followed by a letter (small or capital)
ie. *w23 and should have a single space or tab between each entry. Names
MUST BE LESS than EIGHT characters. Mapmaker, the programme we use for mapping,
is not case sensitive and so reads &quot;a&quot; and &quot;A&quot; as the
same thing. Please check the current names used for Loci, so as not to generate
conflicts. 
<p>
Your file should look like this:
<pre>
RI line Number		4	5	13	14	17	25	29<br>
*A1			A	A	B	-	A	B	B<br>
*b2			B	B	B	A	H	-	B<br>
*C4			B	A	-	A	B	B	A<br>
</pre>

Save as TEXT ONLY. <BR>
<BR>
<BR>
<H4>5. Enter data into Mapmaker.</H4> <p>
If the data is sent to NASC for mapping then this section is for 
interest only.  Mapmaker for the Macintosh and UNIX are somewhat different to run (Lander <I>et al</I> 1987). If the database of <a href="67_loci.html" target="RI_main">67 markers</a> is being used with these programs we
strongly suggest that you READ THE INSTRUCTION 
MANUALS  and DO THE TUTORIALS before attempting to run 
the program.  <P>
Use of the Kosambi mapping function (as opposed to Haldane) 
appears to give the best fit for recombination data generated in 
<I>Arabidopsis</I> (Koorneef and Stam, 1992). Kosambi mapping 
function takes into account the effects of interference; which 
means that after one recombination event has occurred it is less 
likely that a second one will occur in adjacent regions, in the same 
generation.  <P>
Before entering data into Mapmaker one needs to add two lines 
of information above the markers scores. 
The top line indicates what type of mapping population the data is 
from, the options depend on the version of Mapmaker being used 
(see below). The second line indicates the numbers of 
individuals, the number of markers scored and defines the 
genotype symbols. <P>

If Mapmaker V1.0 is being used the data is considered as coming 
from an F2 segregating population. This requires using a LOD 
value of 6.0 (default is 3.0) for the group and  three-point 
commands, to take account of the multiple rounds of 
recombination. The top line of the data file should read:- <P>
	<CODE>data type f2 intercross</CODE> <P>
If Mapmaker V2.0 is being used on a UNIX one can run the data 
as coming from an RI population ( and therefore use the default 
LOD of 3.0). The default genotypes for this version are 
A=Columbia and B=Landsberg. The top line of the file should 
read:- <P>
		<CODE>data type ri self</CODE> <P>
The second line indicates the number of individuals in the 
population (ie. 100), the number of markers being scored (ie. 46) 
and defines the genotype symbols used in the mapping data (ie. 
CLXYHU or AB-). The respective second lines will look 
something like:- <P>
<CODE>(V1.0)   100 46 0 0 CLXYHU	(V2.0)	100 46 0 AB-</CODE><p>

(PLEASE CHECK the manual of the version of Mapmaker being 
used that these lines are correct for that version, they do vary). <P>

At the end the program will produce a list of the markers with the 
centimorgan distances and recombination fractions between them. 
On the Macintosh this can be converted into a map; this is also 
possible on the UNIX but the map produced is not so 
informative. If Mapmaker V1.0 has been used the the 
centimorgan distances will have to be recalculated, they are 
approximately two-fold too big as they were calculated for an F2 
population. The formula is:- <P>
<PRE>
r = R/2(1-R)
	where R is the recombination fraction.
	eg. the distance between two markers is 16.5cM (R=15.9%)
r = 0.159/2(1-0.159)
  = 0.094
  = 9.4cM

</PRE>

<hr>
<H4>REFERENCES</H4>
Chang C., Bowman J.L., DeJohn A.W., Lander E.S. and 
Meyerwitz E.M. (1988). Restriction fragment length 
polymorphism linkage map for Arabidopsis thaliana. PNAS <B>85</B> 
6856-6860	 <P>

Koornneef M. and Stam P. (1992). Genetic analysis.  In 
"Methods in Arabidopsis Research" (Koncz C.,Chua N-H and 
Schell J., eds.)  World Scientific Publishing Co. Pte. Ltd., 
Singapore.  83-99 <P>

Lander E.S., Green P., Abrahamson J., Barlow A., Day M.J., 
Lincoln S.E., and Newberg L. (1987). Mapmaker: an interactive 
computer package for constructing primary genetic linkage maps 
of experimental and natural populations. Genetics <B>121</B> 174-181 <P>

Lister C. and Dean C. (1993). Recombinant inbred lines for 
mapping RFLP and phenotypic markers in Arabidopsis thaliana.  
Plant Journal <B>4 </B>745-750 <P>

Magrath R., Bano F., Morgner M., Parkin I., Sharpe A., Lister 
C., Dean C., Turner J., Lydiate D. and Mithen R. (1994) 
Genetics of aliphatic glucosinolates. 1. side chain elongation in 
Brassica napus and Arabidopsis thaliana. Heredity (in 
press).  <P>

Nam H-G, Giraudat J., den Boer B., Moonan F., Loos 
W.D.B., Hauge B.M. and Goodman H.M. (1989) Restriction 
fragment length polymorphism linkage map of Arabidopsis 
thaliana. Plant Cell <b>1</b> 699-705 <P>

Parker  J., Szabo V.,Staskawicz B., Lister C., Dean C., Daniels 
M. and Jones J. (1993) Phenotypic characterization and molecular 
mapping of the Arabidopsis thaliana locus  RPP5 determining  
resistance to Peronospora parasitica. Plant Journal <B>4  </B>821-831 <P>

</BLOCKQUOTE>
<hr>
</body>

