<TITLE>INRA-Versailles T-DNA lines</TITLE>
<center>
<H2><img src=INRA.gif> <P>History of the Arabidopsis T-DNA lines from 
INRA-Versailles</H2>
</center>

Nicole Bechtold and George Pelletier <P>

Station de Genetique et d'Amelioration des plantes,<br>
INRA,<br>78026 Versailles cedex,<br> France <P>
<hr>
<B>SEE <A HREF="nasc.html">NASC Update</A> FOR RELEASE DETAILS</B>
<hr>
<H2>Origin</H2>
The deposited 148 bulks corresponding to 1480 lines came from a collection 
initiated in 1992 by the discovery of a new transformation method by 
vacuum infiltration (Bechtold <I>et al.</I>1993). In 3 years we selected 16 000 independent 
transformants and screened 6,000 of them <I>in vitro</I> and in the greenhouse 
for mutations affecting hypocotyl elongation or gametogenesis and for 
flower specific promoter trapping. <P>

<H2>Transformation method</H2>


For the <I>in planta</I> transformation method by vacuum infiltration we used 
the <I>Arabidopsis</I> ecotype Wassilevskija and the binary vector pGKB5, 
constructed by D. Bouchez (Bouchez <I>et al.</I> 1993). This vector contains a promoterless GUS 
reporter gene fused to the right border, and the genes conferring kanamycin 
and Basta resistance as plant selection markers. <P>
Four-week old plants, well developed, with the first fruits formed and 
secondary inflorescences appearing, were infiltrated for 20 minutes with an  
<I>Agrobacterium</I>  culture (OD600nm= 0.8) resuspended in one-third of the 
initial volume in infiltration medium. Plants were then transferred to soil 
and 4 weeks later the seeds were harvested. The T1 transformants were then 
selected on sand sub-irrigated with water containing Basta (5-10 mg/l 
phosphinothricin), transferred to soil, and grown in plastic tubes to prevent 
seed contamination. The T2 seeds were harvested 6 weeks later. <P>

<H2>pGKB5</H2>


A new plant transformation vector (see Figure 1 below) was designed, that could be 
used for T-DNA tagging in <I>Arabidopsis</I> . The T-DNA region of this plasmid 
is flanked by fragments containing the right and left borders of the TR-DNA 
of pRiA4 (Jouanin <I>et al.</I> 1989). The GUS-nos3' reporter cassette from pBI101.1 (Jefferson <I>et al.</I> 1987) is inserted 40 
bp away from the right border. As the GUS gene possesses its own ATG 
initiation codon, it is able to produce active transcriptional or translational 
gene fusions upon its insertion in the genome. The T-DNA also contains 
two plant selection markers derived from pGSFR280 (De Blok <I>et al.</I> 1987) that confer 
resistance of plant cells to kanamycin and to the herbicide Basta 
(phosphinothricin). <P>
This binary plasmid derives from a pBGS plasmid (Spratt <I>et al.</I> 1986), harbours a bacterial 
kanamycin resistance gene, and is able to replicate both in <I>Escherichia coli</I> 
and in <I>Agrobacterium</I>. The origin of replication of pRiA4, cloned as a large 
8 kb BamHI fragment from pLJbB11 (Jouanin <I>et al.</I> 1985), confers a very high stability in 
<I>Agrobacterium</I> under non selective conditions : <I>Agrobacterium</I> strains 
containing pGKB5 show no detectable loss of the plasmid after 25 
generations in medium lacking kanamycin. The binary vector was 
introduced into several <I>Agrobacterium</I> disarmed strains by electroporation : 
C58C1 (pMP90) (Koncz and Schell 1986), C58C1 (pGV2260) (Deblaere <I>et al.</I> 1985), LBA4404 (Hoekema <I>et al.</I> 1983) to give respectively the 
strains MP5-1, GV5-2, LB5-1.
Strain MP5-1 was used for obtaining all the T-DNA lines generated in this 
collection. <P>

Figure 1. Functional map of the binary vector pGKB5. <P>
<img src=versailles.gif> <P>
Arrows indicate coding sequences, 
and black boxes promoter / terminator regions. The right and left border fragments derived 
from the TR-DNA of pRiA4 from Agrobacterium rhizogenes strain A4 [3] ; the black strips 
correspond to the 24 bp border sequences that serve as signals for T-DNA transfer. The 
chimeric kanamycin and Basta resistance genes originate from pGSFR280 (De Blok 
<I>et al</I> (1987). Sites for 
EcoRV are indicated above the T-DNA. The hatched area corresponds to the probe used in 
hybridization experiments. A large origin of replication from pRiA4 (ori pRiA4) (Jouanin <I>et al</I> 1985)has 
been used to insure a good stability of the vector in Agrobacterium even without selection 
pressure.
uidA : coding region of the beta-glucuronidase from E. coli ; nos 3' : 3' region of the nopaline 
synthase gene from pTiC58 ; ocs 3' : 3' region of the octopine synthase gene from pTiAch5 ; 
nptII : neomycin phosphotransferase II ; P nos : promoter region of the nopaline synthase 
gene ; P 35S : promoter of the 35S transcript of the Cauliflower Mosaic Virus ; bar : coding 
sequence of the basta resistance gene from Streptomyces hygroscopus ; 3' g7 : 3' region of the 
gene 7 from the T-DNA of pTi15955. <P>


Select <A HREF="bouchez.html">here</A> to view the complete sequence of the T-DNA. <P>


<H2>Mutation screening</H2>


The T2 families for which enough seeds was produced, were screened in 
vitro on medium as described by (Estelle and Somerville 1987) and modified 
according to (Santoni <I>et al.</I> 1994). 
About 100 seeds were sowed on medium containing 100 mg/l kanamycin to 
estimate the number of T-DNA insertions and screen for gametophytic 
mutations. The screening of the hypocotyl development mutations was 
carried out without selection in light and darkness 10 days after sowing.
About 50 seeds of each T2 families were also sowed on compost without 
selection in the greenhouse to screen early development and gametogenesis 
mutations. <P>

<H2>GUS screening</H2>


The GUS activity of 2,000 first T2 lines was tested histochemically on 15-
days-old seedlings grown <I>in vitro</I> either under light or dark conditions (Mollier <I>et al.</I> 1995).
For all the families we tested the GUS activity on flowers and siliques taken 
from 6 weeks old plants grown in the greenhouse. <P>

<H2>Seed production</H2>


Because of the low number of T2 seeds it was necessary to multiply them 
and send the T3 generation to the Nottingham Arabidopsis Stock Centre. T3 seeds come from the bulk 
of about 50 T2 plants for each family. Seeds were stored at 4C and 15 % RH. 
The aliquots contain hemizygous, homozygous and wild type seeds. <P>

<H2>Available data</H2>


The available data were recorded through analysis of T2 seedlings <I>in vitro</I> 
and flowering plants in the greenhouse. This information includes: <P>
<ul>

<LI> <I>in vitro</I> segregation on about 100 seeds sowed on kanamycin 100 
	  mg/l (corresponding to 3:1 segregation, 15:1, 63:1, 2:1, ?= number of
	  resistant plants <number of sensitive plants (contains also all the 
	  lines were the number of resistant plant=0), no notations = c 2> 3.86 
	  for all the possible segregations). <P>

<LI> GUS expression at seedling stage in vitro  for a limited number of 
	  lines [3].  <P>

<LI>  GUS expression in flowers and siliques in the greenhouse. <P>

<LI>  seedling observations in vitro  <P>

<LI>  plant observations in the greenhouse. <P>
</ul>
<H2>Unavailable lines</H2>


A number of lines are not yet available through NASC because we or our 
collaborators were interested in: <P>
<ul>


<LI>  1:1 segregation for kanamycin resistance. <P>

<LI>  lines containing about 1/4 sterile or partial sterile mutants. <P>

<LI>  hypocotyl development mutants. <P>

<LI>  leaf morphology mutants. <P>

<LI>  leaf necrosis mutants. <P>

<LI>  germination mutants. <P>

<LI>  yellow-green mutants. <P>

<LI> specific GUS expression in flower and siliques. <P>

<LI>  some specific GUS expression in leaves. <P>

<LI>  lines we didn't test in vitro because they yielded only a few T2 seeds 
	  (1/3 of the lines, but they will be tested in T3 generation). <P>
</ul>
<H2>Restriction for the use of the Versailles-INRA T-DNA lines</H2>


These lines are freely distributed by the NASC for fundamental research 
purposes. Commercial application of the information obtained from the 
study of these lines is not possible without prior agreement with INRA. <P>

<H2>Important notes</H2>
<ul>

<LI>  The segregation of the kanamycin marker  was estimated on about 100 
seedlings. This is not always enough to distinguish between  segregations 
like 2:1 and 3:1 for example. It is then necessary to verify the segregation 
with more seeds (about 300). <P>

<LI>  For several lines no resistant progeny were recovered on kanamycin, but 
in Southern analysis some of them do contain the T-DNA. We decided to 
send all the lines even if all the plants were kanamycin sensitive. These 
lines were noted "?" in the segregation data. <P>

<LI>  We also have observed differences between GUS expression at seedling 
stage in vitro and in the greenhouse, and some unrepeatable GUS 
expression in flowers and siliques (GUS expression varied with greenhouse 
conditions). <P>

<LI>  Not all the GUS expression observed was necessarily tissue-specific. We 
just tested the specificity of GUS expression in the flowers and siliques. <P>

<LI>  The lines were not always grown and harvested in the same climatic 
conditions, and it is possible that this lead to some differences in the 
germination rate. <P>

<LI>  In this collection, we observed that 56 % of T-DNA integration are at a 
single locus, and by Southern 75 % of them are in tandem (Bechtold <I>et al.</I> 1993). <P>

<LI>  From the first studies of mutants provided by this collection, it seems that 
about 1/4 of the lines showing a segregation of mutants/ wild type near to 
1:3 will be tagged. As already described by Feldmann, the <I>in planta</I> methods 
also appear to have a mutagenic effect. <P>
</ul>
<H2>References</H2>


N Bechtold , J Ellis, G Pelletier. (1993) In planta Agrobacterium mediated gene 
transfer by infiltration of adult <I>Arabidopsis thaliana</I> plants. C. R. Acad. Sci. 
Paris, Sciences de la vie/Life Sciences <B>316</B>, 1194-9 <P>

D. Bouchez, C. Camilleri, M. Caboche.(1993) A binary vector based on Basta 
resistance for in planta transformation of <I>Arabidopsis thaliana</I>  C. R. Acad. 
Sci. Paris, Sciences de la vie/Life sciences <B>316</B>, 1188-93 <P>

Jouanin L., Bouchez D., Drong R. F., Tepfer D., Slightom J. L. (1989)
Analysis of TR-DNA / plant junctions in the genome of a Convolvulus 
arvensis clone transformed by Agrobacterium rhizogenes strain A4. Plant 
Mol. Biol. <B>12</B>,  75-85. <P>

Jefferson R. A., Kavanagh T. A., Bevan M. W.( 1987) GUS fusions ,  beta-
glucuronidase as a sensitive and versatile gene fusion marker in higher 
plants. EMBO J. <B>6</B>,  3901-3907. <P>

De Blok M., Botterman J., Vanderwiele M., Dockx J., Thoen C., Gossele  V., 
Rao Movva N., Thompson C., Van Montagu M., Leemans J. (1987) 
Engineering herbicide resistance in plants by expression of a detoxifying 
enzyme.EMBO J. <B>6</B>,  2513-2518. <P>

Spratt B. G., Hedge P. J., te Heesen S., Edelman A., Broome-Smith J. K. 
(1986) Kanamycin-resistant vectors that are analogs of plasmids pUC8, pUC9, 
pEMBL8 and pEMBL9. Gene <B>41</B>,  337-342. <P>

Jouanin L., Vilaine F., d'Enfert C., Casse-Delbart F. (1985) Localization and 
restriction maps of the replication origin regions of the plasmids of 
Agrobacterium rhizogenes strain A4. Mol. Gen. Genet. <B>210</B>,  370-374. <P>

Koncz C., Schell J.( 1986) The promoter of TL-DNA gene 5 controls the 
tissue-specific expression of chimaeric genes carried by a novel type of 
Agrobacterium binary vector. Mol. Gen. Genet. <B>204</B>,  383-396. <P>

Deblaere R., Bytebier B., De Greve H., Deboeck F., Schell J., Van Montagu 
M., Leemans J. (1985) Efficient octopine Ti plasmid vectors for 
Agrobacterium-mediated gene transfer to plants. Nucl. Acids Res. <B>13</B>,  4777-
4788. <P>

Hoekema A., Hirsch P. R., Hooykas P. J. J., Schilperoort R. A. (1983). A 
binary plant vector strategy based on separation of vir- and T-region of the 
Agrobacterium tumefaciens Ti plasmid.  Nature <B>303</B>,  179-180. <P>

Estelle M.A. and Somerville C.R. (1987) Auxin-resistant mutants of 
<I>Arabidopsis thaliana</I> with an altered morphology. Mol. Gen. Genet. <B>206</B>,  200-6 <P>

Santoni V., Bellini C., Caboche M. (1994) Use of two-dimensional protein-
pattern analysis for the characterization of <I>Arabidopsis thaliana</I>  mutants. 
Planta <B>192</B>,  557-66 <P>

P. Mollier, P. Montoro, M. Delarue, N. Bechtold, C. Bellini, G. Pelletier. 
Promoterless gusA expression in a large number of <I>Arabidopsis thaliana</I> (1995)
transformants obtained by the in planta infiltration method. C.R.Acad.Sci. 
Paris, Sciences de la vie/Life sciences <B>318</B>, 465-74

