<html>
<head>
<title>TAIR: Cereon Arabidopsis Polymorphisms Help</title>
<meta http-equiv="refresh" content="2;URL=http://www.arabidopsis.org/Cereon/help.html">
<script language="JavaScript" src="/js/navbar.js"></script>
<link rel="stylesheet" type="text/css" href="/css/main.css" >
<script language='JavaScript'>
var highlight = 3; var helpfile="";
</script>
</head>

<body leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" bgcolor="#F5F9FF">


<!-- HEADER using external JavaScript file -->

<script language='JavaScript' SRC='/js/header'>
</script><p>

<!-- End of header -->


<p>
<table width="602" align="center" border="0" cellspacing="0" cellpadding="2">
	<tr>
		<td width="602">
		
		<!-- Content goes here //--> 

<span class="header">CEREON ARABIDOPSIS POLYMORPHISM COLLECTION HELP</span>
</td>
</tr>
<tr>
<td colspan=2>
<a href="#release1">Release 1 README</a> | <a href="#release2">Release 2 README</a> | <a href="#release3">Release 3 README</a> | <a href="#columns">Column Headings</a> | <a href="#notes">Notes and Usage</a>
  <P>
The accompanying tables contains potential nucleotide polymorphisms
between Columbia and Landsberg erecta ecotypes. The data were
generated by Cereon Genomics using all the sequenced BACs up to the
point of each release. The clones listed in the table are ordered by
chromosome and their location on a chromosome.
<P>
The data were kindly submitted by Dr. Steve
Rounsley. All questions referring to the data should be sent to:
<a href="mailto:athal@cereon.com">athal@cereon.com</a>
<P>
<a name=release1><b>Release 1: Released May 3, 2000</b></a>
<P><b>SNPs and INDELs</b><br>
There are 39315 entries (from 981 BACs).  25,274 SNPs (Single
Nucleotide Polymorphism) and 14,041 INDELs (small
insertions/deletions).
<br>
  <b>Large INDELs</b><img alt="[NEW]" src="/images/new.gif" border=0>
<br>
632 INDELs > 100bp found in 341 of the 980 BACs used for Release 1.
<br>  
Size range: Min 101bp Max 38Kb 
<br>
This collection of INDELs are larger than 100bp and were 
omitted from the original polymorphism release due to an 
increased level of false positives contained within.  
Repetitive sequence is the major cause for such false 
positives, and the larger the gap between two matches, 
the more likely one of the matches is due to a match against 
a repeated region.
<br>
The data is being provided with these caveats so please use 
this with caution.  There are however true positives in this 
dataset, particularly the INDELs at the lower end of the size 
distribution, and many people had requested access to these for 
the purposes of specific studies such as transposon analysis.
<P>
<a name=release2><b>Release 2: Released on November 16, 2000</b></a>
<P><b>SNPs and INDELs</b><img alt="[NEW]" src="/images/new.gif" border=0>
  <br>
Release 2 contains 4476 predicted polymorphisms from 124 BAC clones
that have been sequenced by the AGI.  1,633 INDELs and 2843 SNPs were
found.  This is an 11% increase over the previous collection of
approximately 39,000.
<br>
The remainder of the Columbia BACs are currently being processed and
there will be a final update of the Cereon collection shortly after
this is completed.
  <br>
<b>Large INDELs</b><img alt="[NEW]" src="/images/new.gif" border=0>
<br>
72 Large INDELs found in 43 of the 124 BACs used for Release 2.
<br>
Also provided here are INDELs greater
than 100 bp.  These should be treated with caution, as they are more
likely to be the result of artifacts of the analysis method.  However,
many will still be true insertions/deletions and may therefore be of
interest for certain kinds of analysis.
<P>
<a name=release3><b>Release 3: Released on March 21, 2001</b></a>
<P><b>SNPs and INDELs</b><img alt="[NEW]" src="/images/new.gif" border=0>
  <br>
Release 3 contains 12175 predicted polymorphisms from 396 BAC clones
that have been sequenced by the AGI.  2905 INDELs, 9227 SNPs, and 43
Large INDELs were found.  This is an 27% increase over the previous
collection of approximately 45,000. The total number of polymorphisms
now is 56,670.
<P>
<a name=columns><b>Column Headings:</b><br></a>
1. SNP_Name: CER(Cereon)+Cereon's internal ID number e.g. CER454879<br>
2. Chromosome: 1 thru 5<br>
3. BAC_Name: in standard AGI format, ordered by position in chromosome<br>
4. BAC_Accession: GenBank/EMBL/DDBJ accession number<br>
5. BAC_Length: in bp<br>
6. Left_Coor*: The coordinate of the base to the left of the
polymorphic location.  See below.<br>
7. Right_Coor*: The coordinate of the base to the left of the
polymorphic location.  See below.<br>
8. SNP_Type: SNP (Single Nucleotide Polymorphism) or IND
(Insertion/Deletion)<br>
9. IND_Size: size of insertion or deletion in Columbia. Col/Ler. e.g. -4/4 means a
4bp deletion in Columbia and 4/-4 means a 4bp insertion in Col. Left blank for SNPs.<br>
10. SNP_Base: changed nucleotide. Col/Ler e.g. A/T means A in Columbia and T in Landsberg.  Left blank for INDELs<br>
11. Left_Flank: 20 bp directly to the left of the polymorphic location<br>
12. Right_Flank: 20bp directly to the right of the polymorphic
location. See Note below.<br>
13. Restriction_sites (Col): restriction sites in Col from the SNP/IND<br>
14. Restriction_sites (Ler): restriction sites in Ler from the SNP/IND<br>
<P>
*Coordinates:<br>
For a SNP - the two coordinates flank the polymorphic base.<br>
For an insertion in Columbia, the two coordinates flank the inserted sequence.<br>
For a deletion in Columbia, the deletion occurs between the two coordinates listed.<br>
<P>
<a name=notes><b>NOTE:</b><br></a>
1. 20mers are provided for locating the correct coordinates just in
case the BAC sequence changes. Also note that the 20mers were supposed
to be directly flanking the changed nucleotide, but the right flanking
sequence may be off by one base for SNPs - but not for INDs.
<P>
Please note that the coordinates used in the datafiles refer to the originally
submitted BAC sequence.  Many BAC sequences at GenBank have been edited by
the AGI groups in order to produce finished chromosome records.  This
involves removing overlapping regions, and flipping some clones in order to
produce a consistent direction along the chromosome.  In addition, AGI
groups may make alterations at any time to the submitted sequence in order
to correct errors.  This can also cause the original coordinates to be
inaccurate. 
<p>
In order to access the original BAC sequence, you need to use the link
provided in the current GenBank record.  The link will look something like
this:
<p>
"COMMENT: On Dec 16, 1999 this sequence version replaced gi:5729683"
<p>
<b>USAGE:</b><br>
The flanking sequence provided in the Cereon data files attempts to provide
an alternative way to locate the polymorphism.  The 20mers can be used to
<a href="http://arabidopsis.org/blast/">BLAST</a> against the Arabidopsis genome to identify the specified location.
There are some caveats to keep in mind when doing this:
<p>
<ul>
<li>1. This sequence should help find the appropriate location in the
BAC of interest. It is not necessarily unique to the genome.  It may also
match other BACs in the genome, but these are not important for locating the
polymorphism.
<li>2. If the 20mer matches more than once in the BAC of interest, try
using the other 20mer as well and combining the results. You can also use TAIR's <a href="http://arabidopsis.org/cgi-bin/patmatch/nph-patmatch.pl">PatMatch</a>, which allows you to put in the polymorphic sequence as well as its approximate length in between the two 20mer set.
<li>3. If the 20mer does not find a match in the BAC of interest, it
could be that the editing mentioned above may have moved this location to a
neighboring BAC.  In this case, check your search results against the
neighboring BACs.
<li>4. If it still does not match, beware that using the default BLAST
parameters does not always work well with such a small query sequence.
Several things can increase your chances of finding a match in the BAC
sequence of interest.
             <ul><li>   A. Use a smaller database.  An example would be a species specific collection at NCBI, or the <a href="http://arabidopsis.org/blast/">TAIR BLAST</a> server selecting only
Arabidopsis genomic sequences > 10kb
               <li> B. Do not filter for low complexity.
               <li> C. Increase the mismatch penalty to -8.  This should force
identical matches.</ul>
    <li>5. If multiple hits to the same BAC occur - do not panic.  Remember,
many INDELs are caused by a different copy number of a direct repeat.  The
flanking sequence may therefore hit multiple places.  The best bet here is
to pick primers several hundred bases either side of this general region.
</ul>
	<p>

</td>
</tr>
<tr>
<td colspan=2 align=center>
<br>
<br><a href="liability.html">General liability statement</a> | <a href="policy.html">Privacy policy and Security</a>
<P>



		</td>
	</tr>
</TABLE>

<!-- FOOTER -->
<script language="javascript" src="/js/footer.js"></script
</body>
</html>
