<title>koorn-1987-aabuy.html</title>
Vol 25: 
<h2>PROCEDURES FOR MAPPING BY USING F2 and F3 POPULATIONS</h2>

M. Koornneef and P. Stam<p>

Department of Genetics, Agricultural University, 53 Generaal
Foulkesweg, 6703 BM Wageningen, The Netherlands<p>

For linkage analyses testcross populations are in generally preferred,
because of their efficiency and the insensitivity to factors
disturbing monogenic segregation ratios. A drawback of testcrossing is
the fact that it requires double recessive genotypes and in addition
one must be sure that enough cross pollinated seeds can be obtained
without contamination of seeds originating from self fertilization. It
will be shown that procedures using F2 and F3 populations can be
efficiently used for rapid gene localization in a species such as
Arabidopsis where often double recessive genotpes are not immediately
available and where making accurate crosses can be difficult. Although
the genetic theory and procedures for this type of analysis have been
worked out by geneticists many years ago (Allard, 1956 and Mather,
1951 for older references), a short summary of the procedures may be
useful for researchers new in the field of Arabidopsis whole plant
geneticists.<p>

When trying to localize an unmapped gene on the linkage map one
generally starts with a (recessive) mutant that is in repulsion with a
tester mutant. The latter may be represented in a multiple marker
line, as described for Arabidopsis by Koornneef and Hanhart, 1985 and
Koornneef et al., 1987. Linkage with such a tester gene will be
detected by a deviation from independent segregation in the F2
populations (as indicated in the following scheme).<p>

<a href="/ais/images/vol25/vol25-pg36-fig0.gif">Click here for figure</a><p>

From the observed segregation frequencies the recombination fraction,
r, can be estimated very simple by the product ratio (PR) method using
the appropriate tables (e.g., Stevens, 1939). The PR method can only
be used for a "classical" F2 with four classes. For more complex
situations e.g., when the heterozygote of one or of both genes can be
distinguished from both homozygotes, maximum likelihood (ML)
procedures have to be applied. Allard (1956) compilated a large number
of formulae for many different situations. Servitova and Cetl (1984)
described formulae for the case that one mutant is a lethal for which
the heterozygous genotype can be recognized with Muller's embryo test
(Muller, 1963).<p>

All procedues mentioned above assume normal monogenic segregation
ratios. However recessive deficits are often observed for individual
genes. These disturbed segregations are in general due to two factors.<p>

a. Reduced viability of the recessive genotype, which in its extreme
implies complete lethality of this genotype.<p>

b. Reduced transmission of the mutant allele through the gametophyt.
This phenomenon, which is called certation, is commonly assumed to
occur only in pollen.<p>

Parameters for phenomena such as certation and reduced viabilitiy can
be, in addition to recombination frequency, used to predict
segregation ratios for situations where disturbed monogenic ratios
occur. However, because of the limited number of degrees of freedom (3
in a classical F2) these parameters often cannot be estimated
simultaneously. Dellaert (1980) showed that for a large collection of
viable Arabidopsis mutants the average frequency of the recessive
phenotype was 21, 5% and that this recessive deficit was mainly due to
certation. Therefore, the most realistic model to describe F2
segregation ratios should include r to identical for the male and
female gametophyts and certation on the male side for both mutant
alleles as shown in Table 1. Note that certation is assumed to act
multiplicatively in the ab carrying pollen.<p>

<a href="/ais/images/vol25/vol25-pg37-tab1.gif">Click here for table</a><p>

Table 1: Gamete frequencies for a F2 repulsion phase with certation
(on the male side only) against both genes.  *alpha* = factor by which
the contribution of the a allele to fertilization is reduced.  *beta*
= factor by which the contribution to fertilization of the b allele is
reduced.  Usually both *alpha* and *beta* are less than one.  r =
recombination fraction.  N = *alpha* + *beta* + r (1 + *alpha**beta* -
*alpha* - *beta*); a normalizing constant such that the male gametic
frequencies add to unity.<p>

Phenotype frequencies can be derived from the gamete frequencies.  For
a epulsion phase F2 with complete dominance these ae:<p>

<a href="/ais/images/vol25/vol25-pg37-fig0.gif">Click here for figure</a><p>

The different parameters (*alpha*, *beta*, and r) can be estimated by
maximum likelihood procedures using a computer program. Questions may
arise about the effect of factors that disturb monogenic segregations
when one does not take them into account in the estimation procedures
for r.<p>

<a href="/ais/images/vol25/vol25-pg38-tab2.gif">Click here for table</a><p>

Table 2: Segegation data for markers with deviating monogenic
segregation ratios.<p>

Table 2 compares three different calculation methods for actual
segregation data of Arabidopsis that show various degrees of disturbed
monogenic ratios. It appears that the three methods yield
approximately the same etimate of r, especially when one takes into
account the large standard errors. By comparing estimates obtained
with these procedures for fictitious data that were based on very
specific parameters for certation and recombination fractions (data
not shown) it was found that the degree of deviation obtained with the
PR method and ML procedure without certation depends on both the
recombination fraction and the degree of certation against mutant
alleles. No general rule can be given as to whether ML (assuming no
certation) on PR performs best when certation disturbs the segregation
ratios. When reduced viability is the only cause of a particular
recessive deficit the PR method still results in a perfect estimate of
r, as the reduction factor occurs both in the numerator and the
denominator of the PR. In this case the two ML estimators are slightly
biased (data not shown).<p>

A problem with F2 repulsion data, especially in cases of close
linkage, is the inaccuracy of the estimate of r. Mather (1936) (cited
by Allard, 1956) already pointed out the efficiency of using F3
progeny tests in this case. Using the same number of plants for F3
progeny tests is more efficient than increasing the size of the F2
population for r 11%. One needs 16 plants per F3 line to have 99%
certainty to classify an F3 line as segregating when it derives from
a heterozygous F2 plant.<p>

F3 progeny tests are most efficient when the F3 lines can be screened
only for an early and easy to score marker. For example when one is
interested in linkage between a seedcoat gene and a hairless gene, the
seedcoat mutant plants of the F2 are harvested and there progeny is
screened for segregation of the hairless gene, scored at the seedling
stage. Recombination factions can be estimated by combining the F3
data with the F2 data using the ML procedure or from the F3 data
only by the following formula.<p>

<a href="/ais/images/vol25/vol25-pg40-fig0.gif">Click here for figure</a><p>

Even in the case of r = 1% the fraction of the lines tested that
segregates is still 2%.<p>

The F3 progeny tests will most of the time enable the isolation of
the double recessive phenotype which can be used for further analysis
e.g., to obtain testcross or F2 populations in coupling that enable a
more accurate estimate of r.<p>

A computer program for application of the ML method (either with or
without estimation of certation parameters) to F2 data is available
from the authors. The program runs on IBM PC/AT compatible PCs under
MS-DOS. It will be forwarded upon request after sending a 5.25 inch
floppy disk.<p>

REFERENCES<p>

Allard, R.W.: Hilgardia 24, 235-278 (1956)<p>

Dellaert, L.M.W.: Theor. Appl. Genet. 57, 137-143 (1980)<p>

Koornneef, M. and C.J. Hanhart: Arabid. Inf. Serv. (Frankfurt am Main)
20, 89-92 (1983)<p>

Koornneef, M., C.J. Hanhart, E.P. Van Loenen-Martinet and J.H. Van der
Veen: Arabid. Inf. Serv. (Frankfurt am Main) 23, 46-50 (1987)<p>

Koornneef, M., J. Van Eden, C.J. Hanhart, P. Stam, F.J. Braaksma and
W.J. Feenstra: J. Heredity 74, 265-272 (1983)<p>

Mather, K.: Ann. Eugenics 7, 251-264 (1936)<p>

Mather, K.: The measurement of linkage in heredity, London; Methuen
Co., Ltd.; New York; John Wiley & Sons, Inc. pp 149 (1951)<p>

Muller, A: Biol. Zbl. 82, 139-163 (1963)<p>

Servitova, J. and J. Cetl: Arabid. Inf. Serv. (Frankfurt am Main ) 21,
59-64 (1984)<p>

