<title>meyer-1987-aabuu.html</title>
Vol 25: 
<h2>PRELIMINARY REPORT ON A RESTRICTION FRAGMENT LENGTH POLYMORPHISM MAP FOR ARABIDOPSIS</h2>

E.M. Meyerowitz, Caren Chang, A.W. Dejohn and J.L. Bowman<p>

Division of Biology, California Institute of Technology, Pasadena,
California 91125 USA<p>

The genomic size of Arabidopsis thaliana can be measured in two ways.
The first is in the units of meiotic recombination, centimorgans. The
current genetic map of Arabidopsis thaliana, based on visible and
physiological mutations induced for the most part in a background of
the Landsberg (Redei, 1970) ecotype, contains approximately 430
centimorgans, distributed among the five linkage groups (Koornneef et
al., 1983). The second measurement of genomic size is in number of
base pairs of double stranded DNA in the nuclear genome; this size is
approximately 70,000,000 base pairs (70,000 kilobase pairs, kb,
Leutwiler, Hough-Evans and Meyerowitz, 1984; Pruitt and Meyerowitz,
1986). Both of these figures are small in comparison to the similar
genomic measurements of other flowering plants, in fact, the DNA
content of the genome is the lowest known for an angiosperm
(Meyerowitz and Pruitt, 1985). The average number of kilobase pairs
per centimorgan in the nuclear genome is likewise very low for a
flowering plant: 160 kb, also the lowest figure known for an
angiosperm. In addition, the Arabidopsis thaliana genome is remarkable
low in its content of dispersed repetitive DNA, with one rough
measurement indicating that single-copy sequences are interruped by
small pieces of middle repetitive DNA only every 120 to 125 kb (Pruitt
and Meyerowitz, 1986). This indicates that it should be possible to
successively isolate overlapping molecular clones containing
Arabidopsis genomic fragments without a serious impediment due to
repeats, and consequently to proceed from any starting point in the
genome to a region several hundred kilobase pairs distant. This
process would be considerably aided by the small genome, since a
complete lambda library of genomic fragments, large enough to have a
99% probability of containing any genomic region, is only 16,000
plaque forming units (Meyerowitz and Pruitt, 1985). Repetitive
screening of such a small library for overlapping clones is not
technically difficult.<p>

Given the ease of such "chromosome walking", and the small genomic
size (in base pairs), it should be possible to obtain molecular clones
of any region of the genome, including any gene for which a map
position is known, if random genetically mapped molecular probes
distributed every few hundred kilobase pairs throughout the genome are
available. Since there are many mapped Arabidopsis genes, including a
number whose mutant phenotypes are of interest to physiologists and
developmental biologists, but not undertood at a biochemical level, we
have started to collect random molecular fragments of the Arabidopsis
genome, and to correlate them with specific genetic map positions.<p>

The method is that of restriction fragment length polymorphism (RFLP)
mapping (Botstein et al., 1980). It starts with two different ecotypes
of Arabidopsis thaliana; we initially used Columbia (Redei, 1962) and
Niederzenz (Kranz, 1978). Genomic DNA from each strain was purified,
digested in separate reactions with the restriction endonucleases
EcoRI, XbaI, and BgIII, and the digested DNA fragments separated by
agarose gel electrophoresis. A nylon filter blot of the gel was made,
and hybridized sequentially with numerous 32 P-labeled genomic lmbda
clones. Many of the clones showed ecotype differences in the
restriction fragment patterns revealed in the blot autoradiographs,
with the data implying that these ecotypes differ by about 1/2 to 1%
in their DNA sequences.<p>

Since blot pattern differences are codominant genetic markers, a
genetic map has been prepared showing the genetic map location of over
50 of these clones. In the first experiments, a triply-marked strain
containing the mapped mutations an, gl-l, and tz, which mark
chromosomes 1, 3, and 5, respectively, in a Columbia background, was
crossed to Niederzenz. The F1 were selfed, and seeds collected from
individual F2 plants. These seeds were used to grow over 100 pools of
F 3 plants, which were scored for the mapped markers, and from which
DNA was extracted. This DNA was restricted, blotted and probed with
DNA fragments capable of detecting polymorphisms, and the segregation
of reach pairwise combination of markers measured. This gave map
distance in centimorgans between all pairs of markers. From this, a
consistent genetic map was constructed with five linkage groups,
containing 41 newly mapped DNA fragments and the three previously
mapped genetic markers.<p>

<a href="/ais/images/vol25/vol25-pg13-fig1.gif">Click here for figure</a><p>

Figure 1:<p>

Chromosomes 2 and 4 were associated with the two unidentified linkage
groups by a different cross (using Landsberg and Niederzenz as the
parental strains), and there are currently over 50 RFLP markers on the
map. The total map from the combined data of both crosses currently
covers 430 centimorgans, and corresponds well to the visible marker
map. Thus, there are starting points for chromosome walks on average
every 1400 kb, and on average any region of the genetic map is within
700 kb or less of a mapped marker. Of course certain regions of the
map are denser than others in markers: a 40 centimorgan region of
chromosome five contains no RFLP markers, while a region of chromosome
1 has eight RFLP markers in a region of about 16 centimorgans. In this
area the greatest distance between two markers is 4.2 cM. If the
average genomic figure of 160 kb per centimorgan applies to this
region, no sequence in it is farther than 340 kb from a mapped DNA
segment, and any gene that maps to this region could in principle be
cloned without great difficulty. The number of markers on the map is
increasing; before long it will be possible to clone genes in most
regions of the genetic map, with no knowledge of the biochemical
nature of the gene product.<p>

REFERENCES<p>

Botstein, D., R.L. White, M. Skolnick and R.W. Davis: Amer J. Hum.
Genet. 32, 314-331 (1980)<p>

Koornneef, M., J. Van Eden, C.J. Hanhart, P. Stam, F.J. Braaksma, and
W.J. Feenstra: J. Hered. 74, 265-272 (1983)<p>

Kranz, A.R.: Arabid. Inf. Serv. (Frankfurt/Main) 15, 118-139<p>

Leutwiler, L.S., B.R. Hough-Evens and E.M. Meyerowitz: Mol. Gen.
Genet. 194, 15-23 (1984)<p>

Meyerowitz, E.M. and R.E. Pruitt: Science 229, 1214-1218 (1985)<p>

Pruitt, R.E. and E.M. Meyerowitz: J. Mol. Biol. 187, 169-183 (1986)<p>

Redei, G.P.: Z. Verebungsl. 93, 164-170 (1962)<p>

Redei, G.P.: Bibliog. Genetica 20, 1-151 (p18) (1970)<p>

