<pre>
MULTIPLEX SCREENING OF A YAC LIBRARY

Andrew Porter, Elaine Durwood, Beata Luczac & Keith Mitchelson
(Department of Molecular & Cell Biology, University of Aberdeen, 
Marischal College, Aberdeen, AB9 1AS, UK.)

1  THE PROBLEM
 Screening YAC libraries for DNA sequences that are ubiquitous to 
eukaryotes presents a problem.   Direct methods such as colony 
hybridisation are not useful as false positives are detected due to  the 
ubiquitous sequences, or close homologues, present in yeast 
chromosomes.  Sequences  such as minisatellites, simple repeats and 
possibly, highly conserved genes may be difficult to  identify 
unambiguously in a colony screen.

2  THE SOLUTION
 PFGE fractionation of YACs away from the host chromosomes would 
overcome the problem.  This  is not practical, however, for individual 
clones in a library screen.  A multiplex approach was  developed to 
simplify the screen.  YAC clones are first pooled then fractionated away 
from host  chromosomes by PFGE.

1.  PROCEDURE:
  (i) A library of YAC clones (96-well microtitre plate format) is grown in 200 
ul of selective media.  In the most simple pooling format, colonies may be 
grouped into pools corresponding to rows and columns of the microtitre 
plate: 8+12 = 20 pools per plate.   Here each clone is represented twice, 
once in a row pool and once in a column pool.
 *  Different alleles of repeated element DNA may give signals of different 
intensity.  A different  pooling format ensures row & column pools 
contain (approx) equal amounts of each YAC clone.   Clones are pooled 
by columns and by half rows to give 12+8+8 = 28 samples per plate.  
This format  ensures signal strength relates to the extent of probe 
hybridisation to each allele.

  (ii) 50 ul aliquots of an overnight culture are pooled and grown for 24 hr 
with 10 ml of fresh selective media.  Alternatively, a scrape pooling of row 
and columns of duplicate plates (see "Yeast chromosome preparation" 
by Gerda Cnops & Renate Schmidt) may be used directly to prepare 
chromosome plugs.  Alternatively, a scrape pool may be used to 
innoculate rich media for further amplification of YAC molecules.
  (iii) Plug are prepared in LMP agarose (Sea-Plaque FMC) using 
standard methods (Cnops & Schmidt).  Samples covering a entire plate 
are slotted into wells (30x comb) of a 1% agarose gel with 0.5 x TBE 
buffer.
  (iv) CHEF electrophoresis at 150 volt for 20 h, 12oC and using a 20/20 
sec switch gives optimal fractionation of YAC molecules.
  (v) The most likely positive clone is located at the intersection of the row 
& the column identified by a positive hybridisation signal.  When 
screening for repeated sequences, where several positives may occur 
together in a pool of clones, the variety of sizes of YAC molecules in a 
pool aids identification of likely candidate clones.
  (vi) The process must then be repeated with individual YAC clones to 
