<pre>
IN SITU HYBRIDISATION TO CHROMOSOMES OF ARABIDOPSIS 
THALIANA

Jola Maluszynska, Andrew R. Leitch, Ilia J. Leitch, Kesera Anamthawat-
Jonsson, Trude Schwarzacher & J.S. (Pat) Heslop-Harrison
(Karyobiology Group, JI Centre for Plant Science Research, Colney 
Lane, Norwich NR4 7UH, U.K.)

*  DNA-DNA in situ hybridisation can be used for localising highly 
repetitive and single copy DNA sequences on chromosomes and 
interphase nuclei of Arabidopsis thaliana.  In our laboratory, we routinely 
use DNA probes labelled with digoxigenin and detected with 
fluorescent tags.  The method is presented in this protocol.  Also given 
is the method for detecting biotin-labelled DNA probes using enzymatic, 
colorimetric detection.

1.  SOLUTIONS
  (i) CITRIC ACID-SODIUM CITRATE BUFFER, pH 4.8
A:  0.1M citric acid-monohydrate
B:  0.1M tri-sodium citrate-2-hydrate
Stock:  40 ml A + 60 ml B
Use:  1:9 dilution in distilled water.

  (ii) ENZYME SOLUTION
20% pectinase (Sigma No. P5146, from Aspergillus niger, solution in 
40% glycerol)
2% cellulase (Calbiochem No. 21947)
made up with 0.01M citric acid-sodium citrate buffer

  (iii) HYBRIDISATION MIX
 *  The hybridisation mixture is usually prepared immediately prior to use 
but can be stored in the  freezer for months.

 Amount recommended per slide (Total 20 ul)
100% formamide (a) 10 ul
50% (w/v) dextran sulphate (b) 4 ul
20 x SSC 2 ul
Probe made up to (c) 2 ul
Blocking DNA (d) 1 ul
10% (w/v) SDS in water 1 ul

 (a)  High grade formamide is used, it must be stored at -20oC, preferably 
after passing through an  ion exchange resin.
 (b) In water (the solution is difficult to dissolve) and stock solution must 
be filtered through a  millipore filter (0.22 um).
 (c) Total amount of probe typically used is 50-100 ng per slide.
 (d) Blocking DNA, e.g., salmon sperm DNA is used at 4-10 ug per slide.

  (iv) MCILVAINE'S BUFFER (100ml-1)
0.1M citric acid (18 ml)
0.2M Na2HPO4.2H20 (82 ml)


  (v) DIAMINO BENZIDINE (DAB)
5 mg DAB in 0.5 ml water, kept in freezer
5 ml water
5 ml 0.1M Tris HCl, pH 7.4

  (vi) SILVER STAIN STOCKS
  A: 0.2% (w/v) ammonium nitrate
0.2% (w/v) silver nitrate
1% (w/v) tungstosilicic acid
0.5 % (v/v) formaldehyde (diluted from stock 30 % (v/v) formaldehyde in 
water)
Made up in deionised water
  B:  5% (w/v) NaCO3 in deionised water

  (vii) GIEMSA COUNTERSTAIN
4% (v/v) Giemsa (Gurr) stain in S&#246;renson's buffer:
0.05M Na2HPO4
0.05M KH2PO4

2.  MATERIAL
  (i) Seedlings of Arabidopsis growing on agar mineral medium (e.g., 
Murashige and Skoog, without sugar), 3-4 days old (radicle 3 to 7 mm 
long) are used.
  (ii) Place seedlings into 2mM 8-hydroxyquinoline for 30 min at r.t. and 
then 30 min at 4oC.
  (iii) Transfer material into cold (4oC) fixative (3:1 methanol:glacial acetic 
acid) for one h at r.t. and then store in the deep freezer (-20oC) until use.

3.  CHROMOSOME PREPARATION
  (i) Rinse fixed seedlings for 15 minutes in 0.01M citric acid-sodium 
citrate buffer, pH 4.8
  (ii) Digest material in enzyme solution at 37oC for 30 min.
  (iii) Remove the enzyme solution carefully with a Pasteur pipette and 
wash material with citric acid-sodium citrate buffer for 30 min at r.t.
  (iv) Isolate root tips and transfer with micro-pipette into 45% acetic acid.
  (v) Transfer a few (3-6) root tips with about 10 ul 45% acetic acid to the 
slide, cover with coverslip (24 x 24) and squash.
  (vi) Examine preparation under a phase contrast microscope; check: 
cell density, metaphase index, and quality of chromosome spread.
  (vii) Remove coverslip from the squash after freezing on dry ice, and air 
dry.

4.  PRETREATMENT
 *  RNase treatment (DNase-free RNase).  Chromosome spreads are 
treated with RNase which  reduces non-specific hybridisation. 
  (i) Add 100 ul of 100 ugml-1 (w/v) RNase A to the slide, cover with 
coverslip and incubate for 1 h at 37oC.
 *  Stock RNase A is prepared by dissolving 10 mgml-1 of DNase-free 
RNase in 10mM Tris.HCl,  pH 7.5, 15mM NaCl.  Boil for 15 min and allow 
to cool.  Store frozen in aliquots and dilute when  required.  Dilute 1:100 
in 2 x SSC (20 x SSC = 3M NaCl, 0.3M sodium citrate).

  (ii) Wash 3 x 5 min in 2 x SSC. 
  (iii) Dehydrate 5 min in 70%, 90%, 96% and 100% ethanol.

5.  DNA PROBE LABELLING
 *  Standard plasmid or phage DNA extraction protocol are used.  Nick 
translation or  oligolabelling of linear DNA also follows standard 
protocols.  After nick translation or  oligolabelling, 1 ug DNA probe is 
resuspended in 20 ul 1 x TE (100 x TE = 1M Tris.HCl, pH 8,  0.1M 
EDTA).  We always test efficiency of labelling with a dot blot.

6.  DENATURATION
 *  Combined denaturation of probe and chromosomal DNA.
 *  This method is the most satisfactory and seems to leave the 
chromosomes optimally denatured with minimal DNA loss.

  (i) Place paper tissues at the bottom of a chamber and soak the paper 
with 2 x SSC.  This makes the chamber humid and prevents the material 
from drying out.
  (ii) Place the chamber into an oven or a water bath to raise its 
temperature to 90oC, as measured by a thermometer in the chamber 
(typically the heater must be set to 95oC). Leave to equilibrate to the 
temperature.
  (iii) Add 20 ul of the hybridisation mix to each slide and cover with a small 
piece of plastic cut from an autoclavable plastic bag.
  (iv) Quickly place slides into preheated humid chamber and incubate at 
90oC for 10 min.  Typically, after the slides are loaded, the temperature 
of the chamber drops to 75oC, but returns to 90oC within 4 min.
  (v) Transfer the humid chamber with slides to a 37oC water bath or 
incubator.  It is important to transfer quickly so that the material does not 
cool down too rapidly.  This will allow the most similar sequences to 
reanneal first.

7.  HYBRIDISATION
  (i) Hybridise at 37oC in a moist chamber for 12-16 h (o/n).

8.  POST-HYBRIDISATION WASHES
 *  After hybridisation, it is necessary to remove non-specifically bound 
and weakly hybridised  probe to reduce background levels.  We normally 
wash at a slightly higher stringency than the  hybridisation conditions by 
regulating the concentration of formamide, ionic conditions and  
temperature.

  (i) Coverslips are removed in 2 x SSC at 37-42oC.
  (ii) A stringent wash is given. Use either:
(a)  50% formamide in 2 x SSC for 10 min at 40-43oC, or
(b)  20% formamide in 0.1 x SSC for 10 min at 40oC.  The stringent wash 
removes weakly hybridised sequences with less then to 85% identity.
  (iii) Wash in 2 x SSC, pH 7.0, at 42oC for 10 min.  Twice during the 10 
min period tip out the old solution and replace with prewarmed fresh.  
Then take out of water bath and leave to cool.
  (iv) Wash in 2 x SSC, pH 7.0, at r.t. for 10 min, changing the solution 
twice during this time.

9.  FLUORESCENT DETECTION AND AMPLIFICATION

 DETECTION
 *  The digoxigenin is detected using a marker, e.g., FITC conjugated to 
an anti-
 -digoxigenin antibody.

  (i) Place slides in 4 x SSC containing 0.2% Tween 20 (4 x SSC/Tween) 
at r.t. for 5 min.
  (ii) Bovine serum albumen (BSA) block.  Drain slides and add 200 ul of 
5% (w/v) BSA in 4 x SSC/Tween to each slide and cover with a plastic 
coverslip.  Incubate for 5 min at r.t.
  (iii) Labelled anti-digoxigenin (e.g., anti-digoxigenin-FITC, raised in 
sheep).  Drain away BSA and add 30 ul labelled anti-digoxigenin diluted 
1:100 in BSA block.  Incubate for 1h at 37oC.
  (iv) Wash in 4 x SSC/Tween at 37oC for 3 x 8 min.
  (v) One can examine the slide here, amplify the signal or counterstain 
the chromosomes.

 AMPLIFICATION
 *  The digoxigenin signal can be amplified. 
  (i) Goat or rabbit serum block.  Drain slides and add 200 ul of 5% (v/v) 
normal goat (rabbit) serum in 4 x SSC/Tween and cover with a plastic 
coverslip.  Incubate at r.t. for 5 min.
  (ii) Labelled antibody raised against sheep (e.g., Rabbit-anti-sheep-
FITC).  Drain away normal goat serum block and add 30 ul labelled 
antibody diluted 1:40 in goat serum block.  Incubate for 1 h at 37oC.
  (ii) Wash 2 x 5 min in 4 x SSC/Tween at 37oC.
  (iii) One can examine the slides here or counterstain the chromosomes.

 DAPI COUNTERSTAINING
 *  DAPI (4'-6-diamidino-2-phenylindole) counterstains chromatin by 
binding to A-T nucleotide  pairs. 
  (i)  Use about 50 ul of 2 ugml-1 (w/v) DAPI solution in McIlvaine's buffer 
for 10-30 min at r.t.
Stock 100 ugml-1 DAPI in water (stored in freezer) is diluted to use in 
McIlvaine's buffer pH 7.
  (ii)  Wash off with distilled water and air dry.
  (iii) Mount in Citifluor antifade.  Apply about 50 ul Citifluor AF1 antifade 
solution per slide.  Place a thin coverslip (preferably UV transparent and 
of high quality; No.0, 24 x 40) over material and gently squeeze excess 
antifade from the slide with filter paper.

 VISUALISATION
 *  The in situ signal should be visualised using high performance 
objectives.  For high  resolution studies the immersion oil should not 
have fluorescent properties (use Leitz  immersion oil). 

10.  AVIDIN-HORSERADISH PEROXIDASE/DIAMINO BENZIDINE 
(HRPO/DAB) DETECTION
 *  This is an alternative, non-fluorescent detection method.
 *  Horseradish peroxidase is used to reduce diamino benzidine (DAB, a 
dangerous carcinogen)  to produce a brown precipitate at the site of in 
situ hybridisation.  It is visualised by transmitted  or reflection contrast 
microscopy.

  (i) Place slides in 1 x PBS containing 0.2% Tween 20 (1 x PBS/Tween) 
at r.t. for 5 min.
  (ii)  Bovine serum albumen (BSA) block.  Add 200 ul of 5% BSA in 1 x 
PBS Tween 20 to each slide and cover with a coverslip.  Incubate for 5 
min at r.t.
  (iii) Avidin-horseradish peroxidase (avidin-HRPO).  Drain away BSA and 
add 40 ul per slide avidin-HRPO, diluted 1:1000 in BSA block.  Cover 
and incubate for 1h at 37oC.
  (iv) Wash 3 x 10 min in 1 x PBS/Tween.
  (v) Drain slides and add 200 ul DAB detection reagent without H2O2, 
cover and incubate in dark for 20 min at 4oC. 
  (vi) Drain slides and add 200 ul DAB detection reagent with H2O2 (1 ul 
30% fresh H2O2 to 2 ml DAB detection reagent).  Incubate at 4oC for 20 
min.
  (vii) Stop reaction with an excess amount of water.

 *  The slide can be counterstained (below) or the DAB precipitate 
amplified with silver.

 SILVER AMPLIFICATION
  (i) Mix an equal volume of silver stain solution A into solution B.  
Immediately add 500 ul to the material, cover and monitor silver 
deposition under the microscope. 
  (ii) Stop the reaction with water followed by 1% (v/v) acetic acid in water 
for 2 min.

 COUNTERSTAINING
  (i) Incubate slides in a Coplin jar containing Giemsa counterstain 
solution for 10 min at r.t.
  (ii) Wash material with distilled water and air dry.
  (iii) Mount in standard microscope mountant e.g., DPX or Euparal

11.  SUPPLIERS OF NON-STANDARD REAGENTS

 Anti-digoxigenin-FITC:
Boehringer Mannheim GmbH, Biochemica, P.O. Box 310 120, D-6800 
Mannheim 31, Germany.
 Avidin-horseradish peroxidase, Normal Goat Serum:
Vector Laboratories Ltd., 16 Wulfric Square, Bretton, Peterborough 
PE3 8RF, U.K.
 Rabbit-antisheep-FITC:
Dakopatts, 42, Produktionsvej, Postbox 1359, 2600 Glostrup, 
Denmark.
 Citifluor AF1:
Citifluor Ltd., Connaught Building, City University, Northampton 
Square,London EC1V 0HB.

 *  All other reagents from Sigma, BRL or companies listed above.

12.  REFERENCES

 Leitch, A.R., Mosgoeller, W., Schwarzacher, T., Bennett, M.D. & 
Heslop-Harrison, J.S., 1990.  Genomic in situ hybridisation to sectioned 
nuclei shows chromosome domains in grass  hybrids. J. Cell Sci. 95, 
335-341.
 Leitch, I.J., Leitch, A.R. & Heslop-Harrison, J.S. 1991. Physical 
mapping of plant DNA  sequences by simultaneous in situ hybridisation 
of two differently labelled fluorescent probes.  Genome (in press).
 Maluszynska, J. & Schweizer, D. 1989. Ribosomal RNA genes in B 
chromosmes of Crepis  capillaris detected by non-radioactive in situ 
hybridisation. Heredity 62, 59-65.
 Schwarzacher, T. & Heslop-Harrison, J.S. 1991. In situ hybridisation to 
plant telomeres using  synthetic oligomeres. Genome (in press).
 Schwarzacher, T., Leitch, A.R., Bennett, M.D. & Heslop-Harrison, J.S. 
1989. In situ  localisation of parental genomes in a wide hybrid. Ann. Bot. 
