<title>koorn-1988-aadea.html</title>
Vol 26: 
<h2>THE EFFECT OF GENETIC BACKGROUND ON RECOMBINATION IN ARABIDOPSIS</h2>

M. Koornneef and C.J. Hanhart<p>

Department of Genetics, Agricultural University, Dreijenlaan 2, 6703
HA Wageningen, The Netherlands<p>

ABSTRACT<p>

Estimates of recombination percentages obtained from F 2 populations
derived from either crosses of Landsberg 'erecta' multiple markerlines
with the wild type Landsberg 'erecta' or hybrids of these markerlines
with the ecotype Niederzenz were compared to study the effect of
genetic background on recombination in Arabidopsis. It was found that
these estimates of recombination percentages did not differ
significantly between both types of hybrids. As Restriction Fragmant
Length Polymorphism (RFLP) maps are constructed on the basis of inter
ecotype hybrids, this result indicates that a quantitative integration
of the classical and RFLP map is possible in Arabidopsis. In the
progeny of the Landsberg/Nierderzenz hybrids a recessive deficit was
much more often observed than in the progeny of intro Landsberg
hybrids, suggesting the presence of genes, linked to the marker genes,
that affect pollen competition.<p>

INTRODUCTION<p>

Genetic linkage maps hae become increasingly important, especially
because procedures are being developed that facilitate the cloning of
genes for which only a mutant phenotype and a genetic map position are
known. For this procedure, known as 'chromosome walking' it is
necessary that molecular markers ahe been mapped in the close
proximity of the gene of interest. Restriciton Fragment Length
Polymorphisms (RFLP's) are an example of such molecular markers, which
can easily be detected between different ecotpes of Arabidopsis
thaliana. Based on molecular differences between the ecotypes
(Landsberg 'erecta' (Ler), Columbia and Niederzena (Nd), Chang et al.
(1988) constructed an RFLP map of Arabidopsis by studying segregation
for these RFLP's in the progeny of hybrids of the different ecotypes.
Integration of the molecular linkage groups in the classical genetic
map (Koornneef, 1987) was done qualitatively by incorporating a number
of previously mapped mutations in the RFLP linkage analysis (Chang et
al., 1988).<p>

For the application of chromosome walking to clone genes were mapped
on the classical map it is important that both maps can also be
integrated in a quantitative way. This can be achieved by mapping a
mutant of interest on the molecular map or by just combining both maps
assuming that recombination frequencies were not different in the
hetrozygotes used to construct each map. A difference in construction
between both maps is that the classical map has been based almost
exclusively on hybrids that were in Landsberg 'erecta' genetic
background in contrast to the RFLP map that was based on hybrids
between different ecotypes. Effects of genetic background on
recombination frequencies are well documented. In tomato e.g.,
recombination is suppressed up to 14% of the control level in the
hybrid Lycopersicon esculentum x L. pennellii for certain chromosome
segments (Rick, 1969). This makes a quantitative integration of the
tomato RFLP map difficult, which was based on this species hybrid
(Zamir and Tanksley, 1988) and the classical map, based on hybrids
within the species L. esculentum. In addition, environmental effects
on recombination frequencies and statistical inaccuracy may explain
differences in the length of classical and RFLP maps.<p>

In the present report the effect of different genetic backgrounds on
recombination frequencies is tested by comparing these frequencies in
hybrids of Ler with hybrids of Ler and Nd that were grown and screened
simultaneously.<p>

MATERIAL AND METHODS<p>

Marker lines that were homozygous recessive for a number of genes on
one chromosome (Koornneef and Hanhart, 1983) were crossed with either
Ler or Nd (obtained from E.M. Meyerowitz). The resulting hybrids were
grown simultaneously in a greenhouse as described before (Koornneef
and Van der Veen, 1980). The resulting F 2 populations (200-500 plants
derived from 3 different F 1 plants) were tested for marker
segregation at the same time for both genetic backgrounds.
Recombination frequencies were estimated according to the procedures
described by Koornneef and Stam (1988) taking into account certation
in the case of recessive deficits.<p>

RESULTS AND DISCUSSION<p>

Segregation for all except one (dis2) gene did not deviate from the
excepted 3:1 ration (Table 1) in the Ler crosses.<p>

<a href="/ais/images/vol26/vol26-pg75-tab1.gif">Click here for table</a><p>

Table 1: Frequency of recessive homozygotes in the F 2 progeny derived
from Landsberg 'erecta' and Landsberg 'erecta' Niederzenz crosses.<p>

However, in the progeny of the Ler/Nd hybrids about half of the genes
showed a recessive deficit (Table 1). As no indication was obtained
that viability of the respective recessive genotypes was more reduced
in these ecotype hybrids, this deficit most likely is due to reduced
transmission of the mutant alleles through the gametophyt (certation).
Apparently this reduced transmission is an effect of the Ler genetic
background and not of the mutant alleles as such, because this
certation should also be found then in the Ler hybrids. Distortion of
the monogenic ratios was also found in the L. esculentum/L. pennellii
hybrid, however, what alleles were reduced depended on the marker gene
(chromosome segment) and not on the origin of the allele in the case
of tomato (Rick, 1969).<p>

Recombination<p>

For none of the marker combinations significant differences were
observed between the two types of hybrids (Table 2).<p>

This implies that between the ecotypes Ler and Nd no genetic
differences, neither as chromosomal structural differences nor as
other genetic factors, affect recombination frequency. The frequencies
observed in the present experiment are of the same order as those
obtained before (Table 2). It can be concluded that mapping
experiments performed with different ecotypes of Arabidopsis are
expected to give comparable results, which implies that such
recombination data can be combined to improve the accuracy of the
Arabidopsis linkage map, and also that the molecular and classical
maps can be quantitatively integrated. This is in agreement with the
observation that the length of the linkage groups on the classical and
molecular map correspond reasonably well (Chang et al., 1988)<p>

However, as these conclusions are based on a comparison of only two
ecotypes, it cannot be excluded that other ecotype combinations may
affect recombination. In addition environmental effects may also
explain differences in recombination data obtained in different
laboratories.<p>

<a href="/ais/images/vol26/vol26-pg77-tab2.gif">Click here for table</a><p>

Table 2: Estimates of recombination percentages in the heterozygotes
of Landsberg'erecta' and Landsberg 'erecta' x Niederzenz hybrids.<p>

REFERENCES<p>

Chang, C., J.L. Bowmen, A.W. DeJohn, E.S. Lander and E.M. Meyerowitz:
Proc. Natl. Acad. Sci. USA 85, 6856-6860 (1988)<p>

Koornneef, M.: Genetic Maps 1987 (ed. O'Brien, S.J.) Cold Spring
Harbor, NY pp 742-745 (1987)<p>

Koornneef, M. and C. J. Hanhart: Arabid. Inf. Serv. (Frankfurt am
Main) 20, 89-92 (1983)<p>

Koornneef, M. and P. Stam: Arabid. Inf. Serv. (Frankfurt am Main) 25,
35-40 (1987)<p>

Koornneef, M. and J.H. Van der Veen: Theor. Appl. Genet. 58, 257-263
(1980) Rick, C.M.: Genetics 62, 753-768 (1969)<p>

Zamir, D. and S.D. Tanksley: Mol. Gen. Genet. 213, 254-261 (1988)<p>

