<HTML>
<HEAD>
   <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
   <META NAME="Generator" CONTENT="Microsoft Word 97">
   <META NAME="GENERATOR" CONTENT="Mozilla/4.02 [en] (Win95; I) [Netscape]">
   <TITLE>bancroft et al.</TITLE>
<script language="javascript">

<!--

function Page_load(page) {
        

                
           if (page == 4)
                page_title = "Contents.html";
           else if (page == 5)
                page_title = "Contents.html";
           else if (page == 6)
                page_title = "fig1.html";

        if (page_title == "")
                return;
        
        
        window.parent.frame_main.location = window.parent.frame_menu.home_page + page_title;
           
}

        var strng;
        function message(id) {
           

           if (id == 4)
                strng = 'WeedsWorld Volume 4ii Contents ( Sept-1997)';
           else if (id == 5)
                strng = 'WeedsWorld Volume 4ii Contents ( Sept-1997)';
           else if (id == 6)
                strng = 'Figure 1, Bancroft et al';

           else 
                return false;

        window.status = strng;
        return true;
}

// end of script -->

</script>

</HEAD>
<BODY bgcolor="#ffffff">

<BLOCKQUOTE>
<H2>
A strategy involving the use of high redundancy YAC subclone libraries
facilitates the contiguous representation in cosmid and BAC clones of 1.7
Mb of the genome of the plant <I>Arabidopsis thaliana</I></H2>
<FONT SIZE=+0>Ian Bancroft<SUP>1</SUP>*, Karina Love<SUP>1</SUP>, Elisabeth
Bent<SUP>1</SUP>, Sarah Sherson<SUP>2</SUP>, Clare Lister<SUP>1</SUP>,
Christopher Cobbett<SUP>2</SUP>, Howard M. Goodman<SUP>3</SUP> and Caroline
Dean<SUP>1</SUP></FONT>

<P><FONT SIZE=+0>1. John Innes Centre, Norwich Research Park, Colney, Norwich
NR4 4UH, UK</FONT>
<BR><FONT SIZE=+0>2. Department of Genetics, University of Melbourne, Parkville,
Victoria 3052, Australia</FONT>
<BR><FONT SIZE=+0>3. Department of Molecular Biology, Massachusetts General
Hospital, Boston, Massachusetts 02114</FONT>
<BR><FONT SIZE=+0>* Tel. + 44 (0)1603 502200</FONT>
<BR><FONT SIZE=+0>Fax. + 44 (0)1603 505725</FONT>
<BR><FONT SIZE=+0>e-mail</FONT> <A HREF="mailto:bancroft@bbsrc.ac.uk">bancroft@bbsrc.ac.uk</A>
<BR>
<HR WIDTH="100%">
<H2>
<FONT SIZE=+0>ABSTRACT</FONT></H2>
<FONT SIZE=+0>A region of over 1.7 Mb of <I>Arabidopsis thaliana</I> chromosome
4 has been assembled into a detailed clone contig. Attempts to assemble
contigs using only clones from genomic cosmid libraries left numerous gaps
in coverage. To achieve complete coverage, we developed a strategy that
involves the subcloning of gel-purified YACs into highly redundant cosmid
libraries. Using such libraries, contig assembly and restriction mapping
can be combined into a simple routine, involving little hybridization analysis.
The cosmid vector used is designed such that the cloned DNA can be inserted
into plant genomes by <I>Agrobacterium</I>-mediated transformation. Thus
the contigs assembled represent a resource for systematic gene identification
by complementation analysis. Other types of clones, such as Bacterial Artificial
Chromosomes, can be integrated into the strategy. This strategy was used
to provide the clones for the first large-scale sequencing of the <I>Arabidopsis</I>
genome and should prove valuable for the detailed analysis of genomes for
which YAC-based physical maps provide the most complete coverage.</FONT>
<H2>
<FONT SIZE=+0>INTRODUCTION</FONT></H2>
<FONT SIZE=+0>The plant <I>Arabidopsis thaliana</I> (<I>Arabidopsis</I>)
has been widely adopted as a model for molecular genetic analysis of plant
development and metabolism. It has a small genome which, based on currently
available data (<A HREF="#dean">1</A>,<A HREF="#schmidt">2</A>) we estimate
consists of around 105 Mb of low copy DNA and 25 Mb of clustered repeat
sequences. The low copy regions of two of the five chromosomes have been
mapped into yeast artificial chromosome (YAC) contigs (<A HREF="#schmidt">2</A>,<A HREF="#zachgo">3</A>)
and the YAC contig mapping of the remaining three chromosomes is almost
complete. Cosmid-based maps provide much greater resolution than YAC-based
maps due to the smaller sizes of the inserts in these clones. Detailed
maps are required for gene cloning or transcript mapping experiments involving
genomes with high gene densities. Large-scale sequencing of the genome
of <I>Arabidopsis</I> is under way, and will provide the first complete
genetic blueprint of a typical flowering plant. Although some preliminary
investigations have been performed (<A HREF="#chen">4</A>,<A HREF="#vaudin">5</A>),
it has not yet been demonstrated that sequencing templates can be derived
directly from a YAC if completely contiguous sequence is required for megabase-scale
regions at optimal cost efficiency. Thorough analysis of the collinearity
of YACs with the genomic DNA of <I>Arabidopsis</I>, as for most organisms,
is also extremely difficult. For sequencing, it is therefore preferable
to convert the YAC-based clone maps of the <I>Arabidopsis</I> chromosomes
to maps consisting of clones propagated in <I>E. coli</I>. A cosmid-based
physical map of the genome of <I>Arabidopsis</I>, consisting of <I>ca.</I>
750 contigs is available (<A HREF="#hauge">6</A>). These contigs were estimated
to contain 90 to 95% of the genome of <I>Arabidopsis</I>, but most of the
contigs are not integrated with either the genetic or YAC-based maps. The
investigation reported here demonstrates that this resource does not allow
complete contiguity to be achieved using these clones, even when supplemented
with further genomic cosmid clones. Other <I>E. coli</I>-based cloning
systems have recently been used to propagate <I>Arabidopsis</I> DNA, including
Bacterial Artificial Chromosomes (BACs, <A HREF="#choi">7</A>). In order
to obtain complete coverage of large regions of the genome of <I>Arabidopsis</I>,
we have developed strategies that have allowed us to construct a contig
of over 1.7 Mb, composed mostly of cosmids, but including some integrated
BAC clones. This contig corresponds to a well-characterised region of the
YAC-based physical map of chromosome 4 and provided the tiling path of
clones for genomic sequencing of this region (<A HREF="#bevan">8</A>).</FONT>
<H2>
<FONT SIZE=+0>RESULTS</FONT></H2>

<H3>
<FONT SIZE=+0>Cosmid identification</FONT></H3>
<FONT SIZE=+0>Cosmids from the region to be represented were identified
by three different strategies, all of which utilize the available YAC-based
physical map. The preferred source of cosmids is the set of contigs already
assembled by a conventional fingerprinting approach (<A HREF="#hauge">6</A>).
The libraries assembled into contigs were constructed using vectors based
on pLorist (<A HREF="#cross">9</A>). We prefix these clones with "G" (after
the <U>G</U>oodman laboratory, which supplied the clones). Most of the
contigs in the target region had been identified during the assembly of
the YAC-based physical map of chromosome 4 (<A HREF="#schmidt">2</A>).
We completed the search by hybridizing gel-purified YACs, selected from
a 1.2 Mb region of the physical map, to dot-blot filters supplied as a
resource to the <I>Arabidopsis</I> research community. In total, cosmid
clones were identified corresponding to approximately 410 kb (34%) of the
1.2 Mb target region. The location of this region, along with regions subject
to other analyses described later, is shown in <A HREF="fig1.html">Figure
1</A>.</FONT>

<P><FONT SIZE=+0>A second strategy was used to identify cosmids representing
an 800 kb contiguous region (see <A HREF="fig1.html">Figure 1</A>). Gel-purified
YACs were used as hybridization probes to a complete library of <I>Arabidopsis</I>
genomic DNA. This library was constructed using vector pLAFR3 (<A HREF="#stask">10</A>).
The clones from this library are prefixed "CC". Cosmids were also identified
from a library constructed using vector pOCA18 (<A HREF="#olszewski">11</A>)
by hybridization with YAC end probes and subsequent chromosome walking.
These clones are prefixed "cAt". Vectors pLAFR3 and pOCA18 are both based
on a very large replicon, pRK290 (<A HREF="#ditta">12</A>) and produce
libraries with very stable inserts. The sizes of inserts that can be cloned
in these vectors (approximately 15 to 25 for pOCA18 and 20 to 30 kb for
pLAFR3) is smaller than can be cloned in the pLorist-derived vectors (35
to 45 kb). Clones totalling 350 kb of coverage (44% of the region) were
identified, in addition to 350 kb (44%) already identified in assembled
G clone contigs. These clones assembled into eight contigs. Additional
libraries were screened, including libraries of bacteriophage lambda clones,
but contig links were not found.</FONT>

<P><FONT SIZE=+0>In order to complete coverage of this 800 kb region, we
found it necessary to subclone YACs selected from the physical map. To
do this we developed a novel protocol that involved preparative gel-purification
of the YACs, partial digestion with <I>Taq</I>I and cloning into a new
cosmid vector called pCLD04541. This vector, shown schematically in <A HREF="fig2.html">Figure
2</A>, is a binary T-DNA vector, so the inserts can be stably introduced
into plant genomes using <I>Agrobacterium</I>-mediated transformation.
Contigs assembled in this vector can therefore be used in complementation
analyses as part of positional cloning experiments. The subclone libraries
generated typically represent 20- to 60-fold redundant coverage of the
subcloned YAC. Subclone cosmids completed the final 100 kb (12%) of the
first 800 kb region assembled.</FONT>
<H3>
<FONT SIZE=+0>Initial cosmid contig assembly and restriction mapping strategy</FONT></H3>
<FONT SIZE=+0>The G cosmid clones were already assembled into contigs.
The cAt and CC clones that had been identified by colony hybridization
were confirmed by hybridization of Southern blots of these clones with
gel-purified YACs. Contigs were assembled on the basis of matches of common-sized
restriction fragments (CC clones) or by chromosome walking using restriction
fragments of cosmids as hybridization probes to the original libraries
(cAt clones). These contigs were confirmed and integrated with G cosmid
contigs by hybridization to Southern blots.</FONT>

<P><FONT SIZE=+0>Where gaps existed between the integrated G, CC and cAt
clone-containing contigs, chromosome walking was performed within the YAC
subclone libraries. This involved the gel-purification of restriction fragments
from the inserts of terminal cosmids in the assembled contigs and hybridization
of these to gridded array colony filters of the subclone library to identify
overlapping clones. The whole insert, or selected restriction fragments
from the inserts of these clones were then gel-purified and again hybridized
to the subclone library filters. This process was repeated, from both ends
of each gap, until the walks reached common clones and complete coverage
was achieved.</FONT>

<P><FONT SIZE=+0>Two cosmids, CC33M3 and CC16N19 had initially been thought
to overlap as they hybridized to each other. Sequencing of both clones
revealed that they did not overlap, but shared highly homologous sequences.
These cosmids were used to identify several BAC clones, including IGF5D3
and IGF1P20, which overlap them both. (The IGF BAC library was obtained
from Thomas Altmann, MPI fuer Molekulare Pflanzenphysiologie, Golm, Germany.)</FONT>

<P><FONT SIZE=+0>For most applications it is necessary to construct restriction
maps of the clones in the assembled contigs. This allows the accurate assessment
of the overlaps between clones and the size of the contig, accurate placement
of transcripts, probe fragments, <I>etc.</I> It also facilitates the analysis
of the collinearity of the contig with the <I>Arabidopsis</I> genome. The
restriction mapping of the clones was conducted using a combination of
conventional approaches: permutations of multiple enzyme digestion, partial
digestion strategies and hybridizations to Southern blots.</FONT>
<BR>&nbsp;
<H3>
<FONT SIZE=+0>Modified cosmid contig assembly and restriction mapping strategy</FONT></H3>
<FONT SIZE=+0>Chromosome walking within YAC subclone libraries in order
to fill gaps between contigs of genomic cosmid clones proved to be an effective
but laborious process. Repeated sequences were encountered both during
the identification of genomic clones and whilst walking within subclone
libraries. These caused two problems. Repeats present in multiple genomic
locations caused the identification of cosmids that were not from our target
region and clustered repeats lead to contigs being incorrectly assembled
by our hybridization-based approach. After completion of the first 800
kb contig, a new strategy was developed. Instead of making contigs of G,
CC and cAt clones, followed by walking within YAC subclone libraries, contigs
of subclone cosmids were assembled first and other clones identified were
aligned with these later.</FONT>

<P><FONT SIZE=+0>The YAC subclone libraries were gridded at low density
and replicas hybridized with several different probes. These included gel-purified
DNA of the subcloned YAC (usually <I>Taq</I>I partially digested DNA of
a size smaller than that used for library construction) and fragments of
the YAC cloning vector, pYAC4, from positions very close to the YAC cloning
site in either the left or right vector arm ( to identify cosmid subclones
containing the ends of the YAC insert and a part of the YAC vector). Since
gel-purified YAC DNA is always contaminated with yeast DNA, pulsed field
gel-recovered yeast DNA was also used as a hybridization probe. Cosmid
clones could then be grouped by hybridization with gel-purified DNA from
several YACs that overlap the subcloned YAC. This analysis also confirms
that the subclone cosmids are genuinely from the target genomic region.
If the subcloned YAC had been chimaeric or misidentified, some or all of
the cosmid subclones would fail to hybridize with overlapping YACs. DNA
was prepared from a sufficient number of cosmids to permit four- to 10-fold
redundant coverage of the YAC. This DNA was digested with a combination
of the restriction endonucleases <I>Hin</I>dIII and <I>Sal</I>I to give
a "fingerprint" of restriction fragments revealed by agarose gel electrophoresis
and staining with ethidium bromide. Putative partial matches in fragment
content were identified by visual inspection of the fingerprints and repeat
digests run on agarose gels in predicted clone order for confirmation.
Where there was insufficient fingerprint data to allow complete contig
assembly, for example due to under representation of some regions, or too
few restriction fragments, gel-purified inserts of clones from the ends
of contigs were used as hybridization probes to the colony filters to identify
overlapping clones. We used this procedure to investigate a <I>ca.</I>
400 kb region and obtained complete coverage in subclone cosmids.</FONT>

<P><FONT SIZE=+0>For some applications, such as large-scale genomic sequencing,
cosmids constructed using vector pCLD04541 are inappropriate as the major
resource. This is because the vector is large (<I>ca.</I> 29 kb) and the
inserts consequently relatively small (on average <I>ca.</I> 17 kb). Other
types of clones can easily be aligned with the subclone contig <I>via</I>
their <I>Hin</I>dIII plus <I>Sal</I>I restriction digest fingerprints.
An example of the alignment of a pLORIST-derived cosmid relative to cosmids
representing a portion of the subclone contig is shown in <A HREF="fig3.html">Figure
3</A>. The comparison of restriction digest fingerprints of clones derived
independently from the genome allows also the analysis of collinearity
of those clones with the genome. If the sets of fragments identified match
exactly, it is highly likely that the clones both faithfully represent
that portion of the genome. Differences would indicate a rearrangement
in one of the clones.</FONT>

<P><FONT SIZE=+0>The determination of a detailed restriction map for the
clones contained in contigs assembled by restriction digest fingerprints
is very efficient. Clones are selected, in contig order, that differ by
one or very few fingerprint restriction fragments. Separate <I>Hin</I>dIII
and <I>Sal</I>I digests, along with a repeat <I>Hin</I>dIII plus <I>Sal</I>I
digest are performed and the fragments resolved by agarose gel electrophoresis
in contig order. Ambiguities in the order of restriction fragments common
to multiple clones are resolved by analysis of the order in which they
appear and disappear when processing along the contig. The endonuclease
<I>Hin</I>dIII cleaves the DNA of <I>Arabidopsis</I> relatively frequently
(on average <I>ca.</I> every 1.5 kb) giving a detailed restriction map.
<I>Sal</I>I cleaves less frequently and the map for this enzyme is convenient
to use as a contig-scale map.</FONT>
<H3>
<FONT SIZE=+0>Integrated BAC and cosmid contig assembly and restriction
mapping</FONT></H3>
<FONT SIZE=+0>BAC clones were identified from two regions, of 400 kb and
100 kb, as shown in <A HREF="fig1.html">Figure 1</A>, by hybridisation
of gel-purified DNA of the YACs YUP16B9 and YUP24F4 to high density colony
filters of the TAMU BAC library (<A HREF="#choi">7</A>). Contigs were assembled
and restriction maps were constructed for <I>Sal</I>I mainly by hybridisation
of BACs and cosmids to Southern blots of <I>Sal</I>I-digested DNA of the
BACs identified. The gaps between the BACs identified were filled using
cosmid subclone libraries constructed from the YACs YUP16B9 and YUP24F4.
These cosmids were identified using a subtractive hybridisation strategy.
The BACs from the regions were used as hybridisation probes to the subclone
libraries to allow the identification of cosmids from the regions they
do not represent. These cosmids were assembled into contigs and restriction
mapped using the restriction fragment fingerprint approach. BAC and cosmid
contigs were integrated by hybridisation of BACs to Southern blots of the
cosmids assembled into contigs. The 1.7 Mb contig that was constructed
from part of <I>Arabidopsis</I> chromosome 4 using the approaches described, is shown in <A HREF="http://synteny.nott.ac.uk/cgi-bin/display?db=agr&class=Map&object=ESSA1-FCA" target="_new">Figure 4</A> as displayed through <a href="http://synteny.nott.ac.uk/arabidopsis.html#Adev" target="_top">AGR</a>. A <I>Bam</I>HI map is projected for the portion of the contig assembled
using our initial approach, a <I>Sal</I>I map is projected for the portions
assembled using restriction digest fingerprint analysis.</FONT>
<H2>
<FONT SIZE=+0>MATERIALS AND METHODS</FONT></H2>

<H3>
<FONT SIZE=+0>Construction of cosmid vector pCLD04541</FONT></H3>
<FONT SIZE=+0>pCLD 04541 was constructed by the insertion of a fragment
containing a <I>cos</I> site into the <I>Sca</I>I site of plasmid pSLJ1711
(<A HREF="#jones">13</A>), which was derived from plasmid pRK290 (<A HREF="#ditta">12</A>).</FONT>
<H3>
<FONT SIZE=+0>YAC subcloning protocol</FONT></H3>
<FONT SIZE=+0>The following method was developed in order to optimize the
key parameters for successful library preparation from gel-purified YAC
DNA. The YAC DNA preparation protocol yields a high concentration of YAC
DNA, with minimal degradation of endogenous yeast chromosomes. The DNA
is resolved using a Pulsed Field Gel Electrophoresis (PFGE) system that
is well-suited to preparative scale separations and is fragmented for cloning
by a protocol that reproducibly yields sufficient partially digested material
of the correct size for cloning into cosmid vector pCLD04541.</FONT>
<H4>
<FONT SIZE=+0>1. Yeast chromosome preparation</FONT></H4>
<FONT SIZE=+0>Cultures were grown at 30<SUP>o</SUP>C in 200 ml of YEPD
medium (in 1 litre conical flask) until in stationary phase (2 to 3 days).
Cells were collected by centrifugation at 3000 rpm for 10 min. in a Sorvall
RC3C centrifuge. The cells were resuspended in 10 ml 1 M Sorbitol and collected
in a 15 ml graduated screw-capped tube by centrifugation at 3000 rpm for
10 min in an MSE Centaur2 bench-top centrifuge. The cells were resuspended
in 1 M Sorbitol to a total volume of 2.5 ml and 2.5 ml of 1% InCert agarose
(FMC) made up in 1 M Sorbitol and cooled to 50<SUP>o</SUP>C was added.
The cells and agarose were mixed thoroughly by gently pipetting, dispensed
into 100 &micro;l block formers (LKB) and left to set on ice for 10 min.
The blocks were pressed out into 45 ml SCEM (1 M Sorbitol, 100mM Sodium
Citrate pH5.8, 10 mM EDTA, 30 mM 2-Mercaptoethanol) in a screw-capped centrifuge
tube. 0.5 ml of 100 mg ml<SUP>-1</SUP> Novozyme 234 (Novo Enzyme Products)
in SCE (filter sterilized after dissolving) was added and the tube placed
on a cell mixer at room temperature for 4 hours. The blocks were then transferred
to a 15 ml screw-capped centrifuge tube containing 7.5 ml ESP ( 1 mgml<SUP>-1</SUP>
Proteinase K in 100 mM EDTA pH8, 1% Sarkosyl). The tube was then filled
with ES (100 mM EDTA pH8, 1% Sarkosyl) and incubated for 2 days with one
change of ESP (containing 0.5 mg ml<SUP>-</SUP> of Proteinase K). The blocks
were then transfered to a 50 ml screw-capped centrifuge tube and washed
2 x 1 hour with 1 mM EDTA pH8 containing 1 mM phenylmethylsulphonyl fluoride
(PMSF) and 6 x 20 min with 1 mM EDTA pH8 and finally 1 x 20 min with TE.
The blocks were stored at 4<SUP>o</SUP>C.</FONT>
<H4>
<FONT SIZE=+0>2. YAC gel-purification and fragmentation</FONT></H4>
<FONT SIZE=+0>Blocks containing yeast chromosome preparations were loaded,
24 per gel, and resolved by POE on a modified PHOGE PFGE apparatus (<A HREF="#bancroft">14</A>,
(<A HREF="#schwartz">15</A>,(<A HREF="#bancroft2">16</A>). In all cases
the gel used was 1% LMP agarose (BRL). POE gels were run at 320 V with
an effective field angle of 130<SUP>o</SUP>. Runs were for 2 to 4 days
using pulse times appropriate to maximize resolution around the size of
the YACs to be purified.</FONT>

<P><FONT SIZE=+0>After electrophoresis, the gels were stained with ethidium
bromide and the DNA was visualized by exposure to long-wave UV. The YAC
band was quickly excised in a minimum-sized gel slice (usually <I>ca.</I>
2 ml) which was then cut into 6 pieces and incubated overnight in 12 ml
of ESP. They were then washed with PMSF and EDTA, as for the chromosome
preparations, before storage overnight at 4 <SUP>o</SUP>C. The blocks were
then</FONT>
<BR><FONT SIZE=+0>equilibrated with water, transfered to two microcentrifuge
tubes and melted at 65 <SUP>o</SUP>C for 20 min. 300 &micro;l aliquots
were added to 300 &micro;l aliquots of 2 x CA buffer pre-warmed to 65 <SUP>o</SUP>C
(1 x CA buffer is 20 mM Tris pH7.5, 7 mM MgCl<SUB>2</SUB>, 100 mM KCl and
200 &micro;g ml<SUP>-1</SUP> BSA.) and incubation continued at 65<SUP>o</SUP>C
for a further 10 min. Methylation buffer was prepared by adding 33 &micro;l
of New England Biolabs (NEB) buffer 4 and 3.3 &micro;l of 20 mg ml<SUP>-1</SUP>
BSA (Boehringer) to 273 &micro;l water. 8.3 &micro;l of 32 mM SAM and 13.2
&micro;l M.<I>Taq</I>I (NEB) was added and the solution quickly dispensed
into 6 x 50 &micro;l aliquots. To each aliquot, 5 &micro;l of an appropriate
dilution of alpha<I>Taq</I>I (NEB) was added to give final enzyme quantities
of 0.64, 0.32, 0.08, 0.02, 0.005 and 0.00125 Units. Each partial digestion
mix was added to an aliquot of DNA and incubated at 65 <SUP>o</SUP>C for
30 min. 35 &micro;l of 0.5 M EDTA pH8 (pre-warmed to 65<SUP>o</SUP>C) was
added to each tube, which was then transferred to a 37 <SUP>o</SUP>C waterbath
to equilibrate for 10 min. 2 Units of -agarase I (Calbiochem or NEB) was
added and incubation continued at 65 <SUP>o</SUP>C overnight. The tubes
were then centrifuged briefly, heated to 65 <SUP>o</SUP>C for 20 min and
equilibrated at 37 <SUP>O</SUP>C for 10 min. 1 Unit of -agarase I was then
added and incubation continued at 37 <SUP>o</SUP>C for 4 hours. The tubes
were heated to 65 <SUP>o</SUP>C and all six aliquots were pooled in a 15
ml centrifuge tube containing 4 ml of phenol and 0.4 ml of 3 M Sodium Acetate.
5 &micro;l of 10 mg ml<SUP>-1</SUP> carrier RNA was added. The tube was
shaken and the phases resolved by centrifugation at 3500 rpm for 10 min
in Sorvall Centaur2 centrifuge. The aqueous phase was transferred to a
tube containing 4 ml of phenol/chloroform, shaken and phases were resolved
as before. The aqueous phase was transferred to a 30 ml corex tube, 2.5
volumes of ethanol was added and placed at -20 <SUP>o</SUP>C overnight.</FONT>
<H4>
<FONT SIZE=+0>3. Size fractionation of YAC partial digest products and
library plating</FONT></H4>
<FONT SIZE=+0>The <I>Taq</I>I partially digested YAC DNA was collected
by centrifugation, dissolved in 200 &micro;l TE and re-precipitated. It
was then dissolved in 10 &micro;l TE, run on a 1% LGT agarose gel and fragments
>15 kb in size were recovered by agarase treatment. The products were co-precipitated
with 1 &micro;g of dephosphorylated vector DNA (pCLD04541 linearized by
cleavage with <I>Cla</I>I and gel-purified). The precipitated DNA was air-dried
and dissolved in 2 &micro;l of water. Ligation was performed by adding
1 &micro;l of 3 x ligation buffer and 0.2 &micro;l of high concentration
T4 DNA Ligase (NEB) and incubation at 4 <SUP>o</SUP>C for 3 to 4 days.
The ligated DNA was packaged using Stratagene XL or Gold extracts and plated
on a Tetracycline-sensitive derivative of Stratagene's "Sure" strain of
<I>E. coli</I>. Colonies were picked into 384-well microtitre plates and
stored at -80<SUP> o</SUP>C</FONT>
<H3>
<FONT SIZE=+0>Other methods</FONT></H3>
<FONT SIZE=+0>Other methods used were essentially standard (<A HREF="#sambrook">17</A>)</FONT>
<H2>
<FONT SIZE=+0>DISCUSSION</FONT></H2>
<FONT SIZE=+0>We initially attempted to use the pre-assembled G cosmid
contigs as the primary resource for long-range contig assembly. These had
been estimated to contain 90 to 95% of the <I>Arabidopsis</I> genome (6).
Over the 1.2 Mb region screened for coverage in these clones, we identified
approximately 410 kb of coverage in these clones (34%). The shortfall of
coverage is probably the result of a combination of several factors. One
is that the coverage provided by the contigs is less than originally estimated.
Clones of DNA of <I>Arabidopsis</I> in pLorist vectors are often unstable
and a proportion of the contigs may be missing from the "representative"
filters available to us. Also some clones identified on these filters could
not be recovered from the glycerol stocks stored in the laboratory of origin.</FONT>

<P><FONT SIZE=+0>Further cosmid coverage was readily obtained by hybridization
to the pOCA18- and pLAFR3-based genomic libraries. Added to the clones
already identified in pLorist-based contigs, 88% of the first 800 kb region
analyzed was represented. The clones assembled into eight contigs. We were
able to bridge the gaps, leading to a single contig, only by identifying
the missing 100 kb in high redundancy YAC subclone cosmid libraries.</FONT>

<P><FONT SIZE=+0>As it proved necessary to subclone YACs to complete coverage
in cosmids, we tested a strategy that involved the use of YAC subclone
libraries from the beginning. Cosmids in individual YAC subclone libraries
were confirmed and grouped by hybridization with DNA of YACs overlapping
the one subcloned. DNA was prepared from a sufficient number of cosmids,
randomly selected from each group, to permit four- to 10-fold redundant
coverage of the YAC. Contigs were assembled mainly by analysis of restriction
endonuclease digest fingerprints. This simple procedure greatly reduced
the number of Southern blots and hybridizations required to assemble and
map the contig. It also makes the approach less sensitive to problems caused
by repetitive sequences. Only those sequences repeated within the region
covered by the subcloned YAC can cause ambiguities. For our subclone libraries
we chose a cosmid vector, pCLD04541, which includes the sequences necessary
for the integration of the insert into plant genomes by <I>Agrobacterium</I>-mediated
transformation. This allows the libraries to be used directly for complementation
analyses in gene cloning experiments. The maximum size of the inserts is
restricted to around 22 kb in this vector, which produces a very detailed
physical map, but one requiring more clones for complete coverage than
would be required if the inserts were larger. Map resolution is increased
further by the construction of a detailed restriction map, based largely
on the fingerprint data used to assemble the contigs.</FONT>

<P><FONT SIZE=+0>For projects such as large-scale genomic sequencing, the
pCLD04541-based cosmids are not ideal. Although very stable, the relatively
small proportion of insert to vector (average 17 kb insert to 29 kb vector)
makes them inefficient to sequence by the preferred "shotgun" strategy.
In such cases, other types of clones can easily be integrated by comparison
of their restriction endonuclease digest fingerprints with those of the
assembled contig. In this way, any available clones that can be more efficiently
sequenced, such as pLorist-based cosmids, can be exploited. Short contigs
of BAC clones can also be linked together using YAC subclone cosmid contigs
assembled by restriction digest fingerprint analysis.</FONT>
<H2>
<FONT SIZE=+0>ACKNOWLEDGEMENTS</FONT></H2>
<FONT SIZE=+0>We would like to thank Deverie K. Bongard-Pierce for supplying
us with dot-blot filters representative of the pLorist-based contigs and
for sending the clones requested, and Neil Olszewski for supplying the
pOCA18-based genomic library, Renate Schmidt for making available to us,
prior to publication, the chromosome 4 YAC contig data, including the identities
of some integrated pLorist-based contigs, Thomas Altmann for supplying
us with the IGF BAC library, Rod Wing for supplying us with the TAMU BAC
library and the Sanger Centre for assistance in producing high density
colony filters of the BAC libraries. This work was supported by the BBSRC,
the EC as part of the European Scientists Sequencing Arabidopsis (ESSA)
project and by a grant from Hoechst AG to HMG.</FONT>
<H2>
<FONT SIZE=+0>REFERENCES</FONT></H2>
<A NAME="dean"></A><FONT SIZE=+0>1. Dean C and Schmidt R: Plant genomes:
a current molecular description. <I>Annu. Rev. Plant Physiol. Plant Mol.
Biol.</I> <B>46</B> 395-418 (1995).</FONT>

<P><A NAME="schmidt"></A><FONT SIZE=+0>2. Schmidt R, West J, Love K, Lenehan
Z, Lister C, Thompson H, Bouchez D and Dean C: Physical map and organisation
of <I>Arabidopsis thaliana</I> chromosome 4. <I>Science</I> <B>270</B>
480-483 (1995).</FONT>

<P><A NAME="zachgo"></A><FONT SIZE=+0>3. Zachgo EA, Wang M-L, Dewdney J,
Bouchez D, Camilleri C, Belmonte S, Huang L, Dolan M and Goodman HM: A
physical map of chromosome 2 of <I>Arabidopsis thaliana</I>. <I>Genome
Research</I> <B>6</B> 19-25 (1996).</FONT>

<P><A NAME="chen"></A><FONT SIZE=+0>4. Chen EY, Schlessinger D and Kere
J: Ordered Shotgun Sequencing, a strategy for integrated mapping and sequencing
of YAC clones. <I>Genomics</I> <B>17</B> 651-656 (1993).</FONT>

<P><A NAME="vaudin"></A><FONT SIZE=+0>5. Vaudin M, Roopra A, Hillier L,
Brinkman R, Sulston J, Wilson RK and Waterston RH: The construction and
analysis of M13 libraries prepared from YAC DNA. <I>Nucl. Acids Res.</I>
<B>23</B> 670-674 (1995).</FONT>

<P><A NAME="hauge"></A><FONT SIZE=+0>6. Hauge BM and Goodman HM: In <I>Methods
in Arabidopsis research</I> (ed. C. Koncz, N.-H. Chua and J. Schell), pp.
191-223. World Scientific Publishing Co. Pty. Ltd., Singapore (1992).</FONT>

<P><A NAME="choi"></A><FONT SIZE=+0>7. Choi SD, Creelman R, Mullet J and
Wing RA: Construction and characterisation of a bacterial artificial chromosome
library for <I>Arabidopsis thaliana</I>. <I>Weeds World</I> <B>2</B> 17-20
(1995).</FONT>

<P><A NAME="bevan"></A><FONT SIZE=+0>8. M. Bevan, I. Bancroft, E. Bent,
K. Love, P. Piffanelli, H. Goodman, C. Dean, [54 members and group leaders
of sequencing laboratories], H. Mewes, C. Scheuller, N. Chalwatzis, Sequence-level
Organisation of a Contiguous 1.9 Mb Region from Chromosome 4 of the Plant
<I>Arabidopsis thaliana</I>. <I>Nature in press</I></FONT>

<P><A NAME="cross"></A><FONT SIZE=+0>9. Cross SH and Little PFR: A cosmid
vector for systematic chromosome walking. <I>Gene</I> <B>49</B> 9-22 (1986).</FONT>

<P><A NAME="stask"></A><FONT SIZE=+0>10. Staskawicz B, Dahlbeck D, Keen
N and Napoli C: Molecular characterization of cloned avirulence genes from
Race 0 and Race 1 of <I>Pseudomonas syringae</I> pv. <I>glycinea</I>. <I>J.
Bacteriol.</I> <B>169</B> 5789-5794 (1987).</FONT>

<P><A NAME="olszewski"></A><FONT SIZE=+0>11. Olszewski NE, Martin FB and
Ausubel FM: Specialized binary vector for plant transformation: expression
of the <I>Arabidopsis thaliana</I> AHAS gene in <I>Nicotiana tabacum</I>.
<I>Nucl. Acids Res.</I> <B>16</B> 10765-10783 (1988).</FONT>

<P><A NAME="ditta"></A><FONT SIZE=+0>12. Ditta G, Stanfield S, Corbin D
and Helinski DR: Broad host range DNA cloning system for Gram-negative
bacteria: construction of a gene bank of <I>Rhizobium meliloti</I>. <I>Proc.
Natl. Acad. Sci. USA</I> <B>77</B> 7347-7351 (1980).</FONT>

<P><A NAME="jones"></A><FONT SIZE=+0>13. Jones JDG, Shlumukov L, Carland
F, English J, Scofield SR, Bishop GJ and Harrison K: Effective vectors
for transformation, expression of heterologous genes, and assaying transposition
excision in transgenic plants. <I>Transgenic Research</I> <B>1</B> 285-297
(1992).</FONT>

<P><A NAME="bancroft"></A><FONT SIZE=+0>14. Bancroft I and Wolk CP: Pulsed
homogeneous orthogonal field gel electrophoresis (PHOGE). <I>Nucl. Acids
Res.</I> <B>16</B> 7405-7418 (1988).</FONT>

<P><A NAME="schwartz"></A><FONT SIZE=+0>15. Schwartz DC and Koval M: Conformational
dynamics of individual DNA molecules during gel electrophoresis. <I>Nature</I>
<B>338</B> 520-522 (1989).</FONT>

<P><A NAME="bancroft2"></A><FONT SIZE=+0>16. Bancroft I, Westphal L, Schmidt
R and Dean C: PFGE-resolved RFLP analysis and long range restriction mapping
of the DNA of <I>Arabidopsis thaliana</I> using whole YAC clones as probes.
<I>Nucl. Acids Res.</I> <B>20</B> 6201-6207 (1992).</FONT>

<P><A NAME="sambrook"></A><FONT SIZE=+0>17. Sambrook J, Fritsch EF and
Maniatis T: <I>Molecular cloning: A laboratory manual</I>, 2nd. Edition.
Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1989).</FONT>
<BR>&nbsp;
<BR>&nbsp;</BLOCKQUOTE>

</BODY>
</HTML>
