<HTML>
<HEAD>
   <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
   <META NAME="Generator" CONTENT="Microsoft Word 97">
   <META NAME="GENERATOR" CONTENT="Mozilla/4.02 [en] (Win95; I) [Netscape]">
   <TITLE>Session 10: Plant-Pathogen Interactions</TITLE>
<script language="javascript">

<!--

function Page_load(page) {
        

                
           if (page == 4)
                page_title = "chory.html";
           else if (page == 5)
                page_title = "madison.html";
           else if (page == 6)
                page_title = "dennis.html";

        if (page_title == "")
                return;
        
        
        window.parent.frame_main.location = window.parent.frame_menu.home_page + page_title;
           
}

        var strng;
        function message(id) {
           

           if (id == 4)
                strng = ' Session 9: Light-Regulated Development';
           else if (id == 5)
                strng = '8th International Arabidopsis Conference, Madison';
           else if (id == 6)
                strng = ' Session 11: Chromosome Structure and Function';

           else 
                return false;

        window.status = strng;
        return true;
}

// end of script -->

</script>

</HEAD>
<BODY bgcolor="#ffffff">

<BLOCKQUOTE>
<H1>
Session 10: Plant-Pathogen Interactions</H1>

<H4>
Chair: Roger Innes, Indiana University, USA</H4>

<H4>
<B>email: <a href="mailto:rinnes@bio.indiana.edu">rinnes@bio.indiana.edu</a></B>
<HR WIDTH="100%"></H4>
This session covered three major areas of inquiry, starting off with the
structure and function of disease resistance genes, followed by genetic
analysis of R-gene signal transduction pathways, and finishing with a talk
on systemic acquired resistance. Each speaker provided the following summaries
of their talks, for which I am most grateful.

<P>Fumiaki Katagiri (University of Maryland, Baltimore County) discussed
molecular mechanisms involved in gene-for gene interactions and novel experimental
systems for their study. The <I>Pseudomonas syringae</I> avirulence genes,
<I>avrRpt2</I> and <I>avrB</I>, and the corresponding <I>Arabidopsis</I>
resistance genes, <I>RPS2</I> and <I>RPM1</I>, respectively, were used
as a model system. Using biolistic transient expression of the avirulence
genes in plants, he showed that the avirulence gene products are the only
bacterial factors required for elicitation of the specific resistance response
in plants as long as the gene products are localized properly. This observation
was confirmed in transgenic plants (<I>RPS2</I> wild type) carrying the
<I>avrRpt2</I> gene linked to the copper-inducible promoter. Such plants
showed a copper-dependent HR-like response. Transient expression of the
avirulence genes in <I>Arabidopsis</I> protoplasts demonstrated that protoplasts
are still capable of specific recognition of the avirulence gene-based
signals. The biolistic transient expression assay was also used for analysis
of the corresponding resistance genes. The result with one <I>RPS2/RPM1</I>
chimeric gene suggested that the LRR of RPS2 determines the specificity
to avrRpt2.

<P>Roger Innes (Indiana University) described the identification of several
<I>Arabidopsis</I> genes that are required for resistance mediated by the
<I>RPS5</I> disease resistance gene. RPS5 confers resistance to strains
of <I>P. syringae </I>that carry the avirulence gene <I>avrPphB</I>. Randy
Warren in his group isolated six mutants that displayed enhanced susceptibility
to such strains. Genetic complementation analyses revealed that two of
these were <I>rps5 </I>mutants, while the remaining four defined at least
three additional genes, which have been designated <I>PBS1</I>, <I>PBS2</I>
and <I>PBS3</I> for avrPphB susceptible. Mutations in PBS1 reduce resistance
mediated by RPS5, but do not affect resistance mediated by other disease
resistance genes in <I>Arabidopsis</I>, indicating that the PBS1 protein
functions in a pathway specific to RPS5, potentially interacting with RPS5
and/or AvrPphB. In contrast to <I>PBS1</I>, mutations in <I>PBS2 </I>and
<I>PBS3</I> affect several <I>Arabidopsis</I> disease resistance gene pathways,
including resistance genes specific to the fungal pathogen <I>Peronospora
parasitica</I>. Interestingly, the <I>rps5-1</I> mutant is phenotypically
similar to the <I>PBS2</I> mutant, affecting multiple disease resistance
genes. The <I>RPS5</I> gene has been isolated and was found to be a member
of the NBS-LRR family of disease resistance genes. The <I>rps5-1 </I>mutant
contains a single amino acid substitution in the third LRR of RPS5 , suggesting
that this region may physically interact with a signal transduction component
shared by other disease resistance gene pathways, perhaps PBS2.

<P>Jane Parker (The Sainsbury Laboratory) described her group's work on
the RPP5 disease resistance pathway. RPP genes confer resistance to the
downy mildew pathogen <I>Peronospora parasitica</I>, with the <I>RPP5</I>
gene specifying resistance to <I>P.</I> <I>parasitica</I> isolate Noco2.
<I>RPP5</I> encodes a protein with a predicted nucleotide binding site
(NBS) and leucine rich repeats (LRRs) that exhibits striking structural
similarity to the tobacco N and flax L6 resistance proteins and is less
similar to the <I>Arabidopsis</I> <I>RPM1</I> and <I>RPS2</I> gene products.
Like N and L6, the RPP5 N-terminal domain resembles the cytoplasmic domains
of the <I>Drosophila </I>Toll and mammalian interleukin-1 transmembrane
receptors. In contrast to N and L6 that produce predicted truncated products
by alternative splicing, <I>RPP5 </I>appears to express only a single transcript
corresponding to the full-length protein. However, a truncated form structurally
similar to those of N and L6 is encoded by one or more other members of
the <I>RPP5</I> gene family that are tightly clustered on chromosome 4.
A candidate <I>RPP14</I> gene that is part of a tightly linked R gene cluster
on chromosome 3, also appears to be of the RPP5/ N gene type. Resistance
conferred by RPP5, RPP14, and a number of other <I>Arabidopsis</I> R genes
is dependent on a second gene, <I>EDS1</I>. The <I>EDS1 </I>gene was cloned
by transposon tagging and sequence analysis reveals a novel protein that
is highly conserved between different <I>Arabidopsis </I>accessions. Two
out of three fast neutron-derived <I>eds1</I> mutant alleles possess extensive
deletions (0.5 and 1 kb, respectively) of the <I>EDS1</I> open reading
frame and several ethyl methane sulphonate-generated mutant alleles are
currently being analysed. The EDS1 protein possesses three appropriately
spaced sequences that show significant similarity to the consensus motifs
comprising a lipase catalytic site. Several site-directed mutations of
the catalytic residues have been made to test whether these abolish EDS1
function in transformed <I>eds1</I> mutant plants. Mutations of <I>EDS1</I>
and a second resistance signalling gene, NDR1, were combined with different
<I>Arabidopsis</I> R genes to assess their relative effects. The results
so far suggest that the two proteins operate preferentially in distinct
resistance signalling pathways. For example, RPP5 requires EDS1 but appears
to operate independently of NDR1,
<BR>whereas RPS2 requires NDR1 but not EDS1. Further R gene-signalling
gene combinations are presently being examined to test the broader requirements
for EDS1 and NDR1. EDS1 is also being combined with several other defence
response mutations in order to position these in the same or distinct resistance
signalling pathways.

<P>Allan Shapiro (U.C. Berkeley, and now at the University of Delaware)
described work performed in the Staskawicz laboratory on the <I>NDR1</I>
gene of <I>Arabidopsis</I>. NDR1 is essential for numerous bacterial and
<I>Peronospora </I>gene-for-gene disease resistance specificities in <I>Arabidopsis</I>.
The <I>NDR1</I> gene was isolated using a positional cloning approach and
was found to encode a putative integral membrane protein. The <I>ndr1-1</I>
allele contains a deletion of the promoter and the N-terminal part of the
predicted protein, while the <I>ndr1-2 </I>and <I>ndr1-3</I> alleles contain
nonsense mutations. A Northern blot showed that <I>NDR1</I> message is
induced by avirulent but not virulent pathogens at an early time point.
Analysis of the <I>ndr1-1</I> mutant was used to dissect the disease resistance
response. This mutant is unable to mount a hypersensitive response (HR)
in response to challenge with bacteria containing avrRpt2. However, an
enhanced HR was seen in response to avrB or any of three other bacterial
avirulence genes. Several other responses normally correlated with disease
resistance were evaluated after inoculating <I>ndr1-1</I> plants with bacteria
containing avrRpt2 or avrB. Bacteria containing avrRpt2 failed to induce
systemic acquired resistance (SAR) and PR-1 driven transcription, and induced
less accumulation of salicylic acid (SA) at an early time point relative
to wild type plants. Substantial catch-up in SA levels were seen at a late
time point. By contrast, avrB-containing bacteria induced SAR and higher
levels of PR-1 driven transcription in <I>ndr1-1</I> plants as compared
to Columbia wild-type. SA levels were comparable to those found in plants
inoculated with avrRpt2-containing bacteria. Data was presented on poster
2-16 showing that <I>ndr1-1</I> plants were partially impaired in induction
of PR-1 driven transcription in response to UV-C light (a mimic of the
oxidative burst) or low levels of BTH, but not impaired in response to
high levels of BTH. A working model incorporating these data would posit
that the <I>ndr1-1</I> plants are partially blocked in disease resistance
signal transduction, downstream of the oxidative burst, but upstream of
SA production. Mutant screens have been initiated to uncover new genes
in this pathway. Data was presented on a screen using a transgenic line
carrying &szlig;-glucuronidase (GUS) under PR-1 control. Mutants which
show reduced GUS activity in response to challenge with avirulent pathogen
have been obtained and are being characterized

<P>Mark Kinkema (Duke University) described his work on characterizing
the NPR1 protein. NPR1 regulates a local and systemic defense response
against pathogen attack in <I>Arabidopsis</I>, and is required for many
of the responses induced by SA. A fusion of <I>NPR1</I> with the green
fluorescent protein (GFP) complements all the <I>npr1</I> mutant phenotypes
in transgenic plants, indicating that NPR1-GFP is functional and, therefore,
reflects the endogenous localization of NPR1. NPR1-GFP is localized in
the nucleus in response to chemical activators of systemic acquired resistance.
Virulent pathogen infection induces the nuclear localization of NPR1-GFP
in cells surrounding the infection site, and nuclear localized NPR1-GFP
correlates with the expression of the PR gene B-1,3-glucanase. These results
suggest that nuclear localization of NPR1 regulates the SA-induced systemic
acquired resistance and local acquired resistance defense responses.
<BR>&nbsp;
<BR><FONT SIZE=-1></FONT>&nbsp;</BLOCKQUOTE>

</BODY>
</HTML>
