<HTML>
<HEAD>
   <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
   <META NAME="Generator" CONTENT="Microsoft Word 97">
   <META NAME="GENERATOR" CONTENT="Mozilla/4.02 [en] (Win95; I) [Netscape]">
   <META NAME="Author" CONTENT="mary anderson">
   <TITLE>Thorlby et al</TITLE>
<script language="javascript">

<!--

function Page_load(page) {
        

                
           if (page == 4)
                page_title = "bancroft.html";
           else if (page == 5)
                page_title = "Contents.html";
           else if (page == 6)
                page_title = "table1.html";

        if (page_title == "")
                return;
        
        
        window.parent.frame_main.location = window.parent.frame_menu.home_page + page_title;
           
}

        var strng;
        function message(id) {
           

           if (id == 4)
                strng = 'bancroft et al.';
           else if (id == 5)
                strng = 'WeedsWorld Volume 4ii Contents ( Sept-1997)';
           else if (id == 6)
                strng = 'Table1 Thorlby et al';

           else 
                return false;

        window.status = strng;
        return true;
}

// end of script -->

</script>

</HEAD>
<BODY bgcolor="#ffffff">

<BLOCKQUOTE>
<H2>
Molecular and physical mapping of the <I>lu</I>-<I>ttg</I> region of <I>Arabidopsis</I>
chromosome 5.</H2>
Glenn J. Thorlby, Leonid Shlumukov, Igor Y. Vizir, Cai Yun Yang , Bernard
J. Mulligan and Zoe A. Wilson.

<P>School of Biology, University of Nottingham, University Park, Nottingham,
NG7 2RD, UK

<P>Corresponding author: Dr. Zoe A. Wilson
<BR>Tel: 0115 9513235
<BR>Fax: 0115 9513251
<BR>Email: Zoe.Wilson@nottingham.ac.uk

<P>
<HR WIDTH="100%">
<BR>&nbsp;
<H2>
Abstract</H2>
Recombinants between the makers <I>lutescens</I> (<I>lu</I>; 17.6 cM) and
<I>transparent testa glabra</I> (<I>ttg</I>; 35.5 cM) on the top arm of
<I>Arabidopsis thaliana</I> chromosome 5 have been used to construct a
molecular genetic map of the region, which encompasses a number of genes
involved in floral development including <I>Ms1</I>, a gene required for
the post-meiotic development of pollen (<A HREF="#dawson">Dawson <I>et
al.</I>, 1993</A>). This recessive mutation results in degeneration of
pollen soon after microspore release and maps at 29.8 &plusmn; 0.8 cM on
<I>Arabidopsis</I> chromosome 5. Clones derived from 4 YAC libraries have
been used to establish a contig between the RFLP markers 4111 and 4556;
this represents a genetic distance of 8.9 cM and a physical distance of
approximately 1.44 Mb. In this region, 1 cM corresponds to a physical distance
of approximately 160 kb.
<BR>&nbsp;
<H2>
Introduction</H2>
Problems have frequently been encountered when low resolution or integrated
genetic maps have been used for the positional cloning of genes; these
include the underestimation of physical distances between markers and the
identification of a high frequency of chimeric, unstable YAC clones or
clones which may map to more than one region of the genome. Therefore fine-scale
molecular genetic mapping of a defined chromosomal region containing the
gene of interest is an essential pre-requisite to positional cloning. As
part of an effort towards the positional cloning of the <I>male sterility
1</I> (<I>Ms1</I>) gene we have been establishing a physical map of the
<I>lu-ttg</I> region on the top arm of <I>Arabidopsis</I> chromosome 5
(<A HREF="#thorlby">Thorlby <I>et al</I>, 1997</A>).

<P>The <I>ms1</I> mutation is a recessive, EMS-derived mutation which results
in the degeneration of pollen immediately after microspore release from
the tetrad (<A HREF="#dawson">Dawson <I>et al.</I>, 1993</A>). Tapetal
development appears aberrant at this stage, however, the identity and function
of the <I>Ms1</I> gene product are unknown. A variety of map positions
have been established for <I>Ms1</I> on chromosome 5, ranging from 25.7
cM on the classical map (<A HREF="#koornneef">Koornneef, 1986</A>) to 29.8
cM on the integrated genetic/RFLP map (<A HREF="#hauge">Hauge <I>et al.</I>,
1993</A>). Using in excess of 400 recombinants between the genetic markers
<I>lutescens </I>(<I>lu</I>; 17.6 cM) and <I>transparent testa glabra</I>
(<I>ttg</I>; 35.5 cM) we have mapped <I>Ms1</I> to 29.8 &plusmn; 0.8 cM
(<A HREF="#thorlby">Thorlby <I>et al.</I>, 1997</A>).

<P>Using a subset of these recombinants we have been conducting molecular
mapping of this region; 183 recombinant plants between the ecotypes Landsberg
<I>erecta</I> , Sn(5)-1 and Wassileskija were screened with DNA markers,
which from published maps, appear to map to this region. Two groups of
recombinants were generated between <I>lu</I> -<I>ms1</I> and <I>ms1</I>-<I>ttg</I>;
the use of recombinant homozygous <I>ms1</I> plants for RFLP mapping was
avoided because of the practical difficulties of maintaining sterile lines.
Markers mapping close to <I>ms1</I> were used to screen YAC libraries and
a 8.9 cM contig of approximately 1.44 Mb was established between the markers
4111 (26.4 cM) and 4556 (35.3 cM) which encompasses the <I>Ms1</I> locus
(<A HREF="#thorlby">Thorlby <I>et al</I>., 1997</A>). These YAC clones
were sized and aligned with the molecular genetic markers to produce an
detailed integrated map of this region.
<H2>
Results</H2>

<H3>
Molecular genetic mapping of the <I>lu-ttg</I> region</H3>
Molecular mapping has been carried out using 149 Ler x Sn(5)-1 recombinants
with break points on either side of <I>ms1</I> and a set of 34 Ler x Ws
lines with breakpoints between <I>ms1</I> -<I>ttg </I>(<A HREF="#thorlby">Thorlby
et al., 1997</A>). Tables <A HREF="table1.html">1</A> and <A HREF="table2.html">2</A>
show the RFLP mapping data generated from the Ler x Sn(5)-1 and Ler x Ws
recombinants, respectively. Map positions were established relative to
the positions of <I>lu</I> (17.6 cM), <I>ms1</I> (29.8 cM) and <I>ttg</I>
(35.5 cM), using the approach of mapping within a defined chromosome segment
(<A HREF="#thorlby">Thorlby et al., 1997</A>). Polymorphisms in the region
close to <I>ms1</I> were extremely difficult to detect, particularly between
the ecotypes Sn(5)-1 and Ler and for some markers, e.g. an 8 kb cosmid
clone containing the <I>TSL</I> gene, no polymorphisms could be found between
Sn(5)-1 and Ler. The map position for <I>TSL</I> was therefore inferred
relative to flanking DNA markers by comparing available data from the RI
lines (<A HREF="#lister">Lister and Dean, 1993</A>), which position <I>TSL</I>
at 0.8 cM away from RFLP marker 4560. This was found to agree with our
physical mapping data of the region (<A HREF="table2.html">Table 2</A>).

<P>Probes pCIT718, 21503 and KG10 were mapped using both the Ler x Sn(5)-1
and the Ler x Ws recombinant lines and slight positional differences were
observed depending on the recombinant ecotypes used. However, a number
of individual lines from both recombinant populations gave anomalous results
with these probes and an additional SSLP marker, CDPK.9. The results suggest
variable marker orders implying that there may be a high level of recombination
or rearrangement occurring in this region<I> </I>of chromosome 5 (manuscript
in preparation). The average data showing the consensus map positions for
these markers is presented in <A HREF="table1.html">Table 1</A> and these
map positions agree with the data obtained from physical mapping of the
<I>ms1</I> to <I>ttg</I> region.
<BR>&nbsp;
<H3>
Physical mapping of <I>ms1</I> region</H3>
YAC clones were identified by screening the Grill, Ward and Ecker YAC libraries,
and by limited hybridisations to the CIC library, with molecular markers
from this region (<A HREF="table3.html">Table 3</A>). Three end-probes
(RI3F12, RI9B3, RI8B11) (<A HREF="#thorlby">Thorlby et al., 1997</A>) were
used to orientate the YAC islands and to screen the libraries to assist
in contig assembly. When used to probe the CIC library the clones RI9B3
and RI8B11 linked two previously isolated YAC islands, spanning the chromosomal
regions between the markers 4111-<I>TSL</I> and pCIT718-4556, respectively.
Comparisons between our genetic and physical maps of the region indicates
that 1 cM equates on average to 160 kb.
<BR>&nbsp;
<H2>
Discussion</H2>
We have established a map position of 29.8 &plusmn; 0.75 cM for <I>ms1</I>,
which agrees with the integrated map position of this gene (<A HREF="#hauge">Hauge
<I>et al</I>., 1993</A>), by measuring recombination frequency relative
to two flanking markers, <I>lu</I> and <I>ttg</I> (<A HREF="#thorlby">Thorlby
et al., 1997</A>).

<P>We have conducted fine-scale molecular genetic mapping of the region
between the markers <I>lu</I> and <I>ttg </I>relative to the genetic distance
between these markers, which by consensus is 17.9 cM (<A HREF="#hauge">Hauge
et al., 1993</A>; <A HREF="#meinke">Meinke and Koornneef, 1994</A>; <A HREF="#meinke1">Meinke,
1995</A>). Our molecular map shows differences between the statistically
derived integrated map (<A HREF="#hauge">Hauge et al., 1993</A>; <A HREF="#meinke">Meinke
and Koornneef, 1994</A>; <A HREF="#meinke1">Meinke, 1995</A>) and the recombinant
inbred map (<A HREF="#lister">Lister and Dean, 1993</A>), which has a lower
density of markers and which was constructed on the basis of significantly
fewer recombinants in this region. Polymorphisms were difficult to detect
in the <I>Ms1</I> region and in some cases larger probes were required
for RFLP screening; for example, a lambda clone was identified corresponding
to the RI9B3 YAC endprobe since polymorphisms could not be detected with
the endprobe itself.

<P>DNA markers mapped to this region were used to identify clones from
four YAC libraries and establish a 8.9 cM contig. Two contig "islands"
were initially established by library screening and these were subsequently
linked by screening the CIC library with the RI9B3 and RI8B11 YAC end-probes.
Polymorphisms could not be detected in the recombinant populations using
any of the three YAC end-probes. However, a larger genomic lambda clone
(lambda 9B3) encompassing the YAC RI9B3 endprobe identified a Ler x Sn(5)-1
polymorphism which co-segregated with the <I>ms1</I> mutation.

<P>The established contig contains 71 YACs and corresponds well with the
YAC clones from contigs 6 and 7 on chromosome 5 identified by <A HREF="#schmidt1">Schmidt
<I>et al</I>. (1997)</A>. Thirteen of the clones have been previously identified
as possibly chimeric (<A HREF="#schmidt">Schmidt <I>et al.</I>, 1994</A>;
<A HREF="#dunn1">Dunn, 1995</A>; Morroll and Wilson, in prep). These YACs
may be due to multiple cloning events or alternatively to genomic rearrangements
such as deletions, recombinations or rearrangements. It may also be that
they represent duplicated regions in the genome and may correspond to alternative
chromosome locations.

<P>Our data indicates that the physical distance between the RFLP markers
4111 and 4556 corresponds to approximately 1.44 Mb and a genetic distance
of 8.9 cM. Therefore in this region 1 cM equates to approximately 160 kb.
Our fine-scale map and the future availability of the recombinant mapping
population (deposited with the <A HREF="http://nasc.nott.ac.uk">Nottingham
Arabidopsis Stock Centre</A>)(<A HREF="#thorlby">Thorlby et al., 1997</A>),
together provide a detailed resource for the positional cloning and mapping
of genes which lie in the <I>lu-ms1-ttg</I> region of chromosome 5.
<BR>&nbsp;
<H2>
<FONT SIZE=+1>Materials and Methods</FONT></H2>

<H3>
Arabidopsis stocks; Plant growth; Generation of recombinants</H3>
<I>Arabidopsis thaliana</I> Ler ( NW20), Sn(5)-1 (N930) and Ws (N1601),
the chromosome 5 marker line (<I>lucoms1ttg</I> in a Ler background, N240)
were obtained from the Nottingham <I>Arabidopsis</I> Stock Centre (NASC),
Nottingham.

<P>Three groups of recombinants (<A HREF="#thorlby">Thorlby et al., 1997</A>)
were used in this study i) Ler/Ws recombinants (provided by David Marks,
University of Minnesota) which were homozygous Ler at the <I>ttg</I> locus
and heterozygous Ws/Ler (or homozygous Ws) at the <I>ms1</I> locus, ii)
<I>lu</I>/<I>ms1</I> recombinants which were homozygous Ler at the <I>lu</I>
locus and heterozygous Ler/Sn(5)-1 (or homozygous Sn(5)-1) at the <I>ms1</I>
locus, and iii) <I>ms1</I>/<I>ttg</I> recombinants which were homozygous
Ler at the <I>ttg</I> locus and heterozygous Ler/Sn(5)-1 (or homozygous
Sn(5)-1) at the <I>ms1</I> locus.
<BR>&nbsp;
<H3>
<FONT SIZE=+1><I>In vitro</I> growth of plants for DNA isolation</FONT></H3>
<FONT SIZE=+0>Plants were grown <I>in vitro</I> for DNA isolation based
on Czako and Marton (1992). DNA was isolated from 1-3g of plant material
by a method based on that of <A HREF="#dellaporta">Dellaporta<I> et al.</I>
(1983)</A>. DNA (1-2 &micro;g) from the parental lines and recombinants
was digested using a range of restriction enzymes and hybridised using
random-primed radiolabelled probes. CAPS primers for NIT4 were as described
by Bartel and Fink (1994). PCR for CAPS analysis was conducted using 25
cycles of 94<SUP>o</SUP>C for 1 min; 55<SUP>o</SUP>C for 1 min.; 72<SUP>o</SUP>C
for 2 min (Techne PHC3 temperature cycler). SSLP primers for CDPK.9 were
as described by <A HREF="#dunn2">Dunn (1996)</A>. Amplification was conducted
using 40 cycles of 94<SUP>o</SUP>C for 15 sec; 55<SUP>o</SUP>C for 15 sec;
72<SUP>o</SUP>C for 30 sec (Techne PHC3 temperature cycler). Protocols
are described in full in <A HREF="#thorlby">Thorlby <I>et al</I> (1997)</A>.</FONT>
<BR>&nbsp;
<H3>
<FONT SIZE=+1>YAC maintenance and DNA preparation</FONT></H3>
YAC libraries (<A HREF="#ecker">Ecker, 1990</A>; <A HREF="#ward">Ward and
Jen, 1990</A>; <A HREF="#grill">Grill and Somerville, 1991</A>) were provided
by the ABRC, Ohio State University. Selected clones from the CIC library
(<A HREF="#creusot">Creusot <I>et al.</I>, 1995</A>) were kindly provided
by Dr. Renate Schmidt (JI Centre, Norwich) and from the ABRC. YAC clones
were grown on selective medium (<A HREF="#ausubel">Ausubel <I>et al.</I>,
1988</A>) at 30<SUP>o</SUP>C and DNA isolated by the Puragene extraction
kit (Flowgen). High molecular weight YAC DNA was prepared using a CHEF
yeast genomic DNA plug kit (Bio-Rad). YAC hybridisations and PFGE was conducted
as described in <A HREF="#thorlby">Thorlby <I>et al</I>. (1997</A>).
<BR>&nbsp;
<H3>
<FONT SIZE=+1>Lambda library screening</FONT></H3>
<FONT SIZE=+0>The Ler genomic l library constructed by <A HREF="#voytas">Voytas
<I>et al</I>. (1990)</A> (ABRC CD4-8) was screened using radioactively
labelled RI9B3 YAC endprobe DNA. Putative positive signals were re-screened
and positive clones selected and confirmed following DNA isolation and
Southern blotting.</FONT>
<BR>&nbsp;
<H2>
Acknowledgements.</H2>
We are grateful to Chris Somerville (Carnegie Institute, Stanford, CA),
Eric Ward (Ciba-Geigy, NC), Joe Ecker (University of Pennsylvania, Philadelphia,
PA) and Francine Creusot (URA CNRS, Orsay, Cedex, France) for use of their
YAC libraries, to Brian Hauge and Howard Goodman (MGH, Boston) for cosmid
clones, to Elliot Meyerowitz (CalTech, Pasadena) for RFLP markers and the
<I>PI</I> clone and to Judy Roe (University of California, Berkeley) for
the <I>TSL</I> clone. Thanks are also due to David Marks (University of
Minnesota, Minneapolis) for the Ler x Ws recombinants. We would like to
acknowledge the ABRC, Ohio State University for clones and libraries. We
are indebted to Drs. Renate Schmidt (Max Delbr&uuml;ck Laboratory, Cologne)
and Caroline Dean (JI, Norwich) for continual collaboration and assistance
on contig establishment and for initial screening using the RI9B3 and RI8B11
end-probes. This work was supported by the UK Biotechnology and Biological
Sciences Research Council (BBSRC) as part of the Plant Molecular Biology
Programmes I and II under grants PG42/536 and PG42/556 to BJM and ZAW.

<P><FONT SIZE=+1>&nbsp;</FONT>
<BR>&nbsp;
<H2>
References</H2>
<A NAME="ausubel"></A>Ausubel, F.M., Brent, R., Kingston, R.E., Moore,
D.D., Seidman, J.G., Smith, J.A. and Struhl, K. (1988) <I>Saccharomyces
cerevisiae</I>. In: <I>Current protocols in molecular biology., </I>2 eds),
John Wiley and Sons, pp. 13.1.2.

<P><A NAME="bartel"></A>Bartel, B. and Fink, G.R. (1994) Differential regulation
of an auxin-producing gene family in <I>Arabidopsis thaliana.</I> <I>Proc.
Natl. Acad. Sci. </I>91, 6649-6653.
<BR>&nbsp;
<BR><A NAME="creusot"></A>Creusot, F., Fouilloux, E., Dron, M., Lafleuriel,
J., Picard, G., Billault, A., Lepaslier, D., Cohen, D., Chaboute, M.E.,
Durr, A., Fleck, J., Gigot, C., Camilleri, C., Bellini, C., Caboche, M.
and Bouchez, D. (1995) The CIC library - a large insert YAC library for
genome mapping in <I>Arabidopsis thaliana</I>. <I>Plant J.</I> 8, 763-770.

<P><A NAME="czako"></A>Czako, M. and Marton, L. (1992) Mass clonal propagation
of <I>Arabidopsis </I>roots. In: <I>Arabidopsis : the compleat guide.,
</I>eds), AFRC PMB.

<P><A NAME="dawson"></A>Dawson, J., Wilson, Z.A., Aarts, M.G.M., Braithwaite,
A.F., Briarty, L.G. and Mulligan, B.J. (1993) Microspore and pollen development
in six male-sterile mutants of <I>Arabidopsis thaliana</I>. <I>Can. J.
Bot.</I> 71, 629-638.

<P><A NAME="dellaporta"></A>Dellaporta, S.L., Wood, J. and Hicks, J.B.
(1983) Isolation of DNA from higher plants. <I>PMB Reporter</I> 4, 19-21.

<P><A NAME="dunn1"></A>Dunn, P. (1995) Chimearic YACs in <I>Arabidopsis</I>
YAC libraries. http://cbil.humgen.upenn.edu/-atgc/physical-mapping/chimericYACs.htm1

<P><A NAME="dunn2"></A>Dunn, P.(1996) SSLP primers., http://cbil.humgen.upenn.edu/~atgc/SSLP_
<BR>info/coming-soon.html

<P><A NAME="ecker"></A>Ecker, J.R. (1990) PFGE and YAC analysis of the
<I>Arabidopsis</I> genome. <I>Methods: A companion to Methods in Enzymology</I>
1, 186-194.

<P><A NAME="grill"></A>Grill, E. and Somerville, C. (1991) Construction
and characterization of a yeast artificial chromosome library of <I>Arabidopsis
</I>which is suitable for chromosome walking. <I>Mol. Gen. Genet.</I> 226,
484-490.

<P><A NAME="hauge"></A>Hauge, B.M., Hanley, S.M., Cartinhour, S., Cherry,
J.M. and Goodman, H.M. (1993) An integrated genetic/RFLP map of the <I>Arabidopsis
thaliana </I>genome. <I>Plant J. </I>3, 745-754.

<P><A NAME="koornneef"></A>Koornneef, M. (1986) Linkage map of <I>Arabidopsis
thaliana</I> . In: <I>Genetic maps 1987: A compilation of linkage and restriction
maps of genetically studied organisms., </I>(O'Brian, SJ, eds). New York,
Cold Spring Harbour, pp. 742-745.

<P><A NAME="lister"></A>Lister, C. and Dean, C. (1993) Recombinant inbred
lines for mapping RFLP and phenotypic markers in <I>Arabidopsis thaliana</I>.
<I>Plant J.</I> 4, 745-750.

<P><A NAME="meinke"></A>Meinke, D. and Koornneef, M. (1994) Genetic map
of visible markers in <I>Arabidopsis</I>. In: <I>The Nottingham Arabidopsis
Stock Centre Seed List, June 1994, </I>(Anderson, M, eds), NASC, pp. 142-143.

<P><A NAME="meinke1"></A>Meinke, D.W. (1995) Embryo lethal map of <I>Arabidopsis
thaliana.</I> http://probe.nalusda.gov:8300/cgi-bin/dbrun/aatdb?find+map

<P><A NAME="schmidt"></A>Schmidt, R., Putterill, J., West, J., Cnops, G.,
Robson, F., Coupland, G. and Dean, C. (1994) Analysis of clones carrying
repeated DNA sequences in 2 YAC libraries of <I>Arabidopsis thaliana</I>.
<I>Plant J.</I> 5, 735-744.

<P><A NAME="schmidt1"></A>Schmidt, R., Love, K., West, J., Lenehan, Z.
and Dean, C. (1997) Description of 31 YAC contigs spanning the majority
of <I>Arabidopsis</I> <I>thaliana</I> chromosome 5. <I>Plant J.</I> 11,
563-572.

<P><A NAME="thorlby"></A>Thorlby, G.J., Shlumukov, L., Vizir, I., Yang,
C.-Y., Mulligan, B.J. and Wilson, Z.A. (1997) Fine-scale molecular genetic
(RFLP) and physical mapping of a 8.9cM region on the top arm of <I>Arabidopsis</I>
chromosome 5 encompassing the male sterility gene, <I>ms1</I>. <I>Plant
J.</I> 12, 471-479.

<P><A NAME="voytas"></A>Voytas, D.F., Konieczny, A., Cummings, M.P. and
Ausubel, F.M. (1990) The structure, distribution and evolution of the Ta1
retrotransposable element family of <I>Arabidopsis thaliana.</I> <I>Genetics</I>
126, 713-721.
<BR>&nbsp;
<BR><A NAME="ward"></A>Ward, E.R. and Jen, G.C. (1990) Isolation of single-copy-sequence
clones from a yeast artificial chromosome library of randomly-sheared <I>Arabidopsis
thaliana</I> DNA. <I>Plant Mol. Biol.</I> 14, 561-568.

<P><FONT SIZE=+1>&nbsp;</FONT></BLOCKQUOTE>

</BODY>
</HTML>
