<html><title>Report on Arabidopsis Database Requirements</title>
<h1>Report of the Ad Hoc Committee on <i>Arabidopsis</i> Database Requirements</h1>
<br>
<h2>Executive Summary</h2>
<p>
An ad hoc committee representing <i>Arabidopsis</i> researchers and
government agencies met in Dallas on 5-6 June 1993 to discuss the
informatics needs of the <i>Arabidopsis</i> research community. The committee
discussed what role the <i>Arabidopsis</i> database (ADB) could serve in the
overall Multinational <i>Arabidopsis thaliana</i> Genome Research Project.
Biological and technical features that should be considered in
proposals for database development were outlined. The committee
recognized that one or more linked <i>Arabidopsis</i> databases will play a
key role in the future of the <i>Arabidopsis</i> genome project. ADB could
also provide an important model for database development in other
areas of biology.
<p>
The committee recommended that:
<p>
<ul>
<li><i>Arabidopsis</i> database(s) be established and maintained for use by the
<i>Arabidopsis</i> research community. Funding of such database services was
considered an appropriate use of federal funds.
<p>
<li>The database(s) have both service and research components, with the
service component modeled on a scientific journal, including a chief
editor, associate editors, and reviewers.
<p>
<li>The database(s) be linked to the needs of the community through an
oversight committee.
<p>
<li>The database(s) be generally accessible both nationally and
internationally through electronic networks, with all data available
on request.
<p>
<li>The design of the database should allow portability to new
generations of database software and to new generations of computers.
</ul>
<p>
Specific recommendations were also made on the content of the
database, the linkages between different types of data, and the design
features that would allow the database to be of maximal use. These are
described in detail in the sections that follow.
<p>
<b>Preface</b>
<p>
Computer databases have become an essential resource in disciplines of
biology such as genetics, ecology and taxonomy, where large amounts of
archival-quality data have accumulated. The rapid growth of
information on the structure and function of the <i>Arabidopsis</i> genome
has resulted in a widely perceived need for the development of one or
more databases which will permit facile access to this information.
This sense of need was prompted by the accumulation of large amounts
of data concerning physical and genetic mapping of the <i>Arabidopsis</i>
genome and large- scale sequencing efforts associated with the
Multinational <i>Arabidopsis</i> Genome Project as described in the most
recent Annual report (NSF Publication 92-112). Several groups within
the <i>Arabidopsis</i> community have responded by adapting preexisting
databases, such as ACEDB, to meet some of the most urgent
requirements. Other groups have initiated the development of new
databases. These initiatives have greatly increased the awareness of
database utility within the <i>Arabidopsis</i> community and have raised many
questions concerning the design and maintenance of public databases
for <i>Arabidopsis</i> and other plants.
<p>
In order to assess the long-term database needs of the <i>Arabidopsis</i>
community, an NSF-sponsored workshop on this topic was convened in
Dallas on 5 June 1993. The workshop participants included the elected
members of the North American <i>Arabidopsis</i> Steering Committee.
Representatives from the National Science Foundation, the U.S.
Department of Agriculture, and the European Community were present as
observers. Two scientists involved with the human Genome Data Base
(GDB) were also present as technical advisors. A list of participants
is given at the end of this document. A number of written suggestions
received from the <i>Arabidopsis</i> community in response to announcements
on the <i>Arabidopsis</i> electronic newsgroup were discussed during the
course of the workshop.
<p>
The general goals of the meeting were to examine the present and
future needs for a database and to outline in general terms the main
issues which should be addressed in any future proposals concerning
the development of new or expanded <i>Arabidopsis</i> databases. The
discussions were intentionally focused on biological and community
issues and there was no attempt to define or specify issues which are
primarily related to specific computer hardware or specific database
programs. In this and related respects, the following report is
intended to be only a guide to assist in the development and review of
grant proposals directed towards the development of expanded database
resources for the <i>Arabidopsis</i> community. It is hoped that the report
may also be of utility to colleagues contemplating the development of
databases for other organisms.
<p>
A central issue of the workshop concerned the question of what purpose
an <i>Arabidopsis</i> database should serve. This was approached by listing
all of the different types of data that can be obtained for
<i>Arabidopsis</i> or other plants, and then considering whether it would be
essential, advantageous, or of little utility to have such data
available in a highly interrelated, easily searchable electronic form.
The outcome of this far-reaching discussion was the recognition that,
if properly conceived and constructed, an encyclopedic database
interrelating everything from nucleotide sequences to ecological data
could provide a completely novel research tool that would potentiate
new kinds of discoveries in biology.
<p>
Because of the relative ease of data access, such a database might
eventually displace scientific journals as a source of certain kinds
of primary scientific information. This scientifically exciting
possibility lies at the heart of the following report. However, in
view of the large costs likely to be incurred in developing and
maintaining an encyclopedic database, and in view of the technical
challenges such a database presents at the present time, the necessity
of assigning priorities to certain aspects of database development was
recognized.
<p>
There were six main conclusions from the workshop:
<p>
First, although all <i>Arabidopsis</i> information need not be archived in a
single database, it is essential that all <i>Arabidopsis</i> databases be
able to communicate with each other and with other major databases,
such as the nucleic acid databases, in a seamless networked fashion.
In this respect, the word "database" may refer to a collection of
separate and specialized databases which are developed and maintained
by different groups, but which operate as one large federated
information resource with a common controlled vocabulary, user
interface and editorial practices. In this and related respects it was
considered essential that <i>Arabidopsis</i> databases be available via
international electronic networks.
<p>
Second, the maintenance of databases involves a major service
component. Because the community will increasingly rely upon such a
database to provide access to much crucial information, it is
imperative that <i>Arabidopsis</i> databases be implemented and managed in
such a way that responsiveness to present and future community needs
will be ensured.
<p>
Third, the work will involve an essential research component. Shorter
term research will be required to determine the best way to implement
many of the current needs for such a database, and longer term
research will be required to ensure that the database can migrate over
time to different and improved hardware and software platforms. The
database must have the ability to grow with the projected increase in
information which would result from the complete sequencing of the
<i>Arabidopsis</i> genome and the analysis of corresponding gene functions.
<p>
Fourth, the databases should provide, in some real sense, an
intellectual focus for the interpretation of biological data. In
particular, the committee noted that mechanisms must be devised to
provide for the synthesis, integration, and editorial control of
information. The concept of structuring a database along the lines of
a scientific journal is attractive in this respect (i.e., a database
should have editors, reviewers, and production staff). It is also
essential that the database be available for the participation of the
international scientific community.
<p>
Fifth, all of the data and knowledge residing in such a database will
form an irreplaceable intellectual resource for the scientific
community. Therefore, federal support for such a database should be
long-term and contingent upon guarantees that all information in the
database will be freely available to the international scientific
community, now and in the future.
<p>
Sixth, the government should support the development of one or more
databases, which would collectively contain all <i>Arabidopsis</i>
information, using a funding vehicle and management plan that would be
designed to provide enough freedom to ensure the success of the
research component, while also providing enough oversight and control
to ensure the adequacy of the service component.
<p>
In addition to examining these general themes, the group considered
the following specific questions pertinent to both the research and
service components of the project. The commentary following each
question constitutes the full recommendations of the committee, which
are also outlined in the Executive Summary.
<p>
<b>I. What should be in the database?</b>
<p>
The <i>Arabidopsis</i> database should be designed to meet the immediate and
long-term needs of individuals working on <i>Arabidopsis</i> as well as the
broader community of biologists with interests in related disciplines.
This fundamental objective should be considered when determining what
specific pieces of information should be included in the database. For
this initial overview, the information content of the database has
been divided into five sections as outlined below. This does not imply
that the design of the database needs to follow these divisions, but
rather that information within each group has been judged by the
<i>Arabidopsis</i> community to be of central importance. The five categories
are: (A) map-based information; (B) stock-based information; (C)
community-based information; (D) sequence-based information; and (E)
biology-based information. Details on what data should be included in
these categories are presented below.
<p>
<b>A. Map-Based Information</b>
<p>
Rapid visualization of updated genetic/molecular maps and ready access
to primary mapping data should be given highest priority in database
development. Map information should be collected and presented in a
manner that allows the user to determine what is known, plus what
remains questionable or unresolved, with respect to map locations of
genetic and molecular markers in combination with a complete physical
map of overlapping cloned sequences. In constructing the database, it
should be remembered that recombination data generally provide only
rough estimates of map location, and that mapping data may differ
widely in quality and reliability. Some database users may therefore
prefer direct access to primary mapping data in order to construct
their own maps of specific regions or compare their results with those
obtained in other laboratories. A database that provides options for
visualizing several different maps constructed with different mapping
functions or subsets of markers and primary mapping data would be
particularly valuable to the <i>Arabidopsis</i> community. Any proposal for
database development should also discuss in some detail how the
integrity of these maps would be verified and maintained through a
network of editors and/or curators with appropriate expertise in
map-based activities. Although it may eventually be possible to
combine all of the physical, genetic, and molecular data generated by
the <i>Arabidopsis</i> genome project into a single definitive map of the
five chromosomes, at the moment it is likely that three complementary
types of maps will need to be included in the database: (1) a genetic
map showing the approximate distances between all well-characterized
mutations (visible or biochemical) and molecular markers (RFLP, RAPD,
cDNAs, or other cloned sequences); (2) a map that presents data on
recombinant inbred (RI) lines in a visual manner; and (3) a physical
map showing the actual arrangement of cloned sequences along the
length of each chromosome. Many researchers may want to start with a
particular mutation or cloned gene and move directly to the
corresponding regions of all three maps. Some mutations and cloned
genes are likely to be known by several different names. It will
therefore be important to establish a database that will accommodate
multiple changes in nomenclature. Provisions should also be made to
add new types of information to genetic and physical maps as they
become available (break points of chromosomal aberrations; regions of
extensive heterochromatin; regions with a high/low degree of sequence
homology to related plants; etc.).
<p>
Other concerns that will need to be addressed in development of the
map portion of the database are listed below:
<p>
<ul>
<li>Mapping data are often generated in bits and pieces; a gene may be
assigned to a particular linkage group long before it becomes
localized to a more specific region. The database should be designed
to include preliminary mapping data with appropriate documentation as
soon as this information becomes available.
<p>
<li>Genetic maps contain some markers for which relative orders (abc vs
acb) have been clearly established through a variety of methods. The
database should allow researchers to determine which orders are fixed
and examine relevant data to know what criteria were used to establish
orders shown in the map. It should also be recognized that different
experiments occasionally suggest contradictory orders. Documentation
of any contradictions should be included in the database. If fixed
orders and map uncertainties cannot be reflected visually, it should
be possible for database users to move readily to supporting
documentation.
<p>
<li>Raw data should be stored in a uniform format so that individual
researchers can construct their own maps of specific regions with
several different mapping programs. It should be possible to extract
various subsets of mapping data(e.g. locations of gene mutations, cDNA
clones, RFLP or RAPD markers, T-DNA insertions, any marker between two
given endpoints on the map, or any marker showing an EcoRI
polymorphism between two particular ecotypes) from the database.
<p>
<li>If possible, maps should be constructed such that individuals can
readily determine what genetic/physical/molecular markers are located
within a given distance from a desired point on a particular version
of the map. Then once the map location of a new sequence or marker has
been determined, the database can be asked to give all relevant
information on the chromosomal region in question.
<p>
<li>A "zoom" feature that allows database users to increase or decrease
the magnification of a particular region of the map with minimal
effort should be included. It would be advantageous to design the
database such that once the nucleotide sequence of large genomic
regions has been established, one could move from the genetic map down
to individual base pairs and back again. The capacity to display on
genetic or physical maps the locations of particular nucleotide
sequences (e.g., endonuclease recognition sites or PCR primer sites)
would be advantageous. - With respect to the physical map, the
database should include all relevant data on contigs of overlapping
cloned sequences: ecotype of origin, sequence, map location, region of
overlap with adjacent clones, reference laboratory, possible sources
of uncertainty, results of other investigators working in given
regions, and availability of specific clones.
</ul>
<p>
<b>B. Stock-Based Information</b>
<p>
The database should contain detailed information on the availability
of biological and chemical materials related to <i>Arabidopsis</i> research.
One should be able to use the database to determine how a wide range
of desired materials can be obtained, propagated, and utilized by the
individual investigator. This may be accomplished either by including
this information directly in the primary database, or by appropriate
linkage with databases at existing stock centers. Development of this
part of the database will require extensive interaction with
individuals at the Nottingham <i>Arabidopsis</i> Stock Center (UK) and the
<i>Arabidopsis</i> Biological Resource Center at Ohio State University (USA).
Major sequencing and molecular mapping centers should also be
consulted on the most efficient means of transferring or linking large
volumes of molecular data to primary databases. Attention must be
given to reducing duplication of effort in order to conserve limited
financial resources.
<p>
The database should include detailed information on the nature,
origin, and availability of seeds, clones, libraries, cDNAs,
oligonucleotides, and any other materials that may require
distribution to the <i>Arabidopsis</i> community. Emphasis should be placed
on careful documentation of biological materials,controlled
vocabularies, and maximal utilization of sophisticated graphics to
display plant phenotypes, molecular hybridization patterns, and other
data where appropriate. With respect to seed stocks, it should be
possible to search the database by general phenotype, not just by gene
symbol, in order to obtain a broad listing of ecotypes and mutant
lines with similar features. To facilitate this, it is important that
phenotypic descriptions involve documented, controlled vocabularies so
that users are not obliged to perform multiple queries using different
terms to describe the same phenotype, then integrate the results later
by hand.
<p>
Information on phenotypes, screening methods, growth conditions, and
differences between alleles should be included for all mutants
available through the stock centers. It should also be possible to
obtain information on additional mutants or alleles that have been
isolated in specific laboratories but are not available from the stock
centers. Nomenclature of genes and alleles must be controlled by the
community together with the stock centers and database managers in
order to avoid duplicate use of the same symbol for different loci and
to ensure that appropriate changes are made when examples of allelism
between mutants with different symbols are found. The database should
be designed to give appropriate information on the availability and
maintenance of any mutation, molecular marker, or cloned sequence
located on the genetic and physical maps. Individuals should be able
to search for specialized libraries, vectors, transgenic lines, and
molecular reagents (antibodies, purified proteins, unusual compounds,
and biochemical standards) required for <i>Arabidopsis</i> research.
<p>
<b>C. Community-Based Information</b>
<p>
The database should provide easy access to information on a wide range
of community issues such as: individual investigators (names,
addresses, research interests), messages sent through the <i>Arabidopsis</i>
electronic network, general announcements (jobs, meetings, seminars,
grants, technical comments, specialized reagents or facilities
available), personal communications (protocols, methods, lab data
relevant to researchers in the field but not included in
publications), and literature (publications on <i>Arabidopsis</i>, meeting
abstracts, grant summaries, complete text of <i>Arabidopsis</i> Information
Service and other reports with limited distribution). Extensive input
from the community will again be needed to keep this portion of the
database current. Investigators should be encouraged to use the
personal communication section to submit important data not included
in any publications, which can then be dated, flagged as unedited,
linked to the name of the contributor, given an accession number, and
searched from all fields. This information could be referenced in
subsequent publications and updated or confirmed when appropriate.
<p>
<b>D. Sequence-Based Information</b>
<p>
There is no requirement for all <i>Arabidopsis</i> sequences to be stored in
the proposed database. In light of the rapid growth of sequence
information and continued budget restrictions, it may be preferable to
focus limited resources on maximizing linkages with existing
databases. If a separate sequence file is not maintained, the primary
emphasis in development of the <i>Arabidopsis</i> database should be on
convenience of linkage to GenBank and other large-scale databases with
a goal of transparent retrieval of sequences. Individuals using the
proposed database should be able to obtain sequence information with
minimal effort, preferably without closing a session in progress. The
<i>Arabidopsis</i> community will require access to extensive sequence data
in relation to other parts of the proposed database. It should
therefore be possible to retrieve from the database sequence
information based on map position, type of sequence, or other specific
requirements. The database should provide access to sequence
information on <i>Arabidopsis</i> nuclear and organelle genomes, specific
cDNAs, genes, oligo- nucleotides, and proteins. Since data will be
obtained from a large number of different laboratories, reference must
be made to which ecotypes were used to generate the sequences
represented in the database. A long-term objective of the <i>Arabidopsis</i>
genome project will be to facilitate the identification and molecular
isolation of genes from a wide range of flowering plants based on
initial studies with <i>Arabidopsis</i>. The database should therefore be
designed to allow individuals to start with a known sequence from
<i>Arabidopsis</i> and search for related sequences from other plants.
<p>
<b>E. Biology-Based Information</b>
<p>
The focus of research with <i>Arabidopsis</i> is likely to change in the
future from the immediate emphasis on mapping, sequencing, and gene
identification, to the long-term questions of general biology and gene
function during plant growth and development. The <i>Arabidopsis</i> database
should therefore include information that may be of critical
importance during this second phase. Examples of topics that might be
included in this category include: information on plant pathogens that
infect <i>Arabidopsis</i> and details on the molecular interactions that take
place between host and pathogen; information on the chemical
composition of specific plant parts (sugars, lipids, proteins,
polysaccharides, specialized compounds, etc.); physiological data on
the normal life cycle and the response of mutant and wild-type plants
to various environmental and experimental treatments; protein profiles
of different plant parts revealed through 2-D gel electrophoresis;
information on the natural distribution and ecology of <i>Arabidopsis</i> and
closely related species; detailed comparisons of the different
ecotypes with respect to morphology, physiology, and molecular
biology; information on the taxonomy of <i>Arabidopsis</i> with particular
attention to related plants used in agriculture; light and electron
micrographs of different types of cells in wild-type plants; records
of expression patterns of specific genes during growth and
development; and computer-enhanced reconstructions of serial sections
through various plant structures. Although some of these features will
not be required for initial database design, their inclusion in
long-term plans would greatly enhance the continued use of <i>Arabidopsis</i>
as a model system for research in plant biology.
<p>
<b>II. How will the database be used? What links should be made between
categories of information?</b>
<p>
In addition to the specific ability to perform searches as described
in the previous sections, the categories of information must be linked
with user-friendly interfaces. Attention should be paid to tight
coordination between the genetic map and related genes, clones, and
sequences, so that selection of any of these will lead transparently
to accession of the others. Also, it is highly desirable for the
database to have simple links for comparative sequence and mutant
analysis with other plants and beyond that, with all organisms. The
interface should allow viewing in a variety of ways.
<p>
As examples of the types of links desired, we add a series of
questions that the system should be able to answer are listed below.
Most of these examples were suggested by current users of AAtDB and
AIMS, the information management database associated with the
<i>Arabidopsis</i> Biological Resource Center at Ohio State University.
<p>
<ul>
<li>If a user enters two cloned markers, the system should return a list
of all markers of a specified type that map between them.
<p>
<li>If a user points to a location on a genetic map, the genes, clones,
and sequences should appear. Likewise, a user should be able to derive
map position if a DNA sequence is used as the starting point.
<p>
<li>If a user finds a specific clone, the expression pattern of the RNA
encoded by the clone should be readily accessed.
<p>
<li>If a user finds a mutant that is altered in a particular way, the
system should retrieve all mutants altered in a similar manner. A
cross-species accession to similar mutants in other plants might be
useful.
<p>
<li>If a user desires to see the map positions for all genes in a given
biochemical or developmental pathway, she should be able to do so.
<p>
<li>If a user has new mapping information, the system should have the
ability to download archived data in that region for manipulation.
</ul>
<p>
<b>III. What community issues must be considered in the design and
operation of the database?</b>
<p>
<b>A. Advisory Committee</b>
<p>
An <i>Arabidopsis</i> database (ADB) proposal should include a provision for
an advisory committee that will represent the community of <i>Arabidopsis</i>
researchers and will advise ADB investigators on priorities and data
to be included.
<p>
One model of how an advisory committee would function is based on an
analogy between ADB and a scientific journal. In this model, the ADB
investigators who are funded by an ADB grant and are responsible for
database assembly, would be analogous to the publisher of the journal.
The ADB curator(s) (a permanent professional position) would be
analogous to the managing editor of the journal. The ADB advisory
committee would be analogous to the editor-in-chief plus the senior
editors of the journal. Finally the ADB advisory committee would
appoint an editorial board which would evaluate specific submissions
to ADB in the same way that the editorial board of a journal reviews
submitted manuscripts for scientific content and appropriateness.
<p>
The Chairperson of the ADB advisory committee could be appointed by
the PI of the ADB grant in consultation with the granting agency and
the North American <i>Arabidopsis</i> Steering Committee (NAASC). In
addition, the advisory committee could consist of four additional
members who would be appointed to four year terms by NAASC, which
should take care to include an international representative.
Initially, two of these committee members would have two-year terms
instead of four year-terms to enable two members to be replaced every
two years.
<p>
The ADB Advisory Committee should provide an annual written report
that would assess the progress of the database and make
recommendations for the coming year. This report would be a part of
the annual progress report of the ADB grant to the funding agency.
<p>
One role of the advisory committee will be to work with the ADB
curator to establish a standardized <i>Arabidopsis</i> nomenclature for
cloned genes that is consistent with other genome databases.
<p>
<b>B. Curation, Entry, Correction, and Long-Term Storage of Data</b>
<p>
Curation: Because ADB will be relatively small compared to the human
genome data base and will most likely have limited funds, developers
of ADB should make every effort to leverage the database activities
undertaken elsewhere and adapt existing software, when appropriate,
for use in the <i>Arabidopsis</i> research community. Thus, the major
activity of ADB will be the collection, entry, and correction of data
rather than writing software for storing, retrieval, and presentation
of data. Therefore, a full-time professional curator(s) will be a key
person for the efficient operation of ADB.
<p>
Ideally, the curator would be an <i>Arabidopsis</i> biologist with extensive
computing experience. Computing experience is important because key
features of the data in ADB will be compatibility with the data in
other genome databases and portability to future generation
<i>Arabidopsis</i> databases. Experience as an <i>Arabidopsis</i> biologist will be
extremely helpful in devising ways of collecting, entering, storing,
retrieving, and presenting data that <i>Arabidopsis</i> biologists would find
useful.
<p>
The ADB curator will work closely with the ADB advisory committee in
determining the categories of data stored in ADB and in designing the
structure of the data storage system. It is important that this design
be forward looking, anticipating the increasing complexity of genomic
data that will surely occur as more regions of the genome become well
known.
<p>
The ADB curator will also interact closely with the ADB editorial
board as well as with individual contributors, in the way that the
managing editor of a journal communicates with both authors and
referees.
<p>
Data entry and correction: It should be assumed that much
responsibility for entry or submission of data to the database will
rest with the research community. The database proposal should
consider how users will simply and efficiently enter data. Data
submission tools should be useful, and carefully thought through.
There should be a robust procedure by which researchers will be able
to oversee the quality of data and make corrections. Corrections
should be entered by the curator, and should not be simple over-writes
of existing information. Instead, author-supplied corrections should
be new entries, providing updated information. In this way, author
updates can be edited and removed, if desired.
<p>
The ADB curator will be responsible for designing user friendly data
entry software that assigns a unique accession number to each entry. A
great deal of thought must be given to the mechanism by which data is
refereed before permanent entry into ADB. Some data such as DNA
sequences may require relatively little, if any, refereeing. A major
decision to be made is whether any data will be entered permanently by
members of the community without the intervention of the curator.
<p>
Some data such as references to published work can be entered directly
by the curator. Other data generated in individual laboratories can be
entered directly by members of the community. In other cases, however,
the curator will have to work closely with particular members of the
advisory committee, the editorial board, or the <i>Arabidopsis</i> community
to collect, collate, and present certain types of data, such as two
and three factor mapping data used in the construction of genetic
maps.
<p>
Individual researchers should not be allowed to make corrections or
alterations in existing data. Rather, corrections or additions to
existing data should be entered by the curator in consultation with
the advisory committee or editorial board if appropriate.
<p>
Special attention should be paid to sites entering major amounts of
data: they should bear much of the responsibility for data entry in
their own areas, perhaps by training a person at that site to be an
"assistant curator." There must be explicit consideration of how the
curator will collect and enter data from major data producers.
<p>
The issue of linking formal publication of information in
peer-reviewed journals to submission of the underlying data in ADB
should be addressed. The requirement by most journals that
descriptions of sequences be accompanied with accession numbers of the
sequences in appropriate databases has had a positive effect on making
sequence data easily available to the community. Some similar system
for linking published papers with material in the ADB should be
considered.
<p>
Long-term storage: Databases are like libraries or journals. Once they
are created, the need to access them will continue indefinitely.
Long-term funding is required to keep information current and to
gather information which is not already in electronic form.
<p>
<b>C. Relation to other Databases and Programs</b>
<p>
When possible, ADB should use industry-standard hardware and software,
so that ADB is both compatible with and can communicate transparently
with other data bases. However, as stated elsewhere in this report,
the primary goal of ADB should be to collect and store data using
currently accepted database models rather than to develop new database
software specifically for ADB. The most important principle,
therefore, in the design of ADB is that the data be entered in a form
that makes it possible to interface easily with other databases and
which makes the data in ADB portable to future generation database
software. Any software that is written specifically for ADB (display
of genetic maps, for example) should be layered and use industry
standard interfaces so that the software, as well as the underlying
data, is also compatible with and portable to future generation
databases. The general principle is that is only makes sense to spend
money on the development of generic databases that can be used for a
variety of different genomes.
<p>
An ADB proposal should discuss the software used in other genome
databases and how ADB will relate to that software. Since funds for
developing an <i>Arabidopsis</i> database from scratch are not available,
programs and methods from other databases must be considered for use
by the <i>Arabidopsis</i> proposal.
<p>
The data from the <i>Arabidopsis</i> database must be accessible by
analytical programs for DNA sequence analysis, genetic mapping, etc.
Therefore, an application programming interface should be included.
<p>
ADB should interface transparently with other genome databases (or
other ADB data bases if there are more than one) such that a user can
ask a question of or retrieve data from more than one database
simultaneously.
<p>
<b>D. Availability of the database.</b>
<p>
Data accumulated by a publicly funded database should be community
property. There should be no restrictions on the availability of the
data in ADB. ADB must be available internationally by INTERNET.
However, for the foreseeable future, not all potential users will have
access to a high quality INTERNET connection that will support a
graphical interface. Therefore, ADB should also be made available in
another format such as 9 track tape or CD.
<p>
Most potential ADB users utilize IBM-PC compatible or Macintosh
computers. PC and Macintosh interfaces should be made available, if
possible, for viewing of graphical or other pictorial data. The
X-windows standard might well solve both the generic network interface
challenge, as well as that for PCs and Macs, since there are now
several good X-windows products available for both PCs and Macs.
<p>
All data should be available for bulk access or bulk downloading.
Screen-by-screen retrieval is not sufficient for transferring large
amounts of information. Bulk access of data is necessary not only for
transfer of data into other databases but for individual researchers
who want to construct a custom database. Custom (private) databases
may form the basis of electronic laboratory notebooks of the future.
<p>
To promote interaction and cooperation between different databases
that contain information relevant to the <i>Arabidopsis</i> community, it may
make sense to create a steering committee, composed of representatives
of the cooperating funding agencies and of advisors to each of the
databases.
<p>
<b>E. One or several databases?</b>
<p>
There does not have to be only a single <i>Arabidopsis</i> database, but each
should have a clear and defined subset of the database task, and
appropriate links to the others.
<p>
If there is more than one <i>Arabidopsis</i> database, it is imperative that
they be integrated and that the staff operating the different
databases be committed to cooperating with each other.
<p>
It makes no sense for two more databases to spend scarce resources
collecting the same data. If there are multiple <i>Arabidopsis</i> databases,
each database should have reciprocal access to the other databases
with full read, query, and dump privileges.
<p>
A proposal must contain provisions for interacting with other
<i>Arabidopsis</i> databases, and must be clear on which of the data types
and services are to be included in the planned database.
<p>
In addition to ADB, simple and fast network access via software like
GOPHER will be required to meet all needs of the <i>Arabidopsis</i>
community.
<p>
<b>F. Education</b>
<p>
The ADB investigators should be provided funding for the provision of
community education and training. This would include the development
of on-line help, training manuals, workshops, and short courses. The
ADB developers should maintain complete documentation and source code.
This information should be in the public domain. Because educators and
students in higher education (including high-school students) may make
use of ADB, sufficient documentation for non-sophisticated users
should be made available.
<p>
<b>IV. What design-feature issues need to be considered?</b>
<p>
As an aid to those who will plan to submit proposals for <i>Arabidopsis</i>
database services, the committee discussed at a general level some of
the design features that would allow the ADB to serve the community
with maximal efficiency, and recommended that any proposal for
database services include a discussion of these design considerations.
In addition, the committee recommended that any <i>Arabidopsis</i> database
proposal would consist of two parts. One should be for biologists,
describing what the database could do, with examples like those in the
list of examples given above. The other should be for database and
computing experts, to show how the goals will be achieved at a
technical level, and how the methods proposed relate to existing
technical methods in use in genome databases.
<p>
<b>A. Design considerations that should be discussed in the proposal:</b>
<p>
1. Controlled vocabularies. The use of plain text comment fields and
other fields with uncontrolled vocabularies must be kept at a minimum.
Any field upon which a user might be expected to initiate a search
should contained controlled-vocabulary entries. It will not do, if
users must, for example, know every synonym for a particular phenotype
in order to obtain information about that phenotype.
<p>
2. System portability. Data must always be in a form that will be
portable to new database systems, and new computers and computer
types. Explicit description of how this will be achieved should be
included in the proposal. Industry standard methods for data storage
and transport should be discussed in relation to the methods proposed
for the <i>Arabidopsis</i> database, with the aim of being able to transport
data and functionality to new database systems. Current industry
trends are towards layered software systems and client-server
databases and we would expect submitters of proposals to explicitly
discuss current trends such as these in the proposal.
<p>
3. Bulk data transfers. All data should be available for bulk access
or bulk downloading, as discussed above. The methods by which this
will be assured should be described.
<p>
4. Data time stamping. The database should have update information on
its own contents: date/time coding should be considered for all data
and links between data, so that update data can be obtained by users
on a regular basis.
<p>
5. Cross references. The database should contain cross-references to
the following databases (where available):
<p>
<ul>
<li>GenBank Nucleotide Sequence Database
<p>
<li><i>Arabidopsis thaliana</i> stock center database
<p>
<li>Cell and/or probe repository catalog number(s)
<p>
<li>Genetic map databases for species showing significant synteny with
<i>Arabidopsis thaliana</i>
</ul>
<p>
6. Database structure. To ensure the development of a robust and
stable production quality system, the database should be based upon
readily available, proven software.
<p>
7. Database interoperability. To facilitate inter-database linking,
and referencing of items in the database in an unambiguous manner for
publications and other reports, and to ensure the long-term ready
availability of the data in the database, primary entities ("unit
records") should be identified by public, unchanging unique
identifiers (accession numbers).
<p>
8. Data dictionary. The database structure should be described in a
data dictionary or repository which would be available to database
users. In addition to documenting the database schema and providing
prose descriptions of the database tables and data elements (fields),
the data dictionary should document integrity constraints implemented
in the database (or in layered software) and the rationale for the
database design. Copies of the data dictionary should be available to
users at a nominal cost.
<p>
9. Software. All software developed under this contract should be
designed and implemented for maximal portability consistent with
timely and cost-effective delivery of service.
<p>
10. Computing facilities. To facilitate user and developer access, the
database should be maintained on a computer (or network of computers)
which uses the Unix operating system and which is connected to the
U.S. Research Internet by communications lines which operate at a
minimum of 1.54 Mbits/sec.
<p>
11. Performance. The database should respond to user queries in a
reasonable time, and the database developers should regularly monitor
the system efficiency and response times and should take corrective
steps whenever response degradation becomes significant.
<p>
12. Documentation. The developer should maintain complete
documentation and source code for all software developed in this
project and complete documentation for all database design and
implementation. This information should be made available to the
funding agencies and to others to ensure the long term functionality
of the system and to ensure long term access by the scientific
community, even beyond the termination of the developer's involvement
in the project.
<p>
13. Security. The database will represent an irreplaceable resource
for the community. Therefore, the developer must take care to ensure
that the data and programs are protected against loss.
<p>
<b>B. Short-term research goals</b>
<p>
The developer should consider and propose to carry out some short-term
research relevant to improving the quality of the <i>Arabidopsis thaliana</i>
database. Some possibilities for short-term research would be:
<p>
1. Add capability of representing and storing data for other plant
species.
<p>
2. Explore the abstract and generic nature of mapping data and develop
generic representation systems in anticipation of adding mapping data
generated by as yet undiscovered mapping techniques.
<p>
3. Develop methods for defining and controlling differential access to
the data.
<p>
4. Develop a means for providing an audit trail, or other historical
record, of all changes to the database.
<p>
5. Investigate methods for facilitating interdatabase interactions and
connections.
<p>
6. Develop a stable, documented application program interface (API) to
the database.
<p>
7. Develop a method for representing variations in data quality and
for recording uncertainty.
<p>
8. Develop means for integration of physical mapping data with genetic
and cytogenetic maps.
<p>
9. Develop means for providing ready user access to underlying
supporting data (maintained in remote laboratory databases) through
the database on-line user interface.
<p>
10. Develop improvements in data presentation, including graphical
representation of maps.
<p>
<b>C. Possible long term research goals</b>
<p>
1. Investigate new database systems and new data models.
<p>
2. Monitor advances in hardware improvement and develop plans for
using new hardware to improve the quality of the database.
<p>
All research projects should include specific plans for production of
prototype systems and for their acceptance testing by the appropriate
user communities.
<p>
<b>Participants</b>
<p>
The <i>Arabidopsis</i> Informatics Needs Assessment Workshop, June 5 and 6,
1993, Dallas, Texas, was attended by the North American <i>Arabidopsis</i>
Steering Committee, the elected representatives of <i>Arabidopsis</i>
researchers:
<p>
Elliot Meyerowitz, California Institute of Technology (Chair)
<p>
Fred Ausubel, Massachusetts General Hospital and Harvard School of
Medicine
<p>
Joanne Chory, Salk Institute
<p>
Joseph Ecker, University of Pennsylvania
<p>
David Meinke, Oklahoma State University
<p>
Chris Somerville, Michigan State University
<p>
and by:
<p>
Machi Dilworth, National Science Foundation
<p>
Steven Heller, U.S. Department of Agriculture
<p>
A. Vassarotti, Commission of the European Communities
<p>
Ken Fasman, Genome Database, Johns Hopkins University
<p>
Robert Robbins, Laboratory for Applied Research in Academic
Information, Johns Hopkins University
<p>
