<html>
<pre>
Dear Mike and David,

First, we want to formally thank you for your efforts to make ATDB an
excellent resource for the Arabidopsis community.  Since we spoke with you
at the meeting in Cold Spring Harbor last December, we have been
considering what resources the community needs now, and will need, as the
genome continues to be sequenced.   At this point in time, we feel that the
most important contribution that ATDB could make is to generate a unified
genetic and physical map, using all of the resources currently available at
the web sites of individual labs.  In this way, ATDB would evolve from a
collection of web links to a single resource site which meets all of the
needs of the genetics and genomics communities.  We are willing to discuss
this with you at length, and are eager for your input.  In particular, we
believe the following resources woud be extremely valuable:

In a clickable, ACeDB type format,

1)  All of the available sequence from AGI labs and GenBank  should be
placed along  all five Arabidopsis chromosomes.  These should not be links
to Entrez, but instead clickable data at ATDB.   If this will data set is
so large that
it will overload the servers, please let us know.  We feel that this is
such an
important resource, that additional hardware purchases would be appropriate.

2) All of the putative open reading frames should be added to this
framework, as well as all EST hits.

3)  Using the TIGR end sequence data base, the Altmann hybridization
database, and other AGI resources, as many BACs as possible should be
placed onto this map.  We know that in some cases these matches will be
hypothetical.  However, many users are spending their time assembling the
same hypothetical collection, and as long as there's a disclaimer, the
community will not be harmed.  In addition, using these framework BACs,
the contigs could be extended with the Wash U and Altmann databases.  In
our hands, the Wash U database is quite reliable, allowing BAC tiling for
megabase distances.

4)  Onto this map,  the YACs should be added.  In this case, you might want
to focus on the published maps of chromosomes 2, 4, and 5 (and 3, which
Bouchez apparently has in press).  The temporary maps that are on the ATGC
site are less useful, as they have not been checked as rigorously.   The
YAC end sequence database that is at Kazusa should also be incorporated to
indicate  the map position of the YAC ends.  In addition, using the CIC to
EST  and PCR to YAC databases,  all putative EST links should be added.

5)  All known CAPS markers, SSLP markers, and RFLP markers should also be
placed on this map (with links to the individuals who developed the
marker).  Sometimes these markers will be placed in regions that are not
currently being sequenced.  In other cases, you could use the existing
sequence to place them precisely on the map.

6)  All cloned genes that have been mapped as phenotypes on the visible map
should also be added.  With these anchor points, you could then place other
visible phenotypic markers on the map in positions that are approximately
accurate.  It may also be appropriate to provide cloned genes as a separate
list,
potentially facilitating interactions between laboratories working in the
same area.

7)  Finally, wherever it's relevant, links to publications and pertinent
information
on other web sites should be added.  These should include not only those
publications dealing with genomic information, but also publications on gene
function and characterization.

If this framework is assembled (or at least started) by this summer's
Arabidopsis meeting, we will then galvanize support from the community to
update the map and add unpublished information.  If you like, we can
arrange time at the meeting for you to demonstrate the database and to call
on the community for contributions.

We are willing to work with you to facilitate this effort.  You have the
full support from the funding agencies to generate a database that serves
the community well, and we will continue to let the funding agencies know
that it's this type of curration that the Arabidopsis community needs most.


Best wishes,

The NAASC
Rick Amasino
Jeff Dangl
Chuck Gasser
Steve Kay
Daphne Preuss
Detlef Weigel
</pre>
