
<HTML>

<TITLE>The Arabidopsis Database moves to Stanford</TITLE>

<H2><img src="/images/tairsmall.gif">The Arabidopsis Database moves to Stanford</H2>

<BODY>
David Flanders, Fabien Petel and Mike Cherry <P>
Dept. of Genetics, School of Medicine, Stanford University, Stanford, CA 904305-5120, USA.
<hr>
As recently announced on the BIOSCI Arabidopsis newsgroup, the formal
and complete transfer has taken place of AAtDB and associated WWW,
Gopher and ftp resources from the Department of Molecular Biology,
MGH, Harvard to the Department of Genetics, in the School of Medicine
at Stanford University.<p>

With this move comes a change in title and its meaning.  The AAtDB
name, which in the past has represented the name of the formal
database, will not be used.  The Stanford effort will be known as the
<i>Arabidopsis thaliana</i> Database Project or <b>AtDB</b> (one A).
The name AtDB will be used for all aspects of the Stanford effort
including a formal database, WWW and Gopher search facilities to
Arabidopsis literature and electronic manuals, plus a variety of
future analysis-resources.  The AtDB name is thus not just a database,
but a collection of information and analytical resources maintained by
the Stanford project.<p>

To access AtDB, point your WWW browser (such as Netscape) to the URL: 
<br>
<a href="/">/</a> 
<br>
This takes you to the Stanford Genomic Resources home page.  From
there, you can access AtDB, the Arabidopsis thaliana Database project,
as well as SGD, the Saccharomyces Genome Database, and the genome
research centers at Stanford. You may want to update your "Bookmarks"
accordingly.<p>


Please note that we are focusing our efforts on the WWW presentation.
A WWW browser is required to have access to all the information we are
providing.  However, the basic information is available via Gopher at
the URL:<br> 
<a href="gopher://genome-gopher.stanford.edu/">
gopher://genome-gopher.stanford.edu/</a>
<br>
We are considering terminating Gopher access in late 1996.<p>

The goal of the current NSF-funded AtDB project is to develop a
prototype of the next generation of Arabidopsis database. This will
provide the user with an easy-to-use interface to the vast amount of
data being produced by the Arabidopsis research
community. Specifically, we will be developing software to provide WWW
interfaces to all the data held on the Stanford AtDB -- with, of
course, links to relevant information elsewhere. This will obviate the
need to have a "central" stand-alone database, with its requirement
for regular updates and the associated lag in the transfer of
knowledge that this inevitably entails.<p>

Two of the new features that we hope to make available soon are: (i) the
ability to perform BLAST and FASTA sequence searching against only
known Arabidopsis DNA and protein sequences; and (ii) the
ability to have zoomable and "clickable" maps available
over the Web, with links to all relevant data.<p>

In parallel with the development of the new database, we will continue
to collect new information, which will be made available through the
AtDB WWW site at Stanford.  In addition, a small proportion of the
funding does enable us to continue to maintain and distribute the
information previously collected by the AAtDB group in Boston.<p>

We'd thus love to have any data you may have that will be of use to
the Arabidopsis community. Please contact us before sending data so we
can discuss the best format for transfer.<p>

The staff on the Stanford AtDB project are: Mike Cherry, PI; David
Flanders, Curator; and Fabien Petel; Scientific Programmer.  AAtDB has
been provided for the past four years by Howard Goodman, MGH/Harvard,
and was created by Mike Cherry and Sam Cartinhour and, for the past
two years, has been curated by John Morris.<p>

Please send all correspondence regarding AtDB to 
<a 
href="mailto:curator@arabidopsis.org">
TAIR Curator</a>

</HTML>

