<title>nagl--1978-aabgq.html</title><p>

<h2>REPETITIVE DNA IN ARABIDOPSIS KORSHINSKYI</h2><p>

W. Nagl<p>

Department of Biology, The University
P.O. Box 3049, D-6750 Kaiserslautern
Federal Republic of Germany<p>

INTRODUCTION<p>

Arabidopsis is the plant genus with the lowest nuclear DNA content
which was detected so far. For A. thaliana, Bennett (1972) established
a 2C value of only 0.53 pg, and Sparrow et al. (1972) estimated a base
number of 2.0 x 10 9 from the nuclear volume, but this is probably an
over estimation.<p>

It has been shown in a series of expriments that the minimum DNA
content of a phylum is correlated with its evolutionary complexity,
but also that the species-specific DNA content varies to a great
extent even in closely related species (known as C-value paradox; for
reviews see Sparow et al. 1972, Nagl 1976, 1978). Recently evidence
has been found in support of the hypothesis that the "primary DNA"
(which may represent the protein-coding genes; Hinegardner 1976)
varies only little among living organisms, while the "secondary DNA"
(that includes the noncoding repetitive sequences) varies extremely
(Narayan and Rees 1976, Rees et al. 1976). Species with low nuclear
DNA contents are, therefore, expected to show only little repetitive
DNA, while those with high nuclear DNA contents should possess much
repetitive DNA. Arabidopsis with its low DNA content can deal as a
model object to test this expectation.<p>

Material and Methods<p>

Seeds of Arabidopsis korshinskyi 3 (code number 76/328, chromosome
number 2N=48) were obtained from the Arabidopsis Information Center at
Frankfurt am Main, and grown in the Botanical Garden of the University
of Kaiserslautern. Four sets of experiments were performed using
between 20 g and 70 g material of fruiting plants.<p>

DNA was extracted according to the method of Wells and Ingle (1970) by
homogenization of the material after freezing in liquid nitrogen,
lysis, second homogenization with a Buhler homogenizer, and several
extractions with chloroform-isoamyl alcohol (24:1, v/v and with
phenol. After RNase and pronase treatment the solution was centrifuged
over night at 100 000 x g, and the pellet dissolved in 0.1 x SSC (SSC
= 0.15 M sodium chloride + 0.015 M sodium citrate). The solution was
extensively dialyzed against 0.12M phosphate buffer and purified by
column chromatography over Sepharose 4-B (Pharmacia).<p>

Thermal denaturation of the DNA was carried out in 0.12 M phosphate
buffer and calibrated with a Gilford spectrophotometer equipped with
the special thermocuvette and the thermoprogrammer. The GC content of
the DNA was calculated by the formula given by Mandel and Marmur
(1968).<p>

For renaturation measurements, the DNA was sonicated with a MSE
ultrasonic disintegrator at medium energy ouput for 8 x 15 sec with
cooling intervals. Then the DNA was denatured for 10 minutes at
105*degrees* C and allowed to renature at Tm (melting temperature) -
25*degrees*C, according to Britten et al. (1970). The hyperchromocity
was recorded in a Gilford spectrophotometer, kept at constant
temperature with an ultrathermostate and controlled with a
thermosensor. E. coli DNA (R. Paesel, Frankfurt) was used as a
standard.<p>

As the haploid genome of Arabidopsis is about 50 times larger than
that of E. coli, the equivalent cot value, at which single copy
sequences are renatured, should be 50 times higher than that of E.
coli (cot = DNA concentration at the beginning of the experiment,
given in mol, x reassociation time in seconds). All experiments were
repeated 6 times.<p>

<a href="/ais/images/vol15/vol15-pg66-fig1.gif">Click here for figure</a><p>

Figure 1: Wave-length scan of DNA extracted from Arabidopsis
korshinskyi plants<p>

<a href="/ais/images/vol15/vol15-pg66-fig2.gif">Click here for figure</a><p>

Figure 2: Thermal denaturation profiles of Arabidopsis korshinskyi DNA
in 0.12 M phosphate buffer (three kuvettes were used simultaneously).
Note the thermal satellite visible at the arrow.<p>

<a href="/ais/images/vol15/vol15-pg66-fig3.gif">Click here for figure</a><p>

Figure 3: Reassociation kinetics of Arabidopsis and E. coli DNAs in
0.12 M phosphate buffer<p>

RESULTS<p>

The wave-length scan as shown in Figure 1 indicates high purity of the
extracted DNA, although the hyperchromicity was only about 20%. The
mean Tm was 8.3.*degrees*C, corresponding to an average GC content of
35.3%. The thermal melting profiles exhibit an irregularity at
89.5*degrees*C (Figure 2, arrow). This indicates the presence of a
GC-rich satellite fraction.<p>

The reassociation kinetics are shown in Figure 3. Arabidopsis
korshinskyi DNA display a prominent proportion of fast annealing,
repetitive sequences. In comparison with E. coli DNA, the cot/2 value
is clearly lower. From the data it can be concluded that at least 60%
of Arabidopsis DNA are composed of repetitive sequences of a frequency
between 100 and 10 000 and a complexity of some hundred to some
thousand nucleotide pairs. Only about 35% of the genome appear to be
composed of unique sequences. If the size of the haploid genome of
Arabidopsis korshinskyi is similar to that of A. thaliana, what is
very likely, than the unique DNA sequences amount to approximately
0.11 pg, or 10 8 nucleotide pairs. Thus, the maximum number of unique
genes of an average size of 1500 nucleotide pairs in Arabidopsis is
about 67 000, but this still is an over-estimation, as many unique
sequences are known, which do not code for proteins or stable RNAs.<p>

DISCUSSION<p>

The present denaturation-reassociation experiments of Arabidopsis
korshinskyi DNA revealed a proportion of 60-65% repetitive sequences
in this species. This is in the same order of magnitude as in other
plant species with relatively low DNA content (Flavell et al. 1974).
Based on the body of data which is known on the percentage of
repetitive DNA sequences in plants and animals so far (reviewed by
Nagl 1976, Straus 1976) it is evident that no higher plant has less
than 50% repetitive sequences in its genome. This is in sharp contrast
to the situation in fungi and lower animals, where normally 5 - 20% of
the nuclear DNA are composed of repetitive sequences.<p>

The biological function of repetitive DNA is not yet understood. It
has been speculated that it is regulatory in nature and controls the
activity of those genes among which it is interspersed (Davidson and
Britten 1973, Davidson et al. 1977, Zuckerkandl 1976). Repetitive DNA
may also play an important role in macro-evolution by providing sites
for chromosome rearrangement, recombination and insertion of
transposable elements (Lima-de Faria 1975, Nagl 1978)). In any case
the presence of a considerable amount of reiterated DNA in higher
organisms indicates that it is a prerequisite for the evolution of
complex organization and its regulation.<p>

References<p>

Bennett, M.D.: Proc. R. Soc. London. Ser. B., 181, 109-135 (1972)<p>

Britten, R.J., Graham, D.E., Neufeld, B.R.: Meth. Enzymol. 29, 363-418
(1974)<p>

Davidson, E.H., Britten, R.J.: Quart. Rev. Biol. 48, 565-613 (1973)<p>

Davidson, E.H., Klein, W.H., Britten, R.J.: Devel. Biol. 55, 69-84
(1977)<p>

Flavell, R.B., Bennett, M.D., Smith, J.B., Smith, D.B.: Biochem.
Genet. 12, 257-279 (1974)<p>

Hinegardner, R.: in Ayala, F.J., ed.: Molecular Evolution, pp 179-199,
Sinauer, Sunderland, USA (1976)<p>

Lima-de-Faria, A.: Hereditas 81, 249-284 (1975)<p>

Mandel, M. Marmur, J.: Meth. Enzymol. 12B, 195-206 (1968)<p>

Nagl, W.: Zellkern und Zellzyklen, Ulmer, Berlin (1976)<p>

Nagl, W.: Endopolyploidy and Polyteny in Differentiation and
Evolution, Elsevier/North Holland, Amsterdam (1978).<p>

Narayan, R.K.J. Rees, H.: Chromosoma 54, 141-154 (1976)<p>

Rees, R.W., Fox, D.P. Maher, E.P.: in Jones, K. Brandham, P.E., eds:
Current Chromosome Research, pp 33-41, North Holland, Amsterdam
(1976).<p>

Sparrow, A.H., Price, H.J. Underbrink, A.G.: Brookhaven Symp. Biol.
23, 451-494 (1972)<p>

Straus, N.A.:. in Fasman, G.D., ed.: Handbook of Biochemistry and
Molecular Biology - Nucleic Acids II, pp. 319-323, CRC Press,
Cleveland, USA (1976)<p>

Wells, R., Ingle, J.: Plant Physiol. 46, 178-179 (1970).<p>

Zuckerkandl, E.: J. Mol. Evol. 9, 73-104 (1976)<p>

