<html>
<script>
<!--

function Page_load(page) {
	

		
	   if (page == 4)
		page_title = "RI_map_move.html";
	   else if (page == 5)
		page_title = "Contents.html";
	   else if (page == 6)
		page_title = "Cherry-Flanders-Petel.WW.html";

	if (page_title == "")
		return;
	
	
	window.parent.frame_main.location = window.parent.frame_menu.home_page + page_title;
	   
}

	var strng;
	function message(id) {
	   

	   if (id == 4)
		strng = 'Lister and Dean RI map project moves to NASC';
	   else if (id == 5)
		strng = 'WeedsWorld Volume3(ii) Contents';
	   else if (id == 6)
		strng = 'Update on the activities at Stanford';

	   else 
		return false;

	window.status = strng;
	return true;
}

// end of script -->
</script>

<head>
<title>Soluble derivatives of green fluorescent protein (GFP) for use in Arabidopsis thaliana ( 5-Sep-1996)</title>
<!-- Created by: Dr Mike Arnold, 29-Aug-1996 -->
<!-- Changed by: Dr Mike Arnold,  5-Sep-1996 -->


</head>
<body bgcolor="#ffffff">
<bodytext><blockquote>

<bodytext><h1>Soluble derivatives of green fluorescent protein (GFP) for use in <i>Arabidopsis thaliana</i></h1>


Seth J. Davis and Richard D. Vierstra<br>
Laboratory of Genetics and Department of Horticulture, University of Wisconsin-Madison 53703
<br>Phone (608) 262-1622, FAX (608) 262-4743
<br><a href="mailto:vierstra@facstaff.wisc.edu">vierstra@facstaff.wisc.edu</a><p></p>
<hr>
<h4>Abstract</h4>
Green fluorescent protein (GFP) from <i>Aequorea victoria</i> has rapidly become a standard reporter in many biological systems.  However, the use of GFP in <i>Arabidopsis thaliana</i> has been limited by aberrant splicing of the mRNA when expressed in dicots and by protein insolubility.  It has been shown that GFP can be expressed in <i>A. thaliana</i> after altering the codon usage in the region that is incorrectly spliced, but the fluorescence signal was weak, possibly due to aggregation of the encoded protein.  Here, we describe soluble versions of GFP, containing a series of site-directed mutations, that show improved fluorescence in <i>A. thaliana</i>.  These mutations can be coupled with two chromophore mutations to shift the spectral qualities of GFP.  We discuss the preliminary characterization of these modified GFPs for use in plants.<p></p>

<h4>Introduction</h4>
Reporter proteins have been recruited to monitor cellular events which would be difficult or impossible to observe.  In plants, the two most widely used reporters are beta-glucuronidase (GUS) and luciferase (LUC).  These reporter systems have been instrumental in many studies, but each has limitations.  The most notable is that neither allows for convenient non-invasive <i>in vivo</i> analysis, because both systems require an added substrate to monitor reporter activity.<p>

Recently a new reporter system, green fluorescent protein (GFP) isolated from <i>Aequorea victoria</i>, has been described which is not limited in this respect (<a href="#chaflie">Chalfie, 1994</a>).  GFP chromophore is assembled by the self-catalyzed covalent modification of amino acids Ser-Tyr-Gly at positions 65-67 to form a <i>p</i>-hydroxybenzylidene-imidazolidinone species (<a href="#cubitt">Cubitt, 1995</a>; <a href="#heim1">Heim, 1994</a>).  The wild-type chromophore is excited with either UV or blue light (maximally at 396nm or 475nm) and emits green fluorescence (maximally at 508nm) (<a href="#heim1">Heim, 1994</a>).  The intrinsic fluorescence of GFP allows for non-invasive analysis which can be monitored without the destruction of the biological sample (<a href="#chaflie">Chalfie, 1994</a>; <a href="#cubitt">Cubitt, 1995</a>).  This reporter system has been shown to function in a wide variety of biological systems, including plants (<a href="#corbett">Corbett, 1995</a>; <a href="#haseloff">Haseloff, 1995</a>; <a href="#kaether">Kaether, 1995</a>; <a href="#wang">Wang, 1994</a>).</p><p>
The first successful reports of GFP activity in monocot protoplasts involved expression from the wild-type GFP cDNA (<a href="#galbraith">Galbraith, 1995</a>; <a href="#hu">Hu, 1995</a>; <a href="#niedz">Niedz, 1995</a>; <a href="#">Sheen, 1995</a>).  However, similar studies in dicot plants failed because these species inappropriately splice out a region within the coding sequence of the GFP mRNA (<a href="#haseloff">Haseloff, 1995</a>).  After the aberrant splicing was corrected by altering the codon usage around the splice junction, transgenic <i>A. thaliana</i> expressed this mutated GFP, but displayed limited fluorescence (<a href="#haseloff">Haseloff, 1995</a>).  Work in <i>E. coli</i> suggests that this low fluorescence level might be caused by protein insolubility, as <i>E. coli</i>-expressed GFP readily aggregates (<a href="#crameri">Crameri, 1996</a>; <a href="#heim1">Heim, 1994</a>).  Aggregated GFP fails to assemble a chromophore and is therefore non-fluorescent (<a href="#heim1">Heim, 1994</a>).  Further, GFP aggregates are cytotoxic to <i>E. coli</i> (<a href="#crameri">Crameri, 1996</a>), Chinese hamster ovary (CHO) cells (<a href="#crameri">Crameri, 1996</a>) and <i>A. thaliana</i> (<a href="#haseloff">Haseloff, 1995</a>).</p><p>
In the last couple of years, a large collection of GFP derivatives have been constructed.  Many of these have optimized codon use (Clonetech) and/or contain one of several identified chromophore changes that alter the spectral properties of GFP for use in specific applications (<a href="#heim1">Heim, 1996; Heim, 1995; Heim, 1994</a>).  Many chromophore transformations were identified by site-directed mutagenesis around the chromophore region (<a href="#cormack">Cormack, 1996</a>), or after random mutagenesis, by selecting for brighter cells after excitation by blue light (<a href="#heim">Heim, 1996; Heim, 1995; Heim, 1994</a>).  Recently, <a href="#crameri">Crameri <i>et al.</i> (1996)</a> screened for increased whole-cell fluorescence under UV light after PCR-induced mutagenesis of the GFP coding region.  This protocol did not select for chromophore mutations; instead, it selected amino acid substitutions that decreased hydrophobicity, resulting in more soluble proteins.  These GFPs aggregated less, causing an increase in fluorescence signal and a decrease in toxicity (<a href="#crameri">Crameri, 1996</a>).  The most active of these soluble GFPs contained three mutations: Phe99 to Ser (F99S), Met153 to Thr (M153T), and Val163 to Ala (V163A).</p><p>
This report describes three improved GFP constructions for use in <i>A. thaliana</i>.  These modified genes contain altered-codon use to avoid improper mRNA processing (<a href="#haseloff">Haseloff, 1995</a>) and three site-directed mutations known to improve protein solubility [Modeled after the triple mutant F99S, M153T, V163A (<a href="#crameri">Crameri, 1996</a>)].  Two of these constructions also contain a chromophore modification causing a spectral shift in the excitation and emission spectra [Modeled after the S65T and the P4 mutants (<a href="#heim1">Heim, 1995; Heim, 1994</a>)].  These chromophore substitutions are established as functional in monocot and dicot protoplasts (<a href="#reichel">Reichel, 1996</a>).  Here, we demonstrate that these soluble GFPs are highly fluorescent in <i>A. thaliana</i> in a transient assay, and therefore should greatly facilitate studies using this reporter protein.</p><p></p>
 
<h4>Materials and Methods</h4>

<h5>Site-Directed Mutagenesis of mGFP4</h5>
The template plasmid for site-directed mutagenesis of the GFP coding region was the plant-modified GFP, mGFP4 (generously donated by J. Haseloff).  It contains altered codon usage in an 84 base-pair interval which is efficiently mis-spliced in dicot plants (<a href="#haseloff">Haseloff, 1995</a>), and an amino acid substitution (Glu80 to Arg) present in the first reporter-clone GFP generated (<a href="#chalfie">Chalfie, 1994</a>; Chalfie, personal communication).  We subcloned the CaMV 35S-mGFP4-NosT region as a HindIII-EcoR1 fragment from pBIN 35S-mGFP4  into the high copy pUC118 plasmid.  Site-directed mutations were introduced by the Quikchange&#129; mutagenesis strategy  (Stratagene).  The "soluble-modified" GFP plasmid (referred as smGFP) was constructed by sequential introduction of three mutations.  Mutations were, in order of generation,
<ul>
<li>V163A using <font><code>GAATGGAATCAAA<u>GCT</u>AACTTCAAAATTAG</code></font> and its complement</li>
<li>M153T using <font><code>CCACAACGTATACATC<u>ACG</u>GCAGACAAACAAAAG</code></font> and its complement</li>
<li>F99S using <font><code>GGAGAGGACCATC<u>TCT</u>TTCAAGGACGACG</code></font> and its complement.</li> 
</ul>
 The chromophore mutations Ser65 to Thr (S65T) and Tyr66 to His (Y66H) were generated from smGFP using the same mutagenesis protocol with the following:
<ul>
<li><font><code>CAGTGATGAAAG<u>TGA</u>ATACCACAAG</code></font> and complement for S65T (referred as smRS-GFP)</li>
<li><font><code>GCAGTGATGAAAGAGA<u>GTA</u>CCACAAGTTACG</code></font> and complement for Y66H (referred as smBFP).<p></p></li>
</ul>

<h5>Plant Material, Particle Bombardment, and Dark Treatment</h5>
All protein-expression experiments used <i>gl1</i>- mutant seedlings of <i>A. thaliana</i> ecotype Columbia.  Transient expression of each GFP construction was monitored following introduction of plasmid DNA by particle bombardment.  Both preparation of DNA-bound gold and the particle-bombardment protocol were based on <a href="#christou">Christou <i>et al</i>. (1990)</a>, using 7-micron gold spheres accelerated by a 7KV discharge towards seedlings placed on 1% agar petri plates.  Following bombardment, these seedlings were maintained in the dark for 6 hours.<p></p>

<h5>Microscopic Analysis</h5>
Microscopic examination of seedling leaves was with an Olympus BX60 microscope fitted with either a DAPI (excitation filter 360-370nm; dichroic mirror 400nm; emission filter 420-460nm) or FITC (excitation filter 470-490nm; dichroic mirror 505nm; emission filter 515-550nm) filter cube.  Images were taken with a Sensys CCD camera (Photometrics Ltd.) and analyzed with both IPlab Spectrum and Adobe Photoshop software.<p></p>

<h5>Results and Discussion</h5>
We tested several of the currently available GFP variants for suitable expression in <i>A. thaliana</i> (data not shown).  The only variant with detectable fluorescence when integrated into the genome was the altered-codon GFP, mGFP4 (<a href="#haseloff">Haseloff, 1995</a>).  However, as observed previously, the fluorescent signal was weak.  A review of the literature revealed improved GFPs for bacterial and mammalian use (<a href="#crameri">Crameri, 1996</a>; <a href="#cubitt">Cubitt, 1995</a>) prompting us to create similarly improved GFPs for plant use.  GFP solubility appears to be one of the limiting factors in whole-cell fluorescence.  A soluble-mutant GFP functioned equally well in <i>E. coli</i> and CHO cell lines (<a href="#crameri">Crameri, 1996</a>), suggesting that these mutations would improve GFP function in plants.<p>
<a name="figure1">
Using the soluble-GFP sequence as a model to improve mGFP4, a triple site-directed mutant was generated from the mGFP4 template (referred as smGFP).  From smGFP, we then generated two chromophore derivatives [S65T and Y66H (see <a href="protocols/figure1.html">Fig. 1</a>)].  The S65T modification is a commonly used red-shifted variant (referred as smRS-GFP), and the Y66H mutant is a blue-shifted variant (referred as smBFP) (<a href="#cubitt">Cubitt, 1995</a>).  Similar chromophore mutations have been constructed for plant use (<a href="#reichel">Reichel, 1996</a>), but none of these clones contain substitutions to improve protein solubility.</a></p><p>
<a name="figure2">
We compared the new soluble-modified GFPs to the original mGFP4.  Particle bombardment of all four GFPs, under the control of the CaMV 35S promoter and the Nos terminator, resulted in transient expression of fluorescent proteins in <i>A. thaliana</i> epidermal cells.  [Since trichomes are the most auto-fluorescent structure on true leaves (see <a href="protocols/figure2.html">Fig. 2</a>), the experiments were conducted with a mutant which lacks trichomes].  Under the FITC filter set, expressed smGFP, smRS-GFP, and smBFP were significantly more fluorescent when compared to mGFP4.  A representative cell harboring each of the constructions is <a name="figure3">shown in <a href="protocols/figure3.html">Fig. 3</a>.  smBFP had an even stronger blue signal under the DAPI filter cube (see <a href="protocols/figure3.html">Fig. 3</a>).  Focusing through the fluorescent cells indicated that GFP was predominately localized to the cytoplasm and not the vacuole (data not shown).</a></a></p><p>
When the soluble mutations were combined with the chromophore mutations S65T and Y66H, several nonpredicted properties in GFP fluorescence arose.  Because the S65T mutation results in a six-fold increase in relative fluorescence compared to the wild-type GFP (<a href="#heim1">Heim, 1995</a>), we originally predicted that smRS-GFP would be more intense than the smGFP.  Under the conditions tested (FITC cube), they had approximately the same signal.  We subsequently discovered that the M153T mutation further red-shifted the excitation spectra for smRS-GFP (<a href="#heim1">Heim, 1996</a>) such that the spectral properties of are not optimal under the FITC filter combination.  Presumably, a more optimal filter set would replace the FITC excitation filter with one that is more red shifted (490-500nm), while using the same FITC emission filter (515-550nm).  The Y66H mutation decreases the relative fluorescence of GFP (<a href="#heim1">Heim, 1994</a>), so we expected smBFP to be less fluorescent, but as with smRS-GFP, the M153T mutation affects the spectral qualities of smBFP (<a href="#heim1">Heim, 1996</a>).  Here, the M153T substitution causes an increase in the fluorescence output of the chromophore, resulting in an improved protein.  Overall, this collection of GFPs was easy to recognize, even under sub-optimal optics.</p><p>

These three soluble-modified GFPs will serve as tools for further <i>A. thaliana</i> research.  Each of these clones shows marked improvement to the existing mGFP4.  In most biological systems, GFP is primarily used for two purposes, either promoter-expression studies or protein localization.  None of these enhanced GFPs contain a retention or localization signal, so they should be well suited for either use.  Given the improved fluorescence output of these constructions, it may even be possible to view these GFP derivatives by eye.  If this is the case, these GFPs could be used in large scale genetic screens.  It may also be possible to use smGFP and smBFP in conjunction with less expensive filter combinations.</p><p>

Each of these improved GFPs have unique properties useful under specific applications.</p> 

<ul>
<li>The wild-type chromophore smGFP is a standard-use GFP, and should function in most reporter applications.  smRS-GFP is exited by visible (bluegreen) light which can interfere with green-fluorescence detection.  In contrast, smGFP can be exited by UV light which reduces this interference.  Therefore, if an FITC cube or band-pass filters are not available, smGFP may be preferred over smRS-GFP (<a href="#crameri">Crameri, 1996</a>). </li>
<li>The smRS-GFP and smBFP clones have nonoverlapping spectra for excitation and emission (<a href="#cubitt">Cubitt, 1995</a>), and should allow for convenient double-labeling experiments in plants.  These chromophore mutations (S65T and Y66H) function for double labeling in bacterial (<a href="#heim1">Heim, 1996</a>) and animal systems (<a href="#rizzuto">Rizzuto, 1996</a>), and add a new value to this set of clones.</li>
</ul>

<p></p>
<hr width="80%">
	  <h4>Clone Details</h4>

<p>
The DNA for the following clones are available from the ABRC.
<menu>
	    <li><b>CD3- 326 </b>for smGFP (soluble-modified GFP)</li>
	    <li><b>CD3- 327</b> for smRS-GFP (soluble-modified-Red Shifted GFP)</li>
	    <li><b>CD3- 328</b> for smBFP (soluble-modified Blue FP)</li>

</menu>
	  The sequence details for the clones are also available from Genbank and EMBL (<b>after 12th October 1996</b>).
	  <menu>
	    <li><b>smGFP</b> U70495 <A HREF="http://www.embl-heidelberg.de/srs/srsc?[EMBL-acc:u70495]+-sf+EMBL" target="_new">EMBL Query</a>  <A HREF="http://www2.ncbi.nlm.nih.gov/cgi-bin/submit_form_query?TITLE=GenBank+Text+Retrieval+Output&INPUTS=4&FR1=n&Q1=u70495&OP1=AND&Q2=&OP2=AND&Q3=&OP3=AND&Q4=&NDOCS=100&DB=gbupdates+genbank&FIELD-ACC=y" target="_new">GenBank Query</a></li>
	   <li><b>smRSGFP</b> U70495 <A HREF="http://www.embl-heidelberg.de/srs/srsc?[EMBL-acc:u70496]+-sf+EMBL" target="_new">EMBL Query</a>  <A HREF="http://www2.ncbi.nlm.nih.gov/cgi-bin/submit_form_query?TITLE=GenBank+Text+Retrieval+Output&INPUTS=4&FR1=n&Q1=u70496&OP1=AND&Q2=&OP2=AND&Q3=&OP3=AND&Q4=&NDOCS=100&DB=gbupdates+genbank&FIELD-ACC=y" target="_new">GenBank Query</a></li>
	    <li><b>smBFP</b> U70495 <A HREF="http://www.embl-heidelberg.de/srs/srsc?[EMBL-acc:u70497]+-sf+EMBL" target="_new">EMBL Query</a>  <A HREF="http://www2.ncbi.nlm.nih.gov/cgi-bin/submit_form_query?TITLE=GenBank+Text+Retrieval+Output&INPUTS=4&FR1=n&Q1=u70497&OP1=AND&Q2=&OP2=AND&Q3=&OP3=AND&Q4=&NDOCS=100&DB=gbupdates+genbank&FIELD-ACC=y" target="_new">GenBank Query</a></li>

	  </menu>
</p>
<hr width="80%">
<h4>References</h4>

<a name="chaflie">Chalfie, M., Tu, Y., Euskirchen, G., Ward, W., Prasher, D. (1994). Green Fluorescent Protein as a Marker for Gene Expression. Science<b> 263</b>, 802-805.</a><p>
<a name="corbett">Corbett, A. H., Koepp, D. M., Schlenstedt, G., Lee, M. S., Hoper, A. K., Silver, P. A. (1995). Rna1p, a Ran/TC4 GTPase activating protein, is required for nuclear import. J. Cell. Biol. <b>130</b>, 1017-1026.</a></p><p>
<a name="cormack">Cormack, B. P., Valdivia, R. H., Falkow, S. (1996). FACS-optimized mutants of the green fluorescent protein (GFP). Gene, In press.</a></p><p>
<a name="crameri">Crameri, A., Whitehorn, E., Tate, E., Stemmer, P. (1996). Improved Green Fluorescent Protein by Molecular Evolution Using DNA Shuffling. Nature Biotechnology <b>14</b>, 315-319.</a></p><p>
<a name="cubitt">Cubitt, A. B., Heim, R., Adams, S. R., Boyd, A. E., Gross, L. A., Tsien, R. Y. (1995). Understanding, improving and using green fluorescent proteins. TIBS <b>20</b>, 448-455.</a></p><p>
<a name="galbraith">Galbraith, D. W., Lambert, G. M., Grebenok, R. J., Sheen, J. (1995). Methods in Plant Cell Biology, Volume 50, pp.3-14, D. W. Galgraith, Bourque, D. P. Bohnert, H. J., ed. (San Diego: Academic Press).</a></p><p>
<a name="haseloff">Haseloff, J., Amos, B. (1995). GFP in plants. TIG <b>11</b>, 328-329.</a></p><p>
<a name="heim1">Heim, R., Tsien, R. Y. (1996). Engineering green fluorescent protein for improved brightness, longer wavelengths and fluorescence resonance energy transfer. Current Biology <b>6</b>, 178-182.</a></p><p>
<a name="heim2">Heim, R., Cubitt, A., Tsien, R. (1995). Improved green fluorescence. Nature <b>373</b>, 663-664.</a></p><p>
<a name="heim3">Heim, R., Prasher, D., Tsien, R. (1994). Wavelength mutations and posttranslational autoxidation of green fluorescent protein. Proc. Natl. Acad. Sci. USA <b>91</b>, 12501-12504.</a></p><p>
<a name="hu">Hu, W., Cheng, C. (1995). Expression of Aequorea fluorescent protein in plant cells. FEBS Letters <b>369</b>, 331-334.</a></p><p>
<a name="kaether">Kaether, C., Gerdes, H. H. (1995). Visualization of protein transport along the secretory pathway using green fluorescent protein. FEBS Lett. <b>369</b>, 267-271.</a></p><p>
<a name="niedz">Niedz, R. P., Sussman, M. R., Satterlee, J., S. (1995). Green fluorescent protein: An <i>in vivo</i> reporter of plant gene expression. Plant Cell Rep. <b>14</b>, 403-406.</a></p><p>
<a name="reichel">Reichel, C., Mathur, J., Eckes, P., Langenkemper, K., Koncz, C., Schell, J., Reiss, B., Maas, C. (1996). Enhanced green fluorescence by the expression of an Aequorea victoria green fluorescent protein mutant in mono- and dicotyledonous plant cells. Proc. Natl. Acad. Sci. USA <b>93</b>, 5888-5893.</a></p><p>
<a name="rizzuto">Rizzuto, R., Brini, M., DeGiorgi, F., Rossi, R., Heim, R., Tsien, R. Y., Pozzan, T. (1996). Double labeling of subcellular structures with organelle-targeted GFP mutants <i>in vivo</i>. Current Biology <b>20</b>, 183-188.</a></p><p>
<a name="sheen">Sheen, J., Hwang, S., Niwa, Y., Kobayashi, H., Galbraith, D. W. (1995). Green-fluorescent protein as a new vital marker in plant cells. Plant J. <b>8</b>, 777-784.</a></p><p>
<a name="wang">Wang, S. X., Hazelrigg, T. (1994). Implications for bcd mRNA localization from spatial distribution of exu protein in Drosophila oogenesis. Nature (London) <b>369</b>, 400-403.</a></p>
<p></p>
<h4>Acknowledgments.</h4>  

We would like to thank Mr. Paul Bates and Dr. Mark Gosink for their critical review of this manuscript.  We are grateful to Dr. Jiming Jiang and Mr. Fenggao Dong for training and use of the CCD camera.  The gl1- seeds were generously provided by Dr. Brian Parks.  Finally, we are very grateful to Dr. James Haseloff for his generous gift of the pBIN 35S-mGFP vector and his discussions on its use.

<hr></bodytext>

</blockquote></bodytext>
</body>
