<HTML>
<HEAD>
    <TITLE>Pulsed-field gel electrophoresis of Arabidopsis thaliana chromosomal DNA</TITLE>
</HEAD>
<BODY>
<H2>Pulsed-field gel electrophoresis of Arabidopsis thaliana chromosomal DNA</H2>
Gregory P. Copenhaver and Craig S. Pikaard<p>
Biology Department,<BR> 
Washington University<BR>
Campus Box 1137,<BR> 
One Brookings Drive,<BR> 
St. Louis, MO 63130<BR>
<HR>
<H3>Introduction </H3>
Pulsed-field gel electrophoresis (PFGE) was developed in 1982 by Schwartz
<I>et. al.</I> as a means of resolving very large DNA molecules [<A HREF="#1">1</A>].  PFGE can
be used to separate DNA molecules from 10 kbp to approximatly 10 Mbp and
is an invaluable tool for genomic analysis.  Contour-clamped Homogeneous
Electric Field (CHEF) electrophoresis  [<A HREF="#2">2</A>], a variety of PFGE, produces sharp,
reproducible bands in straight lanes and can be used in a variety of applications
including RFLP analysis of large restriction fragments, sizing and isolation
of YACS and BACS, and direct visualization of small chromosomes.  The following
is a detailed, &quot;cookbook&quot; protocol for one and two-dimensional
CHEF analyses of Arabidopsis DNA. We have used these protocols in several
papers, referenced at the end [<A HREF="#3">3</A>, <A HREF="#4">4</A>, <A HREF="#5">5</A>].  For simplicity we have optimized
all conditions for Arabidopsis and for the BioRad CHEF DRII system.<BR>
<H3>Materials and Resources</H3>
<H4>Germination Media (GM) plates:</H4>
0.43% MS salts + Gamborg vitamins (Sigma), 2.0% sucrose, pH to 5.7 with KOH, 0.8% Agar (Sigma). Autoclave 20 minutes with no more than 500 ml/ liter bottle, and pour 25 ml/petri dish under sterile conditions in the laminar flow	hood.<BR>
<H4>Protoplasting solution:</H4>
0.4M mannitol, 0.33% cellulysin (Calbiochem), 0.17%
macerozyme R-10, 3 mM MES pH 7.5 (pH with KOH), 7 mM CaCl2.
Optionally, Glutathione (Sigma) may be added to a concentration
of 5 mM (23 mg/15 ml) as a free radical scavenger.<BR>
<BR>
<H4>NDS: </H4>
500 mM EDTA, 1% N-laurylsarcosine, 1 mM Tris. For 1 L dissolve the EDTA and Tris in 700 ml of ddH2O by adding solid NaOH to a pH between 8.0 - 9.0.  In a separate vessel dissolve the N-laurylsarcosine in 100 ml ddH2O (helps to use 65&#176;C bath). Add the sarcosine solution to the EDTA solution and pH to 9.5, and add 	ddH2O to volume.<BR>
<BR>
<H4>T10E10:</H4> 
10 mM Tris pH 8.0, 10 mM EDTA pH 8.0. Restriction Enzyme Reaction Solution:  1X restriction enzyme buffer, 8 mM spermidine*, 0.1 mg/ml BSA, 50 units restriction enzyme<BR>
* Spermidine (Sigma) should only be added to those reactions that have at
least 50 mM salt (spermidine will inhibit endonuclease activity in low salt).<BR>
<H4>STOP:</H4>
O.5X TAE (Tris Acetate EDTA gel buffer), 10 mM EDTA (pH 8.0), 1% Fidye (20 % Ficol 400, 100 mM EDTA (pH 8.0), 1% SDS, 0.1% Orange G)
<BR>
<H4>Gel Denaturing Solution: </H4>
 400 mM NaOH, 600 mM NaCl<BR>
<H4>Gel Neutralizing Solution: </H4>
 1.5 M Nacl, 500 mM Tris pH 7.5<BR>
<H4>20X SSC:</H4>
3 M NaCl, 0.3 M sodium citrate, adjust pH to 7.0<BR>
<BR>
<H4>Prehybridization Solution:</H4>
6X SSC, 0.5% SDS, 5X Denhardt's solution, 100 mg/ml denatured, sheared, salmon sperm DNA or yeast tRNA <BR>
<BR>
<H4>Prehybridization solution:</H4>
should be made fresh and added at a volume equal to 0.2 ml/cm2 of filter membrane.  Also, salmon sperm DNA (Sigma) is a good inexpensive blocking agent but causes reduced signal and high background with some probes (such as telomere probes).  In such cases we substitute yeast tRNA (Boehringer Mannheim) which is considerably more expensive.<BR>
<BR>
<H4>50X Denhardt's: </H4>
1% Ficol 400, 1% polyvinylpyrrolidone, 1% BSA (fraction V)<BR>
<BR>
<H4>Hexamer labeling reaction:</H4>
4X reaction stock* 15 &#181;l<BR>
BSA (10 mg/ml)						2.4 &#181;l<BR>
Klenow fragment of DNA polymerase I(NEB)		1.0 &#181;l<BR>
alpha-32P-dCTP (3000 Ci/mM)				6.0 &#181;l<BR>
alpha-32P-dATP (3000 Ci/mM)				6.0 &#181;l<BR>
<BR>
*4X reaction stock:  200 mM Tris-Cl pH 8.0, 20 mM MgCl2, 20 mM DTT, 80 uM dNTP, 800 mM HEPES (not pH'd), 27 units Hexamer (Pharmacia)<BR>
<BR>
To make the HEPES  stock dissolve 4.766 g of the free acid (Sigma) in 10 ml of ddH20 and store as 1 ml aliquots at -20 C. To make the hexamer stock dissolve 50 A260 units of pd(N)6 (Pharmacia) in 556 ul ddH20 and store at -20 C.<BR>
To make the 1 mM dNTP stock mix equal volumes of 1 mM dGTP and TTP (or which ever two you haven't used as hot nucleotides). <BR>
<BR>
<H4>Hexamer Labeling Stop Solution: </H4>
200 mM EDTA pH 7.2, 1% SDS, 40 mg/ml Blue Dextran, 0.1 mg/ml Bromphenol Blue<BR>
<BR>
<H4>Sephadex: </H4>
Add 100g of dry sephadex G-50 powder to 1 liter of G-50 buffer(10 mM Tris-Cl pH 7.5, 2 mM NaCl, 0.5 mM EDTA, 0.1% SDS).  Swirl, cover and let stand for an hour.  Pour off excess buffer and replace with fresh buffer, mix and let settle again.  Repeat the wash and autoclave.<BR>
<BR>
<H4>Hybridization Mix: </H4>
6X SSC, 10 mM EDTA, probe DNA, 5X Denhardt's solution, 0.5% SDS, 100 mg/ml salmon sperm DNA<BR>Use 0.05 ml/square cm of membrane.  See notes on salmon sperm DNA under pre hybridization. <BR>
<BR>
<H4>Suppliers:</H4>
The Nottingham Arabidopsis Stock Centre<BR>
Dr. Mary Anderson,<BR>
NASC,<BR>
Dept. of Life Science,<BR>
University of Nottingham,<BR>
University Park, <BR>
Nottingham NG7 2RD, <BR>
U.K.<BR>
Tel +44 115  9791216<BR>
email <A HREF="mailto:arabidopsis@nottingham.ac.uk">arabidopsis@nottingham.ac.uk</A><BR>
<A HREF="http://nasc.nott.ac.uk/home.html">http://nasc.nott.ac.uk</A><br>
<BR>
Arabidopsis Biological Resource Center<BR>
The Ohio State University,<BR>
1735 Neil Avenue, <BR>
309 Botany &amp; Zoology Bldg,<BR>
Columbus Ohio 43210, <BR>
USA<BR> 
Tel 614-292-9371; <BR>
FAX 614-292-0603; <BR>
email <A HREF="mailto:arabidopsis+@osu.edu">arabidopsis+@osu.edu</A><BR>
<A HREF="http://aims.cps.msu.edu/aims/">http://aims.cps.msu.edu/aims/</A><BR>
<BR>
Lehle Seeds<BR>
P.O. Box 2366, <BR>
Round Rock TX, <BR>
7680-2366, <BR>
USA. <BR>
Tel: 1-800-881-3945,<BR>
FAX: 1-572-388-3974,<BR>
World Wide Web site <A HREF="http://www.arabidopsis.com">http://www.arabidopsis.com</A><BR>
<BR>
Small Parts Inc.<BR>
13980 N.W. 58th Court, <BR>
P.O. Box 4650, <BR>
Miami Lakes Fl 33014-0650;<BR>
Tel 305-557-8222;
FAX 1-800-423-9009<BR>
<BR>
Grace-Sierra Horticultural Products Company<BR>
1001 Yosemite<BR>
Dr., Milpitas, <BR>
CA 95035<BR>
<H4>I.  Growing plant material</H4>
	Whenever possible choose a reliable source for seed and document strain
and batch numbers.  Often samples which are called by similar names will
be different genetically.  For instance, there are at least 5 different
strains of Arabidopsis thaliana commonly called Landsberg and each one gives
distinctive rDNA polymorphisms (unpublished data).  It is therefore imperative
to document the history of the strains you use.  Our lab uses The Nottingham
Arabidopsis Stock Centre and The Ohio State University Arabidopsis Biological
Resource Center.  Both these resources provide material (seeds, DNA stocks
etc...) free of charge to academic researchers.  Lehle Seeds is a good commercial supplier.<BR>
<BR>
	Optimally plants should be grown in sterile conditions [<A HREF="#6">6</A>, <A HREF="#7">7</A>].  Sterile
growing conditions will prevent errors due to contamination by bacterial,
fungal and insect DNA.  To grow sterile plants, add the desired amount of
seed (~10 &#181;L of seeds/plate to be planted) to an eppendorf tube and fill
with 95% ethanol.  Incubate at room temp for 2-3 minutes.  Remove the ethanol
and add 700 &#181;L of ddH20 and 700 &#181;L of commercial bleach (Chlorox) to the
tube.  Adding 0.01% Triton X-100 makes the seeds much easier to handle but
it is not necessary for sterilization.  Incubate at room temperature for
20-30 minutes with occasional mixing.  From this point on all work should
be done in a laminar flow hood.  In the hood remove the chlorox solution
and rinse the seeds four times with 1 ml  of sterile water, leaving the
final ml in the tube with the seeds.  Using a wide bore pipette tip (these
can be made manually by cutting the top off a 200 &#181;L tip or they can be
purchased) take up 25 &#181;L of seeds and sow the seeds onto GM plates at a
density of about 1-2 seeds/0.5 cm2 or 150 seeds/plate; do this by touching
the tip of the pipette to the media repeatedly.  Generally, it is a good
idea to sow approximatly 5 plates per stain/line to be examined.  This will
give enough agarose-embedded DNA for more than 100 restriction digests.
 If seeds are limiting one can make a single agarose plug from as few as
six two-week old plants.  Also, don't try to sow seeds of too many strains/lines
in one day as it is difficult to process large numbers later in the procedure.
 We usually plant 1-4 lines in a day.  Wrap the plates of sown seeds with
parafilm and poke holes in the parafilm with a needle to allow gas exchange.
 Incubate the plates in a growth chamber (16 hr 24&#176;C days 8 hr 21&#176;C nights)
for 2 weeks.  Any plates with contaminating fungus or bacteria should be
discarded.  If contamination is a reccurring problem, seeds can be sown
on GM plates containing carbenicillin. However this could conceivably change
your experimental results.<BR>
<BR>
	It is possible to harvest DNA from individual plants when the experiment
demands, for example when sib analysis is necessary.  In this case plant
seeds, sterilized as described above, in 3 inch pots which contain autoclaved
potting mixture (rediearth:vermiculite, 60:40) that has been saturated with
sterile water.  Place these pots in a plastic flat and cover the flat with
a clear plastic dome lid.  Place the flat in a short day (8 hr. light, 24&#176;C) growth chamber.  Keep the flat covered until the plants germinate and
then crack the lid for 2-3 days before removing it to acclimate the plants
to a lower humidity.  The soil should be allowed to dry (but not too dry!)
before watering.  Also the plants tend to grow better if they are given
15:16:17 Peters fertilizer (Grace-Sierra Horticulture Products Co.) at 125-150
ppm nitrogen once a week or so.  Once the rosettes are large and bushy (6-9
weeks depending on the ecotype) harvest the leaves giving preference to
the smallest and youngest which will yield better protoplasts.  This method
will yield enough protoplasts to make 1-2 agarose plugs.  However, the quality
of the DNA is somewhat poorer than that derived from young tissue grown
on GM plates such that the background following Southern blotting and hybridization
is higher.     <BR>
<H4>II. Harvesting plant material</H4>
	Generation of plant protoplasts is achieved by the method of Doelling and
Pikaard with slight modification [<A HREF="#6">6</A>, <A HREF="#7">7</A>].  After the plants have grown for
two weeks take them to the hood and pull them from the agar with forceps.
 Combine the plants from 2 plates in the lid of one of the petri dishes
and lightly mince the plants into small pieces using surgical scissors.
 Immediately following mincing place the plant material in a clean petri
dish and add 15 ml (or enough to cover the plant material) 0.5M mannitol
(in water) to plasmolyze the cells for 30 min. - 1 hr.  After plasmolysis
remove the mannitol using a pipette  and add15 ml of Protoplasting Solution.
 Protoplasting solution should be made immediately prior to use and filter
sterilized using a 0.22 or 0.45 um vacuum filtration unit.  Wrap the petri
dish with parafilm, cover with aluminum foil, and place on a rotary shaker
with gentle agitation ( 48 rpm on our Bellco orbital shaker).  Allow the
digestion to proceed overnight.  <BR>
	Separate liberated protoplasts from unwanted plant material by pouring
the digestion mixture through successively finer stainless steel wire mesh.
 We use 1.13 mm, 230 um, and 38 um mesh steps (mesh was purchased from Small
Parts Inc.).  Our filter units were constructed by gluing mesh disks (cut
from sheets supplied by the manufacturer) to the bottoms of 50 ml plastic
conical tubes with the bottom cone cut off.  Minimizing physical trauma
to the protoplasts is critical, therefore avoid pipetting whenever possible.
 When using the smallest mesh it is necessary to pre-wet the filter from
below by placing the filter onto a drop of solution.  This overcomes surface
tension and removes trapped air bubbles so that the protoplasts in solution
can flow through.  Transfer the final filtrate to a clean 50 ml conical
tube and centrifuge at 26 X g for 10 minutes in a swinging bucket rotor
at room temperature.  Remove the supernatant (usually very green)  and resuspend
the pellet containing the protoplasts in 14 ml of 0.34 M mannitol, 0.07
M CaCl2 and transfer to a 15 ml conical tube.  Centrifuge the protoplasts
again at 26 X g for 10 minutes, remove the supernatant and resuspend the
protoplasts as before.  Count the protoplasts on a hemacytometer and adjust
the volume to give the desired protoplast/ml.  For our studies of the chromosomal
arrangement of rRNA genes, a high copy gene, we use 1.8 X 107 protoplasts/ml.
 Anand and Southern [<A HREF="#8">8</A>] recommend a concentration of 1 X 107 protoplasts/ml
for high copy number genes and 3 X 107 for low or single copy number genes.
 Concentrations as high as 5 x 107 can be used but band sharpness and resolution
suffer.  As a general rule lower protoplast concentrations yield better
resolution but the lower the DNA content, the harder it is to detect rare
sequences and autoradiography time increases.  Once you have determined
your desired protoplast concentration adjust the volume of your protoplast
suspension to accommodate this concentration.  To increase concentration
spin the protoplast suspension at 25 X g for 10 minutes and remove the appropriate
amount of supernatant and resuspend the protoplasts in the remaining supernatant.
 To decrease concentration add the appropriate amount of 0.34 M mannitol,
0.07 CaCl2.  <BR>
<H4>III.  Preparing agarose embedded DNA</H4>
	Agarose embedded DNA is prepared according to Anand and Southerns's protocol
for cultured animal cells with minor modifications [<A HREF="#8">8</A>].  Following resuspension
of the protoplasts at the correct concentration, prepare an equal volume
of 2% lowmelt agarose solution (we use Fischer DNA grade low-melt agarose)
in 50 mM TBE (50 mM Tris, 50 mM Borate, 1 mM EDTA, 0.5X TBE is 45 mM with
respect to the Tris and Borate) and heat in a boiling water bath until the
agarose is molten.  Equilibrate the temperature of both the protoplasts
and the agarose to 37 C in a water bath (about 5-10 minutes) then mix the
agarose and the protoplast solution together.  Immediately pipette the mixture
into plug molds.  We use BioRad plug molds with dimensions of 0.9 cm (width)
X 2 cm (height) X 0.1 cm (thickness).  The width and thickness of the plugs
should match the width and thickness of the comb used to form the wells
of the CHEF gel.  Place the filled plug molds on ice to quickly harden the
agarose before the protoplasts cand begin to settle towards the bottom of
the molds.  Once hardened (5-10 minutes) removed the agarose plugs from
the molds with a small spatula into a 50 ml conical tube containing NDS
with 1 mg/ml pronase (pronase is a much cheaper substitute for proteinase
K; we buy ours from Sigma).  We usually put 10-40 plugs into 40 ml of NDS
per 50 ml conical tube.  Place the tube containing the plugs in a 55 C water
bath and let it incubate overnight.  Replace the NDS/pronase with fresh
solution and let it incubate overnight again.  Replace the NDS/pronase with
fresh NDS WITHOUT pronase and incubate at 55&#176;C  overnight again.  Replace
the NDS with fresh NDS and store at 4&#176;C.  At this point the plugs should
have lost all green color.  Plugs made from young tissue grown on GM media
should be a translucent white/tan color whereas plugs made from older soil-grown
tissues are often a more opaque white/tan color.  These plugs in NDS are
stable for years.  We have used 3 year old plugs to resolve pieces of DNA
as large as 3 Mb with no appreciable differences in DNA quality as compared
to fresh plugs.  <BR>
	
<H4>IV.   Restriction digests of agarose embedded DNA</H4>
	Agarose-embedded DNA is digested with restriction endonucleases according
to the methods of Van Daelen and Zable [<A HREF="#9">9</A>].  Place a piece of plug slightly
smaller than the well of a CHEF gel (we use plugs cut into thirds with final
dimensions of ~ 0.9 cm X 0.6 cm X 0.1 cm)  in a 15 ml conical tube containing
10 ml of T10E10 with 2 mM PMSF (Sigma) and allow the plug to equilibrate
1 hour (to overnight) at 4&#176;C (this equilibration is done at 4&#176;C because
PMSF is fairly unstable at room temperature in aqueous solution).  Replace
the solution with fresh T10E10 WITHOUT PMSF and equilibrate for 30 min.
at room temperature.  Repeat the T10E10 washes 3 times.  Place the sample
in a microcentrafuge tube containing 1 ml of the desired 1X restriction
enzyme buffer without the DTT and BSA and let it equilibrate for 1 hour.
 We find that using 2 ml flat bottomed microcentrafuge tubes reduces breakage
of the agarose plug.  Repeat the buffer wash once more.  Remove the buffer
wash and add an appropriate volume of reaction solution.  We use 250 &#181;L
of reaction solution for our sample size (larger sample may require more
solution to cover the plugs).  Incubate the agarose embedded DNA with the
restriction enzyme at the appropriate temperature for 8 hours (or overnight).
 Keep in mind that when using restriction enzymes that require high temperatures,
such as BstE II, the samples are embedded in low-melt agarose.  Therefore,
 reactions should not exceed 55&#176;C.  After the initial incubation, add 50
additional units of restriction enzyme  and incubate for an additional 6
hours.  This results in 100 units of enzyme used per reaction which, based
on our experience performing deliberate partial digestion series, should
be a vast excess of what is necessary to achieve complete digestion [<A HREF="#10">10</A>].
 It should also be noted that when choosing a restriction enzyme, close
attention should be paid to its sensitivity to methylation, particularly
CpG and CpNpG methylation [<A HREF="#11">11</A>].<BR>
<BR>
<H4>V.  Running the CHEF gel</H4>
	Preparation and running of the CHEF gel is based on the manufacturer's
recommendations [<A HREF="#12">12</A>, <A HREF="#13">13</A>, <A HREF="#14">14</A>, <A HREF="#15">15</A>].  To pour the gels, make a fresh agarose solution
for each gel.  This avoids changes in agarose concentration due to multiple
heatings and subsequent buffer evaporation.  We use a 13 cm X 14 cm casting
tray and we pour 110 ml gels.  The casting tray is leveled each time with
a bubble level to assure constant gel thickness.  The well combs are placed
at a distance of 0.8 cm from the top edge of the gel, with a 0.1 cm gap
between the bottom of the comb and the casting tray.  We use 1% pulsed-field
certified DNA grade agarose (BioRad) for 10 kbp - 1.5 Mbp separations and
0.8% chromosomal grade agarose (BioRad) for 1.5 - 3.0 Mbp separations. 
All gels are made in 50 mM TBE.  The agarose solution is heated in a microwave
on medium power with frequent mixing, to avoid boiling over, and allowed
to cool until it is comfortable to handle with bare hands. Once cooled sufficiently,
the agarose is poured into the casting tray.  The gel is allowed to harden
completely before loading.<BR>
<BR>
	After the samples are digested with the appropriate restriction enzyme(s)
remove the reaction solution and replace it with 500 &#181;l STOP solution
and allow the samples to equilibrate for 5 - 10 minutes.  Pour the sample
into a plastic petri dish, and using a plastic micro-spatula, scoop the
plug up and place it into a well in the gel (using a 1 ml pipette tip to
guide the plug helps).  Useful DNA size standards include Saccharomyces
cerevisiae chromosomes (NEB) for 100 kpb - 1.5 Mbp separations and Hansenula
wingei chromosomes (BioRad) for 1 - 3 Mbp separations.  Once the sample
and controls are loaded, seal each well with molten low melt agarose of
the same concentration as the gel using a syringe fitted with a medium gauge
needle.  Be careful to avoid introducing air bubbles.  Allow the seals to
harden for 5 minutes.<BR>
<BR>
	We run our CHEF apparatus in a 4&#176;C cold room with no further cooling.
 The buffer (2 liters of 50 mM TBE) is pre-equilibrated to cold-room temperature
and is circulated in the running tank at a rate of 1 liter per minute. 
Under these conditions the mean temperature of the gel is ~14&#176;C.
 The parameters for switch time, voltage, and run length will, of course,
vary with the size of DNA fragments being separated.  We find that a good
starting program for 10 kbp - 1.5 Mbp separation is as follows:<BR>
<BR>
Stage I:<BR>
	initial switch time		60 seconds<BR>
	final switch time		60 seconds<BR>
	run time				14 hours<BR>
	voltage				200 V<BR>
Stage II:<BR>
	Initial switch time		90 seconds<BR>
	final switch time		90 seconds<BR>
	run time 				8 hours<BR>
	voltage				200 V<BR>
 <BR>
For 1.5 - 3.0 Mbp separations a good starting program is:<BR>
<BR>
Stage I:<BR>
	initial switch time		1200 seconds<BR>
	final switch time		1800 seconds<BR>
	run time				30 hours<BR>
	voltage				67 V<BR>
Stage II:<BR>
	Initial switch time		90 seconds<BR>
	final switch time		180 seconds<BR>
	run time 				11 hours<BR>
	voltage				200 V<BR>
<H4>VI.  Southern analysis of CHEF gels</H4>
	After the CHEF gel run is complete, remove the gel from the running tank
and place it in a deep platic tray containing 500 ml 50 mM TBE containing
1.5 ug/ml ethidium bromide (this solution can be reused several times).
  Incubate the gel in the staining solution for 30 minutes at room temperature
while gently agitating on a rotary shaker.  Following staining, visualize
the DNA in the gel using a short wavelength UV transilluminator.  Record
a photographic image of the gel for your records.  Often for gels resolving
large pieces of DNA, bands will not be visible with EtBr.  <BR>
	<BR>
Because of the large size of the DNA fragments, it is necessary to nick
them to achieve efficient transfer [<A HREF="#16">16</A>].  We use a BioRad GS Gene Linker
UV Chamber on setting 'nic' , which delivers 60 mJoule of 254 nm UV light.
 After nicking, denature the gel in 500 ml of fresh Denaturing Solution
for 30 minutes at room temperature with gentle agitation.  Following denaturation,
neutralize the gel in 500 ml of Neutralizing Solution for 30 minutes at
room temperature with gentle agitation (the gel has a tendency to float
in this high salt solution so agitation should be made sufficient to assure
submersion).  Following neutralization, proceed with standard Southern transfer
using 10X SSC as a transfer buffer.  We use Nitrobind nitrocellulose as
a transfer membrane (MSI) which we prewet with 6X SSC before blotting. 
Following Southern transfer, rinse the membrane briefly with 6X SSC to remove
any agarose and crosslink the DNA to the membrane using UV light.  Again,
we use the BioRad Gene Linker on program 'C3' which delivers 150 mJoule
of 254 nm UV light.  At this point the blot can be stored in 6X SSC in a
sealed plastic bag or it can be probed immediately.  We never allow our
blots to dry.<BR>
<BR>
	To probe the blot, place it in a heat-sealable plastic bag containing Prehybridization
Solution (avoiding bubbles) and incubate in a 65&#176;C water bath overnight
(incubation times as short as 4 hours are sufficient).  While the gel is
pre-hybing label the probe to be used.  We use the hexamer labeling protocol
of Feinberg and Vogelstein [<A HREF="#17">17</A>] but other methods could be used.  For hexamer
labeling dilute the template (probe) DNA to 100 ng/30 &#181;l.  For template
DNA we use plasmid fragments of the appropriate sequence  that have been
isolated on agarose gels and purified using the Gene Clean kit (BIO101).
 Also, template DNA should be at least 400 bp for hexamer labeling.  Boil
30 &#181;l of the template DNA for 5 minutes then immediately cool the sample
on ice.  Once the sample has cooled add the remaining Hexamer Labeling Reaction
ingredients.  Mix the reaction components well and spin briefly in a microcentrafuge
to remove drops from the side of the tube, and incubate overnight.  Note
that both the membrane prehyb and the hexamer labeling reaction are incubating
during the same night.  The following day add 66 &#181;l of Hexamer Stop Solution
to the labeling reaction.  Plug a 5 inch glass Pasteur pipette with glass
wool at the point were the aperture begins to narrow.  Fill the pipette
with Sephadex G-50 in G-50 buffer until the bed of resin reaches 1 cm below
the top the column.  When the meniscus of the G-50 buffer reaches the bed
of resin carefully pipette the stopped reaction onto the column.  Once the
sample has entered the column add more G-50 buffer.  Collect the fractions
that contain the Blue Dextran (faster migrating of the two dyes in the STOP
solution).  These fractions will contain the labeled DNA.  The unincorporated
nucleotides will migrate more slowly, usually with the Bromphenol blue (slower)
dye.  Weigh a microcentrafuge tube, transfer the labeled probe to the tube
and weigh it again to estimate the volume.  Remove 5 ul of the probe and
count it in a scintillation counter to determine the specific activity of
the probe.  Heat the remaining probe in a boiling water bath to denature
the DNA (be careful to secure the top of the tube with a tube lock or use
a screw cap tube) for 5 minutes.  Immediately chill the probe on ice for
10 min.   Add the cooled probe to the Hybridization Mix.  Remove the prehybridzation
solution from the blot, replace with Hybridization Mix and probe, and reseal
the sealable bag (being careful to avoid bubbles).  Place the bag in a shaking
water bath, secure it below the surface of the water, and incubate with
gentle agitation at 67&#176;C overnight.<BR>
<BR>
	Following hybridization, move the blot from the bag to a plastic container
with a sealable lid or to a fresh sealable bag.  Wash the blot once with
300 ml 2X SSC, 0.5% SDS at room temperature for 15 minutes, once with 300
ml 2X SSC, 0.1% SDS at room temperature  for 15 minutes, and once with 300
ml 0.1X SSC, 0.5% SDS at 68&#176;C for 2 hours.  Rinse the blot briefly
with 50 ml of 0.1X SSC, 0.5% SDS and wrap the damp blot in plastic wrap.
 Tape the edges of the plastic wrap to prevent any leakage and visualize
the blot by autoradiography or phosporimaging.<BR>
<BR>
	Stripping and reusing of CHEF gel blots has proved difficult and inconsistent
in our hands so we don't do it.<BR>
		
<H4>VII.  Two Dimensional CHEF analysis</H4>
	For 2D analysis, run the first dimension as described above but before
staining the gel with EtBr use a straight edge and a scalpel to slice half
(lengthwise) of the lane of interest out of the gel.  Using only half of
the first dimension lane for the second dimension serves two purposes. 
First, it leaves half of the lane on the original gel which can then be
processed and used as a control for the second dimension.  Second, it makes
the bands (or spots) in the second dimension much tighter and easier to
differentiate.  Place the first dimension gel slice in a small container
with 100 ml of the 1X restriction enzyme buffer to be used in the second
dimension, amended to contain 2 mM spermidine, 1 mM DTT, and 2 &#181;g/ml BSA,
and incubate at room temperature for 2 hours [<A HREF="#18">18</A>, <A HREF="#19">19</A>].  Repeat the wash
once more for 2 hours and then place the gel slice in a sealable bag containing
10 ml of the appropriate 1X restriction enzyme buffer amended to contain
2 mM spermidine, 1 mM DTT, 2 mg/ml BSA, and 1000 units of restriction enzyme.
 Seal the bag, avoiding bubbles,  and incubate at the appropriate temperature
8 hours to overnight.  Following the initial incubation add 1000 additional
units of restriction enzyme and incubate for an additional 6 hours.  After
digestion remove the gel slice from the bag and place it in a tray with
200 ml of 50 mM TBE and incubate at room temperature with gentle agitation
for 1 hour.  Repeat the wash once more for 1 hour.  Following the final
wash, lay the gel strip in the casting tray perpendicular to the direction
of electrophoresis and pour the gel as before.  It is useful to use the
well combs to hold the gel slice in place while initially pouring the gel
but be sure to remove them before the gel sets.  Once the gel has hardened
use the appropriate electrophoresis regime as described above.  It is also
sometimes useful to run a CHEF gel in the first dimension and a conventional
gel in the second dimension if products resulting from the second digestion
are small.  To see real-life examples of gels resulting from these protocols,
see the recent papers by Gregory Copenhaver [<A HREF="#3">3</A>, <A HREF="#4">4</A>, <A HREF="#5">5</A>].<BR>
<BR>
<H3>References</H3>
<A NAME="1">
1.  Schwartz, D.C., Saffran, W. Welsh, J., Haas, R., Goldenberg, M., and
Cantor, C.R. (1982)  Cold Spring Harbor Symp. Quant. Biol. 47, 198-195<BR>
<BR>
<A NAME="2">
2.  Chu, G., Vollrath, D., and Davis, R.W. (1986) Separation of large DNA
molecules by contour-clamped homogeneous electric fields.  Science, 234,
1582-1585<BR>
<BR>
<A NAME="3">
3.  Copenhaver, Gregory P., Jed H. Doelling, J. Scott Gens and Craig S.
Pikaard (1995). Use of RFLPs larger than 100 kbp to map the position and
internal organization of the nucleolus organizer region on chromosome 2
in Arabidopsis thaliana.  The Plant Journal 7: 273-286. <BR>
<BR>
<A NAME="4">
4.  Copenhaver, Gregory P. and Craig S. Pikaard. RFLP and physical mapping
with an rDNA-specific endonuclease reveals that nucleolus organizer regions
of Arabidopsis thaliana adjoin the telomeres on chromosomes 2 and 4. The
Plant Journal 9, 259-272<BR>
<BR>
<A NAME="5">
5.  Copenhaver, Gregory P. and Craig S. Pikaard (1996). Two-dimensional
RFLP analyses reveal megabase-sized clusters of rRNA gene variants in Arabidopsis
thaliana, suggesting local spreading of variants as the mode for homogenization
during concerted evolution. The Plant Journal 9, 273-282<BR>
<BR>
<A NAME="6">
6.  Doelling, J.H. and Pikaard, C.S.  (1993).  Transient expression in Arabidopsis
thaliana protoplasts derived from rapidly established cell suspension cultures.
 Plant Cell Rep.  12, 241-244.<BR>
<BR>
<A NAME="7">
7.  Doelling, J.H., Gaudino, R.J. and Pikaard, C.S.  (1993).  Functional
analysis of Arabidopsis rRNA gene and spacer promoters in vivo and by transient
expression.  Proc. Natl. Acad. Sci. USA, 90, 7528-7532.<BR>
<BR>
<A NAME="8">
8.  Anand, R. and Southern, E, (1990) Pulsed field gel electrophoresis.
 In Gel Electrophoresis of Nucleic Acids a Practical Approach, 2nd Edition,
(Rickwood, D. and Hames, B.D., ed.) IRL press, Oxford  p. 101-123<BR>
<BR>
<A NAME="9">
9.  Van Daelen, R., and Zabel, P. (1992)  Pulsed-Field Gel Electrophoresis,
in Plant Genomes:  Methods for Genetic and Physical Mapping, (Beckmann,
J.S., and Osborn, T.C., ed.) Dordecht: Kluwer Academic Publishers<BR>
<BR>
<A NAME="10">
10.  New England Biolabs Corporation Bulletin: The NEB Transcritpt.  Volume
3. No. 1  1991<BR>
<BR>
<A NAME="11">
11.  McClelland, M., Nelson, M., Raschke, E. (1994)  Effect of site-specific
modification on restriction endonucleases and DNA modification methyltransferases.
 Nucleic Acid Res. 22, (No. 17) 3640-3659<BR>
<BR>
<A NAME="12">
12.  CHEF-DR II Pulsed Field Electrophoresis System Instruction Manual and
Application Guide.   (1988) BioRad Corporation<BR>
<BR>
<A NAME="13">
13.  Applications in Pulsed Field Electrophoresis.  BioRad Bulletin 1593<BR>
<BR>
<A NAME="14">
14.  Addendum to Pulse Controller Manual.  BioRad Bulletin 1631 A<BR>
<BR>
<A NAME="15">
15.  Birren, B. (1990)  Preparation and Separation of Mammalian DNA by Pulsed
Field Gel Electrophoresis. BioRad Bulletin 1589<BR>
<BR>
<A NAME="16">
16.  UV Nicking of DNA in Agarose Gels for Enhanced Transfer and Detection
of Megabase Size DNAs. (1991) BioRad Bulletin 1672<BR>
<BR>
<A NAME="17">
17.  Feinberg, A.P. and Vogelstein, B. (1983).  A technique for radiolabeling
DNA restriction endnonuclease fragments to high specific activity.  Anal.
BioChem.  132, 6-13.<BR>
<BR>
<A NAME="18">
18.  Warburton, P.E., Wevrick, R., Mahtani, M.M. and Willard, H.F. (1992).
 Pulsed-field and two dimensional gel electrophoresis of long arrays of
tandemly repeated DNA.  In Methods in Molecular Biology, Vol 12:  Pulsed-field
Gel Electrophoresis (Burmeister, M. and Ulanovsky, L. ed.) The Humana Press
Inc. Totowa, NJ p. 299-317<BR>
<BR>
<A NAME="19">
19.  Bautsch, W. (1992) Bacterial Genome Mapping by Two-Dimensional Pulsed-Field
Gel Electrophoresis (2D-PFGE), in Methods in Molecular Biology, Vol. 12:
 Pulsed-Field Gel Electrophoresis, (Burmeister, M., Ulanovsky, L., ed.)
The Humana Press Inc. Totowa, NJ p. 185-201<BR>
<BR>
</BODY>
</HTML>
