<img src = "banner1.gif"><hr>
<H2>Sample Output from University of Minnesota's Sequence Analysis Project</H2>
(A collaboration between the University of Minnesota 
        and Michigan State University, funded by NSF)
<hr>
This page shows only a small portion of the output available from the University of Minnesota's database.  For further information regarding this work, go to the University of Minnesota's  <A Href="http://lenti.med.umn.edu/">Computational Biology Centers' Home Page</A> OR <A Href = "TC8203.report.html">Back to the Sample TC Report</a>
<hr>
<a name="#bof#"></A><TITLE>Sequence Analysis for 40F11T7</TITLE><H2>Sequence Analysis for 40F11T7</H2>
<P>
<H4>Sequence Origin: Michigan State University Arabidopsis cDNA Sequencing Project</H4>
<P>
<H4>Analysis Origin: University of Minnesota Plant Molecular Informatics Center</H4>
dbEST ID for this sequence: 21241<P>
GenBank Accession Number for this sequence: <a href="http://www3.ncbi.nlm.nih.gov/htbin-post/Entrez/query?db=n&form=6&uid=T04426&dopt=g">T04426</a><P>
<H3>Table of Contents</H3>
<dl>
<dd><a href="#ActualSequence">Processed Sequence after Filtering and/or Trimming for Quality</A>
<P>
<dd><a href="#AVPictures">Visualization of BLAST Hits</A>
<P>
<dd><a href="#PercentN">Percentage of Unknown (N) bases in Sequence</A>
<P>
<dd><a href="#LC_Reg">Low Complexity Regions</A>
<P>
<dd><a href="#BlastxSummary">Blastx Results Summary</A>
<P>
<dd><a href="#BlastxHits">Blastx Hits Ordered by P-value</A>
<P>

<P>

<P>
</dl>
<H3>Plus Strand Processed</H3>
<HR>
<A NAME="ActualSequence"><H2>DNA Sequence after Vector Filtering and End Trimming for Quality</H2></A>
<P>
(Each Full Line is 40 bp Long)
<P>
<PRE>
CTCCATTTCTCTATCTTTCTCTCTCGCTGCTTCTCGAATC
TNCTGTATCATCTTCTTCTTCTTCAAGTGAAAAATGGCCG
ATGGTGAGGATATTCAGCCACTTGTCTGTGACAATGGAAC
TGGAATGGTGAAGGCTGGTTTTGCTGGTGATGATGCCCCG
AGAGCAGTGTTCCCAAGTATTGTTGGTCGTCCTAGGCACA
CTGGTGTCATGGTTGGTATGGGTCAGAAAGATGCTTACGT
TGGTGATGAAGCTCAGTCCAAGAGAGGTATCCTCACTCTG
AAGTATCCAATCGAACATGGTATTGTAAGTAACNTGGGAT
GACATGGAAAAGATATGGCATCACACTTTCTACAACGAGC
TTCGTGTTGCCCCTGAGGGAGCACCCAAGTTCTACTCACA
</PRE>
<P>Sequence Length: 400 <P>
<HR>
<A NAME="PercentN"><H2>Percentage of Unknown (N) bases in Sequence</H2></A>
400 original bp (0.5% unknown bases), reduced to 400 bp (0.5% unknown bases).<HR>
<A NAME="AVPictures"><H2>Visualization of Blastx Hits from AlignmentViewer Tool</H2></A>
<img src="40F11T7-IAAI000.seq.blastx.gif">
</a>
<p>
<HR>
<H2>Visualization of Blastp Hits (where Low Complexity Regions have been removed) from AlignmentViewer Tool</H2>
Hits in a frame will be displayed if and only if there were low complexity regions within the BLASTX hits for any given frame. Otherwise nothing will be displayed. Lack of hits may be because the low complexity regions were only found in the incorrect frames. In the future the visualization will be improved to eliminate this and improve clarity.<p>
<img src="40F11T7-IAAI000.seq.blastp.gif">
</a>
<p>
<HR>
<A NAME="LC_Reg"><H2>Low Complexity Regions from XNU program</H2></A>
<H3>Low complexity regions found for translated frame +1 :</H3>
None<P>
<H3>Low complexity regions found for translated frame +2 :</H3>
Translated Protein: start = 16, end = 22<P>
Corresponding DNA : start = 47, end = 67<P>
<P>
<H3>Low complexity regions found for translated frame +3 :</H3>
Translated Protein: start = 45, end = 75<P>
Corresponding DNA : start = 135, end = 227<P>
Translated Protein: start = 2, end = 23<P>
Corresponding DNA : start = 6, end = 71<P>
<P>
<HR>
<H2>The function of this Sequence can possibly be determined by the following BLASTX, BLASTN, AND/OR BLASTP hits.</H2>
<HR>
<A NAME="BlastxSummary"><H2>Summary of Blastx Hits on Plus Strand with P-value <= 1.000e-01</H2></A>
<H3>BLAST DB SEARCHED: Non-redundant PIR (RELEASE 44.00) plus GenPept, from GenBank (RELEASE 89.0)
</H3><P>
<H3>DATE SEARCHED: Mon Jul 10 09:11:08 CDT 1995
</H3><P>
<H4>Click on the "goto" in the first column to see more details: e.g. alignment and longer description of hit sequence from databank.</H4>
<PRE>
<B>                          High  </B>
<B>More DB  Acc    Fr Shift? Score P-value  Description</B>
<B>---- --  ---    -- ------ ----- -------  -----------</B>
<a href="#blx_S31933">goto</a> pir S31933    2     Y      380   1.2e-35   actin - common tobaccoPEP:pir|S35257|S35257 acti... 
<a href="#blx_JA0066">goto</a> pir JA0066    2     Y      379   1.5e-35   actin - Arabidopsis thaliana PEP:gi|166582|gp|M2 ... 
<a href="#blx_S20098">goto</a> pir S20098    2     Y      378   2.0e-35   actin - potato PEP:gi|21544|gp|X55751|STPOAC97_1 ... 
<a href="#blx_S20094">goto</a> pir S20094    2     Y      377   2.7e-35   actin - potato PEP:gi|21536|gp|X55749|STPOAC58_1 ... 
<a href="#blx_S10020">goto</a> pir S10020    2     Y      376   3.5e-35   actin 1 - rice                                    
<a href="#blx_S47090">goto</a> pir S47090    2     Y      376   3.5e-35   actin - sorghum PEP:gi|499012|gp|X79378|SVSOAC1_1... 
<a href="#blx_20322">goto</a>  gi  20322    2     Y      376   3.5e-35   Oryza sativa RAc1 mRNA for actin. [Oryza sativa]  
<a href="#blx_295885">goto</a>  gi 295885    2     Y      376   3.5e-35   actin [Oryza sativa]                              
<a href="#blx_S07002">goto</a> pir S07002    2     Y      376   3.8e-35   actin 1 - carrot                                  
<a href="#blx_S20093">goto</a> pir S20093    2     Y      374   6.1e-35   actin - potato PEP:gi|21534|gp|X55752|STPOAC101_1... 
</PRE>
<HR>
<HR>
<A NAME="BlastxHits"><H2>BLASTX Hits by P-value</H2></A>
<P>
<ul><li>Highest P-value cutoff for reporting hits = 0.100

<P>
</ul>
<a NAME="blx_S31933"><b>Hit Accession Number</b></a> S31933</a> of PIR database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
actin - common tobaccoPEP_:pir|S35257|S35257 actin -
 common tobaccoPEP_:gi|22609|gp|X63603|NTACTI_1 actin [Nicotiana 
 tabacum] </dd>
<P>
<dt>Length of Hit Sequence = 377<p>
<dt>P-value = 1.200000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 380  Frame = 2   Identities = 100%   Positives = 100%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA<p>
      1 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGILTLKYPIEHGIVSN<p>
      1 QSKRGILTLKYPIEHGIVSN 80<p><p>
</PRE>
<PRE>
  Score: 135  Frame = 3   Identities = 91%   Positives = 91%<p>
    315 WDDMEKIWHHTFYNELRVAPEGAP 386<p>
        WDDMEKIWHHTFYNELRVAPE  P<p>
      1 WDDMEKIWHHTFYNELRVAPEEHP 104<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<PRE>
____________________________

</PRE>
<a NAME="blx_JA0066"><b>Hit Accession Number</b></a> JA0066</a> of PIR database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
actin - Arabidopsis
 thalianaPEP_:gi|166582|gp|M20016|ATHACT1A_1 A.thaliana AAc1 gene 
 encoding actin-1, complete cds. [Arabidopsis thaliana] </dd>
<P>
<dt>Length of Hit Sequence = 377<p>
<dt>P-value = 1.500000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 379  Frame = 2   Identities = 98%   Positives = 100%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA<p>
      1 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGILTLKYPIEHGIV+N<p>
      1 QSKRGILTLKYPIEHGIVNN 80<p><p>
</PRE>
<PRE>
  Score: 135  Frame = 3   Identities = 91%   Positives = 91%<p>
    315 WDDMEKIWHHTFYNELRVAPEGAP 386<p>
        WDDMEKIWHHTFYNELRVAPE  P<p>
      1 WDDMEKIWHHTFYNELRVAPEEHP 104<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<PRE>
____________________________

</PRE>
<a NAME="blx_S20098"><b>Hit Accession Number</b></a> S20098</a> of PIR database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
actin - potatoPEP_:gi|21544|gp|X55751|STPOAC97_1 actin
 [Solanum tuberosum] </dd>
<P>
<dt>Length of Hit Sequence = 377<p>
<dt>P-value = 2.000000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 378  Frame = 2   Identities = 98%   Positives = 100%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRH+GVMVGMGQKDAYVGDEA<p>
      1 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHSGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGILTLKYPIEHGIVSN<p>
      1 QSKRGILTLKYPIEHGIVSN 80<p><p>
</PRE>
<PRE>
  Score: 135  Frame = 3   Identities = 91%   Positives = 91%<p>
    315 WDDMEKIWHHTFYNELRVAPEGAP 386<p>
        WDDMEKIWHHTFYNELRVAPE  P<p>
      1 WDDMEKIWHHTFYNELRVAPEEHP 104<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<PRE>
____________________________

</PRE>
<a NAME="blx_S20094"><b>Hit Accession Number</b></a> S20094</a> of PIR database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
actin - potatoPEP_:gi|21536|gp|X55749|STPOAC58_1 actin
 [Solanum tuberosum] </dd>
<P>
<dt>Length of Hit Sequence = 377<p>
<dt>P-value = 2.700000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 377  Frame = 2   Identities = 97%   Positives = 98%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MA+GEDIQPLVCDNGTGMVKAGFAGDDAPRAVFP IVGRPRHTGVMVGMGQKDAYVGDEA<p>
      1 MAEGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPRIVGRPRHTGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGILTLKYPIEHGIVSN<p>
      1 QSKRGILTLKYPIEHGIVSN 80<p><p>
</PRE>
<PRE>
  Score: 135  Frame = 3   Identities = 91%   Positives = 91%<p>
    315 WDDMEKIWHHTFYNELRVAPEGAP 386<p>
        WDDMEKIWHHTFYNELRVAPE  P<p>
      1 WDDMEKIWHHTFYNELRVAPEEHP 104<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<PRE>
____________________________

</PRE>
<a NAME="blx_S10020"><b>Hit Accession Number</b></a> S10020</a> of PIR database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
actin 1 - rice</dd>
<P>
<dt>Length of Hit Sequence = 377<p>
<dt>P-value = 3.500000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 376  Frame = 2   Identities = 98%   Positives = 100%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MAD+EDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA<p>
      1 MADAEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGILTLKYPIEHGIVSN<p>
      1 QSKRGILTLKYPIEHGIVSN 80<p><p>
</PRE>
<PRE>
  Score: 135  Frame = 3   Identities = 91%   Positives = 91%<p>
    315 WDDMEKIWHHTFYNELRVAPEGAP 386<p>
        WDDMEKIWHHTFYNELRVAPE  P<p>
      1 WDDMEKIWHHTFYNELRVAPEEHP 104<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<PRE>
____________________________

</PRE>
<a NAME="blx_S47090"><b>Hit Accession Number</b></a> S47090</a> of PIR database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
actin - sorghumPEP_:gi|499012|gp|X79378|SVSOAC1_1
 actin [Sorghum bicolor] </dd>
<P>
<dt>Length of Hit Sequence = 377<p>
<dt>P-value = 3.500000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 376  Frame = 2   Identities = 98%   Positives = 100%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MAD+EDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA<p>
      1 MADAEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGILTLKYPIEHGIVSN<p>
      1 QSKRGILTLKYPIEHGIVSN 80<p><p>
</PRE>
<PRE>
  Score: 135  Frame = 3   Identities = 91%   Positives = 91%<p>
    315 WDDMEKIWHHTFYNELRVAPEGAP 386<p>
        WDDMEKIWHHTFYNELRVAPE  P<p>
      1 WDDMEKIWHHTFYNELRVAPEEHP 104<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<PRE>
____________________________

</PRE>
<a NAME="blx_20322"><b>Hit Accession Number</b></a> 20322</a> of GenInfo database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
Oryza sativa RAc1 mRNA for actin. [Oryza
 sativa] </dd>
<P>
<dt>Length of Hit Sequence = 377<p>
<dt>P-value = 3.500000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 376  Frame = 2   Identities = 98%   Positives = 100%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MAD+EDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA<p>
      1 MADAEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGILTLKYPIEHGIVSN<p>
      1 QSKRGILTLKYPIEHGIVSN 80<p><p>
</PRE>
<PRE>
  Score: 135  Frame = 3   Identities = 91%   Positives = 91%<p>
    315 WDDMEKIWHHTFYNELRVAPEGAP 386<p>
        WDDMEKIWHHTFYNELRVAPE  P<p>
      1 WDDMEKIWHHTFYNELRVAPEEHP 104<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<PRE>
____________________________

</PRE>
<a NAME="blx_295885"><b>Hit Accession Number</b></a> 295885</a> of GenInfo database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
actin [Oryza sativa]</dd>
<P>
<dt>Length of Hit Sequence = 377<p>
<dt>P-value = 3.500000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 376  Frame = 2   Identities = 98%   Positives = 100%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MAD+EDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA<p>
      1 MADAEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGILTLKYPIEHGIVSN<p>
      1 QSKRGILTLKYPIEHGIVSN 80<p><p>
</PRE>
<PRE>
  Score: 135  Frame = 3   Identities = 91%   Positives = 91%<p>
    315 WDDMEKIWHHTFYNELRVAPEGAP 386<p>
        WDDMEKIWHHTFYNELRVAPE  P<p>
      1 WDDMEKIWHHTFYNELRVAPEEHP 104<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<PRE>
____________________________

</PRE>
<a NAME="blx_S07002"><b>Hit Accession Number</b></a> S07002</a> of PIR database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
actin 1 - carrot</dd>
<P>
<dt>Length of Hit Sequence = 380<p>
<dt>P-value = 3.800000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 376  Frame = 2   Identities = 98%   Positives = 100%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA<p>
      1 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGI+TLKYPIEHGIVSN<p>
      1 QSKRGIITLKYPIEHGIVSN 80<p><p>
</PRE>
<PRE>
  Score: 93  Frame = 3   Identities = 73%   Positives = 84%<p>
    330 KIWHHTFYNELRVAPEGAP 386<p>
        +IWHHTFYNELR +PE  P<p>
      1 RIWHHTFYNELRASPEEHP 103<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<PRE>
____________________________

</PRE>
<a NAME="blx_S20093"><b>Hit Accession Number</b></a> S20093</a> of PIR database
<dl>
<dt>Hit Sequence Description : </dt>
<dd>
actin - potatoPEP_:gi|21534|gp|X55752|STPOAC101_1
 actin [Solanum tuberosum] </dd>
<P>
<dt>Length of Hit Sequence = 377<p>
<dt>P-value = 6.100000e-35<p>

</dl>
HSP Alignments:<p>
<PRE>
  Score: 374  Frame = 2   Identities = 97%   Positives = 100%<p>
     74 MADGEDIQPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 253<p>
        MAD+EDI+PLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA<p>
      1 MADAEDIEPLVCDNGTGMVKAGFAGDDAPRAVFPSIVGRPRHTGVMVGMGQKDAYVGDEA 60<p><p>
</PRE>
<PRE>
    254 QSKRGILTLKYPIEHGIVSN 313<p>
        QSKRGILTLKYPIEHGIVSN<p>
      1 QSKRGILTLKYPIEHGIVSN 80<p><p>
</PRE>
<PRE>
  Score: 133  Frame = 3   Identities = 83%   Positives = 91%<p>
    315 WDDMEKIWHHTFYNELRVAPEGAP 386<p>
        WDDMEKIWHHTFYNELRV+P+  P<p>
      1 WDDMEKIWHHTFYNELRVSPDEHP 104<p><p>
</PRE>
<b>Note:</b> A potential frame shift has occurred amongst these alignments.
<p>
<HR>
<HR>
<A NAME="Citations_and_Credits"><H2>Citations and Credits</H2></A>
The script for creating this results file is the work of 
<A NAME=10 HREF="http://lenti.med.umn.edu/personnel/uofm_personnel.html#lshoop">Libby Shoop</A>.<p>
The BLAST programs whose results are listed here are documented in:<P>
<blockquote>Altschul, Stephen F., Warren Gish, Webb Miller, Eugene W. Myers, and David J. Lipman (1990).  Basic local alignment search tool.  J. Mol. Biol. 215:403-410.</blockquote><p>
<HR>

<A HREF="TC8203.report.html">Back to the Sample TC Report</A>
