<HTML>
<HEAD>
    <TITLE>Searching for tagged male sterile mutants of Arabidopsis</TITLE>
<!-- Changed by: Dr Mike Arnold,  8-May-1996 -->
</HEAD>
<BODY>
<H2>Searching for tagged male sterile mutants of Arabidopsis</H2>
Julie A. Glover (1), Katherina C. Blomer (2), Leigh B. Farrell (3), Abdul M. Chaudhury (4),
&amp; Elizabeth S. Dennis (4*) <BR>
<BR>
1 &amp; 4.  CSIRO Division of Plant Industry,<BR>
GPO Box 1600, <BR>
Canberra ACT 2601 &amp; Cooperative Research Centre for Plant Science,<BR>
PO Box 475,<BR>
Canberra ACT 2601 &amp; Division of Biochemistry and Molecular Biology,<BR>
Australian National University,<BR> 
Canberra ACT 0200, <BR>
Australia.<BR>
<BR>2. Present address: Peter MacCallum Cancer Institute,<BR> 
Research Unit,
<BR> St Andrews Pl, <BR>
East Melbourne VIC 3002,<BR> 
AUSTRALIA<BR>
<BR>3. Present address: Gene Shears Pty. Ltd.<BR> 
PO Box 1238,<BR> 
Neutral Bay NSW 2089,<BR>
Australia.<BR>
<BR>

* Author for correspondence<BR><HR>
<H3>Abstract</H3>
Although many male sterile mutants have been identified in <I>Arabidopsis</I>,
few genes responsible have been cloned. In order to facilitate cloning of
a male sterility gene, 23 of Feldmann's T-DNA-generated, reduced fertility
lines were screened to identify a tagged male sterile mutation. Male sterile
mutants were identified, as well as mutants which were both male and female
sterile. Segregation of the kanamycin marker gene in the progeny of 15 of
these lines was studied. 40% had functional T-DNAs inserted at a single
locus, the remainder segregating for two or more functional T-DNA inserts.
Linkage between T-DNA inserts and mutant phenotype was tested for six lines.
Mutations in three of these lines were not linked to a T-DNA insert, three
lines were identified where the mutation was segregating with a T-DNA insert.
<H3>Introduction</H3>
Fertile pollen is the culmination of physiological, biochemical and morphological
processes requiring the coordinated temporal and spatial expression of a
large number of gametophytic and sporophytic genes. These genes interact
to control the formation of the stamens, differentiation of the sporogenous
cells and meiosis, the post-meiotic development of free microspores, microspore
mitosis, pollen differentiation and anthesis. Although mutants which have
defects in one or more of these processes have been characterised in many
different plant species (<A HREF="#19">Kaul, 1988</A>), very few of the corresponding genes
have been isolated. Isolation of these genes, and characterisation of their
function, expression and interaction will provide important understanding
of the process of pollen formation. Male sterility is an important tool
in plant breeding as it prevents self-pollination, which is known to cause
inbreeding depression. Male sterility has been exploited in the production
of hybrid seed as crosses between inbred plant lines often result in progeny
with increased yield, enhanced disease resistance and better ability to
endure environmental stresses, a general phenomenon known as hybrid vigour.<BR>
<BR>
Many male sterile mutants of <I>Arabidopsis</I> have been characterised
(<A HREF="#34">Van der Veen and Wirtz, 1968</A>, <A HREF="#16">Goldberg<I> et al</I>, 1993</A>, <A HREF="#29"> Preuss <I>et al</I>, 1993</A>, <A HREF="#8">Dawson <I>et al</I>, 1993</A>, <A HREF="#5">Chaudhury <I>et al</I>, 1992</A>, <A HREF="#6">Chaudhury <I>et al</I>, 1994</A>, <A HREF="#30">Regan and Moffatt, 1990</A>, <A HREF="#18">H&uuml;lskamp <I>et al</I>, 1995</A>, <A HREF="#27">Peirson <I>et al</I>, 1996</A>, <A HREF="#17">He <I>et al</I>, 1996</A>) but only
two of the genes responsible have been identified. One of these mutants
lacks the adenine phosphoribosyl transferase enzyme which converts adenine
to AMP, showing the importance of the purine salvage pathway during microspore
development (<A HREF="#25">Moffatt and Somerville, 1988</A>). Another <I>Arabidopsis</I> male
sterile mutant in which the mutated gene has been identified is ms2 which
was generated by tagging with a maize transposable element (<A HREF="#1">Aarts <I>et
al</I>, 1993</A>). The MS2 gene shares a short stretch of homology with a wheat
mitochondrial gene. This may be significant as the mitochondrial genome is involved in most cases of cytoplasmic male sterility (<A HREF="#35">Vedel <I>et al</I>, 1994</A>). However <I>ms2</I> is a nuclear encoded mutation, so the significance
of this homology has yet to be clearly understood.<BR>
<BR>
Recently a large population of T-DNA transformed plants has been produced
by Feldmann and colleagues using a procedure known as seed transformation
(reviewed in <A HREF="#14">Forsthoefel <I>et al</I>, 1992</A>). They screened the progeny
of these transformants for visible differences in phenotype and identified
many mutants, including plants with reduced fertility. These reduced fertility
lines continued to flower later than wild-type plants and produced few (if
any) seed (<A HREF="#10">Feldmann, 1991</A>). Among these mutants are male sterile mutants
where the mutated gene is tagged by a T-DNA insert (<A HREF="#27">Peirson <I>et al</I>, 1996</A>, <A HREF="#17">He <I>et al</I>, 1996</A>). We obtained 23 of these reduced fertility
mutants as part of our quest for identifying genes involved in pollen production
in higher plants. We report here the results of screening these lines for
T-DNA tagged male sterile mutants. 
<H3>Methods</H3>
<H4>Plant Lines</H4>

T-DNA transformed lines, ecotype Wassilewskija (Ws), were kindly provided
by Kenneth Feldmann, University of Arizona. These lines were generated by
seed transformation (<A HREF="#13">Feldmann and Marks, 1987</A>) and categorised as containing
reduced fertility mutants (<A HREF="#10">Feldmann, 1991</A>).
<H4>Kanamycin selection</H4>

Seeds were surface-sterilised with 33:66:1 bleach/sterile H<TT>2</TT>O/10%
SDS for 15 min and rinsed several times in double-distilled H<TT>2</TT>O
before plating onto MS growth medium (<A HREF="#26">Murashige and Skoog, 1962</A>) containing
3% sucrose, 0.4% (w/v) agar and 50 (g/ml kanamycin (kan). In order to overcome
dormancy plated seeds were kept in the dark at 4&#176; C for at least 24
hours and then grown at 22 &#176;C in 16h light/8h dark cycle with an irradiance
of 150 &#181;mol quanta.m-2.s-1 PAR. After 7-10 days plants were scored for kan
resistance. For some mutant lines kanR plants were then transferred to MS
plates and grown a further 10 days and then transferred to soil and grown
to maturity and the fertility phenotype scored.
<H4>Phenotype</H4>

T3 seed from Feldmann's mutant lines were grown on soil, selfed and T4 seed
collected from individual plants. These T4 families were grown at 22 &#176;C
under continuous fluorescent illumination in soil mix and self-sterile plants
were easily detected in a population of self-pollinating plants because
of difference in silique length. Female fertility was then tested by applying
pollen from a wild-type fertile plant to the flowers of a self-sterile plant;
the mutant was scored as female fertile/male sterile if outcrossing produced
viable seed. Mutant plants from some lines were checked for the presence
of pollen under a light microscope. If pollen was detected, these plants
were used as pollen donors to determine pollen viability.
<H4>Southern analysis</H4>

Mutant lines which were shown to contain few inserts were subjected to southern
analysis. Genomic DNA was isolated from plants essentially as described
by <A HREF="#9">Dean <I>et al</I>, 1992</A>, except that purification on a CsCl cushion was added
(<A HREF="#32">Taylor and Powell, 1982</A>). 2 - 5 &#181;g of DNA was digested with 60 units
of the restriction endonuclease EcoRI in a 100 &#181;l reaction volume containing
1 mM spermidine for up to 18 h at 37 &#176;C. Restricted fragments were
separated on an 0.7% agarose gel and transferred onto Hybond N+ (Amersham)
nylon filter and fixed in 0.4M NaOH for 20 min. The right border T-DNA probe
corresponded to the HindIII fragment 23 from pTiC58 containing the nopaline
synthase (NOS) gene (Figure 1). The pBR322 plasmid was also used to detect
the presence of the T-DNA. Probes were radiolabelled using random oligo
priming (Amersham). Hybridisations and washes were done at 65&#176;C either
as in <A HREF="#31">Schmidt <I>et al</I> (1992)</A> or <A HREF="#7">Church and Gilbert (1984)</A>.
<H3>Results</H3>
<H4>Phenotypic Analysis</H4>

Since we completed these experiments, Feldmann has made these lines publicly
available through the Arabidopsis Biological Resource Centre (<A HREF="http://aims.cps.msu.edu/aims/">ABRC</A>) at Ohio
State and the Nottingham Arabidopsis Stock Centre (<A HREF="http://nasc.nott.ac.uk/">NASC</A>). The ABRC has published
limited characterisations of the phenotypes of these line numbers in their
Seed and DNA Stock List (February 1995). For most of the lines our results
matched their data (Table 1).<BR>
<BR>
<BR>
Table 1: Phenotype of the Reduced Fertility Lines Examined (<a href="glover.html.txt">text version of table 1</a>)<BR>
<BR>
<TABLE ID="1" BORDER >
<TR align=center><TH>MUTANT LINE</TH><th>ABRC PHENOTYPE</TH><TH>OUR PHENOTYPE</th>
</TR>
<TR align=center><TD>178</TD><TD>Male sterile</TD><TD>Male sterile/ pollenless</td>
</TR>
<TR align=center><TD>243</TD><TD>Male sterile</TD><TD>Female sterile or male/female sterile</td>
</TR>
<TR align=center><TD>547</TD><TD>Male sterile</TD><TD>Partially male sterile</td> 
</TR>
<TR align=center><TD>783</TD><TD>Partially fertile, small anthers reduced pollen</TD><td>Female sterile or male/female sterile</td>
</TR>
<TR align=center><TD>871</TD><TD>Male sterile</TD><TD>No phenotype detected</td>
</TR>
<TR align=center><TD>932</TD><TD>Male sterile/flower mutant</TD><TD>Partially sterile/dwarfed</td>
</TR>
<TR align=center><TD>1097</TD><TD>Male sterile</TD><TD>Pollenless</td>
</TR>
<TR align=center><TD>1180</TD><TD>Male sterile</TD><TD>Male sterile</td>
</TR>
<TR align=center><TD>1569</TD><TD>Male sterile</TD><TD>Partially sterile/dwarfed/bushy</td>
</TR>
<TR align=center><TD>1728</TD><TD>Reduced fertility mutant</TD><TD>Male sterile/pollenless
</td></TR>
<TR align=center><TD>1746</TD><TD>Male sterile</TD><TD>No phenotype detected</td>
</TR>
<TR align=center><TD>1885</TD><TD>Male sterile</TD><TD>Male sterile/pollenless</td>
</TR>
<TR align=center><TD>1926</TD><TD>Male sterile</TD><TD>Male sterile - dehiscence mutant</td>
</TR>
<TR align=center><TD>2379</TD><TD>Male sterile/root mutant</TD><TD>Pollenless</td>
<TR align=center>
<TR align=center><TD>2415</TD><TD>Male sterile</TD><TD>Female sterile or female/male sterile
</td></TR>
<TR align=center><TD>2522</TD><TD>Male sterile</TD><TD>Male sterile/partially female sterile
</td><TR align=center>
<TR align=center><TD>2836</TD><TD>Partially male sterile</TD><TD>Partially male sterile</td>
</TR>
<TR align=center><TD>3181</TD><TD>Male sterile</TD><TD>No phenotype detected</td>
</TR>
<TR align=center><TD>3529</TD><TD>Male sterile</TD><TD>Partially male sterile</td>
</TR>
<TR align=center><TD>3914</TD><TD>Male sterile/ yellow green</TD><TD>Pollenless/partially female sterile</td>
</TR>
<TR align=center><TD>4416</TD><TD>Anther-affected/embryo defective and size mutant</TD><td>Not determined</td>
</TR>
<TR align=center><TD>4791</TD><TD>Anther -affected</TD><TD>Not determined</td>
</TR>
<TR align=center><TD>4838</TD><TD>Male sterile</TD><TD>Partially male sterile</td>
</TR>
</TABLE>

<BR>
We could not detect any mutant plants segregating in three of the lines
871, 1746, and 3181, even though at least 30 plants from each line were
examined. The ABRC Stock centre reports that these lines have low frequency
of mutations and our data support this.<p>

Mutants in two of the lines, 932 and 1569, had altered overall plant morphology
in addition to fertility abnormalities. Many of the mutants that were scored
in Feldmann's screen as reduced fertility were also identified in other
mutant screens. For example, in addition to being isolated in the fertility
screen, 2379 was scored as short root in the agar screen, 932 was scored
as a flower mutant, and 4416 was scored as a embryo defective and a size
variant. These may represent either pleiotropic phenotype due to mutations
of the same gene or a number of independent mutations in the same line.
As only small numbers of these plants were examined - it was difficult to
differentiate between these two possibilities. <BR>
<br>
Many of the lines showed partial male sterility phenotypes (547, 2836, 3529,
and 4838), which were not analysed further as identification and characterisation
of these mutants was very difficult.<BR>
<br>
We showed some lines, 243, 783 and 2415, to have reduced female fertility,
as applying pollen from a wild-type plant did not induce full seed set.
As the reverse test wasn't performed, that is using the mutant as a pollen
donor on wild-type flowers, the possibility still exists that mutant plants
from these lines could be male sterile as well. This seems likely as the
ABRC seed list reports these lines to be male sterile. In addition, we identified
two mutant lines, 2522 and 3914, in which plants, as well as being male
sterile, showed reduced female fertility. The remainder of the lines studied
were found to be male sterile.
<H4>Scoring for kanamycin resistance</H4>

Figure 1 shows the T-DNA construct that was used to transform these lines.
It contains, within the T-DNA borders, the neomycin phosphotransferase II
(<I>nptII</I>) gene from <I>Escherichia coli</I> which detoxifies kanamycin by phosphorylation
(<A HREF="#2">Bevan <I>et al</I>, 1983</A>). Therefore the presence of a T-DNA insert can
be detected, in most cases, by scoring plants for kanamycin resistance.
The <I>nptII</I> gene segregates as a dominant marker, that is plants containing
only one copy of the gene express resistance levels indistinguishable from
plants homozygous for the marker gene.<p>

<A HREF="figure1.html">Figure 1:Structure of T-DNA in Ti plasmid 3850:1003</A><BR>
Diagram of the T-DNA region. AmpR, <I>E. coli</I> beta-lactamase gene; ORI, <I>E. coli</I>
origin of replication; NPT1, kanamycin resistance gene with a bacterial
promoter; NPT2, kanamycin gene with a dual plant promoter. Size of EcoRI
restriction fragments are shown in kilobases. Region of homology to probe
used to hybridise to filters in Figures 2,3 &amp;4 is shown.<p>
<p>
Over 35% of T2 plants produced by the seed transformation method contained
more than one functional insert per transformed line (<A HREF="#12">Feldmann <I>et al</I>, 1994</A>). In order to determine the number of functional T-DNA inserts in the
reduced fertility lines, seed was collected from at least 8 individual T3
plants from each line and 30 - 200 of these seeds (T4 families) plated onto
media containing kanamycin. Approximately 10 days after germination, kanamycin
resistant (kanR) plants could, in most cases, be easily differentiated from
kanamycin sensitive plants (kanS) plants. The ratio of kanR to kanS plants
in each family was determined, and (chi-squared calculated according to Mendelian
segregation ratios for a dominant gene (P&gt;0.05). Based on these tests,
plants were designated as having one or more T-DNA insertions conferring
kanR (Table 2).<p>

Table 2: Number of T-DNA inserts as estimated from kanamycin segregation
ratios of T4 progeny of the reduced fertility lines.(<a href="glover_table2.html.txt">text version of table 2</a>)<p>

<TABLE ID="2" BORDER>
<TR align=center><TH>Line Number</TH><th>No insert</TH><th>1 insert</th><TH>2 inserts</TH><th>3 or more inserts or non- segregating</TH><th>Other*</th></tr>
<TR align=center><TD>Ratio kanR:kanS</TD><TD>All kanS</TD><td>3:1</TD><TD>Between 3:1 and 15:1</td><TD>63:1 or &gt;63:1</td><td><br></td></tr>
<TR align=center><TD>178</TD><TD>19</TD><TD>2</TD><TD>1</td><td><br></td><td><br></td>
</TR>
<TR align=center><TD>243</TD><td><br></td><TD>3</TD><TD>1</TD><TD>4</TD><TD>2</td>
</TR>
<TR align=center><TD>547</TD><TD>3</TD><TD>7</TD><TD>1</TD><TD>4</TD><TD>1</td>
</TR>
<TR align=center><TD>1097</TD><TD>3</TD><TD>12</TD><TD>2</TD><TD>2</TD><TD>5</td>
</TR>
<TR align=center><TD>1180</TD><TD>9</TD><TD>4</TD><td><br></td><td><br></td><td><br></td>
</TR>
<TR align=center><TD>1885</TD><TD>2</TD><TD>5</TD><TD>2</TD><TD>1</TD><TD>12</td>
</TR>
<TR align=center><TD>1926</TD><TD>17</TD><TD>9</td><td><br></td><td><br></td><td><br></td>
</TR>
<TR align=center><TD>2379</TD><TD>1</TD><TD>5</TD><TD>5</TD><td><br></td><TD>15</td>
</TR>
<TR align=center><TD>2415</TD><TD>2</TD><TD>1</TD><TD>1</TD><TD>7</TD><TD>6</td>
</TR>
<TR align=center><TD>2522</TD><TD>2</td><td>9</td><td><br></td><td><br></td><td><br></td>
</TR>
<TR align=center><TD>2836</TD><TD>6</TD><TD>2</td><td><br></td><td><br></td><td><br></td>
</TR>
<TR align=center><TD>3181</TD><TD>2</TD><TD>8</td><td><br></td><td><br></td><td><br></td>
</TR>
<TR align=center><TD>3529</TD><td><br></td><TD>3</TD><TD>3</TD><TD>5</td><td><br></td>
</TR>
<TR align=center><TD>3914</TD><TD>6</TD><TD>9</td><td><br></td><td><br></td><td><br></td>
</TR>
<TR align=center><TD>4416</TD><td><br></td><TD>4</TD><td><br></td><TD>1</TD><TD>3</td>
</TABLE>

* More plants need to be screened so that chi-squared analyses can 
differentiate between categories.

 <P>
One of the mutant lines, 4791, was totally kanS in our hands. As this line
was initially isolated due to its kanR, one explanation for this is that
the <I>nptII</I> gene in this line has been silenced and is no longer conferring
kanR. Another explanation is that, as we were working with T4 generation
plants, the transgenic plants may have been lost during segregation through
the generations.<br>
<BR>
In all the lines tested except one, kanR plants were easily distinguished
from kanS plants. In this line, 1728, kan ratios were very difficult to
determine as plants were only partially resistant to kan and were difficult
to rescue from plates. <A HREF="#3">Castle <I>et al</I>, 1993</A> showed that embryonic lethal
mutants (also produced by Feldmann) that showed mottling on kan plates contained
truncations near the 3 prime end of the <I>nptII gene</I>. This could also be an
explanation for the partial resistance of this line.<BR>
<BR>
From the kan segregation ratios of the T4 families it was possible to infer
the number of T-DNAs integrated in the T3 plants. Most lines contained functional
T-DNAs integrated into at least two sites in the genome (178, 243, 547,
1097, 1885, 2379, 2415, 3529 and 4416). For other lines the segregation
pattern of either all kanS or three is to one kanR:kanS suggested the integration
of a functional T-DNA at a single locus (1180, 1926, 2522, 2836, 3181, 3914),
although the data is also consistent with multiple T-DNAs inserting at this
single locus.<BR>
<H4>Cosegregation</H4>

The fertility mutants produced in Feldmann's transformation procedure could
have resulted in a number of ways. A gene essential for plant sexual reproduction
could have been disrupted by insertion of a functional or silent T-DNA insert
within or proximal to this gene or else mutagenesis associated with the
transformation process could have occurred. Mutations in the latter category
would not necessarily linked to a T-DNA insertion.<BR>
<BR>
A large number of unlinked mutations have been detected in T-DNA transformed
plants (<A HREF="#21">Koncz <I>et al</I>, 1992</A>). In order to determine if any of
the reduced fertility mutations were due to insertion of a T-DNA into a
gene for male fertility, the segregation of the reduced fertility phenotype
and T-DNA were followed. This T-DNA could either be functional and encoding
for kanR, or not conferring kanR - where the T-DNA is inserted only partially
or the <I>nptII</I> gene has been inactivated. Six mutant lines 178, 1180, 1885,
1926, 2379 and 2522 were chosen for further study on the basis of their
phenotype and/or the number of T-DNA inserts.<BR>
<BR>
<B>Line 178</B>
<p>
Most of the segregating T4 families from this line were kanS. Two were segregating
for a single kanR marker, and one for two unlinked inserts. In order to
determine whether the kanR was linked to the male sterility phenotype, a
fertile wild-type plant was used as a pollen donor for a cross with a mutant
plant from one of the families containing a single functional kanR marker
gene. The F1 seeds were grown, seeds collected and F2 plants grown to seed.
Seeds were collected from 104 individual F2 fertile plants. This seed was
then used to score the F2 parent separately for kanR (by germinating on
kanamycin plates) and for the sterility phenotype (by growing plants without
selection in soil) to test for cosegregation. 66 of these F2 plants were
shown to be heterozygote for the male sterility and heterozygous for the
kan marker gene. 38 F2 plants were homozygous wild type plants, as well
as being all kanS The results of this experiment show that among a large
segregating population, no recombination events had occurred between the
male sterility phenotype and the kanR marker. This suggests that the T-DNA
tag encoding for kanR is very closely linked to the male sterility phenotype
in this mutant line.<BR>
<BR>
In order to detect silent inserts, DNA from pooled F3 plants representing
each of the 104 fertile and heterozygote F2 plants was isolated along with
DNA from 60 male sterile plants from the original T3 population of this
line. This DNA was subjected to Southern analysis; the plant DNA was probed
with the RB of the T-DNA. An example of the results is shown in Figure 2.
All of the male sterile and heterozygous plants were kanR and had a 14kb
EcoRI fragment hybridising to the right end of the T-DNA that was absent
from all of the homozygous fertile plants. Another T-DNA insert was also
present in this population, represented by a 7.2kb EcoRI fragment. This
insert segregated independently of the mutation as it was absent in some
male sterile plants and present in some homozygous fertile plants. Plants
containing only this insert were kanS, indicating that this insert is not
conferring kanR.<BR>
<BR>

<BLOCKQUOTE>
<A HREF="figure2.html">Figure 2:</A> Representative DNA gel blot analysis of DNA from plants from Line
178.
DNA was isolated from individual plants. The genotype of these plants was
determined by scoring in the next generation for those plants that produced
seed. Lanes are labelled as follows : MM - homozygous male fertile plants,
Mm - heterozygous plants, and mm - homozygous male sterile plants. The DNA
was cleaved with EcoRI, resolved on an agarose gel, transferred onto nylon,
and hybridised to a radiolabelled probe derived from the right border of
the T-DNA. Sizes (in kilobases) of the DNA fragments are shown on the left.
DNA representing 210 individual F2 plants was analysed in this manner. The
larger T-DNA hybridising EcoRI fragment segregated with the <I>ms</I> mutation
in these plants.<BR>
</BLOCKQUOTE>

<BR>
<B>Line 1180</B><BR>
<BR>
The segregation of the kanR marker suggested integration of a functional
T-DNA at a single locus. Southern hybridisations on individual mutant plants
using T-DNA segments as probes (<I>NOS</I> and pBR322), revealed mutant plants
without T-DNA inserts, suggesting this line to contain an unlinked mutation.<BR>
<BR>
<B>Line 1885</B><BR>
<BR>
The segregation on kanamycin of plants from this line suggested that functional
T-DNAs were integrated into at least two independent loci. Similar to line
1180 however, southern hybridisations on individual mutant plants using
T-DNA segments as probes (<I>NOS</I> and pBR322) detected mutant plants without
T-DNA inserts, showing that the mutation in this line was unlinked to a
T-DNA insert.<BR>
<BR>
<b>Line 1926</b><BR>
<BR>
Limited study of this mutant line suggests that it is a dehiscence related
mutant as pollen is produced but not effectively released from the anther
locule. Families from this mutant line segregated either 3:1 kanR: kanS,
or 100% kanS, suggesting the T-DNA has integrated at a single locus. For
initial linkage analysis - 11 kanR plants were transferred to soil and the
phenotype scored. 7 of these were fertile and shown to be heterozygous for
male sterility in the next generation, and the other 4 were male sterile.
As this data pointed to the mutant phenotype being linked to the T-DNA -
a mutant plant was crossed with wild-type and the F2 population subjected
to southern analysis and scoring similar to that described for Line 178.
27 of these F2 plants were scored as male sterile and 79 as heterozygous
plants. When DNA from these plants was hybridised with a probe corresponding
to the right border region of the T-DNA, three T-DNA hybridising bands were
detected in all 106 of these plants (Figure 3). These bands were absent
in the 20 homozygous wild-type plants tested. Further analysis showed up
to ten T-DNAs have integrated in a tandem and inverted repeat arrangement
into a single locus in this line. The ms mutation was deemed to be linked
to this T-DNA locus.<BR>
<BR>
<BLOCKQUOTE>
<A HREF="figure3.html">Figure 3:</A> Representative DNA gel blot analysis of DNA from plants from Line
1926.
DNA was isolated from individual plants. The genotype of these plants was
determined by scoring in the next generation for those plants that produced
seed. Lanes are labelled as follows : MM - homozygous male fertile plants,
Mm - heterozygous plants, and mm - homozygous male sterile plants. The DNA
was cleaved with EcoRI, resolved on an agarose gel, transferred onto nylon,
and hybridised to a radiolabelled probe derived from the right border of
the T-DNA. Sizes (in kilobases) of the DNA fragments are shown on the left.
DNA representing 126 individual F2 plants was analysed in this manner. All
three T-DNA hybridising bands segregated with the ms mutation in these plants.<BR>

</BLOCKQUOTE>
<BR>
<BR>
<BR>
<B>Line 2379</B><BR>
<BR>
Kan segregation of progeny from this line suggested that it contained at
least 3 insertion sites of a functional kanR gene. When T4 families containing
only one insert were grown non-selectively on soil and the phenotype determined,
some families were segregating three fertile to one sterile plant in some
of these lines, suggesting the mutation may be linked to a T-DNA insert
(<A HREF="#11">Feldmann, 1992</A>). However, when Southern analysis was performed on individual
sterile and fertile plants from this line, bands hybridising to the T-DNA
were not present in all the male sterile plants (<I>NOS</I> and pBR322 probes were
used for these experiments). These data show that this mutant is unlinked
to a T-DNA insert.<BR>
<BR>
<B>Line 2522</B><BR>
<BR>
Kan segregation suggested a functional T-DNA inserted at a single locus.
Southern analysis of individual plants showed 26 sterile plants contained
a single 23kb band that was also present in three out of four fertile plants
(Figure 4). These three fertile plants were shown in the next generation
to be heterozygous for the mutation and the one without the 23kb band was
shown to be a wild type plant. These data are sufficient to indicate that
this line may be a tagged mutation, because if the T-DNA and the mutation
were segregating independently each mutant would only have a 75% chance
of containing that 23kb band corresponding to the T-DNA (Feldmann, pers.
comm.).<BR>
<BR>
<BLOCKQUOTE>
<A HREF="figure4.html">Figure 4:</A> DNA gel blot analysis of DNA from plants from Line 2522.
DNA was isolated from individual plants. The genotype of these plants was
determined by scoring in the next generation for those plants that produced
seed. Lanes are labelled as follows : MM - homozygous male fertile plants,
Mm - heterozygous plants, and mm - homozygous male sterile plants. The DNA
was cleaved with EcoRI, resolved on an agarose gel, transferred onto nylon,
and hybridised to a radiolabelled probe derived from the right border of
the T-DNA. Sizes (in kilobases) of the DNA fragments are shown on the left.
26/26 ms plants contained the 23kb band, along with 3/4 fertile plants.<BR>

</BLOCKQUOTE>
<BR>
<H3>Discussion</H3>
<H4>Phenotype</H4>

As reported previously, the reduced fertility mutants identified in Feldmann's
massive screen could potentially be male sterile, female sterile or both
female and male sterile. As Arabidopsis is self-pollinating, simple tests
can distinguish between these three categories of mutants. If pollen from
a wild-type plant is applied to a sterile plant and fails to cause silique
elongation and seed formation, this indicates a blockage in female fertility.
If pollen from this mutant plant fails to fertilise wild-type ovules then
this plant can be identified as both male and female sterile. In two lines,
2522 and 3914, mutant plants were identified that were affected in both
male and female fertility. These mutants could be meiotic mutants (<A HREF="#20">Kaul and Murthy, 1985</A>), however further analysis needs to be done to show this
conclusively, as processes other than meiosis may require a common gene,
for example post-pollen mitosis and megaspore mitosis. As the number of
plants studied was small there is also a chance that there are two closely
linked mutations in these lines.<BR>
<BR>

The most common phenotype encountered among these lines was male sterility.
Male sterility can be categorised into many different types including structural
male sterility, microspore and pollen development mutants, dehiscence mutants
and pollen function mutants (<A HREF="#4">Chaudhury, 1993</A>). Out of five of the male sterile
mutants we studied, four of them appeared to be microspore development mutants
(178, 1180, 1885, 2379) and one a dehiscence mutant (1926) (data not shown).
Although not reported here, 178 has a conditional phenotype resulting from
a complex interplay between genetic and environmental factors (in press).<BR>
<BR>

Screening for sterility in the T2 generation will only identify sporophytic
mutants. Male gametophytic mutations will produce 50% non-viable pollen
in the T1 generation, so the egg cells will be fertilised only by the 50%
viable pollen. Even though this type of mutation will be transmitted through
the female line, all the progeny of the T1 plants will set seed, and will
not be detected by a screen of this kind.<BR>
<BR>
<H4>Number of insertion sites</H4>

The testing of around 1000 transformed lines generated from Feldmann's seed
transformation procedure showed 57% of the lines segregated for a single
functional T-DNA insert (ratio 3 kanR:1 kanS), and 35% for two or more inserts
(ratio greater than 3:1) with 9% having non-Mendelian ratios (<A HREF="12">Feldmann <I>et al</I>, 1994</A>). 
As fewer inserts generally facilitate analysis (and cloning)
of a tagged line, we estimated the number of T-DNA inserts in 15 of the
lines by kan segregation ratios (Table 2). From the results of these experiments,
6/15 (40%) of the tested mutant lines were segregating for a functional
T-DNA insert at a single locus, whilst the remainder showed segregation
ratios indicative of two or more functional T-DNA inserts. These results
are comparable to those of other researchers. In some of the lines with
two or more inserts it is difficult to say with certainty just how many
functional inserts are present. Hundreds more seeds would need to be screened
from the non-segregating lines to determine whether they are homozygous
for the T-DNA insert or contain 3 or more inserts.<BR>
<BR>

If some families are homozygous for the kanR marker gene this would indicate
that the sterility phenotype was not caused directly by that T-DNA insertion.
The logic behind this conclusion is simple - if the T-DNA was linked to
the mutation, all plants homozygous for the T-DNA insert would also be homozygote
for the fertility mutation, thus would produce few, if any seed. Thus families
which did not segregate for the T-DNA insert would not be detected. It is
on the basis of this idea that line numbers 1180, 1926 and 2522 were closely
examined for linkage of the sterility phenotype to the T-DNA insert, as
no kanR:kanR plants were detected in these lines.<BR>
<BR>
<H4>Unlinked mutations</H4>

Similar to other transformation procedures, seed transformation generates
a large number of mutations that are not associated with a T-DNA tag. Feldmann
and others have reported that, in some cases 20-30% of lines examined contained
mutations that were unlinked to T-DNA insertions. The analysis of the ms
mutants obtained from Feldmann therefore required extensive studies to determine
linkage between the T-DNA insertion and the mutation. Feldmann reports that
any distortion of segregation ratios of either the mutant phenotype or the
selectable marker suggests that the mutant is unlinked. As the lines, 871,
1746 and 3181 showed no detectable phenotype in our hands, and the report
from the ABRC confirms that these mutations were low in frequency, these
data point to these mutations being unlinked to a functional T-DNA insert.
Southern analyses showed that three other mutant lines, 2379, 1180 and 1885,
were unlinked to either a functional or a silent T-DNA insert.<BR>
<BR>

Several explanations have been postulated for the high percentage of unlinked
mutations in seed transformation of Arabidopsis. One hypothesis suggests
aborted T-DNA integration into the plant genome. T-DNA integration is thought
to occur by illegitimate recombination involving a double stranded break
and repair mechanism (<A HREF="#15">Gheysen <I>et al</I>, 1991</A>, <A HREF="#24">Mayerhofer <I>et al</I>, 1991</A>). If this
process is interrupted, the T-DNA may be removed by a repair mechanism leaving
single-base pair changes or deletions ('footprints') in the nuclear DNA
(<A HREF="#21">Koncz <I>et al</I>, 1992</A>). <A HREF="#36">Weigel <I>et al</I> (1992)</A> sequenced an 'untagged' allele of LEAFY from Feldmann's population and found that it contained a single base change which may have been generated in this manner.<BR>
<BR>

Another way in which untagged mutants could be produced is by the transformation
process activating an endogenous transposon. Transposons have been found
in Arabidopsis, including a superfamily of retrotransposons <I>Ta1-Ta10</I> (<A HREF="#22">Konieczny <I>et al</I>, 1991</A>) and the transposon-like element <I>Tat1</I> (<A HREF="#28">Peleman <I>et al</I>, 1991</A>)
but no evidence has been obtained yet to show that these elements are still
active. More recently, a mobile endogenous transposon, <I>Tag1</I> was also discovered when researchers were screening a T-DNA-<I>Ac</I> transformed mutant population of Arabidopsis for chlorate resistance mutants (<A HREF="#33">Tsay <I>et al</I>, 1993</A>). <I>Tag1</I> inserted into the <I>CHL1</I> gene, and its high activity was shown by the high reversion of this mutation to chlorate sensitivity. One hypothesis was that transposition of the <I>Tag1</I> element may have been stimulated by the integration of the T-DNA (<A HREF="#33">Tsay <I>et al</I>, 1993</A>). The presence of the Ac transposase may also be enhancing the <I>Tag1</I> activity.<BR>
<BR>

Several researchers have reported <I>Agrobacterium</I> sequences other than the
T-DNA have been integrated into plant genomes. Martineau and colleagues
analysed several hundred plants from different plant species and found that
approximately 20-30% of these contained <I>Agrobacterium</I> DNA from beyond the
T-DNA region stably integrated into the plant genome (<A HREF="#23">Martineau <I>et al</I>, 1994</A>).
Similarly the T-DNA region in an embryo lethal mutant of Arabidopsis generated
by seed transformation was found to contain a portion of the virulence gene
region (<A HREF="#3">Castle <I>et al</I>, 1993</A>). The integration of these non-T-DNA bacterial sequences could be responsible for some of the 'untagged' mutants.<BR>
<BR>
<H4>Linked mutations</H4>

In these experiments we identified three T-DNA transformed reduced fertility
lines from Ken Feldmann's population where the mutant phenotype was genetically
linked to a functional T-DNA insert. We identified Line 178 to be segregating
for a male sterile mutation as well as at least two T-DNA insertion loci.
By determining kan ratios and detecting the presence of T-DNA hybridising
bands on southern blots in a population of segregating plants, we established
that one of these T-DNAs was genetically linked to the male sterile mutation,
suggest that the male sterile mutation is tagged by this T-DNA. <BR>
<BR>

The 1926 line also segregated for a male sterile mutation that was closely
linked to a T-DNA insertion locus. However southern analysis from this line
suggests that there are multiple T-DNAs inserted at this locus.<BR>
<BR>

Another mutant line, 2522, was also identified which contained a single
T-DNA insertion linked to the sterility phenotype, established by southern
analysis on individual sterile and fertile plants. Mutant plants in this
line produced abnormal pollen and also showed reduced female fertility in
our hands.<BR>
<BR>
<H4>Further Work</H4>


We report here three mutants of the fertility pathway of Arabidopsis that
are potentially tagged by T-DNA insertions. Although allelism tests have
not been performed between these lines, the very different phenotypes of
these mutants suggest three different genes are involved. It should be possible
to clone the genes corresponding to these mutations using the T-DNA as a
tag.<BR>

<BR>
The work on mutant lines 1926 and 2522 is at a similar stage, plant DNA
adjacent to the T-DNA has been rescued from both these lines, although the
plant DNA from 1926 contains a repeated element - complicating further work.
The work on 178 is the most advanced. Plasmid rescue was used to clone the
T-DNA/plant junction and this fragment used to screen cDNA and genomic libraries.
Candidate clones have been sequenced. Mutant plants have been transformed
with the candidate clones to ascertain by complementation whether a gene
essential for pollen production in Arabidopsis has been cloned.<BR>
<BR>
<H4>Acknowledgments</H4>

JAG was supported by an Australian Postgraduate Award administered by the
Australian National University, as well as a Graduate Assistantship from
the Cooperative Research Centre for Plant Science. We wish to thank Bjrg
Sherman, Ying Luo and Ian Watson for their excellent technical assistance.<BR>
<BR>
<H3>References</H3>
<A NAME="1">
Aarts, M.G., Dirkse, W.G., W.J. Stiekema and A. Pereira. 1993. Transposon
tagging of a male sterility gene in Arabidopsis. Nature <B>363</B>:715-717.<BR>
<BR>
<A NAME="2">
Bevan, M.W., R.B. Flavell and M.D. Chilton. 1983. A chimaeric antibiotic
resistance gene as a selectable marker for plant cell transformation. Nature
<B>394</B>:184-187.<BR>
<BR>
<A NAME="3">
Castle, L.A., Errampalli, D., Atherton, T.L., Franzmann, L.H., E.S. Yoon
and D.W. Meinke. 1993. Genetic and molecular characterisation of embryonic
mutants following seed transformation in Arabidopsis. Mol. Gen. Genet. <B>241</B>:504-514.<BR>
<BR>
<A NAME="4">
Chaudhury, A.M. 1993. Nuclear genes controlling male fertility. Plant Cell
<B>5</B>:1277-1283.<BR>
<BR>
<A NAME="5">
Chaudhury, A.M., Craig, S., Blmer, K.C., L. Farrell and E.S. Dennis. 1992.
Genetic control of male fertility in higher plants. Aust. J. Plant. Physiol.
<B>19</B>:419-426.<BR>
<BR>
<A NAME="6">
Chaudhury, A.M., Lavithis, M., Taylor, P.E., Craig, S., Singh, M.B., Signer,
E.R., R.B Knox and E.S. Dennis. 1994. Genetic control of male fertility
in Arabidopsis thaliana: structural analysis of premeiotic developmental
mutants. Sex. Plant Reprod. <B>7</B>:17-28.<BR>
<BR>
<A NAME="7">
Church, G.M. and W. Gilbert. 1984. Genomic sequencing. Proc. Natl. Acad.
Sci. USA <B>81</B>:1991-1995.<BR>
<BR>
<A NAME="8">
Dawson, J., Wilson, Z.A., Aarts, M.G.M., Braithwaite, A., L.G.B. Briarty
and B.J. Mulligan. 1993. Microspore and pollen development in six male sterile
mutants of Arabidopsis thaliana. Can. J. Bot. <B>71</B>:629-638.<BR>
<BR>
<A NAME="9">
Dean, C., Sjodin, C., Page, T., J. Jones and C. Lister. 1992. Behaviour
of the maize transposable element Ac in Arabidopsis thaliana. The Plant
Journal <B>2</B>:69-81.<BR>
<BR>
<A NAME="10">
Feldmann, K.A. 1991. T-DNA insertion mutagenesis in Arabidopsis: mutational
spectrum. The Plant Journal <B>1</B>:71-82. <BR>
<BR>
<A NAME="11">
Feldmann, K.A. 1992. T-DNA insertion mutagenesis in Arabidopsis :seed infection/transformation.
In: Methods in Arabidopsis research. (eds C. Koncz <I>et al</I>), pp274-289. World
Scientific, London.<BR>
<BR>
<A NAME="12">
Feldmann, K.A., R.I. Malmberg and C. Dean. 1994. Mutagenesis in Arabidopsis.
In: Arabidopsis (eds E.M. Meyerowitz and C.R. Somerville), pp137-172. Cold
Spring Harbour Laboratory Press, New York.<BR>
<BR>
<A NAME="13">
Feldmann, K.A. and M.D. Marks. 1987. Agrobacterium-mediated transformation
of germinating seeds of Arabidopsis thaliana: a non-tissue culture approach.
Mol. Gen. Genet. <B>208</B>:1-9.<BR>
<BR>
<A NAME="14">
Forsthoefel, N.R., Wu, Y., Schulz, B., M.J. Bennett and K.A. Feldmann. 1992.
T-DNA insertion mutagenesis in Arabidopsis: prospects and perspectives.
Aust. J. Plant Physiol. <B>19</B>:353-366.<BR>
<BR>
<A NAME="15">
Gheysen, G., R. Villarroel and M. Van Montagu. 1991. Illegitimate recombination
in plants: a model for T-DNA integration. Genes Dev. <B>5</B>:287-297.<BR>
<BR>
<A NAME="16">
Goldberg, R.B., T.P. Beals and P.M. Sanders. 1993. Anther development: basic
principles and practical applications. Plant Cell <B>5</B>:1217-1229.<BR>
<BR>
<A NAME="17">
He, C., Tirlapur, U., Cresti, M., Peja, M., D.E. Crone and J.P. Mascarenhas.
1996. An Arabidopsis mutant showing aberrations in male meiosis. Sex. Plant
Reprod. <B>9</B>:54-57.<BR>
<BR>
<A NAME="18">
Hulskamp, M., Kopczak, S.D., Horejsi, T.F., B.K. Kihl and R.E. Pruitt.
1995. Identification of genes required for pollen-stigma recognition in
Arabidopsis thaliana. The Plant Journal <B>8</B>:703-714.<BR>
<BR>
<A NAME="19">
Kaul, M.L.H. 1988. Male sterility in higher plants. Springer-Verlag, Berlin
Heidelberg.<BR>
<BR>
<A NAME="20">
Kaul, M.L.H. and T.G.K. Murthy. 1985. Mutant genes affecting higher plant
meiosis. Theor. Appl. Genet. <B>70</B>:449-466.<BR>
<BR>
<A NAME="21">
Koncz, C., Nemeth, K., G.P. Redei and J. Schell. 1992. T-DNA insertional
mutagenesis in Arabidopsis. Plant Mol. Biol. <B>20</B>:963-976.<BR>
<BR>
<A NAME="22">
Konieczny, A., Voytas, D.F., M.P. Cummings and F.M. Ausubel. 1991. A superfamily
of Arabidopsis thaliana retrotransposons. Genetics <B>127</B>:801-809.<BR>
<BR>
<A NAME="23">
Martineau, B., T.A. Voelker and R.A. Sanders. 1994. On defining T-DNA. Plant
Cell <B>6</B>:1032-1033.<BR>
<BR>
<A NAME="24">
Mayerhofer, R., Koncz-Kalman, Z., Nawrath, C., Bakkeren, G., Crameri, A.,
Angelis, K., Redei, G. P., Schell, J., B. Hohn and C. Koncz. 1991. T-DNA
integration: a model of illegitimate recombination in plants. EMBO J. <B>10</B>:697-704.<BR>
<BR>
<A NAME="25">
Moffatt, B. and C. Somerville. 1988. Positive selection for male-sterile
mutants of Arabidopsis lacking adenine phosphoribosyl transferase activity.
Plant Physiol. <B>86</B>:1150-1154.<BR>
<BR>
<A NAME="26">
Murashige, T. and F. Skoog. 1962. A revised medium for rapid growth and
bioassays with tobacco tissue cultures. Physiol. Plant. <B>15</B>:473-497.<BR>
<BR>
<A NAME="27">
Peirson, B.N., Owen, H.A., K.A. Feldmann and C.A. Makaroff. 1996. Characterisation
of three male-sterile mutants of Arabidopsis exhibiting alterations in meiosis.
Sex. Plant Reprod. <B>9</B>:1-16.<BR>
<BR>
<A NAME="28">
Peleman, J., Cottyn, B.,Van Camp, W., M. Van Montagu and D. Inze. 1991.
Transient occurrence of extrachromosomal DNA of Arabidopsis thaliana transposon-like
element, Tat1. Proc. Natl. Acad. Sci. USA <B>88</B>: 3618-3622.<BR>
<BR>
<A NAME="29">
Preuss, D., Lemieux, B., G. Yen and R.W. Davis. 1993. A conditional sterile
mutation eliminates surface components from Arabidopsis pollen and disrupts
cell signalling during fertilisation. Genes Dev. <B>7</B>:974-985.<BR>
<BR>
<A NAME="30">
Regan, S.M. and B.A. Moffatt, 1990. Cytochemical analysis of pollen development
in wild-type Arabidopsis and a male-sterile mutant. Plant Cell <B>2</B>:877-889.<BR>
<BR>
<A NAME="31">
Schmidt, R., Cnops, G., I. Bancroft and C. Dean. 1992. Construction of an
overlapping YAC library of the Arabidopsis thaliana genome. Aust. J. Plant
Physiol. <B>19</B>:341-351.<BR>
<BR>
<A NAME="32">
Taylor, B., Powell, A. 1982. Isolation of plant DNA and RNA. FOCUS <B>4</B>:4-6.<BR>
<BR>
<A NAME="33">
Tsay, Y.F., Frank, M.J., Page, T., C. Dean and N.M. Crawford. 1993. Identification
of a mobile endogenous transposon in Arabidopsis thaliana. Science <B>260</B>:
342-344.<BR>
<BR>
<A NAME="34">
Van der Veen, J.H. and P. Wirtz. 1968. EMS-induced genic male sterility
in Arabidopsis thaliana: a model selection experiment. Euphytica <B>17</B>:371-377.<BR>
<BR>
<A NAME="35">
Vedel, F., Pla, M., Vitart, V., Gutierres, S., P. Chetrit and R. Depaepe.
1994. Molecular basis of nuclear and cytoplasmic male sterility in higher
plants. Plant. Physiol. Biochem. <B>32</B>(5):601-618.<BR>
<BR>
<A NAME="36">
Weigel, D., Alvarez, J., Smyth, D.R., M.F. Yanofsky and E.M. Meyerowitz.
1992. LEAFY controls floral meristem identity in Arabidopsis. Cell <B>69</B>:843-859.
</BODY>
</HTML>
