<HTML>

<HEAD>


<TITLE>Protocols</TITLE>

</HEAD>

<BODY BACKGROUND="/images/arabidopsisbackground.gif">

<a href="/index.html"><img src="/images/tairsmall.gif" border=0 align=left></a><CENTER><H2>STANDARD PLANT MOLECULAR BIOLOGY PROTOCOLS </H2></CENTER>

<CENTER>
<img src="/images/line.red2.gif" alt="[line]" WIDTH="80%" HEIGHT="4">
</CENTER>
<p>

<center>Kindly provided by <a href="http://biobase.dk/~mundy">John Mundy</a>, Institute of Molecular Biology,
Copenhagen, Denmark.
</center>

<p>

<hr size=1 noshade width=90%>
<P>
<A href ="#Agro.cc">Agrobacterium competent cells</a><BR>	

<P>
<A href ="#Agro.elec">Agrobacterium electroporation</a>

<P>
<A href ="#Agro.mp1">Agrobacterium miniprep</a>

<P>
<A href ="#Agro.mp2">Agrobacterium miniprep (phenol free)</a>

<P>
<A href ="#trans.inf">Arabidopsis transformation (infiltration)</a>

<P>
<A href ="#prob1">cDNA probes</a>

<P>
<A href ="#hyb1">Colony hybridization</a>

<P>
<A href ="#conA">Concanavalin A Blots</a>


<P>
<A href ="#dnapage">DNA PAGE</a>

<P>
<A href ="#ccca">E coli competent cells (CaCl2)</a>

<P>
<A href ="#ccru">E coli competent cells (RuCl2)</a>

<P>
<A href ="#colitrans">E coli transformation</a>

<P>
<A href ="#endlabel">End labelled probe preparation</a>

<P>
<A href ="#filterwash">Filter washing</a>

<P>
<A href ="#fluorPAG">Fluorography of PAG</a>

<P>
<A href ="#Fluoromet">Fluorometer/Luminometer Wallac Victor II</a>

<P>
<A href ="#Footprint">Footprint with DNAse1</a>

<P>
<A href ="#Gelshift">Gelshift</a>

<P>
<A href ="#cereal">Genomic DNA from cereal leaves </a>

<P>
<A href ="#gst">GST fusion protein purification </a>

<P>
<A href ="#gushis">GUS histochemical assay</a>

<P>
<A href ="#gusluc">GUS & LUC bombardment assay</a>

<P>
<A href ="#guslumi">GUS luminometric assay</a>

<P>
<A href ="#his">HIS-tagged protein purification</a>

<P>
<A href ="#hormone">Hormone treatments</a> 

<P>
<A href ="#horminhi">Hormones & inhibitors</a> 

<P>
<A href ="#hybselc">Hybridization selection of cloned DNA</a>

<P>
<A href ="#imuprecip">Immunoprecipitation from in vitro translations</a>

<P>
<A href ="#isotacho">Isotachophoresis probe isolation</a>

<P>
<A href ="#vitrotranscr">In vitro transcription</a>

<P>
<A href ="#vitrotransl">In vitro translation</a>

<P>
<A href ="#lamdaDNA">Isolation of Lambda DNA by plate lysate</a>

<P>
<A href ="#lamdalys">Lambda lysate</a>

<P>
<A href ="#leafpcr">Leaf PCR </a>

<P>
<A href ="#Maxam">Maxam Gilbert sequencing</a>

<P>
<A href ="#microsome">Microsomal preps for membrane proteins</a>

<P>
<A href ="#miniprepalk">Miniprep - alkaline lysis</a>

<P>
<A href ="#NEpHGE">NEpHGE Non-equilibrium pH gel electrophoresis</a>

<P>
<A href ="#Northern">Northern blot in formaldehyde gel</a>

<P>
<A href ="#nuclext">Nuclear extract preparation</a>

<P>
<A href ="#nuclei">Nuclei prep</a>

<P>
<A href ="#nuclprot">Nuclear prep from protoplasts</a>

<P>
<A href ="#dTcellulose">Oligo dT cellulose purification of mRNA</a>

<P>
<A href ="#oligoprobe">Oligo probe hybridization</a>

<P>
<A href ="#oligoprobeprep">Oligonucleotide probe preparation</a>

<P>
<A href ="#phagestrain">Phage strains</a>

<P>
<A href ="#plasmidcscl">Plasmid prep with CsCl</a>

<P>
<A href ="#primerxt">Primer extension with synthetic oligonucleotide</a>

<P>
<A href ="#progeny">Progeny analysis of plants</a>

<P>
<A href ="#RNAGT">RNA extraction in GT w/ CsCl spin</a>

<P>
<A href ="#RNAstarch">RNA large scale from starchy tissue</a>

<P>
<A href ="#RNAmini1">RNA miniprep 1</a>

<P>
<A href ="#RNAmini2">RNA miniprep 2</a>

<P>
<A href ="#RTPCR">RT-PCR</a>

<P>
<A href ="#sdspage">SDS-PAGE for proteins</a>

<P>
<A href ="#seedsteril">Seed sterilization for cereals</a>


<P>
<A href ="#silver">Silver staining of proteins in PAG</a>

<P>
<A href ="#solprot">Soluble protein extraction</a>

<P>
<A href ="#Southwestern">Southwestern blotting</a>

<P>
<A href ="#T4pol">T4 polymerase fill in rxn</a>


<P>
<A href ="#western">Western blotting</a>

<P>
<A href ="#wholecell">Whole cell extracts</a>
<P>

<hr size=1 noshade width=90%>

<P>
<H4><A name ="Agro.cc">AGROBACTERIUM COMPETENT CELLS</a></H4>

You should double up this protocol - it is almost the same
amount of work and you 

can thus get some 80 tubes.

<P>
1. Inoculate colony O/N in 2 ml YEP + antibiotics at 28C shaker.
ABI - 50 KAN & 25 

<P>
Chlor, gv3101  - 25GEN

2. Transfer O/N culture to 200ml YEP in a sterile 500ml flask
and shake at 250rpm 

until the OD is 0.3 (4-5hrs)

<P>
3. Spin in sterile 50ml screw cap tubes 4C 5krpm 10&#130;. Check
to make sure cells are 

pelleted, if not repeat at higher speed.

<P>
4. Aspirate supernatant, resuspend pellet in 20ml ice cold 1mM
HEPES pH7 (sterile 
filtered), respin.

<P>
5. Repeat 4. two more times!

<P>
6. After aspirating, resuspend pellet in 2ml ice cold 10% glycerol
(sterile 

filtered).

<P>
7, ASAP dipense in 40ul aliquots in pre-chilled, sterile eppis,
freeze in lN2 and 
store -70C



<P>
<H4><a name="Agro.elec">AGROBACTERIUM ELECTROPORATION</a></H4>
<b>Safety precautions</b>

<P>
1. Never change any other settings than stated while unit is charging
(this may 

damage both the electroporator and power supply)

<P>
2. Keep unit away from water in a dry area and away from flammable
materials.

<P>
3. Never short circuit  terminals.

<P>
4. Whilst delivering pulse, keep hands away from chamber and cuvette.
The result 

may otherwise be shocking.



<P>
<b>DNA preparations</b>

<P>
DNA for electroporation must be free of salt, RNA or protein.
DNA (in TE buffer) 


should be first treated with RNase, then twice extracted with
phenol/chloroform. 

<P>
This will remove protein and RNA. To remove salt, EtOH precipitate
the DNA and 

wash twice with 70% ethanol. Resuspend the DNA at 0.4 -1 ug/ml.




<P>
<b>Preparing the electroporator</b>

<P>
There are two types of cuvettes 1 and 2mm. Most Agro protocols
use 2mm (Invitrogen 

#650009 w/blue lids). 

<P>
1. Make sure power supply is off.

<P>
2. With Charge/Pulse switch of the electroporator in the PULSE
position, connect 

the leads from the power supply to the corresponding coloured
terminals on the 

back of the electroporator.

<P>
3. Set  Arm/Disarm dial to disarmed

<P>
4. Set Capacitance selector to 50 mF

<P>
5. Set Load resistance to 200W

<P>
6. Turn on power supply

<P>
7. Set maximum power to 25 W

<P>
8. Set current to 25 mA

<P>
9. Set voltage to 1800 V

<P>
10. Allow power supply to stabilize (still in pulse position)


<P>
11. Select the CHARGE position using the Charge/Pulse switch.
After 20-30 seconds, 

the charging light will glow.

<P>
12. Check that voltage meter still reads 1800 V

<P>
13. Set Arm/Disarm to the ARMED position and the armed light will
glow.

<P>
14. Switch the Charge/Pulse to the PULSE position. The pulse light
will glow 

briefly and both the charging and armed lights will go out.

<P>
15. Set Arm/Disarm to the DISARMED position (the armed light should
be off)  



<P>
<B>Electroporating</B>

<P>
Electrocompetent bacterial cells, YEP media and DNA solutions
must be kept on ice 


before mixing. Note that the following steps should be carried
out in under 1' and 

that you should be wearing glasses and gloves

<P>
16. mix 1-2ml DNA (600 ng) with 40ml cells

<P>
17. Transfer the DNA/cell mixture to a cuvette on ice avoiding
air bubbles by 

gently shaking the cuvette

<P>
18. Dry outside of the cuvette with tissue paper and insert the
cuvette into the 


cuvette chamber with notch facing towards you

<P>
19. Close cuvette chamber lid

<P>
20. Set Arm/Disarm to ARM (arm light comes on)

<P>
21. Set Charge/Pulse to pulse and the pulse light will come on
briefly

<P>
22. When pulse light is off, set Arm/Disarm to DISARM (arm light
comes on) and 


remove cuvette

<P>
23. With DNA/agro mix still in cuvette, add 500ml cold YEP (no
antibiotics) and 

mix solution by gently pitppeting up and down

<P>
24. Transfer the cells to an eppi and incubate at 28C for 2-4
hour

<P>
25. Leave the electroporator with the switch in the PULSE position


<P>
26. Plate 200ml on YEP + antibiotics

<P>
27. Incubate at 28C and colonies will appear in 2-3 days



<P>
<b>Re-using cuvettes</b>

<P>
Fill a used cuvette w/ 0.1M H2SO4 and let stand 15'. Rinse 6x
w/ dH20, then 2x w/ 

96% EtOH. Store them well-covered in 70% EtOH



<P>
<H4><a name="Agro.mp1">AGROBACTERIUM MINIPREP</a></H4>
Agros to be used for plant transformation should be checked for
the presence of 


the Ti plasmid as plant transformation and the analysis of transgenic
plants is 


time consuming. The easiest way to do this is to make an agro
miniprep and to use 


PCR to determine that the cells contain your construct. PCR is
necessary here 


because the Ti plasmid is single copy and you can barely see it
on agarose gels. 



<P>
1.  Grow cells overnight in 5 ml LB or YEP with antibiotics.

<P>
For pMONs in ABI - 50ug/ml KAN, 50ug/ml Spec, 25ug/ml Chlor

<P>
For pBI types in gv3101 - 50ug/ml KAN, 25ug/ml GEN

<P>
2.  Remove 1 ml cells to two microfuge tube

<P>
3.  Centrifuge 45 sec and remove the supernatant with aspiration


<P>
4.  Add 1 ml cells more to both tubes and repeat step 3

<P>
5.  Vortex the pellet, add 100 l MPS1 solution, vortex again and
incubate the 


tubes at room temperature for 5 min

<P>
6.  Add 20 l of a 20 mg/ml lysozyme solution, vortex-spin 1 sec
and incubate 15 


min at 37C.

<P>
7.  Add 200 l MPS2 solution( freshly made), mix gently by turning
the rack 3-4 


times and incubate 5 min on ice

<P>
8.  Add150 l MPS3, vortex for at least 10 sec and incubate 5 min
on ice

<P>
9.  Centrifuge for 5 min and remove the supernatant to new tubes


<P>
10. Add 400 l phenol/chloroform/isoamyl alcohol (25:24:1), vortex,
centrifuge for 


5 min and remove the supernatant to new tubes

<P>
11. Repeat step 10

<P>
12. Repeat this step with chloroform alone

<P>
13. Add 300 l isopropanol and incubate on ice for 10 min

<P>
14. Centrifuge for 5 min and wash pellet with  70 % EtOH

<P>
15. Dry pellet and resuspend the two tubes in a total of 50 l
TE-buffer+RNase, use 


2ml for a PCR, freeze the rest.


<P>

MPS1 for 50 ml  Stock

<P>
<PRE>
	50 mM glucose 		1M 2,5 ml

	10 mM EDTA 		0,5mM 1 ml
<BR>
	25 mM Tris pH=8.0 	1M 1,25 ml

</PRE>

<P>
MPS2 for 10 ml
<P>
<PRE>
	0,2 N NaOH  	10N  200 ml

	1% SDS 		10%  1 ml

	H2O      	8,8 ml
</PRE>




<P>
MPS3 for 100 ml
<P>
<PRE>
	5 M potassium acetate   60 ml

	glacial acetic acid  	11,5 ml

	H2O     		28,5ml

</PRE>

<P>
<H4><A name="Agro.mp2">AGROBACTERIUM PROTEASE K MINIPREP (PHENOL FREE)</a></H4>
1) Resuspend pellet from 3ml cultures in 200uls STET

<P>
2) Add 10ul Lysozyme/RNase solution and incubate for15min at 37
C.

<P>
3) Add 1ul 10mg/ml Protease K solution and incubate at 50 C for
15min.

<P>
4) Incubate at 94 C 1min then vortex vigourously 30 sec.

<P>
5) Remove gooey supernatant to new tube and add 5uls 5% CTAB.
Wait 5min then spin 


5min.

<P>
6) Remove supernatant and rususpend in 300uls 1.25M NaCl by vigourous
vortexing. 


Spin tubes for 1min and transfer supernatant to new tube with
750uls 96% EtOH + 


1mM PMSF. 5min RT, 30min spin.

<P>
7) Wash pellet in 70% EtOH+ 1mM PMSF, dry and resuspend in 40uls
TE 8.0



<P>
Can visualize 10uls in a restriction digest.



<P>
<H4><a name="trans.inf">ARABIDOPSIS TRANSFORMATION BY VACUUM INFILTRATION</a></H4>
This protocol  is modified from Bechtold, Ellis and Pelletier
(1993). &quot;In


planta Agrobacterium mediated gene transfer by infiltration of
adult


Arabidopsis thaliana plants&quot;. [C.R. Acad. Sci. Paris, Life
Sciences 316:


1194-1199].



<P>
PLANT GROWTH:

<P>
1. Take seeds with a brush and place them into 8cm square pots
filled with


soil. Don't compress the soil too much and water the pots thoroughly
with


2-3 pot-vol to remove excess nutrients. Place 12-16 seeds in each
pot.

<P>
Place the pots in the cold room for two days before transfering
them to the


growth chamber. Grow the plants for three weeks in short days
(10 hr or


less) to get large plants and a greater seed yield. Transfer the
pots to


long days to induce bolting. Grow plants to a stage at which bolts
are


around 10 cm tall.



<P>
 2. Clip off emerging bolts close to rosette leaves to encourage
 growth of


multiple secondary bolts. Infiltration will be done 7 to 9 days
after


clipping (plants will be 10-15 cm high and the biggest of the
inflorescens


will have made the first tiny silique. Do not water the plants
the day


before vacuum infiltration.



<P>
VACUUM INFILTRATION:

<P>
3. Start a 4ml agrobacterium culture (YEP+antibiotics) inoculated
from a


-800C stock or from a plate. Grow cells O/N to 48h depending 
on the


strain. Add this culture to 250 ml of YEP+antibiotics (A 250ml
culture will


give enough cells for infiltration of 6 pots). Grow the culture
between O/N


and 2 days (depending  on the strain) to OD600 = 1.2-1.8. The
culture will


have a mother of pearl appearance (not lumpy or black).



<P>
4. Spin down agros at 5000rpm for 10 min in 250ml centrifuge bottles,



resuspend in infiltration media to an OD600 = 0.8 in a minimum
volume of


300ml.



<P>
5. Poor the agro suspension into a beaker of an appropiate  size
(400ml is


ok). Place the beaker into the vacuum jar. Degass the solution
by drawing


vacuum until bubles form. Place a paper towel under the beaker
to avoid


that the beaker gets stuck in the bottom of the vacuum jar.



<P>
6. Sprinkle the plants with water 5 min prior to infiltration
(optional)


and then invert plants into the culture solution. Make sure that
all the


flowers are submerged and leave 2cm between the rosettes leaves
and the


agro suspension. Don't let the culture contact the rosette or
soil as this


could kill the plants. Avoid that the solution boils over when
you pull the


vacuum. Make sure that the soil is only moist, so that the water
from the


pots does not enter into the culture suspension (therefore we
recommend not


to water the plants the day before infiltation). Draw vacuum for
15-20 min


for WS and 30 min for Col-0 at a pressure close to 0.05 Bar (we
are using


an oil pump).



<P>
 7. Before removing the plants from the vacuum jar place a plastic
bag over


the pot and beaker. Pull out and remove plants from the beaker,
lay pots


on their side (to avoid that excess infiltration media runs down
into the


soil). Fold over the top of the plastic bag and  staple them twice.
The


other possibility is to place the pots laying on their side into
a tray and


cover the whole box with saranwrap. Put them in a growth chamber
for one


night. Next day move them to the green house. Put the plants in
vertical


position and open  the bags. Next day get rid off the bags. In
case you


have the plants in trays: put also the plants in vertical position
and use


sticks and saranwrap to make a kind of a tend around the plants.
Next day


remove the plastic. In hot summers, we recommend to give plants
a shower


after we have placed  them in vertical position (the purpose of
this is to


remove the sugars from the infiltration media which decrease fungal



infection).



<P>
8. Grow plants for approx. four weeks, keeping bolts from each
pot together


but separated from neighbouring pots



<P>
9. When the siliques begin to turn yellow, place the pot on its
side with


the plants inside a big envelope. Leave them for one week to dry
out and


cut off the plants. Let the seeds dry in the envelope and clean
them 10


days later (keep all the seeds from one pot together). Store the
seeds in


the cold room for one week before plating them.



<P>
KANAMYCIN SELECTION PROTOCOL

<P>
1. Sterilisation of seeds:

<P>
aliquot seeds in 15ml falcon tubes (approx 700 seeds/tube, you
can estimate


the ammount of seeds by first drawing a square plate of 9cmx9cm
on a paper


and spreading the seeds on  it). Add 10 ml of hypoclorite solution.
Shake


tubes for 10 min. Remove the solution and add 10ml of 70% ethanol.
Wait 2


minutes. Discard EtOH and wash seeds 2-3 times with 10ml of  sterile
water.

<P>
Resuspend seeds with 8ml 0.7% top agar (no warmer than 55oC ).





2. Spread seeds onto  selection plates (MS+Kan). Dry plates in
laminar flow


hood  until the top agar has solidified.



<P>
3. Vernalize plates for two nights in the cold room at 4oC. Transfer
the


plates to the growth chamber (21oC with continous light).



<P>
4. After aprox 7 days transformants should be clearly identifiable
as dark


green plants with healthy green secondary leaves and roots that
extend into


the selective medium. Root growth is the most clear maker to identify



transformants at an early stages.

<P>
To make sure that the transformants are positive transfer them
to a new


MS+Kan plate and leave them there for a few days (if they turn
yellow is


because they are faulse positives). Transfer the seedlings to
soil.



<P>
If you have contamination on your plates at this step, transfer
the


transformants as early as possible to soil.



<P>
5. Grow the plants and collect the seeds.



<P>
<b>Infiltration Media</b>

<P>
1/2 x Murashige&Skoog salts (SIGMA #5524)

<P>
1X B5 vitamines (1ml of 1000x stock) (SIGMA; #G-2519) Gamborg's
vitamine


powder, to prepare the 1000x stock disolve 11.2g in 100ml water.


<P>
5% sucrose

<P>
adjust to pH 5.7 before autoclaving

<P>
after autoclaving add:

<P>
- Benzylamino Purine (BAP), 10 &#181;l per liter of a 1 mg/ml
stock in DMSO. By


adding the hormone just before use, you can keep infiltration
media as a


stock for at least one week prior to infiltration.

<P>
- we recommend to add  0.01% silwet to the infiltration media
to increase


transformation efficiency especially for Landsberg and colombia
ecotypes.


(silwet is from LEHLE SEEDS, cat no VIS-01 VAC-IN-STUFF (silwet
L-77)



<P>
<b>Selection plates:</b>

<P>
1x Murashige&Skoog salts

<P>
1% sucrose

<P>
adjust pH 5.7 with 1M KOH.

<P>
0.7% Difco agar.

<P>
autoclave, cool, and add:

<P>
1x MS vitamines (SIGMA #M-7150). Take 1ml of 1000x stock prepared
 by


disolving 10.3gr in 100ml of water.

<P>
antibiotic (kanamycin 50mg/l).



<P>
Top agar:

<P>
1x Murashige&Skoog salts.

<P>
1% sucrose.

<P>
adjust pH 5.7 with 1M KOH.

<P>
0.7% Difco agar.

<P>
autoclave.

<P>
before use: boil in the microwave and keep in water bath at 50-550C.




<P>
<b>YEP media (liquid)</b>:

<P>
10 g /l Bacto peptone (Difco)

<P>
10 g/l Yeast extract (Difco)

<P>
5 g /l NaCl

<P>
For YEP plates add 15gr/l Difco bacto agar.



<P>
Hypoclorite solution:

<P>
for 50 ml:

<P>
4ml Na Hypoclorite 15%

<P>
255l Tween-20

<P>
water to 50ml



<P>
<h4><a name="prob1">cDNA PROBES</a></H4>
mRNA 1-5ug

<P>
H2O 50ul

<P>
oligoDt (10mg/ml) 2ul



<P>
heat 90C 2', ice



<P>
5xRT buffer 20

<P>
10mM dA,G,T 2 ea.

<P>
1mM dC  2 

<P>
1M DTT  1

<P>
dCTP* 10

<P>
RT 6


<P>
inc 42C, 1h


<P>
+

<P>
4M NaOH 15

<P>
10% SDS 7.5

<P>
0.25M EDTA 19

<P>
H2O 8.5


<P>
inc 37C 1h


<P>
+ 3ul Hac

<P>
PCHCl3 ext



<P>
separate from free nucs over G-50 column



<P>
use106cpm/mlhyb



<P>
<H4><a name="hyb1">COLONY HYBRIDIZATION</a></H4>
filter prep: S&S BA 85 Nc

<P>
1) lay filters on plates to moisten

<P>
2) replicaplate colonies to them

<P>
3) grow o/n

<P>
4) transfer filters to 3mm paper on saran wrap soaked w/ 10% SDS.
This


will lyze colonies; they subside slightly, inc 5'

<P>
5) trans filter to dry 3mm briefly

<P>
6) trans 2x to 3mm soaked in 0.5M NaOH to denature DNA, inc 5'


<P>
7) trans to dry 3mm briefly

<P>
8) trans to 3mm soaked w/ 1M Tris 7.5 to neutralize, inc 5'

<P>
9) trans to dry 3mm

<P>
10)trans to 3mm w/ o.5M Tris/1.5M NaCl to bind DNA, inc 5'

<P>
11) wash in 1xSSC w/ 1ug/ml Proteinase K RT 1h

<P>
12) wash 10' in 1xSSC

<P>
13) air dry, bake 80C 2h vacuum



<P>

<b>solutions:</b>

<P>
<PRE>
PB 				/ 1l

50% formamide  			500ml
5xSSC   			250ml 20xSSC
200ug/ml ssDNA  		20ml 10mg/ml sheared boiled salmon sperm
5xDenhardts   			100ml 50x
10ug/ml polyadenylic acid 	1ml 10mg/ml
0.1% SDS 			100ml 10%
H2O 				to 1l
</PRE>

<P>
<PRE>
HB				 /1l

same as PB but

100ug/ml ssDNA
10% dextran SO4   		100g, dissolve this first o/n
H2O 
</PRE>


<P>
<H4><a name="ConA">CONCANAVALIN A BLOTS</a></H4>

1) electroblot

<P>
2) block in Con A buffer (CAB) RT 3h, 5&#169;10ml/lane

<P>
3) inc w/ HRP&#169;Con A RT 3h (1ul HRP&#169;Con A/lane in CAB,
approx1:2000 dilution

<P>
4) wash 3 x 200ml, 30' RT w/ CAB

<P>
5) wash 3 x 100ml, 2' RT w/ HRP buffer (HRPB)

<P>
6) develope in DMB solution  (DMBS) 1&#169;10'

<P>
7) wash in H2O to stop reaction



<P>
<b>solutions:</b>

<P>
<PRE>
CAB   			[stock]  		ml/1l

0.5M NaCl  		3M   			160
50mM Tris 7.4 		1M   			50
2.5mM CaCl2  		1M   			2.5
1mM MgCl2  		1M   			1
0.5% Tween 20    				5

</PRE>

<P>
HRPB: 10mM Tris 7.5



<P>
DMBS:

<P>
10mM Tris 7.5, 20ml

<P>
5mg DMB

<P>
3.3ul H2O2, immediately before use




HRPCon A: type 4 labelled, Sigma # L4010. Make 1mg/ml stock




DMB: 3,3'&#169;dimethoxybenzidine dihydrochloride Kodak #X8748




<P>
<H4><a name="dnapage">DNA PAGE</a></H4>

<table><TR><TD>% gel</TD> <TD> 3.5</TD>  <TD>  5.0</TD>  <TD> 8.0</TD>  <TD>  12.0</TD>  <TD> 20.0</TD></TR>
<TR><TD>30% acryl stock</TD> <TD>  11.6 </TD> <TD> 16.6  </TD> <TD> 26.6 </TD> <TD>  40.0 </TD> <TD>  66.6</TD></TR>
<TR><TD>H20 </TD> <TD> 76.3</TD> <TD>71.3 </TD> <TD>61.3 </TD> <TD>47.9 </TD> <TD>  21.3</TD></TR>
<TR><TD>10x TBE</TD> <TD>5.0 </TD> <TD>   &gt;  </TD> <TD>    &gt;</TD> <TD>      &gt; </TD> <TD>     &gt;</TD> </TR> 
<TR><TD>10% APS</TD> <TD>  0.7  </TD> <TD>  &gt; </TD> <TD>     &gt; </TD> <TD>     &gt; </TD> <TD>     &gt;</TD> </TR>


<TR><TD>TEMED (ul)  </TD> <TD>      30   </TD> <TD>  &gt;   </TD> <TD>   &gt; </TD> <TD>     &gt;  </TD> <TD>    &gt;</TD> </TR>
</table>
<P>
seal gel w/ 2mls mix (-TEMED) + 2ul TEMED


<P>
<b>range of separation</B>

<P>
<table>
<TR><TD>% gel</TD> <TD> bp</TD></TR>


<TR><TD>3.5</TD> <TD> 100-1000</TD></TR>


<TR><TD>5.0</TD> <TD>   80-500</TD></TR>


<TR><TD>8.0 </TD> <TD>  60-400</TD></TR>


<TR><TD>12.0 </TD> <TD>  40-200</TD></TR>


<TR><TD>20.0 </TD> <TD>  10-100</TD></TR>
</table>

<P>
<b>migration of marker dyes</B>

<P>
<table>
<TR><TD>% gel </TD> <TD> BPB </TD> <TD> XC</TD></TR>


<TR><TD>3.5 </TD> <TD> 100 </TD> <TD> 460</TD></TR>


<TR><TD>5.0 </TD> <TD> 65 </TD> <TD> 260</TD></TR>


<TR><TD>8.0 </TD> <TD> 45 </TD> <TD> 160</TD></TR>


<TR><TD>12.0 </TD> <TD> 20  </TD> <TD> 70</TD></TR>


<TR><TD>20.0 </TD> <TD> 12 </TD> <TD></TD></TR>
</table>
<P>
1)Plates: Long plates for sequencing and oligo preparations, medium
for S1s and 


strand separations, short for gel shifts and restriction digests.
Clean plates w/ 


soap & H2O, rinse w/ deion H2O, then EtOH, wipe dry. Siliconize
notched plate, 


wash excess w/ EtOH. Taping is unnecessary, use 3 sided spacers
and plug w/ 2ml 


gel mix + 2ul TEMED in pasteur pipette. 



<P>
2) Gels: Acryl stocks are filter sterilized. First warm urea in
H2O to dissolve. 

<P>
Cast gels last 5-7d RT, are made 0.5 x TBE. prerun for sequencing,
unecessary for 


prep gels.

<P>
<PRE>
		[gel]
   		6%

g urea  	21.4
ml 10x TBE  	2.14
/ml acryl 	(38/2%)6.43
ml H2O  	18.21
ul 10% APS  	178.5
ul TEMED  	42.84
</PRE>

<P>
3) Samples: loading buffer is: <P>
<PRE>
		200ul formamide
		12ul 10xTBE
		32ul dyes (0.8% BPB & XLC)
</PRE>
<P>
heat to 95C 2', cool on ice, warm to RT prior to loading



<P>
4) Running: attach aluminum plate. At 2000v-2500v, short sequence
xc 27cm 2hr, 

medium xc 65cm 4hr, long xc 110cm 7hr or 1200v o/n



<P>
5) Fixing: remove notched plate

<P>
leave 15' in 20% EtOH bath

<P>
run-off dry

<P>
transfer to Whatman 3MM paper

<P>
dry at 80C under vaccum

<P>
expose w/ screen o/n

<P>
<table>
<TR><TD></TD> <TD> 5%  </TD> <TD>     6%   </TD> <TD>    8%    </TD> <TD>   12%  </TD> <TD>    16%</TD> </TR>


<TR><TD>g  urea      </TD> <TD>       30    </TD> <TD>    &gt;   </TD> <TD>      &gt;   </TD> <TD>      &gt;    </TD> <TD>  
  &gt; </TD> </TR> 
<TR><TD>ml 10x TBE </TD> <TD>        3    </TD> <TD>    &gt;  </TD> <TD>      &gt;  </TD> <TD>   &gt;</TD> <TD>    &gt;    </TD> </TR> 
 
<TR><TD>ml acryl (38/2%) </TD> <TD>  7.5 </TD> <TD>      9   </TD> <TD>      12     </TD> <TD>   19    </TD> <TD>    25</TD> </TR> 


<TR><TD>ml H2O    </TD> <TD>          27.5  </TD> <TD>    25.5  </TD> <TD>    22.5  </TD> <TD>    15.5  </TD> <TD>    9.5</TD> </TR> 


<TR><TD>ul 10% APS    </TD> <TD>      250   </TD> <TD>    &gt;  </TD> <TD>       &gt;     </TD> <TD>    500    </TD> <TD>   &gt;</TD> </TR> 

<TR><TD>ul TEMED    </TD> <TD>       60    </TD> <TD>   &gt;      </TD> <TD>  &gt;   </TD> <TD>     &gt;   </TD> <TD>  
  &gt;</TD> </TR> 
</table>
<PRE>

</PRE>

<H4><a name="ccca">E COLI COMPETENT CELLS (CACL2)</a></H4>


1) Grow 5 ml culture o/n w/ or w/out antibiotics depending uponstrain


<P>
2) Inoculate 500ml media, grow to 0.5OD600

<P>
3) Spin 10', 3000 rpm

<P>
4) Resuspend in 165 ml 0.1M CaCl2

<P>
5) Stand 20' on ice

<P>
6) Spin 10', 3000rpm 

<P>
7) Resuspend in 30ml 0.1M CaCl2, 20% glycerol

<P>
8) Freeze in 0.2ml aliquots in LN2



<P>
<H4><a name="ccru">E. COLI COMPETENT CELLS (RUCL2)</a></H4>

Double up the following protocol so you get plenty

<P>
The eppis used in step 9 should be pre-chilled at -80C



<P>
1. Streak bacteria on fresh plate and grow O/N

<P>
2. Pick 5-6 fresh colonies and disperse in sterile eppi w/ 1ml
SOB

<P>
3. Inoculate 100ml SOB in sterile 1l flask. Grow 37C 2-3hr to
OD595=0.2 (low 


density critical)

<P>
4. Spin cells in four 50ml sterile screw cap tubes at 2500rpm
15* 4C

<P>
5. Pour off sup, invert tubes briefly to remove excess liquid.
Resuspend pellets 


in 8ml RF1/tube (1/3 vol.)

<P>
6. Inc on ice 15&#130;

<P>
7. Spin cells 2500rpm 15&#130; 4C

<P>
8. Repeat 5 and resuspend pellet in 1ml RF2/tube (1/25 vol.) and
ice cells 15&#130;

<P>
9. Aliquot 100ul well suspended cells in eppis and freeze immediately
in lN2. 

<P>
Store at -80C



<P>
SOB 500ml (use within 2-3 weeks)
<PRE>
Bactotryptone 		10g
Yeast extract 		2.5g
NaCl   			292mg
Kcl   			0.9g
H2O   			to 500ml
</PRE>

<P>

RF1 100ml
<PRE>
RbCl   			1.2g
MnCl4H2O  		0.99g
KOAc   			3ml of 1M pH7.5 (adjusted w/ KOH)
CaCl2 2H2O  		15g

Adjust to pH5.8 w/ 0.2M Hac. Filter through 0.22um. Store RT
</PRE>

<P>
RF2 50ml
<PRE>
RbCl   			60mg
MOPS   			1ml 0.5M pH6.8 (adjust w/ NaOH)
CaCl2 2H2O  		0.55g
Glycerol  		7.5g

Adjust to pH 6.8 w/ NaOH. Filter through 0.22um. Store RT
</PRE>


<P>
<H4><a name="colitrans">E COLI TRANSFORMATION</a></H4>

1) thaw competent cells (-70 C stored) on ice, invert to mix

<P>
2) add 150ul cells to DNA samples in 13ml tubes on ice

<P>
3) inc 25' on ice with occassional mixing

<P>
4) heat shock 5', 37 C

<P>
5) inc ice 5'

<P>
6) add 1ml LB &#137;without antibiotics_, shake 1hr 37 C

<P>
7) spin 30', asp to 200ul, plate 100ul, store the rest



<P>
For blue/white screen, add IPTG and X-Gal to plates before starting
transformation

<P>
200ul 100mM IPTG (0.2g to 8.3ml H2O, 0.22 filter

<P>
62.5ul 4% X-Gal  (0.4g to 10ml DMF, 0.22 filter

<P>
store both at -20C, best if aliquoted. Do not mix before use.




<P>
Positive control uses 10ng supercoiled plasmid



<P>
<H4><a name="endlabel">END LABELLED PROBE PREPARATION</a></H4>

<P>
1) Digest plasmid w/ 5'overhang enzyme

<P>
2) P/CHCl3 ext

<P>
3) EtOH ppt

<P>
4) Redissolve to 0.5mg/ml in H2O



<P>
5) Start label rxn:



<P>
3ul digested plasmid (1ug)

<BR>
7ul H2O

<BR>
2ul each a&#169;32P dNTPs

<BR>
2ul 10x medium salt restriction buffer

<BR>
1 ul klenow

<BR>
15' RT



<P>
6) Add 2ul imM cold dNTPs

<P>
7) 15' RT

<P>
8) Add 35ul TE

<P>
9) P/CHCl3 ext

<P>
10) EtOH ppt



<P>
11) To isolate probe, digest w/ 2nd enzyme to free probe

<P>
12) Separate fragments by 6% non-denaturing PAGE

<P>
<H4><a name="filterwash">FILTER WASHING</a></H4>

1) Remove filter from hyb solution directly into LSWB at RT C


<P>
wash 15'

<P>
2) Discard & rewash w/ 500ml LSWB at RT C

<P>
3) Discard & wash 3x w/ LSWB 15' at 60 C (see below)

<P>
5) Check filter with counter. If more than 10, proceed to 5),



otherwise to 8)

<P>
6) Discard and wash 3x w/ HSWB at 60 C

<P>
7) Check filter w/ counter. Should be less than 10.

<P>
8) Blot dry briefly, pack in clingfilm. Tape pack in cassette
w/ End labelled 


probe preparation_



<P>
1)digest plasmid w/ 5'overhang enzyme

<P>
2)P/CHCl3 ext

<P>
3)EtOH ppt

<P>
4)redissolve to 0.5mg/ml in H2O



<P>
5) start label rxn:



<P>
3ul digested plasmid (1ug)

<P>
7ul H2O

<P>
2ul each a&#169;32P dNTPs

<P>
2ul 10x medium salt restriction buffer

<P>
1 ul klenow

<P>
15' RT



<P>
6)add 2ul imM cold dNTPs

<P>
7)15' RT

<P>
8)add 35ul TE

<P>
9)P/CHCl3 ext

<P>
10)EtOH ppt



<P>
11)to isolate probe, digest w/ 2nd enzyme to free probe

<P>
12)separate fragments by 6% non&#169;denaturing PAGE (see page.
ptc)



<P>
 screens and phosphorink spot tapes to orient.

<P>
9) Insert film (AXR5 Kodak) and store at -70 C till development.




<P>
LSWB 2X SSC, 0.1% SDS

<P>
HSWB 0.2X SSC, 0.1% SDS

<P>
The easiest way to heat these solution is to submerge large,


double zipper plastic bags in water bath.



<P>
The temperature used varies for different probes and samples.
Make sure you know 


what tempt to use. 60 C is standard for homologous DNA/DNA/RNA
hybs.



<P>
<H4><a name="fluorPAG">FLUOROGRAPHY OF PAG</a></H4>
1) fix as usual

<P>
2) rinse 2-3x1h or overnight in 50% EtOH

<P>
3) incubate 1hr in DMSO in fume hood in metal/glass tray. If the
gel starts 


turning white, put in back in 50% EtOH and wash longer. You can
keep reusing the 

<P>
DMSO, but the gel should see some fresh DMSO each time before
it goes into the 


DMSO/PPO.

<P>
4) rinse 2 x 1hr in 20% PPO/DMSO (poisonous), then 1hr in 20%
PPO/DMSO  with 3% 


glycerol. These solutions can be kept separately and used in the
same order 


several times.

<P>
5) soak in H2O w/ washing 10' to ppt PPO in gel. With gloved hand
gently wipe gel 


surface free of excess PPO. Plenty of water and gentle agitation
required here.

<P>
6) dry gel on cellulose sheets (Biorad) above thick paper 60C,
2-3h w/ good 


vacuum. Every geldryer/vaccuum system is different. You need a
good setup for 


this. Handle the dried gel carefully and tape it to the cassette
screens. The gels 


curl like crazy when frozen

<P>
7) expose at 70C to prevent swelling



<P>
<H4><a name="Fluoromet">FLUOROMETER/LUMINOMETER WALLAC VICTOR II</a></H4>
Samples for both Glucuronidase (MUG) and Luciferase assays should
be prepared 


according to the &quot;GUS & LUC bombardment assay&quot; protocol.




<P>
1. Switch the Victor II and the computer on. Windows 95 launches



automatically the &quot;Wallac 1420 manager&quot; program.



<P>
2. In the &quot;Tools&quot; menu, check that the &quot;User level&quot;
is set on &quot;Advanced&quot;. 


Otherwise, the &quot;Routine&quot; menu would disable any system
operation which means 


that you could not change any setting, even in your own protocols.




<P>
3. Select a protocol available in the list in the &quot;Instrument
control&quot; menu 


(then go to step 5) or create your own protocol (see step 4).




<P>
4. Creation of a new protocol menu



<P>
4.1 Click with the left button (LB) on &quot;Explorer&quot; in
the &quot;Tools&quot; menu in 


order to launch the &quot;Wallac 1420 Explorer&quot; program.
By clicking with the 


right button (RB) on the &quot;Users&quot; folder, you will be
able to create a new 


protol and then name it.



<P>
4.2 Open your new protocol (double-LB). This operation starts
the &quot;Protocol 


Editor&quot; window which should be left open all the time you
will work with the 


Victor II. This program enables you to define all the operations
to perform 


(see step 4.4), to select which wells will be measured (step 4.3)
and other 


things such as the format of the saved files, etc.



<P>
4.3 Select the wells which should be measured in the &quot;Sample&quot;
menu (LB), the 


default setting is 96 wells. NB: we use the standard 96 wells
white 


microtiter plates.



<P>
4.4 Define the operations to perform in the &quot;Measurement&quot;
menu. Select 


measurement &quot;by plate&quot;. To add an operation, you can
either click (LB) on the 


small icons displayed on the left or click in the white operation
window 


(RB). The following operations can be chosen : dispense (if injectors



installed), delay (to wait inbetween the measurement of two wells),
shake, or 


label (to select the kind of detection : fluorometry, luminescence,
...). To 


perform a MUG assay, select &quot;label&quot; (LB) then, in the
&quot;fluorometry&quot; menu, 


select the icon represented with a locker &quot;Umbelliferone
(1.0s)&quot; (LB). You 


have now the possibility to select this protocol (double-LB) and
use the 


default Umbelliferone measurement or make a &quot;copy&quot; of
this protocol with an 


other name. Important : you can not change the settings of any
default 


protocol represented as an icon with a locker. If you want to
change some 


parameters (such as the counting time), you have to create a new
protocol (by 


copying a default one). The default &quot;Umbelliferone (1.0s)&quot;
protocol is well 


adapted to our MUG assays.



<P>
4.5 Save your protocol which should now appear in the listing
of the &quot;Wallac 


1420 Manager&quot; window in the &quot;Instrument control&quot;
menu. Select it (LB)



<P>
5. Run your protocol by pressing &quot;Start&quot;. You can have
a &quot;Live display&quot; of 


the measurement. I would recommend to try your protocol once with
an empty 


microtiter plate to check if everything works fine (good connexion
with the 


Victor II, no mistake when you selected which wells should be
measured,...).



<P>
6. How to run a MUG assay : mix both the extract and the substrate
(MUG 


buffer + methanol) according to the related protocol (GUS &
LUC bombardment 


assay) but do not stop the reaction with CaCO3. Indeed, you can
measure the 


activity at various time (t0, t1h, t2h,...) from the same wells.
To do so, 


incubate the microtiter plate at 37C in an incubator (remember
to cover the 


plate to avoid evaporation). It is recommended, especially when
the GUS 


activity of the sample is totaly unknown, to make a series of
dilution 


(dilute the extract in the extraction buffer) to check wether
activities will 


be proportional.



<P>
7. The data will be automatically saved in the same folder than
your protocol 


(classified upon the date). You can open them (double-LB) then
export them as 


an Excel file or other formats.



<P>
8. Leave both Fluorometer and computer on during the week. Switch
them off 


during the weekend.



<P>
<H4><a name="Footprint">FOOTPRINTING WITH DNASE1</a></H4>

1)probe: same as used for gelshift), isolated by isotacelectrophoresis
w/out EtOH 


ppt which can enature dsDNA



<P>
2)probe mix/rxn: volumes x # samples

<P>
1ul    probe (0.1&#169;0.5ng or 10&#169;20kcpm)

<P>
0.15ul 20mM EDTA

<P>
0.4ul  10ug/ul dIdC or dAdT (from gel shift assay)

<P>
0.5ul  H2O



<P>
3)DNAse mix: made up near end of binding incubations. DNAse l(Worthington
DPFF,Cat 

<P>
#LS0006330, lot #58A047,5mg) is 1mg/ml in150mM NaCl, 50% glycerol,
store at &#169;20C. 

<P>
Try 3 different [s] ofDNAse mix (A,B,C)

<P>
1,2 & 3ul stock DNAse1

<P>
2 ul      1M MgCl2

<P>
&#169;&gt; 100ul  H2O



<P>
4)binding rxn: components titrated & optimized by gel shiftassays


<P>
2ul    probe mix

<P>
Xul    extract

<P>
&#169;&gt;18ul NEB (see nucprp.ptc)

<P>
30' RT



<P>
5)DNAse rxn: add 2ul DNAse mix to binding rxn

<P>
inc 1' RT

<P>
stop w/ 100ul DNAse stop mix: 

<P>
<PRE>
                   	stock/50ml

6M Urea 		18g
0.4M NaCl           	6.6ml 3M
1% SDS            	5ml 10%
20mM EDTA           	4ml 250mM
10mM Tris 8         	0.5ml 1M
0.8M NH4OAc         	5ml 8M
10ug/ml glycogen       	50ul 10mg/ml
</PRE>


<P>
5)P/CHCl3 ext

<P>
6)EtOH ppt



<P>
7)PAGE: Resuspend carefully in 8ul sequencing sample buffer (5'vortex,
5' 60C, 1' 

<P>
vortex, 2' 90C, spin, transfer to new tube,count cpm). Load equal
counts on 6% or 


gradient sequencing gel.  

<P>
Notes: If extract inhibits DNAse, add 0.1&#169;0.3ul extra DNAse
mixto binding rxns. 

<P>
DNAse requires  Mg, some factors are inactivatedby it! Remember
ug/KB x 0.66 = 


picomole thus 1ng of 300bp probe =2 femptomole.



<P>
<H4><a name="Gelshift">GELSHIFT</a></H4>

1)Nuclear extract: see nucprep.ptc & nucext.ptc


extracts should be at least 3ug/ul



<P>
Probe preparation: see endlabl.ptc & isotach.ptc. probe shouldbe
10&#169;20k cpm/ul. 

<P>
Fragments larger than 400bp should not be used. Make A stock (25ug/50ul)
of probe 


plasmid digested at one end w/ 5' overhang.



<P>
3)Binding rxn: 

<P>
1-2ul  probe (0.5ng or 20k cpm in isotach 40mM Tris 7.5 buffer)


<P>
1-2ul  poly dIdC or dAdT (3ug/ul in NEB, sonicated to 300&#169;500bp)


<P>
1-6ul  extract (5&#169;10ug/ul in NEB)

<P>
&gt;10ul NEB (see nucext.ptc)

<P>
incubate 30' RT  

<P>
1ul sequencing dyes immediately prior to loading under tension




<P>
4)Titrations: Start w/ extract titration at 3ug/rxn poly dIdC.


<P>
At extract [] w/ max binding, do dIdC titration.

<P>
work for complete probe binding, none free. Try Mg salts later.




<P>
5)Competitions: Fragments should be isolated by PAGE/isotach.
10-100ng DNA/rxn is 


usually necessary.



<P>
6)gels: 

<P>
Acryl



<P>
48.5ml

<BR>
10ml   30/0.8% acryl stock

<BR>
1.5ml  10x TBE

<BR>
50ul   TEMED

<BR>
400 ul 10% APS

<BR>
run 100&#169;200v w/ circulation



<P>
Acryl/agarose gel

<P>
H2O80ml H2O 

<BR>
0.7g agarose

<BR>
boil to dissolve

<BR>
10ml 10x buffer 100mM Tris 7.5, 10mM EDTA, 30mM NaOAc

<BR>
10ml 30/0.8% acryl stock

<BR>
cool to 60C

<BR>
60ul TEMED 

<BR>
100ul 10% APS

<BR>
let set for 2hr, prerun with circulation 30' at 100v and run w/
circulation



<P>
7)Gels are dried unfixed on Whatman DE 81 sheets at 80C on dryer.Expose
o/n -70C 


w/screen.



<P>
<H4><a name="cereal">GENOMIC DNA FROM CEREAL LEAVES</a></H4>

1)extract nuclei in nuclear homogenization buffer NHB from 1kg
leaves (see nuclei 


prep steps 1-8)

<P>
2)resuspend in 35ml TE

<P>
3)add 35ml 2x nuclear lysis buffer NLB (see nucext.rcp)

<P>
4)add proteainase K to 0.2mg/ml

<P>
5)incubate 37C 1hr

<P>
6)spin 3k rpm 5' RT

<P>
7)add 1g/ml ground CsCl to supernatant

<P>
8)spin o/n vti50 45k rpm

<P>
9)puncture w/ 21 gauge needle 1cm from bottom, collect & pool
viscous fractions

<P>
10)dyalize o/n vs. 2l TE in cold room



<P>
<H4><a name="gst">GST FUSION PROTEIN PURIFICATION</a></H4>

1) grow 20ml cells O/N 37 C, dilute 50X into prewarmed LB, grow
to 0.6 OD or about 

<P>
1hr. Induce w/ 2mM IPTG (238mg/0.5l), grow 3hr, spin 6k GS3 10',
freeze at -70 C.

<P>
2) extract cells (from 500ml) in 25 mls. Heintz Buffer plus triton
(HBT) by gentle 


pipette resuspending on ice circa 10'after thawing.

<P>
3) transfer to 50ml conical ss34 flip top tubes.

<P>
4) add 10mg lysozyme powder to the 50ml (cells from 1 liter now
in 1 50ml tube), 


digest 15' on ice.

<P>
5) sonicate with large probe 1' 80% power, freeze in lN, thaw
at 37 C, sonicate 

<P>
again on ice, solution should become viscous.

<P>
6) add 1mg DNAse and RNAse, incubate on ice 15'.

<P>
7) spin 7.5k rpm 4 C ss34 10'.

<P>
8) transfer supernatants to conical screw caps, freeze in lN2,
may store.

<P>
9) Batch adsorb w/ 4ml 50% slurry GT-Sepharose (PL 17-0756-01),
1hr, 4 C, spin 2', 


4k on bench.

<P>
10) aspirate, resuspend in 25ml HBT, spin, repeat.

<P>
11) pour slurry into column (Econo 1.7x20), elute to top, then
with 20 column 


vols. of HB-T.

<P>
12) elute protein in minimal volume (5-10ml) HB+5mM GT (Sigma
G4251, 1.5 mg/ml).

<P>
13) lN2 freeze as 100ml aliquots for GS.



<P>
HB 1 liter

<P>
<table>
<TR><TD>[Final]</TD> <TD>        Stock </TD> <TD>            ml/l</TD> </TR>

<TR><TD>25mM HEPES, pH7.9  </TD> <TD>  1M </TD> <TD>         50</TD> </TR>

<TR><TD>1mM EDTA, pH 8.0    </TD> <TD>      0.5M   </TD> <TD>  2</TD> </TR>

<TR><TD>20% glycerol   </TD> <TD> stock  </TD> <TD>       200</TD> </TR>

<TR><TD>1mM MgCl    </TD> <TD> 1M   </TD> <TD>  1  </TD> </TR>

<TR><TD>60mM KCl </TD> <TD>2M       </TD> <TD>      30</TD> </TR>

<TR><TD>1% Triton   </TD> <TD>  stock    </TD> <TD>10 add before use</TD> </TR>

<TR><TD>0.5mM DTT     </TD> <TD> 1M  </TD> <TD>   0.5    &quot;</TD> </TR>

<TR><TD>0.5mM PMSF</TD> <TD> 0.5mM   </TD> <TD>  10   &quot;  </TD> </TR>

<TR><TD>5ug/ml Leupeptin     </TD> <TD>     5mg/ml     </TD> <TD>        dilute before use</TD> </TR>

<TR><TD>5mg/ml antipain     </TD> <TD>         &quot;       </TD> <TD>
&quot;</TD> </TR>
</table>
<P>
check pH!



<P>
<H4><a name="gushis">GUS HISTOCHEMICAL STAINING</a></H4>

1) determine number of slides needed, multiply by 0.75ml

<P>
2) make up required vol of stain:

<P>
for 10ml(4C) 5mg X-Gus

<P>
   50ul nn dimethyl formamide, dissolve

<P>
   + 10ml 50mM NaPO4 pH 7

<P>
3) sections best cut with vibrating knife

<P>
for sections w/ chlorophyll, put in cell-wells w/ 500ul stain


<P>
for sections w/out chlorophyll, put directly on slides w/ stain


<P>
4) inc o/n 37C in humidity chamber

<P>
5) asp, inc 10'in FAA:

<P>
for 200ml (4C) 10ml formaldehyde

<P>
   10ml HAc

<P>
   75ml EtOH

<P>
   H2O &gt; vol

<P>
6) inc 2' 50% EtOH

<P>
7) inc 2' 100% EtOH

<P>
8) inc 1' H2O



<P>
<H4><a name="gusluc">GUS & LUC BOMBARDMENT ASSAY</a></H4>

1. Bombarded tissue is incubated according to the experiment

<P>
2. grind to a fine powdcr with a mortar and pestle  in lN2

<P>
3. add 500u1 Extract Buffer l00nmM K2HPO4/kH2P04, pH 7.8 1mMDTT.
Keep on ice until 


spin 10' 4C in microfuge. Aliquot about 450ul to a fresh tube.
Freeze in lN2 and 


store -80C.



<P>
<H4><a name="guslumi">Fluorometric GUS</a></H4>

Mix pr assay 125u1 2 x MUG amd 50u1 100% Methanol

<P>
prewarm at 37C and at t0 add 75ul extract, mix.

<P>
When all have been done (up to 50) take out 50u1 aliquots to 250u1
0.3M Na2CO3 


prealiquoted into fluoroscan microtiter plates (Microstrips black
Ps, cat no. 


9502177, Lab systems). These are measured in scanner (Flurorscan
II, Labsystems, 


Finland) and the numbers are t0 values. Remember to include proper
- control 


without bombardment for background (Tback0). Depending upon the
levels of 


activity, generally incubate the rest for 24hr although you can
see a slight color 


change in 2-4 hr if levels are high. Calculate t24-T0 minus Tback24-Tback0.
If the 


values are above 5000, make dilutions as measurements are not
linear at this 


level.



<P>
Extract buffer: good for both GUS and LUC measurements

<P>
275ml O.2M K2HP04 plus ca. 20ml 0.2M KH2PO4, adjust to pH 7.8,
then add 1 vol H20. 

<P>
Autoclave. Just befrore use add DTT to lmM and leupeptin to 20ug/ml.




<P>
2 x MUG: 2mM MUG in 5OmM Na3PO4/Na2HPO4 pH 7, 10mM EDTA, 0.1%
Triton X-100, 0.1% 


Sodium Lauryl Sarkosyl, lOmM DTT. Store in aliquots at -2OC.

<P>
MUG:4-methyl-umbelliferyl B-D-g1ucuronide.

<P>
Luminometric LUC assay:

<P>
1. Mix

<P>
20ul lO x (12.5x) LUC buffer

<P>
10ul 100mM ATP

<P>
1ul 100mg/ml BSA

<P>
169ul H20

<P>
total is 200ul/special tube for luminometer

<P>
2. add 50ul extract, mix

<P>
3. Inject 100ul diluted luciferin solution and take measurement
after 5-15&quot;. This 


time interval must be constant for all samples!



<P>
10 x LUC buffer

<P>
250mM Tricine, pH 7.8

<P>
150mM MgC12.

<P>
lOOmM ATP

<P>
in H20 adjust to pH 7.0, sterile filter, store aliquots at -20C




<P>
lOmM Luciferin stock:

<P>
D(-) Luciferin lOmg (cat 411400 (Boehringer). For 10mM stock,
weigh out 1mg (light 


sensitive!), add 27ul DMSO to solubilize, then add 12ul 3M NaOAc,
pH5, mix, then 


275ul H2O, Don't make up large amountrs as this is not very stable,
even at -80C.



<P>
Diluted luciferin solution: 0.5mM in H2O



<P>
<H4><a name="guslumi">GUS LUMINOMETRIC ASSAY<a></H4>

<P>
GUS-Light is a chemiluminescent reporter gene assay system designed
for rapid, 


sensitive, and non-isotopic detectjon of B-glucuronidase in cell
extracts. The 


GUS-Light reporter gene assay incorporates GlucuronTM chemiluminescent
substrate 


and a proprietary Light Emission Accelerator. The chemiluminescent
assay has a 


wide dynamic range, enabling detecfion of 0.6pg to 2 ng of B-glucuronidase.




<P>
The B-glucuronidase detection assay is simple and fast. Cell lysate
or purified 


glucuronidase is first incubated with Reaction Buffer for 1 hour.
Glucuron 


chemiluminescent substrate which is present in Reaction Buffer
is catalytically 


decomposed by the enzyme. The sample is then placed in a luminometer
chamber and 


GUS Accelerator is added which terminates the B-glucuronidase
activity and 


accelerates the emission of light. Chemiluminescence signal intensity
is measured 


as a 5 second integral. The amount of cell extract used in the
assay should be 


adjusted to keep the assay within the linear range. High intensity
signals may 


saturate the detector of a luminometer resulting in artificially
low values.



<P>
The GUS-Light system has been formulated for luminometers equipped
with automatic 


injectors. Manual injections may be performed, however signal
intensities should 


be measured within approximately the same interval following the
addition of GU5 


Accelerator to each sample. Reaction components should be scaled
down if a 


luminometer with a smaller volume injector is used, however sensitivity
may be 


affected slightly. Alternate lysis buffGrs mav be used, however
we recommend that 


their performance is compared with the CUS Lysis Buffer to ensure
optimum results 


of the assay. A Lysis buffer compatible with the luciferase assay
containing 0.1M 


potassium phosphate, 1mM DTT, and 1mg/ ml BSA has been tested
with equivalent 


performance to the GUS Lysis Buffer. 



<P>
Bacterial contamination of plant material will cause high background.
Best results 

<P>
will be obtained with sterile preparations. Chlorophyll in concentrated
samples 


may interfere with the chemiluminescent signal intensity. Therefore,
if high 


levels of chlorophyll are present, several dilutions of extract
should be assayed.



<P>
II. SYSTEM COMPONENTS




Each GUS-light (Cat. No. Bg100) contains rtagents sufficient for
200 tests. GUS-


Light (Cat. No. BG30O) contains reagents sufficient for 600 tests.
Glucuron 


Chemiluminescent Substrate A l00X concentrate is diluted in GUS
Reaction Buffer 


Diluent prior to use.



<P>
GUS Lysis Solution contains 50 mM sodium phosphate pH 7, 10 mM
EDTA, 0.1% sodium 


lauryl sarcosine, 0.1% Triton X-100 (Store at 4'C). Fresh B-mercaptoethanol
should 


be added prior to use to a final concentration of 10mM (i).



<P>
GUS Reaction Buffer Diluent contains 0.1MNaPO4 pH7.0, 10 mM EDTA
(store at 4 C)




GUS Accelerator contains a ready to use luminescent accelerator
(store at 4C)



<P>
III.PROCEDURE FOR B-GLUCURONIDASE DETECTION



<P>
Positive Control: Add 1ul B-glucuronidase (20 pg of B-glucuronidase,
Sigma G-7896) 


to mock extract equivalent to the volume of experimental extract
used. 

<P>
Negative Control: Assay a volume of mock extract equivalent to
the volume of 


experimental extract used. 



<P>
It is recommended that all assays are performed in triplicate
(step 6).



<P>
1. Grind tissue to a powder in lN2. These can be stored at -80C.


<P>
2 Aliquot the required amount of GUS Lysis Buffer. Add fresh B-mercaptoethanol
to 


a final concentration of 10mM. Note that the lysis buffer for
GUS/LUC can also be 


used.

<P>
3 Add sufficient volume of GUS Lysis Solution to cover the tissue
(250 uL of Lysis 


Buffer per 25 mg of plant material).

<P>
4. centrifuge sample in a microfuge for 2 minutes to pellet debris.


<P>
5. Transfer supernatant to a fresh microfuge tube.

<P>
6. Dilute Glucuron substrate l00 fold with GUS Reaction Buffer
Diluent to make GUS 


Reaction Buffer. This mixture will remain stable for several weeks
if stored 


uncontaminated at 4'C. It is recommended to only dilute the amount
of Glucuron 


substrate that will be used within a one month period.

<P>
7. Warm the volume of GUS Reaction Buffer required for the entire
experiment to 


room temperature.

<P>
8. Aliquot 2 to 20uL of individual cell extracts into luminometer
sample tubes. If 


less than 20 uL of sample is used, Lysis Solution should be added
to 20 uL final 


volume. Note: The amount of extract required may vary depending
upon the degree of 


expression and the specific luminometer utilized. Use 5ul of extract
for positive 


controls and 10 to 20ul of extract for experiments with potentially
low levels of 


enzyme. It is important to vary the concentrations of extract
in order to record 


the signal within the linear range of the assay.

<P>
9. Add 180 ul of GUS Reaction Buffer to a luminometer tube and
mix gently. 

<P>
Incubate for 60 minutes at room temperature. Incubations may be
as short as 15 


minutes (especially if high levels of expression are expected),
but the dynamic 


range of the assay may decrease. Note: Light intensities are time
dependent. 

<P>
Reaction Buffer should be added to sample extracts within the
same time frame as 


they are measured in the luminometer. For example, if it takes
10 seconds to 


complete a measurement. then Reachon Buffer should be added to
tubes every 10 


seconds.

<P>
9. Place tube in a luminometer. Inject 300ul of GUS Accelerator.
After a 2 to 5 


second delay following injection, count the sample for 5 seconds.
If manual 


injection is used, then the Accelerator should be added in the
same consistent 


time frame as the addition of Reaction Buffer.

<P>
REFERENCES
<P>



1.Callagher, S.R. ifl &quot;GUS Protocols: Using the GUS Gene
as a Reporter of Gene 


Expression&quot;, Ed. S.R. Callagher, 1992, Academic Press, 47-59.




<P>
Fulton, R., and B. Van Ness. Luminescent reporter Gene Assavs
for Luciferase and 


B-galactosidase Using a Liquid Scintiliation Counter. BioTechniques
14(5): 762-763 


(1993).


<P>

3. Nguyen, V.T., M. Morange, and OBensaude. Firefly Luciferase
Luminescence Assays 


Using Scintillation Counters for Quantitation in Transfected Mammalian
Cells. 


Anal. Biochem. 171, 404-8 (1988).



<P>
Manufacturer: TR&Oslash;PIX, Inc., 47 Wiggins Avenue, Bedford,
Massachusetts 01730 (617) 


271-0045 or (800) 542-2369 FAX (617) 275-8581. 



Local AH diagnostics 86 101055



<P>
<H4><a name="his">HIS-TAGGED PROTEIN PURIFICATION</a></H4>

1 liter cell prep

<P>
1) grow 20ml cells O/N 37 C, dilute 50X into prewarmed LB, grow
to 0.6 OD or about 


1hr. Induce w/ 2mM IPTG (238mg/0.5l), grow 3hr, spin 6k GS3 10',
freeze at -70 C.

<P>
2) resuspend cells (from 500ml) in fliptop in 25 mls. UPB8.0,
freeze lN2.

<P>
3) thaw at 37 C, sonicate 30&quot; high, freeze lN2, thaw.

<P>
4) spin RT 10' 15k ss34, respin in fresh tube 10' 15k ss34.

<P>
5) decant again to new tube, add 5ml 50% Ni-NTA resin, mix gently
RT 30'.

<P>
6) spin, decant and wash resin 3x in 50ml UPB6.3 w/5' gentle shaking
each time.

<P>
7) resuspend in 25ml UPB6.3, pour into column and elute to top
with another 25ml.s

<P>
8) elute bound proteins 20ml w/ UPB6.3+250mM imidazole.

<P>
9)lN2 freeze and store at -70 C.



<P>
UPB 1 liter

<P>
<PRE>
[final]    		stock   	ml/l

8M UREA    		solid   	480g
10mM TRISHCl 8.0 or 6.3 1M   		10
0.1M Na-PO4		0.2M pHed   	500
1mM bme    		14.27   	70ml
250mM imidazol          solid           0.34g/20ml.
</PRE>


<P>
<H4><a name="hormone">HORMONE TREATMENTS</A></H4>

For aleurones, place layers from 100 seeds in 9cm petri w/ 20ml


<P>
aleurone solution:

<P>
10mM CaCl2 (1.47g/l)

<BR>
1X antibiotic solution



<P>
100X antibiotic stock:

<P>
5mg/ml ampicillin

<BR>
10mg/ml neomycin

<BR>
1mg/ml nystatin



<P>
GA 7mg/1ml EtOH, warm at 37C to dissolve, use 20ul/20ml aleurone
solution



<P>
ABA 5mg/1ml EtOH,  ditto



<P>
cycloheximide, make 5mg/ml stock in EtOH, use 40ul/20ml aleurone
solution



<P>
chloramphenicol, make 50mg/ml stock in EtOH, use 40ul/20ml aleurone
solution



<P>
<H4><a name="horminhi">HORMONES & INHIBITORS</a></H4>



<P>
For protoplasts, dilute stock first 100X in media, then 10x to
protoplasts in 


media.



<P>
<b>Phytohormones</b>



<P>
ABA: [active] 50mM, stock 50mM, MW 264, stock =


Sigma A1012 13.2mg/ml EtOH, dilute 1000x



<P>
GA: [active] 10mM, stock 10mM, MW 346, stock =


Sigma G7645 34.6mg/ml EtOH, dilute 1000x



<P>
Me-JA 98% solution, 5ul/1ml EtOH, 100ul/9.9ml Inc. Firmenich Geneva
buffer/media, 


dile 10 or 100x.



<P>
<b>Protein synthesis inhibitors</b>



<P>
cycloheximide: [active] 20mM, stock 10mM, MW 281, stock =


Boe 103675 2.8mg/ml EtOH, dilute 1000x



<P>
chloramphenicol: [active] 100mM, stock 100mM, MW 323, stock


Sigma C0378 = 32.3mg/ml 50% EtOH, dilute 1000x



<P>
anisomycin:  [active] 60mM, stock 60mM, MW 265, stock =


Sigma A9789 15.9mg/ml 50% EtOH, dilute 1000x



<P>
<b>2nd messengers/inhibitors</b>



<P>
dibut cA/GMP: [active] 1mM, stock 100mM, MW 469, stock =


Sigma B1381 47mg/ml H2O, dilute 100x



<P>
Ca ionophore:  [active] 5mM, stock 5mM, MW 524, stock =


Sigma 7272 2.6mg/ml DMSO, dilute 1000x



<P>
PMA:  [active] 10mM, stock 10mM, MW 616, stock =


6.1mg/ml DMSO, dilute 1000x



<P>
Froskolin: [active] 10mM, stock 10mM, MW 410, stock =


4.1mg/ml DMSO, dilute 1000x



<P>
<H4><a name="hybselc">HYBRIDIZATION SELECTION OF CLONED DNA</a></H4>


1a) dsDNA, 50up plasmid DNA denatured and randomlycleaved by boiling
in 0.33 N 


NaOH 5', cool on ice, bringto 2M NH4OAc



<P>
1b) ssDNA, 250ug ssDNA is boil 5' and diluted to 2MNH4OAc




<P>
2) filter onto wetted Nc disks using cutaway filterholder

<P>
3) wash w/ 10 vol 1M NH4OAc

<P>
4) bake 80C 2h vacuum

<P>
5) cut filter into 6&#169;8 pieces w/ sharp razor on parafi&#136;lm,
careful for flying 


pieces!

<P>
6) inc in 1ml prehyb buffer PB in round bottom 1ml nunctubes o/n
42C

<P>
7) aspirate, add in 100ul hyb buffer HB

<P>
8) warm to 65C, cool to 42C and add 1ul RNAsin

<P>
9) inc 3-6h 42C

<P>
10)low stringency wash

<P>
   2 x w/ 1x SSC, 0.5% SDS, RT, 10'

<P>
   3 x w/ 0.1x SSC, 0.1% SDS, RT 10'

<P>
   1 x w/ 0.1x SSC, 0.1% SDS, 50C 10'

<P>
   high stringency wash if needed

<P>
   1 x w/ 1x TE, 60C 10'

<P>
11) remove filters quickly to eppis, add 300ul H20, boil


<P>
12) remove to new eppi on ice, add 50ug tRNA carrier

<P>
13) EtOH ppt

<P>
14) resuspend in 4ul H20, translate 2ul, load half of this



<P>
Filters:S7S BA 85 13mm, 0.45um, Cat # 4020260



<P>
Cutaway holder: Millipore filter holder Cat # xx3001200.This unit
has has had one 


end cut away making it a funnel.

<P>
Solutions: DEPC treated H20, autoclave


<P>

PB<PRE>				stock  		/2.5ml

50% formamide      				1.25ml
10mM Pipes 6.5    		200mM  		125ul
0.4M NaCl     			5M  		200ul
4mM EDTA     			0.25M  		40ul
500ug/ml wheat germ tRNA*
20ug/ml polyadenylic acid  	10mg/ml 	5ul
0.2% SDS     			10%  		50ul
*exhaustively PCHCl3 exted
</PRE>


<P>
HB
<P>
<PRE>
same as PB but
100ug/ml tRNA
Polyadenylic acid
10-20 ul Poly A+ mRNA/filter
</PRE>


<P>
<H4><a name="imuprecip">IMMUNOPRECIPITATION FROM IN VITRO TRANSLATION </a></H4>

<P>
1) count net inc (TCA filters)

<P>
2) total product lanes require 100kcpm for 2-3 day exposure. Total
product 


aliquots can be cleared of charged globin as follows:

<P>
 make up to 10 or 20ul w/ TE

<P>
 add 2 vols sated AmSO4

<P>
 mix, stand on ice 30'

<P>
 spin 5' 4C, aspirate sup

<P>
 resuspend pellet in SDS-sample buffer

<P>
3) aliquot 200k-5x 106 cpm for imm.ppt. depending upon antigen.
500kcpm good for 


starters. Dilute to 600ul w/ Tris-triton buffer (TTB). Other buffers
better for 


membrane proteins (see Anderson & Blobel, Methods in Enzymol.
vol 93, 111).

<P>
4) + 10-15ul first antibody, usually pre-immune or normal sera
(Dako #902-3) to 


bind non-specific proteins.

<P>
5) inc w/ slow mixing 1h RT

<P>
6) + 50ul 1:1 Protein A Sepharose (PAS) in TTB. resuspend before
pitteting with 


cut-off p200

<P>
7) inc 30' RT 

<P>
8) spin 1' RT

<P>
9) transfer sup to new tube, + 15ul 2nd antibody (specific for
antigen), inc 1hr 


RT.

<P>
10) pellet (proteins bound non-specifically to pre-immune serum
and Sepharose) is 


washed 3 x w/ TTB and 3 x w/ tris buffer (TB). aspirte final pellet
to dryness w/ 


hamilton, resuspend in 45ul SDS-sample buffer

<P>
11) specific antigen in  supernatant from step 9 is adsorbed after
1h w/ PAS for 


30' as before, then washed and prepared for SDS-PAGE as before
(steps 7-10)



<P>
Note: depending upon antigen, it is possible to continue specific



immunoprecipitations w/ a third (second specific) antibody. In
this case, the 


supernatant obtained in step 11 is adsorbed again w/ PAS before



immunoprecipitating out the second antigen



<P>
Solutions:

<P>
PAS swell 1h in TTB before use


<P>
<PRE>
TTB     		/l

50mM Tris 7.5   	50ml 1M
150mM NaCl   		50ml 3M
10mM Methinonie  	1.5g
2% Triton   		20ml


TB

10mM Tris 7.5   	10ml
</PRE>


<P>
SDS-sample buffer (see SDS-PAGE)



<P>
<H4><a name="isotacho">ISOTACHOPHORESIS PROBE ISOLATION</a></H4>

<P>
1)assemble plastic column pieces

<P>
2)pack with Sephadex G50 fine running buffer

<P>
3)equilibrate with 40mM Tris pH 7.5 buffer

<P>
4)secure dialysis tubing with a luer ring or slice of tubing

<P>
5)assure there are no trapped bubbles!

<P>
6)remove top buffer to bed top

<P>
7)load gel slices on top

<P>
8)add 5ul sequencing dyes to slices

<P>
9)overlay w/ 100mM 6&#169;aminocaproic acid (Sigma A2504)

<P>
10)stick column bottom into lower Tris buffer reservoir

<P>
11)connect up to power as shown

<P>
12)run at 3mA (100-300v) 

<P>
13)current may vary, adjust voltage occassionally

<P>
14)knock bubbles from upper(&#169;) electrode occassionally

<P>
15)stop current when dye is 1.5cm from bottom,

<P>
16)replace caproic acid w/ Tris buffer

<P>
17)remove dialysis tubing and elute DNA, 2 drops/eppi



<P>
DNA elutes before dye so careful!



<P>
columns

<P>
Kontes #220160&#169;0000 col. comp. poly 200mm pkg/100

<P>
Kontes #420162&#169;0000 poly bed support filter pkg/100



<P>
dialysis tubing

<P>
Spectropore cutoff 3500 in TE, no azide!



<P>
<H4><a name="vitrotranscr">IN VITRO TRANSCRIPTION</a></H4>

<P>
see Contreras et al. (1982) NAR 10: 6353-63 and Stratagenes Bluescript
(pSK) 


protocols. Stratagene's in vitro transcription kit with protocols
works fine. Use 


CsCl purified plasmids with T7 and/or T3 promoter(s). ORF should
have a 5' ATG 


with a good ribosome-binding site (Joshi (1987) NAR 15: 6643-53)
and a 3' stop 


codon. If plasmid lacks T3/7 terminator, linearize at 3' end of
ORF with a 5' 


overhang enzyme 0CHCl3 ext., and EtOH ppt.



<P>
5 ul  5x TB (250mM Tris 7.9, 30mM MgCl2, 10mM Spermidine, 50mM
NaCl)

<P>
X ul  water to 24 ml total

<P>
1-5 ug digested plasmid DNA

<P>
1 ul  10mM rATP

<P>
1 ul  10mM rUTP

<P>
1 ul  10mM rCTP

<P>
1 ul  0.1mM rGTP

<P>
3 ul  2mM 7mGpppG (Pharmacia 27 4635 01)

<P>
1 ul  0.75M DTT

<P>
1 ul  RNAsin (40U/ml)

<P>
1.5 ul T3 or T7 Polymerase

<P>
inc 37 C 5', then add

<P>
1 ul  10mM rGTP

<P>
inc 37 C 20'. This mix can be frozen away for extended periods
and aliquots used 


to program in vitro translation. For RNA probes, DNAse the plasmid,
0CHCl3 and 


EtOH ppt.



<P>
<H4><a name="vitrotransl">IN VITRO TRANSLATION</a></H4>

<P>
Use Promega's nuclease-treated -Met (L4210) or -Cys (L4240). The
lysate is the 


same, just the amino acid mixes differ. The Promega protocols
work fine as 


follows, although it is often best to scale them down.



<P>
35 ml lysate

<P>
7 ml  H2O (50 ml final depending on mRNA)

<P>
1 ml  RNAsin

<P>
1 ml  1mM amino acid mixture

<P>
2 ml  mRNA (1 mg or less)

<P>
4 ml  Met or Cys (1200Ci/mMol, 10mCi/ml)

<P>
inc 37 C 60', place on ice

<P>
spot 1 ml on Whatman 1MM, air dry on tin foil

<P>
boil in 10% TCA 10', sink filters with ice, rinse 2 x quickly
in tap water, once 


in EtOH and once in acetone, air dry. Count with 5ml Econofluor.




<P>
SDS-PAGE ANALYSIS



<P>
Take desired volume of mix and dilute to 20 ml w/ water. Add 40ml
 of cold, 


saturated AmSO4 (66% final), mix, spin and inc on ice at least
30'. Spin 10' at 


4 C, aspirate supernatant completely off and resuspend pellet
in SDS sample 


buffer. This ppt. reduces background significantly. Follow SDS-PAGE
protocol (see 


note for Cys translations).



<P>
<H4><a name="lamdaDNA">LAMBDA DNA BY PLATE LYSATE</a></H4>

<P>
for 4 large plates, 50-100kpfu/plate:

<P>
1) pool 12ml o/n washes

<P>
2) spin out cells 10k, 15' RT

<P>
3) + to sup: 20ul DNAse 1 (10mg/ml) and 50ul RNAse (10mg/ml)/
10ml sup

<P>
4) inc 30', 37C

<P>
5) transfer to 150ml corex bottles

<P>
6) + 1 vol 2M NaCL/20% PEG

<P>
7) inc 30' on ice

<P>
8) spin 7k 20' 4C

<P>
9) drain & wipe well

<P>
10)resuspend in 5ml TE

<P>
11)transfer to 15ml corex

<P>
12)+ 150ul .25M EDTA, 150ul 10% SDS

<P>
13)+ 0.5ml Proteinase K (10mg/ml)

<P>
14)inc 65C, 20'

<P>
15)P ext

<P>
16)PCHCl3 ext

<P>
17)CHCl3 ext

<P>
18)EtOH ppt -70C

<P>
19)resuspend in 4ml TE, add 4g CsCl, 100ul EtBr

<P>
20) spin in small quickseal vTi65 55k o/n



<P>
Plates are: NZY for 1 liter

<P>
5g NaCl

<P>
2g MgSO4 7H20

<P>
5g yeast extract

<P>
10g NZ amino acids (gibco tryptone, casein hydrolysate peptone
140)

<P>
15g difco agar



<P>
top agarose is for 200ml

<P>
1.6g NaCl

<P>
1.2g agarose litex

<P>
2g tryptone



<P>
<H4><a name="lamdalys">LAMBDA DNA BY LIQUID LYSATE</a></H4>

<P>
1) Add cored single plaques (1 plaque = 107 phage) to 0.5ml fresh
cells (ON 


culture in LB w/ 10mM MgSO4 & 0.2% maltose) in 50ml copolymer
tubes. Make also 


mock - plaque.

<P>
2) inc. 15' 37C

<P>
3) add 5ml LB + 5mM CaCl2

<P>
4) Shake hard 37C 4-5hrs until cleared w/ cell debris -compare
to control

<P>
5) add 100ul ChCl3, shake 5' 37C

<P>
6) spin 3000rpm table top 10'

<P>
7) transfer sup to corex, add 4ml 20% PEG,2M NaCl, invert well,
inc 60' on ice.

<P>
8) spin 3000rpm tabletop 20min, drain and wipe tube dry.

<P>
9) resuspend pellet in 750ul LB, transfer to eppi, add 750ul LB/DE52,
invert 20-


30x.

<P>
10) spin 5', transfer sup, spin again, transfer sup.

<P>
11) add 7ml sup 13ul 0.1mg/ml Poteinase K, 32ul 10% SDS, mix inc
RT 5'

<P>
12) add 130ul 3M KOAc, inc 88C 20', cool on ice 10', spin 10'
4C

<P>
13) transfer 800ul and add 800ul -20C isopropanol, inc -70C 15',
spin 10'

<P>
14) decant well and briefly speedvac pellet, resuspend in 20ul
TE.



<P>
LB/DE52

<P>
Wash (mix/settle/aspirate) 50g Whatman DE52-cellulose in 100ml
blue cap with 0.05 

<P>
N HCL till pH under 4.5. Add 10n NaOH dropwise w/ stirring till
pH 7. Wash 3x w/ 


LB. Add LB to 25% final vol w/ 0.2% NaN3. store 4C



<P>
<H4><a name="leafpcr">LEAF PCR PROTOCOL</a></H4>

<P>
(Klimyuk et al., 1993, TPJ 3: 493-494)

<P>
1) Samples are harvested in 1.5 ml tubes and stored on ice.

<P>
2) 40ul of 1N NaOH was added and the samples boiled for 30 sec.


<P>
3) 40ul of 0.25N HCl then 20ul Tris mix was added and the samples
boiled for 


another 2 min.



<P>
-tissue samples can then be used immediatly or stored at 4 C for
several weeks.

<P>
-The amount of tissue used in each PCR reaction should not exceed
2mm2 or the 


reaction will not work. A small amount of treated material can
be excised for use 


in a PCR reaction with a sterile Gilson tip.



<P>
PCR reaction conditions are as follows:

<P>
 total volume= 50ul

<P>
      for 5.5 reactions

<P>
10X buffer 5ul    27.5uls

<P>
10mM dNTPs 1.25uls   6.875uls

<P>
primer A  2.5uls  13.75uls

<P>
primer B  2.5uls  13.75uls

<P>
dH2O  38.75uls   213.1uls

<P>
taq poly  1.0ul   5.5uls



<P>
95 C 10min 1X

<P>
95 C 30sec

<P>
55 C 30sec

<P>
72 C 45sec 30X

<P>
72 C 10min 1X



<P>
run 15ul on a 2% agarose gel



<P>
note: 2.5 times more primer is used and 2 times more taq polymerase
in the leaf 


PCR protocol. If you could get by with less, Jonathan Jones would
have done so!



<P>
Stocks

<P>
0.25N HCl

<P>
0.25N NaOH



<P>
Tris buffer:

<P>
0.5M Tris pH 8.0

<P>
0.25% Nonidet P-40



<P>
LEAF PCR ON ARABIDOPSIS TISSUE WITHOUT ALKALINE TREATMENT

<P>
Preparation of Master mix:

<P>
       1 x  Taq-buffer

<P>
    1.5 mM  MgCl2

<P>
    200 mM  of each dNTP

<P>
        1 mM  each primer

<P>
 0.5 ml 20 x Taq polymerase



<P>
The mix is stored on ice until use.



<P>
Preparation of leaf tissue:



<P>
Put the leaf in a small Petri-dish. Make a hole in a leaf with
the narrow end 


of the Pasteur pipette (a forceps might be helpful) and place
the leaf in a 


PCR-tube, if necessary by blowing. On ice, add 50 ml the Master
mix.



<P>
Running the cycles:



<P>
 Transfer the tubes directly from ice to the prewarmed 94  C block
on the 


Robocycler and run the following cycles:



<P>
  94  C for 3 min  1 x

<P>
  94  C for 30 s; Tann.* for 1 min; 72  C for 1 min-1 min 30 s
35 x

<P>
  72  C for 10 min (optional)  1 x





<P>
Appropriate controls:



<P>
 Positive Negative



<P>
For screening transgenics: plasmid ColO

<P>
 ColO with endogenous primer-set gDNA

<P>
 gDNA with endogenous primer-set - DNA



<P>
*The annealing temperature (Tann.) should be 2-3  C below the
calculated Tann..



<P>
<H4><a name="Maxam">MAXAM GILBERT SEQUENCING</a></H4>

<P>
NEN kit #NEK010

<P>
1)Modification reactions: volumes are in ul, DNAs are 10&#169;20k&#136;cpm/ul


<P>
Do G, then T&C and C while A&G
is incubating



<P>
<PRE>
G react  	A&G react  	T&C react  	C react

200 G buf  	10 H2O  	10 H2O  	15 5M NaCl
5 DNA   	10 DNA  	10 DNA  	5  DNA
1 DMS   2Pip 	For  30 hydrazine 30 hydrazine
2', RT  	15', 37C  	5', RT  	5', RT

50 DMS stop  snapfreeze  200 hz stop  200 hz stop

750 EtOH     	750 EtOH  	750 EtOH

5' spin     5' spin  5' spin
</PRE>


<P>
2)Cleavage reaction: speedvac all samples

<P>
resuspend in 100ul 1M piperidine (diluted from 10M stock)

<P>
inc 30', 90C, tighly sealed



<P>
3)Cleanup

<P>
+ 1ml butanol, vortex

<P>
spin 3', aspirate

<P>
+ 150ul 0.1% SDS, vortex

<P>
+ 1ml butanol, vortex

<P>
spin 3', aspirate

<P>
+ 500ul EtOH

<P>
spin 3', aspirate

<P>
speedvac



<P>
4)PAGE, see ureapage.ptc



<P>
<H4><a name="microsome">MICROSOMAL PREP</a></H4>

<P>
from Halkier & M&oslash;ller 1989 Plant Physiol. 90, 1552-9




<P>
1) 0.1-1g sample freeze-dried material powdered in motar/pestle
under lN, 


suspended in 20xv/w MEB (microsome extract buffer):

<P>
   250mM sucrose

<P>
   100mM Tricine (7.9)

<P>
    50mM NaCl

<P>
     2mM EDTA

<P>
     2mM DTT

<P>
   100mg/20ml PVPP (BDH Polyclar AT or Sigma 6755)

<P>
   0.8mM PMSF

<P>
   5ug/ml antipain & leupeptin

<P>
2) Spin 10' 10,000g

<P>
3) Transfer supernatant w/ pippette to black double capped polycarbonate
tubes 

<P>
(16x76mm, Cat # 355603). Save pellet.

<P>
4) Spin supernatant 60' at 165,000g (That's 70krpm in a 70ti).


<P>
5) Pippette off supernatant and save.

<P>
6) Resuspend microsomal pellet in 0.1-1ml MEB. Dyalyze against


<P>
    50mM Tricine (7.9)

<P>
     2mM DTT

<P>
     0.8mM PMSF



<P>
<H4><a name="miniprepalk">MINIPREP - ALKALINE LYSIS</a></H4>

<P>
1) 1.5ml o/n culture to eppi, spin 1', aspirate sup

<P>
2) resuspend by vortex  5' RT in 100ul miniprep solution 1 MPS1


<P>
3) + 200ul MPS2, invert tubes rapidly 3x, inc 5' ice

<P>
4) + 150ul MPS3, vortex upsidedown 10&quot;, inc 5' ice

<P>
5) spin 5' RT

<P>
6) transfer to eppi - 7a) for sequencing

<P>
7) PCHCl3 ext

<P>
8) spin 2' RT

<P>
9) transfer eppi

<P>
10)+ 900ul EtOH

<P>
11)inc 2' RT

<P>
12)spin 5' RT

<P>
13) aspirate

<P>
14) 70% EtOH wash & spin

<P>
15) aspirate, speedvac

<P>
16) resuspend in 50ul TE, use 2 for digests



<P>
7a)  +900ul EtOH

<P>
8a)  spin 5' RT

<P>
9a)  aspirate

<P>
10a) + 1mL 70% EtOH, spin

<P>
11a) aspirate, resuspend in 200ml TE, 2mg RNAseA, inc 15' 37 C


<P>
12a) Phenol/CHCl3 extract, add 20ml 3M NaOAc, EtOH ppt, 70% wash


<P>
13a) resuspend in 30ml TE

<P>
14a) see sequenase protocol for denaturation

<P>
16) resuspend in 50ul TE

<P>
solutions:



<P>
MPS1, frozen  stock   /50ml

<P>
50mM glucose  2M    1.25ml

<P>
10mM EDTA   0.25M   2ml

<P>
25mN Tris 8   1M   1.25ml



<P>
MPS2, fresh      /10ml

<P>
0.2N NaOH   10N   200ul

<P>
H20    -   8.8ml

<P>
1% SDS   10%   1ml



<P>
MPS3       /100ml

<P>
60mM KOAc   5M   60ml

<P>
1.2M HAc   []   11.5ml

<P>
H2O    -   28.5ml



<P>
<H4><a name="NEpHGE">NEPHGE NONEQUILIBRIUM PH GEL ELECTROPHORESIS</a></H4>

<P>
1) Gel tubes are 0.25 x 13.5cm, acid (1/1 H2SO4/HNO3) washed,rinsed,
silated, 


rinsed EtOH, H2O, baked, one end plugged w/ 2xparafilm

<P>
2) make gels afternoon of day before electrophoresis

<P>
3) make up gel solution

<P>
4) polymerized w/ 20ul fresh 10% APS, 7ul TEMED

<P>
draw into syringe fitted with 23 gauge and PE tubing, add frombottom
to top. 

<P>
Aspirate solution back down to 1.5cm from top toclear meniscus


<P>
5)overlay w/ H2O 1&#169;2h while polymerizing, replace w/ 20ul
samplebuffer o/n, 


parafilm top

<P>
6)secure dialysis tubing soaked in lower buffer to bottom of gelsw/
tubing slice. 

<P>
Avoid bubbles, best to do this w/ end submergedin lower reservoir


<P>
7) aspirate sample buffer from gel top

<P>
8) load sample (10ul), cover w/ 10ul overlay buffer, upper bufferto
top

<P>
9) connect &#169; electrode to bottom, electrophorese 400v for
2400Vh

<P>
10) aspirate upper buffer, then overlay, then sample, rinse topof
gel w/ sample 


buffer, then fill to top w/ SDS&#169;sample buffer

<P>
11) expel gel from tube slowly using tubing fitted syringe intofalcon
tubes w/ 5ml 


SDS&#169;sample buffer. 

<P>
12) equilibrate w/ rocking 1.5h

<P>
13) Load on 7.5-15% SDS-PAG (notched plates) w/ sample buffercontaining
1% 


pre&#169;electrophoresed agarose. MW standard wells arecast on
left side with plastic 


teeth. Include BPB in some of theagarose. Electrophorese at 20mA,
16h



<P>
Sample prep: salt is not allowed!

<P>
lysate & wgerm, 5ul lysate proteins, 20&#169;100kcpm


protein, 50&#169;200ug



<P>
solutions:



<P>
Gel, fresh

<P>
   [final] [stock] /10ml

<P>
urea   9.2M    5.53g

<P>
NP&#169;40   2%  10%  2ml

<P>
Acryl/Bis  4%  28.4/1.6% 1.33ml

<P>
Amphol 3.5-9.5 1.86%  40%  465ul

<P>
Amphol 5.0-7.0 0.14%  40%  36ul

<P>
methylamine  10mM    7.8ul

<P>
Ampholines are LKB 1818&#169;101 & 1809&#169;121



<P>
Sample buffer, frozen 1ml aliquots

<P>
urea   &quot;  &quot;  &quot;

<P>
NP40  &quot;  &quot;  &quot; 

<P>
amphol 3.5&#169;9.5 &quot;  &quot;  &quot;

<P>
amphol 5.0&#169;7.0 &quot;  &quot;  &quot;

<P>
methylamine &quot;  &quot;  &quot; 

<P>
DTT   10mM  1M  0.1ml



<P>
Overlay buffer, frozen 1ml aliquots

<P>
urea        8M    4.81g

<P>
amphol 3.5-9.5 0.2%  40%  50ul

<P>
methylamine  10mM    7.8ul



<P>
SDS&#169;sample buffer

<P>
glycerol  10%    10ml

<P>
Bme   5%    5ml

<P>
SDS   2.3%  10%  46ml

<P>
Tris 6.8  2.5M 0.5M   12.5ml



<P>
Upper buffer

<P>
36.6M, 2ml/l



<P>
Lower buffer

<P>
83.5mM ethanolamine, 5ml/l



<P>
<H4><a name="Northern">NORTHERN BLOT IN FORMALDEHYDE GEL</a></H4>

<P>
gel prep:

<P>
1) add 2g agarose to 143ml H20, microwave, put at 60C

<P>
2) + 20ml 10x northern running buffer NRB at 60C

<P>
3) + 37ml formaldehyde at 60C

<P>
4) cast well mixed in fume hood



<P>
sample prep:

<P>
1) mix RNA + H20  4.5ul

<P>
  10x NRB      2ul

<P>
  formaldehyde 3.5ul

<P>
  formamide  10ul

<P>
2) heat 60C 15'

<P>
3) quickly + 2ul/northern sample loading buffer NSLB

<P>
4) quickly load

<P>
5) run 2 h 100-150v in 1x running buffer NRB

<P>
6) when run complete, soak gel 1h in 20x SSC

<P>
7) transfer to Nc in 10x SSC

<P>
8) prehyb 6h, hyb 40h, 42C in PB & HB (see colony hybridization)


<P>
9) wash filters 2x 55C, 1xSSC,1% SDS, 20'

<P>
                Nx 55C, 0.1xSSC, 0.1% SDS, until low background





solutions




NRB, autoclave, turns yellow

<P>
0.2M Mops 7

<P>
50mM NaOAc

<P>
5mM EDTA 



<P>
NSLB, DEPC H2O, filter sterilize

<P>
50% glycerol

<P>
1mM EDTA

<P>
0.4% BPB

<P>
0.4% XC



<P>
MW markers:BRL RNA ladder Cat # 5620SA



<P>
<H4><a name="nuclext">NUCLEAR EXTRACT PREPARATION</a></H4>

<P>
Based on Parker & Topol (1984) Cell 36 357&#169;369, &
Green et al.(1987) EMBO J. 6 


2543&#169;2549) using swelling in low osmoticum andlysis at high
osmoticum. AmSO4 ppt 


concentrates and separateshistones from binding proteins. All
steps in cold room 


on ice.Volumes here for 100mg nuclear protein (see nucprep.ptc).




<P>
1) thaw frozen nuclei on ice

<P>
2) add 1 vol NWB  triton, transfer to 15ml corex 

<P>
3) spin 5' 3krpm

<P>
4) decant supernatant, drain well

<P>
5) resuspend in 7ml total of nuclear lysis buffer NLB, transferto
Beckman ti70 

<P>
polycarbonate tubes (16x76mm, Cat # 355603)

<P>
6) add 770ul 4M AmSO4

<P>
7) rock 20', should see viscosity increase

<P>
8) spin 1h, 35krpm ti70. The pellet is hardish, supernatantviscous
w/ DNA

<P>
9) add 0.3g/ml AmSO4 to supernatant w/ stirring in a 25ml beaker.It
takes at least 

<P>
0.5h to dissolve

<P>
10)spin 15', 10krpm in 15ml corex. The supernatant containshistones,
the pellet 


primarily binding proteins. Decant thorough&#136;ly, proteins
in supernatant can be 


ppted w/ 80% AmSO4 

<P>
11)add 250ul nuclear extract buffer NEB to pellet,stand on ice5',
gently resuspend 


and transfer to dialysis tubing (cut&#169;off14kD, no azide)

<P>
12)dialyze 4h x 2 x 2l NEB 

<P>
13)spin 10', 12krpm, l N2 freeze supernatant and store &#169;70C


<P>
14)yeild for pea is 5-10% of nuclei prep protein, 2-5% for rice
&wheat. Good 


extracts are 5-10mg/ml



<P>
Solutions: autoclaved, precool, add DTT, Bme and Pase inhibitorsprior
to use

<P>
4M AmSO4, BRL enzyme grade



<P>
NLB                             Stocks/100ml

<P>
110mM KCl                       3.7ml 3M 

<P>
15mM Hepes/KOH 7.6              7.5ml 0.2M

<P>
5mM MgCl2                       500ul 1M 

<P>
1mM DTT                         100ul 1M

<P>
5ug/ml antipain & leupeptin     100ul 5mg/ml             
            

<P>
NEB

<P>
40mM KCl                        1.2ml  3M

<P>
25mM  Hepes/KOH 7.6             12.5ml 0.2M

<P>
0.1mM EDTA                      50ul   0.2M

<P>
10%   glycerol                  10ml

<P>
1mM   DTT                       100ul  1M for resuspension

<P>
5ug/ml antipain & leupeptin     100ul  5mg/ml for resuspension


<P>
5mM Bme                         30ul   for dialysis

<P>
0.8mM PMSF                      1ml    80mM for dialysis



<P>
<H4><a name="nuclei">NUCLEI PREP</a></H4>

<P>
Based on Watson & Thompson (1986) Methods Enzymol. 118 57&#169;75,Green
et al. (1987) 


EMBO J 6 2543&#169;2549, and Kannangara et al.(1977) Carlsberg
Res. Commun. 42 431&#169;434. 

<P>
Set up cold room daybefore, all solutions at 4C, all tubes on
ice.



<P>
1) 2.5l (15-20 trays) of seed is grown 7 d in the dark (givesless
starch) in 


vermiculite w/ occassional H2O. Harvest w/ scis&#136;sors to yeild
approx. 1kg leaves. 

<P>
After weighing, leaves arefurther cut to approx. 2 cm length.


<P>
2) wash half in 4l H2O

<P>
3) rinse w/ 1.5l homogenization buffer HB 

<P>
4) transfer to large razor blender, add 2l HB

<P>
5) smash w/ 3 1&quot; bursts, then 15&quot; low speed

<P>
6) filter thru 1000u and 80u nytex screens (see diagram)

<P>
7) rinse blender w/ 500ml HB, brush lower screen to free debris


<P>
8) spin 15' (time to smash other half of material) at 3k rpm in
8250ml swing&#169;out 


fuge tubes

<P>
9) decant & resuspend gently (only wide&#169;mouth pipettes)
in 5ml/t&#136;ube (80ml total) 


nuclei wash buffer NWB + Triton, transfer to 430ml Corex tube
and wait for next 


smash

<P>
10) spin corex tubes 5', 3k rpm in swing&#169;out

<P>
11) resuspend gently with small brush in 10ml NWB Triton/tube(40ml
total)

<P>
12) spin 5', 3k rpm

<P>
13) for storage, resuspend in 4ml NWB Triton, mix w/ 1.6mlglycerol,freeze
in lN2 


and store at -70C

<P>
14) take 1 & 5ul for Bradford assay. Yeild for rice approx
80-120mg, twice that 


for pea



<P>
Solutions: autoclaved, precool, Bme and Pase inhibitors addedprior
to use




Homogenization buffer HB              stocks/l



<P>
1M hexylene glycol(Aldrich)           127.5ml 7.8M

<P>
10mM Pipes/KOH 7                      10ml    1M

<P>
10mM MgCl2                            10ml    1M

<P>
0.5% v/v triton X&#169;100                 25ml    20%

<P>
5mM Bme,                              300ul

<P>
0.8mM PMSF                            10ml    80mM



<P>
Nuclei wash buffer NWB



<P>
same as HB but

<P>
0.5M hexylene glycol

<P>
w/ or w/out triton



<P>
<H4><a name="nuclprot">NUCLEAR PREP FROM PROTOPLASTS</a></H4>

<P>
1)  8x106 protos in cellwells, collect and spin in 2x 15ml APM
at 500rpm for 5' in 


bench fuge to pellet. decent supernatant. Subsequent steps at
4 C. Examine cells 


here, and after next step to check lysis.

<P>
2)  Lyse by resuspending in 50ml ice cold NIB. 

<P>
3)  Filter through a 20m screen, rinse sreen w/ 50ml NIB, pool
in 3x 30ml corex 


tubes, 1 tube balance.

<P>
4)  Pellet nuclei in HB4 swingbucket at 2470rpm (1000g), 20',
4 C.

<P>
5)  Gently resuspend in 40ml NIB, pool in 2x 30ml corex.

<P>
6)  Spin to pellet starch and debris in HB4 at 110rpm (1g), 3',
4 C.

<P>
7)  Transfer supernatant containing nuclei to new tubes. Pellet
at 1750rpm (500g) 

<P>
15'. Decant completely, quickly wipe tube dry.

<P>
8)  Resuspend nuclei in 5ml HEN-5. Transfer to ultra tube.

<P>
9)  Add 0.55ml 5M NaCl ([0.5M] final) and 27.5ml 1M spermidine
([5mM] final).

<P>
10) Gently agitate at 4 C, 2hr.

<P>
11) Spin clear in Ti50 rotor at 37.5k rpm (88kg) for 15', 4 C.


<P>
12) Transfer supernatant to fresh ultra tube. Add 0.45g/ml finely
 crushed AmSO4, 


gently agitate at 4 C for 30' after AmSO4 fully dissolved.

<P>
13) Pellet in Ti50 at 28k rpm (50kg) for 30', 4 C.

<P>
14) Resuspend in minimal volume HEN-20. Try 50ml, followed by
50ml to rinse tube.

<P>
15) Dialyze in small tubes against 2 x 100ml HEN-25 for 2x 3hr.


<P>
17) lN2 freeze in 20ml fractions, store at -70 C.



<P>
NIB                          stock           ml/l

<P>
10mM  TRIS-HCl 7.5           1M              10

<P>
10mM  KCl                    2M               5

<P>
10mM  MgCl                   1M              10

<P>
10mM  &szlig;me                    14.27M          700ml

<P>
20%   Glycerol               100%            200

<P>
0.02% Triton X-100           100%            200ml



<P>
HEN                          stock           ml/l

<P>
20mM  HEPES-KOH 7.5          0.5M            40

<P>
1mM   EDTA                   250mM           4

<P>
50mM  NaCl                   5M              10

<P>
5-20% Glycerol               100%            50-200

<P>
1mM   DTT                    1M              1



<P>
<H4><a name="dTcellulose">OLIGO DT CELLULOSE PURIFICATION OF MRNA</a></H4>

<P>
Cat # 20003, oligo(dT)&#169;cellulose type 3, Collaborative research
Inc., 128 Spring 

<P>
St. Lexington Mass 02173



<P>
Column prep: suspend cellulose in elution buffer, wash in dis&#136;posable
spin columns 


set in falcon tubes w/ 10 volumes binding buffer





<P>
Binding buffer: 10mM  Tris 7.5

<P>
                500mM NaCl

<P>
                1mM   EDTA

<P>
               0.5%  SDS



<P>
Wash buffer: same SDS



<P>
Elution buffer: 10mM Tris 7.5

<P>
                1mM  EDTA   



<P>
<H4><a name="oligoprobe">OLIGO PROBE HYBRIDIZATION</a></H4>

<P>
Tm = 4/GC + 2/AT or (IN)



<P>
Hyb buffer (OHB)



<P>
6 x SSC

<P>
10 x Denhardt's

<P>
50ug/ml ssDNA

<P>
0.1% SDS



<P>
Hyb at Tm-10C w/ 2 x 106 cpm/ml



<P>
wash:  RT   20' 6 x SSC x 2

<P>
  Tm -5      5'  6 x SSC x 1

<P>
  RT         5'  6 x SSC x 1



<P>
<H4><a name="oligoprobeprep">OLIGONUCLEOTIDE PROBE PREPARATION</a></H4>

<P>
use #b/A260 = ul containing 100pmole to determine []

<P>
if probe has not be desalted or gel purified, spin 1-5 OD260 u
thru 3ml G-50 


column, washing w/ 0.4ml aliquots, pool most []ed 3



<P>
A)

<P>
1) dry down 100pmole in speedvac

<P>
2) add 1ul 10x buffer: 700mM Tris 7.6

<P>
                       100mM MgCl2

<P>
                       1mM   KCl

<P>
                       50mM  DTT

<P>
and    1ul 660uM gATP

<P>
       1ul 10mM spermidine 7

<P>
       5ul 32-P dATP 

<P>
       1ul T4 kinase

<P>
       1ul H2O

<P>
3) incubate 30', 37C

<P>
4) separate DNA from 32-P dATP by 16% urea PAGE (12% if &gt;30b)


<P>
5) visualize bands by autoradiography

<P>
6) isolate DNA by isotachophoresis (see isotach.ptc) or crush
and soak (see 


crshsoak.ptc)



<P>
b)

<P>
1) dry down 150uCi gATP

<P>
2) add 1.5ul 10x buffer: 500mM Tris 7.5

<P>
     100mM MgCl2

<P>
     1mM Spermidine HCl, pHed

<P>
     1mM EDTA

<P>
     50mM DTT

<P>
and    12ul H20

<P>
   1.5ul kinase

<P>
3) inc 60', 37C



<P>
<H4><a name="phagestrain">PHAGE STRAINS</a></H4>

<P>
XL1-Blue: grow in LB/12.5mg/ml TET, 0.2% Maltose, 10mM MgSO4

<P>
Plate on same media/top agarose

<P>
IPTG: 5ml 0.5M stock per ml top agarose

<P>
X-Gal: 16.6ml 250mg/ml stock/ ml top agarose



<P>
1088-90: grow in LB/100mg/ml AMP, 0.2% Maltose, 10mM MgSO4

<P>
Plate on same media/top agarose

<P>
IPTG: 10ml 0.1M stock per ml top agarose

<P>
X-Gal: 10ml 2% stock per ml top agarose



<P>
<H4><a name="plasmidcscl">PLASMID PREP WITH CSCL</a></H4>

<P>
1)grow o/n in 500 ml lb in 2l flask w/ good aeration

<P>
2)spin cells 5.5k, 10'

<P>
3)pour off, towel dry all supernatant

<P>
4)resuspend in 6ml cold sucrose buffer SB, no bubbles!

<P>
5)transfer to tubes for Beckman ti 70(Nalgene 3118.0028) or fixed
angle low speed 


screw cap tubes

<P>
6)add approx. 15mg lysozyme

<P>
7)vortex

<P>
8)stand 5' on ice

<P>
9)add 7.5ml triton buffer TB

<P>
10)stand 5' on ice w/ mixing

<P>
11)spin 30k, 30' in ti70 or 20k, 1h in fixed angle low speed

<P>
12)decant all supernatant to screw cap 50ml tube

<P>
13)measure volume and add 1g/ml ground CsCl

<P>
14)fill to 36ml with 1g/ml CsCl in TE

<P>
15)add 0.2ml 10mg/ml EtBr

<P>
16)add 2ml 20% Lauryl sarcosine

<P>
17)transfer w/ 50ml syringe w/ 16 gauge needle to vti50 quickseal
tubes (Beckman # 


342414), seal

<P>
18)spin in vti50 49k rpm o/n

<P>
19)harvest lower band to 30ml corex

<P>
20)extract w/ = vol CsCl/TE sat'ed isopropanol

<P>
21)spin 3k, 5', discard top, repeat until clear

<P>
22)add  1 vol H2O, divide into two tubes, and add 2 vol EtOH each


<P>
23)-20C ppt. 3hrs

<P>
24)spin 7.5k, 15', 0C

<P>
25)70% EtOH wash & spin

<P>
26)resuspend in 5 ml TE

<P>
27)EtOH ppt. again

<P>
28)resuspend in 0.3ml TE



<P>
a good yeild is 1mg plasmid DNA



<P>
solutions:autoclave but for isopropanol, precool SB & TB



<P>
Sucrose buffer SB     stock/100ml for 12 preps

<P>
50mM Tris 8           5ml  1M 

<P>
50mM EDTA 8           20ml 0.25M

<P>
15% sucrose           15g 



<P>
Triton buffer TB      

<P>
50mM Tris 8           5ml  1M

<P>
50mM EDTA 8           20ml 0.25M

<P>
0.4% Triton           0.4mltriton



<P>
1g CsCl2/ml TE, 400mls



<P>
isopropanol sated w/ CsCl/TE, 400mls sated w/ 100ml 0.5g CsCl/ml
TE



<P>
<H4><a name="primerxt">PRIMER EXTENSION WITH OLIGONUCLEOTIDE</a></H4>

<P>
all solution 0.1% DEPC treated & autoclaved



<P>
1)purify and kinase 200ng oligo (see oligoprp.ptc and oligokin.p&#136;tc)
 10ul  oligo

<P>
10ul  32PgATP

<P>
3.2ul 10 x kinase buffer

<P>
2ul   T4 kinase (2u)

<P>
7ul   H2O

<P>
inc 30', 37C

<P>
2)test incorporation by DE81 or TLC (see inctest.ptc)

<P>
3)PCHCl3 ext

<P>
4) coEtOH ppt w/ mRNA and carrier tRNA (PCHCl3 pure), as follows:


<P>
tube # 1  2  3  4

<P>
oligo  20ng  7.5ng  1ng  1ng

<P>
mRNA  1ug  &gt;  &gt;  0

<P>
tRNA  10ug  &gt;  &gt;  &gt;

<P>
5)resuspend in 25ul primer annealing buffer PAB, transfer toscrew
cap eppi and 


parafilm

<P>
6)place in boiling H2O bath 2'

<P>
7)transfer quickly to 90C H2O bath, reset to Tm-15C (previouslydetermined
for 


bath). Tm defined:

<P>
Tm = 16.6log[Na] + 0.X(% G+C) + 81.5 &#169; 500/#b of oligo

<P>
at 0.4M NaCl, 1% formamide decreases Tm 0.65C, and 1% mismatchdecreases
Tm 1.5C

<P>
8)leave to anneal at Tm several hr

<P>
9) EtOH ppt w/ 0.2 vol NaOAc

<P>
10) resuspend in:

<P>
2ul reverse transcriptase buffer RTB

<P>
2ul 10mM dNTPs

<P>
0.5ul 14.5 ul H2O

<P>
1ul BSA

<P>
14.5ul H2O

<P>
1ul MMV RT (BRL) or AMV

<P>
11)inc 1h, 37C

<P>
12)1ul 200mM EDTA

<P>
13)PCHCL3 ext

<P>
14)EtOH

<P>
15) run sequencing gel on products depending upon extension length




<P>
solutions: 

<P>
PAB RTB, see BRL catalogue

<P>
0.4M NaCl

<P>
40mM Pipes 6.9

<P>
1mM EDTA



<P>
<H4><a name="progeny">PROGENY ANALYSIS OF PLANTS CARRYING 35S SENSE/ANTISENSE GENES
</a></H4>

<P>
1. Transform plants by infiltration, grow to seed (F1). Transgenes
should be 


dominant (expressed constitutively), therefore phenotypes possible
in these 


seeds although there are so relatively few transformants that
they would be 


virtually impossible to see.



<P>
2. Plate F1 seeds on KAN and select for transformants. Examine
these plants 


carefully during their growth. Transfer the plants to soil and
grow to F2 


seed. The plants are hemizygous (heterozygous for transgene).
Now is the time 


to take leaf material for Southerns and northerns/RNAse protection.
The 


former are performed on DNA digested, for example, with one and
with two 


polylinker enzymes. Northerns for foreign genes require only random-primed



DNA probes. To measure sense/antisense transcripts, strand-specific
probes 


are required. A protocol is described in Curent Protocols (4.7.1)
and good 


kits are available from in vitrogen, or Stratagene. Collect F2
seed of all F1 


plants. Seed from plants showing good mRNA accumulation and/or
single copy 


integration are most interesting. The alleles look like this:



all hemizygous

<P>
   R  r

<P>
R  RR Rr

<P>
r  Rr rr



<P>
3) Plate F2 seed on KAN: 25% should be sensitive (rr), 75% resistant
(Rr or 


RR). Of the 75% resistant, 66% should be hemizygous, 33% homozygous.
These F2 


resistant individuals should be studied for phenotypes, and various
assays 


can be performed with them. Transfer to soil and grow these plants
to F3 


seed. The alleles coming out of these plants should look like
this:

<P>
66% hemizygous     33% homozygous

<P>
    R  r        R  R

<P>
R  RR Rr    R   RR RR

<P>
r  Rr rr    R   RR RR



<P>
4) Plate 50 F3 seed from each F2 line on KAN and score sensitive/resistant.


<P>
F3 single copy homozygotes should all be resistant. These are
the most 


important lines for further crosses. Homozygous, KAN-resistant
plants can be 


transferred to soil and used for crosses.



<P>
<H4><a name="RNAGT">RNA EXTRACTION IN GT W/ CSCL SPIN</a></H4>

<P>
1)freeze 15&#169;20g material l N2, grind to powder

<P>
2)add powder to 50ml flip&#169;top tubes containing 30ml guanidiniumthiocyanate
buffer 


GTB, shake to assure that thawing material isunder buffer

<P>
3)inc 10', 50C

<P>
4)spin 15', 15krpm

<P>
5)pipette off supernatant to 50ml screw cap tubes and add 0.15gCsCl/ml.
Do not 


worry about small amounts of material floating around

<P>
6)layer solution carefully onto 8ml 5.7M CsCL (w/5ul EtBr stock)cushions
in sw28 


open tubes. Tubes should have been inced w/0.1MNaOH/1%SDS for
30', then rinsed 


thoroughly in H2O pror to use

<P>
7)spin sw27 22h, 24krpm 18C

<P>
8)aspirate top solution w/ interface junk to cushion, then quick&#136;ly
aspirate 


cushion around red RNA pellet

<P>
9) immediately add 3ml sarcosyl/urea SU, transfer to 15ml corex.vigorous
pipetting 


required to dissolve gelatinous flakes

<P>
10)add 1.5ml P to fully dissolved RNA, vortex, then 1.5 ml CHCl3,vortex


<P>
11)spin 5', 7.5krpm

<P>
12)back ext PCHCl3 phase w/ 1.5ml SU, vortex, spin, add SU phaseto
first SU phase 

<P>
(4.5ml total)

<P>
13)EtOH ppt, &#169;20C

<P>
14)resuspend in 100ul DEPC&#169;treated H2O

<P>
15)2ul for OD 260/280. ratio should be about 2 for good RNA



<P>
GTB, filter sterilized   stock/500ml

<P>
5M   guanidinium thiocyanate  295g

<P>
25mM Na citrate    3.67g

<P>
0.5% sarcosyl    12.5ml 20%

<P>
2mM  EDTA     4ml    250mM

<P>
5%   Bme     25ml



<P>
5.7M CsCl/0.1M EDTA   84.185g CsCl/100ml



<P>
7M urea/2% sarcosyl   42g urea,10ml 20%sarc/100ml



<P>
<H4><a name="RNAstarch">RNA LARGE SCALE FROM STARCHY TISSUE</a></H4>

<P>
1) determine fresh weight of frozen sample

<P>
2) grind in lN2-cooled mortar, assure total destruction.

<P>
3) mix 1:1 RNA extraction buffer (REB) with phenol, pre-heated
to 50 C.

<P>
4) add powder to REB/phenol (1g/2ml) in chloroform resistant centrifuge
tubes. Mix 


well with polytronuntil powder dispersed  Shake another 5'.

<P>
5) Add 1ml chloroform per g sample, shake 25' RT

<P>
6) spin 20,000xg, 20', RT

<P>
7) transfer top layer to new tube. Back extract interface with
0.5 ml REB/g 


sample, and repeat 5)-6) on this and first supernatant.

<P>
8) combine supernatants, reextract with phenol/chloroform, spin
10,000, 15' 0 C.

<P>
9) For Polyphenol-rich material, add 2ml 50% PolyClar AT in DEPC
H2O/ 100ml 


extract, then chloroform extract.

<P>
10) measure volume of final supernatant, add 0.1 vol 2.5 M NaOAc
and 2.5 vol EtOH. 

<P>
mix well, ppts. O/N -20 C

<P>
9) spin 10,000, 20', 0 C

<P>
10) wash pellet w/ -20C 70 EtOH

<P>
11) spin 10,000, 15', 0C

<P>
12) decant carefully, dry wipe tube, resuspend pellet in GTB,
pursue rnalrgprp.ptc

<P>
13) resuspend RNA in 20 ml. GTB, rinse tubes, transfer to 50ml
screw cap. re-rinse 


with 8 ml GTB, add 0.15g CsCl/ml. Do not worry about small amounts
of material 


floating around. Final volume should be 30ml.

<P>
6) layer solution carefully onto 8ml 5.7M CsCL (w/5ul EtBr stock)
cushions in sw28 


open tubes. Tubes should have been inced w/ 0.1M NaOH/1%SDS for
30', then rinsed 


thoroughly in H2O pror to use.

<P>
7) spin sw27 22h, 24krpm 18C

<P>
8) aspirate top solution w/ interface junk to cushion, then quickly
aspirate 


cushion around red RNA pellet

<P>
9) immediately add 3ml sarcosyl/urea SU, transfer to 15ml corex.
vigorous 


pipetting required to dissolve gelatinous flakes

<P>
10)add 1.5ml Phenol to fully dissolved RNA, vortex, then 1.5 ml
CHCl3, vortex

<P>
11)spin 5', 7.5krpm

<P>
12)back ext PCHCl3 phase w/ 1.5ml SU, vortex, spin, add SU phase
to first SU phase 


(4.5ml total)

<P>
13)EtOH ppt, -20C

<P>
14)resuspend in 100ul DEPC-treated H2O

<P>
15)2ul for OD 260/280. ratio should be about 2 for good RNA



<P>
REB, autoclave                     stock/500ml

<P>
100mM Tris, pH 9.0                 50ml  1M

<P>
100mM NaCl                         13ml  3M

<P>
10mM EDTA                          100ml 0.5M

<P>
1% SDS                             50ml  10% 

<P>
5mM DTT                            2.5ml 1M



<P>
GTB, filter sterilized   stock/500ml

<P>
5M   guanidinium thiocyanate   295g

<P>
25mM Na citrate     3.67g

<P>
0.5% sarcosyl     12.5ml 20%

<P>
2mM  EDTA      4ml 250mM

<P>
5%   Bme      25ml



<P>
5.7M CsCl/0.1M EDTA   84.185g CsCl/100ml



<P>
7M urea/2% sarcosyl   42g urea,10ml 20%sarc/100ml



<P>
<H4><a name="RNAmini1">RNA MINIPREP 1</a></H4>

<P>
1) grind 0.5-2g material in lN2 to fine powder

<P>
2) pour 6 large spatulafulls into 30ml corex tubes w/ 10 ml RNAextraction
buffer 

<P>
REB

<P>
3) inc in H2O bath 5&#169;10', 50C w/ occassional shaking

<P>
4) add 1.4ml 3M KCl,vortex, inc on ice 10'

<P>
5) spin 5', 5krpm fixed angle

<P>
6) decant sup thru miracloth to 30ml corex containing 5.6ml 8M
LiCl

<P>
7) leave o/n at 4C

<P>
8) spin 7.5krpm, 10', 4C

<P>
9) dissolve pellet in 5ml H2O

<P>
10)PCHCl3 ext

<P>
11)EtOH ppt &#169;20C

<P>
12)dissolve pellet in 0.4ml H2O & EtOH ppt &#169;20C

<P>
13)resuspend in 50ul H2O

<P>
14)2.5ul for OD260



<P>
solutions:



<P>
REB      stock ml/l

<P>
50mM    Tris 8    50ml     1M

<P>
300mM   NaCl    100ml    3M

<P>
5mM     EDTA    20ml     0.25M

<P>
2%      SDS_    200ml    10%

<P>
autoclave, then add

<P>
1mM     aurintricarboxylic acid 10ml 100mM (S#a1895)

<P>
14mM    Bme     1ml



<P>
8M LiCl



<P>
<H4><a name="RNAmini2">RNA MINIPREP 2</a></H4>

<P>
0,5-2g plant material 

<P>
Solutions 

<P>
REB buffer

<P>
50mM tris pH8 

<P>
300mM NaCl

<P>
5mM EDTA

<P>
2% SDS

<P>
Autoclave then add to aliquots to final concentration:

<P>
1mM Aurintricarboxylic acid 

<P>
14mM Mercaptoethanol

<P>
ex. aliquot 50ml gives 500ul aurintricarboxylic acid+50ul mercaptoethanol.
 



<P>
Stocks

<P>
100 mM Aurintricarboxylic acid (sigma pH8 w. NaOH)

<P>
3 M KCl

<P>
8MLiCl

<P>
3MNaOc,pH6



<P>
Grind tissue (0,5-2g) with pistil/mortar w. liquid nitrogen to
a fine powder. 

<P>
results are very dependent of this step, so grind well.



<P>
1.Add powder to an ELKAY 0002059 tube containing 4,5ml REB buffer


<P>
2.vortex

<P>
3.+0,7ml KCl

<P>
4.Ice 15min

<P>
5.spin 10-20min 8k rpm

<P>
6.transfer supernatant to a new tube.

<P>
7.+2ml 8mLiCl

<P>
8.let stand o/n or 5h at 4c

<P>
9.spin 8k rpm 20min.

<P>
10.Take pellet. resuspend in 2ml water depc. treated

<P>
11.vortex.

<P>
12.add 2ml phenol/chloroform/isoamylalkhol (25:24:1)

<P>
13 vortex

<P>
14.spin

<P>
15.10min 8k rpm

<P>
16.take upperphase to a new tube.

<P>
17..repeat 12-15

<P>
18.add 200ul 3M NaAc, mix

<P>
19.add 5ml 100% EtOH

<P>
20.8k rpm 20min

<P>
21.wash pellet in 2ml 80% ETOH

<P>
22.dry pellet

<P>
23.add 100-200ul depc water (2g of tissue gives 200ul water)

<P>
24.transfer to a epp. take 4-5ul forOD 260

<P>
End -80c



<P>
<H4><a name="RTPCR">RT-PCR</a></H4>

<P>
Total RNAs should have been isolated with a protocole compatible
with the RTase 

<P>
step (chloroforme extraction following a phenol extraction for
instance). 

<P>
Contamination of RNAs by genomic DNA should not be too important
(almost invisible 


on a gel). The DNase  treatment is performed with a RNase-free
DNase in the RTase 


buffer. Prior to the RTase step, the Dnase is heat inactivated
and therefore does 


not require a phenol/chloroforme extraction.



<P>
1. DNase treatment of total RNAs

<P>
On ice, mix the following :

<P>
RNAs   10 ug (not more than 29.5 ul)

<P>
10xRTase buffer 5 ul

<P>
RQ1 DNnase   2 ul

<P>
H20   up to 36.5 ul



<P>
Mix well, spin down and incubate at 37 C for 15&#130;

<P>
Inactivate the DNase at 65 C for 10&#130; then transfer the tubes
on ice



<P>
2. RTase (synthesis of the first strand of cDNAs)

<P>
on ice, add the following to the DNase-treated RNAs :

<P>
0.1 DTT  5 ul

<P>
10 mM dNTPs  5 ul

<P>
0.5 ug/ul oligo-dT 2 ul

<P>
MMLV-RTase  1 ul

<P>
RNase-inhibitor  0.5 ul



<P>
Incubate the tubes 1 hour at 37 C then 2&#130; at 92 C

<P>
For long storage, store the cDNAs at -80 C, otherwise at -20 C.




<P>
3. PCR

<P>
Perform the PCR by using 1 ul of cDNAs for a 50 ul PCR reaction.
The optimal 


number of cycles should be determined in order to stop the reaction
during the 


exponential part of the amplification, before reaching the plateau
phase, 


otherwise the PCR will not be quantitative. Remember to include
appropriate 


positive and negative PCR controls.



<P>
<H4><a name="sdspage">SDS-PAGE FOR PROTEINS</a></H4>

<P>
PA stock made from biorad 2.6% C, add 362 ml. H2O/150g mix. This
gives 30:0.8%. 

<P>
sterile filter.



<P>
gradient gel  7.5%  15%  standard 12.5%



<P>
30% Acryl stock  11.5ml 22.5ml 37.5ml

<P>
resolving buffer  5.6ml  5.6ml  11.2ml

<P>
60% sucrose   3.8ml  13ml  -

<P>
10% SDS   450ul  450ul  900ul

<P>
H2O    28ml  4ml  40.0ml

<P>
TEMED    15ul  10ul  25ul

<P>
10% APS   0.18ml 0.18ml 360ul



<P>
5% stacking gel

<P>
30% Acryl stock  5.0ml

<P>
stacking buffer  7.5

<P>
10% SDS   300ul

<P>
H20    17.1ml

<P>
TEMED    40.0ul

<P>
10% APS   0.4ml



<P>
8x Resolving buffer, filter, 4 C

<P>
363g Tris/l, pH 8.8



<P>
4x Stacking buffer, filter, 4 C

<P>
60.5g Tris/l pH 6.8



<P>
5x Running buffer, 4 C

<P>
30.25 g Tris/l

<P>
144 g Glycine/l (not HCl salt)



<P>
sample buffer, frozen

<P>
    [final] [stock] /50ml  2x (for dil.)

<P>
Tris, 8.8    0.2M  1M  20ml  20

<P>
sucrose   0.5M  -  18g  36

<P>
EDTA    5mM  0.25M  2ml  4.0

<P>
methinone   1%  -  0.5g  1g

<P>
BPB    0.05%  -  2mg  3mg

<P>
SDS    2.5%  10%  12.5ml  2.5g

<P>
DTT    5mM  1M  250ul  500ug



<P>
for samples containing cysteine or free 35-S cysteine, add


iodoacetamide  50mM  0.5M  2.5ul/25ul sample

<P>
inc 15' 37C, then + 1ul Bme to quench iodoacetamide 



<P>
2M urea may be used in SDS gels. Allow urea to dissolve fully
before adding SDS.



<P>
<H4><a name="seedsteril">SEED STERILIZATION FOR CEREALS</a></H4>

<P>
1) wash 10' with tap H2O

<P>
2) wash w/ 4xV w/ 0.2% AgNO3

<P>
3) wash 2x w/ sterile H2O

<P>
4) wash 1x w/ 100mM NaCl

<P>
5) wash 3x w/ sterile H2O

<P>
6) plate w/ 100ug/ml chloramphenicol

<P>
            100U/ml Nystatin

<P>
OR

<P>
1) wash 1' 96% EtOH, aspirate

<P>
2) wash 10' 30% commercial bleach w/ 1ul/ml 20% TritonX100

<P>
3) wash 5x w/ sterile H2O

<P>
4) plate as above



<P>
<H4><a name="silver">SILVER STAINING OF PROTEINS IN PAG</a></H4>

<P>
adapted from Boulikas et al. (1981) Anal Biochem 118, 197&#169;203




<P>
1) fix gel 30' in fixer, then rinse 2 x 1' in 50% EtOH

<P>
2) shake 15' in solution C

<P>
3) rinse 5' in H2O w/ rapid shaking and 1 flip of gel. Careful,
gel swells

<P>
4) develope in solution D. Careful, stop reaction before gel is
as dark as you 


want it as development continues briefly in fixer

<P>
5) stop development by washing in fixer



<P>
solutions:

<P>
A 2g AgNO3 in 10ml H2O

<P>
B 54ml 0.1N NaOH + 3.5 ml [NH3]

<P>
C add A to B dropwise with stirring. dilute to 250ml w/ H2O. Mustbe
used 


immediately

<P>
D 2.5ml 1% citric acid + 0.25ml 37% formaldehyde. Dilute to 500mlw/
H2O. Must be 


used immediately



<P>
<H4><a name="solprot">SOLUBLE PROTEIN EXTRACTION</a></H4>

<P>
1) grind tissue quickly in chilled mortar in 3ml soluble proteinextract
buffer 

<P>
(SPEB)/gm tissue

<P>
2) transfer to 15ml corex, spin 12k 10' fixed angle

<P>
3) trans sup to 15ml corex, add 0.25 vol 50% TCA, chill 4C 30'


<P>
4) spin 1k 5', gently pour off sup, do not let pellet dry!

<P>
5) + 5ml 80% cold 80% acetone

<P>
6) sonicate in cold, spin 1k, 5', aspirate, repeat 3x

<P>
7) store protein as acetone suspension &#169;70C in 1ml 80% acetone




<P>
determine [protein]

<P>
OD 280: take 10ul to 1ml 0.1N NaOH/1% SDS

<P>
Bradford: dry 10ul acetone suspension, dissolve in 10ul 0.1NNaOH,
+

<P>
+ 3ul 1M Tris 8, + 10ul 0.1N HCl, dilute to 1ml w/ H2O, use 10-100ul
for Bradford





<P>
Solutions:

<P>
SPEB    [stock]  ml/50ml

<P>
50mM Hepes   0.5M   5

<P>
5% Bme   -   2.5

<P>
1x BAM/ECAP   100x    0.5

<P>
1x PMSF   100x   0.5

<P>
H2O    -   41.5



<P>
100X BAM/ECAP: 100mM benzamidine & 500mM e-amino caproic acid
in H2O



<P>
100x PMSF: 80mM/EtOH, &#169;20



<P>
<H4><a name="Southwestern">SOUTHWESTERN BLOTTING</a></H4>

<P>
Silva et al. (1987) PNAS 84: 1744-48

<P>
1) run SDS-PAGE and blot as for westerns

<P>
2) place NC in blocking buffer (SWBB), inc RT 1h

<P>
3) aspirate and + denaturing buffer (SWDB), inc RT 1h

<P>
4) asp and + renaturing buffer (SWRB), inc RT 1h

<P>
5) asp, transfer to sealbag, + plenty of SWRB, inc 4C o/n

<P>
6) asp and + 10-15 ml binding buffer (SWBiB) + milk

<P>
7) asp, + 3 ml (&lt; as possible) SWBiB + milk + DNA probe, inc
RT 1h

<P>
8) wash 3 x 5' in SWBiB. expose up to 1 week



<P>
Note:



<P>
a) milk may inhibit binding. Try 10mg/ml BSA (boiled 10' in 10mM
KHPO4 & 50mM 


Hepes Ph 7.5) in SWBB. If milk is thus replaced, omit from SWBiB.


<P>
b) [NaCl] in SWRB and SWBiB can be optimized

<P>
c) Other variables - Buffer (Tris, Hepes), MgCl2 (Mn, Zn), EDTA,
NP-40



<P>
Solutions:



<P>
SWBB 5% Non-fat dry milk

<P>
 30mM Hepes 7.5



<P>
SWDB 50mM Tris 8.3

<P>
 50mM DTT

<P>
 1mM  EDTA

<P>
 6M   Gua-HCl



<P>
SWRB 50mM Tris

<P>
 1mM  EDTA

<P>
 2mM  DTT

<P>
 0.5M NaCl

<P>
 10%  glycerol

<P>
 0.1% NP-40

<P>
 

<P>
SWBiB 30mM Hepes 7.5

<P>
 50mM NaCl

<P>
 5mM  MgCl

<P>
 2mM  DTT

<P>
 0.25%Non fat dry milk



<P>
<H4><a name="T4pol">T4 POLYMERASE FILL IN RXN</a></H4>

<P>
1) resuspend 40ug DNA in 90ul H2O for bal 31 deletions (see baldel.ptc)
Use less 

<P>
DNA, smaller volumes for normal fillins

<P>
2) add 10ul 10 x T4pol buffer

<P>
3ul T4 polymerase (BM #1004786100)

<P>
3)inc 5' , 37C

<P>
4)add 5ul 2mM dNTPs

<P>
5)inc 10', 37C

<P>
6)add 3ul Klenow enzyme (BM #997463500)

<P>
7)inc 5', 37C

<P>
8)P/CHCl3 ext.

<P>
9)EtOH ppt.

<P>
 

<P>
10x T4 pol buffer

<P>
330mM Tris-acetate pH 7.9

<P>
660mM K-acetate

<P>
100mM Mg-acetate



<P>
<H4><a name="western">WESTERN BLOTTING</a></H4>

<P>
electroblotting: assure that gel is cut to correct size for S&S
BA85 or Millipore 


GSWP 0.22um membrane after they have been washed 10' in transfer
buffer. Transfer 


for 3h at 250mM at RT



<P>
immune assay:use 50ml solutions/12x20cm piece



<P>
1) wash wet membrane (most antigens withstand membrane drying)
in PBS + 0.5% tween 

<P>
2 x 30'.

<P>
2) aspirate, inc in 1st antibody diluted 500-5000x in 1% BSA-PBS
1-3h

<P>
3) asp, wash 3 x 10' in PBS/tween

<P>
4) asp, inc in 2nd antibody diluted 1-10k x in PBS/tween 2h

<P>
5) asp, repeat 3)

<P>
6) asp. for phosphatase wash 5' in 50mM Tris 9.6, 5mM MgCl,asp,
inc in stain A 


stain solution until products clearly visible

<P>
7b)for peroxidase, inc in stain B

<P>
8) asp, rinse 2 x 5' in PBS



<P>
solutions:



<P>
Transfer buffer, 4C

<P>
1 x SDS-PAGE running buffer/20% MeOH



<P>
BCIP (5-bromo, 4-chloro indoxyl phosphate), frozen 5mg.ml DMF




<P>
NBT (nitro blue tetrazolium), frozen 0.5%/EtOH



<P>
PBS/l

<P>
20ml M/15 KH2PO4 (9g/l)

<P>
80ml M/15 Na2HPO4 (9.5g/l)

<P>
30ml 5M NaCl

<P>
H2O &gt; 1l



<P>
stain A, fresh

<P>
50mM Tris 9.6

<P>
5mM MgCl2

<P>
100ul BCIP

<P>
100ul NBT



<P>
Stain B, fresh

<P>
10mg 3amino-9ethylcarbazole(Sigma A5754)/1.25ml DMF

<P>
25ml 50mM NaOAc 5

<P>
12.5ul 30% H2O2



<P>
<H4><a name="wholecell">WHOLE CELL EXTRACTS</a></H4>

<P>
1l rice seed imbibed o/n, grown in dark 6d in 6 trays vermiculite


<P>
All steps at 4C on ice

<P>
1) harvest w/ scissors, weigh (yeild should be approx 200g), thencut
to 1-2cm 


length

<P>
2) wash in 1l H2O

<P>
3) smash in 250ml whole cell extract buffer WCEB, 8 burst 10&quot;
each

<P>
4) filter thru 1000u & 80u screens

<P>
5) measure volume, add 1/10 volume 5M NaCl, rock 30' 4C

<P>
7) spin in quick seal ti70 1h, 40krpm

<P>
8) add to supernatant 0.3g/ml ground AmSO4 and 0.1ml 1N NaOH/10gAmSO4


<P>
9) stir in small beaker 30'

<P>
10)spin 20', 10krpm in swingout 250ml tubes

<P>
11)resuspend pellet in 6ml nuclei extract buffer NEB

<P>
12)dialyze (cutoff 14kD, no azide) against 2 x 2l NEB

<P>
13)spin 15', 12krpm in 15ml corex

<P>
14)measure 1ul for Bradford protein. Yeild should be approx 50mg/ml


<P>
15)aliquot as 0.5ml, l N2 freeze, store at -70C



<P>
solutions: autoclave, precool, add DTT, Bme & Pase inhibitors
priorto use



<P>
WCEB                            stocks/l

<P>
40mM  Tris 7.5                  40ml 1M

<P>
5mM   MgCl2                     5ml  1M

<P>
0.5M  sucrose                   362g

<P>
10mM  Bme                       300ul

<P>
0.8mM PMSF                      10ml  80mM



<P>
NEB

<P>
40mM  KCl                       13ml   3M

<P>
25mM  Hepes 7.5                 125ml  0.2M

<P>
0.1mM EDTA                      500ul  0.2M

<P>
10%   glycerol                  100ml

<P>
1mM   DTT                       1ml    1M for resuspension

<P>
5ug/ml leupeptin & antipain     1ml    5mg/ml for resuspension


<P>
5mM Bme                         300ul  for dialysis

<P>

0.8mM PMSF                      10ml   80mM for dialysis



</BODY>

</HTML>
