<html><head><title>AAtDB, An Arabidopsis thaliana Database</title></head>

<body background="/images/arabidopsisbackground.gif">
<a href=/><img src="/images/tairsmall.gif" border=0 align=left></a>
<h1 align=center>AAtDB, An Arabidopsis thaliana Database</h1>

<br><center><img src="/images/line.red2.gif" alt="[line]"></center>
<p>

The latest version of AAtDB is available for UNIX and Macintosh
operating systems. To get your own copy of the software, connect via
either ftp or WWW to <a
href="ftp://genome-ftp.stanford.edu">genome-ftp.stanford.edu</a>.
Other sites are listed below. The information is also available
through the Internet using the Gopher and WAIS software clients. <P>


<H2>Special features of this new release include:</H2>

<ul>
<li>Updated Chromosome Maps with the new Recombinant Inbred Maps from Clare Lister and Caroline Dean.<p>
<li>Updated Contact information from nearly 200 people on the Arabidopsis BioSci email news list.<p>
<li>yUP Hybridization data from the ATGC web server.<p>
<li>CAPS markers from Drenkard and Ausubel are now included.<p>
<li>Sequence updates from GenBank including over 4000 new EST entries.<p>
<li>New bibliographic information from Agricola and Medline  <p>
</ul>

<H2>
Numbers of Objects in selected Classes for version 3-6:
</H2>
<PRE>

   Map                  31         Locus                 1988
   Allele             1334         Map_Population          26
   Author             5981         Motif                  349
   Clone             23168         Paper                 4441
   Contact            1447         Probe                  528
   DNA_Resource       2869         Sequence             41755
   Gene_Class          244             Sequence_EST     18277
   Gene_Product        830             Sequence_Homol   20780
   Germplasm_Resource 3520             Sequence_Genomic+ 2665
   Image               819         Source                  61
   Journal             411         2_point_data          4877

</PRE>


AAtDB features a wide variety of public information that is presented
using graphical, text, and tabular formats. The user interface, which
was designed to invite browsing, allows users to explore information
by pointing and clicking with the workstation mouse or by using a
versatile query facility. This UNIX release of AAtDB runs using the
ACEDB 3.0 software.  ACEDB allows all parts of the database to be
cross-referenced with each other. The large number of interconnections
provides a dense navigable network in which information can be located
from many different starting points.  <p>

All information contained within AAtDB was obtained directly from the
original authors or from publicly available collections and
databases. Updates to AAtDB are periodically distributed, the current
version of the database is 3-6.<p>

The database features:<p>
<ul>
<li>The Hauge/Goodman cosmid physical map including greater than 14,000
cosmid clones. The contigs of the physical map are associated with the
genetic maps. <p>

<li>Genetic markers, including CAPS, RFLP, RAPD, and "classical"
visible markers.<p>

<li>Genetic maps, including an update to the RI map of Lister and Dean (1993) The Plant Journal, the Integrated map from Hauge et al. (1993) The Plant Journal. The "Classical" marker map from Koornneef (1994) in; Arabidopsis, Meyerowitz and Somerville editors, Cold Spring Harbor Labs, publisher.<p>

<li>Primary F2 and RI population recombination data from the Goodman,
Meyerowitz, Scolnik, and Dean laboratories.  <p>

<li>Primary 2-point data from M. Koorneef, D. Meinke, and others.  <p>

<li>Stock lists, maintained and entered for AAtDB by Mary Anderson at
the Nottingham Arabidopsis Stock Centre. <p>

<li>List of Arabidopsis researchers, including postal mail address,
telephone and FAX numbers, electronic mail address, publications,
research interests, and research associates.<p>

<li>All Arabidopsis DNA sequences from GenBank and EMBL DNA sequence
databases (GenBank version 89, June 1995)<p>

<li>NCBI BLASTX (six frame translations searched against the NCBI
non-redundant peptide database) defined peptide sequence homologies
for all DNA sequences included in AAtDB.  <p>

<li>Phenotype descriptions from the Green Book by Meyerowitz and
Pruitt, updated with descriptions from the Nottingham and Ohio State
stock centers. <p>

<li>Scanned images of RFLP autoradiograms from the Goodman lab,
photographs of mutant plants from G. Redei and M. Anderson, and
ethidum bromide-stained restriction enzyme digests of RFLP probes
distributed by the ABRC. <p>

<li>Bibliographic citations, including references for all articles from
the Arabidopsis Information Service.  <p>
</ul>

 AAtDB uses the ACEDB software developed by Richard Durbin (MRC-LMB,
Cambridge, U.K.) and Jean Thierry-Mieg (CNRS, Montpellier, France).
Precompiled executables of ACEDB are available for Sun SPARC (SunOS
and Solaris), DEC Alpha, and Silicon Graphics workstations under the
X-windows environment. Source code is available for compilation with
other computer architectures<p>

All the AAtDB information in text form is available via the
<i>Arabidopsis thaliana Database</i> Project at Stanford..  The images obtained from the ABRC at
Ohio State University are available as GIF files. All these files are
available via Gopher and anonymous ftp from the host
genome-ftp.stanford.edu  & genome-gopher.stanford.edu.  <p>

<H2>Connecting via the Web</H2>
The AAtDB Research Companion Gopher is also connected to the World
Wide Web.  The URL is /. If
you are unfamiliar with ACEDB software, there is a revised version of
the primer "An Introduction to ACEDB for AAtDB, which is available
through the Web server
(URL=/introaatdb/).  This manual
is approximately 20 pages of text and graphics which briefly describes
the data and how to use the database.<p>

<H2>
 How to obtain the Unix version of AAtDB: </H2> 

UNIX files are available via anonymous FTP from three sites.  Note the
new UK site at the European Bioinformatics Institute, the URLs
are:<br>

<A HREF="ftp://ftp.ebi.ac.uk/pub/databases/ace/aatdb">ftp://ftp.ebi.ac.uk/pub/databases/ace/aatdb</A><br>
<A HREF="ftp://ncbi.nlm.nih.gov/repository/aatdb/aatdb.3x">ftp://ncbi.nlm.nih.gov/repository/aatdb/aatdb.3x</A><br>
<A HREF="ftp://genome-ftp.stanford.edu/arabidopsis/aatdb/aatdb.3x">ftp://genome-ftp.stanford.edu/arabidopsis/aatdb/aatdb.3x</A> <p>

Detailed information about how to retrieve the Unix update information
is included in the file <A
HREF="ftp://genome-ftp.stanford.edu/arabidopsis/aatdb/aatdb.3x/AAtDB.Retrieval_Installation_ver_3-6">AAtDB.Retrieval_Installation_ver_3-6</A>
available via anonymous ftp on the AAtDB server
(genome-ftp.stanford.edu, in the /arabidopsis/aatdb/aatdb.3x directory) .  You will
need to obtain the INSTALL script, the compressed binary for your
machine and the update files.  currently we have binaries for Sun
Sparc running Sun OS or Solaris, DEC mips and alpha and SGI.  You will
need approximately 400 Mb of disk space to load and install,
approximately 310 Mb final if you download all the image files. <p>

<H2>
How to obtain the Macintosh version<p>
</H2>

Macintosh files are available from;<br>
<A HREF="ftp://ncbi.nlm.nih.gov/repository/aatdb/MacAAtDB">ftp://ncbi.nlm.nih.gov/repository/aatdb/MacAAtDB</A><br>
<A HREF="ftp://genome-ftp.stanford.edu/arabidopsis/aatdb/MacAAtDB">ftp://genome-ftp.stanford.edu/arabidopsis/aatdb/MacAAtDB</A> <p>

To install the Macintosh version you need only retrieve the file
MacAAtDB.sea.bin (using binary mode if you are retrieving with ftp).
On the anonymous ftp server, the file is in /arabidopsis/aatdb/MacAAtDB.  Double
clicking on the file should launch the extraction process. Two version
are available; one with EST sequences and homology data, and one
without.  The 'full' version requires 142,456 kb of disk space and the
'demo' version requires 52,587 kb of disk space.  Neither version has
the entire set of Stock Center images due to space limitations.<p>

This was the last update from Mass General Hospital.  Special thanks to all those that have submitted information, and the people at the USDA and the National Agricultural Library who provided funding.  <p>

John W. Morris and Howard M. Goodman, Department of Molecular Biology, Massachusetts General Hospital, 50 Blossom Street, Boston, MA 02114 USA <p>

<p>
<hr size=3 noshade width=95%>
<p>
<a href="/"><img src="/images/arrow.small.up.gif" border=0>Return to TAIR Home Page</a>
</body></html>
