<title>koorn-1984-aabnj.html</title>
Vol 21: 
<h2>THE LOCALIZATION OF TWO ABA INSENSITIVE GENES</h2>

M. Koornneef and C.J. Hanhart<p>

Department of Genetics, Agricultural University, Generaal Foulkesweg
53, 6703 BM Wageningen, The Netherlands<p>

INTRODUCTION<p>

Recently a map of Arabidopsis thaliana was published locating 76 genes
at distinct positions on one of the five Arabidopsis chromosomes
(Koornneef et al., 1983). Further linkage studies now concentrate on
the mapping of genes specially of interest for detailed biochemical
and physiological studies and on making the present map more accurate.
In this report the linkage relations of two abscisic acid
(ABA)-insensitivity genes abi-1 and abi-3 (Koornneef et al., 1984) are
presented.<p>

MATERIAL AND METHODS<p>

The mutants used in the present study and the procedures to estimate
recombination percentages from F 2 populations and F 3 *progeny tests
are described in Koornneef et al. (1983). Final map distances were
estimated by a computer program written by Dr. P Stam, based on the
procedures described by Jensen and Jorgensen (1975).<p>

When two markers segregated in both a similar population (mainly F 2)
and phase as in previous experiments, the new data were combined with
the previously obtained data and a new more accurate recombination
percentage was estimated.<p>

Because of the variation in ABA-sensitivity observed between
individuals of the same genotype (Koornneef et al., 1984), unambiguous
scoring for this character is not possible for every F 2 plant.
Therefore ABA sensitivity was assayed on the behaviour of each of the
F 2 plants progenies on 10 micro M ABA. For DV (abi-3 2), which is
more sensitive to ABA than CIV (abi-3 1) and AII (abi-1 1), the
sensitivity on both 3 and 10 micro M ABA was taken into account.<p>

<a href="/ais/images/vol21/vol21-pg6-tab1.gif">Click here for table</a><p>

Table 1: Segregation of ABA sensitivity in F 2's derived crosses with
chromosome 3 (CIV and DV) and chromosome 4 (AII) multiple marker
lines.<p>

<a href="/ais/images/vol21/vol21-pg6-tab2.gif">Click here for table</a><p>

Table 2: Estimation of recombination percentages between markers of
chromosome 3 and 4.<p>

<a href="/ais/images/vol21/vol21-pg7-fig1.gif">Click here for figure</a><p>

Figure 1: The revised linkage maps of chromosome 3 and 4 of
Arabidopsis thaliana. Brackets indicate that the order within the
braketed group could not yet be conclusively established.<p>

RESULTS AND DISCUSSION<p>

For preliminary linkage tests, the ABA-sensitive mutants AII (abi-1)
and CIV (abi-3) were crossed with an/an, pv/pv, gl-l/gl-l,
cer-2/cer-2, ms-1/ms-1. The resulting F 2 was sown on 10 microM ABA.
Among the insensitive progeny there was a shortage of cer-2/cer-2
plants in the AII F 2 and a shortage of gl-l/gl-l plants in the CIV F2
indicating the location of abi-l on chromosome 4 and of abi-3 on
chromosome 3. Detailed linkage data were obtained from the analysis of
the F2's derived from cp-1/cp-1, cer-2/cer-2, ap-2/ap-2 x AII and
from, hy-2/hy-2, gl-l/gl/l, tt-5/tt-5 x CIV respectively x DV (Table
1). The distribution over the different phenotypic classes did not
differ significantly between CIV and DV (*chi*sq. = 16.3, df = 23, 0.8 P
0.9), confirming allelism of both mutants as was suggested before
(Koornneef et al., 1984). The combined data of CIV and DV were used to
estimate the recombination percentages that are shown in Table 2 for
chromosome 3 together with the estimates for chromosome 4. The
localization of abi-3 between hy-2 and gl-l looks clear.<p>

The two recombinants abi-1/abi-1, cer-2/cer-2 (Table 1) were not
conclusive for the localization of abi-1 between cer-2 and ap-2.
However, the presence of an ap-2/ap-2 plant sensitive to ABA and the
absence of a sensitive cp-l/cp-l plant makes the latter position of
abi-l the most probable one. The revised maps of chromosome 3 and 4
(Figure 1) show only minor differences compared to the previous maps.
The slightly changed position of cp-1 (15.9 instead of 20.3) on
chromosome 4 derived from the new data obtained (Table 2) and from the
deletion of the value cp-1-ag, 10.1 +/- 1.1 (Koornneef et al., 1983,
Table V) from the final estimate of the map distances. This value was
omitted because of its significant *chi*sq. value, when compared to the
ultimate estimate of this distance (*chi*sq. = 6.3 before omitting it from
the data file).<p>

REFERENCES<p>

Jensen, J. and, J.H. Jorgensen: Hereditas 80, 5-16 (1975)<p>

Koornneef, M., J. Van Eden, C.J. Hanhart, P. Stam, F.J. Braaksma and
W.J. Feenstra: The J. of Heredity 74, 265-272 (1983)<p>

Koornneef, M., G. Reuling and C.M. Karssen: Physiol. Plant. 61,
377-383 (1984)<p>

