<HTML>
<HEAD>
<script language="javascript">
<!--

function Page_load(page) {
	

		
	   if (page == 4)
		page_title = "reports/introduction.html";
	   else if (page == 5)
		page_title = "reports/Norwich.html";
	   else if (page == 6)
		page_title = "reports/jurgens.html";

	if (page_title == "")
		return;
	
	
	window.parent.frame_main.location = window.parent.frame_menu.home_page + page_title;
	   
}

	var strng;
	function message(id) {
	   

	   if (id == 4)
		strng = 'Introduction to the Conference Reports';
	   else if (id == 5)
		strng = 'Reports on the 7th International Conference for Arabidopsis Research';
	   else if (id == 6)
		strng = 'Report on Session 4 : Embryogenesis';

	   else 
		return false;

	window.status = strng;
	return true;
}

// end of script -->
</script>

<TITLE>Workshop 1: Electronic Information Resources</TITLE>
<!-- Changed by: Dr Mike Arnold,  5-Sep-1996 -->
</HEAD>
<BODY bgcolor=#ffffff>
<BLOCKQUOTE>

<H1>Workshop 1: Electronic Information Resources</H1>
<H4>Chair: Mary Anderson, Nottingham Arabidopsis Stock Centre, University of Nottingham,
UK.</h4>
<b>email: </b><a href="mailto:arabidopsis@nottingham.ac.uk">arabidopsis@nottingham.ac.uk</a>
<hr>
In the last few years there has been a dramatic increase in the amount of
information which has been generated due to the success of the <I>Arabidopsis</I>
research programme. In keeping with the open philosophy of the programme,
where appropriate this information has been made publically available to
the community. This workshop provided a forum for the introduction to, and
live demonstration of, the various projects which provide electronic access
to <I>Arabidopsis</I> information resources.
 <p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S1.html" target="_top">Randy Scholl</A> (Arabidopsis Biological Resource Center, Ohio State University)
described the <A HREF="http://aims.cps.msu.edu/aims/" target="_top">ABRC WWW server</A>, which provides a WWW interface to their database
management system AIMS (Arabidopsis Information Management System). Randy
combined his demonstration of the workings of the server with a description
of the Stock Centre's activities. He demonstrated how to browse the information
as well as how to search the server to find stocks and images, how to place
orders, check on your own order status or to check on order histories for
a particular stock. Randy also described how ABRC intends to acquire large
numbers of T-DNA transformants and transposon lines for mutant screens and
reverse genetics experiments. DNA isolated from T-DNA lines will also be
made available to allow easy PCR-based screening of these populations for
mutations.
<p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S2.html" target="_top">Mary Anderson</A> (Nottingham Arabidopsis Stock Centre, University of Nottingham)
gave a demonstration of the redesigned <A HREF="http://nasc.nott.ac.uk/" target="_top">NASC WWW server</A>. The new layout has incorporated the use of frames to allow for fast and easy access of information.
Information is provided as a menu on the left hand side of the screen that
drives the loading of information into the main part of the screen on the right. The
new search facility for the on-line catalogue and picture book was also
demonstrated, using the example of the newly released MOGEN lines, as well
as how to interact with the Centre by placing orders or asking questions
on-line. Mary also previewed the abstracts from the conference which had
been prepared by Mike Arnold and are available from the NASC server to be
queried or browsed by session, author or keyword. Mary then changed tack
to talk about Weeds World giving a brief description of the types of information
the newsletter contains, how regularly it is published and how to submit
information for publication.
<p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S3.html" target="_top">Ernie Retzel</A> (University of Minnesota), with his designer crutches and brace
for his broken foot, spoke about their <I>Arabidopsis</I> EST project which has
been conducted in collaboration with Tom Newman at the Michigan State University.
The analysis also includes the EST data from the French consortia. Ernie
described the suite of tools that they have developed to allow easy analysis
of the myriad of data that the ESTs have generated. Most of these tools
are made available through the project's<A HREF="http://lenti.med.umn.edu/arabidopsis/arab_top_page.html" target="_top"> WWW server</A>. An exciting development
has been the creation of a motif and similarity clustering tool to identify and display relationships
between sequences. The relationships are displayed as a &quot;tangled cats
cradle&quot; with the close relationships identifed by coloured groups and short distances,
so that islands of related genes are linked through lines. The close relationships
identified concur with the findings of Steve Rounsley at TIGR (see next
speaker) but also bring out additional findings of less related sequences
that nevertheless show significant relationships. This will be a useful
tool for the discovery of genes and gene-relatedness. 
<p>
Following on from the EST project at Minnesota, <A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S4.html" target="_top">Steve Rounsley</A> from The
Institute for Genome Research (TIGR), spoke about the analysis they have
been conducting on the US and French EST data which is deposited in dbEST.
Inherent to any EST project is a redundancy of information that is generated
due to the fact that cDNA clones from the same gene may be sequenced multiple
times. Steve described how they assembled the ESTs into tentative consensus
sequences to reduce this redundancy. Nucleotide and amino acid similarity
searches were then conducted in the public databases for each of the ESTs
to allow putative gene identifications. The tentative consensus sequences
and homologies are made publically available through the <A HREF="http://www.tigr.org/tdb/at/at.html" target="_top">TIGR WWW server</A>.
<p>
Changing tack, we moved away from sequence information, to physical mapping.
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S5.html" target="_top">Tony Schaeffner</A> (Institut fuer Biochemie, Muenchen) described their<A HREF="http://www.lmb.uni-muenchen.de/groups/schaeffner/main.htm#chr" target="_top"> WWW
server</A> which has been established to provide information about on-going
chromosome walking projects. The idea for the server sprang from discussions
at the last meeting at Madison.Tony described how they have set up five
pages representing the five <I>Arabidopsis</I> chromosomes. Each page carries a
list of ordered markers and a listing of laboratories that are currently
working or interested in a particular region. Tony encouraged anyone who
was interested in including their data to contact him or submit their information
through their WWW server.
<p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S6.html" target="_top">David Meinke</A> (Oklahoma State University) started his talk in his capacity
as curator of gene symbols for the <I>Arabidopsis</I> community, describing how
the <A HREF="http://mutant.lse.okstate.edu/meinke.html" target="_top">Meinke WWW server</A> carries detailed information on rules of nomenclature,
mutant gene symbols, linkage data, genetic maps, and e-mail addresses of
contributing laboratories. He described the philosophy of gene naming, how
to send in data about new gene names and the submission of mapping information.
David then went on to describe the other part of the server, which is devoted
to his research activities. He described how detailed information is provided
on the 250 embryo- defective mutants which have been isolated in the Meinke
Lab including the background, mutagen employed, tagging status, seed and
embryo phenotype, segregation ratio, chromosome location, and original seed
source for mutants in the collection.<BR>
<BR>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S7.html" target="_top">David Flanders</A> (AtDB Project, Stanford) brought up the rear, with a &quot;canned&quot;
live demonstration (didn't trust UK technology now that he is based at Stanford)
of the latest developments from the AtDB Project at Stanford. He described
how, as well as making new information available from their <A HREF="/" target="_top">WWW server</A>,
they are developing a new prototype plant genome database. The database
is being developed using the commercial object-orienated database management
system, <A HREF="http://www.informix.com/" target="_top">Illustra</A> and will be accessed through the WWW. He demonstrated some
early displays that they have been designing for physical map and colleague
information. Full details of the project's developments are given in the
article <a href="../Cherry-Flanders-Petel.WW.html" target ="frame_main">"New Directions in Genome Databases at Stanford"</a>, in this issue of Weeds World.
<HR>A computer demonstration of all the services ran for the entire conference
in the LCR. We are grateful to the staff of the UEA Computer Department
for their help with the demonstration. Many thanks to Sun Microsystems and
Relay Business Systems for the loan of the two Sun machines for the conference.
<hr>

</BLOCKQUOTE></BODY>
</HTML>
