<HTML>
<script language="javascript">
<!--

function Page_load(page) {
	

		
	   if (page == 4)
		page_title = "reports/leyser_and_bennet.html";
	   else if (page == 5)
		page_title = "reports/Norwich.html";
	   else if (page == 6)
		page_title = "RI_map_move.html";

	if (page_title == "")
		return;
	
	
	window.parent.frame_main.location = window.parent.frame_menu.home_page + page_title;
	   
}

	var strng;
	function message(id) {
	   

	   if (id == 4)
		strng = 'Report on Workshop 2 : Gene and Protein Function';
	   else if (id == 5)
		strng = 'Reports on the 7th International conference on Arabidopsis Research';
	   else if (id == 6)
		strng = 'The Lister and Dean RI map moves to NASC';

	   else 
		return false;

	window.status = strng;
	return true;
}

// end of script -->
</script>

<HEAD>
    <TITLE></TITLE>
<!-- Changed by: Dr Mike Arnold,  5-Sep-1996 -->
</HEAD>
<BODY bgcolor=#ffffff>
<BLOCKQUOTE>

<H1>Workshop 5 : Epigenetics</H1>
<H4>Chair: Eric Richards, Washington University, USA.</H4>
<b>email:</b> <a href="mailto:richards@wustlb.wustl.edu">richards@wustlb.wustl.edu</a>
<hr>
The workshop highlighted the recent development of several experimental
tools and systems in <I>Arabidopsis</I> to study epigenetic regulation.  Epigenetic
regulation may be viewed as propagation of alternate information states
of the genome without alteration of primary nucleotide sequence.  The study
of plant epigenetic regulation is most highly developed in maize where a
number of sophisticated and complex epigenetic phenomena have been dissected
for several decades.  <I>Arabidopsis</I>, by comparison, has suffered from the
lack of sufficiently well-described epigenetic phenomena but this situation
is changing as evidenced by this workshop.
<p>
The first two presentations dealt with the most conspicuous candidate mechanisms
for epigenetic regulation:  DNA modification and chromatin structure.  The
first speaker in the workshop, <A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S86.html" target="_top">Eric Richards</A> (Washington University), described
work on <I>Arabidopsis</I> <I>ddm</I> mutants which contain reduced levels of genomic
cytosine methylation.  The current status of investigation of <I>ddm1</I> mutations
was summarized and a description of more recently isolated alleles of a
second locus (<I>DDM2</I>) was presented.  The <I>DDM2</I> locus appears to correspond
to the cytosine methyltransferase gene, <I>MET1</I>, originally isolated by Jean
Finnegan and colleagues (CSIRO).  The second talk, presented by <A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S87.html" target="_top">Rob Oosterling</A>
(Utrecht), focused on chromatin structure.  A picture of higher-order chromatin
organization was sought from the definition of matrix attachment regions
(<I>MARs</I>) in a 16 kb genomic region around the plastocyanin gene.  Three <I>MARs</I>
were found in intergenic regions by a heterologous (rodent) nuclear matrix
binding assay.  While these <I>MARs</I> correspond to the general design of previously
identified <I>MARs</I> in plants and other organisms (e.g., AT-rich) a novel degenerate
21-bp motif was found in all three plastocyanin locus <I>MARs</I>.
<p>
The balance of the workshop was devoted to description of epigenetic systems
and the role of cytosine methylation in regulating these systems.  <A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S88.html" target="_top">Ian Furner</A>
(Cambridge University) began with a description of his group's work on epigenetic
lesions of the <I>TT4</I> locus which encodes an anthocyanin biosynthesis enzyme
chalcone synthetase.  The lesions, which cause reduction in <I>CHS</I> gene expression
and anthocyanin pigmentation and correlate with increased cytosine methylation
of the <I>CHS</I> gene, were apparently established by sense-suppression (or &quot;co-suppression&quot;)
after introduction of a <I>TT4</I> transgene.  The lesions persisted even after
the removal of the transgene.  While 5azadeoxycytidine did not reactivate
the silenced endogenous <I>CHS</I> gene, introduction of the <I>tt4</I> epiallele into
a background homozygous for the hypomethylation mutation, <I>ddm1</I>, led to loss
of <I>CHS</I> gene methylation and activation of the gene.  This work provides
one of the first examples of the formation and inheritance of epigenetic
lesions of an endogenous gene in <I>Arabidopsis</I>.  Further, the sensitivity
of the <I>tt4</I> epialleles to <I>ddm1</I> suggests that at least the maintenance, if
not the establishment, of the silenced state involves DNA methylation.
<p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S89.html" target="_top">Craig Pikaard</A> (Washington University) then presented results on nucleolar
dominance, a classical epigenetic phenomenon involving the silencing of
the entire set of rRNA genes from one parent of a hybrid or allotetraploid.
 The Pikaard lab investigated nucleolar dominance in Brassica allotetraploids
at the molecular level and showed that the rRNA genes of one parent in each
of the three allotetraploid Brassica species are transcriptionally silenced.
 Similar results demonstrating nucleolar dominance in the <I>Arabidopsis</I> allotetraploid,
<I>A. suecica</I>, were presented.  5azadeoxycytidine treatment of the Brassica
allotetraploids leads to activation of the silenced rRNA gene complement
indicating that DNA methylation is involved in maintenance of the transcriptionally
inactive state.
<p>
The importance of DNA methylation for proper <I>Arabidopsis</I> development is
clearly demonstrated by the cytosine methyltransferase antisense gene expression
studies presented by<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S90.html" target="_top"> Jean Finnegan</A> (CSIRO).  A severe reduction in cytosine
methylation occurs in some of the group's <I>MET1</I> methyltransferase antisense
lines (down to 10-15% of wild-type levels).  This reduction in methylation
leads to a number of abnormal morphological phenotypes, including homeotic
transformations in the flowers.  <I>MET1</I> antisense expression also is associated
with the ectopic expression in the leaves of the normally flower-specific
floral identity genes, <I>AG</I> and <I>AP3</I>.  The similarities between the morphological
and gene expression phenotypes of the <I>MET1</I> antisense plants and mutants
in the curly leaf (<I>clf</I>) polycomb-like gene led to the suggestion that DNA
methylation and chromatin factors, such as polycomb-class proteins, may
collaborate to ensure proper gene expression.
<p>
The final speaker of the workshop, <A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S91.html" target="_top">Tetsuji Kakutani </A>(National Institute
of Agrobiological Resources, Tsukba), presented the results of genetic and
phenotypic analysis of <I>ddm1</I> mutants which argue further for the importance
of cytosine methylation in plant development.  <I>ddm1</I> homozygotes exhibit
dramatic morphological phenotypes, including alterations in apical dominance,
leaf shape and flowering time, after propagation for a number of generations
by self-pollination.  Genetic analysis indicates that the morphological
abnormalities are caused by the accumulation of heritable lesions at genomic
sites unlinked to the potentiating <I>ddm1</I> mutations.  The lesions which have
been analyzed are stable in <I>DDM1</i> backgrounds and may correspond to traditional
genetic mutations or meiotically transmissible epimutations.  In any case,
the results draw a potential connection between alteration of DNA methylation
and the maintenance of the genomic information, either genetic or epigenetic.
<hr>

</BLOCKQUOTE>
</BODY>
</HTML>
