<HTML>

<script language="javascript">
<!--

function Page_load(page) {
	

		
	   if (page == 4)
		page_title = "reports/jurgens.html";
	   else if (page == 5)
		page_title = "reports/Norwich.html";
	   else if (page == 6)
		page_title = "reports/meyerowitz.html";

	if (page_title == "")
		return;
	
	
	window.parent.frame_main.location = window.parent.frame_menu.home_page + page_title;
	   
}

	var strng;
	function message(id) {
	   

	   if (id == 4)
		strng = 'Report on Session 4 : Embryogenesis';
	   else if (id == 5)
		strng = 'Reports on the 7th International conference on Arabidopsis Research';
	   else if (id == 6)
		strng = 'Report on Session 6 : Reproduction II';

	   else 
		return false;

	window.status = strng;
	return true;
}

// end of script -->
</script>

<HEAD>
    <TITLE>Session 5: Reproduction I (21st Aug 1996)</TITLE>
<!-- Changed by: Dr Mike Arnold,  5-Sep-1996 -->
</HEAD>
<BODY bgcolor=#ffffff>
<BLOCKQUOTE>



<H1>Session 5: Reproduction I</H1>
<H4>Chair: Detlef Weigel, Salk Institute, USA</H4>
<b>email:</b> <a href="mailto:detlfe_weigel@qm.salk.edu">detlef_weigel@qm.salk.edu</a>
<HR>
This session focused on flowering-time and meristem-identity genes.
<p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S30.html" target="_top">Detlef Weigel</A> (Salk Institute, La Jolla, USA) discussed preliminary results
on the dissection of the promoter of the meristem-identity gene <I>LEAFY</I>
(<I>LFY</I>) (Weigel <I>et al</I>., Cell <B>69</B>: 843-859, 1992) using GUS fusion
genes. The<I> LFY</I> promoter is quickly upregulated in response to long
photoperiods, paralleling the rapid induction to flowering under such conditions.
Surprisingly, there is also a slower upregulation in non-inductive short
days, long before flower primordia are initiated. The expression is detected
at the shoot apex, specifically in young leaf primordia, which is confirmed
by <I>in situ</I> hybridization data that George Coupland showed in his
presentation (see below). However, expression levels in young flowers are
apparently much higher, and the <I>LFY </I>promoter can be used for ablation
of flower primordia when fused to the gene encoding diphtheria toxin A chain.
Weigel also discussed the <I>UFO</I> gene, whose cloning has previously
been reported by the Haughn and Coen laboratories (Ingram <I>et al</I>.,
Plant Cell<B> 7</B>: 1501-1510, 1995) . He showed that <I>UFO</I> function requires
<I>LFY</I> activity, and that simultaneous constitutive expression of <I>LFY</I>
and <I>UFO</I> is sufficient to activate the promoter of the floral homeotic
gene <I>APETALA3</I> in a wide range of tissues outside of flowers.
<p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S31.html" target="_top">George Coupland</A> (John Innes Centre, Norwich, UK) discussed the flowering-time
gene <I>CONSTANS</I> (<I>CO</I>) that his group has recently isolated (Putterill
<I>et al</I>., Cell <B>80</B>: 847-857, 1995) . <I>CO</I> appears to be ubiquitously
expressed, with the level of expression being much higher in long days than
in short days, suggesting that <I>CO</I> levels are directly limiting for
flowering. This is consistent with the finding that transgenic plants in
which <I>CO</I> is constitutively expressed at high levels (35S::CO) flower
early, with three leaves in long days and four leaves in short days. Although
the delay by short days is nowhere as strong as in wild type, the small
delay indicates that other genes contribute to the delay in flowering time
under short days. One such gene might be the <I>EARLY IN SHORT DAYS 4</I>
(<I>ESD4</I>) gene, since 35S::CO <I>esd4</I> double mutants are day-neutral and
flower with merely two leaves in both long and short days. Several other
genes such as the late-flowering gene <I>FCA</I>, which is thought to act
in a parallel pathway, are limiting for floral induction by 35S::CO, as
an <I>fca</I> mutation attenuates the 35S::CO phenotype.<BR>
<BR>
To understand <I>CO</I> function in more detail, Coupland and co-workers
created a modified version of 35S::CO, in which <I>CO</I> is fused to the
ligand binding domain of glucocorticoid receptor, and with which nuclear
localization of <I>CO</I> protein can be regulated. Flowering of these plants
can be induced at will with the glucocorticoid dexamethasone. Obvious targets
of floral induction are the flower-meristem-identity genes <I>LFY</I> and
<I>APETALA1</I> (<I>AP1</I>), and Coupland and co-workers went on to determine
how quickly these genes are activated upon induction of the <I>CO</I> gene
compared to a shift from short to long days. It turns out that the transcriptional
activation of <I>LFY</I> is very similar in both treatments, while the activation
of <I>AP1</I> is considerably slower with the <I>CO</I> fusion than with
photoperiod, suggesting that <I>CO</I> defines a branch in the photoperiod-pathway
that is <I>LFY</I> specific. The activation of <I>AP1</I> in the<I> CO</I>
transgenics might be mediated by <I>LFY</I>, which would fit with results
that <A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S52.html" target="_top">Marty Yanofsky</A> (UC San Diego, USA) reported in the Reproduction II
session.
<p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S32.html" target="_top">Caroline Dean</A> (John Innes Centre, Norwich, UK) reported on the third late-flowering
gene that has been cloned, the <I>FCA</I> gene. <I>FCA</I> acts in the constitutively
promoting, or autonomous, floral induction pathway. <I>FCA</I> is different
from the two previously isolated late-flowering genes <I>LD</I> and <I>CO</I>,
both of which appear to encode transcriptional regulators. The predicted
<I>FCA</I> protein has two RNA binding domains of about 90 amino acids,
and is similar to other developmental regulators such as the Sex-lethal
gene of flies. Characteristic for these genes is that they regulate splicing,
and that their own splicing is regulated as well. At least the latter part
holds true for <I>FCA</I>, as several different, alternatively spliced transcripts
can be detected. The gamma-B transcript, which contains the whole open reading
frame, accounts for about 40% of <I>FCA</I> mRNAs. So far there is no evidence
that the splicing of <I>FCA</I> itself is regulated by floral inductive
conditions, but that might not be expected from the mutant phenotype, since
<I>fca</I> mutants flower late under both long and short days. An exciting
possibility is that <I>FCA</I> splicing is regulated by other flowering-time
genes and/or that splicing of other flowering-time genes is regulated by
<I>FCA</I>. Dean also reported that <I>ap1</I> mutants flower earlier than
wild-type, and that there is a complex interaction of <I>ap1</I> mutations
with different <I>fca</I> alleles.
<p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S33.html" target="_top">Karen Hicks</A> (University of Oregon, Eugene, USA) described an exciting story
that might finally provide a molecular link between photoperiod, floral
induction and circadian rhythm. Plant physiologists have for a long time
invoked a circadian clock in the photoperiodic control of flowering, and
Hicks and her colleagues in the Meeks-Wagner laboratory in collaboration
with Steve Kay's group (University of Virginia, Charlottesville, USA), found
now that the <I>elf3</I> mutant, which was originally isolated because of
its day-neutral early-flowering phenotype (Zagotta <I>et al</I>., Aust J
Plant Physiol 19: 411-418, 1992) , is defective in circadian rhythm, as
assayed by either leaf movement or activity of a <I>cab2</I>::luc transgene (Millar
<I>et al</I>., Plant Cell 4: 1075-1087, 1992) . <I>elf3 </I>mutants are rhythmic
in a light/dark cycle or in constant dark, but not in constant light. An
additional defect is a failure to suppress hypocotyl elongation, especially
in short days, which suggests a link with various photoreceptors, mutations
in which cause a similar phenotype. The cloning of <I>ELF3</I> as well as
the search for genetic suppressors and enhancers are well under way.
<p>
<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S34.html" target="_top">Guillermo Cardon</A> (Max Planck Institute, Cologne, Germany) discussed
an <I>Arabidopsis</I> ortholog of the snapdragon gene encoding <I>SQUAMOSA</I>
promoter binding protein 1 (<I>SBP1</I>) of snapdragon, which has been cloned
in the Huijser laboratory (Klein <I>et al</I>., Mol Gen Genet <B>250</B>: 7-16, 1996)
. <I>SBP</I> genes in snapdragon as well as the <I>SBP-like</I> (<I>SPL</I>)
genes in <I>Arabidopsis</I> constitute a new family of DNA binding proteins.
The protein product of <I>SPL3</I>, which maps to chromosome 2 in a region
where no floral mutant has been previously localized, binds to an element
that is conserved between <I>SQUAMOSA</I> and its <I>Arabidopsis</I> ortholog
<I>AP1</I>, and <I>SPL3</I> RNA expression increases during the life cycle
of long-day grown plants. <I>SPL3</I> transcripts are more widely distributed
than that of its putative target, <I>AP1</I>. Transgenic plants that express
<I>SPL3</I> constitutively (35S::<I>SPL3</I>) flower earlier than wild type, and
the first flower is often subtended by a bract, which might be interpreted
as a transformation of the last lateral shoot into a flower, akin to the
shoot-to-flower conversion seen in 35S::<I>AP1</I> plants. The plants also show
other phenotypes, including curling of the leaf margins, carpelloid transformation
of first whorl organs in the later formed flowers and a terminal flower.
The relationship between the 35S::<I>SPL3</I> effects and <I>AP1</I> expression
is not clear yet, as <I>AP1</I> is not ectopically expressed in the transgenic
plants, although an <I>ap1</I> mutation appears to attenuate the 35S::<I>SPL3</I> phenotype
somewhat. Plants expressing antisense <I>SPL3</I> RNA flower slightly later
than wild type and show a modification of pedicels.
<p>
The final presentation was by<A HREF="http://nasc.nott.ac.uk/Norwich/sessions/S35.html" target="_top"> Oliver Ratcliffe</A> (John Innes Centre, Norwich,
UK), who reported that <I>TERMINAL FLOWER 1</I> (<I>TFL1</I>) is the <I>Arabidopsis</I>
ortholog of the snapdragon gene <I>CENTRORADIALIS</I> (<I>CEN</I>), which
has recently been cloned by Bradley, Carpenter, Coen and colleagues (Bradley
<I>et al</I>., Nature <B>379</B>: 791-797, 1996) . The expression pattern of <I>CEN</I>
and <I>TFL1</I> is similar in plants that have started to flower; both are
expressed in subapical regions of the shoot meristem. In contrast to <I>CEN</I>,
<I>TFL1</I> expression is also detected in the vegetative shoot meristem,
consistent with the early-flowering phenotype of <I>tfl1</I>, but not <I>cen</I>
mutants. Ratcliffe proposed several models for the interaction of <I>TFL1</I>
with the <I>LFY</I> and <I>AP1</I> genes, and announced that &quot;Des Bradley
is a top bloke.&quot;
<p>

</BLOCKQUOTE></BODY>
</HTML>
