<TITLE>Arabidopsis Workshop at PGIII</TITLE>


<H2>Arabidopsis Workshop at PGIII</H2>
Report by Mary Anderson <P>

17 January 1995, San Diego, USA  <P>

Chair: Caroline Dean <P>

This workshop focussed on the more global aspects of Arabidopsis research
looking at progress in the mapping and sequencing of the genome. <P>
Renate Schmidt (John Innes, UK) presented progress on the physical mapping of
chromosome 4 and 5 by the Dean Group. Using 140 markers in hybridisation
experiments on the four available YAC libraries, in combination with
chromosome walking, they have managed to generate eight contigs which span
chromosome 4. Current estimates on the size of this chromosome are around 20.5
Mb. The position of the centromere and the rDNA have been identified. Renate
reported that repeated DNA around the centromere proves problematical in the
chromosome walking experiments, as some tandemly repeated sequences localized
in the centromeric region are likely to be lost from the YAC inserts following
replication. Good progress was also reported for chromosome 5, which now is
covered by 40 contigs.  <P>
Bertrand Lemieux (York University, Canada) presented an overview of his EST
mapping project. He described the use of a PCR-based approach to identify YACs
that contain a limited set of ESTs from the NCBI repository. This work is
being done in collaboration with Ron Davis at Stanford, the French Arabidopsis
genome mapping group and Joe Ecker at Penn State. The YAC libraries being used
are the CIC and the yUP banks with a few EG clones included. So far 600
primers and 140 ESTs have been used to anchor 192 YAC clones.  <P>
Tom Newman (Michigan State) reported on the progress of the MSU Arabidopsis
cDNA project. Their goal is to generate Expressed Sequence Tags (ESTs) for
about 85% of the expressed genes in this model system. This effort is
complementary to the French cDNA sequencing initiative, which was reported on
at one of the main sessions. The MSU group are now sequencing approximately
1000 ESTs per month and have already deposited 10,000 ESTs in dbEST at NCBI.
Sequence analysis before submission to dbEST is conducted through a
collaboration with Ernie Retzel's Biocomputing Group at Minnesota (see his
article). Clones and libraries are available from the Arabidopsis Biological
Resource Center at Ohio State.  The generation of a subtracted library is a
major priority.  Tom reported that all the clones from which ESTs have been
generated will be removed from the cDNA library so that low abundance cDNAs
can be identified.   <P>
Howard Goodman (Massachussets General Hospital, Boston) reported on progress
of mapping chromosome 2. Using anchored markers, the new CIC YAC library, and
walking with end-probes they have established approximately 8 YAC contigs
covering approximately 75% of chromosome 2. Current efforts are directed
toward closing the remaining gaps. In addition, they have initiated a
sequencing effort on chromosome 2 using the multiplex method. Data is in the
process of being collected on 4 cosmids as part of an initial feasibility
study.  <P>
George Murphy (John Innes, UK) reported on the EC-funded European Scientists
Sequencing Arabidopsis (ESSA) program, which involves seven laboratories
across the EC that aim to sequence a contiguous region of two million bases of
chromosome 4 of arabidopsis over a period of three years. The program is
coordinated from Norwich by Mike Bevan and about 40% of the sequencing will be
performed there. Analysis of the first two cosmids sequenced in Norwich shows
that the gene density is about one gene every 5 kb. Interestingly, although a
number of computing techniques were applied to identify coding regions, none
were able to detect two genes eventually located by hybridization of a cDNA
library to the cosmids. <P>

<hr>

<A HREF="contents.html"><IMG SRC="tiny_wwcover.gif"></A>
<A HREF="contents.html"> Return to Contents Page: Weeds World Vol2(i)</A>
