<html>
<meta http-equiv="Refresh" content="3;url=/help/helppages/genedeta.htm">
<head>

<link rel="stylesheet" type="text/css" href="/css/help.css">
<!-- HEADER using external JavaScript file -->
<script language="JavaScript" src="/js/helpheader.js"></script>
<script>
setPrevious("generesu.htm");
setNext("markerse.htm");
writeHeader();
</script>

<!-- End of header -->
</head>

<h2><a name="913144"> </a>The TAIR Gene Details</h2><hr>


<p><a name="447197"> </a><font class="cont">When you select a gene name on the Gene Search Results window, the Gene Details appear. Generally, line titles do not appear in this page if there are no associated data for the gene, as in this example: </font></p>


<p><a name="448172"> </a><font class="cont"><img src="images/genedetb.gif"></font></p>


<h4><a name="448006"> </a>Name</h4>

    <dl>
      <dt> <a name="448057"> </a><font class="cont">This is the mnemonic naming used for gene names by researchers. Examples include AG (Agamous), and QRT1 (Quartet1). A symbol for a gene is designated when a gene has been published or if the name and symbol have been registered at Oklahoma State, currently maintained by David Meinke's group or in GenBank.</font>
      <dt> <a name="448059"> </a><font class="cont">(<font  size="2" face="Verdana, Arial, Helvetica, sans-serif"><a href="http://mutant.lse.okstate.edu/genepage/genepage.html">http://mutant.lse.okstate.edu/genepage/genepage.html</a></font>).</font>
    </dl>

<h4><a name="448011"> </a>Aliases</h4>


<p><a name="448020"> </a><font class="cont">Synonyms of the gene name.</font></p>


<h4><a name="448012"> </a>ORF Name</h4>

  <dl>
    <dt> <a name="448045"> </a><font class="cont">An open reading frame (ORF) name from the Arabidopsis Genome Initiative (AGI) groups' annotations. Usually, the convention for naming an ORF in Arabidopsis has been using the clone name followed by a number suffix (e.g., F23H14.13). For chromosome arms that have been completely sequenced, a standard ORF name designation is used:</font>
<ul>
  <li><a name="448046"> </a><font class="cont">AT (Arabidopsis thaliana)</font>
  <li><a name="448047"> </a><font class="cont">2 (chromosome number)</font>
  <li><a name="448048"> </a><font class="cont">G (for Gene)</font>
  <li><a name="448049"> </a><font class="cont">01130 (Number)</font>
  <dl>
    <dt> <a name="448050"> </a><font class="cont">Examples: At2g01130, AT4g00010</font>
  </dl>
</ul>
  </dl>

<h4><a name="448014"> </a>Product Name</h4>


<p><a name="448022"> </a><font class="cont">Name of a gene product. Examples: ASPARTATE AMINOTRANSFERASE, CYTOPLASMIC ISOZYME 1, CHALCONE SYNTHASE.</font></p>


<p><a name="448023"> </a><font class="cont">For the genes that do not have a full name or symbol--largely the predicted genes (ORFs) from AGI sequencing and annotation)--the following product names have been used:</font></p>

<ul>
<li><a name="448024"> </a><font class="cont">Hypothetical protein</font>
  <dl>
    <dt> <a name="448025"> </a><font class="cont">Gene models without any database matches.</font>
  </dl>
<li><a name="448026"> </a><font class="cont">Unknown protein</font>
  <dl>
    <dt> <a name="448027"> </a><font class="cont">Gene models with only EST matches.</font>
  </dl>
</ul>

<h4><a name="448007"> </a>Product Description</h4>


<p><a name="448008"> </a><font class="cont">Description of the gene product(s). Most of this data in TAIR currently is from AtDB curation of literature information</font></p>


<h4><a name="448015"> </a>Gene Structure Predicted</h4>


<p><a name="448064"> </a><font class="cont">Indicates whether the gene structure has been predicted.</font></p>


<h4><a name="447962"> </a>Has Full-Length cDNA</h4>


<p><a name="448038"> </a><font class="cont">Indicates whether a full-length cDNA sequence for this gene is available.</font></p>


<h4><a name="448131"> </a>Mendel#</h4>


<p><a name="448132"> </a><font class="cont">The gene identifier in the Mendel Plant Gene Nomenclature Database (<font  size="2" face="Verdana, Arial, Helvetica, sans-serif"><a href="http://genome-www.Stanford.edu/Mendel/">http://genome-www.Stanford.edu/Mendel/</a></font>)</font></p>


<h4><a name="447966"> </a>Verification Method</h4>


<p><a name="447967"> </a><font class="cont">In this version of the TAIR database, this method is for primary gene structure (translation start, intron/exon boundaries). </font></p>


<h4><a name="448148"> </a>Derived Markers</h4>


<p><a name="448149"> </a><font class="cont">Any genetic markers that were derived from this gene.</font></p>


<h4><a name="448151"> </a>In Clones</h4>


<p><a name="448152"> </a><font class="cont">The clones that contain this gene.</font></p>


<h4><a name="448018"> </a>Map Locations</h4>


<p><a name="448065"> </a><font class="cont">Details about all mapped location for the gene, including links to the TAIR Map Viewer and further details.</font></p>


<h4><a name="448093"> </a>Sequence Locations</h4>


<p><a name="448079"> </a><font class="cont">Details about sequence locations for the gene, including a link to attribution data.</font></p>


<p><a name="447955"> </a><font class="cont"></font></p>

<!-- footer using external javascript file starts here -->
<script language="JavaScript" src ="/js/helpfoot.js"></script>
<script>
setPrevious("generesu.htm");
setNext("markerse.htm");
writeFooter();
</script>
<!-- end footer -->


</body>
</html>
