<html>
<meta http-equiv="Refresh" content="3;url=/help/helppages/initials.htm">
<head>
<link rel="stylesheet" type="text/css" href="/css/help.css">
<!-- HEADER using external JavaScript file -->
<script language="JavaScript" src="/js/helpheader.js"></script>
<script>
setPrevious("tairseaa.htm");
setNext("clonesea.htm");
writeHeader();
</script>

<!-- End of header -->
</head>

<body>
<br>

<h2><a name="447173"> </a>Using the TAIR Search Page</h2><hr>


<p><a name="447174"> </a><font class = "cont">The TAIR Search Page provides options you can use to search the TAIR database. You can do simple or complex searching. <strong>Simple searching</strong> is a name search of all data types or a specific data type. <strong>Complex searching</strong> uses additional parameters to search a specific data type. In the current version, complex searching is available for Clones, Genes, Genetic Markers, Maps, Publications, and Arabidopsis Organizations and Researchers. Simple search forms for specific searching of Protein data, Gene Ontology (GO) Annotations and Locus Histories is also available. The next version of the search software will include simple searching by keywords and complex searching of multiple data types.</font></p>


<h2><a name="447035"> </a>General Search</h2>


<p><a name="447037"> </a><font class = "cont">At the top of the TAIR Search Page, are the General Search options.</font></p>


<p><a name="447039"> </a><font class = "cont"><img src="images/gensearc.gif"></font></p>


<p><a name="447044"> </a><font class = "cont">General Search lets you search most TAIR data by name. This search is particularly useful if you have the name of an entity but do not know the data class to which it belongs. In the current version, this feature searches clones, genes, genetic markers, people's last names, organization names, and all aliases. TAIR's database contains many types of data, which are defined and described in detail in the <font  size="2" face="Verdana, Arial, Helvetica, sans-serif"><a href="/search/schemas.html">TAIR Data Model and Documentation</a></font>.</font></p>


<h4><a name="447225"> </a>Using Wildcard Drop Down Menu</h4>


<p><a name="447046"> </a><font class = "cont">You can use the wildcard dropdown menu to the left of the search input box in your name searches. Four types of wildcard options are supported:

<p><table border="1" bordercolorlight="#FFFFFF" bordercolordark="#000000"
       cellpadding="5" cellspacing="0">
  <caption><b><i><font class = "cont"></font></i></b></caption>
  <tr>
    <td><font class = "cont"><a name="447053"> </a><strong>exactly</strong></font></td>
    <td><font class = "cont">

<p><a name="447055"> </a><font class = "cont">name matches input string exactly</font></p>
</font></td>
  </tr>
  <tr>
    <td><font class = "cont"><a name="447057"> </a><strong>contains</strong></font></td>
    <td><font class = "cont">

<p><a name="447059"> </a><font class = "cont">name contains the input string at the beginning, middle, or the end</font></p>

</font></td>
  </tr>
  <tr>
    <td><font class = "cont"><a name="447061"> </a><strong>starts with</strong></font></td>
    <td><font class = "cont">
<p><a name="447069"> </a><font class = "cont">name contains the input string at the beginning</font></p>

</font></td>
  </tr>
  <tr>
    <td><font class = "cont"><a name="447065"> </a><strong>ends with</strong></font></td>
    <td><font class = "cont">
<p><a name="447080"> </a><font class = "cont">name contains the input string at the end</font></p>

</font></td>
  </tr>
</table>



</font>
</p>


<h2><a name="447050"> Specific Search</h2>


<p><a name="447094"> </a><font class = "cont">The Specific Search options appear below the General Search options.</font></p>


<p><a name="447096"> </a><font class = "cont"><img src="images/specsearch.gif"></font></p>


<p><a name="447105"> </a><font class = "cont">These options let you search within specific data types in TAIR. The data types (described below) are listed on the left. The search option appears to the right of each data type. When you select the search option for a data type, a search form appears. The specific search is most useful if you know the type of data you seek.</font></p>


<h3><a name="447108"> </a><font color="#003366" face="Verdana, Arial, Helvetica, sans-serif">Data Types</font></h3>


<h4><a name="447236"> Markers</h4>


<p><a name="447237"> </a><font class = "cont">This data type represents genetic markers that can be used to
experimentally distinguish between polymorphisms.  The types of markers
are: CAPS, SSLP, AFLP, RFLP, RAPD, PCR based, and hybridization based.
SNPs and visible markers (polymorphisms of unknown sequence variation but
characterized phenotype) can now be found using the Polymorphism/Allele
Search. Potential polymorphisms (e.g., dinucleotide repeats or satellites)
are also included in this category for searching.</font></p>


<h4><a name="447245"> </a>Genes</h4>


<p><a name="447246"> </a><font class = "cont">The genes data type represents any <em>Arabidopsis</em> gene model or locus. In TAIR, a Gene Model is defined as any description of a gene product from a variety of sources including computational prediction, mRNA sequencing, or genetic characterization. A locus is defined as the genomic sequence corresponding to a transcribed unit (e.g. AT2G03340) in the genome. In TAIR, many gene models can exist for a given locus, therefore a search for a gene may result in multiple hits for the same gene name</p>

<p>
<h4><a name="dna"> </a>DNA</h4>


<p>
  <a name="dnadef"> </a><font class = "cont">This data type represents any object that contains a piece of Arabidopsis DNA and can exist in the form of a clone, clone end, library, purified genomic DNA, filters. Many of these objects are sequenced and are available as DNA stocks from ABRC.</font>
</p>
<p>
<h4><a name="polymorph"> </a>Polymorphism</h4>
<p>
  <a name="polymorphdef"> </a><font class = "cont">Any genetic variation between two ecotypes or within
one ecotype that can be assessed experimentally by sequencing, PCR
amplification, restriction digest analysis or phenotypic inspection.</font>
</p>

<h4><a name="447254"> </a>Clones</h4>


<p><a name="447255"> </a><font class = "cont">This data type represents any piece of DNA that is inserted into a vector.</font></p>

<h4><a name="protname"> </a>Proteins</h4>


<p><a name="protdef"> </a><font class = "cont">This data type represents molecules built from amino acids joined together with peptide bonds. Proteins are associated to physical, structural and other characteristics such as: isoelectric point, molecular weight,domains, motifs, and  subcellular locations.</font></p>

<h4><a name="447264"> </a>Sequences</h4>


<p><a name="447265"> </a><font class = "cont">This data type contains sequence text and information for all <em>Arabidopsis</em> nucleotide and amino acid sequences. Nucleotide sequence types include genomic, cDNA, oligos, polymorphic sequences, and sequences flanking molecular objects such as genetic markers (e.g., primers).</font></p>


<h4><a name="447274"> </a>Maps</h4>


<p><a name="447275"> </a><font class = "cont">This data type represents any entity that has other entities assigned to (or positioned on) it. Maps are largely separated into genetic, physical, and sequence maps in TAIR. However, entites such as clones and genes will also be considered to be maps in TAIR, if the entities have other elements assigned to them.</font></p>


<h4><a name="447109"> </a>Community</h4>


<p><a name="447110"> </a><font class = "cont">This data type includes the organizations and individual researchers who study <em>Arabidopsis</em>. Organizations may include <em>Arabidopsis</em> labs, stock centers, committees, facilities, universities, and non- and for-profit institutes.</font></p>


<h4><a name="Reference"> </a>Reference</h4>


<p><a name="Reference1"> </a><font class = "cont">This data type represents reference information for <em>Arabidopsis</em> data. The types of reference in TAIR include journal article, book, book chapter, thesis, personal communication, electronic references, conference proceedings, analysis reference (for software-analyzed data such as gene annotation), database reference (data from external databases such as GenBank and TIGR), and websites.</font></p>

<h4><a name="447113"> </a>Publication</h4>


<p><a name="447114"> </a><font class = "cont">This data type represents publications for <em>Arabidopsis</em> data. Types of publication in TAIR include journal article, book, book chapter, thesis, conference proceedings, and press releases.</font></p>

<h4><a name="goname"> </a>GO Annotations</h4>


<p>
  <a name="godef"> </a><font class = "cont">Annotations of <em>Arabidopsis</em> gene products with controlled vocabulary terms for gene product molecular functions, cellular components and biological processes. Terms and ontology structures are developed by the<a href="http://www.geneontology.org"> Gene Ontology Consortium</a>.</font>
<p>
<h4><a name="locushisname"> </a>Locus History</h4>


<p>
  <a name="locushisdef"> </a><font class = "cont">A locus is defined in TAIR as a genomic sequence corresponding to a transcribed unit (e.g. AT2G03440) in the genome. Loci may change due to splitting or merging of two or more loci. Also, some loci may be deleted due to insufficient evidence and new ones inserted. The locus history data includes information about loci that have been merged, split,inserted or made obsolete. The locus history search can also be used to find a locus name that has not previously been used when naming a new locus.</font>
<p>
<h4><a name="microarrayname"> </a>Microarray</h4>


<p><a name="locushisdef"> </a><font class = "cont">Array elements are the nucleotide molecules that are printed onto
microarray slides or in situ synthesized onto a chip. In the case of the
Affymetrix chips, the array elements are considered to be the set of
oligonucleotides that define a target gene (probe sets).</font>
<p><a name="447157"> </a><font class = "cont"></font></p>

<!-- footer using external javascript file starts here -->
<script language="JavaScript" src ="/js/helpfoot.js"></script>
<script>
setPrevious("tairseaa.htm");
setNext("clonesea.htm");
writeFooter();
</script>
<!-- end footer -->

</body>
</html>
