<title>abeel-1969-aaaxz</title><p>
Vol 06:
<h2> Existence of DNA transmethylase enzymes in Arabidopsis</h2><p>

M. ABEELS and C. DIGNEFFE<p>

(Laboratoire de Cytogenetique, Universite Catholique, Louvain,
Belgium)<p>

In No. 5 of this newsletter we have described the existence of tRNA
transmethylase enzymes in Arabidopsis thaliana (ABEELS, 1968).  More
informations on those enzymes were also given in this issue (ABEELS
and DUBOIS, 1969), but it was of interest to investigate the
existancej of DNA transmethylase enzymes in Arabidopsis thaliana.<p>

A methylation pattern for a specific strain can be made either after
purification and separation of the deoxyribonucleotides from a DNA
extraction, or with an in vitro reaction using a submethylated source
of DNA, enzymes from the investigated strain and methyl 14C groups
from the sole methyl donor, S-adenosylmethionine methyl 14C.  A
homologous methylation between DNA and enzyme from the same species
seems to be impossible except for the E. coli strain K 12 58/161 which
synthesizes DNA in the absence of methionine (BOREK and RYAN, 1958;
GOLD, HURWITZ and ANDERS, 1963 a, b).<p>

For those reasons in vitro methylation of DNA from other species with
enzymes from Arabidopsis thaliana was preferred for technical
convenience but also because that system supposes the recognition of a
DNA structure and therefore similarities in the genetic coding
properties are tested.<p>

Three different sources of DNA were prepared: (1) E. coli strain K12
58/161 normally grown, (2) E. coli strain K12 grown in starvation
conditions with minimal medium (3) Streptomyces strain S17.
Methylated deoxyribose nucleotides are well known for many species of
bacteria and bactrial viruses (WYATT, 1950; DUNN and SMITH, 1955)
mammalian tissues (CHARGAFF et al., 1953) and recently they had been
found in algae (PAKHOMOVA, 1968 ). As for those organisms we found no
other nucleotides than 6-MAP (6-methylaminopurine) and 5-MC
(5-methylcytosine). The ratio 6-MAP/5MC for the different sources of
DNA methylated by Arabidopsis enzymes were 1.38 for (1) DNA from E.
coli rich in methyl groups, 1.63 for (2) DNA from E. coli poor in
methyl groups, and 1.27 for (3) DNA from Streptemyces.  Methylation
level was another source of difference because the radioactivity of
(1) was the third of (2) and that of (3) the half of (2).<p>

Conclusions:<p>

-- Mathylation activity exists by Arabidopsis thaliana.<p>

-- 6-MAP and 5-MC are found.<p>

-- Extensions of the methylation level are different for the different
species of DNA..<p>

-- Similarities of coding properties must exist.<p>

-- Investigations with endonuclease digestion enzymes are necessary
for more informations.<p>

Experimental procedure:<p>

E. coli growth medium:<p>

--for rich DNA: Difco Bacto Tryptone 10 gr; NaCl 8 gr; Glucose 1 gr;
Distilled water 1000 ml.<p>

-- for poor DNA the medium utilized is a variant of the M9 medium with
0.003 gr of 1-methionine for 1,000 ml of minimal medium.<p>

-- for Streptomyces: Nutrient Broth 8 gr, NacL 1 gr for one liter of
growth medium.<p>

The DNA was extracted following the method of MARMUR (1961) with very
little modifications.  The plants 2 to 5 weeks old were frozen and
homogenized. The medium for the extraction of enzymes contained: 0.1 M
Tris buffer, pH 7.2, with 16 mM MgCl2, 20 mM potassium-metabisulfite
and 0.45 mM sucrose (ANDERSON and ROWAN, 1966).  The assay mixture for
the methylation consisted of 1 micro M 2-mercaptoethanol, 10 micro M
triethanolamin buffer, pH 8.8, 10 m micro M
C14-CH3-S-adenosyl-methionine, 2 mg of heated ribonuclease, 100 m
micro M nucleotides as methyl deficient DNA and sufficient enzyme to
incorporate at least 1 micro M of C14-CH3 groups (HURWITZ et al.,
1964) for a total volume of 0.25 ml.  Aftr incubation for 30 min at
38*degrees*C 5 microM sodium phrophosphate, 0.05 ml of 0.5% bovine
plasma albumin, and 0.2 ml of 7% HC10 4 were added and the insoluble
material removed by centrifugation. The pellet was resuspended and
washed twice again.  One part from the washed pellet was dissolved
with 1.5 ml of 0.2 M NH 4 OH, decanted on metal planchets, dried and
counted. The other part after hydrolysis with ln-HC1, 1 hr at
100*degrees*C was submitted to right angle chromatography; solvent I
n-butanol - H20 86:14 v/v, NH 3 gas, solvent II isopropanol-conc. HC1
- H20 (170:41:39 by vol.).<p>

Purine bases and pyrimidine nucleotides were localized in the
UV-light, a UV-photography was made and the cut out spots were counted
for their radioactivity.<p>

References:<p>

ABEELS, M.: Arabid. Inf. Serv. 5, 27 (1968)<p>

-- , and E. DUBOIS: Arabid. Inf. Serv., this issue, p. 10<p>

ANDERSON, J.W., and K.S. ROWAN; Biochem, J. 101, 9 (1966)<p>

BOREK, E., and A.J. RYAN: Bacteriol. 75, 72 (1958)<p>

CHARGAFF, E., C.F. CRAMPTON, and R. LIPSCHITZ; Nature 172, 289 (1953)<p>

DUNN, D.B., and J.D. SMITH: Nature 175, 336 (1955)<p>

GOLD, M., J. HURWITZ, and M. ANDERS: Proc. nat. Acad. Sci. (Wash.) 50,
164-169 (1963 a)<p>

-- , -- , -- , : Biochem. Biophys. Res. Comm. 11, 107 (1963b)<p>

HURWITZ, J, M. GOLD, and M. ANDERS: J. Biol. Chem. 239, 3462-3474
(1964)<p>

MARMUR, J.: J. mol Biol. 3, 208-218 (1961)<p>

PAKHOMOVA, M.V. et al.: Dokl. Akad. Nauk SSSR 182, 712-715 (1968)<p>

WYATT, G.R.; Nature 166, 237 (1950)<p>

