<html>
<head>
<title>Lab 10: Transposon Tagging and Enhancer Traps</title>
<script language="JavaScript" src="/js/navbar.js"></script>
<link rel="stylesheet" type="text/css" href="/css/main.css" >
<script language='JavaScript'>
var highlight = 0; var helpfile="";
</script>
</head>

<body leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" bgcolor="#F5F9FF">


<!-- HEADER using external JavaScript file -->

<script language='JavaScript' SRC='/js/header'>
</script><p>

<!-- End of header -->


<p>
<table width="602" align="center" border="0" cellspacing="0" cellpadding="2">
	<tr>
		<td width="602">
<P>
<CENTER><B>Lab 10: Transposon Tagging and Enhancer Traps</b>

<P>
Cold Spring Harbor Arabidopsis Molecular Genetics Course</CENTER>
<P>
July, 1996<BR>

<P>
Drs. Patty Springer<br>
and<br>
Rob Martienssen
<P>
Cold Spring Harbor Laboratory<br>
Cold Spring Harbor, NY<br>
phone: (516) 367-8322<br>
e-mail: martiens@cshl.org</CENTER>

<P>
<B>1. OVERVIEW<BR>
</B>
<P>
Experiments:   1.  Screening F<sub>2</sub> families for Nam<sup>R</sup> Kan<sup>R</sup> transposants
<P>
   2.  Examination of F<sub>3</sub> seedlings for mutant phenotypes
<P>
   3.  Staining F<sub>3</sub> seedlings for GUS expression
<P>
   4.  Confirmation of transposants<BR>

<P>
<B>2. SCHEDULE<BR>
</B>
<P>
Monday July 1 Seed sterilization and plating 
<P>
Friday  July 5 Put F<sub>3</sub> plates in incubator 
<P>
Wednesday July 10 Put F<sub>2</sub> selection plates in incubator
<P>
Sunday  July 14 Examine F<sub>3</sub> seedlings plated 7/1 for mutant 
<P>
  phenotypes 
<P>
  Stain F<sub>3</sub> seedlings for GUS activity (overnight) 
<P>
  Examine F<sub>2</sub> seedlings for double resistant 
<P>
  transposants
<P>
  Transplant to small Nam Kan plates to confirm 
<P>
  phenotype
<P>
Monday  July 17 Clear chlorophyll from GUS stained seedlings with

<P>
  EtOH
<P>
  Examine seedlings for GUS expression
<P>
  PCR on selected GUS positive lines to confirm 
<P>
  transposants<BR>

<P>
<B>3. INTRODUCTION</B>
<P>
 Enhancer traps and gene traps are reporter gene constructs that
can respond to <I>cis</I>-acting transcriptional signals when
fused to chromosomal DNA.  Insertional mutagenesis using these
&quot;traps&quot; involves generating a large number of individuals
that have the reporter gene integrated at different sites throughout
the genome.  Their progeny are collected and examined for (i)
expression of the reporter gene; and (ii) mutant phenotypes caused
by insertion.  In lines in which the reporter gene is inserted
within or near a chromosomal gene, reporter gene expression mimics
that of the chromosomal gene.  In the last few years enhancer
traps and gene traps have been extensively exploited in Drosophila
and in the mouse, leading to the isolation of genes important
in development whose phenotypes would have been missed in conventional
mutagenesis screens.  This is because reporter gene expression
patterns can be examined in heterozygous individuals, which should
be viable even if the insertion is homozygous lethal.  Reporter
gene expression patterns can also be informative when the insertion
does not result in an obvious mutant phenotype.  
<P>
 This laboratory will introduce you to our enhancer trap and gene
trap system in Arabidopsis, which was developed by V. Sundaresan
and R. Martienssen at Cold Spring Harbor, in collaboration with
Hong Ma (CSHL), Jonathan Jones and Caroline Dean (John Innes Centre,
UK).  Enhancer trap  reporter genes have a minimal promoter that
is only expressed when inserted near cis-acting chromosomal enhancers.
 Gene trap reporter genes (promoter traps and exon traps) have
no promoter; reporter gene expression can occur only when the
reporter gene inserts within a transcribed chromosomal gene, creating
a transcriptional fusion.  
<P>
 Our system uses both enhancer trap and gene trap reporters, and
is based on the maize Ac  and Ds transposable elements.  The Ds
elements carry the &#223;-glucuronidase (uidA) gene as a reporter and
the neomycin phosphotransferase (NPTII) gene (conferring resistance
to kanamycin) as a selectable marker.  There are two classes of
Ds elements: (i) an enhancer trap element which has a minimal
promoter fused to the uidA gene, and (ii) a gene trap element,
which has a multiple splice acceptor fused to the uidA gene. 
Random insertions of the Ds element throughout the genome allow
us to detect chromosomal gene expression through the activation
of the uidA gene.  We can detect this expression in the absence
of a mutant phenotype, so genes performing  redundant functions
can be identified.  We may also be able to assess the role of
essential genes, because the GUS expression pattern can be examined
in viable heterozygotes.  
<P>
 Ds elements are transactivated by crossing to transgenic plants
that provide a source of transposase, namely an immobilized Ac
 element.  Ac  and Ds transposable elements have a tendency to
transpose to genetically linked locations in the genome approximately
50% of the time, which is a major drawback to using them for random
tagging.  We have developed a selection scheme which allows us
to enrich for unlinked transposition events, by selecting against
the donor locus using a dominant negative selectable marker. 
The indole acetamide hydrolase (iaaH) gene has been incorporated
in the T-DNAs carrying both the Ds elements and the Ac element.
 Selection against the iaaH gene using napthalene acetamide (NAM),
and for the Ds element using kanamycin allows the recovery of
transposition events that have lost (by recombination) the donor
locus, and thereby enriches for unlinked transposition events.
 Since we also select against the Ac element, the insertion is
immediately stabilized.  <BR>
<BR>
<BR>

<P>
<B>4. LABORATORY</B>
<P>
 A mutagenesis is initiated by crossing plants homozygous for
one of the Ds elements to plants containing the Ac transposase.
 The resulting F1 seed are planted, and the plants are allowed
to self-pollinate.  The F<sub>2</sub> seed are harvested, and plated on media
containing kanamycin (selects for the Ds element) and NAM (selects
against the iaaH gene).  The double resistant F<sub>2</sub> seedlings (called
transposants) contain a transposed Ds element.  The  Nam<sup>R</sup> Kan<sup>R</sup>
seedlings are transplanted to soil, and allowed to self-fertilize.
 F<sub>3</sub> seed are collected, and grown, and the resulting plants are
stained for GUS activity and examined for mutant phenotypes. 
<BR>

<P>
 You will perform F<sub>2</sub> selections and screen F<sub>3</sub> seedlings for mutations
and patterns of GUS expression during the laboratory.  The F<sub>3</sub>
seedlings you will screen were derived from F<sub>2</sub> selections that
were made and grown up previously.  You will then use a tissue
PCR technique to confirm the transposants.<BR>
<BR>

<P>
<B>Experiment #1:  Selecting F<sub>2</sub> Families for Nam<sup>R</sup>
Kan<sup>R</sup> transposants <BR>
</B>
<P>
1.  You will be given F<sub>2</sub> seed aliquoted into approximately 700
seed (15 mg) batches.  The seed will be sterilized in 15 ml conical
tubes and plated on 100mm square plates.<BR>

<P>
2.   Sterilize the seed by following the protocol below.  Work
in the hood and use sterile technique.  
<P>
  
<P>
 a.  Wash with 2 ml 95% ethanol for 5 min.  The seed will settle
and the ethanol can be decanted or pipetted off.  <BR>

<P>
 b.  Add 1 ml of bleach solution to the seed, leave 5 min.  Fill
tube with sterile water and allow seed to settle.  Remove liquid
by pipetting or decanting. 
<P>
  
<P>
 c.  Wash 2 times with a large volume of sterile water.  Decant
each wash.<BR>

<P>
3.  Add top agar (at 60&deg; C) to 4 ml.  <BR>

<P>
4.  Pour quickly onto plate and swirl to evenly distribute seeds.
<BR>

<P>
5.  Let agar solidify.<BR>

<P>
6.  Place at 4&deg;C overnight.<BR>

<P>
7.  Move plates to growth chamber for 4-5 days.  <BR>

<P>
8.  Examine plates for seedlings that are resistant to NAM and
kanamycin (transposants).  You will be shown examples of these
phenotypes.   <BR>

<P>
9.  Transfer Nam<sup>R</sup> Kan<sup>R</sup> seedlings carefully to small (60mm) Nam/Kan
MS plates.  Use sterile forceps, and work in the hood.  This will
allow verification of the double resistant phenotype.  <BR>
<BR>

<P>
Note:  Only about 25% of F<sub>2</sub> families give rise to transposants,
so not every plate will have them.  In addition, transposants
arise at frequencies between 0.1% and 2.5%, so the number on any
given plate will vary.  Examine plates from other groups as well
as your own.  <BR>
<BR>

<P>
<B>Experiment #2:  Screening F<sub>3</sub> families for mutant phenotypes.
<BR>
</B>
<P>
 We have approximately 10,000 transposant lines at present.  These
lines are represented by F<sub>3</sub> seed which was collected from individual
enhancer and gene trap transposants (each of which was selected
as Nam<sup>R</sup> Kan<sup>R</sup> F<sub>2</sub> seedling).  You will be given F<sub>3</sub> seed from both
enhancer and gene trap lines.<BR>

<P>
<B>Plating and Sterilization</B>
<P>
1.  Sterilize 20 - 30 seeds (these will be aliquoted for you)
from each family.  The exact number of families will be given
to you in the lab.  Sterilize in an eppendorf tube using the protocol
described below.  <BR>

<P>
 a.  Wash with ~0.5 ml ethanol for 5 min.  Allow seed to settle
and pipette or aspirate off ethanol.<BR>

<P>
 b.  Add ~0.5 ml bleach solution to seed, leave for 5 min.  Do
not leave this step longer than 5 minutes, as germination efficiency
will decrease.   Fill tube with sterile water, allow seed to settle.
 Remove liquid by aspiration or pipetting.  <BR>

<P>
 c.  Wash 3 times with sterile water.   Aspirate or pipette off
each wash.<BR>

<P>
 d.  Resuspend seed in 0.7 ml 0.1% agar.  Spread on MS non-selective
media (60mm plates), distributing evenly.<BR>

<P>
2.  Place plates at 4&deg; C 4 days (until Friday
7/5).  Cold treatment encourages even germination.<BR>

<P>
3.  Transfer plates to growth chamber until Sunday July 14.<BR>

<P>
<B>Phenotypic screen</B>
<P>
 Examine seedlings for visible phenotypes.  We classify these
as size variants, morphological defects and pigment defects. 
Record the number of seedlings that germinate, and the number
of mutant and normal seedlings on the plate.  Individual F<sub>2</sub> plants
can be either homozygous or heterozygous for the Ds element, so
any insertion induced phenotype could either be present in every
F<sub>3</sub> plant (in the case of a F<sub>2</sub> homozygote) or could segregate 3:1
normal to mutant (if the F<sub>2</sub> plant was heterozygous and the mutation
is recessive).  Segregation ratios substantially less than 1 in
4 may indicate that the mutation arose after transposition and
is not associated with the insertion. Mutations in our lines arise
at a frequency of about 4%, not all of which are tagged.  
<P>
<B>Experiment #3. Screening for GUS staining patterns.<BR>
</B>
<P>
 Screening for reporter gene expression patterns in the same seedlings
can now proceed.  Since F<sub>2</sub> plants could be either homozygous or
heterozygous, insertions may be segregating in the F<sub>3</sub>.  Stain
4 seedlings from each line in order to make sure you sample plants
that have the insertion.  Record the number of seedlings that
stain.<BR>
<BR>

<P>
<B>Staining for GUS activity.</B>
<P>
 You will use two different staining conditions that differ in
the concentration of an oxidative catalyst.  <BR>

<P>
1.  With fine forceps, carefully transfer seedlings from each
plate into a microtiter well containing 0.7 ml of GUS stain. 
<BR>

<P>
2.  After seedlings are transferred, place dish in dessicator,
and draw vacuum for 10 minutes.<BR>

<P>
3.  Wrap dishes in aluminum foil (GUS is light sensitive) and
place at 37&deg; C overnight.
<P>
-&gt; Save remaining seedlings on plate, you will need them later.
 <BR>

<P>
<B>Clearing</B>
<P>
 Remove stain solution and replace with 70% EtOH.  Clearing is
speeded up by placing at 37&deg;C.  Several changes
may be necessary.  <BR>

<P>
<B>Examination</B>
<P>
 First look at seedlings in microtiter dishes with a dissecting
scope.  You may want to remove some of the EtOH to get a clearer
view.  Seedlings showing interesting patterns can be mounted on
a microscope slide and looked at under higher magnification.<BR>
<BR>

<P>
<B>Experiment #4:  Confirmation of Transposants<BR>
</B>
<P>
 Ds has a tendency to transpose to closely linked sites (within
5 cM, and often within a few kb).  Therefore, insertions into
the iaaH gene on the T-DNA are possible.  If this happens, the
iaaH gene will be inactivated, conferring NAM resistance.  We
fortuitously select for this event.  In practice, we find about
5 percent of our transposants contain an iaaH insertion (and are
therefore not really transposants).  However, in lines that express
GUS, the percentage is higher.  We need to test each GUS positive
line for the presence of the iaaH gene, which would indicate that
the line is not a true transposant.  This is easily done using
PCR, which can be done on plant tissue using the following protocol
(Klimyuk et al., 1993).  <BR>

<P>
 You will test each line that expresses GUS.  The lines will be
divided up so that each group does approximately the same number.
 <BR>

<P>
1.  Pick a small amount of tissue from each seedling (1/2 cotyledon
works well, <U>too much inhibits the reaction</U>).  Collect tissue
from 3 seedlings from each line (because of Ds segregation) .
 This will be combined in a single PCR reaction.  <BR>

<P>
2.  Collect tissue into a 0.5 ml eppendorf tube containing 40&#181;l
of 0.25N NaOH.  Boil for 30 seconds in a water bath.  <BR>

<P>
3.  Add 40&#181;l 0.25N HCl, and 20&#181;l  0.5M Tris pH8; 0.25% Nonidet
P-40.  Boil for 2 minutes.<BR>

<P>
4.  Spin, and discard liquid.  The plant tissue will be used in
a PCR reaction.<BR>

<P>
5.  Set up the following reaction for each sample.  Make a master
mix containing all reaction components.  Make enough for two additional
samples to allow for pipetting errors.  
<P>
   5.0 &#181;l 10X PCR buffer
<P>
   5.0 &#181;l dNTPs (2 mM stock)
<P>
   2.5 &#181;l Nam1 Primer (10 pmoles/l)
<P>
   2.5 &#181;l Nam2 Primer (10 pmoles/l)
<P>
   2.5 &#181;l Nam3 Primer (10 pmoles/l)
<P>
   2.5 &#181;l Nam4 Primer (10 pmoles/l)
<P>
   2.5 &#181;l 148D Primer  (10 pmoles/l)
<P>
   2.5 &#181;l 148AR Primer (10 pmoles/l)
<P>
   0.2 &#181;l Taq Polymerase
<P>
   24.8 &#181;l Water<BR>

<P>
 The four Nam primers are from two different regions of the iaaH
gene, which will allow detection of the gene even if one region
has been disrupted by the insertion.  The 148 primers are from
a chromosomal gene, and provide an internal control for each reaction.
<BR>

<P>
6.  Add 50 &#181;l of the mix to each tube (containing the treated
tissue).  Overlay with 50 &#181;l mineral oil (if using thermocycler
without heated lid).<BR>
<BR>
<BR>
<BR>
<BR>

<P>
7.  Place tubes in thermocycler.  The following program will be
used
<P>
   95&deg; C  10 min (1 X)
<P>
   95&deg; C  30 sec
<P>
   55&deg; C  30 sec (30 X)
<P>
   72&deg; C  30 sec
<P>
   72&deg; C  10 min (1 X)<BR>

<P>
8.  Run 15 &#181;l of each sample on a 2.0% agarose gel.  <BR>

<P>
9. The 148AR-148D primer pair gives about a 175 bp band.
<P>
 The Nam1 - Nam2 primer pair gives about a 375 bp band.
<P>
 The Nam3 - Nam4 primer pair gives about a 500 bp band.   <BR>
<BR>

<P>
<B>5. REAGENTS<BR>
</B>
<P>
<B>MS medium (also called GM or germination medium)</B>
<P>
 Per liter
<P>
 4.4 g MS salts (Sigma or Carolina Biological Company)
<P>
 10.0 g sucrose (tissue culture grade, Sigma)<BR>

<P>
-&gt; pH to 5.7 with 1 M KOH.  Add 7 g agar per liter and autoclave.
 Allow to cool to 60&deg;C before adding NAM and
kanamycin.<BR>

<P>
<B>NAM (alpha-napthalene acetamide, Sigma)</B>
<P>
 Stock is 2 mg/ml made in 95% EtOH
<P>
 Use 325 &#181;l per liter (final 3.5 &#181;M)
<P>
 It is a good idea to check individual sources of NAM to make
sure this concentration gives the desired effect.  <BR>
<BR>
<BR>

<P>
<B>Kanamycin</B>
<P>
 Stock is 50 mg/ml solution made in water and filter sterilized.
<P>
 Use 1 ml per liter (final concentration 50 &#181;g/ml).<BR>

<P>
<B>Bleach solution</B>
<P>
 Use between 20% and 100% (v/v) bleach (1-5% sodium hypochlorite)
depending on cleanliness of seed.  You will use a 40% solution.
 Add Tween-20 to a concentration of about 0.1%.<BR>

<P>
<B>GUS Stain</B>
<P>
<table border=1>
<tr><th>Stain A</th><th>Stain B</th></tr>
<tr><td>100 mM sodium phosphate pH 7</td><td>100 mM sodium phosphate pH 7</td></tr>
<tr><td>10 mM EDTA</td><td>10 mM EDTA</td></tr>
<tr><td>0.1% Triton X-100</td><td>0.1% Triton X-100</td></tr>
<tr><td>1 mg/ml X-Gluc*</td><td>1 mg/ml X-Gluc</td></tr>
<tr><td>100 &#181;g/ml Chloramphenicol#</td><td>100 &#181;g/ml Chloramphenicol</td></tr>
<tr><td><br></td><td>1 mM potassium ferricyanide+</td></tr>
<tr><td><br></td><td>1 mM potassium ferrocyanide+<BR></td></tr>
</table>

<P>
-&gt; Make a 20 mg/ml stock of X-Gluc in dimethylformamide, store
at -20&deg;C*.<BR>

<P>
* X-Gluc is very expensive.  Treat stocks with care.  
<P>
# Optional (inhibits bacterial growth during incubation).
<P>
+ Oxidative catalyst.  Concentration can be varied from 0-5 mM
<BR>

<P>
-&gt; Some staining protocols call for a mild fixation before
or after staining.  This may be useful for some tissue types.
 The 'GUS Protocols' manual is a good general reference.<BR>

<P>
-&gt; You may also alter the time of staining; for long staining
times (2-3 days) it is advisable to use 2 - 5 mM ferri/ferrocyanide
to limit stain diffusion. <BR>

<P>
<B>6. REFERENCES<BR>
</B>
<P>
This is the reading list for the transposon tagging lecture (R.
Martienssen) and for the laboratory (P. Springer).<BR>

<P>
<B>Transposon tagging in dicots<BR>
</B>
<P>
Aarts, M.G., Dirkse, W.G., Stiekema, W.J., and Pereira, A. (1993).
Transposon tagging of a male sterility gene in Arabidopsis. Nature
363, 715-717.<BR>

<P>
Aarts, M.G., Corzaan, P., Stiekema, W.J., Pereira, A. (1995).
A two-element Enhancer-Inhibitor transposon system in Arabidopsis
thaliana. Mol. Gen. Genet. 247, 555-564.<BR>

<P>
Balcells, L., Sundberg, E., and Coupland, G. (1994). A heat-shock
promoter fusion to the Ac  transposase gene drives inducible transposition
of a Ds element during Arabidopsis embryo development. Plant Journal
5, 755-764.<BR>

<P>
Bancroft, I., Bhatt, A.M., Sjodin, C., Scofield, S., Jones, J.D.G.,
and Dean, C. (1992). Development of an efficient two-element transposon
tagging system in Arabidopsis thaliana. Mol. Gen. Genet. 233,
449-461.<BR>

<P>
Bancroft, I. and Dean, C. (1993). Factors affecting the excision
frequency of the maize
<P>
transposable element Ds in Arabidopsis thaliana. Mol. Gen. Genet.
240, 65-72.<BR>

<P>
Bancroft, I., Jones, J.D.G., and Dean, C. (1993). Heterologous
transposon tagging of the DRL1 locus in Arabidopsis. Plant Cell
5, 631-638.<BR>

<P>
Briza, J., Carroll, B.J., Klimyuk, V.I., Thomas, C.M., Jones,
D.A., and Jones, J.D.G. (1995). Distribution of unlinked transpositions
of a Ds element from a T-DNA locus on tomato chromosome 4. Genetics
141, 383-390<BR>

<P>
Cardon, G.H., Frey, M., Saedler, H., and Gierl, A. (1993). Mobility
of the maize transposable element En/Spm in Arabidopsis thaliana.
Plant Journal 3, 773-784.<BR>

<P>
Chuck, G., Robbins, T., Nijjar, C., Ralston, E., Courtney-Gutterson,
N., and Dooner, H. K. (1993). Tagging and cloning of a Petunia
flower color gene with the maize transposable element Activator.
Plant Cell 5, 371-378.<BR>

<P>
Fedoroff, N.V. and Smith, D.L. (1993). A versatile system for
detecting transposition in Arabidopsis. Plant Journal 3, 273-289.
<BR>

<P>
Feldman, K.A. (1991). T-DNA insertion mutagenesis in Arabidopsis:
mutational spectrum. Plant Journal 1, 71-82.<BR>

<P>
Honma, M.A., Baker, B.J., and Waddell, C.S. (1993). High-frequency
germinal transposition of DsALS in Arabidopsis. Proc. Natl. Acad.
Sci., USA 90, 6242-6246.<BR>

<P>
Klimyuk, V.I., Nussaume, L., Harrison, K., Jones, and J.D.G.(1995).
Novel GUS expression patterns following transposition of an enhancer
trap Ds element in Arabidopsis. Mol Gen Genet 249, 357-365.<BR>

<P>
Lawson, E.J., Scofield, S.R., Sjodin, C., Jones, J.D.G., and Dean,
C. (1994). Modification of the 5' untranslated leader region of
the maize Activator element leads to increased activity in Arabidopsis.
Mol. Gen. Genet. 245, 608-615.<BR>

<P>
Long, D., Martin, M., Sundberg, E., Swinburne, J., Puangsomlee,
P., Coupland, G. (1993). The maize transposable element system
Ac/Ds as a mutagen in Arabidopsis: identification of an albino
mutation induced by Ds insertion. Proc Natl Acad Sci U S A 90,
10370-10374.<BR>

<P>
Long, D., Swinburne, J., Martin, M., Wilson, K., Sundberg, E.,
Lee, K., and Coupland, G. (1993). Analysis of the frequency of
inheritance of transposed Ds elements in Arabidopsis after activation
by a CaMV 35S promoter fusion to the Ac transposase gene. Mol.
Gen. Genet. 241, 627-636.<BR>

<P>
Lucas, H., Feuerbach, F., Kunert, K., Grandbastien, M.A., and
Caboche, M. (1995). RNA-mediated transposition of the tobacco
retrotransposon Tnt1 in Arabidopsis thaliana. EMBO J. 14, 2364-2373.
<BR>

<P>
Nussaume, L., Harrison, K., Klimyuk, V., Martienssen, R., Sundaresan,
V., and Jones, J.D.G.  (1995). Analysis of splice donor and acceptor
site function in a transposable gene trap derived from the maize
element Activator. Mol. Gen. Genet 249, 91-101.<BR>

<P>
Osborne, B.I., Wirtz, U., and Baker, B. (1995). A system for insertional
mutagenesis and chromosomal rearrangement using the Ds transposon
and Cre-lox. Plant J. 7, 687-701.<BR>

<P>
Scofield, S.R., English, J.J., and Jones, J.D.G. (1993). High
level expression of the Activator transposase gene inhibits the
excision of Dissociation in tobacco cotyledons. Cell 75, 507-517.
<BR>

<P>
Smith, D.L. and Fedoroff, N.V. (1995). LRP1, a gene expressed
in lateral and adventitious root primordia of Arabidopsis. Plant
Cell 7, 735-745.<BR>

<P>
Springer, P.S., McCombie, W.R., Sundaresan ,V. and Martienssen,
R.A. (1995). Gene trap tagging of PROLIFERA, an essential MCM2-3-5-like
gene in Arabidopsis.  Science 268, 877-880.<BR>

<P>
Sundaresan, V., Springer, P., Volpe, T., Haward, S., Jones, J.D.G.,
Dean, C., Ma, H., and Martienssen, R. (1995). Patterns of gene
action in plant development revealed by enhancer trap and gene
trap transposable elements. Genes Dev. 9, 1797-1810.<BR>

<P>
Swinburne, J., Balcells, L., Scofield, S.R., Jones, J.D.G., and
Coupland, G. (1992). Elevated levels of Ac  transposase mRNA are
associated with high frequencies of Ds excision in Arabidopsis.
 The Plant Cell 4, 583-595.<BR>

<P>
Tsay, Y-F, Frank, M.J., Page, T., Dean, C., and Crawford, N. (1993).
Identification of a mobile endogenous transposon in Arabidopsis
thaliana. Science 260, 342-344.<BR>

<P>
Wilson, K., Long, D., Swinburne, J., and Coupland, G. (1996).
A Dissociation insertion causes a semidominant mutation that increases
expression of TINY, an Arabidopsis gene related to APETALA2. Plant
Cell 8, 659-671.<BR>
<BR>

<P>
<B>Enhancer Traps and Gene Traps in Drosophila and Mouse<BR>
</B>
<P>
Bellen, H.J., O'Kane, C.J., Wilson, C., Grossniklaus, U., Pearson,
R.K., and Gehring (1989). P-element-mediated enhancer detection:a
versatile method to study development in Drosophila. Genes Dev.
3, 1288-1300.<BR>

<P>
Bier, E., Vaessin, H., Shepherd, S., Lee, K., McCall, K., Barbel,
S., Ackerman, L., Carretto, R., Uemera, T. Grell, E., Jan, L.Y.,
and Jan, Y.N. (1989). Searching for pattern and mutation in the
Drosophila genome with a P-lacz vector. Genes Dev. 3, 1273-1287.
<BR>

<P>
Friedrich, G. and Soriano, P. (1991). Promoter traps in embryonic
stem cells: a genetic screen to identify and mutate developmental
genes in mice. Genes Dev. 5, 1513-1523.<BR>

<P>
Gossler, A., Joyner, A.L., Rossant, J., and Skarnes, W.C. (1989).
Mouse embryonic stem cells and reporter constructs to detect developmentally
regulated genes. Science 244, 463-465.<BR>

<P>
Wilson, C., Pearson, R.K., Bellen, H.J., O'Kane, C.J., Grossniklaus,
U., and Gehring, W.J. (1989). P-element mediated enhancer detection:an
efficient method for isolating and characterizing developmentally
regulated genes in Drosophila. Genes Dev. 3, 1301-1313.<BR>

<P>
Wilson, C., Bellen, H.J., and Gehring, W.J. (1990). Position effects
on eukaryotic gene expression. Ann. Rev. Cell. Biol. 6, 679-714.
<BR>
<BR>

<P>
<B>Tissue PCR in Arabidopsis.<BR>
</B>
<P>
Klimyuk, V.I., Carroll, B.J., Thomas, C.M. and Jones, J.D.G. (1993).
Alkali treatment for rapid preparatin of plant material for reliable
PCR analysis. Plant Journal 3, 493-494.<BR>
<BR>

<P>
<B>GUS staining<BR>
</B>
<P>
Gallagher S.R., ed. &quot;GUS Protocols: using the GUS gene as
a reporter of gene expression&quot;.  San Diego: Academic Press
Inc., 1992.<BR>
<P>
<center>Last modified on January 18, 2002</center>

		</td>
	</tr>
</TABLE>
<script language="javascript" src="/js/footer.js"></script>
</body>
</html>
