<html>
<head>
<title>CSH T-DNA Methods</title>
<script language="JavaScript" src="/js/navbar.js"></script>
<link rel="stylesheet" type="text/css" href="/css/main.css" >
<script language='JavaScript'>
var highlight = 0; var helpfile="";
</script>
</head>

<body leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" bgcolor="#F5F9FF">


<!-- HEADER using external JavaScript file -->

<script language='JavaScript' SRC='/js/header'>
</script><p>

<!-- End of header -->


<p>
<table width="602" align="center" border="0" cellspacing="0" cellpadding="2">
	<tr>
		<td width="602">

<P>
<CENTER><B>Lab 13: Gene Machine</b>
<P>
Cold Spring Harbor Arabidopsis Molecular Genetics Course
<P>
July 1996

<P>

Dr. R. Meagher<br>
University of Georgia<br>
Department of Genetics<br>
Life Sciences B402<br>
Athens, GA 30602-7223<br>
phone: (706) 542-1444
</CENTER>

<P>
<B>EXPERIMENTAL PROTOCOLS</B>
<P>
<B>Preparing DNA From Pools of Transgenic Plant Lines</B>
<P>
T-DNA transformed <I>Arabidopsis
thaliana</I> (ecotype Wassilewskija) populations were generated
by the seed transformation protocol and most of these lines have
been described previously (Forsthoefel et al., 1992) .  Two different
methods have been used to prepare the plant populations.<BR>

<P>
<B>Method #1--Growing the
plant population in liquid</B>
<P>
1.  Forty seeds (T3 or
T4) were aliquoted from each of 5 transformed lines and pooled.
 The pooled seeds were surface sterilized (8 min 50%/Clorox bleach)
and rinsed 5X with sterilized distilled water.  
<P>
2.  Each sterilized seed
pool was transferred to a 250 ml flask containing 100 ml of M.
&amp; S. (Gibco BRL) supplemented with 0.5% sucrose (w/v) (pH
= 6.0 before autoclaving).<B>  </B>
<P>
3.  Following a 24 hr cold
treatment (4&deg;C), the flasks containing the seeds were transferred
to a shaker (30 rpm) and grown for 15 days under continuous florescent
light (35 &#181;mol s-1 m-2) at 23-25&deg;C. 
<P>
4.  Approximately 5 g of
tissue were harvested from each pool, blotted dry, frozen in liquid
nitrogen, and stored at -70&deg;C.
<P>
5.  DNA was extracted from
each of two pools of 5 lines (in the text, termed &quot;a subpool
of 10 lines&quot;).<BR>

<P>
<B>Method #2--Growing the
plant population in flats </B>
<P>
1.  One thousand seeds
from a pool of 100 plant lines were sowed directly into 1 flat
(21&quot; x 10.5&quot; x 2.5&quot;) of fine soil mix (Fafard 3B).
 The seeds were mixed with coarse sand in a small salt shaker
and dispersed evenly across the flat. 
<P>
2.  Seeds were grown at
23&deg;C under florescent lights (Growlux) on a cycle of 8 hr light
x 16 hr darkness. 
<P>
3.  The flats were misted
2X a day for the first 10 days and then once a day until harvest
.
<P>
4.  Three-week-old seedlings
were harvested by cutting off all above-ground parts and freezing
immeditately in liquid nitrogen and grinding with a morter and
pestle.  The frozen powdered tissue was stored at -70&deg;C in 10
g batches.<BR>
<BR>

<P>
<B>DNA extraction </B>(McKinney
et al., 1995) 
<P>
1.  DNA was prepared from
5-10 g of frozen tissue (prepared by either method above) using
a modified CTAB (cetyltrimethylammonium bromide) DNA extraction
procedure  (Murray and Thompson, 1980). If not already
done (i.e., Method #1), the frozen tissue was ground to a fine
powder and frozen in a chilled mortar. 
<P>
2.  Ten ml of extraction
buffer (2% CTAB, 100 mM Tris, 20 mM EDTA, 1.4 M NaCl [pH 8.0])
were added to the powder and the mix was incubated for 10 min
at 60&deg;C.  
<P>
3.  Ten ml of chloroform/isoamyl
alcohol (24:1) were added to the incubated mix and vortexed for
10 sec.  The phases were separated by centrifuge (1,500 x g).
 
<P>
4.  The aqueous phase was
transferred to a clean tube and extracted with chloroform/ isoamylalcohol
for a second time.  
<P>
5.  After centrifuging,
the aqueous phase was transferred to a clean tube and 1 ml of
10% (w/v) CTAB was added.  
<P>
6.  The solution was heated
for 2 min at 60&deg;C and CTAB/DNA precipitation was induced by adding
20 ml of deionized water and placing the tubes on ice from 1 hr
to overnight<B>.  </B>Rushing this step or insufficient chilling
causes a major loss of yield.
<P>
7.  The precipitates were
collected by centrifuge and dissolved by adding 5 mls of 1 N NaCl
and heating for 2 min at 60&deg;C.  
<P>
8.   DNA was precipitated
by adding 10 mls of 95% EtOH and was collected by centrifuge.
 The pellet was washed 2X, gently but thoroughly, with 5 mls 70%
EtOH and air dried.  
<P>
9.  The dried DNA pellet
from 10 g of tissue was resuspended in 200 &#181;l of TE. 
<P>
10.  The DNA was quantified
after visualizing an aliquot from each sample (after RNAase A
treatment) on an ethidium bromide-stained agarose gel and comparing
the high molecular weight band with a standard.  Alternatively,
micro fluorimetry (Hoefer TKO 100 mini-fluorometer) and Hoechst
dye 33258 (final concentration 0.1 &#181;g/ml) were used to quantify
a approximately 10 ng sample of DNA.  The typical yield was 40-250
ng of DNA per ml (8-50) &#181;g per 10 g of tissue).
<P>
11.  The pools of 100 lines
were generated by mixing equal amounts of DNA from each of 10
subpools of 10 lines (Plant Population Method #1).
<P>
<B>DNA for Rapid-Screen
Prepared From Individual Transgenic Plant Lines </B>
<P>
(McKinney et al., 1995)

<P>
1.  DNA samples from individual
plant lines were prepared as follows:  One very young leaf 0.5
cm wide was ground briefly with a stainless steel rod in a 1.5
ml eppendorf tube, and then ground further with 100 &#181;l of extraction
buffer (50 mM Tris-HCl pH = 8.0 at 22&deg;C, 200 mM NaCl, 0.2 mM EDTA,
0.5% sodium dodecyl sulfate, 100 &#181;g/ml Proteinase K) and incubated
at 37&deg;C for 30 min. 
<P>
2.  The leaf sample was
then extracted 1X with 100 &#181;l of buffer-saturated phenol and 1X
with 100 &#181;l chloroform at room temperature.  
<P>
3.  The aqueous solution
was made 300 mM in sodium acetate (3M stock) and the nucleic acids
were precipitated with 2 volumes of ethanol.  
<P>
4.  The ethanol pellet
was washed gently 2X with 70% ethanol and resuspended in 100 &#181;l
TE (10 mM Tris-HCl pH=8.0 22&deg;C, 1 mM EDTA).  
<P>
5.  A small portion of the nucleic
acid preparation, 0.5-1 &#181;l, was used for PCR.  Using more plant
material or more of the final product from the purification may
inhibit the PCR reaction.
<P>
<B>T-DNA and Actin Primers </B>(McKinney
et al., 1995) <BR>

<P>
1.  The actin PCR primers presented
in Table I (McKinney et al., 1995)  were synthesized at the University
of Georgia's Molecular Genetics Instrument Facility (not purifed)
and were already in use in our laboratory as DNA sequencing, mapping,
and cloning primers and were not designed for these experiments.
 This accounts for the extreme variation in length, degeneracy,
and percent GC composition.  Among the sequences of a degenerate
primer is a sequence identical to each of the 10 <I>Arabidopsis</I>
 actin genes, with one exception.  The 284A primer contains several
mismatched nucleotides relative to <I>ACT9 </I> and probably will
not amplify this pseudogene.  
<P>
2.  Actin Primer Design and Controls:
 In reconstruction PCR experiments, the two primers used during
the initial screening, 119S and 284A, where each used in conjunction
with either a T7 or Sp6 vector primer and 10 ng of genomic subclones
of <I>ACT2</I>in Bluescript SKII+ (Promega).
<P>
a.  Using the PCR conditions described
herein, each primer pair (i.e., Sp6/284A) could PCR amplify visually
detectable levels (~50 ng) of 5-6 kb genomic/vector fragments.
 
<P>
b.  Neither actin primer generated
a single strong random amplified polymorphic DNA (RAPD) band*
when used alone to amplify products from <I>Arabidopsis</I> DNA.
 This was taken as evidence that the actin primers and amplification
conditions were efficient enough to detect insertions within a
few kbp of the primers. 
<P>
3.  T-DNA primer design:  The
T-DNA LB (LB102A) and RB (RB16843S) primers (Table I) were synthesized
based on the sequence near-ends of T-DNA element as presented
in Genbank (submitted &amp; TiPost37.ba, respectively).
<P>
a.  We determined that each of
these T-DNA or actin primers did not generate a strong RAPD band(s)
when used alone with <I>Arabidopsis</I> DNA.  
<P>
b.  Other variables, such as locating
the T-DNA primers near the border sequences, are discussed in
the text.  It should also be noted that the highest yield of PCR
is generally for products of 200 to 1,000 bp.  We find the molar
yield of products less than 200 bp drops off rapidly with decreasing
size.  This is another advantage of keeping the T-DNA primers
near the border junctions.<BR>
<BR>

<P>
*If a primer generates a few weak
RAPD bands when used alone against genomic DNA, these bands will
not be present when a real PCR product is generated.  In the past
,when primers have given us problem and generated a strong dominant
RAPD band, we have successfully eliminated the problem by changing
the 3' terminal 3-5 nt.  In a coding region, this is done either
by adding or deleting an additional codon to the 3' end of the
original sequence.  In the original primer design, making longer
primers (26-31 mers) has been more successful at increasing product
yield and decreasing background then finding primer sites with
very low degeneracy.  The 3' ends of primers were designed to
avoid any potential to prime internally on themselves and make
primer dimers (no inverse complement to the 3' end of more than
2-3 bp).
<P>
<B>PCR Amplification and Detection
of T-DNA/actin Junction Sequences</B>
<P>
 (McKinney et al., 1995) <BR>

<P>
1.  PCR was performed in 50 &#181;l
reactions topped with mineral oil and containing 0.02-0.05 &#181;g
of the DNA pool in Promega's buffer, 1 unit of Taq enzyme, 5 &#181;g
RNAaseA, 25 pmoles RB or LB oligos, and 50 pmoles of the degenerate
(or 25 pmoles of unique) actin primers and 0.1 mM dNTPs.  
<P>
2.  The protocol started with
an initial 2 min incubation at 94&deg;C, followed by 45 cycles of
PCR (94&deg;C/30 sec, 42&deg;C/30 sec, 68&deg;C/3 min).  When unique primers
are used, we generally have less background and fewer RAPDs if
the  annealing temperature (42&deg;C) is raised to 52-56&deg;C.
<P>
3.  10 &#181;l of the product (mixed
with 5 &#181;l of sample loading mix) was run on a 1% agarose gel 0.5
x 17 x 17 cm with 2x 27 well combs stacked one above the other
8 cm apart. The gel is run at 150V for 1.5 hr, stained with ethidium
bromide, and then photographed with a  ruler next to the gel for
size reference (or mark standards with india ink).
<P>
<B>Southern imprint</B>
<P>
1.  The DNA samples in the 1%
agarose gel were: 
<P>
a) depurinated in 0.25M HCl 2
x 15 min each; 
<P>
b)  rinsed in distilled water
3X;
<P>
c)  denatured in 0.5M NaOH, 1.5M
NaCl: 20-40 min; and 
<P>
d)  neutralized in 3M NaCl, 0.5M
Tris pH 7.0: 2 x 20 min.
<P>
2.  The transfer membrane Biotrans
plus (0.45&#181;M) (ICN Biomedicals) was cut to the dimensions of the
gel, wet in distilled water, and then soaked in 20XSSC until ready
for transfer.
<P>
3.  The DNA was vacuum-transferred
using the Trans-vac TE80 (Hoefer, Inc.), which takes only 60 min,
or by the following overnight method:
<P>
a.  Whatman 3mm paper to be used
as a wick was cut slightly wider (1/2&quot;) and longer (5&quot;)
than the gel.  The ends of the wick were suspended into buffer
from a piece of glass or plastic that was set on top of a baking
dish.  The wick was wetted and placed in contact with the 20XSSC
in the baking dish reservoir.  The gel was then placed on top
of the wick, avoiding air bubbles.
<P>
b.  20 X SSC was poured onto the
gel and the Biotrans placed on top (avoiding air bubbles between
gel and filter).
<P>
c.  Two pieces of Whatman 3MM,
wetted in 20XSSC, were cut in the same dimensions as the gel and
placed on top of the membrane.
<P>
d.  Four to five inches of paper
towels (slightly smaller than the Whatman paper) were placed carefully
on top of the gel-membrane-filter paper sandwich.
<P>
e.  A piece of plastic or glass
was placed on top of the paper towels and topped with a small
weight.
<P>
f.  The DNA transfer was allowed
to proceed overnight.  The stack was dissembled and the wells
were marked with a nitrocellulose marking pen (S &amp; S).
<P>
4.  After either transfer method,
the Biotrans membrane was remove from gel and the membrane exposed,
nucleic acid side up, to strong UV light (6 &quot;) for 2 min.
 The membrane was then baked for 2 hr in a 60-80&deg;C oven.  The
membrane is now ready for hybridization.
<P>
5.  As a probe, full-length <I>ACT2
</I> cDNA<B> </B>(An et al., 1996)  was PCR amplified with Sp6
and T7 primers from a pcDNAII (Invitrogen) clone and labeled with
<sup>32</sup>P-ATP by the random primer method to a specific activity of
~300 x 10<sup>6</sup>/&#181;g.  
<P>
6.  The DNA containing filters
were prehybridized for 4.5 hrs at 40&deg;C in 35% formamide, 6x SSC,
5x Denhardt's (Maniatis et al., 1989) , 0.5% SDS, 25mM sodium
phosphate pH 6.5, 50 &#181;g/ml tRNA (Sigma), and 0.1% gelatin, hybridized
with probe for 2 days at 40&deg;C in the prehybridization mix
<P>
7.  Filters were washed 4X at
50-56&deg;C for 10 min, 3x SSC, 0.2% sodium dodecyl sulfate (McDowell
et al., 1996) .  
<P>
8.  The filters were typically
exposed to x-ray film with one amplification screen for 30 min
or as indicated in the text.  
<P>
<B>Sequences of actin/T-DNA Junctions</B>
(McKinney et al., 1995) <BR>

<P>
1.  The actin/T-DNA junctions
at both ends of the insertions in <I>act2-1</I> and <I>act4-1
</I>were PCR amplified (100&#181;l reactions) using gene-specific or
degenerate upstream or downstream primers (Table I) and the appropriate
T-DNA border primer.  Sometimes a series of nested PCR reactions
were used to increase the PCR yield.  First reaction with an outer
set of oligos often yields a product so weak it cannot be visualized
on a gel.  This reaction was then isopropanol precipitated and
resuspended in 5-20 &#181;l water.  Then 2-5 &#181;l of this product was
used as template for a PCR reaction using an inner set of oligos.
<P>
2.  The PCR product was phenol
extracted and precipitated with isopropanol.  
<P>
3.  This product was subjected
to cycle sequencing (fmol DNA Sequencing System, Promega, Inc.),
priming the reaction with a <sup>32</sup>P end-labeled T-DNA border, actin
gene-specific, or actin degenerate oligonucleotide.
<P>
<B>References to Methods<BR>
</B>
<P>
<B>An, Y.-Q., Huang, S., McDowell,
J.M., McKinney, E.C., and Meagher, R.B.</B>
(1996). Conserved Expression of the <I>Arabidopsis ACT1 </I>and
<I>ACT3</I> actin subclass in organ primordia and mature pollen.
Plant Cell <B>8</B>, 15-30.
<P>
<B>Forsthoefel, N.R., Wu, Y.,
Schulz, B., Bennett, M.J., and Feldmann, K.A.</B>
(1992). T-DNA insertion mutagenesis in <I>Arabidopsis</I>: prospects
and perspectives. Aust. J. Plant Physiol. <B>19</B>, 353-366.
<P>
<B>Maniatis, T., Fritsch, E.F.,
and Sambrook, J. </B>(1989).
Molecular cloning:  A laboratory Manual.  In:  Molecular cloning:
 A laboratory Manual, <B>2,</B> Editor, eds (Cold Spring Harbor,
New York: Cold Spring Harbor laboratory Press), 
<P>
<B>McDowell, J.M., Huang, S.,
McKinney, E.C., An, Y.-Q., and Meagher, R.B.</B>
(1996). <I>Arabidopsis  thaliana </I>contains ten actin genes
encoding six ancient protein subclasses. Genetics <B>142</B>,
587-602.
<P>
<B>McKinney, E.C., Ali, N., Traut,
A., Feldmann, K.A., Belostotsky, D.A., McDlowell, J.A., and Meagher,
R.B.</B> (1995). Sequence
Based Identification of T-DNA Insertion Mutations in <I>Arabidopsis:</I>
Actin Mutants <I>act2-1</I> and <I>act4-1</I>. Plant J. <B>8</B>,
613-622.
<P>
<B>Murray, M.G., and Thompson,
W.F.</B> (1980). Rapid
isolation of high molecular weight plant DNA. Nuc. Acids Res.
<B>8</B>, 4321-4325. <BR>
<P>
<center>Last modified on January 18, 2002</center>

		</td>
	</tr>
</TABLE>

<!-- FOOTER -->
<script language="javascript" src="/js/footer.js"></script>
</body>
</html>
