<html>
<head>
<title>TAIR</title>
<script language="JavaScript" src="/js/navbar.js"></script>
<link rel="stylesheet" type="text/css" href="/css/main.css" >
<script language='JavaScript'>
var highlight = 0; var helpfile="";
</script>
</head>

<body leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" bgcolor="#F5F9FF">


<!-- HEADER using external JavaScript file -->

<script language='JavaScript' SRC='/js/header'>
</script><p>

<!-- End of header -->


<p>
<table width="602" align="center" border="0" cellspacing="0" cellpadding="2">
	<tr>
		<td width="602">

<P>
<CENTER><B>Lab 5: <I>in situ</I> Hybridization with Nonradioactive Probes</b>
<p>
Cold Spring Harbor Arabidopsis Molecular Genetics Course
<P>
July, 1996
<p>

Drews Lab<br>
Department of Biology<br>
University of Utah<br>
(801) 585-6204

<p>

Okamuro Lab<br>
Department of Biology<br>
UC Santa Cruz<br>
(408) 459-2612

</Center>

<P>
<B>I.  FIXATION AND EMBEDDING
<BR>
</B>
<P>
<B>A.  Required Materials:<BR>
</B>
<P>
Bulk 95% ethanol--use this for
everything except the 100% ethanol steps
<P>
Bulk 100% ethanol --use only for
the 100% ethanol steps
<P>
Glacial acetic acid
<P>
37% formaldehyde
<P>
20 ml glass scintillation vials
(e.g. S/P Baxter #R2550-14)
<P>
Xylenes
<P>
Eosin Y; Eosin Yellow (Manufacturing
Chemists #B286)
<P>
Paraplast Plus Chips (contains
&lt;1% DMSO)
<P>
Paraplast Chips
<P>
Index cards (3x5)
<P>
Superfrost Plus Treated Microscope
Slides (Fisher # 12-550-15)
<P>
Dissecting needle<BR>
<BR>

<P>
<B>B.  Required Equipment:<BR>
</B>
<P>
Oven set to 60&#176;C
<P>
Eberbach Micro Slide Warming Table
(product #2750; phone, 800-422-2558)
<P>
TissuePrep Flotation Bath (e.g.
Lab-Line # 26103/Baxter # M7655) set to 42&#176;C .
<P>
Slide warmer (e.g. Fisher # 12-594)
set to 45-50&#176;C.
<P>
Microtome<BR>
<BR>

<P>
<B>C.  Fixation:<BR>
</B>
<P>
We have tried a variety of fixation
procedures.  For <I>Arabidopsis</I> floral tissue, the best signals
come from tissue fixed in FAA.  Floral tissue fixed in glutaraldehyde
gives no signal.  There are numerous reports that paraformaldehyde-fixed
tissue gives the best signals.  We have not compared FAA-fixed
and paraformaldehyde-fixed tissue.  <BR>

<P>
<B>1.</B>
 Fixative Mix:<BR>

<TABLE>

<TR><TD WIDTH=40></TD><TD WIDTH=108></TD><TD WIDTH=108><CENTER><U>% in Mix</U></CENTER>
</TD><TD WIDTH=108><CENTER><U>Amount Add</U></CENTER>
</TD></TR>

<TR><TD WIDTH=40></TD><TD WIDTH=108>Ethanol
</TD><TD WIDTH=108><CENTER>50.0%</CENTER>
</TD><TD WIDTH=108><CENTER>50 ml</CENTER>
</TD></TR>

<TR><TD WIDTH=40></TD><TD WIDTH=108>Acetic Acid
</TD><TD WIDTH=108><CENTER>  5.0%</CENTER>
</TD><TD WIDTH=108><CENTER> 5 ml</CENTER>
</TD></TR>

<TR><TD WIDTH=40></TD><TD WIDTH=108>37% Formaldehyde
</TD><TD WIDTH=108><CENTER>  3.7%</CENTER>
</TD><TD WIDTH=108><CENTER>10 ml </CENTER>
</TD></TR>

<TR><TD WIDTH=40></TD><TD WIDTH=108>Water
</TD><TD WIDTH=108><CENTER>41.3%</CENTER>
</TD><TD WIDTH=108><CENTER><U>35 ml</U></CENTER>
</TD></TR>

<TR><TD WIDTH=40></TD><TD WIDTH=108></TD><TD WIDTH=108></TD><TD WIDTH=108><CENTER>100 ml</CENTER>
</TD></TR>

</TABLE>

<P>
<B>2.</B>
 Place 10-15 ml of fixative into 20 ml scintillation vials.<BR>

<P>
<B>3</B>.
 Cut a cluster of flowers at the apex of the floral stem that
includes stages 1-14, and immediately immerse in fixative.  Cut
so that 1-2 mm of floral stem is present--this helps to orient
the tissue when sectioning.  Cut off older flowers if you are
not interested in these stages.  If possible, cut tissue while
immersed.  Place as many pieces of tissue that will easily cover
the bottom of a vial as a single layer of tissues.  <BR>

<P>
<B>4.</B>
 The tissue will float in the fixative.  If desired, the tissue
can be forced down with a Nytex screen cut to the size of the
scintillation vials.<BR>

<P>
<B>5.</B>
 Place tissue/fixative in a vacuum (&gt;20 atm) for 15 minutes.
 Pull the vacuum very slowly (this should take at least 10 minutes).
 This step pulls the air out of the tissue, allowing the fixative
to penetrate better.  Usually, air bubbles will come out of the
solution and get trapped in the trichomes, causing the tissue
to float.<BR>

<P>
<B>6</B>.
 After 15 minutes, release the vacuum very slowly.  Again this
should take at least 10 minutes.    <BR>

<P>
<B>7</B>.
 Gently swirl contents of vial to remove trapped air bubbles.
 At this point the tissue will usually sink when the second vacuum
pulse is released.  If the tissue does not sink, repeat steps
5 and 6 until the tissue does sink.  <BR>

<P>
<B>8</B>.
 Incubate at room temperature until the tissue has been exposed
to the fixative for a total of 16 hours.  NO LONGER!  Overfixation
can result in the loss of signal.<BR>

<P>
<B>9</B>.
 Rinse out fixative--remove fixative and add 50% ethanol.  Incubate
at room temperature for 30 minutes.  Repeat this step.  <BR>
<BR>

<P>
<B>D.  Dehydration:<BR>
</B>
<P>
The tissue must be completely
dehydrated before exposure to xylenes; otherwise, the water and
xylenes will form a white emulsion.  We also stain the tissue
with Eosin Y during these steps to help orient the tissue when
sectioning.  This is an optional step.<BR>

<P>
<B>1</B>.
 Remove 50% ethanol and replace with 60% ethanol.  Incubate for
30 minutes.<BR>

<P>
<B>2</B>.
 Repeat for the following ethanol solutions:  70%, 85%, 95%.<BR>

<P>
<B>3</B>.
 Leave overnight in 95% ethanol with 0.1% Eosin Y.<BR>

<P>
<B>4</B>.
 Next day, remove as much as possible of the Eosin Y/95% ethanol
and replace with 100% ethanol.  Incubate for 1 hour.  The tissue
destains slowly in 100% ethanol, so do not incubate too long (e.g.
overnight).  <BR>

<P>
<B>5</B>.
 Remove as much as possible of the 100% ethanol, replace with
fresh 100% ethanol, and incubate for 30 minutes.  Repeat this
step if it is impossible to remove virtually all of the solution
in steps 4 and 5.<BR>
<BR>

<P>
<B>E.  Clearing:</B>
<P>
 
<P>
The tissue must be permeated with
xylenes because paraffin is not miscible in ethanol.  The tissue
does not destain (Eosin) in xylenes.<BR>

<P>
<B>1.</B>
 Remove 100% ethanol and replace with 25% xylenes:75% ethanol.
 Incubate at room temperature for 30 minutes.<BR>

<P>
<B>2.</B>
 Repeat for the following xylene:ethanol solutions:<BR>

<P>
 50% xylenes:50% ethanol
<P>
 75% xylenes:25% ethanol<BR>

<P>
<B>3</B>.
 Remove the 75% xylenes and replace with 90% xylenes:10% chloroform.
 Incubate at room temperature for 1 hour.<BR>

<P>
<B>4</B>.
 Repeat step 3 two times.  Fill the vial half full with 90% xylenes:10%
chloroform the final time.

<p><B>F.  Infiltration:<BR>
</B>
<P>
Xylenes is typically used as an
organic solvent for infiltration of paraffin.  In the presence
of 100% xylenes paraffin will sink to the bottom of the vial,
resting on the tissue.  By contrast, in the presence of 90% xylenes:10%
chloroform, paraffin floats.  For this reason, we use 90% xylenes:10%
chloroform as an organic solvent to allow for a more gradual infiltration
of paraffin into the tissue.  <BR>

<P>
<U>Preliminary step</U>.
 Place a beaker full of Paraplast Plus chips in an oven at 60&#176;C
well in advance; it will take &gt;5 hours to melt.<BR>

<P>
<B>1</B>.
 Add 20 chips of Paraplast Plus to each vial.  Incubate overnight
at room temperature.  If in a hurry, go to step 2 without the
overnight incubation.<BR>

<P>
<B>2</B>.
 Next morning, the paraplast will be only partially into solution.
 Place vials in a 42&#176;C incubator.  After about 30 minutes,
the paraplast chips will be in solution.  <BR>

<P>
<B>3</B>.
 Add 20 more Paraplast Plus chips, and incubate at 42&#176;C until
chips are in solution--about  30 minutes.  Swirl occasionally.
<BR>

<P>
<B>4</B>.
 Repeat step 3 until the vial is full (4-5 times; total of about
100 chips).<BR>

<P>
<B>5</B>.
 Pour off xylene/paraplast solution.  Add molten Paraplast Plus.
 Swirl to mix.  Incubate at 57-62&#176;C for at least 10 hours.
<BR>

<P>
<B>6</B>.
 Change the molten paraplast twice a day (morning and night; minimum
10 hrs. between changes) for 2 days.  If you are able to remove
almost all of the molten paraplast each time, 4 changes should
be adequate.  If your tissue floats and you are not able to remove
most of the wax, change the molten paraplast a few extra times.
<BR>
<BR>

<P>
<B>G.  Pouring Boats:<BR>
</B>
<P>
<B>1</B>.
 Fold up an index card so that it has raised edges of about 1
cm.  Tape the edges with Scotch tape inside and outside.<BR>

<P>
<B>2</B>.
 Place the boat on the hottest part of the Eberbach hot plate.
<BR>

<P>
<B>3</B>.
 Pour the infiltrated tissue into the boat.<BR>

<P>
<B>4</B>.
 Top off the boat with molten <U>Paraplast</U> (not Paraplast
Plus).<BR>

<P>
<B>5</B>.
 Arrange the tissue into a regular array using a dissecting needle.
 Orient tissue pieces with the stems either straight up (for transverse
sections) or lying on their side (for longitudinal sections).
 Tissue pieces must be at least 5 mm apart.<BR>

<P>
<B>6</B>.
 Slowly move the boat to the cooler part of the hot plate.  You
probably will need to arrange the tissue again.<BR>

<P>
<B>7</B>.
 Label each boat by inserting a flagged string.<BR>

<P>
<B>8</B>.
 To harden, float the boat for greater than or equal to 1 hour in water.
<P>
<B>H.  Sectioning:<BR>
</B>
<P>
<B>1</B>.
 Cut out blocks of embedded tissue and mount onto microtome blocks.
<BR>

<P>
<B>2</B>.
 Section tissue at 8-10 &#181;m.<BR>

<P>
<B>3</B>.
 Cut ribbons into ~1.5 cm pieces.  Float ribbon pieces on 42&#176;C
water for &gt;1 minute.  This step takes the compression out of
the tissue.<BR>

<P>
<B>4</B>.
 Put slide in water just under the floating ribbon.<BR>

<P>
<B>5</B>.
 Bring slide up so as to catch the ribbon.  Use a teflon-coated
spatula to position the ribbon.  Minimize the amount of water
trapped between section and slide.  Place slide on slide warmer
for at least 30 minutes before adding more ribbon pieces.  (Note:
 a  convection or gravity air incubator  set at 45-50&#176;C can
also be used to affix ribbons to slides).<BR>

<P>
<B>6</B>.
 Repeat for as many different ribbon pieces as you want to place
on a given slide.<BR>

<P>
<B>7</B>.
 Incubate slide on a slide warmer greater than or equal to 24 hours (the longer the
better) to &quot;bake&quot; the ribbon piece onto the slide. 
This is a very important step.  This must be done immediately,
and the slide warmer must be set to 45-50&#176;C or the sections
may fall off during the hybridization and wash steps.  Use the
dustcover.
<P>
<B>II. PROBE SYNTHESIS<BR>
</B>
<P>
<B>A.  The probes:<BR>
</B>
<P>
Generally two probes are synthesized.
 The first probe hybridizes with the RNA of interest, and is generally
called antisense, anti-mRNA, or (+) strand probes.  You must also
synthesize a control probe that will not hybridize with the RNA
of interest.  The most convenient thing to do is to synthesize
an RNA probe in the opposite orientation as the anti-RNA probe.
 Probes of this type are called sense, mRNA, or (-) strand probes.
 <BR>

<P>
A typical experiment consists
of 10 slides; 8 of these are hybridized with the anti-RNA probe
and 2 are hybridized with the control probe.  <BR>
<BR>

<P>
<B>B.  Solutions:<BR>
</B>
<P>
<B>1.</B>
 200 mM EDTA  (DEPC-treated and autoclaved)<BR>

<P>
<B>2.</B>
 4 M LiCl  (DEPC-treated and autoclaved)<BR>

<P>
<B>3.</B>
 200 mM Na<sub>2</sub>CO<sub>3</sub>  (pH = 11.4)--prepare just prior to use.<BR>

<P>
<B>4.</B>
 200 mM NaHCO<sub>3</sub>   (pH = 8.2)--prepare just prior to use.<BR>

<P>
<B>5.</B>
 1 M NaOAc pH 4.7 (DEPC-treated and autoclaved)
<P>
 pH must be adjusted with acetic
acid<BR>

<P>
<B>6. </B>
TE (DEPC-treated and autoclaved)<BR>
<BR>

<P>
<B>C.  Materials:<BR>
</B>
<P>
<B>1.</B>
 Boehringer Mannheim RNA Labeling Kit (#1175 025) <BR>

<P>
<B>2.</B>
 20 mg/ml RNase-free Glycogen (Boehringer Mannheim #901 393):
<BR>

<P>
<B>D.  Template preparation:<BR>
</B>
<P>
<B>1.</B>
 Digest with the appropriate restriction enzyme to linearize the
template so that a &quot;run-off&quot; transcript can be generated.
 Avoid the use of enzymes that generate 3' overhangs--they can
act as promoters for the polymerases.<BR>

<P>
<B>2.</B>
 Phenol/chloroform extract once, and chloroform extract twice.
<BR>

<P>
<B>3.</B>
 Precipitate the DNA, and resuspend in RNase-free TE at about
1 mg/ml.<BR>

<P>
<B>4.</B>
 Determine the exact [DNA] using the spectrophotometer.<BR>
<BR>

<P>
<B>E.  Probe synthesis:<BR>
</B>
<P>
<B>1.</B>
 Transcription reaction:<BR>

<P>
The 20 &#181;l reaction described
here should yield ~10 &#181;g of DIG-labelled RNA.  If more DIG-labelled
RNA is required, scale up the reaction proportionally.<BR>

<P>
 <B>(a)</B>  Mix:<BR>

<P>
  H<sub>2</sub>O   12 &#181;l
<P>
  10x Transcription Buffer (vial
8)      2 &#181;l
<P>
  NTP Labeling Mix (vial 7)  
 2 &#181;l
<P>
  Template DNA    1 &#181;l  (1 &#181;g)
<P>
  RNase Inhibitor (vial 10)  
 1 &#181;l  (20 units)<BR>

<P>
 <B>(b)</B>  Mix and remove a
0.5 &#181;l sample for a gel.<BR>

<P>
 <B>(c)</B>  Add 2 &#181;l of
RNA polymerase (SP6, T3, or T7).<BR>

<P>
 <B>(d)</B>  Incubate at 37&deg;C
for greater than or equal to 2 hours.<BR>

<P>
 <B>(e)</B>  Remove a 0.5 &#181;l
sample for a gel.<BR>

<P>
<B>2.</B>
 DNase treatment:<BR>

<P>
 <B>(a)</B>  Add 2 &#181;l of
RNase-free DNase (vial 9).<BR>

<P>
 <B>(b)</B>  Incubate at 37&deg;C
for 30 min.<BR>

<P>
 <B>(c)</B>  Remove a 0.5 &#181;l
sample for a gel.<BR>

<P>
<B>3.</B>
 Run all 3 gel samples on a 1% TBE (nondenaturing) minigel.  Include
a DNA size marker on the gel.  Do not bother to run a RNA  size
marker--it will not look good.  If the DNase reaction works, the
template band will disappear.  If the synthesis works well, you
will see a thick RNA band of about the correct size (DNA and RNA
do not migrate equivalently).  The gel is nondenaturing, so the
RNA band may not be tight.  If you see multiple bands, you may
be able to solve the problem by denaturing your samples at 80&#176;C
for 5 minutes followed by quenching on ice before loading.<BR>

<P>
<B>4.</B>
 If everything looks good, add 2 &#181;l of 200 mM EDTA to stop
the reaction.<BR>

<P>
<B>5.</B>
 Precipitate by adding 2.5 &#181;l of 4 M LiCl and 75 &#181;l
of Ethanol.  Mix well and incubate at  -20&deg;C overnight.  Do not
add tRNA carrier because its presence will interfere with monitoring
the hydrolysis reaction in step D.3.<BR>

<P>
<B>6.</B>
 Centrifuge, wash pellet 2X in 70% ethanol, and dry pellet in
the speedvac.  Resuspend in 100 &#181;l of DEPC-treated H<sub>2</sub>O. 
Let resuspend for 1-2 hours on ice.  <BR>

<P>
<B>7.</B>
 Run a 1 &#181;l sample on a gel to check recovery.<BR>

<P>
<B>8.</B>
 If your recovery looks bad, chances are that it is because the
probe was not resuspended well.  If this occurs, incubate the
RNA at 55&#176;C for 5-10 min to get into solution.<BR>

<P>
<B>9. </B>
Store frozen at -80&#176;C until ready for the hydrolysis step.
<BR>
<BR>

<P>
<B>Note:  </B>Boehringer-Mannheim
recommends a modified version of the transcription reaction if
you are synthesizing probes of &lt;500 bp.  The modifications
for a typical 20 &#181;l reaction include:  (1) using 500 ng gel
isolated insert DNA as a template; (2) increasing the reaction
time from 2 to 4-6 hrs.; (3) increasing the spermidine concentration
by 1-5 mM; and (4) adding 40 U RNase inhibitor (Boehringer-Mannheim
#799 017).   <BR>
<BR>

<P>
<B>F.  Probe Hydrolysis:<BR>
</B>
<P>
We reduce the size of our probes
to allow better penetration into the tissue.  Small probes (50-100
nt) give higher signals than larger probes (&gt;150 nt).   We
chemically degrade our probes to a mean length of ~100 nt.  The
RNA probes are chemically degraded by incubation in a pH 10.2
carbonate buffer at 60&deg;C.  <BR>

<P>
<B>1.</B>
 Before doing the hydrolysis reaction, test the hydrolysis buffers
by mixing 5 ml of H<sub>2</sub>O with 2 ml of 200 mM NaHCO<sub>3</sub>  and 3 ml of
200 mM Na<sub>2</sub>CO<sub>3</sub>.  The pH should be ~10.2.   <BR>

<P>
<B>2. </B>
Calculate incubation time:<BR>

<P>
 Incubation time (in minutes)
= (L<sub>o</sub> - Lf) / (K)(L<sub>o</sub>)(Lf)
<P>
   t = hydrolysis in minutes
<P>
   Lo = starting length in kb
<P>
   Lf = final length in kb = 0.100
kb
<P>
   K =  rate constant for hydrolysis
= .11 kb<sup>-1</sup> min<sup>-1</sup>
<P>
 Sample calculation, 1 kb transcript
to a mean size of 100 nt:
<P>
  [(1) - (0.100)]/[(0.11)(1.0)(0.100)]
= 82 minutes.<BR>

<P>
<B>3.</B>
 Hydrolyze half of your probe: <BR>

<P>
 <B>(a)</B>  Take off a 2 &#181;l
sample as an unhydrolysed control.
<P>
 <B>(b)</B>  Take off 50 &#181;l
for hydrolysis.  
<P>
 <B>(c)</B>  To the 50 &#181;l
of RNA, add: 
<P>
  20 &#181;l 200 mM NaHCO<sub>3</sub>  (add
first)
<P>
  30 &#181;l 200 mM Na<sub>2</sub>CO<sub>3</sub>   
<P>
 <B>(d)</B>  Mix and incubate
at 60&#176;C for the calculated time in the paraffin oven.
<P>
 <B>(e)</B>  Stop the reaction
by adding 10 &#181;l of 1 M NaOAc pH 4.7.<BR>
<BR>

<P>
<B>4.</B>
 Precipitate the probes by adding the following:<BR>

<P>
     1 &#181;l 20 mg/ml Glycogen
(optional)
<P>
   10 &#181;l 4 M LiCl<sub>2</sub>    
<P>
 300 &#181;l Ethanol<BR>

<P>
Incubate at -20&#176;C overnight.
<BR>
<BR>

<P>
<B>G.  Prepare Probe for Hybridization:
<BR>
</B>
<P>
<B>1.</B>
 Spin down the probe, wash 2X with 70% ethanol, and resuspend
in 50 &#181;l of TE.<BR>

<P>
<B>2.</B>
 There is no good way to quantify a DIG-labeled probe.  We estimate
the concentration of the probe two ways:  <BR>

<P>
<B>(a)  </B>Remove
greater than or equal to 5 &#181;l and run on a gel to check the size and to estimate
probe concentration.<BR>

<P>
<B>(b)</B>
 [RNA] can be estimated by scanning a 1:50 dilution using 100
&#181;l cuvette in the spectrophotometer.  Expect to obtain between
100-200 &#181;g/ml.<BR>

<P>
<B>3. </B>
To determine the best [probe] to use, we do a pilot hybridization
experiment using 200, 400, and 800 ng/ml per kb of probe complexity.
 <BR>

<P>
<B>4.</B>
 For long-term storage (months), it is essential that the probe
be stored in hybridization solution.  We usually dilute the probe
to 10 &#181;g/ml in hybridization solution for long-term storage.
 RNase inhibitor (1-2 &#181;l) can also be added to inhibit possible
contaminating RNases.<BR>
<BR>

<P>
<B>III. HYBRIDIZATION<BR>
</B>
<P>
<B>A.  Solutions:<BR>
</B>
<P>
<B>1.</B>
 Set up 10 staining dishes containing the following ethanol solutions:
 100%, 100%, 95%, 85%, 70%, 50%, 30%, 15%, H<sub>2</sub>O, H<sub>2</sub>O.  Will need
250 ml of each solution if hybridizing less than or equal to 10 slides (small
staining dishes) or 500 ml if hybridizing 11-20 slides (large
staining dishes).<BR>

<P>
<B>2.</B>
 <U>20X SSPE</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=234>0.2 M NaH<sub>2</sub>PO<sub>4</sub>
<P>
20 mM EDTA
<P>
3.00 M NaCl
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=234> 100 ml per experiment
</TD></TR>

<TR><TD WIDTH=112>Formula for 1 L:
</TD><TD WIDTH=234>  27.6 g NaH<sub>2</sub>PO<sub>4</sub>-H<sub>2</sub>O
<P>
175.3 g NaCl
<P>
    7.4 g EDTA 
<P>
Adjust pH to 7.4, increase vol to 1 L
<P>
Autoclave
</TD></TR>

</TABLE>

<P>
<B>3.</B>
 <U>Proteinase K Solution</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=234>100 mM Tris (7.5)
<P>
  50 mM EDTA
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=234>250 ml for less than or equal to 10 slides
<P>
500 ml  for 11-20 slides
</TD></TR>

<TR><TD WIDTH=112>Formula for 500 ml:
</TD><TD WIDTH=234>50 ml 1 M Tris (7.5) or 6.1 g Tris base
<P>
50 ml 0.5 M EDTA  or 9.3 g EDTA
<P>
Adjust pH to 7.5, increase vol to 500 ml
</TD></TR>

</TABLE>

<P>
<B>4.</B>
 <U>50% Dextran Sulfate</U> <BR>

<P>
We use Dextran Sulfate from Pharmacia
(# 17-0340-01)
<P>
Make up 25-50 ml and store in
the refrigerator
<P>
NOTE:  50% dextran sulfate is
difficult to pipet, especially when cold.  Warm it up by placing
in the microwave for a short time (~15 seconds)<BR>

<P>
<B>5.</B>
 <U>10 mg/ml RNase-free yeast tRNA</U>  <BR>

<P>
We use tRNA from BRL (# 5401)
<P>
Stored in the -20 freezer<BR>
<BR>

<P>
<B>6.</B>
<U>10X Hybridization salts</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=234>100 mM Tris (7.5)
<P>
  10 mM EDTA
<P>
 3.0 M NaCl
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=234>1 ml per experiment
</TD></TR>

<TR><TD WIDTH=112>Formula for 100 ml:
</TD><TD WIDTH=234>10 ml 1 M Tris (7.5) or 1.2 g Tris base 
<P>
  2 ml 0.5 M EDTA or 372 mg EDTA
<P>
17.5 g NaCl
</TD></TR>

</TABLE>

<P>
<B>7</B>.
 10X Blocking Reagent (10X BR);<BR>

<P>
See Immunological Detection section
for composition and formula
<P>
Need &lt;1 ml per experiment
<P>
<B>8.</B>
 <U>Hybridization Solution</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=351>  50 % Formamide
<P>
300 mM NaCl
<P>
  10 mM Tris (7.5)
<P>
    1 mM EDTA
<P>
    5 % Dextran sulfate
<P>
    1 % Blocking reagent
<P>
150 &#181;g/ml tRNA
<P>
500 &#181;g/ml  Poly A (only if hybridizing with cDNA clone sequences)
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=351>Use ~150 &#181;l per slide for a surface area of 24 x 50 mm.  For convenience, make up 10 ml at a time and store frozen.  
</TD></TR>

<TR><TD WIDTH=112>Formula for 10 ml:
</TD><TD WIDTH=351>1.85 ml H<sub>2</sub>O
<P>
5.00 ml Formamide
<P>
1.00 ml 10X Hybridization salts
<P>
1.00 ml 50% Dextran sulfate 
<P>
1.00 ml 10X Blocking reagent
<P>
0.15 ml 10 mg/ml tRNA 
</TD></TR>

</TABLE>

<P>
<B>9.</B>
<U>Other Reagents/Materials</U>:<BR>

<P>
Distilled Formamide  (We use BRL
# 5515)
<P>
Tupperware box for hybridization
(22 x 33 x 6 cm).  We bought our box from Life Science Products,
Inc. (1-800-245-5774), product #T-541
<P>
Plastic test tube rack that fits
in the Tupperware box.  A 13 mm Nalgene Unwire rack (Baxter #S9248-13)
works well, but it must cut in half to shorten its height
<P>
Coverslips
<P>
Sigmacote (Sigma #SL-2)
<P>
<B>B.  Affix sections to slides:
<BR>
</B>
<P>
<B>1.</B>
 Before hybridizing, place the slides in a 45&deg;C incubator overnight
or longer to bake the sections onto the slides.  Should be done
immediately after sectioning--baking later does not help.<BR>

<P>
<B>C.  Set up for experiment:
<BR>
</B>
<P>
<B>1.</B>
 Prewarm proteinase K solution to 37&#176;C in a staining dish.
<BR>

<P>
<B>2.  </B>Set
up staining dishes with alcohol solutions for dehydration step<B>.
<BR>
</B>
<P>
<B>3.  </B>Remove
Hybridization solution from freezer and warm to room temperature.
<BR>

<P>
<B>4.</B>
  Treat coverslips with Sigmacote:<BR>

<P>
 (a)  Dip coverslips in Sigmacote
and air dry.  <BR>

<P>
 (b)  Dip coverslips in 100% alcohol
and air dry.  <BR>

<P>
<B>D.  Dewaxing:<BR>
</B>
<P>
<B>1.</B>
 Put slides in a staining dish slide holder.<BR>

<P>
<B>2.</B>
 Fill a staining dish with xylenes, and place a small stir bar
at the bottom.  Place slides into this staining dish.<BR>

<P>
<B>3.</B>
 Stir for 10 minutes.  Stir very slowly--just fast enough to mix
the solution a bit.<BR>

<P>
<B>4.</B>
 Repeat with fresh xylenes.<BR>

<P>
<B>5.</B>
 If you are processing many racks of slides, you can use the same
two xylenes solutions.<BR>

<P>
<B>E.  Hydration:<BR>
</B>
<P>
<B>1.</B>
 Remove slides from xylenes and place into 100% ethanol (95% ethanol
will not work).  Dip up and down about 15 times or until the &quot;streaks&quot;
go away.  Incubate for 5 minutes to remove Eosin Y. <BR>

<P>
<B>2.</B>
 Repeat step 1.  <BR>

<P>
<B>3.</B>
 Process the slides through the following ethanol solutions: 
95%, 85%, 70%, 50%, 30%, 15%, H<sub>2</sub>O, H<sub>2</sub>O.  Once again, dip up and
down 15 times or until the &quot;streaks&quot; go away.  Begin
with the 95% ethanol and end with the H<sub>2</sub>O, so as to hydrate the
tissue gradually.  If the sections are still orange in the 85%
solution, leave in 85% for no longer than 5 minutes.  <BR>

<P>
<B>4.</B>
 Change the last water for each rack of slides.<BR>

<P>
<B>F.  Proteinase K digestion:
<BR>
</B>
<P>
Proteinase K is used to partially
digest the tissue to allow better probe penetration.  This step
increases the signal.  For <I>Arabidopsis</I> floral tissue, the
best incubation time is 30-45 minutes.  Less time gives a weaker
signal, and greater time tends to destroy the morphology of the
tissue.  The timing was calibrated with a Proteinase K solution
that was prepared just before the incubation.  <BR>

<P>
<B>1.</B>
 Prewarm the Proteinase K Solution to 37&deg;C in a staining dish.
<BR>

<P>
<B>2.</B>
 Add Proteinase K to the prewarmed solution to a final concentration
of 1 &#181;g/ml.  We make up a fresh 5 mg/ml solution and add
50 &#181;l of this to 250 ml of solution (100 &#181;l to 500 ml
if using large staining dishes).  <BR>

<P>
<B>3.</B>
 Incubate at 37&deg;C for 30 minutes.<BR>

<P>
<B>4.</B>
 After the incubation, remove the slides from the Proteinase K
solution and place into a staining dish filled with H<sub>2</sub>O.  Dip
up and down a few times to rinse off the Proteinase K.  Repeat.
<BR>

<P>
<B>G.  Dehydration<BR>
</B>
<P>
<B>1.</B>
  Process the slides through the following ethanol solutions:
 H<sub>2</sub>O, 15%, 30%, 50%, 70%, 85%, 95%, 100%.  You can use the same
solutions used in step E except for the 100% solution.  <BR>

<P>
<B>2.   </B>Air
dry sections in a desiccator under vacuum for about 1 hour.  
<BR>
<BR>

<P>
<B>H.  </B>
<B>Hybridization</B>  <BR>

<P>
<B>1.  </B>Look
at the slides and rank them according to quality.  Plan to hybridize
~8 slides with the anti-mRNA probe and ~2 slides with the control
probe.  Use the best 2-3 slides for hybridizing with the anti-mRNA
probe.  Be sure to save some reasonably good slides for hybridizing
with the control probe.  <BR>
<BR>

<P>
<B>2.</B>
 Apply Hybridization solution to slides:<BR>

<P>
<B>(a) </B>
Add probe to the hybridization solution.  The amount added should
be optimized for each probe preparation (see Probe Synthesis section,
step II.G)<BR>

<P>
<B>(b) </B>Pre-warm
the hybridization solution to 45&deg;C.  This makes it easier to spread.
<P>
<B>(c)</B>
 Elevate one side of the slide about 1 cm and apply the hybridization
solution in a pool to the elevated side of the slide.  Spread
evenly with a yellow tip. This step is made easier by using a
lot (200 &#181;l) of hybridization solution.  The amount required
depends upon the number of sections on the slide.<BR>

<P>
<B>(d)</B>
 Cover slide with a Sigmacote-treated coverslip.  Avoid bubbles--if
you get a lot of bubbles, start over.<BR>

<P>
<B>3.</B>
 Place slides in a humidified box and incubate overnight at 42&deg;C.
 Place several layers of wet paper towels at the bottom of the
tupperware box.  Elevate the slides using plastic test tube racks
cut to fit the box.  If the plastic racks are not available, use
a pair of 10 ml plastic pipets.
<P>
<B>IV. WASH<BR>
</B>
<P>
<B>A.  Solutions:<BR>
</B>
<P>
<B>1.</B>
 <U>20X SSPE</U> (see above)<BR>

<P>
<B>2.</B>
  <U>RNase Buffer</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=234>  10 mM Tris (7.5)
<P>
    1 mM EDTA
<P>
500 mM NaCl
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=234>250 ml for 6-10 slides 
</TD></TR>

<TR><TD WIDTH=112>Formula for 500 ml:
</TD><TD WIDTH=234>5 ml 1 M Tris (7.5) or 0.6 g Tris base
<P>
1 ml 0.5 M EDTA or 0.2 g EDTA 
<P>
14.6 g NaCl
<P>
Adjust pH to 7.5, increase vol to 500 ml
</TD></TR>

</TABLE>

<P>
<B>3.</B>
 <U>10 mg/ml RNase Stock Solution:<BR>
</U>
<P>
Add RNase Buffer directly to the
bottle to achieve a concentration of 10 mg/ml.  Store frozen at
-20&deg;C.  This is a 2000X stock.  This RNase stock will be good
for &gt;1 year.  We use Ribonuclease A from Sigma (catalog # R-5503).
<P>
<B>4.  </B><U>10X
PBS</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=247>1.3 M NaCl
<P>
30 mM NaH<sub>2</sub>PO<sub>4</sub>
<P>
70 mM Na<sub>2</sub>HPO<sub>4</sub>
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=247>50 ml 
</TD></TR>

<TR><TD WIDTH=112>Formula for 1 L:
</TD><TD WIDTH=247>76.0 g NaCl
<P>
  3.6 g NaH<sub>2</sub>PO<sub>4</sub>
<P>
  9.9 g Na<sub>2</sub>HPO<sub>4</sub>
<P>
Autoclave
</TD></TR>

</TABLE>

<P>
<B>B.  Other Materials:<BR>
</B>
<P>
<B> </B>Tupperware
boxes for washes
<P>
 Slide mailers (Baxter # M6271)
<BR>

<P>
<B>C.  Set up for wash:<BR>
</B>
<P>
<B>1.</B>
 Prewarm 100 ml of the RNase buffer to 37&deg;C.<BR>

<P>
<B>2.</B>
 Thaw the RNase stock solution.<BR>

<P>
<B>3.</B>
 Prewarm 3L of 0.2X SSPE to 50&deg;C (2L for use before RNase step,
1L after)<BR>

<P>
<B>D.  Remove coverslips and hybridization
solution:<BR>
</B>
<P>
<B>1.</B>
 Fill two 150 - 250 ml beakers with 2X SSPE.  Dip each slide up
and down in the first beaker until its coverslip comes off.  Dip
each slide a few times in the second beaker of 2X SSPE.<BR>

<P>
<B>2.</B>
 Place slides into a staining dish containing 2X SSPE.  After
all of the slides have been processed, proceed to the next step.
<BR>

<P>
<B>E.  First High Stringency Wash:
<BR>
</B>
<P>
<B>1.</B>
 Wash slides in 1 L of 0.2X SSPE at 55&deg;C for 1 hour with gentle
agitation (can do in tupperware).  <BR>

<P>
<B>2.</B>
Transfer slides to a slide mailer containing RNase buffer preequilibrated
to 37&deg;C.  Rinse slides 2 X in RNase buffer, 37&deg;C.  <BR>
<BR>

<P>
<B>F.  RNase Treatment:<BR>
</B>
<P>
This step significantly reduces
background.  However,  DIG-labeled RNA is apparently more sensitive
to RNase than 35S-labeled probes.  We did an experiment using
0, 5 &#181;g/ml, and 25 &#181;g/ml RNase and found that 0 or 5
&#181;g/ml RNase gave about equal levels of signal and background,
but that there was very little signal at 25 &#181;g/ml RNase.
 We suggest first using 5 &#181;g/ml RNase.  Increase the RNase
concentration if you have background problems; decrease the concentration
if you have problems with low signals.<BR>

<P>
<B>1.</B>
 Prewarm 70 ml of the RNase Buffer to 37&deg;C (35 ml per disposable
plastic tube).  <BR>

<P>
<B>2.</B>
 Thaw the 10 mg/ml RNase stock, dilute to 5 &#181;g/ml by adding
17.5 &#181;l to the 35 ml of prewarmed RNase Buffer, and mix by
swirling gently.  <BR>

<P>
<B>3.</B>
 Using a disposable plastic pipet, dispense 16 ml (16 ml for 5
slides--more solution if fewer slides or you can fill the empty
spots with blank slides) of 5 &#181;g/ml RNase to each of 2 slide
mailers.  <BR>

<P>
<B>4.</B>
 Place slide mailers in a 37&deg;C water bath and incubate for 30
 minutes.<BR>

<P>
<B>5.</B>
 After the 30 minute incubation, pour 5 &#181;g/ml RNase directly
into sink drain and rinse slides in slide mailers two times with
the remaining RNase buffer (pre-warmed to 37&deg;C also.) <BR>

<P>
<B>6.</B>
 Toss the slide mailers into the trash and treat all glassware
with DEPC.<BR>

<P>
<B>G.  Second High Stringency
Wash:<BR>
</B>
<P>
<B>1.  </B>Place
slides into slide rack and wash in 1 L of 0.2X SSPE at 55&deg;C for
1 hr with gentle agitation.<BR>

<P>
<B>2.  </B>Wash
slides for 5 minutes in 0.2X SSPE  at 37&deg;C in a large staining
dish.  Repeat. <BR>
<BR>

<P>
<B>H.  Final Rinse:<BR>
</B>
<P>
<B>1.</B>
Rinse in 1X PBS in staining dish.  <BR>

<P>
<B>2.</B>
 Proceed immediately to the immunological detection.  If you prefer
to do the immunological detection the following day, you can store
the slides in 1X PBS overnight.<BR>

<P>
<B>IV.  IMMUNOLOGICAL DETECTION</B>
<P>
<B> </B>

<P>
<B>A.  Materials:<BR>
</B>
<P>
 Boehringer Mannheim DIG Nucleic
Acid Detection Kit (#1175 041)
<P>
 <BR>

<P>
<B>B.  Buffers:<BR>
</B>
<P>
<B>1.</B>
 <U>Buffer 1</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=247>100 mM Maleic Acid (7.5)
<P>
150 mM NaCl
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=247>500 ml
</TD></TR>

<TR><TD WIDTH=112>Formula for 500 ml:
</TD><TD WIDTH=247>5.8 g Maleic Acid (MW = 116)
<P>
4.4 g NaCl
<P>
3.5 g NaOH (solid)
<P>
Adjust pH to 7.5, increase vol to 500 ml
<P>
Autoclave
</TD></TR>

</TABLE>

<P>
<B>2.</B>
 <U>10X Blocking Reagent (10X BR)</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=247>10% (w/v) Blocking Reagent in Buffer 1
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=247>100 ml
</TD></TR>

<TR><TD WIDTH=112>Formula for 100 ml:
</TD><TD WIDTH=247>10 g of blocking reagent (vial 6)
<P>
Buffer 1 to a final vol of 100 ml
<P>
Will need to heat to ~60&deg;C to get into solution
<P>
Autoclave, store in the refrigerator
</TD></TR>

</TABLE>

<P>
<B>3.</B>
 <U>1X Blocking Reagent (1XBR)</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=247>1% Blocking Reagent in Buffer 1
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=247>100 ml
</TD></TR>

<TR><TD WIDTH=112>Formula for 100 ml:
</TD><TD WIDTH=247>10 ml 10X Blocking Reagent
<P>
90 ml Buffer 1
</TD></TR>

</TABLE>

<P>
<B>4.</B>
 <U>1 M MgCl2<BR>
</U>
<P>
<B>5.</B>
 <U>BSA Wash solution</U>:
<P>
 
<P>
NOTE: The BSA you use is very
critical.  You must use BSA that is free of endogenous phosphatases
or you will have a lot of background.  We use Sigma #A-7030.<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=247>1% BSA 
<P>
0.3% Triton-X-100
<P>
100 mM Tris-Cl, pH 7.5
<P>
150 mM NaCl
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=247>~20 ml for 4 slides per step if using tip box lids (~400 ml for 16 slides)
</TD></TR>

<TR><TD WIDTH=112>Formula for 500 ml:
</TD><TD WIDTH=247> 5 g BSA
<P>
 1.5 ml Triton X-100
<P>
 50 ml 1 M Tris-Cl, pH 7.5
<P>
 15 ml 5 M NaCl
<P>
 Bring volume up to 500 ml.
</TD></TR>

</TABLE>

<P>
<B>6.</B>
 <U>TNM Buffers:<BR>
</U>
<P>
Use TNM-5 if using polyvinyl alcohol
in the color reaction
<P>
Use TMN-50 if <U>not</U> using
polyvinyl alcohol in the color reaction<BR>

<P>
<U>TNM-5 Buffer:<BR>
</U>
<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=247>100 mM Tris (9.0)
<P>
100 mM NaCl
<P>
    5 mM MgCl2
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=247>~ 100 ml
</TD></TR>

<TR><TD WIDTH=112>Formula for 500 ml:
</TD><TD WIDTH=247>50 ml 1 M Tris or 6.1 g Tris base 
<P>
10 ml 5 M NaCl or 2.9 g NaCl 
<P>
2.5 ml 1 M MgCl2
<P>
Adjust pH to 9.0; increase vol to 500 ml
</TD></TR>

</TABLE>

<P>
<U>TNM-50 Buffer:<BR>
</U>
<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=247>100 mM Tris (9.5)
<P>
100 mM NaCl
<P>
  50 mM MgCl2
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=247>~ 100 ml
</TD></TR>

<TR><TD WIDTH=112>Formula for 500 ml:
</TD><TD WIDTH=247>50 ml 1 M Tris or 6.1 g Tris base 
<P>
10 ml 5 M NaCl or 2.9 g NaCl 
<P>
25 ml 1 M MgCl2
<P>
Adjust pH to 9.5; increase vol to 500 ml
</TD></TR>

</TABLE>

<P>
<B>7.</B>
 <U>TNP Buffer</U>:<BR>

<P>
This buffer is needed only if
you are using polyvinyl alcohol in the color reaction<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=351>100 mM Tris (9.0)
<P>
100 mM NaCl
<P>
10% Polyvinyl alcohol M<sub>r</sub> of 70-100kDa (Sigma # P-1763) 
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=351>~20 ml per set of 5 slides
</TD></TR>

<TR><TD WIDTH=112>Formula for 100 ml:
</TD><TD WIDTH=351>10 ml 1 M Tris 
<P>
  2 ml 5 M NaCl
<P>
10 g PVA<BR>

<P>
Add Tris and NaCl; adjust pH to 9.0
<P>
Dissolve PVA at 90&#176;C; increase vol to 100 ml
<P>
Cool to room temperature before making PVA Color-Substrate Solution.
</TD></TR>

</TABLE>

<P>
<B>8.</B>
 <U>T<sub>10</sub>E<sub>1</sub> (8.0)</U>:<BR>

<TABLE>

<TR><TD WIDTH=112>Composition:
</TD><TD WIDTH=247>10 mM Tris (8.0)
<P>
  1 mM EDTA
</TD></TR>

<TR><TD WIDTH=112>Amount Required:
</TD><TD WIDTH=247>500 ml
</TD></TR>

<TR><TD WIDTH=112>Formula for 500 ml:
</TD><TD WIDTH=247>5 ml of 1 M Tris
<P>
1 ml of 0.5 M EDTA
<P>
Adjust pH to 8.0, increase vol to 500 ml
<P>
Autoclave
</TD></TR>

</TABLE>

<P>
<B>B.  Blocking:<BR>
</B>
<P>
1.  Place slides in a tupperware
box of the correct size (or top from a yellow pipette tip box)
and cover with 1X Blocking Reagent.  Incubate at room temperature
with gentle agitation for 45 min.  <BR>

<P>
2.  Pour out blocking solution
and replace with BSA Wash Solution.  Incubate at room temperature
with gentle agitation for 45 min.<BR>

<P>
<B>C.  Antibody reaction:<BR>
</B>
<P>
The optimal antibody dilution
will probably be different with different probes.  Try a 1:600
dilution first.  If your signal is too weak, try a lower dilution.
 If your signal is very strong, try a higher dilution to save
antibody.<BR>

<P>
<B>1.</B>
 Dilute the anti-DIG-AP conjugate (vial 3) in BSA Wash Solution.
 If using slide mailers, make up 15-16 ml for each set of 5 slides.
   <BR>

<P>
<B>2.</B>
 Place slides in slide mailer and add antibody solution.  Incubate
at room temperature for 2 hours with slow rotation.  <BR>

<P>
<B>D.  Antibody wash:<BR>
</B>
<P>
<B>1.  </B>Remove
slides from the antibody solution and place on the bottom of a
Tupperware box.  Add enough BSA Wash Solution to cover the slides.
 Work fast so that the antibody does not dry up on the slides.
 <BR>

<P>
<B>2.</B>
 Incubate for 30 minutes at room temperature with gentle agitation.
<BR>

<P>
<B>3.</B>
 Repeat step 1 twice for a total of 3 washes in BSA Wash Solution.
<BR>
<BR>
<BR>

<P>
<B>E.1  Color reaction <U><B>without</B></U><B>
polyvinyl alcohol:<BR>
</B></B>
<P>
The color reaction can be done
in one of two ways--either with or without polyvinyl alcohol.
 If you wish to do the color reaction with polyvinyl alcohol,
go to section E.2.<BR>

<P>
<B>1.</B>
 Wash slides in TNM-50 Buffer for 5 minutes at room temperature
with gentle agitation.  <BR>

<P>
<B>2.</B>
 Repeat step 1.<BR>

<P>
<B>3.</B>
 Make up Color-Substrate Solution just before use.  Wrap in foil
to keep dark until used.  If you do the color reaction in slide
mailers, you will need 15-16 ml for each set of 5 slides.  For
32 ml, mix the following:<BR>

<P>
   32 ml TNM-50 Buffer
<P>
112 &#181;l X-phosphate-solution
(vial 5)
<P>
144 &#181;l NBT (vial 4)<BR>

<P>
<B>4.</B>
 Place slides in slide mailers and add the Color-Substrate Solution.
 Incubate in the dark at room temperature without shaking.  <BR>
<BR>

<P>
<B>E.2  Color reaction with polyvinyl
alcohol<BR>
</B>
<P>
It has been reported that PVA
decreases the incubation time and increases the cellular localization
of the color reaction by slowing the diffusion of reaction intermediates
(see De Block and Debrouwer, 1993).  In our experience, PVA does
not significantly  decrease the incubation time.  However, we
have observed that PVA does improve cellular resolution.  PVA
can be a hassle to deal with (thick and gummy); thus, for your
first experiment, it might be best to eliminate the PVA.<BR>

<P>
<B>1.</B>
 Wash slides in TNM-5 Buffer for 5 minutes at room temperature
with gentle agitation.  <BR>

<P>
<B>2.</B>
 Repeat step 1.<BR>

<P>
<B>3.</B>
 Make up PVA Color-Substrate Solution just before use.  Wrap in
foil to keep dark until used.  If you do the color reaction in
slide mailers, you will need 15-16 ml for each set of 5 slides.
 For 32 ml, mix the following:<BR>

<P>
  32 ml TNP Buffer
<P>
160 &#181;l 1 M MgCl2  
<P>
112 &#181;l X-phosphate-solution
(vial 5)
<P>
144 &#181;l NBT (vial 4)<BR>

<P>
<B>4.</B>
 Place slides in slide mailers and add PVA Color-Substrate Solution.
 Incubate in the dark at room temperature without shaking.  <BR>
<BR>
<BR>

<P>
<B>F.  Monitor the color reaction:
<BR>
</B>
<P>
<B>1.</B>
 For a typical probe, monitor the reaction progress beginning
at about 6 hours.  If you expect the signal to be very strong,
begin monitoring at 1 hour.<BR>

<P>
<B>2.</B>
 To monitor the reaction progress, choose a test slide, rinse
with T<sub>10</sub>E<sub>1</sub>, wipe off the back of the slide, and observe using
an old microscope.  Never observe unmounted slides using a good
microscope!!!  Put the test slide back into the color solution
if a longer incubation is required.<BR>

<P>
<B>3.</B>
 When the color reaction is complete, wash the slides for 5 minutes
with T<sub>10</sub>E<sub>1</sub>.  Repeat twice.<BR>
<BR>

<P>
<U><B>V.  SLIDE MOUNTING<BR>
</B></U>
<P>
<B>A.  Materials/Solutions:<BR>
</B>
<P>
<B>1.</B>
 Ethanol series in coplin jars: 25%, 50%, 75%, 100%, 100%<BR>

<P>
<B>2.</B>
 Xylenes in a coplin jar<BR>

<P>
<B>3.</B>
 Coverslips<BR>

<P>
<B>4.</B>
 Cytoseal or Permount<BR>
<BR>

<P>
<B>B.  Procedure<BR>
</B>
<P>
<B>1.</B>
 Do this procedure in the hood.  Process 1 slide at a time because
the color fades in the xylene.  <BR>

<P>
<B>2.</B>
 Dehydrate through the ethanol series.  Dip slides in the ethanols
until streaks go away (3-4 times).  <BR>

<P>
<B>3.</B>
 Dip in the xylenes until the streaks go away.  Let sit for another
10-20 seconds.<BR>

<P>
<B>4.</B>
 Remove the slide from the xylenes and place on a clean paper
towel.  Quickly (before the xylenes evaporate) add 1-2 drops of
Cytoseal to the slide and quickly (before the xylenes evaporate)
place a coverslip on the slide.  <BR>

<P>
<B>5.</B>
 Squeeze out air bubbles and excess Cytoseal by pressing down
gently on the coverslip with a kimwipe.  This will usually cause
Cytoseal streaks to form on top of the coverslip.  The Cytoseal
streaks can be removed by rubbing gently with a xylenes-moistened
kimwipe.  <BR>

<P>
<B>6.</B>
 If you wish to observe the slides at this point, use the old
microscope.  Never use the good microscope until step 7 has been
completed.<BR>

<P>
<B>7.</B>
 Place slides in a 42&#176;C incubator overnight to harden the
Cytoseal.  Before observing the slides, dip in xylenes a few times
to remove the excess Cytoseal, and then air dry (in the hood)
for greater than or equal to 5 minutes.  It is now OK to use the good microscope.
<BR>
<BR>

<P>
<U><B>References<BR>
</B></U>
<P>
David Oppenheimer's protocol.
 We essentially follow the protocol developed by David.<BR>

<P>
Cox et al. (1984) Detection of
mRNAs in sea urchin embryos by <I>in</I> <I>situ</I> hybridization
using asymmetric RNA probes. Dev. Biol. 101, 485-502.<BR>

<P>
Cox and Goldberg (1988) Analysis
of plant gene expression. In Plant Molecular Biology, A Practical
Approach. C.H. Shaw, ed. (Oxford: IRL Press), pp 1-34.<BR>

<P>
De Block, M., and Debrouwer, D.
(1993)  RNA-RNA <I>in situ</I> hybridization using digoxigenin-labeled
probes:  The use of high-molecular-weight polyvinyl alcohol in
the alkaline phosphatase indoxyl-nitroblue tetrazolium reaction.
Anal. Biochem. 215, 86-89.<BR>

<P>
Meyerowitz (1987) <I>In situ</I>
hybridization to RNA in plant tissue. Plant Mol. Biol. Rep. 5,
242-250 <BR>

<P>
Moench et al. (1985) Efficiency
of <I>in situ</I> hybridization as a function of probe size and
fixation technique. J Virol. Methods 11, 119-130.<BR>

<P>
Sass, J. E. <U>Botanical Microtechnique</U>.
Iowa State University Press. 1958.<BR>
<P>
<center>Last modified on January 18, 2002</center>

		</td>
	</tr>
</TABLE>

<!-- FOOTER -->
<script language="javascript" src="/js/footer.js"></script>
</body>
</html>
