<html>
<head>
<title>TAIR</title>
<script language="JavaScript" src="/js/navbar.js"></script>
<link rel="stylesheet" type="text/css" href="/css/main.css" >
<script language='JavaScript'>
var highlight = 0; var helpfile="";
</script>
</head>

<body leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" bgcolor="#F5F9FF">


<!-- HEADER using external JavaScript file -->

<script language='JavaScript' SRC='/js/header'>
</script><p>

<!-- End of header -->


<p>
<table width="602" align="center" border="0" cellspacing="0" cellpadding="2">
	<tr>
		<td width="602">
		
<P>
<CENTER><B>Lab 7: Gene Expression and Protein Localization in Arabidopsis</B>
<BR>
Cold Spring Harbor Arabidopsis Molecular Genetics Course
<P>
July, 1996

<P>
Dr. Albrecht von Arnim<br>
Dept. of Botany<br>
University of Tennessee<br>
Hesler Biology Building 437<br>
Knoxville, TN 37996-110<br>
phone: (423) 974-6206

</CENTER>
<P>
<B>OVERVIEW</B>

<P>
To
a large degree, the function of a protein can be inferred from
its cellular compartmentalization and its interactions with other
cellular components.  Therefore, information on the <B>subcellular</B>
distribution of a protein is crucial to determine its overall
role in the cell.  The histological distribution of the mRNA transcript
reflects another important aspect; this aspect, however, has been
covered in the <I>in situ</I> hybridization laboratory and will
not be discussed here.  
<P>
 Several options are available
to probe the subcellular localization of a given protein.  In
a few cases, it has been possible to design a specific way to
localize a protein directly while inside the cell, based on a
unique functional property or physical characteristic of the protein
(light absorption or fluorescence).  In the majority of cases,
however, some form of <B>indirect detection</B> will be necessary.
 Methods that should be applicable to almost any protein will
be the focus in this lab.  <BR>

<P>
There are two major strategies
for indirect detection:<BR>

<P>
<B>A. Detection of the native
protein: </B>A specific
antibody or an equivalent high affinity probe is used to label
the target protein <I>in situ</I>.  The probe is either directly
labelled with a fluorescent tag or detected with secondary antibodies.
 Final observation is by either electron microscopy or light microscopy.
 While electron miscroscopy has higher resolution, it is also
time-consuming and not very sensitive.  In many cases, the light
microscope can offer sufficient spatial resolution (Colasanti
et al., 1993; Neuhaus et al., 1993; Wick, 1993; Matsui et al.,
1995).<BR>

<P>
<B>B. Detection of a recombinant
version: </B>Alternatively,
a tag can be incorporated into the target protein and the modified
protein introduced back into the cell by recombinant DNA technology.
 Thus, the fusion protein can be localized by monitoring the tag.
 There are three types of commonly used tags.  
<P>
 First, <B>epitope tags </B>(8-11
amino acid stretches of well-characterized foreign proteins such
as c-myc or hemagglutinin).  These tags are small, thus minimizing
the chance of altering the gross conformation (or subcellular
localization) of the targeted protein.  Monitoring the tag can
be carried out according to standard immunological procedures
with commercially available monoclonal antibodies specific for
the chosen tag.  The obvious advantage here is elimination of
the need to generate specific antibodies for each new target protein.
  
<P>
 Second, &szlig;-glucuronidase
(GUS) acts as a reliable <B>reporter enzyme</B> in fusion proteins,
especially for nuclear versus non-nuclear localization (Restrepo
et al., 1990; Varagona et al., 1991 and 1992; Abel and Theologis,
1994; Abel et al., 1994; Citovsky et al., 1994; von Arnim and
Deng, 1994).  Although the <I>in situ</I> staining for GUS activity
is easier than immunodetection, there are some potential drawbacks
with this method.  For example, the <I>in situ</I> enzyme assay
can lead to a loss of spatial resolution.  
<P>
 Third, <B>fluorescent proteins</B>
(<I>e.g.</I> GFPs) combine the advantage of high resolution with
detection in vivo (Chalfie et al., 1994, Heim et al., 1994; Prasher,
1995).  This approach has been  succesful in yeast and mammalian
cells (e.g. Olson et al., 1995) and in Drosophila (Wang and Hazelrigg,
1994; Kerrebrock et al., 1995), and has also been developed for
plants.
<P>
 While a tag can obviously improve
the specificity of detection and avoid the need for generating
antibodies for every new protein to study, it can potentially
alter the subcellular localization of the fused protein due to
structural constraints introduced by the tag and by an aberrant
expression level.   However, genetic or biochemical experiments
may clarify whether the fusion protein remains functional.  Ideally,
conclusions using one method should be corroborated by an independent
approach.<BR>

<P>
<B>AIM<BR>
</B>
<P>
 During the two days allocated
to this section, we will provide a theoretical overview and an
experimental entryway into methods for transient gene expression
and subcellular localization in <I>Arabidopsis</I>.  The main
course of events will be dominated by Experiment 1, immunofluorescent
detection of proteins in protoplasts.  In parallel, we will demonstrate
the subcellular detection of proteins using GUS as a fusion marker
(Experiment 2).  Experiment 3 will demonstrate the power of GFP
as a subcellular marker and the transient expression of proteins
using the particle gun.  <BR>

<P>
Experiment 1: Immunofluorescence
detection in Arabidopsis protoplasts  
<P>
Experiment 2: Subcellular localization
of GUS-fusion proteins in 
<P>
   <I>Arabidopsis</I> seedlings
<P>
Experiment 3: Transient gene expression
of GUS and GFP by particle 
<P>
   bombardment<BR>
<BR>

<P>
<B>SCHEDULE<BR>
</B>
<P>
<B>Experiment 1       meanwhile...
<BR>
</B>
<P>
<B>Day 1</B>
<P>
1. Digest tissue 1)   09.00--11.00

<P>
2. Wash protoplasts  11.00 --12.00
<P>
3. Settle cells onto slides  12.00
--15.00 lunch, instruction         set up GUS-staining 
<P>
        (Expt.2), 
<P>
        particle gun (Expt.3)
<P>
4. Fixation etc. of cells   15.00
--17.00 (leave over night)
<P>
5. Analyse GUS-stained 
<P>
 seedlings (Expt.2)  17.00--18.30
<BR>

<P>
<B>Day 2</B>
<P>
5. Rehydration and block 1) 09.00--11.30
 (TA)
<P>
6. Add first antibody  11.30 --13.00
  lunch 
<P>
7. Wash    13.00 --13.30
<P>
8. Add second antibody  13.30
--14.30  inspect cells from 
<P>
         transient assay
<P>
         (particle gun) 2)
<P>
9. Wash    14.30 --15.00
<P>
10. Mount slides   15.00 --15.30
<P>
11. Analyse results  15.30 --18.30
<BR>

<P>
1) This step will be set up by
the TAs
<P>
2) Tissue transformed with the
particle gun (Day 1) has been mounted for microscopy by TAs. 

<P><h3>Experiment #1:  Immunofluorescence detection in Arabidopsis
protoplasts</h3>

<P>
<B>INTRODUCTION</B>
<P>
 We will concentrate on the direct
detection of a protein using specific antibodies.  The major problem
is to make the antigen accessible to the antibody, which is effectively
blocked by cell walls and the cuticle.  The effect of cell walls
can be overcome by: <BR>

<P>
- Digestion of cell walls in a
whole mount specimen
<P>
- Digestion of cell walls under
isolation of protoplasts
<P>
- Sectioning<BR>

<P>
Several key factors need to be
considered when choosing between these options:  
<P>
 First, <B>autofluorescence</B>
in plant specimens can be a serious problem.  Sometimes it is
possible to work with tissues that show little autofluorescence,
for example roots, which lend themselves to the whole-mount approach,
but this is often impractical.  More commonly, it is necessary
to take additional measures to reduce autofluorescence, for example
by washing out the interfering substances before microscopy. 
This is more easily achieved in single cells, such as protoplasts,
than in whole mounts.  In addition, careful selection of fluorophores
and optical filters is necessary to reduce the interference of
autofluorescence.  
<P>
 Second, the specimen has to be
<B>fixed</B> to preserve the subcellular structures during the
antibody incubation and wash steps.  The trade-off:  Gentle fixation
will preserve antigenicity and keep autofluorescence low, but
may not preserve the subcellular structures sufficiently.  Harsh
fixation conditions (especially glutaraldehyde) will increase
autofluorescence.  
<P>
 Third, <B>antigenicity</B> of
the target should be preserved.  Strong fixatives will reduce
antigenicity and may also unmask fortuitous epitopes.  Embedding
of tissue, associated with extreme changes between aqueous and
non-aqueous media and exposure to a wide range of temperatures,
also runs counter the preservation of antigenic sites.  
<P>
 Overall, while detection of a
highly abundant antigen may be straightforward even after rough
treatment of the tissue, only the very optimum of conditions may
allow detection of a protein of low abundance.  <BR>

<P>
 The following strategy has allowed
quite consistent detection of a wide variety of antigens in our
hands.  <BR>

<P>
1.  Isolation of protoplasts from
the tissue of choice
<P>
2.  Binding of protoplasts to
gelatin-coated slides
<P>
3.  Fixation and permeabilization
of the cells
<P>
4.  Antibody incubations and washes
<P>
5.  Mounting and inspection<BR>

<P>
This method has the following
<B>advantages</B>: 
<P>
(1) Access of the antibodies is
very rapid. 
<P>
(2) Fixation is quick and uniform.

<P>
(3) Autofluorescence is kept at
a low level. 
<P>
(4) The whole experiment can be
carried out in one (long) day.  <BR>

<P>
<B>Disadvantages</B>
to be kept in mind are: 
<P>
(1) Protoplasting may alter certain
cellular structures. 
<P>
(2) Analysis is confined to cell
types amenable to protoplasting (mesophyll cells, etc.).<BR>
<BR>

<P>
<B>PROCEDURES<BR>
</B>
<P>
<B>Part A: Isolation of <I><B>Arabidopsis</B></I><B>
seedling protoplasts<BR>
</B></B>
<P>
1. Harvest one plate of seedlings,
grown on solidified agar (GM medium) for 7 to 10 days.  
<P>
-&gt; Ecotypes Columbia and No-O
work well.  Using a small pair of curved scissors, cut the seedlings
just above the agar surface to separate the roots from the rest
of the seedling.  Transfer the green organs to 10 ml enzyme solution
in a 125 ml Erlenmeyer flask or to 5 ml enzyme solution in a 6
cm diameter glass petri dish.  <BR>

<P>
2. Apply vacuum for 1 minute in
a desiccator/bell jar.  <BR>

<P>
3. Shake slowly (40-70 rpm) for
90 to 120 minutes.  <BR>

<P>
4. Shake for 10 minutes at 80
to 100 rpm to release the protoplasts.<BR>

<P>
5. Filter the cell suspension
through a 75 micrometer nylon mesh into a 30 ml Corex glass tube.
 
<P>
-&gt; Hold the folded mesh with
a small plastic funnel or with one hand.  Handle the cells gently
with a pasteur pipette.  Wash the flask or the dish with 5 ml
of wash solution and pool this fraction with the first.  <BR>

<P>
6. Spin for 2 minutes at room
temperature at 150 g  (1000 rpm) in a HB-6 swingout rotor with
rubber adapters (up to 500 g is acceptable). <BR>

<P>
7. Pipette off and discard the
10 ml supernatant, resuspend protoplasts gently in 10 ml wash
solution and incubate for 5 minutes.<BR>

<P>
8. Spin as above in step 6, remove
supernatant, and resuspend the protoplasts in 0.5 ml wash solution.
 If necessary, the cells can be stored on ice.  <BR>

<P>
Note:<B>  </B>We thank Dr. Jen
Sheen, Massachusetts General Hospital, Boston, MA, for communicating
the basis of this protocol. <BR>
<BR>

<P>
<B>SOLUTIONS AND REAGENTS-1A<BR>
</B>
<P>
<B>GM medium </B>(1
liter)<BR>

<P>
sucrose 10 g
<P>
MS salts (Sigma M-5519) 4.3 g
 
<P>
MES buffer 0.5 g  
<P>
1000x vitamins 1 ml  <BR>

<P>
Adjust pH to 5.7 with KOH, add
8 g Bacto-Agar, autoclave in a 1 l bottle, cool to about 50&deg;C,
and pour about 30 ml each into deep petri dishes (Nunc 4026).
 <BR>

<P>
1000x vitamins:
<P>
100 mg/ml myo-inositol
<P>
1 mg/ml thiamine
<P>
0.5 mg/ml nicotinic acid
<P>
0.5 mg/ml pyridoxine
<P>
<B>Enzyme solution<BR>
</B>
<P>
cellulase R10 1%
<P>
macerozyme R10 0.25%
<P>
mannitol 0.4 M
<P>
MES  10 mM  pH 5.7<BR>

<P>
Heat to 55&deg;C for 10 minutes to
inactivate proteases and to accelerate enzyme solubilization,
then cool to room temperature. <BR>

<P>
Add:
<P>
CaCl<sub>2</sub> 30 mM
<P>
&#223;-mercaptoethanol 5 mM
<P>
BSA 0.1%<BR>

<P>
Filter through 0.45 micron filter.
 The solution can be stored for several months at -20&deg;C without
apparent loss of activity. 
<P>
Cellulase 'Onozuka' R-10 and Macerozyme
R-10  are supplied by Yakult Honsha Inc. Co, Ltd., 1-1-19 Higashi
Shinbashi Minato-ku; Tokyo; 105 Japan.<BR>
<BR>

<P>
<B>Wash solution<BR>
</B>
<P>
mannitol 0.5 M
<P>
MES, pH 5.7 4 mM
<P>
KCl 2 mM<BR>

<P>
optional:
<P>
5 mM CaCl<sub>2</sub>, 2 mM malate, 2 mM
ammonium sulfate, 1 mM potassium phosphate.<BR>

<P>
<B>Part B: Immunofluorescence
labeling of Arabidopsis protoplasts<BR>
</B>
<P>
1. <B>Coat slides</B>: Dip 8-well
slides (Carlson Scientific, #100806) into a narrow beaker with
coating solution, air dry for 30 minutes by leaning them against
a support;  rinse with water, then air dry and store at 4&deg;C in
a microscope slide holder, wrapped in cling film.
<P>
-&gt; Coated slides will be provided.
 <BR>

<P>
2. Place slides onto a pair of
pasteur pipettes over wet paper towels in a dish with a tight
lid (<B>humid chamber</B>).  <BR>

<P>
3. With a pasteur pipette, apply
a drop of protoplasts (ca. 50 &#181;l in wash solution) to each of
the 8 wells on the slide, cover the container and let the cells
<B>settle</B> and adhere to the slide for 3 hours at room temperature.
 
<P>
-&gt; Make sure that the cells
will stay moist throughout, if not stated otherwise.  
<P>
-&gt; Label slides with a pencil.
 
<P>
-&gt; It is convenient to apply
only one type of cells per slide and apply different antisera
onto the different wells, rather than the other way round.   
<BR>

<P>
4. Gently remove most of the excess
solution with a yellow tip from the wells of one slide.  With
a blue tip, immediately add 50 &#181;l <B>fixation</B> <B>solution</B>
to each well (need 400 &#181;l per slide).  Incubate for 5 to 10 minutes
at room temperature.  Repeat for the other slides.  
<P>
-&gt; Lots of protoplasts don't
stick to the slide and are lost at this step.  
<P>
-&gt; Work <I>quickly</I> to prevent
cells from drying out while they are not protected by solution.
 Add and remove droplets of solution from the edge of the well,
avoiding to touch the well itself.  
<P>
-&gt; Incubation times should
be reproducible, but can be optimized for each antigen.  <BR>

<P>
5. Permeabilize/destain the cells.
 Gently remove fixative solution with a yellow tip/Pipetman, again
working from the edge of the well.  Immediately add 25 &#181;l
<B>permeabilization solution</B> to each well and incubate for
5 to 10 minutes. 
<P>
-&gt; Keep the slide horizontal
to make sure that this solution doesn't run off the slide.<BR>

<P>
6. Remove the permeabilization
solution from one slide by aspiration or with a yellow tip/Pipetman.
 Immerse the slide into a glass slide holder/Coplin jar with <B>acetone/
methanol</B> (1:1 v/v, preferably kept at -20<FONT SIZE=1 FACE="New Century Schlbk">o</FONT>C).
 Repeat for the other slides.  Incubate slides for 10 minutes
in the freezer (or on the bench).  Using forceps, transfer the
slides to a second Coplin jar with the same solution and incubate
for another 5 minutes. 
<P>
-&gt; The slide holder is designed
to hold up to 10 slides in back-to-back pairs.  <BR>

<P>
7. Pull out the slides with forceps
and <B>air dry</B> for at least 20 minutes.  
<P>
-&gt; At this stage all green
pigment should be gone.
<P>
-&gt; The slides can now be stored
in a slide holder, wrapped in clingfilm, at 4<FONT SIZE=1 FACE="New Century Schlbk">o</FONT>C
over night.  <BR>

<P>
8. <B>Wash</B> the slides once
for 30 minutes with PBS (using the slide holder), then place them
into the humid chamber.
<P>
-&gt; Keep the samle moist in
the humid chamber from now on.
<P>
-&gt; It is most convenient and
safe to remove solutions from the wells with a yellow tip at the
end of a piece of tubing attached to an aspirator.<BR>

<P>
9. Apply 50 &#181;l <B>blocking
solution</B> per well (PHEM, 2% BSA); incubate for at least 30
minutes.<BR>

<P>
10. While the slides are being
blocked, prepare the <B>antibody solutions</B>.  About 10 &#181;l
per well is needed.  Make a dilution in blocking solution in a
0.6 ml eppendorf tube.  Spin the antiserum for 10 minutes at 13,000
rpm to remove any insoluble aggregates of antibody and and transfer
the supernatant to a new tube.
<P>
-&gt; Leave the two left wells
with blocking solution lacking antibody.  These will serve as
controls for autofluorescence and for the secondary antibody alone.
<P>
-&gt; To reduce background fluorescence
resulting from antibody aggregates, it helps to store (a 1 in
5 dilution of) the antibody stock in smalll aliquots at -80<FONT SIZE=1 FACE="New Century Schlbk">o</FONT>C,
after filtering it through a 0.2 &#181;m filter.
<P>
-&gt; The concentration of antibody
has to be determined empirically. A concentration of 5 &#181;g
per ml or a dilution of 1:100 is a point to begin with. Try 1:30
and 1:300 as well.  <BR>

<P>
11. remove blocking solution by
aspiration and apply 8&#181;l of <B>antibody</B> per well.  Incubate
for 1 hour at room temperature or over night at 4<FONT SIZE=1 FACE="New Century Schlbk">o</FONT>C.
<P>
-&gt; Make sure that the antibody
cannot leak from one well to the next.
<P>
-&gt; Overnight incubation tends
to increase unspecific binding.<BR>

<P>
12. <B>Wash</B>.  Remove first
antibody solution, taking care that the sample doesn't dry out
completely.  Add 50&#181;l PBS to each well and remove, then was
with PBS containing 0.1% NP-40, then again with PBS (5 minutes
each).
<P>
-&gt; The second and third wash
are done by immersion of the slide into the slide holder.  <BR>

<P>
13. While the slides are being
incubated with the first antibody and washed, prepare the <B>secondary
antibody</B>.  Again, make 10 ml per well of a (suggested) 1:200
or 1:100 dilution in PBS, 0.5% BSA.  Spin for 10 minutes at 13,000
rpm and take only the supernatant to remove any insoluble aggregates.
 Apply 8 &#181;l antibody to each well for 1 hour, making sure that
it doesn't leak from one well to the next.  
<P>
-&gt; Incubate one of the two
wells lacking first antibody with secondary antibody, and leave
the other one with blocking solution alone (autofluorescence control).
 <BR>

<P>
14. <B>Wash</B> slides once with
PBS, twice with PBS, 0.1% NP-40, and then once with PBS again,
5 minutes each.  
<P>
-&gt; The first wash is done on
individual wells, the other washes in the slide holder.  <BR>

<P>
15. Remove PBS by aspiration and
quickly add about 2 to 4 &#181;l <B>mount solution</B> to each well.
 
<P>
-&gt; Spread out the mount solution
with a yellow tip and add a coverslip.  Add as little mount solution
as possible, but try to avoid trapping air bubbles.  To do this,
initially add 2 &#181;l; if this is not enough to cover the well, add
another 2 &#181;l along the edge of the well.  Seal the coverslip with
nail polish and store slides in the dark.    <BR>

<P>
16. Inspect slides at 1000x magnification
(100x objective/lens) with immersion oil on a microscope with
epifluorescence optics.  
<P>
-&gt; Beforehand, clean any gelatin
from the back of the slide with water and a Kimwipe.  Try to limit
exposure of the samples to the microscope light, because this
tends to produce a hazy background, rather than the more desirable
black background.  It is convenient to look for nicely fixed cells
under the UV/blue channel (DAPI), then switch to the blue/green
(fluorescein) or green/red channels (Texas Red, rhodamine).  
If desired, photograph with 400 ASA color slide film.  
<P>
-&gt; fluorescein bleaches out
very easily, even in the anti-fade solution.  Expose only briefly
to epifluorescence light source.  
<P>
<B>SOLUTION AND REAGENTS-1B<BR>
</B>
<P>
<B>Coating solution<BR>
</B>
<P>
Polylysine (reusable): 50 mg per
liter in 10 mM Tris, pH 8.0; store at 4&deg;C.
<P>
Gelatin: Dissolve 1 g gelatin,
0.1 g chromic potassium sulfate (Chrome alum, Aldrich) in 100
ml warm water.  Make this solution fresh each time.  <BR>

<P>
<B>PHEM </B>(100
ml)
<P>
PIPES (MW 302.4) 60 mM 1.81 g
<P>
HEPES (MW 238.3) 25 mM 0.60 g
<P>
EGTA (MW 380.4) 10 mM 0.38 g
<P>
Mg2Cl (MW 203.3) 2 mM 40.6 mg
<P>
adjust pH to 6.9 with NaOH.<BR>

<P>
<B>10x PBS </B>(1
liter)
<P>
Na2HPO4 74 mM 10.6 g
<P>
NaH2PO4.H2O 14 mM 2.0 g
<P>
NaCl 1.5 M 90 g<BR>
<BR>

<P>
<B>Fixation solution: </B>2%
paraformaldehyde in PHEM.  
<P>
-&gt; To dissolve paraformaldehyde,
heat gently to not more than 55&deg;C and vortex occasionally.  The
choice of fixative and the design of the fixation step are probably
the most critical for the success or failure of the experiment.
 <BR>

<P>
<B>Permeabilization solution:
</B>0.5% NP-40 in PHEM
<BR>

<P>
<B>Acetone/methanol 1:1 (v/v)
<BR>
</B>
<P>
<B>Blocking solution:</B>
2% BSA in PHEM<BR>

<P>
<B>Mount solution: </B>Antifade'
(Molecular Probes, S-2828), freshly mixed to contain 1 mg/l DAPI
(4',6-diamidino-2-phenylindole).
<P>
<B>Experiment #2: <B>Subcellular
localization of GUS-fusion proteins in <I>Arabidopsis</I> seedlings
<BR>
</B></B>
<P>
<B>INTRODUCTION<BR>
</B>
<P>
 GUS-fusion proteins have been
employed extensively to ask whether a test protein can drive nuclear
localization of the GUS enzyme marker (Restrepo et al., 1990).
 The fusion protein genes can be expressed by particle bombardment
(Varagona et al., 1992), transformation of protoplasts (Abel et
al., 1994) or from chromosomal transgenes (Varagona et al., 1991).
<P>
 In this experiment, transgenic
<I>Arabidopsis</I> seedlings carrying genes for GUS-COP1, GUS-NIa,
or GUS will be provided. <BR>

<P>
<B>PROCEDURES<BR>
</B>
<P>
<B>Localization of <I><B>Arabidopsis</B></I><B>
proteins with GUS <I>in situ</I> enzyme assay  <BR>
</B></B>
<P>
1. For each sample, add 300 &#181;l
of stain solution to a well of a 24-well tissue culture dish.
<BR>

<P>
2. Using forceps, harvest seedlings
grown on GM for 6 days and immediately place them into the stain
solution.
<P>
-&gt; Note: in general, a gentle
prefixation of the tissue (2% formaldehyde in wash solution for
2 minutes), followed by washing, may preserve the subcellular
localization better, but at the expense of the sensitivity of
the detection.  We did not notice significant differences in the
relative GUS staining patterns with or without the fixation.<BR>

<P>
3. Incubate seedlings in stain
solution at room temperature until the desired staining intensity
is achieved.  
<P>
-&gt; Monitor progress under the
stereomicroscope and/or: 
<P>
-&gt; Inspect under the light
microscope in the presence of 1 &#181;g/ml DAPI.<BR>

<P>
4. If desired, replace stain with
wash solution. <BR>

<P>
5. Replace wash solution with
fixation solution, incubate for 30'.  
<P>
-&gt; Wear gloves when handling
fixation solution.  <BR>

<P>
6. Replace fixation solution with
30% ethanol, leave on a rocking table for 10 minutes, then change
to 50% and 70% ethanol.  Incubate in 70% ethanol overnight, gradually
transfer back to wash solution and mount the seedlings in wash
solution with 1 &#181;g/ml DAPI on regular microscope slides.  Seal
the slides and store at 4&deg;C. <BR>

<P>
7. Observe under light microscope.
<BR>

<P>
<B>SOLUTIONS AND REAGENTS</B>
<BR>

<P>
<B>Stain solution:<BR>
</B>
<P>
sodium phosphate, pH 7.0 100 mM

<P>
EDTA 1 mM 
<P>
potassium ferrocyanide 5 mM 
<P>
potassium ferricyanide 5 mM 
<P>
Triton-X-100 1 %
<P>
X-Gluc 1 mg/ml
<P>
-&gt; dissolve X-Gluc in a small
volume of dimethylformamide by warming it for a few seconds at
50&deg;C.<BR>

<P>
<B>Wash solution:  </B>100
mM sodium phosphate, 1 mM EDTA, pH 7<BR>

<P>
<B>Fixation solution:  </B>3.7%
formaldehyde in wash solution<B>Experiment #3: Transient expression
by particle bombardment<BR>
</B>
<P>
 Particle bombardment has been
used to introduce test plasmids into various tissues of Arabidopsis.
 Potential applications:  <BR>

<P>
- Cell type specificity of promoter
fragments
<P>
- Strength of promoter fragments
in a defined cell type
<P>
- Subcellular localization of
proteins<BR>

<P>
<B>PROCEDURE<BR>
</B>
<P>
<B>Particle bombardment<BR>
</B>
<P>
1. Prepare DNA:<BR>

<P>
For 5 shots @ 2 microgram plasmid
DNA<BR>

<P>
- Place 50 &#181;l of ethanol-washed
tungsten particles into a 1.5 ml tube and vortex heavily.
<P>
-&gt; Tungsten is prepared by
suspension of commercially available tungsten powder at 50 mg
per ml ethanol, thorough sonication, and washing with ethanol
for four times.  The particles are stored at -20&deg;C in 1 ml aliquots.
<P>
- Spin in microfuge for a few
seconds and wash in 50 &#181;l water.
<P>
- Repeat wash
<P>
- Resuspend in 50 &#181;l water
<P>
- Combine:
<P>
 50 &#181;l tungsten particles in water
<P>
 12 &#181;l of DNA solution (1 mg/ml
of a GUS-expression plasmid in water)
<P>
 50 &#181;l 2.5M CaCl<sub>2</sub>
<P>
 20 &#181;l 0.1M spermidine, free base
<BR>

<P>
- Incubate 20 minutes on ice with
occasional vortexing
<P>
- Add 200 &#181;l ethanol, spin two
minutes, and discard supernatant
<P>
- Wash particles 3 times in ethanol
<P>
- Resuspend particles in 30 &#181;l
ethanol, vortex well
<P>
- Pipette 5 &#181;l of the particles
onto each of five macrocarriers (flying disks (orange)).
<P>
-&gt; Make sure the particles
are well suspended, while dispensing them.  Discard the last 5
&#181;l (too lumpy).
<P>
-&gt; Dry particles on macrocarrier
on filter paper first in air then in a petri dish with drierite.
<BR>

<P>
2. Harvest one plate of seedlings
(ecotype No-O or Columbia), grown on solidified agar (GM medium)
for 7 to 10 days.  Use forceps to gently pull out the seedlings
from the agar and arrange them (crowded) in the center of a petri
dish of solidified MS medium with the roots in the center and
the shoots on the periphery.   <BR>

<P>
3. Bombardment with 1200 psi rupture
discs, following specific instructions for BioRad particle gun.
<BR>

<P>
4. After bombardment, seal the
plates with parafilm and incubate over night under white light
at 22&deg;C.  <BR>

<P>
5. Stain the seedlings in a plastic
dish with 3-5 ml of GUS-stain solution (see Experiment 2).  
<P>
-&gt; Monitor progress of staining
under the stereomicroscope.  To stop the staining, wash the seedlings
in 100 mM phosphate, 1 mM EDTA (wash solution) and fix them in
3.7% formaldehyde in wash solution for 30 minutes, then wash them
again before mounting onto microscope slides.  <BR>
<BR>

<P>
<B>Bombardment procedure<BR>
</B>
<P>
A.  Setup<BR>

<P>
- Open pressurized helium tank,
turning grey knob
<P>
- adjust pressure setting on helium
tank gauge to 200 psi above the value of the rupture disk to be
used.
<P>
- Switch on vacuum pump
<P>
- Switch on particle gun (left
of three red buttons).<BR>

<P>
B.  Shooting<BR>

<P>
1. Insert rupture disk (pressure
disk, e.g. BioRad 165-2329): Unscrew grey disk holder from top
of vacuum chamber, insert disk (can be sterilized in isopropanol
beforehand), and screw grey disk holder back on.  
<P>
-&gt; Make sure disk makes a tight
seal with holder and doesn't move while the holder is screwed
back on.<BR>

<P>
2. Place macrocarrier (orange
flying disk) with DNA side up into the small metal holder.  Make
edge of disk flush with holder by simply pressing the disk down
with the red 'hat' supplied in the accessories box.<BR>

<P>
3. Unscrew lid (ca. 6cm diameter)
from white Teflon plate. Insert stopping screen (BioRad 165-2336,
package of 500) into bottom of the plate, then insert  macrocarrier
holder (with flying disk and DNA), DNA-side down, then screw lid
back on.  Place the whole set into the first shelf of the vacuum
chamber with the tiny screw facing the door.  -&gt; The tiny screw
can be loosened with an allen key in the accessories kit; then
the position of the DNA can be adjusted up and down, if desired.
   <BR>

<P>
4. Place sample tissue into second
shelf from the bottom.  Close chamber door.<BR>

<P>
5. Pull vacuum (upper position
on three-way middle switch of three red buttons).
<P>
-&gt; I use 28 in Hg vacuum (pump
will only deliver 27 in right now).  Hold vacuum by quickly pressing
three-way switch all the way down.  <BR>

<P>
6. press fire switch (right button)
continuously.  Pressure will rise behind pressure disk, disk will
burst, etc.<BR>

<P>
C. Open chamber, repeat procedure
for second sample etc.<BR>

<P>
D. Shut down:
<P>
1. Close Helium tank (grey knob)
.
<P>
2. Pull vacuum (about 15-20in)
and hold.
<P>
3. Press fire switch to bleed
helium line until pressure on gauge is close to zero.  
<P>
4. Open pressure adjustment screw
until handle runs freely 
<P>
5. Bleed residual vacuum from
chamber.
<P>
6. Shut off gun and vacuum pump.
<BR>
<BR>
<BR>

<P>
<B>SOLUTIONS AND REAGENTS<BR>
</B>
<P>
<B>MS medium plates </B>

<P>
 
<P>
4.3 g MS salts (Sigma), 1 mg thiamine,
10 mg myo-inositol, 180 mg potassium dihydrogenphosphate, 30 g
sucrose, adjust pH to 5.7 with KOH, 1.2% bacto-agar; autoclave,
and pour into shallow petri dishes.
<P>
-&gt; optional: add 2.5 mg amphotericinB
and 100 mg carbenicillin per liter.<BR>

<P>
<B>Other solutions have been described
in Experiment 2.</B>
<P>
<B>REFERENCES<BR>
</B>
<P>
Abel, S. and Theologis, A. (1994).
 Transient transformation of Arabidopsis leaf protoplasts: a versatile
experimental system to study gene expression.  Plant Journal 5,
421-427.  <BR>

<P>
Abel, S., Oeller, P., and Theologis,
A. (1994).  Early auxin-induced genes encode short-lived nuclear
proteins.  Proc. Natl. Acad. Sci. (USA) 91, 326-330. <BR>

<P>
Chalfie, M., Tu, Y., Euskirchen,
G., Ward, W.W., and Prasher, D.C. (1994).  Green fluorescent protein
as a marker for gene expression.  Science 263, 802-805.  
<P>
-&gt;  GFP as a reporter for cell
type specific promoter activity<BR>

<P>
Citovsky, V, Warnick, D, and Zambryski,
P. (1994).  Nuclear import of Agrobacterium VirD2 and VirE2 proteins
in maize and tobacco.  Proc. Natl. Acad. Sci. (USA) 91, 3210-3214.
<BR>

<P>
Colasanti, J., Cho, S.-O., Wick,
S., and Sundaresan, V. (1993).  Localization of the functional
p34(cdc2) homolog of maize in root tip and stomatal complex cells:
 association with predicted division sites.  Plant Cell 5, 1101-1111.
<BR>

<P>
Doonan, J., Zhang, H., and Traas,
J.   Indirect immunofluorescence on Arabidopsis seedlings.  Arabidopsis
- The Compleat guide.  <BR>

<P>
Heim, R., Prasher, D.C., and Tsien,
R.Y. (1994).  Wavelength mutations and posttranslational autoxidation
of green fluorescent protein.  Proc. Natl. Acad. Sci. (USA) 91,
12501-12504.  <BR>

<P>
Jefferson, R.A. (1987).  Assaying
chimeric genes in plants: The GUS gene fusion system.  Plant Mol.
Biol. Reporter 5, 387-405.   <BR>

<P>
Kerrebrock, A.W., Moore, D.P.,
Wu, J.S., and Orr-Weaver, T.L. (1995).  Mei-S332, a Drosophila
protein required for sister-chromatid cohesion, can localize to
meiotic centromere regions.  Cell 83, 247-256.  <BR>

<P>
Matsui, M., Stoop, C.D., von Arnim,
A.G., Wei, N., and Deng, X.-W. (1995).  <I>Arabidopsis </I>COP1
specifically interacts <I>in vitro</I> with a novel cytoskeleton
associated protein.  Proc. Natl. Acad. Sci. (USA) 92, 4239-4243.
 <BR>

<P>
Neuhaus, G., Bowler, C., Kern,
R., and Chua, N.-H. (1993).  Calcium/calmodulin-dependent and
-independent phytochrome signal transduction pathways.  Cell 73,
937-952.
<P>
-&gt; This report describes a
cryosectioning protocol, after stringent fixation of the tissue
with formaldehyde, followed by detection with fluorescent antibodies.
 The antigens are abundant plastid proteins.   <BR>

<P>
Olson, K.R., McIntosh, J.R., and
Olmsted, J.B. (1995).  Analysis of MAP4 function in living cells
using green fluorescent protein (GFP) chimeras.  J. Cell Biol.
130, 639-650.<BR>

<P>
Prasher, D.C. (1995).  Using GFP
to see the light.  Trends Genet. 11, 320-323.  
<P>
-&gt; This review is accompanied
by a series of others describing the application of GFP in different
species.  <BR>

<P>
Restrepo, M.A., Freed, D.D., and
Carrington, J.C. (1990).  Nuclear transport of plant potyviral
proteins.  Plant Cell 2, 987-998.  
<P>
-&gt; One of the first papers
to describe the use of GUS-fusions for protein localization.<BR>

<P>
Varagona, M.J., Schmidt, R.J.,
and Raikhel, N.V. (1991).  Monocot regulatory protein Opaque-2
is localized in the nucleus of maize endosperm and transformed
tobacco plants. Plant Cell 3, 105-113.<BR>

<P>
Varagona, M.J., Schmidt, R.J.,
and Raikhel, N.V. (1992).  Nuclear localization signal(s) required
for nuclear targeting of the maize regulatory protein Opaque-2.
 Plant Cell 4, 1213-1227.  <BR>

<P>
von Arnim, A.G. and Deng, X.-W.
(1994).  Light inactivation of <I>Arabidopsis</I> photomorphogenic
repressor COP1 involves a cell-specific regulation of its nucleocytoplasmic
partitioning.  Cell 79, 1035-1045.<BR>

<P>
Wang, S. and Hazelrigg, T. (1994).
 Implications for bcd mRNA localization from spatial distribution
of exu protein in Drosophila oogenesis.  Nature 369, 400-403.
 
<P>
-&gt; GFP as a reporter for subcellular
protein localization.<BR>

<P>
Wick, S.W. (1993).  Methods in
Cell Biology 37, 171-200.  Chapter 10: Immunolabeling of antigens
in plant cells.  
<P>
-&gt; An excellent overview on
techniques for immunolabeling with fluorescent probes in plant
cells.  <BR>
<BR>
<P>
<center>Last modified on January 18, 2002</center>

		</td>
	</tr>
</TABLE>

<!-- FOOTER -->
<script language="javascript" src="/js/footer.js"></script>
</body>
</html>
