<html>
<head>
<title>TAIR</title>
<script language="JavaScript" src="/js/navbar.js"></script>
<link rel="stylesheet" type="text/css" href="/css/main.css" >
<script language='JavaScript'>
var highlight = 0; var helpfile="";
</script>
</head>

<body leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" bgcolor="#F5F9FF">


<!-- HEADER using external JavaScript file -->

<script language='JavaScript' SRC='/js/header'>
</script><p>

<!-- End of header -->


<p>
<table width="602" align="center" border="0" cellspacing="0" cellpadding="2">
	<tr>
		<td width="602">

<P>
<CENTER><B>Lab 8: Plant Defense Responses</B>
<BR>
Cold Spring Harbor Arabidopsis Molecular Genetics Course
<P>
July, 1996
<P>
Jane Glazebrook
<P>
Center for Agricultural Biotechnology
<br>
University of Maryland
<br>
Rm 2114A Ag/Life Sciences Surge Building
<br>
College Park, MD, 20742
<br>
(301)-405-7659 (office)
<br>
(301)-405-1587 (lab)
<br>
(301)-314-9075 (FAX)
<br>
glazebro@umbi.umd.edu
</CENTER>
<p>

<B>1.  Introduction</B>
<P>
Like other organisms, plants are subject to attack by
a wide variety of microorganisms, including viruses, bacteria,
and fungi.  Plants differ from animals in two ways which have
profound effects on the strategies plants use to defend themselves
from pathogen attack.  First, plants do not have an immune system,
so, since there are no cells specialized for defense of the organism,
each cell must have the capacity to respond to a pathogen attack.
 Second, plants can suffer loss of significant fractions of the
plant body without dire consequences for the survival of the organism.
 This allows &quot;scorched earth&quot; strategies, in which infected
cells are killed along with the pathogen, to be effective as plant
defense responses.
<P>
 Plant defense reponses include the hypersensitive response, synthesis
of phytoalexins (small antimicrobial compounds), lignification
of plant cell walls, synthesis of lytic enxymes such as chitinases
and glucanases, an oxidative burst (release of active oxygen species),
and expression of a plethora of pathogenesis related proteins
and transcripts of unknown function (4, 10).  Elucidation of the
regulatory mechanisms controlling expression of these defense
responses, and the extent to which partiuclar responses are effective
in limiting growth of various pathogens is important for the design
of strategies to improve disease resistance in crops.  Development
of disease-resistant lines allows efficient crop production with
reduced reliance on environmentally undesirable toxic agrichemicals.
<P>
 In this laboratory excercise, you will observe two plant defense
responses, the hypersenstive response (HR) and phytoalexin synthesis.
 The HR occurs in plant pathogen interactions in which the plant
host specifically recognizes the pathogen and responds by rapidly
expressing a battery of defense responses.  Recognition requires
the presence of an avirulence gene in the pathogen and a corresponding
resistance gene in the host.  In most cases, a particular plant
resistance gene confers resistance only against pathogens carrying
a particular avirulence gene, and vice versa.  For this reason,
this type of resistance has been called &quot;gene-for-gene&quot;.
 A simple molecular model that explains these phenomena is that
the plant resistance genes encode receptors that bind the products
of the pathogen avirulence genes, and trigger a signal transduction
cascade leading to expression of defense responses (9).  In nearly
all gene-for-gene resistance responses, the defense responses
induced in response to pathogen recognition include the HR, a
rapid programmed cell death at the infection site.  The HR is
thought to limit pathogen growth by depriving the pathogen of
water and nutrients.  The correlation between gene-for-gene resistance
and the HR suggests that the HR is an important component of resistance
to avirulent pathogens.  In the field, plants normally encounter
low titers of pathogens, so only isolated plant cells undergo
the HR and the plant shows few, if any, symptoms resulting from
its interaction with a pathogen.  In the laboratory, plants can
be infected with extremely high pathogen titers, so that the HR
can be detected macroscopically as dry necrotic patches of tissue.
<P>
 In experiment 1, you will infect various wild-type <I>Arabidopsis</I>
plants, and plants with one of the resistance gene mutations <I>rps2</I>
(14) or <I>rpm1</I>  (1)  with a virulent <I>P. syringae  </I>strain
and with isogenic strains carrying one of the avirulence genes
<I>avrRpt2 </I>(5, 13) <I> </I>or <I>avrRpm1 </I>(3).  You will
observe disease symptoms caused by the virulent strain, the HR
and resistance that result from avirulence gene-resistance gene
interactions, and the gene-for-gene specificity of the resistance.
<P>
 Experiments 2 and 3 address phytoalexin synthesis and its antimicrobial
activity.  Phytoalexins are low molecular weight antimicrobial
compounds produced by plants in response to pathogen attack (11)
.  Different plant families produce different classes of phytoalexins.
 For example, legumes produce isoflavonoids, potatoes produce
sesquiterpenoids, and brassicas produce indolics.  Several lines
of evidence suggest that phytoalexins play an important role in
plant defense.  Phytoalexins inhibit growth of bacterial and fungal
phytopathogens in vitro (2).  In many cases, phytoalexins accumulate
more rapidly in response to infection by avirulent pathogens than
in response to virulent ones (2, 10) .  Constitutive expression
of a gene encoding stilbene synthase, a phytoalexin biosynthetic
enzyme from grape, in transgenic tobacco conferred increased resistance
against a tobacco fungal pathogen (8) .
<P>
 Many plants produce multiple, structurally related phytoalexins,
but only one phytoalexin (3-thiazol-2-ylindole, or camalexin)
has been detected in significant quantities in <I>Arabidopsis
</I>(12).  Camalexin inhibits the growth of the fungal pathogen,
<I>Cladosporium cladosporoides</I>, and the bacterial pathogen,
<I>Pseudomonas syringae</I>, in vitro (12).  Camalexin is synthesiszed
in response to <I>P. syringae </I> infection, but there is no
clear correlation between pathogen avirulence and camalexin accumulation
(6).  Mutations in at least five <I>Arabidopsis</I> genes (<I>pad</I>
genes) cause defects in camalexin synthesis in response to <I>P.
syringae </I>infection.  None of the <I>pad</I> mutants are compromised
for resistance to <I>P. syringae </I>strains carrying avirulence
genes, demonstrating that camalexin does not play a crucial role
in conferring gene-for-gene type resistance to these pathogens
(6) .  However, mutations in four of the five <I>pad</I> genes
cause increased susceptibility to virulent <I>P. syringae </I>strains,
suggesting that camalexin may contribute to limiting the growth
of these pathogens (6, 7).  
<P>
 In experiment 2 of the laboratory excercise, you will isolate
camalexin from <I>Arabidopsis</I> leaves using extraction with
organic solvents and thin layer chromatography.  You will observe
that camalexin is synthesized in response to infection.  In experiment
3, you will use a bioassay to show that camalexin inhibits fungal
growth in vitro.
<P>
<B>2.  Experiment 1:</B>  <B>Inoculation of plants with bacterial
pathogens.</B>
<P>
<B> </B>In this experiment, wild-type plants will be infected
with a virulent bacterial pathogen, and with isogenic strains
carrying one of the avirulence genes <I>avrRpt2</I> or <I>avrRpm1</I>.
 Symptoms caused by virulent bacteria, resistance conferred by
recognition of an avirulence gene, and the hypersensitive response
associated with resistance will be observed.  In addition, <I>rps2</I>
mutants (with loss-of-function mutations in the <I>RPS2</I> resistance
gene, which is required for resistance to bacteria carrying <I>avrRpt2</I>),
and <I>rpm1</I> mutants (with loss-of-function mutations in the
<I>RPM1</I>, resistance gene, which is required for resistance
to bacteria carrying <I>avrRpm1</I>) will be infected with the
virulent and avirulent strains.  The gene-for-gene specificity
of the hypersensitive response and resistance will be observed.
<BR>

<P>
<B>PROCEDURE</B>
<P>
1. The night before the experiment, inoculate 5 ml cultures of
<I>Psm</I>ES4326, <I>Psm</I>ES4326(<I>avrRpt2</I>), and <I>Psm</I>ES4326(<I>avrRpm1</I>).
 Use King's B medium, and add tetracycline to 5 &#181;g/ml for the
<I>Psm</I>ES4326(<I>avrRpt2</I>) and <I>Psm</I>ES4326(<I>avrRpm1</I>)
cultures.  Shake at 28&deg; overnight.
<P>
2. Pellet 1 ml of each culture in a microcentrifuge for 1 minute.
 Remove the supernatant, resuspend the cells in 1 ml of 10mM MgSO<sub>4</sub>,
and pellet again.  Resuspend cells in 1 ml of 10mM MgSO<sub>4</sub>.
<P>
3. Dilute the cells 1:10 into 10mM MgSO<sub>4</sub>, and measure OD<sub>600</sub>. 
For each strain, prepare 10 ml of an OD<sub>600</sub>=0.1 suspension, and
10 ml of an OD<sub>600</sub>=0.002 suspension.
<P>
4. Inoculate several mature leaves of each plant genotype (Wild-type
Columbia, <I>rps2</I> mutant, and <I>rpm1</I> mutant) with each
bacterial suspension, as well as sterile 10 mM MgSO<sub>4</sub> solution.
 Inoculate by filling a 1 ml syringe (without a needle) with the
bacterial suspension, pressing it against the abaxial surface
(underside) of the leaf, and gently depressing the plunger.  Label
the infected leaves by attaching tape to toothpicks, writing the
inoculum used on the tape, and inserting the toothpicks into the
soil on the clockwise side of the infected leaves.  
<P>
Expected results:  Hypersensitive responses should be visible
by 16 hours after inoculation.  Leaves infected with OD<sub>600</sub>=0.1
concentrations of virulent strains will show a water-soaked collapse
starting at 24 hours after inoculation.  Virulent strains infected
with OD<sub>600</sub>= 0.002 concentrations will cause water-soaking and
chlorosis visible after 2-3 days.<BR>

<P>
<B>Medium</B>
<P>
<U>King's B</U>
<P>
10 g Difco proteose peptone #3
<P>
1.5 g K<sub>2</sub>HPO<sub>4</sub>, anhydrous
<P>
15 g glycerol
<P>
H2O to 1 liter
<P>
Adjust pH to 7.0 with HCl
<P>
After autoclaving, add sterile MgSO<sub>4</sub> to 5mM<BR>

<P>
<B>3.  Experiment 2:  Detection of camalexin by1 fluorescence.</B>
<P>
 The phytoalexin produced by <I>Arabidopsis</I>, camalexin, is
easily detected because it emits a bright blue fluorescence under
illumination with long-wave ultraviolet light.  In this experiment,
camalexin will be extracted from infected leaf tissue, and separated
from other components of the extract by thin-layer chromatography
(TLC).  Since steps 1-3 require more time than is available for
this laboratory excercise, extracts will be provided, and the
experiment will begin at step 4.
<P>
<B>PROCEDURE</B>
<P>
1. Two days before the experiment, inoculate three wild-type <I>Arabidopsis</I>
leaves with <I>Psm</I>ES4326 at a concentration of OD<sub>600</sub>=0.02.
<P>
2. Excise the infected leaves, and three uninfected leaves.  Place
each leaf into a 1.5 ml eppendorf tube, add 350 &#181;l of 80% methanol
(v/v), and heat at 65&deg; for 20 minutes.  
<P>
3. Remove the leaves, and place the methanol extracts under reduced
pressure until the volume is reduced to approximately 100 &#181;l.
<P>
4. Extract each sample with 75 &#181;l of choroform, and remove the
chloroform phases (the green lower phases) to clean tubes.  Allow
the choroform to evaporate under reduced pressure (in a glass
desiccator, or a vacuum oven with the heat off, attached to house
vacuum).  This should take about one hour.
<P>
5. While the chloroform is evaporating, add 200 ml ethyl acetate
and 30 ml hexane to a thin layer chromatography tank in a fume
hood.  Place the lid on the tank.  It is important to have the
liquid and gas phases in the tank equilibrated.
<P>
6. Dissolve the samples (now a green residue) in 5 &#181;l of choroform.
 Spot all 5 &#181;l onto a TLC plate, approximately 2 cm from the bottom
of the plate, leaving 2 cm between each sample.  Samples may be
labeled using a #2 pencil (gently, so as not to scrape the silica
off the plate).  Place the labels below the samples, so that the
samples do not pass over them as they develop.
<P>
7. Place the plate into the TLC tank, minimizing the time that
the lid is off the tank.  Allow the chromatogram to develop until
the solvent front is 2-4 cm from the top of the plate.  This usually
takes 45-60 minutes, depending on the temperature in the fume
hood.
<P>
8. Remove the plate from the tank, and allow it to dry in the
fume hood for 10 minutes.  Examine the plate in a darkened room
using a hand-held long-wave ultraviolet lamp.  Camalexin should
be visible as a bright blue spot, present only in samples prepared
from infected leaves.  Mark the center of the camalexin spots
lightly with a pencil.
<P>
<B>4.  Experiment 3:</B> <B>Bioassay for compounds with antimicrobial
activity</B>
<P>
 Compounds with anti-microbial activity can be detected on TLC
plates using a bioassay.  In this experiment, TLC plates are sprayed
with a suspension of fungal spores and hypae in a nutrient solution.
 The fungus grows over the surface of the plate, except in areas
where inhibitory compounds are present.  Since the particular
fungus used has dark-colored hypae, inhibitory spots are visible
as white areas in a dark background.  
<P>
<B>PROCEDURE</B>
<P>
1. Pipet 10 ml of 2X potato-dextrose medium onto the surface of
a petri dish that is completely covered by a 2-week-old culture
of <I>Cladosporium cladosporoides </I>grown at room temperature
on 1X potato-dextrose agar.  Gently scrape the surface of the
fungal mat with a pipet to suspend spores and hypae in the solution.
 Filter the suspension through cheesecloth into a chromatography
sprayer.  Repeat with a second 10 ml aliquot of 2X potato-dextrose
medium.
<P>
2.  Place the TLC plate from Experiment 2 in a vertical position.
 Spray the plate with the fungal suspension until the plate looks
uniformly wet.  Do not spray the plate so heavily that liquid
runs down it.
<P>
3. Line the bottom of a 20 x 20 cm petri dish with a moistened
paper towel.  Place the sprayed plate on it, cover with the lid,
and put in a dark place at room temperature.  Zones of inhibition
around the camalexin spots should be visible in 1-3 days.
<P>
<B>Media</B>
<P>
<U>2X potato-dextrose broth</U>
<P>
48 g Difco potato-dextrose broth, dehydrated
<P>
H<sub>2</sub>O to 1 liter, Autoclave
<P>
<U>1X potato-dextrose agar</U>
<P>
24 g Difco potato-dextrose broth, dehydrated
<P>
15 g Agar
<P>
H<sub>2</sub>O to 1 liter
<P>
Autoclave<BR>
<BR>

<P>
<B>References<BR>
</B>
<P>
1.  Bisgrove, S.R., Simonich, M.T., Smith, N.M., Sattler, A. and
Innes, R.W. (1994). A disease resistance gene in <I>Arabidopsis</I>
with specificity for two different pathogen avirulence genes.
Plant Cell<I> 6</I>, 927-933.<BR>

<P>
2.  Darvill, A.G. and Albersheim, P. (1984). Phytoalexins and
their elicitors - a defense against microbial infection in plants.
Ann. Rev. Plant Physiol.<I> 35</I>, 243-275.<BR>

<P>
3.  Debener, T., Lehnackers, H., Arnold, M. and Dangl, J.L. (1991).
Identification and molecular mapping of a single <I>Arabidopsis
thaliana</I> locus determining resistance to a phytopathogenic
<I>Pseudomonas syringae</I> isolate. Plant Journal<I> 1</I>, 289-302.
<BR>

<P>
4.  Dixon, R.A. and Lamb, C.J. (1990). Molecular communication
in interactions between plants and microbial pathogens. Ann. Rev.
Plant Physiol. Plant Mol. Biol.<I> 41</I>, 339-367.<BR>

<P>
5.  Dong, X., Mindrinos, M., Davis, K.R. and Ausubel, F.M. (1991).
Induction of <I>Arabidopsis</I> defense genes by virulent and
avirulent<I> Pseudomonas syringae</I> strains and by a cloned
avirulence gene. Plant Cell<I> 3</I>, 61-72.<BR>

<P>
6.  Glazebrook, J. and Ausubel, F.M. (1994). Isolation of phytoalexin-deficient
mutants of <I>Arabidopsis thaliana</I> and characterization of
their interactions with bacterial pathogens. Proc. Natl. Acad.
Sci. USA<I> 91</I>, 8955-8959.<BR>

<P>
7.  Glazebrook, J., Rogers, E.E. and Ausubel, F.M. (1996). Isolation
of Arabidopsis mutants with enhanced disease susceptibility by
direct screening. Genetics<I> in press</I>, <BR>

<P>
8.  Hain, R., Reif, H.-J., Krause, E., Langebartels, R., Kindl,
H., Vornam, B., Wiese, W., Schmelzer, E., Schreier, P.H., Stocker,
R.H. and Stenzel, K. (1993). Disease resistance results from foreign
phytoalexin expression in a novel plant. Nature<I> 361</I>, 153-156.
<BR>

<P>
9.  Keen, N.T. (1990). Gene-for-gene complementarity in plant-pathogen
interactions. Annu. Rev. Genet.<I> 24</I>, 447-463.<BR>

<P>
10.  Lamb, C.J., Lawton, M.A., Dron, M. and Dixon, R.A. (1989).
Signals and transduction mechanisms for activation of plant defenses
against microbial attack. Cell<I> 56</I>, 215-224.<BR>

<P>
11.  Paxton, J.D. (1981). Phytoalexins- a working redefinition.
Phytopathol. Z<I> 101</I>, 106-109.<BR>

<P>
12.  Tsuji, J., Jackson, E.P., Gage, D.A., Hammerschmidt, R. and
Somerville, S.C. (1992). Phytoalexin accumulation in <I>Arabidopsis
thaliana</I> during the hypersensitive reaction to <I>Pseudomonas
syringae </I>pv.<I>syringae</I>. Plant Physiol.<I> 98</I>, 1304-1309.
<BR>

<P>
13.  Whalen, M.C., Innes, R.W., Bent, A.F. and Staskawicz, B.J.
(1991). Identification of <I>Pseudomonas syringae</I> pathogens
of <I>Arabidopsis</I> and a bacterial locus determining avirulence
on both <I>Arabidopsis</I> and soybean.  The Plant Cell<I> 3</I>,
49-59.<BR>

<P>
14.  Yu, G.-L., Katagiri, F. and Ausubel, F.M. (1993). <I>Arabidopsis</I>
mutations at the <I>RPS2</I> locus result in loss of resistance
to <I>Pseudomonas syringae</I> strains expressing the avirulence
gene <I>avrRpt2</I>. Mol. Plant-Microbe Interact.<I> 6</I>, 434-443.
<BR>
<P>
<center>Last modified on January 18, 2002</center>

		</td>
	</tr>
</TABLE>

<!-- FOOTER -->
<script language="javascript" src="/js/footer.js"></script>
</body>
</html>
