<html>
<head>
<title>TAIR</title>
<script language="JavaScript" src="/js/navbar.js"></script>
<link rel="stylesheet" type="text/css" href="/css/main.css" >
<script language='JavaScript'>
var highlight = 0; var helpfile="";
</script>
</head>

<body leftmargin="0" topmargin="0" marginwidth="0" marginheight="0" bgcolor="#F5F9FF">


<!-- HEADER using external JavaScript file -->

<script language='JavaScript' SRC='/js/header'>
</script><p>

<!-- End of header -->


<p>
<table width="602" align="center" border="0" cellspacing="0" cellpadding="2">
	<tr>
		<td width="602">
		

<center><h3>Lab 9: Analysis of the Regulation of Tryptophan Biosynthetic 
Proteins by Immunoblot and Isolation of Mutants</h3>

<p>
Cold Spring Harbor Arabidopsis Molecular Genetics Course<br>
July, 1996

<p>
Jianmin Zhao and Chad Williams <br>
Boyce Thompson Institute for Plant Research<br>
Tower Road<br>
Ithaca, NY 14853-1801<br>
phone: (607) 254-1326
</center>

<p>
<h2>1. INTRODUCTION</h2>

<p>

Antibodies are valuable tools to study the properties, localization
and abundance of proteins (Harlow and Lane, 1988).  The immunoblot
(western blot) is one of the most frequently used immunotechniques for
the following reasons: - Proteins are separated by electrophoresis
prior to immunodetection, eliminating many problems related to
non-specific binding of the antibody.  - The detection is not limited
to soluble proteins because detergent and reducing agent are present
in the denaturing gel sample buffer.  - Antigens can be quantified
from a very small amount of tissue (as little as 10 mg or less).  -
The same membrane can be stripped and reprobed with several
antibodies.

<p>

Screening for Regulatory Mutants is a very useful approach to study
the regulation of a gene.  The biochemical study of a gene's
regulation often provides clues on the possible ways of isolating
regulatory mutants.  One strategy that has been applied to study gene
regulation utilizes the promoter-reporter transgenic plant as a
starting material.  This strategy's advantage is that the expression
of the reporter gene reflects the regulation of the gene in study, and
that the activity of the reporter gene can be easily evaluated.  One
reporter gene commonly used for this type of screening is &#223;-
glucorunadase (GUS) (Bowling et al., 1994; Cao et al., 1994).  The
detection of GUS activity can be accomplished by histochemical
staining with X-Gluc or by enzyme assays using MUG.  The advantages of
using GUS as the reporter gene in a mutant screen include: 1)
transgenic plants are usually already constructed in order to study
the regulation of genes of interest; and 2) X- Gluc staining and MUG
assay require very little tissue so the screen can be done with a
single small leaf.  One disadvantage of the GUS system is that the
screening is very labor intensive.  An alternative reporter gene is
the hph gene which confers resistance to hygromycin B (Susek et al.,
1993).  Thus, selection on hygromycin B is possible.  However,
transgenic plants that have hph as the reporter gene are uncommon, and
therefore it is almost certain that the transgenic plants will have to
be constructed.

<p>

The tryptophan biosynthetic pathway leads to the biosynthesis of many
secondary metabolites, in addition to tryptophan (Radwanski and Last,
1995).  For example, the major phytoalexin from Arabidopsis,
camalexin, is derived from the tryptophan biosynthetic pathway (Tsuji
et al., 1993; Glazebrook and Ausubel, 1994).  The diverse functions of
the products derived from this pathway probably contribute to the fact
that many conditions such as amino acid starvation and pathogen
infection have been found to induce the tryptophan pathway enzymes at
both the mRNA and protein levels.

<p>

In this lab, we will work with two polyclonal antibodies, anti-TSA and
-TSB (Zhao and Last, 1995), against the Arabidopsis tryptophan
synthase alpha (TSA) and &#223; (TSB) subunits, respectively.  We will use
these antibodies to characterize proteins from Arabidopsis Col-0 and
two Arabidopsis mutants, <i>trp2-8</i> and <i>trp3-1</i>, defective in TSB and TSA
respectively (Last and Fink, 1988; Barczak et al., 1995; Radwanski et
al., 1995).  We will also analyze the induction of TSA and TSB in
plants treated with a pathogen, glyphosate, and lysine + threonine.
In addition, we are going to screen for constitutive and
non-responsive mutants using PAT-GUS transgenic plants (aka AR208).


<h2>2. OVERVIEW AND TIME TABLE</h2>

<dl>
<p>
<dt>Day 1
<dd>(these steps will be carried out by the lab instructor)
Sterilize and plate out 50 seed/plate of mutagenized AR208, 
unmutagenized AR208, and Col-0.  Place these plates in a growth chamber.

<p>

<dt>Day 12
<dd>Treat some of the Col-0 plates with 15 mM glyphosate.  Add 3 ml 
sterilized glyphosate to each petri dish with a pipette.  Gently tilt
and shake each plate horizontally in order to completely cover the
agar top with the glyphosate.  Return plates to growth chamber.

<p>

<dt>Day 13
<dd>1.  Treat some of the AR208 (both mutagenized and 
unmutagenized) and Col-0 plates with 40 mM Lysine + Threonine (K + T).  
Add 3 ml sterilized K + T to each petri dish with a pipette.  Gently tilt and 
shake each plate horizontally in order to completely cover the agar top with 
the K + T.  Return plates to growth chamber.
	
<p>

<dt>Day 15
<dd>
<table>
<tr>
  <td>12:30 to 4:15 pm</td>
  <td>extract proteins from plant tissue; determine concentration</td>
</tr>
<tr>
  <td>4:15 to 5:30 pm</td>
  <td>run the gel</td>
</tr>
<tr>
  <td>5:30 to 6:00 pm</td>
  <td>start transfer</td>
</tr>
</table>

<p>

<dt>Day 16
<dd>
<table>
<tr>
  <td>11:00 to 11:30 am</td>
  <td>stain the membrane with Ponceau S and put the membrane into Blotto.</td>
</tr>
<tr>
  <td>1:30 to 3:30 pm</td>
  <td>incubate blot with primary antibodies.</td>
</tr>
<tr>
  <td>3:30 to 4:30 pm</td>
  <td>incubate blot with secondary antibody.</td>
</tr>
<tr>
  <td>4:30 to 5:30 pm</td>
  <td>develop the blots and analyze the results. Initiate screening of putative mutants (during incubations)</td>
</tr>
</table>

<p>

<dt>Day 17
<dd>screen mutants via GUS staining identify putative mutants and transfer to soil

</dl>

<p>

<h2>3. EXPERIMENTAL PROTOCOL</h2>

<p>

<b>Part A.1:  Screening for Regulatory Mutants of the Tryptophan 
Biosynthetic Pathway</b>

<p>

<b>1.</b>  Sterilize M2 seeds from mutagenized PAT-GUS transgenic plants (AR208) 
and resuspend in 0.1% agar.  Use a pasteur pipette to plate out 50 seeds on a 
PN plate in a sterile hood.  Use a grid (fifty 1cm X 1cm squares) to guide the 
plating.  Grow the plants for 2 weeks under constant light (about 90 &#181;E).  
Always plate out unmutagenized AR208 as a control.  (this step has already 
been done).

<p>

<b>2.</b>  To screen for constitutive mutants, excise 1 cotyledon from each plant 
using sterile tissue culture scissors.  Stain the cotyledons for GUS in 150 &#181;l 
X-Gluc solution in a 96 well plate.  Plants with a higher GUS staining when 
compared to the AR208 control are putative constitutive mutants.  Transfer 
these putative mutants onto soil.

<p>

<b>3.</b>  To screen for nonresponsive mutants, treat the plates with 3 ml 40 mM 
K+T.  Two days after the K+T treatment, excise one cotyledon from each 
plant and stain for GUS.  The GUS transgene in parental AR208 (control) 
should be induced by K+T.  M2 plants that show a lower GUS staining are 
picked as the non-responsive putative mutants.  Transfer these putative 
mutants onto soil.

<p>

<b>**</b> Constitutive mutants and nonresponsive mutants can be screened in a 
sequential order using the same plate.  In this case, screen for constitutives 
first and then do the K+T treatment and the nonresponsive mutant screen.

<p>

<b>4.</b>  Grow the putative mutants and harvest the M3 seeds.

<p>

<b>5.</b>  Rescreen the M3 seed of the putative mutants using steps #1-3.  After 
GUS staining, perform Western blots on any interesting lines to check the 
levels of endogenous protein expression.  In this way, you can eliminate any 
mutations which only affect the expression/activity of GUS.

<p>

<b>Part A.2: Chemical treatment of Arabidopsis plants</b>

<p>

Arabidopsis seeds were surface sterilized (please refer to enhancer-trap lab 
protocol for sterilizing seeds) and fifty seeds were plated out on each PN(S) 
plate.  The plates were grown under 60-120 &#181;E light for 14 days.  Three ml of 
a chemical solution (glyphosate or lysine + threonine) were added to each 
plate and the solution was evenly spread out by vigorously shaking the plates 
horizontally.  Glyphosate and lysine + threonine solutions were used at 15 
mM and 40 mM respectively.  The final concentration will be 10 times lower 
then the above concentration due to the 30 ml agar volume in each plate. The 
treated plants were put back to the growth chamber.  Plant tissue for 
immunoblot analysis can be harvested and frozen in liquid nitrogen and 
stored in a -80 &deg;C freezer.


<p>

<b>Part B: Protein Sample Preparation from Arabidopsis Leaves</b>

<p>

Each student will prepare 6 samples; each group of two students will need 
nine different samples for their protein gel (see Part D, step 7 for list of 
samples).

<p>

<b>1.</b>  Harvest 0.1 g of leaf tissue, add 300 &#181;l of lysis buffer and homogenize 
with a pestle for Eppendorf tubes.  Keep tubes on ice while you finish the 
homogenization steps for the other samples.  However, it is best to proceed to 
the next step upon accumulating a group of 4 or 6 protein samples.

<p>

<b>2.</b>  Spin the tubes for 5 min in a microfuge (12, 000g) at 4 &deg;C.

<p>

<b>3.</b>  Add 80 &#181;l of 2X SDS sample buffer to a new tube for each sample.

<p>

<b>4.</b>  Transfer 80 &#181;l of the supernatent to the tubes with the 2X SDS sample 
buffer (henceforth known as <b>sample A</b>).  Add 10 &#181;l of the supernatent from 
the spun tubes to another tube (<b>sample B</b>).

<p>

<b>5.</b>  Cap each <b>sample A</b> tube with a safety lock and boil for five minutes.  
After boiling, these can sit at room temperature.  These will be used for the 
protein gel.

<p>

<b>6.</b>  Keep <b>sample B</b> tubes on ice.  These will used later to determine the 
protein concentration.

<p>

<b>7.</b>  Continue steps #1-6 until done extracting protein from the plant tissue 
samples.

<p>

<b>Part C: Bradford Protein Concentration Assay</b>

<p>

<b>1.</b>  Dilute BioRad protein assay stock solution with double distilled water (1 
part stock sol.: 9 parts water).

<p>

<b>2.</b>  Add 1 ml of the diluted assay solution to an eppendorf tube.  You will need 
one tube for each sample and two tubes to use as a blank.

<p>

<b>3.</b>  Add 5 &#181;l of <b>sample B</b> to the assay solution.

<p>

<b>4.</b>  Measure A<sub>595</sub> using a disposable cuvette.

<p>

<b>5.</b>  Determine the protein concentration for each sample.  Add 1X SDS sample 
buffer, if necessary, to each <b>sample A</b> tube in order to equilibrate the 
concentrations to one another.

<p>

<b>Part D: SDS-PAGE (Laemmli, 1970)</b>

<p>

<b>1.</b> Assemble the gel apparatus (TA will demonstrate).

<p>

<b>2.</b> Make up the separating gel solution (12% for the lab) according to the 
following table. 

<p>

<table cellpadding=5 border=1>
<th>
  <td>10%</td>
  <td>12%*</td>
  <td>15%</td>
  <td>Stacking gel</td>
</th>
<tr>
  <td>30% Acrylamide</td>
  <td>3.00 ml</td>
  <td>3.45 ml</td>
  <td>4.25 ml</td>
  <td>0.5 ml</td>
</tr>
<tr>
  <td>4X Running Buffer</td>
  <td>2.25 ml</td>
  <td>2.25 ml</td>
  <td>2.25 ml</td>
  <td>*1 ml 4X Stacking</td>
</tr>
<tr>
  <td>ddH<sub>2</sub>O</td>
  <td>3.25 ml</td>
  <td>2.78 ml</td>
  <td>1.90 ml</td>
  <td>2.0 ml</td>
</tr>
<tr>
  <td>10% SDS</td>
  <td>90 &#181;l</td>
  <td>90 &#181;l</td>
  <td>90 &#181;l</td>
  <td>35.0 &#181;l</td>
</tr>
<tr>
  <td>10% APS</td>
  <td>90 &#181;l</td>
  <td>90 &#181;l</td>
  <td>90 &#181;l</td>
  <td>32.5 &#181;l</td>
</tr>
<tr>
  <td>TEMED</td>
  <td>4.5 &#181;l</td>
  <td>4.5 &#181;l</td>
  <td>4.5 &#181;l</td>
  <td>3.5 &#181;l</td>
</tr>
</table>

<p>

<b>3.</b> Pour the gel and gently overlay 0.5-1 ml of ddH<sub>2</sub>O on top of the gel 
solution. 

<p>

<b>4.</b> When the separating gel has polymerized (about 15 min), pour off the 
ddH<sub>2</sub>O on top of the gel and prepare the stacking gel solution. 

<p>

<b>5.</b> Pour the stacking gel (3 ml) and insert the comb. 

<p>

<b>6.</b> When the stacking gel has polymerized, set up the running unit. 

<p>

<b>7.</b> Load 25 &#181;l of each sample and 7 &#181;l of prestained molecular weight 
standards in the following order:

<p>
<table cellpadding=3>
<tr>
  <td>lane 1</td>
  <td>standard</td>
</tr>
<tr>
  <td>lane 2</td>
  <td>Col-0 (control - plate)</td>
</tr>
<tr>
  <td>lane 3</td>
  <td>Col-0 (K+T)</td>
</tr>
<tr>
  <td>lane 4</td>
  <td>Col-0 (glyphosate)</td>
</tr>
<tr>
  <td>lane 5</td>
  <td>mutant 1</td>
</tr>
<tr>
  <td>lane 6</td>
  <td>mutant 2</td>
</tr>
<tr>
  <td>lane 7</td>
  <td>Col-0 (control - soil grown)</td>
</tr>
<tr>
  <td>lane 8</td>
  <td>Col-0 (pathogen treated)</td>
</tr>
<tr>
  <td>lane 9</td>
  <td>AR208 (untreated control)</td>
</tr>
<tr>
  <td>lane 10</td>
  <td>C16-7 (constitutive. mutant)</td>
</tr>
</table>

<p>

Run the gel at 120 V for 1 hour.

<p>

<b>Part E: Protein Transfer</b>

<p>

<b>1.</b> After running the gel, put the gel in transfer buffer for about 20 min with 
gentle shaking. Soak the nitrocellulose membrane in the transfer buffer for 
about 10 min. <br>
-> Use S&S NC 0.2 &#181;m pore; Note:  this is not the same as S&S BAS NC 
used for RNA and DNA gel blots. 

<p>
 
<b>2.</b> Assemble the transfer sandwich in a glass dish containing transfer buffer 
with the order of: black side of plastic sandwich down in tray then wet fiber 
pad, wet Whatman 3MM filter paper, gel, wet membrane, wet filter paper, 
wet fiber pad then clear side of plastic sandwich. All components are pre-wet 
in transfer buffer.

<p>

<b>3.</b> Put the sandwich into the blotting chamber with the membrane on the 
anode (+) side; in other words, place the black side of the sandwich facing the 
black side of the sandwich holder. Put cold transfer buffer and the ice cooler 
into the chamber. 

<p>

<b>4.</b> Blotting conditions: <br>
1 h blot: 100V with prechilled buffer, ice cooler, and constant stirring. <br>
3-5 h blot: 40-50V. <br>
Overnight blot: 30V <br>
-> We are going to do the overnight transfer.

<p>

<h3>Day 2</h3>

<p>

<b>Part F: Immunoblot</b>

<p>

<b>1.</b> Stain the membrane in Ponceau S solution for 10 min and rinse with 
water. <br>
-> Take a picture of the stained membrane.

<p>

<b>2.</b> Block the membrane in Blotto for >1 h at room temperature. <br>
-> 4 h to overnight can be used to reduce the background. 

<p>

<b>3.</b> Incubate the membrane with 1000X diluted anti-TSA and anti-TSB 
(together in Blotto) with gentle shaking for 2 hours at room temperature. 

<p>

<b>4.</b> Wash the membrane with TBS or Blotto for three times with gentle 
shaking for 5-10 minutes each time. 

<p>

<b>5.</b> Put the membrane into fresh Blotto containing the secondary antibody 
(5000X dilution of an alkaline phosphatase conjugated goat anti-rabbit 
antibody) and incubate with gentle shaking for at least 30 minutes. 

<p>

<b>6.</b> Wash with 1X TBS for three times with gentle shaking for 5-10 min. 

<p>

<b>7.</b> Develop color in 20 ml alkaline phosphatase buffer with 50 &#181;l of each 
substrate, p-nitro blue tetrazolium chloride (NBT) and 5-bromo-4-chloro-3-
indolylphosphate p-toluidine salt (BCIP). Let the color develop for 10 min or 
until the band of interest shows up. Stop the reaction by rinsing the 
membrane with water. 

<p>

<b>8.</b> Analyze the results and identify the two mutants (determine whether they 
are <i>trp2-8</i> or <i>trp3-1</i>).

<p>

<h2>4. ADDITIONAL PROTOCOLS</h2>

<br>

<b>Part A: Using Radiolabeled Secondary Reagents</b>

<p>

<b>1.</b> After the first antibody binding and washing step (step 33), transfer the 
nitrocellulose membrane to a square petri-dish. Add 30 ml of fresh Blotto and 
1 mCi of <sup>125</sup>I protein A.<br>
->  <sup>125</sup>I protein A is supplied by New England 
Nuclear (NEN), NEX-146L, 100 mCi.

<p>

<b>2.</b> Incubate the membrane for 1-2 hours at room temperature with gentle 
shaking. 

<p>

<b>3.</b> Remove the <sup>125</sup>I protein A solution into a radioactive liquid waste 
container.  Add 15 ml TBS to the dish and shake for 10 min.  Remove the TBS 
into a radioactive waste container. <br>
-> The <sup>125</sup>I protein A solution can be reused several times. 

<p>

<b>5.</b> Transfer the membrane to a tray containing 250 ml of TBS and wash for 10 
minutes.  Discard the TBS into the sink and flush with water.  

<p>

<b>6.</b> Wash 2-3 times in TBS, 10 minutes each. 

<p>

<b>7.</b> Remove the filter and seal it in a heat seal-bag. 

<p>

<b>8.</b> Expose the membrane to a phosphoimager film at room temperature or to 
an X-ray film at -70&deg;C with an intensifying screen. 

<p>

<b>Part B: Stripping and Reprobing Membranes with Antibodies</b>
<p>

<b>1.</b> Submerge the membrane in stripping buffer and incubate at 50&deg;C for 30 
min.

<p>

<b>2.</b> Wash the membrane in TBS at room temperature twice for 10 minutes 
each. 

<p>

<b>3.</b> Block the membrane in blocking solution (Blotto) for at least 1 hour. 

<p>

<b>4.</b> Perform regular immunoblot. 

<p>

<b>5.</b> SOLUTIONS

<p>


<table cellpadding=5 border=1>
<caption align=top><b>PLANT NUTRIENT AGAR MEDIUM WITH SUCROSE (PNS)</b></caption>
<tr>
  <th>Reagent</td>
  <th>Volume/l</td>
  <th>Stock Solution</td>
</tr>
<tr>
  <td>1 M KNO3</td>
  <td>5.0 ml</td>
  <td>-</td>
</tr>
<tr>
  <td>1 M MgSO<sub>4</sub>.7H<sub>2</sub>O</td>
  <td>2.0 ml</td>
  <td>-</td>
</tr>
<tr>
  <td>1 M Ca(NO<sub>3</sub>)<sub>2</sub>.4H<sub>2</sub>O</td>
  <td>2.0 ml</td>
  <td>-</td>
</tr>
<tr>
  <td>20 mM Fe.EDTA (fresh)</td>
  <td>2.5 ml</td>
  <td>0.1836g/25ml; store 4&deg;C 2 weeks</td>
</tr>
<tr>
  <td>Micronutrients</td>
  <td>1.0 ml</td>
  <td>-</td>
</tr>
<tr>
  <td>Agar</td>
  <td>7.5 grams</td>
  <td>-</td>
</tr>
<tr>
  <td>Sucrose (optional)</td>
  <td>5 grams</td>
  <td>-</td>
</tr>
</table>

<p>

Autoclave one liter in a 2 liter Erlenmeyer flask for (30) minutes, swirl to mix 
agar thoroughly and cool to 50&deg;C.

<p>

After autoclaving, add 2.5 ml of 1 M KPO<sub>4</sub> pH 5.5 <sup>*</sup> per liter. 

<p>

<sup>*</sup> Titrate monobasic potassium phosphate with 1M KOH to achieve pH 5.5 
and then bring to 100 ml final volume.

<p>

<sup>***</sup> Ca(NO<sub>3</sub>)<sub>2</sub>.4H<sub>2</sub>O forms a precipitate with KPO<sub>4</sub> under high 
temperatures. To avoid this, filter sterilize the KPO<sub>4</sub>, and add after 
autoclaving the rest of the solution.

<p>

<table cellpadding=5 border=1>
  <caption align=top>Micronutrient Stock (1000X):</caption>
<tr>
  <th>Final Conc.</th>
  <th>For One Liter</th>
  <th>Molecular Mass</th>
</tr>
<tr>
  <td>70 mM H<sub>3</sub>BO<sub>4</sub></td>
  <td>4.3 g</td>
  <td>62 g/mol</td>
</tr>
<tr>
  <td>14 mM MnSO<sub>4</sub>.H<sub>2</sub>O</td>
  <td>2.4g</td>
  <td>169 g/mol</td>
</tr>
<tr>
  <td>0.5 mM CuSO<sub>4</sub>.5H<sub>2</sub>O</td>
  <td>0.125g</td>
  <td>250 g/mol</td>
</tr>
<tr>
  <td>1.0 mM ZnSO<sub>4</sub>.6H<sub>2</sub>O</td>
  <td>0.288g</td>
  <td>288 g/mol</td>
</tr>
<tr>
  <td>0.2 mM Na<sub>2</sub>MoO<sub>4</sub></td>
  <td>0.041g</td>
  <td>206 g/mol</td>
</tr>
<tr>
  <td>10 mM NaCl</td>
  <td>0.58g</td>
  <td>58 g/mol</td>
</tr>
<tr>
  <td>0.01 mM CoCl<sub>2</sub>.6H<sub>2</sub>O</td>
  <td>2.4 mg</td>
  <td>238g/mol</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>Glyphosate:</th>
  <td>15 mM, pH 7.0<td>
</tr>
</table>

<table cellpadding=2>
<tr>
  <th>K+T:</th>
  <td>40 mM, pH 7.0</td>
</tr>
</table>

<table cellpadding=2>
<tr>
  <th>Lysis Buffer:</th>
  <td>2 mM</td>
  <td>Tris.HCl pH8.0</td>
</tr>
<tr>
  <td rowspan=2>
  <td>0.14 M</td>
  <td>NaCl</td>
</tr>
<tr>
  <td>1 mM</td>
  <td>phenylmethylsulfonyl flouride (PMSF)</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>30% acrylamide (0.8% Bis):</th>
  <td>Dissolve 29.2 g acrylamide and 0.8 g bis-acrylamide in 100 ml water, store at 4&deg;C in the dark.</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>4X Separating Buffer:</th>
  <td>Dissolve 18.7 g Tris base in 100 ml water and adjust 
pH to 8.8 with hydrochloric acid.</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>4X Stacking Buffer:</th>
  <td>Dissolve 6.06g Tris base in 100ml water and adjust pH 
to 6.8 with hydrochloric acid.</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>Running buffer (5X):</th>
  <td>15 g</td>
  <td>Tris base</td>
</tr>
<tr>
  <td rowspan=3>
  <td>72 g</td>
  <td>glycine</td>
</tr>
<tr>
  <td>1 l</td>
  <td>water</td>
</tr>
<tr>
  <td></td>
  <td>add SDS to 0.1% final concentration.</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>SDS-PAGE Sample</th>
  <td>1.00 ml</td>
  <td>0.5M Tris/HCl (pH 6.8)</td>
</tr>
<tr>
  <th>Buffer (2X):</th>
  <td>0.80 ml</td>
  <td>glycerol</td>
</tr>
<tr>
  <td rowspan=4></td>
  <td>1.60 ml</td>
  <td>10% SDS</td>
</tr>
<tr>
  <td>0.20 ml</td>
  <td>0.05% bromophenol blue</td>
</tr>
<tr>
  <td>0.32 ml</td>
  <td>water</td>
</tr>
<tr>
  <td>0.08 ml</td>
  <td>&#223;-mercaptoethanol (add fresh each time)</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>Transfer Buffer:</th>
  <td></td>
  <td>for 1 liter</td>
</tr>
<tr>
  <td rowspan=3></td>
  <td>3.03 g</td>
  <td>Tris Base</td>
</tr>
<tr>
  <td>14.4 g</td>
  <td>glycine</td>
</tr>
<tr>
  <td>200 ml</td>
  <td>methanol</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>Blotto:</th>
  <td>Dissolve 25 g non-fat dry milk in 500 ml 1X TBS (optional: 0.02% NaN<sub>3</sub>)</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>10X TBS:</th>
  <td>0.2 M Tris HCl pH 7.4, 1.5 M NaCl</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>Alkaline Phosphatase Reaction Buffer:</th>
  <td>0.1 M Tris/HCl, 0.1 M NaCl, 50 mM MgCl<sub>2</sub>, pH 9.5.</td>
</tr>
</table>

<p>

<table cellpadding=2>
<tr>
  <th>Stripping Buffer:</th>
  <td></td>
  <td>for 100 ml</td>
</tr>
<tr>
  <td rowspan=3>
  <td>100  mM</td>
  <td>&#223;-mercaptoethanol</td>
  <td>0.64 ml</td>
</tr>
<tr>
  <td>62.5 mM</td>
  <td>Tris</td>
  <td>6.25 ml of 1M Tris, pH 6.7</td>
</tr>
<tr>
  <td>2%</td>
  <td>SDS</td>
  <td>20.0 ml of 10% SDS</td>
</tr>
<tr>
  <td></td>
  <td></td></td>
  <td>73.0 ml water</td>
</tr>
</table>

<p>

<h2>6. REFERENCES</h2>

<p>

<b>Ausubel, F. M., Brent, R., Kingston, R. E., Moore, D. D., Seidman, J. G., 
Smith, J. A., and Struhl, K. (eds) </b>(1993). Current Protocols in Molecular 
Biology, Wiley Interscience, New York.

<p>

<b>Barczak, A. J., Zhao, J., Pruitt, K. D., and Last, R. L. </b>(1995). 5-
Fluoroindole Resistance Identifies Tryptophan Synthase Beta Subunit 
Mutants in Arabidopsis thaliana.Genetics <b>140</b>, 303-313.

<p>

<b>Bowling, S. A., Guo, A., Cao, H., Gordon, S., Klessig, D. F., and Dong, 
X. </b>(1994). A mutation in Arabidopsis that leads to constitutive expression of 
systemic acquired resistance. Plant Cell <b>6</b>, 1845-1857.

<p>

<b>Cao, H., Bowling, S. A., Gordon, S., and Dong, X. </b>(1994). 
Characterization of an Arabidopsis mutant that is nonresponsive to inducers 
of systemic acquired resistance. Plant Cell <b>6</b>, 1583-1592.

<p>

<b>Glazebrook, J., and Ausubel, F. M. </b>(1994). Isolation of phytoalexin-
deficient mutants of Arabidopsis thaliana and characterization of their 
interactions with bacterial pathogens. Proc. Natl. Acad. of Sci. USA <b>91</b>, 8955-8959.

<p>

<b>Harlow, E., and Lane, D. </b>(1988). Antibodies: A Laboratory Manual. (Cold 
Spring Harbor, New York: Cold Spring Harbor Laboratory).

<p>

<b>Laemmli, U. K. </b>(1970). Cleavage of structural proteins during the assembly 
of the head of bacteriophage T4. Nature <b>227</b>, 680-685.

<p>

<b>Last, R. L., and Fink, G. R. </b>(1988). Tryptophan-requiring mutants of the 
plant Arabidopsis thaliana. Science <b>240</b>, 305-310.

<p>

<b>Radwanski, E. R., and Last, R. L. </b>(1995). Tryptophan biosynthesis and 
metabolism: Biochemical and molecular genetics. Plant Cell <b>7</b>, 921-934.

<p>

<b>Radwanski, E. R., Zhao, J., and Last, R. L. </b>(1995). Arabidopsis thaliana 
tryptophan synthase alpha: Gene cloning and expression analysis. Molec. 
Gen. Genet. <b>248</b>, 657-667.

<p>

<b>Susek, R. E., Ausubel, F. M., and Chory, J. </b>(1993). Signal transduction 
mutants of arabidopsis uncouple nuclear cab and rbcs gene expression from 
chloroplast development. Cell <b>74</b>, 787-799.

<p>

<b>Tsuji, J., Zook, M., Somerville, S. C., Last, R. L., and Hammerschmidt, 
R. </b>(1993). Evidence that tryptophan is not a direct biosynthetic intermediate 
of camalexin in Arabidopsis thaliana. Physiol. Molec. Plant Path. <b>43</b>, 221-229.

<p>

<b>Zhao, J., and Last, R. L. </b>(1995). Immunological characterization and 
chloroplast import of the tryptophan biosynthetic enzymes of the flowering 
plant Arabidopsis thaliana. J. Biol. Chem. <b>270</b>, 6081-6087.
<P>
<center>Last modified on January 18, 2002</center>

		</td>
	</tr>
</TABLE>

<!-- FOOTER -->
<script language="javascript" src="/js/footer.js"></script>
</body>
</html>
