  <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
<script language="javascript">
<!--

function Page_load(page) {
	

		
	   if (page == 4)
		page_title = "genome_V.html";
	   else if (page == 5)
		page_title = "Contents.html";
	   else if (page == 6)
		page_title = "prakash.html";

	if (page_title == "")
		return;
	
	
	window.parent.frame_main.location = window.parent.frame_menu.home_page + page_title;
	   
}

	var strng;
	function message(id) {
	   

	   if (id == 4)
		strng = 'Plant and Animal Genome V - Arabidopsis Workshop';
	   else if (id == 5)
		strng = 'WeedsWorld Volume 4(i) Contents';
	   else if (id == 6)
		strng = 'Towards the development of cold tolerant plants';

	   else 
		return false;

	window.status = strng;
	return true;
}

// end of script -->
</script>

  <head>
    <title>Mapping of icr genes reveals that tt6 should be placed on Chromosome 5</title>
  </head>

  <body bgcolor="#ffffff">
    <blockquote>

      <h1>Mapping of icr genes reveals that tt6 should be placed on Chromosome 5</h1>
    
James J. Campanella(1), Weiqin Dong and Christopher D. Town<br> Biology
Department, Case Western Reserve University, Cleveland, OH 44106.<br>(1)
Current address: Department of Biological Sciences, Lehigh University,
Bethlehem, PA 18015.
<hr>
      <p>We recently described the identification of two genes, icr1
      and icr2, mutations which confer increased resistance to growth
      inhibition by amino acid conjugates of indole-3-acetic acid
      (IAA) <a href="#campanella">(Campanella <i>et al</i>.,
      1996)</a>. In a cross between icr1 and MSU25 (er, gl1, hy2,
      tt6), we found that icr1 was linked to tt6, but not to hy2 or
      gl1.  We then used RFLP, CAPS <a href="#konieczny">(Konieczny
      and Ausubel, 1993)</a> and SSLP <a href="#bell">(Bell and Ecker,
      1994)</a> markers to more precisely locate icr1. Surpringly, we
      found that neither icr1 nor tt6 were linked to either pCIT1210
      or nga6, both located close to the stated position of tt6 on the
      unified map (Table 1).


</p>

<center>
      <table align=center>
	<tr><td colspan=2 bgcolor="#cccccc"><font color="#790000"><b>Table 1. Linkage between icr1, tt6 and markers on Chromosome 3</b></font></td></tr>
	<tr><th align=left>Loci Examined</th><th align=left> r &plusmn; S.E.1</th></tr>
	<tr><th align=left>tt6/icr1</th><td>        0.12 &plusmn; 0.07</td></tr>
	<tr><th align=left>pCIT1210*/gl1</th><td>    0.12 &plusmn; 0.08</td></tr>
	<tr><th align=left>gl1/nga6</th><td>         0.19 &plusmn; 0.01</td></tr>
	<tr><th align=left>pCIT1210/nga6</th><td>    0.02 &plusmn; 0.02</td></tr>
	<tr><th align=left>icr1/nga6 </th><td>       0.33 &plusmn; 0.10</td></tr>
	<tr><th align=left>icr1/pCIT1210</th><td>    0.35 &plusmn; 0.10</td></tr>
	<tr><th align=left>tt6/pCIT1210</th><td>     0.50 &plusmn; 0.13</td></tr>
	<tr><th align=left>tt6/nga6</th><td>         0.46 &plusmn; 0.19</td></tr>
	
      </table> 
</center>

      <p>(*Using XbaI, pCIT1210 detects a polymorphism between Ws-2
      and Landsberg erecta. Using a set of SSLP markers covering the remaining
      four chromosomes, both icr1 and tt6 were found to be linked to
      nga 151, which is located near the top of chromosome 5. (Table
      2). Correspondence with Dr. Maarten Koornneef revealed that the
      original placement of tt6 on chromsome 3 was based on loose
      linkage to gl1 (38cM), and is thus not inconsistent with our
      current observations. We therefore believe that tt6 and icr1
      should be placed above nga 151 on chromsome 5.</p>


<center><table align="center">
<tr><td colspan=2 bgcolor="#cccccc"><font color="#790000"><b>Table 2 Linkage between icr1, tt6 and markers on Chromosome 5</b></font></td></tr>
<tr><th align="left">Loci Examined</th><th align="left">   r &plusmn; S.E.</th></tr>
<tr><th align="left">nga225/icr1</th><td>     0.31 &plusmn; 0.05</td></tr>
<tr><th align="left">nga225/tt6</th><td>      0.24 &plusmn; 0.05</td></tr>
<tr><th align="left">nga158/icr1</th><td>     0.30 &plusmn; 0.05</td></tr>
<tr><th align="left">nga158/tt6</th><td>      0.23 &plusmn; 0.05</td></tr>
<tr><th align="left">nga158/nga151</th><td>   0.19 &plusmn; 0.05</td></tr>
<tr><th align="left">icr1/tt6</th><td>        0.18 &plusmn; 0.05</td></tr>
<tr><th align="left">icr1/nga151</th><td>     0.42 &plusmn; 0.08</td></tr>
<tr><th align="left">tt6/nga151</th><td>      0.44 &plusmn; 0.10</td></tr>
<tr><th align="left">nga225/nga158</th><td>   0.09 &plusmn; 0.02</td></tr>
<tr><th align="left">nga225/nga151</th><td>   0.19 &plusmn; 0.05</td></tr>
	
</table>
</center><p>
Similar mapping of icr2 places this marker towards the bottom of chromosome 
5 (Table 3).<p>


	<center>
      <table>
	<tr><td colspan="2" bgcolor="#cccccc" ><font color="#79000"><b>Table 3 Linkage between icr2 and markers on Chromosome 5</b></font></tr></td>
	<tr><th align="left">Loci Examined</th><th align="left">   r &plusmn; S.E.</th</tr>
	<tr><th align="left">icr2/DFR </th><td>       39 &plusmn; 13</td></tr>
	<tr><th align="left">icr2/LFY3</th><td>       29 &plusmn; 8</td></tr>
	<tr><th align="left">icr2/ILL1/2 </th><td>    14 &plusmn; 6</td></tr>
	<tr><th align="left">icr2/g2368</th><td>      15.7 &plusmn; 6.5</td></tr>
	<tr><th align="left">icr2/CATHANKA</th><td>   26 &plusmn; 8</td></tr>
	<tr><th align="left">cathanka-g2368</th><td>  12&plusmn;4</td></tr>
	<tr><th align="left">cathanka-ill1/2</th><td> 25 &plusmn; 6</td></tr>
	<tr><th align="left">g2368-ill1/2 </th><td>   12 &plusmn; 4</td></tr>
	
      </table>
    </center>

      <p>We would like to point out that these map locations are not
      precise, since due to the incomplete dominance of both icr1 and
      icr2, it is sometimes difficult to distinguish heterozygotes
      from homozyotes. We report here the re-location of tt6 since it
      may have bearing on the research of other groups <a href="#pelletier">(e.g. Pelletier
      <i>et al</i>., 1997)</a>. We have included our current data on icr1
      and icr2 since we are not pursuing this at present and the
      information may be of interest to others.
</p>
      <h3>Literature Cited</h3>

<a name="bell">Bell, C.J. and Ecker, J.R. (1994) Assignment of 30 Microsatellite Loci to 
the Linkage Map of <i>Arabidopsis</i> . Genomics <b>19</b>137-144.<p>
<a name="campanella">Campanella, J.J., Ludwig-Mueller, J and Town, C.D. (1996). Isolation and 
Characterization of Mutants of <i>Arabidopsis thaliana</i> with Increased 
Resistance to Growth Inhibition by IAA-amino acid Conjugates. Plant Physiol. 
<b>112</b> 735-745.<p>
<a name="konieczny">Konieczny A. and Ausubel, F.M.  (1993). A Procedure for Mapping <i>Arabidopsis</i> 
Mutations Using Co-dominant Ecotype-specific PCR-based Markers. Plant J. <b>4</b>
403-410.<p>
<a name="pelletier">Pelletier, M.K., Murrell, J.R., Shirley, B.W. (1997). Characterization of 
flavonol synthase and lecuoanthocyanidin dioxygenase genes in <i>Arabidopsis</i>. 
Plant Physiol. (in press).


      


      <hr>
	  </blockquote>
  </body> 
</html>

